<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Yuming Jiang" />
  <meta name="author" content="Dina Schuster" />
  <meta name="author" content="Benjamin A. Neely" />
  <meta name="author" content="Germán L. Rosano" />
  <meta name="author" content="Norbert Volkmar" />
  <meta name="author" content="Devasahayam Arokia Balaya Rex" />
  <meta name="author" content="Susan B. Egbert" />
  <meta name="author" content="Trenton M. Peters-Clarke" />
  <meta name="author" content="Simion Kreimer" />
  <meta name="author" content="Emma H. Doud" />
  <meta name="author" content="Oliver M. Crook" />
  <meta name="author" content="Amit Kumar Yadav" />
  <meta name="author" content="Muralidharan Vanuopadath" />
  <meta name="author" content="Martín L. Mayta" />
  <meta name="author" content="Anna G. Duboff" />
  <meta name="author" content="Nicholas M. Riley" />
  <meta name="author" content="Jesse G. Meyer" />
  <meta name="dcterms.date" content="2023-10-02" />
  <meta name="keywords" content="markdown, publishing, manubot" />
  <title>A Comprehensive Overview of Modern Proteomics</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta property="og:type" content="article" />
  <meta name="dc.title" content="A Comprehensive Overview of Modern Proteomics" />
  <meta name="citation_title" content="A Comprehensive Overview of Modern Proteomics" />
  <meta property="og:title" content="A Comprehensive Overview of Modern Proteomics" />
  <meta property="twitter:title" content="A Comprehensive Overview of Modern Proteomics" />
  <meta name="dc.date" content="2023-10-02" />
  <meta name="citation_publication_date" content="2023-10-02" />
  <meta property="article:published_time" content="2023-10-02" />
  <meta name="dc.modified" content="2023-10-02T04:36:20+00:00" />
  <meta property="article:modified_time" content="2023-10-02T04:36:20+00:00" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Yuming Jiang" />
  <meta name="citation_author_institution" content="Department of Computational Biomedicine, Cedars Sinai Medical Center" />
  <meta name="citation_author_orcid" content="0000-0001-7444-3849" />
  <meta name="twitter:creator" content="@yumingjiang94" />
  <meta name="citation_author" content="Dina Schuster" />
  <meta name="citation_author_institution" content="Department of Biology, Institute of Molecular Systems Biology, ETH Zurich, Zurich 8093, Switzerland" />
  <meta name="citation_author_institution" content="Department of Biology, Institute of Molecular Biology and Biophysics, ETH Zurich, Zurich 8093, Switzerland" />
  <meta name="citation_author_institution" content="Laboratory of Biomolecular Research, Division of Biology and Chemistry, Paul Scherrer Institute, Villigen 5232, Switzerland" />
  <meta name="citation_author_orcid" content="0000-0001-6611-8237" />
  <meta name="twitter:creator" content="@dina_sch" />
  <meta name="citation_author" content="Benjamin A. Neely" />
  <meta name="citation_author_institution" content="Chemical Sciences Division, National Institute of Standards and Technology, NIST Charleston" />
  <meta name="citation_author_orcid" content="0000-0001-6120-7695" />
  <meta name="twitter:creator" content="@neely615" />
  <meta name="citation_author" content="Germán L. Rosano" />
  <meta name="citation_author_institution" content="Mass Spectrometry Unit, Institute of Molecular and Cellular Biology of Rosario, Rosario, Argentina" />
  <meta name="citation_author_orcid" content="0000-0002-8313-6813" />
  <meta name="twitter:creator" content="@GermanRosano" />
  <meta name="citation_author" content="Norbert Volkmar" />
  <meta name="citation_author_institution" content="Department of Biology, Institute of Molecular Systems Biology, ETH Zurich, Zurich 8093, Switzerland" />
  <meta name="citation_author_institution" content="DISCO Pharmaceuticals, Zurich 8093, Switzerland" />
  <meta name="citation_author_orcid" content="0000-0003-0766-5606" />
  <meta name="twitter:creator" content="@NorbertVolkmar" />
  <meta name="citation_author" content="Devasahayam Arokia Balaya Rex" />
  <meta name="citation_author_institution" content="Center for Systems Biology and Molecular Medicine, Yenepoya Research Centre, Yenepoya (Deemed to be University), Mangalore 575018, India" />
  <meta name="citation_author_orcid" content="0000-0002-9556-3150" />
  <meta name="twitter:creator" content="@rexprem" />
  <meta name="citation_author" content="Susan B. Egbert" />
  <meta name="citation_author_institution" content="Department of Chemistry, University of Manitoba, Winnipeg, Cananda" />
  <meta name="citation_author_orcid" content="0000-0001-5458-1099" />
  <meta name="twitter:creator" content="@lichenlady94" />
  <meta name="citation_author" content="Trenton M. Peters-Clarke" />
  <meta name="citation_author_institution" content="Department of Pharmaceutical Chemistry, University of California-San Francisco" />
  <meta name="citation_author_orcid" content="0000-0002-9153-2525" />
  <meta name="twitter:creator" content="@trentmpc" />
  <meta name="citation_author" content="Simion Kreimer" />
  <meta name="citation_author_institution" content="Smidt Heart Institute, Cedars Sinai Medical Center" />
  <meta name="citation_author_institution" content="Advanced Clinical Biosystems Research Institute, Cedars Sinai Medical Center" />
  <meta name="citation_author_orcid" content="0000-0001-6627-3771" />
  <meta name="twitter:creator" content="@KreimerSimion" />
  <meta name="citation_author" content="Emma H. Doud" />
  <meta name="citation_author_institution" content="Center for Proteome Analysis, Indiana University School of Medicine, Indianapolis, Indiana, USA" />
  <meta name="citation_author_orcid" content="0000-0003-0049-0073" />
  <meta name="twitter:creator" content="@fireinlab" />
  <meta name="citation_author" content="Oliver M. Crook" />
  <meta name="citation_author_institution" content="Oxford Protein Informatics Group, Department of Statistics, University of Oxford, Oxford OX1 3LB, United Kingdom" />
  <meta name="citation_author_orcid" content="0000-0001-5669-8506" />
  <meta name="twitter:creator" content="@OllyMCrook" />
  <meta name="citation_author" content="Amit Kumar Yadav" />
  <meta name="citation_author_institution" content="Translational Health Science and Technology Institute" />
  <meta name="citation_author_orcid" content="0000-0002-9445-8156" />
  <meta name="twitter:creator" content="@theoneamit" />
  <meta name="citation_author" content="Muralidharan Vanuopadath" />
  <meta name="citation_author_institution" content="School of Biotechnology, Amrita Vishwa Vidyapeetham, Kollam-690 525, Kerala, India" />
  <meta name="citation_author_orcid" content="0000-0002-9364-917X" />
  <meta name="twitter:creator" content="@V_MuraleeDhar" />
  <meta name="citation_author" content="Martín L. Mayta" />
  <meta name="citation_author_institution" content="School of Medicine and Health Sciences, Center for Health Sciences Research, Universidad Adventista del Plata, Libertador San Martín 3103, Argentina" />
  <meta name="citation_author_institution" content="Molecular Biology Department, School of Pharmacy and Biochemistry, Universidad Nacional de Rosario, Rosario 2000, Argentina" />
  <meta name="citation_author_orcid" content="0000-0002-7986-4551" />
  <meta name="twitter:creator" content="@MartinMayta2" />
  <meta name="citation_author" content="Anna G. Duboff" />
  <meta name="citation_author_institution" content="Department of Chemistry, University of Washington" />
  <meta name="citation_author_orcid" content="0009-0002-7316-3831" />
  <meta name="citation_author" content="Nicholas M. Riley" />
  <meta name="citation_author_institution" content="Department of Chemistry, University of Washington" />
  <meta name="citation_author_orcid" content="0000-0002-1536-2966" />
  <meta name="twitter:creator" content="@riley_nm1" />
  <meta name="citation_author" content="Jesse G. Meyer" />
  <meta name="citation_author_institution" content="Department of Computational Biomedicine, Cedars Sinai Medical Center" />
  <meta name="citation_author_orcid" content="0000-0003-2753-3926" />
  <meta name="twitter:creator" content="@j_my_sci" />
  <link rel="canonical" href="https://jessegmeyerlab.github.io/proteomics-tutorial/" />
  <meta property="og:url" content="https://jessegmeyerlab.github.io/proteomics-tutorial/" />
  <meta property="twitter:url" content="https://jessegmeyerlab.github.io/proteomics-tutorial/" />
  <meta name="citation_fulltext_html_url" content="https://jessegmeyerlab.github.io/proteomics-tutorial/" />
  <meta name="citation_pdf_url" content="https://jessegmeyerlab.github.io/proteomics-tutorial/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://jessegmeyerlab.github.io/proteomics-tutorial/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://jessegmeyerlab.github.io/proteomics-tutorial/v/074652281b943b17a0946804791cd184f69537ee/" />
  <meta name="manubot_html_url_versioned" content="https://jessegmeyerlab.github.io/proteomics-tutorial/v/074652281b943b17a0946804791cd184f69537ee/" />
  <meta name="manubot_pdf_url_versioned" content="https://jessegmeyerlab.github.io/proteomics-tutorial/v/074652281b943b17a0946804791cd184f69537ee/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">A Comprehensive Overview of Modern Proteomics</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://jessegmeyerlab.github.io/proteomics-tutorial/v/074652281b943b17a0946804791cd184f69537ee/">permalink</a>)
was automatically generated
from <a href="https://github.com/jessegmeyerlab/proteomics-tutorial/tree/074652281b943b17a0946804791cd184f69537ee">jessegmeyerlab/proteomics-tutorial@0746522</a>
on October 2, 2023.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Yuming Jiang</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7444-3849">0000-0001-7444-3849</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/jymbcrc">jymbcrc</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/yumingjiang94">yumingjiang94</a>
<br>
<small>
Department of Computational Biomedicine, Cedars Sinai Medical Center
</small></p></li>
<li><p><strong>Dina Schuster</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6611-8237">0000-0001-6611-8237</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/dschust-r">dschust-r</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/dina_sch">dina_sch</a>
<br>
<small>
Department of Biology, Institute of Molecular Systems Biology, ETH Zurich, Zurich 8093, Switzerland; Department of Biology, Institute of Molecular Biology and Biophysics, ETH Zurich, Zurich 8093, Switzerland; Laboratory of Biomolecular Research, Division of Biology and Chemistry, Paul Scherrer Institute, Villigen 5232, Switzerland
</small></p></li>
<li><p><strong>Benjamin A. Neely</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6120-7695">0000-0001-6120-7695</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/neely">neely</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/neely615">neely615</a>
<br>
<small>
Chemical Sciences Division, National Institute of Standards and Technology, NIST Charleston
· Funded by NIST
</small></p></li>
<li><p><strong>Germán L. Rosano</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-8313-6813">0000-0002-8313-6813</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/ger225">ger225</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/GermanRosano">GermanRosano</a>
<br>
<small>
Mass Spectrometry Unit, Institute of Molecular and Cellular Biology of Rosario, Rosario, Argentina
· Funded by Grant PICT 2019-02971 (Agencia I+D+i)
</small></p></li>
<li><p><strong>Norbert Volkmar</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0766-5606">0000-0003-0766-5606</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/norbertvolkmar">norbertvolkmar</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/NorbertVolkmar">NorbertVolkmar</a>
<br>
<small>
Department of Biology, Institute of Molecular Systems Biology, ETH Zurich, Zurich 8093, Switzerland; DISCO Pharmaceuticals, Zurich 8093, Switzerland
</small></p></li>
<li><p><strong>Devasahayam Arokia Balaya Rex</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9556-3150">0000-0002-9556-3150</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/ArokiaRex">ArokiaRex</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/rexprem">rexprem</a>
<br>
<small>
Center for Systems Biology and Molecular Medicine, Yenepoya Research Centre, Yenepoya (Deemed to be University), Mangalore 575018, India
</small></p></li>
<li><p><strong>Susan B. Egbert</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-5458-1099">0000-0001-5458-1099</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/lichenlady94">lichenlady94</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/lichenlady94">lichenlady94</a>
<br>
<small>
Department of Chemistry, University of Manitoba, Winnipeg, Cananda
</small></p></li>
<li><p><strong>Trenton M. Peters-Clarke</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9153-2525">0000-0002-9153-2525</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/trentmpc">trentmpc</a>
<br>
<small>
Department of Pharmaceutical Chemistry, University of California-San Francisco
</small></p></li>
<li><p><strong>Simion Kreimer</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6627-3771">0000-0001-6627-3771</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/KreimerSimion">KreimerSimion</a>
<br>
<small>
Smidt Heart Institute, Cedars Sinai Medical Center; Advanced Clinical Biosystems Research Institute, Cedars Sinai Medical Center
</small></p></li>
<li><p><strong>Emma H. Doud</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0049-0073">0000-0003-0049-0073</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/edoud1">edoud1</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/fireinlab">fireinlab</a>
<br>
<small>
Center for Proteome Analysis, Indiana University School of Medicine, Indianapolis, Indiana, USA
</small></p></li>
<li><p><strong>Oliver M. Crook</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-5669-8506">0000-0001-5669-8506</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/ococrook">ococrook</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/OllyMCrook">OllyMCrook</a>
<br>
<small>
Oxford Protein Informatics Group, Department of Statistics, University of Oxford, Oxford OX1 3LB, United Kingdom
</small></p></li>
<li><p><strong>Amit Kumar Yadav</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9445-8156">0000-0002-9445-8156</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/aky">aky</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/theoneamit">theoneamit</a>
<br>
<small>
Translational Health Science and Technology Institute
· Funded by Grant BT/PR16456/BID/7/624/2016 (Department of Biotechnology, India); Grant Translational Research Program (TRP) at THSTI funded by DBT
</small></p></li>
<li><p><strong>Muralidharan Vanuopadath</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9364-917X">0000-0002-9364-917X</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/vanuopadathmurali">vanuopadathmurali</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/V_MuraleeDhar">V_MuraleeDhar</a>
<br>
<small>
School of Biotechnology, Amrita Vishwa Vidyapeetham, Kollam-690 525, Kerala, India
</small></p></li>
<li><p><strong>Martín L. Mayta</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-7986-4551">0000-0002-7986-4551</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/martinmayta">martinmayta</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/MartinMayta2">MartinMayta2</a>
<br>
<small>
School of Medicine and Health Sciences, Center for Health Sciences Research, Universidad Adventista del Plata, Libertador San Martín 3103, Argentina; Molecular Biology Department, School of Pharmacy and Biochemistry, Universidad Nacional de Rosario, Rosario 2000, Argentina
</small></p></li>
<li><p><strong>Anna G. Duboff</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0009-0002-7316-3831">0009-0002-7316-3831</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/agduboff">agduboff</a>
<br>
<small>
Department of Chemistry, University of Washington
· Funded by Summer Research Acceleration Fellowship, Department of Chemistry, University of Washington
</small></p></li>
<li><p><strong>Nicholas M. Riley</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1536-2966">0000-0002-1536-2966</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/rileynm">rileynm</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/riley_nm1">riley_nm1</a>
<br>
<small>
Department of Chemistry, University of Washington
· Funded by Grant R00 GM147304
</small></p></li>
<li><p><strong>Jesse G. Meyer</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2753-3926">0000-0003-2753-3926</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/jessegmeyerlab">jessegmeyerlab</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/j_my_sci">j_my_sci</a>
<br>
<small>
Department of Computational Biomedicine, Cedars Sinai Medical Center
· Funded by Grant R21 AG074234; Grant R35 GM142502
</small></p></li>
</ul>
<div id="correspondence">
<p>✉ — Correspondence possible via <a href="https://github.com/jessegmeyerlab/proteomics-tutorial/issues">GitHub Issues</a></p>
</div>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<p>Proteomics is the large scale study of protein structure and function from biological systems.<br />
“Shotgun proteomics” or “bottom-up proteomics” is the prevailing strategy, in which proteins are hydrolyzed into peptide that are analyzed by mass spectrometry.
Proteomics studies can be applied to diverse studies ranging from simple protein identification to studies of protein-protein interactions, absolute and relative protein quantification, post-translational modifications, and protein stability.
To enable this range of different experiments, there are diverse strategies for proteome analysis.
The nuances of how proteomic workflows differ may be difficult to understand for new practitioners.
Here, we provide a comprehensive tutorial of different proteomics methods.
Our tutorial covers all necessary steps starting from protein extraction and ending with biological interpretation.
We expect that this work will serve as a basic resource for new practitioners of the field of shotgun or bottom-up proteomics.</p>
<h2 class="page_break_before" id="introduction">Introduction</h2>
<p>Proteomics is the large scale study of protein structure and function.
Proteins are translated from mRNAs that are transcribed from the genome.
Although the genome encodes potential cellular functions and states, the study of proteins is necessary to truly understand biology.
Currently, proteomic studies are facilitated by mass spectrometry, although alternative methods are being developed.</p>
<p>Modern proteomics started around the year 1990 with the introduction of soft ionization methods that enabled, for the first time, transfer of large biomolecules into the gas phase without destroying them <span class="citation" data-cites="4Ajjx5fo 15Ye4Za7Y">[<a href="#ref-4Ajjx5fo" role="doc-biblioref">1</a>,<a href="#ref-15Ye4Za7Y" role="doc-biblioref">2</a>]</span>.
Shortly afterward, the first computer algorithm for matching peptides to a database was introduced <span class="citation" data-cites="pG7zq91Q">[<a href="#ref-pG7zq91Q" role="doc-biblioref">3</a>]</span>.
Another major milestone that allowed identification of over 1000 proteins were actually improvements to chromatography <span class="citation" data-cites="ujahSo3G">[<a href="#ref-ujahSo3G" role="doc-biblioref">4</a>]</span>.
As the volume of data exploded, methods for statistical analysis transitioned use from the wild west to modern informatics based on statistical models <span class="citation" data-cites="mw5jxASm">[<a href="#ref-mw5jxASm" role="doc-biblioref">5</a>]</span> and the false discovery rate <span class="citation" data-cites="y9Bki24q">[<a href="#ref-y9Bki24q" role="doc-biblioref">6</a>]</span>.
<!-- Todo: figure 1: major milestones in proteomics technology since 1990 --></p>
<p>Two strategies of mass spectrometry-based proteomics differ fundamentally by whether proteins are cleaved into peptides before analysis: “top-down” and “bottom-up”.
Bottom-up proteomics (also refered to as shotgun proteomics) is defined by the hydrolysis of proteins into peptide pieces <span class="citation" data-cites="16QSCuG38">[<a href="#ref-16QSCuG38" role="doc-biblioref">7</a>]</span>.
Therefore, bottom-up proteomics does not actually measure proteins, but must infer their presence <span class="citation" data-cites="mw5jxASm">[<a href="#ref-mw5jxASm" role="doc-biblioref">5</a>]</span>.
Sometimes proteins are inferred from only one peptide sequence representing a small fraction of the total protein sequence predicted from the genome.
In contrast, top-down proteomics attempts to measure all proteins intact <span class="citation" data-cites="a6Z8FNxW">[<a href="#ref-a6Z8FNxW" role="doc-biblioref">8</a>]</span>.
The potential benefit of top-down proteomics is the ability to measure proteoforms <span class="citation" data-cites="h00Trxkt">[<a href="#ref-h00Trxkt" role="doc-biblioref">9</a>]</span>.
However, due to myriad analytical challenges, the depth of protein coverage that is achievable by top-down proteomics is generally less than that of bottom-up proteomics <span class="citation" data-cites="14nZGo469 CwZoo7w">[<a href="#ref-14nZGo469" role="doc-biblioref">10</a>,<a href="#ref-CwZoo7w" role="doc-biblioref">11</a>]</span>.</p>
<p>In this tutorial we focus on the bottom-up proteomics workflow.
The most common version of this workflow is generally comprised of the following steps.
First, proteins in a biological sample must be extracted.
Usually this is done by denaturing and solubilizing the proteins while disrupting DNA and tissue.
Next, proteins are hydrolyzed into peptides, usually using a protease like trypsin.
Peptides from proteome hydrolysis must be purified.
Most often this is done with reversed phase chromatography cartridges or tips.
The peptides are then almost always separated by liquid chromatography before they are ionized and introduced into a mass spectrometer.
The mass spectrometer then collects precursor and fragment ion data from those peptides.
The data analysis is usually the rate limiting step.
Peptides must be identified, and proteins are inferred and quantities are assigned.
Changes in proteins across conditions are determined with statistical tests, and results must be interpreted in the context of the relevant biology.</p>
<p>There are many variations on this workflow. The wide variety of experimental goals that are achievable with proteomics technology leads to a wide variety of potential proteomics workflows. Even choice is important and every choice will affect the results. In this tutorial, we cover all of the required steps in detail to serve as a tutorial for new proteomics practioners.</p>
<ol type="1">
<li>Biochemistry basics</li>
<li>Types of experiments enabled by proteomics</li>
<li>Protein extraction</li>
<li>proteolysis</li>
<li>Isotopic Labeling</li>
<li>Enrichments</li>
<li>Peptide purification</li>
<li>Mass Spectrometry</li>
<li>Peptide Ionization</li>
<li>Data Acquisition</li>
<li>Basic Data Analysis</li>
<li>Biological Interpretation</li>
<li>Sample Fractionation</li>
<li>Tandem MS</li>
<li>Experimental considerations and design</li>
</ol>
<h2 class="page_break_before" id="biochemistry-basics">Biochemistry Basics</h2>
<h3 id="proteins">Proteins</h3>
<p>Proteins are large biomolecules or biopolymers made up of amino acids which are linked by peptide bonds.
They perform various functions in living organisms ranging from having structural roles to functional involvement in cellular signaling and the catalysis of chemical reactions (enzymes).
Proteins are made up of 20 different amino acids (not counting pyrrolysine and selenocysteine, which only occur in specific organisms) and their sequence is encoded in their corresponding genes.
The human genome encodes more than 20,000 different proteins.
Each protein is present at a different abundances.
Previous studies have shown that the concentration range of proteins can span over a range of at least seven orders of magnitude to up to 20 000 000 copies per cell and that their distribution is tissue-specific[<span class="citation" data-cites="Iei0Ms6X">[<a href="#ref-Iei0Ms6X" role="doc-biblioref">12</a>]</span>]<span class="citation" data-cites="OQN3Pb5c">[<a href="#ref-OQN3Pb5c" role="doc-biblioref">13</a>]</span>.
Due to genetic variations, as well as alternative splicing and post-translational modifications, multiple different proteoforms can be produced from one single gene (Figure 1) [<span class="citation" data-cites="6a8Xt7SG">[<a href="#ref-6a8Xt7SG" role="doc-biblioref">14</a>]</span>; https://doi.org/10.1038/s41587-023-01714-x].</p>
<div id="fig:proteoforms" class="fignos">
<figure>
<img src="images/intro-centralDogma-proteoforms.png" style="width:100.0%" data-tag="1" alt="Figure 1: Proteome Complexity There is more than one protein product from each gene product due to alternative splicing and post-translational modifications. This means there are likely many more unique “proteoforms” than there are genes. Some estimates are upwards of 1,000,000 unique possible protein sequences." />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>Proteome Complexity</strong>
There is more than one protein product from each gene product due to alternative splicing and post-translational modifications.
This means there are likely many more unique “proteoforms” than there are genes.
Some estimates are upwards of 1,000,000 unique possible protein sequences.</figcaption>
</figure>
</div>
<h4 id="ptms">PTMs</h4>
<p>After protein biosynthesis, enzymatic and nonenzymatic processes change the protein sequence through proteolysis or chemical modification of the amino acid side chains (Figure X).
PTMs are an important biological process contributing to the diversity and control of the cellular proteome.
Proteins can be post-translationally modified through enzymatic and non-enzymatic reactions <em>in vivo</em> and <em>in vitro</em> <span class="citation" data-cites="m2ZOJt1R">[<a href="#ref-m2ZOJt1R" role="doc-biblioref">15</a>]</span>.
PTMs can be reversible or irreversible, and they change protein function in several ways, for example, by altering substrate–enzyme(s) interactions or altering protein-protein interactions <span class="citation" data-cites="uLIaWFNx OEP3MV7r">[<a href="#ref-uLIaWFNx" role="doc-biblioref">16</a>,<a href="#ref-OEP3MV7r" role="doc-biblioref">17</a>]</span>.</p>
<p>More than 400 PTMs have been discovered in both prokaryotic and eukaryotic cells.
These modifications are crucial in controlling protein functions and signal transduction pathways <span class="citation" data-cites="fAHUecLj">[<a href="#ref-fAHUecLj" role="doc-biblioref">18</a>]</span>.
The most commonly studied and biologically relevant post-translational modifications include ubiquitinations (Lys, Cys, Ser, Thr, N-term), succinylations (Lys), methylations (Arg, Lys, His, Glu, Asn, Cys), disulfide bonds (Cys-Cys), methylation (Arg, Lys), oxidations (any amino acid, but especially Met, Trp, His, Cys), phosphorylations (Ser, Thr, Tyr, His), acetylations (Lys, N-term), glycosylations (Arg, Asp, Cys, Ser, Thr, Tyr, Trp) and lipidations.</p>
<p>Post-translational modification of a protein can alter its function, activity, structure, location and interactions.
PTMs alter signal transduction pathways and gene expression control <span class="citation" data-cites="e078CIsS">[<a href="#ref-e078CIsS" role="doc-biblioref">19</a>]</span> regulation of apoptosis <span class="citation" data-cites="oTq7yQcv 1DDSEV4N5">[<a href="#ref-oTq7yQcv" role="doc-biblioref">20</a>,<a href="#ref-1DDSEV4N5" role="doc-biblioref">21</a>]</span> by phosphorylation.
Ubiquitination regulates protein degradation <span class="citation" data-cites="1CHaUvQ1A">[<a href="#ref-1CHaUvQ1A" role="doc-biblioref">22</a>]</span>, SUMOylation regulates chromatin structure, DNA repair, transcription, cell-cycle progression <span class="citation" data-cites="M4mSYOgE 4iAess79">[<a href="#ref-M4mSYOgE" role="doc-biblioref">23</a>,<a href="#ref-4iAess79" role="doc-biblioref">24</a>]</span>, and palmitoylation regulates maintenance of the structural organization of exosome-like extracellular vesicle membranes by <span class="citation" data-cites="blcqlkZJ">[<a href="#ref-blcqlkZJ" role="doc-biblioref">25</a>]</span>.
Glycosylation is a ubiquitous modification that regulates a variety of T cell functions, such as cellular migration, T cell receptor signalling, cell survival, and apoptosis <span class="citation" data-cites="V8FVI0EN 889bLrDQ">[<a href="#ref-V8FVI0EN" role="doc-biblioref">26</a>,<a href="#ref-889bLrDQ" role="doc-biblioref">27</a>]</span>.
Deregulation of PTMs is linked to cellular stress and diseases <span class="citation" data-cites="1FiLH8A0G">[<a href="#ref-1FiLH8A0G" role="doc-biblioref">28</a>]</span>.</p>
<p>Several non-MS methods exist to study PTMs, including in vitro PTM reaction tests with radioactive isotope-labelled substrates, western blot with PTM-specific antibodies, and peptide and protein arrays <span class="citation" data-cites="fzBRT8ZP Wcw3KfEB">[<a href="#ref-fzBRT8ZP" role="doc-biblioref">29</a>,<a href="#ref-Wcw3KfEB" role="doc-biblioref">30</a>]</span>.
While effective, these approaches have many limitations, such as inefficiency and difficulty in producing pan-specific antibodies.
MS-based proteomics approaches are currently the predominant tool for identifying and quantifying changes in PTMs.</p>
<h4 id="protein-structure">Protein Structure</h4>
<p>Almost all proteins (except for intrinsically disordered proteins<span class="citation" data-cites="nOIi7YMn">[<a href="#ref-nOIi7YMn" role="doc-biblioref">31</a>]</span>) fold into 3D structures either by themselves or assisted through chaperones<span class="citation" data-cites="aq0DyuaG">[<a href="#ref-aq0DyuaG" role="doc-biblioref">32</a>]</span>.
There are four levels relevant to the folding of any protein:</p>
<ul>
<li><p>Primary structure:
The protein’s linear amino acid sequence, with amino acids connected through peptide bonds.</p></li>
<li><p>Secondary structure:
The amino acid chain’s folding: α-helix, β-sheet or turn.</p></li>
<li><p>Tertiary structure:
The three-dimensional structure of the protein.</p></li>
<li><p>Quarternary structure:
The structure of several protein molecules/subunits in one complex.</p></li>
</ul>
<h2 class="page_break_before" id="types-of-experiments">Types of Experiments</h2>
<p>A wide range of questions are addressable with proteomics technology, which translates to a wide range of variations of proteomics workflows.
Sometimes identifying what proteins are present is desired, and sometimes the quantities of as many proteins as possible are desired.
Proteomics experiments can be both qualitative and quantitative.
The following sections give an overview of some common proteomics experiments.</p>
<h3 id="protein-abundance-changes">Protein abundance changes</h3>
<p>A common experiment is unbiased mapping of proteins along with detection of changes in their abundance across sample groups.
This is achieved using one of the quantitative methods mentioned elsewhere in this manuscript such as LFQ or TMT.
In these experiments, data should be collected from at least three biological replicates of each condition to estimate the variance of measuring each protein.
Depending on the experiment design, different statistical tests are used to ask if there are changes in the measured protein abundances between groups.
If there are only two groups, the quantities might be compared with a t-test or with a Wilcoxon signed-rank test.
The latter is a non-parametric version of the t-test.
If there are more than two sample groups, then often Analysis of Variance (ANOVA) is used instead.
With either testing scheme, the p-values from the first set of tests must be corrected for multiple tests.
A common method for p value correction is the Benjamini-Hochberg method <span class="citation" data-cites="UeFCwwR7">[<a href="#ref-UeFCwwR7" role="doc-biblioref">33</a>]</span>.
These types of experiments have revealed wide ranges of proteomic remodeling from various biological systems.</p>
<h3 id="post-translational-modifications-ptms">Post-Translational Modifications (PTMs)</h3>
<p>Proteins are modified after they are translated with various chemical groups, or even by proteolytic cleavage such as n-terminal methionine removal.
There are several proteomics methods that are developed specific to each type of modification.
See also the section on Protein/Peptide Enrichment and Depletion.</p>
<h4 id="phosphoproteomics">Phosphoproteomics</h4>
<p>Phosphoproteomics is the study of the phosphorylation of proteins.
Currently, there are two techniques that are useful for phosphoproteomics: mass spectrometry and western blotting.
A key challenge with phosphoproteomics is sensitivity; it is important to ensure that there is sufficient amount of protein before completing a phosphoproteomics project, typically at least 1 mg of total protein per sample.
After proteolysis of the proteome, phosphopeptides need to be enriched to be detected by mass spectrometry.
Various methods of enrichment have been developed <span class="citation" data-cites="E5n1UfBA N0xHE81w 4q5oxW6 1E7oJsvFz">[<a href="#ref-E5n1UfBA" role="doc-biblioref">34</a>,<a href="#ref-N0xHE81w" role="doc-biblioref">35</a>,<a href="#ref-4q5oxW6" role="doc-biblioref">36</a>,<a href="#ref-1E7oJsvFz" role="doc-biblioref">37</a>]</span>.<br />
See the Peptide/Protein Enrichment and Depletion section for more details.
Despite advancement in technology of phosphoproteomics, the following challenges still exist: limited sample amounts, highly complex samples, and huge dynamic range <span class="citation" data-cites="EiFGJCed">[<a href="#ref-EiFGJCed" role="doc-biblioref">38</a>]</span>.
Additionally, phosphoproteomic analysis is often time-consuming and requires the use of expensive equipment such as enrichment kits.</p>
<h4 id="glycoproteomics">Glycoproteomics</h4>
<p>One PTM gaining interest due to its ubiquity and emerging functional roles is glycosylation.
Protein glycosylation sites can be N-linked glycosylation sites (asparagine-linked), O-linked glycosylation sites (serine/threonine-linked).
Understanding the function of protein glycosylation will help us understand numerous biological processes since this is a universal protein modification across all domains of life <span class="citation" data-cites="PNr94DXb MH9QSnbc nSDlzuSO buMJaA2S">[<a href="#ref-PNr94DXb" role="doc-biblioref">39</a>,<a href="#ref-MH9QSnbc" role="doc-biblioref">40</a>,<a href="#ref-nSDlzuSO" role="doc-biblioref">41</a>,<a href="#ref-buMJaA2S" role="doc-biblioref">42</a>]</span>.</p>
<p>Studies of phosphorylation and glycosylation share several experimental pipeline steps including sample preparation.
Protein clean-up approaches for glycoproteomics may differ from other proteomics experiments because glycopeptides are more hydrophilic than most peptides.
Some approaches mentioned in literature include: filter-aided sample preparation (FASP), suspension traps (S-traps), and protein aggregation capture (PAC) <span class="citation" data-cites="PNr94DXb Iygi3cGd 1FU2NfVYY yO8bBfo8 7zlczgkb t0hz9mu6 VTprpeAT">[<a href="#ref-PNr94DXb" role="doc-biblioref">39</a>,<a href="#ref-Iygi3cGd" role="doc-biblioref">43</a>,<a href="#ref-1FU2NfVYY" role="doc-biblioref">44</a>,<a href="#ref-yO8bBfo8" role="doc-biblioref">45</a>,<a href="#ref-7zlczgkb" role="doc-biblioref">46</a>,<a href="#ref-t0hz9mu6" role="doc-biblioref">47</a>,<a href="#ref-VTprpeAT" role="doc-biblioref">48</a>]</span>.
Multiple proteases may be used to increase the sequence coverage and detect more modification sites, such as: trypsin, chymotrypsin, Pepsin, WaLP/MaLP <span class="citation" data-cites="yeQvImXS">[<a href="#ref-yeQvImXS" role="doc-biblioref">49</a>]</span>, GluC, AspN, Pronase, Proteinase K, OgpA, StcEz, BT4244, AM0627, AM1514, AM0608, Pic, ZmpC, CpaA, IMPa, PNGase F, Endo F, Endo H, and OglyZOR <span class="citation" data-cites="PNr94DXb">[<a href="#ref-PNr94DXb" role="doc-biblioref">39</a>]</span>.
Mass spectrometry has improved over the past decade so that now more strategies are available for glycoprotein structure elucidation and also for glycosylation site quantification <span class="citation" data-cites="PNr94DXb">[<a href="#ref-PNr94DXb" role="doc-biblioref">39</a>]</span>.</p>
<h3 id="structural-techniques">Structural techniques</h3>
<p>Several proteomics methods have been developed to reveal protein structure information for simple and complex systems.</p>
<h4 id="xl-ms-cross-linking-mass-spectrometry">XL-MS (Cross-linking mass spectrometry)</h4>
<p>XL-MS is an emerging technology for the field of proteomics.
It can be used to determine changes in protein-protein interactions.
XL-MS covalently locks interacting proteins together to preserve interactions and proximity during MS analysis.
XL-MS is different from traditional MS in that it requires the identification of chimeric MS/MS spectra from cross-linked peptides <span class="citation" data-cites="waP2bBmk XYcoBTz1">[<a href="#ref-waP2bBmk" role="doc-biblioref">50</a>,<a href="#ref-XYcoBTz1" role="doc-biblioref">51</a>]</span>.</p>
<p>The common steps in a XL-MS workflow are as follows <span class="citation" data-cites="v6mVdK34">[<a href="#ref-v6mVdK34" role="doc-biblioref">52</a>]</span>:<br />
1. Generate a system with protein-protein interactions of interest (<em>in vitro</em> or <em>in vivo</em> <span class="citation" data-cites="JnaXzrse">[<a href="#ref-JnaXzrse" role="doc-biblioref">53</a>]</span>)<br />
2. Add a cross-linking reagent to covalently connect adjacent protein regions (such as disuccinimidyl sulfoxide, DSSO) <span class="citation" data-cites="XYcoBTz1">[<a href="#ref-XYcoBTz1" role="doc-biblioref">51</a>]</span><br />
3. Proteolysis to produce peptides<br />
4. MS/MS data collection<br />
5. Identify cross-linked peptide pairs using special software (i.e. pLink <span class="citation" data-cites="KW2lStHJ">[<a href="#ref-KW2lStHJ" role="doc-biblioref">54</a>]</span>)<br />
6. Generate cross-link maps for structural modeling</p>
<h4 id="hydrogen-deuterium-exchange-mass-spectrometry-hdx-ms">Hydrogen deuterium exchange mass spectrometry (HDX-MS)</h4>
<p>HDX-MS works by detecting changes in peptide mass due to exchange of amide hydrogens of the protein backbone is with deuterium from D2O <span class="citation" data-cites="1Dqlo9aRm">[<a href="#ref-1Dqlo9aRm" role="doc-biblioref">55</a>]</span>.
The rate of exchange depends on the protein solvent accessible surface area, dynamics, and the properties of the amino acid sequence <span class="citation" data-cites="1Dqlo9aRm cZM0UPqp SZzVEJCj H647Pppu">[<a href="#ref-1Dqlo9aRm" role="doc-biblioref">55</a>,<a href="#ref-cZM0UPqp" role="doc-biblioref">56</a>,<a href="#ref-SZzVEJCj" role="doc-biblioref">57</a>,<a href="#ref-H647Pppu" role="doc-biblioref">58</a>]</span>.
Although using D2O as a solvent to make deuterium labeled sample is simple, HDX-MS requires a number of controls to ensure that experimental conditions capture the dynamics of interest <span class="citation" data-cites="1Dqlo9aRm 1CHLgMzps 19wzQMZwm m2HWEBQZ">[<a href="#ref-1Dqlo9aRm" role="doc-biblioref">55</a>,<a href="#ref-1CHLgMzps" role="doc-biblioref">59</a>,<a href="#ref-19wzQMZwm" role="doc-biblioref">60</a>,<a href="#ref-m2HWEBQZ" role="doc-biblioref">61</a>]</span>.
If the peptide dissociation process is tuned appropriately, residue-level quantification of changes in solvent accessibility are possible within a measured peptide <span class="citation" data-cites="whtWKxfa">[<a href="#ref-whtWKxfa" role="doc-biblioref">62</a>]</span>.
If done correctly, HDX can produce precise protein structure measurements with high reproducibility.
Masson <em>et al.</em> gave recommendations on how to prep samples, conduct data analysis, and present findings in a detailed stepwise manner that will be useful to have if you are new to using this technique <span class="citation" data-cites="1Dqlo9aRm">[<a href="#ref-1Dqlo9aRm" role="doc-biblioref">55</a>]</span>.</p>
<h4 id="radical-footprinting">Radical Footprinting</h4>
<p>This technique uses hydroxyl radical footprinting and MS in order to elucidate protein structures, assembly, and interactions within a large macromolecule <span class="citation" data-cites="17FfyYJKn dyFOYdP5">[<a href="#ref-17FfyYJKn" role="doc-biblioref">63</a>,<a href="#ref-dyFOYdP5" role="doc-biblioref">64</a>]</span>.
In addition to proteomics applications, various approaches to make hydroxide radicals have also been applied for footprinting studies in nucleic acid/ligand interactions <span class="citation" data-cites="8tWrfuT2 b8XE9cl8 14ACSyAnR">[<a href="#ref-8tWrfuT2" role="doc-biblioref">65</a>,<a href="#ref-b8XE9cl8" role="doc-biblioref">66</a>,<a href="#ref-14ACSyAnR" role="doc-biblioref">67</a>]</span>.
This chapter is very useful in learning more on this topic <span class="citation" data-cites="muDfX3XR">[<a href="#ref-muDfX3XR" role="doc-biblioref">68</a>]</span>.</p>
<p>There are several methods of producing radicals for protein footprinting:<br />
1. Fenton and Fenton‐like Chemistry <span class="citation" data-cites="17FfyYJKn zy4wero bAhX0GGr">[<a href="#ref-17FfyYJKn" role="doc-biblioref">63</a>,<a href="#ref-zy4wero" role="doc-biblioref">69</a>,<a href="#ref-bAhX0GGr" role="doc-biblioref">70</a>]</span><br />
2. Electron-Pulse Radiolysis <span class="citation" data-cites="17FfyYJKn GyzG2odq">[<a href="#ref-17FfyYJKn" role="doc-biblioref">63</a>,<a href="#ref-GyzG2odq" role="doc-biblioref">71</a>]</span><br />
3. High‐Voltage Electrical Discharge <span class="citation" data-cites="17FfyYJKn tcDiyhQT">[<a href="#ref-17FfyYJKn" role="doc-biblioref">63</a>,<a href="#ref-tcDiyhQT" role="doc-biblioref">72</a>]</span><br />
4. Synchrotron X‐ray Radiolysis of Water <span class="citation" data-cites="17FfyYJKn gryR5tjl">[<a href="#ref-17FfyYJKn" role="doc-biblioref">63</a>,<a href="#ref-gryR5tjl" role="doc-biblioref">73</a>]</span><br />
5. Plasma Formation of OH Radicals <span class="citation" data-cites="17FfyYJKn 43HI8GWO">[<a href="#ref-17FfyYJKn" role="doc-biblioref">63</a>,<a href="#ref-43HI8GWO" role="doc-biblioref">74</a>]</span><br />
6. Photolysis of Hydrogen Peroxide <span class="citation" data-cites="17FfyYJKn pFnbzp8Y">[<a href="#ref-17FfyYJKn" role="doc-biblioref">63</a>,<a href="#ref-pFnbzp8Y" role="doc-biblioref">75</a>]</span></p>
<h4 id="fast-photochemical-oxidation-of-proteins-fpop-doi10.1074jbc.rev119.006218">Fast photochemical oxidation of proteins (FPOP) <span class="citation" data-cites="IH1lj0uY">[<a href="#ref-IH1lj0uY" role="doc-biblioref">76</a>]</span></h4>
<p>FPOP is an example of a radical footprinting method.
In FPOP, a laser-based hydroxyl radical protein footprinting MS method that relies on the irreversible labeling of solvent-exposed amino acid side chains by hydroxyl radicals in order to understand structure of proteins.
A laser produces 248 nm light that causes hydrogen peroxide to break into a pair of hydroxyl radicals <span class="citation" data-cites="pFnbzp8Y cMJbHB1a">[<a href="#ref-pFnbzp8Y" role="doc-biblioref">75</a>,<a href="#ref-cMJbHB1a" role="doc-biblioref">77</a>]</span>.
The flow rate of solution through the capillary and laser frequency are adjusted so that each protein molecule is irradiated only once.
After they are irradiated, the sample is collected to a tube that has catalase and free methionine in the buffer so that the H2O2 and hydroxyl radical are quenched to prevent further unwarranted oxidation, which may then modify residues that become exposed due unfolding after the initial labeling.
Control samples are made by running the sample through the flow system without any irradiation.
Another experimental control is done by adding a radical scavenger to tune the extent of protein oxidation <span class="citation" data-cites="LgS9wDAM sqSJTywu">[<a href="#ref-LgS9wDAM" role="doc-biblioref">78</a>,<a href="#ref-sqSJTywu" role="doc-biblioref">79</a>]</span>.
FPOP has wide application for proteins including measurements of fast protein folding and transient dynamics.</p>
<h4 id="protein-painting-doi10.1038ncomms5413-doi10.1074jbc.ra118.007310">Protein Painting <span class="citation" data-cites="sZQji0TC 17xQ2aw0Q">[<a href="#ref-sZQji0TC" role="doc-biblioref">80</a>,<a href="#ref-17xQ2aw0Q" role="doc-biblioref">81</a>]</span></h4>
<p>Protein painting uses “molecular paints” to noncovalently coat the solvent accessible surface of proteins.
These paint molecules will coat the protein surfaces but will not have access to the hydrophobic cores or protein-protein interface regions that solvents cannot access.
If the “paint” covers free amines of lysine side chains, the parts that are painted will not be subjected to trypsin cleavage, while the unpainted areas will.
After proteolysis, the peptides samples will be subjected to MS.
A lack of proteolysis in a region is interpreted as solvent accessibility, which gives rough structural information about complex protein mixtures or even a whole proteome.</p>
<h4 id="lip-ms-limited-proteolysis-mass-spectrometry-doi10.1016j.tibs.2020.05.006-doi10.1038nprot.2017.100-doi10.1038s41596-022-00771-x-doi10.1038nprot.2017.100">LiP-MS (limited proteolysis mass spectrometry) <span class="citation" data-cites="gdTpkDDu xTQMmTt5 6IZfi1O xTQMmTt5">[<a href="#ref-gdTpkDDu" role="doc-biblioref">82</a>,<a href="#ref-xTQMmTt5" role="doc-biblioref">83</a>,<a href="#ref-xTQMmTt5" role="doc-biblioref">83</a>,<a href="#ref-6IZfi1O" role="doc-biblioref">84</a>]</span></h4>
<p>Limited proteolysis coupled to mass spectrometry (LiP-MS) is a method that tracks structural changes in complex proteomes in response to arbitrary stimuli.
The idea is to detect changes in protease-accessible protein regions due to the perturbation.
For example, a drug binding may make a disordered region protected from non-specific proteolysis.
LiP-MS can therefore provide a somewhat unbiased view of structural changes at the proteome scale.
This may be useful when attempting to discover a mechanism for some treatment that causes a phenotype.
For additional information about LiP-MS, please refer to this document: <span class="citation" data-cites="1BxnTxtGI">[<a href="#ref-1BxnTxtGI" role="doc-biblioref">85</a>]</span></p>
<h3 id="protein-stability-and-small-molecule-binding">Protein stability and small molecule binding</h3>
<h4 id="thermal-proteome-profiling-tpp-doi10.15252msb.20199232-doi10.1016j.cbpa.2022.102225-doi10.1021jacs.1c10621-doi10.10029781118970195.ch11">Thermal proteome profiling (TPP) <span class="citation" data-cites="JOskUci 8UcGwfym 89c9sDdK Ssg2QwSE">[<a href="#ref-JOskUci" role="doc-biblioref">86</a>,<a href="#ref-8UcGwfym" role="doc-biblioref">87</a>,<a href="#ref-89c9sDdK" role="doc-biblioref">88</a>,<a href="#ref-Ssg2QwSE" role="doc-biblioref">89</a>]</span></h4>
<p>Thermal proteome profiling (TPP) involves subjecting a protein sample to a thermal shift assay (TSA), in which the protein is exposed to a range of temperatures, and the resulting changes in protein stability are monitored using mass spectrometry.
During a typical TPP experiment, a protein sample is first treated with a drug of interest to stabilize protein-ligand interactions.
The sample is then divided into multiple aliquots, which are subjected to different temperatures in order to induce thermal denaturation.
The resulting drug induced changes in protein stability curves are detected using mass spectrometry.
By comparing protein stability curves across the temperatures between treatment conditions, TPP can provide insight into the proteins that bind a ligand.</p>
<h4 id="cellular-thermal-shift-assay-cetsa-doi10.1016j.cbpa.2019.11.004">Cellular Thermal Shift Assay (CETSA) <span class="citation" data-cites="rMCVhIUo">[<a href="#ref-rMCVhIUo" role="doc-biblioref">90</a>]</span></h4>
<p>CETSA is the same idea as TPP, but specifically for studies of protein stability in live cells.
The assay is capable of detecting shifts in the thermal equilibrium of cellular proteins in response to a variety of perturbations, but most commonly in response to drug treatments.</p>
<h3 id="protein-protein-interactions-ppis">Protein-protein interactions (PPIs)</h3>
<h4 id="affinity-purification-coupled-to-mass-spectrometry-ap-ms-doi10.1002iub.1081-doi10.1038nrg3356-doi10.1038nmeth.2400">Affinity purification coupled to mass spectrometry (AP-MS) <span class="citation" data-cites="ieBDHPPg DORXXewQ 1BsZO9WFn">[<a href="#ref-ieBDHPPg" role="doc-biblioref">91</a>,<a href="#ref-DORXXewQ" role="doc-biblioref">92</a>,<a href="#ref-1BsZO9WFn" role="doc-biblioref">93</a>]</span></h4>
<p>AP-MS is an approach that involves the purification of a target protein or protein complex using a specific antibody followed by mass spectrometry analysis to identify the interacting proteins.
In a typical AP-MS experiment, a protein or protein complex of interest is first tagged with a specific epitope or affinity tag, such as a FLAG or HA tag, which is used to selectively capture the target protein using an antibody.
The protein complex is then purified from the sample using a series of wash steps, and the interacting proteins are identified using mass spectrometry.
The success of AP-MS experiments depends on a number of factors, including the quality of the antibody or tag used for purification, the specificity and efficiency of the resin used for capture, and the sensitivity and resolution of the mass spectrometer.
In addition, careful experimental design and data analysis are critical for accurate identification and interpretation of protein-protein interactions.
AP-MS has been used to study a wide range of biological processes, including signal transduction pathways, protein complexes, and protein post-translational modifications.
AP-MS has been performed on a whole proteome scale as part of the BioPlex project <span class="citation" data-cites="yeZfXPV6 m46yvHfn VMc1jEkd">[<a href="#ref-yeZfXPV6" role="doc-biblioref">94</a>,<a href="#ref-m46yvHfn" role="doc-biblioref">95</a>,<a href="#ref-VMc1jEkd" role="doc-biblioref">96</a>]</span>.
Despite its widespread use, AP-MS has some limitations, including the potential for non-specific interactions, the difficulty in interpreting complex data sets, and the possibility of missing important interacting partners due to limitations in sensitivity or specificity.
However, with continued advances in technology and data analysis methods, AP-MS is likely to remain a valuable tool for the study of protein-protein interactions.</p>
<h4 id="apex-peroxidase-doi10.1038nbt.2375-doi10.1126science.1230593">APEX peroxidase <span class="citation" data-cites="LkwAXvyl 1BbkEbfDz">[<a href="#ref-LkwAXvyl" role="doc-biblioref">97</a>,<a href="#ref-1BbkEbfDz" role="doc-biblioref">98</a>]</span></h4>
<p>APEX-MS is a labeling technique that enables genetically attaches a peroxidase to a protein of interest.
When biotin-phenol is transiently added in the presence of hydrogen peroxide, nearby proteins are covalently biotinylated <span class="citation" data-cites="D5NLgmhd">[<a href="#ref-D5NLgmhd" role="doc-biblioref">99</a>]</span>.
APEX thereby enables discovery of interacting proteins in living cells.
One of the major advantages of APEX is its ability to label proteins in their native environment, allowing for the identification of interactions that occur under physiological conditions.
Despite its advantages, APEX has some limitations, including the potential for non-specific labeling, the difficulty in distinguishing between direct and indirect interactions, and the possibility of missing interactions that occur at low abundance or in regions of the cell that are not effectively labeled.</p>
<h4 id="proximity-dependent-biotin-identification-bioid-doi10.1083jcb.201112098-doi10.1074mcp.r120.001941-doi10.1002pmic.201600123-doi10.1091mbc.e15-12-0844">Proximity-dependent biotin identification (BioID) <span class="citation" data-cites="zSgLeSta NFFhvScy 1FVEjFnHE jJJVaczi">[<a href="#ref-zSgLeSta" role="doc-biblioref">100</a>,<a href="#ref-NFFhvScy" role="doc-biblioref">101</a>,<a href="#ref-1FVEjFnHE" role="doc-biblioref">102</a>,<a href="#ref-jJJVaczi" role="doc-biblioref">103</a>]</span></h4>
<p>BioID is a proximity labeling technique that allows for the identification of protein-protein interactions.
BioID involves the genetic tagging of a protein of interest with a promiscuous biotin ligase in live cells, which then biotinylates proteins in close proximity to the protein of interest.
One of the advantages of BioID is its ability to label proteins in their native environment, allowing for the identification of interactions that occur under physiological conditions.
BioID has been used to identify a wide range of protein interactions, including receptor-ligand interactions, signaling complexes, and protein localization.
BioID is a slower reaction than APEX and therefore may pick up more transient interactions.
BioID has the same limitations as APEX.
For more information on BioID, please refer to this chapter: <span class="citation" data-cites="sas7Wv3H">[<a href="#ref-sas7Wv3H" role="doc-biblioref">104</a>]</span></p>
<h2 class="page_break_before" id="protein-extraction">Protein Extraction</h2>
<p>Protein extraction from the sample of interest is the initial phase of any mass spectrometry-based proteomics experiment.
Thought should be given to any planned downstream assays, specific needs of proteolysis (LiP-MS, post translational modification enrichments, enzymatic reactions, glycan purification or hydrogen-deuterium exchange experiments) long term project goals (reproducibility, multiple sample types, low abundance samples) as well as to the initial experimental question (coverage of a specific protein, subcellular proteomics, global proteomics, protein-protein interactions or immune or affinity enrichment of a specific classes of modifications.)
The 2009 version of Methods in Enzymology: guide to Protein Purification <span class="citation" data-cites="emz5eXnO">[<a href="#ref-emz5eXnO" role="doc-biblioref">105</a>]</span> serves as a deep dive into how molecular biologists and biochemists traditionally thought about protein extraction.
The Protein Protocols handbook <span class="citation" data-cites="LvSVXQeu">[<a href="#ref-LvSVXQeu" role="doc-biblioref">106</a>]</span> and the excellent review by Linn <span class="citation" data-cites="YFivV9n5">[<a href="#ref-YFivV9n5" role="doc-biblioref">107</a>]</span> are good sources of general proteomics protocols for a scientist new to the field.
Any change in extraction conditions should be expected to create potential changes in downstream results.
Be sure to think about and optimize the protein extraction step first and stick with a protocol that works for your needs.
If a collaborator is attempting to reproduce your results, make sure they begin with the same extraction protocols.</p>
<h3 id="buffer-choice">Buffer choice</h3>
<h4 id="general-proteomics">General proteomics</h4>
<p>A common question to proteomics core facilities is, “What is the best buffer for protein extraction?”
Unfortunately, there is no one correct answer.
For global proteomics experiments where maximizing the number of protein or peptide identifications is a goal, a buffer of neutral pH (50-100 mM PBS, Tris, HEPES, ammonium bicarbonate, triethanolamine bicarbonate; pH 7.5-8.5) is used in conjunction with a chaotrope or surfactant to denature and solubilize proteins (e.g., 8 M urea, 6 M guanidine, 5% SDS) <span class="citation" data-cites="OZvOVLht bpTXhTR5">[<a href="#ref-OZvOVLht" role="doc-biblioref">108</a>,<a href="#ref-bpTXhTR5" role="doc-biblioref">109</a>]</span>.
Often other salts like 50-150 mM NaCl are also added.
Complete denaturation of the proteins in the sample in a timely fashion is an advantage as it generally prevents changes to protein status by endogenous proteases, kinases, phosphotases, and other enzymes.
If intact protein separations are planned (based on size or isoelectric point) choose a denaturant compatible with those methods, such as SDS<span class="citation" data-cites="Igj2yRI0">[<a href="#ref-Igj2yRI0" role="doc-biblioref">110</a>]</span>.
Compatibility with protease (typically trypsin) and peptide cleanup steps will need to be considered.
8 M urea must be diluted to 2 M or less for trypsin and chymotrypsin digestions, while guanidine and SDS should be removed either through protein precipitation, through filter-assisted sample preparation (FASP), or similar solid phase digestion techniques.
Note that some buffers can potentially introduce modifications onto proteins such as carbamylation from urea at high temperatures <span class="citation" data-cites="B8hLhFif">[<a href="#ref-B8hLhFif" role="doc-biblioref">111</a>]</span>.</p>
<h4 id="protein-protein-interactions">Protein-protein interactions</h4>
<p>Denaturing conditions will efficiently extract proteins – but they will denature/disrupt most protein-protein interactions.
If you are working on an immune- or affinity purification of a specific protein and expect to analyze enzymatic activity, structural features, and/or protein-protein interactions, a non-denaturing lysis buffer should be utilized <span class="citation" data-cites="G9HWYdpG 15gFWnWy1">[<a href="#ref-G9HWYdpG" role="doc-biblioref">112</a>,<a href="#ref-15gFWnWy1" role="doc-biblioref">113</a>]</span>.
Check the calculated pI and hydrophobicity (the Expasy.org resource ProtParam is useful fo this) for a good idea of starting pH/conductivity, but you may need to perform a stability screen.
In general, a good starting point for the buffer will still be close to neutral pH with 50-250 mM NaCl, but specific proteins may require pH as low as 2 or as high as 9 for stable extraction.
A low percent of mass spec compatible detergent may also be used.
Newer mass spectrometry compatible detergents are also useful for protein extraction and ease of downstream processing – including Rapigest® (Waters), N-octyl-β-glucopyranoside, Azo <span class="citation" data-cites="1FdkzkZPe">[<a href="#ref-1FdkzkZPe" role="doc-biblioref">114</a>]</span>, PPS silent surfactant <span class="citation" data-cites="KbqdsNwY">[<a href="#ref-KbqdsNwY" role="doc-biblioref">115</a>]</span>, sodium laurate <span class="citation" data-cites="1FX8gYsTy">[<a href="#ref-1FX8gYsTy" role="doc-biblioref">116</a>]</span>, and sodium deoxycholate<span class="citation" data-cites="7nXVfPN7">[<a href="#ref-7nXVfPN7" role="doc-biblioref">117</a>]</span>.
AVOID the use of tween-20, triton-X, NP-40, and PEGs as these compounds are challenging to remove after digestion <span class="citation" data-cites="EZCh7C0x">[<a href="#ref-EZCh7C0x" role="doc-biblioref">118</a>]</span>.</p>
<h4 id="optional-additives">Optional additives</h4>
<p>For non-denaturing buffer conditions, which preserve tertiary and quaternary protein structures, additional additives may not be necessary for successful extraction and to prevent proteolysis or PTM modifications throughout the extraction process.
Protease, phosphatase and deubiquitinase inhibitors are optional additives in less denaturing conditions or in experiments focused on specific post-translational modifications.
Keep in mind that protease inhibitors may impact digestion conditions and will need to be diluted or removed prior to trypsin addition.
For extraction of DNA or RNA binding proteins, addition of a small amount of nuclease or benzonase might be useful for degradation of any bound nucleic acids and result in a more consistent digestion <span class="citation" data-cites="EK9zCZWn">[<a href="#ref-EK9zCZWn" role="doc-biblioref">119</a>]</span>.</p>
<h3 id="mechanical-or-sonic-disruption">Mechanical or Sonic Disruption</h3>
<h4 id="cell-lysis">Cell lysis</h4>
<p>One typical lysis buffer is 8 M urea in 100 mM Tris, pH 8.5; the pH based on optimum trypsin activity <span class="citation" data-cites="Cj5ZGyu8">[<a href="#ref-Cj5ZGyu8" role="doc-biblioref">120</a>]</span>
Small mammalian cell pellets and exosomes will lyse almost instantly upon addition denaturing buffer.
If non-denaturing conditions are desired, osmotic swelling and subsequent shearing or sonication can be applied <span class="citation" data-cites="17GoHwva3">[<a href="#ref-17GoHwva3" role="doc-biblioref">121</a>]</span>.
Efficiency of extraction and degradation of nucleic acids can be improved using various sonication methods: 1) probe sonicator with ice; 2) water bath sonicator with ice or cooling; 3) bioruptor® sonication device 4) Adaptive focused acoustics (AFA®) <span class="citation" data-cites="195QiKJpv">[<a href="#ref-195QiKJpv" role="doc-biblioref">122</a>]</span>.
Key to these additional lysis techniques are to keep the temperature of the sample from rising significantly which can cause proteins to aggregate or degrade.
Some cell types may require additional force for effective lysis (see below).
For cells with cell walls (i.e. bacteria or yeast), lysozyme is often added in the lysis buffer.
Any added protein will be present in downstream results, however, so excessive addition of lysozyme is to be avoided unless tagged protein purification will occur.</p>
<h4 id="tissueother-lysis">Tissue/other lysis</h4>
<p>Although small pieces of soft tissue can often be successfully extracted with the probe and sonication methods described above, larger/harder tissues as well as plants/yeast/fungi are better extracted with some form of additional mechanical force.
If proteins are to be extracted from a large amount of sample, such as soil, feces, or other diffuse input, one option is to use a dedicated blender and filter the sample, followed by centrifugation.
If samples are smaller, such as tissue, tumors, etc., cryo-homogenization is recommended.
The simplest form of this is grinding the sample with liquid nitrogen and a mortar and pestle.
Tools such as bead beaters (i.e. FastPrep-24®) are also used, where the sample is placed in a tube with appropriately sized glass or ceramics beads and shaken rapidly.
Cryo-mills are chambers where liquid nitrogen is applied around a vessel and large bead or beads.
Cryo-fractionators homogenize samples in special bags that are frozen in liquid nitrogen and smashed with various degrees of force <span class="citation" data-cites="4EGQHHdM">[<a href="#ref-4EGQHHdM" role="doc-biblioref">123</a>]</span>.
After homogenization, samples can be sonicated by one of the methods above to fragment DNA and increase solubilization of proteins.</p>
<h3 id="measuring-the-efficiency-of-protein-extraction">Measuring the efficiency of protein extraction</h3>
<p>Following protein extraction, samples should be centrifuged (10-14,000 g for 10-30 min depending on sample type) to remove debris and any unlysed material prior to determinining protein concentration.
The amount of remaining insoluble material should be noted throughout an experiment as a large change may indicate protein extraction issues.
Protein concentration can be calculated using a number of assays or tools <span class="citation" data-cites="wXcRYG3x DRVtai1B">[<a href="#ref-wXcRYG3x" role="doc-biblioref">124</a>,<a href="#ref-DRVtai1B" role="doc-biblioref">125</a>]</span>; generally absorbance measuremnts are facile, fast and affordable, such as Bradford or BCA assays.
Protein can also be estimated by tryptophan fluorescence, which has the benefit of not consuming sample <span class="citation" data-cites="7YHVbaDH">[<a href="#ref-7YHVbaDH" role="doc-biblioref">126</a>]</span>.
A nanodrop UV spectrophotometer may be used to measure absorbance at UV280.
Consistency in this method is important as each method will have inherent bias and error <span class="citation" data-cites="qFKCAO2X jQLWiiId">[<a href="#ref-qFKCAO2X" role="doc-biblioref">127</a>,<a href="#ref-jQLWiiId" role="doc-biblioref">128</a>]</span>.
Extraction buffer components will need to be compatible with any assay chosen; alternatively, buffer may be removed (see below) prior to protein concentration calculation.</p>
<h3 id="reduction-and-alkylation">Reduction and alkylation</h3>
<p>Typically, disulfide bonds in proteins are reduced and alkylated prior to proteolysis in order to disrupt structures and simplify peptide analysis.
This allows better access to all residues during proteolysis and removes the crosslinked peptides created by S-S inter peptide linkages.
There are a variety of reagent options for these steps.
For reduction, the typical agents used are 5-15 mM concentration of tris(2-carboxyethyl)phosphine hydrochloride (TCEP-HCl), dithiothreitol (DTT), or 2-mercaptoethanol (2BME).
TCEP-HCl is an efficient reducing agent, but it also significantly lowers sample pH, which can be abated by increasing sample buffer concentration or resuspending TCEP-HCl in an appropriate buffer system (i.e 1M HEPES pH 7.5).<br />
Following the reducing step, a slightly higher 10-20mM concentration of alkylating agent such as chloroacetamide/iodoacetamide or n-ethyl maleimide is used to cap the free thiols <span class="citation" data-cites="OglKN2v5 2cCnZ6ft 9vICnnYM">[<a href="#ref-OglKN2v5" role="doc-biblioref">129</a>,<a href="#ref-2cCnZ6ft" role="doc-biblioref">130</a>,<a href="#ref-9vICnnYM" role="doc-biblioref">131</a>]</span>.
In order to monitor which cysteine residues are linked or modified in a protein, it is also possible to alkylate free cysteines with one reagent, reduce di-sulfide bonds (or other cysteine modifications) and alkylate with a different reagent <span class="citation" data-cites="GkFAexwC Z6nk0TFI s4OLOlUG">[<a href="#ref-GkFAexwC" role="doc-biblioref">132</a>,<a href="#ref-Z6nk0TFI" role="doc-biblioref">133</a>,<a href="#ref-s4OLOlUG" role="doc-biblioref">134</a>]</span>.
Alkylation reactions are generally carried out in the dark at room temperature to avoid excessive off-target alkylation of other amino acids.</p>
<h3 id="removal-of-bufferinterfering-small-molecules">Removal of buffer/interfering small molecules</h3>
<p>If extraction must take place in a buffer which is incompatible for efficient proteolysis (check the guidelines for the protease of choice), then protein cleanup should occur prior to digestion.
This is generally performed through precipitation of proteins.
The most common types are 1) acetone, 2) trichloroacetic acid (TCA), and 3) methanol/chloroform/water <span class="citation" data-cites="xvenBhjQ OMzPHtgd">[<a href="#ref-xvenBhjQ" role="doc-biblioref">135</a>,<a href="#ref-OMzPHtgd" role="doc-biblioref">136</a>]</span>.
Proteins are generally insoluble in most pure organic solvents, so cold ethanol or methanol are sometimes used.
Pellets should be washed with organic solvent for complete removal especially of detergents.
Alternatively, solid phase based digestion methods such as S-trap <span class="citation" data-cites="Llj72YHx">[<a href="#ref-Llj72YHx" role="doc-biblioref">137</a>]</span>, FASP <span class="citation" data-cites="rWTbDNu2 bkON7ZR2">[<a href="#ref-rWTbDNu2" role="doc-biblioref">138</a>,<a href="#ref-bkON7ZR2" role="doc-biblioref">139</a>]</span>, SP3 <span class="citation" data-cites="29YpOcx3 R9HFU5Rw">[<a href="#ref-29YpOcx3" role="doc-biblioref">140</a>,<a href="#ref-R9HFU5Rw" role="doc-biblioref">141</a>]</span> and on column/bead can allow for proteins to be applied to a solid phase and buffers removed prior to proteolysis <span class="citation" data-cites="IRVtdB7A">[<a href="#ref-IRVtdB7A" role="doc-biblioref">142</a>]</span>.
Specialty detergent removal columns exist (Pierce/Thermo Fisher Scientific) but add expense and time consuming steps to the process.
Relatively low concentrations of specific detergents, such as 1% deoxycholate (DOC), or chaotropes (i.e. 1M urea) are compatible with proteolysis by trypsin/Lys-C.
Often proteolysis-compatible concentrations of these detergents and chaotropes are achieved by diluting the sample in appropriate buffer (i.e. 100 mM ammonium bicarbonate, pH 8.5) after cell or tissue lysis in a higher concentration.
DOC can then be easily removed by precipitation or phase separation <span class="citation" data-cites="GcubNF0j">[<a href="#ref-GcubNF0j" role="doc-biblioref">143</a>]</span> following digestion by acidification of the sample to pH 2-3.<br />
Any small-molecule removal protocol should be tested for efficiency prior to implementing in a workflow with many samples as avoiding detergent (or polymer) contamination in the LC/MS is very important.</p>
<h3 id="protein-quantification">Protein quantification</h3>
<p>After proteins are isolated from the sample matrix, they are often quantified.
Protein quantification is important to assess the yeild of an extraction procedure, and to adjust the scale of the downstream processing steps to match the amount of protein.
For example, when purifying peptides, the amount of sorbent should match the amount of material to be bound.
Presently, there is a wide variety of techniques to quantitate the amount of protein present in a given sample.
These methods can be broadly divided into three types as follows:</p>
<h4 id="colorimetry-based-methods">Colorimetry-based methods:</h4>
<p>The method includes different assays like Coomassie Blue G-250 dye binding (the Bradford assay), the Folin-Lowry assay, the bicinchoninic acid (BCA) assay and the biuret assay <span class="citation" data-cites="1F0WRIEv4">[<a href="#ref-1F0WRIEv4" role="doc-biblioref">144</a>]</span>.
The most commonly used method is the BCA assay.
In the BCA method the peptide bonds of the protein reduce cupric ions [Cu2+] to cuprous ions [Cu+] at a rate which is proportional to the amount of protein present in the sample.
Subsequently, the BCA reagent binds to the cuprous ions, leading to the formation of a complex which absorbs 562 nm wavelength light.
This permits a direct correlation between sample protein concentration and absorbance <span class="citation" data-cites="MgeRCWZg 7aeNLPmC">[<a href="#ref-MgeRCWZg" role="doc-biblioref">145</a>,<a href="#ref-7aeNLPmC" role="doc-biblioref">146</a>]</span>.
The Bradford assay is another method for protein quantification also based on colorimetry principle.
It relies on the interaction between the Coomassie brilliant blue dye and the protein based on hydrophobic and electrostatic interactions. Dye binding shifts the absorption maxima from 470 nm to 595 nm <span class="citation" data-cites="10DIgRvDQ L2cR80nH">[<a href="#ref-10DIgRvDQ" role="doc-biblioref">147</a>,<a href="#ref-L2cR80nH" role="doc-biblioref">148</a>]</span>.
Similarly, the Folin- Lowry method is a two-step colorimetric assay.
Step one is the biuret reaction wherein complexes of copper with the nitrogen in the protein molecule are formed.
In the second step, the complexed tyrosine and tryptophan amino acids react with Folin–Ciocalteu phenol reagent generating an intense, blue-green color absorbing light at 650–750 nm <span class="citation" data-cites="2lZxrpnu">[<a href="#ref-2lZxrpnu" role="doc-biblioref">149</a>]</span>.</p>
<p>Another simple but less reliable protein quantification method of UV-Vis Absorbance at 280 nm estimates the protein concentration by measuring the absorption of the aromatic residues; tyrosine, and tryptophan, at 280 nm <span class="citation" data-cites="q1g1rJrT">[<a href="#ref-q1g1rJrT" role="doc-biblioref">150</a>]</span>.</p>
<h4 id="fluorescence-based-methods">Fluorescence-based methods:</h4>
<p>Colorimetric assays are inexpensive and require common lab equipment, but colorimetric detection is less sensitive than fluorescence.
Protein in proteomic samples can be quantified using intrinsic fluorescence of tryptophan based on the assumption that approximately 1% of all amino acids in the proteome are tryptophan <span class="citation" data-cites="NLQoCcYj">[<a href="#ref-NLQoCcYj" role="doc-biblioref">151</a>]</span>.</p>
<p>NanoOrange is an assay for the quantitative measurement of proteins in solution using the NanoOrange reagent, a merocyanine dye that produces a large increase in fluorescence quantum yield when it interacts with detergent-coated proteins.
Fluorescence is measured using 485-nm excitation and 590-nm emission wavelengths.
The NanoOrange assay can be performed using fluorescence microplate readers, fluorometers, and laser scanners that are standard in the laboratory <span class="citation" data-cites="DRVtai1B">[<a href="#ref-DRVtai1B" role="doc-biblioref">125</a>]</span>.</p>
<p>3-(4-carboxybenzoyl)quinoline-2-carboxaldehyde (CBQCA) is a sensitive fluorogenic reagent for amine detection, which can be used for analyzing proteins in solution.
As the number of accessible amines in a protein is modulated by its concentration, CBQCA has a greater sensitivity and dynamic range when measuring protein concentration <span class="citation" data-cites="iRPPTjzF">[<a href="#ref-iRPPTjzF" role="doc-biblioref">152</a>]</span>.</p>
<h3 id="summary">Summary</h3>
<p>Often you will be given protein extraction conditions from molecular biologists or biochemistry which you will have to make work with downstream mass spectrometry applications.
For bottom-up proteomics, the overreaching goal is efficient and consistent extraction and digestion.
A range of mechanical and non-mechanical extraction protocols have been developed and the use any one specific technique is generally dictated by sample type or assay requirements (i.e. native versus non-native extraction).
Extraction can be aided by the addition of detergents and/or chaotropes to the sample, but care should be taken that these additives do not interfere with the sample digestion step or downstream mass-spectrometry applications.</p>
<h2 class="page_break_before" id="proteolysis">Proteolysis</h2>
<p>Proteolysis is the defining step that differentiates bottom-up or shotgun proteomics from top-down proteomics.
Hydrolysis of proteins is extremely important because it defines the population of potentially identifyable peptides.
Generally peptides between a length of 7-35 amino acids are considered useful for mass spectrometry analysis.
Peptides that are too long are difficult to identify by tandem mass spectrometry, or may be lost during sample preparation due to irreversible binding with solid-phase extraction sorbents.
Peptides that are too short are also not useful because they may match to many proteins during protein inference.
There are many choices of enzymes and chemicals that hydrolyze proteins into peptides.
This section summarizes potential choices and their strengths and weaknesses.</p>
<p>Trypsin is the most common choice of protease for proteome hydrolysis <span class="citation" data-cites="vpBg0CP7">[<a href="#ref-vpBg0CP7" role="doc-biblioref">153</a>]</span>.
Trypsin is favorable because of its specificity, availability, efficiency and low cost.
Trypsin cleaves at the C-terminus of basic amino acids, Arg and Lys.
Many of the peptides generated from trypsin are short in length (less than ~ 20 amino acids), which is ideal for chromatographic separation, MS-based peptide fragmentation and identification by database search.
The main drawback of trypsin is that majority (56%) of the tryptic peptides are ≤ 6 amino acids, and hence using trypsin alone limits the observable proteome <span class="citation" data-cites="16FpKYZCe lSFMVWD1 10vHwy5gN">[<a href="#ref-16FpKYZCe" role="doc-biblioref">154</a>,<a href="#ref-lSFMVWD1" role="doc-biblioref">155</a>,<a href="#ref-10vHwy5gN" role="doc-biblioref">156</a>]</span>.
This limits the number of identifiable protein isoforms and post-translational modifications.</p>
<p>Although trypsin is the most common protease used for proteomics, in theory it can only cover a fraction of the proteome predicted from the genome <span class="citation" data-cites="y9KaXde4">[<a href="#ref-y9KaXde4" role="doc-biblioref">157</a>]</span>.
This is due to production of peptides that are too short to be unique, for example due to R and K immediately next to each other.
Peptides below a certain length are likely to happen many times in the whole proteome, meaning that even if we identify them we cannot know their origin protein.
In protein regions devoid of R/K, trypsin may also result in very long peptides that are then lost do to irreversible binding to the solid phase extraction device, or that become difficult to identify due to complicated fragmentation patterns.
Thus, parts of the true proteome sequences that are present are lost after trypsin digestion due to both production of very long and very short peptides.</p>
<p>Many alternative proteases are available with different specificities that complement trypsin to reveal different proteomic sequences <span class="citation" data-cites="7aKTZT1i 16FpKYZCe">[<a href="#ref-16FpKYZCe" role="doc-biblioref">154</a>,<a href="#ref-7aKTZT1i" role="doc-biblioref">158</a>]</span>, which can help distinguish protein isoforms <span class="citation" data-cites="gXhdeFOF">[<a href="#ref-gXhdeFOF" role="doc-biblioref">159</a>]</span>.
The enzyme choice mostly depends on the application.
In general, for a mere protein identification mostly trypsin is the choice due to the reasons aforementioned.
However, alternative enzymes can facilitate <em>de novo</em> assembly when the genomic data information is limited in the public database repositories <span class="citation" data-cites="14bc70ELO zmo81rNN IZw9sVjg 19KCAADV7 7BuX2TBR">[<a href="#ref-14bc70ELO" role="doc-biblioref">160</a>,<a href="#ref-zmo81rNN" role="doc-biblioref">161</a>,<a href="#ref-IZw9sVjg" role="doc-biblioref">162</a>,<a href="#ref-19KCAADV7" role="doc-biblioref">163</a>,<a href="#ref-7BuX2TBR" role="doc-biblioref">164</a>]</span>.
Use of multiple proteases for proteome digestion also can improve the sensitivity and accuracy of protein quantification <span class="citation" data-cites="1GLf6FYas">[<a href="#ref-1GLf6FYas" role="doc-biblioref">165</a>]</span>.
Moreover, by providing an increased peptide diversity, the use of multiple proteases can expand sequence coverage and increase the probability of finding peptides which are unique to single proteins <span class="citation" data-cites="WubQ7yFP yeQvImXS y9KaXde4">[<a href="#ref-yeQvImXS" role="doc-biblioref">49</a>,<a href="#ref-y9KaXde4" role="doc-biblioref">157</a>,<a href="#ref-WubQ7yFP" role="doc-biblioref">166</a>]</span>.
A multi-protease approach can also improve the identification of N-Termini and signal peptides for small proteins <span class="citation" data-cites="Ac0S3PrF">[<a href="#ref-Ac0S3PrF" role="doc-biblioref">167</a>]</span>.
Overall, integrating multiple-protease data can increase the number of proteins identified <span class="citation" data-cites="P861T4mW 14eCb7Jfc">[<a href="#ref-P861T4mW" role="doc-biblioref">168</a>,<a href="#ref-14eCb7Jfc" role="doc-biblioref">169</a>]</span>, the number of identified post-translational modifications detected <span class="citation" data-cites="WubQ7yFP 18s9UXUGf yeQvImXS">[<a href="#ref-yeQvImXS" role="doc-biblioref">49</a>,<a href="#ref-WubQ7yFP" role="doc-biblioref">166</a>,<a href="#ref-18s9UXUGf" role="doc-biblioref">170</a>]</span> and decrease the ambiguity of the protein group list <span class="citation" data-cites="WubQ7yFP">[<a href="#ref-WubQ7yFP" role="doc-biblioref">166</a>]</span>.</p>
<p>Lysyl endopeptidase (Lys-C) obtained from <em>Lysobacter enzymogenesis</em> is a serine protease involved in cleaving carboxyl terminus of Lys <span class="citation" data-cites="ZgqlEERS lSFMVWD1">[<a href="#ref-lSFMVWD1" role="doc-biblioref">155</a>,<a href="#ref-ZgqlEERS" role="doc-biblioref">171</a>]</span>.
Like trypsin, the optimum pH range required for its activity is from 7 to 9.
A major advantage of Lys-C is its resistance to denaturing agents, including 8 M urea - a chaotrope commonly used to denature proteins <em>prior</em> to digestion <span class="citation" data-cites="gXhdeFOF">[<a href="#ref-gXhdeFOF" role="doc-biblioref">159</a>]</span>.
Trypsin is less efficient at cleaving Lys than Arg, which could limit the quality of quantitation from tryptic peptides.
Hence, to achieve complete protein digestion with minimal missed cleavages, Lys-C is often used simultaneously with trypsin digestion <span class="citation" data-cites="nobcST81">[<a href="#ref-nobcST81" role="doc-biblioref">172</a>]</span>.</p>
<p>Alpha-lytic protease (aLP) is also secreted by the soil bacterial <em>Lysobacter enzymogenesis</em> <span class="citation" data-cites="oDNy6K9V">[<a href="#ref-oDNy6K9V" role="doc-biblioref">173</a>]</span>.
Wild-type aLP (WaLP) and an active site mutant of aLP, M190A (MaLP), have been used to expand proteome coverage <span class="citation" data-cites="yeQvImXS">[<a href="#ref-yeQvImXS" role="doc-biblioref">49</a>]</span>.
Based on observed peptide sequences from yeast proteome digestion, WaLP showed a specificity for small aliphatic amino acids like alanine, valine, and glycine, but also threonine and serine.
MaLP showed specificity for slightly larger amino acids like methionine, phenylalanine, and surprisingly, a preference for leucine over isoleucine.
The specificity of WaLP for threonine enabled the first method for mapping endogenous human SUMO sites <span class="citation" data-cites="4iAess79">[<a href="#ref-4iAess79" role="doc-biblioref">24</a>]</span>.</p>
<p>Glutamyl peptidase I, commonly known as Glu-C or V8 protease, is a serine protease obtained from <em>Staphyloccous aureus</em> <span class="citation" data-cites="UeP5oN6q">[<a href="#ref-UeP5oN6q" role="doc-biblioref">174</a>]</span>.
Glu-C cleaves at the C-terminus of glutamate, but also after aspartate <span class="citation" data-cites="UeP5oN6q FcovvAwu">[<a href="#ref-UeP5oN6q" role="doc-biblioref">174</a>,<a href="#ref-FcovvAwu" role="doc-biblioref">175</a>]</span>.</p>
<p>Peptidyl-Asp metallopeptidase, commonly known as Asp-N, is a metalloprotease obtained from <em>Pseudomonas fragi</em> <span class="citation" data-cites="Z8khRFem">[<a href="#ref-Z8khRFem" role="doc-biblioref">176</a>]</span>.
Asp-N catalyzes the hydrolysis of peptide bonds at the N-terminal of aspartate residues.
The optimum activity of this enzyme occurs at a pH range between 4 and 9.
As with any metalloprotease, chelators like EDTA should be avoided for digestion buffers when using Asp-N.
Studies also suggest that Asp-N cleaves at the amino terminus of glutamate when a detergent is present in the proteolysis buffer <span class="citation" data-cites="Z8khRFem">[<a href="#ref-Z8khRFem" role="doc-biblioref">176</a>]</span>.
Asp-N often leaves many missed cleavages <span class="citation" data-cites="gXhdeFOF">[<a href="#ref-gXhdeFOF" role="doc-biblioref">159</a>]</span>.</p>
<p>Chymotrypsin or chymotrypsinogen A is a serine protease obtained from porcine or bovine pancreas with an optimum pH range from 7.8 to 8.0 <span class="citation" data-cites="129jHkJ7J">[<a href="#ref-129jHkJ7J" role="doc-biblioref">177</a>]</span>.
It cleaves at the C-terminus of hydrphobic amino acids Phe, Trp, Tyr and barely Met and Leu residues.
Since the transmembrane region of membrane proteins commonly lacks tryptic cleavage sites, this enzyme works well with membrane proteins having more hydrophobic residues <span class="citation" data-cites="n8tCUNeh Ddlp2JQj gXhdeFOF">[<a href="#ref-gXhdeFOF" role="doc-biblioref">159</a>,<a href="#ref-n8tCUNeh" role="doc-biblioref">178</a>,<a href="#ref-Ddlp2JQj" role="doc-biblioref">179</a>]</span>.
The chymotryptic peptides generated after proteolysis will cover the proteome space orthogonal to that of tryptic peptides both in a quantitative and qualitative manner <span class="citation" data-cites="VbXzZ784 1BDWCCTHa Ddlp2JQj">[<a href="#ref-Ddlp2JQj" role="doc-biblioref">179</a>,<a href="#ref-VbXzZ784" role="doc-biblioref">180</a>,<a href="#ref-1BDWCCTHa" role="doc-biblioref">181</a>]</span></p>
<p>Clostripain, commonly known as Arg-C, is a cysteine protease obtained from <em>Clostridium histolyticum</em> <span class="citation" data-cites="qJADzs4">[<a href="#ref-qJADzs4" role="doc-biblioref">182</a>]</span>.
It hydrolyses mostly the C-terminal Arg residues and sometimes Lys residues, but with less efficiency.
The peptides generated are generally longer than that of tryptic peptides.
Arg-C is often used with other proteases for improving qualitative proteome data and also for investigating PTMs <span class="citation" data-cites="lSFMVWD1">[<a href="#ref-lSFMVWD1" role="doc-biblioref">155</a>]</span>.</p>
<p>LysargiNase, also known as Ulilysin, is a recently discovered protease belonging to the metalloprotease family.
It is a thermophilic protease derived from <em>Methanosarcina acetivorans</em> that specifically cleaves at the N-terminus of Lys and Arg residues <span class="citation" data-cites="MqKkSQNK">[<a href="#ref-MqKkSQNK" role="doc-biblioref">183</a>]</span>.
Hence, it enabled discovery of C-terminal peptides that were not observed using trypsin.
In addition, it can also cleave modified amino acids such as methylated or dimethylated Arg and Lys <span class="citation" data-cites="MqKkSQNK">[<a href="#ref-MqKkSQNK" role="doc-biblioref">183</a>]</span>.</p>
<p>Peptidyl-Lys metalloendopeptidase, or Lys-N, is an metalloprotease obtained from <em>Grifola frondosa</em> <span class="citation" data-cites="3xdcLBaR">[<a href="#ref-3xdcLBaR" role="doc-biblioref">184</a>]</span>.
It cleaves N-terminally of Lys and has an optimal activity at pH 9.0.
Unlike trypsin, Lys-N is more resistant to denaturing agents and can be heated up to 70 °C <span class="citation" data-cites="lSFMVWD1">[<a href="#ref-lSFMVWD1" role="doc-biblioref">155</a>]</span>.
Reports suggest that the peptides generated after Lys-N digestion produces more of c-type ions in a ETD-based mass spectrometer <span class="citation" data-cites="O6S1GvPn">[<a href="#ref-O6S1GvPn" role="doc-biblioref">185</a>]</span>.
Hence this can be used for analysing PTMs, identification of C-terminal peptides and also for <em>de novo</em> sequencing strategies <span class="citation" data-cites="O6S1GvPn 8DGqkN3D">[<a href="#ref-O6S1GvPn" role="doc-biblioref">185</a>,<a href="#ref-8DGqkN3D" role="doc-biblioref">186</a>]</span>.</p>
<p>Pepsin A, commonly known as pepsin, is an aspartic protease obtained from bovine or porcine pancreas <span class="citation" data-cites="eMC9mqfj">[<a href="#ref-eMC9mqfj" role="doc-biblioref">187</a>]</span>.
Pepsin was one of several proteins crystalized by John Northrop, who shared the 1946 Nobel prize in chemistry for this work <span class="citation" data-cites="pnU7apE 9Yf3fEMC 10CHY6r2E 1EtaxGZW0">[<a href="#ref-pnU7apE" role="doc-biblioref">188</a>,<a href="#ref-9Yf3fEMC" role="doc-biblioref">189</a>,<a href="#ref-10CHY6r2E" role="doc-biblioref">190</a>,<a href="#ref-1EtaxGZW0" role="doc-biblioref">191</a>/]</span>.
Pepsin works at an optimum pH range from 1 to 4 and specifically cleaves Trp, Phe, Tyr and Leu <span class="citation" data-cites="lSFMVWD1">[<a href="#ref-lSFMVWD1" role="doc-biblioref">155</a>]</span>.
Since it possess high enzyme activity and broad specificity at lower pH, it is preferred over other proteases for MS-based disulphide mapping <span class="citation" data-cites="itW7kK3s Vske1oHJ">[<a href="#ref-itW7kK3s" role="doc-biblioref">192</a>,<a href="#ref-Vske1oHJ" role="doc-biblioref">193</a>]</span>.
Pepsin is also used extensively for structural mass spectrometry studies with hydrogen-deuterium exchange (HDX) because the rate of back exchange of the amide deuteron is minimized at low pH <span class="citation" data-cites="Azi1X1R4 1nMCmYmb">[<a href="#ref-Azi1X1R4" role="doc-biblioref">194</a>,<a href="#ref-1nMCmYmb" role="doc-biblioref">195</a>]</span>.</p>
<p>Proteinase K was first isolated from the mold <em>Tritirachium album</em> Limber <span class="citation" data-cites="YOXX6An6">[<a href="#ref-YOXX6An6" role="doc-biblioref">196</a>]</span>.
The epithet ‘K’ is derived from its ability to efficiently hydrolyse keratin <span class="citation" data-cites="YOXX6An6">[<a href="#ref-YOXX6An6" role="doc-biblioref">196</a>]</span>.
It is a member of the subtilisin family of proteases and is relatively unspecific with a preference for proteolysis at hydrophobic and aromatic amino acid residues <span class="citation" data-cites="1CgjQBy2S">[<a href="#ref-1CgjQBy2S" role="doc-biblioref">197</a>]</span>.
The optimal enzyme activity is between pH 7.5 and 12.
Proteinase K is used at low concentrations for limited proteolysis (LiP) and the detection of protein structural changes in the eponymous technique LiP-MS <span class="citation" data-cites="1BhNuatKH">[<a href="#ref-1BhNuatKH" role="doc-biblioref">198</a>]</span>.</p>
<div id="fig:Multiple-protease-proteolysis" class="fignos">
<figure>
<img src="images/Proteolysis_v03.png" style="width:100.0%" data-tag="1" alt="Figure 1: Multiple protease proteolysis improves protein inference The use of other proteases beyond Trypsin such as Lysyl endopeptidase (Lys-C), Peptidyl-Asp metallopeptidase (Asp-N), Glutamyl peptidase I, (Glu-C), Chymotrypsin, Clostripain (Arg-C) or Peptidyl-Lys metalloendopeptidase (Lys-N) can generate a greater diversity of peptides. This improves protein sequence coverage and allows for the correct identification of their N-termini. Increasing the number of complimentary enzymes used will increase the number of proteins identified by single peptides and decreases the ambiguity of the assignment of protein groups. Therefore, this will allow more protein isoforms and post-translational modifications to be identified than using Trypsin alone." />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>Multiple protease proteolysis improves protein inference</strong>
The use of other proteases beyond Trypsin such as Lysyl endopeptidase (Lys-C), Peptidyl-Asp metallopeptidase (Asp-N), Glutamyl peptidase I, (Glu-C), Chymotrypsin, Clostripain (Arg-C) or Peptidyl-Lys metalloendopeptidase (Lys-N) can generate a greater diversity of peptides.
This improves protein sequence coverage and allows for the correct identification of their N-termini.
Increasing the number of complimentary enzymes used will increase the number of proteins identified by single peptides and decreases the ambiguity of the assignment of protein groups.
Therefore, this will allow more protein isoforms and post-translational modifications to be identified than using Trypsin alone.</figcaption>
</figure>
</div>
<p>Although different specificity is useful in theory to enable improved proteome sequence coverage, there are practical challenges because most of the standard workflow is optimized for tryptic peptides.
For example, peptides that lack a c-terminal positive charge due to arginine or lysine side chains can have a less pronounced y-ion series.
This can lead to lower scoring peptide-spectra matches because some peptide identification algorithms preferentially score y ions higher.</p>
<h2 class="page_break_before" id="peptide-quantification">Peptide Quantification</h2>
<!-- todo: change the label of this file eventually to peptide quant instead of peptide labeling
 -->
<h3 id="label-free-quantification-lfq-of-peptides">Label-free quantification (LFQ) of peptides</h3>
<p>LFQ of peptide precursors requires no additional steps in the protein extraction, digestion, and peptide purification workflow.
Samples can be taken straight to the mass spectrometer and are injected one at a time, each sample necessitating their own LC-MS/MS experiment and raw file.
Quantification of peptides by LFQ is routinely performed by many commercial and freely available proteomics software (see Data Analysis section below).
In LFQ, peptide abundances across LC-MS/MS experiments are calculated by aligning windows of accurate peptide mass and retention time.</p>
<h3 id="stable-isotope-labeling-of-peptides">Stable isotope labeling of peptides</h3>
<p>One approach to improve the throughput and quantitative completeness within a group of samples is sample multiplexing via stable isotope labeling.
Multiplexing enables pooling of samples and parallel LC-MS/MS analysis within one run.
Quantification can be achieved at the MS1- or MSn-level, dictated by the upstream labeling strategy.</p>
<p>Stable isotope labeling methods produce peptides that are chemically identical from each sample that differ only in their mass due.
Methods include stable isotope labeling in amino acid cell culture (SILAC) <span class="citation" data-cites="YTQzoOxb">[<a href="#ref-YTQzoOxb" role="doc-biblioref">199</a>]</span> and chemical labeling such as amine-modifying tags for relative and absolute quantification (mTRAQ) <span class="citation" data-cites="1FOQ1H9mS">[<a href="#ref-1FOQ1H9mS" role="doc-biblioref">200</a>]</span> or dimethyl labeling <span class="citation" data-cites="Qqbj9JVK">[<a href="#ref-Qqbj9JVK" role="doc-biblioref">201</a>]</span>.
The labeling of each sample imparts mass shifts (e.g. 4 Da, 8 Da) which can be detected within the MS1 full scan.
The ability to label samples in cell culture has enabled impactful quantitative biology experiments <span class="citation" data-cites="E7KmdZas TGkcmyuH">[<a href="#ref-E7KmdZas" role="doc-biblioref">202</a>,<a href="#ref-TGkcmyuH" role="doc-biblioref">203</a>]</span>.
These approaches have nearly exclusively been performed using data-dependent acquisition (DDA) strategies.
However, recent work employing faster instrumentation has shown the benefits of chemical labeling with 3-plex mTRAQ or dimethyl labels for data-independent acquisition (DIA) <span class="citation" data-cites="QipLkzBy ZhSU2auj">[<a href="#ref-QipLkzBy" role="doc-biblioref">204</a>,<a href="#ref-ZhSU2auj" role="doc-biblioref">205</a>]</span>, an idea originally developed nearly a decade earlier using chemical labels to quantify lysine acetylation and succinylation stoichiometry <span class="citation" data-cites="WPmaUWBS">[<a href="#ref-WPmaUWBS" role="doc-biblioref">206</a>]</span>.
As new tags with higher plexing become available, strategies like plexDIA and mDIA are sure to benefit <span class="citation" data-cites="QipLkzBy ZhSU2auj">[<a href="#ref-QipLkzBy" role="doc-biblioref">204</a>,<a href="#ref-ZhSU2auj" role="doc-biblioref">205</a>]</span>.</p>
<h3 id="peptide-labeling-with-isobaric-tags">Peptide labeling with isobaric tags</h3>
<p>Another approach to improve throughput and quantitative completeness within a group of samples is multiplexing via isobaric labels, a strategy which enables parallel data acquisition after pooling of samples.
Commercial isobaric tags include tandem mass tags (TMT) <span class="citation" data-cites="j9N2kPSr">[<a href="#ref-j9N2kPSr" role="doc-biblioref">207</a>]</span> and isobaric tags for relative and absolute quantification (iTRAQ) <span class="citation" data-cites="V8IykxaB">[<a href="#ref-V8IykxaB" role="doc-biblioref">208</a>]</span> amongst others, and several non-commercial options have also been developed <span class="citation" data-cites="lbOXqNSw">[<a href="#ref-lbOXqNSw" role="doc-biblioref">209</a>]</span>.
10- and 11-plex TMT kits were recently supplanted by proline-based TMT tags (TMTpro), originally introduced as 16-plex kits in 2019 <span class="citation" data-cites="12bJg7wN8">[<a href="#ref-12bJg7wN8" role="doc-biblioref">210</a>]</span> and upgraded to an 18-plex platform in 2021 <span class="citation" data-cites="1HEOHr3h0">[<a href="#ref-1HEOHr3h0" role="doc-biblioref">211</a>]</span>.</p>
<p>The isobaric tag labeling-based peptide quantitation strategy uses derivatization of every peptide sample with a different isotopic incorporation from a set of isobaric mass tags.
All isobaric tags have a common structural theme consisting of 1) an amine-reactive groups (usually triazine ester or N-hydroxysuccinimide [NHS] esters) which react with peptide N-termini and ε-amino group of the lysine side chain of peptides, 2) a balancer group, and 3) a reporter ion group.<br />
Peptide labeling is followed by pooling the labelled samples, which undergo MS and MS/MS analysis simultaneously.
As the isobaric tags are used, peptides labeled with these tags give a single MS peak with the same precursor m/z value in an MS1 scan and identical retention time of liquid chromatography analysis.
The modified parent ions undergo fragmentation during MS/MS analysis generating two kinds of fragment ions: (a) reporter ions and (b) peptide fragment ions.
Each reporter ions’ relative intensity is directly proportional to the peptide abundance in each of the starting samples that were pooled.
As usual, the b- and y-type fragment ion peaks are still used to identify amino acid sequences of peptides, from which proteins can be inferred.
Since it is possible to label most tryptic peptides with an isobaric mass tag at least at the n-termini, numerous peptides from the same protein can be detected and quantified, thus leading to an increase in the confidence in both protein identification and quantification <span class="citation" data-cites="15rS5UMb9">[<a href="#ref-16D3xg72c" role="doc-biblioref">213</a>]</span>.</p>
<p>Because the size of the reporter ions is small and sometimes the mass difference between reporter ions is small (i.e., a ~6 mDa difference when using 13C versus 15N), these methods neary exclusively employ high-resolution mass analyzers, not classical ion traps <span class="citation" data-cites="kIpNiFeU">[<a href="#ref-kIpNiFeU" role="doc-biblioref">214</a>]</span>.
There are examples, however, of using isobaric tags with pulsed q dissociation on linear ion traps (LTQs) <span class="citation" data-cites="ESY1qR9T">[<a href="#ref-ESY1qR9T" role="doc-biblioref">215</a>]</span>.
Suitable instruments are the Thermo Q-Exactive, Exploris, Tribrid, and Astral lines, or TOFs such as the TripleTOF or timsTOFs <span class="citation" data-cites="FMSpJgZ4 OlzbV8x4">[<a href="#ref-FMSpJgZ4" role="doc-biblioref">216</a>,<a href="#ref-OlzbV8x4" role="doc-biblioref">217</a>]</span>.</p>
<!-- todo: put the isobaric tagging figure Rex made here and refer to it -->
<p>The following are some of the isobaric labeling techniques:</p>
<h4 id="isobaric-tags-for-relative-and-absolute-quantitation-itraq">isobaric Tags for Relative and Absolute Quantitation (iTRAQ)</h4>
<p>The iTRAQ tagging method covalently labels the peptide N-terminus and side-chain primary amines with tags of different masses through the NHS-ester bond.
This is followed by mass spectrometry analysis <span class="citation" data-cites="JokZNPi7">[<a href="#ref-JokZNPi7" role="doc-biblioref">218</a>]</span>.
Reporter ions for an 8plex iTRAQ are measured at roughly 113, 114, 115, 116, 117, 118, 119, and 121 Thompsons.
At the moment, two kinds of iTRAQ reagents are available: 4-plex and 8-plex.
Using 4-plex reagents, a maximum of four different biological conditions can be analyzed simultaneously, whereas using 8plex reagents, eight different biological conditions can be analysed <span class="citation" data-cites="43RxThXr 1At6lP9vn">[<a href="#ref-43RxThXr" role="doc-biblioref">219</a>,<a href="#ref-1At6lP9vn" role="doc-biblioref">220</a>]</span>.</p>
<h4 id="itraq-hydrazide-itraqh">iTRAQ hydrazide (iTRAQH)</h4>
<p>iTRAQH is an isobaric tagging reagent for the selective labeling and relative quantification of carbonyl (CO) groups in proteins <span class="citation" data-cites="5gl2gxA4">[<a href="#ref-5gl2gxA4" role="doc-biblioref">221</a>]</span>.
The reactive CO and oxygen groups which are generated as the byproducts of oxidation of lipids at the time of oxidative stress causes protein carbonylation <span class="citation" data-cites="t4QMhmYv">[<a href="#ref-t4QMhmYv" role="doc-biblioref">222</a>]</span>.
iTRAQH is produced from iTRAQ and surplus of hydrazine.
This reagent reacts with peptides which are carbonylated, thus forming a hydrazone group.
iTRAQH is a novel method for analyzing carbonylation sites in proteins utilizing an isobaric tag for absolute and relative quantitation iTRAQ derivative, iTRAQH, and the analytical power of linear ion trap instruments (QqLIT).
This new strategy seems to be well suited for quantifying carbonylation at large scales because it avoids time-consuming enrichment procedures <span class="citation" data-cites="5gl2gxA4">[<a href="#ref-5gl2gxA4" role="doc-biblioref">221</a>]</span>.
Thus, there is no need for enriching modified peptides before LC-MS/MS analysis.</p>
<h4 id="tandem-mass-tag-tmt">Tandem Mass Tag (TMT)</h4>
<p>TMT labeling is based on a similar principle as that of iTRAQ.
In the case of 6-plex-TMT, the masses of reporter groups are roughly 126, 127, 128, 129, 130, and 131 Thompsons <span class="citation" data-cites="kIpNiFeU">[<a href="#ref-kIpNiFeU" role="doc-biblioref">214</a>]</span>.
TMT works best with MS which allow quantitation at MS3-level with higher accuracy (e.g.: Thermo’s Orbitrap Tribrid instruments) <span class="citation" data-cites="15rS5UMb9 9irghCdu">[<a href="#ref-15rS5UMb9" role="doc-biblioref">212</a>,<a href="#ref-9irghCdu" role="doc-biblioref">223</a>]</span> by limiting the ratio distortion of co-isolated precursors within isobaric multiplexed quantitative proteomics <span class="citation" data-cites="1GJhuH2mX agiL3n2A">[<a href="#ref-1GJhuH2mX" role="doc-biblioref">224</a>,<a href="#ref-agiL3n2A" role="doc-biblioref">225</a>]</span>.
Infrared photoactivation of co-isolated TMT fragment ions generates more quantitative reporter ion generation and sensitivity relative to standard beam-type collisional activation <span class="citation" data-cites="1EjowCoSS">[<a href="#ref-1EjowCoSS" role="doc-biblioref">226</a>]</span>
TMT is widely used for quantitative protein biomarker discovery.
In addition, TMT labeling technique helps multiplex sample analysis enabling efficient use of instrument time.
TMT labelling also controls for technical variation because after samples are mixed the ratios are locked in, and any sample loss would be equal across channels.
A wide range of TMT reagents with different multiplexing capabilities are available, such as TMT zero, TMT duplex, TMT six plex, TMT 10-plex, and TMT 11-plex are available along with the recent addition of TMT 16-plex <span class="citation" data-cites="12bJg7wN8">[<a href="#ref-12bJg7wN8" role="doc-biblioref">210</a>]</span> and now TMT 18-plex <span class="citation" data-cites="1HEOHr3h0">[<a href="#ref-1HEOHr3h0" role="doc-biblioref">211</a>]</span>.
These TMT reagents have a similar chemical structure, which allows the efficient transition from method development to multiplex quantification <span class="citation" data-cites="OlzbV8x4">[<a href="#ref-OlzbV8x4" role="doc-biblioref">217</a>]</span>.</p>
<h4 id="iodotmt">iodoTMT</h4>
<p>IodoTMT reagents are isobaric reagents used for tagging cysteine residues of peptides.
The commercially available IodoTMT reagents are iodoTMTzero and iodoTMT 6-plex <span class="citation" data-cites="ghHhZLY4 RikGmYWm">[<a href="#ref-ghHhZLY4" role="doc-biblioref">227</a>,<a href="#ref-RikGmYWm" role="doc-biblioref">228</a>]</span>.</p>
<h4 id="aminoxytmt-isobaric-mass-tags">aminoxyTMT Isobaric Mass Tags</h4>
<p>Also referred to as glyco-TMTs, these reagents have chemistry similar to iTRAQH.
The stable isotope-labeled glyco-TMTs are utilized for quantitating N-linked glycans.
They are derived from the original TMT reagents with an addition of carbonyl-reactive groups, which involve either hydrazide or aminoxy chemistry as functional groups.
These aminooxy TMTs show a better performance as compared to its iTRAQH counterparts in terms of efficiency of labeling and quantification.
The glyco-TMT compounds consist of stable isotopes thus enabling (i) isobaric quantification using MS/MS spectra and (ii) quantification in MS1 spectra using heavy/light pairs.
Aminoxy TMT6-128 and TMT6-131 along with the hydrazide TMT2-126 and TMT2-127 reagents can be used for isobaric quantification.
In the quantification at MS1 level, the light TMT0 and the heavy TMT6 reagents have a difference in mass of 5.0105 Da which is sufficient to separate the isotopic patterns of all common N-glycans.
Glycan quantification based on glyco-TMTs generates more accurate quantification in MS1 spectra over a broad dynamic range.
Intact proteins or their digests obtained from biological samples are treated with PNGase F/A glycosidases to release the N-linked glycans during the process of labeling using aminoxyTMT reagents.
The free glycans are then purified and labeled with the aminoxyTMT reagent at the reducing end.
The labeled glycans from individual samples are subsequently pooled and then undergo analysis in MS for identification of glycoforms in the sample and quantification of relative abundance of reporter ions at MS/MS level <span class="citation" data-cites="XS3gqf5Y">[<a href="#ref-XS3gqf5Y" role="doc-biblioref">229</a>]</span>.</p>
<h4 id="nn-dimethyl-leucine-dileu">N,N-Dimethyl leucine (DiLeu)</h4>
<p>The N,N-Dimethyl leucine, also referred to as DiLeu, is an tandem mass tag reagent which is isobaric and has reporter ions of isotope-encoded dimethylated leucine <span class="citation" data-cites="WvWnJWa8">[<a href="#ref-WvWnJWa8" role="doc-biblioref">230</a>]</span>.
Each incorporated label produces a 145.1 Da mass shift.
A maximum of four samples can be simultaneously analyzed using DiLeu at a highly reduced cost.
MS/MS analysis shows intense reporter ions i.e., dimethylated leucine a1 ions at 115, 116, 117, and 118 m/z.
The DiLeu tag labeling efficiency is similar to that of the iTRAQ.
Although, DiLeu-labeled peptides offer increased confidence for identification of peptides and more reliable quantification as they undergo better fragmentation thus generating higher reporter ion intensities <span class="citation" data-cites="WvWnJWa8">[<a href="#ref-WvWnJWa8" role="doc-biblioref">230</a>]</span>.</p>
<h4 id="deuterium-isobaric-amine-reactive-tag-diart">Deuterium isobaric Amine Reactive Tag (DiART)</h4>
<p>DiART is an isobaric tagging method used in quantitative proteomics <span class="citation" data-cites="OhLoelJI 1FMnd7AHB">[<a href="#ref-OhLoelJI" role="doc-biblioref">231</a>,<a href="#ref-1FMnd7AHB" role="doc-biblioref">232</a>]</span>.
The reporter group in DiART tags is a N,N′-dimethyl leucine reporter group with a mass to charge range of 114–119. DiART reagents can a label a maximum of six samples and further analyzed by MS.
The isotope purity of DiART reagents is very high hence correction of isotopic impurities is not needed at the time of data analysis <span class="citation" data-cites="1CFiIlIUn">[<a href="#ref-1CFiIlIUn" role="doc-biblioref">233</a>]</span>.
The performances of DiART including the mechanism of fragmentation, the number of proteins identified and the quantification accuracy are similar to iTRAQ.
Irrespective of the sequence of the peptide, reporter ions of high-intensity are produced by DiART tags in comparison to those with iTRAQ and thus, DiART labeling can be used to quantify more peptides as well as those with lower abundance, and with reliable results <span class="citation" data-cites="OhLoelJI">[<a href="#ref-OhLoelJI" role="doc-biblioref">231</a>]</span>.
DiART serves as a cheaper alternative to TMT and iTRAQ while also having a comparable labeling efficiency.
It has been observed that these tags are useful in labeling huge protein quantities from cell lysates before TiO2 enrichment in quantitative phosphoproteomics studies <span class="citation" data-cites="bHqseFm6">[<a href="#ref-bHqseFm6" role="doc-biblioref">234</a>]</span>.</p>
<!-- todo: not sure if this is the best place for this figure about quantification strategies, do we need a whole section on quant? -->
<div id="fig:quant-summary" class="fignos">
<figure>
<img src="images/Summay_peptide-protein-labeling_.svg" style="width:100.0%" data-tag="2" alt="Figure 2: Quantitative strategies commonly used in proteomics. A) Label-free quantitation. Proteins are extracted from samples, enzymatically hydrolyzed into peptides and analyzed by mass spectrometry. Chromatographic peak areas from peptides are compared across samples that are analyzed sequentially. B) Metabolic labelling. Stable isotope labeling with amino acids in cell culture (SILAC) is based on feeding cells stable isotope labeled amino acids (“light” or “heavy”). Samples grown with heavy or light amino acids are mixed before cell lysis. The relative intensities of the heavy and light peptide are used to compute protein changes between samples. C) Isobaric or chemical labelling. Proteins are isolated separately from samples, enzymatically hydrolyzed into peptides, and then chemically tagged with isobaric stable isotope labels. These isobaric tags produce unique reporter mass-to-charge (m/z) signals that are produced upon fragmentation with MS/MS. Peptide fragment ions are used to identify peptides, and the relative reporter ion signals are used for quantification." />
<figcaption aria-hidden="true"><span>Figure 2:</span> <strong>Quantitative strategies commonly used in proteomics.</strong>
A) Label-free quantitation.
Proteins are extracted from samples, enzymatically hydrolyzed into peptides and analyzed by mass spectrometry.
Chromatographic peak areas from peptides are compared across samples that are analyzed sequentially.
B) Metabolic labelling.
Stable isotope labeling with amino acids in cell culture (SILAC) is based on feeding cells stable isotope labeled amino acids (“light” or “heavy”).
Samples grown with heavy or light amino acids are mixed before cell lysis.
The relative intensities of the heavy and light peptide are used to compute protein changes between samples.
C) Isobaric or chemical labelling.
Proteins are isolated separately from samples, enzymatically hydrolyzed into peptides, and then chemically tagged with isobaric stable isotope labels.
These isobaric tags produce unique reporter mass-to-charge (m/z) signals that are produced upon fragmentation with MS/MS.
Peptide fragment ions are used to identify peptides, and the relative reporter ion signals are used for quantification.</figcaption>
</figure>
</div>
<h2 class="page_break_before" id="peptideprotein-enrichment-and-depletion">Peptide/Protein Enrichment and Depletion</h2>
<p>In order to study low abundance protein modifications, or to study rare proteins in complex mixtures, various methods have been developed to enrich or deplete specific proteins or peptides.</p>
<h3 id="peptide-enrichment">Peptide enrichment</h3>
<h4 id="glycosylation">Glycosylation</h4>
<p>Mass spectrometry-based analysis of protein glycosylation has emerged as the premier technology to characterize such a universal and diverse class of biomolecules.
Glycosylation is a heterogenous post-translational modification that decorates many proteins within the proteome, conferring broad changes in protein activity. <span class="citation" data-cites="2h6JMw35 nSDlzuSO DOI:10.1101/9781621824213">[<a href="#ref-nSDlzuSO" role="doc-biblioref">41</a>,<a href="#ref-2h6JMw35" role="doc-biblioref">235</a>,<a href="#ref-DOI:10.1101/9781621824213" role="doc-biblioref"><strong>DOI:10.1101/9781621824213?</strong></a>]</span>
This PTM can take many forms.
The covalent linkage of mono- or oligosaccharides to polypeptide backbones through a nitrogen atom of asparagine (N) or an oxygen atom of serine (S) or threonine (T) side-chains creates N- and O-glycans, respectively.<br />
The heterogenity of proteoglycans is not directly tied to the genome, and thus cannot be inferred.
Rather, the abundance and activity of protein glycosylation is governed by glycosyltransferases and glycosidases which add and remove glycans, respectively.
The fields of glycobiology and bioanalytical chemistry are intricately intertwined with mass spectrometry at the center thanks in part to its power of detecting any modification that imparts a mass shift.</p>
<p>Due to the myriad glycan structures and proteins which harbor them, the enrichment of glycoproteins or glycopeptides is not as streamlined as that of other PTMs <span class="citation" data-cites="DC1ukRtO">[<a href="#ref-DC1ukRtO" role="doc-biblioref">236</a>]</span>.
The enrichment of glycoproteome from the greater proteome inherently introduces bias prior to the LC-MS/MS analysis.
One must take into account which class or classes of glycopeptides they are interested in analyzing before enrichment for optimal LC-MS/MS results.
Glycopeptides can be enriched via glycan affinity, for example to glycan-binding proteins, chemical properties like charge or hydrophilicity, chemical coupling of glycans to stationary phases, and by bioorthogonal, chemical biology approaches.
Glycan affinity-based enrichment strategies include the use of lectins, antibodies, inactivated enzymes, immobilized metal affinity chromatography (IMAC), and metal oxide affinity chromatography (MOAC).
The enrichment of glycopeptides by their chemical properties, for example by biopolymer charge and hydrophobicity, include hydrophilic interaction chromatography (HILIC), electrostatic repulsion-hydrophilic interaction chromatography (ERLIC), and porous graphitic carbon (PGC).
One variation of ERLIC combines strong anion exchange, electrostatic repulsion, and hydrophilic interaction chromatography (SAX-ERLIC) has risen in popularity thanks to robustness and commercially available enrichment kits <span class="citation" data-cites="pgTlWbfO">[<a href="#ref-pgTlWbfO" role="doc-biblioref">237</a>]</span>.</p>
<p>Chemical coupling methods most often used to enrich the glycoproteome employ hydrazide chemistry for sialylated glycopeptides.
Glycan are cleaved from the stationary phase by PNGase F.
The dependence of chemical coupling methods on PNGase F biases their output toward N-glycopeptides.
Alkoxyamine compounds and boronic acid-based methods have also shown utility.
We direct readers to several reviews on glycopeptide enrichment strategies <span class="citation" data-cites="DC1ukRtO 3T8OfCLe 17QEVShGe ZH7oVCIW 4qdFwSjo">[<a href="#ref-DC1ukRtO" role="doc-biblioref">236</a>,<a href="#ref-3T8OfCLe" role="doc-biblioref">238</a>,<a href="#ref-17QEVShGe" role="doc-biblioref">239</a>,<a href="#ref-ZH7oVCIW" role="doc-biblioref">240</a>,<a href="#ref-4qdFwSjo" role="doc-biblioref">241</a>]</span></p>
<h3 id="phosphoproteomics-1">Phosphoproteomics</h3>
<p>Protein phosphorylation, a hallmark of protein regulation, dictates protein interactions, signaling, and cellular viability.
This post-translational modificaiton (PTM) involves the installation of a negatively charged phosphate moiety (PO 4-) onto the hydroxyl side-chain of serine (Ser, S), threonine (Thr, T), and tyrosine (Tyr, Y), residues on target proteins.
Protein kinases catalyze the transfer of PO 4- group from ATP to the nucleophile (OH) group of serine, threonine, and tyrosine residues, while protein phosphatases catalyze the removal of PO 4 -.
Phosphorylation changes the charge of a protein, often altering protein conformation and therefore function <span class="citation" data-cites="8FBpI8YZ">[<a href="#ref-8FBpI8YZ" role="doc-biblioref">242</a>]</span>.
Protein phosphorylation is one of the major PTMs that alters the stability, subcellular location, enzymatic activity complex formation, degradation of protein, and cell signalling of protein with a diverse role in cells <span class="citation" data-cites="ahQQ60nZ">[<a href="#ref-FzJAr4ba" role="doc-biblioref">244</a>]</span>.
Phosphorylation can regulate almost all cellular processes, including metabolism, growth, division, differentiation, apoptosis, and signal transduction pathways <span class="citation" data-cites="e078CIsS">[<a href="#ref-e078CIsS" role="doc-biblioref">19</a>]</span>.
Rapid changes in protein phosphorylation are associated with several diseases <span class="citation" data-cites="CG2bNzkD">[<a href="#ref-CG2bNzkD" role="doc-biblioref">245</a>]</span>.</p>
<p>Several methods are used to characterize phosphorylation using modification-specific enrichment techniques combined with advanced MS/MS methods and computational data analysis <span class="citation" data-cites="gPKAjoqC">[<a href="#ref-gPKAjoqC" role="doc-biblioref">246</a>]</span>.
MS-based phosphoproteomics tools are pivotal for the comprehensive study for the structural and dynamics of cellular signalling networks <span class="citation" data-cites="9tmuuRlu">[<a href="#ref-9tmuuRlu" role="doc-biblioref">247</a>]</span>, but there are many challenges <span class="citation" data-cites="ZhEOEHzl">[<a href="#ref-ZhEOEHzl" role="doc-biblioref">248</a>]</span>.
For example, phosphopeptides are low stoichiometry compared to non-phosphorylated peptides, which makes them difficult to identify.
Phosphopeptides also exhibit low ionization efficiency <span class="citation" data-cites="IZ8H05sX">[<a href="#ref-IZ8H05sX" role="doc-biblioref">249</a>]</span>.
To overcome these challenges, it is important to reduce sample complexity to detect large numbers of phosphorylation sites.
This is accomplished using enrichment the modified proteins and/or peptides <span class="citation" data-cites="1DdZgE3oc 1AhoIVYp8 2bm470Yr">[<a href="#ref-1DdZgE3oc" role="doc-biblioref">250</a>,<a href="#ref-1AhoIVYp8" role="doc-biblioref">251</a>,<a href="#ref-2bm470Yr" role="doc-biblioref">252</a>]</span>.
Prefractionation techniques such as strong anionic ion-exchange chromatography (SAX), strong cationic ion-exchange chromatography (SCX), and hydrophilic interaction liquid chromatography (HILIC) are also often useful for reducing sample complexity before enrichment to observe more phosphorylation sites <span class="citation" data-cites="OoEPX07F">[<a href="#ref-OoEPX07F" role="doc-biblioref">253</a>]</span>.</p>
<p>As with any proteomics experiment, phosphoproteomics studies require protein extraction, proteolytic enzyme digestion, phosphopeptide enrichment, peptide fractionation, LC-MS/MS, bioinformatics data analysis, and biological function inference.
Special consideration is required during protein extraction where the cell lysis buffer should include phosphatase inhibitors such as sodium orthovanadate, sodium pyrophosphatase and beta-glycerophosphate <span class="citation" data-cites="cXqDwSab">[<a href="#ref-cXqDwSab" role="doc-biblioref">254</a>]</span>.</p>
<p>Enrichment can be done at the protein level before proteolysis.
Phosphoprotein enrichment typically involves the use of immobilized metal-affinity chromatography (IMAC) to selectively capture phosphorylated proteins based on their high-affinity binding to metal ions such as Ga(III), Fe(III), Zn(II) and Al(III) <span class="citation" data-cites="vghskTda 1Biu5x5kI GINSjn64 tH2WxrEl LbRH4fW1">[<a href="#ref-vghskTda" role="doc-biblioref">255</a>,<a href="#ref-1Biu5x5kI" role="doc-biblioref">256</a>,<a href="#ref-GINSjn64" role="doc-biblioref">257</a>,<a href="#ref-tH2WxrEl" role="doc-biblioref">258</a>,<a href="#ref-LbRH4fW1" role="doc-biblioref">259</a>]</span>.</p>
<p>Enrichment is more commonly done at the peptide level because there are several advantages over phosphoprotein enrichment.
First, peptides have simpler three-dimensional structures than proteins, which makes them easier to separate and analyze.
Second, phosphopeptide enrichment is not hindered by small, lipophilic, and very acidic or alkaline proteins <span class="citation" data-cites="2bm470Yr">[<a href="#ref-2bm470Yr" role="doc-biblioref">252</a>]</span>.
Third, prefractionation techniques such as strong anionic ion-exchange chromatography (SAX), strong cationic ion-exchange chromatography (SCX) and hydrophilic interaction liquid chromatography (HILIC) are easier to use for peptide separation than they are for protein separation, and they are more sensitive than 2D-gel electrophoresis, allowing for the identification of less abundant phosphopeptides <span class="citation" data-cites="f75VaG7P">[<a href="#ref-f75VaG7P" role="doc-biblioref">260</a>]</span>.
As a result, phosphopeptide enrichment has yielded more experimental data than phosphoprotein enrichment <span class="citation" data-cites="tH2WxrEl">[<a href="#ref-tH2WxrEl" role="doc-biblioref">258</a>]</span>.
Phosphopeptide enrichment is typically done after any isobaric labeling strategy, although several have investigated the importance of order at these stages.</p>
<p>Phosphopeptide enrichment often uses titanium dioxide (TiO2) <span class="citation" data-cites="CkrjTtPg">[<a href="#ref-CkrjTtPg" role="doc-biblioref">261</a>]</span> and/or IMAC such as Fe3+ coupled to solid-phase materials <span class="citation" data-cites="GINSjn64 cXqDwSab 11CslpEEW">[<a href="#ref-cXqDwSab" role="doc-biblioref">254</a>,<a href="#ref-GINSjn64" role="doc-biblioref">257</a>,<a href="#ref-11CslpEEW" role="doc-biblioref">262</a>]</span>.
Often organic acids such as glutamic acid, lactic acid, glycolic acid is added to compete with acidic non-phosphopeptides for binding to the metal-ions.
Carr and coworkers even demonstrated phosphoproteome analysis without any enrichment <span class="citation" data-cites="SLLSf3H4">[<a href="#ref-SLLSf3H4" role="doc-biblioref">263</a>]</span>.</p>
<p>The use of Fe-IMAC column chromatography allows for the improved phosphopeptide from complex peptide mixtures <span class="citation" data-cites="Bij8Oa4F">[<a href="#ref-Bij8Oa4F" role="doc-biblioref">264</a>]</span>.
Compared to other formats like StageTips or batch incubations with TiO2 or Ti-IMAC beads, Fe-IMAC columns have do not suffer from problems with poor binding or elution of phosphopeptides, and the efficiency of enrichment increases linearly with the amount of starting material <span class="citation" data-cites="15ptCbE4S">[<a href="#ref-15ptCbE4S" role="doc-biblioref">265</a>]</span>.</p>
<p>Multiple IMAC steps can be used in parallel or sequentially to improve phosphopeptide coverage.
Lai et al., (2012) showed that the combined use of Fe3+-IMAC and Ti(4+)-IMAC chromatography enables complementary identification of more phosphorylation sites than either technique alone <span class="citation" data-cites="fPTYjwxV">[<a href="#ref-fPTYjwxV" role="doc-biblioref">266</a>]</span>.
A novel phosphopeptide enrichment technique using sequential enrichment with magnetic Fe3O4 and TiO2 particles was developed to detect mono- and multi-phosphorylated peptides <span class="citation" data-cites="Hii2H7k2">[<a href="#ref-Hii2H7k2" role="doc-biblioref">267</a>]</span>.</p>
<h4 id="general-protocol-for-phosphoproteomics">General protocol for phosphoproteomics:</h4>
<p>• Collect cell line or tissue samples.<br />
• For cell samples, suspend the cell pellet in 2% SDS lysis buffer, heat at 90°C for 5 min, and sonicate.<br />
• For tissue samples, homogenize in liquid nitrogen and add to 4% SDS lysis buffer, then heat at 90°C for 5 min and sonicate.<br />
• Centrifuge the samples at 3,000 rcf to remove insoluble material and remove supernatant to a new tube.<br />
• Add 5 mM Tris (2-carboxyethyl) phosphine (TCEP) and 10 mM chloroacetamide to the samples for alkylation and reduction in the dark at room temperature for 30 minutes.<br />
• Precipitate the protein with acetone to remove SDS.<br />
• Dissolve the protein pellet in 8M Urea with 50 mM TEAB.<br />
• Quantify the protein using a BCA assay.<br />
• Perform proteolysis with trypsin using 1:100 trypsin:substrate (wt:wt). For example, if you have 1 mg of protein, add 10 ug of trypsin.<br />
• Desalt the resulting peptides using Sep-Pak C18 columns or stage tips.<br />
• Subject the fractions to phosphopeptide enrichment using TiO2/ Fe‐IMAC beads.<br />
• Desalt the peptides again using C18 stage tips.<br />
• Perform LC-MS/MS.</p>
<h4 id="tips-for-studying-phosphorylation">Tips for studying phosphorylation:</h4>
<p>• Cell lysates should always be prepared using phosphatase inhibitors and samples should be placed on the ice during sonication for protein extraction.<br />
• Increase the amount of starting material of your sample for phosphoenrichment to at least 1 mg of protein or more for optimal results.<br />
• If using anti-phosphorylation antibodies, ensure their specificity is confirmed with other methods.<br />
• Make sure to select a suitable method for the phosphoenrichment that fits the experiment goals.<br />
• TiO2-based phosphopeptide enrichment methods have different enrichment specificities; selecting non-phosphopeptide excluders such as glutamic acid, lactic acid, glycolic acid, and dihydroxybenzoic acid are the key part of the study <span class="citation" data-cites="G33wPuEv">[<a href="#ref-G33wPuEv" role="doc-biblioref">268</a>]</span>.<br />
• Do not use milk as a blocking agent when western blotting for phosphorylation because milk contains the phosphoprotein casein and can lead to a higher background due to non-specific binding.</p>
<h3 id="antibody-enrichments-of-modifications">Antibody enrichments of modifications</h3>
<p>Western blot analysis is used to detect the PTMs in a protein by using antibodies <span class="citation" data-cites="LeABskLB">[<a href="#ref-LeABskLB" role="doc-biblioref">269</a>]</span>.
As an extension of this, pan-PTM antibodies have been used to isolate peptides bearing the PTM of interest <span class="citation" data-cites="xEkij3xI">[<a href="#ref-xEkij3xI" role="doc-biblioref">270</a>]</span>.
One benefit of this approach is that peptides are less likely to experience non-specific binding than proteins <span class="citation" data-cites="gPKAjoqC">[<a href="#ref-gPKAjoqC" role="doc-biblioref">246</a>]</span>.
Initially peptide immunoaffinity precipitation was developed to enrich for phosphotyrosine-containing peptides.
This protocol was initially designed to enrich for phosphotyrosine-containing peptides <span class="citation" data-cites="oN0RedlD">[<a href="#ref-oN0RedlD" role="doc-biblioref">271</a>]</span>.
Peptide immunoprecipitation yielded significantly greater coverage of the phosphotyrosine proteome than global phosphorylation enrichment strategies by enriching for a subset of the phosphoproteome.<br />
Since then, peptide immunoaffinity precipitation has been used successfully to enrich for peptides with other phosphorylation motifs <span class="citation" data-cites="11yMtXtzL nEmRnHBE">[<a href="#ref-11yMtXtzL" role="doc-biblioref">272</a>,<a href="#ref-nEmRnHBE" role="doc-biblioref">273</a>]</span> as well as peptides with other modifications such as the diglycyl-lysine residue of ubiquitin modification after trypsin proteolysis <span class="citation" data-cites="fZ6aM0wq JMsurOWF 11FwNutcB">[<a href="#ref-fZ6aM0wq" role="doc-biblioref">274</a>,<a href="#ref-JMsurOWF" role="doc-biblioref">275</a>,<a href="#ref-11FwNutcB" role="doc-biblioref">276</a>]</span>, acetyl-lysine <span class="citation" data-cites="tWUlvotu 36Wx4WMF bAU4w5zL RMueHUu6 58knZUn0">[<a href="#ref-tWUlvotu" role="doc-biblioref">277</a>,<a href="#ref-36Wx4WMF" role="doc-biblioref">278</a>,<a href="#ref-bAU4w5zL" role="doc-biblioref">279</a>,<a href="#ref-RMueHUu6" role="doc-biblioref">280</a>,<a href="#ref-58knZUn0" role="doc-biblioref">281</a>]</span>, arginine methylation <span class="citation" data-cites="I7T7nB5">[<a href="#ref-I7T7nB5" role="doc-biblioref">282</a>]</span>, tyrosine nitration <span class="citation" data-cites="82ISyiE">[<a href="#ref-82ISyiE" role="doc-biblioref">283</a>]</span>, and tyrosine phosphorylation <span class="citation" data-cites="1CF6MyQQ3 PgRmFpK6">[<a href="#ref-1CF6MyQQ3" role="doc-biblioref">284</a>,<a href="#ref-PgRmFpK6" role="doc-biblioref">285</a>]</span>.</p>
<p>The O-linked β-D-N-acetylglucosamine (O-GlcNac) is found on serine and threonine residues of nucleocytoplasmic proteins a one of PTMs is involved in involved in the occurrence and progression of cancers in multiple systems throughout the body <span class="citation" data-cites="hAQepsrJ">[<a href="#ref-hAQepsrJ" role="doc-biblioref">286</a>]</span>.
Anti-O-GlcNAc monoclonal antibody enables enrichment from O-GlcNAcylated peptides of cells and tissues.
These antibodies have high sensitivity and specificity toward O-GlcNAc-modified peptides and do not identify O-GalNAc or GlcNAc in extended glycans <span class="citation" data-cites="UV2xOP8u">[<a href="#ref-UV2xOP8u" role="doc-biblioref">287</a>]</span>.</p>
<h3 id="abundant-protein-depletion-blood-samples">Abundant protein depletion (Blood samples)</h3>
<p>The range abundances of proteins in the blood/plasma proteome exceeds 10 orders of magnitude.
Due to this wide dynamic range, detection of proteins with medium and low abundance by proteomic analyses is difficult <span class="citation" data-cites="xwhmqv6G">[<a href="#ref-xwhmqv6G" role="doc-biblioref">288</a>]</span>.
Identifying protein biomarkers from biological samples such as blood is often obstructed by proteins present at higher concentrations.
The removal of these high abundant proteins enables the detection of less abundant and unique proteins.
The ability to deplete abundant proteins with specificity, reproducibility, and selectivity is extremely important in proteomic studies <span class="citation" data-cites="MZe4do4v">[<a href="#ref-MZe4do4v" role="doc-biblioref">289</a>]</span>.</p>
<p>The following are some of the methods used for abundant protein depletion:</p>
<h4 id="dye-ligand-depletion">Dye-ligand depletion:</h4>
<p>This method is used for the depletion of serum albumin based on the interaction between albumin and dyes like Cibacron Blue (CB) through electrostatic force, hydrogen bonding and hydrophobic interactions.
The method is relatively low cost, widely available, robust and has high binding capacity.
However, it lacks specificity and has varying efficiency <span class="citation" data-cites="CkoUIvms mxcAPQRD">[<a href="#ref-CkoUIvms" role="doc-biblioref">290</a>,<a href="#ref-mxcAPQRD" role="doc-biblioref">291</a>]</span>.</p>
<h4 id="protein-ligand-depletion">Protein-ligand depletion:</h4>
<p>This method is used for depletion of immunoglobulins (Ig) based on the interaction between the Fragment crystallizable (Fc) region of these Igs <span class="citation" data-cites="LonwNp1">[<a href="#ref-LonwNp1" role="doc-biblioref">292</a>]</span> and cell wall protein A, G or A/G of Staphylococcus aureus and Streptococcus spp <span class="citation" data-cites="23zI8SGX LRMcEp9I">[<a href="#ref-23zI8SGX" role="doc-biblioref">293</a>,<a href="#ref-LRMcEp9I" role="doc-biblioref">294</a>]</span>.
It is highly selective and has high yield and purity.
However, non-specific binding may occur due to co-absorption of other proteins <span class="citation" data-cites="gxra2RsF">[<a href="#ref-gxra2RsF" role="doc-biblioref">295</a>]</span>.</p>
<h4 id="immunodepletion">Immunodepletion:</h4>
<p>This method is used for depletion of proteins having high abundance in plasma or serum on the basis of the specific interaction of these proteins with their respective antibodies (antigen-antibody interaction) <span class="citation" data-cites="13El9kECw">[<a href="#ref-13El9kECw" role="doc-biblioref">296</a>]</span>.
It has high specificity and commercial kits are also readily available, but it is expensive, has limited sample loading and can result in non-specific binding <span class="citation" data-cites="gxra2RsF">[<a href="#ref-gxra2RsF" role="doc-biblioref">295</a>]</span>.</p>
<h4 id="combinatorial-peptide-ligand-library">Combinatorial peptide ligand library:</h4>
<p>This method is used for partial depletion of major proteins i.e., those with high abundance and for relative enrichment of lower and medium abundant proteins <span class="citation" data-cites="xeb6grTF">[<a href="#ref-xeb6grTF" role="doc-biblioref">297</a>]</span>.
It is based on the interaction with an array of ligands which are essentially peptides of 6 amino acids in length.
It is also used for normalization of the global protein abundance <span class="citation" data-cites="1AFK16JwO">[<a href="#ref-1AFK16JwO" role="doc-biblioref">298</a>]</span>.
However, the drawbacks include non-specific binding as well as loss of proteins due to incomplete elution or inefficient binding <span class="citation" data-cites="gxra2RsF">[<a href="#ref-gxra2RsF" role="doc-biblioref">295</a>]</span>.</p>
<h4 id="precipitation">Precipitation:</h4>
<p>This method of abundant protein depletion works by altering the solubility of proteins using a chemical reagent including inorganic salt solution <span class="citation" data-cites="hcs5DCXe">[<a href="#ref-hcs5DCXe" role="doc-biblioref">299</a>]</span>, organic solvents <span class="citation" data-cites="jOj0DZcZ">[<a href="#ref-jOj0DZcZ" role="doc-biblioref">300</a>]</span>, non-ionic polymer <span class="citation" data-cites="1DUeuj8TB">[<a href="#ref-1DUeuj8TB" role="doc-biblioref">301</a>]</span> and reducing agents <span class="citation" data-cites="jv9v3twJ">[<a href="#ref-jv9v3twJ" role="doc-biblioref">302</a>]</span>.
It is extremely simple and cost-effective.
However, it is less specific with a risk of protein loss, difficulty in protein resolubilization as well as time consuming <span class="citation" data-cites="gxra2RsF">[<a href="#ref-gxra2RsF" role="doc-biblioref">295</a>]</span>.</p>
<h4 id="new-technologies">New technologies:</h4>
<p>Newer methods of highly abundant protein depletion are based on the interaction between polymers such as bacterial cellulose nanofibers <span class="citation" data-cites="a6kEMboa">[<a href="#ref-a6kEMboa" role="doc-biblioref">303</a>]</span>, cryogels <span class="citation" data-cites="WwW8brVC 1KROnN78">[<a href="#ref-WwW8brVC" role="doc-biblioref">304</a>,<a href="#ref-1KROnN78" role="doc-biblioref">305</a>]</span> and nanomaterials <span class="citation" data-cites="7cVO0ON7">[<a href="#ref-7cVO0ON7" role="doc-biblioref">306</a>]</span>.<br />
These techniques are highly specific, relatively cheap, and very stable.
They can also be reused since they have larger binding capacity and less cross-reactivity <span class="citation" data-cites="gxra2RsF">[<a href="#ref-gxra2RsF" role="doc-biblioref">295</a>]</span>.</p>
<h2 class="page_break_before" id="peptide-purification-and-fractionation">Peptide Purification and Fractionation</h2>
<h3 id="peptide-purification-methods">Peptide purification methods</h3>
<h4 id="solid-phase-extraction-spe">Solid phase extraction (SPE)</h4>
<p>Solid phase extraction (SPE) is a common MS-based proteomics technique employed in the sample preparation.
In this method, compound isolation is based on chemical and physical properties, which determines the distribution of compounds between a mobile phase (liquid) and a stationary phase (solid).
After the molecules bind, washing of the bound compounds is performed and then molecules are made to elute from the stationary phase after replacing the mobile phase with the elution buffer.
The material used for SPE is usually discarded after every sample and no gradient is applied for elution (single-step procedure of elution) <span class="citation" data-cites="nCqm9thz">[<a href="#ref-nCqm9thz" role="doc-biblioref">307</a>]</span>.
Thus, using SPE only a specific analyte group gets separated, which depends on the stationary phase.
Hence, SPE is primarily used for sample clean-up and for reducing complexity of the sample.
For MS-based proteomic analysis, it is largely used to get rid of salts and other contaminants that might lead to ion suppression.
The major drawback of this technique is that with SPE only a small fraction of the sample is examined because not all compounds are captured, but only those with binding capabilities same as that of the sorbent.
The material for SPE is available in various types, including (micro-) columns, cartridges, plates, micropipette tips, and functionalized magnetic beads (MBs) <span class="citation" data-cites="167ADUQ9x klXQ49fD">[<a href="#ref-167ADUQ9x" role="doc-biblioref">308</a>,<a href="#ref-klXQ49fD" role="doc-biblioref">309</a>]</span>.
Reversed-phase is the most widely used material for SPE in proteomic studies for the proteins and peptide fractionation and rarely, ion-exchange material.
For the separation of glycosylated proteins and peptides, the preferred material is normal phase such as HILIC <span class="citation" data-cites="18rJRCf42 kPeMPVtT">[<a href="#ref-18rJRCf42" role="doc-biblioref">310</a>,<a href="#ref-kPeMPVtT" role="doc-biblioref">311</a>]</span>.
SPE materials which are less commonly used are silica- or polystyrene-based ones <span class="citation" data-cites="16XWFxoc3 us64YkZ0">[<a href="#ref-16XWFxoc3" role="doc-biblioref">312</a>,<a href="#ref-us64YkZ0" role="doc-biblioref">313</a>]</span>.
The other types of SPE methods are IEX, metal chelation, and affinity-based <span class="citation" data-cites="nPfahhRa">[<a href="#ref-nPfahhRa" role="doc-biblioref">314</a>]</span>.</p>
<p>The basic idea behind the choice of binding and wash versus elution solutions for SPE is that that the binding and wash solutions should favor the interaction between the analytes of interest and the solid phase, whereas the elution solution should favor the interaction of the analyte with the liquid phase (<strong>Figure 3</strong>).
For example, with reversed phase SPE, the solid phase is C18 or some other hydrophic chemistry.
Binding of peptides to this solid phase is based on the hydrophobicity of peptides, mostly due to their peptide backbone, but also due to the presence of amino acid side chains like leucine and phenylalanine.
To encourage peptides to ‘like’ the stationary phase more than the liquid phase, the peptides are loaded in aqueous solution.
This will enable washing of the hydrophilic contaminants like salts, small polar buffer molecules, and polar denaturants like urea.
After washing the bound peptides, they can be eluted by switching the liquid phase to something hydrophobic, which allows the peptides to partition more into the liquid phase and elute from the solid phase.</p>
<div id="fig:SPE" class="fignos">
<figure>
<img src="images/SPE_v03.png" style="width:100.0%" data-tag="3" alt="Figure 3: Solid phase extraction (SPE) steps. SPE is a sample preparation technique that uses a solid adsorbent contained most commonly in a cartridge device to selectively adsorb certain molecules from solution. The first step is the conditioning of the cartridge which involves wetting the adsorbent to solvate its functional groups and filling the void spaces with solvent thereby removing any air in the column. This is necessary to produce a suitable environment for adsorption and thus ensure reproducible interaction with the analytes. After conditioning, the sample is loaded in the cartridge. This can be performed with the aid of positive or negative pressure to ensure a constant flow rate. In this step molecules bind the adsorbent and interferences pass through. Next, the column is washed with the mobile phase to eliminate the contaminants while ensuring the analyte remains bound. Finally, peptides are eluted in an appropriate buffer solution with polarity or charge that competes with interaction with the solid phase." />
<figcaption aria-hidden="true"><span>Figure 3:</span> <strong>Solid phase extraction (SPE) steps.</strong>
SPE is a sample preparation technique that uses a solid adsorbent contained most commonly in a cartridge device to selectively adsorb certain molecules from solution.
The first step is the conditioning of the cartridge which involves wetting the adsorbent to solvate its functional groups and filling the void spaces with solvent thereby removing any air in the column.
This is necessary to produce a suitable environment for adsorption and thus ensure reproducible interaction with the analytes.
After conditioning, the sample is loaded in the cartridge.
This can be performed with the aid of positive or negative pressure to ensure a constant flow rate.
In this step molecules bind the adsorbent and interferences pass through.
Next, the column is washed with the mobile phase to eliminate the contaminants while ensuring the analyte remains bound.
Finally, peptides are eluted in an appropriate buffer solution with polarity or charge that competes with interaction with the solid phase.</figcaption>
</figure>
</div>
<h4 id="specific-types-of-peptide-purification">Specific Types of peptide purification</h4>
<p>There are many additional peptide purification methods that are commonly used in proteomics currently.
These methods include the following:</p>
<ol type="1">
<li>StageTips, in-stagetip (iST) <span class="citation" data-cites="TY5C1ch7 j0ZcPOCn">[<a href="#ref-TY5C1ch7" role="doc-biblioref">315</a>,<a href="#ref-j0ZcPOCn" role="doc-biblioref">316</a>]</span></li>
<li>SP2 or SP3 <span class="citation" data-cites="SWsK5zPt">[<a href="#ref-SWsK5zPt" role="doc-biblioref">317</a>]</span></li>
<li>Suspension trapping (S-trap) <span class="citation" data-cites="1FU2NfVYY">[<a href="#ref-1FU2NfVYY" role="doc-biblioref">44</a>]</span></li>
</ol>
<h3 id="peptide-fractionation-methods">Peptide fractionation methods</h3>
<p>The number of peptides produced from proteolysis of the whole proteome is immense.
Thus, after peptides are cleaned from interferences, they are often fractionated into subsets to enable increased proteome coverage.
The characterization of the whole proteome is expected from higher order organisms, and with rising interest in post-translational modifications, an elaborate coverage of protein sequence is required.
There are different methods for peptide fractionation as follows:</p>
<h4 id="ion-exchange-chromatography-iec">Ion-exchange chromatography (IEC)</h4>
<p>This method involves the separation based on contrasting electric charge <span class="citation" data-cites="ncGFPcxC">[<a href="#ref-ncGFPcxC" role="doc-biblioref">318</a>]</span>.
In this approach, the mechanism of analyte retention is based on the principle of electrostatic attraction between the sample and the stationary phase functional groups (FGs), having opposite charges.
IEC is classified into two types: cation-exchange and anion-exchange chromatography.
In cation-exchange chromatography, at an acidic pH, the negatively charged functional groups such as sulfates are attracted to positively charged peptides, whereas, in anion-exchange chromatography, positively charged FGs such as quaternary ammoniums are attracted to peptides with negative charge at an alkaline pH.
These techniques are further classified into: strong (cation [SCX] and anion [SAX] exchange), and weak exchangers (cation [WCX] and anion [WAX] exchange), based on the type of FG attached <span class="citation" data-cites="Bm9Z2DVx">[<a href="#ref-Bm9Z2DVx" role="doc-biblioref">319</a>]</span>.
These functional groups are most commonly supported in resins made up of silica and synthetic polymers, however, some inorganic materials are sometimes used <span class="citation" data-cites="ncGFPcxC">[<a href="#ref-ncGFPcxC" role="doc-biblioref">318</a>]</span>.
In the IEC method, peptide elution is performed using a mobile phase with higher ionic strength, to ensure peptide partition into the liquid phase. SCX along with a salt gradient/plug is a routinely used proteomics technique.
In the SCX method, peptides are resolved according to their net charge, in which the peptide with the lowest positive charge is eluted first.
Increasing the salt concentration decreases the peptide retention time due to competition with the electrostatic interactions between the peptides and the solid phase.
However, SCX resolution is limited compared to reversed phase chromatography and will thus limit the suitability of this technique for complex mixtures <span class="citation" data-cites="L1FaRtpm">[<a href="#ref-L1FaRtpm" role="doc-biblioref">320</a>]</span>.</p>
<h4 id="reversed-phase-chromatography-rplc">Reversed-phase chromatography (RPLC)</h4>
<p>Reversed-phase chromatography is the most widely commonly used chromatographic technique which separates molecules in solution having neutral pH based on their hydrophobicity.
The separation occurs on the basis of the partition coefficient of analytes between the mobile phase and the hydrophobic stationary phase.
Highly polar peptides elute before the ones having less polarity because of the strong interaction with the hydrophobic functional groups forming a layer similar to a liquid around the silica resin <span class="citation" data-cites="GjvtcEz1">[<a href="#ref-GjvtcEz1" role="doc-biblioref">321</a>]</span>.
RPLC has been widely used in separation of peptides because of its compatibility with gradient elution and aqueous samples and its retention mechanism, which modulates separation owing to changes in the properties like pH, additives and organic modifier <span class="citation" data-cites="1FrVJlffz">[<a href="#ref-1FrVJlffz" role="doc-biblioref">322</a>]</span>.
Numerous factors influence the capacity of chromatographic peaks, such as temperature, column length, stationary phase, particle size, mobile-phase ion-pairing reagent, mobile-phase modifier and gradient slope<span class="citation" data-cites="1HPaIsist">[<a href="#ref-1HPaIsist" role="doc-biblioref">323</a>]</span>.
Usually online RPLC is done at acidic pH to ensure peptide ionization, but it can be paired with offline high pH RPLC and multiple fraction concatenation to produce orthogonal separation due to altered ionization of amino acids changing peptide hydrophobicity <span class="citation" data-cites="190oCO8Ki">[<a href="#ref-190oCO8Ki" role="doc-biblioref">324</a>]</span>.</p>
<h4 id="hydrophilic-interaction-liquid-chromatography-hilic">Hydrophilic interaction liquid chromatography (HILIC)</h4>
<p>HILIC is similar in its principle to normal-phase chromatography.
It is used for the separation of hydrophilic peptides and polar analytes <span class="citation" data-cites="16k9ga0WM">[<a href="#ref-16k9ga0WM" role="doc-biblioref">325</a>]</span>.
This separation is achieved by a stationary phase that is hydrophilic in nature, for example: cyano-, diol-, amino- bonded phases <span class="citation" data-cites="nrDJzBh3">[<a href="#ref-nrDJzBh3" role="doc-biblioref">326</a>]</span>, and an organic and hydrophobic mobile phase <span class="citation" data-cites="g9BMjmYA">[<a href="#ref-g9BMjmYA" role="doc-biblioref">327</a>]</span>.
The elution of bonded peptides occurs by increasing the mobile phase polarity in a reversed elution order as compared to RPLC <span class="citation" data-cites="g9BMjmYA KqYbI8ep">[<a href="#ref-g9BMjmYA" role="doc-biblioref">327</a>,<a href="#ref-KqYbI8ep" role="doc-biblioref">328</a>]</span>.
Thus, the peptides with less polarity elute before the more polar peptides.
HILIC can also be used for enrichment and targeted proteomic analysis of PTMs, such as glycosylation, N-acetylation and phosphorylation, which increase the polarity of peptides and therefore also their retention on HILIC <span class="citation" data-cites="GjvtcEz1">[<a href="#ref-GjvtcEz1" role="doc-biblioref">321</a>]</span>.</p>
<h4 id="isoelectric-focusing-ief">Isoelectric focusing (IEF)</h4>
<p>IEF is a type of high-resolution (HR) technique of electrophoresis used for the separation as well as concentration of peptides that are amphoteric in nature on the basis of their isoelectric point (pI) using a solution without buffer consisting of either carrier ampholytes or a gel with immobilized pH gradient (IPG).
After IEF separation, the separated amphoteric peptides in the liquid phase are recovered for further analysis by RPLC-MS/MS <span class="citation" data-cites="Nk7uUlzO">[<a href="#ref-Nk7uUlzO" role="doc-biblioref">329</a>]</span>.
Along with being a technique with improved resolution and capacity, for separation of peptides, IEF provides with additional information on physicochemical properties of the peptides, for example: peptide iso electric point (pI) which acts as a tool for validation and filtration for identifying MS/MS peptide sequence during the step of database search <span class="citation" data-cites="Qf7HZAh">[<a href="#ref-Qf7HZAh" role="doc-biblioref">330</a>]</span>.
The IEF system is not only used for increasing the coverage of proteome but also in quantitative label-free <span class="citation" data-cites="LSDxad5T">[<a href="#ref-LSDxad5T" role="doc-biblioref">331</a>]</span> and stable isobaric labeling experiments <span class="citation" data-cites="Qf7HZAh">[<a href="#ref-Qf7HZAh" role="doc-biblioref">330</a>]</span>.
IEF and gel-based separations have fallen out of favor in the last decade due to improvements in liquid chromatography.</p>
<h4 id="electrostatic-repulsion-hydrophilic-interaction-chromatography-erlic">Electrostatic repulsion-hydrophilic interaction chromatography (ERLIC)</h4>
<p>ERLIC is a method based on use of a weak anion exchange column operated at low pH with high organic solvent enabling isocratic elution <span class="citation" data-cites="Kf2BNqap">[<a href="#ref-Kf2BNqap" role="doc-biblioref">332</a>]</span>.
Acidic peptides are retained by electrostatic interaction, basic and neutral peptides are retained through hydrophilic interaction made favorable by high organic solvent.
This improves retention of acidic peptides and reduces retention of basic peptides compared to normal HILIC <span class="citation" data-cites="1HJnUznNp">[<a href="#ref-1HJnUznNp" role="doc-biblioref">333</a>]</span>.</p>
<h2 class="page_break_before" id="types-of-mass-spectrometers-used-for-proteomics">Types of Mass Spectrometers used for Proteomics</h2>
<h3 id="mass-spectrometry">Mass spectrometry</h3>
<p>Mass spectrometry is a science of ions; it serves as a sophisticated instrument for determining the masses of compounds and elements.
It can be likened to an ultra-precise weigh scale that can differentiate mass variations down to a single electron, or even lower.
Since J.J. Thomson’s initial exploration in 1912, the field of mass spectrometry has undergone numerous improvements, spanning from isotope assessment to the interpretation of biomacromolecules <span class="citation" data-cites="VhI3IC9H">[<a href="#ref-VhI3IC9H" role="doc-biblioref">334</a>]</span>, all thanks to the combined efforts of diverse fields like chemistry, physics, electronic engineering, and computer science.
Nowadays, with the rapid improvement of sensitivity, mass resolution, tandem mass spectrometry methods and dissociation methods, mass spectrometers have evolved as a core tool for proteomics (and metabolomic) analysis.
It is precisely the widespread application of mass spectrometry in proteomics analysis that has given rise to more instrument manufacturers and a greater diversity of mass spectrometer types.
This also brings a happy annoyance to many beginners or researchers in other fields who have no background in mass spectrometry: which manufacturer and which type of mass spectrometry should I choose to analyze my samples?
Here, to help new learners build a basic understanding faster, we will briefly introduce some basic concepts, common types of mass spectrometers, and their suitable application scenarios.</p>
<h3 id="mass-spectrometer-structure-and-basic-principles">Mass Spectrometer Structure and Basic Principles</h3>
<p>The fundamental principle of mass spectrometry revolves around specific physical processes that can be described by various mathematical formulas.
Since this article serves as a guide for those new to the field, particularly those from a biology background, we’ve chosen to steer clear of delving too deeply into intricate mathematical and physical explanations.
However, for those keen on a deeper understanding, we’ve included references pertaining to these foundational principles.
In this piece, our focus lies on introducing fundamental concepts and outlining the typical workflow in mass spectrometry.</p>
<p>The process of mass spectrometry (MS) is to generate gas phase ions from compounds in samples by any suitable method, to separate these ions by their mass-to-charge ratio (m/z), and then detect them by their respective m/z and abundance.
The successful implementation and demonstration of this process requires participation of five fundamental systems (Figure X):</p>
<h4 id="the-ion-source.">1) The ion source.</h4>
<p>This is where gas phase ions are generated.
Common methods of ionization in mass spectrometry includes electrospray ionization (ESI), matrix assisted laser desorption ionization (MALDI), atmospheric pressure chemical ionization (APCI), electron ionization (EI) and chemical ionization (CI) <span class="citation" data-cites="lKt4oMBO 1B2HdTk7M">[<a href="#ref-lKt4oMBO" role="doc-biblioref">335</a>,<a href="#ref-1B2HdTk7M" role="doc-biblioref">336</a>]</span>.
For proteomic analysis, soft ionization methods such as ESI and MALDI are the most widely applied techniques <span class="citation" data-cites="4Ajjx5fo 15Ye4Za7Y">[<a href="#ref-4Ajjx5fo" role="doc-biblioref">1</a>,<a href="#ref-15Ye4Za7Y" role="doc-biblioref">2</a>]</span>, which will be discussed in next chapter and no more redundance here.</p>
<h4 id="the-mass-analyzer.">2) The mass analyzer.</h4>
<p>This is where gas phase ions are separated according to their mass-to-charge ratio (m/z) based on physics principles.
There are several types of mass analyzers applied in mass spectrometry, such as time-of-flight (TOF), quadrupole, ion trap, orbitrap, Fourier transform-ion cyclotron resonance (FT-ICR), and magnetic sector analyzers <span class="citation" data-cites="18qirHBwQ 5vv7ID3o">[<a href="#ref-18qirHBwQ" role="doc-biblioref">337</a>,<a href="#ref-5vv7ID3o" role="doc-biblioref">338</a>]</span>, each with unique advantages and applications (Table 10.1).
For proteomic analysis, tandem mass spectrometry, which involves combining two or more mass analyzers, is typically used to achieve precursor selection, structural analysis, improved sensitivity, and better mass resolution <span class="citation" data-cites="MooVAdBf">[<a href="#ref-MooVAdBf" role="doc-biblioref">339</a>]</span>.
Mass analyzer is the core component of a mass spectrometer, it is also the most important factor that we need to take into consideration when choosing a mass spectrometer for a specific project.</p>
<h4 id="the-detector.">3) The detector.</h4>
<p>This is where separated ions are detected and their respective m/z values and abundances are recorded, generating a mass spectrum.
Common types of ion detectors, including the Electron Multiplier (EM), Photomultiplier Tube (PMT), Microchannel Plate (MP) and Faraday Cup (FC), along with their strengths and limitations, are illustrated in Table 10.2.
It is worth noting that Orbitrap and FT-ICR mass analyzers don’t use conventional detectors as listed above.
Instead, they detect ions in a fundamentally different way by detecting image current produced by oscillating ions <span class="citation" data-cites="Uaum1m63 svXqfJ4D erGVtl2k">[<a href="#ref-Uaum1m63" role="doc-biblioref">340</a>,<a href="#ref-svXqfJ4D" role="doc-biblioref">341</a>,<a href="#ref-erGVtl2k" role="doc-biblioref">342</a>]</span>.
In both mass analyzers, the detector is essentially measuring an electrical current (or more accurately, a voltage that’s proportional to the current) that’s induced by the motion of the ions.
This signal is then processed to extract the frequencies of oscillation and fourier-transformed into a mass spectrum, which is quite different from other types of detectors that count individual ions or particles striking a surface.</p>
<h4 id="the-vacuum-system.">4) The vacuum system.</h4>
<p>This is designed to maintain a vacuum environment for ions’ movement inside the instrument, consists of different type of pumps including rough vacuum pumps (rotary vane pumps, scroll pumps) and high vacuum pumps (turbo molecular pumps, diffusion pumps).
The reason for maintaining a high vacuum is to reduce the collision between ions and gas molecules during their travel, which can affect their trajectory.
FT-ICR and Orbitrap instruments usually require higher vacuum in the 10-9 to 10-11 Torr level, while TOFs require medium vacuum in the 10-7 to 10-8 Torr range, and Quadrupole and Ion trap require a relative lower vacuum in the 10-5 to 10-6 Torr range.</p>
<h4 id="the-control-system.">5) The control system.</h4>
<p>This is needed to regulate and coordinate the various parts of the mass spectrometer to ensure seamless functioning.
This typically includes ion source control, mass analyzer control, detector control, data acquisition control, interfacing with auxiliary systems (such as a liquid chromatograph and gas chromatograph), and modules for instrument diagnostics and calibration.</p>
<div id="fig:MS-diagram" class="fignos">
<figure>
<img src="images/MS_f101.svg" style="width:100.0%" data-tag="4" alt="Figure 4: Diagram of typical mass spectrometer modules. Systems must have an ion source, mass analyzer, detector, vacuum system, and control system." />
<figcaption aria-hidden="true"><span>Figure 4:</span> <strong>Diagram of typical mass spectrometer modules.</strong>
Systems must have an ion source, mass analyzer, detector, vacuum system, and control system.</figcaption>
</figure>
</div>
<p>Table 10-1 Common mass analyzers.</p>
<table>
<colgroup>
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 32%" />
<col style="width: 35%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Type</th>
<th style="text-align: center;">Acronym</th>
<th style="text-align: center;">Principle</th>
<th style="text-align: center;">Characteristics</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Time-of-flight</td>
<td style="text-align: center;">TOF</td>
<td style="text-align: center;">Time dispersion of a pulsed ion beam; separation by the time it takes for ions to travel a fixed distance</td>
<td style="text-align: center;">High-speed analysis, large mass range and good sensitivity. Suited for fast data acquisition and high-throughput applications. Modern TOF systems usually can achieve mass resolution well over 10,000 (m/Δm) or even higher.</td>
</tr>
<tr class="even">
<td style="text-align: center;">Linear quadrupole</td>
<td style="text-align: center;">Q</td>
<td style="text-align: center;">Continuous ion beam in linear radio frequency quadrupole field; separation due to instability of ion trajectories</td>
<td style="text-align: center;">High transmission efficiency, simple design, good sensitivity, and tunable mass range; relatively low mass resolution ranges from several hundreds to a thousand; often used in tandem mass spectrometry (MS/MS) experiments</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Quadrupole ion trap</td>
<td style="text-align: center;">QIT</td>
<td style="text-align: center;">Traps ions by electromagnetic fields; separation in three-dimensional radio frequency quadrupole field by resonant excitation</td>
<td style="text-align: center;">Efficient for fragmenting ions and structural elucidation, higher sensitivity, and relatively compact which good for benchtop instruments. Relatively a low mass resolution around 1,000 - 3,000.</td>
</tr>
<tr class="even">
<td style="text-align: center;">Fourier transform-ion cyclotron resonance</td>
<td style="text-align: center;">FT-ICR</td>
<td style="text-align: center;">Traps ions in a strong magnetic field by Lorentz force; separation by cyclotron frequency, image current detection and Fourier transformation of transient signal</td>
<td style="text-align: center;">Ultimate high mass resolution (up to 1,000,000), making it ideal for elemental and isotopic analysis. Large size, low speed, and expensive in terms of both initial purchase cost and ongoing operation and maintenance costs.</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Orbitrap</td>
<td style="text-align: center;">Orbitrap</td>
<td style="text-align: center;">Axial oscillation in inhomogeneous electric field; detection of frequency after Fourier transformation of transient signal</td>
<td style="text-align: center;">Extremely high resolution and accuracy (up to 500,000), capable of resolving complex mixtures with high sensitivity. Relatively low speed, expensive in terms of both initial purchase cost and ongoing operation and maintenance costs. Need high vacuum.</td>
</tr>
</tbody>
</table>
<p>Table 10.2 Common detectors.</p>
<table>
<colgroup>
<col style="width: 25%" />
<col style="width: 27%" />
<col style="width: 47%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Type</th>
<th style="text-align: center;">Principle</th>
<th style="text-align: center;">Characteristics</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Electron Multiplier</td>
<td style="text-align: center;">Amplifies signals by utilizing a sequence of dynodes that emit secondary electrons when struck by an incident electron, creating a cascading effect. This results in an amplified output current at the final anode, proportional to the intensity of the initial signal.</td>
<td style="text-align: center;">Very good signal amplification to even one electron (may cause more noise dependent on gain), high sensitivity, need high vacuum and high voltage, expensive. Limited dynamic range, finite lifespan and need to be replaced periodically</td>
</tr>
<tr class="even">
<td style="text-align: center;">Faraday Cup</td>
<td style="text-align: center;">Charged particles, such as ions or electrons, enter the cup and transfer their charge to it, causing a change in electric potential that can be measured over time to infer the number of particles.</td>
<td style="text-align: center;">Suitable for particles and charge state detection. Simplicity and robustness, Wide dynamic range, no need for high voltage and high vacuum. Lower sensitivity. Sensitive to Secondary Emission directional sensitivity (direction of incoming particles)</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Microchannel Plate</td>
<td style="text-align: center;">Similar to electron multiplier, a two-dimensional matrix or “plate” of many tiny, parallel, hollow channels made from a type of glass that can generate secondary electron emissions upon incident particles striking the channel walls. These secondary emissions create an electron avalanche down the channels and amplifies the original signal.</td>
<td style="text-align: center;">Signal Amplification (Not as good as electron multiplier, but lower noise), Spatial Resolution ability, shorter life expectancy due to channel aging and depletion of the secondary emission material, smaller and cheaper than electron multiplier</td>
</tr>
<tr class="even">
<td style="text-align: center;">Daly Detector</td>
<td style="text-align: center;">Directing ions onto a surface (Doorknob) to trigger the emission of electrons, which are then accelerated towards a phosphor screen to produce photons, that are subsequently detected and amplified by a photomultiplier tube, thereby converting the ion signal into a measurable electrical signal.</td>
<td style="text-align: center;">High gain, ruggedness, wide dynamic range, suitable for high mass and high energy ions. Limited mass resolution, larger size and need high voltage, finite lifespan.</td>
</tr>
</tbody>
</table>
<h3 id="types-of-mass-spectrometers-used-for-proteomics.">Types of mass spectrometers used for proteomics.</h3>
<p>Typically, mass spectrometers are named based on the abbreviations of their principal or tandem mass analyzers.
This naming convention stems from the fact that the mass analyzer forms the core component of a mass spectrometer, and it also dictates key performance attributes such as mass resolution, scanning speed, sensitivity, and cycle time.
These performance metrics, in turn, determine what type of analysis we can conduct, its speed and its accuracy.
Next, we will focus on introducing several classic tandem mass spectrometry types commonly used in proteomics.</p>
<h3 id="triple-quadrupole-qqq.">1. Triple quadrupole (QQQ).</h3>
<p>Triple quadrupole mass spectrometer (often abbreviated as QqQ, QQQ, TQ, or TQMS), as suggested by the name, is a type of tandem mass spectrometer where three quadrupole mass analyzers are combined in series (Figure 10.2).
Each quadrupole is essentially a set of four parallel metal rods to which radio frequency (RF) and direct current (DC) voltages are applied to each opposing pair of rods.
The QqQ operate in a synchronized manner to isolate ions of interest (according to the Mathieu function), induce fragmentation, and then detect the resulting fragment ions.
Specifically, first quadrupole (Q1) is a mass filter, where ions of a specific mass-to-charge ratio (m/z) are selected from the incoming ion beam.
This is achieved by adjusting the voltage applied to the pair rods within the quadrupole, allowing ions with a particular m/z value to pass through while deflecting others.
The second quadrupole (Q2), also known as the collision cell, is where selected ions from Q1 are fragmented into product ions.
This fragmentation happens due to the collisions between inert gas molecules (nitrogen, argon, or helium) and ions, which causes the ions to break up (fragment) into smaller pieces (fragment ions).
For more detail about peptide fragmentation, see the Tandem Mass Spectrometry section.
This process is known as collision-induced dissociation (CID) <span class="citation" data-cites="EmIJDW71 17hxbWgXF">[<a href="#ref-EmIJDW71" role="doc-biblioref">343</a>,<a href="#ref-17hxbWgXF" role="doc-biblioref">344</a>]</span>.
The Q2 is usually only subjected to RF potential and does not filter ions; instead, it transmits the product ions to the third quadrupole.
In some tandem mass spectrometry, hexapole or octupole are also using to replace quadrupole as the collision cell.
Lastly, the third quadrupole (Q3) acts as a secondary mass filter, similar to Q1, but with the purpose of selecting specific fragment ions produced in the collision cell while excluding other ions.
The chosen ions are then directed to the detector, where their abundance is measured (Figure 10.2).
This process, involving precursor ion selection, precursor ion fragmentation, and product ion detection, is a general operating principle in tandem mass spectrometry and determines what kind of scan mode you can utilize.
A key characteristic and advantage of QqQ is the flexibility of choosing various scan modes <span class="citation" data-cites="1ENeUykUA 1GjxHekcM 17hxbWgXF">[<a href="#ref-17hxbWgXF" role="doc-biblioref">344</a>,<a href="#ref-1ENeUykUA" role="doc-biblioref">345</a>,<a href="#ref-1GjxHekcM" role="doc-biblioref">346</a>]</span>, such as the following.</p>
<h4 id="product-ion-scan">1. Product Ion Scan:</h4>
<p>Q1 is set to filter a specific precursor ion, which is then fragmented in Q2.
Q3 scans the full range of product ion masses.
This mode is usually used to identify the structure of a particular compound.</p>
<h4 id="precursor-ion-scan">2. Precursor Ion Scan:</h4>
<p>Q3 is set to filter a specific product ion.
Q1 scans the full range of precursor ions, that when fragmented in Q2, yield the selected product ion.
This mode is used to find compounds that yield a specific fragment ion, which can be particularly useful when looking for compounds with a common structural motif.</p>
<h4 id="neutral-loss-scan">3. Neutral Loss Scan:</h4>
<p>Both Q1 and Q3 scan the full range of ions, but with a mass difference equal to a specific “neutral loss”.
This mode is used to identify compounds that, when fragmented, lose a specific neutral molecule.</p>
<h4 id="multiple-reaction-monitoring-mrm">4. Multiple Reaction Monitoring (MRM):</h4>
<p>Both Q1 and Q3 are set to filter specific ions (precursor and product, respectively).
This highly selective mode is used for quantitative analysis of specific compounds, offering excellent sensitivity and specificity<span class="citation" data-cites="1B9P7OPqR 9yFTjbpI">[<a href="#ref-1B9P7OPqR" role="doc-biblioref">347</a>,<a href="#ref-9yFTjbpI" role="doc-biblioref">348</a>]</span>.</p>
<p>The triple quadrupole mass spectrometer is a highly versatile instrument, capable of both qualitative and quantitative analysis.
Enke and Yost at Michigan State University developed the first commercial triple-quadrupole mass spectrometer in the late 1970s<span class="citation" data-cites="Ddpn8u0F">[<a href="#ref-Ddpn8u0F" role="doc-biblioref">349</a>]</span>.
QqQ is particularly well-suited for targeted quantitative analysis due to its high sensitivity, selectivity, and dynamic range, which has made it a go-to instrument in areas such as drug metabolism studies, environmental monitoring, food safety analysis, pharmaceuticals, and clinical diagnostics <span class="citation" data-cites="1AOQTDs57 11skpaXK6 BaGpN1HP 1ETD5TZZ8">[<a href="#ref-1AOQTDs57" role="doc-biblioref">350</a>,<a href="#ref-11skpaXK6" role="doc-biblioref">351</a>,<a href="#ref-BaGpN1HP" role="doc-biblioref">352</a>,<a href="#ref-1ETD5TZZ8" role="doc-biblioref">353</a>]</span>.<br />
However, Quadrupole suffer from inherent limitations in mass resolution due to the constraints of principles and precision in mechanical manufacturing.
Consequently, they face difficulties in accurately identifying unknown molecules within complex mixtures and thus not appropriate for applications like structure analysis and biomarker discovery.</p>
<div id="fig:QQQ-diagram" class="fignos">
<figure>
<img src="images/MS_f102.svg" style="width:100.0%" data-tag="2" alt="Figure 2: Schematic diagram of typical QqQ system. Three quadrupoles enable precursor selection, fragmentation, and the fragment ion selection." />
<figcaption aria-hidden="true"><span>Figure 2:</span> <strong>Schematic diagram of typical QqQ system.</strong>
Three quadrupoles enable precursor selection, fragmentation, and the fragment ion selection.</figcaption>
</figure>
</div>
<h3 id="q-tof">2. Q-TOF</h3>
<p>Even though quadrupoles face difficulties in accurately identifying unknown peptides within complex mixtures due to its mass resolution, they serve effectively as mass filters, making them an excellent choice for combining with other high-resolution mass analyzers to form tandem mass spectrometry systems.
One commonly used approach is Quadrupole-Time-of-Flight Mass Spectrometer (Q-TOF-MS), a ‘hybrid’ device, integrating quadrupole techniques with a time-of-flight mass analyzer.
W.E. Stephens constructed and published the design of the first time-of-flight (TOF) analyzer in 1946 <span class="citation" data-cites="EeH1Mu0v kFLq1VDE">[<a href="#ref-EeH1Mu0v" role="doc-biblioref">354</a>,<a href="#ref-kFLq1VDE" role="doc-biblioref">355</a>]</span>.
The principle of TOF is quite straightforward: ions of different mass-to-charge ratios (m/z) are imparted with the same initial kinetic energy (E = Uq = ½ mv2) and then separated over time as they travel along a field-free drift path of known length.
If all ions begin their flight simultaneously, or at least within a short enough time span, the lighter ions will reach the detector before the heavier ones due to their faster velocity (V)<span class="citation" data-cites="15YzDfo72">[<a href="#ref-15YzDfo72" role="doc-biblioref">356</a>]</span>.
Based on this principle, the m/z of different ions can be calculated according to the order in which they reach the detector.
Similarly, we can easily conclude that the longer of the drift path, the higher of the mass resolution can reach if keep the response time of detector the same.
In fact, in pursuit of higher mass resolution, researchers have indeed built time-of-flight (TOF) drift tubes that are tens of meters long.
However, apparently, this is not practical for widely application in a regular lab place.
An alternative way to expand drift length and achieve higher resolution is to apply reflector (often called a reflectron).
The principles and advantages of using a reflector can be summarized as follows.</p>
<p>Under ideal circumstances within a TOF mass spectrometer, ions sharing the same m/z would reach the detector concurrently post-acceleration, thus generating a sharp peak on the mass spectrum.
However, the inherent oscillation path variability of ions within the mass spectrometer makes it challenging to maintain uniform initial kinetic energy amongst all ions, leading to peak broadening and a substantial reduction in mass resolution.
The reflector is designed to rectify this issue. Comprising a series of electrodes that set to different voltages, the reflector generates a retarding electric field that reverses ion trajectories back through the flight tube.
Notably, the reflector is engineered such that ions carrying lower kinetic energy delve less into the reflector and have a reduced flight path, while those with higher kinetic energy permeate more deeply and follow a longer flight path.
This equalizes the variances in initial kinetic energy, enabling ions of the same m/z to hit the detector almost simultaneously, thereby enhancing the resolution of TOF.</p>
<p>Furthermore, the usage of reflector effectively expands the flight path length within the same physical confines, resulting in superior ion separation and consequently, higher resolution.
This reflection comes at the cost of some ion loss, and therefore some sensitivity loss.
As such, reflecting TOFs are the basis of most commercial instruments currently in use.</p>
<p>The construction of a Q-TOF bears significant resemblance to a triple-quadrupole mass spectrometer, with the critical distinction that the third quadrupole has been replaced by a time-of-flight tube.
Figure 10.3 delineates the schematic of a typical Quadrupole-Time-of-Flight (Q-TOF) mass spectrometer, which comprises three fundamental components:</p>
<h4 id="quadrupole-mass-analyzer-q.">1. Quadrupole mass analyzer (Q).</h4>
<p>This part of the instrument is basically the same to the Q1 in QqQ, which select specific m/z values to pass through by applying a combination of DC and RF voltages across the rods.</p>
<h4 id="collision-cell.">2. Collision cell.</h4>
<p>Here, selected ions undergo collision-induced dissociation (CID) by interacting with a neutral gas, leading to their fragmentation into smaller constituents.
This process yields structural information about the original molecules.
Usually, quadrupole, hexapole, or even octopole are used as the collision cell for better focusing and transporting.</p>
<h4 id="time-of-flight-tof-mass-analyzer.">3. Time-of-Flight (TOF) mass analyzer.</h4>
<p>Upon exiting the collision cell, the fragmented ions are reaccelerated into the ion modulator region of the time-of-flight analyzer.
There, they undergo pulsing by a strong electric field (typically 20 kV or higher) and get accelerated to a field free drift tube, and then reflected to the detector.</p>
<p>TOFs generally offer mass resolutions surpassing 50,000, rendering it a reliable instrument for identifying unknown compounds.
Moreover, the rapid travel time of ions in the vacuum tube (at the nanosecond level) confers the Q-TOF with distinctive benefits in short gradient and high-throughput analyses <span class="citation" data-cites="eJibibTn KJYZ12ho cXsqAa1Y">[<a href="#ref-eJibibTn" role="doc-biblioref">357</a>,<a href="#ref-KJYZ12ho" role="doc-biblioref">358</a>,<a href="#ref-cXsqAa1Y" role="doc-biblioref">359</a>]</span>.
Another advantage of TOF is its broad mass range, which allows for the detection of large proteins, nanoclusters, and even large particles<span class="citation" data-cites="OeUnezUK 2oFwzqFP 3O301V9H">[<a href="#ref-OeUnezUK" role="doc-biblioref">360</a>,<a href="#ref-2oFwzqFP" role="doc-biblioref">361</a>,<a href="#ref-3O301V9H" role="doc-biblioref">362</a>]</span>.
However, it should be noted that due to ion numbers and detector limitations, mass resolution is typically difficult to maintain over a wide mass range.</p>
<p>Presently, Q-TOF related instruments are available from all leading instrument manufacturers, and the main models are listed below:
Sciex: “TripleTOF® 6600+”, “TripleTOF® 5600+” System and “X500R QTOF” System.
Bruker Corporation: “Impact II”, “timsTOF” series, “microTOF-Q III”, “ultrafleXtreme-MALDI-TOF/TOF” and “maXis II”.
Agilent Technologies: “Agilent 6530 Accurate-Mass Q-TOF”, “Agilent 6545 Accurate-Mass Q-TOF”, and “Agilent 6550 iFunnel Q-TOF”.
Waters Corporation: “SYNAPT G2-Si HDMS”, “Xevo G2-XS QToF” and “SYNAPT XS”.</p>
<div id="fig:q-tof-diagram" class="fignos">
<figure>
<img src="images/MS_f103.svg" style="width:100.0%" data-tag="3" alt="Figure 3: Schematic diagram of a typical quadrupole time-of-flight mass spectrometer. Like a QQQ, a Q-TOF will have two quadrupoles for selection and fragmentation followed by the TOF for the final higher resolution separation and detection." />
<figcaption aria-hidden="true"><span>Figure 3:</span> <strong>Schematic diagram of a typical quadrupole time-of-flight mass spectrometer.</strong>
Like a QQQ, a Q-TOF will have two quadrupoles for selection and fragmentation followed by the TOF for the final higher resolution separation and detection.</figcaption>
</figure>
</div>
<h3 id="q-orbitrap">3. Q-Orbitrap</h3>
<p>Mass spectrometry that uses Orbitrap as the core mass analyzer is another critical pillar in the field of proteomics.
In the late 20th century, Russian scientist Alexander Makarov invented the Orbitrap <span class="citation" data-cites="1CdqYFQjZ">[<a href="#ref-1CdqYFQjZ" role="doc-biblioref">363</a>]</span>, which is a novel mass analyzer that operates based on the principle of electrodynamic ion trapping and Fourier Transform.
The orbitrap consists of two main components: an inner spindle-like electrode and a coaxial outer barrel-like electrode (Figure 10.4 A).
The ions are trapped in an orbit around the spindle electrode due to the electrostatic attraction.
Once inside, the ions begin oscillating along the central axis of the device, or “orbiting”, due to the electric field formed by the inner and outer electrodes.
The oscillation frequency of an ion is inversely proportional to the square root of its mass-to-charge ratio.
The frequency at which each ion oscillates induces an image current on the detector, which can be measured and transformed into a mass spectrum using Fourier transform.</p>
<p>The biggest difference between Orbitrap and other mass spectrometers (TOF, Q) is that it does not use ions to hit an induction device like an electron multiplier.
One of the main advantages of the Orbitrap is its ultra-high mass resolution, often exceeding 240,000 or even higher.
This gives the Orbitrap a significant superiority in the identification of unknown molecules such as peptides and metabolites<span class="citation" data-cites="5vv7ID3o 1GyycMiIQ">[<a href="#ref-5vv7ID3o" role="doc-biblioref">338</a>,<a href="#ref-1GyycMiIQ" role="doc-biblioref">364</a>]</span>.
Moreover, Orbitrap spectrometers are also appreciated for their compact structure, small size, robustness, and reliability.
Just like the Q-TOF, the Orbitrap is also usually used for tandem mass spectrometry.
Figure 10.4 B demonstrates a typical 2D schematic diagram of Q-Orbitrap.
Ions first pass through an ion optics module, which consists of a high-capacity ion transfer tube (HCTT), an electrodynamic ion funnel (EDIF), and an advanced active beam guide (AABG).
These are designed to capture ions, reduce ion losses, prevent neutrals and high-velocity clusters from entering the quadrupole, and increase sensitivity.
The ions are then segmented by the quadrupole for precursor ion selection, and the selected ions are trapped by the ion-routing multipole for higher energy collisional dissociation.
Finally, the fragmented ions are captured once again by the C-trap and injected into the Orbitrap batch-by-batch for accurate mass-to-charge analysis.
Overall, this process still follows the logical sequence of precursor ions selection, precursor ions fragmentation, and fragment ions detection.</p>
<p>Compared to a TOF, one disadvantage of the Orbitrap is its longer cycle time (AGC pre-scan, ion injection, ion isolation, ion activation and mass analysis, usually &gt;100ms), which is a negative factor for the currently favored short gradient, high-throughput analysis.
Another minor flaw of Orbitrap is the challenge encountered when trying to pair it with MALDI.
This primarily stems from the fact that MALDI uses a pulsed ionization technique, whereas Orbitrap operates continuously.
This mismatch can lead to inefficiencies and challenges in coupling the two techniques.
At present, the Orbitrap still dominates important applications in almost all aspects of proteomics including biomarker discovery<span class="citation" data-cites="KRgCNqM9">[<a href="#ref-KRgCNqM9" role="doc-biblioref">365</a>]</span>, post-translational modification (PTM) analysis<span class="citation" data-cites="m2ZOJt1R 184JFw6LL">[<a href="#ref-m2ZOJt1R" role="doc-biblioref">15</a>,<a href="#ref-184JFw6LL" role="doc-biblioref">366</a>]</span>, quantitative proteomics (LFQ, TMT, iTRAQ)<span class="citation" data-cites="DqzDdsRo 4x97ohIq pRTFSroJ">[<a href="#ref-DqzDdsRo" role="doc-biblioref">367</a>,<a href="#ref-4x97ohIq" role="doc-biblioref">368</a>,<a href="#ref-pRTFSroJ" role="doc-biblioref">369</a>]</span>, protein-protein interaction studies<span class="citation" data-cites="1B5HMmjdK">[<a href="#ref-1B5HMmjdK" role="doc-biblioref">370</a>]</span> and structural proteomics<span class="citation" data-cites="BtlT8wee 1GNpqhnTj">[<a href="#ref-BtlT8wee" role="doc-biblioref">371</a>,<a href="#ref-1GNpqhnTj" role="doc-biblioref">372</a>]</span>.
It can perform both top-down and bottom-up analyses owing to its broad mass range, and is suitable for both Data-Dependent Acquisition (DDA) and Data-Independent Acquisition (DIA) methods.
Right now, the Orbitrap is still under patent protection and only one company, ThermoFisher, is allowed to manufacture related products.
Classic models from ThermoFisher include Orbitrap Ascend Tribrid, Orbitrap Eclipse™ Tribrid™, Orbitrap Fusion™ Lumos™, Orbitrap Exploris series (120, 240, 480) and Q Exactive™ series.</p>
<div id="fig:QE-diagram" class="fignos">
<figure>
<img src="images/MS_f104.svg" style="width:100.0%" data-tag="4" alt="Figure 4: Schematic diagram of orbitrap. (A) Close up of a orbitrap. (B) General schematic of complete Q-Orbitrap system." />
<figcaption aria-hidden="true"><span>Figure 4:</span> <strong>Schematic diagram of orbitrap.</strong>
(A) Close up of a orbitrap.
(B) General schematic of complete Q-Orbitrap system.</figcaption>
</figure>
</div>
<h3 id="quadrupole-fourier-transform-ion-cyclotron-resonance-q-ft-icr">4. Quadrupole Fourier Transform Ion Cyclotron Resonance (Q-FT-ICR)</h3>
<p>The Fourier Transform Ion Cyclotron Resonance (FT-ICR) mass spectrometer is a type of mass spectrometry that uses magnetic fields to separate ions based on their mass-to-charge ratio.
FT-ICR was first invented in 1974 by Alan G. Marshall and Melvin B. Comisarow from the University of British Columbia <span class="citation" data-cites="wknKHPF0">[<a href="#ref-wknKHPF0" role="doc-biblioref">373</a>]</span> and is widely recognized for its high mass resolution and precision, making it a highly valuable tool in many scientific fields including proteomics, metabolomics, petroleum analysis, and environmental science.
The central feature of an FT-ICR mass spectrometer is a superconducting magnet coupled with an ICR cell (Figure 10.5A).
This magnet creates a strong and homogeneous magnetic field in which ions are injected.
Once the ions are inside ICR cell, under the influence of the strong magnetic field, they follow a circular path with a very small orbital radius at a specific frequency directly proportional to their mass-to-charge ratio.
At this point, no detectable image current signal is generated by detector plates located inside the ICR cell.
To improve the signal, a voltage is applied by excitation plates and resonance occurs when the frequency of the strong magnetic field matches the cyclotron frequency of the ions.
The ions absorb radio frequency energy, which increases the radius of their circular path, and consequently, the excited ions move closer to the detector plates and generate a current.
The resulting signal is an oscillating pattern or a time-domain signal.</p>
<p>Similar to Orbitrap, this time-domain signal is then transformed into a frequency-domain signal using Fourier transform, hence the name Fourier Transform ion cyclotron resonance (ICR).
The Fourier transformed data forms a mass spectrum where each peak corresponds to a specific ion present in the sample.
One of the most important advantages of FT-ICR mass spectrometry is its exceptionally high mass resolution and mass accuracy, even for large and complex molecules.
This enables precise identification and characterization of a wide range of compounds in complex mixtures <span class="citation" data-cites="HLEPjTl5 64lSPaQQ">[<a href="#ref-HLEPjTl5" role="doc-biblioref">374</a>,<a href="#ref-64lSPaQQ" role="doc-biblioref">375</a>]</span>.
Moreover, FT-ICR mass spectrometry can be used for multiple stages of mass analysis (MSn), including tandem mass spectrometry (MS/MS), providing detailed information about the structure of ions.
Another significant benefit of FT-ICR is its broad mass range, making it possible to identify macromolecules like proteins for top-down proteomics <span class="citation" data-cites="Gw2v7q21 CwZoo7w">[<a href="#ref-CwZoo7w" role="doc-biblioref">11</a>,<a href="#ref-Gw2v7q21" role="doc-biblioref">376</a>]</span>.</p>
<p>Despite its advantages, FT-ICR mass spectrometry is not without challenges.
The technique requires high-performance superconducting magnets, which are expensive for both initial purchase and further maintenance.
This is because FT-ICR requires liquid nitrogen and liquid helium cooling systems to keep the magnet at a sufficiently low temperature to maintain its superconducting state.
Moreover, the device demands high vacuum conditions and careful temperature control to maintain the stability of the magnetic field and the ion trajectories.
A schematic representation of a Q-FT-ICR system is shown in Figure 10.5 B.
In congruence with the tandem mass spectrometers elucidated earlier, ions pass through an array of ion optics modules which designed for ion focusing and purification.
Following this, the ions are selectively filtered by the first quadrupole.
After this filtration, precursor ions undergo fragmentation in the collision cell, which can be a quadrupole, hexapole, or octopole.
The fragmented ions are subsequently re-concentrated by the ensuing focusing lens.
Ultimately, these fragmented ions are trapped, excited, and detected within the ICR cell.
At present, commercial FT-ICR mass spectrometers are available in both Thermo Fisher Scientific (“LTQ FT Ultra” and “LTQ FT Ultra Hybrid” systems) and Bruker Daltonics (“solariX” and “apex” series).</p>
<div id="fig:FT-ICR-diagram" class="fignos">
<figure>
<img src="images/MS_f105.svg" style="width:100.0%" data-tag="5" alt="Figure 5: Schematic of FT-ICR. (A) Typical FT-ICR cell. (B) Example of complete FT-ICR system." />
<figcaption aria-hidden="true"><span>Figure 5:</span> <strong>Schematic of FT-ICR.</strong>
(A) Typical FT-ICR cell.
(B) Example of complete FT-ICR system.</figcaption>
</figure>
</div>
<h3 id="ion-mobility">5. Ion mobility</h3>
<p>In the context of omics research, a fundamental task is the separation, identification, and quantification of molecules in complex mixtures.
Mass spectrometry alone can only provide two-dimensional data including mass-to-charge ratio and their intensity.
Liquid chromatography contributes to the separation of compounds and further provides the third dimension of information, retention time (RT), which make LC-MS evolves as the “golden standard” for proteomic analysis <span class="citation" data-cites="2Qw9ppKa ovFVXSFD">[<a href="#ref-2Qw9ppKa" role="doc-biblioref">377</a>,<a href="#ref-ovFVXSFD" role="doc-biblioref">378</a>]</span>.
Despite the substantial improvements in mass spectrometry resolution and liquid chromatography consistency, accurately identifying extremely similar molecules such as isomers with LC-MS remains a challenge.
Ion mobility mass spectrometry (IM-MS), a technique that utilizes electric fields to transport analytes through a buffer gas, is beneficial for separating and identifying ions based on their size, shape, and charge state.
This technique provides the fourth dimension of information, collision cross section (CCS), which allows for more comprehensive characterization of molecules <span class="citation" data-cites="ODpFT5rK">[<a href="#ref-ODpFT5rK" role="doc-biblioref">379</a>]</span>.
Apparently, multi-dimensional data is always beneficial for us to understand things comprehensively and accurately, thus getting closer to the truth.</p>
<p>In terms of mass spectrometry based proteomic analysis, adding CCS data can help us better separate, identify, and quantify peptides.</p>
<p>The core principle of ion mobility spectrometry is to separate ions in an inert gas under the influence of an electric field (E), and then measure the amount of time it takes for each ion to pass through drift tube, which is defined to be the steady-state drift velocity (Vd) correlated to the specific analyte’s mobility (K), as shown in Eq. 1.
Vd = KE (Eq.1)
While the primary measurement in IMS analyses is the mobility (K), for many analytical applications, it has become routine to convert K into the calculated collision cross-section value (CCS or Ω) using Mason-Schamp equation (Eq. 3)<span class="citation" data-cites="rfF30WBX">[<a href="#ref-rfF30WBX" role="doc-biblioref">380</a>]</span>.<br />
Ω=(3/16 〖(2π/μKT)〗^(1/2) ze)/N0K0 (Eq.2)<br />
The components of the equation are defined as follows: e, charge of an electron; z, ion charge; N0, buffer gas density; μ, reduced mass of the collision partners; kb, Boltzmann’s constant; and T, the drift region temperature.
Although the Mason-Schamp equation isn’t universally embraced, it is currently the primary formula the community uses to compute CCS.
In basic terms, the CCS serves as a standard metric for the size in the gas phase, generally expressed in units of square Angströms (Å2).
However, according to the Eq.2, parameters including gas composition, working pressure, temperature within the mobility region, path of analyte movement, and the strength of the applied field can influence the final CCS value and may differ for each specific IMS platform.
Hence, direct comparison of CCS value between different platforms often requires calibration.</p>
<p>Generally, ion mobility techniques can be categorized into three separation concepts: (1) temporally dispersive, (2) spatially dispersive, and (3) ion confinement (trapping) and selective release (Figure 10.6 A) <span class="citation" data-cites="2Qw9ppKa">[<a href="#ref-2Qw9ppKa" role="doc-biblioref">377</a>]</span>.
Temporally dispersive methods produce an arrival time spectrum based on differences in the time it takes for ions to traverse a similar gas-filled drift region under the influence of an electric field.
Time-dispersive technique inherently provides an extensive examination of all signals detected during a given observation window.
However, a fundamental limitation of this wide-ranging analysis is the diminished sensitivity linked to a single time dispersion occurrence, which usually requires many (10−100) events to be aggregated to achieve statistically significant ion mobility measurements.
In contrast, spatially dispersive methods separate ions based on mobility differences (charge, shape and size), leading them on distinct drift paths or trajectories, but without significant time differences.
A characteristic of spatially dispersive techniques is the scanning of voltage to obtain a broad-band ion mobility spectrum.
Types of spatially dispersive ion mobility include High Field Asymmetric Waveform Ion Mobility Spectrometry (FAIMS), uniform-field differential mobility analyzers (DMA), and the newly introduced scanned frequency ion mobility filter called transverse modulation ion mobility spectrometry (TMIMS).
Ion confinement and release strategies are recently developed techniques which trap ions in a pressurized drift cell by electric field, and then releasing them based on mobility distinctions.
This technique relies on the ability to control the position of ions under elevated pressure conditions using precisely adjustable electrodynamic fields.
It requires a precise fabrication craft and more complicated control system.
While it has only been perfected recently, typical products like trapped ion mobility spectrometry (TIMS) <span class="citation" data-cites="D1WlSrbO 2aWp1Ns2">[<a href="#ref-D1WlSrbO" role="doc-biblioref">381</a>,<a href="#ref-2aWp1Ns2" role="doc-biblioref">382</a>]</span> and cyclic traveling wave IMS have become commercially available <span class="citation" data-cites="Gl9lNiar">[<a href="#ref-Gl9lNiar" role="doc-biblioref">383</a>]</span>.
Table 10.3 summarized typical ion mobility separation techniques, their separation concept, electric field direction, gas flow direction, strengths, and drawbacks.
Also, for three categories of ion mobility techniques, we have selected a typical technique from each for brief introduction.</p>
<p>Table 10.3 Typical ion mobility separation techniques.</p>
<table>
<colgroup>
<col style="width: 12%" />
<col style="width: 12%" />
<col style="width: 12%" />
<col style="width: 12%" />
<col style="width: 12%" />
<col style="width: 37%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Separation concept</th>
<th style="text-align: center;">Ion mobility techniques</th>
<th style="text-align: center;">Ion movement direction</th>
<th style="text-align: center;">Electric field direction</th>
<th style="text-align: center;">Drift Gas direction</th>
<th style="text-align: center;">Characteristics</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Temporally Dispersive</td>
<td style="text-align: center;">drift tube IMS (DTIMS)</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">#</td>
<td style="text-align: center;">High mobility resolution (need long drift tube), direct measurement of CCS. Low speed, large size, low sensitivity,</td>
</tr>
<tr class="even">
<td style="text-align: center;">Temporally Dispersive</td>
<td style="text-align: center;">traveling wave IMS (TWIMS)</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">→→→</td>
<td style="text-align: center;">#</td>
<td style="text-align: center;">High mobility resolution, faster than DTIMS. Low sensitivity, large size, low sensitivity, traveling electric field waves.</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Spatially Dispersive</td>
<td style="text-align: center;">high-field asymmetric IMS (FAIMS)</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">↑↓</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">Good as mass filter, Fast. No CCS measurement (Compensation voltage instead), low mobility resolution.</td>
</tr>
<tr class="even">
<td style="text-align: center;">Spatially Dispersive</td>
<td style="text-align: center;">transverse modulation IMS (TMIMS)</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">→ and ↑↓</td>
<td style="text-align: center;">#</td>
<td style="text-align: center;">Transverse Modulation, compact instrumentation, orthogonal Separation, fast and high resolution</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Confinement and Selective Release</td>
<td style="text-align: center;">trapped ion mobility spectrometry (TIMS)</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">←</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">High mobility resolution, compact instrumentation, high sensitivity, high speed, high ion utilization rate</td>
</tr>
<tr class="even">
<td style="text-align: center;">Confinement and Selective Release</td>
<td style="text-align: center;">multi-pass cyclic traveling wave IMS</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">→→→</td>
<td style="text-align: center;">#</td>
<td style="text-align: center;">High mobility resolution, improved Signal-to-Noise ratio and sensitivity, versatility (from small molecules to large biomolecules) and adjustability (number of passes can often be adjusted)</td>
</tr>
</tbody>
</table>
<p>‘#’ means stationary drift gas; →, ←, ↑↓ indicates drift gas direction or electric force direction; →→→ represents a wave and gradient electric field.</p>
<div id="fig:IMS-diagram" class="fignos">
<figure>
<img src="images/MS_f106.svg" style="width:100.0%" data-tag="6" alt="Figure 6: Ion Mobility. (A) Conceptional diagram of three types of ion mobility strategies. (B) Schematic of drift tube ion mobility spectrometry. (C) Schematic of high field asymmetric waveform ion mobility spectrometry (FAIMS). (D) Schematic of trapped ion mobility spectrometry (TIMS)." />
<figcaption aria-hidden="true"><span>Figure 6:</span> <strong>Ion Mobility.</strong>
(A) Conceptional diagram of three types of ion mobility strategies.
(B) Schematic of drift tube ion mobility spectrometry.
(C) Schematic of high field asymmetric waveform ion mobility spectrometry (FAIMS).
(D) Schematic of trapped ion mobility spectrometry (TIMS).</figcaption>
</figure>
</div>
<h4 id="drift-tube-ion-mobility-spectrometry-dtims">10.5.1. Drift Tube Ion Mobility Spectrometry (DTIMS)</h4>
<p>The principle of Drift Tube Ion Mobility Spectrometry (DTIMS) is based on the differential migration (time) of ions through a neutral buffer gas (commonly helium or nitrogen) under the influence of a weak uniform electric field (typically tens of V/cm).
The mobility (K) of an ion is proportional to its drift velocity (V) and inversely proportional to the strength of the applied electric field (E).
For ions with same charge states, the drift velocities are primarily determined by their collisional interactions with a buffer gas, namely, mainly affected by their shape and size.
To illustrate this process, imagine two objects with identical mass: a solid metal ball and a feather. Due to its lower density, the feather should have a larger volume than the ball.
When both are dropped from the same height, the solid ball reaches the ground before the feather because of air resistance.
This observation doesn’t contradict Newton’s law of universal gravitation, as we have accounted for air resistance.
In the context of DTIMS, the buffer gas in the drift tube acts as the “air resistance”, while the uniform electric field represents the “gravity”.
Hence, ions with the same mass-to-charge ratio are separated based on their shape and size.
This capability allows DTIMS to distinguish between isomeric compounds with identical masses but different structural configurations, given that these isomers might have distinct interactions with the drift gas.
Also, follow the intuition of the free fall example, in DTIMS, smaller ions will move faster and hit the detector earlier than larger ions in DTIMS (Figure 10.6 B).<br />
DTIMS possess the strengths including high resolving power and allows for straightforward measurement of an ion’s CCS from first principles<span class="citation" data-cites="NcqTCZMl 6pMoYbxP">[<a href="#ref-NcqTCZMl" role="doc-biblioref">384</a>,<a href="#ref-6pMoYbxP" role="doc-biblioref">385</a>]</span>.
However, DTIMS also suffers from disadvantages including: 1) separation time is too long for all ions passing through the drift tube, relative to the accumulation time, which decreases the duty cycle.
2) A longer drift tube or higher pressure is needed for greater resolving power.
However, this inevitably increases ion diffusion and ion losses unless ion focusing techniques are employed.
3) Segmentation and collision between ions and gas molecules during the traveling process in drift tube reduces the sensitivity.
Continual advancements in DTIMS design and application of ion focusing techniques further pushed the mobility resolution of same DTIMS platforms to 100 to 250 (t/Δt) range or even greater.</p>
<h4 id="high-field-asymmetric-waveform-ion-mobility-spectrometry-faims">10.5.2 High Field Asymmetric Waveform Ion Mobility Spectrometry (FAIMS)</h4>
<p>High Field Asymmetric Waveform Ion Mobility Spectrometry (FAIMS) represents a distinct version of spatially dispersive ion mobility spectrometry (IMS).
This technique differentiates ions utilizing a pronounced asymmetric oscillating electric field combined with a moving gas.
The principle of FAIMS is based on the different trajectories of ions as they move through a high asymmetric electric field, which are determined by their physical structure and charge states <span class="citation" data-cites="KCpIS4fP VdxvCWch E9GVLceZ">[<a href="#ref-KCpIS4fP" role="doc-biblioref">386</a>,<a href="#ref-VdxvCWch" role="doc-biblioref">387</a>,<a href="#ref-E9GVLceZ" role="doc-biblioref">388</a>]</span>.
In FAIMS, gas-phase ions are carried by a flow of carrier gas between two electrodes in a direction orthogonal to the direction of asymmetric electric field (E).
The asymmetric waveform electric field is typically characterized by a short, high-voltage pulse of one polarity followed by a longer, lower-voltage pulse of the opposite polarity.
An ion’s mobility within such an electric field is determined by its charge state, its physical structure, and the properties of the surrounding gas it moves through.
Once the ions are subjected to an asymmetric electric field, the ions will alternate between travelling toward one electrode or the other as the field oscillates in polarity, resulting in a curved trajectories between the electrodes.
Some ions move more in the high field relative to the low field, and vice versa (Figure 10.6 C).
To differentiate between ions, a so-called “compensation voltage” (CV), which is a DC offset voltage that compensates for the differential ion movement in the high and low fields, is applied <span class="citation" data-cites="EUzlbRbi">[<a href="#ref-EUzlbRbi" role="doc-biblioref">389</a>]</span>.
In this case, only ions with a specific response to the changing electric field and those that match the applied compensation voltage (CV) will have a zero net movement and are able to traverse the drift region to the detector, while others hit the electrode plate and be neutralized.
By scanning or modulating the CV, different ion species can be selectively transmitted through the FAIMS device.
In contrast to drift tube IMS in which the ion stream is sampled in discrete packets and all ions reach the detector, FAIMS is a continuous filtration technique that allows uninterrupted sampling of the ion stream, but only for a selected subset of the ion population.
One of the good advantages of this continuous collection technique is greatly increase the signal-to-noise ratio for the ion(s) of interest by removing unwanted chemical noise, which make FAIMS more like a mass filter rather than ion mobility spectrometry.
The drawback of FAIMS is it cannot produce any CCS value and its low resolving power.
Commercial FAIMS products from vendors including Thermo Fisher and WATERS are available now.</p>
<h4 id="trapped-ion-mobility-spectrometry-tims">10.5.3 Trapped ion mobility spectrometry (TIMS)</h4>
<p>Trapped ion mobility spectrometry (TIMS) is a typical type of ion mobility which uses ion refinement and release strategy <span class="citation" data-cites="CzOaytxB">[<a href="#ref-CzOaytxB" role="doc-biblioref">390</a>]</span>.
The basic idea behind TIMS is a combination of traditional ion mobility spectrometry and ion trapping techniques.
Instead of driving ions through a drift tube filled with stationary gas, TIMS holds the ions stationary in a drift cell under a moving buffer gas and then release them by adjusting electric fields (voltages on electrodes).
This process was realized by applying two different electric fields:
1) Radially confining pseudopotential.
An RF (radio frequency) voltage is applied to the electrodes of the TIMS analyzer to generate a radially confining pseudopotential, which essentially no axial component and only use for “focusing” ions in the central region of TIMS tube, preventing them from diffusion or hitting electrodes.
2) Axially electric field.
An axially electric field gradient, produced by superimposing DC potentials on tunnel electrodes, is applied for “trapping” ions based on the equilibrium between the force of drift gas and the opposing force from the electric field gradient, which is stronger at the entrance and becomes progressively weaker moving deeper into the tunnel.</p>
<p>As a result, once ions entered the device, lower mobilities ones are trapped at positions where the magnitude of axially electric field is larger, while higher mobilities ones are confined to deeper positions of tunnel where axially electric field is lower.
Then, after enough ions have been accumulated in the TIMS tunnel, additional ions are prevented from entering the tunnel region and residing ions are trapped for a short time (usually few milliseconds) which can be defined by users.
Finally, the magnitude of axially electric field is decreased at a user defined rate so that ions are eluted as an order of mobilities value (K) from high to low (Figure 10.6D).
The axially electric field gradient is set by a resistor divider.
Importantly, like other ion mobility strategies, the resolving power of TIMS is highly dependent on the length of the gas column through which the ions traverse.
In TIMS, ions are trapped in a specific location while buffer gas continuously flows past them.
Thus, the resolving power achieved by TIMS depends on the “quantity” of gas, specifically the length of the gas column, that passes by the ions during the separation time.
This offers the direct benefit of allowing the analyzer to maintain a compact physical size (around 5 cm) and achieve a high resolving power (R ∼ 300), while the analytical gas column – the portion that flows during an analysis – can be extensive (up to 10 m) and tailored to the user’s needs.
Moreover, by leveraging the “trapping” capability (trapping time) of TIMS and the high scanning speeds of TOF, platforms such as TIMS-Q-TOF can implement a full duty cycle acquisition protocol known as Parallel Accumulation-Serial Fragmentation (PASEF) <span class="citation" data-cites="D1WlSrbO 1FmNlvpSt">[<a href="#ref-D1WlSrbO" role="doc-biblioref">381</a>,<a href="#ref-1FmNlvpSt" role="doc-biblioref">391</a>]</span>.
This is particularly meaningful for identifying more peptides within a given time frame, such as capture more precursors from co-eluted peptides in the same liquid chromatography peak.
Currently, Bruker is the primary provider of commercial mass spectrometers that utilizing TIMS-tof technology. (TIMS-tof pro, TIMS-tof pro2, SCP. etc.).</p>
<h4 id="structures-for-lossless-ion-manipulation-slim">10.5.4 Structures for Lossless Ion Manipulation (SLIM)</h4>
<p>A final type of ion mobility invented by Richard Smith and team at Pacific Northwest National Labs is called SLIM <span class="citation" data-cites="KgcabL7g">[<a href="#ref-KgcabL7g" role="doc-biblioref">392</a>]</span>.
This technology uses printed circuit boards to confine ions in long path lengths for high resolution ion mobility.
Ions can be passed through the board multiple times to achieve over 1 km of pathlength for IMS separation <span class="citation" data-cites="XOhnpKc9">[<a href="#ref-XOhnpKc9" role="doc-biblioref">393</a>]</span>.
This technology is currently under commercial development by Mobilion known as Mobie <span class="citation" data-cites="GwEAWw2z">[<a href="#ref-GwEAWw2z" role="doc-biblioref">394</a>]</span>.</p>
<h2 class="page_break_before" id="peptide-ionization">Peptide Ionization</h2>
<p>Until the early 1990s, peptides analysis by mass spectrometry was challenging.
Hard ionization techniques in use at the time, like fast atom bombardment, were not directly applicable to peptides without destroying or breaking them.
The soft ionization tehniques however, revolutionized the proteomics field and it became possible to routinely ionize and analyze peptides using MALDI and ESI techniques at high-throughput scale.
These two techniques were so impactful that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) “for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules” <span class="citation" data-cites="YdLSljVE">[<a href="#ref-YdLSljVE" role="doc-biblioref">395</a>]</span>.</p>
<h3 id="maldi">MALDI</h3>
<p>The term, Matrix-assisted LASER desorption ionization (MALDI), was coined by Hillenkamp and Karas in 1985<span class="citation" data-cites="zfiYj2co">[<a href="#ref-zfiYj2co" role="doc-biblioref">396</a>]</span>.
Karas and Hillenkamp discovered the MALDI technique first, although a similar ionization method was shown by Koichi Tanaka in 1988 <span class="citation" data-cites="15Ye4Za7Y">[<a href="#ref-15Ye4Za7Y" role="doc-biblioref">2</a>]</span>.
A few months later, Karas and Hillenkamp also demonstrated MALDI applied to protein ionization <span class="citation" data-cites="tnQHCFV7">[<a href="#ref-tnQHCFV7" role="doc-biblioref">397</a>]</span>.
It also created a controversy that the widely used method of MALDI from these two people had been overlooked, and the Nobel prize was awarded to Tanaka, whose system was rarely used<span class="citation" data-cites="1HBJs2tAB">[<a href="#ref-1HBJs2tAB" role="doc-biblioref">398</a>]</span>.</p>
<p>MALDI first requires the peptide sample to be co-crystallized with a matrix molecule, which is usually a volatile, low molecular-weight, organic aromatic compound.
Some examples of such compounds are cyno-hydroxycinnamic acid, dihyrobenzic acid, sinapinic acid, alpha-hydroxycinnamic acid, ferulic acid etc <span class="citation" data-cites="5ZJdaya1">[<a href="#ref-5ZJdaya1" role="doc-biblioref">399</a>]</span>.
Subsequently, the analyte is placed in a vacuum chamber in which it is irradiated with a LASER, usually at 337nm <span class="citation" data-cites="wGHAVD9r">[<a href="#ref-wGHAVD9r" role="doc-biblioref">400</a>]</span>.
This laser energy is absorbed by the matrix, which then transfers that energy along with its free protons to the co-crystalized peptides without significantly breaking them.
The matrix and co-crystallized sample generate plumes, and the volatile matrix imparts its protons to the peptides as it gets ionized first.
The weak acidic conditions used as well as the acidic nature of the matrix allows easy exchange of protons for the peptides to get ionized and fly under the electrical field in the mass spectrometer.
These ionized peptides generally form the metastable ions, most of them will fragment quickly <span class="citation" data-cites="DEnAe0bs">[<a href="#ref-DEnAe0bs" role="doc-biblioref">401</a>]</span>.
However, it can take several milliseconds and the mass spectrometry analysis can be performed before this time.
Peptides ionized by MALDI almost always take up a single charge and thus observed and detected as [M+H]+ species.</p>
<h4 id="maldi-mechanism">MALDI Mechanism</h4>
<div id="fig:MALDI-mechanism" class="fignos">
<figure>
<img src="images/MALDI.png" style="width:100.0%" data-tag="1" alt="Figure 1: MALDI The analyte-matrix mixture is irradiated by a laser source, leading to ablation. Desorption and proton transfer ionize the analyte molecules that can then be accelerated into a mass spectrometer." />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>MALDI</strong>
The analyte-matrix mixture is irradiated by a laser source, leading to ablation.
Desorption and proton transfer ionize the analyte molecules that can then be accelerated into a mass spectrometer.</figcaption>
</figure>
</div>
<h3 id="electrospray-ionization">Electrospray Ionization</h3>
<p>ESI was first applied to peptides by John Fenn and coworkers in 1989 <span class="citation" data-cites="4Ajjx5fo">[<a href="#ref-4Ajjx5fo" role="doc-biblioref">1</a>]</span>.
Concepts related to electrospray Ionization (ESI) were published at least as early as 1882, when Lord Rayleigh described the number of charges that could assemble on the surface of a droplet <span class="citation" data-cites="BslEV2vj">[<a href="#ref-BslEV2vj" role="doc-biblioref">402</a>]</span>.
ESI is usually coupled with reverse-phase liquid-chromatography of peptides directly interfaced to a mass spectrometer.
A high voltage (~ 2 kV) is applied between the spray needle and the mass spectrometer.
As solvent exits the needle, it forms droplets that take on charge at the surface, and through a debated mechanism, those charges are imparted to peptide ions.
The liquid phase is generally kept acidic to help impart protons easily to the analytes.</p>
<p>Tryptic peptides ionized by ESI usually carry one charge one the side chain of their c-terminal residue (Arg or Lys) and one charge at their n-terminal amine.
Peptides can have more than one charge if they have a longer peptide backbone, have histidine residues, or have missed cleavages leaving extra Arg and Lys.
In most cases, peptides ionized by ESI are observed at more than one charge state.
Evidence suggests that the distribution of peptide charge states can be maniupulated through chemical additives <span class="citation" data-cites="11vyuKWyw">[<a href="#ref-11vyuKWyw" role="doc-biblioref">403</a>]</span>.</p>
<h4 id="electrospray-mechanism">Electrospray Mechanism</h4>
<div id="fig:ESI-mechanism" class="fignos">
<figure>
<img src="images/ESI.png" style="width:100.0%" data-tag="1" alt="Figure 1: Electrospray Ionization Charged droplets are formed, their size is reduced due to evaporation until charge repulsion leads to Coulomb fission and results in charged analyte molecules." />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>Electrospray Ionization</strong>
Charged droplets are formed, their size is reduced due to evaporation until charge repulsion leads to Coulomb fission and results in charged analyte molecules.</figcaption>
</figure>
</div>
<p>The main goal of ESI is the production of gas-phase ions from electrolyte ions in solution.
During the process of ionization, the solution emerging from the electrospray needle or capillary is distorted into a Taylor cone and charged droplets are formed.
The charged droplets subsequently decrease in size due to solvent evaporation.
As the droplets shrink, the charge density and Coulombic repulsion increase.
This process destabilizes the droplets until the repulsion between the charges is higher than the surface tension and they fission (Coulomb explosion) <span class="citation" data-cites="4mpHldYP">[<a href="#ref-4mpHldYP" role="doc-biblioref">404</a>]</span> <span class="citation" data-cites="93IPP3bH">[<a href="#ref-93IPP3bH" role="doc-biblioref">405</a>]</span>.
Typical bottom-up proteomics experiments make use of acidic analyte solutions which leads to the formation of positively charged analyte molecules due to an excess presence of protons.</p>
<!-- TODO, explicitly describe the taylor cone and coulombic explosion -->
<!-- TODO: discussion of ionizaation mechanisms, ion evaporation versus charge residue model -->
<h2 class="page_break_before" id="data-acquisition">Data Acquisition</h2>
<!-- Figures:
1. show a plot of the polularity of MRM/DIA/DDA/ over time? 
-->
<p>Hybrid mass spectrometers used for modern proteome analysis offer the flexibility to collect data in many different ways.
Data acquisition strategies differ in the sequence of precursor scans and fragment ion scans, and in how analytes are chosen for MS/MS.
Constant innovation to develop better data collection methods improves our view of the proteome, but many method options may confuse newcomers.
This section provides an overview of the general classes of data collection methods.</p>
<p>Data acquisition strategies for proteomics fall into one of two groups.</p>
<ol type="1">
<li>Data dependent acquisition (DDA), in which the exact scan sequence in each analysis depends on the data that the mass spectrometer observes.</li>
<li>Data independent acquisition (DIA), in which the exact scan sequence in each analysis DOES NOT depend on the data; the collected scans are the same whether you inject yeast peptides, human peptides, or a solvent blank.</li>
</ol>
<p>DDA and DIA can both be futher subdivided in to targeted and untargeted methods.</p>
<!-- to do: figure showing the four basic types of data acquisition and their names, like four squares? -->
<h3 id="dda">DDA</h3>
<p>In most cases, the peptide masses that will be observed are not known before doing the experiment.
Data collection methods must account for this.
DDA was invented in the early 1990s, which enabled collecting MS/MS spectra for observed peptides as they eluting from the LC column <span class="citation" data-cites="JpKe7ymh byV7q4rP 1BU9KEnwO">[<a href="#ref-JpKe7ymh" role="doc-biblioref">406</a>,<a href="#ref-byV7q4rP" role="doc-biblioref">407</a>,<a href="#ref-1BU9KEnwO" role="doc-biblioref">408</a>]</span>.</p>
<h4 id="untargeted-dda">Untargeted DDA</h4>
<p>A common method currently used in modern proteomics is untargeted DDA.
The MS collects precursor (MS1) scans iteratively until precursor mass envelopes meeting certain criteria are detected.
Criteria for selection are usually specific charge states and a minimum signal intensity.
When those ions meet these criteria, the MS selects those masses for fragmentation.</p>
<p>Because ions are selected as they are observed, repeated DDA of the same sample will produce a different set of identifications.
This stochasticity is the main drawback of DDA.
To ameliorate this issue, often strategies are used to transfer identifications between multiple sample analyses.
This transfer of IDs across runs is known as “match between runs”, which was originally made famous by the processing software MaxQaunt <span class="citation" data-cites="vO5QsPvL Gs7HWzAD">[<a href="#ref-vO5QsPvL" role="doc-biblioref">409</a>,<a href="#ref-Gs7HWzAD" role="doc-biblioref">410</a>]</span>.
There are several other similar tools and strategies, including the accurate mass and time approach <span class="citation" data-cites="66LthRpU">[<a href="#ref-66LthRpU" role="doc-biblioref">411</a>]</span>, Q-MEND <span class="citation" data-cites="MaoP0MGM">[<a href="#ref-MaoP0MGM" role="doc-biblioref">412</a>]</span>, IDEAL-Q <span class="citation" data-cites="5H13CEWA">[<a href="#ref-5H13CEWA" role="doc-biblioref">413</a>]</span> and superHIRN <span class="citation" data-cites="rRs3KNfz">[<a href="#ref-rRs3KNfz" role="doc-biblioref">414</a>]</span>.
More recent work has introducted statistical assessment of MBR methods using a two-proteome model <span class="citation" data-cites="1EHOiWEPV">[<a href="#ref-1EHOiWEPV" role="doc-biblioref">415</a>]</span>.
Statistically controlled MBR is currently available in the IonQuant tool <span class="citation" data-cites="XGf3EEUa">[<a href="#ref-XGf3EEUa" role="doc-biblioref">416</a>]</span>.</p>
<p>Because DDA is required for quantification of proteins using isobaric tags like TMT, this stochasiticity of DDA limits the ability to compare quantities across batches.
For example, if you have 30 samples, you can use two sets of the 16-plex kit to label 15 samples in each set with one channel labeled by a pooled sample to enable comparison across the groups.
When you collect DDA data from each of those sets, each set will have MS/MS data from an overlapping but different set of peptides.
If one set has MS/MS from a peptide but the other set does not, then that peptide cannot be quantified in the whole sample group.
This limits the number of quantified proteins in large TMT experiments with multiple batches.
<!-- mention intelligent DDA and real time search? --></p>
<h4 id="targeted-dda">Targeted DDA</h4>
<p>Targeted DDA is not common in modern proteomics.
In targeted DDA, in addition to general criteria like a minimum intensity and a certain charge state, the mass spectrometer looks for specific masses.
These masses might be previously observed signals that were previously missed by MS/MS <span class="citation" data-cites="2D6WnK0w yqW0xFtN">[<a href="#ref-2D6WnK0w" role="doc-biblioref">417</a>,<a href="#ref-yqW0xFtN" role="doc-biblioref">418</a>]</span>.
In these studies, the sample is first analyzed by LC-MS to detect precursor ion features with some software, and then subsequent analyses target those masses for fragmentation with inclusion lists until they are all fragmeted.
This was shown to increase proteome converage.</p>
<h4 id="dda-methods-for-modifications">DDA methods for modifications</h4>
<p>Resonant CID <span class="citation" data-cites="h7UB5vUH">[<a href="#ref-h7UB5vUH" role="doc-biblioref">419</a>]</span> and beam-type HCD <span class="citation" data-cites="C0Uvh6dQ">[<a href="#ref-C0Uvh6dQ" role="doc-biblioref">420</a>]</span> are the most popular methods for unmodified and modified peptides due to their speed, accessibility, and efficiency.
Due to the weak phosphoester bond relative to the peptide backbone, resonant CID usually produces spectra that are dominated by only the neutral loss of the phosphate.
For this reason, the optimal dissociation methods for phosphopeptide identification and phosphosite localization include HCD or ExD-based methods, discussed later in more depth <span class="citation" data-cites="YU1KHJ6Z 18YSDGQ7f">[<a href="#ref-YU1KHJ6Z" role="doc-biblioref">421</a>,<a href="#ref-18YSDGQ7f" role="doc-biblioref">422</a>]</span>.
ExD methods generate phosphopeptide MS/MS spectra with many c- and z•-type fragment ions for peptide sequencing and localization of labile phosphate modifications, typically disrupted with CID <span class="citation" data-cites="GH22tfkO">[<a href="#ref-GH22tfkO" role="doc-biblioref">423</a>]</span>.
Gas-phase phosphate rearrangement induced by collisional activation represents a glaring challenge for the field and several have explored site localization in the face of rearrangement <span class="citation" data-cites="12LsVxcez 1BSd7ZIJx XAV4WUNo">[<a href="#ref-12LsVxcez" role="doc-biblioref">424</a>,<a href="#ref-1BSd7ZIJx" role="doc-biblioref">425</a>,<a href="#ref-XAV4WUNo" role="doc-biblioref">426</a>]</span>.</p>
<p>Advanced data acquisition schemes trigger predetermined MS/MS events when a specific fragment ion or neutral loss is detected in a spectrum.
Certain decision-tree strategies have arisen to increase data acquisition efficiency, including pseudo-MS3 scans which are triggered on detection of phosphate losses <span class="citation" data-cites="Oadeotpz">[<a href="#ref-Oadeotpz" role="doc-biblioref">427</a>]</span> and the use of site-specific x-type ions <span class="citation" data-cites="DcoxXgj">[<a href="#ref-DcoxXgj" role="doc-biblioref">428</a>]</span>.
For example, when linear ion traps were the main proteomics workhorses, resonant CID analysis of phosphopeptides would result in predominantly neutral loss of the phosphate with limited sequence ion information.
To gain sequence ions in these experiments, instruments could be set to isolate a loss of 98 Thompsons for MS3 activation <span class="citation" data-cites="SngtJdsy WtrZNDoq">[<a href="#ref-SngtJdsy" role="doc-biblioref">429</a>,<a href="#ref-WtrZNDoq" role="doc-biblioref">430</a>]</span>.
The newer collisional dissociation technique HCD, or beam-type collisional activation, significantly improves the detection of peptide fragments with the phosphorylation intact on fragment ions, and thus, this neutral loss scanning technique is no longer common.</p>
<p>Recently developed approaches to phosphopeptide identification include DIA-based phosphoproteomics with Spectronaut <span class="citation" data-cites="OCeJYJsg GD2lTg3N">[<a href="#ref-OCeJYJsg" role="doc-biblioref">431</a>,<a href="#ref-GD2lTg3N" role="doc-biblioref">432</a>]</span>, “plug-and-play” high-resolution MS <span class="citation" data-cites="18oI6YEEE">[<a href="#ref-18oI6YEEE" role="doc-biblioref">433</a>]</span>, SureQuant for phosphotyrosine <span class="citation" data-cites="nO23Yma7">[<a href="#ref-nO23Yma7" role="doc-biblioref">434</a>]</span>, PIQED for direct identification and quantification of phosphorylation from DIA without a prior spectral library <span class="citation" data-cites="184JFw6LL">[<a href="#ref-184JFw6LL" role="doc-biblioref">366</a>]</span>, and FAIMS front-end separations which yield 15-20% more phosphosite identifications than non-FAIMS experiments <span class="citation" data-cites="E9GVLceZ">[<a href="#ref-E9GVLceZ" role="doc-biblioref">388</a>]</span>.
For quantification of phosphoproteins, Hogrebe et al. investigated several of the most common strategies and concluded that TMT-based MS2 strategies may be the current best approach <span class="citation" data-cites="rydRUvTU">[<a href="#ref-rydRUvTU" role="doc-biblioref">435</a>]</span>.</p>
<p>Additionally, while less commonly modified than serine and threonine, histidine <span class="citation" data-cites="aoK1gRMs 17Un7LUDR 14IWO8iZc">[<a href="#ref-aoK1gRMs" role="doc-biblioref">436</a>,<a href="#ref-17Un7LUDR" role="doc-biblioref">437</a>,<a href="#ref-14IWO8iZc" role="doc-biblioref">438</a>]</span>, arginine <span class="citation" data-cites="xcaW4mpA">[<a href="#ref-xcaW4mpA" role="doc-biblioref">439</a>]</span>, and tyrosine <span class="citation" data-cites="1ELOuUl8I nO23Yma7 uOTrEnEQ">[<a href="#ref-nO23Yma7" role="doc-biblioref">434</a>,<a href="#ref-1ELOuUl8I" role="doc-biblioref">440</a>,<a href="#ref-uOTrEnEQ" role="doc-biblioref">441</a>]</span> phosphorylation also represent intriguing cell signaling biology.
Going forward, we expect that faster instruments will enable investigations of high phosphoproteomic depth and reproducibility in rapid timeframes, such that many proteomes can be analyzed for temporal and spatial insight.</p>
<p>A similar product-dependent MS/MS triggering strategy was introduced for N-linked glycopeptides <span class="citation" data-cites="GL3PYaNa">[<a href="#ref-GL3PYaNa" role="doc-biblioref">442</a>]</span>.
Collisional dissociation of glycosylated peptides produces oxonium ions, for example at m/z 204.09 (HexNAc) or m/z 366.14 (HexHexNAc).
If oxonium ions from the fragmented glycan are detected among the most abundant fragment ions of the HCD spectra, then an ETD scan is triggered.
This ETD scan provides information about the peptide sequence, while the original HCD scan provides glycan structure information.</p>
<h3 id="dia">DIA</h3>
<p>The simplest method to operate a mass spectrometer is to have predefined scans that are collected for each sample analysis.
This is data-independent acquisition (DIA); the scans that are collected do not depend on the data that the instrument observes.
Thus, the scan sequence is repetitive, looping through binned windows of predetermined width, and/or a predetermined m/z range.
Although simple in terms of data collection, when the scan sequence includes MS/MS, sophisticated software is required to analyze the data.
Like DDA, DIA can also be either targeted or untargeted <span class="citation" data-cites="wAKbWLXc">[<a href="#ref-wAKbWLXc" role="doc-biblioref">443</a>]</span>:
The two targeted DIA methods are selected reaction monitoring (SRM) or multiple reaction monitoring (MRM), and untargeted DIA (uDIA) is often referred to simply as “DIA” or SWATH (Figure 4).</p>
<div id="fig:DIA-types" class="fignos">
<figure>
<img src="images/data-acquisition-DIA-types.svg" style="width:100.0%" data-tag="4" alt="Figure 4: Types of DIA. A) SRM/MRM. Peptides are ionized by ESI and although there are many peptides entering the mass spectrometer at any time, the first quadruople (Q1) isolates one mass, which is then fragmented by HCD. Fragment masses from the peptide are then selected in the third quadruople (Q3). This leads to very low noise and high sensitivity. B) PRM. Like MRM, peptides are selected in the first quadruople, but this analysis is done on a high resolution instrument like an Orbitrap or TOF. Selectivity is gained by exploiting the high mass accuracy and resolution to monitor multiple fragment ions. C) uDIA/SWATH. Like MRM and PRM, peptides are isolated with Q1, but in this case a much wider isolation window is used. This usually results in co-isolation of many peptides simultaneously. Fragments from many peptides are measured with high resolution and high mass accuracy. Special software is used to get peptide identities and quantities from the fragment ions." />
<figcaption aria-hidden="true"><span>Figure 4:</span> <strong>Types of DIA.</strong>
A) SRM/MRM.
Peptides are ionized by ESI and although there are many peptides entering the mass spectrometer at any time, the first quadruople (Q1) isolates one mass, which is then fragmented by HCD.
Fragment masses from the peptide are then selected in the third quadruople (Q3).
This leads to very low noise and high sensitivity.
B) PRM.
Like MRM, peptides are selected in the first quadruople, but this analysis is done on a high resolution instrument like an Orbitrap or TOF.
Selectivity is gained by exploiting the high mass accuracy and resolution to monitor multiple fragment ions.
C) uDIA/SWATH.
Like MRM and PRM, peptides are isolated with Q1, but in this case a much wider isolation window is used.
This usually results in co-isolation of many peptides simultaneously.
Fragments from many peptides are measured with high resolution and high mass accuracy.
Special software is used to get peptide identities and quantities from the fragment ions.</figcaption>
</figure>
</div>
<h4 id="targeted-dia">Targeted DIA</h4>
<p>The first type of targeted DIA is called SRM or MRM <span class="citation" data-cites="NQj9Wees">[<a href="#ref-NQj9Wees" role="doc-biblioref">444</a>]</span>.
The popularity of this method in the literature peaked in 2014, with just under 1,500 documents on pubmed that year resulting from a search for “MRM”.
In this strategy, the QQQ MS is set so that the first quadrupole selects the precursor mass of the peptide(s) of interest, the second quadrupole fragments the peptide, and the third quadruople monitors the product of specific fragments from that peptide.
This strategy is very sensitive and has the benefit of very low noise.
The fragments monitored in Q3 are chosen such that it is unlikely these fragments could arise from another peptide.
Usually at least a few transitions are monitored for each peptide in order to get multiple measures for that peptide.</p>
<p>An early example of MRM applied to quantify c-reactive protein was in 2004 <span class="citation" data-cites="1FAMk6Qey">[<a href="#ref-1FAMk6Qey" role="doc-biblioref">445</a>]</span>.
Around the same time, SRM was combined with antibody enrichment of peptides from target proteins <span class="citation" data-cites="63Pqc9VE">[<a href="#ref-63Pqc9VE" role="doc-biblioref">446</a>]</span>.
This approach was popular for analysis of plasma proteins <span class="citation" data-cites="1B9P7OPqR">[<a href="#ref-1B9P7OPqR" role="doc-biblioref">347</a>]</span>.
These early examples led to many more studies that used QQQ MS instruments to get accurate quantitation of many proteins in one injection <span class="citation" data-cites="iJJwYdQ1 ba7F2Epn">[<a href="#ref-iJJwYdQ1" role="doc-biblioref">447</a>,<a href="#ref-ba7F2Epn" role="doc-biblioref">448</a>]</span>.
Scheduling MRM measurement when chromatography is stable additionally enabled better utilization of instrument duty cycle and therefore monitoring of more peptides per injection <span class="citation" data-cites="jcu6ZT0x">[<a href="#ref-jcu6ZT0x" role="doc-biblioref">449</a>]</span>.
Efforts even developed libraries of transitions that allow quantification of any protein in model organisms <span class="citation" data-cites="eFcH0xbv">[<a href="#ref-eFcH0xbv" role="doc-biblioref">450</a>]</span>.</p>
<p>Another similar targeted DIA method is called parallel reaction monitoring (PRM) <span class="citation" data-cites="1Bt8NAhNo">[<a href="#ref-1Bt8NAhNo" role="doc-biblioref">451</a>]</span>.
Instead of using a QQQ instrument, PRM uses a hybrid MS with a quadrupole and a high resolution mass analyzer, such as an Q-TOF or Q-Exactive.
The idea is that instead of monitoring specific fragments in Q3, the high mass accuracy can be used to filter peptide fragments for high selectivity and accurate quantification.
Studies have found that PRM and MRM/SRM have comparable dynamic range and linearity <span class="citation" data-cites="HB3XvRF2">[<a href="#ref-HB3XvRF2" role="doc-biblioref">452</a>]</span>.</p>
<h4 id="untargeted-dia">Untargeted DIA</h4>
<p>There were many implementations of uDIA over the years, starting in 2003 by Purvine et al from the Goodlett lab <span class="citation" data-cites="dWzeUq6d">[<a href="#ref-dWzeUq6d" role="doc-biblioref">453</a>]</span>.
In this first work they demonstrated uDIA using a Q-TOF with in source fragmentation, and showed that extracted ion chromatograms of precursor and fragment ions matched in shape suggesting that this could be used to identify and quantify peptides.
The following year, Venable et al from the Yates lab introduced uDIA with an ion trap <span class="citation" data-cites="yn40ByaQ">[<a href="#ref-yn40ByaQ" role="doc-biblioref">454</a>]</span>.
Subsequent methods include MSE <span class="citation" data-cites="Z3Hfnak1">[<a href="#ref-Z3Hfnak1" role="doc-biblioref">455</a>]</span>, PAcIFIC <span class="citation" data-cites="Wb9egjJg">[<a href="#ref-Wb9egjJg" role="doc-biblioref">456</a>]</span>, all ions fragmentation (AIF) <span class="citation" data-cites="OubW7j0g">[<a href="#ref-OubW7j0g" role="doc-biblioref">457</a>]</span>.
Computational methods were also developed to automate interpretation of this data, such as DeMux <span class="citation" data-cites="OubW7j0g">[<a href="#ref-OubW7j0g" role="doc-biblioref">457</a>]</span>, XDIA <span class="citation" data-cites="y71ZDsTJ">[<a href="#ref-y71ZDsTJ" role="doc-biblioref">458</a>]</span>, and ETISEQ <span class="citation" data-cites="ckj0XYOM">[<a href="#ref-ckj0XYOM" role="doc-biblioref">459</a>]</span>.</p>
<p>The paper that is often cited for uDIA that led to widespread adoption was by Gillet et al. from the Aebersold group in 2012 <span class="citation" data-cites="11ccXv5xm">[<a href="#ref-11ccXv5xm" role="doc-biblioref">460</a>]</span>.
In this paper they branded the idea as SWATH.
Widespread adoption may have been facilitated by the co-marketing of this idea by ABSciex as a proteomics solution on their new 5600 Q-TOF (called “tripleTOF” despite containing only one TOF, likely a portmanteau of “triple quadrupole” and “Q-TOF”).
Importantly, in the Gillet et al. paper the authors described a computational method to extract information from SWATH where peptides of interest were queried against the data.
They also demonstrated the application of SWATH to measure proteomic changes that happen in diauxic shift, and showed that SWATH can reveal modified peptides, in this case a methionine oxidation.</p>
<p>There are also many papers describing uDIA with orbitraps.
One early example described combining random isolation windows together and then demultiplexing the chimeric spectra <span class="citation" data-cites="1GCjdtGKY">[<a href="#ref-1GCjdtGKY" role="doc-biblioref">461</a>]</span>.
In another landmark paper, over 6,000 proteins were identified from mouse tissue by at least 2 peptides <span class="citation" data-cites="1GvzNjeV3">[<a href="#ref-1GvzNjeV3" role="doc-biblioref">462</a>]</span>.
In 2018, the new model orbitrap at that time (HF-X) enabled identification of nearly 6,000 human proteins in only 30 minutes.
Currently orbitraps have all but replaced the Sciex Q-TOFs for DIA data collection.</p>
<p>A new direction in uDIA is the addition of ion separation by ion mobility.
This has appeared in two forms.
On the timsTOF, diaPASEF makes use of the trapped ion mobility to increase speed and sensitivity of analysis <span class="citation" data-cites="YKRUqViz">[<a href="#ref-YKRUqViz" role="doc-biblioref">463</a>]</span>.
On the orbitrap, the combination of FAIMS and DIA has enabled the identification of over 10,000 proteins from one sample, which is a major milestone <span class="citation" data-cites="44oI1g68">[<a href="#ref-44oI1g68" role="doc-biblioref">464</a>]</span>.</p>
<h2 class="page_break_before" id="analysis-of-raw-data">Analysis of Raw Data</h2>
<p>The goal of basic data analysis is to convert raw spectral data into identities and quantities of peptides and proteins that can be used for biologically-focused analysis.
This step may often include measures of quality control, cross-run data normalization, quantification on different levels (precursor, peptide, protein), protein inference, PTM (post translational modification) localization and also first steps of data analysis, such as statistical hypothesis tests.</p>
<p>In typical bottom-up proteomics experiments, proteins are digested into peptides and further analyzed with LC-MS/MS systems.
Peptides can have different PTMs and ionize differently depending on their length and amino acid distributions.
Therefore, mass spectrometers often record different charge and modification states of one single peptide.
The entity that is recorded on a mass spectrometer is usually referred to as a precursor ion (peptide with its modification and charge state).
This precursor ion is fragmented and the precursor or peptide sequences are obtained though spectral matching.
The quantity of a precursor is estimated with various methods.
The measured precursor quantities are combined to generate a peptide quantity.
Peptides are also often combined into a protein group through protein inference, which combines multiple peptide identifications into a single protein identification <span class="citation" data-cites="imR1dFei wtH5a4Ax">[<a href="#ref-imR1dFei" role="doc-biblioref">465</a>,<a href="#ref-wtH5a4Ax" role="doc-biblioref">466</a>]</span>.
Protein inference is still a challenge in bottom-up proteomics.</p>
<p>Due to the inherent differences in the data structures of DDA and DIA measurements, there exist different types of software that can facilitate the steps mentioned above.
The existing software for DDA and DIA analysis can be further divided into freeware and non-freeware:</p>
<h4 id="dda-freeware">DDA freeware</h4>
<table>
<colgroup>
<col style="width: 13%" />
<col style="width: 40%" />
<col style="width: 46%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Publication</th>
<th style="text-align: center;">Website</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">MaxQuant</td>
<td style="text-align: center;">Cox and Mann, 2008<span class="citation" data-cites="18PIzg2Co">[<a href="#ref-18PIzg2Co" role="doc-biblioref">467</a>]</span></td>
<td style="text-align: center;"><a href="https://www.maxquant.org/">MaxQuant</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">MSFragger</td>
<td style="text-align: center;">Kong et al., 2017<span class="citation" data-cites="bKpYR6PB">[<a href="#ref-bKpYR6PB" role="doc-biblioref">468</a>]</span></td>
<td style="text-align: center;"><a href="https://msfragger.nesvilab.org/">MSFragger</a></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Mascot</td>
<td style="text-align: center;">Perkins et al., 1999<span class="citation" data-cites="rKtsv6G">[<a href="#ref-rKtsv6G" role="doc-biblioref">469</a>]</span></td>
<td style="text-align: center;"><a href="https://www.matrixscience.com/">Mascot</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">MS-GF+</td>
<td style="text-align: center;">Kim et al., <span class="citation" data-cites="1B2RVFOKK">[<a href="#ref-1B2RVFOKK" role="doc-biblioref">470</a>]</span></td>
<td style="text-align: center;"><a href="https://github.com/MSGFPlus/msgfplus">MS-GF+</a></td>
</tr>
<tr class="odd">
<td style="text-align: center;">X!Tandem</td>
<td style="text-align: center;">Craig et al., <span class="citation" data-cites="IcvYCrHi iK4IJ8ED">[<a href="#ref-IcvYCrHi" role="doc-biblioref">471</a>,<a href="#ref-iK4IJ8ED" role="doc-biblioref">472</a>]</span></td>
<td style="text-align: center;"><a href="https://www.thegpm.org/tandem/">GPMDB</a></td>
</tr>
</tbody>
</table>
<h4 id="dia-freeware">DIA freeware:</h4>
<table>
<colgroup>
<col style="width: 13%" />
<col style="width: 40%" />
<col style="width: 46%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Publication</th>
<th style="text-align: center;">Website</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">MaxDIA</td>
<td style="text-align: center;">Cox and Mann, 2008<span class="citation" data-cites="18PIzg2Co">[<a href="#ref-18PIzg2Co" role="doc-biblioref">467</a>]</span></td>
<td style="text-align: center;"><a href="https://www.maxquant.org/">MaxQuant</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">Skyline</td>
<td style="text-align: center;">MacLean et al., 2010<span class="citation" data-cites="CObAggjA">[<a href="#ref-CObAggjA" role="doc-biblioref">473</a>]</span></td>
<td style="text-align: center;"><a href="https://skyline.ms/project/home/software/Skyline/begin.view">Skyline</a></td>
</tr>
<tr class="odd">
<td style="text-align: center;">DIA-NN</td>
<td style="text-align: center;">Demichev et al., 2019<span class="citation" data-cites="1HLH2DWkK">[<a href="#ref-1HLH2DWkK" role="doc-biblioref">474</a>]</span></td>
<td style="text-align: center;"><a href="https://github.com/vdemichev/DiaNN">DIA-NN</a></td>
</tr>
</tbody>
</table>
<h4 id="targeted-proteomics-freeware">Targeted proteomics freeware:</h4>
<table>
<colgroup>
<col style="width: 13%" />
<col style="width: 40%" />
<col style="width: 46%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Publication</th>
<th style="text-align: center;">Website</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Skyline</td>
<td style="text-align: center;">MacLean et al., 2010<span class="citation" data-cites="CObAggjA">[<a href="#ref-CObAggjA" role="doc-biblioref">473</a>]</span></td>
<td style="text-align: center;"><a href="https://skyline.ms/project/home/software/Skyline/begin.view">Skyline</a></td>
</tr>
</tbody>
</table>
<h4 id="dda-non-freeware">DDA non-freeware:</h4>
<table>
<colgroup>
<col style="width: 13%" />
<col style="width: 40%" />
<col style="width: 46%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Publication</th>
<th style="text-align: center;">Website</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">ProteomeDiscoverer</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="https://www.thermofisher.com/ch/en/home/industrial/mass-spectrometry/liquid-chromatography-mass-spectrometry-lc-ms/lc-ms-software/multi-omics-data-analysis/proteome-discoverer-software.html">ProteomeDiscoverer</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">Mascot</td>
<td style="text-align: center;">Perkins et al., 1999<span class="citation" data-cites="rKtsv6G">[<a href="#ref-rKtsv6G" role="doc-biblioref">469</a>]</span></td>
<td style="text-align: center;"><a href="https://www.matrixscience.com/">Mascot</a></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Spectromine</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="ttps://biognosys.com/software/spectromine/?gclid=Cj0KCQiAoY-PBhCNARIsABcz770mjUz6iavBr9Ql7RPUdMvaHu9RYgPNrEfZco1wExEeoFwnQXuCHscaAlgBEALw_wcB">Spectromine</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">PEAKS</td>
<td style="text-align: center;">Tran et al., 2018<span class="citation" data-cites="IUqTUA43">[<a href="#ref-IUqTUA43" role="doc-biblioref">475</a>]</span></td>
<td style="text-align: center;"><a href="https://www.bioinfor.com/peaks-studio/">PEAKS</a></td>
</tr>
</tbody>
</table>
<h4 id="dia-non-freeware">DIA non-freeware:</h4>
<table>
<colgroup>
<col style="width: 13%" />
<col style="width: 40%" />
<col style="width: 46%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Publication</th>
<th style="text-align: center;">Website</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Spectronaut</td>
<td style="text-align: center;">Bruderer et al., 2015<span class="citation" data-cites="UKcaDUJ7">[<a href="#ref-UKcaDUJ7" role="doc-biblioref">476</a>]</span></td>
<td style="text-align: center;"><a href="https://biognosys.com/software/spectronaut/">Spectronaut</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">PEAKS</td>
<td style="text-align: center;">Tran et al., 2018<span class="citation" data-cites="IUqTUA43">[<a href="#ref-IUqTUA43" role="doc-biblioref">475</a>]</span></td>
<td style="text-align: center;"><a href="https://www.bioinfor.com/peaks-studio/">PEAKS</a></td>
</tr>
</tbody>
</table>
<h4 id="data-summary-and-interpretation">Data Summary and Interpretation</h4>
<table>
<colgroup>
<col style="width: 15%" />
<col style="width: 35%" />
<col style="width: 48%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Publication</th>
<th style="text-align: center;">Website</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Peptide Shaker</td>
<td style="text-align: center;">Vaudel <em>et al.</em>, 2015<span class="citation" data-cites="B10jxZjh tZ6JV4UF">[<a href="#ref-B10jxZjh" role="doc-biblioref">477</a>,<a href="#ref-tZ6JV4UF" role="doc-biblioref">478</a>]</span></td>
<td style="text-align: center;"><a href="http://compomics.github.io/projects/peptide-shaker.html">PeptideShaker</a>, <a href="https://github.com/barsnes-group/peptide-shaker-online">Peptide Shaker Online</a></td>
</tr>
</tbody>
</table>
<h3 id="analysis-of-dda-data">Analysis of DDA data</h3>
<p>DDA data analysis either directly uses the vendor proprietary data format directly with a proprietary search engine like Mascot, Sequest (through Proteome Discoverer), Paragon (through Protein Pilot), or it can be processed through one of the many freely available search engines or pipelines, for example, MaxQuant, MSGF+, X!Tandem, Morpheus, MSFragger, and OMSSA.
Tables 1 and 4 give weblinks and citations for these software tools.
<!-- todo: need to make sure all these are listed in the table and cited -->
<!-- todo: need a paragraph discussing intergration environments like the transproteomics pipeline (TPP) and IDpicker -->
For analysis with freeware, raw data is converted to either text-based MGF (mascot generic format) or into a standard open XML format like mzML <span class="citation" data-cites="zsmAmqZJ fQHXeOG1 r2hPariY">[<a href="#ref-zsmAmqZJ" role="doc-biblioref">479</a>,<a href="#ref-fQHXeOG1" role="doc-biblioref">480</a>,<a href="#ref-r2hPariY" role="doc-biblioref">481</a>]</span>.
The appropriate FASTA file containing proteins predicted from that organism’s genome is chosen as a reference database to search the experimental spectra.
All search parameters like peptide and fragment mass errors (i.e. MS1 and MS2 tolerances), enzyme specificity, number of missed cleavages, chemical artefacts (fixed modifications) and potential biological modifications (variable/dynamic modifications) are specified before executing the search.
The search algorithm scores each query spectrum against its possible peptide matches <span class="citation" data-cites="1GxDrwPIW">[<a href="#ref-1GxDrwPIW" role="doc-biblioref">482</a>]</span>.
A spectrum and its best scoring candidate peptide are called a peptide spectrum match (PSM).
The scores reflect a <em>goodness-of-fit</em> between an experimental spectrum and a theoretical one and do not necessarily depict the correctness of the peptide assignment.</p>
<p>For evaluating the matches, a decoy database is preferred as a null model for peptide matching.
A randomized or reversed version of target database is used as a nonparametric null model.
The decoy database can be searched separate from the target database (Kall’s method)<span class="citation" data-cites="bjANWlBN">[<a href="#ref-bjANWlBN" role="doc-biblioref">483</a>]</span> or it can be combined with the target database before search (Elias and Gygi method)<span class="citation" data-cites="BrkRiRe8">[<a href="#ref-BrkRiRe8" role="doc-biblioref">484</a>]</span>.
Using either separate method or concatenated database search method, an estimate of false hits can be calculated which is used to estimate the false discovery rate (FDR) <span class="citation" data-cites="5mQtZh5o">[<a href="#ref-5mQtZh5o" role="doc-biblioref">485</a>]</span>.
The FDR denotes the proportion of false hits in the population accepted as true.
For Kall’s method: the false hits are estimated to be the number of decoys above a given threshold.
It is assumed that the number of decoy hits that pass a threshold are the false hits.
A similar number of target population may also be false.
Therefore, the FDR is calculated as <span class="citation" data-cites="B2OZyvg5">[<a href="#ref-B2OZyvg5" role="doc-biblioref">486</a>]</span>:</p>
<p><span class="math display">\[FDR = \frac{Decoy PSMs + 1}{Target PSMs}\]</span></p>
<p>For Elias and Gygi Method, the target population in which FDR is estimated changes.
The target and decoy hits coming from a joint database compete against each other.
For any spectrum, either a target or a decoy peptide can be the best hit.
It is argued that the joint target-decoy population has decoy hits as confirmed false hits.
However, due to the joint database search, the target database may also have equal number of false hits.
Thus, the number of false hits is multiplied by two for FDR estimation.</p>
<p><span class="math display">\[FDR = \frac{2 * Decoy PSMs}{Target + Decoy PSMs}\]</span></p>
<h3 id="strategies-for-analysis-of-dia-data">Strategies for analysis of DIA data</h3>
<h3 id="targeted-proteomics-data-analysis">Targeted proteomics data analysis</h3>
<h3 id="quality-control">Quality control</h3>
<p>Quality control should be a central aspect of any mass spectrometry-based study to ensure reproducibility of generated results.
There are two types of quality controls that can be conducted for any kind of mass spectrometry experiment.
The first one is focused on monitoring the perfomance of the instruments themselves (e.g. HPLC and mass spectrometer), whereas the second one is focused on your experiments.
For further reading, we recommend to take a look at issue 11 on quality control published in the journal <em>Proteomics</em> in 2011 <span class="citation" data-cites="JXLIvVI6">[<a href="#ref-JXLIvVI6" role="doc-biblioref">487</a>]</span>, especially the review by Köcher <em>et al.</em> <span class="citation" data-cites="8dt10izl">[<a href="#ref-8dt10izl" role="doc-biblioref">488</a>]</span>, as well as the review published by Bittremieux <em>et al.</em> in 2017 <span class="citation" data-cites="196I1uWdL">[<a href="#ref-196I1uWdL" role="doc-biblioref">489</a>]</span>.</p>
<h4 id="instrument-performance">Instrument Performance</h4>
<p>It is generally advisable to monitor instrument performance regularly.
Instrument calibrations in regular intervals help ensure that performance is maintained.
Often basic calibration and sensitivity can be checked by direct infusion of a standard.
During the calibration you can check injection times (for ion trap instruments) and intensity of the ions in the calibration mix.</p>
<p>After ensuring good calibration and signal with the simple calibration mixture, it is advisable to analyze complex samples, such as tryptic digests of whole-cell lysates (e.g. HeLa cells, HEK cells, yeast, etc.) or tryptic digests of purified proteins.
The additional check with a complex sample ensures all aspects of the system are working together correctly, especially the liquid chromatography and emitter.
These digests should be analyzed after every instrument calibration and periodically between samples when acquiring more extensive batches.
Data measured from tryptic digests should be analyzed by the software of your choice and the numbers of identified peptide precursors and proteins can be compared with previous controls for consistency.</p>
<p>Another strategy is to analyze digested purified proteins, which easily enable discovery of retention time shifts and mass accuracy problems.
In case you are working with a Thermo mass spectrometer, you can open the acquired .raw file directly either in FreeStyle or in Qual Browser and look for specific m/z values of your peptides.
Looking at the intensity of the extracted peaks will help identify sensitivity fluctuations.</p>
<p>Carry-over between different measurments can be identified from blank measurements which are subsequently analyzed with your search software of choice.
Blank measurements can be injections of different buffers, water or the starting conditions of your liquid chromatography.
In case of increased detection of carry-over, injections with trifluoroethanol can be performed.</p>
<p>Another factor to take into consideration is the stability of your electrospray.
Electrospray stability tends to worsen over time as columns wear, as well as when measuring samples with residual contaminants, such as salts or detergents.
You will notice spray instabilities either in the total ion chromatogram (TIC) as thin spikes with short periods of no measured signal or if you install cameras at your ESI source.
Suboptimal spray conditions will usually result in droplets forming on the emitter, being released into the mass spectrometer (also referred to as “spitting”).
Real-time quality control software (listed in the table below) can help you identify instrument issues right away.</p>
<h4 id="data-quality-control">Data Quality Control</h4>
<p>Apart from instrument performance, any kind of data analysis should have proper quality control in place to identify problematic measurements and to exclude them if necessary.
It is recommended to develop a standardized system for data quality control early on and to keep this consistent over time.
Adding indexed retention time (iRT) peptides can help identify and correct gradient and retention time inconsistencies between samples at the data analysis stage.
Decoy searches help monitor and control the false-discovery rate.
Inlcuding common contaminants, such as keratins, in the FASTA files used for searches can help identify sample preparation issues.
Other parameters to check in your analysis are the consistency of the number of peptide-spectrum matches, identified peptides and proteins over all samples of your study, as well as your coefficients of variation between your replicates.
Before and after data normalization (if normalization is performed) it is good to compare the median intensities of all measurments to identify potential measurement or normalization issues.
Precursor charge distributions, missed cleavage numbers, peak width, as well as the number of points per peak are additional parameters that can be checked.
In case you are analyzing different conditions, you can perform hierarchical clustering or a principal component analysis to check if your samples cluster as expected.</p>
<h4 id="quality-control-software">Quality Control Software</h4>
<h5 id="raw-file-and-real-time-analysis">Raw file and real-time analysis</h5>
<table>
<colgroup>
<col style="width: 7%" />
<col style="width: 27%" />
<col style="width: 24%" />
<col style="width: 21%" />
<col style="width: 18%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Supported instrument vendors</th>
<th style="text-align: center;">Website/Download</th>
<th style="text-align: center;">publication</th>
<th style="text-align: center;">Note</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">QuiC</td>
<td style="text-align: center;">Thermo Scientific, AB SCIEX, Agilent, Bruker, Waters</td>
<td style="text-align: center;"><a href="https://biognosys.com/resources/quic-manual/">QuiC</a></td>
<td style="text-align: center;"></td>
<td style="text-align: center;">requires Biognosys iRT peptides</td>
</tr>
<tr class="even">
<td style="text-align: center;">AlphaPept</td>
<td style="text-align: center;">Thermo Scientific, Bruker</td>
<td style="text-align: center;"><a href="https://github.com/MannLabs/alphapept">AlphaPept</a></td>
<td style="text-align: center;"><span class="citation" data-cites="9ocWQ3g2">[<a href="#ref-9ocWQ3g2" role="doc-biblioref">490</a>]</span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: center;">RawMeat 2.1</td>
<td style="text-align: center;">Thermo Scientific</td>
<td style="text-align: center;"><a href="https://proteomicsresource.washington.edu/protocols06/RawMeat_1007.exe">RawMeat</a></td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: center;">rawDiag</td>
<td style="text-align: center;">Thermo Scientific</td>
<td style="text-align: center;"><a href="https://github.com/fgcz/rawDiag">rawDiag</a></td>
<td style="text-align: center;"><span class="citation" data-cites="mKMrKqLf">[<a href="#ref-mKMrKqLf" role="doc-biblioref">491</a>]</span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: center;">rawrr</td>
<td style="text-align: center;">Thermo Scientific</td>
<td style="text-align: center;"><a href="https://github.com/fgcz/rawrr">rawrr</a></td>
<td style="text-align: center;"><span class="citation" data-cites="AI1wddvh">[<a href="#ref-AI1wddvh" role="doc-biblioref">492</a>]</span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: center;">rawBeans</td>
<td style="text-align: center;">Thermo or mzML</td>
<td style="text-align: center;"><a href="https://bitbucket.org/incpm/prot-qc/downloads/">rawBeans</a></td>
<td style="text-align: center;"><span class="citation" data-cites="MotUU8WD">[<a href="#ref-MotUU8WD" role="doc-biblioref">493</a>]</span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: center;">SIMPATIQCO</td>
<td style="text-align: center;">Thermo Scientific</td>
<td style="text-align: center;"><a href="https://ms.imp.ac.at/index.php?action=simpatiqco">SIMPATIQCO</a></td>
<td style="text-align: center;"><span class="citation" data-cites="fVgT4RCS">[<a href="#ref-fVgT4RCS" role="doc-biblioref">494</a>]</span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: center;">QC-ART</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="https://github.com/stanfill/QC-ART">QC-ART</a></td>
<td style="text-align: center;"><span class="citation" data-cites="gR6OKDGy">[<a href="#ref-gR6OKDGy" role="doc-biblioref">495</a>]</span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: center;">SprayQc</td>
<td style="text-align: center;">Thermo Scientific, AB SCIEX, extensible to other instrumentation</td>
<td style="text-align: center;"><a href="http://sourceforge.org/projects/sprayqc">SprayQc</a></td>
<td style="text-align: center;"><span class="citation" data-cites="IrACRvhj">[<a href="#ref-IrACRvhj" role="doc-biblioref">496</a>]</span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: center;">Metriculator</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="http://github.com/princelab/metriculator">Metriculator</a></td>
<td style="text-align: center;"><span class="citation" data-cites="N0V314Nq">[<a href="#ref-N0V314Nq" role="doc-biblioref">497</a>]</span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: center;">MassQC</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="https://massqc.proteomesoftware.com/">MassQC</a></td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: center;">OpenMS</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="https://www.openms.de/">OpenMS</a></td>
<td style="text-align: center;"><span class="citation" data-cites="ZJ3s73nL">[<a href="#ref-ZJ3s73nL" role="doc-biblioref">498</a>]</span></td>
<td style="text-align: center;"></td>
</tr>
</tbody>
</table>
<h5 id="search-result-qc">Search result QC</h5>
<table>
<colgroup>
<col style="width: 10%" />
<col style="width: 36%" />
<col style="width: 27%" />
<col style="width: 25%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Website/Download/publication</th>
<th style="text-align: center;">publication</th>
<th style="text-align: center;">Note</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">MSStats</td>
<td style="text-align: center;"><a href="https://github.com/Vitek-Lab/MSstats">MSStats</a></td>
<td style="text-align: center;"><span class="citation" data-cites="LpMjs9Ht">[<a href="#ref-LpMjs9Ht" role="doc-biblioref">499</a>]</span></td>
<td style="text-align: center;">can use output from MaxQuant, Proteome Discoverer, Skyline, Progenesis, Spectronaut</td>
</tr>
<tr class="even">
<td style="text-align: center;">MSStatsQC</td>
<td style="text-align: center;"><a href="https://msstats.org/msstatsqc/">MSStatsQC</a></td>
<td style="text-align: center;"><span class="citation" data-cites="BVhBtZ2w">[<a href="#ref-BVhBtZ2w" role="doc-biblioref">500</a>]</span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: center;">PTXQC</td>
<td style="text-align: center;"><a href="https://github.com/cbielow/PTXQC">PTXQC</a></td>
<td style="text-align: center;"><span class="citation" data-cites="R1U4dX3A">[<a href="#ref-R1U4dX3A" role="doc-biblioref">501</a>]</span></td>
<td style="text-align: center;">requires MaxQuant search engine output</td>
</tr>
<tr class="even">
<td style="text-align: center;">protti</td>
<td style="text-align: center;"><a href="https://github.com/jpquast/protti">protti</a></td>
<td style="text-align: center;"><span class="citation" data-cites="Nix1GwUR">[<a href="#ref-Nix1GwUR" role="doc-biblioref">502</a>]</span></td>
<td style="text-align: center;"></td>
</tr>
</tbody>
</table>
<!-- todo: this table is probably very incomplete. Add more tools -->
<h3 id="statistical-hypothesis-testing">Statistical hypothesis testing</h3>
<h2 class="page_break_before" id="databases">Databases</h2>
<h3 id="what-are-they-and-where-do-you-get-them">What are they and where do you get them?</h3>
<h4 id="protein-database-sources-and-types">Protein Database Sources and Types</h4>
<p>Many mass spectrometry-based proteomic techniques use search algorithms that require a defined theoretical search space to identify peptide sequences based on precursor mass and fragmentation patterns, which are then used to infer the presence and abundance of a protein.
The search space is calculated from the potential proteins in a sample, which includes the proteome (often a single species) and expected contaminants.
This is called database searching and the flat file of protein sequences in FASTA format acts as a protein database.
In this section, we will describe major resources for proteome FASTA files (protein sequence collections), how to retrieve them, and suggested best practices for preserving FASTA file provenance to improve reproducibility.</p>
<p>In general, FASTA sequence collections can be retrieved from three central clearing houses: UniProt, RefSeq, and Ensembl.
These will be discussed separately below as they each have specific design goals, data products, and unique characteristics.
It is important to learn the following three points for each resource: the source of the underlying data, canonical versus non-canonical sequences, and how versioning works.
These points, along with general best practices, such as using a taxonomic identifier, are essential to understand and communicate search settings used in analyses of proteomic datasets.
Finally, it is critical to understand that sequence collections from these three resources are not the same, nor do they offer the same sets of species.</p>
<p>Key terminology may vary between resources, so these terms are defined here.
The term “taxon identifier” is used across resources and is based on the NCBI taxonomy database.
Every taxonomic node has a number, e.g., Homo sapiens (genus species) is 9606 and Mammalia (class) is 40674.
This can be useful when retrieving and describing protein sequence collections.
Another term used is “annotation”, which has different meanings in different contexts.
Broadly, a “genome annotation” is the result of an annotation pipeline to predict coding sequences, and often a gene name/symbol if possible.
Two examples are MAKER <span class="citation" data-cites="KDnDuk25">[<a href="#ref-KDnDuk25" role="doc-biblioref">503</a>]</span> and the RefSeq annotation pipeline <span class="citation" data-cites="12mAW7BXf">[<a href="#ref-12mAW7BXf" role="doc-biblioref">504</a>]</span>.
Alternatively, “protein annotation” (or gene annotation) often refers to the annotation of proteins (gene products) using names and ontology (i.e., protein names, gene names/symbols, functional domains, gene onotology, keywords, etc.).
Protein annotation is termed “biocuration” and described in detail by UniProt <span class="citation" data-cites="BRlMNUOW">[<a href="#ref-BRlMNUOW" role="doc-biblioref">505</a>]</span>.
Lastly, there are established minimum reporting guidelines for referring to FASTA files established in MIAPE: Mass Spectrometry Informatics that are taxon identifier and number of sequences <span class="citation" data-cites="1EbCD9r1v paTtvUb8">[<a href="#ref-1EbCD9r1v" role="doc-biblioref">506</a>,<a href="#ref-paTtvUb8" role="doc-biblioref">507</a>]</span>.
The FASTA file naming suggestions below are not official but are suggested as a best practice.</p>
<h5 id="uniprot">UniProt</h5>
<p>The Universal Protein Resource (UniProt) <span class="citation" data-cites="16uefmiYk">[<a href="#ref-16uefmiYk" role="doc-biblioref">508</a>]</span>, has three different products: UniProt Knowledgebase (UniProtKB), the UniProt Reference Clusters (UniRef), and the UniProt Archive (UniParc).
The numerous resources and capabilities associated with the UniProt are not explored in this section, but these are well described on UniProt’s website.
UniProtKB is the source of proteomes across the Tree of Life and is the resource we will be describing herein.
There are broadly two types of proteome sequence collections: Swiss-Prot/TrEMBL and designated proteomes.
The Swiss-Prot/TrEMBL type can be understood by discussing how data is integrated into UniProt.
Most protein sequences in UniProt are derived from coding sequences submitted to EMBL-Bank, GenBank and DDBJ.
These translated sequences are initially imported into TrEMBL database, which is why TrEMBL is also termed “unreviewed”.
There are other sources of protein sequences, as described by UniProt <span class="citation" data-cites="1BIF8v4YE">[<a href="#ref-1BIF8v4YE" role="doc-biblioref">509</a>]</span>.
These include the Protein Data Bank (PDB), direct protein sequencing, sequences derived from the literature, gene prediction (from sources such as Ensembl) or in-house prediction by UniProt itself.
Protein sequences can then be manually curated into the Swiss-Prot database using multiple outlined steps (described in detail by UniProt here <span class="citation" data-cites="Q2scu90J">[<a href="#ref-Q2scu90J" role="doc-biblioref">510</a>]</span>) and is why Swiss-Prot is also termed “reviewed”.
Note that more than one TrEMBL entry may be removed and replaced by a single Swiss-Prot entry during curation.
A search of “organism:9606” at UniProtKB will retrieve both the Swiss-Prot/reviewed and TrEMBL/unreviewed sequences for Homo sapiens.
The entries do not overlap, so users often either use just Swiss-Prot or Swiss-Prot combined with TrEMBL, the latter being the most exhaustive option.
With ever-increasing numbers of high-quality genome assemblies processed with robust automated annotation pipelines, TrEMBL entries will contain higher quality protein sequences than in the past.
In other words, if a mammal species has 20 000 to 40 000 entries in UniProtKB and many of these are TrEMBL, users should be comfortable using all the protein entries to define their search space (more on this later when discussing proteomes at UniProtKB).
Determining the expected size of a well-annotated proteome requires additional knowledge, but tools to answer these questions continue to improve.
As more and more genome annotations are generated, the backlog of manual curation continues to increase.
However, automated genome annotations are also rapidly improving, blurring the line between Swiss-Prot and TrEMBL utility.</p>
<p>The second type of protein sequence collections available at UniProtKB are designated proteomes, with subclasses of “proteome”, “reference proteome” or “pan-proteome”.
As defined by UniProt, a proteome is the set of proteins derived from the annotation of a completely sequenced genome assembly (one proteome per genome assembly).
This means that a proteome will include both Swiss-Prot and TrEMBL entries present in a single genome annotation, and that all entries in the proteome can be traced to a single complete genome assembly.
This aids in tracking provenance as assemblies change, and metrics of these assemblies are available.
These metrics include Benchmarking Universal Single-Copy Ortholog (BUSCO) score, and “Completeness” as Standard, Close Standard or Outlier based on the Complete Proteome Detector (CPD).
Given the quality of genome annotation pipelines, using a proteome as a FASTA file for a species is the preferred method of defining search spaces now.
Outside of humans, no higher eukaryotic Swiss-Prot sequence collections are complete enough for use in proteomics analyses, but this does not mean that the available Swiss-Prot plus TrEMBL protein sequence collection precludes accurate proteomic data analysis.
Lastly, the difference between reference proteome and proteome is used to highlight model organisms or organisms of interest, but not to imply improved quality.
UniProt also has support for the concept of “pan proteomes” (consensus proteomes for a closely related set of organisms) but this is mostly used for bacteria (e.g., strains of a given species will share a pan proteome).</p>
<p>When retrieving protein sequence collections as Swiss-Prot/TrEMBL or designated proteomes, there is an option of downloading “FASTA (canonical)” or “FASTA (canonical &amp; isoform)”.
The later includes additional manually annotated isoforms for Swiss-Prot sequences.
Each Swiss-Prot entry has one canonical sequence chosen by the manual curator.
Any additional sequence variants (mostly from alternative slicing) are annotated as differences with respect to the canonical sequence.
Specifying “canonical” will select only one protein sequence per Swiss-Prot entry while specifying “canonical &amp; isoforms” will download additional protein sequences by including isoforms for Swiss-Prot entries.
Recently, an option to “download one protein sequence per gene (FASTA)” has been added.
These FASTA files include Swiss-Prot and TrEMBL sequences to number about 20 000 protein sequences for a wide range of higher eukaryotic organisms.</p>
<p>The number of additional isoforms varies considerably by species.
In the human, mouse, and rat proteomes of the total number of entries, 26 %, 40 % and 72 % are canonical, respectively.
The choice of including isoforms is related to the search algorithm and experimental goals.
For instance, if differentiating isoforms is relevant, they should be included otherwise they will not be detected.
In cases where isoforms are present in the FASTA (evident by shared protein names) but these cannot be removed prior to downloading (e.g., California sea lion, Zalophus californianus, proteome UP000515165, release 2022_01), non-redundant FASTA files can be manually generated (i.e., “remove_duplicates.py” via <span class="citation" data-cites="16BHXvDGS">[<a href="#ref-16BHXvDGS" role="doc-biblioref">511</a>]</span>).
If possible, retrieving canonical protein sequences via proteomes is the most straight forward approach and in general appropriate for most search algorithms, versus the method of searching and downloading Swiss-Prot and/or TrEMBL entries.</p>
<p>Though FASTA files are the typical input of many search algorithms, UniProt also offers an XML and GFF format download.
In contrast to the flat FASTA file format, the XML format includes sequence information as well as associated information like PTMs, which is used in some search algorithms like MetaMorpheus <span class="citation" data-cites="1GfGl6YGi">[<a href="#ref-1GfGl6YGi" role="doc-biblioref">512</a>]</span>.</p>
<p>Once a protein sequence collection has been selected and retrieved, there is the evergreen question of how to name and report this to others in a way that allows them to reproduce the retrieval. The minimum reporting information is the taxon identified and number of sequences used <span class="citation" data-cites="1EbCD9r1v paTtvUb8">[<a href="#ref-1EbCD9r1v" role="doc-biblioref">506</a>,<a href="#ref-paTtvUb8" role="doc-biblioref">507</a>]</span>. The following naming format (and those below) augments this and is suggested for UniProtKB FASTA files (the use of underscores or hyphens is not critical):
[common or scientific name]-[taxon id]-uniprot-[swiss-prot/trembl/proteome]-[UP# if used]-[canonical/canonical plus isoform]-[release]
example of a Homo sapiens (human) protein fasta from UniProtKB:</p>
<p>Human-9606-uniprot-proteome-UP000005640-canonical-2022_01.fasta</p>
<p>The importance of the taxon identifier has already been described above and is a consistent identifier across time and shared across resources. The choices of Swiss-Prot and TrEMBL in some combination was discussed above, and Proteome can be “proteome”, “reference proteome” or “pan-proteome”. The proteome identifier (‘UP’ followed by 9 digits) is conserved across releases, and release information should also be included. A confusing issue to newcomers is what the term “release” means. This is a year_month format (e.g., 2022_01), but it is not the date a FASTA file was downloaded or created, nor does it imply there are monthly updates. This release “date” is a traceable release identifier that is listed on UniProt’s website. Including all this information ensures that the exact provenance of a FASTA file is known and allows the FASTA file to be regenerated.</p>
<h5 id="refseq">RefSeq</h5>
<p>NCBI is a clearing house of numerous types of data and databases.
Specific to protein sequence collections, NCBI Reference Sequence Database (RefSeq) provides annotated genomes across the Tree of Life.
The newly developed NCBI Datasets portal <span class="citation" data-cites="uPHDl2Vm">[<a href="#ref-uPHDl2Vm" role="doc-biblioref">513</a>]</span> is the preferred method for accessing the myriad of NCBI data products, though protein sequence collections can also be retrieved from RefSeq directly<span class="citation" data-cites="tg8zlLqI xpZWsuGI">[<a href="#ref-tg8zlLqI" role="doc-biblioref">514</a>,<a href="#ref-xpZWsuGI" role="doc-biblioref">515</a>]</span>.
Like UniProt described above, most of the additional functionality and information available through NCBI Datasets and RefSeq will not be described here, although the Eukaryotic RefSeq annotation dashboard <span class="citation" data-cites="BfmLo0uT">[<a href="#ref-BfmLo0uT" role="doc-biblioref">516</a>]</span> is a noteworthy resource to monitor the progress of new or re-annotations.
We recommend exploring the resources available from NCBI <span class="citation" data-cites="fWjGnKWy">[<a href="#ref-fWjGnKWy" role="doc-biblioref">517</a>]</span>, utilizing their tutorials and help requests.</p>
<p>RefSeq is akin to the “proteome” sequence collection from UniProtKB, where a release is based on a single genome assembly.
If a more complete genome assembly is deposited or additional secondary evidence (e.g., RNA sequencing) is deposited, RefSeq can update the annotation with a new annotation release.
Every annotation release will have an annotation report that contains information on the underlying genome assembly, the new genome annotation, secondary evidence used, and various statistics about what was updated.
The current annotation release is referred to as the “reference annotation”, but each annotation is numbered sequentially starting at 100 (the first release).
Certain species are on scheduled re-annotation, like human and mouse, while other species are updated as needed based on new data and community feedback (ex. release 100 of taxon 9704 was in 2018, but a more contiguous genome assembly resulted in re-annotation to release 101 in 2020).
This general process for new and existing species is described in Heck and Neely <span class="citation" data-cites="17gB3bs6x">[<a href="#ref-17gB3bs6x" role="doc-biblioref">518</a>]</span>.</p>
<p>Since RefSeq is genome assembly-centric, its protein sequence collections are retrieved for each species.
This contrasts with being able to use a higher-level taxon identifier like 40674 (Mammalia) in UniProt to retrieve a single FASTA.
To accomplish this same search in NCBI Datasets requires a Mammalia search, followed by browsing all 2083 genomes and then filtering the results to reference genomes with annotations, and those resulting 188 could be bulk downloaded, though this will still be 188 individual FASTA files.
It is possible to download a single FASTA from an upper-level taxon identifier using the NCBI Taxonomy Browser, though this service may be redundant with the new NCBI Datasets portal.
Given the constant development of NCBI Datasets, these functionalities may change, but the general RefSeq philosophy of single species FASTA should be kept in mind.
Likewise, when retrieving genome annotations there is no ability to specify canonical entries only, but it is possible to use computational tools to remove redundant entries (“remove_duplicates.py” from <span class="citation" data-cites="16BHXvDGS">[<a href="#ref-16BHXvDGS" role="doc-biblioref">511</a>]</span>).</p>
<p>Similar to the UniProtKB FASTA file naming suggestion, the following naming format is suggested for RefSeq protein sequence collection FASTA (the use of underscores or hyphens is not critical):
[common or scientific name]-[taxon id]-refseq-[release number]
example of a Equus caballus (horse) protein FASTA from RefSeq:
Equus_caballus-9796-refseq-103.fasta
The release number starts at 100 and is consecutively numbered.
Note, the human releases only recently began following this consecutive numbering for Release 110, and previously had a much longer number to be included (e.g., NCBI Release 109.20211119).
Also, in a few species (Human and Chinese hamster, currently), there is a reference and an alternate assembly, both with an available annotation.
In these cases, including the underlying assembly identifier would be needed. Note that when you retrieve the protein FASTA from NCBI it will include two more identifiers that aren’t required in the file name since it can be determined from the taxon identifier and release number.
These are the genome assembly used (this is generated by the depositor and follows no naming scheme) and the RefSeq identifier (GCF followed by a number string).
These aren’t essential for FASTA naming, but are for comparing between UniProt, RefSeq and Ensembl when the same underlying assembly is used (or not, indicating how up to date one is versus the other).</p>
<h5 id="ensembl">Ensembl</h5>
<p>There are two main web portals for Ensembl sequence collections: the Ensembl genome browser <span class="citation" data-cites="69obv795">[<a href="#ref-69obv795" role="doc-biblioref">519</a>]</span> has vertebrate organisms and the Ensemble Genome project <span class="citation" data-cites="1EGyqGdLr">[<a href="#ref-1EGyqGdLr" role="doc-biblioref">520</a>]</span> has specific web portals for different non-vertebrate branches of the Tree of Life.
This contrasts with NCBI and UniProt where all branches are centrally available.
Recently, Ensembl has created a new portal “Rapid Release” focusing on quickly making annotations available (replacing the “Pre-Ensemble” portal), albeit without the full functionality of the primary Ensembl resources.
Overall, Ensembl provides diverse comparative and genomic tools that should be explored, but, specific to this discussion, they provide species-specific genome annotation products similar to RefSeq.</p>
<p>To retrieve a protein sequence collection from Ensemble at any of the portals, a species can be searched using a name, which will then have taxon identifier displayed (but searching by identifier is not readily apparent).
From the results you can select your species and follow links for genome annotation.
Caution should be used when browsing the annotation products since the protein coding sequence (abbreviated “cds”) annotations are nucleic acid sequences (a useable via 3-frame translation if using certain software), while actual translated peptide sequences are in the “pep” folders.
The pep folders contain file names with “ab initio” and “all” in the FASTA file names (file extensions are “fa” for FASTA and “gz” indicating gzip compression algorithm), while there may only be one pep product for certain species in the “Rapid Release” portal.
The “ab initio” FASTA files contain mostly predicted gene products.
The “all” FASTA files are the usable protein sequence collections.
Ensembl FASTA files usually have some protein sequence redundancy.</p>
<p>Ensembl provides a release number for all the databases within each portal.
Similar to the UniProt file naming suggestion, the following naming format is suggested for Ensembl protein sequence collection FASTA (the use of underscores or hyphens is not critical):</p>
<p>[common or scientific name]-[taxon id]-ensembl-[abinitio/all]-[rapid]-[release number]</p>
<p>example of a Sus scrofa (pig) protein FASTA from Ensembl:</p>
<p>Pig-9823-ensembl-all-106.fasta</p>
<p>Similar to the FASTA download from RefSeq, the downloaded file name can include additional identifying information related to the underlying genome assembly.
Again, this is not required for labeling, but is useful to easily compare assembly versions.</p>
<p>Since much of the data from Ensembl is also regularly processed into UniProt, using UniProt sequence collections instead may be preferred.
That said, they are not on the same release schedule nor will the FASTA files contain the same proteins.
Ensembl sequences still must go through the established protein sequence pipeline at UniProt to remove redundancy and conform to UniProt accession and FASTA header formats.
Moreover, the gene-centric and comparative tools built into Ensembl may be more experimentally appropriate and using an Ensembl protein sequence collection can better leverage those tools.</p>
<h4 id="other-resources">Other resources</h4>
<p>There are other locations of protein sequence collections, and these will likewise have different FASTA file formatting; sequences may have unusual characters, and formats of accessions and FASTA header lines may need to be reformatted to be compatible with search software.
These alternatives include institutes like the Joint Genome Institute’s microbial genome clearing house, species-specific community resource (e.g., PomBase, FlyBase, WormBase, TryTrypDB, etc.), and one-off websites tenuously hosting in-house annotations.
It is preferred to use protein sequence collection from the main three sources described here, since provenance can be tracked, and versions maintained.
It is beyond the scope of this discussion to address other genome annotation resources, how they are versioned, or the best way to describe FASTA files retrieved from those sources.
In these cases, defaulting to the minimum requirements of listing number of entries and supplying the FASTA along with data are necessary.</p>
<h4 id="contaminants">Contaminants</h4>
<p>Samples are rarely comprised of only proteins from the species of interest.
There can be protein contamination during sample collection or processing.
This may include proteins from human skin, wool from clothing, particles from latex, or even porcine trypsin itself, all of which contain proteins that can be digested along with the intended sample and analyzed in the mass spectrometer.
Avoiding unwanted matching of mass spectra originating from contaminant proteins to the cellular proteins due to sequence similarities is important to the identification and quantitation of as many cellular proteins as possible.
To avoid random matching, repositories of supplementary sequences for contaminant proteins have been added to a reference database for MS data searches.
Appending a contaminants database to the reference database allows the identification of peptides that are not exclusive to one species.
Peptides that are exclusive to the organism of interest are used to calculate abundance to avoid inflated quantitative results due to potential contaminant peptides.</p>
<p>As early as 2004, The Global Proteome Machine was providing a protein sequence collection of these common Repository of Adventitious Proteins (cRAP), while another contaminant list was published in 2008 <span class="citation" data-cites="OeXy2hrh">[<a href="#ref-OeXy2hrh" role="doc-biblioref">521</a>]</span>.
The current cRAP version (v1.0) was described in 2012 <span class="citation" data-cites="jwgBjKOg">[<a href="#ref-jwgBjKOg" role="doc-biblioref">522</a>]</span> and is still widely in use today.
cRAP is the contaminant protein list used in nearly all modern database searching software, though the documentation, versioning or updating of many of these “built-in” contaminant sequence collections is difficult to follow.
There is also another contaminant sequence collection distributed with MaxQuant.
Together, the cRAP and MaxQuant contaminant protein sequence collections are found in some form across most software, including MetaMorpheus and Philosopher (available in FragPipe) <span class="citation" data-cites="WdwXXrRx">[<a href="#ref-WdwXXrRx" role="doc-biblioref">523</a>]</span>.
This list of known frequently contaminating proteins can either be automatically included by the software or can be retrieved as a FASTA to be used along with the primary search FASTA(s).
Recently the Hao Lab has revisited these common contaminant sequences in an effort to update the protein sequences, test their utility on experimental data, and add or remove entries <span class="citation" data-cites="m8bjbzJ5">[<a href="#ref-m8bjbzJ5" role="doc-biblioref">524</a>]</span>.</p>
<p>In addition to these environmentally unintended contaminants, there are known contaminants that also have available protein sequence collections (or can be generated using the steps above) and should be included in the search space.
These can include the media cells were grown in (e.g., fetal bovine serum <span class="citation" data-cites="CnwgtVyY 1CIFGLtYQ">[<a href="#ref-CnwgtVyY" role="doc-biblioref">525</a>,<a href="#ref-1CIFGLtYQ" role="doc-biblioref">526</a>]</span>, food fed to cells/animals (e.g., Caenorhabditis elegans grown on Escherichia coli) or known non-specific binders in affinity purification (i.e., CRAPome <span class="citation" data-cites="wUllSafI">[<a href="#ref-wUllSafI" role="doc-biblioref">527</a>]</span>).
The common Repository of Fetal Bovine Serum Proteins (cRFP)<span class="citation" data-cites="EyM6Y0Xm">[<a href="#ref-EyM6Y0Xm" role="doc-biblioref">528</a>]</span> are protein lists of common protein contaminants and fetal serum bovine sequences used to reduced the number of falsely identified proteins in cell culture experiments.
Cells washed or cultured in contaminant free media before harvest or the collection of secreted proteins depletes most high abundance contaminant proteins but the sequence similarity between contaminant and secreted proteins can cause false identifications and overestimation of the true protein abundance leading to wasted resources and time on validating false leads.
As emphasized throughout this section, accurately defining the search space is essential for accurate results and, especially in the case of contaminants, requires knowledge of the experiment and sample processing to adequately define possible background proteins.</p>
<h3 id="choosing-the-right-database">Choosing the right database</h3>
<!-- add greater exploration of how to choose a database -->
<p>Proteomics data analysis requires carefully matching the search space (defined by the database choice) with the expected proteins.
A properly chosen database will minimize false positives and false negatives.
Choosing a database that is too large will increase the number of false positives, or decoy hits, which in turn will reduce the total number of identifiable proteins.
For this reason it is ill advised to search against all possible protein sequences ever predicted from any genomic sequence.
On the other hand, choosing a database that is too small may increase false negatives, or missed protein identifications, because in order for a protein to be identified it must be present in the database.
Thus, proteomics practitioners must do their best to predict the proteins that might be in their sample before they analyze their data.</p>
<p>Proteomics data analysis requires carefully aligning the search space with the expected proteome and the statistical approach of the search algorithm.
Search algorithms can self-correct when a database is overly large such that higher identity thresholds are required for identification to minimize false positives (e.g., Mascot), while smaller experiment-specific search spaces (also referred to as “subsets”) can have unintended effects on false positives if not managed appropriately <span class="citation" data-cites="qgZxt4hu 12o52Fhcj Lovg2liS">[<a href="#ref-qgZxt4hu" role="doc-biblioref">529</a>,<a href="#ref-12o52Fhcj" role="doc-biblioref">530</a>,<a href="#ref-Lovg2liS" role="doc-biblioref">531</a>]</span> or may even improve protein identifications <span class="citation" data-cites="nMWQwf9T">[<a href="#ref-nMWQwf9T" role="doc-biblioref">532</a>]</span>.
Whether to employ a search space that is sample-specific (i.e., subset), species-specific (with only canonical proteins, described below), exhaustive species-specific (including all isoforms), or even larger clade-level protein sequence set (e.g., the over 14 million protein sequences associated with Fungi, taxon identifier 4751) is a complex issue that is experiment and software dependent.
Moreover, in cases where no species-specific protein sequence collection exists, homology-based searching can be used (as described in <span class="citation" data-cites="17gB3bs6x">[<a href="#ref-17gB3bs6x" role="doc-biblioref">518</a>]</span>).
In each of these cases, proteomics practitioners must understand their specific experimental sample and search algorithm in order to know how to best define the search space, which is essential to yielding accurate results. See more discussion of database choice in the following section.</p>
<h2 class="page_break_before" id="biological-interpretation">Biological Interpretation</h2>
<p>The most common untargeted proteomics experiment will produce a list of proteins or peptides of interest which require further validation and biological interpretation.
This list usually results from statistical data analysis; the typical output of differentially expressed proteins usually contains hundreds of hits.
In this section, we aim to present a concise overview of how proteomic data can be effectively contextualised and used to generate new hypotheses.</p>
<p>The simplest approach is to start manual lookup of every protein in the list to uncover groups that function together.
Starting with a list of hundreds of protein changes, a smaller list can be prioritized by considering the level of significance and effect size.
For example, proteins with the smallest p-values (significance) and largest abundance fold-changes (effect size).
It is tempting to focus on proteins with the most extreme fold changes.
In this case, the assumption is that the more significant the fold change (in either direction, up- or down-regulation), the higher the impact of those proteins on cellular behavior.
This assumption is not always valid because protein signal in MS depends on abundance.
The manual data interpretation approach is typically infeasible due to the number of proteins that would need to be individually looked up one-by-one.</p>
<p>A better strategy is to use computational methods.
These methods may consider the whole list of proteins including some ranking by significance or fold change.
One common interpretation method is to construct a protein network, which then lends itself to network analyses.
Another method is to consider functional enrichment through annotation databases.
These databases offer insights by examining the enrichment of certain functional annotations amongst the interesting proteins.
Secondly, one could consider other evolutionary, structurally or regulatory based methods to identify interpretation of the data.
To fully interpret analysis, it may be required to perform or examine other data such as data from biophysical, biochemical and alternative proteomic approaches.
Finally, the data can further be interpreted using multi-omic, native or clinical approaches.
Below we summarize these approaches and point out potential pitfalls with these methods.</p>
<h3 id="constructing-a-protein-network">Constructing a protein network</h3>
<p>A network is a representation of the relations between objects.
Nodes are the entities of the network (e.g., users of a social platform, train stations, proteins), while edges are the connections between them (e.g., friendship, routes, and protein interactions, respectively).
In the case of protein-protein interactions, the evidence for the functional associations between proteins can be obtained experimentally.
For example, co-immunoprecipitation, crosslinking, and proximity labeling can be used to reveal physical interactions <span class="citation" data-cites="43dscula">[<a href="#ref-43dscula" role="doc-biblioref">533</a>]</span>.
The data is presented in a table with nodes and edges (e.g., “protein A interacts with protein B”) from which the network can be constructed.
A considerable wealth of protein-protein association data is stored in free databases like IntAct, which contain interactions derived from literature curation or direct user submissions <span class="citation" data-cites="xeKEaOVF">[<a href="#ref-xeKEaOVF" role="doc-biblioref">534</a>]</span>.
Protein interactions can also be predicted by classifiers that consider many features, like orthology and co-localization, to produce a posterior odds ratio of interaction <span class="citation" data-cites="D6gNmWdp uSY25K4H">[<a href="#ref-D6gNmWdp" role="doc-biblioref">535</a>,<a href="#ref-uSY25K4H" role="doc-biblioref">536</a>]</span>.
Finally, large repositories like STRING (Search Tool for the Retrieval of Interacting Genes/Proteins) collect and integrate protein-protein interaction data from several databases <span class="citation" data-cites="uSY25K4H">[<a href="#ref-uSY25K4H" role="doc-biblioref">536</a>]</span>.
STRING also provides a web-based interface to survey the data, and users only have to feed a search box with the identifiers of the protein(s) of interest.
STRING will retrieve the network and show the evidence supporting each interaction.
Importantly, these databases do not indicate the direction of the interaction, so they produce undirected networks.
If edges have directions (e.g., A influences B and not vice versa), then the network is directed.
Signaling pathways are examples of directed graphs.</p>
<h3 id="network-analysis">Network analysis</h3>
<p>Network analysis is a group of techniques that explore and investigate the network, yielding valuable knowledge about its structure and unveiling key players regulating the flow of information.
One of the first steps in network analysis relates to centrality measurements.
Centralities are indicators of the relative importance of a node corresponding to its position in the network, and each centrality measure provides new insights to interpret the data in new ways <span class="citation" data-cites="Es98brAH awUpRsoz">[<a href="#ref-Es98brAH" role="doc-biblioref">537</a>,<a href="#ref-awUpRsoz" role="doc-biblioref">538</a>]</span>.</p>
<h4 id="degree-centrality">Degree centrality</h4>
<p>The degree of a node measures the number of edges incident to that node.
Nodes with a high degree interact with many other nodes, called first neighbors.
In particular, the node degree distribution in protein networks is highly skewed, with most nodes having a low degree and a few having high degrees, known as hubs.
Hubs are usually regulatory proteins, being notable examples oncogenes and transcription factors.
Moreover, hubs are attractive targets for directed interventions, as their alteration has a profound effect on the stability of the network <span class="citation" data-cites="12ufdjSBQ">[<a href="#ref-12ufdjSBQ" role="doc-biblioref">539</a>]</span>.</p>
<h4 id="closeness-centrality">Closeness centrality</h4>
<p>The route from one node to another is a path, and the shortest path is the one connecting them in the least amount of steps.
Closeness centrality is the inverse of the average length of a node’s shortest paths to all other nodes in the network.
Nodes with a high closeness score have the shortest distances to all the others, so closeness centrality calculations detect nodes that can spread information very efficiently, as they are in a better position in the network for this task <span class="citation" data-cites="1FZ0vcua3 KQtBgFa0">[<a href="#ref-1FZ0vcua3" role="doc-biblioref">540</a>,<a href="#ref-KQtBgFa0" role="doc-biblioref">541</a>]</span>.</p>
<h4 id="betweenness-centrality">Betweenness centrality</h4>
<p>This centrality index is related to the amount of shortest paths transversing a node.
Nodes with a high betweenness centrality usually bridge different parts of the network and strongly influence the flow of information, as they lie in communication paths.
These connector hubs (or bottlenecks) are also interesting for follow–up experiments because their removal can disconnect different regions of the network <span class="citation" data-cites="1BmdBEJEB">[<a href="#ref-1BmdBEJEB" role="doc-biblioref">542</a>]</span>.</p>
<p>Centrality measurements add new layers of information and allow for ranking differentially expressed proteins apart from their fold-change in abundance.
Figure <a href="#fig:simple-network">4</a>X depicts a simple network consisting of proteins A to L, with A having the highest fold change (10) and L the lowest (2).
In Panel A, the fill color for the nodes indicates this metric, where it can be easily seen that A stands out.
However, protein A is a peripheral protein, only interacting with B.
In Panel B, nodes are colored according to node degree.
Clearly, protein F has the highest number of interactions and is also the closest to all other nodes, which can be appreciated when nodes are colored according to closeness centrality (Panel C).
On the other hand, protein G acts as a bridge between two regions of the network and thus, has the highest betweenness centrality (Panel D).
Except for fold change, node A has the lowest indices, and it will be up to the researcher to decide whether this protein warrants further examination.</p>
<div id="fig:simple-network" class="fignos">
<figure>
<img src="images/Simple%20network.png" style="width:100.0%" data-tag="4" alt="Figure 4: Analysis of a simple network using different centrality measurements. Nodes are colored according to each metric using a yellow-to-red gradient (yellow: lowest value, red: highest value). Network visualization and analysis were performed in Cytoscape." />
<figcaption aria-hidden="true"><span>Figure 4:</span> <strong>Analysis of a simple network using different centrality measurements.</strong>
Nodes are colored according to each metric using a yellow-to-red gradient (yellow: lowest value, red: highest value).
Network visualization and analysis were performed in Cytoscape.</figcaption>
</figure>
</div>
<h3 id="network-clustering">Network clustering</h3>
<p>In the small network presented in Figure <a href="#fig:simple-network">4</a>X, two groups of densely connected nodes exist.
This topology suggests that these communities (or “clusters”) work together or participate in a protein complex.
Dividing a network into clusters helps identify underlying relationships among nodes, which is especially useful in large networks.
In a broad sense, network clustering groups nodes according to a topological property, generally interconnectedness.
There are many network clustering algorithms, each with its own merits and approaches <span class="citation" data-cites="1H6bETSCb 11gPEmZWK">[<a href="#ref-1H6bETSCb" role="doc-biblioref">543</a>,<a href="#ref-11gPEmZWK" role="doc-biblioref">544</a>]</span>.
The MCL (Markov CLustering) algorithm is suitable for protein networks in most situations .
On the other hand, the Molecular COmplex DEtection (MCODE) algorithm helps detect very densely connected nodes, thus unveiling protein complexes <span class="citation" data-cites="iqdatwR1">[<a href="#ref-iqdatwR1" role="doc-biblioref">545</a>]</span>.
In this regard, network clustering is useful for tentatively assigning the function of an uncharacterized protein.
If the protein appears in a cluster, its function should be closely related to the cluster members, a principle known as “guilty by association.” <span class="citation" data-cites="WJOtOvRY">[<a href="#ref-WJOtOvRY" role="doc-biblioref">546</a>]</span></p>
<h3 id="network-visualization">Network visualization</h3>
<p>A critical step in network analysis is to display the data in a structured and uncluttered graph.
Networks can rapidly become a hairball unamenable to interpretation.
Software platforms like Cytoscape can be used to visualize networks orderly by applying layout algorithms and format styles <span class="citation" data-cites="8oE5lYNw">[<a href="#ref-8oE5lYNw" role="doc-biblioref">547</a>]</span>.
Since many of these platforms are open source, community-designed plugins enhance their capabilities.
In Cytoscape, the stringApp adds a search bar to query the STRING database with accession numbers or protein names <span class="citation" data-cites="jrtjRoft">[<a href="#ref-jrtjRoft" role="doc-biblioref">548</a>]</span>.
The network is directly retrieved into Cytoscape, where its built-in network analyzer can be used to calculate centralities.
Moreover, user-defined information, like fold-change values, can be integrated and mapped into the network.</p>
<h3 id="functional-term-enrichment-analysis-kegg-string-go-gsea-ora-reactome-and-others">Functional term enrichment analysis: KEGG, String, GO, GSEA, ORA, Reactome, and others</h3>
<!-- To-Do: this section is redundant with the next section -->
<p>Term enrichment analysis is performed to assess whether particular ‘functional terms’ are over-represented in a list of proteins (e.g. from a proteomics experiment) <span class="citation" data-cites="h0C6mgc4 CaFLiife QlVqCOZZ">[<a href="#ref-h0C6mgc4" role="doc-biblioref">549</a>,<a href="#ref-CaFLiife" role="doc-biblioref">550</a>,<a href="#ref-QlVqCOZZ" role="doc-biblioref">551</a>]</span>.
For example, after a differential abundance analysis, we may wish to examine whether there is any shared function amongst the proteins which were determined to have significant changes.
The simplest analysis to test whether this subset contains more of any particular functional terms than we would expect given the background of proteins.
For example, the Gene Ontology is split into the classes: Cellular Component, Molecular Function and Biological Function and we might be interested as to whether our proteins may be more likely to localize to a particular subcellular niche <span class="citation" data-cites="IFevKgND">[<a href="#ref-IFevKgND" role="doc-biblioref">552</a>]</span>.
The Cellular Component terms could give us a starting point if this might be the case, by examining if Cellular Component annotations are enriched.</p>
<p>There are a number of databases and tools to perform such analysis, which can even be extended to examine whole pathways, networks, post-translational modification and literature representation.
For example, databses such as KEGG <span class="citation" data-cites="15ZNlTPZi">[<a href="#ref-15ZNlTPZi" role="doc-biblioref">553</a>]</span>, String <span class="citation" data-cites="uSY25K4H">[<a href="#ref-uSY25K4H" role="doc-biblioref">536</a>]</span>, Reactome [PMID:31691815] and PhosphoSitePlus <span class="citation" data-cites="f3o81IaL">[<a href="#ref-f3o81IaL" role="doc-biblioref">554</a>]</span> can be used to test or annotate a list of proteins.
For example, proteomics analysis of human cardiac 3D microtissue exposed to anthracyclines (drugs used in cancer chemotherapy) unearthed several proteins with altered levels <span class="citation" data-cites="10dRpS3qN">[<a href="#ref-10dRpS3qN" role="doc-biblioref">555</a>]</span>.
Many of these were specifically grouped under GO terms related to mitochondrial dysfunction, indicating the detrimental effects of these drugs on the organelle.
GO terms <span class="citation" data-cites="IFevKgND">[<a href="#ref-IFevKgND" role="doc-biblioref">552</a>]</span> or descriptors from other annotation libraries (like KEGG <span class="citation" data-cites="15ZNlTPZi">[<a href="#ref-15ZNlTPZi" role="doc-biblioref">553</a>]</span> or REACTOME <span class="citation" data-cites="DhTe2WLI">[<a href="#ref-DhTe2WLI" role="doc-biblioref">556</a>]</span>) can be retrieved from STRING when constructing a network or from other freely available compendiums.
We refer to a number of articles on the topics, including tools, reviews and best-practice <span class="citation" data-cites="rlKsqE1k 1FmAqWcpa 7xIEHm2N">[<a href="#ref-rlKsqE1k" role="doc-biblioref">557</a>,<a href="#ref-1FmAqWcpa" role="doc-biblioref">558</a>,<a href="#ref-7xIEHm2N" role="doc-biblioref">559</a>]</span>.
The main points from such analysis is that we can obtain an insight about protein function by looking at whether our list of proteins have similar or the same annotations.
A number of limitations should be taken into account for interpretation.
The first is that proteins that are more abundant are more likely to be studied, measured and examined in the literature.
Hence, abundant proteins will have more annotations than less abundant ones.
One key part of the analysis is also to correctly select the background set; that is, the universe of protein which our list is being compared against.
By including contaminates or proteins that are not expressed in our system within the list, the results may be unfaithful.</p>
<p>We may also have access to our own curated set of annotations derived either computational or experimental.
One may be interested in seeing whether we have enrichment of these annotations amongst the differentially abundant proteins.
Our list of proteins could be divided into two groups: differentially abundant or not. These groups could be divided into whether they have a particular annotation: yes or no.
This information can be summarised in a two-by-two table, to which we can apply a statistical test to examine whether that annotation is enriched within our differentially abundant proteins.
One test that could be used is the hypergeometric test, and another would be a Fisher Exact test.</p>
<p>There are many methods for performing functional enrichment analysis on the data, but they can mainly be classified into three categories (Figure XI{fig:enrich_methods}), as follows.</p>
<div id="fig:enrich_methods" class="fignos">
<figure>
<img src="images/enrich_methods_v4.png" style="width:100.0%" data-tag="4" alt="Figure 4: Types of functional enrichment methods. In the volcano plot (left), proteins with altered values are colored blue or red according to arbitrarily chosen cut-off values for significance and fold change. Black bars or thick-bordered nodes indicate members of a GO category." />
<figcaption aria-hidden="true"><span>Figure 4:</span> <strong>Types of functional enrichment methods.</strong>
In the volcano plot (left), proteins with altered values are colored blue or red according to arbitrarily chosen cut-off values for significance and fold change.
Black bars or thick-bordered nodes indicate members of a GO category.</figcaption>
</figure>
</div>
<h4 id="over-representation-analysis">Over-representation analysis</h4>
<p>In a modern proteomics analysis, usually thousands of proteins are identified and quantified.
Fold-change and significance thresholds are chosen (e.g., fold-change ≥ 2 and p ≤ 0.05) to obtain a list of proteins with altered levels among the tested conditions.
In over-representation methods, a contingency table is created for every protein set to establish whether proteins with altered abundance show an enrichment or a depletion of the ontology term compared to the background observed proteome <span class="citation" data-cites="HNlvs4Yk">[<a href="#ref-HNlvs4Yk" role="doc-biblioref">560</a>]</span>.
For example, suppose that 2000 proteins were quantified in a proteomics analysis, being 40 of these members of the set “tricarboxylic acid cycle (TCA).”
Also, let us assume that 200 proteins showed altered abundance, with 15 belonging to the TCA set.
Then, the contingency table can be constructed as follows:</p>
<table>
<colgroup>
<col style="width: 25%" />
<col style="width: 31%" />
<col style="width: 33%" />
<col style="width: 9%" />
</colgroup>
<thead>
<tr class="header">
<th></th>
<th style="text-align: center;"><strong>Proteins with altered abundance</strong></th>
<th style="text-align: center;"><strong>Proteins with unaltered abundance</strong></th>
<th style="text-align: center;"><strong>Total</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>Proteins in TCA set</strong></td>
<td style="text-align: center;">15</td>
<td style="text-align: center;">25</td>
<td style="text-align: center;">40</td>
</tr>
<tr class="even">
<td><strong>Proteins not in TCA set</strong></td>
<td style="text-align: center;">185</td>
<td style="text-align: center;">1775</td>
<td style="text-align: center;">1960</td>
</tr>
<tr class="odd">
<td><strong>Total</strong></td>
<td style="text-align: center;">200</td>
<td style="text-align: center;">1800</td>
<td style="text-align: center;">2000</td>
</tr>
</tbody>
</table>
<p>Then, a suitable statistical test is conducted to ascertain if proteins with altered levels are enriched in members of the TCA cycle (in this case, they are; p &lt; 0.00001).
This is commonly achieved using Fisher’s exact test <span class="citation" data-cites="TWAPY3zr">[<a href="#ref-TWAPY3zr" role="doc-biblioref">561</a>]</span>.
The process is then repeated for every set as desired.
Since multiple comparisons are tested, p values must be adjusted by a false discovery rate <span class="citation" data-cites="ArmIWAfa">[<a href="#ref-ArmIWAfa" role="doc-biblioref">562</a>]</span>.</p>
<h4 id="functional-class-scoring">Functional class scoring</h4>
<p>The caveat of over-representation methods is that they rely on a list of differentially expressed genes or proteins with altered abundance, selected due to arbitrarily chosen cut-off values.
For example, if we set a fold change cutoff of 2, a protein with a fold-change of 1.99 would not be included in the analysis.
Moreover, several proteins belonging to the same set may have altered levels but are below the fold change threshold.
However, moderate alterations of their abundance as a group could drive the observed phenotype, even more so than a single protein over the cut-off.
Functional class scoring strategies aim at countering these limitations by disregarding thresholds altogether.
GSEA (Gene Set Enrichment Analysis) is a widely used functional class scoring method in which all detected entities are first ranked according to a quantitative measurement (fold change, p-value, or their combination) <span class="citation" data-cites="tt0Vdf4C">[<a href="#ref-tt0Vdf4C" role="doc-biblioref">563</a>]</span>.
Then, the distribution of members of a set is obtained.
A scoring scheme based on the Kolmogorov – Smirnov test is used to assess whether there is an enrichment of the category towards the top or bottom of the ranked list.</p>
<h4 id="pathway-topology-based-methods">Pathway topology-based methods</h4>
<p>Both methods mentioned above do not consider the functional relationships among proteins put forth by network analysis; i.e., they assume functional independence.
Topology-based enrichment methods incorporate this information by, for example, assigning an importance value to a set when its members also participate in a pathway or cluster together in a network <span class="citation" data-cites="6gI0wLAR">[<a href="#ref-6gI0wLAR" role="doc-biblioref">564</a>]</span>.
In Figure <a href="#fig:enrich_methods">4</a>XI, it can be shown that topology-based methods consider non-significant hits (grey nodes) that other strategies may not pick up, due to their position in a network.</p>
<h3 id="other-computational-approaches-network-analysis-isoform-correlation-analysis-alphafold-blast-protein-language-models">Other computational approaches: Network analysis, Isoform correlation analysis, AlphaFold, BLAST, protein language models</h3>
<p>Additional computational analysis of a list of interesting proteins may uncover additional substructure, correlation or biologically useful hypothesis.
Building a network between the proteins based on the experiments performed, might be a useful approach to identify additional structure.
For example, co-expression network analysis can be used to build a network from these proteins <span class="citation" data-cites="1BeN7jMiD">[<a href="#ref-1BeN7jMiD" role="doc-biblioref">565</a>]</span>.
In these networks, proteins are nodes and edges describe relationships between those proteins.
Network-specific methods can then be applied, such as community detection algorithms which could uncover clusters of proteins with shared functions <span class="citation" data-cites="15Ki75h8 fXCPx62A">[<a href="#ref-15Ki75h8" role="doc-biblioref">566</a>,<a href="#ref-fXCPx62A" role="doc-biblioref">567</a>]</span>.</p>
<p>One way the proteome generates complexity is through alternative-splicing, which results in protein isoforms <span class="citation" data-cites="eZTqkAjM">[<a href="#ref-eZTqkAjM" role="doc-biblioref">568</a>]</span>.</p>
<p>Recently, a number of tools have been proposed to identify peptide isoforms that are quantitatively different across conditions by using a principle called peptide correlation analysis <span class="citation" data-cites="kBc66YMI YKUlYryb">[<a href="#ref-kBc66YMI" role="doc-biblioref">569</a>,<a href="#ref-YKUlYryb" role="doc-biblioref">570</a>]</span>.
The idea is that the quantitative behavior of peptides should match each other.
If there are subgroups that behave coherently within the group but not across groups suggest that peptide may have come from a different proteoform.
These approaches can be used to identify specific proteoforms that are functional across different conditions.</p>
<p>For many, a protein’s structure reveals important functional details <span class="citation" data-cites="vegaUWlG">[<a href="#ref-vegaUWlG" role="doc-biblioref">571</a>]</span>.
There are a plethora of approaches to predict a protein’s structure <span class="citation" data-cites="PnnbHO9a yZfcMIwh 1AwSDaTyL">[<a href="#ref-PnnbHO9a" role="doc-biblioref">572</a>,<a href="#ref-yZfcMIwh" role="doc-biblioref">573</a>,<a href="#ref-1AwSDaTyL" role="doc-biblioref">574</a>]</span>.
Recently, AlphaFold and RoseTTAFold have become dominant methods for predicting protein structures with high resolution <span class="citation" data-cites="yZfcMIwh 1AwSDaTyL">[<a href="#ref-yZfcMIwh" role="doc-biblioref">573</a>,<a href="#ref-1AwSDaTyL" role="doc-biblioref">574</a>]</span>.
If intrinsically disordered domains are of particular interest, methods explicitly designed for this task are recommended <span class="citation" data-cites="tIYr5DF7">[<a href="#ref-tIYr5DF7" role="doc-biblioref">575</a>]</span>.
Once a structure is obtained more elaborate computational methods might be useful such as docking or molecular dynamics <span class="citation" data-cites="CRgkwTQA iOQeUEPU">[<a href="#ref-CRgkwTQA" role="doc-biblioref">576</a>,<a href="#ref-iOQeUEPU" role="doc-biblioref">577</a>]</span>.
These approaches can give insight into how protein or molecules fit together and the dynamics of a protein’s structure (conformational heterogeneity).
A complete discussion of these topics is beyond the scope of this section.</p>
<p>Another way to obtain insights into a protein function is to look for protein with similar sequences or motifs.
Using BLAST, a sequence alignment tool, one can align two or more protein sequences and determine their level of similarity <span class="citation" data-cites="jcOtxABy">[<a href="#ref-jcOtxABy" role="doc-biblioref">578</a>]</span>.
For example, if a human protein of unknown function has a similar sequence to a yeast protein with known function this may be a starting place for the putative function of that protein.</p>
<p>Novel approaches to representing the similarity of proteins have proved successful at predicting the functional properties of proteins.
Protein language models seek to learn “representation” of proteins, these are usually numerical vectors that represent a protein sequence <span class="citation" data-cites="daqSjHIW nf5EVAMJ">[<a href="#ref-daqSjHIW" role="doc-biblioref">579</a>,<a href="#ref-nf5EVAMJ" role="doc-biblioref">580</a>]</span>.
Abstractly, these vectors preserve protein similarity or a notion of “proteinness”. This usually means that two proteins that have a close vector may share similarities in protein function.
These representations are also advantageous because they can easily become the inputs for machine learning algorithms to predict valuable protein properties; for example, thermal stability values <span class="citation" data-cites="wLlg6SzE">[<a href="#ref-wLlg6SzE" role="doc-biblioref">581</a>]</span>, protein-protein binding affinities <span class="citation" data-cites="YBdYtUv">[<a href="#ref-YBdYtUv" role="doc-biblioref">582</a>]</span>, secondary protein structure, and more.</p>
<h2 id="orthogonal-experimental-methods">Orthogonal experimental methods</h2>
<h3 id="the-importance-of-orthogonal-experimental-validation">The importance of orthogonal experimental validation</h3>
<p>The computational workflows to interpret mass spectrometry data are sophisticated, powerful tools, but also show important limitations and caveats due to their dependence on limited prior knowledge, specific experimental parameters or data quality restraints (see section ‘Analysis of Raw Data’).
These inherent biases can give rise to ambiguous or spurious interpretation of the data even when these workflows are applied correctly and to the best of the experimenter’s knowledge.
Therefore, researchers will oftentimes be asked by scientific journals to provide independent orthogonal validation of their proteomics data and not performing such can be a major roadblock in the publication process.</p>
<p>The aim of validating data obtained by proteomics approaches should always be two-fold by demonstrating that the conclusions arrived at by proteomics data acquisition and analysis are, firstly, valid and, secondly, relevant.
Depending on the question at hand, researchers can draw on an overabundance of techniques to validate MS-derived hypotheses in appropriate cellular, organismal or in vitro models.
In the the following paragraphs we aim to present only a high-level, stringent, non-exhaustive selection of orthogonal validation approaches and emphasise the importance of implementing assays that challenge assumptions gained from proteomics data analysis pipelines.</p>
<p>Before embarking on orthogonal validation of any hit, the success of the experiment should be established by assessing (internal) positive controls.
Internal positive controls can be proteins whose behaviour under the experimental conditions applied can be deduced from prior knowledge (i.e. the scientific literature or public databases).
Once the expected changes in internal controls have been confirmed by computational analysis (see the above section), the orthogonal experimental validation of novel, perhaps unexpected findings can begin.</p>
<p>Orthogonal validation of new insights obtained from quantitative proteomics experiments can be a very time-consuming process and often requires familiarity with techniques not directly related to proteomics workflows.
Given these challenges, the method(s) of choice warrant(s) careful consideration and is highly context-dependent.
Importantly, proteomics experiments in one way or another generally yield comprehensive lists of potentially interesting candidate proteins or pathways, the researcher will have to shortlist candidates to be taken forward to the validation stage of the project.
Which candidates should you validate by an orthogonal approach and which ones might not require further validation?</p>
<p>In general, candidates representing abundant proteins that show high sequence coverage and are detected with high confidence might not necessarily need extensive orthogonal validation when compared with proteins of intermediate to low abundance that might be more challenging to faithfully quantify by proteomics alone (i.e. many membrane proteins or transcription factors).
Similarly, since the proteome is rarely comprehensively quantified in any single proteomics experiment, proteins of interest (POIs) that are critical for an observed biological change might not be part of the dataset.
In these cases, additional, targeted analyses might help to support or discredit proteomics-based hypotheses.</p>
<p>Validation techniques are as manifold as biological questions and discussions thereof may easily fill multiple textbooks.
The following sections are therefore merely meant to paint with a broad brush stroke a picture of useful methodolgies with which to validate and follow up MS-data derived observations.
As this is meant to orient the reader, whereever possible, we will explicitly point out useful literature reviews for a deeper dive into each of these techniques.</p>
<h3 id="general-considerations">General considerations</h3>
<p>Once POIs have been selected based on prior agreed-upon selection criteria (i.e. (adjusted) p value and/or fold change thresholds), orthogonal validation experiments should ideally be conducted under physiologically relevant conditions to mitigate artifical and misleading outcomes.
Therefore, in vitro experiments, while useful to isolate and dissect particular aspects of a biological system, can give highly artificial results as conditions are far removed from the POI’s native environment.
To investigate the biological function of a protein or pathway, direct genetic manipulation of the biological system at hand (e.g. modulating the expression of a POI by overexpression or knockout-/down experiments) can be minimally invasive when performed correctly.
Should the POI be encoded by an essential gene, by definition, a complete and stable knockout might not be advisable <span class="citation" data-cites="bUq8ZzIL 171wjW4dP">[<a href="#ref-bUq8ZzIL" role="doc-biblioref">583</a>,<a href="#ref-171wjW4dP" role="doc-biblioref">584</a>]</span>.
In these extreme cases, attenuated expression (i.e. using RNA interference (RNAi) or controlled degradation, see below) rather than complete repression of a gene can be used to probe for protein function.
Epitope tagging and/or exogenous expression of a gene of interest can be a powerful approach in assessing PPIs and investigating proteins of low abundance.
However, overexpression artifacts are common <span class="citation" data-cites="pGJ2GFbE">[<a href="#ref-pGJ2GFbE" role="doc-biblioref">585</a>]</span>.</p>
<p>It is not always possible to fully avoid the pleiotropic effects of protein (over-)expression or depletion, but a number of mitigation strategies (i.e. inducible expression, the use of multiple independent RNAi strategies) will be discussed below.</p>
<p>Extensive biochemical characterization of any overexpressed gene is critical to ensure it closely reflects the functions of its endogenous counterpart.
These assays might involve assessing protein localization (i.e. by imaging techniques such as microscopy and flow cytometry), protein abundance (i.e. by mass spectrometry or immunoblot analysis) and phenotypic assays where applicable and practical.</p>
<h3 id="functional-genomics-techniques-in-the-validation-of-ms-hits">Functional genomics techniques in the validation of MS hits</h3>
<p>Typical follow-up experiments to validate mass-spectrometry derived insights often involve the acute depletion or induction of a POI and assessing the impact on specific cellular phenotypes.
Here we present a selection of methodologies to effectively modulate gene expression and discuss important considerations when planning functional genomics experiments for target validation.</p>
<p>Gene deletion or knockdown to prevent production of a functional protein is a powerful means to interrogate the role of one or more proteins in the phenotype(s) under investigation.
To this end, well-established technologies deserving mention at this point are RNA interference (RNAi) in the form of siRNA/shRNA- or miRNA-mediated gene knockdown abd CRISPR/Cas9-or TALEN-mediated gene knockout <span class="citation" data-cites="DdiSq79q">[<a href="#ref-DdiSq79q" role="doc-biblioref">586</a>]</span>.
Since each one of these technologies comes with their own unique advantages and caveats, the approach taken depends on the biological question at hand.</p>
<p>Clustered regularly interspaced short palindromic repeats (CRISPR)/Cas-based gene deletion technologies allow for the targeting of individual genes with relative ease, high efficiency and specificity <span class="citation" data-cites="15T32hvm6">[<a href="#ref-15T32hvm6" role="doc-biblioref">587</a>]</span>.
When expressed in mammalian cells, the bacterially-derived Cas9 endonuclease can be guided with the help of a short guide RNA (gRNA) to a genomic location of interest, where it creates a DNA double strand break in a highly controlled manner (for a detailed discussion see <span class="citation" data-cites="hRIRSidU">[<a href="#ref-hRIRSidU" role="doc-biblioref">588</a>]</span>).
The cell’s DNA double-stand break repair machinery then introduces base pair insertions or deletions (indels) via non-homologous-end-joining (NHEJ), thus causing missense, and frameshift mutations (i.e. resulting in premaure stop codons), leading to premature termination of gene expression or non-functional, aberrant gene products.
Similarly, the concomitant provision of a complementary DNA donor template encoding a desired gene modification (i.e. insertion of a stretch of DNA or base pair modification) will trigger homology-directed repair (HDR), resulting in gene knockin or base editing <span class="citation" data-cites="hRIRSidU">[<a href="#ref-hRIRSidU" role="doc-biblioref">588</a>]</span>.<br />
Practical considerations of CRISPR/Cas9-mediated gene knock-in and base editing will not be addressed in detail but are expertly discussed in <span class="citation" data-cites="1uwXNnEt EPXqJmJj rUUeN4Sh VFV60BPa">[<a href="#ref-1uwXNnEt" role="doc-biblioref">589</a>,<a href="#ref-EPXqJmJj" role="doc-biblioref">590</a>,<a href="#ref-rUUeN4Sh" role="doc-biblioref">591</a>,<a href="#ref-VFV60BPa" role="doc-biblioref">592</a>]</span>.</p>
<p>The relative ease-of-use and high efficiency of the CRISPR/Cas9 gene editing technology has rendered it the method of choice for gene manipulation in many fields of cell biology.
However,it should be noted that CRISPR/Cas9-mediated gene deletion is not free from off-target effects (<span class="citation" data-cites="sAa18GbX">[<a href="#ref-sAa18GbX" role="doc-biblioref">593</a>]</span> for advice on how to minimise these off-target effects).
Moreover, long-term depletion (or upregulation) of a POI itself can in some cases have dramatic systemic consequences and constitute an acute selection pressure leading to compensatory stress-induced adaptation that might obfuscate primary loss-of-function phenotypes and pose a substantial hurdle to the interpretability of biological data.
As these compensatory mechanisms often manifest with time, controlled, transient genetic manipulation (gene depletion or transgene expression) is advised.
Small interfering RNA (siRNA)-mediated knockdown by transient transfection is typically achieved at shorter time frames (24 – 96h), depending on the turnover of the POI.
On an even shorter time-scale, targeted, degron-based degradation systems enable depletion of a POI within minutes and further reduce off-target effects, but require the exogenous expression of a transgene and therefore some genetic manipulation.
A more comprehensive discussion of a selection of these systems (anchor-away, deGradFP, auxin-inducible degron (AID), dTAG technologies) and their advantages and potential pitfalls is presented in <span class="citation" data-cites="n2p9cQ7">[<a href="#ref-n2p9cQ7" role="doc-biblioref">594</a>]</span>.</p>
<p>Multiple eukaryotic and prokaryotic transcription-based systems have been developed that allow for the controlled biosynthesis or depletion of one or more POIs.
Amongst these, a popular and dependable choice for mammalian cells are tetracycline-controlled operon systems, which allow up- or downregulation of a POI in the presence of the antibiotic tetracycline or its derivative doxycycline.
These systems rely on the insertion of a bacteria-derived Tet operon (TetO) between the promoter and coding sequence of a GOI.
In this configuration, the TetO binds a co-expressed Tet-repressor protein blocking transcription the of GOI.
When tetracycline is added to the cells, the repressor then dissociates from the operon, thus de-repressing the GOI.
Different variations of this potent system exist, allowing for more flexibility in experimental design.
For instance, in the Tet-OFF system, the Tet repressor is fused to a eukaryotic transactivator (the chimeric fusion construct is termed tTA) and addition of tetracycline, or the related doxycycline, abolishes TetO binding and thus suppresses transcriptional activation <span class="citation" data-cites="1G6PkBlEo">[<a href="#ref-1G6PkBlEo" role="doc-biblioref">595</a>]</span>.
Alternatively, a mutant form of tTA (rtTA) binds the TetO only in the presence of tetracycline, allowing for tetracycline-induced gene expression.
For a detailed discussion of these systems, we refer the reader to an excellent review <span class="citation" data-cites="6NVK2tsy">[<a href="#ref-6NVK2tsy" role="doc-biblioref">596</a>]</span>.</p>
<p>When generating stable expression cell lines, being able to precisely control the genomic integration site of the transgene reduces overall genetic heterogeneity in a cell population and thereby reduces potential off-target or pleiotropic effects.
This ability is realised in the FlpIn-T-REx technology which harnesses Flp-recombinase mediated DNA recombination at a strictly defined genomic locus (the FRT site) <span class="citation" data-cites="idh43eOQ">[<a href="#ref-idh43eOQ" role="doc-biblioref">597</a>]</span>.
Site-directed isogenic integration of any GOI at the FRT site, which is under a tetracycline-inducible promoter and a hygromycin resistance gene, allows for facile generation of tetracycline/doxycycline-inducible isogeneic expression cell lines with minimal leaky expression (for an example, see <span class="citation" data-cites="S1IrclG6">[<a href="#ref-S1IrclG6" role="doc-biblioref">598</a>]</span>).</p>
<h3 id="validation-and-interpretation-of-protein-abundance-changes">Validation and interpretation of protein abundance changes</h3>
<p>To validate protein abundance changes observed by quantitative bottom-up proteomics or simply assess the success of targeted genetic manipulation as part of an orthogonal follow-up experiment (see above), the experimenter typically resorts to antibody-based techniques such as immunoblotting analysis or immunofluorescence and immunohistological imaging of POIs.
The latter also allows for validation of protein expression and localisation in intact tissue or cells.
However, these semi-quantitative methods are strongly influenced by the quality of the antibodies used and might not be sensitive enough to detect small changes in protein levels.
In this case, more accurate orthogonal quantitation of proteins might be achieved by stable isotope labelling (SILAC/TMT/iTRAQ) and/or SRM/PRM (see section ‘Experiment Types’).
SDS-PAGE and immunoblot analysis are powerful and facile low-throughput tools to quickly validate protein abundance changes.
However, short of introducing epitope tags to the endogenous POI, the success of immunoblotting is contingent on the availability of specific antibodies, which can present a formidable problem when investigating poorly characterised proteins or working with model organisms for which the commercial availability of specific antibodies is limited (this is particularly problematic for ‘unconventional’ or even well-establishedmodel organisms such as yeast).
A detailed discussion of the strengths and pitfalls of immunoblotting for validation of semi-quantitative proteomics data can be found in an excellent review by Handler <em>et al.</em> <span class="citation" data-cites="BwuqP6B4">[<a href="#ref-BwuqP6B4" role="doc-biblioref">599</a>]</span>.</p>
<p>Protein abundance changes detected in a proteomics experiment can be the result of a range of different cellular processes.
The abundance of a protein in a complex sample (e.g. cell lysate or biological fluid) directly reflects a combination of the protein’s intrinsic stability and the translational rate under the conditions of interest.</p>
<p>Both protein stability as well as gene expression activity can be quantified independently.
Altered protein stability might be a direct consequence of specific or global changes in protein turnover.
Radioisotope labelling is a well-established, accurate way to monitor protein synthesis, maturation and turnover <span class="citation" data-cites="gFNDbBK5 TJFHVsUt">[<a href="#ref-gFNDbBK5" role="doc-biblioref">600</a>,<a href="#ref-TJFHVsUt" role="doc-biblioref">601</a>]</span>.
This ‘pulse-chase’ methodology relies on the incorporation (‘pulsing’) of radioisotopes (typically 35S-labelled cysteine and methionine) into de-novo synthesised proteins.
Upon withdrawal of the labelleled amino acids from the culture medium, the decay of signal is monitored over time (‘the chase’) by SDS-PAGE and phosphoimaging, resulting in a temporal readout of protein abundances.
The advantage of this technology is that a subpopulation (newly synthesised proteins) can be monitored directly, giving an accurate assessment of protein stability.
Once a change in protein stability has been validated, the underlying mechanisms can be addressed by inhibiting protein degradation pathways; prominently proteasome-mediated degradation (using specific proteasome inhibitors such as bortemzomib/velcade or MG132), autophagy (pharmacologically inhibiting autophagic flux) or degradation by proteases (using protease inhibitors).
The type of radiolabeling described above is relatively labor-intense, of low-throughput and has the obvious disadvantage of requiring radioactive material, which needs to be handled under strict safety precautions.
Moreover, it critically depends on the presence of one or more methionines and/or cysteines in the POIs.</p>
<p>It is also possible to measure protein stability within complex protein mixtures (i.e. cell lysates or biological fluids) using an array of specialized mass spectrometry techniques as discussed in <span class="citation" data-cites="T6IYjbu4">[<a href="#ref-T6IYjbu4" role="doc-biblioref">602</a>]</span> and <span class="citation" data-cites="8UcGwfym">[<a href="#ref-8UcGwfym" role="doc-biblioref">87</a>]</span>.</p>
<p>For purified proteins, well-established in vitro spectrometric and calorimetric methods such as circular dichroism, differential scanning calorimetry or differential scanning fluorometry can be used, but the relatively high sample amounts might be restrictive.</p>
<p>Finally, gene expression changes can also be determined with high fidelity using quantitative real-time PCR (qRT-PCR) or RNA-Seq can measure changes in gene transcription or mRNA turnover (for an extensive discussion of both technologies, please see <span class="citation" data-cites="fYmIhIvC">[<a href="#ref-fYmIhIvC" role="doc-biblioref">603</a>]</span> and <span class="citation" data-cites="9ByCq3fI">[<a href="#ref-9ByCq3fI" role="doc-biblioref">604</a>]</span>, respectively).</p>
<h3 id="validation-of-protein-protein-interactions">Validation of protein-protein interactions</h3>
<p>The interaction of a protein with other proteins determines its function.
Protein-protein interactions (PPIs) can be either mostly static (i.e. core subunits of a protein complex) or dynamic, varying with cellular state (i.e. cell cycle phase or cellular stress responses, posttranslational modifications) or environmental factors (i.e. availability of nutrients, presence of extracellular ligands of cell-surface receptors).
Therefore, any given protein can typically bind a range of interaction partners in a spatially and temporally restricted manner, thus forming complex PPI networks (the interactome of a protein).
The method of choice to experimentally examine altered PPI states depends on the model system and biological question (i.e. purified proteins vs complex protein mixtures, monitoring of PPIs in live cells or cell lysate etc).
Popular methods for the validation of PPIs in vivo include protein fragment complementation (split protein systems), 2-hybrid assays (mammalian, yeast and bacterial), proximity ligation, proximity labelling and FRET / BRET.
Protein fragment complementation assays rely on the principle that the two self-associating halves of reporter proteins can be expressed in an inactive form but when in spatial proximity bind one another to complement the functional, active reporter.
When these split reporters are fused to two interacting proteins (so-called bait and prey proteins), the binding of bait to prey induces the spatial restriction needed to fully complement the reporter. Commonly used reporter complementation systems are split fluorescent proteins (i.e. GFP, YFP) <span class="citation" data-cites="1Hf5Cuj4t">[<a href="#ref-1Hf5Cuj4t" role="doc-biblioref">605</a>]</span>, ubiquitin <span class="citation" data-cites="UJonPw7t">[<a href="#ref-UJonPw7t" role="doc-biblioref">606</a>]</span>, luciferase <span class="citation" data-cites="btMVdgTk">[<a href="#ref-btMVdgTk" role="doc-biblioref">607</a>]</span>, TEV protease <span class="citation" data-cites="zrB7Yy5d">[<a href="#ref-zrB7Yy5d" role="doc-biblioref">608</a>]</span>, beta-lactamase <span class="citation" data-cites="hPao2o3e">[<a href="#ref-hPao2o3e" role="doc-biblioref">609</a>]</span>, beta-galactosidase, Gal4, or DHFR <span class="citation" data-cites="LY7mqhu6">[<a href="#ref-LY7mqhu6" role="doc-biblioref">610</a>]</span>.
The resulting functional readout of these complementation system depends on which split reporter is used.
In general, the split luciferase system shows enhanced sensitivity over fluorescence-based systems as background luminescence is low.</p>
<p>Two-hybrid assays are based on a similar functional complementation strategy as fragment complementation systems.
Conventionally, two self-complementing transcription factor fragments are fused to bait and prey proteins, respectively, leading to the restoration of a functional transcription factor only upon prey-bait interaction.
The complemented transcription factor then induces the expression of a reporter gene that can be measured.
Multiple variations of this system abound for different model organisms, but they almost always involve transcriptional activation or repression of a reporter gene (<span class="citation" data-cites="18Csai5qq">[<a href="#ref-18Csai5qq" role="doc-biblioref">611</a>]</span> for a detailed discussion).</p>
<p>The yeast-2-hybrid system (Y2H) is deserving of mention here as it had been the very first 2-hybrid system established <span class="citation" data-cites="19CwS2HX">[<a href="#ref-19CwS2HX" role="doc-biblioref">612</a>]</span> and has ever since proven to be extremely versatile (multiple auxotrophic reporters and markers of phenotypic sensitivity available), cheap, lends itself to functional high-throughput screening and variants have been developed that allow for the investigation of membrane-protein interactions (i.e. membrane Y2H) <span class="citation" data-cites="2PC5fx0I 18Csai5qq">[<a href="#ref-18Csai5qq" role="doc-biblioref">611</a>,<a href="#ref-2PC5fx0I" role="doc-biblioref">613</a>]</span>.</p>
<p>Despite the many advantages the Y2H offers, critical drawbacks include the potential of misfolding of bait and prey proteins when fused to a complementation reporter, expression at non-physiological levels, the lack of control over posttranslational modifications that might be important for the PPI under investigation, and the potential requirement of kingdom- or species-specific folding factors for the bait/prey under investigation (i.e. when probing PPI of mammalian proteins in Y2H).
Principles of the Y2H technology have also been adapted to mammalian systems, which circumvent some of the aforementioned drawbacks of Y2H <span class="citation" data-cites="1DnPkHRsA">[<a href="#ref-1DnPkHRsA" role="doc-biblioref">614</a>]</span>.</p>
<p>Perhaps the most commonly applied method of detecting and validating PPIs in vitro is affinity purification (AP, also known as affinity chromatography) of co-immunoprecipitation (Co-IP) either coupled with SDS-PAGE/immunoblotting or mass spectrometry to determine the identity of interacting proteins.
AP typically relies on the isolation of a transgenic POI by an epitope tag (using epitope-specific matrix-conjugated proteins (antibodies or epitope-binding proteins)), while Co-IP harnesses specific antibodies directly targeting the POI.
Specific interactors are expected to be enriched compared to the negative control (i.e an isotype control antibody, a knockout cell line or empty matrix).
AP is not solely restricted to detecting PPIs, but can also be adapted to protein interactions with other biomolecules such as RNA <span class="citation" data-cites="1G5hcMdry">[<a href="#ref-1G5hcMdry" role="doc-biblioref">615</a>]</span>.
It should be noted that AP and Co-IP can return multiple potential binding partners, many of which might be artefactual due to loss of cellular compartmentalisation during sample preparation.</p>
<p>To reduce the probability of such artefacts and increase the confidence of a specific interaction, reciprocal affinity purification (by pulldown of each interaction partner) or in situ imaging might be performed (i.e. using fluorescence resonance energy transfer (FRET) <span class="citation" data-cites="636RraOs">[<a href="#ref-636RraOs" role="doc-biblioref">616</a>]</span>, split-protein systems <span class="citation" data-cites="zTlZpc1k">[<a href="#ref-zTlZpc1k" role="doc-biblioref">617</a>]</span>, proximity ligation assay <span class="citation" data-cites="13oBuESdH">[<a href="#ref-13oBuESdH" role="doc-biblioref">618</a>]</span> and immunofluorescence microscopy).</p>
<p>Forster and bioluminescence resonance energy transfer (FRET / BRET) can be used for in situ visualisation of protein proximities and therefore PPIs.
In FRET, non-radiative energy transfer between donor and receptor chromophores (each fused to prey and bait proteins, respectively), results in the emission of a characteristic fluorescence signal only when both prey and bait are in very close proximity (1-10 nm distance) and a suitable light source for donor excitation is provided <span class="citation" data-cites="feT5EHvH">[<a href="#ref-feT5EHvH" role="doc-biblioref">619</a>]</span>.</p>
<p>The underlying principle of BRET is similar to that of FRET but with the exception of using a chemical substrate which activates bioluminescent donor, such as luciferase, resulting in energy transfer to a fluorescent acceptor molecule <span class="citation" data-cites="o1iijZBV 168qPRxga">[<a href="#ref-o1iijZBV" role="doc-biblioref">620</a>,<a href="#ref-168qPRxga" role="doc-biblioref">621</a>]</span>.
The main advantages of BRET over FRET are independence from an external light source (which can result in photobleaching), but requires at least one of the POIs to be fused to the donor (while in FRET, donor and acceptor can be chemically conjugated to POI-specific antibodies) <span class="citation" data-cites="o1iijZBV">[<a href="#ref-o1iijZBV" role="doc-biblioref">620</a>]</span>.
FRET can be particularly useful in investigating cell surface protein interactions when using specific antibodies conjugated to donor and acceptor probes as antibodies are not cell-permeable and therefore restricted to targets presented on the cell surface in the absence of membrane permeabilization agents.
Other fluorescence-based PPI assays encompass Fluorescence correlation spectroscopy (FCS) and fluorescence cross-correlation spectroscopy (FCCS).
These methods use small volumes of fluorescently labelled proteins and can determine their diffusion coefficients, which change in when proteins form a complex <span class="citation" data-cites="6egqfZ9W">[<a href="#ref-6egqfZ9W" role="doc-biblioref">622</a>]</span>.</p>
<p>Proximity labelling methods (Proximity ligation and enzymatic proximity labelling (BirA, APEX2, HRP) can surveil labile or transient interaction in live cells in a high-throughput format when coupled with target identification by MS <span class="citation" data-cites="ZhoKgqsC">[<a href="#ref-ZhoKgqsC" role="doc-biblioref">623</a>]</span>, <span class="citation" data-cites="RXiPqFRh">[<a href="#ref-RXiPqFRh" role="doc-biblioref">624</a>]</span>.
These approaches harness a biotin ligase (i.e. BirA, BioID2, AirID, BASU, APEX2, HRP) fused to a POI whose interactome is to be determined. In the presence of biotin (for BirA, BioID2, AirID, BASU, APEX2 and HRP) or a biotin-phenol derivative (for APEX2), the biotin ligase will activate the biotin(-phenol) which then covalently biotinylates any protein in close proximity.
The activated biotin has a short half-life, ensuring that the effective labelling radius is typically restricted to approximately 10 nm.
Biotinylated proteins are isolated by affinity purification with streptavidin-conjugated beads and identified by mass spectrometry or SDS-PAGE/immunoblotting.
TurboID, miniTurboID and ultraID, promiscuous biotin ligases faster than BirA, have been developed allowing for shorter treatment times and decreased background signal.
The choice of a biotin ligase variant depends on the POI and experimental setup, but in general HRP does not work in cytoplasmic environments where conditions are chemically reducing, but is suitable for labelling proteins extracellular face of the plasma membrane or in the endoplasmic reticulum and golgi apparatus.
While TurboID and similar variants have fast kinetics, they can cause depletion of endogenous biotin and therefore cytotoxicity.
A major drawback shared by all variants described above is that they necessitate fusion to the POI, which might alter its physiological behaviour and give rise to false positives or false negatives.
Moreover, detecting a biotin-labelled protein does not unequivocally designate it as an interaction partner as spatial proximity to the POI-biotin ligase fusion protein without direct binding can result in biotinylation.
The inclusion of controls, such as expression of the biotinylating enzyme alone in the cellular compartment of interest, is therefore particularly important for enzymatic proximity labelling methods.</p>
<p>The in situ proximity ligation assay (PLA) combines the specificity of antibodies with the signal amplification capacity of a DNA polymerase reaction.
Here, two antibodies, each conjugated to a short single-strand DNA (ssDNA) tag and each specific to one of the two proteins whose interaction is under investigation, are added to fixed cells or tissue.
Once bound to their respective targets and only when in direct proximity, the addition of two connector oligonucleotides complementary to each tag ssDNA tag and phi29 DNA polymerase, triggers isothermal rolling circle amplification, eventually resulting in the generation of continuous stretches of repetitive DNA.
These DNA products can then be visualised by in situ hybridisation with fluorescently labelled oligonucleotides (see <span class="citation" data-cites="1F8ZvcEeH">[<a href="#ref-1F8ZvcEeH" role="doc-biblioref">625</a>]</span> for a detailed discussion).
PLA has the advantage of visualising the two interacting proteins in their native environment when high-resolution microscopy is used as a readout.</p>
<p>Chemical crosslinking (XL) of proteins can determine PPIs with amino-acid level resolution, and can thereby give valuable insights into the orientation of two or more proteins relative to one another <span class="citation" data-cites="3LymzGVH">[<a href="#ref-3LymzGVH" role="doc-biblioref">626</a>]</span>.
Recent technical advances also enabled the visualisation of protein-RNA interaction <span class="citation" data-cites="yxw6FRju">[<a href="#ref-yxw6FRju" role="doc-biblioref">627</a>]</span>.
Various XL chemistries are available (amine-reactive, sulfhydryl and photoreactive crosslinkers; reversible vs irreversible) and cross-linked proteins detected by mass spectrometry <span class="citation" data-cites="4T3MaN5d">[<a href="#ref-4T3MaN5d" role="doc-biblioref">628</a>]</span>.
In general, applying XL-MS to a mixture of interacting, purified proteins is preferable to in situ XL of complex protein mixtures (i.e. cell lysate) as detection and deconvolution of XL peptides is technically and computationally challenging.</p>
<p>Surface plasmon resonance can accurately measure several key kinetics of PPIs with high accuracy (e.g. association and dissociation kinetics, stoichiometry, affinity) <span class="citation" data-cites="H3H5bwtx">[<a href="#ref-H3H5bwtx" role="doc-biblioref">629</a>]</span>.
It relies on the quantification of refractive index changes of polarised light shone onto a sensor chip containing a prey protein immobilised on a metal surface (typically gold).
When prey and bait proteins interact, the mass concentration at the metal interface changes, altering the refractive index and SPR angle (intensity of the refracted light).
 </p>
<h2 class="page_break_before" id="liquid-chromatography-lc">Liquid Chromatography (LC)</h2>
<p>Chromatography is the physical sorting of a mixture of molecular species that are dissolved in a mobile phase through the strength of binding, or affinity, to the chromatographic column’s stationary phase <span class="citation" data-cites="rNop1hIz">[<a href="#ref-rNop1hIz" role="doc-biblioref">630</a>]</span>.
The mobile phase is pressure driven through the column and molecular species, or analytes, that have a strong affinity to the stationary phase are retained, or slowed, while those with a weak affinity pass through quickly.
Thusly the analytes are separated by order of elution from the column.
Chromatography can exploit most physical properties of the analytes, including ionic charge (anion/cation exchange chromatography), hydrogen binding (hydrophilic interaction), and size (size exclusion chromatography, capillary electrophoresis).
In some chromatographic separations the mobile phase composition is adjusted by mixing two or more buffers at different ratios to influence the strength of affinity of individual analytes to the stationary phase and exquisitely regulate retention.</p>
<p>Mass spectrometers suffer from ion suppression, a phenomenon where the over-abundance of one or a few species within the ion population entering the mass spectrometer masks the presence of less abundant species <span class="citation" data-cites="71TlBiJy">[<a href="#ref-71TlBiJy" role="doc-biblioref">631</a>]</span>.
Complex biological samples, such as tissue, cell lysate, or physiological fluids contain a wide dynamic range of molecule concentrations that span many orders of magnitude.
The physical separation of analytes from biological samples by LC reduces the complexity of the ion population presented to the mass spectrometer at a given time, thus allowing the instrument to carry out the necessary fragmentation scans to identify and quantify the detectable species.
Therefore, one major benefit of LC is that it allows detection of low abundant analytes in other elution windows.</p>
<p>The field of proteomics predominantly separates peptides using reversed phase liquid chromatography <span class="citation" data-cites="pRj1odLy T9Yv2Omz WG5jq46X">[<a href="#ref-pRj1odLy" role="doc-biblioref">632</a>,<a href="#ref-T9Yv2Omz" role="doc-biblioref">633</a>,<a href="#ref-WG5jq46X" role="doc-biblioref">634</a>]</span>.
Reversed stationary phase is most commonly composed of microscopic (1-3 μm) silica beads coated with covalently bound long (e.g. C18) hydrophobic alkyl chains.
The hydrophobic side chains of certain residues and the peptide backbone bind to this stationary phase through non-polar interactions.
These interactions are strong in an aqueous solvent, but are disrupted when the organic composition of the solvent is increased.
Thus, in a reversed phase separation the proportion of non-polar, or organic, solvent in the mobile phase is gradually increased to release analytes from the stationary phase based on the strength of hydrophobic binding: weakly bound hydrophilic analytes elute with a low organic level in the mobile phase and strongly bound hydrophobic analytes only elute when the organic composition reaches a higher percentage.
By far the most popular combination of solvents for peptide analysis is water and acetonitrile with dilute acid modifier (such as 0.1% formic acid or 0.5% acetic acid).
The programmed rate at which the proportion of organic solvent is increased in the mobile phase is called the “gradient”, which you will often find described in the methods sections for reversed phase separations.</p>
<h3 id="electrospray-ionization-1">Electrospray Ionization</h3>
<p>LC is paired to MS through ESI, and LC parameters greatly influence ESI.
The analytes are eluted in a liquid mobile phase and must be released into the gas phase as charged ions for detection by mass spectrometry.
This is achieved by spraying the eluent from the chromatographic separation through a narrow nozzle under a high voltage potential (1,000-4,000 volts) between the nozzle, or emitter, and the mass spectrometer inlet.
The eluent is sprayed as a mist of small charged droplets that explode into smaller droplets as the solvent evaporates and the repelling columbic force of the charged analytes increases <span class="citation" data-cites="BslEV2vj">[<a href="#ref-BslEV2vj" role="doc-biblioref">402</a>]</span>.
The droplets become progressively smaller until individual analyte molecules are ejected.
The ejected analytes are ionized by the retained charge and can thus be manipulated by the electric fields in the mass spectrometer to measure their mass and perform the necessary fragmentations to elucidate structure.
The chromatographic flowrate (the volume of mobile phase driven through the chromatographic column per unit time e.g. uL/min) dictates the efficiently of electrospray ionization (proportion of analytes eluting from the column that are ionized and into the gas phase) and is thus a key consideration for sensitivity of analysis <span class="citation" data-cites="hwO7hkQN">[<a href="#ref-hwO7hkQN" role="doc-biblioref">635</a>]</span>.
Reduced flowrates generate smaller droplets which degrade into ejected charged analytes rapidly, thus resulting in more detectable analytes and higher ionization efficiency.
Electrospray ionization efficiency is also aided by an inert sheath gas, high temperature, and reduced pressure between the nozzle and ion lensing elements, thus decent sensitivity can still be achieved at high flowrates.
For more detailed discussion of ionization, see the “Ionization” section.</p>
<h3 id="quality-attributes-of-chromatographic-separation">Quality Attributes of Chromatographic Separation</h3>
<p>The quality of chromatographic separation defines the number of analytes that are identified and quantified by LC-MS analysis.
The theory around chromatographic separation was developed when LCs were paired with spectrophotometer detectors that only measure the combined signal intensity from all co-eluting analytes.
The ability of MS to simultaneously detect the masses of individual components re-defines the significance of certain LC attributes.
For those looking for mathematical descriptions of chromatographic quality, refer to the “Van Deemter equation”, which we do not cover here to maintain simplicity <span class="citation" data-cites="156fxE3m7">[<a href="#ref-156fxE3m7" role="doc-biblioref">636</a>]</span>.
The following attributes are the most important to consider in LC-MS.</p>
<h4 id="chromatographic-resolution">Chromatographic Resolution</h4>
<p>Chromatographic resolution is defined as the ability to fully resolve adjacent chromatographic peaks containing analytes with nearly equal affinities to the solid phase.
In mass spectrometry analytes are distinguished by mass even if they are not resolved by LC.
Thus in LC-MS, the more relevant, but closely related concept is the peak width at the half maximum (FWHM) of the peak.
A low FWHM indicates a sharp elution peak. In a sharp peak the entirety of the analyte population is electrosprayed into the mass spectrometer in a short time thus increasing the signal.
Low FWHM of high abundance species also confines their ionization suppression to narrow time windows, which means a lower number of co-eluting analytes are hidden.
Conversely, high FWHM means that the analyte signal is spread out over time, thus reducing sensitivity.
Furthermore, at a high FWHM, high abundance species mask analytes through ion suppression over a larger portion of the separation.</p>
<h4 id="peak-capacity">Peak Capacity</h4>
<p>Peak capacity is the number of peaks detectable over the analytical separation.
A long separation in which FWHM remains low would have a large peak capacity and thus allow identification of many species.
Unfortunately increasing the length of a reversed phase gradient also increases the FWHM due to an increase in diffusion, which results in a diminishing return for longer analytical methods.
A longer separation provides more time and opportunities for the mass spectrometer to sample each analyte to acquire fragmentation spectra required for identification and the selection of gradient length should consider both the desired throughput and the speed of the MS data acquisition strategy.</p>
<h4 id="reproducibility-and-robustness">Reproducibility and Robustness</h4>
<p>Reproducibility is defined as the ability to repeatedly obtain the same measurement for the same analytes each time that the analysis is repeated.
In liquid chromatography this means that each analyte should elute at nearly the same retention time (the time elapsed since the start of the analysis until the analyte’s elution from the chromatographic column) with the same peak width.
Robustness is the ability of the system to maintain reproducible performance despite nonoptimal conditions.
The most typical obstacles to robustness are mechanical wear of the system components and the analytical column, fouling of the system by contaminants introduced in the samples, and clogging due to accumulation of contaminants.
High flow methods tend to be more robust due to reduced impact of changes in dwell volumes, and the wider bore of the components used is more resilient to clogging.
However, higher flowrate comes at the cost of reduced ionization efficiency and sensitivity, thus nanoflow (100-300 nL/min flowrate) chromatography remains a widely utilized strategy in proteomics.</p>
<h4 id="throughput-and-instrument-utilization">Throughput and Instrument Utilization</h4>
<p>Throughput is the number of samples that are analyzed in a given timeframe, for example samples per day.
High throughput is required to analyze thousands of samples that truly represent biological diversity in a timely manner.
Increasing throughput means less data are collected for individual samples.
Furthermore, many steps in the LC process are required for sample analysis in which no useful data is collected including sample injection, and system cleaning and equilibration, which reduce the ratio of data collected to instrument operation time, or instrument utilization.
The ability to perform these steps while a different sample is analyzed, or parallelization, increases instrument utilization and the amount of data collected by several minutes which is a significant increase when several samples are analyzed per hour.</p>
<h3 id="trapping-and-pre-columns">Trapping and Pre-Columns</h3>
<p>Trapping and pre-columns are short chromatographic columns that are used to increase robustness of an LC-MS system.
A pre-column is connected directly to the front of the analytical column and is intended to be disposable and to absorb contaminants and protect the analytical column.
The trapping column is a connected indirectly to the analytical column through a valve. The valve can be switched to redirect the flow through the trapping column away from the analytical column.
This allows analytes to be loaded on the trapping column while analytes that are hydrophilic and poorly retained are washed away and do not contaminate the analytical column or the mass spectrometer.
This process is referred to as desalting, and once it is complete, the valve configuration is changed to connect the trapping column to the analytical column, and analytes captured on the trapping column can be eluted off the trap and through the analytical column for analysis by MS.
Certain trapping columns can be operated in both directions, which allows aggregates to be flushed away when the trapping column is cleaned in the reverse direction.
Additionally trapping columns are shorter and have less backpressure so they can be loaded with sample quickly at a fast flowrate.
Whereas loading the sample directly on the analytical column requires a slower flowrate.
Two trapping columns can be used in tandem to provide parallelization, while one trapping column is cleaned and loaded with samples the second trapping column is in line with the analytical column analyzing the sample that was loaded on it in the previous run <span class="citation" data-cites="dvtPvLcY L2sT906m">[<a href="#ref-dvtPvLcY" role="doc-biblioref">637</a>,<a href="#ref-L2sT906m" role="doc-biblioref">638</a>]</span>.</p>
<h3 id="multi-dimensional-lc">Multi Dimensional LC</h3>
<p>Depth of profiling has previously been increased by combining two or more orthogonal LC separations.
Orthogonal in this context means that each separation sorts the analytes into different populations <span class="citation" data-cites="1Gtuii4F4">[<a href="#ref-1Gtuii4F4" role="doc-biblioref">639</a>]</span>.
For example, a separation based on positive charge (strong cation exchange, SCX) separates analytes based on positive charge, and when paired with reversed phase chromatography results a higher peak capacity and more analytes identified.
The first highly popular method was multidimensional protein identification technology (MudPIT), which used online separation by SCX followed by C18 reversed phase <span class="citation" data-cites="1FnaHa1s9">[<a href="#ref-1FnaHa1s9" role="doc-biblioref">640</a>]</span>.
However, the resolution of peptide separation by SCX is low, leading to the presence of peptides in many fractions.
The currently accepted most popular method for two-dimensional separation combines iterative reversed phase at different high and then low pH to sort analytes by changes in hydrophobicity due to changes in amino acid side chain ionization.
Although the separations are not entirely orthogonal, multiple fraction concatenation across the high pH elution can produce entirely orthogonal peptide sets <span class="citation" data-cites="dl5RDy6j">[<a href="#ref-dl5RDy6j" role="doc-biblioref">641</a>]</span>.
In recent years the focus of proteomics has shifted from deep profiling of fewer samples to rapid profiling of large cohorts.
Thus, lengthy multidimensional methods have been replaced with single shot experiments only using one dimension of high resolution reversed phase separation <span class="citation" data-cites="44oI1g68">[<a href="#ref-44oI1g68" role="doc-biblioref">464</a>]</span>.
However peak capacity is regained by using ion mobility spectrometry (separation of ionized peptides in the gas phase).</p>
<h2 class="page_break_before" id="tandem-mass-spectrometry-and-peptide-fragmentation">Tandem Mass Spectrometry and Peptide Fragmentation</h2>
<h3 id="tandem-mass-spectrometry">Tandem Mass Spectrometry</h3>
<p>Tandem MS, where precursor ions are selected and fragmented to generate an MS/MS spectrum containing peptide-derived product ions, is a fundamental process in modern proteomics <span class="citation" data-cites="qAVrmvwM V3WGOS1f">[<a href="#ref-qAVrmvwM" role="doc-biblioref">642</a>,<a href="#ref-V3WGOS1f" role="doc-biblioref">643</a>]</span>.
This is largely because intact peptide mass alone cannot unambiguously provide a peptide’s sequence <span class="citation" data-cites="1DSKc93ie">[<a href="#ref-1DSKc93ie" role="doc-biblioref">644</a>]</span>; however, MS/MS spectra provide more information due to predictable fragmentation behavior of peptide ions to generate sequence-informative fragments <span class="citation" data-cites="2MR1Pihs qAVrmvwM">[<a href="#ref-qAVrmvwM" role="doc-biblioref">642</a>,<a href="#ref-2MR1Pihs" role="doc-biblioref">645</a>]</span>.
Some more advanced proteomic acquisition methods use MS1-only feature detection in combination with retention time to maximize information used for downstream quantitation <span class="citation" data-cites="zCNyqvc9">[<a href="#ref-zCNyqvc9" role="doc-biblioref">646</a>]</span>.
In most of these, identifications are fundamentally based on MS/MS spectra, either acquired as part of a specific LC-MS/MS analysis that contains the MS/MS spectra themselves or on a spectral library of MS/MS spectra acquired previously <span class="citation" data-cites="175VGosTe ylZpW7Vi">[<a href="#ref-175VGosTe" role="doc-biblioref">647</a>,<a href="#ref-ylZpW7Vi" role="doc-biblioref">648</a>]</span>.
True MS1-only methods that use only accurate mass and retention time for identification have been discussed, but these have yet to be widely adopted <span class="citation" data-cites="zCNyqvc9">[<a href="#ref-zCNyqvc9" role="doc-biblioref">646</a>]</span>.</p>
<p>The value of MS/MS spectra for peptide identification comes from predictable fragmentation behavior of peptide ions to generate sequence-informative fragments <span class="citation" data-cites="2MR1Pihs qAVrmvwM">[<a href="#ref-qAVrmvwM" role="doc-biblioref">642</a>,<a href="#ref-2MR1Pihs" role="doc-biblioref">645</a>]</span>.
Multiple dissociation methods exist to generate product ions in MS/MS spectra through various mechanisms (Figure X).
In non-modified peptides, the most labile bonds are typically peptide bonds (i.e., amide bonds) between amino acids.
Depending on where peptides dissociate along the peptide backbone, the fragments are assigned different ion types (Figure XA).
Fragment ion nomenclature was first developed by Roepstorff and Fohlman in 1984 <span class="citation" data-cites="mZdxW2jk">[<a href="#ref-mZdxW2jk" role="doc-biblioref">649</a>]</span> and then refined by Biemann in 1990 <span class="citation" data-cites="ivBdKZgo">[<a href="#ref-ivBdKZgo" role="doc-biblioref">650</a>]</span>.
The main ion types are the peptides that contain the original peptide n-terminus (i.e. a, b, and c ions), or the original peptide c-terminus (i.e. x, y, and z ions).
The number associated with each fragment ion indicates how many amino acids from each end are included.
The most common ions from collisional methods are the b and y ions, which result from fragmentation of the amide between the carbonyl and nitrogen.
The most common ions from electron transfer methods are c and z ions, which occur between the nitrogen and the alpha-carbon of the peptide backbone.</p>
<div id="fig:SPE" class="fignos">
<figure>
<img src="images/Tandem_MS.svg" style="width:100.0%" data-tag="3" alt="Figure 3: Peptide Fragmentation Methods. (A) Sequence-informative fragment ions are termed a/x-, b/y-, and c/z-type fragments depending on which bond along the peptide backbone breaks. Fragments that explain the intact N-terminus of the peptide are a-, b-, and c-type ions, while x-, y-, and z-type ions explain the intact C-terminus of the peptide. Other panels show common dissociation methods, including collision, electron, and photon-based fragmentation. (B) Resonant collision-induced dissociation (resCID) and beam-type CID (beamCID) both produce mainly b/y-type sequencing ions through collisions with background gases like helium and nitrogen that increase the internal energy of peptide cations. (C) Electron capture and electron transfer dissociation (ECD and ETD) generate mainly c/z-type fragments through electron-mediated radical driven cleavage of the peptide backbone. (D) Infrared multiphoton dissociation (IRMPD) is a slow heating method similar in dissociation mechanism to resCID, but very different in implementation due to the IR lasers required (often with lower energy 10.6 micron photons). Ultraviolet photodissociation (UVPD) can use a range of wavelengths (popular options shown) to introduce higher energy photons to peptide cations, causing vibrational and electronic excitation that can generate all major fragment ion types depending on wavelength used." />
<figcaption aria-hidden="true"><span>Figure 3:</span> <strong>Peptide Fragmentation Methods.</strong>
(A) Sequence-informative fragment ions are termed a/x-, b/y-, and c/z-type fragments depending on which bond along the peptide backbone breaks.
Fragments that explain the intact N-terminus of the peptide are a-, b-, and c-type ions, while x-, y-, and z-type ions explain the intact C-terminus of the peptide.
Other panels show common dissociation methods, including collision, electron, and photon-based fragmentation.
(B) Resonant collision-induced dissociation (resCID) and beam-type CID (beamCID) both produce mainly b/y-type sequencing ions through collisions with background gases like helium and nitrogen that increase the internal energy of peptide cations.
(C) Electron capture and electron transfer dissociation (ECD and ETD) generate mainly c/z-type fragments through electron-mediated radical driven cleavage of the peptide backbone.
(D) Infrared multiphoton dissociation (IRMPD) is a slow heating method similar in dissociation mechanism to resCID, but very different in implementation due to the IR lasers required (often with lower energy 10.6 micron photons).
Ultraviolet photodissociation (UVPD) can use a range of wavelengths (popular options shown) to introduce higher energy photons to peptide cations, causing vibrational and electronic excitation that can generate all major fragment ion types depending on wavelength used.</figcaption>
</figure>
</div>
<p>One of the earliest and most ubiquitous peptide fragmentation methods is collision-induced dissociation (CID, also called collisionally-activated dissociation, CAD) <span class="citation" data-cites="EmIJDW71">[<a href="#ref-EmIJDW71" role="doc-biblioref">343</a>]</span> (Figure XB).
Here, collisions with inert gas molecules are used to increase the internal energy of peptide ions to reach bond dissociation energies that fragment them into products.
Various inert gases can be used; helium, nitrogen, and argon are the most common.
Preferences for which gas is used is often a function of how much energy per collision is desired.
Two main versions of CID are used in proteomics, with the most common being beam-type CID (beamCID, sometimes called higher-energy collisional dissociation, HCD) <span class="citation" data-cites="12yIRpFcQ 1DIcKpBMl">[<a href="#ref-12yIRpFcQ" role="doc-biblioref">651</a>,<a href="#ref-1DIcKpBMl" role="doc-biblioref">652</a>]</span>.
BeamCID typically uses nitrogen or argon as a collision gas, and peptide ions are accelerated in to a collision cell filled with several mTorr of bath gas.
The kinetic energy used to accelerate precursor ions (often generated using direct current voltage differentials between the source of the ions and the collision cell) determines the energy imparted through collisions with the bath gas, which in turn governs their fragmentation behavior.</p>
<p>In non-modified peptides, the most labile bonds are typically peptide bonds (i.e., amide bonds) between amino acids, so the increase in internal energy from beamCID generates b- and y-type ions that represent this peptide bond cleavage, as shown in Biemann fragment ion nomenclature (Figure X).
b-type ions provide sequence information for fragments that have an intact N-terminus, while y-type ions denote fragment ions with an intact C-terminus.
Collisions in beamCID cause near instantaneous generation of primary fragment ions.
Because the increase in internal energy happens rapidly before energy can be redistributed, beamCID can generate fragments that are not necessarily derived from cleavage of the most labile bonds (e.g., PTM-modified peptides, discussed below), but spectra are often dominated by b/y-type ions from amide bond cleavage.
BeamCID can also generate secondary fragments, such as immonium ions from side chain losses <span class="citation" data-cites="CrVLd8Zt">[<a href="#ref-CrVLd8Zt" role="doc-biblioref">653</a>]</span> or a-type fragment ions that come from water loss from b-type ions due to multiple collision events (note: a-type ions can form as primary fragmentation products in other dissociation methods).
The simplicity of beamCID, which requires simply an rf-only collision cell, has made it widely implemented on most instrument platforms used in modern proteomics.</p>
<p>A second form of CID is called resonant CID (resCID), where the internal energy of peptide ions is slowly increased through multiple low-energy collisions.
Here, helium gas is most often used, as it imparts less energy per collision, and activation typically happens in ion trap devices where supplemental frequencies can be used to excite ions.
In other words, ions are trapped using axial rf-frequencies, and an additional rf-frequency is applied to the electrodes of the ion trap <span class="citation" data-cites="Lx0XDSc1">[<a href="#ref-Lx0XDSc1" role="doc-biblioref">654</a>]</span>.
This supplemental rf is selected to have a frequency resonant with the fundamental frequency of the ions to be fragmented, as determined by the Mathieu equations, which excites the ions of interest so that they have increased kinetic energy as they move in the ion trap <span class="citation" data-cites="8fRsJhJf 1DjnN7u2e">[<a href="#ref-8fRsJhJf" role="doc-biblioref">655</a>,<a href="#ref-1DjnN7u2e" role="doc-biblioref">656</a>]</span>.
The increased kinetic energy creates more collisions with the background helium gas to slowly build up the internal energy of the precursor ions until the dissociation energy of the most labile bond is reached, causing fragmentation.
Once ions dissociate, the fragments have different m/z values than the precursor ions, meaning they fall out of resonance with the supplemental rf and are no longer activated.
Thus, resCID typically fragments only the most labile bonds in precursor ions and does not have secondary fragmentation behavior.
As above, for non-modified peptide ions, this typically generates sequence-informative b- and y-type product ions.
For modified peptides where the bonds connecting the modification to an amino acid are more labile than peptide bonds, e.g., phosphopeptides and glycopeptides, resCID MS/MS spectra can be dominated by products ions only of the PTM-loss rather than sequence informative fragment ions, although many factors govern this behavior <span class="citation" data-cites="16Wowt3CO Oadeotpz">[<a href="#ref-Oadeotpz" role="doc-biblioref">427</a>,<a href="#ref-16Wowt3CO" role="doc-biblioref">657</a>]</span>.
Because of this, and because this method requires an ion trap device with the ability to apply supplemental rfs, resCID is less used than beamCID.
For both beamCID and resCID, the mobile proton model has been widely accepted to explain fragmentation behavior <span class="citation" data-cites="Ud7cg5Ze">[<a href="#ref-Ud7cg5Ze" role="doc-biblioref">658</a>]</span>, and this largely predictable behavior has greatly helped in manual and algorithm-assisted spectral interpretation.</p>
<p>Despite the utility and broad adoption of CID, there are alternative dissociation methods that have been explored for a variety of uses, including applications where CID is inadequate for the experimental question <span class="citation" data-cites="VOcnSdY9 oNkSkkq8 XHvsh4uo">[<a href="#ref-VOcnSdY9" role="doc-biblioref">659</a>,<a href="#ref-oNkSkkq8" role="doc-biblioref">660</a>,<a href="#ref-XHvsh4uo" role="doc-biblioref">661</a>]</span>.
The most popular of these alternative dissociation methods are electron-based dissociation (ExD) approaches, which include electron capture dissociation (ECD) and electron transfer dissociation (ETD).
In both of these, peptide cations capture thermal electrons (ECD, <span class="citation" data-cites="qWde4x">[<a href="#ref-qWde4x" role="doc-biblioref">662</a>]</span>) or abstract an electron from a reagent anion (ETD, <span class="citation" data-cites="GH22tfkO">[<a href="#ref-GH22tfkO" role="doc-biblioref">423</a>]</span>) to generate radical-driven dissociation of the N-Ca bond that predominantly generates sequence-informative c- and z-type product ions (Figure XC).
The mechanisms of ExD methods have been widely explored <span class="citation" data-cites="OVsIcoNm xhXJM8vz">[<a href="#ref-OVsIcoNm" role="doc-biblioref">663</a>,<a href="#ref-xhXJM8vz" role="doc-biblioref">664</a>]</span>, and the preferential cleavage of N-Ca bonds along the peptide backbone have been particularly useful for PTM-modified species because the modifications remain largely intact even during peptide backbone bond fragmentation.
ExD methods have shown promise for analysis of numerous PTMs, including phosphorylation, glycosylation, ADP-ribosylation, and more <span class="citation" data-cites="rqc6ScJr HNURrSN7">[<a href="#ref-rqc6ScJr" role="doc-biblioref">665</a>,<a href="#ref-HNURrSN7" role="doc-biblioref">666</a>]</span>.</p>
<p>Two fundamental challenges exist with ExD methods.
First, ExD implementation requires instruments that can manipulate cations and anions (or free electrons) within the same scan sequence and can trap both simultaneously for electron capture/transfer events to occur.
This has been successfully accomplished on a number of instruments, including FT-ICR, ion trap, ToFs with quadrupole ion traps, and hybrid Orbitrap instruments, but it is not a ubiquitous feature of all platforms.
That said, several exciting advances in recent years have made ExD methods more accessible on numerous instrument configurations <span class="citation" data-cites="rqc6ScJr HNURrSN7 UqS9JT46 1Gs2JKxdE Vu5rNLFc">[<a href="#ref-rqc6ScJr" role="doc-biblioref">665</a>,<a href="#ref-HNURrSN7" role="doc-biblioref">666</a>,<a href="#ref-UqS9JT46" role="doc-biblioref">667</a>,<a href="#ref-1Gs2JKxdE" role="doc-biblioref">668</a>,<a href="#ref-Vu5rNLFc" role="doc-biblioref">669</a>]</span>.
A second challenge is the dependence of ExD dissociation efficiency on precursor ion charge density <span class="citation" data-cites="eSdFYuAj">[<a href="#ref-eSdFYuAj" role="doc-biblioref">670</a>]</span>.
ExD methods generally produce robust fragmentation for charge dense precursor ions, i.e., those with relatively low m/z values and higher z.
Alternatively, precursors with low charge density, i.e., higher m/z values, have relatively condensed secondary gas-phase structure that leads to non-covalent interactions.
Even when ExD methods drive peptide backbone cleavage in these cases, product ions (i.e. c- and z-type fragments) are held together by the non-covalent interactions so that few (or no) sequence-informative product ions are produced.
This process is called non-dissociative electron-capture/transfer (ECnoD/ETnoD)<span class="citation" data-cites="qyjSSKs7">[<a href="#ref-qyjSSKs7" role="doc-biblioref">671</a>]</span>.
Several strategies to mitigate ECnoD/ETnoD have been successfully explored, including supplemental activation of product ions with resCID (ETcaD <span class="citation" data-cites="c8aJxeOj">[<a href="#ref-c8aJxeOj" role="doc-biblioref">672</a>]</span>) or beamCID (EThcD <span class="citation" data-cites="TEQyYNF9 bsAhaObV">[<a href="#ref-TEQyYNF9" role="doc-biblioref">673</a>,<a href="#ref-bsAhaObV" role="doc-biblioref">674</a>]</span>), supplemental activation with infrared photons (AI-ECD <span class="citation" data-cites="oRY0m4xv F58LowfZ">[<a href="#ref-oRY0m4xv" role="doc-biblioref">675</a>,<a href="#ref-F58LowfZ" role="doc-biblioref">676</a>]</span> and AI-ETD <span class="citation" data-cites="3mhY8AZ1 pXrOdleW 1HZ2A5iku PWUQDl3j">[<a href="#ref-3mhY8AZ1" role="doc-biblioref">677</a>,<a href="#ref-pXrOdleW" role="doc-biblioref">678</a>,<a href="#ref-1HZ2A5iku" role="doc-biblioref">679</a>,<a href="#ref-PWUQDl3j" role="doc-biblioref">680</a>]</span>) or ultraviolet photons (ETuvPD<span class="citation" data-cites="1CD0nWWIP">[<a href="#ref-1CD0nWWIP" role="doc-biblioref">681</a>]</span>), and use of higher energy electrons <span class="citation" data-cites="a9481fQj 3Q6WdA4H Vu5rNLFc">[<a href="#ref-Vu5rNLFc" role="doc-biblioref">669</a>,<a href="#ref-a9481fQj" role="doc-biblioref">682</a>,<a href="#ref-3Q6WdA4H" role="doc-biblioref">683</a>]</span>.
Despite their successes, these methods still require instrumentation capable of ExD in addition to extra hardware needed for a given strategy (e.g., a CO2 laser in AI-ETD <span class="citation" data-cites="1zm3p02B">[<a href="#ref-1zm3p02B" role="doc-biblioref">684</a>]</span>).
As with ExD in general, recent advances in supplemental activation strategies for ExD are making these tools more accessible <span class="citation" data-cites="rqc6ScJr HNURrSN7">[<a href="#ref-rqc6ScJr" role="doc-biblioref">665</a>,<a href="#ref-HNURrSN7" role="doc-biblioref">666</a>]</span>.</p>
<p>Photoactivation is another family of alternative dissociation strategies that have been steadily gaining popularity <span class="citation" data-cites="M8hC7nxG pP0K4nse">[<a href="#ref-M8hC7nxG" role="doc-biblioref">685</a>,<a href="#ref-pP0K4nse" role="doc-biblioref">686</a>]</span>.
Infrared multi-photon dissociation (IRMPD) is canonically the photodissociation method used in early proteomic applications <span class="citation" data-cites="pP0K4nse">[<a href="#ref-pP0K4nse" role="doc-biblioref">686</a>]</span>, but ultraviolet photodissociation (UVPD) has been the more widely used approach in the recent decade <span class="citation" data-cites="YqEIuvCO">[<a href="#ref-YqEIuvCO" role="doc-biblioref">687</a>]</span>.
IRMPD functions similarly to resCID; it is a slow heating approach that causes vibrational excitation due to absorption of low energy photons, generally 10.6 um photons from a CO2 laser <span class="citation" data-cites="zA5rBLH4 XXMVxnKl">[<a href="#ref-zA5rBLH4" role="doc-biblioref">688</a>,<a href="#ref-XXMVxnKl" role="doc-biblioref">689</a>]</span>.
Predominant fragments are b- and y-type fragments, although secondary fragmentation occurs because fragment ions remain in the photon path after the initial dissociation event (Figure XD).
Despite limited use in the past decade, recent work shows that IRMPD, or more generally activation with IR photons, may still have value in the proteomics toolkit <span class="citation" data-cites="1EjowCoSS 1HZ2A5iku">[<a href="#ref-1EjowCoSS" role="doc-biblioref">226</a>,<a href="#ref-1HZ2A5iku" role="doc-biblioref">679</a>]</span>.
UVPD has been explored with a number of wavelengths, including 157 nm, 193 nm, 213 nm, 266 nm, and 355 nm <span class="citation" data-cites="UC6BHa63 145wzrCEm UoKcvT55 168MaNzTm UrNcqFO2 cZOW0Gem">[<a href="#ref-UC6BHa63" role="doc-biblioref">690</a>,<a href="#ref-145wzrCEm" role="doc-biblioref">691</a>,<a href="#ref-UoKcvT55" role="doc-biblioref">692</a>,<a href="#ref-168MaNzTm" role="doc-biblioref">693</a>,<a href="#ref-UrNcqFO2" role="doc-biblioref">694</a>,<a href="#ref-cZOW0Gem" role="doc-biblioref">695</a>]</span>.
Higher-energy UVPD approaches, like 193 and 213 nm photons, are typically used for underivatized peptide and protein ions <span class="citation" data-cites="YqEIuvCO">[<a href="#ref-YqEIuvCO" role="doc-biblioref">687</a>]</span>, while others, like 266 and 355 nm, can be used for directed fragmentation at specific residues with natural chromophores (e.g., tyrosine) or exogenously added chromophore tags <span class="citation" data-cites="1ACL74GmA 18sf2lBbl">[<a href="#ref-1ACL74GmA" role="doc-biblioref">696</a>,<a href="#ref-18sf2lBbl" role="doc-biblioref">697</a>]</span>.
UVPD with 193 and 213 nm generate multiple fragment types, including sequence-informative a-, b-, c-, x-, y-, and z-ions in addition to other fragmentation pathways, which occur through vibrational and electronic excitation <span class="citation" data-cites="5SkV76Q3">[<a href="#ref-5SkV76Q3" role="doc-biblioref">698</a>]</span>.
UVPD has been explored for bottom-up proteomic applications, but its more impactful utility, arguably, has been realized for intact protein characterization <span class="citation" data-cites="10TkrAfIp">[<a href="#ref-10TkrAfIp" role="doc-biblioref">699</a>]</span>.
The laser needed for UVPD (i.e., the photon wavelength desired) determines much about its implementation.
193 nm photons are typically generated using an Excimer laser with ArF gas <span class="citation" data-cites="KWENOLlt">[<a href="#ref-KWENOLlt" role="doc-biblioref">700</a>]</span>, while 213 nm photons can be generated with a solid-state laser that is easier to integrate into an instrument platform and maintain <span class="citation" data-cites="1A6jk9UQ3 UoKcvT55">[<a href="#ref-UoKcvT55" role="doc-biblioref">692</a>,<a href="#ref-1A6jk9UQ3" role="doc-biblioref">701</a>]</span>.
That said, 213 nm photons tend to provide more directed, preferential cleavage pathways compared to 193 nm photons that cleave more broadly in non-directed fashion <span class="citation" data-cites="zJsJYkGj">[<a href="#ref-zJsJYkGj" role="doc-biblioref">702</a>]</span>.
Outside of ExD and photoactivation approaches, other alternative dissociation methods have been explored for various proteomic applications, although they are not as widely adopted at ExD and UVPD methods<span class="citation" data-cites="M8hC7nxG">[<a href="#ref-M8hC7nxG" role="doc-biblioref">685</a>]</span>.</p>
<h2 id="experiment-design">Experiment Design</h2>
<p>This section should discuss trade offs and balancing them to design an experiment.
1. constraints: Each experiment will have different constraints, which may include the number of samples needed for analysis, or desire to quantify a specific subset of proteins within a sample.
2. sample size
3. statistics
4. costs</p>
<h2 class="page_break_before" id="acknowledgements">Acknowledgements</h2>
<p>The authors thank Phil Wilmarth for helpful input.
Identification of certain commercial equipment, instruments, software, or materials does not imply recommendation or endorsement by the National Institute of Standards and Technology, nor does it imply that the products identified are necessarily the best available for the purpose.
The authors thank Dasom Hwang for help with graphic design.
The authors thank Anthony Gitter and Daniem Himmelstein for assistance using manubot.
The authors thank Jordan Burton and Pierre-Alexander Mücke for minor edits to the text.
This manuscript was written collaboratively using manubot <span class="citation" data-cites="YuJbg3zO">[<a href="#ref-YuJbg3zO" role="doc-biblioref">703</a>]</span>.
The live and evolving version where anyone can contribute can be found here https://github.com/jessegmeyerlab/proteomics-tutorial/tree/main.</p>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-4Ajjx5fo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline"><strong>Electrospray Ionization for Mass Spectrometry of Large Biomolecules</strong> <div class="csl-block">John B Fenn, Matthias Mann, Chin Kai Meng, Shek Fu Wong, Craig M Whitehouse</div> <em>Science</em> (1989-10-06) <a href="https://doi.org/cq2q43">https://doi.org/cq2q43</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.2675315">10.1126/science.2675315</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2675315">2675315</a></div></div>
</div>
<div id="ref-15Ye4Za7Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline"><strong>Protein and polymer analyses up tom/z 100 000 by laser ionization time-of-flight mass spectrometry</strong> <div class="csl-block">Koichi Tanaka, Hiroaki Waki, Yutaka Ido, Satoshi Akita, Yoshikazu Yoshida, Tamio Yoshida, T Matsuo</div> <em>Rapid Communications in Mass Spectrometry</em> (1988-08) <a href="https://doi.org/ffbwwr">https://doi.org/ffbwwr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/rcm.1290020802">10.1002/rcm.1290020802</a></div></div>
</div>
<div id="ref-pG7zq91Q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline"><strong>An approach to correlate tandem mass spectral data of peptides with amino acid sequences in a protein database.</strong> <div class="csl-block">JK Eng, AL McCormack, JR Yates</div> <em>Journal of the American Society for Mass Spectrometry</em> (1994-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24226387">https://www.ncbi.nlm.nih.gov/pubmed/24226387</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/1044-0305(94)80016-2">10.1016/1044-0305(94)80016-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24226387">24226387</a></div></div>
</div>
<div id="ref-ujahSo3G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline"><strong>An Automated Multidimensional Protein Identification Technology for Shotgun Proteomics</strong> <div class="csl-block">Dirk A Wolters, Michael P Washburn, John R Yates</div> <em>Analytical Chemistry</em> (2001-10-25) <a href="https://doi.org/bn4kq6">https://doi.org/bn4kq6</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac010617e">10.1021/ac010617e</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11774908">11774908</a></div></div>
</div>
<div id="ref-mw5jxASm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline"><strong>A Statistical Model for Identifying Proteins by Tandem Mass Spectrometry</strong> <div class="csl-block">Alexey I Nesvizhskii, Andrew Keller, Eugene Kolker, Ruedi Aebersold</div> <em>Analytical Chemistry</em> (2003-07-15) <a href="https://doi.org/b2xv45">https://doi.org/b2xv45</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac0341261">10.1021/ac0341261</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14632076">14632076</a></div></div>
</div>
<div id="ref-y9Bki24q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline"><strong>Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry</strong> <div class="csl-block">Joshua E Elias, Steven P Gygi</div> <em>Nature Methods</em> (2007-03) <a href="https://doi.org/djz7fz">https://doi.org/djz7fz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth1019">https://doi.org/10.1038/nmeth1019</a></div></div>
</div>
<div id="ref-16QSCuG38" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline"><strong>Mass-spectrometric exploration of proteome structure and function</strong> <div class="csl-block">Ruedi Aebersold, Matthias Mann</div> <em>Nature</em> (2016-09) <a href="https://doi.org/f83zqm">https://doi.org/f83zqm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nature19949">10.1038/nature19949</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27629641">27629641</a></div></div>
</div>
<div id="ref-a6Z8FNxW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline"><strong>High-throughput quantitative top-down proteomics</strong> <div class="csl-block">Kellye A Cupp-Sutton, Si Wu</div> <em>Molecular Omics</em> (2020) <a href="https://doi.org/gnx98p">https://doi.org/gnx98p</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/c9mo00154a">10.1039/c9mo00154a</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31932818">31932818</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7529119">PMC7529119</a></div></div>
</div>
<div id="ref-h00Trxkt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline"><strong>Proteoforms as the next proteomics currency</strong> <div class="csl-block">Lloyd M Smith, Neil L Kelleher</div> <em>Science</em> (2018-03-09) <a href="https://doi.org/gn6p4x">https://doi.org/gn6p4x</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.aat1884">10.1126/science.aat1884</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29590032">29590032</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5944612">PMC5944612</a></div></div>
</div>
<div id="ref-14nZGo469" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline"><strong>Novel Strategies to Address the Challenges in Top-Down Proteomics</strong> <div class="csl-block">Jake A Melby, David S Roberts, Eli J Larson, Kyle A Brown, Elizabeth F Bayne, Song Jin, Ying Ge</div> <em>Journal of the American Society for Mass Spectrometry</em> (2021-05-13) <a href="https://doi.org/gj2mjj">https://doi.org/gj2mjj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/jasms.1c00099">10.1021/jasms.1c00099</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33983025">33983025</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8310706">PMC8310706</a></div></div>
</div>
<div id="ref-CwZoo7w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline"><strong>Top-Down Proteomics: Ready for Prime Time?</strong> <div class="csl-block">Bifan Chen, Kyle A Brown, Ziqing Lin, Ying Ge</div> <em>Analytical Chemistry</em> (2017-11-21) <a href="https://doi.org/gssrqr">https://doi.org/gssrqr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.7b04747">10.1021/acs.analchem.7b04747</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29161012">29161012</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6138622">PMC6138622</a></div></div>
</div>
<div id="ref-Iei0Ms6X" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline"><strong>The quantitative proteome of a human cell line</strong> <div class="csl-block">Martin Beck, Alexander Schmidt, Johan Malmstroem, Manfred Claassen, Alessandro Ori, Anna Szymborska, Franz Herzog, Oliver Rinner, Jan Ellenberg, Ruedi Aebersold</div> <em>Molecular Systems Biology</em> (2011-01) <a href="https://doi.org/drjmfj">https://doi.org/drjmfj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/msb.2011.82">10.1038/msb.2011.82</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22068332">22068332</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3261713">PMC3261713</a></div></div>
</div>
<div id="ref-OQN3Pb5c" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline"><strong>A Quantitative Proteome Map of the Human Body</strong> <div class="csl-block">Lihua Jiang, Meng Wang, Shin Lin, Ruiqi Jian, Xiao Li, Joanne Chan, Guanlan Dong, Huaying Fang, Aaron E Robinson, Michael P Snyder, … Simona Volpi</div> <em>Cell</em> (2020-10) <a href="https://doi.org/ghbjrk">https://doi.org/ghbjrk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2020.08.036">10.1016/j.cell.2020.08.036</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32916130">32916130</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7575058">PMC7575058</a></div></div>
</div>
<div id="ref-6a8Xt7SG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline"><strong>Proteoform: a single term describing protein complexity</strong> <div class="csl-block">Lloyd M Smith, Neil L Kelleher</div> <em>Nature Methods</em> (2013-02-27) <a href="https://doi.org/gwcc">https://doi.org/gwcc</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.2369">10.1038/nmeth.2369</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23443629">23443629</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4114032">PMC4114032</a></div></div>
</div>
<div id="ref-m2ZOJt1R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline"><strong>Post-translational modifications in proteins: resources, tools and prediction methods</strong> <div class="csl-block">Shahin Ramazi, Javad Zahiri</div> <em>Database</em> (2021-01-01) <a href="https://doi.org/gpjs7g">https://doi.org/gpjs7g</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/database/baab012">10.1093/database/baab012</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33826699">33826699</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8040245">PMC8040245</a></div></div>
</div>
<div id="ref-uLIaWFNx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline"><strong>Posttranslational modifications in proteins: resources, tools and prediction methods.</strong> <div class="csl-block">Shahin Ramazi, Javad Zahiri</div> <em>Database : the journal of biological databases and curation</em> (2021-04-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/33826699">https://www.ncbi.nlm.nih.gov/pubmed/33826699</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/database/baab012">10.1093/database/baab012</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33826699">33826699</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8040245">PMC8040245</a></div></div>
</div>
<div id="ref-OEP3MV7r" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline"><strong>Protein post-translational modifications and regulation of pluripotency in human stem cells.</strong> <div class="csl-block">Yu-Chieh Wang, Suzanne E Peterson, Jeanne F Loring</div> <em>Cell research</em> (2013-11-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24217768">https://www.ncbi.nlm.nih.gov/pubmed/24217768</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/cr.2013.151">10.1038/cr.2013.151</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24217768">24217768</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3915910">PMC3915910</a></div></div>
</div>
<div id="ref-fAHUecLj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline"><strong>Proteolytic post-translational modification of proteins: proteomic tools and methodology.</strong> <div class="csl-block">Lindsay D Rogers, Christopher M Overall</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2013-07-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23887885">https://www.ncbi.nlm.nih.gov/pubmed/23887885</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m113.031310">10.1074/mcp.m113.031310</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23887885">23887885</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3861706">PMC3861706</a></div></div>
</div>
<div id="ref-e078CIsS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline"><strong>The crucial role of protein phosphorylation in cell signaling and its use as targeted therapy (Review).</strong> <div class="csl-block">Fatima Ardito, Michele Giuliani, Donatella Perrone, Giuseppe Troiano, Lorenzo Lo Muzio</div> <em>International journal of molecular medicine</em> (2017-06-22) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28656226">https://www.ncbi.nlm.nih.gov/pubmed/28656226</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3892/ijmm.2017.3036">10.3892/ijmm.2017.3036</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28656226">28656226</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5500920">PMC5500920</a></div></div>
</div>
<div id="ref-oTq7yQcv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline"><strong>Mitochondrial phosphorylation in apoptosis: flipping the death switch.</strong> <div class="csl-block">Natalie M Niemi, Jeffrey P MacKeigan</div> <em>Antioxidants &amp; redox signaling</em> (2012-12-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23088365">https://www.ncbi.nlm.nih.gov/pubmed/23088365</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1089/ars.2012.4982">10.1089/ars.2012.4982</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23088365">23088365</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3717198">PMC3717198</a></div></div>
</div>
<div id="ref-1DDSEV4N5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline"><strong>Phosphorylation of Bcl2 and regulation of apoptosis.</strong> <div class="csl-block">PP Ruvolo, X Deng, WS May</div> <em>Leukemia</em> (2001-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/11368354">https://www.ncbi.nlm.nih.gov/pubmed/11368354</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/sj.leu.2402090">10.1038/sj.leu.2402090</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11368354">11368354</a></div></div>
</div>
<div id="ref-1CHaUvQ1A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline"><strong>Protein degradation by the ubiquitin-proteasome pathway in normal and disease states.</strong> <div class="csl-block">Stewart H Lecker, Alfred L Goldberg, William E Mitch</div> <em>Journal of the American Society of Nephrology : JASN</em> (2006-05-31) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16738015">https://www.ncbi.nlm.nih.gov/pubmed/16738015</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1681/asn.2006010083">10.1681/asn.2006010083</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16738015">16738015</a></div></div>
</div>
<div id="ref-M4mSYOgE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline"><strong>SUMOylation-Mediated Regulation of Cell Cycle Progression and Cancer.</strong> <div class="csl-block">Karolin Eifler, Alfred CO Vertegaal</div> <em>Trends in biochemical sciences</em> (2015-10-22) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26601932">https://www.ncbi.nlm.nih.gov/pubmed/26601932</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.tibs.2015.09.006">10.1016/j.tibs.2015.09.006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26601932">26601932</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4874464">PMC4874464</a></div></div>
</div>
<div id="ref-4iAess79" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline"><strong>Site-specific identification and quantitation of endogenous SUMO modifications under native conditions.</strong> <div class="csl-block">Ryan J Lumpkin, Hongbo Gu, Yiying Zhu, Marilyn Leonard, Alla S Ahmad, Karl R Clauser, Jesse G Meyer, Eric J Bennett, Elizabeth A Komives</div> <em>Nature communications</em> (2017-10-27) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29079793">https://www.ncbi.nlm.nih.gov/pubmed/29079793</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-017-01271-3">10.1038/s41467-017-01271-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29079793">29079793</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5660086">PMC5660086</a></div></div>
</div>
<div id="ref-blcqlkZJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline"><strong>Palmitoylation is a post-translational modification of Alix regulating the membrane organization of exosome-like small extracellular vesicles.</strong> <div class="csl-block">Daniele P Romancino, Valentina Buffa, Stefano Caruso, Ines Ferrara, Samuele Raccosta, Antonietta Notaro, Yvan Campos, Rosina Noto, Vincenzo Martorana, Antonio Cupane, … Antonella Bongiovanni</div> <em>Biochimica et biophysica acta. General subjects</em> (2018-09-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30251702">https://www.ncbi.nlm.nih.gov/pubmed/30251702</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.bbagen.2018.09.004">10.1016/j.bbagen.2018.09.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30251702">30251702</a></div></div>
</div>
<div id="ref-V8FVI0EN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline"><strong>T cells modulate glycans on CD43 and CD45 during development and activation, signal regulation, and survival.</strong> <div class="csl-block">Mary C Clark, Linda G Baum</div> <em>Annals of the New York Academy of Sciences</em> (2012-01-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22288421">https://www.ncbi.nlm.nih.gov/pubmed/22288421</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/j.1749-6632.2011.06304.x">10.1111/j.1749-6632.2011.06304.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22288421">22288421</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4190024">PMC4190024</a></div></div>
</div>
<div id="ref-889bLrDQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline"><strong>Mammalian glycosylation in immunity.</strong> <div class="csl-block">Jamey D Marth, Prabhjit K Grewal</div> <em>Nature reviews. Immunology</em> (2008-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18846099">https://www.ncbi.nlm.nih.gov/pubmed/18846099</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nri2417">10.1038/nri2417</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18846099">18846099</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2768770">PMC2768770</a></div></div>
</div>
<div id="ref-1FiLH8A0G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline"><strong>Deciphering protein post-translational modifications using chemical biology tools</strong> <div class="csl-block">Anne C Conibear</div> <em>Nature Reviews Chemistry</em> (2020-10-06) <a href="https://doi.org/gqnxhb">https://doi.org/gqnxhb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41570-020-00223-8">10.1038/s41570-020-00223-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37127974">37127974</a></div></div>
</div>
<div id="ref-fzBRT8ZP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline"><strong>Analysis of yeast protein kinases using protein chips.</strong> <div class="csl-block">H Zhu, JF Klemic, S Chang, P Bertone, A Casamayor, KG Klemic, D Smith, M Gerstein, MA Reed, M Snyder</div> <em>Nature genetics</em> (2000-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/11062466">https://www.ncbi.nlm.nih.gov/pubmed/11062466</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/81576">10.1038/81576</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11062466">11062466</a></div></div>
</div>
<div id="ref-Wcw3KfEB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline"><strong>Peptide arrays: from macro to micro.</strong> <div class="csl-block">Ulf Reimer, Ulrich Reineke, Jens Schneider-Mergener</div> <em>Current opinion in biotechnology</em> (2002-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/12323352">https://www.ncbi.nlm.nih.gov/pubmed/12323352</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0958-1669(02)00339-7">10.1016/s0958-1669(02)00339-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12323352">12323352</a></div></div>
</div>
<div id="ref-nOIi7YMn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline"><strong>Intrinsically Disordered Proteins and Intrinsically Disordered Protein Regions</strong> <div class="csl-block">Christopher J Oldfield, AKeith Dunker</div> <em>Annual Review of Biochemistry</em> (2014-06-02) <a href="https://doi.org/gfw6t4">https://doi.org/gfw6t4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1146/annurev-biochem-072711-164947">10.1146/annurev-biochem-072711-164947</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24606139">24606139</a></div></div>
</div>
<div id="ref-aq0DyuaG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline"><strong>A Review: Molecular Chaperone-mediated Folding, Unfolding and Disaggregation of Expressed Recombinant Proteins</strong> <div class="csl-block">Komal Fatima, Fatima Naqvi, Hooria Younas</div> <em>Cell Biochemistry and Biophysics</em> (2021-02-25) <a href="https://doi.org/gpd4pb">https://doi.org/gpd4pb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s12013-021-00970-5">10.1007/s12013-021-00970-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33634426">33634426</a></div></div>
</div>
<div id="ref-UeFCwwR7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline"><strong>Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing</strong> <div class="csl-block">Yoav Benjamini, Yosef Hochberg</div> <em>Journal of the Royal Statistical Society: Series B (Methodological)</em> (1995-01) <a href="https://doi.org/gfpkdx">https://doi.org/gfpkdx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/j.2517-6161.1995.tb02031.x">10.1111/j.2517-6161.1995.tb02031.x</a></div></div>
</div>
<div id="ref-E5n1UfBA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline"><strong>Large-scale phosphorylation analysis of mouse liver</strong> <div class="csl-block">Judit Villén, Sean A Beausoleil, Scott A Gerber, Steven P Gygi</div> <em>Proceedings of the National Academy of Sciences</em> (2007-01-30) <a href="https://doi.org/c7zbpm">https://doi.org/c7zbpm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.0609836104">10.1073/pnas.0609836104</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17242355">17242355</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1785252">PMC1785252</a></div></div>
</div>
<div id="ref-N0xHE81w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline"><strong>PhosphoPep—a phosphoproteome resource for systems biology research in
&lt;i&gt;Drosophila&lt;/i&gt;
Kc167 cells</strong> <div class="csl-block">Bernd Bodenmiller, Johan Malmstrom, Bertran Gerrits, David Campbell, Henry Lam, Alexander Schmidt, Oliver Rinner, Lukas N Mueller, Paul T Shannon, Patrick G Pedrioli, … Ruedi Aebersold</div> <em>Molecular Systems Biology</em> (2007-01) <a href="https://doi.org/fg6rnn">https://doi.org/fg6rnn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/msb4100182">10.1038/msb4100182</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17940529">17940529</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2063582">PMC2063582</a></div></div>
</div>
<div id="ref-4q5oxW6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline"><strong>Quantitative Analysis of Phosphotyrosine Signaling Networks Triggered by CD3 and CD28 Costimulation in Jurkat Cells</strong> <div class="csl-block">Ji-Eun Kim, Forest M White</div> <em>The Journal of Immunology</em> (2006-03-01) <a href="https://doi.org/gr3m64">https://doi.org/gr3m64</a> <div class="csl-block">DOI: <a href="https://doi.org/10.4049/jimmunol.176.5.2833">10.4049/jimmunol.176.5.2833</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16493040">16493040</a></div></div>
</div>
<div id="ref-1E7oJsvFz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">37. </div><div class="csl-right-inline"><strong>Quantitative Phosphoproteomic Analysis of T Cell Receptor Signaling Reveals System-Wide Modulation of Protein-Protein Interactions</strong> <div class="csl-block">Viveka Mayya, Deborah H Lundgren, Sun-Il Hwang, Karim Rezaul, Linfeng Wu, Jimmy K Eng, Vladimir Rodionov, David K Han</div> <em>Science Signaling</em> (2009-08-18) <a href="https://doi.org/d3czmn">https://doi.org/d3czmn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/scisignal.2000007">10.1126/scisignal.2000007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19690332">19690332</a></div></div>
</div>
<div id="ref-EiFGJCed" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">38. </div><div class="csl-right-inline"><strong>Quantitative phosphoproteomic analysis of signaling network dynamics</strong> <div class="csl-block">Forest M White</div> <em>Current Opinion in Biotechnology</em> (2008-08) <a href="https://doi.org/bhd9vx">https://doi.org/bhd9vx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.copbio.2008.06.006">10.1016/j.copbio.2008.06.006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18619541">18619541</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2567239">PMC2567239</a></div></div>
</div>
<div id="ref-PNr94DXb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline"><strong>Glycoproteomics</strong> <div class="csl-block">Ieva Bagdonaite, Stacy A Malaker, Daniel A Polasky, Nicholas M Riley, Katrine Schjoldager, Sergey Y Vakhrushev, Adnan Halim, Kiyoko F Aoki-Kinoshita, Alexey I Nesvizhskii, Carolyn R Bertozzi, … Nichollas E Scott</div> <em>Nature Reviews Methods Primers</em> (2022-06-23) <a href="https://doi.org/gr3m6w">https://doi.org/gr3m6w</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s43586-022-00128-4">10.1038/s43586-022-00128-4</a></div></div>
</div>
<div id="ref-MH9QSnbc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">40. </div><div class="csl-right-inline"><strong>Biological roles of glycans</strong> <div class="csl-block">Ajit Varki</div> <em>Glycobiology</em> (2016-08-24) <a href="https://doi.org/f9v9j5">https://doi.org/f9v9j5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/glycob/cww086">10.1093/glycob/cww086</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27558841">27558841</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5884436">PMC5884436</a></div></div>
</div>
<div id="ref-nSDlzuSO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">41. </div><div class="csl-right-inline"><strong>Glycosylation in Cellular Mechanisms of Health and Disease</strong> <div class="csl-block">Kazuaki Ohtsubo, Jamey D Marth</div> <em>Cell</em> (2006-09) <a href="https://doi.org/d2qc7z">https://doi.org/d2qc7z</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2006.08.019">10.1016/j.cell.2006.08.019</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16959566">16959566</a></div></div>
</div>
<div id="ref-buMJaA2S" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline"><strong>Neurological Aspects of Human Glycosylation Disorders</strong> <div class="csl-block">Hudson H Freeze, Erik A Eklund, Bobby G Ng, Marc C Patterson</div> <em>Annual Review of Neuroscience</em> (2015-07-08) <a href="https://doi.org/f3pb3r">https://doi.org/f3pb3r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1146/annurev-neuro-071714-034019">10.1146/annurev-neuro-071714-034019</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25840006">25840006</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4809143">PMC4809143</a></div></div>
</div>
<div id="ref-Iygi3cGd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">43. </div><div class="csl-right-inline"><strong>Universal sample preparation method for proteome analysis</strong> <div class="csl-block">Jacek R Wiśniewski, Alexandre Zougman, Nagarjuna Nagaraj, Matthias Mann</div> <em>Nature Methods</em> (2009-04-19) <a href="https://doi.org/cfbrvh">https://doi.org/cfbrvh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.1322">10.1038/nmeth.1322</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19377485">19377485</a></div></div>
</div>
<div id="ref-1FU2NfVYY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">44. </div><div class="csl-right-inline"><strong>Suspension trapping (STrap) sample preparation method for bottom-up proteomics analysis</strong> <div class="csl-block">Alexandre Zougman, Peter J Selby, Rosamonde E Banks</div> <em>PROTEOMICS</em> (2014-03-26) <a href="https://doi.org/f2rdw4">https://doi.org/f2rdw4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.201300553">10.1002/pmic.201300553</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24678027">24678027</a></div></div>
</div>
<div id="ref-yO8bBfo8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">45. </div><div class="csl-right-inline"><strong>Protein Aggregation Capture on Microparticles Enables Multipurpose Proteomics Sample Preparation*</strong> <div class="csl-block">Tanveer S Batth, MaximAX Tollenaere, Patrick Rüther, Alba Gonzalez-Franquesa, Bhargav S Prabhakar, Simon Bekker-Jensen, Atul S Deshmukh, Jesper V Olsen</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2019-05) <a href="https://doi.org/gf3vvs">https://doi.org/gf3vvs</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.tir118.001270">10.1074/mcp.tir118.001270</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30833379">30833379</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6495262">PMC6495262</a></div></div>
</div>
<div id="ref-7zlczgkb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">46. </div><div class="csl-right-inline"><strong>S-Trap, an Ultrafast Sample-Preparation Approach for Shotgun Proteomics</strong> <div class="csl-block">Milkessa HaileMariam, Rodrigo Vargas Eguez, Harinder Singh, Shiferaw Bekele, Gobena Ameni, Rembert Pieper, Yanbao Yu</div> <em>Journal of Proteome Research</em> (2018-08-16) <a href="https://doi.org/gd9kf5">https://doi.org/gd9kf5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.8b00505">10.1021/acs.jproteome.8b00505</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30114372">30114372</a></div></div>
</div>
<div id="ref-t0hz9mu6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">47. </div><div class="csl-right-inline"><strong>Multiplexed CuAAC Suzuki–Miyaura Labeling for Tandem Activity-Based Chemoproteomic Profiling</strong> <div class="csl-block">Jian Cao, Lisa M Boatner, Heta S Desai, Nikolas R Burton, Ernest Armenta, Neil J Chan, José O Castellón, Keriann M Backus</div> <em>Analytical Chemistry</em> (2021-01-20) <a href="https://doi.org/gr3m6q">https://doi.org/gr3m6q</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.0c04726">10.1021/acs.analchem.0c04726</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33470097">33470097</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8849040">PMC8849040</a></div></div>
</div>
<div id="ref-VTprpeAT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">48. </div><div class="csl-right-inline"><strong>SP3-FAIMS Chemoproteomics for High Coverage Profiling of the Human Cysteinome</strong> <div class="csl-block">Tianyang Yan, Heta Desai, Lisa Boatner, Stephanie Yen, Jian Cao, Maria Palafox, Yasaman Jami-Alahmadi, Keriann Backus</div> <em>American Chemical Society (ACS)</em> (2020-12-28) <a href="https://doi.org/gr3m63">https://doi.org/gr3m63</a> <div class="csl-block">DOI: <a href="https://doi.org/10.26434/chemrxiv.13487364.v1">10.26434/chemrxiv.13487364.v1</a></div></div>
</div>
<div id="ref-yeQvImXS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">49. </div><div class="csl-right-inline"><strong>Expanding Proteome Coverage with Orthogonal-specificity α-Lytic Proteases</strong> <div class="csl-block">Jesse G Meyer, Sangtae Kim, David A Maltby, Majid Ghassemian, Nuno Bandeira, Elizabeth A Komives</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2014-03) <a href="https://doi.org/f5vgcg">https://doi.org/f5vgcg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m113.034710">10.1074/mcp.m113.034710</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24425750">24425750</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3945911">PMC3945911</a></div></div>
</div>
<div id="ref-waP2bBmk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">50. </div><div class="csl-right-inline"><strong>XL-MS: Protein cross-linking coupled with mass spectrometry</strong> <div class="csl-block">Andrew N Holding</div> <em>Methods</em> (2015-11) <a href="https://doi.org/gftzb6">https://doi.org/gftzb6</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ymeth.2015.06.010">10.1016/j.ymeth.2015.06.010</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26079926">26079926</a></div></div>
</div>
<div id="ref-XYcoBTz1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">51. </div><div class="csl-right-inline"><strong>To Cleave or Not To Cleave in XL-MS?</strong> <div class="csl-block">B Steigenberger, P Albanese, AJR Heck, RA Scheltema</div> <em>Journal of the American Society for Mass Spectrometry</em> (2019-12-27) <a href="https://doi.org/gr3m6s">https://doi.org/gr3m6s</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/jasms.9b00085">10.1021/jasms.9b00085</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32031400">32031400</a></div></div>
</div>
<div id="ref-v6mVdK34" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">52. </div><div class="csl-right-inline"><strong>Cross-Linking Mass Spectrometry: An Emerging Technology for Interactomics and Structural Biology</strong> <div class="csl-block">Clinton Yu, Lan Huang</div> <em>Analytical Chemistry</em> (2017-11-21) <a href="https://doi.org/gfvmcj">https://doi.org/gfvmcj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.7b04431">10.1021/acs.analchem.7b04431</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29160693">29160693</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6022837">PMC6022837</a></div></div>
</div>
<div id="ref-JnaXzrse" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">53. </div><div class="csl-right-inline"><strong>A New in Vivo Cross-linking Mass Spectrometry Platform to Define Protein–Protein Interactions in Living Cells</strong> <div class="csl-block">Robyn M Kaake, Xiaorong Wang, Anthony Burke, Clinton Yu, Wynne Kandur, Yingying Yang, Eric J Novtisky, Tonya Second, Jicheng Duan, Athit Kao, … Lan Huang</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2014-12) <a href="https://doi.org/f6r7qw">https://doi.org/f6r7qw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m114.042630">10.1074/mcp.m114.042630</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25253489">25253489</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4256503">PMC4256503</a></div></div>
</div>
<div id="ref-KW2lStHJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">54. </div><div class="csl-right-inline"><strong>Identification of cross-linked peptides from complex samples</strong> <div class="csl-block">Bing Yang, Yan-Jie Wu, Ming Zhu, Sheng-Bo Fan, Jinzhong Lin, Kun Zhang, Shuang Li, Hao Chi, Yu-Xin Li, Hai-Feng Chen, … Meng-Qiu Dong</div> <em>Nature Methods</em> (2012-07-08) <a href="https://doi.org/f38rv2">https://doi.org/f38rv2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.2099">10.1038/nmeth.2099</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22772728">22772728</a></div></div>
</div>
<div id="ref-1Dqlo9aRm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">55. </div><div class="csl-right-inline"><strong>Recommendations for performing, interpreting and reporting hydrogen deuterium exchange mass spectrometry (HDX-MS) experiments</strong> <div class="csl-block">Glenn R Masson, John E Burke, Natalie G Ahn, Ganesh S Anand, Christoph Borchers, Sébastien Brier, George M Bou-Assaf, John R Engen, SWalter Englander, Johan Faber, … Kasper D Rand</div> <em>Nature Methods</em> (2019-06-27) <a href="https://doi.org/ghh62q">https://doi.org/ghh62q</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-019-0459-y">10.1038/s41592-019-0459-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31249422">31249422</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6614034">PMC6614034</a></div></div>
</div>
<div id="ref-cZM0UPqp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">56. </div><div class="csl-right-inline"><strong>Hydrogen exchange mass spectrometry for studying protein structure and dynamics</strong> <div class="csl-block">Lars Konermann, Jingxi Pan, Yu-Hong Liu</div> <em>Chem. Soc. Rev.</em> (2011) <a href="https://doi.org/db943b">https://doi.org/db943b</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/c0cs00113a">10.1039/c0cs00113a</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21173980">21173980</a></div></div>
</div>
<div id="ref-SZzVEJCj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">57. </div><div class="csl-right-inline"><strong>Hydrogen exchange in proteins.</strong> <div class="csl-block">A Hvidt, SO Nielsen</div> <em>Advances in protein chemistry</em> (1966) <a href="https://www.ncbi.nlm.nih.gov/pubmed/5333290">https://www.ncbi.nlm.nih.gov/pubmed/5333290</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0065-3233(08)60129-1">10.1016/s0065-3233(08)60129-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/5333290">5333290</a></div></div>
</div>
<div id="ref-H647Pppu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">58. </div><div class="csl-right-inline"><strong>Hydrogen exchange and structural dynamics of proteins and nucleic acids</strong> <div class="csl-block">SWalter Englander, Neville R Kallenbach</div> <em>Quarterly Reviews of Biophysics</em> (1983-11) <a href="https://doi.org/cxj4z7">https://doi.org/cxj4z7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1017/s0033583500005217">10.1017/s0033583500005217</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/6204354">6204354</a></div></div>
</div>
<div id="ref-1CHLgMzps" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">59. </div><div class="csl-right-inline"><strong>Differential hydrogen/deuterium exchange mass spectrometry analysis of protein–ligand interactions</strong> <div class="csl-block">Michael J Chalmers, Scott A Busby, Bruce D Pascal, Graham M West, Patrick R Griffin</div> <em>Expert Review of Proteomics</em> (2011-02) <a href="https://doi.org/btrkr9">https://doi.org/btrkr9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1586/epr.10.109">10.1586/epr.10.109</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21329427">21329427</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3113475">PMC3113475</a></div></div>
</div>
<div id="ref-19wzQMZwm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">60. </div><div class="csl-right-inline"><strong>Hydrogen Exchange Mass Spectrometry: Are We Out of the Quicksand?</strong> <div class="csl-block">Roxana E Iacob, John R Engen</div> <em>Journal of the American Society for Mass Spectrometry</em> (2012-04-03) <a href="https://doi.org//10.1007/s13361-012-0377-z">https://doi.org//10.1007/s13361-012-0377-z</a> <div class="csl-block">DOI: <a href="https://doi.org//10.1007/s13361-012-0377-z">/10.1007/s13361-012-0377-z</a></div></div>
</div>
<div id="ref-m2HWEBQZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">61. </div><div class="csl-right-inline"><strong>Replication in bioanalytical studies with HDX MS: aim as high as possible</strong> <div class="csl-block">Jamie A Moroco, John R Engen</div> <em>Bioanalysis</em> (2015-05) <a href="https://doi.org/gr3m65">https://doi.org/gr3m65</a> <div class="csl-block">DOI: <a href="https://doi.org/10.4155/bio.15.46">10.4155/bio.15.46</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26039804">26039804</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4837975">PMC4837975</a></div></div>
</div>
<div id="ref-whtWKxfa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">62. </div><div class="csl-right-inline"><strong>Development of a Peptide Probe for the Occurrence of Hydrogen (&lt;sup&gt;1&lt;/sup&gt;H/&lt;sup&gt;2&lt;/sup&gt;H) Scrambling upon Gas-Phase Fragmentation</strong> <div class="csl-block">Kasper D Rand, Thomas JD Jørgensen</div> <em>Analytical Chemistry</em> (2007-10-13) <a href="https://doi.org/c24d6z">https://doi.org/c24d6z</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac0710782">10.1021/ac0710782</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17935303">17935303</a></div></div>
</div>
<div id="ref-17FfyYJKn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">63. </div><div class="csl-right-inline"><strong>Hydroxyl Radical-Mediated Modification of Proteins as Probes for Structural Proteomics</strong> <div class="csl-block">Guozhong Xu, Mark R Chance</div> <em>Chemical Reviews</em> (2007-08-01) <a href="https://doi.org/c769n8">https://doi.org/c769n8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/cr0682047">10.1021/cr0682047</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17683160">17683160</a></div></div>
</div>
<div id="ref-dyFOYdP5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">64. </div><div class="csl-right-inline"><strong>Structural Mass Spectrometry of Proteins Using Hydroxyl Radical Based Protein Footprinting</strong> <div class="csl-block">Liwen Wang, Mark R Chance</div> <em>Analytical Chemistry</em> (2011-08-01) <a href="https://doi.org/fvrx3k">https://doi.org/fvrx3k</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac200567u">10.1021/ac200567u</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21770468">21770468</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3184339">PMC3184339</a></div></div>
</div>
<div id="ref-8tWrfuT2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">65. </div><div class="csl-right-inline"><strong>Mapping nucleic acid structure by hydroxyl radical cleavage</strong> <div class="csl-block">Thomas D Tullius, Jason A Greenbaum</div> <em>Current Opinion in Chemical Biology</em> (2005-04) <a href="https://doi.org/cdcbtp">https://doi.org/cdcbtp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cbpa.2005.02.009">10.1016/j.cbpa.2005.02.009</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15811796">15811796</a></div></div>
</div>
<div id="ref-b8XE9cl8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">66. </div><div class="csl-right-inline"><strong>Probing DNA Structure with Hydroxyl Radicals</strong> <div class="csl-block">Thomas D Tullius</div> <em>Current Protocols in Nucleic Acid Chemistry</em> (2001-12) <a href="https://doi.org/bf493d">https://doi.org/bf493d</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/0471142700.nc0607s07">10.1002/0471142700.nc0607s07</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18428898">18428898</a></div></div>
</div>
<div id="ref-14ACSyAnR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">67. </div><div class="csl-right-inline"><strong>Targeted DNA oxidation and trajectory of radical DNA using DFT based QM/MM dynamics</strong> <div class="csl-block">Pradip K Biswas, Sandipan Chakraborty</div> <em>Nucleic Acids Research</em> (2019-02-18) <a href="https://doi.org/gr3m6z">https://doi.org/gr3m6z</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/gkz089">10.1093/nar/gkz089</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30773597">30773597</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6451130">PMC6451130</a></div></div>
</div>
<div id="ref-muDfX3XR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">68. </div><div class="csl-right-inline"><strong>&lt;scp&gt;MS&lt;/scp&gt;
‐Based Hydroxyl Radical Footprinting: Methodology and Application of Fast Photochemical Oxidation of Proteins (
&lt;scp&gt;FPOP&lt;/scp&gt;
)</strong> <div class="csl-block">Ben Niu, Michael L Gross</div> <em>Mass Spectrometry‐Based Chemical Proteomics</em> (2019-07-26) <a href="https://doi.org/gr3m6d">https://doi.org/gr3m6d</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/9781118970195.ch15">10.1002/9781118970195.ch15</a></div></div>
</div>
<div id="ref-zy4wero" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">69. </div><div class="csl-right-inline"><strong>Fenton and Fenton-like catalysts for electrochemical immunoassay: A mini review</strong> <div class="csl-block">Jiejie Feng, Changshun Chu, Zhanfang Ma</div> <em>Electrochemistry Communications</em> (2021-04) <a href="https://doi.org/gr3m6h">https://doi.org/gr3m6h</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.elecom.2021.106970">10.1016/j.elecom.2021.106970</a></div></div>
</div>
<div id="ref-bAhX0GGr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">70. </div><div class="csl-right-inline"><strong>Inactivation of MS2 coliphage in Fenton and Fenton-like systems: role of transition metals, hydrogen peroxide and sunlight</strong> <div class="csl-block">Jessica I Nieto-Juarez, Katarzyna Pierzchła, Andrzej Sienkiewicz, Tamar Kohn</div> <em>Environmental Science &amp;amp; Technology</em> (2010-03-31) <a href="https://doi.org/fmmv29">https://doi.org/fmmv29</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/es903739f">10.1021/es903739f</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20356037">20356037</a></div></div>
</div>
<div id="ref-GyzG2odq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">71. </div><div class="csl-right-inline"><strong>The use of the methods of radiolysis to explore the mechanisms of free radical modifications in proteins</strong> <div class="csl-block">Chantal Houée-Levin, Krzysztof Bobrowski</div> <em>Journal of Proteomics</em> (2013-10) <a href="https://doi.org/f5kn8p">https://doi.org/f5kn8p</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jprot.2013.02.014">10.1016/j.jprot.2013.02.014</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23454334">23454334</a></div></div>
</div>
<div id="ref-tcDiyhQT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">72. </div><div class="csl-right-inline"><strong>High voltage electrical discharges followed by deep eutectic solvents extraction for the valorization of pomegranate seeds (Punica granatum L.)</strong> <div class="csl-block">Ester Hernández-Corroto, Nadia Boussetta, María Luisa Marina, María Concepción García, Eugène Vorobiev</div> <em>Innovative Food Science &amp;amp; Emerging Technologies</em> (2022-07) <a href="https://doi.org/gr3m6j">https://doi.org/gr3m6j</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ifset.2022.103055">10.1016/j.ifset.2022.103055</a></div></div>
</div>
<div id="ref-gryR5tjl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">73. </div><div class="csl-right-inline"><strong>Structural proteomics: Topology and relative accessibility of plant lipid droplet associated proteins</strong> <div class="csl-block">Pascale Jolivet, Laure Aymé, Alexandre Giuliani, Frank Wien, Thierry Chardot, Yann Gohon</div> <em>Journal of Proteomics</em> (2017-10) <a href="https://doi.org/gcnk7p">https://doi.org/gcnk7p</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jprot.2017.09.005">10.1016/j.jprot.2017.09.005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28918933">28918933</a></div></div>
</div>
<div id="ref-43HI8GWO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">74. </div><div class="csl-right-inline"><strong>Proteomic analysis of Saccharomyces cerevisiae response to plasma treatment</strong> <div class="csl-block">Višnja Stulić, Tomislava Vukušić, Ana Butorac, Dean Popović, Zoran Herceg</div> <em>International Journal of Food Microbiology</em> (2019-03) <a href="https://doi.org/gr3m6k">https://doi.org/gr3m6k</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ijfoodmicro.2018.12.017">10.1016/j.ijfoodmicro.2018.12.017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30639916">30639916</a></div></div>
</div>
<div id="ref-pFnbzp8Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">75. </div><div class="csl-right-inline"><strong>Laser flash photolysis of hydrogen peroxide to oxidize protein solvent-accessible residues on the microsecond timescale</strong> <div class="csl-block">David M Hambly, Michael L Gross</div> <em>Journal of the American Society for Mass Spectrometry</em> (2005-12-01) <a href="https://doi.org/cjbbvw">https://doi.org/cjbbvw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jasms.2005.09.008">10.1016/j.jasms.2005.09.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16263307">16263307</a></div></div>
</div>
<div id="ref-IH1lj0uY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">76. </div><div class="csl-right-inline"><strong>Fast photochemical oxidation of proteins (FPOP): A powerful mass spectrometry–based structural proteomics tool</strong> <div class="csl-block">Danté T Johnson, Luciano H Di Stefano, Lisa M Jones</div> <em>Journal of Biological Chemistry</em> (2019-08) <a href="https://doi.org/ghzz8z">https://doi.org/ghzz8z</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/jbc.rev119.006218">10.1074/jbc.rev119.006218</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31262727">31262727</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6690683">PMC6690683</a></div></div>
</div>
<div id="ref-cMJbHB1a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">77. </div><div class="csl-right-inline"><strong>Laser flash photochemical oxidation to locate heme binding and conformational changes in myoglobin</strong> <div class="csl-block">David Hambly, Michael Gross</div> <em>International Journal of Mass Spectrometry</em> (2007-01) <a href="https://doi.org/fvzqsq">https://doi.org/fvzqsq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ijms.2006.08.018">10.1016/j.ijms.2006.08.018</a></div></div>
</div>
<div id="ref-LgS9wDAM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">78. </div><div class="csl-right-inline"><strong>Modifications generated by fast photochemical oxidation of proteins reflect the native conformations of proteins</strong> <div class="csl-block">Emily E Chea, Lisa M Jones</div> <em>Protein Science</em> (2018-04-14) <a href="https://doi.org/gdpkmn">https://doi.org/gdpkmn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pro.3408">10.1002/pro.3408</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29575296">29575296</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5980583">PMC5980583</a></div></div>
</div>
<div id="ref-sqSJTywu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">79. </div><div class="csl-right-inline"><strong>Fast Photochemical Oxidation of Protein Footprints Faster than Protein Unfolding</strong> <div class="csl-block">Brian C Gau, Joshua S Sharp, Don L Rempel, Michael L Gross</div> <em>Analytical Chemistry</em> (2009-07-23) <a href="https://doi.org/djvkzx">https://doi.org/djvkzx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac901054w">10.1021/ac901054w</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20337372">20337372</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3164994">PMC3164994</a></div></div>
</div>
<div id="ref-sZQji0TC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">80. </div><div class="csl-right-inline"><strong>Protein painting reveals solvent-excluded drug targets hidden within native protein–protein interfaces</strong> <div class="csl-block">Alessandra Luchini, Virginia Espina, Lance A Liotta</div> <em>Nature Communications</em> (2014-07-22) <a href="https://doi.org/gr3m6t">https://doi.org/gr3m6t</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/ncomms5413">10.1038/ncomms5413</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25048602">25048602</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4109009">PMC4109009</a></div></div>
</div>
<div id="ref-17xQ2aw0Q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">81. </div><div class="csl-right-inline"><strong>Protein painting, an optimized MS-based technique, reveals functionally relevant interfaces of the PD-1/PD-L1 complex and the YAP2/ZO-1 complex</strong> <div class="csl-block">Amanda Haymond, Douglass Dey, Rachel Carter, Angela Dailing, Vaishnavi Nara, Pranavi Nara, Sravani Venkatayogi, Mikell Paige, Lance Liotta, Alessandra Luchini</div> <em>Journal of Biological Chemistry</em> (2019-07) <a href="https://doi.org/ghmhrd">https://doi.org/ghmhrd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/jbc.ra118.007310">10.1074/jbc.ra118.007310</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31167787">31167787</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6643031">PMC6643031</a></div></div>
</div>
<div id="ref-gdTpkDDu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">82. </div><div class="csl-right-inline"><strong>Detecting Protein–Small Molecule Interactions Using Limited Proteolysis–Mass Spectrometry (LiP-MS)</strong> <div class="csl-block">Monika Pepelnjak, Natalie de Souza, Paola Picotti</div> <em>Trends in Biochemical Sciences</em> (2020-10) <a href="https://doi.org/gr3m6m">https://doi.org/gr3m6m</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.tibs.2020.05.006">10.1016/j.tibs.2020.05.006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32475682">32475682</a></div></div>
</div>
<div id="ref-xTQMmTt5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">83. </div><div class="csl-right-inline"><strong>Measuring protein structural changes on a proteome-wide scale using limited proteolysis-coupled mass spectrometry</strong> <div class="csl-block">Simone Schopper, Abdullah Kahraman, Pascal Leuenberger, Yuehan Feng, Ilaria Piazza, Oliver Müller, Paul J Boersema, Paola Picotti</div> <em>Nature Protocols</em> (2017-10-26) <a href="https://doi.org/gcf7zn">https://doi.org/gcf7zn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nprot.2017.100">10.1038/nprot.2017.100</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29072706">29072706</a></div></div>
</div>
<div id="ref-6IZfi1O" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">84. </div><div class="csl-right-inline"><strong>Proteome-wide structural changes measured with limited proteolysis-mass spectrometry: an advanced protocol for high-throughput applications</strong> <div class="csl-block">Liliana Malinovska, Valentina Cappelletti, Devon Kohler, Ilaria Piazza, Tsung-Heng Tsai, Monika Pepelnjak, Patrick Stalder, Christian Dörig, Fabian Sesterhenn, Franziska Elsässer, … Paola Picotti</div> <em>Nature Protocols</em> (2022-12-16) <a href="https://doi.org/gr3m6v">https://doi.org/gr3m6v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41596-022-00771-x">10.1038/s41596-022-00771-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36526727">36526727</a></div></div>
</div>
<div id="ref-1BxnTxtGI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">85. </div><div class="csl-right-inline"><a href="https://www.eubopen.org/sites/www.eubopen.org/files/attachments/2022/LiP-MS%20protocol.pdf">https://www.eubopen.org/sites/www.eubopen.org/files/attachments/2022/LiP-MS%20protocol.pdf</a></div>
</div>
<div id="ref-JOskUci" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">86. </div><div class="csl-right-inline"><strong>Thermal proteome profiling for interrogating protein interactions</strong> <div class="csl-block">André Mateus, Nils Kurzawa, Isabelle Becher, Sindhuja Sridharan, Dominic Helm, Frank Stein, Athanasios Typas, Mikhail M Savitski</div> <em>Molecular Systems Biology</em> (2020-03) <a href="https://doi.org/gr3m62">https://doi.org/gr3m62</a> <div class="csl-block">DOI: <a href="https://doi.org/10.15252/msb.20199232">10.15252/msb.20199232</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32133759">32133759</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7057112">PMC7057112</a></div></div>
</div>
<div id="ref-8UcGwfym" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">87. </div><div class="csl-right-inline"><strong>Thermal proteome profiling: Insights into protein modifications, associations, and functions</strong> <div class="csl-block">Cecile Le Sueur, Henrik M Hammarén, Sindhuja Sridharan, Mikhail M Savitski</div> <em>Current Opinion in Chemical Biology</em> (2022-12) <a href="https://doi.org/grhp63">https://doi.org/grhp63</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cbpa.2022.102225">10.1016/j.cbpa.2022.102225</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36368297">36368297</a></div></div>
</div>
<div id="ref-89c9sDdK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">88. </div><div class="csl-right-inline"><strong>Thermal Proteome Profiling Reveals the O-GlcNAc-Dependent Meltome</strong> <div class="csl-block">Dustin T King, Jesús E Serrano-Negrón, Yanping Zhu, Christopher L Moore, Matthew D Shoulders, Leonard J Foster, David J Vocadlo</div> <em>Journal of the American Chemical Society</em> (2022-03-01) <a href="https://doi.org/gqqm4n">https://doi.org/gqqm4n</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/jacs.1c10621">10.1021/jacs.1c10621</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35230102">35230102</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8969899">PMC8969899</a></div></div>
</div>
<div id="ref-Ssg2QwSE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">89. </div><div class="csl-right-inline"><strong>Target Discovery Using Thermal Proteome Profiling</strong> <div class="csl-block">Sindhuja Sridharan, Ina Günthner, Isabelle Becher, Mikhail Savitski, Marcus Bantscheff</div> <em>Mass Spectrometry‐Based Chemical Proteomics</em> (2019-07-26) <a href="https://doi.org/gr3m6c">https://doi.org/gr3m6c</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/9781118970195.ch11">10.1002/9781118970195.ch11</a></div></div>
</div>
<div id="ref-rMCVhIUo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">90. </div><div class="csl-right-inline"><strong>CETSA in integrated proteomics studies of cellular processes</strong> <div class="csl-block">Nayana Prabhu, Lingyun Dai, Pär Nordlund</div> <em>Current Opinion in Chemical Biology</em> (2020-02) <a href="https://doi.org/gh9nqs">https://doi.org/gh9nqs</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cbpa.2019.11.004">10.1016/j.cbpa.2019.11.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31838273">31838273</a></div></div>
</div>
<div id="ref-ieBDHPPg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">91. </div><div class="csl-right-inline"><strong>Quantitative proteomics: A strategic ally to map protein interaction networks</strong> <div class="csl-block">Miguel Marcilla, Juan Pablo Albar</div> <em>IUBMB Life</em> (2012-12-24) <a href="https://doi.org/f4gpwr">https://doi.org/f4gpwr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/iub.1081">10.1002/iub.1081</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23281033">23281033</a></div></div>
</div>
<div id="ref-DORXXewQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">92. </div><div class="csl-right-inline"><strong>Next-generation proteomics: towards an integrative view of proteome dynamics</strong> <div class="csl-block">AFMaarten Altelaar, Javier Munoz, Albert JR Heck</div> <em>Nature Reviews Genetics</em> (2012-12-04) <a href="https://doi.org/ghkhz2">https://doi.org/ghkhz2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nrg3356">10.1038/nrg3356</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23207911">23207911</a></div></div>
</div>
<div id="ref-1BsZO9WFn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">93. </div><div class="csl-right-inline"><strong>Interlaboratory reproducibility of large-scale human protein-complex analysis by standardized AP-MS</strong> <div class="csl-block">Markku Varjosalo, Roberto Sacco, Alexey Stukalov, Audrey van Drogen, Melanie Planyavsky, Simon Hauri, Ruedi Aebersold, Keiryn L Bennett, Jacques Colinge, Matthias Gstaiger, Giulio Superti-Furga</div> <em>Nature Methods</em> (2013-03-03) <a href="https://doi.org/f4t647">https://doi.org/f4t647</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.2400">10.1038/nmeth.2400</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23455922">23455922</a></div></div>
</div>
<div id="ref-yeZfXPV6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">94. </div><div class="csl-right-inline"><strong>The BioPlex Network: A Systematic Exploration of the Human Interactome</strong> <div class="csl-block">Edward L Huttlin, Lily Ting, Raphael J Bruckner, Fana Gebreab, Melanie P Gygi, John Szpyt, Stanley Tam, Gabriela Zarraga, Greg Colby, Kurt Baltier, … Steven P Gygi</div> <em>Cell</em> (2015-07) <a href="https://doi.org/f7kc5h">https://doi.org/f7kc5h</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2015.06.043">10.1016/j.cell.2015.06.043</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26186194">26186194</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4617211">PMC4617211</a></div></div>
</div>
<div id="ref-m46yvHfn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">95. </div><div class="csl-right-inline"><strong>Architecture of the human interactome defines protein communities and disease networks</strong> <div class="csl-block">Edward L Huttlin, Raphael J Bruckner, Joao A Paulo, Joe R Cannon, Lily Ting, Kurt Baltier, Greg Colby, Fana Gebreab, Melanie P Gygi, Hannah Parzen, … JWade Harper</div> <em>Nature</em> (2017-05) <a href="https://doi.org/f96974">https://doi.org/f96974</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nature22366">10.1038/nature22366</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28514442">28514442</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5531611">PMC5531611</a></div></div>
</div>
<div id="ref-VMc1jEkd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">96. </div><div class="csl-right-inline"><strong>Dual proteome-scale networks reveal cell-specific remodeling of the human interactome</strong> <div class="csl-block">Edward L Huttlin, Raphael J Bruckner, Jose Navarrete-Perea, Joe R Cannon, Kurt Baltier, Fana Gebreab, Melanie P Gygi, Alexandra Thornock, Gabriela Zarraga, Stanley Tam, … Steven P Gygi</div> <em>Cell</em> (2021-05) <a href="https://doi.org/gkcn2g">https://doi.org/gkcn2g</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2021.04.011">10.1016/j.cell.2021.04.011</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33961781">33961781</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8165030">PMC8165030</a></div></div>
</div>
<div id="ref-LkwAXvyl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">97. </div><div class="csl-right-inline"><strong>Engineered ascorbate peroxidase as a genetically encoded reporter for electron microscopy</strong> <div class="csl-block">Jeffrey D Martell, Thomas J Deerinck, Yasemin Sancak, Thomas L Poulos, Vamsi K Mootha, Gina E Sosinsky, Mark H Ellisman, Alice Y Ting</div> <em>Nature Biotechnology</em> (2012-10-21) <a href="https://doi.org/f4czsn">https://doi.org/f4czsn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nbt.2375">10.1038/nbt.2375</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23086203">23086203</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3699407">PMC3699407</a></div></div>
</div>
<div id="ref-1BbkEbfDz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">98. </div><div class="csl-right-inline"><strong>Proteomic Mapping of Mitochondria in Living Cells via Spatially Restricted Enzymatic Tagging</strong> <div class="csl-block">Hyun-Woo Rhee, Peng Zou, Namrata D Udeshi, Jeffrey D Martell, Vamsi K Mootha, Steven A Carr, Alice Y Ting</div> <em>Science</em> (2013-03-15) <a href="https://doi.org/f4qdfc">https://doi.org/f4qdfc</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.1230593">10.1126/science.1230593</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23371551">23371551</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3916822">PMC3916822</a></div></div>
</div>
<div id="ref-D5NLgmhd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">99. </div><div class="csl-right-inline"><strong>Proteomic Mapping of the Human Mitochondrial Intermembrane Space in Live Cells via Ratiometric APEX Tagging</strong> <div class="csl-block">Victoria Hung, Peng Zou, Hyun-Woo Rhee, Namrata D Udeshi, Valentin Cracan, Tanya Svinkina, Steven A Carr, Vamsi K Mootha, Alice Y Ting</div> <em>Molecular Cell</em> (2014-07) <a href="https://doi.org/f6d4pk">https://doi.org/f6d4pk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.molcel.2014.06.003">10.1016/j.molcel.2014.06.003</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25002142">25002142</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4743503">PMC4743503</a></div></div>
</div>
<div id="ref-zSgLeSta" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">100. </div><div class="csl-right-inline"><strong>A promiscuous biotin ligase fusion protein identifies proximal and interacting proteins in mammalian cells</strong> <div class="csl-block">Kyle J Roux, Dae In Kim, Manfred Raida, Brian Burke</div> <em>Journal of Cell Biology</em> (2012-03-12) <a href="https://doi.org/f3wp8h">https://doi.org/f3wp8h</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1083/jcb.201112098">10.1083/jcb.201112098</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22412018">22412018</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3308701">PMC3308701</a></div></div>
</div>
<div id="ref-NFFhvScy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">101. </div><div class="csl-right-inline"><strong>Proximity Dependent Biotinylation: Key Enzymes and Adaptation to Proteomics Approaches</strong> <div class="csl-block">Payman Samavarchi-Tehrani, Reuben Samson, Anne-Claude Gingras</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2020-05) <a href="https://doi.org/gr3r3r">https://doi.org/gr3r3r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.r120.001941">10.1074/mcp.r120.001941</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32127388">32127388</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7196579">PMC7196579</a></div></div>
</div>
<div id="ref-1FVEjFnHE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">102. </div><div class="csl-right-inline"><strong>Meet the neighbors: Mapping local protein interactomes by proximity‐dependent labeling with BioID</strong> <div class="csl-block">Renata Varnaitė, Stuart A MacNeill</div> <em>PROTEOMICS</em> (2016-07-27) <a href="https://doi.org/f3qr9p">https://doi.org/f3qr9p</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.201600123">10.1002/pmic.201600123</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27329485">27329485</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5053326">PMC5053326</a></div></div>
</div>
<div id="ref-jJJVaczi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">103. </div><div class="csl-right-inline"><strong>An improved smaller biotin ligase for BioID proximity labeling</strong> <div class="csl-block">Dae In Kim, Samuel C Jensen, Kyle A Noble, Birendra KC, Kenneth H Roux, Khatereh Motamedchaboki, Kyle J Roux</div> <em>Molecular Biology of the Cell</em> (2016-04-15) <a href="https://doi.org/f8ndt9">https://doi.org/f8ndt9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1091/mbc.e15-12-0844">10.1091/mbc.e15-12-0844</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26912792">26912792</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4831873">PMC4831873</a></div></div>
</div>
<div id="ref-sas7Wv3H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">104. </div><div class="csl-right-inline"><strong>BioID: A Proximity-Dependent Labeling Approach in Proteomics Study</strong> <div class="csl-block">Peipei Li, Yuan Meng, Li Wang, Li-jun Di</div> <em>Functional Proteomics</em> (2018-10-02) <a href="https://doi.org/gr3m6f">https://doi.org/gr3m6f</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-4939-8814-3_10">10.1007/978-1-4939-8814-3_10</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30276738">30276738</a></div></div>
</div>
<div id="ref-emz5eXnO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">105. </div><div class="csl-right-inline"><strong>Guide to protein purification</strong> <div class="csl-block">Richard R Burgess, Murray P Deutscher</div> <em>Elsevier/Academic Press</em> (2009) <div class="csl-block">ISBN: 9780123745361</div></div>
</div>
<div id="ref-LvSVXQeu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">106. </div><div class="csl-right-inline"><strong>The protein protocols handbook</strong> <div class="csl-block">John M Walker (editor)</div> <em>Humana Press</em> (2009) <div class="csl-block">ISBN: 9781597451987</div></div>
</div>
<div id="ref-YFivV9n5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">107. </div><div class="csl-right-inline"><strong>Strategies and considerations for protein purifications.</strong> <div class="csl-block">Stuart Linn</div> <em>Methods in enzymology</em> (2009) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19892162">https://www.ncbi.nlm.nih.gov/pubmed/19892162</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0076-6879(09)63002-0">10.1016/s0076-6879(09)63002-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19892162">19892162</a></div></div>
</div>
<div id="ref-OZvOVLht" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">108. </div><div class="csl-right-inline"><strong>Effective interactions between chaotropic agents and proteins.</strong> <div class="csl-block">Giovanni Salvi, Paolo De Los Rios, Michele Vendruscolo</div> <em>Proteins</em> (2005-11-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16152629">https://www.ncbi.nlm.nih.gov/pubmed/16152629</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/prot.20626">10.1002/prot.20626</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16152629">16152629</a></div></div>
</div>
<div id="ref-bpTXhTR5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">109. </div><div class="csl-right-inline"><strong>A quantitative study of the effects of chaotropic agents, surfactants, and solvents on the digestion efficiency of human plasma proteins by trypsin.</strong> <div class="csl-block">Jennifer L Proc, Michael A Kuzyk, Darryl B Hardie, Juncong Yang, Derek S Smith, Angela M Jackson, Carol E Parker, Christoph H Borchers</div> <em>Journal of proteome research</em> (2010-10-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20722421">https://www.ncbi.nlm.nih.gov/pubmed/20722421</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr100656u">10.1021/pr100656u</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20722421">20722421</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2996461">PMC2996461</a></div></div>
</div>
<div id="ref-Igj2yRI0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">110. </div><div class="csl-right-inline"><strong>Best practices and benchmarks for intact protein analysis for top-down mass spectrometry.</strong> <div class="csl-block">Daniel P Donnelly, Catherine M Rawlins, Caroline J DeHart, Luca Fornelli, Luis F Schachner, Ziqing Lin, Jennifer L Lippens, Krishna C Aluri, Richa Sarin, Bifan Chen, … Jeffrey N Agar</div> <em>Nature methods</em> (2019-06-27) <a href="https://www.ncbi.nlm.nih.gov/pubmed/31249407">https://www.ncbi.nlm.nih.gov/pubmed/31249407</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-019-0457-0">10.1038/s41592-019-0457-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31249407">31249407</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6719561">PMC6719561</a></div></div>
</div>
<div id="ref-B8hLhFif" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">111. </div><div class="csl-right-inline"><strong>Inhibition of protein carbamylation in urea solution using ammonium-containing buffers.</strong> <div class="csl-block">Shisheng Sun, Jian-Ying Zhou, Weiming Yang, Hui Zhang</div> <em>Analytical biochemistry</em> (2013-10-23) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24161613">https://www.ncbi.nlm.nih.gov/pubmed/24161613</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ab.2013.10.024">10.1016/j.ab.2013.10.024</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24161613">24161613</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4072244">PMC4072244</a></div></div>
</div>
<div id="ref-G9HWYdpG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">112. </div><div class="csl-right-inline"><strong>Integrated analysis of gene expression, CpG island methylation, and gene copy number in breast cancer cells by deep sequencing.</strong> <div class="csl-block">Zhifu Sun, Yan W Asmann, Krishna R Kalari, Brian Bot, Jeanette E Eckel-Passow, Tiffany R Baker, Jennifer M Carr, Irina Khrebtukova, Shujun Luo, Lu Zhang, … EAubrey Thompson</div> <em>PloS one</em> (2011-02-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21364760">https://www.ncbi.nlm.nih.gov/pubmed/21364760</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pone.0017490">10.1371/journal.pone.0017490</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21364760">21364760</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3045451">PMC3045451</a></div></div>
</div>
<div id="ref-15gFWnWy1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">113. </div><div class="csl-right-inline"><strong>A generic protein purification method for protein complex characterization and proteome exploration.</strong> <div class="csl-block">G Rigaut, A Shevchenko, B Rutz, M Wilm, M Mann, B Séraphin</div> <em>Nature biotechnology</em> (1999-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10504710">https://www.ncbi.nlm.nih.gov/pubmed/10504710</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/13732">10.1038/13732</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10504710">10504710</a></div></div>
</div>
<div id="ref-1FdkzkZPe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">114. </div><div class="csl-right-inline"><strong>Photocleavable Surfactant-Enabled Extracellular Matrix Proteomics.</strong> <div class="csl-block">Samantha J Knott, Kyle A Brown, Harini Josyer, Austin Carr, David Inman, Song Jin, Andreas Friedl, Suzanne M Ponik, Ying Ge</div> <em>Analytical chemistry</em> (2020-11-24) <a href="https://www.ncbi.nlm.nih.gov/pubmed/33232116">https://www.ncbi.nlm.nih.gov/pubmed/33232116</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.0c03104">10.1021/acs.analchem.0c03104</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33232116">33232116</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7961849">PMC7961849</a></div></div>
</div>
<div id="ref-KbqdsNwY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">115. </div><div class="csl-right-inline"><strong>Investigation of solubilization and digestion methods for microsomal membrane proteome analysis using data-independent LC-MSE.</strong> <div class="csl-block">Flaubert Mbeunkui, Michael B Goshe</div> <em>Proteomics</em> (2011-01-31) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21280217">https://www.ncbi.nlm.nih.gov/pubmed/21280217</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200900698">10.1002/pmic.200900698</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21280217">21280217</a></div></div>
</div>
<div id="ref-1FX8gYsTy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">116. </div><div class="csl-right-inline"><strong>Sodium laurate, a novel protease- and mass spectrometry-compatible detergent for mass spectrometry-based membrane proteomics.</strong> <div class="csl-block">Yong Lin, Linju Huo, Zhonghua Liu, Jianglin Li, Yi Liu, Quanze He, Xianchun Wang, Songping Liang</div> <em>PloS one</em> (2013-03-28) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23555778">https://www.ncbi.nlm.nih.gov/pubmed/23555778</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pone.0059779">10.1371/journal.pone.0059779</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23555778">23555778</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3610932">PMC3610932</a></div></div>
</div>
<div id="ref-7nXVfPN7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">117. </div><div class="csl-right-inline"><strong>Evaluation of the application of sodium deoxycholate to proteomic analysis of rat hippocampal plasma membrane.</strong> <div class="csl-block">Jian Zhou, Tieyang Zhou, Rui Cao, Zhen Liu, Jianying Shen, Ping Chen, Xianchun Wang, Songping Liang</div> <em>Journal of proteome research</em> (2006-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17022626">https://www.ncbi.nlm.nih.gov/pubmed/17022626</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr060112a">10.1021/pr060112a</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17022626">17022626</a></div></div>
</div>
<div id="ref-EZCh7C0x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">118. </div><div class="csl-right-inline"><strong>Identification of Poly(ethylene glycol) and Poly(ethylene glycol)-Based Detergents Using Peptide Search Engines.</strong> <div class="csl-block">Shiva Ahmadi, Dominic Winter</div> <em>Analytical chemistry</em> (2018-05-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29726681">https://www.ncbi.nlm.nih.gov/pubmed/29726681</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.8b00365">10.1021/acs.analchem.8b00365</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29726681">29726681</a></div></div>
</div>
<div id="ref-EK9zCZWn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">119. </div><div class="csl-right-inline"><strong>Quantitative assessment of in-solution digestion efficiency identifies optimal protocols for unbiased protein analysis.</strong> <div class="csl-block">Ileana R León, Veit Schwämmle, Ole N Jensen, Richard R Sprenger</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2013-06-21) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23792921">https://www.ncbi.nlm.nih.gov/pubmed/23792921</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m112.025585">10.1074/mcp.m112.025585</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23792921">23792921</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3790306">PMC3790306</a></div></div>
</div>
<div id="ref-Cj5ZGyu8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">120. </div><div class="csl-right-inline"><strong>Proteomic challenges: sample preparation techniques for microgram-quantity protein analysis from biological samples.</strong> <div class="csl-block">Peter Feist, Amanda B Hummon</div> <em>International journal of molecular sciences</em> (2015-02-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25664860">https://www.ncbi.nlm.nih.gov/pubmed/25664860</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/ijms16023537">10.3390/ijms16023537</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25664860">25664860</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4346912">PMC4346912</a></div></div>
</div>
<div id="ref-17GoHwva3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">121. </div><div class="csl-right-inline"><strong>An overview of cell disruption methods for intracellular biomolecules recovery</strong> <div class="csl-block">Tatiane Aparecida Gomes, Cristina Maria Zanette, Michele Rigon Spier</div> <em>Preparative Biochemistry &amp;amp; Biotechnology</em> (2020-02-19) <a href="https://doi.org/gqpdr5">https://doi.org/gqpdr5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1080/10826068.2020.1728696">10.1080/10826068.2020.1728696</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32074000">32074000</a></div></div>
</div>
<div id="ref-195QiKJpv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">122. </div><div class="csl-right-inline"><strong>Mechanical forces used for cell fractionation can create hybrid membrane vesicles.</strong> <div class="csl-block">Izhar Salomon, Hans Janssen, Jacques Neefjes</div> <em>International journal of biological sciences</em> (2010-10-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21060726">https://www.ncbi.nlm.nih.gov/pubmed/21060726</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7150/ijbs.6.649">10.7150/ijbs.6.649</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21060726">21060726</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2974167">PMC2974167</a></div></div>
</div>
<div id="ref-4EGQHHdM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">123. </div><div class="csl-right-inline"><strong>Lyophilization and homogenization of biological samples improves reproducibility and reduces standard deviation in molecular biology techniques.</strong> <div class="csl-block">Agnes Molnar, Tamas Lakat, Adam Hosszu, Beata Szebeni, Anna Balogh, Laszlo Orfi, Attila J Szabo, Andrea Fekete, Judit Hodrea</div> <em>Amino acids</em> (2021-05-17) <a href="https://www.ncbi.nlm.nih.gov/pubmed/34002278">https://www.ncbi.nlm.nih.gov/pubmed/34002278</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00726-021-02994-w">10.1007/s00726-021-02994-w</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34002278">34002278</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8128086">PMC8128086</a></div></div>
</div>
<div id="ref-wXcRYG3x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">124. </div><div class="csl-right-inline"><strong>Assays for determination of protein concentration.</strong> <div class="csl-block">Bradley JSC Olson, John Markwell</div> <em>Current protocols in protein science</em> (2007-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18429326">https://www.ncbi.nlm.nih.gov/pubmed/18429326</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/0471140864.ps0304s48">10.1002/0471140864.ps0304s48</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18429326">18429326</a></div></div>
</div>
<div id="ref-DRVtai1B" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">125. </div><div class="csl-right-inline"><strong>Development and characterization of the NanoOrange protein quantitation assay: a fluorescence-based assay of proteins in solution.</strong> <div class="csl-block">Laurie J Jones, Richard P Haugland, Victoria L Singer</div> <em>BioTechniques</em> (2003-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/12703310">https://www.ncbi.nlm.nih.gov/pubmed/12703310</a> <div class="csl-block">DOI: <a href="https://doi.org/10.2144/03344pt03">10.2144/03344pt03</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12703310">12703310</a></div></div>
</div>
<div id="ref-7YHVbaDH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">126. </div><div class="csl-right-inline"><strong>Fast and Sensitive Total Protein and Peptide Assays for Proteomic Analysis</strong> <div class="csl-block">Jacek R Wiśniewski, Fabienne Z Gaugaz</div> <em>Analytical Chemistry</em> (2015-04-09) <a href="https://doi.org/f3nsk2">https://doi.org/f3nsk2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac504689z">10.1021/ac504689z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25837572">25837572</a></div></div>
</div>
<div id="ref-qFKCAO2X" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">127. </div><div class="csl-right-inline"><strong>Evaluation of colorimetric assays for analyzing reductively methylated proteins: Biases and mechanistic insights.</strong> <div class="csl-block">Pamlea N Brady, Megan A Macnaughtan</div> <em>Analytical biochemistry</em> (2015-09-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26342307">https://www.ncbi.nlm.nih.gov/pubmed/26342307</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ab.2015.08.027">10.1016/j.ab.2015.08.027</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26342307">26342307</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4631703">PMC4631703</a></div></div>
</div>
<div id="ref-jQLWiiId" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">128. </div><div class="csl-right-inline"><strong>Quantification of Intrinsically Disordered Proteins: A Problem Not Fully Appreciated.</strong> <div class="csl-block">Sara Contreras-Martos, Hung H Nguyen, Phuong N Nguyen, Nevena Hristozova, Mauricio Macossay-Castillo, Denes Kovacs, Angela Bekesi, Jesper S Oemig, Dominique Maes, Kris Pauwels, … Pierre Lebrun</div> <em>Frontiers in molecular biosciences</em> (2018-09-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30234128">https://www.ncbi.nlm.nih.gov/pubmed/30234128</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3389/fmolb.2018.00083">10.3389/fmolb.2018.00083</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30234128">30234128</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6131523">PMC6131523</a></div></div>
</div>
<div id="ref-OglKN2v5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">129. </div><div class="csl-right-inline"><strong>Evaluation and optimization of reduction and alkylation methods to maximize peptide identification with MS-based proteomics.</strong> <div class="csl-block">Suttipong Suttapitugsakul, Haopeng Xiao, Johanna Smeekens, Ronghu Wu</div> <em>Molecular bioSystems</em> (2017-11-21) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29019370">https://www.ncbi.nlm.nih.gov/pubmed/29019370</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/c7mb00393e">10.1039/c7mb00393e</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29019370">29019370</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5698164">PMC5698164</a></div></div>
</div>
<div id="ref-2cCnZ6ft" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">130. </div><div class="csl-right-inline"><strong>A simplified procedure for the reduction and alkylation of cysteine residues in proteins prior to proteolytic digestion and mass spectral analysis.</strong> <div class="csl-block">John E Hale, Jon P Butler, Valentina Gelfanova, Jin-Sam You, Michael D Knierman</div> <em>Analytical biochemistry</em> (2004-10-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/15351294">https://www.ncbi.nlm.nih.gov/pubmed/15351294</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ab.2004.04.013">10.1016/j.ab.2004.04.013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15351294">15351294</a></div></div>
</div>
<div id="ref-9vICnnYM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">131. </div><div class="csl-right-inline"><strong>Systematic Evaluation of Protein Reduction and Alkylation Reveals Massive Unspecific Side Effects by Iodine-containing Reagents.</strong> <div class="csl-block">Torsten Müller, Dominic Winter</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2017-05-24) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28539326">https://www.ncbi.nlm.nih.gov/pubmed/28539326</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m116.064048">10.1074/mcp.m116.064048</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28539326">28539326</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5500753">PMC5500753</a></div></div>
</div>
<div id="ref-GkFAexwC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">132. </div><div class="csl-right-inline"><strong>Discovery of a Redox Thiol Switch: Implications for Cellular Energy Metabolism.</strong> <div class="csl-block">Xing-Huang Gao, Ling Li, Marc Parisien, Jing Wu, Ilya Bederman, Zhaofeng Gao, Dawid Krokowski, Steven M Chirieleison, Derek Abbott, Benlian Wang, … Maria Hatzoglou</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2020-03-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/32132231">https://www.ncbi.nlm.nih.gov/pubmed/32132231</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.ra119.001910">10.1074/mcp.ra119.001910</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32132231">32132231</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7196587">PMC7196587</a></div></div>
</div>
<div id="ref-Z6nk0TFI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">133. </div><div class="csl-right-inline"><strong>Residue Modification and Mass Spectrometry for the Investigation of Structural and Metalation Properties of Metallothionein and Cysteine-Rich Proteins.</strong> <div class="csl-block">Gordon W Irvine, Martin J Stillman</div> <em>International journal of molecular sciences</em> (2017-04-26) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28445428">https://www.ncbi.nlm.nih.gov/pubmed/28445428</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/ijms18050913">10.3390/ijms18050913</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28445428">28445428</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5454826">PMC5454826</a></div></div>
</div>
<div id="ref-s4OLOlUG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">134. </div><div class="csl-right-inline"><strong>Chasing cysteine oxidative modifications: proteomic tools for characterizing cysteine redox status.</strong> <div class="csl-block">Christopher I Murray, Jennifer E Van Eyk</div> <em>Circulation. Cardiovascular genetics</em> (2012-10-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23074338">https://www.ncbi.nlm.nih.gov/pubmed/23074338</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1161/circgenetics.111.961425">10.1161/circgenetics.111.961425</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23074338">23074338</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3500588">PMC3500588</a></div></div>
</div>
<div id="ref-xvenBhjQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">135. </div><div class="csl-right-inline"><strong>Comparison of protein precipitation methods for sample preparation prior to proteomic analysis.</strong> <div class="csl-block">Lei Jiang, Lin He, Michael Fountoulakis</div> <em>Journal of chromatography. A</em> (2004-01-16) <a href="https://www.ncbi.nlm.nih.gov/pubmed/14753699">https://www.ncbi.nlm.nih.gov/pubmed/14753699</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.chroma.2003.10.029">10.1016/j.chroma.2003.10.029</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14753699">14753699</a></div></div>
</div>
<div id="ref-OMzPHtgd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">136. </div><div class="csl-right-inline"><strong>Protein precipitation techniques.</strong> <div class="csl-block">Richard R Burgess</div> <em>Methods in enzymology</em> (2009) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19892180">https://www.ncbi.nlm.nih.gov/pubmed/19892180</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0076-6879(09)63020-2">10.1016/s0076-6879(09)63020-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19892180">19892180</a></div></div>
</div>
<div id="ref-Llj72YHx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">137. </div><div class="csl-right-inline"><strong>Facile Preparation of Peptides for Mass Spectrometry Analysis in Bottom-Up Proteomics Workflows.</strong> <div class="csl-block">Melinda Wojtkiewicz, Linda Berg Luecke, Maia I Kelly, Rebekah L Gundry</div> <em>Current protocols</em> (2021-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/33750040">https://www.ncbi.nlm.nih.gov/pubmed/33750040</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/cpz1.85">10.1002/cpz1.85</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33750040">33750040</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8011989">PMC8011989</a></div></div>
</div>
<div id="ref-rWTbDNu2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">138. </div><div class="csl-right-inline"><strong>Universal sample preparation method for proteome analysis.</strong> <div class="csl-block">Jacek R Wiśniewski, Alexandre Zougman, Nagarjuna Nagaraj, Matthias Mann</div> <em>Nature methods</em> (2009-04-19) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19377485">https://www.ncbi.nlm.nih.gov/pubmed/19377485</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.1322">10.1038/nmeth.1322</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19377485">19377485</a></div></div>
</div>
<div id="ref-bkON7ZR2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">139. </div><div class="csl-right-inline"><strong>Filter-Aided Sample Preparation for Proteome Analysis.</strong> <div class="csl-block">Jacek R Wiśniewski</div> <em>Methods in molecular biology (Clifton, N.J.)</em> (2018) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30259475">https://www.ncbi.nlm.nih.gov/pubmed/30259475</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-4939-8695-8_1">10.1007/978-1-4939-8695-8_1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30259475">30259475</a></div></div>
</div>
<div id="ref-29YpOcx3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">140. </div><div class="csl-right-inline"><strong>[Group and type distribution of hemolytic streptococci isolated from clinical specimens--prevalence of group A type 3 isolates in 1985 in Toyama Prefecture].</strong> <div class="csl-block">H Kodama, N Tokuman, I Yasui, Y Gyobu, Y Kashiwagi</div> <em>Kansenshogaku zasshi. The Journal of the Japanese Association for Infectious Diseases</em> (1987-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/3117935">https://www.ncbi.nlm.nih.gov/pubmed/3117935</a> <div class="csl-block">DOI: <a href="https://doi.org/10.11150/kansenshogakuzasshi1970.61.482">10.11150/kansenshogakuzasshi1970.61.482</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3117935">3117935</a></div></div>
</div>
<div id="ref-R9HFU5Rw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">141. </div><div class="csl-right-inline"><strong>Evaluation of FASP, SP3, and iST Protocols for Proteomic Sample Preparation in the Low Microgram Range.</strong> <div class="csl-block">Malte Sielaff, Jörg Kuharev, Toszka Bohn, Jennifer Hahlbrock, Tobias Bopp, Stefan Tenzer, Ute Distler</div> <em>Journal of proteome research</em> (2017-10-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28948796">https://www.ncbi.nlm.nih.gov/pubmed/28948796</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.7b00433">10.1021/acs.jproteome.7b00433</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28948796">28948796</a></div></div>
</div>
<div id="ref-IRVtdB7A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">142. </div><div class="csl-right-inline"><strong>Comparison of In-Solution, FASP, and S-Trap Based Digestion Methods for Bottom-Up Proteomic Studies.</strong> <div class="csl-block">Katelyn R Ludwig, Monica M Schroll, Amanda B Hummon</div> <em>Journal of proteome research</em> (2018-05-24) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29754492">https://www.ncbi.nlm.nih.gov/pubmed/29754492</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.8b00235">10.1021/acs.jproteome.8b00235</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29754492">29754492</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9319029">PMC9319029</a></div></div>
</div>
<div id="ref-GcubNF0j" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">143. </div><div class="csl-right-inline"><strong>Phase transfer surfactant-aided trypsin digestion for membrane proteome analysis.</strong> <div class="csl-block">Takeshi Masuda, Masaru Tomita, Yasushi Ishihama</div> <em>Journal of proteome research</em> (2008-02) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18183947">https://www.ncbi.nlm.nih.gov/pubmed/18183947</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr700658q">10.1021/pr700658q</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18183947">18183947</a></div></div>
</div>
<div id="ref-1F0WRIEv4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">144. </div><div class="csl-right-inline"><strong>Colorimetric protein assay techniques.</strong> <div class="csl-block">CV Sapan, RL Lundblad, NC Price</div> <em>Biotechnology and applied biochemistry</em> (1999-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10075906">https://www.ncbi.nlm.nih.gov/pubmed/10075906</a> <div class="csl-block">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10075906">10075906</a></div></div>
</div>
<div id="ref-MgeRCWZg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">145. </div><div class="csl-right-inline"><strong>Measurement of protein using bicinchoninic acid.</strong> <div class="csl-block">PK Smith, RI Krohn, GT Hermanson, AK Mallia, FH Gartner, MD Provenzano, EK Fujimoto, NM Goeke, BJ Olson, DC Klenk</div> <em>Analytical biochemistry</em> (1985-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/3843705">https://www.ncbi.nlm.nih.gov/pubmed/3843705</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/0003-2697(85)90442-7">10.1016/0003-2697(85)90442-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3843705">3843705</a></div></div>
</div>
<div id="ref-7aeNLPmC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">146. </div><div class="csl-right-inline"><strong>The bicinchoninic acid (BCA) assay for protein quantitation.</strong> <div class="csl-block">JM Walker</div> <em>Methods in molecular biology (Clifton, N.J.)</em> (1994) <a href="https://www.ncbi.nlm.nih.gov/pubmed/7951748">https://www.ncbi.nlm.nih.gov/pubmed/7951748</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1385/0-89603-268-x:5">10.1385/0-89603-268-x:5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/7951748">7951748</a></div></div>
</div>
<div id="ref-10DIgRvDQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">147. </div><div class="csl-right-inline"><strong>A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding.</strong> <div class="csl-block">MM Bradford</div> <em>Analytical biochemistry</em> (1976-05-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/942051">https://www.ncbi.nlm.nih.gov/pubmed/942051</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1006/abio.1976.9999">10.1006/abio.1976.9999</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/942051">942051</a></div></div>
</div>
<div id="ref-L2cR80nH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">148. </div><div class="csl-right-inline"><strong>Bradford Assay for Determining Protein Concentration.</strong> <div class="csl-block">Clara L Kielkopf, William Bauer, Ina L Urbatsch</div> <em>Cold Spring Harbor protocols</em> (2020-04-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/32238597">https://www.ncbi.nlm.nih.gov/pubmed/32238597</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/pdb.prot102269">10.1101/pdb.prot102269</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32238597">32238597</a></div></div>
</div>
<div id="ref-2lZxrpnu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">149. </div><div class="csl-right-inline"><strong>An assay for the measurement of the protein content of cells immobilized in carrageenan.</strong> <div class="csl-block">A Jones, T Razniewska, BH Lesser, R Siqueira, D Berk, LA Behie, GM Gaucher</div> <em>Canadian journal of microbiology</em> (1984-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/6744121">https://www.ncbi.nlm.nih.gov/pubmed/6744121</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1139/m84-069">10.1139/m84-069</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/6744121">6744121</a></div></div>
</div>
<div id="ref-q1g1rJrT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">150. </div><div class="csl-right-inline"><strong>UV-Vis Spectra-Activated Droplet Sorting for Label-Free Chemical Identification and Collection of Droplets.</strong> <div class="csl-block">Todd A Duncombe, Aaron Ponti, Florian P Seebeck, Petra S Dittrich</div> <em>Analytical chemistry</em> (2021-09-17) <a href="https://www.ncbi.nlm.nih.gov/pubmed/34533299">https://www.ncbi.nlm.nih.gov/pubmed/34533299</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.1c02822">10.1021/acs.analchem.1c02822</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34533299">34533299</a></div></div>
</div>
<div id="ref-NLQoCcYj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">151. </div><div class="csl-right-inline"><strong>Fast and sensitive total protein and Peptide assays for proteomic analysis.</strong> <div class="csl-block">Jacek R Wiśniewski, Fabienne Z Gaugaz</div> <em>Analytical chemistry</em> (2015-04-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25837572">https://www.ncbi.nlm.nih.gov/pubmed/25837572</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac504689z">10.1021/ac504689z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25837572">25837572</a></div></div>
</div>
<div id="ref-iRPPTjzF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">152. </div><div class="csl-right-inline"><strong>3-(4-Carboxybenzoyl)quinoline-2-carboxaldehyde, a reagent with broad dynamic range for the assay of proteins and lipoproteins in solution.</strong> <div class="csl-block">WW You, RP Haugland, DK Ryan, RP Haugland</div> <em>Analytical biochemistry</em> (1997-01-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/9025944">https://www.ncbi.nlm.nih.gov/pubmed/9025944</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1006/abio.1996.9920">10.1006/abio.1996.9920</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/9025944">9025944</a></div></div>
</div>
<div id="ref-vpBg0CP7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">153. </div><div class="csl-right-inline"><strong>Getting intimate with trypsin, the leading protease in proteomics</strong> <div class="csl-block">Elien Vandermarliere, Michael Mueller, Lennart Martens</div> <em>Mass Spectrometry Reviews</em> (2013-06-15) <a href="https://doi.org/gn64qb">https://doi.org/gn64qb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.21376">10.1002/mas.21376</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23775586">23775586</a></div></div>
</div>
<div id="ref-16FpKYZCe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">154. </div><div class="csl-right-inline"><strong>Value of using multiple proteases for large-scale mass spectrometry-based proteomics.</strong> <div class="csl-block">Danielle L Swaney, Craig D Wenger, Joshua J Coon</div> <em>Journal of proteome research</em> (2010-03-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20113005">https://www.ncbi.nlm.nih.gov/pubmed/20113005</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr900863u">10.1021/pr900863u</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20113005">20113005</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2833215">PMC2833215</a></div></div>
</div>
<div id="ref-lSFMVWD1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">155. </div><div class="csl-right-inline"><strong>Proteomics beyond trypsin.</strong> <div class="csl-block">Liana Tsiatsiani, Albert JR Heck</div> <em>The FEBS journal</em> (2015-04-14) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25823410">https://www.ncbi.nlm.nih.gov/pubmed/25823410</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/febs.13287">10.1111/febs.13287</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25823410">25823410</a></div></div>
</div>
<div id="ref-10vHwy5gN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">156. </div><div class="csl-right-inline"><div class="csl-block">Jesse G Meyer</div> <em>ISRN computational biology</em> (2014-04-22) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30687733">https://www.ncbi.nlm.nih.gov/pubmed/30687733</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1155/2014/960902">10.1155/2014/960902</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30687733">30687733</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6347401">PMC6347401</a></div></div>
</div>
<div id="ref-y9KaXde4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">157. </div><div class="csl-right-inline"><strong>&lt;i&gt;In Silico&lt;/i&gt;Proteome Cleavage Reveals Iterative Digestion Strategy for High Sequence Coverage</strong> <div class="csl-block">Jesse G Meyer</div> <em>ISRN Computational Biology</em> (2014-04-22) <a href="https://doi.org/gb6s2r">https://doi.org/gb6s2r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1155/2014/960902">10.1155/2014/960902</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30687733">30687733</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6347401">PMC6347401</a></div></div>
</div>
<div id="ref-7aKTZT1i" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">158. </div><div class="csl-right-inline"><strong>Multiple enzymatic digestion for enhanced sequence coverage of proteins in complex proteomic mixtures using capillary LC with ion trap MS/MS.</strong> <div class="csl-block">Gargi Choudhary, Shiaw-Lin Wu, Paul Shieh, William S Hancock</div> <em>Journal of proteome research</em> (2003) <a href="https://www.ncbi.nlm.nih.gov/pubmed/12643544">https://www.ncbi.nlm.nih.gov/pubmed/12643544</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr025557n">10.1021/pr025557n</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12643544">12643544</a></div></div>
</div>
<div id="ref-gXhdeFOF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">159. </div><div class="csl-right-inline"><strong>Six alternative proteases for mass spectrometry-based proteomics beyond trypsin.</strong> <div class="csl-block">Piero Giansanti, Liana Tsiatsiani, Teck Yew Low, Albert JR Heck</div> <em>Nature protocols</em> (2016-04-28) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27123950">https://www.ncbi.nlm.nih.gov/pubmed/27123950</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nprot.2016.057">10.1038/nprot.2016.057</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27123950">27123950</a></div></div>
</div>
<div id="ref-14bc70ELO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">160. </div><div class="csl-right-inline"><strong>Combination of Proteogenomics with Peptide </strong> <div class="csl-block">B Blank-Landeshammer, I Teichert, R Märker, M Nowrousian, U Kück, A Sickmann</div> <em>mBio</em> (2019-10-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/31615963">https://www.ncbi.nlm.nih.gov/pubmed/31615963</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1128/mbio.02367-19">10.1128/mbio.02367-19</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31615963">31615963</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6794485">PMC6794485</a></div></div>
</div>
<div id="ref-zmo81rNN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">161. </div><div class="csl-right-inline"><strong>Precision </strong> <div class="csl-block">Hao Yang, Yan-Chang Li, Ming-Zhi Zhao, Fei-Lin Wu, Xi Wang, Wei-Di Xiao, Yi-Hao Wang, Jun-Ling Zhang, Fu-Qiang Wang, Feng Xu, … Ping Xu</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2019-01-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30622160">https://www.ncbi.nlm.nih.gov/pubmed/30622160</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.tir118.000918">10.1074/mcp.tir118.000918</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30622160">30622160</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6442358">PMC6442358</a></div></div>
</div>
<div id="ref-IZw9sVjg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">162. </div><div class="csl-right-inline"><strong>Mass spectrometry-assisted venom profiling of Hypnale hypnale found in the Western Ghats of India incorporating de novo sequencing approaches.</strong> <div class="csl-block">Muralidharan Vanuopadath, Nithin Sajeev, Athira Radhamony Murali, Nayana Sudish, Nithya Kangosseri, Ivy Rose Sebastian, Nidhi Dalpatraj Jain, Amit Pal, Dileepkumar Raveendran, Bipin Gopalakrishnan Nair, Sudarslal Sadasivan Nair</div> <em>International journal of biological macromolecules</em> (2018-07-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29990557">https://www.ncbi.nlm.nih.gov/pubmed/29990557</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ijbiomac.2018.07.016">10.1016/j.ijbiomac.2018.07.016</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29990557">29990557</a></div></div>
</div>
<div id="ref-19KCAADV7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">163. </div><div class="csl-right-inline"><strong>Venomics and antivenomics of Indian spectacled cobra (Naja naja) from the Western Ghats</strong> <div class="csl-block">Muralidharan Vanuopadath, Dileepkumar Raveendran, Bipin Gopalakrishnan Nair, Sudarslal Sadasivan Nair</div> <em>Acta Tropica</em> (2022-04) <a href="https://doi.org/gpbzf7">https://doi.org/gpbzf7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.actatropica.2022.106324">10.1016/j.actatropica.2022.106324</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35093326">35093326</a></div></div>
</div>
<div id="ref-7BuX2TBR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">164. </div><div class="csl-right-inline"><strong>Sequencing-Grade &lt;i&gt;De novo&lt;/i&gt; Analysis of MS/MS Triplets (CID/HCD/ETD) From Overlapping Peptides</strong> <div class="csl-block">Adrian Guthals, Karl R Clauser, Ari M Frank, Nuno Bandeira</div> <em>Journal of Proteome Research</em> (2013-05-30) <a href="https://doi.org/f47kqd">https://doi.org/f47kqd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr400173d">10.1021/pr400173d</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23679345">23679345</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4591044">PMC4591044</a></div></div>
</div>
<div id="ref-1GLf6FYas" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">165. </div><div class="csl-right-inline"><strong>Multiple-Enzyme-Digestion Strategy Improves Accuracy and Sensitivity of Label- and Standard-Free Absolute Quantification to a Level That Is Achievable by Analysis with Stable Isotope-Labeled Standard Spiking.</strong> <div class="csl-block">Jacek R Wiśniewski, Christine Wegler, Per Artursson</div> <em>Journal of proteome research</em> (2018-10-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30336047">https://www.ncbi.nlm.nih.gov/pubmed/30336047</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.8b00549">10.1021/acs.jproteome.8b00549</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30336047">30336047</a></div></div>
</div>
<div id="ref-WubQ7yFP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">166. </div><div class="csl-right-inline"><strong>Improved Protein Inference from Multiple Protease Bottom-Up Mass Spectrometry Data</strong> <div class="csl-block">Rachel M Miller, Robert J Millikin, Connor V Hoffmann, Stefan K Solntsev, Gloria M Sheynkman, Michael R Shortreed, Lloyd M Smith</div> <em>Journal of Proteome Research</em> (2019-08-04) <a href="https://doi.org/gpqmp2">https://doi.org/gpqmp2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.9b00330">10.1021/acs.jproteome.9b00330</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31378069">31378069</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6733628">PMC6733628</a></div></div>
</div>
<div id="ref-Ac0S3PrF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">167. </div><div class="csl-right-inline"><strong>Multi-protease Approach for the Improved Identification and Molecular Characterization of Small Proteins and Short Open Reading Frame-Encoded Peptides</strong> <div class="csl-block">Philipp T Kaulich, Liam Cassidy, Jürgen Bartel, Ruth A Schmitz, Andreas Tholey</div> <em>Journal of Proteome Research</em> (2021-03-24) <a href="https://doi.org/gpqmpz">https://doi.org/gpqmpz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.1c00115">10.1021/acs.jproteome.1c00115</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33760615">33760615</a></div></div>
</div>
<div id="ref-P861T4mW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">168. </div><div class="csl-right-inline"><strong>A Multiple Protease Strategy to Optimise the Shotgun Proteomics of Mature Medicinal Cannabis Buds</strong> <div class="csl-block">Delphine Vincent, Vilnis Ezernieks, Simone Rochfort, German Spangenberg</div> <em>International Journal of Molecular Sciences</em> (2019-11-11) <a href="https://doi.org/gpqmp3">https://doi.org/gpqmp3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/ijms20225630">10.3390/ijms20225630</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31717952">31717952</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6888629">PMC6888629</a></div></div>
</div>
<div id="ref-14eCb7Jfc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">169. </div><div class="csl-right-inline"><strong>Confetti: A Multiprotease Map of the HeLa Proteome for Comprehensive Proteomics</strong> <div class="csl-block">Xiaofeng Guo, David C Trudgian, Andrew Lemoff, Sivaramakrishna Yadavalli, Hamid Mirzaei</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2014-06) <a href="https://doi.org/f56bwx">https://doi.org/f56bwx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m113.035170">10.1074/mcp.m113.035170</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24696503">24696503</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4047476">PMC4047476</a></div></div>
</div>
<div id="ref-18s9UXUGf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">170. </div><div class="csl-right-inline"><strong>An Augmented Multiple-Protease-Based Human Phosphopeptide Atlas</strong> <div class="csl-block">Piero Giansanti, Thin Thin Aye, Henk van den Toorn, Mao Peng, Bas van Breukelen, Albert JR Heck</div> <em>Cell Reports</em> (2015-06) <a href="https://doi.org/gpqmpx">https://doi.org/gpqmpx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.celrep.2015.05.029">10.1016/j.celrep.2015.05.029</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26074081">26074081</a></div></div>
</div>
<div id="ref-ZgqlEERS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">171. </div><div class="csl-right-inline"><strong>Use of endoproteinase Lys-C from Lysobacter enzymogenes in protein sequence analysis.</strong> <div class="csl-block">PA Jekel, WJ Weijer, JJ Beintema</div> <em>Analytical biochemistry</em> (1983-10-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/6359954">https://www.ncbi.nlm.nih.gov/pubmed/6359954</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/0003-2697(83)90308-1">10.1016/0003-2697(83)90308-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/6359954">6359954</a></div></div>
</div>
<div id="ref-nobcST81" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">172. </div><div class="csl-right-inline"><strong>Large-scale quantitative assessment of different in-solution protein digestion protocols reveals superior cleavage efficiency of tandem Lys-C/trypsin proteolysis over trypsin digestion.</strong> <div class="csl-block">Timo Glatter, Christina Ludwig, Erik Ahrné, Ruedi Aebersold, Albert JR Heck, Alexander Schmidt</div> <em>Journal of proteome research</em> (2012-10-16) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23017020">https://www.ncbi.nlm.nih.gov/pubmed/23017020</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr300273g">10.1021/pr300273g</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23017020">23017020</a></div></div>
</div>
<div id="ref-oDNy6K9V" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">173. </div><div class="csl-right-inline"><strong>The alpha-lytic protease gene of Lysobacter enzymogenes. The nucleotide sequence predicts a large prepro-peptide with homology to pro-peptides of other chymotrypsin-like enzymes.</strong> <div class="csl-block">DM Epstein, PC Wensink</div> <em>The Journal of biological chemistry</em> (1988-11-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/3053694">https://www.ncbi.nlm.nih.gov/pubmed/3053694</a> <div class="csl-block">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3053694">3053694</a></div></div>
</div>
<div id="ref-UeP5oN6q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">174. </div><div class="csl-right-inline"><strong>Purification and properties of an extracellular protease of Staphylococcus aureus.</strong> <div class="csl-block">GR Drapeau, Y Boily, J Houmard</div> <em>The Journal of biological chemistry</em> (1972-10-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/4627743">https://www.ncbi.nlm.nih.gov/pubmed/4627743</a> <div class="csl-block">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/4627743">4627743</a></div></div>
</div>
<div id="ref-FcovvAwu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">175. </div><div class="csl-right-inline"><strong>Mildly acidic conditions eliminate deamidation artifact during proteolysis: digestion with endoprotease Glu-C at pH 4.5.</strong> <div class="csl-block">Shanshan Liu, Kevin Ryan Moulton, Jared Robert Auclair, Zhaohui Sunny Zhou</div> <em>Amino acids</em> (2016-01-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26748652">https://www.ncbi.nlm.nih.gov/pubmed/26748652</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00726-015-2166-z">10.1007/s00726-015-2166-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26748652">26748652</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4795971">PMC4795971</a></div></div>
</div>
<div id="ref-Z8khRFem" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">176. </div><div class="csl-right-inline"><strong>Specificity of endoproteinase Asp-N (Pseudomonas fragi): cleavage at glutamyl residues in two proteins.</strong> <div class="csl-block">D Ingrosso, AV Fowler, J Bleibaum, S Clarke</div> <em>Biochemical and biophysical research communications</em> (1989-08-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/2669754">https://www.ncbi.nlm.nih.gov/pubmed/2669754</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/0006-291x(89)90848-6">10.1016/0006-291x(89)90848-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2669754">2669754</a></div></div>
</div>
<div id="ref-129jHkJ7J" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">177. </div><div class="csl-right-inline"><strong>Chymotrypsin: molecular and catalytic properties.</strong> <div class="csl-block">W Appel</div> <em>Clinical biochemistry</em> (1986-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/3555886">https://www.ncbi.nlm.nih.gov/pubmed/3555886</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0009-9120(86)80002-9">10.1016/s0009-9120(86)80002-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3555886">3555886</a></div></div>
</div>
<div id="ref-n8tCUNeh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">178. </div><div class="csl-right-inline"><strong>Mass-spectrometry-based draft of the human proteome.</strong> <div class="csl-block">Mathias Wilhelm, Judith Schlegl, Hannes Hahne, Amin Moghaddas Gholami, Marcus Lieberenz, Mikhail M Savitski, Emanuel Ziegler, Lars Butzmann, Siegfried Gessulat, Harald Marx, … Bernhard Kuster</div> <em>Nature</em> (2014-05-29) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24870543">https://www.ncbi.nlm.nih.gov/pubmed/24870543</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nature13319">10.1038/nature13319</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24870543">24870543</a></div></div>
</div>
<div id="ref-Ddlp2JQj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">179. </div><div class="csl-right-inline"><strong>Confetti: a multiprotease map of the HeLa proteome for comprehensive proteomics.</strong> <div class="csl-block">Xiaofeng Guo, David C Trudgian, Andrew Lemoff, Sivaramakrishna Yadavalli, Hamid Mirzaei</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2014-04-02) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24696503">https://www.ncbi.nlm.nih.gov/pubmed/24696503</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m113.035170">10.1074/mcp.m113.035170</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24696503">24696503</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4047476">PMC4047476</a></div></div>
</div>
<div id="ref-VbXzZ784" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">180. </div><div class="csl-right-inline"><strong>Quantitative and qualitative proteome characteristics extracted from in-depth integrated genomics and proteomics analysis.</strong> <div class="csl-block">Teck Yew Low, Sebastiaan van Heesch, Henk van den Toorn, Piero Giansanti, Alba Cristobal, Pim Toonen, Sebastian Schafer, Norbert Hübner, Bas van Breukelen, Shabaz Mohammed, … Victor Guryev</div> <em>Cell reports</em> (2013-11-27) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24290761">https://www.ncbi.nlm.nih.gov/pubmed/24290761</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.celrep.2013.10.041">10.1016/j.celrep.2013.10.041</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24290761">24290761</a></div></div>
</div>
<div id="ref-1BDWCCTHa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">181. </div><div class="csl-right-inline"><strong>Protease bias in absolute protein quantitation.</strong> <div class="csl-block">Mao Peng, Nadia Taouatas, Salvatore Cappadona, Bas van Breukelen, Shabaz Mohammed, Arjen Scholten, Albert JR Heck</div> <em>Nature methods</em> (2012-05-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22669647">https://www.ncbi.nlm.nih.gov/pubmed/22669647</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.2031">10.1038/nmeth.2031</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22669647">22669647</a></div></div>
</div>
<div id="ref-qJADzs4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">182. </div><div class="csl-right-inline"><strong>Studies on the active site of clostripain. The specific inactivation by the chloromethyl ketone derived from -N-tosyl-L-lysine.</strong> <div class="csl-block">WH Porter, LW Cunningham, WM Mitchell</div> <em>The Journal of biological chemistry</em> (1971-12-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/4332560">https://www.ncbi.nlm.nih.gov/pubmed/4332560</a> <div class="csl-block">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/4332560">4332560</a></div></div>
</div>
<div id="ref-MqKkSQNK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">183. </div><div class="csl-right-inline"><strong>LysargiNase mirrors trypsin for protein C-terminal and methylation-site identification.</strong> <div class="csl-block">Pitter F Huesgen, Philipp F Lange, Lindsay D Rogers, Nestor Solis, Ulrich Eckhard, Oded Kleifeld, Theodoros Goulas, FXavier Gomis-Rüth, Christopher M Overall</div> <em>Nature methods</em> (2014-11-24) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25419962">https://www.ncbi.nlm.nih.gov/pubmed/25419962</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.3177">10.1038/nmeth.3177</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25419962">25419962</a></div></div>
</div>
<div id="ref-3xdcLBaR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">184. </div><div class="csl-right-inline"><strong>Proteomic analyses using Grifola frondosa metalloendoprotease Lys-N.</strong> <div class="csl-block">Laura Hohmann, Carly Sherwood, Ashley Eastham, Amelia Peterson, Jimmy K Eng, James S Eddes, David Shteynberg, Daniel B Martin</div> <em>Journal of proteome research</em> (2009-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19195997">https://www.ncbi.nlm.nih.gov/pubmed/19195997</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr800774h">10.1021/pr800774h</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19195997">19195997</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2798736">PMC2798736</a></div></div>
</div>
<div id="ref-O6S1GvPn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">185. </div><div class="csl-right-inline"><strong>Straightforward ladder sequencing of peptides using a Lys-N metalloendopeptidase.</strong> <div class="csl-block">Nadia Taouatas, Madalina M Drugan, Albert JR Heck, Shabaz Mohammed</div> <em>Nature methods</em> (2008-04-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18425140">https://www.ncbi.nlm.nih.gov/pubmed/18425140</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.1204">10.1038/nmeth.1204</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18425140">18425140</a></div></div>
</div>
<div id="ref-8DGqkN3D" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">186. </div><div class="csl-right-inline"><strong>Cleavage specificities of the brother and sister proteases Lys-C and Lys-N.</strong> <div class="csl-block">Reinout Raijmakers, Pieter Neerincx, Shabaz Mohammed, Albert JR Heck</div> <em>Chemical communications (Cambridge, England)</em> (2010-10-18) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20953479">https://www.ncbi.nlm.nih.gov/pubmed/20953479</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/c0cc02523b">10.1039/c0cc02523b</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20953479">20953479</a></div></div>
</div>
<div id="ref-eMC9mqfj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">187. </div><div class="csl-right-inline"><strong>A history of pepsin and related enzymes.</strong> <div class="csl-block">Joseph S Fruton</div> <em>The Quarterly review of biology</em> (2002-06) <a href="https://www.ncbi.nlm.nih.gov/pubmed/12089768">https://www.ncbi.nlm.nih.gov/pubmed/12089768</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1086/340729">10.1086/340729</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12089768">12089768</a></div></div>
</div>
<div id="ref-pnU7apE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">188. </div><div class="csl-right-inline"><strong>CRYSTALLINE PEPSIN : I. ISOLATION AND TESTS OF PURITY.</strong> <div class="csl-block">JH Northrop</div> <em>The Journal of general physiology</em> (1930-07-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19872561">https://www.ncbi.nlm.nih.gov/pubmed/19872561</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1085/jgp.13.6.739">10.1085/jgp.13.6.739</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19872561">19872561</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2141071">PMC2141071</a></div></div>
</div>
<div id="ref-9Yf3fEMC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">189. </div><div class="csl-right-inline"><strong>CRYSTALLINE PEPSIN : II. GENERAL PROPERTIES AND EXPERIMENTAL METHODS.</strong> <div class="csl-block">JH Northrop</div> <em>The Journal of general physiology</em> (1930-07-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19872562">https://www.ncbi.nlm.nih.gov/pubmed/19872562</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1085/jgp.13.6.767">10.1085/jgp.13.6.767</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19872562">19872562</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2141088">PMC2141088</a></div></div>
</div>
<div id="ref-10CHY6r2E" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">190. </div><div class="csl-right-inline"><strong>CRYSTALLINE PEPSIN.</strong> <div class="csl-block">JH Northrop</div> <em>Science (New York, N.Y.)</em> (1929-05-31) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17758437">https://www.ncbi.nlm.nih.gov/pubmed/17758437</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.69.1796.580">10.1126/science.69.1796.580</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17758437">17758437</a></div></div>
</div>
<div id="ref-1EtaxGZW0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">191. </div><div class="csl-right-inline"><strong>The Nobel Prize in Chemistry 1946</strong> <div class="csl-block">NobelPrize.org</div> <a href="https://www.nobelprize.org/prizes/chemistry/1946/speedread/">https://www.nobelprize.org/prizes/chemistry/1946/speedread/</a></div>
</div>
<div id="ref-itW7kK3s" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">192. </div><div class="csl-right-inline"><strong>Protein disulfide bond determination by mass spectrometry.</strong> <div class="csl-block">Jeffrey J Gorman, Tristan P Wallis, James J Pitt</div> <em>Mass spectrometry reviews</em> (2002) <a href="https://www.ncbi.nlm.nih.gov/pubmed/12476442">https://www.ncbi.nlm.nih.gov/pubmed/12476442</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.10025">10.1002/mas.10025</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12476442">12476442</a></div></div>
</div>
<div id="ref-Vske1oHJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">193. </div><div class="csl-right-inline"><strong>Facilitating protein disulfide mapping by a combination of pepsin digestion, electron transfer higher energy dissociation (EThcD), and a dedicated search algorithm SlinkS.</strong> <div class="csl-block">Fan Liu, Bas van Breukelen, Albert JR Heck</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2014-06-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24980484">https://www.ncbi.nlm.nih.gov/pubmed/24980484</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.o114.039057">10.1074/mcp.o114.039057</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24980484">24980484</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4189002">PMC4189002</a></div></div>
</div>
<div id="ref-Azi1X1R4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">194. </div><div class="csl-right-inline"><strong>Online, High-Pressure Digestion System for Protein Characterization by Hydrogen/Deuterium Exchange and Mass Spectrometry</strong> <div class="csl-block">Lisa M Jones, Hao Zhang, Ilan Vidavsky, Michael L Gross</div> <em>Analytical Chemistry</em> (2010-01-22) <a href="https://doi.org/b993rm">https://doi.org/b993rm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac902477u">10.1021/ac902477u</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20095571">20095571</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2826105">PMC2826105</a></div></div>
</div>
<div id="ref-1nMCmYmb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">195. </div><div class="csl-right-inline"><strong>Hydrogen/deuterium exchange in mass spectrometry</strong> <div class="csl-block">Yury Kostyukevich, Thamina Acter, Alexander Zherebker, Arif Ahmed, Sunghwan Kim, Eugene Nikolaev</div> <em>Mass Spectrometry Reviews</em> (2018-03-30) <a href="https://doi.org/gffzx8">https://doi.org/gffzx8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.21565">10.1002/mas.21565</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29603316">29603316</a></div></div>
</div>
<div id="ref-YOXX6An6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">196. </div><div class="csl-right-inline"><strong>Proteinase K from Tritirachium album Limber.</strong> <div class="csl-block">W Ebeling, N Hennrich, M Klockow, H Metz, HD Orth, H Lang</div> <em>European journal of biochemistry</em> (1974-08-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/4373242">https://www.ncbi.nlm.nih.gov/pubmed/4373242</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/j.1432-1033.1974.tb03671.x">10.1111/j.1432-1033.1974.tb03671.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/4373242">4373242</a></div></div>
</div>
<div id="ref-1CgjQBy2S" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">197. </div><div class="csl-right-inline"><strong>Proteinase K</strong> <div class="csl-block">W Saenger</div> <em>Handbook of Proteolytic Enzymes</em> (2013) <a href="https://doi.org/gkfkcz">https://doi.org/gkfkcz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/b978-0-12-382219-2.00714-6">https://doi.org/10.1016/b978-0-12-382219-2.00714-6</a> · ISBN: 9780123822192</div></div>
</div>
<div id="ref-1BhNuatKH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">198. </div><div class="csl-right-inline"><strong>Measuring protein structural changes on a proteome-wide scale using limited proteolysis-coupled mass spectrometry.</strong> <div class="csl-block">Simone Schopper, Abdullah Kahraman, Pascal Leuenberger, Yuehan Feng, Ilaria Piazza, Oliver Müller, Paul J Boersema, Paola Picotti</div> <em>Nature protocols</em> (2017-10-26) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29072706">https://www.ncbi.nlm.nih.gov/pubmed/29072706</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nprot.2017.100">10.1038/nprot.2017.100</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29072706">29072706</a></div></div>
</div>
<div id="ref-YTQzoOxb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">199. </div><div class="csl-right-inline"><strong>Stable Isotope Labeling by Amino Acids in Cell Culture, SILAC, as a Simple and Accurate Approach to Expression Proteomics</strong> <div class="csl-block">Shao-En Ong, Blagoy Blagoev, Irina Kratchmarova, Dan Bach Kristensen, Hanno Steen, Akhilesh Pandey, Matthias Mann</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2002-05) <a href="https://doi.org/djw454">https://doi.org/djw454</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m200025-mcp200">10.1074/mcp.m200025-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12118079">12118079</a></div></div>
</div>
<div id="ref-1FOQ1H9mS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">200. </div><div class="csl-right-inline"><strong>iTRAQ Labeling is Superior to mTRAQ for Quantitative Global Proteomics and Phosphoproteomics</strong> <div class="csl-block">Philipp Mertins, Namrata D Udeshi, Karl R Clauser, DR Mani, Jinal Patel, Shao-en Ong, Jacob D Jaffe, Steven A Carr</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2012-06) <a href="https://doi.org/fx5pb3">https://doi.org/fx5pb3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m111.014423">10.1074/mcp.m111.014423</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22210691">22210691</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3433912">PMC3433912</a></div></div>
</div>
<div id="ref-Qqbj9JVK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">201. </div><div class="csl-right-inline"><strong>Multiplex peptide stable isotope dimethyl labeling for quantitative proteomics.</strong> <div class="csl-block">Paul J Boersema, Reinout Raijmakers, Simone Lemeer, Shabaz Mohammed, Albert JR Heck</div> <em>Nature protocols</em> (2009) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19300442">https://www.ncbi.nlm.nih.gov/pubmed/19300442</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nprot.2009.21">10.1038/nprot.2009.21</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19300442">19300442</a></div></div>
</div>
<div id="ref-E7KmdZas" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">202. </div><div class="csl-right-inline"><strong>In Vivo SILAC-Based Proteomics Reveals Phosphoproteome Changes during Mouse Skin Carcinogenesis</strong> <div class="csl-block">Sara Zanivan, Alexander Meves, Kristina Behrendt, Erwin M Schoof, Lisa J Neilson, Jürgen Cox, Hao R Tang, Gabriela Kalna, Janine H van Ree, Jan M van Deursen, … Matthias Mann</div> <em>Cell Reports</em> (2013-02) <a href="https://doi.org/f4r3vk">https://doi.org/f4r3vk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.celrep.2013.01.003">10.1016/j.celrep.2013.01.003</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23375375">23375375</a></div></div>
</div>
<div id="ref-TGkcmyuH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">203. </div><div class="csl-right-inline"><strong>Large-scale phosphosite quantification in tissues by a spike-in SILAC method</strong> <div class="csl-block">Mara Monetti, Nagarjuna Nagaraj, Kirti Sharma, Matthias Mann</div> <em>Nature Methods</em> (2011-07-10) <a href="https://doi.org/d28v54">https://doi.org/d28v54</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.1647">10.1038/nmeth.1647</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21743459">21743459</a></div></div>
</div>
<div id="ref-QipLkzBy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">204. </div><div class="csl-right-inline"><strong>Increasing the throughput of sensitive proteomics by plexDIA</strong> <div class="csl-block">Jason Derks, Andrew Leduc, Georg Wallmann, RGray Huffman, Matthew Willetts, Saad Khan, Harrison Specht, Markus Ralser, Vadim Demichev, Nikolai Slavov</div> <em>Nature Biotechnology</em> (2022-07-14) <a href="https://doi.org/gss5jp">https://doi.org/gss5jp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41587-022-01389-w">10.1038/s41587-022-01389-w</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35835881">35835881</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9839897">PMC9839897</a></div></div>
</div>
<div id="ref-ZhSU2auj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">205. </div><div class="csl-right-inline"><strong>Robust dimethyl‐based multiplex‐DIA doubles single‐cell proteome depth via a reference channel</strong> <div class="csl-block">Marvin Thielert, Ericka CM Itang, Constantin Ammar, Florian A Rosenberger, Isabell Bludau, Lisa Schweizer, Thierry M Nordmann, Patricia Skowronek, Maria Wahle, Wen‐Feng Zeng, … Matthias Mann</div> <em>Molecular Systems Biology</em> (2023-08-21) <a href="https://doi.org/gss5jt">https://doi.org/gss5jt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.15252/msb.202211503">10.15252/msb.202211503</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37602975">37602975</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10495816">PMC10495816</a></div></div>
</div>
<div id="ref-WPmaUWBS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">206. </div><div class="csl-right-inline"><strong>Quantification of Lysine Acetylation and Succinylation Stoichiometry in Proteins Using Mass Spectrometric Data-Independent Acquisitions (SWATH)</strong> <div class="csl-block">Jesse G Meyer, Alexandria K D’Souza, Dylan J Sorensen, Matthew J Rardin, Alan J Wolfe, Bradford W Gibson, Birgit Schilling</div> <em>Journal of the American Society for Mass Spectrometry</em> (2016-09-02) <a href="https://doi.org/f86vtw">https://doi.org/f86vtw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s13361-016-1476-z">10.1007/s13361-016-1476-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27590315">27590315</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5059418">PMC5059418</a></div></div>
</div>
<div id="ref-j9N2kPSr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">207. </div><div class="csl-right-inline"><strong>Tandem Mass Tags:  A Novel Quantification Strategy for Comparative Analysis of Complex Protein Mixtures by MS/MS</strong> <div class="csl-block">Andrew Thompson, Jürgen Schäfer, Karsten Kuhn, Stefan Kienle, Josef Schwarz, Günter Schmidt, Thomas Neumann, Christian Hamon</div> <em>Analytical Chemistry</em> (2003-03-01) <a href="https://doi.org/bghhcr">https://doi.org/bghhcr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac0262560">10.1021/ac0262560</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12713048">12713048</a></div></div>
</div>
<div id="ref-V8IykxaB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">208. </div><div class="csl-right-inline"><strong>Multiplexed Protein Quantitation in Saccharomyces cerevisiae Using Amine-reactive Isobaric Tagging Reagents</strong> <div class="csl-block">Philip L Ross, Yulin N Huang, Jason N Marchese, Brian Williamson, Kenneth Parker, Stephen Hattan, Nikita Khainovski, Sasi Pillai, Subhakar Dey, Scott Daniels, … Darryl J Pappin</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2004-12) <a href="https://doi.org/cqjzsb">https://doi.org/cqjzsb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m400129-mcp200">10.1074/mcp.m400129-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15385600">15385600</a></div></div>
</div>
<div id="ref-lbOXqNSw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">209. </div><div class="csl-right-inline"><strong>&lt;i&gt;N&lt;/i&gt;,&lt;i&gt;N&lt;/i&gt;-Dimethyl Leucines as Novel Isobaric Tandem Mass Tags for Quantitative Proteomics and Peptidomics</strong> <div class="csl-block">Feng Xiang, Hui Ye, Ruibing Chen, Qiang Fu, Lingjun Li</div> <em>Analytical Chemistry</em> (2010-03-10) <a href="https://doi.org/b7smr6">https://doi.org/b7smr6</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac902778d">10.1021/ac902778d</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20218596">20218596</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2859709">PMC2859709</a></div></div>
</div>
<div id="ref-12bJg7wN8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">210. </div><div class="csl-right-inline"><strong>TMTpro: Design, Synthesis, and Initial Evaluation of a Proline-Based Isobaric 16-Plex Tandem Mass Tag Reagent Set</strong> <div class="csl-block">Andrew Thompson, Nikolai Wölmer, Sasa Koncarevic, Stefan Selzer, Gitte Böhm, Harald Legner, Peter Schmid, Stefan Kienle, Petra Penning, Claudia Höhle, … Ian Pike</div> <em>Analytical Chemistry</em> (2019-11-18) <a href="https://doi.org/gss5jj">https://doi.org/gss5jj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.9b04474">10.1021/acs.analchem.9b04474</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31738517">31738517</a></div></div>
</div>
<div id="ref-1HEOHr3h0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">211. </div><div class="csl-right-inline"><strong>TMTpro-18plex: The Expanded and Complete Set of TMTpro Reagents for Sample Multiplexing</strong> <div class="csl-block">Jiaming Li, Zhenying Cai, Ryan D Bomgarden, Ian Pike, Karsten Kuhn, John C Rogers, Thomas M Roberts, Steven P Gygi, Joao A Paulo</div> <em>Journal of Proteome Research</em> (2021-04-26) <a href="https://doi.org/gjt6rs">https://doi.org/gjt6rs</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.1c00168">10.1021/acs.jproteome.1c00168</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33900084">33900084</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8210943">PMC8210943</a></div></div>
</div>
<div id="ref-15rS5UMb9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">212. </div><div class="csl-right-inline"><strong>Isobaric labeling-based relative quantification in shotgun proteomics.</strong> <div class="csl-block">Navin Rauniyar, John R Yates</div> <em>Journal of proteome research</em> (2014-11-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25337643">https://www.ncbi.nlm.nih.gov/pubmed/25337643</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr500880b">10.1021/pr500880b</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25337643">25337643</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4261935">PMC4261935</a></div></div>
</div>
<div id="ref-16D3xg72c" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">213. </div><div class="csl-right-inline"><strong>Computational Methods for Understanding Mass Spectrometry–Based Shotgun Proteomics Data</strong> <div class="csl-block">Pavel Sinitcyn, Jan Daniel Rudolph, Jürgen Cox</div> <em>Annual Review of Biomedical Data Science</em> (2018-07-20) <a href="https://doi.org/gjtq4h">https://doi.org/gjtq4h</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1146/annurev-biodatasci-080917-013516">10.1146/annurev-biodatasci-080917-013516</a></div></div>
</div>
<div id="ref-kIpNiFeU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">214. </div><div class="csl-right-inline"><strong>Peptide Labeling Using Isobaric Tagging Reagents for Quantitative Phosphoproteomics.</strong> <div class="csl-block">Lei Cheng, Trairak Pisitkun, Mark A Knepper, Jason D Hoffert</div> <em>Methods in molecular biology (Clifton, N.J.)</em> (2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26584918">https://www.ncbi.nlm.nih.gov/pubmed/26584918</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-4939-3049-4_4">10.1007/978-1-4939-3049-4_4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26584918">26584918</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6169521">PMC6169521</a></div></div>
</div>
<div id="ref-ESY1qR9T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">215. </div><div class="csl-right-inline"><strong>Discovery- and target-based protein quantification using iTRAQ and pulsed Q collision induced dissociation (PQD).</strong> <div class="csl-block">Wells W Wu, Guanghui Wang, Paul A Insel, Cheng-Te Hsiao, Sige Zou, Bronwen Martin, Stuart Maudsley, Rong-Fong Shen</div> <em>Journal of proteomics</em> (2012-03-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22397766">https://www.ncbi.nlm.nih.gov/pubmed/22397766</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jprot.2012.02.027">10.1016/j.jprot.2012.02.027</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22397766">22397766</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3321122">PMC3321122</a></div></div>
</div>
<div id="ref-FMSpJgZ4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">216. </div><div class="csl-right-inline"><strong>Insights into protein post-translational modification landscapes of individual human cells by trapped ion mobility time-of-flight mass spectrometry</strong> <div class="csl-block">Benjamin C Orsburn, Yuting Yuan, Namandjé N Bumpus</div> <em>Nature Communications</em> (2022-11-25) <a href="https://doi.org/grhnzj">https://doi.org/grhnzj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-022-34919-w">10.1038/s41467-022-34919-w</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36433961">36433961</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9700839">PMC9700839</a></div></div>
</div>
<div id="ref-OlzbV8x4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">217. </div><div class="csl-right-inline"><strong>TMT Labeling for the Masses: A Robust and Cost-efficient, In-solution Labeling Approach.</strong> <div class="csl-block">Jana Zecha, Shankha Satpathy, Tamara Kanashova, Shayan C Avanessian, MHarry Kane, Karl R Clauser, Philipp Mertins, Steven A Carr, Bernhard Kuster</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2019-04-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30967486">https://www.ncbi.nlm.nih.gov/pubmed/30967486</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.tir119.001385">10.1074/mcp.tir119.001385</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30967486">30967486</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6601210">PMC6601210</a></div></div>
</div>
<div id="ref-JokZNPi7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">218. </div><div class="csl-right-inline"><strong>Multiplexed protein quantitation in Saccharomyces cerevisiae using amine-reactive isobaric tagging reagents.</strong> <div class="csl-block">Philip L Ross, Yulin N Huang, Jason N Marchese, Brian Williamson, Kenneth Parker, Stephen Hattan, Nikita Khainovski, Sasi Pillai, Subhakar Dey, Scott Daniels, … Darryl J Pappin</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2004-09-22) <a href="https://www.ncbi.nlm.nih.gov/pubmed/15385600">https://www.ncbi.nlm.nih.gov/pubmed/15385600</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m400129-mcp200">10.1074/mcp.m400129-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15385600">15385600</a></div></div>
</div>
<div id="ref-43RxThXr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">219. </div><div class="csl-right-inline"><strong>Peptide labeling with isobaric tags yields higher identification rates using iTRAQ 4-plex compared to TMT 6-plex and iTRAQ 8-plex on LTQ Orbitrap.</strong> <div class="csl-block">Peter Pichler, Thomas Köcher, Johann Holzmann, Michael Mazanek, Thomas Taus, Gustav Ammerer, Karl Mechtler</div> <em>Analytical chemistry</em> (2010-08-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20593797">https://www.ncbi.nlm.nih.gov/pubmed/20593797</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac100890k">10.1021/ac100890k</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20593797">20593797</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3093924">PMC3093924</a></div></div>
</div>
<div id="ref-1At6lP9vn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">220. </div><div class="csl-right-inline"><strong>Comparison of 4-plex to 8-plex iTRAQ quantitative measurements of proteins in human plasma samples.</strong> <div class="csl-block">Gwenael Pottiez, Jayme Wiederin, Howard S Fox, Pawel Ciborowski</div> <em>Journal of proteome research</em> (2012-05-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22594965">https://www.ncbi.nlm.nih.gov/pubmed/22594965</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr300414z">10.1021/pr300414z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22594965">22594965</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3390908">PMC3390908</a></div></div>
</div>
<div id="ref-5gl2gxA4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">221. </div><div class="csl-right-inline"><strong>Novel method to investigate protein carbonylation by iTRAQ strategy.</strong> <div class="csl-block">Angelo Palmese, Chiara De Rosa, Giovanni Chiappetta, Gennaro Marino, Angela Amoresano</div> <em>Analytical and bioanalytical chemistry</em> (2012-08-29) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22926130">https://www.ncbi.nlm.nih.gov/pubmed/22926130</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00216-012-6324-9">10.1007/s00216-012-6324-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22926130">22926130</a></div></div>
</div>
<div id="ref-t4QMhmYv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">222. </div><div class="csl-right-inline"><strong>Role of oxidative carbonylation in protein quality control and senescence.</strong> <div class="csl-block">Thomas Nyström</div> <em>The EMBO journal</em> (2005-03-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/15775985">https://www.ncbi.nlm.nih.gov/pubmed/15775985</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/sj.emboj.7600599">10.1038/sj.emboj.7600599</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15775985">15775985</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1142534">PMC1142534</a></div></div>
</div>
<div id="ref-9irghCdu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">223. </div><div class="csl-right-inline"><strong>Evaluation of the Orbitrap Ascend Tribrid Mass Spectrometer for Shotgun Proteomics</strong> <div class="csl-block">Yuchen He, Evgenia Shishkova, Trenton M Peters-Clarke, Dain R Brademan, Michael S Westphall, David Bergen, Jingjing Huang, Romain Huguet, Michael W Senko, Vlad Zabrouskov, … Joshua J Coon</div> <em>Analytical Chemistry</em> (2023-06-30) <a href="https://doi.org/gss5jf">https://doi.org/gss5jf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.3c01155">10.1021/acs.analchem.3c01155</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37389810">37389810</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10528367">PMC10528367</a></div></div>
</div>
<div id="ref-1GJhuH2mX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">224. </div><div class="csl-right-inline"><strong>MS3 eliminates ratio distortion in isobaric multiplexed quantitative proteomics.</strong> <div class="csl-block">Lily Ting, Ramin Rad, Steven P Gygi, Wilhelm Haas</div> <em>Nature methods</em> (2011-10-02) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21963607">https://www.ncbi.nlm.nih.gov/pubmed/21963607</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.1714">10.1038/nmeth.1714</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21963607">21963607</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3205343">PMC3205343</a></div></div>
</div>
<div id="ref-agiL3n2A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">225. </div><div class="csl-right-inline"><strong>MultiNotch MS3 Enables Accurate, Sensitive, and Multiplexed Detection of Differential Expression across Cancer Cell Line Proteomes</strong> <div class="csl-block">Graeme C McAlister, David P Nusinow, Mark P Jedrychowski, Martin Wühr, Edward L Huttlin, Brian K Erickson, Ramin Rad, Wilhelm Haas, Steven P Gygi</div> <em>Analytical Chemistry</em> (2014-07-03) <a href="https://doi.org/f59sb8">https://doi.org/f59sb8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac502040v">10.1021/ac502040v</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24927332">24927332</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4215866">PMC4215866</a></div></div>
</div>
<div id="ref-1EjowCoSS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">226. </div><div class="csl-right-inline"><strong>Infrared Photoactivation Boosts Reporter Ion Yield in Isobaric Tagging</strong> <div class="csl-block">Kenneth W Lee, Trenton M Peters-Clarke, Keaton L Mertz, Graeme C McAlister, John EP Syka, Michael S Westphall, Joshua J Coon</div> <em>Analytical Chemistry</em> (2022-02-10) <a href="https://doi.org/gssbgz">https://doi.org/gssbgz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.1c05398">10.1021/acs.analchem.1c05398</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35142486">35142486</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8985229">PMC8985229</a></div></div>
</div>
<div id="ref-ghHhZLY4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">227. </div><div class="csl-right-inline"><strong>Mass spectrometry-based quantitative proteomics for dissecting multiplexed redox cysteine modifications in nitric oxide-protected cardiomyocyte under hypoxia.</strong> <div class="csl-block">Kuan-Ting Pan, Yi-Yun Chen, Tsung-Hsien Pu, Yu-Shu Chao, Chun-Yi Yang, Ryan D Bomgarden, John C Rogers, Tzu-Ching Meng, Kay-Hooi Khoo</div> <em>Antioxidants &amp; redox signaling</em> (2013-10-23) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24152285">https://www.ncbi.nlm.nih.gov/pubmed/24152285</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1089/ars.2013.5326">10.1089/ars.2013.5326</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24152285">24152285</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3936484">PMC3936484</a></div></div>
</div>
<div id="ref-RikGmYWm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">228. </div><div class="csl-right-inline"><strong>Proteomic quantification and site-mapping of S-nitrosylated proteins using isobaric iodoTMT reagents.</strong> <div class="csl-block">Zhe Qu, Fanjun Meng, Ryan D Bomgarden, Rosa I Viner, Jilong Li, John C Rogers, Jianlin Cheng, CMichael Greenlief, Jiankun Cui, Dennis B Lubahn, … Zezong Gu</div> <em>Journal of proteome research</em> (2014-06-13) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24926564">https://www.ncbi.nlm.nih.gov/pubmed/24926564</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr401179v">10.1021/pr401179v</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24926564">24926564</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4084841">PMC4084841</a></div></div>
</div>
<div id="ref-XS3gqf5Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">229. </div><div class="csl-right-inline"><strong>Carbonyl-reactive tandem mass tags for the proteome-wide quantification of N-linked glycans.</strong> <div class="csl-block">Hannes Hahne, Patrick Neubert, Karsten Kuhn, Chris Etienne, Ryan Bomgarden, John C Rogers, Bernhard Kuster</div> <em>Analytical chemistry</em> (2012-04-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22455665">https://www.ncbi.nlm.nih.gov/pubmed/22455665</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac300197c">10.1021/ac300197c</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22455665">22455665</a></div></div>
</div>
<div id="ref-WvWnJWa8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">230. </div><div class="csl-right-inline"><strong>N,N-dimethyl leucines as novel isobaric tandem mass tags for quantitative proteomics and peptidomics.</strong> <div class="csl-block">Feng Xiang, Hui Ye, Ruibing Chen, Qiang Fu, Lingjun Li</div> <em>Analytical chemistry</em> (2010-04-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20218596">https://www.ncbi.nlm.nih.gov/pubmed/20218596</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac902778d">10.1021/ac902778d</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20218596">20218596</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2859709">PMC2859709</a></div></div>
</div>
<div id="ref-OhLoelJI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">231. </div><div class="csl-right-inline"><strong>Comparative evaluation of two isobaric labeling tags, DiART and iTRAQ.</strong> <div class="csl-block">Zhen Chen, Quanhui Wang, Liang Lin, Qi Tang, James L Edwards, Shuwei Li, Siqi Liu</div> <em>Analytical chemistry</em> (2012-03-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22404494">https://www.ncbi.nlm.nih.gov/pubmed/22404494</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac203467q">10.1021/ac203467q</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22404494">22404494</a></div></div>
</div>
<div id="ref-1FMnd7AHB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">232. </div><div class="csl-right-inline"><strong>Deuterium isobaric amine-reactive tags for quantitative proteomics.</strong> <div class="csl-block">Junxiang Zhang, Yan Wang, Shuwei Li</div> <em>Analytical chemistry</em> (2010-09-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20715779">https://www.ncbi.nlm.nih.gov/pubmed/20715779</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac101306x">10.1021/ac101306x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20715779">20715779</a></div></div>
</div>
<div id="ref-1CFiIlIUn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">233. </div><div class="csl-right-inline"><strong>A robust error model for iTRAQ quantification reveals divergent signaling between oncogenic FLT3 mutants in acute myeloid leukemia.</strong> <div class="csl-block">Yi Zhang, Manor Askenazi, Jingrui Jiang, CJohn Luckey, James D Griffin, Jarrod A Marto</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2009-12-17) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20019052">https://www.ncbi.nlm.nih.gov/pubmed/20019052</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m900452-mcp200">10.1074/mcp.m900452-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20019052">20019052</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2871413">PMC2871413</a></div></div>
</div>
<div id="ref-bHqseFm6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">234. </div><div class="csl-right-inline"><strong>Cost-effective isobaric tagging for quantitative phosphoproteomics using DiART reagents.</strong> <div class="csl-block">Nikhil Ramsubramaniam, Feng Tao, Shuwei Li, Mark R Marten</div> <em>Molecular bioSystems</em> (2013-10-16) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24129742">https://www.ncbi.nlm.nih.gov/pubmed/24129742</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/c3mb70358d">10.1039/c3mb70358d</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24129742">24129742</a></div></div>
</div>
<div id="ref-2h6JMw35" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">235. </div><div class="csl-right-inline"><strong>Protein glycosylation: nature, distribution, enzymatic formation, and disease implications of glycopeptide bonds</strong> <div class="csl-block">RG Spiro</div> <em>Glycobiology</em> (2002-04-01) <a href="https://doi.org/dvf2t7">https://doi.org/dvf2t7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/glycob/12.4.43r">10.1093/glycob/12.4.43r</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12042244">12042244</a></div></div>
</div>
<div id="ref-DC1ukRtO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">236. </div><div class="csl-right-inline"><strong>A Pragmatic Guide to Enrichment Strategies for Mass Spectrometry–Based Glycoproteomics</strong> <div class="csl-block">Nicholas M Riley, Carolyn R Bertozzi, Sharon J Pitteri</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2021) <a href="https://doi.org/gjtq4p">https://doi.org/gjtq4p</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.r120.002277">10.1074/mcp.r120.002277</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33583771">33583771</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8724846">PMC8724846</a></div></div>
</div>
<div id="ref-pgTlWbfO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">237. </div><div class="csl-right-inline"><strong>Enrichment of Intact Glycopeptides Using Strong Anion Exchange and Electrostatic Repulsion Hydrophilic Interaction Chromatography</strong> <div class="csl-block">Abel Bermudez, Sharon J Pitteri</div> <em>Methods in Molecular Biology</em> (2021) <a href="https://doi.org/gss5jb">https://doi.org/gss5jb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-0716-1241-5_8">10.1007/978-1-0716-1241-5_8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33908003">33908003</a></div></div>
</div>
<div id="ref-3T8OfCLe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">238. </div><div class="csl-right-inline"><strong>Quantitative site-specific analysis of protein glycosylation by LC-MS using different glycopeptide-enrichment strategies</strong> <div class="csl-block">Jessica Wohlgemuth, Michael Karas, Thomas Eichhorn, Robertus Hendriks, Sven Andrecht</div> <em>Analytical Biochemistry</em> (2009-12) <a href="https://doi.org/c4hqcn">https://doi.org/c4hqcn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ab.2009.08.023">10.1016/j.ab.2009.08.023</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19699707">19699707</a></div></div>
</div>
<div id="ref-17QEVShGe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">239. </div><div class="csl-right-inline"><strong>Advances in LC–MS/MS-based glycoproteomics: Getting closer to system-wide site-specific mapping of the N- and O-glycoproteome</strong> <div class="csl-block">Morten Thaysen-Andersen, Nicolle H Packer</div> <em>Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics</em> (2014-09) <a href="https://doi.org/f6ckz2">https://doi.org/f6ckz2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.bbapap.2014.05.002">10.1016/j.bbapap.2014.05.002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24830338">24830338</a></div></div>
</div>
<div id="ref-ZH7oVCIW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">240. </div><div class="csl-right-inline"><strong>Structural analysis of glycoprotein sialylation – Part I: pre-LC-MS analytical strategies</strong> <div class="csl-block">Morten Thaysen-Andersen, Martin R Larsen, Nicolle H Packer, Giuseppe Palmisano</div> <em>RSC Advances</em> (2013) <a href="https://doi.org/gss5jr">https://doi.org/gss5jr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/c3ra42960a">10.1039/c3ra42960a</a></div></div>
</div>
<div id="ref-4qdFwSjo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">241. </div><div class="csl-right-inline"><strong>Recent Advances in the Analysis of Complex Glycoproteins</strong> <div class="csl-block">Stefan Gaunitz, Gabe Nagy, Nicola LB Pohl, Milos V Novotny</div> <em>Analytical Chemistry</em> (2016-11-23) <a href="https://doi.org/gss5jg">https://doi.org/gss5jg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.6b04343">10.1021/acs.analchem.6b04343</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28105826">28105826</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5609817">PMC5609817</a></div></div>
</div>
<div id="ref-8FBpI8YZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">242. </div><div class="csl-right-inline"><strong>Phosphorylation of Single Stranded RNA Virus Proteins and Potential for Novel Therapeutic Strategies.</strong> <div class="csl-block">Forrest Keck, Pouya Ataey, Moushimi Amaya, Charles Bailey, Aarthi Narayanan</div> <em>Viruses</em> (2015-10-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26473910">https://www.ncbi.nlm.nih.gov/pubmed/26473910</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/v7102872">10.3390/v7102872</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26473910">26473910</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4632380">PMC4632380</a></div></div>
</div>
<div id="ref-ahQQ60nZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">243. </div><div class="csl-right-inline"><strong>The functional landscape of the human phosphoproteome.</strong> <div class="csl-block">David Ochoa, Andrew F Jarnuczak, Cristina Viéitez, Maja Gehre, Margaret Soucheray, André Mateus, Askar A Kleefeldt, Anthony Hill, Luz Garcia-Alonso, Frank Stein, … Pedro Beltrao</div> <em>Nature biotechnology</em> (2019-12-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/31819260">https://www.ncbi.nlm.nih.gov/pubmed/31819260</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41587-019-0344-3">10.1038/s41587-019-0344-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31819260">31819260</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7100915">PMC7100915</a></div></div>
</div>
<div id="ref-FzJAr4ba" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">244. </div><div class="csl-right-inline"><strong>A review on recent trends in the phosphoproteomics workflow. From sample preparation to data analysis.</strong> <div class="csl-block">Jiří Urban</div> <em>Analytica chimica acta</em> (2021-07-17) <a href="https://www.ncbi.nlm.nih.gov/pubmed/35227377">https://www.ncbi.nlm.nih.gov/pubmed/35227377</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.aca.2021.338857">10.1016/j.aca.2021.338857</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35227377">35227377</a></div></div>
</div>
<div id="ref-CG2bNzkD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">245. </div><div class="csl-right-inline"><strong>The role of protein phosphorylation in human health and disease. The Sir Hans Krebs Medal Lecture.</strong> <div class="csl-block">P Cohen</div> <em>European journal of biochemistry</em> (2001-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/11589691">https://www.ncbi.nlm.nih.gov/pubmed/11589691</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1046/j.0014-2956.2001.02473.x">10.1046/j.0014-2956.2001.02473.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11589691">11589691</a></div></div>
</div>
<div id="ref-gPKAjoqC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">246. </div><div class="csl-right-inline"><strong>Modification-specific proteomics: strategies for characterization of post-translational modifications using enrichment techniques.</strong> <div class="csl-block">Yingming Zhao, Ole N Jensen</div> <em>Proteomics</em> (2009-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19743430">https://www.ncbi.nlm.nih.gov/pubmed/19743430</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200900398">10.1002/pmic.200900398</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19743430">19743430</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2892724">PMC2892724</a></div></div>
</div>
<div id="ref-9tmuuRlu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">247. </div><div class="csl-right-inline"><strong>Using phosphoproteomics data to understand cellular signaling: a comprehensive guide to bioinformatics resources.</strong> <div class="csl-block">Sara R Savage, Bing Zhang</div> <em>Clinical proteomics</em> (2020-07-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/32676006">https://www.ncbi.nlm.nih.gov/pubmed/32676006</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/s12014-020-09290-x">10.1186/s12014-020-09290-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32676006">32676006</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7353784">PMC7353784</a></div></div>
</div>
<div id="ref-ZhEOEHzl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">248. </div><div class="csl-right-inline"><strong>Analytical challenges translating mass spectrometry-based phosphoproteomics from discovery to clinical applications.</strong> <div class="csl-block">Anton B Iliuk, Justine V Arrington, Weiguo Andy Tao</div> <em>Electrophoresis</em> (2014-07-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24890697">https://www.ncbi.nlm.nih.gov/pubmed/24890697</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/elps.201400153">10.1002/elps.201400153</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24890697">24890697</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4250476">PMC4250476</a></div></div>
</div>
<div id="ref-IZ8H05sX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">249. </div><div class="csl-right-inline"><strong>Estimating the Efficiency of Phosphopeptide Identification by Tandem Mass Spectrometry.</strong> <div class="csl-block">Chuan-Chih Hsu, Liang Xue, Justine V Arrington, Pengcheng Wang, Juan Sebastian Paez Paez, Yuan Zhou, Jian-Kang Zhu, WAndy Tao</div> <em>Journal of the American Society for Mass Spectrometry</em> (2017-03-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28283928">https://www.ncbi.nlm.nih.gov/pubmed/28283928</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s13361-017-1603-5">10.1007/s13361-017-1603-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28283928">28283928</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5438756">PMC5438756</a></div></div>
</div>
<div id="ref-1DdZgE3oc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">250. </div><div class="csl-right-inline"><strong>Preparation of proteins and peptides for mass spectrometry analysis in a bottom-up proteomics workflow.</strong> <div class="csl-block">Rebekah L Gundry, Melanie Y White, Christopher I Murray, Lesley A Kane, Qin Fu, Brian A Stanley, Jennifer E Van Eyk</div> <em>Current protocols in molecular biology</em> (2009-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19816929">https://www.ncbi.nlm.nih.gov/pubmed/19816929</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/0471142727.mb1025s88">10.1002/0471142727.mb1025s88</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19816929">19816929</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2905857">PMC2905857</a></div></div>
</div>
<div id="ref-1AhoIVYp8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">251. </div><div class="csl-right-inline"><strong>Phosphopeptide Fragmentation and Site Localization by Mass Spectrometry: An Update.</strong> <div class="csl-block">Clement M Potel, Simone Lemeer, Albert JR Heck</div> <em>Analytical chemistry</em> (2018-12-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30457327">https://www.ncbi.nlm.nih.gov/pubmed/30457327</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.8b04746">10.1021/acs.analchem.8b04746</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30457327">30457327</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6322148">PMC6322148</a></div></div>
</div>
<div id="ref-2bm470Yr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">252. </div><div class="csl-right-inline"><strong>Enrichment techniques employed in phosphoproteomics.</strong> <div class="csl-block">Jan Fíla, David Honys</div> <em>Amino acids</em> (2011-10-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22002794">https://www.ncbi.nlm.nih.gov/pubmed/22002794</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00726-011-1111-z">10.1007/s00726-011-1111-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22002794">22002794</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3418503">PMC3418503</a></div></div>
</div>
<div id="ref-OoEPX07F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">253. </div><div class="csl-right-inline"><strong>Off-Line High-pH Reversed-Phase Fractionation for In-Depth Phosphoproteomics</strong> <div class="csl-block">Tanveer S Batth, Chiara Francavilla, Jesper V Olsen</div> <em>Journal of Proteome Research</em> (2014-11-04) <a href="https://doi.org/f72pvg">https://doi.org/f72pvg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr500893m">10.1021/pr500893m</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25338131">25338131</a></div></div>
</div>
<div id="ref-cXqDwSab" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">254. </div><div class="csl-right-inline"><strong>Temporal Quantitative Phosphoproteomics Profiling of Interleukin-33 Signaling Network Reveals Unique Modulators of Monocyte Activation.</strong> <div class="csl-block">Devasahayam Arokia Balaya Rex, Yashwanth Subbannayya, Prashant Kumar Modi, Akhina Palollathil, Lathika Gopalakrishnan, Yashodhar P Bhandary, Thottethodi Subrahmanya Keshava Prasad, Sneha M Pinto</div> <em>Cells</em> (2022-01-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/35011700">https://www.ncbi.nlm.nih.gov/pubmed/35011700</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/cells11010138">10.3390/cells11010138</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35011700">35011700</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8749991">PMC8749991</a></div></div>
</div>
<div id="ref-vghskTda" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">255. </div><div class="csl-right-inline"><strong>Immobilized gallium(III) affinity chromatography of phosphopeptides.</strong> <div class="csl-block">MC Posewitz, P Tempst</div> <em>Analytical chemistry</em> (1999-07-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10424175">https://www.ncbi.nlm.nih.gov/pubmed/10424175</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac981409y">10.1021/ac981409y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10424175">10424175</a></div></div>
</div>
<div id="ref-1Biu5x5kI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">256. </div><div class="csl-right-inline"><strong>Effective disruption of phosphoprotein-protein surface interaction using Zn(II) dipicolylamine-based artificial receptors via two-point interaction.</strong> <div class="csl-block">Akio Ojida, Masa-aki Inoue, Yasuko Mito-oka, Hiroshi Tsutsumi, Kazuki Sada, Itaru Hamachi</div> <em>Journal of the American Chemical Society</em> (2006-02-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16464107">https://www.ncbi.nlm.nih.gov/pubmed/16464107</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ja056585k">10.1021/ja056585k</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16464107">16464107</a></div></div>
</div>
<div id="ref-GINSjn64" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">257. </div><div class="csl-right-inline"><strong>Isolation of phosphoproteins by immobilized metal (Fe3+) affinity chromatography.</strong> <div class="csl-block">L Andersson, J Porath</div> <em>Analytical biochemistry</em> (1986-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/3085541">https://www.ncbi.nlm.nih.gov/pubmed/3085541</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/0003-2697(86)90523-3">10.1016/0003-2697(86)90523-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3085541">3085541</a></div></div>
</div>
<div id="ref-tH2WxrEl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">258. </div><div class="csl-right-inline"><strong>Purification of phosphoproteins by immobilized metal affinity chromatography and its application to phosphoproteome analysis.</strong> <div class="csl-block">Mitsuyo Machida, Hidetaka Kosako, Kyoko Shirakabe, Michimoto Kobayashi, Masato Ushiyama, Junichi Inagawa, Joe Hirano, Tomoyo Nakano, Yasuhiko Bando, Eisuke Nishida, Seisuke Hattori</div> <em>The FEBS journal</em> (2007-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17480206">https://www.ncbi.nlm.nih.gov/pubmed/17480206</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/j.1742-4658.2007.05705.x">10.1111/j.1742-4658.2007.05705.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17480206">17480206</a></div></div>
</div>
<div id="ref-LbRH4fW1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">259. </div><div class="csl-right-inline"><strong>Efficient enrichment of intact phosphorylated proteins by modified immobilized metal-affinity chromatography.</strong> <div class="csl-block">Anna Dubrovska, Serhiy Souchelnytskyi</div> <em>Proteomics</em> (2005-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16252304">https://www.ncbi.nlm.nih.gov/pubmed/16252304</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200500002">10.1002/pmic.200500002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16252304">16252304</a></div></div>
</div>
<div id="ref-f75VaG7P" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">260. </div><div class="csl-right-inline"><strong>A quantitative investigation into the losses of proteins at different stages of a two-dimensional gel electrophoresis procedure.</strong> <div class="csl-block">Shaobo Zhou, Matthew J Bailey, Michael J Dunn, Victor R Preedy, Peter W Emery</div> <em>Proteomics</em> (2005-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/15966007">https://www.ncbi.nlm.nih.gov/pubmed/15966007</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200401178">10.1002/pmic.200401178</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15966007">15966007</a></div></div>
</div>
<div id="ref-CkrjTtPg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">261. </div><div class="csl-right-inline"><strong>Highly selective enrichment of phosphorylated peptides from peptide mixtures using titanium dioxide microcolumns.</strong> <div class="csl-block">Martin R Larsen, Tine E Thingholm, Ole N Jensen, Peter Roepstorff, Thomas JD Jørgensen</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2005-04-27) <a href="https://www.ncbi.nlm.nih.gov/pubmed/15858219">https://www.ncbi.nlm.nih.gov/pubmed/15858219</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.t500007-mcp200">10.1074/mcp.t500007-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15858219">15858219</a></div></div>
</div>
<div id="ref-11CslpEEW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">262. </div><div class="csl-right-inline"><strong>Phosphopeptide Enrichment by Immobilized Metal Affinity Chromatography.</strong> <div class="csl-block">Tine E Thingholm, Martin R Larsen</div> <em>Methods in molecular biology (Clifton, N.J.)</em> (2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26584922">https://www.ncbi.nlm.nih.gov/pubmed/26584922</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-4939-3049-4_8">10.1007/978-1-4939-3049-4_8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26584922">26584922</a></div></div>
</div>
<div id="ref-SLLSf3H4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">263. </div><div class="csl-right-inline"><strong>Selective Detection and Sequencing of Phosphopeptides at the Femtomole Level by Mass Spectrometry</strong> <div class="csl-block">Steven A Carr, Michael J Huddleston, Roland S Annan</div> <em>Analytical Biochemistry</em> (1996-08) <a href="https://doi.org/cp2r54">https://doi.org/cp2r54</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1006/abio.1996.0313">10.1006/abio.1996.0313</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8811904">8811904</a></div></div>
</div>
<div id="ref-Bij8Oa4F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">264. </div><div class="csl-right-inline"><strong>Comprehensive and reproducible phosphopeptide enrichment using iron immobilized metal ion affinity chromatography (Fe-IMAC) columns.</strong> <div class="csl-block">Benjamin Ruprecht, Heiner Koch, Guillaume Medard, Max Mundt, Bernhard Kuster, Simone Lemeer</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2014-11-13) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25394399">https://www.ncbi.nlm.nih.gov/pubmed/25394399</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m114.043109">10.1074/mcp.m114.043109</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25394399">25394399</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4288255">PMC4288255</a></div></div>
</div>
<div id="ref-15ptCbE4S" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">265. </div><div class="csl-right-inline"><strong>Optimized Enrichment of Phosphoproteomes by Fe-IMAC Column Chromatography.</strong> <div class="csl-block">Benjamin Ruprecht, Heiner Koch, Petra Domasinska, Martin Frejno, Bernhard Kuster, Simone Lemeer</div> <em>Methods in molecular biology (Clifton, N.J.)</em> (2017) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28188522">https://www.ncbi.nlm.nih.gov/pubmed/28188522</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-4939-6747-6_5">10.1007/978-1-4939-6747-6_5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28188522">28188522</a></div></div>
</div>
<div id="ref-fPTYjwxV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">266. </div><div class="csl-right-inline"><strong>Complementary Fe(3+)- and Ti(4+)-immobilized metal ion affinity chromatography for purification of acidic and basic phosphopeptides.</strong> <div class="csl-block">Alan Chuan-Ying Lai, Chia-Feng Tsai, Chuan-Chih Hsu, Yu-Ni Sun, Yu-Ju Chen</div> <em>Rapid communications in mass spectrometry : RCM</em> (2012-09-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22886815">https://www.ncbi.nlm.nih.gov/pubmed/22886815</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/rcm.6327">10.1002/rcm.6327</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22886815">22886815</a></div></div>
</div>
<div id="ref-Hii2H7k2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">267. </div><div class="csl-right-inline"><strong>Sequential Fe3O4/TiO2 enrichment for phosphopeptide analysis by liquid chromatography/tandem mass spectrometry.</strong> <div class="csl-block">Sunkyu Choi, Jaeyoon Kim, Kun Cho, Gunwook Park, Jong Hyuk Yoon, Sehoon Park, Jong Shin Yoo, Sung Ho Ryu, Young Hwan Kim, Jeongkwon Kim</div> <em>Rapid communications in mass spectrometry : RCM</em> (2010-05-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20411586">https://www.ncbi.nlm.nih.gov/pubmed/20411586</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/rcm.4541">10.1002/rcm.4541</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20411586">20411586</a></div></div>
</div>
<div id="ref-G33wPuEv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">268. </div><div class="csl-right-inline"><strong>Comprehensive Evaluation of Different TiO</strong> <div class="csl-block">Jiaran Li, Jifeng Wang, Yumeng Yan, Na Li, Xiaoqing Qing, Ailikemu Tuerxun, Xiaojing Guo, Xiulan Chen, Fuquan Yang</div> <em>Cells</em> (2022-06-28) <a href="https://www.ncbi.nlm.nih.gov/pubmed/35805136">https://www.ncbi.nlm.nih.gov/pubmed/35805136</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/cells11132047">10.3390/cells11132047</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35805136">35805136</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9265536">PMC9265536</a></div></div>
</div>
<div id="ref-LeABskLB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">269. </div><div class="csl-right-inline"><strong>The necessity of and strategies for improving confidence in the accuracy of western blots.</strong> <div class="csl-block">Rajeshwary Ghosh, Jennifer E Gilda, Aldrin V Gomes</div> <em>Expert review of proteomics</em> (2014-07-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25059473">https://www.ncbi.nlm.nih.gov/pubmed/25059473</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1586/14789450.2014.939635">10.1586/14789450.2014.939635</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25059473">25059473</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4791038">PMC4791038</a></div></div>
</div>
<div id="ref-xEkij3xI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">270. </div><div class="csl-right-inline"><strong>Mapping protein-protein interactions by affinity-directed mass spectrometry.</strong> <div class="csl-block">Y Zhao, TW Muir, SB Kent, E Tischer, JM Scardina, BT Chait</div> <em>Proceedings of the National Academy of Sciences of the United States of America</em> (1996-04-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8633009">https://www.ncbi.nlm.nih.gov/pubmed/8633009</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.93.9.4020">10.1073/pnas.93.9.4020</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8633009">8633009</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC39479">PMC39479</a></div></div>
</div>
<div id="ref-oN0RedlD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">271. </div><div class="csl-right-inline"><strong>Immunoaffinity profiling of tyrosine phosphorylation in cancer cells.</strong> <div class="csl-block">John Rush, Albrecht Moritz, Kimberly A Lee, Ailan Guo, Valerie L Goss, Erik J Spek, Hui Zhang, Xiang-Ming Zha, Roberto D Polakiewicz, Michael J Comb</div> <em>Nature biotechnology</em> (2004-12-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/15592455">https://www.ncbi.nlm.nih.gov/pubmed/15592455</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nbt1046">10.1038/nbt1046</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15592455">15592455</a></div></div>
</div>
<div id="ref-11yMtXtzL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">272. </div><div class="csl-right-inline"><strong>ATM and ATR substrate analysis reveals extensive protein networks responsive to DNA damage.</strong> <div class="csl-block">Shuhei Matsuoka, Bryan A Ballif, Agata Smogorzewska, ERobert McDonald, Kristen E Hurov, Ji Luo, Corey E Bakalarski, Zhenming Zhao, Nicole Solimini, Yaniv Lerenthal, … Stephen J Elledge</div> <em>Science (New York, N.Y.)</em> (2007-05-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17525332">https://www.ncbi.nlm.nih.gov/pubmed/17525332</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.1140321">10.1126/science.1140321</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17525332">17525332</a></div></div>
</div>
<div id="ref-nEmRnHBE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">273. </div><div class="csl-right-inline"><strong>Akt-RSK-S6 kinase signaling networks activated by oncogenic receptor tyrosine kinases.</strong> <div class="csl-block">Albrecht Moritz, Yu Li, Ailan Guo, Judit Villén, Yi Wang, Joan MacNeill, Jon Kornhauser, Kam Sprott, Jing Zhou, Anthony Possemato, … Michael J Comb</div> <em>Science signaling</em> (2010-08-24) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20736484">https://www.ncbi.nlm.nih.gov/pubmed/20736484</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/scisignal.2000998">10.1126/scisignal.2000998</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20736484">20736484</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3137639">PMC3137639</a></div></div>
</div>
<div id="ref-fZ6aM0wq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">274. </div><div class="csl-right-inline"><strong>Global identification of modular cullin-RING ligase substrates.</strong> <div class="csl-block">Michael J Emanuele, Andrew EH Elia, Qikai Xu, Claudio R Thoma, Lior Izhar, Yumei Leng, Ailan Guo, Yi-Ning Chen, John Rush, Paul Wei-Che Hsu, … Stephen J Elledge</div> <em>Cell</em> (2011-09-29) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21963094">https://www.ncbi.nlm.nih.gov/pubmed/21963094</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2011.09.019">10.1016/j.cell.2011.09.019</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21963094">21963094</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3226719">PMC3226719</a></div></div>
</div>
<div id="ref-JMsurOWF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">275. </div><div class="csl-right-inline"><strong>Systematic and quantitative assessment of the ubiquitin-modified proteome.</strong> <div class="csl-block">Woong Kim, Eric J Bennett, Edward L Huttlin, Ailan Guo, Jing Li, Anthony Possemato, Mathew E Sowa, Ramin Rad, John Rush, Michael J Comb, … Steven P Gygi</div> <em>Molecular cell</em> (2011-09-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21906983">https://www.ncbi.nlm.nih.gov/pubmed/21906983</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.molcel.2011.08.025">10.1016/j.molcel.2011.08.025</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21906983">21906983</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3200427">PMC3200427</a></div></div>
</div>
<div id="ref-11FwNutcB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">276. </div><div class="csl-right-inline"><strong>A proteome-wide, quantitative survey of in vivo ubiquitylation sites reveals widespread regulatory roles.</strong> <div class="csl-block">Sebastian A Wagner, Petra Beli, Brian T Weinert, Michael L Nielsen, Jürgen Cox, Matthias Mann, Chunaram Choudhary</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2011-09-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21890473">https://www.ncbi.nlm.nih.gov/pubmed/21890473</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m111.013284">10.1074/mcp.m111.013284</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21890473">21890473</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3205876">PMC3205876</a></div></div>
</div>
<div id="ref-tWUlvotu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">277. </div><div class="csl-right-inline"><strong>Lysine acetylation targets protein complexes and co-regulates major cellular functions.</strong> <div class="csl-block">Chunaram Choudhary, Chanchal Kumar, Florian Gnad, Michael L Nielsen, Michael Rehman, Tobias C Walther, Jesper V Olsen, Matthias Mann</div> <em>Science (New York, N.Y.)</em> (2009-07-16) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19608861">https://www.ncbi.nlm.nih.gov/pubmed/19608861</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.1175371">10.1126/science.1175371</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19608861">19608861</a></div></div>
</div>
<div id="ref-36Wx4WMF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">278. </div><div class="csl-right-inline"><strong>Calorie restriction and SIRT3 trigger global reprogramming of the mitochondrial protein acetylome.</strong> <div class="csl-block">Alexander S Hebert, Kristin E Dittenhafer-Reed, Wei Yu, Derek J Bailey, Ebru Selin Selen, Melissa D Boersma, Joshua J Carson, Marco Tonelli, Allison J Balloon, Alan J Higbee, … Joshua J Coon</div> <em>Molecular cell</em> (2012-11-29) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23201123">https://www.ncbi.nlm.nih.gov/pubmed/23201123</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.molcel.2012.10.024">10.1016/j.molcel.2012.10.024</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23201123">23201123</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3704155">PMC3704155</a></div></div>
</div>
<div id="ref-bAU4w5zL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">279. </div><div class="csl-right-inline"><strong>Deep, Quantitative Coverage of the Lysine Acetylome Using Novel Anti-acetyl-lysine Antibodies and an Optimized Proteomic Workflow.</strong> <div class="csl-block">Tanya Svinkina, Hongbo Gu, Jeffrey C Silva, Philipp Mertins, Jana Qiao, Shaunt Fereshetian, Jacob D Jaffe, Eric Kuhn, Namrata D Udeshi, Steven A Carr</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2015-05-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25953088">https://www.ncbi.nlm.nih.gov/pubmed/25953088</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.o114.047555">10.1074/mcp.o114.047555</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25953088">25953088</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4563726">PMC4563726</a></div></div>
</div>
<div id="ref-RMueHUu6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">280. </div><div class="csl-right-inline"><strong>High-Resolution Mass Spectrometry to Identify and Quantify Acetylation Protein Targets.</strong> <div class="csl-block">Birgit Schilling, Jesse G Meyer, Lei Wei, Melanie Ott, Eric Verdin</div> <em>Methods in molecular biology (Clifton, N.J.)</em> (2019) <a href="https://www.ncbi.nlm.nih.gov/pubmed/31087289">https://www.ncbi.nlm.nih.gov/pubmed/31087289</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-4939-9434-2_1">10.1007/978-1-4939-9434-2_1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31087289">31087289</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6825508">PMC6825508</a></div></div>
</div>
<div id="ref-58knZUn0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">281. </div><div class="csl-right-inline"><strong>Simultaneous Quantification of the Acetylome and Succinylome by ‘One‐Pot’ Affinity Enrichment</strong> <div class="csl-block">Nathan Basisty, Jesse G Meyer, Lei Wei, Bradford W Gibson, Birgit Schilling</div> <em>PROTEOMICS</em> (2018-08-19) <a href="https://doi.org/gn4cmb">https://doi.org/gn4cmb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.201800123">10.1002/pmic.201800123</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30035354">30035354</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6175148">PMC6175148</a></div></div>
</div>
<div id="ref-I7T7nB5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">282. </div><div class="csl-right-inline"><strong>Immunoaffinity enrichment and mass spectrometry analysis of protein methylation.</strong> <div class="csl-block">Ailan Guo, Hongbo Gu, Jing Zhou, Daniel Mulhern, Yi Wang, Kimberly A Lee, Vicky Yang, Mike Aguiar, Jon Kornhauser, Xiaoying Jia, … Michael J Comb</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2013-10-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24129315">https://www.ncbi.nlm.nih.gov/pubmed/24129315</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.o113.027870">10.1074/mcp.o113.027870</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24129315">24129315</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3879628">PMC3879628</a></div></div>
</div>
<div id="ref-82ISyiE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">283. </div><div class="csl-right-inline"><strong>Selective Affinity Enrichment of Nitrotyrosine-Containing Peptides for Quantitative Analysis in Complex Samples.</strong> <div class="csl-block">Yingxin Zhao, Yueqing Zhang, Hong Sun, Rosario Maroto, Allan R Brasier</div> <em>Journal of proteome research</em> (2017-07-17) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28714690">https://www.ncbi.nlm.nih.gov/pubmed/28714690</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.7b00275">10.1021/acs.jproteome.7b00275</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28714690">28714690</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5656043">PMC5656043</a></div></div>
</div>
<div id="ref-1CF6MyQQ3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">284. </div><div class="csl-right-inline"><strong>In-depth qualitative and quantitative profiling of tyrosine phosphorylation using a combination of phosphopeptide immunoaffinity purification and stable isotope dimethyl labeling.</strong> <div class="csl-block">Paul J Boersema, Leong Yan Foong, Vanessa MY Ding, Simone Lemeer, Bas van Breukelen, Robin Philp, Jos Boekhorst, Berend Snel, Jeroen den Hertog, Andre BH Choo, Albert JR Heck</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2009-09-21) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19770167">https://www.ncbi.nlm.nih.gov/pubmed/19770167</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m900291-mcp200">10.1074/mcp.m900291-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19770167">19770167</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2808269">PMC2808269</a></div></div>
</div>
<div id="ref-PgRmFpK6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">285. </div><div class="csl-right-inline"><strong>Time-resolved mass spectrometry of tyrosine phosphorylation sites in the epidermal growth factor receptor signaling network reveals dynamic modules.</strong> <div class="csl-block">Yi Zhang, Alejandro Wolf-Yadlin, Phillip L Ross, Darryl J Pappin, John Rush, Douglas A Lauffenburger, Forest M White</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2005-06-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/15951569">https://www.ncbi.nlm.nih.gov/pubmed/15951569</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m500089-mcp200">10.1074/mcp.m500089-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15951569">15951569</a></div></div>
</div>
<div id="ref-hAQepsrJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">286. </div><div class="csl-right-inline"><strong>O-GlcNAcylation: an important post-translational modification and a potential therapeutic target for cancer therapy.</strong> <div class="csl-block">Qingsong Lu, Xiaozhen Zhang, Tingbo Liang, Xueli Bai</div> <em>Molecular medicine (Cambridge, Mass.)</em> (2022-09-14) <a href="https://www.ncbi.nlm.nih.gov/pubmed/36104770">https://www.ncbi.nlm.nih.gov/pubmed/36104770</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/s10020-022-00544-y">10.1186/s10020-022-00544-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36104770">36104770</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9476278">PMC9476278</a></div></div>
</div>
<div id="ref-UV2xOP8u" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">287. </div><div class="csl-right-inline"><strong>Novel Antibodies for the Simple and Efficient Enrichment of Native O-GlcNAc Modified Peptides.</strong> <div class="csl-block">Rajan A Burt, Borislav Dejanovic, Hayley J Peckham, Kimberly A Lee, Xiang Li, Johain R Ounadjela, Anjana Rao, Stacy A Malaker, Steven A Carr, Samuel A Myers</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2021-10-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/34678516">https://www.ncbi.nlm.nih.gov/pubmed/34678516</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.mcpro.2021.100167">10.1016/j.mcpro.2021.100167</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34678516">34678516</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8605273">PMC8605273</a></div></div>
</div>
<div id="ref-xwhmqv6G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">288. </div><div class="csl-right-inline"><strong>Depletion of abundant plasma proteins and limitations of plasma proteomics.</strong> <div class="csl-block">Chengjian Tu, Paul A Rudnick, Misti Y Martinez, Kristin L Cheek, Stephen E Stein, Robbert JC Slebos, Daniel C Liebler</div> <em>Journal of proteome research</em> (2010-10-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20677825">https://www.ncbi.nlm.nih.gov/pubmed/20677825</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr100646w">10.1021/pr100646w</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20677825">20677825</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2948641">PMC2948641</a></div></div>
</div>
<div id="ref-MZe4do4v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">289. </div><div class="csl-right-inline"><strong>Differences among techniques for high-abundant protein depletion.</strong> <div class="csl-block">Nina Zolotarjova, James Martosella, Gordon Nicol, Jerome Bailey, Barry E Boyes, William C Barrett</div> <em>Proteomics</em> (2005-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16052628">https://www.ncbi.nlm.nih.gov/pubmed/16052628</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200402021">10.1002/pmic.200402021</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16052628">16052628</a></div></div>
</div>
<div id="ref-CkoUIvms" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">290. </div><div class="csl-right-inline"><strong>Dye-ligand affinity systems.</strong> <div class="csl-block">A Denizli, E Pişkin</div> <em>Journal of biochemical and biophysical methods</em> (2001-10-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/11694290">https://www.ncbi.nlm.nih.gov/pubmed/11694290</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0165-022x(01)00209-3">10.1016/s0165-022x(01)00209-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11694290">11694290</a></div></div>
</div>
<div id="ref-mxcAPQRD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">291. </div><div class="csl-right-inline"><strong>Cibacron blue immobilized poly(glycidyl-methacrylate) nanobeads for albumin removal in proteome studies.</strong> <div class="csl-block">Müge Andaç</div> <em>Artificial cells, nanomedicine, and biotechnology</em> (2013-10-29) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24168355">https://www.ncbi.nlm.nih.gov/pubmed/24168355</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3109/21691401.2013.852102">10.3109/21691401.2013.852102</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24168355">24168355</a></div></div>
</div>
<div id="ref-LonwNp1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">292. </div><div class="csl-right-inline"><strong>Protein G genes: structure and distribution of IgG-binding and albumin-binding domains.</strong> <div class="csl-block">U Sjöbring, L Björck, W Kastern</div> <em>Molecular microbiology</em> (1989-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/2473373">https://www.ncbi.nlm.nih.gov/pubmed/2473373</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/j.1365-2958.1989.tb00177.x">10.1111/j.1365-2958.1989.tb00177.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2473373">2473373</a></div></div>
</div>
<div id="ref-23zI8SGX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">293. </div><div class="csl-right-inline"><strong>Staphylococcal protein A consists of five IgG-binding domains.</strong> <div class="csl-block">T Moks, L Abrahmsén, B Nilsson, U Hellman, J Sjöquist, M Uhlén</div> <em>European journal of biochemistry</em> (1986-05-02) <a href="https://www.ncbi.nlm.nih.gov/pubmed/2938951">https://www.ncbi.nlm.nih.gov/pubmed/2938951</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/j.1432-1033.1986.tb09625.x">10.1111/j.1432-1033.1986.tb09625.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2938951">2938951</a></div></div>
</div>
<div id="ref-LRMcEp9I" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">294. </div><div class="csl-right-inline"><strong>Crystal structure of a Staphylococcus aureus protein A domain complexed with the Fab fragment of a human IgM antibody: structural basis for recognition of B-cell receptors and superantigen activity.</strong> <div class="csl-block">M Graille, EA Stura, AL Corper, BJ Sutton, MJ Taussig, JB Charbonnier, GJ Silverman</div> <em>Proceedings of the National Academy of Sciences of the United States of America</em> (2000-05-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10805799">https://www.ncbi.nlm.nih.gov/pubmed/10805799</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.97.10.5399">10.1073/pnas.97.10.5399</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10805799">10805799</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC25840">PMC25840</a></div></div>
</div>
<div id="ref-gxra2RsF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">295. </div><div class="csl-right-inline"><strong>Plasma/serum proteomics: depletion strategies for reducing high-abundance proteins for biomarker discovery.</strong> <div class="csl-block">Pey Yee Lee, Junaida Osman, Teck Yew Low, Rahman Jamal</div> <em>Bioanalysis</em> (2019-10-16) <a href="https://www.ncbi.nlm.nih.gov/pubmed/31617391">https://www.ncbi.nlm.nih.gov/pubmed/31617391</a> <div class="csl-block">DOI: <a href="https://doi.org/10.4155/bio-2019-0145">10.4155/bio-2019-0145</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31617391">31617391</a></div></div>
</div>
<div id="ref-13El9kECw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">296. </div><div class="csl-right-inline"><strong>Depletion of Highly Abundant Proteins of the Human Blood Plasma: Applications in Proteomics Studies of Psychiatric Disorders.</strong> <div class="csl-block">Sheila Garcia, Paulo A Baldasso, Paul C Guest, Daniel Martins-de-Souza</div> <em>Methods in molecular biology (Clifton, N.J.)</em> (2017) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27896769">https://www.ncbi.nlm.nih.gov/pubmed/27896769</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-4939-6730-8_16">10.1007/978-1-4939-6730-8_16</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27896769">27896769</a></div></div>
</div>
<div id="ref-xeb6grTF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">297. </div><div class="csl-right-inline"><strong>Reduction of dynamic protein concentration range of biological extracts for the discovery of low-abundance proteins by means of hexapeptide ligand library.</strong> <div class="csl-block">Luc Guerrier, Pier Giorgio Righetti, Egisto Boschetti</div> <em>Nature protocols</em> (2008) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18451796">https://www.ncbi.nlm.nih.gov/pubmed/18451796</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nprot.2008.59">10.1038/nprot.2008.59</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18451796">18451796</a></div></div>
</div>
<div id="ref-1AFK16JwO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">298. </div><div class="csl-right-inline"><strong>Sample treatment methods involving combinatorial peptide ligand libraries for improved proteomes analyses.</strong> <div class="csl-block">Pier Giorgio Righetti, Egisto Boschetti</div> <em>Methods in molecular biology (Clifton, N.J.)</em> (2015) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25384740">https://www.ncbi.nlm.nih.gov/pubmed/25384740</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-4939-1872-0_4">10.1007/978-1-4939-1872-0_4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25384740">25384740</a></div></div>
</div>
<div id="ref-hcs5DCXe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">299. </div><div class="csl-right-inline"><strong>Depletion of highly abundant proteins in blood plasma by ammonium sulfate precipitation for 2D-PAGE analysis.</strong> <div class="csl-block">Andrea Mahn, Maritza Ismail</div> <em>Journal of chromatography. B, Analytical technologies in the biomedical and life sciences</em> (2011-09-18) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21963274">https://www.ncbi.nlm.nih.gov/pubmed/21963274</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jchromb.2011.09.024">10.1016/j.jchromb.2011.09.024</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21963274">21963274</a></div></div>
</div>
<div id="ref-jOj0DZcZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">300. </div><div class="csl-right-inline"><strong>A novel and cost effective method of removing excess albumin from plasma/serum samples and its impacts on LC-MS/MS bioanalysis of therapeutic proteins.</strong> <div class="csl-block">Guowen Liu, Yue Zhao, Aida Angeles, Lora L Hamuro, Mark E Arnold, Jim X Shen</div> <em>Analytical chemistry</em> (2014-08-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25083595">https://www.ncbi.nlm.nih.gov/pubmed/25083595</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac501837t">10.1021/ac501837t</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25083595">25083595</a></div></div>
</div>
<div id="ref-1DUeuj8TB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">301. </div><div class="csl-right-inline"><strong>Enhanced Detection of Low-Abundance Human Plasma Proteins by Integrating Polyethylene Glycol Fractionation and Immunoaffinity Depletion.</strong> <div class="csl-block">Zhao Liu, Songhua Fan, Haipeng Liu, Jia Yu, Rui Qiao, Mi Zhou, Yongtao Yang, Jian Zhou, Peng Xie</div> <em>PloS one</em> (2016-11-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27832179">https://www.ncbi.nlm.nih.gov/pubmed/27832179</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pone.0166306">10.1371/journal.pone.0166306</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27832179">27832179</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5104378">PMC5104378</a></div></div>
</div>
<div id="ref-jv9v3twJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">302. </div><div class="csl-right-inline"><strong>Reducing agent-mediated precipitation of high-abundance plasma proteins.</strong> <div class="csl-block">Scott E Warder, Lora A Tucker, Tamara J Strelitzer, Evelyn M McKeegan, Joseph L Meuth, Paul M Jung, Anita Saraf, Bhawani Singh, Jie Lai-Zhang, Gerard Gagne, John C Rogers</div> <em>Analytical biochemistry</em> (2009-01-19) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19454248">https://www.ncbi.nlm.nih.gov/pubmed/19454248</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ab.2009.01.013">10.1016/j.ab.2009.01.013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19454248">19454248</a></div></div>
</div>
<div id="ref-a6kEMboa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">303. </div><div class="csl-right-inline"><strong>Protein depletion with bacterial cellulose nanofibers.</strong> <div class="csl-block">Ilgım Göktürk, Emel Tamahkar, Fatma Yılmaz, Adil Denizli</div> <em>Journal of chromatography. B, Analytical technologies in the biomedical and life sciences</em> (2018-08-29) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30219335">https://www.ncbi.nlm.nih.gov/pubmed/30219335</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jchromb.2018.08.030">10.1016/j.jchromb.2018.08.030</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30219335">30219335</a></div></div>
</div>
<div id="ref-WwW8brVC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">304. </div><div class="csl-right-inline"><strong>Supermacroporous Composite Cryogels in Biomedical Applications.</strong> <div class="csl-block">Yeşeren Saylan, Adil Denizli</div> <em>Gels (Basel, Switzerland)</em> (2019-04-17) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30999704">https://www.ncbi.nlm.nih.gov/pubmed/30999704</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/gels5020020">10.3390/gels5020020</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30999704">30999704</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6630583">PMC6630583</a></div></div>
</div>
<div id="ref-1KROnN78" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">305. </div><div class="csl-right-inline"><strong>Molecularly imprinted poly(hydroxyethyl methacrylate) based cryogel for albumin depletion from human serum.</strong> <div class="csl-block">Muge Andac, Igor Yu Galaev, Adil Denizli</div> <em>Colloids and surfaces. B, Biointerfaces</em> (2013-04-17) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23668981">https://www.ncbi.nlm.nih.gov/pubmed/23668981</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.colsurfb.2013.03.054">10.1016/j.colsurfb.2013.03.054</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23668981">23668981</a></div></div>
</div>
<div id="ref-7cVO0ON7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">306. </div><div class="csl-right-inline"><strong>Bacterial cellulose nanofibers for albumin depletion from human serum</strong> <div class="csl-block">Emel Tamahkar, Ceyhun Babaç, Tülin Kutsal, Erhan Pişkin, Adil Denizli</div> <em>Process Biochemistry</em> (2010-10) <a href="https://doi.org/bzqhtf">https://doi.org/bzqhtf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.procbio.2010.07.007">10.1016/j.procbio.2010.07.007</a></div></div>
</div>
<div id="ref-nCqm9thz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">307. </div><div class="csl-right-inline"><strong>Ultrasensitive proteomics using high-efficiency on-line micro-SPE-nanoLC-nanoESI MS and MS/MS.</strong> <div class="csl-block">Yufeng Shen, Nikola Tolić, Christophe Masselon, Ljiljana Pasa-Tolić, David G Camp, Kim K Hixson, Rui Zhao, Gordon A Anderson, Richard D Smith</div> <em>Analytical chemistry</em> (2004-01-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/14697044">https://www.ncbi.nlm.nih.gov/pubmed/14697044</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac030096q">10.1021/ac030096q</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14697044">14697044</a></div></div>
</div>
<div id="ref-167ADUQ9x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">308. </div><div class="csl-right-inline"><strong>Biomarkers discovery by peptide and protein profiling in biological fluids based on functionalized magnetic beads purification and mass spectrometry.</strong> <div class="csl-block">Fulvio Magni, Yuri EM Van Der Burgt, Clizia Chinello, Veronica Mainini, Erica Gianazza, Valeria Squeo, André M Deelder, Marzia Galli Kienle</div> <em>Blood transfusion = Trasfusione del sangue</em> (2010-06) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20606758">https://www.ncbi.nlm.nih.gov/pubmed/20606758</a> <div class="csl-block">DOI: <a href="https://doi.org/10.2450/2010.015s">10.2450/2010.015s</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20606758">20606758</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2897205">PMC2897205</a></div></div>
</div>
<div id="ref-klXQ49fD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">309. </div><div class="csl-right-inline"><strong>Magnetic particles as powerful purification tool for high sensitive mass spectrometric screening procedures.</strong> <div class="csl-block">Jochen F Peter, Angela M Otto</div> <em>Proteomics</em> (2010-02) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20099258">https://www.ncbi.nlm.nih.gov/pubmed/20099258</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200900535">10.1002/pmic.200900535</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20099258">20099258</a></div></div>
</div>
<div id="ref-18rJRCf42" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">310. </div><div class="csl-right-inline"><strong>Robust and high-throughput sample preparation for (semi-)quantitative analysis of N-glycosylation profiles from plasma samples.</strong> <div class="csl-block">LRenee Ruhaak, Carolin Huhn, Carolien AM Koeleman, André M Deelder, Manfred Wuhrer</div> <em>Methods in molecular biology (Clifton, N.J.)</em> (2012) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22665312">https://www.ncbi.nlm.nih.gov/pubmed/22665312</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-61779-885-6_23">10.1007/978-1-61779-885-6_23</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22665312">22665312</a></div></div>
</div>
<div id="ref-kPeMPVtT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">311. </div><div class="csl-right-inline"><strong>Utilizing ion-pairing hydrophilic interaction chromatography solid phase extraction for efficient glycopeptide enrichment in glycoproteomics.</strong> <div class="csl-block">Simon Mysling, Giuseppe Palmisano, Peter Højrup, Morten Thaysen-Andersen</div> <em>Analytical chemistry</em> (2010-07-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20536156">https://www.ncbi.nlm.nih.gov/pubmed/20536156</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac100530w">10.1021/ac100530w</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20536156">20536156</a></div></div>
</div>
<div id="ref-16XWFxoc3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">312. </div><div class="csl-right-inline"><strong>On-line 1D and 2D porous layer open tubular/LC-ESI-MS using 10-microm-i.d. poly(styrene-divinylbenzene) columns for ultrasensitive proteomic analysis.</strong> <div class="csl-block">Quanzhou Luo, Guihua Yue, Gary A Valaskovic, Ye Gu, Shiaw-Lin Wu, Barry L Karger</div> <em>Analytical chemistry</em> (2007-07-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17625912">https://www.ncbi.nlm.nih.gov/pubmed/17625912</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac070583w">10.1021/ac070583w</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17625912">17625912</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2570646">PMC2570646</a></div></div>
</div>
<div id="ref-us64YkZ0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">313. </div><div class="csl-right-inline"><strong>Development of a monolithic silica extraction tip for the analysis of proteins.</strong> <div class="csl-block">Shota Miyazaki, Kei Morisato, Norio Ishizuka, Hiroyoshi Minakuchi, Yukihiro Shintani, Masahiro Furuno, Kazuki Nakanishi</div> <em>Journal of chromatography. A</em> (2004-07-16) <a href="https://www.ncbi.nlm.nih.gov/pubmed/15317408">https://www.ncbi.nlm.nih.gov/pubmed/15317408</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.chroma.2004.03.025">10.1016/j.chroma.2004.03.025</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15317408">15317408</a></div></div>
</div>
<div id="ref-nPfahhRa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">314. </div><div class="csl-right-inline"><strong>Solid-phase extraction strategies to surmount body fluid sample complexity in high-throughput mass spectrometry-based proteomics.</strong> <div class="csl-block">Marco R Bladergroen, Yuri EM van der Burgt</div> <em>Journal of analytical methods in chemistry</em> (2015-01-27) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25692071">https://www.ncbi.nlm.nih.gov/pubmed/25692071</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1155/2015/250131">10.1155/2015/250131</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25692071">25692071</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4322654">PMC4322654</a></div></div>
</div>
<div id="ref-TY5C1ch7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">315. </div><div class="csl-right-inline"><strong>Stop and Go Extraction Tips for Matrix-Assisted Laser Desorption/Ionization, Nanoelectrospray, and LC/MS Sample Pretreatment in Proteomics</strong> <div class="csl-block">Juri Rappsilber, Yasushi Ishihama, Matthias Mann</div> <em>Analytical Chemistry</em> (2002-12-24) <a href="https://doi.org/frp8hj">https://doi.org/frp8hj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac026117i">10.1021/ac026117i</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12585499">12585499</a></div></div>
</div>
<div id="ref-j0ZcPOCn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">316. </div><div class="csl-right-inline"><strong>Minimal, encapsulated proteomic-sample processing applied to copy-number estimation in eukaryotic cells</strong> <div class="csl-block">Nils A Kulak, Garwin Pichler, Igor Paron, Nagarjuna Nagaraj, Matthias Mann</div> <em>Nature Methods</em> (2014-02-02) <a href="https://doi.org/f5vwds">https://doi.org/f5vwds</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.2834">10.1038/nmeth.2834</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24487582">24487582</a></div></div>
</div>
<div id="ref-SWsK5zPt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">317. </div><div class="csl-right-inline"><strong>Automated sample preparation with SP3 for low-input clinical proteomics.</strong> <div class="csl-block">Torsten Müller, Mathias Kalxdorf, Rémi Longuespée, Daniel N Kazdal, Albrecht Stenzinger, Jeroen Krijgsveld</div> <em>Molecular systems biology</em> (2020-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/32129943">https://www.ncbi.nlm.nih.gov/pubmed/32129943</a> <div class="csl-block">DOI: <a href="https://doi.org/10.15252/msb.20199111">10.15252/msb.20199111</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32129943">32129943</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6966100">PMC6966100</a></div></div>
</div>
<div id="ref-ncGFPcxC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">318. </div><div class="csl-right-inline"><strong>National continuous surveys on internal quality control for HbA1c in 306 clinical laboratories of China from 2012 to 2016: Continual improvement.</strong> <div class="csl-block">Tingting Li, Wei Wang, Haijian Zhao, Falin He, Kun Zhong, Shuai Yuan, Zhiguo Wang</div> <em>Journal of clinical laboratory analysis</em> (2016-11-21) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27868236">https://www.ncbi.nlm.nih.gov/pubmed/27868236</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/jcla.22099">10.1002/jcla.22099</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27868236">27868236</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6817230">PMC6817230</a></div></div>
</div>
<div id="ref-Bm9Z2DVx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">319. </div><div class="csl-right-inline"><strong>Abstracts of Presentations at the Association of Clinical Scientists 143</strong> <div class="csl-block">Annals of clinical and laboratory science</div> (2022-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/35777803">https://www.ncbi.nlm.nih.gov/pubmed/35777803</a> <div class="csl-block">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35777803">35777803</a></div></div>
</div>
<div id="ref-L1FaRtpm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">320. </div><div class="csl-right-inline"><strong>A comparison of immobilized pH gradient isoelectric focusing and strong-cation-exchange chromatography as a first dimension in shotgun proteomics.</strong> <div class="csl-block">Amal S Essader, Benjamin J Cargile, Jonathan L Bundy, James L Stephenson</div> <em>Proteomics</em> (2005-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/15672457">https://www.ncbi.nlm.nih.gov/pubmed/15672457</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200400888">10.1002/pmic.200400888</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15672457">15672457</a></div></div>
</div>
<div id="ref-GjvtcEz1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">321. </div><div class="csl-right-inline"><strong>Peptide fractionation in proteomics approaches.</strong> <div class="csl-block">Bruno Manadas, Vera M Mendes, Jane English, Michael J Dunn</div> <em>Expert review of proteomics</em> (2010-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20973639">https://www.ncbi.nlm.nih.gov/pubmed/20973639</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1586/epr.10.46">10.1586/epr.10.46</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20973639">20973639</a></div></div>
</div>
<div id="ref-1FrVJlffz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">322. </div><div class="csl-right-inline"><strong>The challenges of the analysis of basic compounds by high performance liquid chromatography: some possible approaches for improved separations.</strong> <div class="csl-block">David V McCalley</div> <em>Journal of chromatography. A</em> (2009-12-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20031138">https://www.ncbi.nlm.nih.gov/pubmed/20031138</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.chroma.2009.11.068">10.1016/j.chroma.2009.11.068</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20031138">20031138</a></div></div>
</div>
<div id="ref-1HPaIsist" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">323. </div><div class="csl-right-inline"><strong>Two-dimensional separation of peptides using RP-RP-HPLC system with different pH in first and second separation dimensions.</strong> <div class="csl-block">Martin Gilar, Petra Olivova, Amy E Daly, John C Gebler</div> <em>Journal of separation science</em> (2005-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16224963">https://www.ncbi.nlm.nih.gov/pubmed/16224963</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/jssc.200500116">10.1002/jssc.200500116</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16224963">16224963</a></div></div>
</div>
<div id="ref-190oCO8Ki" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">324. </div><div class="csl-right-inline"><strong>High-pH reversed-phase chromatography with fraction concatenation for 2D proteomic analysis.</strong> <div class="csl-block">Feng Yang, Yufeng Shen, David G Camp, Richard D Smith</div> <em>Expert review of proteomics</em> (2012-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22462785">https://www.ncbi.nlm.nih.gov/pubmed/22462785</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1586/epr.12.15">10.1586/epr.12.15</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22462785">22462785</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3337716">PMC3337716</a></div></div>
</div>
<div id="ref-16k9ga0WM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">325. </div><div class="csl-right-inline"><strong>Hydrophilic interaction liquid chromatography (HILIC)--a powerful separation technique.</strong> <div class="csl-block">Bogusław Buszewski, Sylwia Noga</div> <em>Analytical and bioanalytical chemistry</em> (2011-08-31) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21879300">https://www.ncbi.nlm.nih.gov/pubmed/21879300</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00216-011-5308-5">10.1007/s00216-011-5308-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21879300">21879300</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3249561">PMC3249561</a></div></div>
</div>
<div id="ref-nrDJzBh3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">326. </div><div class="csl-right-inline"><strong>Stationary phases for hydrophilic interaction chromatography, their characterization and implementation into multidimensional chromatography concepts.</strong> <div class="csl-block">Pavel Jandera</div> <em>Journal of separation science</em> (2008-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18428181">https://www.ncbi.nlm.nih.gov/pubmed/18428181</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/jssc.200800051">10.1002/jssc.200800051</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18428181">18428181</a></div></div>
</div>
<div id="ref-g9BMjmYA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">327. </div><div class="csl-right-inline"><strong>Hydrophilic interaction liquid chromatography (HILIC) in proteomics.</strong> <div class="csl-block">Paul J Boersema, Shabaz Mohammed, Albert JR Heck</div> <em>Analytical and bioanalytical chemistry</em> (2008-02-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18264818">https://www.ncbi.nlm.nih.gov/pubmed/18264818</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00216-008-1865-7">10.1007/s00216-008-1865-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18264818">18264818</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2324128">PMC2324128</a></div></div>
</div>
<div id="ref-KqYbI8ep" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">328. </div><div class="csl-right-inline"><strong>ERK1/2 regulates intracellular ATP levels through alpha-enolase expression in cardiomyocytes exposed to ischemic hypoxia and reoxygenation.</strong> <div class="csl-block">Yoichi Mizukami, Akihiro Iwamatsu, Toshihiko Aki, Masayasu Kimura, Kazuyuki Nakamura, Tomoko Nao, Tomoko Okusa, Masunori Matsuzaki, Ken-Ichi Yoshida, Sei Kobayashi</div> <em>The Journal of biological chemistry</em> (2004-09-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/15459207">https://www.ncbi.nlm.nih.gov/pubmed/15459207</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/jbc.m402299200">10.1074/jbc.m402299200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15459207">15459207</a></div></div>
</div>
<div id="ref-Nk7uUlzO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">329. </div><div class="csl-right-inline"><strong>Efficient fractionation and improved protein identification by peptide OFFGEL electrophoresis.</strong> <div class="csl-block">Patric Hörth, Christine A Miller, Tobias Preckel, Christian Wenz</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2006-07-14) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16849286">https://www.ncbi.nlm.nih.gov/pubmed/16849286</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.t600037-mcp200">10.1074/mcp.t600037-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16849286">16849286</a></div></div>
</div>
<div id="ref-Qf7HZAh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">330. </div><div class="csl-right-inline"><strong>Improved proteome coverage by using iTRAQ labelling and peptide OFFGEL fractionation.</strong> <div class="csl-block">Emilie Ernoult, Erick Gamelin, Catherine Guette</div> <em>Proteome science</em> (2008-10-13) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18851748">https://www.ncbi.nlm.nih.gov/pubmed/18851748</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/1477-5956-6-27">10.1186/1477-5956-6-27</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18851748">18851748</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2572582">PMC2572582</a></div></div>
</div>
<div id="ref-LSDxad5T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">331. </div><div class="csl-right-inline"><strong>Combination of peptide OFFGEL fractionation and label-free quantitation facilitated proteomics profiling of extraocular muscle.</strong> <div class="csl-block">Sven Fraterman, Ulrike Zeiger, Tejvir S Khurana, Neal A Rubinstein, Matthias Wilm</div> <em>Proteomics</em> (2007-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17708596">https://www.ncbi.nlm.nih.gov/pubmed/17708596</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200700382">10.1002/pmic.200700382</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17708596">17708596</a></div></div>
</div>
<div id="ref-Kf2BNqap" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">332. </div><div class="csl-right-inline"><strong>Electrostatic Repulsion Hydrophilic Interaction Chromatography for Isocratic Separation of Charged Solutes and Selective Isolation of Phosphopeptides</strong> <div class="csl-block">Andrew J Alpert</div> <em>Analytical Chemistry</em> (2007-11-21) <a href="https://doi.org/bt84c3">https://doi.org/bt84c3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac070997p">10.1021/ac070997p</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18027909">18027909</a></div></div>
</div>
<div id="ref-1HJnUznNp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">333. </div><div class="csl-right-inline"><strong>Novel Application of Electrostatic Repulsion-Hydrophilic Interaction Chromatography (ERLIC) in Shotgun Proteomics: Comprehensive Profiling of Rat Kidney Proteome</strong> <div class="csl-block">Piliang Hao, Tiannan Guo, Xin Li, Sunil S Adav, Jie Yang, Meng Wei, Siu Kwan Sze</div> <em>Journal of Proteome Research</em> (2010-06-10) <a href="https://doi.org/bgkxkx">https://doi.org/bgkxkx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr100037h">10.1021/pr100037h</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20450224">20450224</a></div></div>
</div>
<div id="ref-VhI3IC9H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">334. </div><div class="csl-right-inline"><strong>A Brief History of Mass Spectrometry</strong> <div class="csl-block">Jennifer Griffiths</div> <em>Analytical Chemistry</em> (2008-07-31) <a href="https://doi.org/btwqsg">https://doi.org/btwqsg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac8013065">10.1021/ac8013065</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18671338">18671338</a></div></div>
</div>
<div id="ref-lKt4oMBO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">335. </div><div class="csl-right-inline"><strong>Direct ionization methods in mass spectrometry: An overview</strong> <div class="csl-block">Christian W Klampfl, Markus Himmelsbach</div> <em>Analytica Chimica Acta</em> (2015-08) <a href="https://doi.org/f7qnxc">https://doi.org/f7qnxc</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.aca.2015.07.012">10.1016/j.aca.2015.07.012</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26347167">26347167</a></div></div>
</div>
<div id="ref-1B2HdTk7M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">336. </div><div class="csl-right-inline"><strong>Mass Spectrometry, Review of the Basics: Ionization</strong> <div class="csl-block">Hanan Awad, Mona M Khamis, Anas El-Aneed</div> <em>Applied Spectroscopy Reviews</em> (2014-09-25) <a href="https://doi.org/gh28k8">https://doi.org/gh28k8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1080/05704928.2014.954046">10.1080/05704928.2014.954046</a></div></div>
</div>
<div id="ref-18qirHBwQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">337. </div><div class="csl-right-inline"><strong>Towards Higher Sensitivity of Mass Spectrometry: A Perspective From the Mass Analyzers</strong> <div class="csl-block">Chang Li, Shiying Chu, Siyuan Tan, Xinchi Yin, You Jiang, Xinhua Dai, Xiaoyun Gong, Xiang Fang, Di Tian</div> <em>Frontiers in Chemistry</em> (2021-12-21) <a href="https://doi.org/gr7v7s">https://doi.org/gr7v7s</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3389/fchem.2021.813359">10.3389/fchem.2021.813359</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34993180">34993180</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8724130">PMC8724130</a></div></div>
</div>
<div id="ref-5vv7ID3o" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">338. </div><div class="csl-right-inline"><strong>Evolution of Orbitrap Mass Spectrometry Instrumentation</strong> <div class="csl-block">Shannon Eliuk, Alexander Makarov</div> <em>Annual Review of Analytical Chemistry</em> (2015-07-22) <a href="https://doi.org/ggn4nz">https://doi.org/ggn4nz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1146/annurev-anchem-071114-040325">10.1146/annurev-anchem-071114-040325</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26161972">26161972</a></div></div>
</div>
<div id="ref-MooVAdBf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">339. </div><div class="csl-right-inline"><strong>Hybrid mass spectrometers for tandem mass spectrometry</strong> <div class="csl-block">Gary L Glish, David J Burinsky</div> <em>Journal of the American Society for Mass Spectrometry</em> (2008-02-01) <a href="https://doi.org/cd66p8">https://doi.org/cd66p8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jasms.2007.11.013">10.1016/j.jasms.2007.11.013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18187337">18187337</a></div></div>
</div>
<div id="ref-Uaum1m63" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">340. </div><div class="csl-right-inline"><strong>Fundamentals and Advances of Orbitrap Mass Spectrometry</strong> <div class="csl-block">Elizabeth S Hecht, Michaela Scigelova, Shannon Eliuk, Alexander Makarov</div> <em>Encyclopedia of Analytical Chemistry</em> (2019-09-16) <a href="https://doi.org/gssrqd">https://doi.org/gssrqd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/9780470027318.a9309.pub2">10.1002/9780470027318.a9309.pub2</a></div></div>
</div>
<div id="ref-svXqfJ4D" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">341. </div><div class="csl-right-inline"><strong>Orbitrap Mass Spectrometry</strong> <div class="csl-block">Roman A Zubarev, Alexander Makarov</div> <em>Analytical Chemistry</em> (2013-05-13) <a href="https://doi.org/f24tv5">https://doi.org/f24tv5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac4001223">10.1021/ac4001223</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23590404">23590404</a></div></div>
</div>
<div id="ref-erGVtl2k" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">342. </div><div class="csl-right-inline"><strong>Advanced fundamentals in Fourier transform mass spectrometry</strong> <div class="csl-block">Yury O Tsybin, Konstantin O Nagornov, Anton N Kozhinov</div> <em>Fundamentals and Applications of Fourier Transform Mass Spectrometry</em> (2019) <a href="https://doi.org/gssrqg">https://doi.org/gssrqg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/b978-0-12-814013-0.00005-3">10.1016/b978-0-12-814013-0.00005-3</a></div></div>
</div>
<div id="ref-EmIJDW71" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">343. </div><div class="csl-right-inline"><strong>Collision-induced dissociation (CID) of peptides and proteins.</strong> <div class="csl-block">JMitchell Wells, Scott A McLuckey</div> <em>Methods in enzymology</em> (2005) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16401509">https://www.ncbi.nlm.nih.gov/pubmed/16401509</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0076-6879(05)02005-7">10.1016/s0076-6879(05)02005-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16401509">16401509</a></div></div>
</div>
<div id="ref-17hxbWgXF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">344. </div><div class="csl-right-inline"><strong>Improved Collisionally Activated Dissociation Efficiency and Mass Resolution on a Triple Quadrupole Mass Spectrometer System</strong> <div class="csl-block">Bruce A Thomson, DJ Douglas, Jay J Corr, James W Hager, Charles L Jolliffe</div> <em>Analytical Chemistry</em> (1995-05-15) <a href="https://doi.org/cmhbkv">https://doi.org/cmhbkv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac00106a008">10.1021/ac00106a008</a></div></div>
</div>
<div id="ref-1ENeUykUA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">345. </div><div class="csl-right-inline"><strong>Tandem-in-space and tandem-in-time mass spectrometry: triple quadrupoles and quadrupole ion traps</strong> <div class="csl-block">Jodie V Johnson, Richard A Yost, Paul E Kelley, Donald C Bradford</div> <em>Analytical Chemistry</em> (1990-10-15) <a href="https://doi.org/cp87w5">https://doi.org/cp87w5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac00219a003">10.1021/ac00219a003</a></div></div>
</div>
<div id="ref-1GjxHekcM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">346. </div><div class="csl-right-inline"><strong>Ion activation methods for tandem mass spectrometry</strong> <div class="csl-block">Lekha Sleno, Dietrich A Volmer</div> <em>Journal of Mass Spectrometry</em> (2004) <a href="https://doi.org/fv3vrk">https://doi.org/fv3vrk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/jms.703">10.1002/jms.703</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15481084">15481084</a></div></div>
</div>
<div id="ref-1B9P7OPqR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">347. </div><div class="csl-right-inline"><strong>Quantitative Mass Spectrometric Multiple Reaction Monitoring Assays for Major Plasma Proteins</strong> <div class="csl-block">Leigh Anderson, Christie L Hunter</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2006-04) <a href="https://doi.org/c8f5g7">https://doi.org/c8f5g7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m500331-mcp200">10.1074/mcp.m500331-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16332733">16332733</a></div></div>
</div>
<div id="ref-9yFTjbpI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">348. </div><div class="csl-right-inline"><strong>Recent trends in mass spectrometer development</strong> <div class="csl-block">James W Hager</div> <em>Analytical and Bioanalytical Chemistry</em> (2003-11-04) <a href="https://doi.org/dt2dkm">https://doi.org/dt2dkm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00216-003-2287-1">10.1007/s00216-003-2287-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14598011">14598011</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7079868">PMC7079868</a></div></div>
</div>
<div id="ref-Ddpn8u0F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">349. </div><div class="csl-right-inline"><strong>Selected ion fragmentation with a tandem quadrupole mass spectrometer</strong> <div class="csl-block">RA Yost, CG Enke</div> <em>Journal of the American Chemical Society</em> (1978-03) <a href="https://doi.org/cm6pp4">https://doi.org/cm6pp4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ja00475a072">10.1021/ja00475a072</a></div></div>
</div>
<div id="ref-1AOQTDs57" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">350. </div><div class="csl-right-inline"><strong>Triple quadrupole linear ion trap mass spectrometer for the analysis of small molecules and macromolecules</strong> <div class="csl-block">Gérard Hopfgartner, Emmanuel Varesio, Viviane Tschäppät, Chantal Grivet, Emmanuel Bourgogne, Luc Alexis Leuthold</div> <em>Journal of Mass Spectrometry</em> (2004-08) <a href="https://doi.org/fq2dw3">https://doi.org/fq2dw3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/jms.659">10.1002/jms.659</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15329837">15329837</a></div></div>
</div>
<div id="ref-11skpaXK6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">351. </div><div class="csl-right-inline"><strong>Recent advances in applications of liquid chromatography–tandem mass spectrometry to the analysis of reactive drug metabolites</strong> <div class="csl-block">Shuguang Ma, Mingshe Zhu</div> <em>Chemico-Biological Interactions</em> (2009-04) <a href="https://doi.org/bwf9sp">https://doi.org/bwf9sp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cbi.2008.09.014">10.1016/j.cbi.2008.09.014</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18848531">18848531</a></div></div>
</div>
<div id="ref-BaGpN1HP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">352. </div><div class="csl-right-inline"><strong>The role of GC-MS/MS with triple quadrupole in pesticide residue analysis in food and the environment</strong> <div class="csl-block">F Hernández, MI Cervera, T Portolés, J Beltrán, E Pitarch</div> <em>Analytical Methods</em> (2013) <a href="https://doi.org/gssrqx">https://doi.org/gssrqx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/c3ay41104d">10.1039/c3ay41104d</a></div></div>
</div>
<div id="ref-1ETD5TZZ8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">353. </div><div class="csl-right-inline"><strong>Tandem mass spectrometry in the clinical laboratory: A tutorial overview</strong> <div class="csl-block">JGrace van der Gugten</div> <em>Clinical Mass Spectrometry</em> (2020-01) <a href="https://doi.org/gssrqh">https://doi.org/gssrqh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.clinms.2019.09.002">10.1016/j.clinms.2019.09.002</a></div></div>
</div>
<div id="ref-EeH1Mu0v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">354. </div><div class="csl-right-inline"><strong>Proceedings of the American Physical Society</strong> <div class="csl-block">Anonymous</div> <em>Physical Review</em> (1946-06-01) <a href="https://doi.org/ddzhj3">https://doi.org/ddzhj3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1103/physrev.69.674.2">10.1103/physrev.69.674.2</a></div></div>
</div>
<div id="ref-kFLq1VDE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">355. </div><div class="csl-right-inline"><strong>A Pulsed Mass Spectrometer with Time Dispersion</strong> <div class="csl-block">MM Wolff, WE Stephens</div> <em>Review of Scientific Instruments</em> (1953-08-01) <a href="https://doi.org/bxz73r">https://doi.org/bxz73r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1063/1.1770801">10.1063/1.1770801</a></div></div>
</div>
<div id="ref-15YzDfo72" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">356. </div><div class="csl-right-inline"><strong>Time-of-Flight Mass Spectrometer with Improved Resolution</strong> <div class="csl-block">WC Wiley, IH McLaren</div> <em>Review of Scientific Instruments</em> (1955-12-01) <a href="https://doi.org/dxcbgx">https://doi.org/dxcbgx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1063/1.1715212">10.1063/1.1715212</a></div></div>
</div>
<div id="ref-eJibibTn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">357. </div><div class="csl-right-inline"><strong>Simplified high-throughput methods for deep proteome analysis on the timsTOF Pro</strong> <div class="csl-block">Jarrod J Sandow, Giuseppe Infusini, Laura F Dagley, Rune Larsen, Andrew I Webb</div> <em>Cold Spring Harbor Laboratory</em> (2019-06-03) <a href="https://doi.org/ggr98j">https://doi.org/ggr98j</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/657908">10.1101/657908</a></div></div>
</div>
<div id="ref-KJYZ12ho" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">358. </div><div class="csl-right-inline"><strong>Ultrafast and Reproducible Proteomics from Small Amounts of Heart Tissue Enabled by Azo and timsTOF Pro</strong> <div class="csl-block">Timothy J Aballo, David S Roberts, Jake A Melby, Kevin M Buck, Kyle A Brown, Ying Ge</div> <em>Journal of Proteome Research</em> (2021-07-08) <a href="https://doi.org/gmqvmf">https://doi.org/gmqvmf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.1c00446">10.1021/acs.jproteome.1c00446</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34236868">34236868</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8349881">PMC8349881</a></div></div>
</div>
<div id="ref-cXsqAa1Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">359. </div><div class="csl-right-inline"><strong>High Throughput Proteome Screening for Biomarker Detection</strong> <div class="csl-block">Sheng Pan, Hui Zhang, John Rush, Jimmy Eng, Ning Zhang, Dale Patterson, Michael J Comb, Ruedi Aebersold</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2005-02) <a href="https://doi.org/brcr5t">https://doi.org/brcr5t</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m400161-mcp200">10.1074/mcp.m400161-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15637048">15637048</a></div></div>
</div>
<div id="ref-OeUnezUK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">360. </div><div class="csl-right-inline"><strong>Mass Spectrometry-Based Structural Virology</strong> <div class="csl-block">Tobias P Wörner, Tatiana M Shamorkina, Joost Snijder, Albert JR Heck</div> <em>Analytical Chemistry</em> (2020-12-04) <a href="https://doi.org/gssrqn">https://doi.org/gssrqn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.0c04339">10.1021/acs.analchem.0c04339</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33275424">33275424</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7807421">PMC7807421</a></div></div>
</div>
<div id="ref-2oFwzqFP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">361. </div><div class="csl-right-inline"><strong>Analytical Approaches for Size and Mass Analysis of Large Protein Assemblies</strong> <div class="csl-block">Joost Snijder, Albert JR Heck</div> <em>Annual Review of Analytical Chemistry</em> (2014-06-12) <a href="https://doi.org/gssrqz">https://doi.org/gssrqz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1146/annurev-anchem-071213-020015">10.1146/annurev-anchem-071213-020015</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25014341">25014341</a></div></div>
</div>
<div id="ref-3O301V9H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">362. </div><div class="csl-right-inline"><strong>The emergence of mass spectrometry for characterizing nanomaterials. Atomically precise nanoclusters and beyond</strong> <div class="csl-block">Clothilde Comby-Zerbino, Xavier Dagany, Fabien Chirot, Philippe Dugourd, Rodolphe Antoine</div> <em>Materials Advances</em> (2021) <a href="https://doi.org/gsrjtb">https://doi.org/gsrjtb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/d1ma00261a">10.1039/d1ma00261a</a></div></div>
</div>
<div id="ref-1CdqYFQjZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">363. </div><div class="csl-right-inline"><strong>Electrostatic Axially Harmonic Orbital Trapping:  A High-Performance Technique of Mass Analysis</strong> <div class="csl-block">Alexander Makarov</div> <em>Analytical Chemistry</em> (2000-02-10) <a href="https://doi.org/cs45nz">https://doi.org/cs45nz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac991131p">10.1021/ac991131p</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10740853">10740853</a></div></div>
</div>
<div id="ref-1GyycMiIQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">364. </div><div class="csl-right-inline"><strong>Coupling liquid chromatography to Orbitrap mass spectrometry</strong> <div class="csl-block">Alexander Makarov, Michaela Scigelova</div> <em>Journal of Chromatography A</em> (2010-06) <a href="https://doi.org/ff4bjw">https://doi.org/ff4bjw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.chroma.2010.02.022">10.1016/j.chroma.2010.02.022</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20299023">20299023</a></div></div>
</div>
<div id="ref-KRgCNqM9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">365. </div><div class="csl-right-inline"><strong>Advances in mass spectrometry-based clinical biomarker discovery</strong> <div class="csl-block">Christopher A Crutchfield, Stefani N Thomas, Lori J Sokoll, Daniel W Chan</div> <em>Clinical Proteomics</em> (2016-01-07) <a href="https://doi.org/gh624w">https://doi.org/gh624w</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/s12014-015-9102-9">10.1186/s12014-015-9102-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26751220">26751220</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4705754">PMC4705754</a></div></div>
</div>
<div id="ref-184JFw6LL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">366. </div><div class="csl-right-inline"><strong>PIQED: automated identification and quantification of protein modifications from DIA-MS data</strong> <div class="csl-block">Jesse G Meyer, Sushanth Mukkamalla, Hanno Steen, Alexey I Nesvizhskii, Bradford W Gibson, Birgit Schilling</div> <em>Nature Methods</em> (2017-07) <a href="https://doi.org/gssrqv">https://doi.org/gssrqv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.4334">10.1038/nmeth.4334</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28661500">28661500</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5663633">PMC5663633</a></div></div>
</div>
<div id="ref-DqzDdsRo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">367. </div><div class="csl-right-inline"><strong>Label-Free Quantification from Direct Infusion Shotgun Proteome Analysis (DISPA-LFQ) with CsoDIAq Software</strong> <div class="csl-block">Yuming Jiang, Alexandre Hutton, Caleb W Cranney, Jesse G Meyer</div> <em>Analytical Chemistry</em> (2022-12-17) <a href="https://doi.org/gssrqq">https://doi.org/gssrqq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.2c02249">10.1021/acs.analchem.2c02249</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36527718">36527718</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9850400">PMC9850400</a></div></div>
</div>
<div id="ref-4x97ohIq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">368. </div><div class="csl-right-inline"><strong>A Complete Workflow for High Throughput Human Single Skeletal Muscle Fiber Proteomics</strong> <div class="csl-block">Amanda Momenzadeh, Yuming Jiang, Simion Kreimer, Laura E Teigen, Carlos S Zepeda, Ali Haghani, Mitra Mastali, Yang Song, Alexandre Hutton, Sarah J Parker, … Jesse G Meyer</div> <em>Journal of the American Society for Mass Spectrometry</em> (2023-07-18) <a href="https://doi.org/gssrqt">https://doi.org/gssrqt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/jasms.3c00072">10.1021/jasms.3c00072</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37463334">37463334</a></div></div>
</div>
<div id="ref-pRTFSroJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">369. </div><div class="csl-right-inline"><strong>Quantitative shotgun proteome analysis by direct infusion</strong> <div class="csl-block">Jesse G Meyer, Natalie M Niemi, David J Pagliarini, Joshua J Coon</div> <em>Nature Methods</em> (2020-11-23) <a href="https://doi.org/ghnrz4">https://doi.org/ghnrz4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-020-00999-z">10.1038/s41592-020-00999-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33230323">33230323</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8009190">PMC8009190</a></div></div>
</div>
<div id="ref-1B5HMmjdK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">370. </div><div class="csl-right-inline"><strong>Characterizing Protein–Protein Interactions Using Mass Spectrometry: Challenges and Opportunities</strong> <div class="csl-block">Arne H Smits, Michiel Vermeulen</div> <em>Trends in Biotechnology</em> (2016-10) <a href="https://doi.org/f863x9">https://doi.org/f863x9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.tibtech.2016.02.014">10.1016/j.tibtech.2016.02.014</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26996615">26996615</a></div></div>
</div>
<div id="ref-BtlT8wee" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">371. </div><div class="csl-right-inline"><strong>Nanoparticles and photochemistry for native-like transmembrane protein footprinting</strong> <div class="csl-block">Jie Sun, Xiaoran Roger Liu, Shuang Li, Peng He, Weikai Li, Michael L Gross</div> <em>Nature Communications</em> (2021-12-14) <a href="https://doi.org/gssrqw">https://doi.org/gssrqw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-021-27588-8">10.1038/s41467-021-27588-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34907205">34907205</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8671412">PMC8671412</a></div></div>
</div>
<div id="ref-1GNpqhnTj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">372. </div><div class="csl-right-inline"><strong>Carbocation Footprinting of Soluble and Transmembrane Proteins</strong> <div class="csl-block">Jie Sun, Shuang Li, Weikai Li, Michael L Gross</div> <em>Analytical Chemistry</em> (2021-09-24) <a href="https://doi.org/gssrqp">https://doi.org/gssrqp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.1c03274">10.1021/acs.analchem.1c03274</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34558889">34558889</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8900486">PMC8900486</a></div></div>
</div>
<div id="ref-wknKHPF0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">373. </div><div class="csl-right-inline"><strong>Fourier transform ion cyclotron resonance spectroscopy</strong> <div class="csl-block">Melvin B Comisarow, Alan G Marshall</div> <em>Chemical Physics Letters</em> (1974-03-15) <a href="https://www.sciencedirect.com/science/article/pii/0009261474891372">https://www.sciencedirect.com/science/article/pii/0009261474891372</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/0009-2614(74)89137-2">10.1016/0009-2614(74)89137-2</a></div></div>
</div>
<div id="ref-HLEPjTl5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">374. </div><div class="csl-right-inline"><strong>Developments in FT‐ICR MS instrumentation, ionization techniques, and data interpretation methods for petroleomics</strong> <div class="csl-block">Yunju Cho, Arif Ahmed, Annana Islam, Sunghwan Kim</div> <em>Mass Spectrometry Reviews</em> (2014-06-18) <a href="https://doi.org/f6zb44">https://doi.org/f6zb44</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.21438">10.1002/mas.21438</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24942384">24942384</a></div></div>
</div>
<div id="ref-64lSPaQQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">375. </div><div class="csl-right-inline"><strong>Molecular composition of urban organic aerosols on clear and hazy days in Beijing: a comparative study using FT-ICR MS</strong> <div class="csl-block">Bin Jiang, Bin Yu Kuang, Yongmei Liang, Jingyi Zhang, XHHilda Huang, Chunming Xu, Jian Zhen Yu, Quan Shi</div> <em>Environmental Chemistry</em> (2016) <a href="https://doi.org/f8523d">https://doi.org/f8523d</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1071/en15230">10.1071/en15230</a></div></div>
</div>
<div id="ref-Gw2v7q21" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">376. </div><div class="csl-right-inline"><strong>Proteomics by FTICR mass spectrometry: Top down and bottom up</strong> <div class="csl-block">Bogdan Bogdanov, Richard D Smith</div> <em>Mass Spectrometry Reviews</em> (2005) <a href="https://doi.org/c2nx8k">https://doi.org/c2nx8k</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.20015">10.1002/mas.20015</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15389855">15389855</a></div></div>
</div>
<div id="ref-2Qw9ppKa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">377. </div><div class="csl-right-inline"><strong>Multidimensional LC Separations in Shotgun Proteomics</strong> <div class="csl-block">Akira Motoyama, John R Yates III</div> <em>Analytical Chemistry</em> (2008-09-30) <a href="https://doi.org/d3ngn8">https://doi.org/d3ngn8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac8013669">10.1021/ac8013669</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18826178">18826178</a></div></div>
</div>
<div id="ref-ovFVXSFD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">378. </div><div class="csl-right-inline"><strong>Application of LC/MS to proteomics studies: current status and future prospects</strong> <div class="csl-block">Guodong Chen, Birendra N Pramanik</div> <em>Drug Discovery Today</em> (2009-05) <a href="https://doi.org/dj8wq2">https://doi.org/dj8wq2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.drudis.2009.02.007">10.1016/j.drudis.2009.02.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19429505">19429505</a></div></div>
</div>
<div id="ref-ODpFT5rK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">379. </div><div class="csl-right-inline"><strong>Ion Mobility-Mass Spectrometry: Time-Dispersive Instrumentation</strong> <div class="csl-block">Jody C May, John A McLean</div> <em>Analytical Chemistry</em> (2015-01-09) <a href="https://doi.org/gcbv8t">https://doi.org/gcbv8t</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac504720m">10.1021/ac504720m</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25526595">25526595</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4318620">PMC4318620</a></div></div>
</div>
<div id="ref-rfF30WBX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">380. </div><div class="csl-right-inline"><strong>Ion Mobility Spectrometry: Fundamental Concepts, Instrumentation, Applications, and the Road Ahead</strong> <div class="csl-block">James N Dodds, Erin S Baker</div> <em>Journal of the American Society for Mass Spectrometry</em> (2019-09-06) <a href="https://doi.org/ggfrf7">https://doi.org/ggfrf7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s13361-019-02288-2">10.1007/s13361-019-02288-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31493234">31493234</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6832852">PMC6832852</a></div></div>
</div>
<div id="ref-D1WlSrbO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">381. </div><div class="csl-right-inline"><strong>Parallel Accumulation–Serial Fragmentation (PASEF): Multiplying Sequencing Speed and Sensitivity by Synchronized Scans in a Trapped Ion Mobility Device</strong> <div class="csl-block">Florian Meier, Scarlet Beck, Niklas Grassl, Markus Lubeck, Melvin A Park, Oliver Raether, Matthias Mann</div> <em>Journal of Proteome Research</em> (2015-11-13) <a href="https://doi.org/f7289w">https://doi.org/f7289w</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.5b00932">10.1021/acs.jproteome.5b00932</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26538118">26538118</a></div></div>
</div>
<div id="ref-2aWp1Ns2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">382. </div><div class="csl-right-inline"><strong>Trends in trapped ion mobility – Mass spectrometry instrumentation</strong> <div class="csl-block">Mark E Ridgeway, Christian Bleiholder, Matthias Mann, Melvin A Park</div> <em>TrAC Trends in Analytical Chemistry</em> (2019-07) <a href="https://doi.org/gssrqk">https://doi.org/gssrqk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.trac.2019.03.030">10.1016/j.trac.2019.03.030</a></div></div>
</div>
<div id="ref-Gl9lNiar" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">383. </div><div class="csl-right-inline"><strong>A Cyclic Ion Mobility-Mass Spectrometry System</strong> <div class="csl-block">Kevin Giles, Jakub Ujma, Jason Wildgoose, Steven Pringle, Keith Richardson, David Langridge, Martin Green</div> <em>Analytical Chemistry</em> (2019-05-29) <a href="https://doi.org/gssrqs">https://doi.org/gssrqs</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.9b01838">10.1021/acs.analchem.9b01838</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31141659">31141659</a></div></div>
</div>
<div id="ref-NcqTCZMl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">384. </div><div class="csl-right-inline"><strong>Utilizing Drift Tube Ion Mobility Spectrometry for the Evaluation of Metabolites and Xenobiotics</strong> <div class="csl-block">Melanie T Odenkirk, Erin S Baker</div> <em>Methods in Molecular Biology</em> (2019-11-16) <a href="https://doi.org/gssrqf">https://doi.org/gssrqf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-0716-0030-6_2">10.1007/978-1-0716-0030-6_2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31729652">31729652</a></div></div>
</div>
<div id="ref-6pMoYbxP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">385. </div><div class="csl-right-inline"><strong>Ion Mobility–Mass Spectrometry</strong> <div class="csl-block">Wentao Jiang, Nadjali A Chung, Jody C May, John A McLean, Renã AS Robinson</div> <em>Encyclopedia of Analytical Chemistry</em> (2019-06-13) <a href="https://doi.org/gn2fcc">https://doi.org/gn2fcc</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/9780470027318.a9292.pub2">10.1002/9780470027318.a9292.pub2</a></div></div>
</div>
<div id="ref-KCpIS4fP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">386. </div><div class="csl-right-inline"><strong>High-field asymmetric waveform ion mobility spectrometry for mass spectrometry-based proteomics</strong> <div class="csl-block">Kristian E Swearingen, Robert L Moritz</div> <em>Expert Review of Proteomics</em> (2012-10) <a href="https://doi.org/f4hx6v">https://doi.org/f4hx6v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1586/epr.12.50">10.1586/epr.12.50</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23194268">23194268</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4777519">PMC4777519</a></div></div>
</div>
<div id="ref-VdxvCWch" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">387. </div><div class="csl-right-inline"><strong>High-Field Asymmetric Waveform Ion Mobility Spectrometry and Native Mass Spectrometry: Analysis of Intact Protein Assemblies and Protein Complexes</strong> <div class="csl-block">Oliver J Hale, Eva Illes-Toth, Todd H Mize, Helen J Cooper</div> <em>Analytical Chemistry</em> (2020-04-28) <a href="https://doi.org/gssrqm">https://doi.org/gssrqm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.0c00649">10.1021/acs.analchem.0c00649</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32343119">32343119</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7304667">PMC7304667</a></div></div>
</div>
<div id="ref-E9GVLceZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">388. </div><div class="csl-right-inline"><strong>Comprehensive Single-Shot Proteomics with FAIMS on a Hybrid Orbitrap Mass Spectrometer</strong> <div class="csl-block">Alexander S Hebert, Satendra Prasad, Michael W Belford, Derek J Bailey, Graeme C McAlister, Susan E Abbatiello, Romain Huguet, Eloy R Wouters, Jean-Jacques Dunyach, Dain R Brademan, … Joshua J Coon</div> <em>Analytical Chemistry</em> (2018-07-03) <a href="https://doi.org/gd4chm">https://doi.org/gd4chm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.8b02233">10.1021/acs.analchem.8b02233</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29969236">29969236</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6145172">PMC6145172</a></div></div>
</div>
<div id="ref-EUzlbRbi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">389. </div><div class="csl-right-inline"><strong>High-field asymmetric waveform ion mobility spectrometry: A new tool for mass spectrometry</strong> <div class="csl-block">R GUEVREMONT</div> <em>Journal of Chromatography A</em> (2004-11-26) <a href="https://doi.org/dsdp9m">https://doi.org/dsdp9m</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.chroma.2004.08.119">10.1016/j.chroma.2004.08.119</a></div></div>
</div>
<div id="ref-CzOaytxB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">390. </div><div class="csl-right-inline"><strong>Trapped ion mobility spectrometry: A short review</strong> <div class="csl-block">Mark E Ridgeway, Markus Lubeck, Jan Jordens, Mattias Mann, Melvin A Park</div> <em>International Journal of Mass Spectrometry</em> (2018-02) <a href="https://doi.org/gc5nw5">https://doi.org/gc5nw5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ijms.2018.01.006">10.1016/j.ijms.2018.01.006</a></div></div>
</div>
<div id="ref-1FmNlvpSt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">391. </div><div class="csl-right-inline"><strong>Trapped Ion Mobility Spectrometry and Parallel Accumulation–Serial Fragmentation in Proteomics</strong> <div class="csl-block">Florian Meier, Melvin A Park, Matthias Mann</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2021) <a href="https://doi.org/gssrqj">https://doi.org/gssrqj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.mcpro.2021.100138">10.1016/j.mcpro.2021.100138</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34416385">34416385</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8453224">PMC8453224</a></div></div>
</div>
<div id="ref-KgcabL7g" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">392. </div><div class="csl-right-inline"><strong>Characterization of Ion Dynamics in Structures for Lossless Ion Manipulations</strong> <div class="csl-block">Aleksey V Tolmachev, Ian K Webb, Yehia M Ibrahim, Sandilya VB Garimella, Xinyu Zhang, Gordon A Anderson, Richard D Smith</div> <em>Analytical Chemistry</em> (2014-09-04) <a href="https://doi.org/f6jr4m">https://doi.org/f6jr4m</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac502054p">10.1021/ac502054p</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25152178">25152178</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4175726">PMC4175726</a></div></div>
</div>
<div id="ref-XOhnpKc9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">393. </div><div class="csl-right-inline"><strong>Serpentine Ultralong Path with Extended Routing (SUPER) High Resolution Traveling Wave Ion Mobility-MS using Structures for Lossless Ion Manipulations</strong> <div class="csl-block">Liulin Deng, Ian K Webb, Sandilya VB Garimella, Ahmed M Hamid, Xueyun Zheng, Randolph V Norheim, Spencer A Prost, Gordon A Anderson, Jeremy A Sandoval, Erin S Baker, … Richard D Smith</div> <em>Analytical Chemistry</em> (2017-04-05) <a href="https://doi.org/f9zgxj">https://doi.org/f9zgxj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.7b00185">10.1021/acs.analchem.7b00185</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28332832">28332832</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5627996">PMC5627996</a></div></div>
</div>
<div id="ref-GwEAWw2z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">394. </div><div class="csl-right-inline"><strong>High-Resolution Ion-Mobility-Enabled Peptide Mapping for High-Throughput Critical Quality Attribute Monitoring</strong> <div class="csl-block">James R Arndt, Kelly L Wormwood Moser, Gregory Van Aken, Rory M Doyle, Tatjana Talamantes, Daniel DeBord, Laura Maxon, George Stafford, John Fjeldsted, Bryan Miller, Melissa Sherman</div> <em>Journal of the American Society for Mass Spectrometry</em> (2021-04-09) <a href="https://doi.org/gssrrb">https://doi.org/gssrrb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/jasms.0c00434">10.1021/jasms.0c00434</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33835810">33835810</a></div></div>
</div>
<div id="ref-YdLSljVE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">395. </div><div class="csl-right-inline"><strong>The Nobel Prize in Chemistry 2002</strong> <div class="csl-block">NobelPrize.org</div> <a href="https://www.nobelprize.org/prizes/chemistry/2002/summary/">https://www.nobelprize.org/prizes/chemistry/2002/summary/</a></div>
</div>
<div id="ref-zfiYj2co" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">396. </div><div class="csl-right-inline"><strong>Influence of the wavelength in high-irradiance ultraviolet laser desorption mass spectrometry of organic molecules</strong> <div class="csl-block">Michael Karas, Doris Bachmann, Franz Hillenkamp</div> <em>Analytical Chemistry</em> (1985-12-01) <a href="https://pubs.acs.org/doi/abs/10.1021/ac00291a042">https://pubs.acs.org/doi/abs/10.1021/ac00291a042</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac00291a042">10.1021/ac00291a042</a></div></div>
</div>
<div id="ref-tnQHCFV7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">397. </div><div class="csl-right-inline"><strong>Laser desorption ionization of proteins with molecular masses exceeding 10,000 daltons</strong> <div class="csl-block">Michael Karas, Franz Hillenkamp</div> <em>Analytical Chemistry</em> (1988-10-15) <a href="https://doi.org/d577jp">https://doi.org/d577jp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac00171a028">10.1021/ac00171a028</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3239801">3239801</a></div></div>
</div>
<div id="ref-1HBJs2tAB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">398. </div><div class="csl-right-inline"><strong>The Scientist :: Nobel Prize controversy</strong> (2007-05-17) <a href="https://web.archive.org/web/20070517202246/http://cmbi.bjmu.edu.cn/news/0212/55.htm">https://web.archive.org/web/20070517202246/http://cmbi.bjmu.edu.cn/news/0212/55.htm</a></div>
</div>
<div id="ref-5ZJdaya1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">399. </div><div class="csl-right-inline"><strong>α-Cyano-4-hydroxycinnamic acid, sinapinic acid, and ferulic acid as matrices and alkylating agents for matrix-assisted laser desorption/ionization time-of-flight mass spectrometric analysis of cysteine-containing peptides.</strong> <div class="csl-block">Hongmei Yang, Debin Wan, Fengrui Song, Zhiqiang Liu, Shuying Liu</div> <em>Rapid communications in mass spectrometry : RCM</em> (2013-06-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23681820">https://www.ncbi.nlm.nih.gov/pubmed/23681820</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/rcm.6587">10.1002/rcm.6587</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23681820">23681820</a></div></div>
</div>
<div id="ref-wGHAVD9r" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">400. </div><div class="csl-right-inline"><strong>The Desorption Process in MALDI</strong> <div class="csl-block">Klaus Dreisewerd</div> <em>Chemical Reviews</em> (2003-01-24) <a href="https://doi.org/cpzqmq">https://doi.org/cpzqmq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/cr010375i">10.1021/cr010375i</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12580636">12580636</a></div></div>
</div>
<div id="ref-DEnAe0bs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">401. </div><div class="csl-right-inline"><strong>Matrix Dependence of Metastable Fragmentation of Glycoproteins in MALDI TOF Mass Spectrometry</strong> <div class="csl-block">Michael Karas, Ute Bahr, Kerstin Strupat, Franz Hillenkamp, Anthony Tsarbopoulos, Birendra N Pramanik</div> <em>Analytical Chemistry</em> (1995-02-01) <a href="https://doi.org/b54gnt">https://doi.org/b54gnt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac00099a029">10.1021/ac00099a029</a></div></div>
</div>
<div id="ref-BslEV2vj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">402. </div><div class="csl-right-inline"><strong>XX. &lt;i&gt;On the equilibrium of liquid conducting masses charged with electricity&lt;/i&gt;</strong> <div class="csl-block">Lord Rayleigh</div> <em>The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science</em> (1882-09) <a href="https://doi.org/c6bp6h">https://doi.org/c6bp6h</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1080/14786448208628425">10.1080/14786448208628425</a></div></div>
</div>
<div id="ref-11vyuKWyw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">403. </div><div class="csl-right-inline"><strong>Charge state coalescence during electrospray ionization improves peptide identification by tandem mass spectrometry.</strong> <div class="csl-block">Jesse G Meyer, Elizabeth A Komives</div> <em>Journal of the American Society for Mass Spectrometry</em> (2012-05-18) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22610994">https://www.ncbi.nlm.nih.gov/pubmed/22610994</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s13361-012-0404-0">10.1007/s13361-012-0404-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22610994">22610994</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6345509">PMC6345509</a></div></div>
</div>
<div id="ref-4mpHldYP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">404. </div><div class="csl-right-inline"><strong>Electrospray: from ions in solution to ions in the gas phase, what we know now.</strong> <div class="csl-block">Paul Kebarle, Udo H Verkerk</div> <em>Mass spectrometry reviews</em> (2009) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19551695">https://www.ncbi.nlm.nih.gov/pubmed/19551695</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.20247">10.1002/mas.20247</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19551695">19551695</a></div></div>
</div>
<div id="ref-93IPP3bH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">405. </div><div class="csl-right-inline"><strong>Unraveling the mechanism of electrospray ionization.</strong> <div class="csl-block">Lars Konermann, Elias Ahadi, Antony D Rodriguez, Siavash Vahidi</div> <em>Analytical chemistry</em> (2012-11-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23134552">https://www.ncbi.nlm.nih.gov/pubmed/23134552</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac302789c">10.1021/ac302789c</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23134552">23134552</a></div></div>
</div>
<div id="ref-JpKe7ymh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">406. </div><div class="csl-right-inline"><strong>Capillary Liquid Chromatography/Mass Spectrometry for Peptide and Protein Characterization</strong> <div class="csl-block">MT Davis, DC Stahl, KM Swiderek, TD Lee</div> <em>Methods</em> (1994-09) <a href="https://doi.org/fmtw5k">https://doi.org/fmtw5k</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1006/meth.1994.1031">10.1006/meth.1994.1031</a></div></div>
</div>
<div id="ref-byV7q4rP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">407. </div><div class="csl-right-inline"><strong>Method to Correlate Tandem Mass Spectra of Modified Peptides to Amino Acid Sequences in the Protein Database</strong> <div class="csl-block">John R Yates, Jimmy K Eng, Ashley L McCormack, David Schieltz</div> <em>Analytical Chemistry</em> (1995-04-15) <a href="https://doi.org/dtcrm9">https://doi.org/dtcrm9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac00104a020">10.1021/ac00104a020</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/7741214">7741214</a></div></div>
</div>
<div id="ref-1BU9KEnwO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">408. </div><div class="csl-right-inline"><strong>Data-controlled automation of liquid chromatography/tandem mass spectrometry analysis of peptide mixtures.</strong> <div class="csl-block">DC Stahl, KM Swiderek, MT Davis, TD Lee</div> <em>Journal of the American Society for Mass Spectrometry</em> (1996-06) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24203425">https://www.ncbi.nlm.nih.gov/pubmed/24203425</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/1044-0305(96)00057-8">10.1016/1044-0305(96)00057-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24203425">24203425</a></div></div>
</div>
<div id="ref-vO5QsPvL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">409. </div><div class="csl-right-inline"><strong>Accurate Proteome-wide Label-free Quantification by Delayed Normalization and Maximal Peptide Ratio Extraction, Termed MaxLFQ</strong> <div class="csl-block">Jürgen Cox, Marco Y Hein, Christian A Luber, Igor Paron, Nagarjuna Nagaraj, Matthias Mann</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2014-09) <a href="https://doi.org/f6hmm5">https://doi.org/f6hmm5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m113.031591">10.1074/mcp.m113.031591</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24942700">24942700</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4159666">PMC4159666</a></div></div>
</div>
<div id="ref-Gs7HWzAD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">410. </div><div class="csl-right-inline"><strong>The MaxQuant computational platform for mass spectrometry-based shotgun proteomics</strong> <div class="csl-block">Stefka Tyanova, Tikira Temu, Juergen Cox</div> <em>Nature Protocols</em> (2016-10-27) <a href="https://doi.org/f89fwn">https://doi.org/f89fwn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nprot.2016.136">10.1038/nprot.2016.136</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27809316">27809316</a></div></div>
</div>
<div id="ref-66LthRpU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">411. </div><div class="csl-right-inline"><strong>Advances in proteomics data analysis and display using an accurate mass and time tag approach</strong> <div class="csl-block">Jennifer SD Zimmer, Matthew E Monroe, Wei-Jun Qian, Richard D Smith</div> <em>Mass Spectrometry Reviews</em> (2006) <a href="https://doi.org/bqv76w">https://doi.org/bqv76w</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.20071">10.1002/mas.20071</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16429408">16429408</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1829209">PMC1829209</a></div></div>
</div>
<div id="ref-MaoP0MGM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">412. </div><div class="csl-right-inline"><strong>A New Algorithm Using Cross-Assignment for Label-Free Quantitation with LC−LTQ-FT MS</strong> <div class="csl-block">Victor P Andreev, Lingyun Li, Lei Cao, Ye Gu, Tomas Rejtar, Shiaw-Lin Wu, Barry L Karger</div> <em>Journal of Proteome Research</em> (2007-04-19) <a href="https://doi.org/d3fshp">https://doi.org/d3fshp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr0606880">10.1021/pr0606880</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17441747">17441747</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2563808">PMC2563808</a></div></div>
</div>
<div id="ref-5H13CEWA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">413. </div><div class="csl-right-inline"><strong>IDEAL-Q, an Automated Tool for Label-free Quantitation Analysis Using an Efficient Peptide Alignment Approach and Spectral Data Validation</strong> <div class="csl-block">Chih-Chiang Tsou, Chia-Feng Tsai, Ying-Hao Tsui, Putty-Reddy Sudhir, Yi-Ting Wang, Yu-Ju Chen, Jeou-Yuan Chen, Ting-Yi Sung, Wen-Lian Hsu</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2010-01) <a href="https://doi.org/d9gxpm">https://doi.org/d9gxpm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m900177-mcp200">10.1074/mcp.m900177-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19752006">19752006</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2808259">PMC2808259</a></div></div>
</div>
<div id="ref-rRs3KNfz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">414. </div><div class="csl-right-inline"><strong>&lt;b&gt;
&lt;i&gt;SuperHirn&lt;/i&gt;
&lt;/b&gt;
- a novel tool for high resolution LC-MS-based peptide/protein profiling</strong> <div class="csl-block">Lukas N Mueller, Oliver Rinner, Alexander Schmidt, Simon Letarte, Bernd Bodenmiller, Mi-Youn Brusniak, Olga Vitek, Ruedi Aebersold, Markus Müller</div> <em>PROTEOMICS</em> (2007-08-28) <a href="https://doi.org/c5zp29">https://doi.org/c5zp29</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200700057">10.1002/pmic.200700057</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17726677">17726677</a></div></div>
</div>
<div id="ref-1EHOiWEPV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">415. </div><div class="csl-right-inline"><strong>Evaluating False Transfer Rates from the Match-between-Runs Algorithm with a Two-Proteome Model</strong> <div class="csl-block">Matthew Y Lim, João A Paulo, Steven P Gygi</div> <em>Journal of Proteome Research</em> (2019-09-23) <a href="https://doi.org/ggd6nh">https://doi.org/ggd6nh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.9b00492">10.1021/acs.jproteome.9b00492</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31547658">31547658</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7346880">PMC7346880</a></div></div>
</div>
<div id="ref-XGf3EEUa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">416. </div><div class="csl-right-inline"><strong>IonQuant Enables Accurate and Sensitive Label-Free Quantification With FDR-Controlled Match-Between-Runs</strong> <div class="csl-block">Fengchao Yu, Sarah E Haynes, Alexey I Nesvizhskii</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2021) <a href="https://doi.org/gqfrh3">https://doi.org/gqfrh3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.mcpro.2021.100077">10.1016/j.mcpro.2021.100077</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33813065">33813065</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8131922">PMC8131922</a></div></div>
</div>
<div id="ref-2D6WnK0w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">417. </div><div class="csl-right-inline"><strong>Post Analysis Data Acquisition for the Iterative MS/MS Sampling of Proteomics Mixtures</strong> <div class="csl-block">Michael R Hoopmann, Gennifer E Merrihew, Priska D von Haller, Michael J MacCoss</div> <em>Journal of Proteome Research</em> (2009-03-03) <a href="https://doi.org/fgtwdr">https://doi.org/fgtwdr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr800828p">10.1021/pr800828p</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19256536">19256536</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2671646">PMC2671646</a></div></div>
</div>
<div id="ref-yqW0xFtN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">418. </div><div class="csl-right-inline"><strong>The Implications of Proteolytic Background for Shotgun Proteomics</strong> <div class="csl-block">Paola Picotti, Ruedi Aebersold, Bruno Domon</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2007-09) <a href="https://doi.org/ffrsx7">https://doi.org/ffrsx7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m700029-mcp200">10.1074/mcp.m700029-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17533221">17533221</a></div></div>
</div>
<div id="ref-h7UB5vUH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">419. </div><div class="csl-right-inline"><strong>Evaluation of HCD- and CID-type Fragmentation Within Their Respective Detection Platforms For Murine Phosphoproteomics</strong> <div class="csl-block">Mark P Jedrychowski, Edward L Huttlin, Wilhelm Haas, Mathew E Sowa, Ramin Rad, Steven P Gygi</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2011-12) <a href="https://doi.org/fw5pzp">https://doi.org/fw5pzp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m111.009910">10.1074/mcp.m111.009910</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21917720">21917720</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3237076">PMC3237076</a></div></div>
</div>
<div id="ref-C0Uvh6dQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">420. </div><div class="csl-right-inline"><strong>Feasibility of Large-Scale Phosphoproteomics with Higher Energy Collisional Dissociation Fragmentation</strong> <div class="csl-block">Nagarjuna Nagaraj, Rochelle CJ D’Souza, Juergen Cox, Jesper V Olsen, Matthias Mann</div> <em>Journal of Proteome Research</em> (2010-10-26) <a href="https://doi.org/d3xcz7">https://doi.org/d3xcz7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr100637q">10.1021/pr100637q</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20873877">20873877</a></div></div>
</div>
<div id="ref-YU1KHJ6Z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">421. </div><div class="csl-right-inline"><strong>Phosphoproteomics with Activated Ion Electron Transfer Dissociation</strong> <div class="csl-block">Nicholas M Riley, Alexander S Hebert, Gerhard Dürnberger, Florian Stanek, Karl Mechtler, Michael S Westphall, Joshua J Coon</div> <em>Analytical Chemistry</em> (2017-04-17) <a href="https://doi.org/f9353n">https://doi.org/f9353n</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.7b00212">10.1021/acs.analchem.7b00212</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28383256">28383256</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5555596">PMC5555596</a></div></div>
</div>
<div id="ref-18YSDGQ7f" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">422. </div><div class="csl-right-inline"><strong>Unambiguous Phosphosite Localization using Electron-Transfer/Higher-Energy Collision Dissociation (EThcD)</strong> <div class="csl-block">Christian K Frese, Houjiang Zhou, Thomas Taus, AFMaarten Altelaar, Karl Mechtler, Albert JR Heck, Shabaz Mohammed</div> <em>Journal of Proteome Research</em> (2013-02-07) <a href="https://doi.org/f3sk5m">https://doi.org/f3sk5m</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr301130k">10.1021/pr301130k</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23347405">23347405</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3588588">PMC3588588</a></div></div>
</div>
<div id="ref-GH22tfkO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">423. </div><div class="csl-right-inline"><strong>Peptide and protein sequence analysis by electron transfer dissociation mass spectrometry</strong> <div class="csl-block">John EP Syka, Joshua J Coon, Melanie J Schroeder, Jeffrey Shabanowitz, Donald F Hunt</div> <em>Proceedings of the National Academy of Sciences</em> (2004-06-21) <a href="https://doi.org/ds3jcv">https://doi.org/ds3jcv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.0402700101">10.1073/pnas.0402700101</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15210983">15210983</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC470779">PMC470779</a></div></div>
</div>
<div id="ref-12LsVxcez" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">424. </div><div class="csl-right-inline"><strong>Evaluation of Gas-Phase Rearrangement and Competing Fragmentation Reactions on Protein Phosphorylation Site Assignment Using Collision Induced Dissociation-MS/MS and MS&lt;sup&gt;3&lt;/sup&gt;</strong> <div class="csl-block">Amanda M Palumbo, Gavin E Reid</div> <em>Analytical Chemistry</em> (2008-11-14) <a href="https://doi.org/fh634h">https://doi.org/fh634h</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac801768s">10.1021/ac801768s</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19012417">19012417</a></div></div>
</div>
<div id="ref-1BSd7ZIJx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">425. </div><div class="csl-right-inline"><strong>Examining factors that influence erroneous phosphorylation site localization via competing fragmentation and rearrangement reactions during ion trap &lt;scp&gt;CID&lt;/scp&gt;‐&lt;scp&gt;MS&lt;/scp&gt;/&lt;scp&gt;MS&lt;/scp&gt; and ‐&lt;scp&gt;MS&lt;/scp&gt;&lt;sup&gt;3&lt;/sup&gt;</strong> <div class="csl-block">Li Cui, Gavin E Reid</div> <em>PROTEOMICS</em> (2013-03) <a href="https://doi.org/f2d3hv">https://doi.org/f2d3hv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.201200384">10.1002/pmic.201200384</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23335301">23335301</a></div></div>
</div>
<div id="ref-XAV4WUNo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">426. </div><div class="csl-right-inline"><strong>Comparative Assessment of Site Assignments in CID and Electron Transfer Dissociation Spectra of Phosphopeptides Discloses Limited Relocation of Phosphate Groups</strong> <div class="csl-block">Nikolai Mischerikow, AFMaarten Altelaar, JDaniel Navarro, Shabaz Mohammed, Albert JR Heck</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2010-10) <a href="https://doi.org/bv4grk">https://doi.org/bv4grk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m900619-mcp200">10.1074/mcp.m900619-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20233845">20233845</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2953911">PMC2953911</a></div></div>
</div>
<div id="ref-Oadeotpz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">427. </div><div class="csl-right-inline"><strong>A Neutral Loss Activation Method for Improved Phosphopeptide Sequence Analysis by Quadrupole Ion Trap Mass Spectrometry</strong> <div class="csl-block">Melanie J Schroeder, Jeffrey Shabanowitz, Jae C Schwartz, Donald F Hunt, Joshua J Coon</div> <em>Analytical Chemistry</em> (2004-05-18) <a href="https://doi.org/dsk6xz">https://doi.org/dsk6xz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac0497104">10.1021/ac0497104</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15228329">15228329</a></div></div>
</div>
<div id="ref-DcoxXgj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">428. </div><div class="csl-right-inline"><strong>Pinpointing Phosphorylation Sites: Quantitative Filtering and a Novel Site-specific x-Ion Fragment</strong> <div class="csl-block">Christian D Kelstrup, Omid Hekmat, Chiara Francavilla, Jesper V Olsen</div> <em>Journal of Proteome Research</em> (2011-04-28) <a href="https://doi.org/bckqkr">https://doi.org/bckqkr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr200154t">10.1021/pr200154t</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21526838">21526838</a></div></div>
</div>
<div id="ref-SngtJdsy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">429. </div><div class="csl-right-inline"><strong>Large-scale characterization of HeLa cell nuclear phosphoproteins</strong> <div class="csl-block">Sean A Beausoleil, Mark Jedrychowski, Daniel Schwartz, Joshua E Elias, Judit Villén, Jiaxu Li, Martin A Cohn, Lewis C Cantley, Steven P Gygi</div> <em>Proceedings of the National Academy of Sciences</em> (2004-08-09) <a href="https://doi.org/cjmg8h">https://doi.org/cjmg8h</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.0404720101">10.1073/pnas.0404720101</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15302935">15302935</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC514446">PMC514446</a></div></div>
</div>
<div id="ref-WtrZNDoq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">430. </div><div class="csl-right-inline"><strong>Evaluation of the utility of neutral‐loss‐dependent MS3 strategies in large‐scale phosphorylation analysis</strong> <div class="csl-block">Judit Villén, Sean A Beausoleil, Steven P Gygi</div> <em>PROTEOMICS</em> (2008-10-29) <a href="https://doi.org/c52q48">https://doi.org/c52q48</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200800283">10.1002/pmic.200800283</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18972524">18972524</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2745099">PMC2745099</a></div></div>
</div>
<div id="ref-OCeJYJsg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">431. </div><div class="csl-right-inline"><strong>Targeted phosphoproteomics of insulin signaling using data-independent acquisition mass spectrometry</strong> <div class="csl-block">Benjamin L Parker, Guang Yang, Sean J Humphrey, Rima Chaudhuri, Xiuquan Ma, Scott Peterman, David E James</div> <em>Science Signaling</em> (2015-06-09) <a href="https://doi.org/f7qzfx">https://doi.org/f7qzfx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/scisignal.aaa3139">10.1126/scisignal.aaa3139</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26060331">26060331</a></div></div>
</div>
<div id="ref-GD2lTg3N" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">432. </div><div class="csl-right-inline"><strong>Data Processing and Analysis for DIA-Based Phosphoproteomics Using Spectronaut</strong> <div class="csl-block">Ana Martinez-Val, Dorte Breinholdt Bekker-Jensen, Alexander Hogrebe, Jesper Velgaard Olsen</div> <em>Methods in Molecular Biology</em> (2021) <a href="https://doi.org/gss5jc">https://doi.org/gss5jc</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-0716-1641-3_6">10.1007/978-1-0716-1641-3_6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34236657">34236657</a></div></div>
</div>
<div id="ref-18oI6YEEE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">433. </div><div class="csl-right-inline"><strong>Plug-and-play analysis of the human phosphoproteome by targeted high-resolution mass spectrometry</strong> <div class="csl-block">Robert T Lawrence, Brian C Searle, Ariadna Llovet, Judit Villén</div> <em>Nature Methods</em> (2016-03-28) <a href="https://doi.org/gss5jn">https://doi.org/gss5jn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.3811">10.1038/nmeth.3811</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27018578">27018578</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5915315">PMC5915315</a></div></div>
</div>
<div id="ref-nO23Yma7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">434. </div><div class="csl-right-inline"><strong>High-Density, Targeted Monitoring of Tyrosine Phosphorylation Reveals Activated Signaling Networks in Human Tumors</strong> <div class="csl-block">Lauren E Stopfer, Cameron T Flower, Aaron S Gajadhar, Bhavin Patel, Sebastien Gallien, Daniel Lopez-Ferrer, Forest M White</div> <em>Cancer Research</em> (2021-01-28) <a href="https://doi.org/gss5js">https://doi.org/gss5js</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1158/0008-5472.can-20-3804">10.1158/0008-5472.can-20-3804</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33509940">33509940</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8137532">PMC8137532</a></div></div>
</div>
<div id="ref-rydRUvTU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">435. </div><div class="csl-right-inline"><strong>Benchmarking common quantification strategies for large-scale phosphoproteomics</strong> <div class="csl-block">Alexander Hogrebe, Louise von Stechow, Dorte B Bekker-Jensen, Brian T Weinert, Christian D Kelstrup, Jesper V Olsen</div> <em>Nature Communications</em> (2018-03-13) <a href="https://doi.org/gc8gfs">https://doi.org/gc8gfs</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-018-03309-6">10.1038/s41467-018-03309-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29535314">29535314</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5849679">PMC5849679</a></div></div>
</div>
<div id="ref-aoK1gRMs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">436. </div><div class="csl-right-inline"><strong>Widespread bacterial protein histidine phosphorylation revealed by mass spectrometry-based proteomics</strong> <div class="csl-block">Clement M Potel, Miao-Hsia Lin, Albert JR Heck, Simone Lemeer</div> <em>Nature Methods</em> (2018-01-29) <a href="https://doi.org/gcwt3t">https://doi.org/gcwt3t</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.4580">10.1038/nmeth.4580</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29377012">29377012</a></div></div>
</div>
<div id="ref-17Un7LUDR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">437. </div><div class="csl-right-inline"><strong>Analysis of Histidine Phosphorylation Using Tandem MS and Ion−Electron Reactions</strong> <div class="csl-block">Anne J Kleinnijenhuis, Frank Kjeldsen, Birgitte Kallipolitis, Kim F Haselmann, Ole N Jensen</div> <em>Analytical Chemistry</em> (2007-09-07) <a href="https://doi.org/cpm6vt">https://doi.org/cpm6vt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac0707838">10.1021/ac0707838</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17822303">17822303</a></div></div>
</div>
<div id="ref-14IWO8iZc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">438. </div><div class="csl-right-inline"><strong>Histidine phosphorylation in human cells; a needle or phantom in the haystack?</strong> <div class="csl-block">Niels M Leijten, Albert JR Heck, Simone Lemeer</div> <em>Nature Methods</em> (2022-06-20) <a href="https://doi.org/gss5jq">https://doi.org/gss5jq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-022-01524-0">10.1038/s41592-022-01524-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35726056">35726056</a></div></div>
</div>
<div id="ref-xcaW4mpA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">439. </div><div class="csl-right-inline"><strong>Studying the fragmentation behavior of peptides with arginine phosphorylation and its influence on phospho‐site localization</strong> <div class="csl-block">Andreas Schmidt, Gustav Ammerer, Karl Mechtler</div> <em>PROTEOMICS</em> (2013-02-18) <a href="https://doi.org/f2d3h9">https://doi.org/f2d3h9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.201200240">10.1002/pmic.201200240</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23172725">23172725</a></div></div>
</div>
<div id="ref-1ELOuUl8I" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">440. </div><div class="csl-right-inline"><strong>Profiling of tyrosine phosphorylation pathways in human cells using mass spectrometry</strong> <div class="csl-block">Arthur R Salomon, Scott B Ficarro, Laurence M Brill, Achim Brinker, Qui T Phung, Christer Ericson, Karsten Sauer, Ansgar Brock, David M Horn, Peter G Schultz, Eric C Peters</div> <em>Proceedings of the National Academy of Sciences</em> (2003-01-09) <a href="https://doi.org/b559nw">https://doi.org/b559nw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.2436191100">10.1073/pnas.2436191100</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12522270">12522270</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC141014">PMC141014</a></div></div>
</div>
<div id="ref-uOTrEnEQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">441. </div><div class="csl-right-inline"><strong>Global, In Vivo, and Site-Specific Phosphorylation Dynamics in Signaling Networks</strong> <div class="csl-block">Jesper V Olsen, Blagoy Blagoev, Florian Gnad, Boris Macek, Chanchal Kumar, Peter Mortensen, Matthias Mann</div> <em>Cell</em> (2006-11) <a href="https://doi.org/fk9n7v">https://doi.org/fk9n7v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2006.09.026">10.1016/j.cell.2006.09.026</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17081983">17081983</a></div></div>
</div>
<div id="ref-GL3PYaNa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">442. </div><div class="csl-right-inline"><strong>Higher Energy Collision Dissociation (HCD) Product Ion-Triggered Electron Transfer Dissociation (ETD) Mass Spectrometry for the Analysis of &lt;i&gt;N&lt;/i&gt;-Linked Glycoproteins</strong> <div class="csl-block">Charandeep Singh, Cleidiane G Zampronio, Andrew J Creese, Helen J Cooper</div> <em>Journal of Proteome Research</em> (2012-08-06) <a href="https://doi.org/f36tfq">https://doi.org/f36tfq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr300257c">10.1021/pr300257c</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22800195">22800195</a></div></div>
</div>
<div id="ref-wAKbWLXc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">443. </div><div class="csl-right-inline"><strong>Clinical applications of quantitative proteomics using targeted and untargeted data-independent acquisition techniques</strong> <div class="csl-block">Jesse G Meyer, Birgit Schilling</div> <em>Expert Review of Proteomics</em> (2017-05-04) <a href="https://doi.org/gk6gdn">https://doi.org/gk6gdn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1080/14789450.2017.1322904">10.1080/14789450.2017.1322904</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28436239">28436239</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5671767">PMC5671767</a></div></div>
</div>
<div id="ref-NQj9Wees" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">444. </div><div class="csl-right-inline"><strong>Review of software tools for design and analysis of large scale MRM proteomic datasets</strong> <div class="csl-block">Christopher M Colangelo, Lisa Chung, Can Bruce, Kei-Hoi Cheung</div> <em>Methods</em> (2013-06) <a href="https://doi.org/f449z8">https://doi.org/f449z8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ymeth.2013.05.004">10.1016/j.ymeth.2013.05.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23702368">23702368</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3775261">PMC3775261</a></div></div>
</div>
<div id="ref-1FAMk6Qey" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">445. </div><div class="csl-right-inline"><strong>Quantification of C-reactive protein in the serum of patients with rheumatoid arthritis using multiple reaction monitoring mass spectrometry and13C-labeled peptide standards</strong> <div class="csl-block">Eric Kuhn, Jiang Wu, Johann Karl, Hua Liao, Werner Zolg, Brad Guild</div> <em>PROTEOMICS</em> (2004-04) <a href="https://doi.org/cxn7fw">https://doi.org/cxn7fw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200300670">10.1002/pmic.200300670</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15048997">15048997</a></div></div>
</div>
<div id="ref-63Pqc9VE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">446. </div><div class="csl-right-inline"><strong>Mass Spectrometric Quantitation of Peptides and Proteins Using Stable Isotope Standards and Capture by Anti-Peptide Antibodies (SISCAPA)</strong> <div class="csl-block">NLeigh Anderson, Norman G Anderson, Lee R Haines, Darryl B Hardie, Robert W Olafson, Terry W Pearson</div> <em>Journal of Proteome Research</em> (2004-02-06) <a href="https://doi.org/c494px">https://doi.org/c494px</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr034086h">10.1021/pr034086h</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15113099">15113099</a></div></div>
</div>
<div id="ref-iJJwYdQ1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">447. </div><div class="csl-right-inline"><strong>A review on mass spectrometry-based quantitative proteomics: Targeted and data independent acquisition</strong> <div class="csl-block">Veronika Vidova, Zdenek Spacil</div> <em>Analytica Chimica Acta</em> (2017-04) <a href="https://doi.org/f93hpx">https://doi.org/f93hpx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.aca.2017.01.059">10.1016/j.aca.2017.01.059</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28351641">28351641</a></div></div>
</div>
<div id="ref-ba7F2Epn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">448. </div><div class="csl-right-inline"><strong>The current status of clinical proteomics and the use of MRM and MRM&lt;sup&gt;3&lt;/sup&gt;for biomarker validation</strong> <div class="csl-block">Jérôme Lemoine, Tanguy Fortin, Arnaud Salvador, Aurore Jaffuel, Jean-Philippe Charrier, Geneviève Choquet-Kastylevsky</div> <em>Expert Review of Molecular Diagnostics</em> (2012-05) <a href="https://doi.org/f337q3">https://doi.org/f337q3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1586/erm.12.32">10.1586/erm.12.32</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22616699">22616699</a></div></div>
</div>
<div id="ref-jcu6ZT0x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">449. </div><div class="csl-right-inline"><strong>High Sensitivity Detection of Plasma Proteins by Multiple Reaction Monitoring of N-Glycosites</strong> <div class="csl-block">Jianru Stahl-Zeng, Vinzenz Lange, Reto Ossola, Katrin Eckhardt, Wilhelm Krek, Ruedi Aebersold, Bruno Domon</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2007-10) <a href="https://doi.org/bwgf39">https://doi.org/bwgf39</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m700132-mcp200">10.1074/mcp.m700132-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17644760">17644760</a></div></div>
</div>
<div id="ref-eFcH0xbv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">450. </div><div class="csl-right-inline"><strong>A database of mass spectrometric assays for the yeast proteome</strong> <div class="csl-block">Paola Picotti, Henry Lam, David Campbell, Eric W Deutsch, Hamid Mirzaei, Jeff Ranish, Bruno Domon, Ruedi Aebersold</div> <em>Nature Methods</em> (2008-11) <a href="https://doi.org/dvs3c7">https://doi.org/dvs3c7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth1108-913">10.1038/nmeth1108-913</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18974732">18974732</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2770732">PMC2770732</a></div></div>
</div>
<div id="ref-1Bt8NAhNo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">451. </div><div class="csl-right-inline"><strong>Parallel Reaction Monitoring for High Resolution and High Mass Accuracy Quantitative, Targeted Proteomics</strong> <div class="csl-block">Amelia C Peterson, Jason D Russell, Derek J Bailey, Michael S Westphall, Joshua J Coon</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2012-11) <a href="https://doi.org/f4gzw2">https://doi.org/f4gzw2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.o112.020131">10.1074/mcp.o112.020131</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22865924">22865924</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3494192">PMC3494192</a></div></div>
</div>
<div id="ref-HB3XvRF2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">452. </div><div class="csl-right-inline"><strong>Parallel reaction monitoring (PRM) and selected reaction monitoring (SRM) exhibit comparable linearity, dynamic range and precision for targeted quantitative HDL proteomics</strong> <div class="csl-block">Graziella E Ronsein, Nathalie Pamir, Priska D von Haller, Daniel S Kim, Michael N Oda, Gail P Jarvik, Tomas Vaisar, Jay W Heinecke</div> <em>Journal of Proteomics</em> (2015-01) <a href="https://doi.org/f6wq8n">https://doi.org/f6wq8n</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jprot.2014.10.017">10.1016/j.jprot.2014.10.017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25449833">25449833</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4259393">PMC4259393</a></div></div>
</div>
<div id="ref-dWzeUq6d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">453. </div><div class="csl-right-inline"><strong>Shotgun collision‐induced dissociation of peptides using a time of flight mass analyzer</strong> <div class="csl-block">Samuel Purvine, Jason‐Thomas Eppel*, Eugene C Yi, David R Goodlett</div> <em>PROTEOMICS</em> (2003-06) <a href="https://doi.org/c37bqm">https://doi.org/c37bqm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200300362">10.1002/pmic.200300362</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12833507">12833507</a></div></div>
</div>
<div id="ref-yn40ByaQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">454. </div><div class="csl-right-inline"><strong>Automated approach for quantitative analysis of complex peptide mixtures from tandem mass spectra</strong> <div class="csl-block">John D Venable, Meng-Qiu Dong, James Wohlschlegel, Andrew Dillin, John R Yates III</div> <em>Nature Methods</em> (2004-09-29) <a href="https://doi.org/dm8rm4">https://doi.org/dm8rm4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth705">10.1038/nmeth705</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15782151">15782151</a></div></div>
</div>
<div id="ref-Z3Hfnak1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">455. </div><div class="csl-right-inline"><strong>UPLC/MSE; a new approach for generating molecular fragment information for biomarker structure elucidation</strong> <div class="csl-block">Robert S Plumb, Kelly A Johnson, Paul Rainville, Brian W Smith, Ian D Wilson, Jose M Castro-Perez, Jeremy K Nicholson</div> <em>Rapid Communications in Mass Spectrometry</em> (2006) <a href="https://doi.org/c8gx39">https://doi.org/c8gx39</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/rcm.2550">10.1002/rcm.2550</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16755610">16755610</a></div></div>
</div>
<div id="ref-Wb9egjJg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">456. </div><div class="csl-right-inline"><strong>Precursor Acquisition Independent From Ion Count: How to Dive Deeper into the Proteomics Ocean</strong> <div class="csl-block">Alexandre Panchaud, Alexander Scherl, Scott A Shaffer, Priska D von Haller, Hemantha D Kulasekara, Samuel I Miller, David R Goodlett</div> <em>Analytical Chemistry</em> (2009-07-02) <a href="https://doi.org/dwbrjn">https://doi.org/dwbrjn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac900888s">10.1021/ac900888s</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19572557">19572557</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3086478">PMC3086478</a></div></div>
</div>
<div id="ref-OubW7j0g" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">457. </div><div class="csl-right-inline"><strong>Proteomics on an Orbitrap Benchtop Mass Spectrometer Using All-ion Fragmentation</strong> <div class="csl-block">Tamar Geiger, Juergen Cox, Matthias Mann</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2010-10) <a href="https://doi.org/fqrb3c">https://doi.org/fqrb3c</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m110.001537">10.1074/mcp.m110.001537</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20610777">20610777</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2953918">PMC2953918</a></div></div>
</div>
<div id="ref-y71ZDsTJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">458. </div><div class="csl-right-inline"><strong>XDIA: improving on the label-free data-independent analysis</strong> <div class="csl-block">Paulo C Carvalho, Xuemei Han, Tao Xu, Daniel Cociorva, Maria da Gloria Carvalho, Valmir C Barbosa, John R Yates III</div> <em>Bioinformatics</em> (2010-01-26) <a href="https://doi.org/cn8x3r">https://doi.org/cn8x3r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bioinformatics/btq031">10.1093/bioinformatics/btq031</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20106817">20106817</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2832823">PMC2832823</a></div></div>
</div>
<div id="ref-ckj0XYOM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">459. </div><div class="csl-right-inline"><strong>ETISEQ – an algorithm for automated elution time ion sequencing of concurrently fragmented peptides for mass spectrometry-based proteomics</strong> <div class="csl-block">Jason WH Wong, Alexander B Schwahn, Kevin M Downard</div> <em>BMC Bioinformatics</em> (2009-08-10) <a href="https://doi.org/c3rshm">https://doi.org/c3rshm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/1471-2105-10-244">10.1186/1471-2105-10-244</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19664259">19664259</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2731054">PMC2731054</a></div></div>
</div>
<div id="ref-11ccXv5xm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">460. </div><div class="csl-right-inline"><strong>Targeted Data Extraction of the MS/MS Spectra Generated by Data-independent Acquisition: A New Concept for Consistent and Accurate Proteome Analysis</strong> <div class="csl-block">Ludovic C Gillet, Pedro Navarro, Stephen Tate, Hannes Röst, Nathalie Selevsek, Lukas Reiter, Ron Bonner, Ruedi Aebersold</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2012-06) <a href="https://doi.org/fzqdc5">https://doi.org/fzqdc5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.o111.016717">10.1074/mcp.o111.016717</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22261725">22261725</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3433915">PMC3433915</a></div></div>
</div>
<div id="ref-1GCjdtGKY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">461. </div><div class="csl-right-inline"><strong>Multiplexed MS/MS for improved data-independent acquisition</strong> <div class="csl-block">Jarrett D Egertson, Andreas Kuehn, Gennifer E Merrihew, Nicholas W Bateman, Brendan X MacLean, Ying S Ting, Jesse D Canterbury, Donald M Marsh, Markus Kellmann, Vlad Zabrouskov, … Michael J MacCoss</div> <em>Nature Methods</em> (2013-06-23) <a href="https://doi.org/gjqdct">https://doi.org/gjqdct</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.2528">10.1038/nmeth.2528</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23793237">23793237</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3881977">PMC3881977</a></div></div>
</div>
<div id="ref-1GvzNjeV3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">462. </div><div class="csl-right-inline"><strong>Optimization of Experimental Parameters in Data-Independent Mass Spectrometry Significantly Increases Depth and Reproducibility of Results</strong> <div class="csl-block">Roland Bruderer, Oliver M Bernhardt, Tejas Gandhi, Yue Xuan, Julia Sondermann, Manuela Schmidt, David Gomez-Varela, Lukas Reiter</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2017-12) <a href="https://doi.org/gcqvm4">https://doi.org/gcqvm4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.ra117.000314">10.1074/mcp.ra117.000314</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29070702">29070702</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5724188">PMC5724188</a></div></div>
</div>
<div id="ref-YKRUqViz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">463. </div><div class="csl-right-inline"><strong>diaPASEF: parallel accumulation–serial fragmentation combined with data-independent acquisition</strong> <div class="csl-block">Florian Meier, Andreas-David Brunner, Max Frank, Annie Ha, Isabell Bludau, Eugenia Voytik, Stephanie Kaspar-Schoenefeld, Markus Lubeck, Oliver Raether, Nicolai Bache, … Matthias Mann</div> <em>Nature Methods</em> (2020-11-30) <a href="https://doi.org/gmhqdm">https://doi.org/gmhqdm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-020-00998-0">10.1038/s41592-020-00998-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33257825">33257825</a></div></div>
</div>
<div id="ref-44oI1g68" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">464. </div><div class="csl-right-inline"><strong>Single-Shot 10K Proteome Approach: Over 10,000 Protein Identifications by Data-Independent Acquisition-Based Single-Shot Proteomics with Ion Mobility Spectrometry</strong> <div class="csl-block">Yusuke Kawashima, Hirotaka Nagai, Ryo Konno, Masaki Ishikawa, Daisuke Nakajima, Hironori Sato, Ren Nakamura, Tomoyuki Furuyashiki, Osamu Ohara</div> <em>Journal of Proteome Research</em> (2022-05-06) <a href="https://doi.org/gqgg5x">https://doi.org/gqgg5x</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.2c00023">10.1021/acs.jproteome.2c00023</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35522919">35522919</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9171847">PMC9171847</a></div></div>
</div>
<div id="ref-imR1dFei" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">465. </div><div class="csl-right-inline"><strong>Interpretation of shotgun proteomic data: the protein inference problem.</strong> <div class="csl-block">Alexey I Nesvizhskii, Ruedi Aebersold</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2005-07-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16009968">https://www.ncbi.nlm.nih.gov/pubmed/16009968</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.r500012-mcp200">10.1074/mcp.r500012-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16009968">16009968</a></div></div>
</div>
<div id="ref-wtH5a4Ax" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">466. </div><div class="csl-right-inline"><strong>In-depth analysis of protein inference algorithms using multiple search engines and well-defined metrics</strong> <div class="csl-block">Enrique Audain, Julian Uszkoreit, Timo Sachsenberg, Julianus Pfeuffer, Xiao Liang, Henning Hermjakob, Aniel Sanchez, Martin Eisenacher, Knut Reinert, David L Tabb, … Yasset Perez-Riverol</div> <em>Journal of Proteomics</em> (2017-01) <a href="https://doi.org/f9r8r6">https://doi.org/f9r8r6</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jprot.2016.08.002">10.1016/j.jprot.2016.08.002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27498275">27498275</a></div></div>
</div>
<div id="ref-18PIzg2Co" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">467. </div><div class="csl-right-inline"><strong>MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification</strong> <div class="csl-block">Jürgen Cox, Matthias Mann</div> <em>Nature Biotechnology</em> (2008-11-30) <a href="https://doi.org/crn24x">https://doi.org/crn24x</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nbt.1511">10.1038/nbt.1511</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19029910">19029910</a></div></div>
</div>
<div id="ref-bKpYR6PB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">468. </div><div class="csl-right-inline"><strong>MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry–based proteomics</strong> <div class="csl-block">Andy T Kong, Felipe V Leprevost, Dmitry M Avtonomov, Dattatreya Mellacheruvu, Alexey I Nesvizhskii</div> <em>Nature Methods</em> (2017-04-10) <a href="https://doi.org/f9z6p7">https://doi.org/f9z6p7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.4256">10.1038/nmeth.4256</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28394336">28394336</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5409104">PMC5409104</a></div></div>
</div>
<div id="ref-rKtsv6G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">469. </div><div class="csl-right-inline"><strong>Probability-based protein identification by searching sequence databases using mass spectrometry data.</strong> <div class="csl-block">DN Perkins, DJ Pappin, DM Creasy, JS Cottrell</div> <em>Electrophoresis</em> (1999-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10612281">https://www.ncbi.nlm.nih.gov/pubmed/10612281</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/(sici)1522-2683(19991201)20:18&lt;3551::aid-elps3551&gt;3.0.co;2-2">10.1002/(sici)1522-2683(19991201)20:18&lt;3551::aid-elps3551&gt;3.0.co;2-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10612281">10612281</a></div></div>
</div>
<div id="ref-1B2RVFOKK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">470. </div><div class="csl-right-inline"><strong>MS-GF+ makes progress towards a universal database search tool for proteomics</strong> <div class="csl-block">Sangtae Kim, Pavel A Pevzner</div> <em>Nature Communications</em> (2014-10-31) <a href="https://doi.org/ggkdq8">https://doi.org/ggkdq8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/ncomms6277">10.1038/ncomms6277</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25358478">25358478</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5036525">PMC5036525</a></div></div>
</div>
<div id="ref-IcvYCrHi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">471. </div><div class="csl-right-inline"><strong>A method for reducing the time required to match protein sequences with tandem mass spectra</strong> <div class="csl-block">Robertson Craig, Ronald C Beavis</div> <em>Rapid Communications in Mass Spectrometry</em> (2003) <a href="https://doi.org/b7bgb9">https://doi.org/b7bgb9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/rcm.1198">10.1002/rcm.1198</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14558131">14558131</a></div></div>
</div>
<div id="ref-iK4IJ8ED" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">472. </div><div class="csl-right-inline"><strong>TANDEM: matching proteins with tandem mass spectra</strong> <div class="csl-block">Robertson Craig, Ronald C Beavis</div> <em>Bioinformatics</em> (2004-02-19) <a href="https://doi.org/cthw6n">https://doi.org/cthw6n</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bioinformatics/bth092">10.1093/bioinformatics/bth092</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14976030">14976030</a></div></div>
</div>
<div id="ref-CObAggjA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">473. </div><div class="csl-right-inline"><strong>Skyline: an open source document editor for creating and analyzing targeted proteomics experiments</strong> <div class="csl-block">Brendan MacLean, Daniela M Tomazela, Nicholas Shulman, Matthew Chambers, Gregory L Finney, Barbara Frewen, Randall Kern, David L Tabb, Daniel C Liebler, Michael J MacCoss</div> <em>Bioinformatics</em> (2010-02-09) <a href="https://doi.org/bqx9rq">https://doi.org/bqx9rq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bioinformatics/btq054">10.1093/bioinformatics/btq054</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20147306">20147306</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2844992">PMC2844992</a></div></div>
</div>
<div id="ref-1HLH2DWkK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">474. </div><div class="csl-right-inline"><strong>DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput</strong> <div class="csl-block">Vadim Demichev, Christoph B Messner, Spyros I Vernardis, Kathryn S Lilley, Markus Ralser</div> <em>Nature Methods</em> (2019-11-25) <a href="https://doi.org/gj9xgj">https://doi.org/gj9xgj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-019-0638-x">10.1038/s41592-019-0638-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31768060">31768060</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6949130">PMC6949130</a></div></div>
</div>
<div id="ref-IUqTUA43" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">475. </div><div class="csl-right-inline"><strong>Deep learning enables de novo peptide sequencing from data-independent-acquisition mass spectrometry</strong> <div class="csl-block">Ngoc Hieu Tran, Rui Qiao, Lei Xin, Xin Chen, Chuyi Liu, Xianglilan Zhang, Baozhen Shan, Ali Ghodsi, Ming Li</div> <em>Nature Methods</em> (2018-12-20) <a href="https://doi.org/gftvmn">https://doi.org/gftvmn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-018-0260-3">10.1038/s41592-018-0260-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30573815">30573815</a></div></div>
</div>
<div id="ref-UKcaDUJ7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">476. </div><div class="csl-right-inline"><strong>Extending the Limits of Quantitative Proteome Profiling with Data-Independent Acquisition and Application to Acetaminophen-Treated Three-Dimensional Liver Microtissues</strong> <div class="csl-block">Roland Bruderer, Oliver M Bernhardt, Tejas Gandhi, Saša M Miladinović, Lin-Yang Cheng, Simon Messner, Tobias Ehrenberger, Vito Zanotelli, Yulia Butscheid, Claudia Escher, … Lukas Reiter</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2015-05) <a href="https://doi.org/f7b76h">https://doi.org/f7b76h</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m114.044305">10.1074/mcp.m114.044305</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25724911">25724911</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4424408">PMC4424408</a></div></div>
</div>
<div id="ref-B10jxZjh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">477. </div><div class="csl-right-inline"><strong>PeptideShaker enables reanalysis of MS-derived proteomics data sets</strong> <div class="csl-block">Marc Vaudel, Julia M Burkhart, René P Zahedi, Eystein Oveland, Frode S Berven, Albert Sickmann, Lennart Martens, Harald Barsnes</div> <em>Nature Biotechnology</em> (2015-01) <a href="https://doi.org/ggkds8">https://doi.org/ggkds8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nbt.3109">10.1038/nbt.3109</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25574629">25574629</a></div></div>
</div>
<div id="ref-tZ6JV4UF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">478. </div><div class="csl-right-inline"><strong>PeptideShaker Online: A User-Friendly Web-Based Framework for the Identification of Mass Spectrometry-Based Proteomics Data</strong> <div class="csl-block">Yehia Mokhtar Farag, Carlos Horro, Marc Vaudel, Harald Barsnes</div> <em>Journal of Proteome Research</em> (2021-10-28) <a href="https://doi.org/gpdd85">https://doi.org/gpdd85</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.1c00678">10.1021/acs.jproteome.1c00678</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34709836">34709836</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8650087">PMC8650087</a></div></div>
</div>
<div id="ref-zsmAmqZJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">479. </div><div class="csl-right-inline"><strong>mzML—a Community Standard for Mass Spectrometry Data</strong> <div class="csl-block">Lennart Martens, Matthew Chambers, Marc Sturm, Darren Kessner, Fredrik Levander, Jim Shofstahl, Wilfred H Tang, Andreas Römpp, Steffen Neumann, Angel D Pizarro, … Eric W Deutsch</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2011-01) <a href="https://doi.org/dxkg99">https://doi.org/dxkg99</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.r110.000133">10.1074/mcp.r110.000133</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20716697">20716697</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3013463">PMC3013463</a></div></div>
</div>
<div id="ref-fQHXeOG1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">480. </div><div class="csl-right-inline"><strong>Mass spectrometer output file format mzML.</strong> <div class="csl-block">Eric W Deutsch</div> <em>Methods in molecular biology (Clifton, N.J.)</em> (2010) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20013381">https://www.ncbi.nlm.nih.gov/pubmed/20013381</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-60761-444-9_22">10.1007/978-1-60761-444-9_22</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20013381">20013381</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073315">PMC3073315</a></div></div>
</div>
<div id="ref-r2hPariY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">481. </div><div class="csl-right-inline"><strong>File Formats Commonly Used in Mass Spectrometry Proteomics</strong> <div class="csl-block">Eric W Deutsch</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2012-12) <a href="https://doi.org/ggkdvv">https://doi.org/ggkdvv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.r112.019695">10.1074/mcp.r112.019695</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22956731">22956731</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3518119">PMC3518119</a></div></div>
</div>
<div id="ref-1GxDrwPIW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">482. </div><div class="csl-right-inline"><strong>Anatomy and evolution of database search engines—a central component of mass spectrometry based proteomic workflows</strong> <div class="csl-block">Kenneth Verheggen, Helge Ræder, Frode S Berven, Lennart Martens, Harald Barsnes, Marc Vaudel</div> <em>Mass Spectrometry Reviews</em> (2017-09-13) <a href="https://doi.org/gbwkmf">https://doi.org/gbwkmf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.21543">10.1002/mas.21543</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28902424">28902424</a></div></div>
</div>
<div id="ref-bjANWlBN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">483. </div><div class="csl-right-inline"><strong>Assigning Significance to Peptides Identified by Tandem Mass Spectrometry Using Decoy Databases</strong> <div class="csl-block">Lukas Käll, John D Storey, Michael J MacCoss, William Stafford Noble</div> <em>Journal of Proteome Research</em> (2007-12-08) <a href="https://doi.org/fbxhxp">https://doi.org/fbxhxp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr700600n">10.1021/pr700600n</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18067246">18067246</a></div></div>
</div>
<div id="ref-BrkRiRe8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">484. </div><div class="csl-right-inline"><strong>Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry.</strong> <div class="csl-block">Joshua E Elias, Steven P Gygi</div> <em>Nature methods</em> (2007-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17327847">https://www.ncbi.nlm.nih.gov/pubmed/17327847</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth1019">10.1038/nmeth1019</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17327847">17327847</a></div></div>
</div>
<div id="ref-5mQtZh5o" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">485. </div><div class="csl-right-inline"><strong>False Discovery Rate Estimation in Proteomics</strong> <div class="csl-block">Suruchi Aggarwal, Amit Kumar Yadav</div> <em>Methods in Molecular Biology</em> (2016) <a href="https://doi.org/f79mzp">https://doi.org/f79mzp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-4939-3106-4_7">10.1007/978-1-4939-3106-4_7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26519173">26519173</a></div></div>
</div>
<div id="ref-B2OZyvg5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">486. </div><div class="csl-right-inline"><strong>Unbiased False Discovery Rate Estimation for Shotgun Proteomics Based on the Target-Decoy Approach</strong> <div class="csl-block">Lev I Levitsky, Mark V Ivanov, Anna A Lobas, Mikhail V Gorshkov</div> <em>Journal of Proteome Research</em> (2016-12-13) <a href="https://doi.org/gqtfbj">https://doi.org/gqtfbj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.6b00144">10.1021/acs.jproteome.6b00144</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27959540">27959540</a></div></div>
</div>
<div id="ref-JXLIvVI6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">487. </div><div class="csl-right-inline"><strong>Quality Control in Proteomics</strong> <div class="csl-block">PROTEOMICS</div> (2011-03) <a href="https://doi.org/bzmh75">https://doi.org/bzmh75</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.201190020">10.1002/pmic.201190020</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21374817">21374817</a></div></div>
</div>
<div id="ref-8dt10izl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">488. </div><div class="csl-right-inline"><strong>Quality control in LC‐MS/MS</strong> <div class="csl-block">Thomas Köcher, Peter Pichler, Remco Swart, Karl Mechtler</div> <em>PROTEOMICS</em> (2011-02-07) <a href="https://doi.org/c54ck5">https://doi.org/c54ck5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.201000578">10.1002/pmic.201000578</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21360669">21360669</a></div></div>
</div>
<div id="ref-196I1uWdL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">489. </div><div class="csl-right-inline"><strong>Quality control in mass spectrometry‐based proteomics</strong> <div class="csl-block">Wout Bittremieux, David L Tabb, Francis Impens, An Staes, Evy Timmerman, Lennart Martens, Kris Laukens</div> <em>Mass Spectrometry Reviews</em> (2017-09-07) <a href="https://doi.org/gbs3vs">https://doi.org/gbs3vs</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.21544">10.1002/mas.21544</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28802010">28802010</a></div></div>
</div>
<div id="ref-9ocWQ3g2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">490. </div><div class="csl-right-inline"><strong>AlphaPept, a modern and open framework for MS-based proteomics</strong> <div class="csl-block">Maximilian T Strauss, Isabell Bludau, Wen-Feng Zeng, Eugenia Voytik, Constantin Ammar, Julia Schessner, Rajesh Ilango, Michelle Gill, Florian Meier, Sander Willems, Matthias Mann</div> <em>Cold Spring Harbor Laboratory</em> (2021-07-26) <a href="https://doi.org/gqmt2q">https://doi.org/gqmt2q</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2021.07.23.453379">10.1101/2021.07.23.453379</a></div></div>
</div>
<div id="ref-mKMrKqLf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">491. </div><div class="csl-right-inline"><strong>rawDiag: An R Package Supporting Rational LC–MS Method Optimization for Bottom-up Proteomics</strong> <div class="csl-block">Christian Trachsel, Christian Panse, Tobias Kockmann, Witold E Wolski, Jonas Grossmann, Ralph Schlapbach</div> <em>Journal of Proteome Research</em> (2018-07-06) <a href="https://doi.org/gd39tz">https://doi.org/gd39tz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.8b00173">10.1021/acs.jproteome.8b00173</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29978702">29978702</a></div></div>
</div>
<div id="ref-AI1wddvh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">492. </div><div class="csl-right-inline"><strong>The rawrr R Package: Direct Access to Orbitrap Data and Beyond</strong> <div class="csl-block">Tobias Kockmann, Christian Panse</div> <em>Journal of Proteome Research</em> (2021-03-09) <a href="https://doi.org/gjgdxj">https://doi.org/gjgdxj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.0c00866">10.1021/acs.jproteome.0c00866</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33686856">33686856</a></div></div>
</div>
<div id="ref-MotUU8WD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">493. </div><div class="csl-right-inline"><strong>RawBeans: A Simple, Vendor-Independent, Raw-Data Quality-Control Tool</strong> <div class="csl-block">David Morgenstern, Rotem Barzilay, Yishai Levin</div> <em>Journal of Proteome Research</em> (2021-03-04) <a href="https://doi.org/gmwh5f">https://doi.org/gmwh5f</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.0c00956">10.1021/acs.jproteome.0c00956</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33657803">33657803</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8041395">PMC8041395</a></div></div>
</div>
<div id="ref-fVgT4RCS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">494. </div><div class="csl-right-inline"><strong>SIMPATIQCO: A Server-Based Software Suite Which Facilitates Monitoring the Time Course of LC–MS Performance Metrics on Orbitrap Instruments</strong> <div class="csl-block">Peter Pichler, Michael Mazanek, Frederico Dusberger, Lisa Weilnböck, Christian G Huber, Christoph Stingl, Theo M Luider, Werner L Straube, Thomas Köcher, Karl Mechtler</div> <em>Journal of Proteome Research</em> (2012-10-22) <a href="https://doi.org/f3sk5j">https://doi.org/f3sk5j</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr300163u">10.1021/pr300163u</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23088386">23088386</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3558011">PMC3558011</a></div></div>
</div>
<div id="ref-gR6OKDGy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">495. </div><div class="csl-right-inline"><strong>Quality Control Analysis in Real-time (QC-ART): A Tool for Real-time Quality Control Assessment of Mass Spectrometry-based Proteomics Data</strong> <div class="csl-block">Bryan A Stanfill, Ernesto S Nakayasu, Lisa M Bramer, Allison M Thompson, Charles K Ansong, Therese R Clauss, Marina A Gritsenko, Matthew E Monroe, Ronald J Moore, Daniel J Orton, … Thomas O Metz</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2018-09) <a href="https://doi.org/gd7n2k">https://doi.org/gd7n2k</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.ra118.000648">10.1074/mcp.ra118.000648</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29666158">29666158</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6126382">PMC6126382</a></div></div>
</div>
<div id="ref-IrACRvhj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">496. </div><div class="csl-right-inline"><strong>SprayQc: A Real-Time LC–MS/MS Quality Monitoring System To Maximize Uptime Using Off the Shelf Components</strong> <div class="csl-block">Richard A Scheltema, Matthias Mann</div> <em>Journal of Proteome Research</em> (2012-05-11) <a href="https://doi.org/gqmt2m">https://doi.org/gqmt2m</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr201219e">10.1021/pr201219e</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22515319">22515319</a></div></div>
</div>
<div id="ref-N0V314Nq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">497. </div><div class="csl-right-inline"><strong>Metriculator: quality assessment for mass spectrometry-based proteomics</strong> <div class="csl-block">Ryan M Taylor, Jamison Dance, Russ J Taylor, John T Prince</div> <em>Bioinformatics</em> (2013-09-02) <a href="https://doi.org/f5gpzz">https://doi.org/f5gpzz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bioinformatics/btt510">10.1093/bioinformatics/btt510</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24002108">24002108</a></div></div>
</div>
<div id="ref-ZJ3s73nL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">498. </div><div class="csl-right-inline"><strong>OpenMS: a flexible open-source software platform for mass spectrometry data analysis</strong> <div class="csl-block">Hannes L Röst, Timo Sachsenberg, Stephan Aiche, Chris Bielow, Hendrik Weisser, Fabian Aicheler, Sandro Andreotti, Hans-Christian Ehrlich, Petra Gutenbrunner, Erhan Kenar, … Oliver Kohlbacher</div> <em>Nature Methods</em> (2016-08-30) <a href="https://doi.org/f82r32">https://doi.org/f82r32</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.3959">10.1038/nmeth.3959</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27575624">27575624</a></div></div>
</div>
<div id="ref-LpMjs9Ht" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">499. </div><div class="csl-right-inline"><strong>MSstats: an R package for statistical analysis of quantitative mass spectrometry-based proteomic experiments</strong> <div class="csl-block">Meena Choi, Ching-Yun Chang, Timothy Clough, Daniel Broudy, Trevor Killeen, Brendan MacLean, Olga Vitek</div> <em>Bioinformatics</em> (2014-05-02) <a href="https://doi.org/f6j737">https://doi.org/f6j737</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bioinformatics/btu305">10.1093/bioinformatics/btu305</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24794931">24794931</a></div></div>
</div>
<div id="ref-BVhBtZ2w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">500. </div><div class="csl-right-inline"><strong>MSstatsQC: Longitudinal System Suitability Monitoring and Quality Control for Targeted Proteomic Experiments</strong> <div class="csl-block">Eralp Dogu, Sara Mohammad-Taheri, Susan E Abbatiello, Michael S Bereman, Brendan MacLean, Birgit Schilling, Olga Vitek</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2017-07) <a href="https://doi.org/gbmgrh">https://doi.org/gbmgrh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m116.064774">10.1074/mcp.m116.064774</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28483925">28483925</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5500765">PMC5500765</a></div></div>
</div>
<div id="ref-R1U4dX3A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">501. </div><div class="csl-right-inline"><strong>Proteomics Quality Control: Quality Control Software for MaxQuant Results</strong> <div class="csl-block">Chris Bielow, Guido Mastrobuoni, Stefan Kempa</div> <em>Journal of Proteome Research</em> (2015-12-28) <a href="https://doi.org/f8c54w">https://doi.org/f8c54w</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.5b00780">10.1021/acs.jproteome.5b00780</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26653327">26653327</a></div></div>
</div>
<div id="ref-Nix1GwUR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">502. </div><div class="csl-right-inline"><strong>protti: an R package for comprehensive data analysis of peptide- and protein-centric bottom-up proteomics data</strong> <div class="csl-block">Jan-Philipp Quast, Dina Schuster, Paola Picotti</div> <em>Bioinformatics Advances</em> (2021-12-10) <a href="https://doi.org/gqmt2n">https://doi.org/gqmt2n</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bioadv/vbab041">10.1093/bioadv/vbab041</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36699412">36699412</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9710675">PMC9710675</a></div></div>
</div>
<div id="ref-KDnDuk25" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">503. </div><div class="csl-right-inline"><strong>MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.</strong> <div class="csl-block">Carson Holt, Mark Yandell</div> <em>BMC bioinformatics</em> (2011-12-22) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22192575">https://www.ncbi.nlm.nih.gov/pubmed/22192575</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/1471-2105-12-491">10.1186/1471-2105-12-491</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22192575">22192575</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3280279">PMC3280279</a></div></div>
</div>
<div id="ref-12mAW7BXf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">504. </div><div class="csl-right-inline"><strong>The NCBI Eukaryotic Genome Annotation Pipeline</strong> <a href="https://www.ncbi.nlm.nih.gov/genome/annotation_euk/process/">https://www.ncbi.nlm.nih.gov/genome/annotation_euk/process/</a></div>
</div>
<div id="ref-BRlMNUOW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">505. </div><div class="csl-right-inline"><strong>UniProt</strong> <a href="https://www.uniprot.org/help/biocuration">https://www.uniprot.org/help/biocuration</a></div>
</div>
<div id="ref-1EbCD9r1v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">506. </div><div class="csl-right-inline"><a href="https://www.psidev.info/sites/default/files/2018-03/MIAPE_MSI_1.1.pdf">https://www.psidev.info/sites/default/files/2018-03/MIAPE_MSI_1.1.pdf</a></div>
</div>
<div id="ref-paTtvUb8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">507. </div><div class="csl-right-inline"><strong>Guidelines for reporting quantitative mass spectrometry based experiments in proteomics.</strong> <div class="csl-block">Salvador Martínez-Bartolomé, Eric W Deutsch, Pierre-Alain Binz, Andrew R Jones, Martin Eisenacher, Gerhard Mayer, Alex Campos, Francesc Canals, Joan-Josep Bech-Serra, Montserrat Carrascal, … Juan P Albar</div> <em>Journal of proteomics</em> (2013-03-14) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23500130">https://www.ncbi.nlm.nih.gov/pubmed/23500130</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jprot.2013.02.026">10.1016/j.jprot.2013.02.026</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23500130">23500130</a></div></div>
</div>
<div id="ref-16uefmiYk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">508. </div><div class="csl-right-inline"><strong>UniProt: the Universal Protein knowledgebase.</strong> <div class="csl-block">Rolf Apweiler, Amos Bairoch, Cathy H Wu, Winona C Barker, Brigitte Boeckmann, Serenella Ferro, Elisabeth Gasteiger, Hongzhan Huang, Rodrigo Lopez, Michele Magrane, … Lai-Su L Yeh</div> <em>Nucleic acids research</em> (2004-01-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/14681372">https://www.ncbi.nlm.nih.gov/pubmed/14681372</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/gkh131">10.1093/nar/gkh131</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14681372">14681372</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC308865">PMC308865</a></div></div>
</div>
<div id="ref-1BIF8v4YE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">509. </div><div class="csl-right-inline"><strong>UniProt</strong> <a href="https://www.uniprot.org/help/sequence_origin">https://www.uniprot.org/help/sequence_origin</a></div>
</div>
<div id="ref-Q2scu90J" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">510. </div><div class="csl-right-inline"><strong>UniProt</strong> <a href="https://www.uniprot.org/help/manual_curation">https://www.uniprot.org/help/manual_curation</a></div>
</div>
<div id="ref-16BHXvDGS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">511. </div><div class="csl-right-inline"><strong>fasta_utilities</strong> <div class="csl-block">Phillip Wilmarth</div> (2023-08-01) <a href="https://github.com/pwilmart/fasta_utilities">https://github.com/pwilmart/fasta_utilities</a></div>
</div>
<div id="ref-1GfGl6YGi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">512. </div><div class="csl-right-inline"><strong>Global Identification of Protein Post-translational Modifications in a Single-Pass Database Search.</strong> <div class="csl-block">Michael R Shortreed, Craig D Wenger, Brian L Frey, Gloria M Sheynkman, Mark Scalf, Mark P Keller, Alan D Attie, Lloyd M Smith</div> <em>Journal of proteome research</em> (2015-09-29) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26418581">https://www.ncbi.nlm.nih.gov/pubmed/26418581</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.5b00599">10.1021/acs.jproteome.5b00599</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26418581">26418581</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4642219">PMC4642219</a></div></div>
</div>
<div id="ref-uPHDl2Vm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">513. </div><div class="csl-right-inline"><strong>Datasets</strong> <div class="csl-block">NCBI</div> <a href="https://www.ncbi.nlm.nih.gov/datasets">https://www.ncbi.nlm.nih.gov/datasets</a></div>
</div>
<div id="ref-tg8zlLqI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">514. </div><div class="csl-right-inline"><strong>Eukaryotic genomes annotated at NCBI</strong> <a href="https://www.ncbi.nlm.nih.gov/genome/annotation_euk/all/">https://www.ncbi.nlm.nih.gov/genome/annotation_euk/all/</a></div>
</div>
<div id="ref-xpZWsuGI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">515. </div><div class="csl-right-inline"><strong>Prokaryotic RefSeq Genomes</strong> <a href="https://www.ncbi.nlm.nih.gov/refseq/about/prokaryotes/">https://www.ncbi.nlm.nih.gov/refseq/about/prokaryotes/</a></div>
</div>
<div id="ref-BfmLo0uT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">516. </div><div class="csl-right-inline"><strong>Eukaryotic RefSeq Genome Annotation Status</strong> <a href="https://www.ncbi.nlm.nih.gov/genome/annotation_euk/status/">https://www.ncbi.nlm.nih.gov/genome/annotation_euk/status/</a></div>
</div>
<div id="ref-fWjGnKWy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">517. </div><div class="csl-right-inline"><strong>Training &amp; Tutorials - Site Guide - NCBI</strong> <a href="https://www.ncbi.nlm.nih.gov/guide/training-tutorials/">https://www.ncbi.nlm.nih.gov/guide/training-tutorials/</a></div>
</div>
<div id="ref-17gB3bs6x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">518. </div><div class="csl-right-inline"><strong>Proteomics in Non-model Organisms: A New Analytical Frontier.</strong> <div class="csl-block">Michelle Heck, Benjamin A Neely</div> <em>Journal of proteome research</em> (2020-08-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/32786681">https://www.ncbi.nlm.nih.gov/pubmed/32786681</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.0c00448">10.1021/acs.jproteome.0c00448</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32786681">32786681</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7874939">PMC7874939</a></div></div>
</div>
<div id="ref-69obv795" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">519. </div><div class="csl-right-inline"><strong>Ensembl genome browser 110</strong> <a href="https://useast.ensembl.org/index.html">https://useast.ensembl.org/index.html</a></div>
</div>
<div id="ref-1EGyqGdLr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">520. </div><div class="csl-right-inline"><strong>Ensembl Genomes</strong> <a href="http://ensemblgenomes.org/">http://ensemblgenomes.org/</a></div>
</div>
<div id="ref-OeXy2hrh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">521. </div><div class="csl-right-inline"><strong>Interferences and contaminants encountered in modern mass spectrometry.</strong> <div class="csl-block">Bernd O Keller, Jie Sui, Alex B Young, Randy M Whittal</div> <em>Analytica chimica acta</em> (2008-04-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18790129">https://www.ncbi.nlm.nih.gov/pubmed/18790129</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.aca.2008.04.043">10.1016/j.aca.2008.04.043</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18790129">18790129</a></div></div>
</div>
<div id="ref-jwgBjKOg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">522. </div><div class="csl-right-inline"><strong>cRAP protein sequences</strong> <a href="https://www.thegpm.org/crap/">https://www.thegpm.org/crap/</a></div>
</div>
<div id="ref-WdwXXrRx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">523. </div><div class="csl-right-inline"><strong>Philosopher: a versatile toolkit for shotgun proteomics data analysis.</strong> <div class="csl-block">Felipe da Veiga Leprevost, Sarah E Haynes, Dmitry M Avtonomov, Hui-Yin Chang, Avinash K Shanmugam, Dattatreya Mellacheruvu, Andy T Kong, Alexey I Nesvizhskii</div> <em>Nature methods</em> (2020-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/32669682">https://www.ncbi.nlm.nih.gov/pubmed/32669682</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-020-0912-y">10.1038/s41592-020-0912-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32669682">32669682</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7509848">PMC7509848</a></div></div>
</div>
<div id="ref-m8bjbzJ5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">524. </div><div class="csl-right-inline"><strong>Protein Contaminants Matter: Building Universal Protein Contaminant Libraries for DDA and DIA Proteomics</strong> <div class="csl-block">Ashley M Frankenfield, Jiawei Ni, Mustafa Ahmed, Ling Hao</div> <em>Cold Spring Harbor Laboratory</em> (2022-04-28) <a href="https://doi.org/gp4xcp">https://doi.org/gp4xcp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2022.04.27.489766">10.1101/2022.04.27.489766</a></div></div>
</div>
<div id="ref-CnwgtVyY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">525. </div><div class="csl-right-inline"><strong>The minotaur proteome: avoiding cross-species identifications deriving from bovine serum in cell culture models.</strong> <div class="csl-block">Jakob Bunkenborg, Guadalupe Espadas García, Marcia Ivonne Peña Paz, Jens S Andersen, Henrik Molina</div> <em>Proteomics</em> (2010-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20641139">https://www.ncbi.nlm.nih.gov/pubmed/20641139</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.201000103">10.1002/pmic.201000103</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20641139">20641139</a></div></div>
</div>
<div id="ref-1CIFGLtYQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">526. </div><div class="csl-right-inline"><strong>Foetal bovine serum influence on in vitro extracellular vesicle analyses.</strong> <div class="csl-block">Brandon M Lehrich, Yaxuan Liang, Massimo S Fiandaca</div> <em>Journal of extracellular vesicles</em> (2021-01-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/33532042">https://www.ncbi.nlm.nih.gov/pubmed/33532042</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/jev2.12061">10.1002/jev2.12061</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33532042">33532042</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7830136">PMC7830136</a></div></div>
</div>
<div id="ref-wUllSafI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">527. </div><div class="csl-right-inline"><strong>The CRAPome: a contaminant repository for affinity purification-mass spectrometry data.</strong> <div class="csl-block">Dattatreya Mellacheruvu, Zachary Wright, Amber L Couzens, Jean-Philippe Lambert, Nicole A St-Denis, Tuo Li, Yana V Miteva, Simon Hauri, Mihaela E Sardiu, Teck Yew Low, … Alexey I Nesvizhskii</div> <em>Nature methods</em> (2013-07-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23921808">https://www.ncbi.nlm.nih.gov/pubmed/23921808</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.2557">10.1038/nmeth.2557</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23921808">23921808</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3773500">PMC3773500</a></div></div>
</div>
<div id="ref-EyM6Y0Xm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">528. </div><div class="csl-right-inline"><strong>Common Repository of FBS Proteins (cRFP) To Be Added to a Search Database for Mass Spectrometric Analysis of Cell Secretome.</strong> <div class="csl-block">Jihye Shin, Yumi Kwon, Seonjeong Lee, Seungjin Na, Eun Young Hong, Shinyeong Ju, Hyun-Gyo Jung, Prashant Kaushal, Sungho Shin, Ji Hyun Back, … Cheolju Lee</div> <em>Journal of proteome research</em> (2019-09-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/31475827">https://www.ncbi.nlm.nih.gov/pubmed/31475827</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.9b00475">10.1021/acs.jproteome.9b00475</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31475827">31475827</a></div></div>
</div>
<div id="ref-qgZxt4hu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">529. </div><div class="csl-right-inline"><strong>Accurately Assigning Peptides to Spectra When Only a Subset of Peptides Are Relevant.</strong> <div class="csl-block">Andy Lin, Deanna L Plubell, Uri Keich, William S Noble</div> <em>Journal of proteome research</em> (2021-07-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/34236864">https://www.ncbi.nlm.nih.gov/pubmed/34236864</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.1c00483">10.1021/acs.jproteome.1c00483</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34236864">34236864</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8489664">PMC8489664</a></div></div>
</div>
<div id="ref-12o52Fhcj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">530. </div><div class="csl-right-inline"><strong>Averaging Strategy To Reduce Variability in Target-Decoy Estimates of False Discovery Rate.</strong> <div class="csl-block">Uri Keich, Kaipo Tamura, William Stafford Noble</div> <em>Journal of proteome research</em> (2019-01-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30560673">https://www.ncbi.nlm.nih.gov/pubmed/30560673</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.8b00802">10.1021/acs.jproteome.8b00802</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30560673">30560673</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6919216">PMC6919216</a></div></div>
</div>
<div id="ref-Lovg2liS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">531. </div><div class="csl-right-inline"><strong>Mass spectrometrists should search only for peptides they care about.</strong> <div class="csl-block">William Stafford Noble</div> <em>Nature methods</em> (2015-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26125591">https://www.ncbi.nlm.nih.gov/pubmed/26125591</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.3450">10.1038/nmeth.3450</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26125591">26125591</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4711994">PMC4711994</a></div></div>
</div>
<div id="ref-nMWQwf9T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">532. </div><div class="csl-right-inline"><strong>Choosing an Optimal Database for Protein Identification from Tandem Mass Spectrometry Data.</strong> <div class="csl-block">Dhirendra Kumar, Amit Kumar Yadav, Debasis Dash</div> <em>Methods in molecular biology (Clifton, N.J.)</em> (2017) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27975281">https://www.ncbi.nlm.nih.gov/pubmed/27975281</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-4939-6740-7_3">10.1007/978-1-4939-6740-7_3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27975281">27975281</a></div></div>
</div>
<div id="ref-43dscula" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">533. </div><div class="csl-right-inline"><strong>Recent progress in mass spectrometry-based strategies for elucidating protein-protein interactions.</strong> <div class="csl-block">Teck Yew Low, Saiful Effendi Syafruddin, MAiman Mohtar, Adaikkalam Vellaichamy, Nisa Syakila A Rahman, Yuh-Fen Pung, Chris Soon Heng Tan</div> <em>Cellular and molecular life sciences : CMLS</em> (2021-05-27) <a href="https://www.ncbi.nlm.nih.gov/pubmed/34046695">https://www.ncbi.nlm.nih.gov/pubmed/34046695</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00018-021-03856-0">10.1007/s00018-021-03856-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34046695">34046695</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8159249">PMC8159249</a></div></div>
</div>
<div id="ref-xeKEaOVF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">534. </div><div class="csl-right-inline"><strong>The IntAct database: efficient access to fine-grained molecular interaction data.</strong> <div class="csl-block">Noemi Del Toro, Anjali Shrivastava, Eliot Ragueneau, Birgit Meldal, Colin Combe, Elisabet Barrera, Livia Perfetto, Karyn How, Prashansa Ratan, Gautam Shirodkar, … Henning Hermjakob</div> <em>Nucleic acids research</em> (2022-01-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/34761267">https://www.ncbi.nlm.nih.gov/pubmed/34761267</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/gkab1006">10.1093/nar/gkab1006</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34761267">34761267</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8728211">PMC8728211</a></div></div>
</div>
<div id="ref-D6gNmWdp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">535. </div><div class="csl-right-inline"><strong>Systematic computational prediction of protein interaction networks.</strong> <div class="csl-block">JG Lees, JK Heriche, I Morilla, JA Ranea, CA Orengo</div> <em>Physical biology</em> (2011-05-13) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21572181">https://www.ncbi.nlm.nih.gov/pubmed/21572181</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1088/1478-3975/8/3/035008">10.1088/1478-3975/8/3/035008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21572181">21572181</a></div></div>
</div>
<div id="ref-uSY25K4H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">536. </div><div class="csl-right-inline"><strong>The STRING database in 2021: customizable protein-protein networks, and functional characterization of user-uploaded gene/measurement sets.</strong> <div class="csl-block">Damian Szklarczyk, Annika L Gable, Katerina C Nastou, David Lyon, Rebecca Kirsch, Sampo Pyysalo, Nadezhda T Doncheva, Marc Legeay, Tao Fang, Peer Bork, … Christian von Mering</div> <em>Nucleic acids research</em> (2021-01-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/33237311">https://www.ncbi.nlm.nih.gov/pubmed/33237311</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/gkaa1074">10.1093/nar/gkaa1074</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33237311">33237311</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7779004">PMC7779004</a></div></div>
</div>
<div id="ref-Es98brAH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">537. </div><div class="csl-right-inline"><strong>Evolution of Centrality Measurements for the Detection of Essential Proteins in Biological Networks.</strong> <div class="csl-block">Mahdi Jalili, Ali Salehzadeh-Yazdi, Shailendra Gupta, Olaf Wolkenhauer, Marjan Yaghmaie, Osbaldo Resendis-Antonio, Kamran Alimoghaddam</div> <em>Frontiers in physiology</em> (2016-08-26) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27616995">https://www.ncbi.nlm.nih.gov/pubmed/27616995</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3389/fphys.2016.00375">10.3389/fphys.2016.00375</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27616995">27616995</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4999434">PMC4999434</a></div></div>
</div>
<div id="ref-awUpRsoz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">538. </div><div class="csl-right-inline"><strong>A systematic survey of centrality measures for protein-protein interaction networks.</strong> <div class="csl-block">Minoo Ashtiani, Ali Salehzadeh-Yazdi, Zahra Razaghi-Moghadam, Holger Hennig, Olaf Wolkenhauer, Mehdi Mirzaie, Mohieddin Jafari</div> <em>BMC systems biology</em> (2018-07-31) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30064421">https://www.ncbi.nlm.nih.gov/pubmed/30064421</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/s12918-018-0598-2">10.1186/s12918-018-0598-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30064421">30064421</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6069823">PMC6069823</a></div></div>
</div>
<div id="ref-12ufdjSBQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">539. </div><div class="csl-right-inline"><strong>Why do hubs tend to be essential in protein networks?</strong> <div class="csl-block">Xionglei He, Jianzhi Zhang</div> <em>PLoS genetics</em> (2006-04-26) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16751849">https://www.ncbi.nlm.nih.gov/pubmed/16751849</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pgen.0020088">10.1371/journal.pgen.0020088</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16751849">16751849</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1473040">PMC1473040</a></div></div>
</div>
<div id="ref-1FZ0vcua3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">540. </div><div class="csl-right-inline"><strong>Comparative genomics of centrality and essentiality in three eukaryotic protein-interaction networks.</strong> <div class="csl-block">Matthew W Hahn, Andrew D Kern</div> <em>Molecular biology and evolution</em> (2004-12-22) <a href="https://www.ncbi.nlm.nih.gov/pubmed/15616139">https://www.ncbi.nlm.nih.gov/pubmed/15616139</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/molbev/msi072">10.1093/molbev/msi072</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15616139">15616139</a></div></div>
</div>
<div id="ref-KQtBgFa0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">541. </div><div class="csl-right-inline"><strong>Closeness centrality | Network analysis of protein interaction data</strong> <div class="csl-block">EMBL-EBI</div> <a href="https://www.ebi.ac.uk/training/online/courses/network-analysis-of-protein-interaction-data-an-introduction/building-and-analysing-ppins/topological-ppin-analysis/closeness-centrality/">https://www.ebi.ac.uk/training/online/courses/network-analysis-of-protein-interaction-data-an-introduction/building-and-analysing-ppins/topological-ppin-analysis/closeness-centrality/</a></div>
</div>
<div id="ref-1BmdBEJEB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">542. </div><div class="csl-right-inline"><strong>The importance of bottlenecks in protein networks: correlation with gene essentiality and expression dynamics.</strong> <div class="csl-block">Haiyuan Yu, Philip M Kim, Emmett Sprecher, Valery Trifonov, Mark Gerstein</div> <em>PLoS computational biology</em> (2007-02-14) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17447836">https://www.ncbi.nlm.nih.gov/pubmed/17447836</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pcbi.0030059">10.1371/journal.pcbi.0030059</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17447836">17447836</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1853125">PMC1853125</a></div></div>
</div>
<div id="ref-1H6bETSCb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">543. </div><div class="csl-right-inline"><strong>Computational approaches for detecting protein complexes from protein interaction networks: a survey.</strong> <div class="csl-block">Xiaoli Li, Min Wu, Chee-Keong Kwoh, See-Kiong Ng</div> <em>BMC genomics</em> (2010-02-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20158874">https://www.ncbi.nlm.nih.gov/pubmed/20158874</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/1471-2164-11-s1-s3">10.1186/1471-2164-11-s1-s3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20158874">20158874</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2822531">PMC2822531</a></div></div>
</div>
<div id="ref-11gPEmZWK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">544. </div><div class="csl-right-inline"><strong>Evaluation of clustering algorithms for protein-protein interaction networks.</strong> <div class="csl-block">Sylvain Brohée, Jacques van Helden</div> <em>BMC bioinformatics</em> (2006-11-06) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17087821">https://www.ncbi.nlm.nih.gov/pubmed/17087821</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/1471-2105-7-488">10.1186/1471-2105-7-488</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17087821">17087821</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1637120">PMC1637120</a></div></div>
</div>
<div id="ref-iqdatwR1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">545. </div><div class="csl-right-inline"><strong>An automated method for finding molecular complexes in large protein interaction networks.</strong> <div class="csl-block">Gary D Bader, Christopher WV Hogue</div> <em>BMC bioinformatics</em> (2003-01-13) <a href="https://www.ncbi.nlm.nih.gov/pubmed/12525261">https://www.ncbi.nlm.nih.gov/pubmed/12525261</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/1471-2105-4-2">10.1186/1471-2105-4-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12525261">12525261</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC149346">PMC149346</a></div></div>
</div>
<div id="ref-WJOtOvRY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">546. </div><div class="csl-right-inline"><strong>Protein function prediction using guilty by association from interaction networks.</strong> <div class="csl-block">Damiano Piovesan, Manuel Giollo, Carlo Ferrari, Silvio CE Tosatto</div> <em>Amino acids</em> (2015-07-28) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26215734">https://www.ncbi.nlm.nih.gov/pubmed/26215734</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00726-015-2049-3">10.1007/s00726-015-2049-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26215734">26215734</a></div></div>
</div>
<div id="ref-8oE5lYNw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">547. </div><div class="csl-right-inline"><strong>Cytoscape: a software environment for integrated models of biomolecular interaction networks.</strong> <div class="csl-block">Paul Shannon, Andrew Markiel, Owen Ozier, Nitin S Baliga, Jonathan T Wang, Daniel Ramage, Nada Amin, Benno Schwikowski, Trey Ideker</div> <em>Genome research</em> (2003-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/14597658">https://www.ncbi.nlm.nih.gov/pubmed/14597658</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/gr.1239303">10.1101/gr.1239303</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14597658">14597658</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC403769">PMC403769</a></div></div>
</div>
<div id="ref-jrtjRoft" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">548. </div><div class="csl-right-inline"><strong>Cytoscape stringApp 2.0: Analysis and Visualization of Heterogeneous Biological Networks.</strong> <div class="csl-block">Nadezhda T Doncheva, John H Morris, Henrietta Holze, Rebecca Kirsch, Katerina C Nastou, Yesid Cuesta-Astroz, Thomas Rattei, Damian Szklarczyk, Christian von Mering, Lars J Jensen</div> <em>Journal of proteome research</em> (2022-12-13) <a href="https://www.ncbi.nlm.nih.gov/pubmed/36512705">https://www.ncbi.nlm.nih.gov/pubmed/36512705</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.2c00651">10.1021/acs.jproteome.2c00651</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36512705">36512705</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9904289">PMC9904289</a></div></div>
</div>
<div id="ref-h0C6mgc4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">549. </div><div class="csl-right-inline"><strong>Protein Bioinformatics</strong> <div class="csl-block">Cathy H Wu, Cecilia N Arighi, Karen E Ross (editors)</div> <em>Methods in Molecular Biology</em> (2017) <a href="https://doi.org/grmckg">https://doi.org/grmckg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-4939-6783-4">10.1007/978-1-4939-6783-4</a></div></div>
</div>
<div id="ref-CaFLiife" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">550. </div><div class="csl-right-inline"><strong>Recent advances in mass-spectrometry based proteomics software, tools and databases</strong> <div class="csl-block">Ankit Halder, Ayushi Verma, Deeptarup Biswas, Sanjeeva Srivastava</div> <em>Drug Discovery Today: Technologies</em> (2021-12) <a href="https://doi.org/grmckh">https://doi.org/grmckh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ddtec.2021.06.007">10.1016/j.ddtec.2021.06.007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34906327">34906327</a></div></div>
</div>
<div id="ref-QlVqCOZZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">551. </div><div class="csl-right-inline"><strong>Data analysis and bioinformatics tools for tandem mass spectrometry in proteomics</strong> <div class="csl-block">Eric W Deutsch, Henry Lam, Ruedi Aebersold</div> <em>Physiological Genomics</em> (2008-03) <a href="https://doi.org/d8zpvp">https://doi.org/d8zpvp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1152/physiolgenomics.00298.2007">10.1152/physiolgenomics.00298.2007</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18212004">18212004</a></div></div>
</div>
<div id="ref-IFevKgND" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">552. </div><div class="csl-right-inline"><strong>Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.</strong> <div class="csl-block">M Ashburner, CA Ball, JA Blake, D Botstein, H Butler, JM Cherry, AP Davis, K Dolinski, SS Dwight, JT Eppig, … G Sherlock</div> <em>Nature genetics</em> (2000-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10802651">https://www.ncbi.nlm.nih.gov/pubmed/10802651</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/75556">10.1038/75556</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10802651">10802651</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3037419">PMC3037419</a></div></div>
</div>
<div id="ref-15ZNlTPZi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">553. </div><div class="csl-right-inline"><strong>KEGG: kyoto encyclopedia of genes and genomes.</strong> <div class="csl-block">M Kanehisa, S Goto</div> <em>Nucleic acids research</em> (2000-01-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10592173">https://www.ncbi.nlm.nih.gov/pubmed/10592173</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/28.1.27">10.1093/nar/28.1.27</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10592173">10592173</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC102409">PMC102409</a></div></div>
</div>
<div id="ref-f3o81IaL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">554. </div><div class="csl-right-inline"><strong>15 years of PhosphoSitePlus®: integrating post-translationally modified sites, disease variants and isoforms.</strong> <div class="csl-block">Peter V Hornbeck, Jon M Kornhauser, Vaughan Latham, Beth Murray, Vidhisha Nandhikonda, Alex Nord, Elzbieta Skrzypek, Travis Wheeler, Bin Zhang, Florian Gnad</div> <em>Nucleic acids research</em> (2019-01-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30445427">https://www.ncbi.nlm.nih.gov/pubmed/30445427</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/gky1159">10.1093/nar/gky1159</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30445427">30445427</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6324072">PMC6324072</a></div></div>
</div>
<div id="ref-10dRpS3qN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">555. </div><div class="csl-right-inline"><strong>Network integration and modelling of dynamic drug responses at multi-omics levels.</strong> <div class="csl-block">Nathalie Selevsek, Florian Caiment, Ramona Nudischer, Hans Gmuender, Irina Agarkova, Francis L Atkinson, Ivo Bachmann, Vanessa Baier, Gal Barel, Chris Bauer, … Jos Kleinjans</div> <em>Communications biology</em> (2020-10-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/33060801">https://www.ncbi.nlm.nih.gov/pubmed/33060801</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s42003-020-01302-8">10.1038/s42003-020-01302-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33060801">33060801</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7567116">PMC7567116</a></div></div>
</div>
<div id="ref-DhTe2WLI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">556. </div><div class="csl-right-inline"><strong>The reactome pathway knowledgebase 2022.</strong> <div class="csl-block">Marc Gillespie, Bijay Jassal, Ralf Stephan, Marija Milacic, Karen Rothfels, Andrea Senff-Ribeiro, Johannes Griss, Cristoffer Sevilla, Lisa Matthews, Chuqiao Gong, … Peter D'Eustachio</div> <em>Nucleic acids research</em> (2022-01-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/34788843">https://www.ncbi.nlm.nih.gov/pubmed/34788843</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/gkab1028">10.1093/nar/gkab1028</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34788843">34788843</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8689983">PMC8689983</a></div></div>
</div>
<div id="ref-rlKsqE1k" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">557. </div><div class="csl-right-inline"><strong>Bioinformatics approaches for the functional interpretation of protein lists: from ontology term enrichment to network analysis.</strong> <div class="csl-block">Kris Laukens, Stefan Naulaerts, Wim Vanden Berghe</div> <em>Proteomics</em> (2015-02-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25430566">https://www.ncbi.nlm.nih.gov/pubmed/25430566</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.201400296">10.1002/pmic.201400296</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25430566">25430566</a></div></div>
</div>
<div id="ref-1FmAqWcpa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">558. </div><div class="csl-right-inline"><strong>The Gene Ontology resource: enriching a GOld mine.</strong> <em>Nucleic acids research</em> (2021-01-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/33290552">https://www.ncbi.nlm.nih.gov/pubmed/33290552</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/gkaa1113">10.1093/nar/gkaa1113</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33290552">33290552</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7779012">PMC7779012</a></div></div>
</div>
<div id="ref-7xIEHm2N" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">559. </div><div class="csl-right-inline"><strong>Urgent need for consistent standards in functional enrichment analysis</strong> <div class="csl-block">Kaumadi Wijesooriya, Sameer A Jadaan, Kaushalya L Perera, Tanuveer Kaur, Mark Ziemann</div> <em>PLOS Computational Biology</em> (2022-03-09) <a href="https://doi.org/gpm77p">https://doi.org/gpm77p</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pcbi.1009935">10.1371/journal.pcbi.1009935</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35263338">35263338</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8936487">PMC8936487</a></div></div>
</div>
<div id="ref-HNlvs4Yk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">560. </div><div class="csl-right-inline"><strong>Gene Set Analysis: Challenges, Opportunities, and Future Research.</strong> <div class="csl-block">Farhad Maleki, Katie Ovens, Daniel J Hogan, Anthony J Kusalik</div> <em>Frontiers in genetics</em> (2020-06-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/32695141">https://www.ncbi.nlm.nih.gov/pubmed/32695141</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3389/fgene.2020.00654">10.3389/fgene.2020.00654</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32695141">32695141</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7339292">PMC7339292</a></div></div>
</div>
<div id="ref-TWAPY3zr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">561. </div><div class="csl-right-inline"><strong>On the Interpretation of χ 2 from Contingency Tables, and the Calculation of P</strong> <div class="csl-block">RA Fisher</div> <em>Journal of the Royal Statistical Society</em> (1922-01) <a href="https://doi.org/frpswx">https://doi.org/frpswx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.2307/2340521">10.2307/2340521</a></div></div>
</div>
<div id="ref-ArmIWAfa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">562. </div><div class="csl-right-inline"><strong>Urgent need for consistent standards in functional enrichment analysis.</strong> <div class="csl-block">Kaumadi Wijesooriya, Sameer A Jadaan, Kaushalya L Perera, Tanuveer Kaur, Mark Ziemann</div> <em>PLoS computational biology</em> (2022-03-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/35263338">https://www.ncbi.nlm.nih.gov/pubmed/35263338</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pcbi.1009935">10.1371/journal.pcbi.1009935</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35263338">35263338</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8936487">PMC8936487</a></div></div>
</div>
<div id="ref-tt0Vdf4C" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">563. </div><div class="csl-right-inline"><strong>Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.</strong> <div class="csl-block">Aravind Subramanian, Pablo Tamayo, Vamsi K Mootha, Sayan Mukherjee, Benjamin L Ebert, Michael A Gillette, Amanda Paulovich, Scott L Pomeroy, Todd R Golub, Eric S Lander, Jill P Mesirov</div> <em>Proceedings of the National Academy of Sciences of the United States of America</em> (2005-09-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16199517">https://www.ncbi.nlm.nih.gov/pubmed/16199517</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.0506580102">10.1073/pnas.0506580102</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16199517">16199517</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1239896">PMC1239896</a></div></div>
</div>
<div id="ref-6gI0wLAR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">564. </div><div class="csl-right-inline"><strong>Methods and approaches in the topology-based analysis of biological pathways.</strong> <div class="csl-block">Cristina Mitrea, Zeinab Taghavi, Behzad Bokanizad, Samer Hanoudi, Rebecca Tagett, Michele Donato, Călin Voichiţa, Sorin Drăghici</div> <em>Frontiers in physiology</em> (2013-10-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24133454">https://www.ncbi.nlm.nih.gov/pubmed/24133454</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3389/fphys.2013.00278">10.3389/fphys.2013.00278</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24133454">24133454</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3794382">PMC3794382</a></div></div>
</div>
<div id="ref-1BeN7jMiD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">565. </div><div class="csl-right-inline"><strong>Faculty Opinions recommendation of Gene co-expression analysis for functional classification and gene-disease predictions.</strong> <div class="csl-block">Naoki Masuda</div> <em>Faculty Opinions – Post-Publication Peer Review of the Biomedical Literature</em> (2021-11-13) <a href="https://doi.org/grmckm">https://doi.org/grmckm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3410/f.727201931.793589848">10.3410/f.727201931.793589848</a></div></div>
</div>
<div id="ref-15Ki75h8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">566. </div><div class="csl-right-inline"><strong>Robust gene coexpression networks using signed distance correlation</strong> <div class="csl-block">Javier Pardo-Diaz, Lyuba V Bozhilova, Mariano Beguerisse-Díaz, Philip S Poole, Charlotte M Deane, Gesine Reinert</div> <em>Cold Spring Harbor Laboratory</em> (2020-06-22) <a href="https://doi.org/grmckj">https://doi.org/grmckj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2020.06.21.163543">10.1101/2020.06.21.163543</a></div></div>
</div>
<div id="ref-fXCPx62A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">567. </div><div class="csl-right-inline"><strong>Community detection in graphs</strong> <div class="csl-block">Santo Fortunato</div> <em>Physics Reports</em> (2010-02) <a href="https://doi.org/b48x7f">https://doi.org/b48x7f</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.physrep.2009.11.002">10.1016/j.physrep.2009.11.002</a></div></div>
</div>
<div id="ref-eZTqkAjM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">568. </div><div class="csl-right-inline"><strong>Fundamental differences</strong> <div class="csl-block">Peter W Gunning, Edna C Hardeman</div> <em>eLife</em> (2018-02-01) <a href="https://doi.org/10.7554/eLife.34477">https://doi.org/10.7554/eLife.34477</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7554/eLife.34477">10.7554/elife.34477</a></div></div>
</div>
<div id="ref-kBc66YMI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">569. </div><div class="csl-right-inline"><strong>Systematic detection of functional proteoform groups from bottom-up proteomic datasets</strong> <div class="csl-block">Isabell Bludau, Max Frank, Christian Dörig, Yujia Cai, Moritz Heusel, George Rosenberger, Paola Picotti, Ben C Collins, Hannes Röst, Ruedi Aebersold</div> <em>Cold Spring Harbor Laboratory</em> (2020-12-22) <a href="https://doi.org/grmckk">https://doi.org/grmckk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2020.12.22.423928">10.1101/2020.12.22.423928</a></div></div>
</div>
<div id="ref-YKUlYryb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">570. </div><div class="csl-right-inline"><strong>Peptide Correlation Analysis (PeCorA) Reveals Differential Proteoform Regulation.</strong> <div class="csl-block">Maria Dermit, Trenton M Peters-Clarke, Evgenia Shishkova, Jesse G Meyer</div> <em>Journal of proteome research</em> (2020-12-16) <a href="https://www.ncbi.nlm.nih.gov/pubmed/33325715">https://www.ncbi.nlm.nih.gov/pubmed/33325715</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.0c00602">10.1021/acs.jproteome.0c00602</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33325715">33325715</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8592057">PMC8592057</a></div></div>
</div>
<div id="ref-vegaUWlG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">571. </div><div class="csl-right-inline"><strong>Overview of protein structural and functional folds.</strong> <div class="csl-block">Peter D Sun, Christine E Foster, Jeffrey C Boyington</div> <em>Current protocols in protein science</em> (2004-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18429251">https://www.ncbi.nlm.nih.gov/pubmed/18429251</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/0471140864.ps1701s35">10.1002/0471140864.ps1701s35</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18429251">18429251</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7162418">PMC7162418</a></div></div>
</div>
<div id="ref-PnnbHO9a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">572. </div><div class="csl-right-inline"><strong>Distance-based protein folding powered by deep learning</strong> <div class="csl-block">Jinbo Xu</div> <em>Proceedings of the National Academy of Sciences</em> (2019-08-09) <a href="https://doi.org/ggr3kr">https://doi.org/ggr3kr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.1821309116">10.1073/pnas.1821309116</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31399549">31399549</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6708335">PMC6708335</a></div></div>
</div>
<div id="ref-yZfcMIwh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">573. </div><div class="csl-right-inline"><strong>Highly accurate protein structure prediction with AlphaFold</strong> <div class="csl-block">John Jumper, Richard Evans, Alexander Pritzel, Tim Green, Michael Figurnov, Olaf Ronneberger, Kathryn Tunyasuvunakool, Russ Bates, Augustin Žídek, Anna Potapenko, … Demis Hassabis</div> <em>Nature</em> (2021-07-15) <a href="https://doi.org/gk7nfp">https://doi.org/gk7nfp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41586-021-03819-2">10.1038/s41586-021-03819-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34265844">34265844</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8371605">PMC8371605</a></div></div>
</div>
<div id="ref-1AwSDaTyL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">574. </div><div class="csl-right-inline"><strong>Accurate prediction of protein structures and interactions using a three-track neural network.</strong> <div class="csl-block">Minkyung Baek, Frank DiMaio, Ivan Anishchenko, Justas Dauparas, Sergey Ovchinnikov, Gyu Rie Lee, Jue Wang, Qian Cong, Lisa N Kinch, RDustin Schaeffer, … David Baker</div> <em>Science (New York, N.Y.)</em> (2021-07-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/34282049">https://www.ncbi.nlm.nih.gov/pubmed/34282049</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.abj8754">10.1126/science.abj8754</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34282049">34282049</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7612213">PMC7612213</a></div></div>
</div>
<div id="ref-tIYr5DF7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">575. </div><div class="csl-right-inline"><strong>D²P²: database of disordered protein predictions.</strong> <div class="csl-block">Matt E Oates, Pedro Romero, Takashi Ishida, Mohamed Ghalwash, Marcin J Mizianty, Bin Xue, Zsuzsanna Dosztányi, Vladimir N Uversky, Zoran Obradovic, Lukasz Kurgan, … Julian Gough</div> <em>Nucleic acids research</em> (2012-11-29) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23203878">https://www.ncbi.nlm.nih.gov/pubmed/23203878</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/gks1226">10.1093/nar/gks1226</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23203878">23203878</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531159">PMC3531159</a></div></div>
</div>
<div id="ref-CRgkwTQA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">576. </div><div class="csl-right-inline"><strong>Molecular docking: a powerful approach for structure-based drug discovery.</strong> <div class="csl-block">Xuan-Yu Meng, Hong-Xing Zhang, Mihaly Mezei, Meng Cui</div> <em>Current computer-aided drug design</em> (2011-06) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21534921">https://www.ncbi.nlm.nih.gov/pubmed/21534921</a> <div class="csl-block">DOI: <a href="https://doi.org/10.2174/157340911795677602">10.2174/157340911795677602</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21534921">21534921</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3151162">PMC3151162</a></div></div>
</div>
<div id="ref-iOQeUEPU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">577. </div><div class="csl-right-inline"><strong>Molecular dynamics simulations: advances and applications</strong> <div class="csl-block">Josep Gelpi, Adam Hospital, Ramón Goñi, Modesto Orozco</div> <em>Advances and Applications in Bioinformatics and Chemistry</em> (2015-11) <a href="https://doi.org/gb9wxp">https://doi.org/gb9wxp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.2147/aabc.s70333">10.2147/aabc.s70333</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26604800">26604800</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4655909">PMC4655909</a></div></div>
</div>
<div id="ref-jcOtxABy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">578. </div><div class="csl-right-inline"><strong>Basic local alignment search tool.</strong> <div class="csl-block">SF Altschul, W Gish, W Miller, EW Myers, DJ Lipman</div> <em>Journal of molecular biology</em> (1990-10-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/2231712">https://www.ncbi.nlm.nih.gov/pubmed/2231712</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0022-2836(05)80360-2">10.1016/s0022-2836(05)80360-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2231712">2231712</a></div></div>
</div>
<div id="ref-daqSjHIW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">579. </div><div class="csl-right-inline"><strong>Learning functional properties of proteins with language models</strong> <div class="csl-block">Serbulent Unsal, Heval Atas, Muammer Albayrak, Kemal Turhan, Aybar C Acar, Tunca Doğan</div> <em>Nature Machine Intelligence</em> (2022-03-21) <a href="https://doi.org/grckz2">https://doi.org/grckz2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s42256-022-00457-9">10.1038/s42256-022-00457-9</a></div></div>
</div>
<div id="ref-nf5EVAMJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">580. </div><div class="csl-right-inline"><strong>Learning the protein language: Evolution, structure, and function</strong> <div class="csl-block">Tristan Bepler, Bonnie Berger</div> <em>Cell Systems</em> (2021-06) <a href="https://doi.org/gqbqpp">https://doi.org/gqbqpp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cels.2021.05.017">10.1016/j.cels.2021.05.017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34139171">34139171</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8238390">PMC8238390</a></div></div>
</div>
<div id="ref-wLlg6SzE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">581. </div><div class="csl-right-inline"><strong>Thermal proteome profiling for interrogating protein interactions.</strong> <div class="csl-block">André Mateus, Nils Kurzawa, Isabelle Becher, Sindhuja Sridharan, Dominic Helm, Frank Stein, Athanasios Typas, Mikhail M Savitski</div> <em>Molecular systems biology</em> (2020-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/32133759">https://www.ncbi.nlm.nih.gov/pubmed/32133759</a> <div class="csl-block">DOI: <a href="https://doi.org/10.15252/msb.20199232">10.15252/msb.20199232</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32133759">32133759</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7057112">PMC7057112</a></div></div>
</div>
<div id="ref-YBdYtUv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">582. </div><div class="csl-right-inline"><strong>Computational prediction of protein–protein binding affinities</strong> <div class="csl-block">Till Siebenmorgen, Martin Zacharias</div> <em>WIREs Computational Molecular Science</em> (2019-10-29) <a href="https://doi.org/grmckf">https://doi.org/grmckf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/wcms.1448">10.1002/wcms.1448</a></div></div>
</div>
<div id="ref-bUq8ZzIL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">583. </div><div class="csl-right-inline"><strong>DEG 10, an update of the database of essential genes that includes both protein-coding genes and noncoding genomic elements: Table 1.</strong> <div class="csl-block">Hao Luo, Yan Lin, Feng Gao, Chun-Ting Zhang, Ren Zhang</div> <em>Nucleic Acids Research</em> (2013-11-15) <a href="https://doi.org/f5shvv">https://doi.org/f5shvv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/gkt1131">10.1093/nar/gkt1131</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24243843">24243843</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3965060">PMC3965060</a></div></div>
</div>
<div id="ref-171wjW4dP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">584. </div><div class="csl-right-inline"><strong>The phenotypic landscape of essential human genes</strong> <div class="csl-block">Luke Funk, Kuan-Chung Su, Jimmy Ly, David Feldman, Avtar Singh, Brittania Moodie, Paul C Blainey, Iain M Cheeseman</div> <em>Cell</em> (2022-11) <a href="https://doi.org/gq68c5">https://doi.org/gq68c5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cell.2022.10.017">10.1016/j.cell.2022.10.017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36347254">36347254</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10482496">PMC10482496</a></div></div>
</div>
<div id="ref-pGJ2GFbE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">585. </div><div class="csl-right-inline"><strong>Transient and stable vector transfection: Pitfalls, off-target effects, artifacts</strong> <div class="csl-block">Aleksei A Stepanenko, Henry H Heng</div> <em>Mutation Research/Reviews in Mutation Research</em> (2017-07) <a href="https://doi.org/gb4hwj">https://doi.org/gb4hwj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.mrrev.2017.05.002">10.1016/j.mrrev.2017.05.002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28927539">28927539</a></div></div>
</div>
<div id="ref-DdiSq79q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">586. </div><div class="csl-right-inline"><strong>Choosing the Right Tool for the Job: RNAi, TALEN, or CRISPR</strong> <div class="csl-block">Michael Boettcher, Michael T McManus</div> <em>Molecular Cell</em> (2015-05) <a href="https://doi.org/f7dvmk">https://doi.org/f7dvmk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.molcel.2015.04.028">10.1016/j.molcel.2015.04.028</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26000843">26000843</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4441801">PMC4441801</a></div></div>
</div>
<div id="ref-15T32hvm6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">587. </div><div class="csl-right-inline"><strong>A Programmable Dual-RNA–Guided DNA Endonuclease in Adaptive Bacterial Immunity</strong> <div class="csl-block">Martin Jinek, Krzysztof Chylinski, Ines Fonfara, Michael Hauer, Jennifer A Doudna, Emmanuelle Charpentier</div> <em>Science</em> (2012-08-17) <a href="https://doi.org/f22dgd">https://doi.org/f22dgd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.1225829">10.1126/science.1225829</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22745249">22745249</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6286148">PMC6286148</a></div></div>
</div>
<div id="ref-hRIRSidU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">588. </div><div class="csl-right-inline"><strong>A glance at genome editing with CRISPR–Cas9 technology</strong> <div class="csl-block">Antara Barman, Bornali Deb, Supriyo Chakraborty</div> <em>Current Genetics</em> (2019-11-05) <a href="https://doi.org/gjrcz5">https://doi.org/gjrcz5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00294-019-01040-3">10.1007/s00294-019-01040-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31691023">31691023</a></div></div>
</div>
<div id="ref-1uwXNnEt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">589. </div><div class="csl-right-inline"><strong>From DNA break repair pathways to CRISPR/Cas-mediated gene knock-in methods</strong> <div class="csl-block">Mahere Rezazade Bazaz, Hesam Dehghani</div> <em>Life Sciences</em> (2022-04) <a href="https://doi.org/gr5j2p">https://doi.org/gr5j2p</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.lfs.2022.120409">10.1016/j.lfs.2022.120409</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35182556">35182556</a></div></div>
</div>
<div id="ref-EPXqJmJj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">590. </div><div class="csl-right-inline"><strong>Gene editing and its applications in biomedicine</strong> <div class="csl-block">Guanglei Li, Xiangyang Li, Songkuan Zhuang, Liren Wang, Yifan Zhu, Yangcan Chen, Wen Sun, Zeguang Wu, Zhuo Zhou, Jia Chen, … Wensheng Wei</div> <em>Science China Life Sciences</em> (2022-02-18) <a href="https://doi.org/gp8msx">https://doi.org/gp8msx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s11427-021-2057-0">10.1007/s11427-021-2057-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35235150">35235150</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8889061">PMC8889061</a></div></div>
</div>
<div id="ref-rUUeN4Sh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">591. </div><div class="csl-right-inline"><strong>A Review on Advanced CRISPR-Based Genome-Editing Tools: Base Editing and Prime Editing</strong> <div class="csl-block">Ali Saber Sichani, Maryam Ranjbar, Maryam Baneshi, Farid Torabi Zadeh, Jafar Fallahi</div> <em>Molecular Biotechnology</em> (2022-12-22) <a href="https://doi.org/gr5j2d">https://doi.org/gr5j2d</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s12033-022-00639-1">10.1007/s12033-022-00639-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36547823">36547823</a></div></div>
</div>
<div id="ref-VFV60BPa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">592. </div><div class="csl-right-inline"><strong>Recent Advances in Genome-Engineering Strategies</strong> <div class="csl-block">Michaela A Boti, Konstantina Athanasopoulou, Panagiotis G Adamopoulos, Diamantis C Sideris, Andreas Scorilas</div> <em>Genes</em> (2023-01-02) <a href="https://doi.org/gr5j2w">https://doi.org/gr5j2w</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/genes14010129">10.3390/genes14010129</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36672870">36672870</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9859587">PMC9859587</a></div></div>
</div>
<div id="ref-sAa18GbX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">593. </div><div class="csl-right-inline"><strong>Latest Developed Strategies to Minimize the Off-Target Effects in CRISPR-Cas-Mediated Genome Editing</strong> <div class="csl-block">Muhammad Naeem, Saman Majeed, Mubasher Zahir Hoque, Irshad Ahmad</div> <em>Cells</em> (2020-07-02) <a href="https://doi.org/gmhjsc">https://doi.org/gmhjsc</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/cells9071608">10.3390/cells9071608</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32630835">32630835</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7407193">PMC7407193</a></div></div>
</div>
<div id="ref-n2p9cQ7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">594. </div><div class="csl-right-inline"><strong>Targeted Protein Degradation Tools: Overview and Future Perspectives</strong> <div class="csl-block">Yuri Prozzillo, Gaia Fattorini, Maria Virginia Santopietro, Luigi Suglia, Alessandra Ruggiero, Diego Ferreri, Giovanni Messina</div> <em>Biology</em> (2020-11-26) <a href="https://doi.org/gmfb42">https://doi.org/gmfb42</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/biology9120421">10.3390/biology9120421</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33256092">33256092</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7761331">PMC7761331</a></div></div>
</div>
<div id="ref-1G6PkBlEo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">595. </div><div class="csl-right-inline"><strong>Tet-On Systems For Doxycycline-inducible Gene Expression</strong> <div class="csl-block">Atze T. Das, Liliane Tenenbaum, Ben Berkhout</div> <em>Current Gene Therapy</em> (2016-06-16) <a href="https://doi.org/f8sfzz">https://doi.org/f8sfzz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.2174/1566523216666160524144041">10.2174/1566523216666160524144041</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27216914">27216914</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5070417">PMC5070417</a></div></div>
</div>
<div id="ref-6NVK2tsy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">596. </div><div class="csl-right-inline"><strong>How to Choose the Right Inducible Gene Expression System for Mammalian Studies?</strong> <div class="csl-block">Kallunki, Barisic, Jäättelä, Liu</div> <em>Cells</em> (2019-07-30) <a href="https://doi.org/gr5j2v">https://doi.org/gr5j2v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/cells8080796">10.3390/cells8080796</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31366153">31366153</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6721553">PMC6721553</a></div></div>
</div>
<div id="ref-idh43eOQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">597. </div><div class="csl-right-inline"><strong>Rapid Creation of Stable Mammalian Cell Lines for Regulated Expression of Proteins Using the Gateway® Recombination Cloning Technology and Flp-In T-REx® Lines</strong> <div class="csl-block">Jessica Spitzer, Markus Landthaler, Thomas Tuschl</div> <em>Methods in Enzymology</em> (2013) <a href="https://doi.org/f5fkw5">https://doi.org/f5fkw5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/b978-0-12-418687-3.00008-2">10.1016/b978-0-12-418687-3.00008-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24011039">24011039</a></div></div>
</div>
<div id="ref-S1IrclG6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">598. </div><div class="csl-right-inline"><strong>Using the Flp-In™ T-Rex™ System to Regulate GPCR Expression</strong> <div class="csl-block">Richard J Ward, Elisa Alvarez-Curto, Graeme Milligan</div> <em>Methods in Molecular Biology</em> (2011) <a href="https://doi.org/dsvq7p">https://doi.org/dsvq7p</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-61779-126-0_2">10.1007/978-1-61779-126-0_2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21607850">21607850</a></div></div>
</div>
<div id="ref-BwuqP6B4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">599. </div><div class="csl-right-inline"><strong>The Art of Validating Quantitative Proteomics Data</strong> <div class="csl-block">David C Handler, Dana Pascovici, Mehdi Mirzaei, Vivek Gupta, Ghasem Hosseini Salekdeh, Paul A Haynes</div> <em>PROTEOMICS</em> (2018-11-25) <a href="https://doi.org/gr5j2b">https://doi.org/gr5j2b</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.201800222">10.1002/pmic.201800222</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30352137">30352137</a></div></div>
</div>
<div id="ref-gFNDbBK5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">600. </div><div class="csl-right-inline"><strong>Pulse‐Chase Analysis for Studying Protein Synthesis and Maturation</strong> <div class="csl-block">Susanne Fritzsche, Sebastian Springer</div> <em>Current Protocols in Protein Science</em> (2014-11) <a href="https://doi.org/gr5jz9">https://doi.org/gr5jz9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/0471140864.ps3003s78">10.1002/0471140864.ps3003s78</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25367008">25367008</a></div></div>
</div>
<div id="ref-TJFHVsUt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">601. </div><div class="csl-right-inline"><strong>Degradation Parameters from Pulse-Chase Experiments</strong> <div class="csl-block">Celine Sin, Davide Chiarugi, Angelo Valleriani</div> <em>PLOS ONE</em> (2016-05-16) <a href="https://doi.org/gbmt2f">https://doi.org/gbmt2f</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pone.0155028">10.1371/journal.pone.0155028</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27182698">27182698</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4868333">PMC4868333</a></div></div>
</div>
<div id="ref-T6IYjbu4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">602. </div><div class="csl-right-inline"><strong>Mass Spectrometry Methods for Measuring Protein Stability</strong> <div class="csl-block">Daniel D Vallejo, Carolina Rojas Ramírez, Kristine F Parson, Yilin Han, Varun V Gadkari, Brandon T Ruotolo</div> <em>Chemical Reviews</em> (2022-03-22) <a href="https://doi.org/gr5j2q">https://doi.org/gr5j2q</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.chemrev.1c00857">10.1021/acs.chemrev.1c00857</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35316030">35316030</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9197173">PMC9197173</a></div></div>
</div>
<div id="ref-fYmIhIvC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">603. </div><div class="csl-right-inline"><strong>New Developments in Quantitative Real-time Polymerase Chain Reaction Technology</strong> <div class="csl-block">Current Issues in Molecular Biology</div> (2014) <a href="https://doi.org/gr5j2t">https://doi.org/gr5j2t</a> <div class="csl-block">DOI: <a href="https://doi.org/10.21775/cimb.016.001">10.21775/cimb.016.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23562919">23562919</a></div></div>
</div>
<div id="ref-9ByCq3fI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">604. </div><div class="csl-right-inline"><strong>RNA-Seq: a revolutionary tool for transcriptomics</strong> <div class="csl-block">Zhong Wang, Mark Gerstein, Michael Snyder</div> <em>Nature Reviews Genetics</em> (2009-01) <a href="https://doi.org/crrz3v">https://doi.org/crrz3v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nrg2484">10.1038/nrg2484</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19015660">19015660</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2949280">PMC2949280</a></div></div>
</div>
<div id="ref-1Hf5Cuj4t" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">605. </div><div class="csl-right-inline"><strong>Development and Applications of Superfolder and Split Fluorescent Protein Detection Systems in Biology</strong> <div class="csl-block">Jean-Denis Pedelacq, Stéphanie Cabantous</div> <em>International Journal of Molecular Sciences</em> (2019-07-15) <a href="https://doi.org/ggks55">https://doi.org/ggks55</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/ijms20143479">10.3390/ijms20143479</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31311175">31311175</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6678664">PMC6678664</a></div></div>
</div>
<div id="ref-UJonPw7t" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">606. </div><div class="csl-right-inline"><strong>Analysis of protein-protein proximities using the split-ubiquitin system</strong> <div class="csl-block">N Lehming</div> <em>Briefings in Functional Genomics and Proteomics</em> (2002-01-01) <a href="https://doi.org/b9btsv">https://doi.org/b9btsv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bfgp/1.3.230">10.1093/bfgp/1.3.230</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15239890">15239890</a></div></div>
</div>
<div id="ref-btMVdgTk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">607. </div><div class="csl-right-inline"><strong>Split-luciferase complementary assay: applications, recent developments, and future perspectives</strong> <div class="csl-block">Taha Azad, Amin Tashakor, Saman Hosseinkhani</div> <em>Analytical and Bioanalytical Chemistry</em> (2014-07-08) <a href="https://doi.org/f6gkk9">https://doi.org/f6gkk9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00216-014-7980-8">10.1007/s00216-014-7980-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25002334">25002334</a></div></div>
</div>
<div id="ref-zrB7Yy5d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">608. </div><div class="csl-right-inline"><strong>Tobacco Etch Virus protease: A shortcut across biotechnologies</strong> <div class="csl-block">Francesca Cesaratto, Oscar R Burrone, Gianluca Petris</div> <em>Journal of Biotechnology</em> (2016-08) <a href="https://doi.org/gr5j2k">https://doi.org/gr5j2k</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jbiotec.2016.06.012">10.1016/j.jbiotec.2016.06.012</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27312702">27312702</a></div></div>
</div>
<div id="ref-hPao2o3e" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">609. </div><div class="csl-right-inline"><strong>A split β-lactamase sensor for the detection of DNA modification by cisplatin and ruthenium-based chemotherapeutic drugs</strong> <div class="csl-block">Samuel R Hinton, Elizabeth LS Corpuz, Karen L McFarlane Holman, Scott C Meyer</div> <em>Journal of Inorganic Biochemistry</em> (2022-11) <a href="https://doi.org/gr5j2m">https://doi.org/gr5j2m</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jinorgbio.2022.111986">10.1016/j.jinorgbio.2022.111986</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36084568">36084568</a></div></div>
</div>
<div id="ref-LY7mqhu6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">610. </div><div class="csl-right-inline"><strong>Detecting Protein-Protein Interaction Based on Protein Fragment Complementation Assay</strong> <div class="csl-block">Tianwen Wang, Ningning Yang, Chen Liang, Hongjv Xu, Yafei An, Sha Xiao, Mengyuan Zheng, Lu Liu, Gaozhan Wang, Lei Nie</div> <em>Current Protein &amp;amp; Peptide Science</em> (2020-08-21) <a href="https://doi.org/gr5j2s">https://doi.org/gr5j2s</a> <div class="csl-block">DOI: <a href="https://doi.org/10.2174/1389203721666200213102829">10.2174/1389203721666200213102829</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32053071">32053071</a></div></div>
</div>
<div id="ref-18Csai5qq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">611. </div><div class="csl-right-inline"><strong>Technologies for the identification and validation of protein-protein interactions</strong> <div class="csl-block">Karoline Pichlerova, Jozef Hanes</div> <em>General physiology and biophysics</em> (2021) <a href="https://doi.org/gr5j2z">https://doi.org/gr5j2z</a> <div class="csl-block">DOI: <a href="https://doi.org/10.4149/gpb_2021035">10.4149/gpb_2021035</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34897023">34897023</a></div></div>
</div>
<div id="ref-19CwS2HX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">612. </div><div class="csl-right-inline"><strong>A novel genetic system to detect protein–protein interactions</strong> <div class="csl-block">Stanley Fields, Ok-kyu Song</div> <em>Nature</em> (1989-07) <a href="https://doi.org/c3ntbc">https://doi.org/c3ntbc</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/340245a0">10.1038/340245a0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2547163">2547163</a></div></div>
</div>
<div id="ref-2PC5fx0I" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">613. </div><div class="csl-right-inline"><strong>Yeast Two-Hybrid, a Powerful Tool for Systems Biology</strong> <div class="csl-block">Anna Brückner, Cécile Polge, Nicolas Lentze, Daniel Auerbach, Uwe Schlattner</div> <em>International Journal of Molecular Sciences</em> (2009-06-18) <a href="https://doi.org/d45mqr">https://doi.org/d45mqr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/ijms10062763">10.3390/ijms10062763</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19582228">19582228</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705515">PMC2705515</a></div></div>
</div>
<div id="ref-1DnPkHRsA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">614. </div><div class="csl-right-inline"><strong>The mammalian two-hybrid system as a powerful tool for high-throughput drug screening</strong> <div class="csl-block">Daniela Patrício, Margarida Fardilha</div> <em>Drug Discovery Today</em> (2020-04) <a href="https://doi.org/gr5j2j">https://doi.org/gr5j2j</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.drudis.2020.01.022">10.1016/j.drudis.2020.01.022</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32032707">32032707</a></div></div>
</div>
<div id="ref-1G5hcMdry" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">615. </div><div class="csl-right-inline"><strong>Isolation of Endogenously Assembled RNA-Protein Complexes Using Affinity Purification Based on Streptavidin Aptamer S1</strong> <div class="csl-block">Yangchao Dong, Jing Yang, Wei Ye, Yuan Wang, Chuantao Ye, Daihui Weng, Huan Gao, Fanglin Zhang, Zhikai Xu, Yingfeng Lei</div> <em>International Journal of Molecular Sciences</em> (2015-09-16) <a href="https://doi.org/f7xz9m">https://doi.org/f7xz9m</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/ijms160922456">10.3390/ijms160922456</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26389898">26389898</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4613318">PMC4613318</a></div></div>
</div>
<div id="ref-636RraOs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">616. </div><div class="csl-right-inline"><strong>FRET as a biomolecular research tool — understanding its potential while avoiding pitfalls</strong> <div class="csl-block">WRuss Algar, Niko Hildebrandt, Steven S Vogel, Igor L Medintz</div> <em>Nature Methods</em> (2019-08-30) <a href="https://doi.org/ggdxxw">https://doi.org/ggdxxw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-019-0530-8">10.1038/s41592-019-0530-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31471616">31471616</a></div></div>
</div>
<div id="ref-zTlZpc1k" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">617. </div><div class="csl-right-inline"><strong>Split-protein systems: beyond binary protein–protein interactions</strong> <div class="csl-block">Sujan S Shekhawat, Indraneel Ghosh</div> <em>Current Opinion in Chemical Biology</em> (2011-12) <a href="https://doi.org/cpvhs8">https://doi.org/cpvhs8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cbpa.2011.10.014">10.1016/j.cbpa.2011.10.014</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22070901">22070901</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3237955">PMC3237955</a></div></div>
</div>
<div id="ref-13oBuESdH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">618. </div><div class="csl-right-inline"><strong>Proximity ligation assay: an ultrasensitive method for protein quantification and its applications in pathogen detection</strong> <div class="csl-block">Pengzhi Wang, Yi Yang, Tianqi Hong, Guoqiang Zhu</div> <em>Applied Microbiology and Biotechnology</em> (2021-01-11) <a href="https://doi.org/gr5j2c">https://doi.org/gr5j2c</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00253-020-11049-1">10.1007/s00253-020-11049-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33427935">33427935</a></div></div>
</div>
<div id="ref-feT5EHvH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">619. </div><div class="csl-right-inline"><strong>Understanding FRET as a Research Tool for Cellular Studies</strong> <div class="csl-block">Dilip Shrestha, Attila Jenei, Péter Nagy, György Vereb, János Szöllősi</div> <em>International Journal of Molecular Sciences</em> (2015-03-25) <a href="https://doi.org/f7cbff">https://doi.org/f7cbff</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/ijms16046718">10.3390/ijms16046718</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25815593">25815593</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4424985">PMC4424985</a></div></div>
</div>
<div id="ref-o1iijZBV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">620. </div><div class="csl-right-inline"><strong>Illuminating insights into protein-protein interactions using bioluminescence resonance energy transfer (BRET)</strong> <div class="csl-block">Kevin DG Pfleger, Karin A Eidne</div> <em>Nature Methods</em> (2006-02-17) <a href="https://doi.org/fq5kjq">https://doi.org/fq5kjq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth841">10.1038/nmeth841</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16489332">16489332</a></div></div>
</div>
<div id="ref-168qPRxga" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">621. </div><div class="csl-right-inline"><strong>Developments in FRET- and BRET-Based Biosensors</strong> <div class="csl-block">Yuexin Wu, Tianyu Jiang</div> <em>Micromachines</em> (2022-10-20) <a href="https://doi.org/gr5j2x">https://doi.org/gr5j2x</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/mi13101789">10.3390/mi13101789</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36296141">36296141</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9610962">PMC9610962</a></div></div>
</div>
<div id="ref-6egqfZ9W" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">622. </div><div class="csl-right-inline"><strong>Fluorescence correlation spectroscopy as a tool for the study of the intracellular dynamics and biological fate of protein corona</strong> <div class="csl-block">Marta Martinez-Moro, Desiré Di Silvio, Sergio E Moya</div> <em>Biophysical Chemistry</em> (2019-10) <a href="https://doi.org/gr5j2f">https://doi.org/gr5j2f</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.bpc.2019.106218">10.1016/j.bpc.2019.106218</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31325709">31325709</a></div></div>
</div>
<div id="ref-ZhoKgqsC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">623. </div><div class="csl-right-inline"><strong>Proximity labeling methods for proteomic analysis of membrane proteins</strong> <div class="csl-block">Francisco Zafra, Dolores Piniella</div> <em>Journal of Proteomics</em> (2022-07) <a href="https://doi.org/gr5j2n">https://doi.org/gr5j2n</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jprot.2022.104620">10.1016/j.jprot.2022.104620</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35598870">35598870</a></div></div>
</div>
<div id="ref-RXiPqFRh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">624. </div><div class="csl-right-inline"><strong>Research Techniques Made Simple: Emerging Methods to Elucidate Protein Interactions through Spatial Proximity</strong> <div class="csl-block">Yonglu Che, Paul A Khavari</div> <em>Journal of Investigative Dermatology</em> (2017-12) <a href="https://doi.org/gcmmkp">https://doi.org/gcmmkp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jid.2017.09.028">10.1016/j.jid.2017.09.028</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29169465">29169465</a></div></div>
</div>
<div id="ref-1F8ZvcEeH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">625. </div><div class="csl-right-inline"><strong>Analysis of Protein Interactions in situ by Proximity Ligation Assays</strong> <div class="csl-block">Björn Koos, Linda Andersson, Carl-Magnus Clausson, Karin Grannas, Axel Klaesson, Gaëlle Cane, Ola Söderberg</div> <em>Current Topics in Microbiology and Immunology</em> (2013) <a href="https://doi.org/f25nfj">https://doi.org/f25nfj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/82_2013_334">10.1007/82_2013_334</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23921974">23921974</a></div></div>
</div>
<div id="ref-3LymzGVH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">626. </div><div class="csl-right-inline"><strong>Cross-linking and mass spectrometry as a tool for studying the structural biology of ribonucleoproteins</strong> <div class="csl-block">Chris P Sarnowski, Maria Bikaki, Alexander Leitner</div> <em>Structure</em> (2022-04) <a href="https://doi.org/gr5jz6">https://doi.org/gr5jz6</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.str.2022.03.003">10.1016/j.str.2022.03.003</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35366400">35366400</a></div></div>
</div>
<div id="ref-yxw6FRju" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">627. </div><div class="csl-right-inline"><strong>Protein–RNA interactions: from mass spectrometry to drug discovery</strong> <div class="csl-block">Benjamin Steinmetz, Izabela Smok, Maria Bikaki, Alexander Leitner</div> <em>Essays in Biochemistry</em> (2023-03) <a href="https://doi.org/gr5jz8">https://doi.org/gr5jz8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1042/ebc20220177">10.1042/ebc20220177</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36866608">36866608</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10070478">PMC10070478</a></div></div>
</div>
<div id="ref-4T3MaN5d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">628. </div><div class="csl-right-inline"><strong>Anatomy of a crosslinker</strong> <div class="csl-block">Adam Belsom, Juri Rappsilber</div> <em>Current Opinion in Chemical Biology</em> (2021-02) <a href="https://doi.org/gk6ddr">https://doi.org/gk6ddr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cbpa.2020.07.008">10.1016/j.cbpa.2020.07.008</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32829152">32829152</a></div></div>
</div>
<div id="ref-H3H5bwtx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">629. </div><div class="csl-right-inline"><strong>Analysis of Protein Interactions by Surface Plasmon Resonance</strong> <div class="csl-block">Dennis G Drescher, Dakshnamurthy Selvakumar, Marian J Drescher</div> <em>Protein-Protein Interactions in Human Disease, Part A</em> (2018) <a href="https://doi.org/gr5jz7">https://doi.org/gr5jz7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/bs.apcsb.2017.07.003">10.1016/bs.apcsb.2017.07.003</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29412994">29412994</a></div></div>
</div>
<div id="ref-rNop1hIz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">630. </div><div class="csl-right-inline"><strong>Comprehensive liquid chromatography: Fundamental aspects and practical considerations—A review</strong> <div class="csl-block">Isabelle François, Koen Sandra, Pat Sandra</div> <em>Analytica Chimica Acta</em> (2009-05) <a href="https://doi.org/b8mj7m">https://doi.org/b8mj7m</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.aca.2009.03.041">10.1016/j.aca.2009.03.041</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19393362">19393362</a></div></div>
</div>
<div id="ref-71TlBiJy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">631. </div><div class="csl-right-inline"><strong>Ion Suppression in Mass Spectrometry</strong> <div class="csl-block">Thomas M Annesley</div> <em>Clinical Chemistry</em> (2003-07-01) <a href="https://doi.org/dh9v2v">https://doi.org/dh9v2v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1373/49.7.1041">10.1373/49.7.1041</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12816898">12816898</a></div></div>
</div>
<div id="ref-pRj1odLy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">632. </div><div class="csl-right-inline"><strong>Advanced proteomic liquid chromatography</strong> <div class="csl-block">Fang Xie, Richard D Smith, Yufeng Shen</div> <em>Journal of Chromatography A</em> (2012-10) <a href="https://doi.org/gsr7w3">https://doi.org/gsr7w3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.chroma.2012.06.098">10.1016/j.chroma.2012.06.098</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22840822">22840822</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3463731">PMC3463731</a></div></div>
</div>
<div id="ref-T9Yv2Omz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">633. </div><div class="csl-right-inline"><strong>Multi-dimensional liquid chromatography in proteomics—A review</strong> <div class="csl-block">Xiang Zhang, Aiqin Fang, Catherine P Riley, Mu Wang, Fred E Regnier, Charles Buck</div> <em>Analytica Chimica Acta</em> (2010-04) <a href="https://doi.org/fsb5nk">https://doi.org/fsb5nk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.aca.2010.02.001">10.1016/j.aca.2010.02.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20363391">20363391</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2852180">PMC2852180</a></div></div>
</div>
<div id="ref-WG5jq46X" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">634. </div><div class="csl-right-inline"><strong>The role of liquid chromatography in proteomics</strong> <div class="csl-block">Y SHI, R XIANG, C HORVATH, J WILKINS</div> <em>Journal of Chromatography A</em> (2004-10-22) <a href="https://doi.org/fq93q6">https://doi.org/fq93q6</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.chroma.2004.07.044">10.1016/j.chroma.2004.07.044</a></div></div>
</div>
<div id="ref-hwO7hkQN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">635. </div><div class="csl-right-inline"><strong>More Sensitive and Quantitative Proteomic Measurements Using Very Low Flow Rate Porous Silica Monolithic LC Columns with Electrospray Ionization-Mass Spectrometry</strong> <div class="csl-block">Quanzhou Luo, Keqi Tang, Feng Yang, Ayesha Elias, Yufeng Shen, Ronald J Moore, Rui Zhao, Kim K Hixson, Sandra S Rossie, Richard D Smith</div> <em>Journal of Proteome Research</em> (2006-04-01) <a href="https://doi.org/bf97wg">https://doi.org/bf97wg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr050424y">10.1021/pr050424y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16674098">16674098</a></div></div>
</div>
<div id="ref-156fxE3m7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">636. </div><div class="csl-right-inline"><strong>Longitudinal diffusion and resistance to mass transfer as causes of nonideality in chromatography</strong> <div class="csl-block">JJ van Deemter, FJ Zuiderweg, A Klinkenberg</div> <em>Chemical Engineering Science</em> (1956-09-01) <a href="https://www.sciencedirect.com/science/article/pii/0009250956800031">https://www.sciencedirect.com/science/article/pii/0009250956800031</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/0009-2509(56)80003-1">10.1016/0009-2509(56)80003-1</a></div></div>
</div>
<div id="ref-dvtPvLcY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">637. </div><div class="csl-right-inline"><strong>Parallelization with Dual-Trap Single-Column Configuration Maximizes Throughput of Proteomic Analysis</strong> <div class="csl-block">Simion Kreimer, Ali Haghani, Aleksandra Binek, Alisse Hauspurg, Saeed Seyedmohammad, Alejandro Rivas, Amanda Momenzadeh, Jesse G Meyer, Koen Raedschelders, Jennifer E Van Eyk</div> <em>Analytical Chemistry</em> (2022-08-31) <a href="https://doi.org/gsr77d">https://doi.org/gsr77d</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.2c02609">10.1021/acs.analchem.2c02609</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36044770">36044770</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9900495">PMC9900495</a></div></div>
</div>
<div id="ref-L2sT906m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">638. </div><div class="csl-right-inline"><strong>High-Throughput Single-Cell Proteomic Analysis of Organ-Derived Heterogeneous Cell Populations by Nanoflow Dual-Trap Single-Column Liquid Chromatography</strong> <div class="csl-block">Simion Kreimer, Aleksandra Binek, Blandine Chazarin, Jae Hyung Cho, Ali Haghani, Alexandre Hutton, Eduardo Marbán, Mitra Mastali, Jesse G Meyer, Thassio Mesquita, … Sarah Parker</div> <em>Analytical Chemistry</em> (2023-06-08) <a href="https://doi.org/gsr77f">https://doi.org/gsr77f</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.3c00213">10.1021/acs.analchem.3c00213</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37289937">37289937</a></div></div>
</div>
<div id="ref-1Gtuii4F4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">639. </div><div class="csl-right-inline"><strong>“Orthogonal” separations for reversed-phase liquid chromatography</strong> <div class="csl-block">Jackson Pellett, Patrick Lukulay, Yun Mao, William Bowen, Robert Reed, M Ma, RC Munger, JW Dolan, Loren Wrisley, K Medwid, … LR Snyder</div> <em>Journal of Chromatography A</em> (2006-01) <a href="https://doi.org/c76zht">https://doi.org/c76zht</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.chroma.2005.09.080">10.1016/j.chroma.2005.09.080</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16236292">16236292</a></div></div>
</div>
<div id="ref-1FnaHa1s9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">640. </div><div class="csl-right-inline"><strong>Large-scale analysis of the yeast proteome by multidimensional protein identification technology</strong> <div class="csl-block">Michael P Washburn, Dirk Wolters, John R Yates III</div> <em>Nature Biotechnology</em> (2001-03) <a href="https://doi.org/cpwfkn">https://doi.org/cpwfkn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/85686">10.1038/85686</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11231557">11231557</a></div></div>
</div>
<div id="ref-dl5RDy6j" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">641. </div><div class="csl-right-inline"><strong>Reversed-phase chromatography with multiple fraction concatenation strategy for proteome profiling of human MCF10A cells</strong> <div class="csl-block">Yuexi Wang, Feng Yang, Marina A Gritsenko, Yingchun Wang, Therese Clauss, Tao Liu, Yufeng Shen, Matthew E Monroe, Daniel Lopez-Ferrer, Theresa Reno, … Richard D Smith</div> <em>PROTEOMICS</em> (2011-04-18) <a href="https://doi.org/c54ggx">https://doi.org/c54ggx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.201000722">10.1002/pmic.201000722</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21500348">21500348</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3120047">PMC3120047</a></div></div>
</div>
<div id="ref-qAVrmvwM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">642. </div><div class="csl-right-inline"><strong>An Introduction to Mass Spectrometry-Based Proteomics</strong> <div class="csl-block">Steven R Shuken</div> <em>Journal of Proteome Research</em> (2023-06-01) <a href="https://doi.org/gsr77x">https://doi.org/gsr77x</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.2c00838">10.1021/acs.jproteome.2c00838</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37260118">37260118</a></div></div>
</div>
<div id="ref-V3WGOS1f" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">643. </div><div class="csl-right-inline"><strong>Protein Analysis by Shotgun/Bottom-up Proteomics</strong> <div class="csl-block">Yaoyang Zhang, Bryan R Fonslow, Bing Shan, Moon-Chang Baek, John R Yates III</div> <em>Chemical Reviews</em> (2013-02-26) <a href="https://doi.org/ggkd3r">https://doi.org/ggkd3r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/cr3003533">10.1021/cr3003533</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23438204">23438204</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3751594">PMC3751594</a></div></div>
</div>
<div id="ref-1DSKc93ie" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">644. </div><div class="csl-right-inline"><strong>Periodicity of Monoisotopic Mass Isomers and Isobars in Proteomics</strong> <div class="csl-block">Long Yu, Yan-Mei Xiong, Nick C Polfer</div> <em>Analytical Chemistry</em> (2011-09-20) <a href="https://doi.org/dbf94b">https://doi.org/dbf94b</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac201624t">10.1021/ac201624t</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21932815">21932815</a></div></div>
</div>
<div id="ref-2MR1Pihs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">645. </div><div class="csl-right-inline"><strong>Protein sequencing by tandem mass spectrometry.</strong> <div class="csl-block">DF Hunt, JR Yates 3rd, J Shabanowitz, S Winston, CR Hauer</div> <em>Proceedings of the National Academy of Sciences</em> (1986-09) <a href="https://doi.org/bhq2fw">https://doi.org/bhq2fw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.83.17.6233">10.1073/pnas.83.17.6233</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3462691">3462691</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC386476">PMC386476</a></div></div>
</div>
<div id="ref-zCNyqvc9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">646. </div><div class="csl-right-inline"><strong>DirectMS1: MS/MS-Free Identification of 1000 Proteins of Cellular Proteomes in 5 Minutes</strong> <div class="csl-block">Mark V Ivanov, Julia A Bubis, Vladimir Gorshkov, Irina A Tarasova, Lev I Levitsky, Anna A Lobas, Elizaveta M Solovyeva, Marina L Pridatchenko, Frank Kjeldsen, Mikhail V Gorshkov</div> <em>Analytical Chemistry</em> (2020-02-20) <a href="https://doi.org/gsr77v">https://doi.org/gsr77v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.9b05095">10.1021/acs.analchem.9b05095</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32077687">32077687</a></div></div>
</div>
<div id="ref-175VGosTe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">647. </div><div class="csl-right-inline"><strong>Robust Algorithm for Alignment of Liquid Chromatography−Mass Spectrometry Analyses in an Accurate Mass and Time Tag Data Analysis Pipeline</strong> <div class="csl-block">Navdeep Jaitly, Matthew E Monroe, Vladislav A Petyuk, Therese RW Clauss, Joshua N Adkins, Richard D Smith</div> <em>Analytical Chemistry</em> (2006-10-04) <a href="https://doi.org/dhxw24">https://doi.org/dhxw24</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac052197p">10.1021/ac052197p</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17073405">17073405</a></div></div>
</div>
<div id="ref-ylZpW7Vi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">648. </div><div class="csl-right-inline"><strong>BoxCar acquisition method enables single-shot proteomics at a depth of 10,000 proteins in 100 minutes</strong> <div class="csl-block">Florian Meier, Philipp E Geyer, Sebastian Virreira Winter, Juergen Cox, Matthias Mann</div> <em>Nature Methods</em> (2018-05-07) <a href="https://doi.org/gdqs9d">https://doi.org/gdqs9d</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-018-0003-5">10.1038/s41592-018-0003-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29735998">29735998</a></div></div>
</div>
<div id="ref-mZdxW2jk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">649. </div><div class="csl-right-inline"><strong>Letter to the editors</strong> <div class="csl-block">P Roepstorff, J Fohlman</div> <em>Biological Mass Spectrometry</em> (1984-11) <a href="https://doi.org/fh4rbm">https://doi.org/fh4rbm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/bms.1200111109">10.1002/bms.1200111109</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/6525415">6525415</a></div></div>
</div>
<div id="ref-ivBdKZgo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">650. </div><div class="csl-right-inline"><strong>Mass spectrometry</strong> <div class="csl-block">James A McCloskey</div> <em>Academic Press</em> (1990) <div class="csl-block">ISBN: 9780121820947</div></div>
</div>
<div id="ref-12yIRpFcQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">651. </div><div class="csl-right-inline"><strong>Evolution of ion internal energy during collisional excitation in the Paul ion trap: A stochastic approach</strong> <div class="csl-block">Douglas E Goeringer, Scott A McLuckey</div> <em>The Journal of Chemical Physics</em> (1996-02-08) <a href="https://doi.org/fj8nsp">https://doi.org/fj8nsp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1063/1.471812">10.1063/1.471812</a></div></div>
</div>
<div id="ref-1DIcKpBMl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">652. </div><div class="csl-right-inline"><strong>Higher-energy C-trap dissociation for peptide modification analysis</strong> <div class="csl-block">Jesper V Olsen, Boris Macek, Oliver Lange, Alexander Makarov, Stevan Horning, Matthias Mann</div> <em>Nature Methods</em> (2007-08-26) <a href="https://doi.org/csj7f3">https://doi.org/csj7f3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth1060">10.1038/nmeth1060</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17721543">17721543</a></div></div>
</div>
<div id="ref-CrVLd8Zt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">653. </div><div class="csl-right-inline"><strong>Quantification of the Compositional Information Provided by Immonium Ions on a Quadrupole-Time-of-Flight Mass Spectrometer</strong> <div class="csl-block">Laura J Hohmann, Jimmy K Eng, Andrew Gemmill, John Klimek, Olga Vitek, Gavin E Reid, Daniel B Martin</div> <em>Analytical Chemistry</em> (2008-06-20) <a href="https://doi.org/bjb3rm">https://doi.org/bjb3rm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac8006076">10.1021/ac8006076</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18564857">18564857</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2638499">PMC2638499</a></div></div>
</div>
<div id="ref-Lx0XDSc1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">654. </div><div class="csl-right-inline"><strong>Ion traps in modern mass spectrometry</strong> <div class="csl-block">Dirk Nolting, Robert Malek, Alexander Makarov</div> <em>Mass Spectrometry Reviews</em> (2017-10-30) <a href="https://doi.org/gg2pjd">https://doi.org/gg2pjd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.21549">10.1002/mas.21549</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29084367">29084367</a></div></div>
</div>
<div id="ref-8fRsJhJf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">655. </div><div class="csl-right-inline"><strong>Mathieu's Equation and Its Generalizations: Overview of Stability Charts and Their Features</strong> <div class="csl-block">Ivana Kovacic, Richard Rand, Si Mohamed Sah</div> <em>Applied Mechanics Reviews</em> (2018-02-14) <a href="https://doi.org/ggj6w7">https://doi.org/ggj6w7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1115/1.4039144">10.1115/1.4039144</a></div></div>
</div>
<div id="ref-1DjnN7u2e" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">656. </div><div class="csl-right-inline"><strong>Ion Trap Mass Spectrometers</strong> <div class="csl-block">Raymond E March</div> <em>Encyclopedia of Spectroscopy and Spectrometry</em> (1999) <a href="https://doi.org/bpb2z8">https://doi.org/bpb2z8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1006/rwsp.2000.0143">10.1006/rwsp.2000.0143</a></div></div>
</div>
<div id="ref-16Wowt3CO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">657. </div><div class="csl-right-inline"><strong>Mechanistic Insights into the Multistage Gas-Phase Fragmentation Behavior of Phosphoserine- and Phosphothreonine-Containing Peptides</strong> <div class="csl-block">Amanda M Palumbo, Jetze J Tepe, Gavin E Reid</div> <em>Journal of Proteome Research</em> (2008-01-09) <a href="https://doi.org/d5qsw7">https://doi.org/d5qsw7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr0705136">10.1021/pr0705136</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18181561">18181561</a></div></div>
</div>
<div id="ref-Ud7cg5Ze" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">658. </div><div class="csl-right-inline"><strong>Mobile and localized protons: a framework for understanding peptide dissociation.</strong> <div class="csl-block">VH Wysocki, G Tsaprailis, LL Smith, LA Breci</div> <em>Journal of mass spectrometry : JMS</em> (2000-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/11180630">https://www.ncbi.nlm.nih.gov/pubmed/11180630</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/1096-9888(200012)35:12&lt;1399::aid-jms86&gt;3.0.co;2-r">10.1002/1096-9888(200012)35:12&lt;1399::aid-jms86&gt;3.0.co;2-r</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11180630">11180630</a></div></div>
</div>
<div id="ref-VOcnSdY9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">659. </div><div class="csl-right-inline"><strong>Collisions or Electrons? Protein Sequence Analysis in the 21st Century</strong> <div class="csl-block">Joshua J Coon</div> <em>Analytical Chemistry</em> (2009-04-13) <a href="https://doi.org/dn6rpw">https://doi.org/dn6rpw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac802330b">10.1021/ac802330b</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19364119">19364119</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2714553">PMC2714553</a></div></div>
</div>
<div id="ref-oNkSkkq8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">660. </div><div class="csl-right-inline"><strong>Methods for analyzing peptides and proteins on a chromatographic timescale by electron-transfer dissociation mass spectrometry</strong> <div class="csl-block">Namrata D Udeshi, Philip D Compton, Jeffrey Shabanowitz, Donald F Hunt, Kristie L Rose</div> <em>Nature Protocols</em> (2008-10-16) <a href="https://doi.org/dnnp4j">https://doi.org/dnnp4j</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nprot.2008.159">10.1038/nprot.2008.159</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18927556">18927556</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2860270">PMC2860270</a></div></div>
</div>
<div id="ref-XHvsh4uo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">661. </div><div class="csl-right-inline"><strong>Recent developments and applications of electron transfer dissociation mass spectrometry in proteomics</strong> <div class="csl-block">Mirela Sarbu, Roxana M Ghiulai, Alina D Zamfir</div> <em>Amino Acids</em> (2014-04-01) <a href="https://doi.org/f57nd8">https://doi.org/f57nd8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00726-014-1726-y">10.1007/s00726-014-1726-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24687149">24687149</a></div></div>
</div>
<div id="ref-qWde4x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">662. </div><div class="csl-right-inline"><strong>Electron Capture Dissociation of Multiply Charged Protein Cations. A Nonergodic Process</strong> <div class="csl-block">Roman A Zubarev, Neil L Kelleher, Fred W McLafferty</div> <em>Journal of the American Chemical Society</em> (1998-03-24) <a href="https://doi.org/c886tf">https://doi.org/c886tf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ja973478k">10.1021/ja973478k</a></div></div>
</div>
<div id="ref-OVsIcoNm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">663. </div><div class="csl-right-inline"><strong>Where Does the Electron Go? Electron Distribution and Reactivity of Peptide Cation Radicals Formed by Electron Transfer in the Gas phase</strong> <div class="csl-block">Xiaohong Chen, Changtong Hao</div> <em>Journal of the American Chemical Society</em> (2008-06-14) <a href="https://doi.org/dk2t5c">https://doi.org/dk2t5c</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ja8019005">10.1021/ja8019005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18597436">18597436</a></div></div>
</div>
<div id="ref-xhXJM8vz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">664. </div><div class="csl-right-inline"><strong>Toward a general mechanism of electron capture dissociation</strong> <div class="csl-block">Erik A Syrstad, František Turecček</div> <em>Journal of the American Society for Mass Spectrometry</em> (2005-02-01) <a href="https://doi.org/cqk9pp">https://doi.org/cqk9pp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jasms.2004.11.001">10.1016/j.jasms.2004.11.001</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15694771">15694771</a></div></div>
</div>
<div id="ref-rqc6ScJr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">665. </div><div class="csl-right-inline"><strong>The Role of Electron Transfer Dissociation in Modern Proteomics</strong> <div class="csl-block">Nicholas M Riley, Joshua J Coon</div> <em>Analytical Chemistry</em> (2017-12-12) <a href="https://doi.org/gsr77t">https://doi.org/gsr77t</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.7b04810">10.1021/acs.analchem.7b04810</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29172454">29172454</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5750139">PMC5750139</a></div></div>
</div>
<div id="ref-HNURrSN7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">666. </div><div class="csl-right-inline"><strong>Radical solutions: Principles and application of electron‐based dissociation in mass spectrometry‐based analysis of protein structure</strong> <div class="csl-block">Frederik Lermyte, Dirk Valkenborg, Joseph A Loo, Frank Sobott</div> <em>Mass Spectrometry Reviews</em> (2018-02-09) <a href="https://doi.org/gsr77q">https://doi.org/gsr77q</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.21560">10.1002/mas.21560</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29425406">29425406</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6131092">PMC6131092</a></div></div>
</div>
<div id="ref-UqS9JT46" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">667. </div><div class="csl-right-inline"><strong>Sequencing Grade Tandem Mass Spectrometry for Top–Down Proteomics Using Hybrid Electron Capture Dissociation Methods in a Benchtop Orbitrap Mass Spectrometer</strong> <div class="csl-block">Jared B Shaw, Neha Malhan, Yury V Vasil’ev, Nathan I Lopez, Alexander Makarov, Joseph S Beckman, Valery G Voinov</div> <em>Analytical Chemistry</em> (2018-08-17) <a href="https://doi.org/gfcbtt">https://doi.org/gfcbtt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.8b01901">10.1021/acs.analchem.8b01901</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30118589">30118589</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6604833">PMC6604833</a></div></div>
</div>
<div id="ref-1Gs2JKxdE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">668. </div><div class="csl-right-inline"><strong>Exploring ECD on a Benchtop Q Exactive Orbitrap Mass Spectrometer</strong> <div class="csl-block">Kyle L Fort, Christian N Cramer, Valery G Voinov, Yury V Vasil’ev, Nathan I Lopez, Joseph S Beckman, Albert JR Heck</div> <em>Journal of Proteome Research</em> (2017-12-28) <a href="https://doi.org/gcq6gv">https://doi.org/gcq6gv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.7b00622">10.1021/acs.jproteome.7b00622</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29249155">29249155</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5799867">PMC5799867</a></div></div>
</div>
<div id="ref-Vu5rNLFc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">669. </div><div class="csl-right-inline"><strong>Dissociation of Biomolecules by an Intense Low-Energy Electron Beam in a High Sensitivity Time-of-Flight Mass Spectrometer</strong> <div class="csl-block">Takashi Baba, Pavel Ryumin, Eva Duchoslav, Keqin Chen, Anjali Chelur, Bill Loyd, Igor Chernushevich</div> <em>Journal of the American Society for Mass Spectrometry</em> (2021-06-03) <a href="https://doi.org/gkcwb4">https://doi.org/gkcwb4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/jasms.0c00425">10.1021/jasms.0c00425</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34080873">34080873</a></div></div>
</div>
<div id="ref-eSdFYuAj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">670. </div><div class="csl-right-inline"><strong>Performance Characteristics of Electron Transfer Dissociation Mass Spectrometry</strong> <div class="csl-block">David M Good, Matthew Wirtala, Graeme C McAlister, Joshua J Coon</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2007-11) <a href="https://doi.org/c34s8v">https://doi.org/c34s8v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m700073-mcp200">10.1074/mcp.m700073-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17673454">17673454</a></div></div>
</div>
<div id="ref-qyjSSKs7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">671. </div><div class="csl-right-inline"><strong>Electron-Transfer Ion/Ion Reactions of Doubly Protonated Peptides:  Effect of Elevated Bath Gas Temperature</strong> <div class="csl-block">Sharon J Pitteri, Paul A Chrisman, Scott A McLuckey</div> <em>Analytical Chemistry</em> (2005-08-05) <a href="https://doi.org/fgkb9f">https://doi.org/fgkb9f</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac050666h">10.1021/ac050666h</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16131079">16131079</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1356655">PMC1356655</a></div></div>
</div>
<div id="ref-c8aJxeOj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">672. </div><div class="csl-right-inline"><strong>Supplemental Activation Method for High-Efficiency Electron-Transfer Dissociation of Doubly Protonated Peptide Precursors</strong> <div class="csl-block">Danielle L Swaney, Graeme C McAlister, Matthew Wirtala, Jae C Schwartz, John EP Syka, Joshua J Coon</div> <em>Analytical Chemistry</em> (2006-12-08) <a href="https://doi.org/b8f7sj">https://doi.org/b8f7sj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac061457f">10.1021/ac061457f</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17222010">17222010</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2522365">PMC2522365</a></div></div>
</div>
<div id="ref-TEQyYNF9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">673. </div><div class="csl-right-inline"><strong>On performing simultaneous electron transfer dissociation and collision-induced dissociation on multiply protonated peptides in a linear ion trap</strong> <div class="csl-block">JLarry Campbell, James W Hager, JCYves Le Blanc</div> <em>Journal of the American Society for Mass Spectrometry</em> (2009-09-01) <a href="https://doi.org/ftnx68">https://doi.org/ftnx68</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jasms.2009.05.009">10.1016/j.jasms.2009.05.009</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19539496">19539496</a></div></div>
</div>
<div id="ref-bsAhaObV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">674. </div><div class="csl-right-inline"><strong>Toward Full Peptide Sequence Coverage by Dual Fragmentation Combining Electron-Transfer and Higher-Energy Collision Dissociation Tandem Mass Spectrometry</strong> <div class="csl-block">Christian K Frese, AFMaarten Altelaar, Henk van den Toorn, Dirk Nolting, Jens Griep-Raming, Albert JR Heck, Shabaz Mohammed</div> <em>Analytical Chemistry</em> (2012-10-31) <a href="https://doi.org/gsr77s">https://doi.org/gsr77s</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac3025366">10.1021/ac3025366</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23106539">23106539</a></div></div>
</div>
<div id="ref-oRY0m4xv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">675. </div><div class="csl-right-inline"><strong>Activated ion electron capture dissociation (AI ECD) of proteins: Synchronization of infrared and electron irradiation with ion magnetron motion</strong> <div class="csl-block">Victor A Mikhailov, Helen J Cooper</div> <em>Journal of the American Society for Mass Spectrometry</em> (2009-05-01) <a href="https://doi.org/d4m48s">https://doi.org/d4m48s</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jasms.2008.12.015">10.1016/j.jasms.2008.12.015</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19200749">19200749</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2674148">PMC2674148</a></div></div>
</div>
<div id="ref-F58LowfZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">676. </div><div class="csl-right-inline"><strong>Activated Ion Electron Capture Dissociation for Mass Spectral Sequencing of Larger (42 kDa) Proteins</strong> <div class="csl-block">David M Horn, Ying Ge, Fred W McLafferty</div> <em>Analytical Chemistry</em> (2000-09-21) <a href="https://doi.org/fh8828">https://doi.org/fh8828</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac000494i">10.1021/ac000494i</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11055690">11055690</a></div></div>
</div>
<div id="ref-3mhY8AZ1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">677. </div><div class="csl-right-inline"><strong>Activated Ion Electron Transfer Dissociation for Improved Fragmentation of Intact Proteins</strong> <div class="csl-block">Nicholas M Riley, Michael S Westphall, Joshua J Coon</div> <em>Analytical Chemistry</em> (2015-06-26) <a href="https://doi.org/f7kv7q">https://doi.org/f7kv7q</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.5b00881">10.1021/acs.analchem.5b00881</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26067513">26067513</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9488116">PMC9488116</a></div></div>
</div>
<div id="ref-pXrOdleW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">678. </div><div class="csl-right-inline"><strong>Infrared Photoactivation Reduces Peptide Folding and Hydrogen-Atom Migration following ETD Tandem Mass Spectrometry</strong> <div class="csl-block">Aaron R Ledvina, Graeme C McAlister, Myles W Gardner, Suncerae I Smith, James A Madsen, Jae C Schwartz, George C Stafford Jr., John E P Syka, Jennifer S Brodbelt, Joshua J Coon</div> <em>Angewandte Chemie International Edition</em> (2009-10-26) <a href="https://doi.org/b7wvdw">https://doi.org/b7wvdw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/anie.200903557">10.1002/anie.200903557</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19795429">19795429</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2788484">PMC2788484</a></div></div>
</div>
<div id="ref-1HZ2A5iku" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">679. </div><div class="csl-right-inline"><strong>Optical Fiber-Enabled Photoactivation of Peptides and Proteins</strong> <div class="csl-block">Trenton M Peters-Clarke, Kevin L Schauer, Nicholas M Riley, Jean M Lodge, Michael S Westphall, Joshua J Coon</div> <em>Analytical Chemistry</em> (2020-08-07) <a href="https://doi.org/gssbgx">https://doi.org/gssbgx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.0c02087">10.1021/acs.analchem.0c02087</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32786458">32786458</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8008966">PMC8008966</a></div></div>
</div>
<div id="ref-PWUQDl3j" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">680. </div><div class="csl-right-inline"><strong>Practical Effects of Intramolecular Hydrogen Rearrangement in Electron Transfer Dissociation-Based Proteomics</strong> <div class="csl-block">Trenton M Peters-Clarke, Nicholas M Riley, Michael S Westphall, Joshua J Coon</div> <em>Journal of the American Society for Mass Spectrometry</em> (2021-12-07) <a href="https://doi.org/gss5jm">https://doi.org/gss5jm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/jasms.1c00284">10.1021/jasms.1c00284</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34874726">34874726</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10291708">PMC10291708</a></div></div>
</div>
<div id="ref-1CD0nWWIP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">681. </div><div class="csl-right-inline"><strong>Hybridizing Ultraviolet Photodissociation with Electron Transfer Dissociation for Intact Protein Characterization</strong> <div class="csl-block">Joe R Cannon, Dustin D Holden, Jennifer S Brodbelt</div> <em>Analytical Chemistry</em> (2014-10-13) <a href="https://doi.org/f6n992">https://doi.org/f6n992</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac5036082">10.1021/ac5036082</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25270663">25270663</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4222623">PMC4222623</a></div></div>
</div>
<div id="ref-a9481fQj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">682. </div><div class="csl-right-inline"><strong>Dissociative capture of hot (3–13 eV) electrons by polypeptide polycations: an efficient process accompanied by secondary fragmentation</strong> <div class="csl-block">Frank Kjeldsen, Kim F Haselmann, Bogdan A Budnik, Frank Jensen, Roman A Zubarev</div> <em>Chemical Physics Letters</em> (2002-04-22) <a href="https://www.sciencedirect.com/science/article/pii/S0009261402001495">https://www.sciencedirect.com/science/article/pii/S0009261402001495</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/S0009-2614(02)00149-5">10.1016/s0009-2614(02)00149-5</a></div></div>
</div>
<div id="ref-3Q6WdA4H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">683. </div><div class="csl-right-inline"><strong>Electron Ionization Dissociation of Singly and Multiply Charged Peptides</strong> <div class="csl-block">YMEva Fung, Christopher M Adams, Roman A Zubarev</div> <em>Journal of the American Chemical Society</em> (2009-07-01) <a href="https://doi.org/d62f44">https://doi.org/d62f44</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ja8087407">10.1021/ja8087407</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19621955">19621955</a></div></div>
</div>
<div id="ref-1zm3p02B" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">684. </div><div class="csl-right-inline"><strong>Implementation of Activated Ion Electron Transfer Dissociation on a Quadrupole-Orbitrap-Linear Ion Trap Hybrid Mass Spectrometer</strong> <div class="csl-block">Nicholas M Riley, Michael S Westphall, Alexander S Hebert, Joshua J Coon</div> <em>Analytical Chemistry</em> (2017-04-17) <a href="https://doi.org/f93cx8">https://doi.org/f93cx8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.7b00213">10.1021/acs.analchem.7b00213</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28383247">28383247</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5560271">PMC5560271</a></div></div>
</div>
<div id="ref-M8hC7nxG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">685. </div><div class="csl-right-inline"><strong>Ion Activation Methods for Peptides and Proteins</strong> <div class="csl-block">Luis A Macias, Inês C Santos, Jennifer S Brodbelt</div> <em>Analytical Chemistry</em> (2019-10-30) <a href="https://doi.org/gp6ppg">https://doi.org/gp6ppg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.9b04859">10.1021/acs.analchem.9b04859</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31665881">31665881</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6949381">PMC6949381</a></div></div>
</div>
<div id="ref-pP0K4nse" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">686. </div><div class="csl-right-inline"><strong>Photodissociation mass spectrometry: new tools for characterization of biological molecules</strong> <div class="csl-block">Jennifer S Brodbelt</div> <em>Chem. Soc. Rev.</em> (2014) <a href="https://doi.org/ghh6xb">https://doi.org/ghh6xb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/c3cs60444f">10.1039/c3cs60444f</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24481009">24481009</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3966968">PMC3966968</a></div></div>
</div>
<div id="ref-YqEIuvCO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">687. </div><div class="csl-right-inline"><strong>Ultraviolet Photodissociation Mass Spectrometry for Analysis of Biological Molecules</strong> <div class="csl-block">Jennifer S Brodbelt, Lindsay J Morrison, Inês Santos</div> <em>Chemical Reviews</em> (2019-12-18) <a href="https://doi.org/gmntb8">https://doi.org/gmntb8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.chemrev.9b00440">10.1021/acs.chemrev.9b00440</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31851501">31851501</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7145764">PMC7145764</a></div></div>
</div>
<div id="ref-zA5rBLH4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">688. </div><div class="csl-right-inline"><strong>Applications of Infrared Multiple Photon Dissociation (IRMPD) to the Detection of Posttranslational Modifications</strong> <div class="csl-block">Philippe Maitre, Debora Scuderi, Davide Corinti, Barbara Chiavarino, Maria Elisa Crestoni, Simonetta Fornarini</div> <em>Chemical Reviews</em> (2019-12-06) <a href="https://doi.org/gsr77w">https://doi.org/gsr77w</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.chemrev.9b00395">10.1021/acs.chemrev.9b00395</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31809038">31809038</a></div></div>
</div>
<div id="ref-XXMVxnKl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">689. </div><div class="csl-right-inline"><strong>Infrared multiphoton dissociation (IRMPD) and collisionally activated dissociationof peptides in a quadrupole ion trapwith selective IRMPD of phosphopeptides</strong> <div class="csl-block">Matthew C Crowe, Jennifer S Brodbelt</div> <em>Journal of the American Society for Mass Spectrometry</em> (2004-11-01) <a href="https://doi.org/d6qb5t">https://doi.org/d6qb5t</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jasms.2004.07.016">10.1016/j.jasms.2004.07.016</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15519225">15519225</a></div></div>
</div>
<div id="ref-UC6BHa63" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">690. </div><div class="csl-right-inline"><strong>Fragmentation of Singly Charged Peptide Ions by Photodissociation atλ=157 nm</strong> <div class="csl-block">Matthew S Thompson, Weidong Cui, James P Reilly</div> <em>Angewandte Chemie International Edition</em> (2004) <a href="https://doi.org/ck67xt">https://doi.org/ck67xt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/anie.200460788">10.1002/anie.200460788</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15366089">15366089</a></div></div>
</div>
<div id="ref-145wzrCEm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">691. </div><div class="csl-right-inline"><strong>Ultrafast Ultraviolet Photodissociation at 193 nm and its Applicability to Proteomic Workflows</strong> <div class="csl-block">James A Madsen, Daniel R Boutz, Jennifer S Brodbelt</div> <em>Journal of Proteome Research</em> (2010-07-15) <a href="https://doi.org/bqqprx">https://doi.org/bqqprx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr100515x">10.1021/pr100515x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20578723">20578723</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2917496">PMC2917496</a></div></div>
</div>
<div id="ref-UoKcvT55" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">692. </div><div class="csl-right-inline"><strong>Thorough Performance Evaluation of 213 nm Ultraviolet Photodissociation for Top-down Proteomics</strong> <div class="csl-block">Luca Fornelli, Kristina Srzentić, Timothy K Toby, Peter F Doubleday, Romain Huguet, Christopher Mullen, Rafael D Melani, Henrique dos Santos Seckler, Caroline J DeHart, Chad R Weisbrod, … Neil L Kelleher</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2020-02) <a href="https://doi.org/gsr773">https://doi.org/gsr773</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.tir119.001638">10.1074/mcp.tir119.001638</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31888965">31888965</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7000117">PMC7000117</a></div></div>
</div>
<div id="ref-168MaNzTm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">693. </div><div class="csl-right-inline"><strong>Rapid peptide fragmentation without electrons, collisions, infrared radiation, or native chromophores</strong> <div class="csl-block">Geoffrey K Yeh, Qingyu Sun, Claudia Meneses, Ryan R Julian</div> <em>Journal of the American Society for Mass Spectrometry</em> (2009-03-01) <a href="https://doi.org/fc6krk">https://doi.org/fc6krk</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jasms.2008.10.019">10.1016/j.jasms.2008.10.019</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19036607">19036607</a></div></div>
</div>
<div id="ref-UrNcqFO2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">694. </div><div class="csl-right-inline"><strong>Ultraviolet photodissociation at 266 nm of phosphorylated peptide cations</strong> <div class="csl-block">Soojin Park, Wha-Keun Ahn, Sunyoung Lee, Sang Yun Han, Bum Ku Rhee, Han Bin Oh</div> <em>Rapid Communications in Mass Spectrometry</em> (2009-11-04) <a href="https://doi.org/dqr3cf">https://doi.org/dqr3cf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/rcm.4184">10.1002/rcm.4184</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19890956">19890956</a></div></div>
</div>
<div id="ref-cZOW0Gem" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">695. </div><div class="csl-right-inline"><strong>MS/MS Simplification by 355 nm Ultraviolet Photodissociation of Chromophore-Derivatized Peptides in a Quadrupole Ion Trap</strong> <div class="csl-block">Jeffrey J Wilson, Jennifer S Brodbelt</div> <em>Analytical Chemistry</em> (2007-09-11) <a href="https://doi.org/cbf8kj">https://doi.org/cbf8kj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac071241t">10.1021/ac071241t</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17845006">17845006</a></div></div>
</div>
<div id="ref-1ACL74GmA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">696. </div><div class="csl-right-inline"><strong>Residue-Specific Radical-Directed Dissociation of Whole Proteins in the Gas Phase</strong> <div class="csl-block">Tony Ly, Ryan R Julian</div> <em>Journal of the American Chemical Society</em> (2007-12-14) <a href="https://doi.org/ddngs4">https://doi.org/ddngs4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ja076535a">10.1021/ja076535a</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18078340">18078340</a></div></div>
</div>
<div id="ref-18sf2lBbl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">697. </div><div class="csl-right-inline"><strong>Ultraviolet Photodissociation: Developments towards Applications for Mass‐Spectrometry‐Based Proteomics</strong> <div class="csl-block">Tony Ly, Ryan R Julian</div> <em>Angewandte Chemie International Edition</em> (2009-09-09) <a href="https://doi.org/c5fncw">https://doi.org/c5fncw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/anie.200900613">10.1002/anie.200900613</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19610000">19610000</a></div></div>
</div>
<div id="ref-5SkV76Q3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">698. </div><div class="csl-right-inline"><strong>The Mechanism Behind Top-Down UVPD Experiments: Making Sense of Apparent Contradictions</strong> <div class="csl-block">Ryan R. Julian</div> <em>Journal of the American Society for Mass Spectrometry</em> (2017-07-12) <a href="https://doi.org/gbtc6z">https://doi.org/gbtc6z</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s13361-017-1721-0">10.1007/s13361-017-1721-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28702929">28702929</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5711567">PMC5711567</a></div></div>
</div>
<div id="ref-10TkrAfIp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">699. </div><div class="csl-right-inline"><strong>Deciphering combinatorial post-translational modifications by top-down mass spectrometry</strong> <div class="csl-block">Jennifer S Brodbelt</div> <em>Current Opinion in Chemical Biology</em> (2022-10) <a href="https://doi.org/gsr77r">https://doi.org/gsr77r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.cbpa.2022.102180">10.1016/j.cbpa.2022.102180</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35779351">35779351</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9489649">PMC9489649</a></div></div>
</div>
<div id="ref-KWENOLlt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">700. </div><div class="csl-right-inline"><strong>Complete Protein Characterization Using Top-Down Mass Spectrometry and Ultraviolet Photodissociation</strong> <div class="csl-block">Jared B Shaw, Wenzong Li, Dustin D Holden, Yan Zhang, Jens Griep-Raming, Ryan T Fellers, Bryan P Early, Paul M Thomas, Neil L Kelleher, Jennifer S Brodbelt</div> <em>Journal of the American Chemical Society</em> (2013-06-04) <a href="https://doi.org/f48shf">https://doi.org/f48shf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ja4029654">10.1021/ja4029654</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23697802">23697802</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3757099">PMC3757099</a></div></div>
</div>
<div id="ref-1A6jk9UQ3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">701. </div><div class="csl-right-inline"><strong>Ultraviolet Photodissociation of Tryptic Peptide Backbones at 213 nm</strong> <div class="csl-block">Lars Kolbowski, Adam Belsom, Juri Rappsilber</div> <em>Journal of the American Society for Mass Spectrometry</em> (2020-04-30) <a href="https://doi.org/gsr77z">https://doi.org/gsr77z</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/jasms.0c00106">10.1021/jasms.0c00106</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32352297">32352297</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7273743">PMC7273743</a></div></div>
</div>
<div id="ref-zJsJYkGj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">702. </div><div class="csl-right-inline"><strong>Comparison of Top-Down Protein Fragmentation Induced by 213 and 193 nm UVPD</strong> <div class="csl-block">Michael Lanzillotti, Jennifer S Brodbelt</div> <em>Journal of the American Society for Mass Spectrometry</em> (2023-01-03) <a href="https://doi.org/gsr772">https://doi.org/gsr772</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/jasms.2c00288">10.1021/jasms.2c00288</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36594540">36594540</a></div></div>
</div>
<div id="ref-YuJbg3zO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">703. </div><div class="csl-right-inline"><strong>Open collaborative writing with Manubot</strong> <div class="csl-block">Daniel S Himmelstein, Vincent Rubinetti, David R Slochower, Dongbo Hu, Venkat S Malladi, Casey S Greene, Anthony Gitter</div> <em>PLOS Computational Biology</em> (2019-06-24) <a href="https://doi.org/c7np">https://doi.org/c7np</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pcbi.1007128">10.1371/journal.pcbi.1007128</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31233491">31233491</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6611653">PMC6611653</a></div></div>
</div>
</div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
