## Types of Experiments {.page_break_before}
A wide range of questions are addressable with proteomics technology, which translates to a wide range of variations of proteomics workflows. 
Sometimes identifying what proteins are present is desired, and sometimes the quantities of as many proteins as possible are desired. 
Proteomics experiments can be both qualitative and quantitative. 

### Qualitative experiments
- Identifying proteins 
- Identifying post translational modifications
- Identifying protein isoforms

### Quantitative experiments
- Protein abundance changes
- Phosphoproteomics
- Glycoproteomics
- Structural techniques (XL-MS, HDX-MS, FPOP, protein-painting, LiP-MS, radical footprinting, ion mobility)
- Protein stability and small molecule binding (Thermal proteome profiling, TPP, or cellular thermal shift assay, CETSA)
- Protein-protein interactions (PPIs): AP-MS, APEX, BioID

###Protein abundance changes
A common experiment is unbiased mapping of proteins along with detection of changes in their abundance across sample groups. 
This is achieved using one of the quantitative methods mentioned elsewhere in this manuscript such as LFQ or TMT. 
In these experiments often at least three replicates of each condition are tested to enable estimating the variance of measuring each protein. 
Depending on the experiment design, different statistical tests are used to ask if there are changes in the measured protein abundances between groups. 
If there are only two groups, the quantities might be compared with a t-test or with a Wilcoxon signed-rank test. 
The latter is a non-parametric version of the t-test. 
If there are more than two sample groups, then often Analysis of Variance (ANOVA) is used instead. 
With either testing scheme, the p-values from the first set of tests must be corrected for multiple tests. 
A common method for p value correction is the Benjamini-Hochberg method [@DOI:10.1111/j.2517-6161.1995.tb02031.x]. 
These types of experiments have revealed wide ranges of proteomic remodeling from various biological systems. 
### Post-Translational Modifications (PTMs)
Proteins are modified after they are translated with various chemical groups, or even by proteolytic cleavage such as n-terminal methionine removal. 
There are several proteomics methods that are developed specific to each type of modification. 

#### Phosphoproteomics
Phosphoproteomics is the study of the phosphorylation of proteins. 
Currently, there are two techniques that are useful for phosphoproteomics: mass spectrometry and western blotting. 
As seen in the figure above, quantitative proteomics over a variety of analytical techniques. 
A key challenge with phosphoproteomics is sensitivity; it is important to ensure that there is sufficient amount of protein before completing a phosphoproteomics project, typically at least 1 mg of total protein per sample. 
- Mass spectrometry
One of the ways to use mass spectrometry to detect phosphorylation is by adding a stable isotope in order to see where phosphorylation would happen that can be seen at the proteome level [@doi:10.1073/pnas.0609836104; @doi:10.1038/msb4100182; @doi:10.4049/jimmunol.176.5.2833; @doi:10.1126/scisignal.2000007].  
It is important to understand what methodology you are planning to use before running the experiments.
Despite advancement in technology of phosphoproteomics, the following challenges still exist: limited sample amounts, highly complex samples, and huge dynamic range [@doi:10.1016/j.copbio.2008.06.006]. 
Current approaches are limited in their ability to detect phosphoproteins at a resolution of 10-15 kD. 
Additionally, phosphoproteomic analysis is often time-consuming and requires the use of expensive equipment such as enrichment kits.

### Glycoproteomics
One PTM gaining interest due to its high occurrence and emerging functional roles is glycosylation. 
Protein glycosylation sites can be glycosylphosphates, N-linked glycosylation sites (asparagine-linked), O-linked glycosylation sites (serine/threonine-linked), or Z-linked glycosylation sites. 
Understanding protein glycosylation will help us understand numerous biological processes since this is a universal protein modification that can be seen across all domains of life [@doi:10.1038/s43586-022-00128-4; @doi:10.1093/glycob/cww086; @doi:10.1016/j.cell.2006.08.019; @doi:10.1146/annurev-neuro-071714-034019].


Studies of these two PTMs share several experimental pipelines steps including sample preparation. 
Protein clean-up approaches for glycoproteomics may differ from other proteomics experiments because glycopeptides are more hydrophilic than most peptides. 
Some approaches that have been mentioned in literature include: filter-aided sample preparation (FASP), suspension traps (S-traps), and protein aggregation capture (PAC) [@doi:10.1038/s43586-022-00128-4; @doi:10.1038/nmeth.1322 ; @doi:10.1002/pmic.201300553; @doi:10.1074/mcp.TIR118.001270; @doi:10.1021/acs.jproteome.8b00505; @doi:10.1021/acs.analchem.0c04726; @doi:10.26434/chemrxiv.13487364.v1].
Some enzymatic digestion to use include: trypsin, chymotrypsin, tryptophan, tyrosine, GluC, AspN, Pronase, Proteinase K,  OgpA, StcEz, BT4244, AM0627, AM1514, AM0608, Pic, ZmpC, CpaA, IMPa, PNGase F, Endo F, Endo H, and OglyZOR. [@doi:10.1038/s43586-022-00128-4]
Mass spectrometry has improved over the past decade so that now more strategies are available for glycoprotein structure elucidation and also for glycosylation site quantification [@doi:10.1038/s43586-022-00128-4].


### Structural techniques
1. XL-MS (Cross-linking mass spectrometry)
XL-MS is an emerging technology for the field of proteomics. It can be used to determine the protein folding and assembly, the dynamics of macromolecules in solution, and the effects of protein modifications on protein function. 
XL-MS allows molecules to be more stable which then can be ionized and fragmented. XL-MS is different from traditional MS in that it can resolve smaller peptides and proteins; can identify peptides in complex mixtures more accurately; can identify proteins in the presence of other proteins or peptides. [@doi:10.1016/j.ymeth.2015.06.010; @doi:10.1021/jasms.9b00085] 

Following are some suggestive steps as to creating a XL-MS workflow:
1. Select ion of interest if it is targeted proteomics.
2. Cross-link the molecular ion through a cross-linking agent (DSS, BS3, EDC, DMTMM, DSG, BS2G, DSSO, DSBU, CBDPS, PIR). [@doi:10.1021/jasms.9b00085]
3. Determine the cross-linking efficiency through binding energy and calculations.
4. Determine the number is cross-links which then will help you know the molecular weight.

There are still some challenges in XL-MS such as:
1. minimum amount of cross-linked peptides in complex mixtures [@doi:10.1021/acs.analchem.7b04431]
2. diversity of cross linked products [@doi:10.1021/acs.analchem.7b04431]

However, further advancements will continue to minimize the impact of these issues.

2. Hydrogen deuterium exchange mass spectrometry (HDX-MS)
HDX-MS works by detecting changes in the mass of the ion in which the amide hydrogens of the protein backbone is exchanged with its surrounding solvents [@doi:10.1038/s41592-019-0459-y]. 
The rate of exchange depends on the protein solvent accessible surface area, dynamics, and the properties of the amino acid sequence [@doi:10.1038/s41592-019-0459-y; @doi:10.1039/C0CS00113A; @doi:10.1016/S0065-3233(08)60129-1; @doi:10.1017/S0033583500005217].
Although using D2O as a solvent to make deuterium labeled sample is simple, HDX-MS requires a number of controls to ensure that experimental conditions are done in an unbiased manner [@doi:10.1038/s41592-019-0459-y; @doi:10.1586/epr.10.109; @doi:/10.1007/s13361-012-0377-z; @doi:10.4155/bio.15.46]. 
If done correctly, HDX can produce precise protein structure measurements with high reproducibility.
Masson **et al.** gave recommendations on how to prep samples, conduct data analysis, and present findings in a detailed stepwise manner that will be useful to have if you are new to using this technique [@doi:10.1038/s41592-019-0459-y].

3. Fast photochemical oxidation of proteins (FPOP) [@doi:10.1074/jbc.REV119.006218].
FPOP is a laser based hydroxyl radical protein footprinting MS method that relies on the irreversible labeling of solvent-exposed amino acid side chains by hydroxyl radicals in order to understand structure elucidation of proteins.
A laser produces 248 nm light to cause hydrogen peroxide to become hydroxyl radicals [@doi:10.1016/j.jasms.2005.09.008; @doi:10.1016/j.ijms.2006.08.018]. 
The flow rate of solution through the capillary and laser frequency are adjusted so that each protein molecule is radiated only once. 
After they are radiated, the sample is collected to a tube that has catalase and free methionine in the buffer so that the H2O2 and hydroxyl radical are quenched to prevent further unwarranted oxidation, which may then modify residues that become exposed due unfolding after the initial labeling. 
Control samples are made through running the sample through the flow system without any radiation. 
Another experimental control is done by adding a radical scavenger with the purpose of preventing over oxidation [@doi:10.1002/pro.3408; @doi:10.1021/ac901054w].
FPOP has wide application for proteins, to include being able to report fast protein folding and protein transient dynamics.

4. Protein Painting [@doi:10.1038/ncomms5413; @doi:10.1074/jbc.RA118.007310].
Protein painting uses "molecular paints" to noncovalently coat the solvent accessible surface of proteins. 
These paint molecules will coat the protein surfaces but will not have access to the hydrophobic cores or protein-protein interface regions that solvents cannot access. 
If the "paint" cover free amines of lysine side chains, the parts that are painted will not be subjected to trypsin cleavage, while the unpainted areas will. 
After proteolysis, the peptides samples will be subjected to MS. 
The samples will only show the unmodified opposing regions in the protein-protein interaction at the time of the painting. 

5. LiP-MS (limited proteolysis mass spectrometry) [@doi:10.1016/j.tibs.2020.05.006; @DOI:10.1038/nprot.2017.100; @doi:10.1038/s41596-022-00771-x; @doi:10.1038/nprot.2017.100].
Limited proteolysis coupled to mass spectrometry (LiP-MS) is a method that tracks structural changes in complex proteoms in response to stimuli.
This method relies on the usage of stimuli (small molecules and proteinase K (PK)) as the additions would cause tertiary and quaternary structures. 
Samples that undergo LiP-MS will show at where the structural changes will happen and give an understanding as to why PK bind to that spot.

For protocols and ways to run LiP-MS, please refer to this document: [@url:https://www.eubopen.org/sites/www.eubopen.org/files/attachments/2022/LiP-MS%20protocol.pdf]

6. Radical Footprinting 
This technique uses hydroxyl radical footprinting and MS in order to elucidate protein structures, assembly, and interactions within a large macromolecule [@doi:10.1021/cr0682047; @DOI:10.1021/ac200567u].
In addition to proteomics applications, various approaches to make hydroxide radicals have also been applied for footprinting studies in nucleic acid/ligand interactions [@doi:10.1016/j.cbpa.2005.02.009; @doi:10.1002/0471142700.nc0607s07; @doi:10.1093/nar/gkz089].
Below are known methods of producing radicals for footprinting.
2dli
1. Fenton and Fenton‐like Chemistry [@doi:10.1021/cr0682047; @doi:10.1016/j.elecom.2021.106970; @doi:10.1021/es903739f]
2. Electron-Pulse Radiolysis [@doi:10.1021/cr0682047; @doi:10.1016/j.jprot.2013.02.014]
3. High‐Voltage Electrical Discharge [@doi:10.1021/cr0682047; @doi:10.1016/j.ifset.2022.103055]
4. Synchrotron X‐ray Radiolysis of Water [@doi:10.1021/cr0682047; @doi:10.1016/j.jprot.2017.09.005]
5. Plasma Formation of OH Radicals [@doi:10.1021/cr0682047; @doi:10.1016/j.ijfoodmicro.2018.12.017]
6. Photolysis of Hydrogen Peroxide [@doi:10.1021/cr0682047; @doi:10.1016/j.jasms.2005.09.008]

This chapter is very useful in learning more on this topic. [@doi:10.1002/9781118970195.ch15]


### Protein stability and small molecule binding
1. Thermal proteome profiling (TPP) [@doi:10.15252/msb.20199232; @doi:10.1016/j.cbpa.2022.102225; @doi:10.1021/jacs.1c10621; @doi:10.1002/9781118970195.ch11]
Thermal proteome profiling (TPP)  involves subjecting a protein sample to a thermal shift assay (TSA), in which the protein is exposed to a range of temperatures, and the resulting changes in protein stability are monitored using mass spectrometry.
During a typical TPP experiment, a protein sample is first treated with a crosslinker to stabilize protein-ligand interactions. 
The sample is then divided into multiple aliquots, which are subjected to different temperatures in order to induce thermal denaturation. 
As the protein denatures, the ligand is released, and the resulting changes in protein stability are detected using mass spectrometry. 
By comparing the protein-ligand interactions across the different temperature conditions, TPP can provide insight into the binding affinity and specificity of a given ligand.

2. Cellular Thermal Shift Assay (CETSA) [@doi:10.1016/j.cbpa.2019.11.004]
CETSA is a powerful tool for studying protein-protein interactions in live cells. 
Although CTSA has been widely used to study protein-protein interactions in a proteome-wide manner, little is known about the global changes of protein interaction states with the energy landscape. 
The assay is capable of detecting shifts in the thermal equilibrium of cellular proteins in response to a variety of environmental perturbations, including the application of voltage, temperature, or magnetic fields. 
This assay is able to detect both small and large changes in protein interaction states, making it an ideal tool for investigating the effects of environmental stressors on protein interactions.

### Protein-protein interactions (PPIs)
1. Affinity purification coupled to mass spectrometry (AP-MS) [@doi:10.1002/iub.1081; @doi:10.1038/nrg3356; @doi:10.1038/nmeth.2400]  
AP-MS is an approach that involves the purification of a target protein or protein complex using a specific antibody followed by mass spectrometry analysis to identify the interacting proteins.
In a typical AP-MS experiment, a protein or protein complex of interest is first tagged with a specific epitope or affinity tag, such as a FLAG or HA tag, that can be used to selectively capture the target protein using a corresponding antibody or resin. 
The protein complex is then purified from the sample using a series of wash steps, and the interacting proteins are identified using mass spectrometry.
The success of AP-MS experiments depends on a number of factors, including the quality of the antibody or tag used for purification, the specificity and efficiency of the resin used for capture, and the sensitivity and resolution of the mass spectrometer. 
In addition, careful experimental design and data analysis are critical for accurate identification and interpretation of protein-protein interactions.
AP-MS has been used to study a wide range of biological processes, including signal transduction pathways, protein complexes, and protein post-translational modifications. 
Despite its widespread use, AP-MS has some limitations, including the potential for non-specific interactions, the difficulty in interpreting complex data sets, and the possibility of missing important interacting partners due to limitations in sensitivity or specificity. 
However, with continued advances in technology and data analysis methods, AP-MS is likely to remain a valuable tool for the study of protein-protein interactions and for the development of new therapeutics.

2. Avidity-based Extracellular Interaction Screen (APEX) [@doi:10.1186/1471-2105-9-529; @doi:10.1007/978-1-4939-9537-0_4; @doi:10.1155/2010/840518]
Avidity-based Extracellular Interaction Screen (APEX) is a proximity labeling technique that allows for the identification of cell surface protein interactions in living cells. 
By labeling cell surface proteins, APEX enables the identification and characterization of protein interactions that occur on the extracellular side of the cell membrane.
One of the major advantages of APEX is its ability to label proteins in their native environment, allowing for the identification of interactions that occur under physiological conditions. 
In addition, APEX is able to label proteins on the extracellular side of the cell membrane, allowing for the identification of cell surface protein interactions. APEX has been used to identify a wide range of protein interactions, including receptor-ligand interactions, signaling complexes, and extracellular matrix interactions.
Despite its advantages, APEX has some limitations, including the potential for non-specific labeling, the difficulty in distinguishing between direct and indirect interactions, and the possibility of missing interactions that occur at low abundance or in regions of the cell that are not effectively labeled. 

3. Biochemical Identification of Neighbors by proximity Labeling (BioID) [@doi: 10.1074/mcp.R120.001941; @doi:10.1002/pmic.201600123; @doi:10.1091/mbc.E15-12-0844]
Biochemical Identification of Neighbors by proximity Labeling (BioID) is a proximity labeling technique that allows for the identification of protein-protein interactions. 
BioID involves the introduction of a genetically-encoded biotin ligase enzyme into cells, which biotinylates proteins in close proximity to the enzyme. 
By labeling proteins that are in close proximity to the biotin ligase, BioID enables the identification and characterization of protein interactions that occur in living cells.
One of the advantages of BioID is its ability to label proteins in their native environment, allowing for the identification of interactions that occur under physiological conditions. In addition, BioID is able to label proteins that are not easily accessible by other proximity labeling techniques, such as APEX. 
BioID has been used to identify a wide range of protein interactions, including receptor-ligand interactions, signaling complexes, and protein localization.
Despite its many advantages, BioID has some limitations, including the potential for non-specific labeling, the difficulty in distinguishing between direct and indirect interactions, and the possibility of missing interactions that occur at low abundance or in regions of the cell that are not effectively labeled. 

For more information on BioID, please refer to this chapter: [@doi:10.1007/978-1-4939-8814-3_10]
