## Proteolysis {.page_break_before}

Proteolysis is the defining step that differentiates bottom-up or shotgun proteomics from top-down proteomics. 
Hydrolysis of proteins is extremely important because it defines the population of potentially identifyable peptides. 
Generally peptides between a length of 7-35 amino acid are considered useful for mass spectrometry analysis.
Peptides that are too long are difficult to identify by tandem mass spectrometry, or may be lost during sample preparation due to 
Peptides that are too short are less likely to uniquely match to a single protein. 
There are many choices of enzymes and chemicals that hydrolyze proteins into peptides. 
This section summarizes potential choices and their strengths and weaknesses. 

Trypsin is the most common choice of protease for proteome hydrolysis [@DOI:10.1002/mas.21376].
Trypsin is favorable because of its specificity, availability, efficiency and is comparatively cheaper than other enzymes. 
Trypsin cleaves at the C-terminus of basic amino acids, arginine and lysine. 
Many of the peptides generated from trypsin are short in length (less than ~ 20 amino acids), which is ideal for chromatographic separation, MS-based peptide fragmentation and identification by database search.
But the drawback of trypsin is that majority (56%) of the tryptic peptides are â‰¤ 6 amino acids, and hence using trypsin alone limits the observable proteome [@PMID: 20113005; @PMID: 25823410; @PMID:30687733].
This limits the number of identifiable protein isoforms and post translational modifications.


3. theoretical studies of proteolysis enzymes [@DOI:https://doi.org/10.1155/2014/960902]
4. Challenges associated with alternative enzyme choices (non-specific and semi-specific enzymes)
5. Alternative enzyme choices (one paragraph each?) - 
The enzyme choice mostly depends on the application.
In general, for a mere protein identification mostly trypsin is the choice due to the reasons aforementioned.
However, alternative enzymes are used for de novo assembly when the genomic data information is limited in the public database repositories.
Especially, they serve as an advantage in improving the sequence coverage [@PMID:12643544; @PMID:20113005] and also helps in distinguishing protein isoforms [@PMID:27123950].
In addition to that they also helps in improving the sensitivity and accuracy of protein quantification in terms of the number of petides derived [@PMID:30336047].
6. Lys-C
Lysyl endopeptidase (Lys-C) obtained from  Lysobacter enzymogenesis is a serine protease invloved in cleaving carboxyl terminus of lysine [@PMID:25823410].
Like trypsin, the optimum pH required for its activity is from pH 7-9. 
One among the major advantage of Lys-C is that it is resistant to denaturing agents, including 8 M urea [@PMID: 27123950].
Compared to Arg residues, trypsin is less efficient in cleaving Lys.
This also interfere in protein quantification in terms of sensitivity and accuracy.
Hence to achieve complete protein digestion with minimal missed cleavages, Lys-C is often used in parallel to complement tryspin digestion [@PMID:23017020; @PMID:16512685]. 
7. Glu-C
Glutamyl peptidase I, commonly known as Glu-C or V8 protease, is a serine protease obtained from Staphyloccous aureus.
This specifically cleaves at the C-terimus of Glu. 
However, depending on the pH conditions, in addition to Glu (pH 4.0) it is invloved in cleaving Asp (pH 8.0) residues also [@PMID:4627743].
8. Asp-N

9. Alpha-lytic protease [@DOI:https://doi.org/10.1074/mcp.m113.034710] and how it enables mapping human SUMO sites [@PMID:29079793].
10. Chymotrypsin
11. Arg-C
12. LysargiNase
13. Lys-N
14. Pepsin
 

