## Biological Interpretation {.page_break_before}

Nearly every untargeted proteomics experiment will result in a list of proteins or peptides of interest which require further, orthogonal validation and biological interpretation.
In this section, we aim to present a concise overview of how proteomic data can be effectively contextualised and used to generate new hypotheses. 

1. term enrichment analysis (KEGG, GO, GSEA, ORA)

Term enrichment analysis is performed to assess whether particular 'functional terms' are over-represented in a list of proteins (e.g. from a proteomics experiment). 
GO -
KEGG - 
GSEA - Gene Set Enrichment Analysis

- based on ranked list of proteins

ORA - Over-representation Analysis

- independent of ranking 

3. network analysis methods
4. structure analysis
5. isoform analysis

5. follow-up experiments

Orthogonal validation techniques of discovery proteomic data sets are highly context-dependent.
Depending on the question, researchers can draw on a myriad of techniques to validate MS-derived hypotheses in appropriate cellular, organismal or in vitro models. 
Protein abundance changes measured by bottom-up proteomics are commonly validated on a protein-to-protein basis using techniques such as immunoblotting analysis or immunofluorescence or immunohistological stainings. 
The latter also allows for validation of protein expression and localisation in intact tissue or cells. 
However, these semiquantitative methods are commonly strongly influenced by the quality of the antibodies used and might not be sensitive enough to detect small changes in protein levels. 

Other validation methods (single proteins): 
-	MS-based: PRM, SRM
- Non-MS: BN-PAGE/grdadient sedimatation assays for changes in protein complex composition/stoichiometry

Altered accessibility of broad-specificity proteases (i.e. proteinase K) to regions of a native protein can be used as a proxy readout for dynamic structural alterations (see LiP-MS). However, the interpretation of differential cleavage patterns still remains challenging as a range of biological processes can effectively modulate protease accessibility. For instance, differential protease cleavage patterns in a native protein can be affected by a change in its interaction with other proteins or metabolites (occluding or exposing protease cleavage sites), conformational changes, and a host of posttranslational modifications (i.e. phosphorylation). Therefore, orthogonal approaches are necessary to assess which of these possibilities are at the root of any observed ‘structural change’. 

Discussed below are multiple methodologies harnessed to further validate and assess structural changes in individual proteins.

-	NMR
-	Semiquantitative methods (IB with conformation-specific antibodies)
-	In silico molecular dynamics simulations 


[in progress]
