<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Yuming Jiang" />
  <meta name="author" content="Devasahayam Arokia Balaya Rex" />
  <meta name="author" content="Dina Schuster" />
  <meta name="author" content="Benjamin A. Neely" />
  <meta name="author" content="Germán L. Rosano" />
  <meta name="author" content="Norbert Volkmar" />
  <meta name="author" content="Amanda Momenzadeh" />
  <meta name="author" content="Trenton M. Peters-Clarke" />
  <meta name="author" content="Susan B. Egbert" />
  <meta name="author" content="Simion Kreimer" />
  <meta name="author" content="Emma H. Doud" />
  <meta name="author" content="Oliver M. Crook" />
  <meta name="author" content="Amit Kumar Yadav" />
  <meta name="author" content="Muralidharan Vanuopadath" />
  <meta name="author" content="Adrian D. Hegeman" />
  <meta name="author" content="Martín L. Mayta" />
  <meta name="author" content="Anna G. Duboff" />
  <meta name="author" content="Nicholas M. Riley" />
  <meta name="author" content="Robert L. Moritz" />
  <meta name="author" content="Jesse G. Meyer" />
  <meta name="dcterms.date" content="2024-05-07" />
  <meta name="keywords" content="markdown, publishing, manubot" />
  <title>Comprehensive Overview of Bottom-Up Proteomics using Mass Spectrometry</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta property="og:type" content="article" />
  <meta name="dc.title" content="Comprehensive Overview of Bottom-Up Proteomics using Mass Spectrometry" />
  <meta name="citation_title" content="Comprehensive Overview of Bottom-Up Proteomics using Mass Spectrometry" />
  <meta property="og:title" content="Comprehensive Overview of Bottom-Up Proteomics using Mass Spectrometry" />
  <meta property="twitter:title" content="Comprehensive Overview of Bottom-Up Proteomics using Mass Spectrometry" />
  <meta name="dc.date" content="2024-05-07" />
  <meta name="citation_publication_date" content="2024-05-07" />
  <meta property="article:published_time" content="2024-05-07" />
  <meta name="dc.modified" content="2024-05-07T21:25:09+00:00" />
  <meta property="article:modified_time" content="2024-05-07T21:25:09+00:00" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Yuming Jiang" />
  <meta name="citation_author_institution" content="Department of Computational Biomedicine, Cedars Sinai Medical Center" />
  <meta name="citation_author_orcid" content="0000-0001-7444-3849" />
  <meta name="twitter:creator" content="@yumingjiang94" />
  <meta name="citation_author" content="Devasahayam Arokia Balaya Rex" />
  <meta name="citation_author_institution" content="Center for Systems Biology and Molecular Medicine, Yenepoya Research Centre, Yenepoya (Deemed to be University), Mangalore 575018, India" />
  <meta name="citation_author_orcid" content="0000-0002-9556-3150" />
  <meta name="twitter:creator" content="@rexprem" />
  <meta name="citation_author" content="Dina Schuster" />
  <meta name="citation_author_institution" content="Department of Biology, Institute of Molecular Systems Biology, ETH Zurich, Zurich 8093, Switzerland" />
  <meta name="citation_author_institution" content="Department of Biology, Institute of Molecular Biology and Biophysics, ETH Zurich, Zurich 8093, Switzerland" />
  <meta name="citation_author_institution" content="Laboratory of Biomolecular Research, Division of Biology and Chemistry, Paul Scherrer Institute, Villigen 5232, Switzerland" />
  <meta name="citation_author_orcid" content="0000-0001-6611-8237" />
  <meta name="twitter:creator" content="@dina_sch" />
  <meta name="citation_author" content="Benjamin A. Neely" />
  <meta name="citation_author_institution" content="Chemical Sciences Division, National Institute of Standards and Technology, NIST Charleston" />
  <meta name="citation_author_orcid" content="0000-0001-6120-7695" />
  <meta name="twitter:creator" content="@neely615" />
  <meta name="citation_author" content="Germán L. Rosano" />
  <meta name="citation_author_institution" content="Mass Spectrometry Unit, Institute of Molecular and Cellular Biology of Rosario, Rosario, Argentina" />
  <meta name="citation_author_orcid" content="0000-0002-8313-6813" />
  <meta name="twitter:creator" content="@GermanRosano" />
  <meta name="citation_author" content="Norbert Volkmar" />
  <meta name="citation_author_institution" content="Department of Biology, Institute of Molecular Systems Biology, ETH Zurich, Zurich 8093, Switzerland" />
  <meta name="citation_author_orcid" content="0000-0003-0766-5606" />
  <meta name="twitter:creator" content="@NorbertVolkmar" />
  <meta name="citation_author" content="Amanda Momenzadeh" />
  <meta name="citation_author_institution" content="Department of Computational Biomedicine, Cedars Sinai Medical Center, Los Angeles, California, USA" />
  <meta name="citation_author_orcid" content="0000-0002-8614-0690" />
  <meta name="citation_author" content="Trenton M. Peters-Clarke" />
  <meta name="citation_author_institution" content="Department of Pharmaceutical Chemistry, University of California-San Francisco" />
  <meta name="citation_author_orcid" content="0000-0002-9153-2525" />
  <meta name="twitter:creator" content="@trentmpc" />
  <meta name="citation_author" content="Susan B. Egbert" />
  <meta name="citation_author_institution" content="Department of Chemistry, University of Manitoba, Winnipeg, Cananda" />
  <meta name="citation_author_orcid" content="0000-0001-5458-1099" />
  <meta name="twitter:creator" content="@lichenlady94" />
  <meta name="citation_author" content="Simion Kreimer" />
  <meta name="citation_author_institution" content="Smidt Heart Institute, Cedars Sinai Medical Center" />
  <meta name="citation_author_institution" content="Advanced Clinical Biosystems Research Institute, Cedars Sinai Medical Center" />
  <meta name="citation_author_orcid" content="0000-0001-6627-3771" />
  <meta name="twitter:creator" content="@KreimerSimion" />
  <meta name="citation_author" content="Emma H. Doud" />
  <meta name="citation_author_institution" content="Center for Proteome Analysis, Indiana University School of Medicine, Indianapolis, Indiana, USA" />
  <meta name="citation_author_orcid" content="0000-0003-0049-0073" />
  <meta name="twitter:creator" content="@fireinlab" />
  <meta name="citation_author" content="Oliver M. Crook" />
  <meta name="citation_author_institution" content="Oxford Protein Informatics Group, Department of Statistics, University of Oxford, Oxford OX1 3LB, United Kingdom" />
  <meta name="citation_author_orcid" content="0000-0001-5669-8506" />
  <meta name="twitter:creator" content="@OllyMCrook" />
  <meta name="citation_author" content="Amit Kumar Yadav" />
  <meta name="citation_author_institution" content="Translational Health Science and Technology Institute" />
  <meta name="citation_author_orcid" content="0000-0002-9445-8156" />
  <meta name="twitter:creator" content="@theoneamit" />
  <meta name="citation_author" content="Muralidharan Vanuopadath" />
  <meta name="citation_author_institution" content="School of Biotechnology, Amrita Vishwa Vidyapeetham, Kollam-690 525, Kerala, India" />
  <meta name="citation_author_orcid" content="0000-0002-9364-917X" />
  <meta name="twitter:creator" content="@V_MuraleeDhar" />
  <meta name="citation_author" content="Adrian D. Hegeman" />
  <meta name="citation_author_institution" content="Departments of Horticultural Science and Plant and Microbial Biology, University of Minnesota – Twin Cities, United States" />
  <meta name="citation_author_orcid" content="0000-0003-1008-6066" />
  <meta name="citation_author" content="Martín L. Mayta" />
  <meta name="citation_author_institution" content="School of Medicine and Health Sciences, Center for Health Sciences Research, Universidad Adventista del Plata, Libertador San Martín 3103, Argentina" />
  <meta name="citation_author_institution" content="Molecular Biology Department, School of Pharmacy and Biochemistry, Universidad Nacional de Rosario, Rosario 2000, Argentina" />
  <meta name="citation_author_orcid" content="0000-0002-7986-4551" />
  <meta name="twitter:creator" content="@MartinMayta2" />
  <meta name="citation_author" content="Anna G. Duboff" />
  <meta name="citation_author_institution" content="Department of Chemistry, University of Washington" />
  <meta name="citation_author_orcid" content="0009-0002-7316-3831" />
  <meta name="citation_author" content="Nicholas M. Riley" />
  <meta name="citation_author_institution" content="Department of Chemistry, University of Washington" />
  <meta name="citation_author_orcid" content="0000-0002-1536-2966" />
  <meta name="twitter:creator" content="@riley_nm1" />
  <meta name="citation_author" content="Robert L. Moritz" />
  <meta name="citation_author_institution" content="Institute for Systems biology, Seattle, WA, USA, 98109" />
  <meta name="citation_author_orcid" content="0000-0002-3216-9447" />
  <meta name="twitter:creator" content="@r_l_moritz" />
  <meta name="citation_author" content="Jesse G. Meyer" />
  <meta name="citation_author_institution" content="Department of Computational Biomedicine, Cedars Sinai Medical Center" />
  <meta name="citation_author_orcid" content="0000-0003-2753-3926" />
  <meta name="twitter:creator" content="@j_my_sci" />
  <link rel="canonical" href="https://jessegmeyerlab.github.io/proteomics-tutorial/" />
  <meta property="og:url" content="https://jessegmeyerlab.github.io/proteomics-tutorial/" />
  <meta property="twitter:url" content="https://jessegmeyerlab.github.io/proteomics-tutorial/" />
  <meta name="citation_fulltext_html_url" content="https://jessegmeyerlab.github.io/proteomics-tutorial/" />
  <meta name="citation_pdf_url" content="https://jessegmeyerlab.github.io/proteomics-tutorial/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://jessegmeyerlab.github.io/proteomics-tutorial/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://jessegmeyerlab.github.io/proteomics-tutorial/v/b43d842d9c8577826b88636f5aac8085e64e0aa3/" />
  <meta name="manubot_html_url_versioned" content="https://jessegmeyerlab.github.io/proteomics-tutorial/v/b43d842d9c8577826b88636f5aac8085e64e0aa3/" />
  <meta name="manubot_pdf_url_versioned" content="https://jessegmeyerlab.github.io/proteomics-tutorial/v/b43d842d9c8577826b88636f5aac8085e64e0aa3/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Comprehensive Overview of Bottom-Up Proteomics using Mass Spectrometry</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://jessegmeyerlab.github.io/proteomics-tutorial/v/b43d842d9c8577826b88636f5aac8085e64e0aa3/">permalink</a>)
was automatically generated
from <a href="https://github.com/jessegmeyerlab/proteomics-tutorial/tree/b43d842d9c8577826b88636f5aac8085e64e0aa3">jessegmeyerlab/proteomics-tutorial@b43d842</a>
on May 7, 2024.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Yuming Jiang</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-7444-3849">0000-0001-7444-3849</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/jymbcrc">jymbcrc</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/yumingjiang94">yumingjiang94</a>
<br>
<small>
Department of Computational Biomedicine, Cedars Sinai Medical Center
</small></p></li>
<li><p><strong>Devasahayam Arokia Balaya Rex</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9556-3150">0000-0002-9556-3150</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/ArokiaRex">ArokiaRex</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/rexprem">rexprem</a>
<br>
<small>
Center for Systems Biology and Molecular Medicine, Yenepoya Research Centre, Yenepoya (Deemed to be University), Mangalore 575018, India
</small></p></li>
<li><p><strong>Dina Schuster</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6611-8237">0000-0001-6611-8237</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/dschust-r">dschust-r</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/dina_sch">dina_sch</a>
<br>
<small>
Department of Biology, Institute of Molecular Systems Biology, ETH Zurich, Zurich 8093, Switzerland; Department of Biology, Institute of Molecular Biology and Biophysics, ETH Zurich, Zurich 8093, Switzerland; Laboratory of Biomolecular Research, Division of Biology and Chemistry, Paul Scherrer Institute, Villigen 5232, Switzerland
</small></p></li>
<li><p><strong>Benjamin A. Neely</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6120-7695">0000-0001-6120-7695</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/neely">neely</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/neely615">neely615</a>
<br>
<small>
Chemical Sciences Division, National Institute of Standards and Technology, NIST Charleston
· Funded by NIST
</small></p></li>
<li><p><strong>Germán L. Rosano</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-8313-6813">0000-0002-8313-6813</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/ger225">ger225</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/GermanRosano">GermanRosano</a>
<br>
<small>
Mass Spectrometry Unit, Institute of Molecular and Cellular Biology of Rosario, Rosario, Argentina
· Funded by Grant PICT 2019-02971 (Agencia I+D+i)
</small></p></li>
<li><p><strong>Norbert Volkmar</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0766-5606">0000-0003-0766-5606</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/norbertvolkmar">norbertvolkmar</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/NorbertVolkmar">NorbertVolkmar</a>
<br>
<small>
Department of Biology, Institute of Molecular Systems Biology, ETH Zurich, Zurich 8093, Switzerland
</small></p></li>
<li><p><strong>Amanda Momenzadeh</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-8614-0690">0000-0002-8614-0690</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/amandamomenzadeh">amandamomenzadeh</a>
<br>
<small>
Department of Computational Biomedicine, Cedars Sinai Medical Center, Los Angeles, California, USA
</small></p></li>
<li><p><strong>Trenton M. Peters-Clarke</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9153-2525">0000-0002-9153-2525</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/petersclarke">petersclarke</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/trentmpc">trentmpc</a>
<br>
<small>
Department of Pharmaceutical Chemistry, University of California-San Francisco
</small></p></li>
<li><p><strong>Susan B. Egbert</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-5458-1099">0000-0001-5458-1099</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/lichenlady94">lichenlady94</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/lichenlady94">lichenlady94</a>
<br>
<small>
Department of Chemistry, University of Manitoba, Winnipeg, Cananda
</small></p></li>
<li><p><strong>Simion Kreimer</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6627-3771">0000-0001-6627-3771</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/KreimerSimion">KreimerSimion</a>
<br>
<small>
Smidt Heart Institute, Cedars Sinai Medical Center; Advanced Clinical Biosystems Research Institute, Cedars Sinai Medical Center
</small></p></li>
<li><p><strong>Emma H. Doud</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0049-0073">0000-0003-0049-0073</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/edoud1">edoud1</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/fireinlab">fireinlab</a>
<br>
<small>
Center for Proteome Analysis, Indiana University School of Medicine, Indianapolis, Indiana, USA
</small></p></li>
<li><p><strong>Oliver M. Crook</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-5669-8506">0000-0001-5669-8506</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/ococrook">ococrook</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/OllyMCrook">OllyMCrook</a>
<br>
<small>
Oxford Protein Informatics Group, Department of Statistics, University of Oxford, Oxford OX1 3LB, United Kingdom
</small></p></li>
<li><p><strong>Amit Kumar Yadav</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9445-8156">0000-0002-9445-8156</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/aky">aky</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/theoneamit">theoneamit</a>
<br>
<small>
Translational Health Science and Technology Institute
· Funded by Grant BT/PR16456/BID/7/624/2016 (Department of Biotechnology, India); Grant Translational Research Program (TRP) at THSTI funded by DBT
</small></p></li>
<li><p><strong>Muralidharan Vanuopadath</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9364-917X">0000-0002-9364-917X</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/vanuopadathmurali">vanuopadathmurali</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/V_MuraleeDhar">V_MuraleeDhar</a>
<br>
<small>
School of Biotechnology, Amrita Vishwa Vidyapeetham, Kollam-690 525, Kerala, India
· Funded by Department of Health Research, Indian Council of Medical Research, Government of India (File No.R.12014/31/2022-HR)
</small></p></li>
<li><p><strong>Adrian D. Hegeman</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1008-6066">0000-0003-1008-6066</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/HegemanLab">HegemanLab</a>
<br>
<small>
Departments of Horticultural Science and Plant and Microbial Biology, University of Minnesota – Twin Cities, United States
· Funded by Nation Science Foundation MCB-2225057 and IOS-2025297
</small></p></li>
<li><p><strong>Martín L. Mayta</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-7986-4551">0000-0002-7986-4551</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/martinmayta">martinmayta</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/MartinMayta2">MartinMayta2</a>
<br>
<small>
School of Medicine and Health Sciences, Center for Health Sciences Research, Universidad Adventista del Plata, Libertador San Martín 3103, Argentina; Molecular Biology Department, School of Pharmacy and Biochemistry, Universidad Nacional de Rosario, Rosario 2000, Argentina
</small></p></li>
<li><p><strong>Anna G. Duboff</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0009-0002-7316-3831">0009-0002-7316-3831</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/agduboff">agduboff</a>
<br>
<small>
Department of Chemistry, University of Washington
· Funded by Summer Research Acceleration Fellowship, Department of Chemistry, University of Washington
</small></p></li>
<li><p><strong>Nicholas M. Riley</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-1536-2966">0000-0002-1536-2966</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/rileynm">rileynm</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/riley_nm1">riley_nm1</a>
<br>
<small>
Department of Chemistry, University of Washington
· Funded by National Institutes of Health Grant R00 GM147304
</small></p></li>
<li><p><strong>Robert L. Moritz</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-3216-9447">0000-0002-3216-9447</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/r_l_moritz">r_l_moritz</a>
<br>
<small>
Institute for Systems biology, Seattle, WA, USA, 98109
· Funded by National Institutes of Health Grants R01GM087221, R24GM127667, U19AG023122, S10OD026936; National Science Foundation Award 1920268
</small></p></li>
<li><p><strong>Jesse G. Meyer</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2753-3926">0000-0003-2753-3926</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/jessegmeyerlab">jessegmeyerlab</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/j_my_sci">j_my_sci</a>
<br>
<small>
Department of Computational Biomedicine, Cedars Sinai Medical Center
· Funded by National Institutes of Health Grant R21 AG074234; National Institutes of Health Grant R35 GM142502
</small></p></li>
</ul>
<div id="correspondence">
<p>✉ — Correspondence possible via <a href="https://github.com/jessegmeyerlab/proteomics-tutorial/issues">GitHub Issues</a></p>
</div>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<p>Proteomics is the large scale study of protein structure and function from biological systems through protein identification and quantification.
“Shotgun proteomics” or “bottom-up proteomics” is the prevailing strategy, in which proteins are hydrolyzed into peptides that are analyzed by mass spectrometry.
Proteomics studies can be applied to diverse studies ranging from simple protein identification to studies of proteoforms, protein-protein interactions, protein structural alterations, absolute and relative protein quantification, post-translational modifications, and protein stability.
To enable this range of different experiments, there are diverse strategies for proteome analysis.
The nuances of how proteomic workflows differ may be challenging to understand for new practitioners.
Here, we provide a comprehensive overview of different proteomics methods.
We cover from biochemistry basics and protein extraction to biological interpretation and orthogonal validation.
We expect this manuscript will serve as a handbook for researchers who are new to the field of bottom-up proteomics.</p>
<h2 class="page_break_before" id="introduction">Introduction</h2>
<p>Proteomics is the large-scale study of protein structure and function.
The term “proteomics” is thought to have been coined by Marc R. Wilkins.
Proteins are translated from messenger RNA (mRNA) transcripts that are transcribed from the complementary DNA-based genome.
Although the genome encodes potential cellular functions and states, the study of proteins in all their forms is necessary to truly understand biology.</p>
<p>Currently, proteomics can be performed with various methods.
Mass spectrometry has emerged within the past few decades as the premier tool for comprehensive proteome analysis.
The ability of mass spectrometry (MS) to detect charged chemicals enables the identification of peptide sequences and modifications for diverse biological investigations.
Alternative (commercial) methods based on affinity interactions of antibodies or DNA aptamers have been developed, namely <a href="https://olink.com/">Olink</a> and <a href="https://somalogic.com/somascan-platform/https://somalogic.com/somascan-platform/">SomaScan</a>.
There are also nascent methods that are either recently commercialized or still under development and not yet applicable to whole proteomes, such as motif scanning using antibodies, variants of N-terminal degradation, and nanopores<span class="citation" data-cites="tzsjhzo9 dZHIMQZQ XUmjaJtI vmYTdWAl"><sup><a href="#ref-tzsjhzo9" role="doc-biblioref">1</a>–<a href="#ref-vmYTdWAl" role="doc-biblioref">4</a></sup></span>.
Another approach uses parallel immobilization of peptides with total internal reflection microscopy and sequential Edman degradation<span class="citation" data-cites="rpZdECmv"><sup><a href="#ref-rpZdECmv" role="doc-biblioref">5</a></sup></span>.
However, by far the most common method for proteomics is based on mass spectrometry coupled to liquid chromatography (LC).</p>
<p>Modern proteomics had its roots in the early 1980s with the analysis of peptides by mass spectrometry and low efficiency ion sources.
One pioneer in the field was Don Hunt, who described sequencing of peptides using tandem mass spectrometry after chemical ionization with isobutane in 1981<span class="citation" data-cites="qRXV6ZBf"><sup><a href="#ref-qRXV6ZBf" role="doc-biblioref">6</a></sup></span>.
Another pioneer was Klaus Biemann, who for example worked with Brad Gibson to report peptide identification from fast atom bombardment<span class="citation" data-cites="FV1jAbXp"><sup><a href="#ref-FV1jAbXp" role="doc-biblioref">7</a></sup></span>.
Progress started ramping up around the year 1990 with the introduction of soft ionization methods that enabled, for the first time, efficient transfer of large biomolecules into the gas phase without destroying them<span class="citation" data-cites="4Ajjx5fo 15Ye4Za7Y"><sup><a href="#ref-4Ajjx5fo" role="doc-biblioref">8</a>,<a href="#ref-15Ye4Za7Y" role="doc-biblioref">9</a></sup></span>.
Shortly afterward, the first computer algorithm for matching peptides to a database was introduced<span class="citation" data-cites="pG7zq91Q"><sup><a href="#ref-pG7zq91Q" role="doc-biblioref">10</a></sup></span>.
Another major milestone that allowed identification of over 1,000 proteins were improvements to chromatography upstream of MS anlaysis<span class="citation" data-cites="ujahSo3G"><sup><a href="#ref-ujahSo3G" role="doc-biblioref">11</a></sup></span>.
As the volume of data exploded, methods for statistical analysis transitioned from the wild west of ad hoc empirical analysis to modern informatics based on statistical models<span class="citation" data-cites="mw5jxASm"><sup><a href="#ref-mw5jxASm" role="doc-biblioref">12</a></sup></span> and false discovery rate<span class="citation" data-cites="15RjczuIy"><sup><a href="#ref-15RjczuIy" role="doc-biblioref">13</a></sup></span>.
<!-- Todo: figure 1: major milestones in proteomics technology since 1990 --></p>
<p>Two strategies of mass spectrometry-based proteomics differ fundamentally by whether proteins are analyzed as a whole chain or cleaved into peptides before analysis: “top-down” versus “bottom-up”.
Bottom-up proteomics (also referred to as shotgun proteomics) is defined by the intentional hydrolysis of proteins into peptide pieces using enzymes called proteases<span class="citation" data-cites="16QSCuG38"><sup><a href="#ref-16QSCuG38" role="doc-biblioref">14</a></sup></span>.
Therefore, bottom-up proteomics does not actually measure proteins, but instead infers protein presence and abundance from identified peptides<span class="citation" data-cites="mw5jxASm"><sup><a href="#ref-mw5jxASm" role="doc-biblioref">12</a></sup></span>.
Sometimes, proteins are inferred from only one peptide sequence representing a small fraction of the total protein sequence predicted from the genome.
In contrast, top-down proteomics attempts to measure intact proteins<span class="citation" data-cites="UsGpigc0 6a8Xt7SG 6hFikhYz a6Z8FNxW"><sup><a href="#ref-UsGpigc0" role="doc-biblioref">15</a>–<a href="#ref-a6Z8FNxW" role="doc-biblioref">18</a></sup></span>.
The potential benefit of top-down proteomics is the ability to measure the many varied proteoforms<span class="citation" data-cites="6a8Xt7SG h00Trxkt oe85BKlv"><sup><a href="#ref-6a8Xt7SG" role="doc-biblioref">16</a>,<a href="#ref-h00Trxkt" role="doc-biblioref">19</a>,<a href="#ref-oe85BKlv" role="doc-biblioref">20</a></sup></span>.
However, due to myriad analytical challenges, the depth of protein coverage that is achievable by top-down proteomics is considerably less than that of bottom-up proteomics<span class="citation" data-cites="CwZoo7w"><sup><a href="#ref-CwZoo7w" role="doc-biblioref">21</a></sup></span>.</p>
<p>In this tutorial we focus on the bottom-up proteomics workflow.
The most common version of this workflow is generally comprised of the following steps.
First, proteins in a biological sample must be extracted.
Usually this is achieved by mechanically lysing cells or tissue while denaturing and solubilizing the proteins and disrupting DNA to minimize interference in analysis procedures.
Next, proteins are hydrolyzed into peptides, most often using the protease trypsin, which generates peptides with basic C-terminal amino acids (arginine and lysine) to aid in fragment ion series production during tandem mass spectrometry (MS/MS).
Peptides can also be generated by chemical reactions that induce residue specific hydrolysis, such as cyanogen bromide that cleaves after methionine.
Peptides from proteome hydrolysis must be purified; this is often accomplished with reversed-phase liquid chromatography (RPLC) cartridges or tips to remove interfering molecules in the sample such as salts and buffers.
The peptides are then almost always separated by reversed-phase LC before they are ionized and introduced into a mass spectrometer, although recent reports also describe LC-free proteomics by direct infusion<span class="citation" data-cites="pRTFSroJ DqzDdsRo 1339IM4Kb"><sup><a href="#ref-pRTFSroJ" role="doc-biblioref">22</a>–<a href="#ref-1339IM4Kb" role="doc-biblioref">24</a></sup></span>.
The mass spectrometer then collects precursor and fragment ion data from those peptides.
Peptides must be identified from the tandem mass spectra, protein groups are inferred from a proteome database, and then quantitative values are assigned.
Changes in protein abundances across conditions are determined with statistical tests, and results must be interpreted in the context of the relevant biology.
Data interpretation is the rate limiting step; data collected in less than one week can take months or years to understand.</p>
<p>There are many variations to this workflow.
The diversity of experimental goals that are achievable with proteomics technology drives an expansive array of workflows.
Every choice is important as every choice will affect the results, from instrument procurement to choice of data processing software and everything in between.
In this tutorial, we detail all the required steps to serve as a comprehensive overview for new proteomics practitioners.</p>
<p>There are 17 sections in total:</p>
<ol type="1">
<li>Biochemistry basics</li>
<li>Types of experiments</li>
<li>Protein extraction</li>
<li>Proteolysis</li>
<li>Peptide Quantification Methods</li>
<li>Enrichments</li>
<li>Peptide purification</li>
<li>Liquid Chromatography</li>
<li>Peptide Ionization</li>
<li>Mass Spectrometry</li>
<li>Peptide Fragmentation (MS/MS)</li>
<li>Data Acquisition</li>
<li>Raw Data Analysis</li>
<li>Protein Databases</li>
<li>Proteomics Knowledge Bases</li>
<li>Biological Interpretation</li>
<li>Orthogonal Validation Experiments</li>
</ol>
<h2 class="page_break_before" id="biochemistry-basics">1. Biochemistry Basics</h2>
<h3 id="proteins">Proteins</h3>
<p>Proteins are large biomolecules or biopolymers made up of a backbone of amino acids which are linked by peptide bonds.
They perform various functions in living organisms ranging from structural roles to functional involvement in cellular signaling and the catalysis of chemical reactions (enzymes).
Proteins are made up of 20 different amino acids (not counting pyrrolysine, hydroxyproline, and selenocysteine, which only occur in specific organisms) and their sequence is encoded in their corresponding genes.
The human genome encodes approximately 19,778 of the predicted canonical proteins coded in the human genome (see www.neXtProt.org)<span class="citation" data-cites="JTW0FD8g"><sup><a href="#ref-JTW0FD8g" role="doc-biblioref">25</a></sup></span>.
Each protein is present at a different abundance depending on the cell type or bodily fluid.
Previous studies have shown that the concentration range of proteins can span at least seven orders of magnitude to up to 20,000,000 copies per cell, and that their distribution is tissue-specific<span class="citation" data-cites="Iei0Ms6X OQN3Pb5c"><sup><a href="#ref-Iei0Ms6X" role="doc-biblioref">26</a>,<a href="#ref-OQN3Pb5c" role="doc-biblioref">27</a></sup></span>.
Protein abundances can span more than ten orders of magnitude in human blood, while a few proteins make up most of the protein by weight in these fluids, making blood and plasma proteomics one of the most challenging matrices for mass spectrometry to analyze.
Due to genetic variation, alternative splicing, and co- and post-translational modifications (PTMs), multiple different proteoforms can be produced from a single gene (<strong>Figure 1</strong>)<span class="citation" data-cites="6a8Xt7SG i4DXPyG5"><sup><a href="#ref-6a8Xt7SG" role="doc-biblioref">16</a>,<a href="#ref-i4DXPyG5" role="doc-biblioref">28</a></sup></span>.</p>
<div id="fig:proteoforms" class="fignos">
<figure>
<img src="images/intro-centralDogma-proteoforms.png" style="width:100.0%" data-tag="1" alt="Figure 1: Proteome Complexity. Each gene may be expressed in the form of multiple protein products, or proteoforms, through alternative splicing and incorporation of post-translational modifications. As such, there are many more unique proteoforms than genes. While there exist 20,000 - 23,000 coding genes in the human genome, upwards of 1,000,000 unique human proteoforms may exist. The study of the structure, function, and spatial and temporal regulation of these proteins is the subject of mass spectrometry-based proteomics" />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>Proteome Complexity.</strong>
Each gene may be expressed in the form of multiple protein products, or proteoforms, through alternative splicing and incorporation of post-translational modifications.
As such, there are many more unique proteoforms than genes.
While there exist 20,000 - 23,000 coding genes in the human genome, upwards of 1,000,000 unique human proteoforms may exist.
The study of the structure, function, and spatial and temporal regulation of these proteins is the subject of mass spectrometry-based proteomics</figcaption>
</figure>
</div>
<h4 id="ptms">PTMs</h4>
<p>After protein biosynthesis, enzymatic and nonenzymatic processes change the protein sequence through proteolysis or covalent chemical modification of amino acid side chains.
Post-translational modifications (PTMs) are important biological regulators contributing to the diversity and function of the cellular proteome.
Proteins can be post-translationally modified through enzymatic and non-enzymatic reactions <em>in vivo</em> and <em>in vitro</em><span class="citation" data-cites="m2ZOJt1R"><sup><a href="#ref-m2ZOJt1R" role="doc-biblioref">29</a></sup></span>.
PTMs can be reversible or irreversible, and they change protein function in multiple ways, for example by altering substrate–enzyme interactions, subcellular localization or protein-protein interactions<span class="citation" data-cites="uLIaWFNx OEP3MV7r"><sup><a href="#ref-uLIaWFNx" role="doc-biblioref">30</a>,<a href="#ref-OEP3MV7r" role="doc-biblioref">31</a></sup></span>.</p>
<p>More than 400 biological PTMs have been discovered in both prokaryotic and eukaryotic cells.
There are many more chemical artifact PTMs that occur during sample preparation, such as carbamylation.
Biological modifications are crucial in controlling protein functions and signal transduction pathways<span class="citation" data-cites="fAHUecLj"><sup><a href="#ref-fAHUecLj" role="doc-biblioref">32</a></sup></span>.
The most commonly studied and biologically relevant post-translational modifications include phosphorylation (Ser, Thr, Tyr, His), glycosylation (Arg, Asp, Cys, Ser, Thr, Tyr, Trp), disulfide bonds (Cys-Cys), ubiquitination (Lys, Cys, Ser, Thr, N-term), succinylation (Lys), methylation (Arg, Lys, His, Glu, Asn, Cys), oxidation (especially Met, Trp, His, Cys), acetylation (Lys, N-term), and lipidations<span class="citation" data-cites="yYHlJgbC"><sup><a href="#ref-yYHlJgbC" role="doc-biblioref">33</a></sup></span>.</p>
<p>Protein PTMs can alter its function, activity, structure, spatiotemporal status and interaction with proteins or small molecules.
PTMs alter signal transduction pathways and gene expression control<span class="citation" data-cites="e078CIsS"><sup><a href="#ref-e078CIsS" role="doc-biblioref">34</a></sup></span> regulation of apoptosis<span class="citation" data-cites="oTq7yQcv 1DDSEV4N5"><sup><a href="#ref-oTq7yQcv" role="doc-biblioref">35</a>,<a href="#ref-1DDSEV4N5" role="doc-biblioref">36</a></sup></span> by phosphorylation.
Ubiquitination generally regulates protein degradation<span class="citation" data-cites="1CHaUvQ1A"><sup><a href="#ref-1CHaUvQ1A" role="doc-biblioref">37</a></sup></span>, SUMOylation regulates chromatin structure, DNA repair, transcription, and cell-cycle progression<span class="citation" data-cites="M4mSYOgE 4iAess79"><sup><a href="#ref-M4mSYOgE" role="doc-biblioref">38</a>,<a href="#ref-4iAess79" role="doc-biblioref">39</a></sup></span>, and palmitoylation regulates the maintenance of the structural organization of exosome-like extracellular vesicle membranes<span class="citation" data-cites="blcqlkZJ"><sup><a href="#ref-blcqlkZJ" role="doc-biblioref">40</a></sup></span>.
Glycosylation is a ubiquitous modification that regulates various T cell functions, such as cellular migration, T cell receptor signaling, cell survival, and apoptosis<span class="citation" data-cites="V8FVI0EN 889bLrDQ"><sup><a href="#ref-V8FVI0EN" role="doc-biblioref">41</a>,<a href="#ref-889bLrDQ" role="doc-biblioref">42</a></sup></span>.
Deregulation of PTMs is linked to cellular stress and diseases<span class="citation" data-cites="1FiLH8A0G"><sup><a href="#ref-1FiLH8A0G" role="doc-biblioref">43</a></sup></span>.</p>
<p>Several non-MS methods exist to study PTMs, including in vitro PTM reaction tests with colorimetric assays, radioactive isotope-labeled substrates, western blot with PTM-specific antibodies and superbinders, and peptide and protein arrays<span class="citation" data-cites="fzBRT8ZP Wcw3KfEB QjEZsYB2"><sup><a href="#ref-fzBRT8ZP" role="doc-biblioref">44</a>–<a href="#ref-QjEZsYB2" role="doc-biblioref">46</a></sup></span>.
While effective, these approaches have many limitations, such as inefficiency and difficulty in producing pan-specific antibodies.
MS-based proteomics approaches are currently the predominant tool for identifying and quantifying changes in PTMs.</p>
<h2 class="page_break_before" id="types-of-experiments">2. Types of Experiments</h2>
<p>A wide range of questions are addressable with proteomics technology, which translates to a wide range of variations of proteomics workflows.
In some workflows, the identification of proteins in a given sample is desired.
For other experiments, the quantification of as many proteins as possible is essential for the success of the study.
Therefore, proteomic experiments can be both qualitative and quantitative.
The following sections give an overview of several common proteomics experiments.</p>
<h3 id="protein-abundance-changes">Protein abundance changes</h3>
<p>A common experiment is a discovery-based, unbiased mapping of proteins along with detection of changes in their abundance across sample groups.
This is achieved using methods such as label free quantification (LFQ) or isobaric tagging, which are described in more detail in subsequent sections.
In these experiments, data should be collected from at least three biological replicates of each condition to estimate the variance of measuring each protein.
Depending on the experiment design, different statistical tests are used to calculate changes in measured protein abundances between groups.
If there are only two groups, the quantities might be compared with a t-test or with a Wilcoxon signed-rank test.
The latter is a non-parametric version of the Student’s t-test.
If there are more than two sample groups, then Analysis of Variance (ANOVA) is used instead, followed by a post-hoc test such as Tukey’s honestly significant difference test to discover pairwise differences.
With either testing scheme, the p-values from the first set of tests must be corrected for multiple testing.
A common method for p-value correction is the Benjamini-Hochberg method<span class="citation" data-cites="UeFCwwR7"><sup><a href="#ref-UeFCwwR7" role="doc-biblioref">47</a></sup></span>.
These types of experiments have revealed wide ranges of proteomic remodeling from various biological systems.</p>
<h3 id="ptms-1">PTMs</h3>
<p>Proteins may become decorated with various chemical modifications during or after translation<span class="citation" data-cites="yYHlJgbC"><sup><a href="#ref-yYHlJgbC" role="doc-biblioref">33</a></sup></span>, or through proteolytic cleavage such as N-terminal methionine removal<span class="citation" data-cites="13Yxz6MsM"><sup><a href="#ref-13Yxz6MsM" role="doc-biblioref">48</a></sup></span>.
Several proteomics methods are available to detect and quantify each specific type of modification.
See also the section on Protein/Peptide Enrichment and Depletion.
For a good online resource listing potential modifications, sites of attachment, and their mass differences, the website www.unimod.org is an excellent curated and freely accessible database.</p>
<h4 id="phosphoproteomics">Phosphoproteomics</h4>
<p>Phosphoproteomics is the study of protein phosphorylation, wherein a phosphate group is covalently attached to a protein side-chain (most commonly serine, threonine, or tyrosine).
Although western blotting can measure one phosphorylation site at a time (if using a monoclonal antibody), mass spectrometry-based proteomics can measure thousands of sites from a sample at the same time.
After proteolysis of the proteome, to achieve competitive coverage of the phosphoproteome, phosphopeptides need to be enriched to be detected by mass spectrometry.
Various methods of enrichment have been developed<span class="citation" data-cites="E5n1UfBA N0xHE81w 4q5oxW6 1E7oJsvFz"><sup><a href="#ref-E5n1UfBA" role="doc-biblioref">49</a>–<a href="#ref-1E7oJsvFz" role="doc-biblioref">52</a></sup></span>.</p>
<p>A key challenge of phosphoproteomics is the limit of detection.
It is important to ensure that there is a sufficient amount of protein before conducting a phosphoproteomics project because phosphorylated proteins and peptides may represent only ~1% of the total protein.
Many phosphoproteomics workflows start with at least 1 mg of total protein per sample.
In addition to low stoichiometry, phosphorylation is very labile, and for this reason, great care must be taken in the collection and storage of samples for phosphoproteomics, where proteome denaturation should be rapid and aggressive while including phosphatase inhibitors.
Newer, more sensitive instrumentation is enabling detection of protein phosphosites from much less material, down to the nanogram-level of peptide loading on the the LC-MS system.
Despite advancement in phosphoproteomics technology, the following challenges still exist: limited sample amounts, highly complex samples, and wide dynamic range<span class="citation" data-cites="EiFGJCed"><sup><a href="#ref-EiFGJCed" role="doc-biblioref">53</a></sup></span>.
Additionally, phosphoproteomic analysis is often time-consuming and requires the use of expensive equipment such as enrichment kits.</p>
<p>See the Peptide/Protein Enrichment and Depletion section for more details.</p>
<h4 id="glycoproteomics">Glycoproteomics</h4>
<p>The importance of protein glycosylation in health and disease has been known for a long time, but do to high analytical difficulty, only recently has their large scale analysis been gaining traction.
Protein glycosylation sites can be N-linked (asparagine-linked) or O-linked (serine/threonine-linked).
Understanding the function of protein glycosylation will help us understand numerous biological processes since this is a universal protein modification across all domains of life, especially at the cell surface<span class="citation" data-cites="PNr94DXb MH9QSnbc nSDlzuSO buMJaA2S"><sup><a href="#ref-PNr94DXb" role="doc-biblioref">54</a>–<a href="#ref-buMJaA2S" role="doc-biblioref">57</a></sup></span>.</p>
<p>Studies of phosphorylation and glycosylation share several experimental pipeline steps including sample preparation.
Protein clean-up approaches for glycoproteomics may differ from other proteomics experiments because glycopeptides are more hydrophilic than most peptides.
Some approaches mentioned in the literature include: filter-aided sample preparation (FASP), suspension traps (S-traps), and protein aggregation capture (PAC)<span class="citation" data-cites="PNr94DXb Iygi3cGd 1FU2NfVYY yO8bBfo8 7zlczgkb t0hz9mu6 VTprpeAT"><sup><a href="#ref-PNr94DXb" role="doc-biblioref">54</a>,<a href="#ref-Iygi3cGd" role="doc-biblioref">58</a>,<a href="#ref-1FU2NfVYY" role="doc-biblioref">59</a>–<a href="#ref-VTprpeAT" role="doc-biblioref">63</a></sup></span>.
Multiple proteases may be used to increase the sequence coverage and detect more modification sites, such as: trypsin, chymotrypsin, pepsin, WaLP/MaLP<span class="citation" data-cites="yeQvImXS"><sup><a href="#ref-yeQvImXS" role="doc-biblioref">64</a></sup></span>, GluC, AspN, pronase, proteinase K, OgpA, StcEz, BT4244, AM0627, AM1514, AM0608, Pic, ZmpC, CpaA, IMPa, PNGase F, Endo F, Endo H, and OglyZOR<span class="citation" data-cites="PNr94DXb"><sup><a href="#ref-PNr94DXb" role="doc-biblioref">54</a></sup></span>.
Mass spectrometry has improved over the past decade, and now many strategies are available for glycoprotein structure elucidation and glycosylation site quantification<span class="citation" data-cites="PNr94DXb"><sup><a href="#ref-PNr94DXb" role="doc-biblioref">54</a></sup></span>.
See also the section on “AminoxyTMT Isobaric Mass Tags” as an example quantitative glycoproteomics method.</p>
<h3 id="structural-techniques">Structural techniques</h3>
<p>Almost all proteins (except for intrinsically disordered proteins<span class="citation" data-cites="nOIi7YMn"><sup><a href="#ref-nOIi7YMn" role="doc-biblioref">65</a></sup></span>) fold into three-dimensional (3D) structures either by themselves or assisted by molecular chaperones<span class="citation" data-cites="aq0DyuaG"><sup><a href="#ref-aq0DyuaG" role="doc-biblioref">66</a></sup></span>.
There are four levels relevant to the folding of any protein:</p>
<ul>
<li><p>Primary structure:
The protein’s linear amino acid sequence, with amino acids connected through peptide bonds.</p></li>
<li><p>Secondary structure:
The amino acid chain’s folding: α-helix, β-sheet or turn.</p></li>
<li><p>Tertiary structure:
The three-dimensional structure of the protein.</p></li>
<li><p>Quaternary structure:
The structure of several protein molecules/subunits in one complex.</p></li>
</ul>
<p>Of recent note, the development of AlphaFold, has enabled the high-accuracy three-dimensional structural prediction of all human proteins and for proteins of many other species, enabling a more thorough study of protein folding and is used to predict the relationship between fold and function<span class="citation" data-cites="vsPwR5xs 11Etsn1JV"><sup><a href="#ref-vsPwR5xs" role="doc-biblioref">67</a>,<a href="#ref-11Etsn1JV" role="doc-biblioref">68</a></sup></span>.
Several proteomics methods have been developed to reveal protein structure information for simple and complex systems.</p>
<h4 id="cross-linking-mass-spectrometry-xl-ms">Cross-linking mass spectrometry (XL-MS)</h4>
<p>XL-MS is an emerging technology in the field of proteomics.
It can be used to determine changes in protein-protein interactions and/or protein structure.
XL-MS covalently locks interacting proteins together to preserve interactions and proximity during MS analysis.
XL-MS is different from traditional MS in that it requires the identification of chimeric MS/MS spectra from cross-linked peptides<span class="citation" data-cites="waP2bBmk XYcoBTz1"><sup><a href="#ref-waP2bBmk" role="doc-biblioref">69</a>,<a href="#ref-XYcoBTz1" role="doc-biblioref">70</a></sup></span>.
XL-MS can be used to gain structural contraints in purified protein systems or at the whole proteome scale.</p>
<p>The common steps in a XL-MS workflow are as follows<span class="citation" data-cites="v6mVdK34"><sup><a href="#ref-v6mVdK34" role="doc-biblioref">71</a></sup></span>:<br />
1. Generate a system with protein-protein interactions of interest (<em>in vitro</em> or <em>in vivo</em><span class="citation" data-cites="JnaXzrse"><sup><a href="#ref-JnaXzrse" role="doc-biblioref">72</a></sup></span>)<br />
2. Add a cross-linking reagent to covalently connect adjacent protein regions (such as disuccinimidyl sulfoxide, DSSO)<span class="citation" data-cites="XYcoBTz1"><sup><a href="#ref-XYcoBTz1" role="doc-biblioref">70</a></sup></span><br />
3. Proteolysis to produce peptides<br />
4. MS/MS data collection<br />
5. Identify cross-linked peptide pairs using special software (i.e. pLink<span class="citation" data-cites="KW2lStHJ"><sup><a href="#ref-KW2lStHJ" role="doc-biblioref">73</a></sup></span>, Kojak<span class="citation" data-cites="10TqPZVdy KIekIPp9"><sup><a href="#ref-10TqPZVdy" role="doc-biblioref">74</a>,<a href="#ref-KIekIPp9" role="doc-biblioref">75</a></sup></span>, xQuest<span class="citation" data-cites="jq9D2t6h"><sup><a href="#ref-jq9D2t6h" role="doc-biblioref">76</a></sup></span>, XlinkX<span class="citation" data-cites="19ZFgWCt6"><sup><a href="#ref-19ZFgWCt6" role="doc-biblioref">77</a></sup></span>)<br />
6. Generate cross-link maps for structural modeling and visualization<span class="citation" data-cites="1F05pgVZ9 1EF7o7CB4"><sup><a href="#ref-1F05pgVZ9" role="doc-biblioref">78</a>,<a href="#ref-1EF7o7CB4" role="doc-biblioref">79</a></sup></span>
(optional: 7. Use detected cross-links for protein-protein docking<span class="citation" data-cites="RInMQu7q"><sup><a href="#ref-RInMQu7q" role="doc-biblioref">80</a></sup></span>)</p>
<h4 id="hydrogen-deuterium-exchange-mass-spectrometry-hdx-ms">Hydrogen deuterium exchange mass spectrometry (HDX-MS)</h4>
<p>HDX-MS works by detecting changes in peptide mass due to exchange of amide hydrogens of the protein backbone with deuterium from D<sub>2</sub>O<span class="citation" data-cites="1Dqlo9aRm"><sup><a href="#ref-1Dqlo9aRm" role="doc-biblioref">81</a></sup></span>.
The exchange rate depends on the protein solvent accessible surface area, dynamics, and the properties of the amino acid sequence<span class="citation" data-cites="1Dqlo9aRm cZM0UPqp SZzVEJCj H647Pppu"><sup><a href="#ref-1Dqlo9aRm" role="doc-biblioref">81</a>–<a href="#ref-H647Pppu" role="doc-biblioref">84</a></sup></span>.
Although using D<sub>2</sub>O to make deuterium-labeled samples is simple, HDX-MS requires several controls to ensure that experimental conditions capture the dynamics of interest<span class="citation" data-cites="1Dqlo9aRm 1CHLgMzps 19wzQMZwm m2HWEBQZ"><sup><a href="#ref-1Dqlo9aRm" role="doc-biblioref">81</a>,<a href="#ref-1CHLgMzps" role="doc-biblioref">85</a>–<a href="#ref-m2HWEBQZ" role="doc-biblioref">87</a></sup></span>.
If the peptide dissociation process is tuned appropriately, residue-level quantification of changes in solvent accessibility are possible within a measured peptide<span class="citation" data-cites="whtWKxfa"><sup><a href="#ref-whtWKxfa" role="doc-biblioref">88</a></sup></span>.
HDX can produce precise protein structure measurements with high reproducibility.
Masson <em>et al.</em> gave recommendations on how to prep samples, conduct data analysis, and present findings in a detailed stepwise manner<span class="citation" data-cites="1Dqlo9aRm"><sup><a href="#ref-1Dqlo9aRm" role="doc-biblioref">81</a></sup></span>.</p>
<h4 id="radical-footprinting">Radical Footprinting</h4>
<p>This technique uses hydroxyl radical footprinting and MS to elucidate protein structures, assembly, and interactions within a large macromolecule<span class="citation" data-cites="17FfyYJKn dyFOYdP5"><sup><a href="#ref-17FfyYJKn" role="doc-biblioref">89</a>,<a href="#ref-dyFOYdP5" role="doc-biblioref">90</a></sup></span>.
In addition to proteomics applications, various approaches to make hydroxide radicals have also been applied for footprinting studies in nucleic acid/ligand interactions<span class="citation" data-cites="8tWrfuT2 b8XE9cl8 14ACSyAnR"><sup><a href="#ref-8tWrfuT2" role="doc-biblioref">91</a>–<a href="#ref-14ACSyAnR" role="doc-biblioref">93</a></sup></span>.
This chapter is very useful in learning more about this topic:<span class="citation" data-cites="muDfX3XR"><sup><a href="#ref-muDfX3XR" role="doc-biblioref">94</a></sup></span>.</p>
<p>There are several methods of producing radicals for protein footprinting:<br />
1. Fenton and Fenton‐like Chemistry<span class="citation" data-cites="17FfyYJKn zy4wero bAhX0GGr"><sup><a href="#ref-17FfyYJKn" role="doc-biblioref">89</a>,<a href="#ref-zy4wero" role="doc-biblioref">95</a>,<a href="#ref-bAhX0GGr" role="doc-biblioref">96</a></sup></span><br />
2. Electron-Pulse Radiolysis<span class="citation" data-cites="17FfyYJKn GyzG2odq"><sup><a href="#ref-17FfyYJKn" role="doc-biblioref">89</a>,<a href="#ref-GyzG2odq" role="doc-biblioref">97</a></sup></span><br />
3. High‐Voltage Electrical Discharge<span class="citation" data-cites="17FfyYJKn tcDiyhQT"><sup><a href="#ref-17FfyYJKn" role="doc-biblioref">89</a>,<a href="#ref-tcDiyhQT" role="doc-biblioref">98</a></sup></span><br />
4. Synchrotron X‐ray Radiolysis of Water<span class="citation" data-cites="17FfyYJKn gryR5tjl"><sup><a href="#ref-17FfyYJKn" role="doc-biblioref">89</a>,<a href="#ref-gryR5tjl" role="doc-biblioref">99</a></sup></span><br />
5. Plasma Formation of OH Radicals<span class="citation" data-cites="17FfyYJKn 43HI8GWO"><sup><a href="#ref-17FfyYJKn" role="doc-biblioref">89</a>,<a href="#ref-43HI8GWO" role="doc-biblioref">100</a></sup></span><br />
6. Photolysis of Hydrogen Peroxide<span class="citation" data-cites="17FfyYJKn pFnbzp8Y"><sup><a href="#ref-17FfyYJKn" role="doc-biblioref">89</a>,<a href="#ref-pFnbzp8Y" role="doc-biblioref">101</a></sup></span></p>
<h4 id="fast-photochemical-oxidation-of-proteins-fpop-doi10.1074jbc.rev119.006218">Fast photochemical oxidation of proteins (FPOP)<span class="citation" data-cites="IH1lj0uY"><sup><a href="#ref-IH1lj0uY" role="doc-biblioref">102</a></sup></span></h4>
<p>FPOP is an example of a radical footprinting method.
In FPOP, a laser-based hydroxyl radical protein footprinting MS method that relies on the irreversible labeling of solvent-exposed amino acid side chains by hydroxyl radicals in order to understand structure of proteins.
A laser produces 248 nm light that causes hydrogen peroxide to break into a pair of hydroxyl radicals<span class="citation" data-cites="pFnbzp8Y cMJbHB1a"><sup><a href="#ref-pFnbzp8Y" role="doc-biblioref">101</a>,<a href="#ref-cMJbHB1a" role="doc-biblioref">103</a></sup></span>.
The flow rate of solution through the capillary and laser frequency are adjusted such that each protein molecule is irradiated only once.
After they are irradiated, the sample is collected in a tube that contains catalase and free methionine in the buffer, quenching the H2O2 and hydroxyl radicals and preventing secondary modification of residues that become exposed due to unfolding after the initial labeling.
Control samples are made by running the sample through the flow system without any irradiation.
Another experimental control involves the addition of a radical scavenger to tune the extent of protein oxidation<span class="citation" data-cites="LgS9wDAM sqSJTywu"><sup><a href="#ref-LgS9wDAM" role="doc-biblioref">104</a>,<a href="#ref-sqSJTywu" role="doc-biblioref">105</a></sup></span>.
FPOP has wide application for proteins including measurements of fast protein folding and transient dynamics.</p>
<h4 id="protein-painting-doi10.1038ncomms5413-doi10.1074jbc.ra118.007310">Protein Painting<span class="citation" data-cites="sZQji0TC 17xQ2aw0Q"><sup><a href="#ref-sZQji0TC" role="doc-biblioref">106</a>,<a href="#ref-17xQ2aw0Q" role="doc-biblioref">107</a></sup></span></h4>
<p>Protein painting uses “molecular paints” to noncovalently coat the solvent-accessible surface of proteins.
Chemically, these paints may be small aryl hydrocarbon dyes with fast on-rates with very slow off-rates<span class="citation" data-cites="sZQji0TC"><sup><a href="#ref-sZQji0TC" role="doc-biblioref">106</a></sup></span>.
These paint molecules will coat the protein surfaces but will not have access to the hydrophobic cores or protein-protein interface regions that solvents cannot access.
If the “paint” covers free amines of lysine side chains, the “painted” parts will be protected from trypsin cleavage, while the unpainted areas will.
After proteolysis, the peptides samples will be subjected to MS.
A lack of proteolysis in a region is interpreted as solvent accessibility, which gives rough structural information about complex protein mixtures or even a whole proteome.</p>
<h4 id="lip-ms-limited-proteolysis-mass-spectrometry-doi10.1016j.tibs.2020.05.006-doi10.1038nprot.2017.100-doi10.1038s41596-022-00771-x-doi10.1038nprot.2017.100">LiP-MS (Limited Proteolysis Mass Spectrometry)<span class="citation" data-cites="gdTpkDDu xTQMmTt5 6IZfi1O xTQMmTt5"><sup><a href="#ref-gdTpkDDu" role="doc-biblioref">108</a>,<a href="#ref-xTQMmTt5" role="doc-biblioref">109</a>,<a href="#ref-xTQMmTt5" role="doc-biblioref">109</a>,<a href="#ref-6IZfi1O" role="doc-biblioref">110</a></sup></span></h4>
<p>Limited proteolysis coupled to mass spectrometry (LiP-MS) is a method that tracks structural changes in complex proteomes in response to a variety of perturbations or stimuli.
The underlying tenet of LiP-MS is that a stimuli-induced change in native protein structure (i.e. protein-protein interaction, introduction of a PTM, ligand/substrate binding, or changes in osmolarity or ambient temperature) can be detected by a change in accessibility of a broad-specificity protease (i.e. proteinase K) to the region(s) of the protein where the structural change occurs.
For example, small molecule binding may render a disordered region protected from non-specific proteolysis by directly blocking access of the protease to the cleavage site.
LiP-MS can therefore provide a somewhat unbiased view of structural changes at the proteome scale.
Importantly, LiP-MS necessitates cell lysates or individual proteins be maintained in their native state prior to or during perturbation and protease treatment.
LiP-MS can also be applied to membrane suspensions, to facilitate the study of membrane proteins without the need for purification or detergents<span class="citation" data-cites="UETOKcpN"><sup><a href="#ref-UETOKcpN" role="doc-biblioref">111</a></sup></span>.<br />
For additional information about LiP-MS, please refer to the following article:<span class="citation" data-cites="1BxnTxtGI"><sup><a href="#ref-1BxnTxtGI" role="doc-biblioref">112</a></sup></span></p>
<h3 id="protein-stability-and-small-molecule-binding">Protein stability and small molecule binding</h3>
<h4 id="cellular-thermal-shift-assay-cetsa-pmid23828940-doi10.1016j.cbpa.2019.11.004">Cellular Thermal Shift Assay (CETSA)<span class="citation" data-cites="1HcZz8Apd rMCVhIUo"><sup><a href="#ref-1HcZz8Apd" role="doc-biblioref">113</a>,<a href="#ref-rMCVhIUo" role="doc-biblioref">114</a></sup></span></h4>
<p>CETSA involves subjecting a protein sample to a thermal shift assay (TSA), in which the protein is exposed to a range of temperatures, and the resulting changes in protein stability by quantifying protein remaining in the soluble fraction.
This is done in live cells immediately before lysis, or in non-denaturing lysates.
The original paper reported this method using immunoaffinity approaches for detecting changes in soluble protein.
The assay is capable of detecting shifts in the thermal equilibrium of cellular proteins in response to a variety of perturbations, but most commonly in response to in vitro drug treatments.</p>
<h4 id="thermal-proteome-profiling-tpp-doi10.15252msb.20199232-doi10.1016j.cbpa.2022.102225-doi10.1021jacs.1c10621-doi10.10029781118970195.ch11">Thermal proteome profiling (TPP)<span class="citation" data-cites="JOskUci 8UcGwfym 89c9sDdK Ssg2QwSE"><sup><a href="#ref-JOskUci" role="doc-biblioref">115</a>–<a href="#ref-Ssg2QwSE" role="doc-biblioref">118</a></sup></span></h4>
<p>Thermal proteome profiling (TPP) follows the same principle as CETSA, but has been extended to use an unbiased mass spectrometry readout of many proteins.
By measuring changes in thermal stability of thousands of proteins, binding to an unknown or unexpected protein can be discovered.
During a typical TPP experiment, a protein sample is first treated with a drug of interest to stabilize protein-ligand interactions.
The sample is then divided into multiple aliquots, which are subjected to different temperatures to induce thermal denaturation.
The resulting drug-induced changes in protein stability curves are detected using mass spectrometry.
By comparing protein stability curves across the temperatures between treatment conditions, TPP can provide insight into the proteins that bind a ligand.</p>
<h3 id="protein-protein-interactions-ppis">Protein-protein interactions (PPIs)</h3>
<h4 id="affinity-purification-coupled-to-mass-spectrometry-ap-ms-doi10.1002iub.1081-doi10.1038nrg3356-doi10.1038nmeth.2400">Affinity purification coupled to mass spectrometry (AP-MS)<span class="citation" data-cites="ieBDHPPg DORXXewQ 1BsZO9WFn"><sup><a href="#ref-ieBDHPPg" role="doc-biblioref">119</a>–<a href="#ref-1BsZO9WFn" role="doc-biblioref">121</a></sup></span></h4>
<p>AP-MS is an approach that involves enrichment of a target protein or protein complex using an antibody with specificity toward a protein of interest followed by mass spectrometry to identify the interacting proteins.
If there are no good antibodies for immunoprecipitation of a protein of interest, it may be genetically tagged with an affinity epitope, such as a FLAG or hemagglutinin, which is used to selectively capture the target protein using an antibody against that epitope.
In either case, the protein complex is then purified from the sample using a series of wash steps, and the interacting proteins are identified using mass spectrometry.
The success of AP-MS experiments depends on many factors, including the quality of the antibody used for purification, the specificity and efficiency of the resin used for capture, and the sensitivity and resolution of the mass spectrometer.
In addition, careful experimental design and data analysis are critical for accurately identifying and interpreting protein-protein interactions.</p>
<p>AP-MS has been used to study a wide range of biological processes, including signal transduction pathways, protein complex dynamics, and protein post-translational modifications.
AP-MS has been performed on a whole proteome scale as part of the BioPlex project<span class="citation" data-cites="yeZfXPV6 m46yvHfn VMc1jEkd"><sup><a href="#ref-yeZfXPV6" role="doc-biblioref">122</a>–<a href="#ref-VMc1jEkd" role="doc-biblioref">124</a></sup></span>.</p>
<p>Despite its widespread use, AP-MS has some limitations, including non-specific interactions, the difficulty in interpreting complex data sets, and the possibility of missing important interacting partners due to constraints in sensitivity or specificity.
However, with continued advances in technology and data analysis methods, AP-MS is likely to remain a valuable tool for studying protein-protein interactions.</p>
<p>There are other variants of this experiment where instead of an antibody against the protein of interest, the protein of interest can itself be conjugated to a solid phase by expression and purification with a his-tag or fusion to a glutathione s-transferase (GST) domain.
These approaches may be useful when good antibodies for IP are not available.</p>
<p>The interaction of any two proteins depends both on their concentrations and their affinity for each other; two proteins could have low affinity for each other, but if present at high concentrations, they will be found together in AP-MS.
Therefore, key considerations for AP-MS studies are to include negative control antibodies to help distinguish true interactions from background, and including many replicates to assess reproducibility.</p>
<h4 id="apex-peroxidase-doi10.1038nbt.2375-doi10.1126science.1230593">APEX peroxidase<span class="citation" data-cites="LkwAXvyl 1BbkEbfDz"><sup><a href="#ref-LkwAXvyl" role="doc-biblioref">125</a>,<a href="#ref-1BbkEbfDz" role="doc-biblioref">126</a></sup></span></h4>
<p>APEX-MS is a labeling technique that utilizes a peroxidase genetically fused to a protein of interest.
When biotin-phenol is transiently added in the presence of hydrogen peroxide, nearby proteins are covalently biotinylated<span class="citation" data-cites="D5NLgmhd"><sup><a href="#ref-D5NLgmhd" role="doc-biblioref">127</a></sup></span>.
APEX thereby enables the discovery of interacting proteins in living cells.
One of the major advantages of APEX is its ability to label proteins in their native environment, allowing for the identification of interactions that occur under physiological conditions.
A key benefit of APEX is that it can detect transient or weak interactors, unlike AP-MS that detects strong and stable interactions.
Despite its advantages, APEX has some limitations, including the potential for non-specific labeling, the difficulty in distinguishing between direct and indirect interactions, and the possibility of missing interactions that occur at low abundance or in regions of the cell that are not effectively labeled.</p>
<h4 id="proximity-dependent-biotin-identification-bioid-doi10.1083jcb.201112098-doi10.1074mcp.r120.001941-doi10.1002pmic.201600123-doi10.1091mbc.e15-12-0844">Proximity-dependent biotin identification (BioID)<span class="citation" data-cites="zSgLeSta NFFhvScy 1FVEjFnHE jJJVaczi"><sup><a href="#ref-zSgLeSta" role="doc-biblioref">128</a>–<a href="#ref-jJJVaczi" role="doc-biblioref">131</a></sup></span></h4>
<p>BioID is a proximity labeling technique that allows for the identification of protein-protein interactions.
BioID involves the genetic tagging of a protein of interest with a promiscuous biotin ligase in live cells, which then biotinylates proteins in close proximity to the protein of interest.
One of the advantages of BioID is its ability to label proteins in their native environment, allowing for the identification of interactions that occur under physiological conditions.
BioID has been used to identify a wide range of protein interactions, including receptor-ligand interactions, signaling complexes, and protein localization.
BioID is a slower reaction than APEX and therefore may pick up even more transient interactions that occur on longer timescales.
A newer alternative to BioID called TurboID has much higher activity, and is now more commonly used<span class="citation" data-cites="BlJpACnb"><sup><a href="#ref-BlJpACnb" role="doc-biblioref">132</a></sup></span>.
BioID has the same limitations as APEX.
For more information on BioID, please refer to<span class="citation" data-cites="sas7Wv3H"><sup><a href="#ref-sas7Wv3H" role="doc-biblioref">133</a></sup></span>.</p>
<h2 class="page_break_before" id="protein-extraction">3. Protein Extraction</h2>
<p>Protein extraction is the initial phase of any mass spectrometry-based proteomics experiment.
Protein extraction is sample dependent; a solution that is effective for plasma proteomics may not work well for plant tissue proteomics.
Thought should be given to any planned downstream assays, such as specific proteolysis requirements (LiP-MS, PTM enrichments, enzymatic reactions, glycan purification or hydrogen-deuterium exchange experiments), long-term project goals (reproducibility, multiple sample types, low abundance samples), as well as to the experimental question (coverage of a specific protein, subcellular proteomics, global proteomics, protein-protein interactions or affinity enrichment of specific classes of modifications).
The 2009 version of Methods in Enzymology: guide to Protein Purification<span class="citation" data-cites="emz5eXnO"><sup><a href="#ref-emz5eXnO" role="doc-biblioref">134</a></sup></span> serves as a deep dive into how molecular biologists and biochemists traditionally carried out protein extraction.
The Protein Protocols handbook<span class="citation" data-cites="LvSVXQeu"><sup><a href="#ref-LvSVXQeu" role="doc-biblioref">135</a></sup></span> and the excellent review by Linn<span class="citation" data-cites="YFivV9n5"><sup><a href="#ref-YFivV9n5" role="doc-biblioref">136</a></sup></span> are good sources of general proteomics protocols.
Another excellent resource is the “Proteins and Proteomics: A Laboratory Manual” by Richard J. Simpson<span class="citation" data-cites="PUTrxDaK PUTrxDaK"><sup><a href="#ref-PUTrxDaK" role="doc-biblioref">137</a>,<a href="#ref-PUTrxDaK" role="doc-biblioref">137</a></sup></span>.
This manual is 926 pages packed full of bench tested protocols and procedures for carrying out protein centric studies.
Any change in extraction conditions should be expected to create potential changes in downstream results.
Be sure to plan and optimize the protein extraction step first and use a protocol that works for your needs.
To reproduce the results of another study, one should begin with the same extraction protocols.</p>
<p>Learning the fundamentals and mechanisms of how and why sample preparation steps are performed is vital because it enables flexibility to perform proteomics from a wide range of samples.
For bottom-up proteomics, the overreaching goal is efficient and consistent extraction and digestion.
A range of mechanical and non-mechanical extraction protocols have been developed and the choice of technique is generally dictated by sample type or assay requirements (i.e. native versus non-native extraction).
Extraction can be aided by the addition of detergents and/or chaotropes to the sample, but care should be taken that these additives do not interfere with the sample digestion step or downstream mass-spectrometry analysis.</p>
<p>In general, a safe and common choice for standard proteomic protein extraction would be to use 8 M urea in 100 mM Tris, pH 8.5; the pH is based on optimum trypsin activity<span class="citation" data-cites="Cj5ZGyu8"><sup><a href="#ref-Cj5ZGyu8" role="doc-biblioref">138</a></sup></span>.
Desalting with StageTips, Waters’ SepPaks, or similiar would yield clean peptides.
Triton X-100 and NP-40 should be avoided at all costs.
The following sections detail the range of choices that are available.</p>
<h3 id="buffer-and-denaturant-choice">Buffer and denaturant choice</h3>
<h4 id="general-proteomics">General proteomics</h4>
<p>A common question to proteomics core facilities is, “What is the best buffer for protein extraction?”
Unfortunately, there is no one correct answer.
For global proteomics experiments where maximizing the number of protein or peptide identifications is a goal, 50-100 mM of a neutral pH buffer (pH 7.5-8.5) is often used with a strong denaturant.
Relevant factors for buffer choice include cost, volatility, and reactivity such as primary amine containing.
Volatile choices like ammonia bicarbonate are desirable because they can be removed by lyophilization.
However, ammonium bicarbonate promotes methionine oxidation and we generally suggest Tris instead to minimize oxidation.
Tris is desirable due to low cost but can act as a chelator and contains a primary amine, which may be incompatible with some conditions, like TMT labeling.
Table 1 summarizes common buffers.
A great online resource to help calculate buffer compositions and pH values is the website by Robert Beynon at http://phbuffers.org.
Although there are a range of buffers that can be used to provide the correct working pH and ionic strength, not all buffers are compatible with downstream workflows.</p>
<p>Table 1: Common buffers used for proteomic sample preparation.</p>
<table>
<colgroup>
<col style="width: 56%" />
<col style="width: 43%" />
</colgroup>
<thead>
<tr class="header">
<th>Buffer</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Phosphate buffered saline (PBS)</td>
<td>Non-volatile, inert</td>
</tr>
<tr class="even">
<td>tris(hydroxymethyl)aminomethane (Tris)</td>
<td>Cheap, non-volatile, primary amine containing</td>
</tr>
<tr class="odd">
<td>4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES)</td>
<td>More expensive, non-volatile</td>
</tr>
<tr class="even">
<td>ammonium bicarbonate</td>
<td>Cheap, volatile, primary amine containing</td>
</tr>
<tr class="odd">
<td>triethanolamine bicarbonate</td>
<td>cheap, volatile, non-primary amine containing</td>
</tr>
</tbody>
</table>
<p>Complete and quick denaturation of proteins in the sample is required to limit changes to protein status by endogenous proteases, kinases, phosphatases, and other enzymes.
For this reason, buffers must be used in conjunction with a chaotrope or surfactant to denature and solubilize proteins<span class="citation" data-cites="OZvOVLht bpTXhTR5"><sup><a href="#ref-OZvOVLht" role="doc-biblioref">139</a>,<a href="#ref-bpTXhTR5" role="doc-biblioref">140</a></sup></span>.
The choice of denaturant should be governed by compatibility with the protease (typically trypsin) and peptide cleanup steps must be considered.
Table 2 lists common denaturants used for proteomics.
Urea is an easy and a common choice because it is compatible with trypsin at &lt;2M and it can be removed by desalting.
However, urea induces carbamylation, which is made worse with sample heating<span class="citation" data-cites="B8hLhFif"><sup><a href="#ref-B8hLhFif" role="doc-biblioref">141</a></sup></span>.
If intact protein separations are planned (based on size or isoelectric point), choose a denaturant compatible with those methods, such as sodium dodecyl sulphate (SDS)<span class="citation" data-cites="Igj2yRI0"><sup><a href="#ref-Igj2yRI0" role="doc-biblioref">142</a></sup></span>.
SDS is a strong denaturant, but it is not compatible with trypsin or reversed phase materials.
Sodium deoxycholate (SDC) and sodium laurate (SL) are also strong denaturants with the added benefit of compatibility with trypsin.
For non-MS workflows, detergents containing poly ethylene glycol tails are common, such as triton X-100.
SDS, SDC, SL, and triton X-100 are incompatible with LC-MS workflows as they can cause ion suppression and column clogging.
Therefore detergents must be removed before further protein processing.
Detergent removal options differ based on the chemistry of the detergent.
Alternatively, mass-spectrometry-compatible detergents may be used, such as n-dodecyl-beta-maltoside (DDM)<span class="citation" data-cites="ifXiaFI9"><sup><a href="#ref-ifXiaFI9" role="doc-biblioref">143</a></sup></span>.</p>
<p>Table 2: Common denaturants used for proteomic sample preparation.</p>
<table>
<colgroup>
<col style="width: 56%" />
<col style="width: 43%" />
</colgroup>
<thead>
<tr class="header">
<th>Denaturant</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>8M Urea</td>
<td>Non-volatile, chemically reactive, limit heating, must be diluted to &lt; 2M before trypsin addition</td>
</tr>
<tr class="even">
<td>1-5% sodium dodecyl sulfate (SDS)</td>
<td>Cheap, strong denaturation and hydrophobicity, must be removed before trypsin</td>
</tr>
<tr class="odd">
<td>1-5% Sodium Deoxycholate</td>
<td>Hydrophobic for membrane proteins, easy to remove due to precipiation with acid</td>
</tr>
<tr class="even">
<td>n-dodecyl-beta-maltoside</td>
<td>expensive, low amounts can be used with trypsin and LC-MS</td>
</tr>
<tr class="odd">
<td>Triton X-100</td>
<td>Do not use this. If samples already have this or NP-40, proceed with protein precipitation</td>
</tr>
</tbody>
</table>
<h4 id="detergent-removal">Detergent removal</h4>
<p>Relatively low concentrations of some detergents, such as 1% SDC, or chaotropes such as 1M urea, are compatible with proteolysis by trypsin/Lys-C.
Often proteolysis-compatible concentrations of these detergents and chaotropes are achieved by diluting the sample in appropriate buffer (i.e. 100 mM ammonium bicarbonate, pH 8.5) after cell or tissue lysis in a higher concentration.
However, most detergents should be removed prior to enzymatic hydrolysis.
This is generally performed through precipitation of proteins.
The most common types are 1) acetone, 2) trichloroacetic acid (TCA), and 3) chloroform/methanol/water (Folch)<span class="citation" data-cites="xvenBhjQ OMzPHtgd"><sup><a href="#ref-xvenBhjQ" role="doc-biblioref">144</a>,<a href="#ref-OMzPHtgd" role="doc-biblioref">145</a></sup></span>.
Proteins are generally insoluble in most pure organic solvents, so cold ethanol or methanol are sometimes used.
Pellets should be washed with organic solvent for complete removal of detergents.
Alternatively, solid phase based digestion methods such as S-trap (ProtiFi)<span class="citation" data-cites="Llj72YHx"><sup><a href="#ref-Llj72YHx" role="doc-biblioref">146</a></sup></span>, FASP<span class="citation" data-cites="rWTbDNu2 bkON7ZR2"><sup><a href="#ref-rWTbDNu2" role="doc-biblioref">147</a>,<a href="#ref-bkON7ZR2" role="doc-biblioref">148</a></sup></span>, SP3<span class="citation" data-cites="29YpOcx3 R9HFU5Rw"><sup><a href="#ref-29YpOcx3" role="doc-biblioref">149</a>,<a href="#ref-R9HFU5Rw" role="doc-biblioref">150</a></sup></span> and on column/bead such as protein aggregation capture (PAC)<span class="citation" data-cites="2ZApD7In"><sup><a href="#ref-2ZApD7In" role="doc-biblioref">151</a></sup></span> can allow for proteins to be applied to a solid phase for detergent removal prior to proteolysis<span class="citation" data-cites="IRVtdB7A"><sup><a href="#ref-IRVtdB7A" role="doc-biblioref">152</a></sup></span>.
Specialty detergent removal columns exist (Pierce/Thermo Fisher Scientific) but add expense and time-consuming steps to the process.
SDC can then be easily removed by precipitation or phase separation<span class="citation" data-cites="GcubNF0j"><sup><a href="#ref-GcubNF0j" role="doc-biblioref">153</a></sup></span> following digestion by acidification of the sample to pH 2-3.<br />
Ethyl acetate can also remove several common detergents<span class="citation" data-cites="120jHNa4n"><sup><a href="#ref-120jHNa4n" role="doc-biblioref">154</a></sup></span>.
Any small-molecule removal protocol should be tested for efficiency prior to implementing in a workflow with many samples as avoiding detergent (or polymer) contamination in the LC/MS is very important.</p>
<h4 id="protein-protein-interactions">Protein-protein interactions</h4>
<p>Denaturing conditions will efficiently extract proteins, but will denature proteins and therefore disrupt most protein-protein interactions.
If you are working on an antibody or affinity purification of a specific protein and expect to analyze enzymatic activity, structural features, and/or protein-protein interactions, a non-denaturing lysis buffer should be utilized<span class="citation" data-cites="G9HWYdpG 15gFWnWy1"><sup><a href="#ref-G9HWYdpG" role="doc-biblioref">155</a>,<a href="#ref-15gFWnWy1" role="doc-biblioref">156</a></sup></span>.
Check the calculated isoelectric point (pI) and hydrophobicity (e.g., try the Expasy.org resource ProtParam) for a good idea of starting pH/conductivity, but a stability screen may be needed.
In general, a good starting point for the buffer will still be close to neutral pH with 50-250 mM NaCl, but specific proteins may require pH as low as 2 or as high as 9 for stable extraction.
A low percent of mass spectrometry compatible detergent may also be used, such as n-dodecyl-beta-maltoside.
Newer mass spectrometry-compatible detergents are also useful for protein extraction and ease of downstream processing – including Rapigest® (Waters), N-octyl-β-glucopyranoside, MS-compatible degradable surfactant (MaSDeS)<span class="citation" data-cites="12BtFuhxe"><sup><a href="#ref-12BtFuhxe" role="doc-biblioref">157</a></sup></span>, Azo<span class="citation" data-cites="1FdkzkZPe"><sup><a href="#ref-1FdkzkZPe" role="doc-biblioref">158</a></sup></span>, PPS silent surfactant<span class="citation" data-cites="KbqdsNwY"><sup><a href="#ref-KbqdsNwY" role="doc-biblioref">159</a></sup></span>, sodium laurate<span class="citation" data-cites="1FX8gYsTy"><sup><a href="#ref-1FX8gYsTy" role="doc-biblioref">160</a></sup></span>, and sodium deoxycholate<span class="citation" data-cites="7nXVfPN7"><sup><a href="#ref-7nXVfPN7" role="doc-biblioref">161</a></sup></span>.
Avoid using tween-20, triton-X, NP-40, and polyethylene glycols (PEGs) as these compounds are challenging to remove after digestion<span class="citation" data-cites="EZCh7C0x"><sup><a href="#ref-EZCh7C0x" role="doc-biblioref">162</a></sup></span>.</p>
<h4 id="optional-additives">Optional additives</h4>
<p>There are several additional additives that are often found in protein extraction buffer solutions.
Salts like 50-150 mM sodium chloride (NaCl) may be used to mimic physiological ionic strength.
Protease, phosphatase and deubiquitinase inhibitors are optional additives in less denaturing conditions or in experiments focused on specific PTMs.
For a broad range of inhibitors, a premixed tablet can be added to the lysis buffer, such as Roche cOmplete Mini Protease Inhibitor Cocktail tablets.
Protease inhibitors may impact desired proteolysis from the added protease, and will need to be diluted or removed prior to protease addition.
To improve extraction of DNA- or RNA-binding proteins, adding a small amount of nuclease or benzonase is useful for degradation of any bound nucleic acids and results in a more consistent digestion<span class="citation" data-cites="EK9zCZWn"><sup><a href="#ref-EK9zCZWn" role="doc-biblioref">163</a></sup></span>.
For non-denaturing buffer conditions, which preserve tertiary and quaternary protein structures, additional additives may still be neccessary to prevent proteolysis or PTMs throughout the extraction process.</p>
<h4 id="extraction-of-plant-proteins">Extraction of plant proteins</h4>
<p>Protein extraction from plant tissues generally more challenging due to the presence of cell walls, large vacuoles, and several different classes of interfering substances that are often present in these materials.
Cell walls require vigorous disruption techniques such as grinding with or without an abrasive, use of a bead mill, or homogenizers, which, while they do release the cellular contents, also rupture and mix the contents of organelles and other subcellular compartments.
As a result, isolation of proteins from organelles or other subcellular fractions of plant materials can be fairly specialized<span class="citation" data-cites="1A6wXg5ZL WjXlUEEW TdvEcc7C"><sup><a href="#ref-1A6wXg5ZL" role="doc-biblioref">164</a>–<a href="#ref-TdvEcc7C" role="doc-biblioref">166</a></sup></span>.
Plant tissues have lower protein content compared with tissues from other organisms as only a small fraction of the tissue volume is cytoplasm with the apoplast (cell exterior and wall) and vacuole using much of the tissue volume.
Depending on tissue, cell type and maturity, a plant cell’s vacuole accounts for most of the cell interior space and typically contains substances that degrade or denature proteins upon tissue disruption.
Isolation of functional native proteins from plants usually requires use of plant-targeted protease inhibitors<span class="citation" data-cites="12PeVjdCj iTMONsOP"><sup><a href="#ref-12PeVjdCj" role="doc-biblioref">167</a>,<a href="#ref-iTMONsOP" role="doc-biblioref">168</a></sup></span>, and strategies for preventing protein modification and precipitation by phenolic compounds and their oxidation products<span class="citation" data-cites="iuZ9vx4h"><sup><a href="#ref-iuZ9vx4h" role="doc-biblioref">169</a></sup></span> in addition to the buffers, reductants and other additives discussed previously.</p>
<p>Methodology for whole-tissue protein extraction of plants has been extensively reviewed<span class="citation" data-cites="J2KykVTl 1CzJQ1fro 14eFecfij lDCDjYu"><sup><a href="#ref-J2KykVTl" role="doc-biblioref">170</a>–<a href="#ref-lDCDjYu" role="doc-biblioref">173</a></sup></span>.
These procedures avoid the sample degradation by protease or phenol oxidase activities that can plague native plant protein purification by using extraction at low temperature followed by protein denaturation and removal of contaminating compound classes using precipitation strategies.
Protein is extracted under denaturing conditions and precipitated using combinations of trichloroacetic acid (TCA), ammonium acetate, or acetone (or other solvent) precipitation.
Initial protein extraction and/or resolubilization of protein precipitates is accomplished using detergents, phenol or other chaotropic agents.
Extraction protocols have been shown to influence proteomic results<span class="citation" data-cites="14eFecfij"><sup><a href="#ref-14eFecfij" role="doc-biblioref">172</a></sup></span>, and the compatibility of extracts with subsequent analytical strategies can vary significantly since protocols that were initially developed for 2D-gel electrophoretic analysis often use detergents that can be problematic for peptide LC-MS/MS proteomic approaches.
More recently developed strategies make use of filters (filter-aided sample preparation, FASP,<span class="citation" data-cites="1EeOoOwwB"><sup><a href="#ref-1EeOoOwwB" role="doc-biblioref">174</a></sup></span>) or coated magnetic beads (single-pot-solid-phase-enhanced sample preparation, SP3,<span class="citation" data-cites="1Be3nuQby tqd1QDxn"><sup><a href="#ref-1Be3nuQby" role="doc-biblioref">175</a>,<a href="#ref-tqd1QDxn" role="doc-biblioref">176</a></sup></span>) for higher throughput shotgun proteomic sample preparation.
Strategies for overcoming the dynamic range limitations caused by plant-specific hyperabundant proteins have been developed both for RuBisCO<span class="citation" data-cites="3rOHISrh"><sup><a href="#ref-3rOHISrh" role="doc-biblioref">177</a></sup></span>, which makes up ~50% of the protein in green tissues of C3 plants, and also for seed storage proteins<span class="citation" data-cites="5VlfbJMW"><sup><a href="#ref-5VlfbJMW" role="doc-biblioref">178</a></sup></span>.</p>
<h3 id="mechanical-or-sonic-disruption">Mechanical or Sonic Disruption</h3>
<h4 id="cell-lysis">Cell lysis</h4>
<p>Small mammalian cell pellets and exosomes will lyse almost instantly upon addition denaturing buffer.
If non-denaturing conditions are desired, osmotic swelling and subsequent shearing or sonication can be applied<span class="citation" data-cites="17GoHwva3"><sup><a href="#ref-17GoHwva3" role="doc-biblioref">179</a></sup></span>.
Efficiency of extraction and degradation of nucleic acids can be improved using various sonication methods: 1) probe sonicator with ice; 2) water bath sonicator with ice or cooling; 3) bioruptor® sonication device 4) Adaptive focused acoustics (AFA®)<span class="citation" data-cites="195QiKJpv"><sup><a href="#ref-195QiKJpv" role="doc-biblioref">180</a></sup></span>.
Key to these additional lysis techniques is to keep the temperature of the sample from rising significantly which can cause proteins to aggregate or degrade.
Some cell types may require additional force for effective lysis (see below).
For cells with cell walls (i.e. bacteria or yeast), lysozyme is often added in the lysis buffer.
Any added protein will be present in downstream results, however, so excessive addition of lysozyme is to be avoided unless tagged protein purification will occur.</p>
<h4 id="tissueother-lysis">Tissue/other lysis</h4>
<p>Although small pieces of soft tissue can often be successfully extracted with the probe and sonication methods described above, larger/harder tissues as well as plants/yeast/fungi are better extracted with some form of additional mechanical force.
If proteins are to be extracted from a large amount of sample, such as soil, feces, or other diffuse input, one option is to use a dedicated blender and filter the sample, followed by centrifugation.
If samples are smaller, such as tissue, tumors, etc., cryo-homogenization is recommended.
The simplest form of this is grinding the sample with liquid nitrogen and a mortar and pestle.
Tools such as bead beaters (i.e. FastPrep-24®) are also used, where the sample is placed in a tube with appropriately sized glass or ceramics beads and shaken rapidly.
Cryo-mills are chambers where liquid nitrogen is applied around a vessel and large bead or beads.
Cryo-fractionators homogenize samples in special bags that are frozen in liquid nitrogen and smashed with various degrees of force<span class="citation" data-cites="4EGQHHdM"><sup><a href="#ref-4EGQHHdM" role="doc-biblioref">181</a></sup></span>.
In addition, rapid bead beating mills such as the Bertin Precellys Evolution are both economical, effective and detergent compatible for many types of proteomics experiments at a scale of 96 samples per batch.
Finally, pressure cycling such as the option from pressure biosciences is useful for homogenization of many small tissue pieces<span class="citation" data-cites="157qUOoZe"><sup><a href="#ref-157qUOoZe" role="doc-biblioref">182</a></sup></span>.
After homogenization, samples can be sonicated by one of the methods above to fragment DNA and increase solubilization of proteins.</p>
<h3 id="measuring-the-efficiency-of-protein-extraction">Measuring the efficiency of protein extraction</h3>
<p>Following protein extraction, samples should be centrifuged (10-14,000 g for 10-30 min depending on sample type) to remove debris and insoluble protein prior to determining protein concentration.
Protein quantification is important to assess the yield of an extraction procedure, to match the amount of protein per sample, and to adjust the scale of the downstream processing steps to match the amount of protein.
For example, when purifying peptides, the amount of sorbent should match the amount of material to be bound.
Protein concentration can be calculated using a number of assays or tools<span class="citation" data-cites="wXcRYG3x DRVtai1B"><sup><a href="#ref-wXcRYG3x" role="doc-biblioref">183</a>,<a href="#ref-DRVtai1B" role="doc-biblioref">184</a></sup></span>;
Extraction solution components will need to be compatible with any assay chosen; alternatively, small molecule interferences may be removed (see above) prior to protein concentration calculation.
Each method will have inherent bias and error<span class="citation" data-cites="qFKCAO2X jQLWiiId"><sup><a href="#ref-qFKCAO2X" role="doc-biblioref">185</a>,<a href="#ref-jQLWiiId" role="doc-biblioref">186</a></sup></span>.
These methods can be broadly divided into three types as follows.</p>
<h4 id="colorimetry-based-methods">Colorimetry-based methods:</h4>
<p>The method includes different assays like Coomassie Blue G-250 dye binding (the Bradford assay), the Folin-Lowry assay, the bicinchoninic acid (BCA) assay and the biuret assay<span class="citation" data-cites="1F0WRIEv4"><sup><a href="#ref-1F0WRIEv4" role="doc-biblioref">187</a></sup></span>.
The most commonly used method is the BCA assay.
In the BCA method the peptide bonds of the protein reduce cupric ions [Cu2+] to cuprous ions [Cu+] at a rate which is proportional to the amount of protein present in the sample.
Subsequently, the BCA reagent binds to the cuprous ions, leading to the formation of a complex which absorbs 562 nm wavelength light.
This permits a direct correlation between sample protein concentration and absorbance<span class="citation" data-cites="MgeRCWZg 7aeNLPmC"><sup><a href="#ref-MgeRCWZg" role="doc-biblioref">188</a>,<a href="#ref-7aeNLPmC" role="doc-biblioref">189</a></sup></span>.
The Bradford assay is another method for protein quantification also based on colorimetry principle.
It relies on the interaction between the Coomassie brilliant blue dye and the protein based on hydrophobic and electrostatic interactions. Dye binding shifts the absorption maxima from 470 nm to 595 nm<span class="citation" data-cites="10DIgRvDQ L2cR80nH"><sup><a href="#ref-10DIgRvDQ" role="doc-biblioref">190</a>,<a href="#ref-L2cR80nH" role="doc-biblioref">191</a></sup></span>.
Similarly, the Folin- Lowry method is a two-step colorimetric assay.
Step one is the biuret reaction wherein complexes of copper with the nitrogen in the protein molecule are formed.
In the second step, the complexed tyrosine and tryptophan amino acids react with Folin–Ciocalteu phenol reagent generating an intense, blue-green color absorbing light at 650–750 nm<span class="citation" data-cites="2lZxrpnu"><sup><a href="#ref-2lZxrpnu" role="doc-biblioref">192</a></sup></span>.</p>
<p>Another simple but less reliable protein quantification method of UV-Vis Absorbance at 280 nm estimates the protein concentration by measuring the absorption of the aromatic residues: phenylalanine, tyrosine, and tryptophan<span class="citation" data-cites="q1g1rJrT"><sup><a href="#ref-q1g1rJrT" role="doc-biblioref">193</a></sup></span>.
This is innacurate because different complements of proteins will have different proportions of aromatic amino acids.
This approach is also sensitive to small molecule interferences that may absorb a similar wavelength.</p>
<h4 id="fluorescence-based-methods">Fluorescence-based methods:</h4>
<p>Colorimetric assays are inexpensive and require common lab equipment, but colorimetric detection is less sensitive than fluorescence.
Total protein in proteomic samples can be quantified using intrinsic fluorescence of tryptophan based on the assumption that approximately 1% of all amino acids in the proteome are tryptophan<span class="citation" data-cites="NLQoCcYj"><sup><a href="#ref-NLQoCcYj" role="doc-biblioref">194</a></sup></span>.</p>
<p>NanoOrange (Invitrogen) is an assay for the quantitative measurement of proteins in solution using a merocyanine dye that produces a large increase in fluorescence quantum yield when it interacts with detergent-coated proteins.
Fluorescence is measured using 485-nm excitation and 590-nm emission wavelengths.
The NanoOrange assay can be performed using fluorescence microplate readers, fluorometers, and laser scanners that are standard in the laboratory<span class="citation" data-cites="DRVtai1B"><sup><a href="#ref-DRVtai1B" role="doc-biblioref">184</a></sup></span>.</p>
<p>3-(4-carboxybenzoyl)quinoline-2-carboxaldehyde (CBQCA) is a sensitive fluorogenic reagent for amine detection, which can be used for analyzing proteins in solution.
As the number of accessible amines in a protein is modulated by its concentration, CBQCA has a greater sensitivity and dynamic range when measuring protein concentration<span class="citation" data-cites="iRPPTjzF"><sup><a href="#ref-iRPPTjzF" role="doc-biblioref">195</a></sup></span>.</p>
<h3 id="reduction-and-alkylation">Reduction and alkylation</h3>
<p>Typically, disulfide bonds in proteins are reduced and alkylated prior to proteolysis in order to disrupt structures and simplify peptide analysis.
This allows better access to all residues during proteolysis and removes the crosslinked peptides created by S-S inter peptide linkages.
There are a variety of reagent options for these steps.
For reduction, the typical agents used are 5-15 mM concentration of tris(2-carboxyethyl)phosphine hydrochloride (TCEP-HCl), dithiothreitol (DTT), or 2-beta-mercaptoethanol (2BME).
TCEP-HCl is an efficient reducing agent, but it also significantly lowers sample pH, which can be abated by increasing sample buffer concentration or resuspending TCEP-HCl in an appropriate buffer system (i.e. 1M HEPES pH 7.5).<br />
Following the reducing step, a slightly higher 10-20mM concentration of alkylating agent such as chloroacetamide/iodoacetamide or n-ethyl maleimide is used to cap the free thiols<span class="citation" data-cites="OglKN2v5 2cCnZ6ft 9vICnnYM"><sup><a href="#ref-OglKN2v5" role="doc-biblioref">196</a>–<a href="#ref-9vICnnYM" role="doc-biblioref">198</a></sup></span>.
In order to monitor which cysteine residues are linked or modified in a protein, it is also possible to alkylate free cysteine residues with one reagent, reduce di-sulfide bonds (or other cysteine modifications) and alkylate with a different reagent<span class="citation" data-cites="GkFAexwC Z6nk0TFI s4OLOlUG"><sup><a href="#ref-GkFAexwC" role="doc-biblioref">199</a>–<a href="#ref-s4OLOlUG" role="doc-biblioref">201</a></sup></span>.
Alkylation reactions are generally carried out in the dark at room temperature to avoid excessive off-target alkylation of other amino acids.</p>
<h2 class="page_break_before" id="proteolysis">4. Proteolysis</h2>
<p>Proteolysis is the defining step that differentiates bottom-up or shotgun proteomics from top-down proteomics.
Hydrolysis of proteins is extremely important because it defines the population of potentially identifiable peptides.
Generally, peptides between a length of 7-35 amino acids are considered useful for mass spectrometry analysis.
Peptides that are too long are difficult to identify by tandem mass spectrometry or may be lost during sample preparation due to irreversible binding with solid-phase extraction sorbents.
Peptides that are too short are also not useful because they may match to many proteins during protein inference.
There are many choices of enzymes and chemicals that hydrolyze proteins into peptides.
This section summarizes potential choices and their strengths and weaknesses.</p>
<p>Before we get into details of various choices for proteolysis, we must discuss terminology.
While it’s true that “digestion” is commonly used in proteomics, it’s important to note that “hydrolysis” is a more specific word choice to describe the chemical process because it refers to breaking peptide bonds within proteins using water.
Although hydrolysis may be associated with the complete chemical hydrolysis of proteins into amino acids, for example using high temperature and acid, hydrolysis reactions catalyzed by enzymes such as pepsin and trypsin are specific for certain amino acid residues.
In fact, all methods of protein cleavage to shorter peptides require a water molecule for their mechanism of action.
In contrast, the definition of “digestion” relates to food breakdown into subunits usable by the body or any chemical process that breaks down substances.
Therefore, while “digestion” is indeed a widely used term for the conversion of the proteome to peptides, “hydrolysis” more accurately describes the specific biochemical process that occurs.
We believe that this terminology choice enhances clarity and precision in scientific communication within the field of proteomics.</p>
<p>Trypsin is the most common choice of protease for proteome hydrolysis<span class="citation" data-cites="vpBg0CP7"><sup><a href="#ref-vpBg0CP7" role="doc-biblioref">202</a></sup></span>.
Trypsin is favorable because of its specificity, availability, efficiency and low cost.
Trypsin is a sufficient choice for most proteomics experiments.
Trypsin cleaves at the C-terminus of basic amino acids, Arg and Lys, if not immediately followed by proline (although there is debate whether a small number of R/K-P sites are actually cleaved).
Many of the peptides generated from trypsin are appropriate in length and hydrophobicity for chromatographic separation, MS-based peptide fragmentation and identification by database search.
The main drawback of trypsin is that majority (56%) of the tryptic peptides are ≤ 6 amino acids, and hence using trypsin alone limits the observable proteome<span class="citation" data-cites="16FpKYZCe lSFMVWD1 10vHwy5gN"><sup><a href="#ref-16FpKYZCe" role="doc-biblioref">203</a>–<a href="#ref-10vHwy5gN" role="doc-biblioref">205</a></sup></span>.
This limits the number of identifiable protein isoforms and post-translational modifications.</p>
<p>Although trypsin is the most common protease used for proteomics, in theory it can only cover a fraction of the proteome predicted from the genome<span class="citation" data-cites="y9KaXde4"><sup><a href="#ref-y9KaXde4" role="doc-biblioref">206</a></sup></span>.
This is due to production of peptides that are too short to be unique, for example due to R and K immediately next to each other.
Peptides below a certain length are likely to occur many times in the whole proteome, meaning that even if we identify them we cannot know their protein of origin.
In protein regions devoid of R/K, trypsin may also result in very long peptides that are then lost due to irreversible binding to the solid phase extraction device, or that become difficult to identify due to complicated fragmentation patterns.
Thus, parts of the true proteome sequences that are present are lost after trypsin digestion due to both production of very long and very short peptides.</p>
<p>Many alternative proteases are available with different specificities that complement trypsin to reveal different protein sequences<span class="citation" data-cites="7aKTZT1i 16FpKYZCe"><sup><a href="#ref-16FpKYZCe" role="doc-biblioref">203</a>,<a href="#ref-7aKTZT1i" role="doc-biblioref">207</a></sup></span>, which can help distinguish protein isoforms<span class="citation" data-cites="gXhdeFOF"><sup><a href="#ref-gXhdeFOF" role="doc-biblioref">208</a></sup></span> (<strong>Figure 2, Table 3</strong>).
The enzyme choice mostly depends on the application.
In general, for a mere protein identification, trypsin is often chosen due to the aforementioned reasons.
However, alternative enzymes can facilitate <em>de novo</em> assembly when the genomic data information is limited in the public database repositories<span class="citation" data-cites="14bc70ELO zmo81rNN IZw9sVjg 19KCAADV7 7BuX2TBR"><sup><a href="#ref-14bc70ELO" role="doc-biblioref">209</a>–<a href="#ref-7BuX2TBR" role="doc-biblioref">213</a></sup></span>.
Use of multiple proteases for proteome digestion also can improve the sensitivity and accuracy of protein quantification<span class="citation" data-cites="1GLf6FYas"><sup><a href="#ref-1GLf6FYas" role="doc-biblioref">214</a></sup></span>.
Moreover, by providing an increased peptide diversity, the use of multiple proteases can expand sequence coverage and increase the probability of finding peptides which are unique to single proteins<span class="citation" data-cites="WubQ7yFP yeQvImXS y9KaXde4"><sup><a href="#ref-yeQvImXS" role="doc-biblioref">64</a>,<a href="#ref-y9KaXde4" role="doc-biblioref">206</a>,<a href="#ref-WubQ7yFP" role="doc-biblioref">215</a></sup></span>.
A multi-protease approach can also improve the identification of N-Termini and signal peptides for small proteins<span class="citation" data-cites="Ac0S3PrF"><sup><a href="#ref-Ac0S3PrF" role="doc-biblioref">216</a></sup></span>.
Overall, integrating multiple-protease data can increase the number of proteins identified<span class="citation" data-cites="P861T4mW 14eCb7Jfc"><sup><a href="#ref-P861T4mW" role="doc-biblioref">217</a>,<a href="#ref-14eCb7Jfc" role="doc-biblioref">218</a></sup></span>, increase the identified post-translational modifications<span class="citation" data-cites="WubQ7yFP 18s9UXUGf yeQvImXS"><sup><a href="#ref-yeQvImXS" role="doc-biblioref">64</a>,<a href="#ref-WubQ7yFP" role="doc-biblioref">215</a>,<a href="#ref-18s9UXUGf" role="doc-biblioref">219</a></sup></span> and decrease the ambiguity of the inferred protein groups<span class="citation" data-cites="WubQ7yFP"><sup><a href="#ref-WubQ7yFP" role="doc-biblioref">215</a></sup></span>.</p>
<p>There are, however, many challenges associated with using alternative proteases.
Since peptides are not cleaved after a positively charged residue (like the R/K targeted by trypsin), they may only obtain one precursor charge and be ineffectively fragmented.
The lack of a c-terminal positive charge will lead to less consistent y-ion series.
Other peptides may obtain too many charges and produce highly charged fragments that are not scored well by search engines.
Another common issue with alternative proteases is the potential for producing “shredded” peptides where multiple peptides differ only by a few residues at either end, thus decreasing the quantity of each species and limiting sensitivity.
This problem is worse with proteases that target uncharged residues, because ionic interactions are much stronger than dispersion forces used for binding aliphatic residues.</p>
<div id="fig:Multiple-protease-proteolysis" class="fignos">
<figure>
<img src="images/Proteolysis_v03.png" style="width:100.0%" data-tag="2" alt="Figure 2: Multiple protease proteolysis improves protein inference The use of other proteases beyond trypsin such as lysyl endopeptidase (Lys-C), peptidyl-Asp metallopeptidase (Asp-N), glutamyl peptidase I, (Glu-C), chymotrypsin, clostripain (Arg-C) or peptidyl-Lys metalloendopeptidase (Lys-N) can generate a greater diversity of peptides. This improves protein sequence coverage and allows for the correct identification of their N-termini. Increasing the number of complimentary enzymes used will increase the number of proteins identified by single peptides and decreases the ambiguity of the assignment of protein groups. Therefore, this will allow more protein isoforms and post-translational modifications to be identified than using Trypsin alone." />
<figcaption aria-hidden="true"><span>Figure 2:</span> <strong>Multiple protease proteolysis improves protein inference</strong>
The use of other proteases beyond trypsin such as lysyl endopeptidase (Lys-C), peptidyl-Asp metallopeptidase (Asp-N), glutamyl peptidase I, (Glu-C), chymotrypsin, clostripain (Arg-C) or peptidyl-Lys metalloendopeptidase (Lys-N) can generate a greater diversity of peptides.
This improves protein sequence coverage and allows for the correct identification of their N-termini.
Increasing the number of complimentary enzymes used will increase the number of proteins identified by single peptides and decreases the ambiguity of the assignment of protein groups.
Therefore, this will allow more protein isoforms and post-translational modifications to be identified than using Trypsin alone.</figcaption>
</figure>
</div>
<p>Table 3: Common proteases used for proteomics.</p>
<table style="width:100%;">
<colgroup>
<col style="width: 14%" />
<col style="width: 11%" />
<col style="width: 31%" />
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Protease</th>
<th style="text-align: center;">source</th>
<th style="text-align: center;">class</th>
<th style="text-align: center;">specificity</th>
<th style="text-align: center;">optimal pH</th>
<th style="text-align: center;">notes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Trypsin</td>
<td style="text-align: center;">mammal pancreas</td>
<td style="text-align: center;">serine protease</td>
<td style="text-align: center;">c-term of R/K, not before P</td>
<td style="text-align: center;">7-9</td>
<td style="text-align: center;">most common protease</td>
</tr>
<tr class="even">
<td style="text-align: center;">LysC</td>
<td style="text-align: center;"><em>Lysobacter enzymogenesis</em></td>
<td style="text-align: center;">serine protease</td>
<td style="text-align: center;">c-term of K</td>
<td style="text-align: center;">7-9</td>
<td style="text-align: center;">high stability</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Alpha-lytic protease</td>
<td style="text-align: center;"><em>Lysobacter enzymogenesis</em></td>
<td style="text-align: center;">serine protease</td>
<td style="text-align: center;">c-term of small side chains</td>
<td style="text-align: center;">7-9</td>
<td style="text-align: center;">high stability</td>
</tr>
<tr class="even">
<td style="text-align: center;">GluC</td>
<td style="text-align: center;"><em>Staphyloccous aureus</em></td>
<td style="text-align: center;">serine protease</td>
<td style="text-align: center;">c-term of D/E</td>
<td style="text-align: center;">4-8</td>
<td style="text-align: center;">specificity for Glu depends on buffer</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Asp-N</td>
<td style="text-align: center;"><em>Pseudomonas fragi</em></td>
<td style="text-align: center;">metalloprotease</td>
<td style="text-align: center;">n-term of D</td>
<td style="text-align: center;">4-9</td>
<td style="text-align: center;">avoid chelators</td>
</tr>
<tr class="even">
<td style="text-align: center;">chymotrypsin</td>
<td style="text-align: center;">mammal pancreas</td>
<td style="text-align: center;">serine protease</td>
<td style="text-align: center;">c-term of larger hydroponics</td>
<td style="text-align: center;">7-9</td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Arg-C</td>
<td style="text-align: center;"><em>Clostridium histolyticum</em></td>
<td style="text-align: center;">cysteine protease</td>
<td style="text-align: center;">c-term or R</td>
<td style="text-align: center;">7.2-7.8</td>
<td style="text-align: center;">avoid oxidation</td>
</tr>
<tr class="even">
<td style="text-align: center;">Ulilysin</td>
<td style="text-align: center;"><em>Methanosarcina acetivorans</em></td>
<td style="text-align: center;">metalloprotease</td>
<td style="text-align: center;">N-term of R/K</td>
<td style="text-align: center;">6-9</td>
<td style="text-align: center;">stable to 55℃</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Lys-N</td>
<td style="text-align: center;"><em>Grifola frondosa</em></td>
<td style="text-align: center;">metalloprotease</td>
<td style="text-align: center;">N-term or K</td>
<td style="text-align: center;">7-9</td>
<td style="text-align: center;">stable to 70℃</td>
</tr>
<tr class="even">
<td style="text-align: center;">Pepsin A</td>
<td style="text-align: center;">mammal pancreas</td>
<td style="text-align: center;">aspartic acid protease</td>
<td style="text-align: center;">broad including W, F, Y, L</td>
<td style="text-align: center;">1-4</td>
<td style="text-align: center;">common for HDX</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Proteinase K</td>
<td style="text-align: center;"><em>Tritirachium album</em></td>
<td style="text-align: center;">serine protease</td>
<td style="text-align: center;">broadest</td>
<td style="text-align: center;">4-12</td>
<td style="text-align: center;">common for limited proteolysis</td>
</tr>
</tbody>
</table>
<p>Lysyl endopeptidase (Lys-C) obtained from <em>Lysobacter enzymogenesis</em> is a serine protease involved in cleaving carboxyl terminus of Lys<span class="citation" data-cites="ZgqlEERS lSFMVWD1"><sup><a href="#ref-lSFMVWD1" role="doc-biblioref">204</a>,<a href="#ref-ZgqlEERS" role="doc-biblioref">220</a></sup></span>.
Like trypsin, the optimum pH range required for its activity is from 7 to 9.
A major advantage of Lys-C is its resistance to denaturing agents, including 8 M urea - a chaotrope commonly used to denature proteins <em>prior</em> to digestion<span class="citation" data-cites="gXhdeFOF"><sup><a href="#ref-gXhdeFOF" role="doc-biblioref">208</a></sup></span>.
Trypsin is less efficient at cleaving Lys than Arg, which could limit the quality of quantitation from tryptic peptides.
Hence, to achieve complete protein digestion with minimal missed cleavages, Lys-C is often used simultaneously with trypsin digestion<span class="citation" data-cites="nobcST81"><sup><a href="#ref-nobcST81" role="doc-biblioref">221</a></sup></span>.</p>
<p>Alpha-lytic protease (aLP) is another serine protease secreted by the soil bacterial <em>Lysobacter enzymogenesis</em><span class="citation" data-cites="oDNy6K9V"><sup><a href="#ref-oDNy6K9V" role="doc-biblioref">222</a></sup></span>.
Wild-type aLP (WaLP) and an active site mutant of aLP, M190A (MaLP), have been used to expand proteome coverage<span class="citation" data-cites="yeQvImXS"><sup><a href="#ref-yeQvImXS" role="doc-biblioref">64</a></sup></span>.
Based on observed peptide sequences from yeast proteome digestion, WaLP showed a specificity for small aliphatic amino acids like alanine, valine, and glycine, but also threonine and serine.
MaLP showed specificity for slightly larger amino acids like methionine, phenylalanine, and surprisingly, a preference for leucine over isoleucine.
The specificity of WaLP for threonine enabled the first method for mapping endogenous human SUMO sites<span class="citation" data-cites="4iAess79"><sup><a href="#ref-4iAess79" role="doc-biblioref">39</a></sup></span>.</p>
<p>Glutamyl peptidase I, commonly known as Glu-C or V8 protease, is a serine protease obtained from <em>Staphyloccous aureus</em><span class="citation" data-cites="UeP5oN6q"><sup><a href="#ref-UeP5oN6q" role="doc-biblioref">223</a></sup></span>.
Glu-C cleaves at the C-terminus of glutamate, but also after aspartate<span class="citation" data-cites="UeP5oN6q FcovvAwu"><sup><a href="#ref-UeP5oN6q" role="doc-biblioref">223</a>,<a href="#ref-FcovvAwu" role="doc-biblioref">224</a></sup></span>.</p>
<p>Peptidyl-Asp metallopeptidase, commonly known as Asp-N, is a metalloprotease obtained from <em>Pseudomonas fragi</em><span class="citation" data-cites="Z8khRFem"><sup><a href="#ref-Z8khRFem" role="doc-biblioref">225</a></sup></span>.
Asp-N catalyzes the hydrolysis of peptide bonds at the N-terminal of aspartate residues.
The optimum activity of this enzyme occurs at a pH range between 4 and 9.
As with any metalloprotease, chelators like EDTA should be avoided for digestion buffers when using Asp-N.
Studies also suggest that Asp-N cleaves at the amino terminus of glutamate when a detergent is present in the proteolysis buffer<span class="citation" data-cites="Z8khRFem"><sup><a href="#ref-Z8khRFem" role="doc-biblioref">225</a></sup></span>.
Asp-N often leaves many missed cleavages<span class="citation" data-cites="gXhdeFOF"><sup><a href="#ref-gXhdeFOF" role="doc-biblioref">208</a></sup></span>.</p>
<p>Chymotrypsin or chymotrypsinogen A is a serine protease obtained from porcine or bovine pancreas with an optimum pH range from 7-9<span class="citation" data-cites="129jHkJ7J"><sup><a href="#ref-129jHkJ7J" role="doc-biblioref">226</a></sup></span>.
It cleaves at the C-terminus of hydrophobic amino acids Phe, Trp, Tyr and barely Met and Leu residues.
Since the transmembrane region of membrane proteins commonly lacks tryptic cleavage sites, this enzyme works well with membrane proteins having more hydrophobic residues<span class="citation" data-cites="n8tCUNeh Ddlp2JQj gXhdeFOF"><sup><a href="#ref-gXhdeFOF" role="doc-biblioref">208</a>,<a href="#ref-n8tCUNeh" role="doc-biblioref">227</a>,<a href="#ref-Ddlp2JQj" role="doc-biblioref">228</a></sup></span>.
The chymotryptic peptides generated after proteolysis will cover the proteome space orthogonal to that of tryptic peptides both in a quantitative and qualitative manner<span class="citation" data-cites="VbXzZ784 1BDWCCTHa Ddlp2JQj"><sup><a href="#ref-Ddlp2JQj" role="doc-biblioref">228</a>–<a href="#ref-1BDWCCTHa" role="doc-biblioref">230</a></sup></span></p>
<p>Clostripain, commonly known as Arg-C, is a cysteine protease obtained from <em>Clostridium histolyticum</em><span class="citation" data-cites="qJADzs4"><sup><a href="#ref-qJADzs4" role="doc-biblioref">231</a></sup></span>.
It hydrolyses mostly the C-terminal Arg residues and sometimes Lys residues, but with less efficiency.
The peptides generated are generally longer than that of tryptic peptides.
Arg-C is often used with other proteases for improving qualitative proteome data and also for investigating PTMs<span class="citation" data-cites="lSFMVWD1"><sup><a href="#ref-lSFMVWD1" role="doc-biblioref">204</a></sup></span>.</p>
<p>LysargiNase, also known as Ulilysin, is a recently discovered protease belonging to the metalloprotease family.
It is a thermophilic protease derived from <em>Methanosarcina acetivorans</em> that specifically cleaves at the N-terminus of Lys and Arg residues<span class="citation" data-cites="MqKkSQNK"><sup><a href="#ref-MqKkSQNK" role="doc-biblioref">232</a></sup></span>.
Hence, it enabled discovery of C-terminal peptides that were not observed using trypsin.
In addition, it can also cleave modified amino acids such as methylated or dimethylated Arg and Lys<span class="citation" data-cites="MqKkSQNK"><sup><a href="#ref-MqKkSQNK" role="doc-biblioref">232</a></sup></span>.</p>
<p>Peptidyl-Lys metalloendopeptidase, or Lys-N, is an metalloprotease obtained from <em>Grifola frondosa</em><span class="citation" data-cites="3xdcLBaR"><sup><a href="#ref-3xdcLBaR" role="doc-biblioref">233</a></sup></span>.
It cleaves N-terminally of Lys and has an optimal activity at pH 9.0.
Unlike trypsin, Lys-N is more resistant to denaturing agents and can be heated up to 70°C<span class="citation" data-cites="lSFMVWD1"><sup><a href="#ref-lSFMVWD1" role="doc-biblioref">204</a></sup></span>.
Peptides generated from Lys-N digestion produce more c-type ions using ETD fragmentation<span class="citation" data-cites="O6S1GvPn"><sup><a href="#ref-O6S1GvPn" role="doc-biblioref">234</a></sup></span>.
Hence this can be used for analysing PTMs, identification of C-terminal peptides and also for <em>de novo</em> sequencing strategies<span class="citation" data-cites="O6S1GvPn 8DGqkN3D"><sup><a href="#ref-O6S1GvPn" role="doc-biblioref">234</a>,<a href="#ref-8DGqkN3D" role="doc-biblioref">235</a></sup></span>.</p>
<p>Pepsin A, commonly known as pepsin, is an aspartic protease obtained from bovine or porcine pancreas<span class="citation" data-cites="eMC9mqfj"><sup><a href="#ref-eMC9mqfj" role="doc-biblioref">236</a></sup></span>.
Pepsin was one of several proteins crystalized by John Northrop, who shared the 1946 Nobel prize in chemistry for this work<span class="citation" data-cites="pnU7apE 9Yf3fEMC 10CHY6r2E 1EtaxGZW0"><sup><a href="#ref-pnU7apE" role="doc-biblioref">237</a>–<a href="#ref-1EtaxGZW0" role="doc-biblioref">240</a></sup></span>.
Pepsin works at an optimum pH range from 1 to 4 and specifically cleaves Trp, Phe, Tyr and Leu<span class="citation" data-cites="lSFMVWD1"><sup><a href="#ref-lSFMVWD1" role="doc-biblioref">204</a></sup></span>.
Since it possess high enzyme activity and broad specificity at lower pH, it is preferred over other proteases for MS-based disulphide mapping<span class="citation" data-cites="itW7kK3s Vske1oHJ"><sup><a href="#ref-itW7kK3s" role="doc-biblioref">241</a>,<a href="#ref-Vske1oHJ" role="doc-biblioref">242</a></sup></span>.
Pepsin is also used extensively for structural mass spectrometry studies with hydrogen-deuterium exchange (HDX) because the rate of back exchange of the amide deuteron is minimized at low pH<span class="citation" data-cites="Azi1X1R4 1nMCmYmb"><sup><a href="#ref-Azi1X1R4" role="doc-biblioref">243</a>,<a href="#ref-1nMCmYmb" role="doc-biblioref">244</a></sup></span>.</p>
<p>Proteinase K was first isolated from the mold <em>Tritirachium album</em> Limber<span class="citation" data-cites="YOXX6An6"><sup><a href="#ref-YOXX6An6" role="doc-biblioref">245</a></sup></span>.
The epithet ‘K’ is derived from its ability to efficiently hydrolyze keratin<span class="citation" data-cites="YOXX6An6"><sup><a href="#ref-YOXX6An6" role="doc-biblioref">245</a></sup></span>.
It is a member of the subtilisin family of proteases and is relatively unspecific with a preference for proteolysis at hydrophobic and aromatic amino acid residues<span class="citation" data-cites="1CgjQBy2S"><sup><a href="#ref-1CgjQBy2S" role="doc-biblioref">246</a></sup></span>.
The optimal enzyme activity is between pH 7.5 and 12.
Proteinase K is used at low concentrations for limited proteolysis (LiP) and the detection of protein structural changes in the eponymous technique LiP-MS<span class="citation" data-cites="1BhNuatKH"><sup><a href="#ref-1BhNuatKH" role="doc-biblioref">247</a></sup></span>.</p>
<h3 id="peptide-quantitation-assays">Peptide quantitation assays</h3>
<p>After peptide production from proteomes, it may be desirable to quantify the peptide yield.
Quantitation of peptide assays is not as easy as protein lysate assays.
BCA protein assays perform poorly with peptide solutions and report erroneous values.
A simplistic measurement is to use a nanodrop device, but absorbance measurements from a drop of solution does not report accurate values either.
Especially given that low amounts of peptides are often produced for proteomics, more sensitive methods based on fluorescence are prefered.
One reliable approach is to Fluorescamine based assay for peptide solutions for higher accuracy<span class="citation" data-cites="Ixo1jeWd dBvmYbi"><sup><a href="#ref-Ixo1jeWd" role="doc-biblioref">248</a>,<a href="#ref-dBvmYbi" role="doc-biblioref">249</a></sup></span>.
This assay is based on the reaction between a labeling reagent and the N-terminal primary amine in the peptide(s); therefore, samples must be free of amine-containing buffers (e.g., Tris-based buffer and/or amino acids).
This procedure has performance similar to the Pierce Quantitative Fluorometric Peptide Assay (Cat 23290).
A second option is also easy to use tryptophan fluorescence to quantify peptide yields<span class="citation" data-cites="7YHVbaDH"><sup><a href="#ref-7YHVbaDH" role="doc-biblioref">250</a></sup></span>, which is useful because it does not consume the sample because it uses intrinsic fluorescence.</p>
<h2 class="page_break_before" id="peptide-quantification">5. Peptide Quantification</h2>
<h3 id="label-free-quantification-lfq-of-peptides">Label-free quantification (LFQ) of peptides</h3>
<p>LFQ of peptide precursors requires no additional steps in the protein extraction, digestion, and peptide purification workflow (<strong>Figure 3</strong>).
Samples can be taken straight to the mass spectrometer and are injected one at a time, each sample necessitating their own LC-MS/MS experiment and raw file.
Quantification of peptides by LFQ is routinely performed by many commercial and freely available proteomics software (see Data Analysis section below).
In LFQ, peptide abundances across LC-MS/MS experiments are usually calculated by computing the area under the extracted ion chromatograms for signals that are specific to each peptide; this involves aligning windows of accurate peptide mass and retention time.
LFQ can be performed using precursor MS1 signals from DDA, or using multiple fragment ion signals from DIA (see Data Acquisition section).
It is important to note that due to differences in peptide ionization efficiency, LFQ only provides relative quantification, not absolute quantities.</p>
<div id="fig:quant-summary" class="fignos">
<figure>
<img src="images/Summay_peptide-protein-labeling_.svg" style="width:100.0%" data-tag="3" alt="Figure 3: Quantitative strategies commonly used in proteomics. A few non-comprehensive examples are of quantification methods are shown. A) Label-free quantification. Proteins are extracted from samples, enzymatically hydrolyzed into peptides and analyzed by mass spectrometry. Extracted ion chromatograms from peptides are compared across samples that are analyzed sequentially. B) Metabolic labelling. Stable isotope labeling by amino acids in cell culture (SILAC) is based on feeding cells stable isotope labeled amino acids (“light” or “heavy”). Samples grown with heavy or light amino acids are mixed before cell lysis. The relative intensities of the heavy and light peptide are used to compute protein differences between samples. C) Isobaric or chemical labelling. Proteins are isolated separately from samples, enzymatically hydrolyzed into peptides, and then chemically tagged with isobaric stable isotope labels. These isobaric tags produce unique reporter mass-to-charge (m/z) signals that are produced upon fragmentation with MS/MS. Peptide fragment ions are used to identify peptides, and the relative reporter ion signals are used for quantification." />
<figcaption aria-hidden="true"><span>Figure 3:</span> <strong>Quantitative strategies commonly used in proteomics.</strong>
A few non-comprehensive examples are of quantification methods are shown. A) Label-free quantification.
Proteins are extracted from samples, enzymatically hydrolyzed into peptides and analyzed by mass spectrometry.
Extracted ion chromatograms from peptides are compared across samples that are analyzed sequentially.
B) Metabolic labelling.
Stable isotope labeling by amino acids in cell culture (SILAC) is based on feeding cells stable isotope labeled amino acids (“light” or “heavy”).
Samples grown with heavy or light amino acids are mixed before cell lysis.
The relative intensities of the heavy and light peptide are used to compute protein differences between samples.
C) Isobaric or chemical labelling.
Proteins are isolated separately from samples, enzymatically hydrolyzed into peptides, and then chemically tagged with isobaric stable isotope labels.
These isobaric tags produce unique reporter mass-to-charge (<em>m/z</em>) signals that are produced upon fragmentation with MS/MS.
Peptide fragment ions are used to identify peptides, and the relative reporter ion signals are used for quantification.</figcaption>
</figure>
</div>
<h3 id="stable-isotope-labeling-of-peptides">Stable isotope labeling of peptides</h3>
<p>One approach to improve the throughput and quantitative completeness within a group of samples is sample multiplexing via stable isotope labeling.
Multiplexing enables pooling of samples and parallel LC-MS/MS analysis within one run.
Quantification can be achieved at the MS1- or MSn-level, dictated by the upstream labeling strategy.</p>
<p>Stable isotope labeling methods produce peptides that are chemically identical from each sample that differ only in their mass.
Methods include stable isotope labeling by amino acids in cell culture (SILAC)<span class="citation" data-cites="YTQzoOxb"><sup><a href="#ref-YTQzoOxb" role="doc-biblioref">251</a></sup></span> and chemical labeling such as amine-modifying tags for relative and absolute quantification (mTRAQ)<span class="citation" data-cites="1FOQ1H9mS"><sup><a href="#ref-1FOQ1H9mS" role="doc-biblioref">252</a></sup></span> or dimethyl labeling<span class="citation" data-cites="Qqbj9JVK"><sup><a href="#ref-Qqbj9JVK" role="doc-biblioref">253</a></sup></span>.
The latter two methods are chemical labeling processes after proteome or peptide purification.
In all these aproaches, the labeling of each sample imparts mass shifts (e.g. 4 Da, 8 Da) which can be detected within the MS1 full scan.
The ability to label samples in cell culture has enabled impactful quantitative biology experiments<span class="citation" data-cites="E7KmdZas TGkcmyuH"><sup><a href="#ref-E7KmdZas" role="doc-biblioref">254</a>,<a href="#ref-TGkcmyuH" role="doc-biblioref">255</a></sup></span>.
These approaches have nearly exclusively been performed using data-dependent acquisition (DDA) strategies.
However, recent work employing faster instrumentation has shown the benefits of chemical labeling with 3-plex mTRAQ or dimethyl labels for data-independent acquisition (DIA)<span class="citation" data-cites="QipLkzBy ZhSU2auj"><sup><a href="#ref-QipLkzBy" role="doc-biblioref">256</a>,<a href="#ref-ZhSU2auj" role="doc-biblioref">257</a></sup></span>, an idea originally developed nearly a decade earlier using chemical labels to quantify lysine acetylation and succinylation stoichiometry<span class="citation" data-cites="WPmaUWBS"><sup><a href="#ref-WPmaUWBS" role="doc-biblioref">258</a></sup></span>.
As new tags with higher plexing become available, strategies like plexDIA and mDIA are sure to benefit<span class="citation" data-cites="QipLkzBy ZhSU2auj"><sup><a href="#ref-QipLkzBy" role="doc-biblioref">256</a>,<a href="#ref-ZhSU2auj" role="doc-biblioref">257</a></sup></span>.</p>
<h3 id="peptide-labeling-with-isobaric-tags">Peptide labeling with isobaric tags</h3>
<p>Another approach is multiplexing via isobaric labels, a strategy which enables parallel data acquisition after pooling of samples.
Commercial isobaric tags include tandem mass tags (TMT)<span class="citation" data-cites="j9N2kPSr"><sup><a href="#ref-j9N2kPSr" role="doc-biblioref">259</a></sup></span> and isobaric tags for relative and absolute quantification (iTRAQ)<span class="citation" data-cites="V8IykxaB"><sup><a href="#ref-V8IykxaB" role="doc-biblioref">260</a></sup></span> amongst others, and several non-commercial options have also been developed<span class="citation" data-cites="lbOXqNSw"><sup><a href="#ref-lbOXqNSw" role="doc-biblioref">261</a></sup></span>.
Although isobaric tags enable collection of data from many samples at once, to improve depth, fractionation by high pH reversed phase is often used, which limits the benefit in throughput.</p>
<p>The isobaric tag labeling-based peptide quantitation strategy uses derivatization of every peptide sample with a different isotopic incorporation from a set of isobaric mass tags.
All isobaric tags have a common structural theme consisting of 1) an amine-reactive groups (usually triazine ester or N-hydroxysuccinimide [NHS] esters) which react with peptide N-termini and ε-amino group of the lysine side chain of peptides, 2) a balancer group, and 3) a reporter ion group (<strong>Figure 4</strong>).</p>
<p>Peptide labeling is followed by pooling the labelled samples, which undergo MS and MS/MS analysis.
Often the pooled samples are fractionated into multiple samples to increase the depth.
As the isobaric tags are used, peptides labeled with these tags give a single MS peak with the same precursor <em>m/z</em> value in an MS1 scan and identical retention time of liquid chromatography analysis.
The modified parent ions undergo fragmentation during MS/MS analysis generating two kinds of fragment ions: (a) reporter ions and (b) peptide fragment ions.
Each reporter ions’ relative intensity is directly proportional to the peptide abundance in each of the starting samples that were pooled.
As usual, b- and y-type fragment ion peaks are still used to identify amino acid sequences of peptides, from which proteins can be inferred.
Since it is possible to label most tryptic peptides with an isobaric mass tag at least at the N-termini, numerous peptides from the same protein can be detected and quantified, thus leading to an increase in the confidence in both protein identification and quantification<span class="citation" data-cites="15rS5UMb9"><sup><a href="#ref-16D3xg72c" role="doc-biblioref">263</a></sup></span>.</p>
<p>Because the size of the reporter ions is small and sometimes the mass difference between reporter ions is small (i.e., a ~6 mDa difference when using 13C versus 15N), these methods almost exclusively employ high-resolution mass analyzers, not classical ion traps<span class="citation" data-cites="kIpNiFeU"><sup><a href="#ref-kIpNiFeU" role="doc-biblioref">264</a></sup></span>.
There are examples, however, of using isobaric tags with pulsed q dissociation on linear ion traps (LTQs)<span class="citation" data-cites="ESY1qR9T"><sup><a href="#ref-ESY1qR9T" role="doc-biblioref">265</a></sup></span>.
Suitable instruments are the Thermo Q-Exactive, Exploris, Tribrid, and Astral lines, or Q-TOFs such as the TripleTOF or timsTOF platforms<span class="citation" data-cites="FMSpJgZ4 OlzbV8x4"><sup><a href="#ref-FMSpJgZ4" role="doc-biblioref">266</a>,<a href="#ref-OlzbV8x4" role="doc-biblioref">267</a></sup></span>.</p>
<div id="fig:DIA-types" class="fignos">
<figure>
<img src="images/TMT_Figure.png" style="width:100.0%" data-tag="4" alt="Figure 4: Chemical structure of isobaric tags. This shows the TMT 6-plex from ThermoFisher, which is an example of an isobaric tag. The structure has three elements, the reactive group (in this case N-hydroy-succinimide), the balancer, and the reporter. The reactive group enables quick covalent conjugation to nucleophic amines found at peptide n-termimi and lysine side chains. The balance and reporter groups together contain a total of six heavy isotopes. The stars in the structures indicate the positions of all six heavy atoms for each TMT form. For this reason, a sample labeled by the any version will have the same precursor mass. However, upon fragmentation, the balance group is lost and the reporter retains a charge. The reporter group is measured in the low mass region and is proportional to the starting amount of each sample before mixing This ratio of reporter signals enables quantification." />
<figcaption aria-hidden="true"><span>Figure 4:</span> <strong>Chemical structure of isobaric tags.</strong>
This shows the TMT 6-plex from ThermoFisher, which is an example of an isobaric tag.
The structure has three elements, the reactive group (in this case N-hydroy-succinimide), the balancer, and the reporter.
The reactive group enables quick covalent conjugation to nucleophic amines found at peptide n-termimi and lysine side chains.
The balance and reporter groups together contain a total of six heavy isotopes.
The stars in the structures indicate the positions of all six heavy atoms for each TMT form.
For this reason, a sample labeled by the any version will have the same precursor mass.
However, upon fragmentation, the balance group is lost and the reporter retains a charge.
The reporter group is measured in the low mass region and is proportional to the starting amount of each sample before mixing
This ratio of reporter signals enables quantification.</figcaption>
</figure>
</div>
<p>The following are some of the isobaric labeling techniques:</p>
<h4 id="isobaric-tags-for-relative-and-absolute-quantitation-itraq">isobaric Tags for Relative and Absolute Quantitation (iTRAQ)</h4>
<p>The iTRAQ tagging method covalently labels the peptide N-terminus and side-chain primary amines with tags of different masses through the NHS-ester bond.
This was the first isobaric tagging method to find widespread use, but it isn’t used as much anymore.
This is followed by mass spectrometry analysis<span class="citation" data-cites="JokZNPi7"><sup><a href="#ref-JokZNPi7" role="doc-biblioref">268</a></sup></span>.
Reporter ions for an 8-plex iTRAQ are measured at nominally 113, 114, 115, 116, 117, 118, 119, and 121 m/z.
Currently, two kinds of iTRAQ reagents are available: 4-plex and 8-plex<span class="citation" data-cites="ZPmx6w38"><sup><a href="#ref-ZPmx6w38" role="doc-biblioref">269</a></sup></span>.
Using 4-plex reagents, a maximum of four different biological conditions can be analyzed simultaneously (i.e., multiplexed), whereas using 8-plex reagents enables the simultaneous analysis of eight different biological conditions<span class="citation" data-cites="43RxThXr 1At6lP9vn"><sup><a href="#ref-43RxThXr" role="doc-biblioref">270</a>,<a href="#ref-1At6lP9vn" role="doc-biblioref">271</a></sup></span>.</p>
<h4 id="itraq-hydrazide-itraqh">iTRAQ hydrazide (iTRAQH)</h4>
<p>iTRAQH is an isobaric tagging reagent for the selective labeling and relative quantification of carbonyl (CO) groups in proteins<span class="citation" data-cites="5gl2gxA4"><sup><a href="#ref-5gl2gxA4" role="doc-biblioref">272</a></sup></span>.
The reactive CO and oxygen groups which are generated as the byproducts of oxidation of lipids at the time of oxidative stress causes protein carbonylation<span class="citation" data-cites="t4QMhmYv"><sup><a href="#ref-t4QMhmYv" role="doc-biblioref">273</a></sup></span>.
iTRAQH is produced from iTRAQ and surplus of hydrazine.
This reagent reacts with peptides which are carbonylated, thus forming a hydrazone group.
iTRAQH is a novel method for analyzing carbonylation sites in proteins utilizing an isobaric tag for absolute and relative quantitation iTRAQ derivative, iTRAQH, and the analytical power of linear ion trap instruments (QqLIT).
This new strategy seems to be well suited for quantifying carbonylation at large scales because it avoids time-consuming enrichment procedures<span class="citation" data-cites="5gl2gxA4"><sup><a href="#ref-5gl2gxA4" role="doc-biblioref">272</a></sup></span>.
Thus, there is no need for enriching modified peptides before LC-MS/MS analysis.</p>
<h4 id="tandem-mass-tag-tmt">Tandem Mass Tag (TMT)</h4>
<p>TMT labeling is based on a similar principle as that of iTRAQ.
The TMT label is based on a glycine backbone and this limits the amount of sites for heavy atom incorporation
In the case of 6-plex-TMT, the masses of reporter groups are nominally 126, 127, 128, 129, 130, and 131 Da<span class="citation" data-cites="kIpNiFeU"><sup><a href="#ref-kIpNiFeU" role="doc-biblioref">264</a></sup></span>.
10- and 11-plex TMT kits were recently supplanted by proline-based TMT tags (TMTpro), originally introduced as 16-plex kits in 2019<span class="citation" data-cites="12bJg7wN8"><sup><a href="#ref-12bJg7wN8" role="doc-biblioref">274</a></sup></span> and upgraded to an 18-plex platform in 2021<span class="citation" data-cites="1HEOHr3h0"><sup><a href="#ref-1HEOHr3h0" role="doc-biblioref">275</a></sup></span>.
Due to co-isolation of multiple precursors leading to reporter ion compression, TMT works best with MS platforms which allow quantitation at the MS3 level (e.g., Thermo Fisher Orbitrap Tribrid instruments)<span class="citation" data-cites="15rS5UMb9 9irghCdu"><sup><a href="#ref-15rS5UMb9" role="doc-biblioref">262</a>,<a href="#ref-9irghCdu" role="doc-biblioref">276</a></sup></span>.
In experiments performed on Q-Orbitrap or Q-TOF platforms, MS2-based sequence identification (via b- and y-type ions) and quantitation (via low m/z reporter ion intensities) is performed.
In experiments performed on Q-Orbitrap-LIT platfroms, MS3-based quantitation can be performed wherein the top ~10 most abundant b- and y-type ions are synchronously co-isolated in the linear ion trap and fragmented once more before product ions are scanned out in the Orbitrap mass analzer.
Adding an additional layer of gas-phase purification limits the ratio distortion of co-isolated precursors within isobaric multiplexed quantitative proteomics<span class="citation" data-cites="1GJhuH2mX agiL3n2A"><sup><a href="#ref-1GJhuH2mX" role="doc-biblioref">277</a>,<a href="#ref-agiL3n2A" role="doc-biblioref">278</a></sup></span>.
Infrared photoactivation of co-isolated TMT fragment ions generates more quantitative reporter ion generation and sensitivity relative to standard beam-type collisional activation<span class="citation" data-cites="1EjowCoSS"><sup><a href="#ref-1EjowCoSS" role="doc-biblioref">279</a></sup></span>.
High-field asymmetric waveform spectrometry (FAIMS) also aids the accuracy of TMT-based quantitation on Tribrid systems<span class="citation" data-cites="VW0HtJln"><sup><a href="#ref-VW0HtJln" role="doc-biblioref">280</a></sup></span>.
TMT is widely used for quantitative protein biomarker discovery.
In addition, TMT labeling technique helps multiplex sample analysis enabling efficient use of instrument time.
TMT labelling also controls for technical variation because after samples are mixed the ratios are locked in, and any sample loss would be equal across channels.
A wide range of TMT reagents with different multiplexing capabilities are available, such as TMT zero, TMT duplex, TMT 6-plex, TMT 10-plex, and TMT 11-plex.
The recent addition of TMTpro tags, slight adaptations of the TMT structure based on proline, allow for higher plexing with TMTpro 16-plex<span class="citation" data-cites="12bJg7wN8"><sup><a href="#ref-12bJg7wN8" role="doc-biblioref">274</a></sup></span> and now TMTpro 18-plex<span class="citation" data-cites="1HEOHr3h0"><sup><a href="#ref-1HEOHr3h0" role="doc-biblioref">275</a></sup></span>.
These TMT reagents have a similar chemical structure, which allows the efficient transition from method development to multiplexed peptide quantification<span class="citation" data-cites="OlzbV8x4"><sup><a href="#ref-OlzbV8x4" role="doc-biblioref">267</a></sup></span>.</p>
<h4 id="iodotmt">iodoTMT</h4>
<p>IodoTMT reagents are isobaric reagents used for tagging cysteine residues of peptides.
The commercially available IodoTMT reagents are iodoTMTzero and iodoTMT 6-plex<span class="citation" data-cites="ghHhZLY4 RikGmYWm"><sup><a href="#ref-ghHhZLY4" role="doc-biblioref">281</a>,<a href="#ref-RikGmYWm" role="doc-biblioref">282</a></sup></span>.
These reagents are useful for studies of cysteine oxidation modifications because only unoxidized cysteine is modified.</p>
<h4 id="aminoxytmt-isobaric-mass-tags">aminoxyTMT Isobaric Mass Tags</h4>
<p>Also referred to as glyco-TMTs, these reagents have chemistry similar to iTRAQH.
The stable isotope-labeled glyco-TMTs are utilized for quantitating N-linked glycans.
They are derived from the original TMT reagents with an addition of carbonyl-reactive groups, which involve either hydrazide or aminoxy chemistry as functional groups.
These aminoxy TMTs show a better performance as compared to its iTRAQH counterparts in terms of efficiency of labeling and quantification.
The glyco-TMT compounds consist of stable isotopes thus enabling (i) isobaric quantification using MS/MS spectra and (ii) quantification in MS1 spectra using heavy/light pairs.
Aminoxy TMT6-128 and TMT6-131 along with the hydrazide TMT2-126 and TMT2-127 reagents can be used for isobaric quantification.
In the quantification at MS1 level, the light TMT0 and the heavy TMT6 reagents have a difference in mass of 5.0105 Da which is sufficient to separate the isotopic patterns of all common N-glycans.
Glycan quantification based on glyco-TMTs generates more accurate quantification in MS1 spectra over a broad dynamic range.
Intact proteins or their digests obtained from biological samples are treated with PNGase F/A glycosidases to release the N-linked glycans during the process of labeling using aminoxyTMT reagents.
The free glycans are then purified and labeled with the aminoxyTMT reagent at the reducing end.
The labeled glycans from individual samples are subsequently pooled and then undergo analysis in MS for identification of glycoforms in the sample and quantification of relative abundance of reporter ions at MS/MS level<span class="citation" data-cites="XS3gqf5Y"><sup><a href="#ref-XS3gqf5Y" role="doc-biblioref">283</a></sup></span>.</p>
<h4 id="nn-dimethyl-leucine-dileu">N,N-Dimethyl leucine (DiLeu)</h4>
<p>The N,N-Dimethyl leucine, also referred to as DiLeu, is an tandem mass tag reagent which is isobaric and has reporter ions of isotope-encoded dimethylated leucine<span class="citation" data-cites="WvWnJWa8"><sup><a href="#ref-WvWnJWa8" role="doc-biblioref">284</a></sup></span>.
Each incorporated label produces a 145.1 Da mass shift.
A maximum of four samples can be simultaneously analyzed using DiLeu at a highly reduced cost.
MS/MS analysis shows intense reporter ions i.e., dimethylated leucine a1 ions at 115, 116, 117, and 118 <em>m/z</em>.
The labeling efficiency of DiLeu tags are similar to that of the iTRAQ tags.
Although, DiLeu-labeled peptides offer increased confidence of identification of peptides and more reliable quantification as they undergo better fragmentation, generating higher reporter ion intensities<span class="citation" data-cites="WvWnJWa8"><sup><a href="#ref-WvWnJWa8" role="doc-biblioref">284</a></sup></span>.</p>
<h4 id="deuterium-isobaric-amine-reactive-tag-diart">Deuterium isobaric Amine Reactive Tag (DiART)</h4>
<p>DiART is an isobaric tagging method used in quantitative proteomics<span class="citation" data-cites="OhLoelJI 1FMnd7AHB"><sup><a href="#ref-OhLoelJI" role="doc-biblioref">285</a>,<a href="#ref-1FMnd7AHB" role="doc-biblioref">286</a></sup></span>.
The reporter group in DiART tags is a N,N′-dimethyl leucine reporter group with a mass to charge range of 114–119. DiART reagents can a label a maximum of six samples and further analyzed by MS.
The isotope purity of DiART reagents is very high hence correction of isotopic impurities is not needed at the time of data analysis<span class="citation" data-cites="1CFiIlIUn"><sup><a href="#ref-1CFiIlIUn" role="doc-biblioref">287</a></sup></span>.
The performances of DiART including the mechanism of fragmentation, the number of proteins identified and the quantification accuracy are similar to iTRAQ.
Irrespective of the sequence of the peptide, reporter ions of high-intensity are produced by DiART tags in comparison to those with iTRAQ and thus, DiART labeling can be used to quantify more peptides as well as those with lower abundance, and with reliable results<span class="citation" data-cites="OhLoelJI"><sup><a href="#ref-OhLoelJI" role="doc-biblioref">285</a></sup></span>.
DiART serves as a cheaper alternative to TMT and iTRAQ while also having a comparable labeling efficiency.
It has been observed that these tags are useful in labeling huge protein quantities from cell lysates before TiO2 enrichment in quantitative phosphoproteomics studies<span class="citation" data-cites="bHqseFm6"><sup><a href="#ref-bHqseFm6" role="doc-biblioref">288</a></sup></span>.</p>
<h4 id="hyperplexing-or-higher-order-multiplexing">Hyperplexing or higher-order multiplexing</h4>
<p>Some studies have combined metabolic labels (i.e., SILAC) with chemical tags (i.e., iTRAQ or TMT) to expand the multiplexing capacity of proteomics experiments referred to as hyperplexing<span class="citation" data-cites="1E4PqaQ4R AP3g38e9"><sup><a href="#ref-1E4PqaQ4R" role="doc-biblioref">289</a>,<a href="#ref-AP3g38e9" role="doc-biblioref">290</a></sup></span> or higher order multiplexing<span class="citation" data-cites="KGUz4OlJ RdDAao5G TL5Hhri7"><sup><a href="#ref-KGUz4OlJ" role="doc-biblioref">291</a>–<a href="#ref-TL5Hhri7" role="doc-biblioref">293</a></sup></span>.
This technique combines MS1- and MS2-based quantitative methods to achieve enhanced multiplexing by multiplying the channels used in each dimension.
This allows for the quantitation of proteomes across multiple samples in a single MS run.
The technique uses two types of mass encoding to label different biological samples.
The labeled samples are then mixed together, which increases the MS1 peptide signal.
Protein turnover rates were studied using SILAC-iTRAQ multitagging<span class="citation" data-cites="oFWp3zgI"><sup><a href="#ref-oFWp3zgI" role="doc-biblioref">294</a></sup></span>, while various combined precursor isotopic labeling and isobaric tagging (cPILOT) studies employed MS1 dimethyl labeling with iTRAQ<span class="citation" data-cites="kwtDZh6G SOsi8PV2 MmHm4JiO tQGfQU9D"><sup><a href="#ref-kwtDZh6G" role="doc-biblioref">295</a>–<a href="#ref-tQGfQU9D" role="doc-biblioref">298</a></sup></span>.
SILAC-TMT hyperplexing was used to study the temporal response to rapamycin in yeast<span class="citation" data-cites="1CCC3i2yw"><sup><a href="#ref-1CCC3i2yw" role="doc-biblioref">299</a></sup></span>.
SILAC-iTRAQ-TAILS method was developed to study matrix metalloproteinases in the secretomes of keratinocytes and fibroblasts<span class="citation" data-cites="a0wHWh6L"><sup><a href="#ref-a0wHWh6L" role="doc-biblioref">300</a></sup></span>.
TMT-SILAC hyperplexing was used to study synthesis and degradation rates in human fibroblasts<span class="citation" data-cites="EcjnY7Z5"><sup><a href="#ref-EcjnY7Z5" role="doc-biblioref">301</a></sup></span>.
Variants of SILAC-iTRAQ and BONCAT, namely BONPlex<span class="citation" data-cites="bnkhDKrK"><sup><a href="#ref-bnkhDKrK" role="doc-biblioref">302</a></sup></span> and MITNCAT<span class="citation" data-cites="sUDoGv6r"><sup><a href="#ref-sUDoGv6r" role="doc-biblioref">303</a></sup></span> were also developed to study temporal proteome dynamics.</p>
<h2 class="page_break_before" id="enrichment-and-depletion">6. Enrichment and Depletion</h2>
<p>In order to study low abundance protein modifications, or to study rare proteins in complex mixtures, various methods have been developed to enrich or deplete specific proteins or peptides.</p>
<h3 id="peptide-enrichment">Peptide enrichment</h3>
<h4 id="phosphoproteomics-1">Phosphoproteomics</h4>
<p>Protein phosphorylation, a hallmark of protein regulation, dictates protein interactions, signaling, and cellular viability.
This post-translational modification (PTM) involves the installation of a negatively charged phosphate moiety (PO 4-) onto the hydroxyl side-chain of serine or threonine residues on target proteins.
Additionally, while less commonly modified than serine and threonine, histidine<span class="citation" data-cites="aoK1gRMs 17Un7LUDR 14IWO8iZc"><sup><a href="#ref-aoK1gRMs" role="doc-biblioref">304</a>–<a href="#ref-14IWO8iZc" role="doc-biblioref">306</a></sup></span>, arginine<span class="citation" data-cites="xcaW4mpA"><sup><a href="#ref-xcaW4mpA" role="doc-biblioref">307</a></sup></span>, and tyrosine<span class="citation" data-cites="1ELOuUl8I nO23Yma7 uOTrEnEQ"><sup><a href="#ref-1ELOuUl8I" role="doc-biblioref">308</a>–<a href="#ref-uOTrEnEQ" role="doc-biblioref">310</a></sup></span> phosphorylation also represent important cell signaling biology.
Protein kinases catalyze the transfer of PO 4- group from ATP to the nucleophile (OH) group of serine, threonine, and tyrosine residues, while protein phosphatases catalyze the removal of PO4-.
Phosphorylation changes the charge of a protein, often altering protein conformation and therefore function<span class="citation" data-cites="8FBpI8YZ"><sup><a href="#ref-8FBpI8YZ" role="doc-biblioref">311</a></sup></span>.
Protein phosphorylation is one of the major PTMs that alters the stability, subcellular location, enzymatic activity complex formation, degradation of protein, and cell signaling of protein with a diverse role in cells<span class="citation" data-cites="ahQQ60nZ 9tmuuRlu"><sup><a href="#ref-FzJAr4ba" role="doc-biblioref">314</a></sup></span>.
Phosphorylation can regulate almost all cellular processes, including metabolism, growth, division, differentiation, apoptosis, and signal transduction pathways<span class="citation" data-cites="e078CIsS"><sup><a href="#ref-e078CIsS" role="doc-biblioref">34</a></sup></span>.
Rapid changes in protein phosphorylation are associated with several diseases<span class="citation" data-cites="CG2bNzkD"><sup><a href="#ref-CG2bNzkD" role="doc-biblioref">315</a></sup></span>.</p>
<p>Several methods are used to characterize phosphorylation using modification-specific enrichment techniques combined with advanced MS/MS methods and computational data analysis<span class="citation" data-cites="gPKAjoqC"><sup><a href="#ref-gPKAjoqC" role="doc-biblioref">316</a></sup></span>.
There are many challenges with studying phosphorylation<span class="citation" data-cites="ZhEOEHzl"><sup><a href="#ref-ZhEOEHzl" role="doc-biblioref">317</a></sup></span>.
For example, many phosphopeptides are low stoichiometry compared to non-phosphorylated peptides, which makes them difficult to identify.
Phosphopeptides also exhibit low ionization efficiency<span class="citation" data-cites="IZ8H05sX"><sup><a href="#ref-IZ8H05sX" role="doc-biblioref">318</a></sup></span>.
To overcome these challenges, it is important to reduce sample complexity to detect large numbers of phosphorylation sites.
This is accomplished using enrichment the modified proteins and/or peptides<span class="citation" data-cites="1DdZgE3oc 1AhoIVYp8 2bm470Yr"><sup><a href="#ref-1DdZgE3oc" role="doc-biblioref">319</a>–<a href="#ref-2bm470Yr" role="doc-biblioref">321</a></sup></span>.</p>
<p>As with any proteomics experiment, phosphoproteomics studies require protein extraction, proteolytic enzyme digestion, phosphopeptide enrichment, peptide fractionation, LC-MS/MS, bioinformatics data analysis, and biological function inference.
Special consideration is required during protein extraction where the cell lysis buffer should include phosphatase inhibitors such as sodium orthovanadate, sodium pyrophosphatase and beta-glycerophosphate<span class="citation" data-cites="cXqDwSab"><sup><a href="#ref-cXqDwSab" role="doc-biblioref">322</a></sup></span>.</p>
<p>Enrichment can be done at the protein level before proteolysis.
Phosphoprotein enrichment typically involves the use of immobilized metal-affinity chromatography (IMAC) to selectively capture phosphorylated proteins based on their high-affinity binding to metal ions such as Ga(III), Fe(III), Zn(II) and Al(III)<span class="citation" data-cites="vghskTda 1Biu5x5kI GINSjn64 tH2WxrEl LbRH4fW1"><sup><a href="#ref-vghskTda" role="doc-biblioref">323</a>–<a href="#ref-LbRH4fW1" role="doc-biblioref">327</a></sup></span>.</p>
<p>Enrichment is more commonly performed at the peptide level because there are several advantages over phosphoprotein-level enrichment.
First, peptides have simpler three-dimensional structures than proteins, which makes them easier to separate and analyze.
Second, phosphopeptide enrichment is not hindered by small, lipophilic, and very acidic or alkaline proteins<span class="citation" data-cites="2bm470Yr"><sup><a href="#ref-2bm470Yr" role="doc-biblioref">321</a></sup></span>.
Third, prefractionation techniques such as strong anion exchange chromatography (SAX), strong cation exchange chromatography (SCX) and hydrophilic interaction chromatography (HILIC) are easier to use for peptide separation than they are for protein separation, and they are more sensitive than 2D-gel electrophoresis that is often used for intact proteins<span class="citation" data-cites="f75VaG7P OoEPX07F"><sup><a href="#ref-f75VaG7P" role="doc-biblioref">328</a>,<a href="#ref-OoEPX07F" role="doc-biblioref">329</a></sup></span>.
As a result, phosphopeptide enrichment has yielded more experimental data than phosphoprotein enrichment<span class="citation" data-cites="tH2WxrEl"><sup><a href="#ref-tH2WxrEl" role="doc-biblioref">326</a></sup></span>.
Phosphopeptide enrichment is typically done after any isobaric labeling strategy, although several have investigated the importance of order at these stages.</p>
<p>Phosphopeptide enrichment often uses titanium dioxide (TiO2)<span class="citation" data-cites="CkrjTtPg"><sup><a href="#ref-CkrjTtPg" role="doc-biblioref">330</a></sup></span> and/or IMAC such as Fe3+ coupled to solid-phase materials<span class="citation" data-cites="GINSjn64 cXqDwSab 11CslpEEW"><sup><a href="#ref-cXqDwSab" role="doc-biblioref">322</a>,<a href="#ref-GINSjn64" role="doc-biblioref">325</a>,<a href="#ref-11CslpEEW" role="doc-biblioref">331</a></sup></span>.
The most common cost-effective beads for phosphopeptide extraction with Ti are ReSyn and GL Sciences, and CubeBio for Fe-based beads.
Often organic acids such as glutamic acid, lactic acid, glycolic acid is added to compete with acidic non-phosphopeptides for binding to the metal-ions.
Carr and coworkers even demonstrated phosphoproteome analysis without any enrichment<span class="citation" data-cites="SLLSf3H4"><sup><a href="#ref-SLLSf3H4" role="doc-biblioref">332</a></sup></span>.</p>
<p>The use of Fe-IMAC column chromatography allows for the improved phosphopeptide from complex peptide mixtures<span class="citation" data-cites="Bij8Oa4F"><sup><a href="#ref-Bij8Oa4F" role="doc-biblioref">333</a></sup></span>.
Compared to other formats like StageTips or batch incubations with TiO2 or Ti-IMAC beads, Fe-IMAC columns do not suffer from problems with poor binding or elution of phosphopeptides, and the efficiency of enrichment increases linearly with the amount of starting material<span class="citation" data-cites="15ptCbE4S"><sup><a href="#ref-15ptCbE4S" role="doc-biblioref">334</a></sup></span>.
Also with recent improvements to Ti based beads, the MagReSyn Ti-IMAC HP with Ti4+ attached with a flexible linker (to reduce steric hindrance) activated with phosphonate groups for Ti4+ chelation, and the MagReSyn Zr-IMAC HP, also with a flexible linker activated with phosphonate groups for Zr4+ chelation, have shown superior phosphopeptide extraction as compared to FE-IMAC.</p>
<p>Multiple IMAC steps can be used in parallel or sequentially to improve phosphopeptide coverage.
Lai et al., (2012) showed that the combined use of Fe3+-IMAC and Ti(4+)-IMAC chromatography enables complementary identification of more phosphorylation sites than either technique alone<span class="citation" data-cites="fPTYjwxV"><sup><a href="#ref-fPTYjwxV" role="doc-biblioref">335</a></sup></span>.
A novel phosphopeptide enrichment technique using sequential enrichment with magnetic Fe3O4 and TiO2 particles was developed to detect mono- and multi-phosphorylated peptides<span class="citation" data-cites="Hii2H7k2"><sup><a href="#ref-Hii2H7k2" role="doc-biblioref">336</a></sup></span>.</p>
<p>More recently, the use of Src Homology 2 (SH2) domains as specific affinity reagents for phosphotyrosine is an emerging technology allowing an expanded fractionation of tyrosine phosphopeptides. Here, short chain protein domains have been constructed and affinity enhanced through yeast two hybrid screening to arrive at high affinity matrices capable of outperforming traditional IMAC approaches<span class="citation" data-cites="vuiYaWET 1G78xLdle"><sup><a href="#ref-vuiYaWET" role="doc-biblioref">337</a>,<a href="#ref-1G78xLdle" role="doc-biblioref">338</a></sup></span>.</p>
<h4 id="tips-for-studying-phosphorylation">Tips for studying phosphorylation:</h4>
<p>• Cell lysates should always be prepared using phosphatase inhibitors and samples should be placed on the ice during sonication for protein extraction.
• Increase the amount of starting material of your sample for phosphoenrichment to at least 1 mg of protein or more for optimal results.
• If using anti-phosphorylation antibodies, ensure their specificity is confirmed with other methods.
• Make sure to select a suitable method for the phosphoenrichment that fits the experiment goals.
• TiO2-based phosphopeptide enrichment methods have different enrichment specificities; selecting non-phosphopeptide excluders such as glutamic acid, lactic acid, glycolic acid, and dihydroxybenzoic acid are the key part of the study<span class="citation" data-cites="G33wPuEv"><sup><a href="#ref-G33wPuEv" role="doc-biblioref">339</a></sup></span>.
• Do not use milk as a blocking agent when western blotting for phosphorylation because milk contains the phosphoprotein casein and can lead to a higher background due to non-specific binding.</p>
<h4 id="glycosylation">Glycosylation</h4>
<p>Mass spectrometry-based analysis of protein glycosylation has emerged as the premier technology to characterize such a universal and diverse class of biomolecules.
Glycosylation is a heterogenous post-translational modification that decorates many proteins within the proteome, conferring broad changes in protein activity.<span class="citation" data-cites="2h6JMw35 nSDlzuSO"><sup><a href="#ref-nSDlzuSO" role="doc-biblioref">56</a>,<a href="#ref-2h6JMw35" role="doc-biblioref">340</a></sup></span>
This PTM can take many forms.
The covalent linkage of mono- or oligosaccharides to polypeptide backbones through a nitrogen atom of asparagine (N) or an oxygen atom of serine (S) or threonine (T) side-chains creates N- and O-glycans, respectively.
The heterogenity of proteoglycans is not directly tied to the genome, and thus cannot be inferred.
Rather, the abundance and activity of protein glycosylation is governed by glycosyltransferases and glycosidases which add and remove glycans, respectively.
The fields of glycobiology and bioanalytical chemistry are intricately intertwined with mass spectrometry at the center thanks in part to its power of detecting any modification that imparts a mass shift.</p>
<p>Due to the myriad glycan structures and proteins which harbor them, the enrichment of glycoproteins or glycopeptides is not as streamlined as that of other PTMs<span class="citation" data-cites="DC1ukRtO"><sup><a href="#ref-DC1ukRtO" role="doc-biblioref">341</a></sup></span>.
The enrichment of glycoproteome from the greater proteome inherently introduces bias prior to the LC-MS/MS analysis.
One must take into account which class or classes of glycopeptides they are interested in analyzing before enrichment for optimal LC-MS/MS results.
Glycopeptides can be enriched via glycan affinity, for example to glycan-binding proteins, chemical properties like charge or hydrophilicity, chemical coupling of glycans to stationary phases, and by bioorthogonal, chemical biology approaches.
Glycan affinity-based enrichment strategies include the use of lectins, antibodies, inactivated enzymes, immobilized metal affinity chromatography (IMAC), and metal oxide affinity chromatography (MOAC).
The enrichment of glycopeptides by their chemical properties, for example by biopolymer charge and hydrophobicity, include hydrophilic interaction chromatography (HILIC), electrostatic repulsion-hydrophilic interaction chromatography (ERLIC), and porous graphitic carbon (PGC).
One variation of ERLIC combines strong anion exchange, electrostatic repulsion, and hydrophilic interaction chromatography (SAX-ERLIC) has risen in popularity thanks to robustness and commercially available enrichment kits<span class="citation" data-cites="pgTlWbfO 1Bj6B87md"><sup><a href="#ref-pgTlWbfO" role="doc-biblioref">342</a>,<a href="#ref-1Bj6B87md" role="doc-biblioref">343</a></sup></span>.</p>
<p>Chemical coupling methods most often used to enrich the glycoproteome employ hydrazide chemistry for sialylated glycopeptides.
Glycan are cleaved from the stationary phase by PNGase F.
The dependence of chemical coupling methods on PNGase F biases their output toward N-glycopeptides.
Alkoxyamine compounds and boronic acid-based methods have also shown utility.
We direct readers to several reviews on glycopeptide enrichment strategies<span class="citation" data-cites="DC1ukRtO 3T8OfCLe 17QEVShGe ZH7oVCIW 4qdFwSjo"><sup><a href="#ref-DC1ukRtO" role="doc-biblioref">341</a>,<a href="#ref-3T8OfCLe" role="doc-biblioref">344</a>–<a href="#ref-4qdFwSjo" role="doc-biblioref">347</a></sup></span></p>
<h4 id="antibody-enrichments-of-modifications">Antibody enrichments of modifications</h4>
<p>Western blot analysis is used to detect the PTMs in a protein through the use of antibodies<span class="citation" data-cites="LeABskLB"><sup><a href="#ref-LeABskLB" role="doc-biblioref">348</a></sup></span>.
As an extension, pan-PTM antibodies have been used to isolate peptides bearing the PTM of interest<span class="citation" data-cites="xEkij3xI"><sup><a href="#ref-xEkij3xI" role="doc-biblioref">349</a></sup></span>.
One benefit of this approach is that peptides are less likely to experience non-specific binding than proteins<span class="citation" data-cites="gPKAjoqC"><sup><a href="#ref-gPKAjoqC" role="doc-biblioref">316</a></sup></span>.
Initially peptide immunoaffinity precipitation was developed to enrich for phosphotyrosine-containing peptides.
This protocol was initially designed to enrich for phosphotyrosine-containing peptides<span class="citation" data-cites="oN0RedlD"><sup><a href="#ref-oN0RedlD" role="doc-biblioref">350</a></sup></span>.
Peptide immunoprecipitation yielded significantly greater coverage of the phosphotyrosine proteome than global phosphorylation enrichment strategies by enriching for a subset of the phosphoproteome.<br />
Since then, peptide immunoaffinity precipitation has been used successfully to enrich for peptides with other phosphorylation motifs<span class="citation" data-cites="11yMtXtzL nEmRnHBE"><sup><a href="#ref-11yMtXtzL" role="doc-biblioref">351</a>,<a href="#ref-nEmRnHBE" role="doc-biblioref">352</a></sup></span> as well as peptides with other modifications such as the diglycyl-lysine residue of ubiquitin modification after trypsin proteolysis<span class="citation" data-cites="fZ6aM0wq JMsurOWF 11FwNutcB"><sup><a href="#ref-fZ6aM0wq" role="doc-biblioref">353</a>–<a href="#ref-11FwNutcB" role="doc-biblioref">355</a></sup></span>, acetyl-lysine<span class="citation" data-cites="tWUlvotu 36Wx4WMF bAU4w5zL RMueHUu6 58knZUn0"><sup><a href="#ref-tWUlvotu" role="doc-biblioref">356</a>–<a href="#ref-58knZUn0" role="doc-biblioref">360</a></sup></span>, arginine methylation<span class="citation" data-cites="I7T7nB5"><sup><a href="#ref-I7T7nB5" role="doc-biblioref">361</a></sup></span>, tyrosine nitration<span class="citation" data-cites="82ISyiE"><sup><a href="#ref-82ISyiE" role="doc-biblioref">362</a></sup></span>, and tyrosine phosphorylation<span class="citation" data-cites="1CF6MyQQ3 PgRmFpK6"><sup><a href="#ref-1CF6MyQQ3" role="doc-biblioref">363</a>,<a href="#ref-PgRmFpK6" role="doc-biblioref">364</a></sup></span>.</p>
<p>The O-linked β-D-N-acetylglucosamine (O-GlcNac) is found on serine and threonine residues and is involved in involved in the progression of cancers in multiple systems throughout the body<span class="citation" data-cites="hAQepsrJ"><sup><a href="#ref-hAQepsrJ" role="doc-biblioref">365</a></sup></span>.
Anti-O-GlcNAc monoclonal antibody enables enrichment from O-GlcNAcylated peptides of cells and tissues.
These antibodies have high sensitivity and specificity toward O-GlcNAc-modified peptides and do not identify O-GalNAc or GlcNAc in extended glycans<span class="citation" data-cites="UV2xOP8u"><sup><a href="#ref-UV2xOP8u" role="doc-biblioref">366</a></sup></span>.</p>
<h3 id="protein-depletion-blood-samples">Protein depletion (Blood samples)</h3>
<p>Many plasma proteomics studies involve the analysis of plasma<span class="citation" data-cites="Zhiw1CQm 3DOhQZCg"><sup><a href="#ref-Zhiw1CQm" role="doc-biblioref">367</a>,<a href="#ref-3DOhQZCg" role="doc-biblioref">368</a></sup></span>.
However, the abundance range of proteins in the blood/plasma proteome exceeds 10 orders of magnitude.
Due to this wide dynamic range, detection of proteins with medium and low abundance by proteomic analyses is difficult<span class="citation" data-cites="xwhmqv6G"><sup><a href="#ref-xwhmqv6G" role="doc-biblioref">369</a></sup></span>, and identifying protein biomarkers from biological samples such as blood is often obstructed by proteins present at higher concentrations.
In fact, the top 14 most abundant proteins in human plasma constitute over 99% of the total protein mass.
The removal of these high-abundant proteins enables the detection of less abundant proteins.
The ability to deplete abundant proteins with specificity, reproducibility, and selectivity is extremely important in proteomic studies<span class="citation" data-cites="MZe4do4v"><sup><a href="#ref-MZe4do4v" role="doc-biblioref">370</a></sup></span>.</p>
<p>The following are some of the methods used for abundant protein depletion:</p>
<h4 id="dye-ligand-depletion">Dye-ligand depletion:</h4>
<p>This method is used for the depletion of serum albumin based on the interaction between albumin and dyes like Cibacron Blue (CB) through electrostatic force, hydrogen bonding and hydrophobic interactions.
The method is relatively low cost, widely available, robust and has high binding capacity.
However, it lacks specificity and has varying efficiency<span class="citation" data-cites="CkoUIvms mxcAPQRD"><sup><a href="#ref-CkoUIvms" role="doc-biblioref">371</a>,<a href="#ref-mxcAPQRD" role="doc-biblioref">372</a></sup></span>.</p>
<h4 id="protein-ligand-depletion">Protein-ligand depletion:</h4>
<p>This method is used for depletion of immunoglobulins (Ig) based on the interaction between the Fragment crystallizable (Fc) region of these Igs<span class="citation" data-cites="LonwNp1"><sup><a href="#ref-LonwNp1" role="doc-biblioref">373</a></sup></span> and cell wall protein A, G or A/G of Staphylococcus aureus and Streptococcus spp<span class="citation" data-cites="23zI8SGX LRMcEp9I"><sup><a href="#ref-23zI8SGX" role="doc-biblioref">374</a>,<a href="#ref-LRMcEp9I" role="doc-biblioref">375</a></sup></span>.
It is highly selective and has high yield and purity.
However, non-specific binding may occur due to co-absorption of other proteins<span class="citation" data-cites="gxra2RsF"><sup><a href="#ref-gxra2RsF" role="doc-biblioref">376</a></sup></span>.</p>
<h4 id="immunodepletion">Immunodepletion:</h4>
<p>This method is used for depletion of proteins having high abundance in plasma or serum on the basis of the specific interaction of these proteins with their respective antibodies (antigen-antibody interaction)<span class="citation" data-cites="13El9kECw"><sup><a href="#ref-13El9kECw" role="doc-biblioref">377</a></sup></span>.
Immunodepletion has high specificity and commercial kits based on columns (Agilent MARS column) or loose beads (Thermo-Fisher High-Select depletions beads) both deplete the Top 14 most abundant protein in blood and are also readily available but expensive.
For some protocols, non-specific binding to these immunodepletion columns or beads by proteins of interest is highly dependent on washing conditions<span class="citation" data-cites="gxra2RsF"><sup><a href="#ref-gxra2RsF" role="doc-biblioref">376</a></sup></span>.</p>
<h4 id="combinatorial-peptide-ligand-library">Combinatorial peptide ligand library:</h4>
<p>This method is used for partial depletion of major proteins i.e., those with high abundance and for relative enrichment of lower and medium abundant proteins<span class="citation" data-cites="xeb6grTF"><sup><a href="#ref-xeb6grTF" role="doc-biblioref">378</a></sup></span>.
It is based on the interaction with an array of ligands which are essentially peptides of 6 amino acids in length.
It is also used for normalization of the global protein abundance<span class="citation" data-cites="1AFK16JwO"><sup><a href="#ref-1AFK16JwO" role="doc-biblioref">379</a></sup></span>.
However, the drawbacks include non-specific binding as well as loss of proteins due to incomplete elution or inefficient binding<span class="citation" data-cites="gxra2RsF"><sup><a href="#ref-gxra2RsF" role="doc-biblioref">376</a></sup></span>.</p>
<h4 id="precipitation">Precipitation:</h4>
<p>This method of abundant protein depletion works by altering the solubility of proteins using a chemical reagent including inorganic salt solution<span class="citation" data-cites="hcs5DCXe"><sup><a href="#ref-hcs5DCXe" role="doc-biblioref">380</a></sup></span>, organic solvents<span class="citation" data-cites="jOj0DZcZ"><sup><a href="#ref-jOj0DZcZ" role="doc-biblioref">381</a></sup></span>, non-ionic polymer<span class="citation" data-cites="1DUeuj8TB"><sup><a href="#ref-1DUeuj8TB" role="doc-biblioref">382</a></sup></span> and reducing agents<span class="citation" data-cites="jv9v3twJ"><sup><a href="#ref-jv9v3twJ" role="doc-biblioref">383</a></sup></span>.
It is extremely simple and cost-effective.
However, it is less specific with a risk of protein loss, difficulty in protein resolubilization as well as time consuming<span class="citation" data-cites="gxra2RsF"><sup><a href="#ref-gxra2RsF" role="doc-biblioref">376</a></sup></span>.</p>
<h4 id="new-technologies">New technologies:</h4>
<p>Newer methods of highly abundant protein depletion are based on the interaction between polymers such as bacterial cellulose nanofibers<span class="citation" data-cites="a6kEMboa"><sup><a href="#ref-a6kEMboa" role="doc-biblioref">384</a></sup></span>, cryogels<span class="citation" data-cites="1KROnN78"><sup><a href="#ref-1KROnN78" role="doc-biblioref">385</a></sup></span>, and nanomaterials<span class="citation" data-cites="7cVO0ON7"><sup><a href="#ref-7cVO0ON7" role="doc-biblioref">386</a></sup></span>.<br />
These techniques are highly specific, relatively cheap, and very stable.
They can also be reused since they have larger binding capacity and less cross-reactivity<span class="citation" data-cites="gxra2RsF"><sup><a href="#ref-gxra2RsF" role="doc-biblioref">376</a></sup></span>.</p>
<p>Protein enrichment/depletion strategies which make use of protein coronas<span class="citation" data-cites="iOe9QsbS 37v9j349"><sup><a href="#ref-iOe9QsbS" role="doc-biblioref">387</a>,<a href="#ref-37v9j349" role="doc-biblioref">388</a></sup></span> or extracellular vesicle enrichment<span class="citation" data-cites="1EP8DknKE"><sup><a href="#ref-1EP8DknKE" role="doc-biblioref">389</a></sup></span> are enabling researchers to probe deeper into the plasma, serum, lymph, and cerebrospinal fluid proteomes.
Automated nanoparticle (NP) protein corona-based proteomics workflows are a novel approach to perform deep blood-based proteomics analysis at unprecedented protein IDs above 6,000 proteins<span class="citation" data-cites="d0vk8527"><sup><a href="#ref-d0vk8527" role="doc-biblioref">390</a></sup></span>.
NPs can efficiently compress the dynamic range of protein abundances into a mass spectrometry accessible detection range and allow full automation of the protein preparation process providing a platform that can rival affinity based approaches with equivalent reproducibility and sensitivity<span class="citation" data-cites="12wGIM8jB"><sup><a href="#ref-12wGIM8jB" role="doc-biblioref">391</a></sup></span>.</p>
<h2 class="page_break_before" id="peptide-purification-and-fractionation">7. Peptide Purification and Fractionation</h2>
<h3 id="peptide-purification-methods">Peptide purification methods</h3>
<p>Before peptide analysis, interferences from sample preparation must be removed.
There are several approaches to purify peptides.</p>
<h4 id="solid-phase-extraction-spe">Solid phase extraction (SPE)</h4>
<p>Solid phase extraction (SPE) is a common MS-based proteomics technique employed in the sample preparation.
In this method, compound isolation is based on chemical and physical properties, which determines the distribution of compounds between a mobile phase (liquid) and a stationary phase (solid).
After the molecules bind, washing of the bound compounds is performed and then molecules are made to elute from the stationary phase after replacing the mobile phase with the elution buffer.
The material used for SPE is usually discarded after every sample and no gradient is applied for elution (single-step procedure of elution)<span class="citation" data-cites="nCqm9thz"><sup><a href="#ref-nCqm9thz" role="doc-biblioref">392</a></sup></span>.
Thus, using SPE only a specific analyte group gets separated, which depends on the stationary phase.
Hence, SPE is primarily used for sample clean-up and for reducing complexity of the sample.
For MS-based proteomic analysis, it is largely used to get rid of salts and other contaminants that might lead to ion suppression.
The major drawback of this technique is that with SPE only a small fraction of the sample is examined because not all compounds are captured, but only those with binding capabilities same as that of the sorbent.
The material for SPE is available in various types, including (micro-) columns, cartridges, plates, micropipette tips, and functionalized magnetic beads (MBs)<span class="citation" data-cites="167ADUQ9x klXQ49fD"><sup><a href="#ref-167ADUQ9x" role="doc-biblioref">393</a>,<a href="#ref-klXQ49fD" role="doc-biblioref">394</a></sup></span>.
Reversed-phase is the most widely used material for SPE in proteomic studies for the proteins and peptide fractionation and rarely, ion-exchange material.
For the separation of glycosylated proteins and peptides, the preferred material is normal phase such as HILIC<span class="citation" data-cites="18rJRCf42 kPeMPVtT"><sup><a href="#ref-18rJRCf42" role="doc-biblioref">395</a>,<a href="#ref-kPeMPVtT" role="doc-biblioref">396</a></sup></span>.
SPE materials which are less commonly used are silica- or polystyrene-based ones<span class="citation" data-cites="16XWFxoc3 us64YkZ0"><sup><a href="#ref-16XWFxoc3" role="doc-biblioref">397</a>,<a href="#ref-us64YkZ0" role="doc-biblioref">398</a></sup></span>.
The other types of SPE methods are IEX, metal chelation, and affinity-based<span class="citation" data-cites="nPfahhRa"><sup><a href="#ref-nPfahhRa" role="doc-biblioref">399</a></sup></span>.</p>
<p>The basic idea behind the choice of binding and wash versus elution solutions for SPE is that that the binding and wash solutions should favor the interaction between the analytes of interest and the solid phase, whereas the elution solution should favor the interaction of the analyte with the liquid phase (<strong>Figure 5</strong>).
For example, with reversed phase SPE, the solid phase is C18 or some other hydrophobic chemistry.
Binding of peptides to this solid phase is based on the hydrophobicity of peptides, mostly due to the presence of hydrophobic amino acid side chains; leucine is the most common amino acid in human proteins.
To encourage peptides to ‘like’ the stationary phase more than the liquid phase, the peptides are loaded in aqueous solution.
This will enable washing of the hydrophilic contaminants like salts, small polar buffer molecules, and polar denaturants like urea.
After washing the bound peptides, they can be eluted by switching the liquid phase to something hydrophobic, which allows the peptides to partition more into the liquid phase and elute from the solid phase.</p>
<div id="fig:SPE" class="fignos">
<figure>
<img src="images/SPE_v05.png" style="width:100.0%" data-tag="5" alt="Figure 5: Solid phase extraction (SPE). SPE is a sample preparation technique that uses a solid adsorbent contained most commonly in a cartridge device to selectively adsorb certain molecules from solution. The first step is the conditioning of the cartridge which involves wetting the adsorbent to solvate its functional groups and filling the void spaces with solvent thereby removing any air in the column. This is necessary to produce a suitable environment for adsorption and thus ensure reproducible interaction with the analytes. After conditioning, the sample is loaded in the cartridge. This can be performed with the aid of positive or negative pressure to ensure a constant flow rate. In this step molecules bind the adsorbent and interferences pass through. Next, the column is washed with the mobile phase to eliminate the contaminants while ensuring the analyte remains bound. Finally, peptides are eluted in an appropriate buffer solution with polarity or charge that competes with interaction with the solid phase." />
<figcaption aria-hidden="true"><span>Figure 5:</span> <strong>Solid phase extraction (SPE).</strong>
SPE is a sample preparation technique that uses a solid adsorbent contained most commonly in a cartridge device to selectively adsorb certain molecules from solution.
The first step is the conditioning of the cartridge which involves wetting the adsorbent to solvate its functional groups and filling the void spaces with solvent thereby removing any air in the column.
This is necessary to produce a suitable environment for adsorption and thus ensure reproducible interaction with the analytes.
After conditioning, the sample is loaded in the cartridge.
This can be performed with the aid of positive or negative pressure to ensure a constant flow rate.
In this step molecules bind the adsorbent and interferences pass through.
Next, the column is washed with the mobile phase to eliminate the contaminants while ensuring the analyte remains bound.
Finally, peptides are eluted in an appropriate buffer solution with polarity or charge that competes with interaction with the solid phase.</figcaption>
</figure>
</div>
<h4 id="specific-types-of-peptide-purification">Specific Types of peptide purification</h4>
<p>There are many additional peptide purification methods that are commonly used in proteomics currently.
These methods include the following:</p>
<ol type="1">
<li>StageTips, in-stagetip (iST)<span class="citation" data-cites="TY5C1ch7 j0ZcPOCn"><sup><a href="#ref-TY5C1ch7" role="doc-biblioref">400</a>,<a href="#ref-j0ZcPOCn" role="doc-biblioref">401</a></sup></span></li>
<li>SP2 or SP3<span class="citation" data-cites="SWsK5zPt"><sup><a href="#ref-SWsK5zPt" role="doc-biblioref">402</a></sup></span></li>
<li>Suspension trapping (S-trap)<span class="citation" data-cites="1FU2NfVYY"><sup><a href="#ref-1FU2NfVYY" role="doc-biblioref">59</a></sup></span></li>
</ol>
<h3 id="peptide-fractionation-methods">Peptide fractionation methods</h3>
<p>The number of peptides produced from proteolysis of the whole proteome is immense.
Thus, after peptides are cleaned from interferences, they are often fractionated into subsets to enable increased proteome coverage.
The characterization of the whole proteome is expected from higher order organisms, and with rising interest in post-translational modifications, an elaborate coverage of protein sequence is required.
There are different methods for peptide fractionation as follows:</p>
<h4 id="ion-exchange-chromatography-iec">Ion-exchange chromatography (IEC)</h4>
<p>This method involves the separation based on contrasting electric charge<span class="citation" data-cites="ncGFPcxC"><sup><a href="#ref-ncGFPcxC" role="doc-biblioref">403</a></sup></span>.
In this approach, the mechanism of analyte retention is based on the principle of electrostatic attraction between the sample and the stationary phase functional groups (FGs), having opposite charges.
IEC is classified into two types: cation-exchange and anion-exchange chromatography.
In cation-exchange chromatography, at an acidic pH, the negatively charged functional groups such as sulfates are attracted to positively charged peptides, whereas, in anion-exchange chromatography, positively charged FGs such as quaternary ammoniums are attracted to peptides with negative charge at an alkaline pH.
These techniques are further classified into: strong (cation [SCX] and anion [SAX] exchange), and weak exchangers (cation [WCX] and anion [WAX] exchange), based on the type of FG attached<span class="citation" data-cites="Bm9Z2DVx"><sup><a href="#ref-Bm9Z2DVx" role="doc-biblioref">404</a></sup></span>.
These functional groups are most commonly supported in resins made up of silica and synthetic polymers, however, some inorganic materials are sometimes used<span class="citation" data-cites="ncGFPcxC"><sup><a href="#ref-ncGFPcxC" role="doc-biblioref">403</a></sup></span>.
In the IEC method, peptide elution is performed using a mobile phase with higher ionic strength, to ensure peptide partition into the liquid phase. SCX along with a salt gradient/plug is a routinely used proteomics technique.
In the SCX method, peptides are resolved according to their net charge, in which the peptide with the lowest positive charge is eluted first.
Increasing the salt concentration decreases the peptide retention time due to competition with the electrostatic interactions between the peptides and the solid phase.
However, SCX resolution is limited compared to reversed phase chromatography and will thus limit the suitability of this technique for complex mixtures<span class="citation" data-cites="L1FaRtpm"><sup><a href="#ref-L1FaRtpm" role="doc-biblioref">405</a></sup></span>.</p>
<h4 id="reversed-phase-chromatography-rplc">Reversed-phase chromatography (RPLC)</h4>
<p>Reversed-phase chromatography is the most commonly used chromatographic technique which separates molecules in solution having neutral pH based on their hydrophobicity.
The separation occurs on the basis of the partition coefficient of analytes between the mobile phase and the hydrophobic stationary phase.
Highly polar peptides elute before the ones having less polarity because of the strong interaction with the hydrophobic functional groups forming a layer similar to a liquid around the silica resin<span class="citation" data-cites="GjvtcEz1"><sup><a href="#ref-GjvtcEz1" role="doc-biblioref">406</a></sup></span>.
RPLC has been widely used in separation of peptides because of its compatibility with gradient elution and aqueous samples and its retention mechanism, which modulates separation owing to changes in the properties like pH, additives and organic modifier<span class="citation" data-cites="1FrVJlffz"><sup><a href="#ref-1FrVJlffz" role="doc-biblioref">407</a></sup></span>.
Numerous factors influence the capacity of chromatographic peaks, such as temperature, column length, stationary phase, particle size, mobile-phase ion-pairing reagent, mobile-phase modifier and gradient slope<span class="citation" data-cites="1HPaIsist"><sup><a href="#ref-1HPaIsist" role="doc-biblioref">408</a></sup></span>.
Usually online RPLC is done at acidic pH to ensure peptide ionization, but it can be paired with offline high pH RPLC and multiple fraction concatenation to produce orthogonal separation due to altered ionization of amino acids changing peptide hydrophobicity<span class="citation" data-cites="190oCO8Ki"><sup><a href="#ref-190oCO8Ki" role="doc-biblioref">409</a></sup></span>.</p>
<h4 id="hydrophilic-interaction-liquid-chromatography-hilic">Hydrophilic interaction liquid chromatography (HILIC)</h4>
<p>Inverse-gradient chromatography was the forerunner to HILIC<span class="citation" data-cites="15mGcvpNQ"><sup><a href="#ref-15mGcvpNQ" role="doc-biblioref">410</a></sup></span>
HILIC is similar in its principle to normal-phase chromatography where the stationary phase is polar and the intitial solvent conditions are nonpolar.
Gradient elution in HILIC is accomplished by increasing the polarity of the mobile phase, by decreasing the concentration of organic solvent, i.e., in the “opposite” direction compared to RPLC separations.
With charged HILIC stationary phases there is also a possibility of increasing the salt or buffer concentration during a gradient to disrupt electrostatic interactions with the solute<span class="citation" data-cites="g9BMjmYA KqYbI8ep"><sup><a href="#ref-g9BMjmYA" role="doc-biblioref">411</a>,<a href="#ref-KqYbI8ep" role="doc-biblioref">412</a></sup></span>.
Thus, the peptides with less polarity elute before the more polar peptides.
It is used for the separation of hydrophilic peptides and polar analytes<span class="citation" data-cites="16k9ga0WM"><sup><a href="#ref-16k9ga0WM" role="doc-biblioref">413</a></sup></span>.
This separation is achieved by a stationary phase that is hydrophilic in nature, for example: cyano-, diol-, amino- bonded phases<span class="citation" data-cites="nrDJzBh3"><sup><a href="#ref-nrDJzBh3" role="doc-biblioref">414</a></sup></span>, and an organic and hydrophobic mobile phase<span class="citation" data-cites="g9BMjmYA"><sup><a href="#ref-g9BMjmYA" role="doc-biblioref">411</a></sup></span>.
HILIC can also be used for enrichment and targeted proteomic analysis of PTMs, such as glycosylation, N-acetylation and phosphorylation, which increase the polarity of peptides and therefore also their retention on HILIC<span class="citation" data-cites="GjvtcEz1"><sup><a href="#ref-GjvtcEz1" role="doc-biblioref">406</a></sup></span>.</p>
<h4 id="electrostatic-repulsion-hydrophilic-interaction-chromatography-erlic">Electrostatic repulsion-hydrophilic interaction chromatography (ERLIC)</h4>
<p>ERLIC is a method based on use of a weak anion exchange column operated at low pH with high organic solvent enabling isocratic elution<span class="citation" data-cites="Kf2BNqap"><sup><a href="#ref-Kf2BNqap" role="doc-biblioref">415</a></sup></span>.
Acidic peptides are retained by electrostatic interaction, basic and neutral peptides are retained through hydrophilic interaction made favorable by high organic solvent.
This improves retention of acidic peptides and reduces retention of basic peptides compared to normal HILIC<span class="citation" data-cites="1HJnUznNp"><sup><a href="#ref-1HJnUznNp" role="doc-biblioref">416</a></sup></span>.</p>
<h4 id="isoelectric-focusing-ief">Isoelectric focusing (IEF)</h4>
<p>IEF is a type of high-resolution (HR) technique of electrophoresis used for the separation as well as concentration of peptides that are amphoteric in nature on the basis of their isoelectric point (pI) using a solution without buffer consisting of either carrier ampholytes or a gel with immobilized pH gradient (IPG).
After IEF separation, the separated amphoteric peptides in the liquid phase are recovered for further analysis by RPLC-MS/MS<span class="citation" data-cites="Nk7uUlzO"><sup><a href="#ref-Nk7uUlzO" role="doc-biblioref">417</a></sup></span>.
Along with being a technique with improved resolution and capacity, for separation of peptides, IEF provides with additional information on physicochemical properties of the peptides, for example: peptide iso electric point (pI) which acts as a tool for validation and filtration for identifying MS/MS peptide sequence during the step of database search<span class="citation" data-cites="Qf7HZAh"><sup><a href="#ref-Qf7HZAh" role="doc-biblioref">418</a></sup></span>.
The IEF system is not only used for increasing the coverage of proteome but also in quantitative label-free<span class="citation" data-cites="LSDxad5T"><sup><a href="#ref-LSDxad5T" role="doc-biblioref">419</a></sup></span> and stable isobaric labeling experiments<span class="citation" data-cites="Qf7HZAh"><sup><a href="#ref-Qf7HZAh" role="doc-biblioref">418</a></sup></span>.
IEF and gel-based separations have fallen out of favor in the last decade due to improvements in liquid chromatography.</p>
<h2 class="page_break_before" id="liquid-chromatography-lc">8. Liquid Chromatography (LC)</h2>
<p>Chromatography is the physical sorting of a mixture of molecular species that are dissolved in a mobile phase through the strength of binding, or affinity, to the chromatographic column’s stationary phase<span class="citation" data-cites="rNop1hIz"><sup><a href="#ref-rNop1hIz" role="doc-biblioref">420</a></sup></span>.
The mobile phase is pressure driven through the column and molecular species, or analytes, that have a strong affinity to the stationary phase are retained, or slowed, while those with a weak affinity pass through quickly.
Thusly the analytes are separated by order of elution from the column.
Chromatography can exploit most physical properties of the analytes, including ionic charge (anion/cation exchange chromatography), hydrogen binding (hydrophilic interaction), and size (size exclusion chromatography, capillary electrophoresis).
In some chromatographic separations the mobile phase composition is adjusted by mixing two or more buffers at different ratios to influence the strength of affinity of individual analytes to the stationary phase and exquisitely regulate retention.</p>
<p>Mass spectrometers suffer from ion suppression, a phenomenon where the over-abundance of one or a few species within the ion population entering the mass spectrometer masks the presence of less abundant species<span class="citation" data-cites="71TlBiJy"><sup><a href="#ref-71TlBiJy" role="doc-biblioref">421</a></sup></span>.
Complex biological samples, such as tissue, cell lysate, or physiological fluids contain a wide dynamic range of molecule concentrations that span many orders of magnitude.
The physical separation of analytes from biological samples by LC reduces the complexity of the ion population presented to the mass spectrometer at a given time, thus allowing the instrument to carry out the necessary fragmentation scans to identify and quantify the detectable species.
Therefore, one major benefit of LC is that it allows detection of low abundant analytes in other elution windows.</p>
<p>The field of proteomics predominantly separates peptides using reversed phase liquid chromatography<span class="citation" data-cites="pRj1odLy T9Yv2Omz WG5jq46X"><sup><a href="#ref-pRj1odLy" role="doc-biblioref">422</a>–<a href="#ref-WG5jq46X" role="doc-biblioref">424</a></sup></span>.
Reversed stationary phase is most commonly composed of microscopic (1-3 μm) silica beads coated with covalently bound long (e.g. C18) hydrophobic alkyl chains.
The hydrophobic side chains of certain residues and the peptide backbone bind to this stationary phase through non-polar interactions.
These interactions are strong in an aqueous solvent but are disrupted when the organic composition of the solvent is increased.
Thus, in a reversed phase separation the proportion of non-polar, or organic, solvent in the mobile phase is gradually increased to release analytes from the stationary phase based on the strength of hydrophobic binding: weakly bound hydrophilic analytes elute with a low organic level in the mobile phase and strongly bound hydrophobic analytes only elute when the organic composition reaches a higher percentage.
By far the most popular combination of solvents for peptide analysis is water and acetonitrile with dilute acid modifier (such as 0.1% formic acid or 0.5% acetic acid).
The programmed rate at which the proportion of organic solvent is increased in the mobile phase is called the “gradient”, which you will often find described in the methods sections for reversed phase separations.</p>
<h3 id="lc-considerations-related-to-electrospray-ionization-esi">LC Considerations Related to Electrospray Ionization (ESI)</h3>
<p>LC is paired to MS through ESI, and LC parameters greatly influence ESI.
The analytes are eluted in a liquid mobile phase and must be released into the gas phase as charged ions for detection by mass spectrometry.
This is achieved by spraying the eluent from the chromatographic separation through a narrow nozzle under a high voltage potential (1,000-4,000 volts) between the nozzle, or emitter, and the mass spectrometer inlet.
The eluent is sprayed as a mist of small charged droplets that explode into smaller droplets as the solvent evaporates and the repelling columbic force of the charged analytes increases<span class="citation" data-cites="BslEV2vj"><sup><a href="#ref-BslEV2vj" role="doc-biblioref">425</a></sup></span>.
The droplets become progressively smaller until individual analyte molecules are ejected.
The ejected analytes are ionized by the retained charge and can thus be manipulated by the electric fields in the mass spectrometer to measure their mass and perform the necessary fragmentations to elucidate structure.</p>
<p>The chromatographic flowrate (the volume of mobile phase driven through the chromatographic column per unit time e.g. uL/min) dictates the efficiency of electrospray ionization (proportion of analytes eluting from the column that are ionized and into the gas phase) and is thus a key consideration for sensitivity of analysis<span class="citation" data-cites="hwO7hkQN"><sup><a href="#ref-hwO7hkQN" role="doc-biblioref">426</a></sup></span>.
Reduced flowrates generate smaller droplets which degrade into ejected charged analytes rapidly, thus resulting in more detectable analytes and higher ionization efficiency.
Electrospray ionization efficiency is also aided by an inert sheath gas, high temperature, and reduced pressure between the nozzle and ion lensing elements, thus decent sensitivity can still be achieved at high flowrates.
For more detailed discussion of ionization, see the “Ionization” section.</p>
<h3 id="quality-attributes-of-chromatographic-separation">Quality Attributes of Chromatographic Separation</h3>
<p>The quality of chromatographic separation defines the number of analytes that are identified and quantified by LC-MS analysis.
The theory around chromatographic separation was developed when LCs were paired with spectrophotometer detectors that only measure the combined signal intensity from all co-eluting analytes.
The ability of MS to simultaneously detect the masses of individual components re-defines the significance of certain LC attributes.
For those looking for mathematical descriptions of chromatographic quality, refer to the “Van Deemter equation”, which we do not cover here to maintain simplicity<span class="citation" data-cites="156fxE3m7"><sup><a href="#ref-156fxE3m7" role="doc-biblioref">427</a></sup></span>.
The following attributes are the most important to consider in LC-MS.</p>
<h4 id="chromatographic-resolution">Chromatographic Resolution</h4>
<p>Chromatographic resolution is defined as the ability to fully resolve adjacent chromatographic peaks containing analytes with nearly equal affinities to the solid phase.
In mass spectrometry analytes are distinguished by mass even if they are not resolved by LC.
Thus in LC-MS, the more relevant, but closely related concept is the peak width at the half maximum (FWHM) of the peak.
A low FWHM indicates a sharp elution peak. In a sharp peak the entirety of the analyte population is electrosprayed into the mass spectrometer in a short time thus increasing the signal.
Low FWHM of high abundance species also confines their ionization suppression to narrow time windows, which means a lower number of co-eluting analytes are hidden.
Conversely, high FWHM means that the analyte signal is spread out over time, thus reducing sensitivity.
Furthermore, at a high FWHM, high abundance species mask analytes through ion suppression over a larger portion of the separation.</p>
<h4 id="peak-capacity">Peak Capacity</h4>
<p>Peak capacity is defined as the maximal number of peaks that ideally can be completely resolved in a pre-established time window.
A long separation in which FWHM remains low would have a large peak capacity and thus allow identification of many species.
Unfortunately increasing the length of a reversed phase gradient also increases the FWHM due to an increase in diffusion, which results in a diminishing return for longer analytical methods.
A longer separation provides more time and opportunities for the mass spectrometer to sample each analyte to acquire fragmentation spectra required for identification and the selection of gradient length should consider both the desired throughput and the speed of the MS data acquisition strategy.</p>
<h4 id="reproducibility-and-robustness">Reproducibility and Robustness</h4>
<p>Reproducibility is defined as the ability to repeatedly obtain the same measurement for the same analytes each time that the analysis is repeated.
In liquid chromatography this means that each analyte should elute at nearly the same retention time (the time elapsed since the start of the analysis until the analyte’s elution from the chromatographic column) with the same peak width.
Robustness is the ability of the system to maintain reproducible performance despite nonoptimal conditions.
The most typical obstacles to robustness are mechanical wear of the system components and the analytical column, fouling of the system by contaminants introduced in the samples, and clogging due to accumulation of contaminants.
High flow methods tend to be more robust due to reduced impact of pump and plumbing configurations and changes in dwell volumes, and the wider bore of the components used is more resilient to clogging.
However, higher flowrate comes at the cost of reduced sensitivity due to reduced ionization efficiency at higher flow rates and increases in the overall peak volume at constant sample loading, thus nanoflow (100-300 nL/min flowrate) chromatography remains a widely utilized strategy in proteomics.
For applications where sample is not limited, slightly higher amounts of applied samples can take advantage of robustness of higher flow rates in the microflow range using newer optimized electrospray sources<span class="citation" data-cites="XQiQIxLc"><sup><a href="#ref-XQiQIxLc" role="doc-biblioref">428</a></sup></span>.</p>
<h4 id="throughput-and-instrument-utilization">Throughput and Instrument Utilization</h4>
<p>Throughput is the number of samples that are analyzed in a given timeframe, for example samples per day.
High throughput is required to analyze thousands of samples that truly represent biological diversity in a timely manner.
Increasing throughput means less data are collected for individual samples.
Furthermore, many steps in the LC process are required for sample analysis in which no useful data is collected including sample injection, and system cleaning and equilibration, which reduce the ratio of data collected to instrument operation time, or instrument utilization.
The ability to perform these steps while a different sample is analyzed, or parallelization, increases instrument utilization and the amount of data collected by several minutes which is a significant increase when several samples are analyzed per hour.</p>
<h3 id="trapping-and-pre-columns">Trapping and Pre-Columns</h3>
<p>Trapping and pre-columns are short chromatographic columns that are used to increase robustness of an LC-MS system.
A pre-column is connected directly to the front of the analytical column and is intended to be disposable and to absorb contaminants and protect the analytical column.
The trapping column is connected indirectly to the analytical column through a valve. The valve can be switched to redirect the flow through the trapping column away from the analytical column.
This allows analytes to be loaded on the trapping column while analytes that are hydrophilic and poorly retained are washed away and do not contaminate the analytical column or the mass spectrometer.
This process is referred to as desalting, and once it is complete, the valve configuration is changed to connect the trapping column to the analytical column, and analytes captured on the trapping column can be eluted off the trap and through the analytical column for analysis by MS.
Certain trapping columns can be operated in both directions, which allows aggregates to be flushed away when the trapping column is cleaned in the reverse direction.
Additionally trapping columns are shorter and have less backpressure so they can be loaded with sample quickly at a fast flowrate.
Whereas loading the sample directly on the analytical column requires a slower flowrate.
Two trapping columns can be used in tandem to provide parallelization, while one trapping column is cleaned and loaded with samples the second trapping column is in line with the analytical column analyzing the sample that was loaded on it in the previous run<span class="citation" data-cites="dvtPvLcY L2sT906m"><sup><a href="#ref-dvtPvLcY" role="doc-biblioref">429</a>,<a href="#ref-L2sT906m" role="doc-biblioref">430</a></sup></span>.</p>
<h3 id="multi-dimensional-lc">Multi Dimensional LC</h3>
<p>Depth of profiling has previously been increased by combining two or more orthogonal LC separations.
Orthogonal in this context means that each separation sorts the analytes into different populations<span class="citation" data-cites="1Gtuii4F4"><sup><a href="#ref-1Gtuii4F4" role="doc-biblioref">431</a></sup></span>.
For example, a separation based on positive charge (strong cation exchange, SCX) separates analytes based on positive charge, and when paired with reversed phase chromatography results a higher peak capacity and more analytes identified.
The first highly popular method was multidimensional protein identification technology (MudPIT), which used online separation by SCX followed by C18 reversed phase<span class="citation" data-cites="1FnaHa1s9"><sup><a href="#ref-1FnaHa1s9" role="doc-biblioref">432</a></sup></span>.
However, the resolution of peptide separation by SCX is low, leading to the presence of peptides in many fractions.
The currently accepted most popular method for two-dimensional separation combines iterative reversed phase at different high and then low pH to sort analytes by changes in hydrophobicity due to changes in amino acid side chain ionization.
Although the separations are not entirely orthogonal, multiple fraction concatenation across the high pH elution can produce entirely orthogonal peptide sets<span class="citation" data-cites="dl5RDy6j"><sup><a href="#ref-dl5RDy6j" role="doc-biblioref">433</a></sup></span>.
In recent years the focus of proteomics has shifted from deep profiling of fewer samples to rapid profiling of large cohorts.
Thus, lengthy multidimensional methods have been replaced with single shot experiments only using one dimension of high resolution reversed phase separation<span class="citation" data-cites="44oI1g68"><sup><a href="#ref-44oI1g68" role="doc-biblioref">434</a></sup></span>.
However peak capacity is regained by using ion mobility spectrometry (separation of ionized peptides in the gas phase).</p>
<h2 class="page_break_before" id="peptide-ionization">9. Peptide Ionization</h2>
<p>As early as the late 1950s, derivitization reagents were used to make peptides volatile enough for electron impact ionization analysis<span class="citation" data-cites="11c7CVfcj"><sup><a href="#ref-11c7CVfcj" role="doc-biblioref">435</a></sup></span>.
Eventually this led to GC-MS analysis of derivatized peptides for sequencing<span class="citation" data-cites="189tZteJw"><sup><a href="#ref-189tZteJw" role="doc-biblioref">436</a></sup></span>.
In the early 1980s, fast atom bombardment (FAB) enabled peptide ionization and sequencing by MS/MS<span class="citation" data-cites="Tr33h2Fs"><sup><a href="#ref-Tr33h2Fs" role="doc-biblioref">437</a></sup></span>, but difficulty interfacing FAB with LC limited its utility<span class="citation" data-cites="lZwv1GwF"><sup><a href="#ref-lZwv1GwF" role="doc-biblioref">438</a></sup></span>.
New soft ionization techniques called matrix-assisted laser desorption (MALDI) and electrospray ionization (ESI) were applied to peptides around 1990, which revolutionized the field of proteomics by making high throughput ionization of peptides easy.
These two techniques were so impactful that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) “for their development of soft desorption ionization methods for mass spectrometric analyses of biological macromolecules”<span class="citation" data-cites="YdLSljVE"><sup><a href="#ref-YdLSljVE" role="doc-biblioref">439</a></sup></span>.</p>
<h3 id="maldi">MALDI</h3>
<p>The term “Matrix-assisted laser desorption” was coined by Hillenkamp and Karas in 1985, although this orignal paper only applied the technique to dipeptides<span class="citation" data-cites="zfiYj2co"><sup><a href="#ref-zfiYj2co" role="doc-biblioref">440</a></sup></span>.
It was Koichi Tanaka who first applied this idea to proteins above 10,000 Daltons in size and published a paper in the Proceedings of the 2nd Japan-China Joint Symposium on Mass spectrometry in 1987 (Tanaka, K., Ido, Y., Akita, S., Yoshida, Y. and Yoshida, T. (1987) Detection of high mass molecules by laser desorption time-of-flight mass spectrometry. Proceedings of the 2nd Japan-China Joint Symposium on Mass spectrometry, 185-187), and then in a follow-up paper published in 1988<span class="citation" data-cites="15Ye4Za7Y"><sup><a href="#ref-15Ye4Za7Y" role="doc-biblioref">9</a></sup></span>.
A few months later, Karas and Hillenkamp also demonstrated MALDI applied to proteins above 10kDa with MALDI<span class="citation" data-cites="tnQHCFV7"><sup><a href="#ref-tnQHCFV7" role="doc-biblioref">441</a></sup></span>.
This resulted in some controversy about who should have won the Nobel prize<span class="citation" data-cites="1HBJs2tAB"><sup><a href="#ref-1HBJs2tAB" role="doc-biblioref">442</a></sup></span> as it was felt by the community that Hillenkamp and Karas had provided the technology several years before but it was Koichi Tanaka that was the first to apply the MALDI technology to proteins a year before Hillenkamp and Karas.</p>
<p>MALDI first requires the peptide sample to be co-crystallized with a matrix molecule, which is usually a volatile, low molecular-weight, organic aromatic compound (<strong>Figure 6</strong>).
Some examples of such compounds are cyno-hydroxycinnamic acid, dihyrobenzic acid, sinapinic acid, alpha-hydroxycinnamic acid, ferulic acid etc<span class="citation" data-cites="5ZJdaya1"><sup><a href="#ref-5ZJdaya1" role="doc-biblioref">443</a></sup></span>.
Subsequently, the analyte is placed in a vacuum chamber in which it is irradiated with a laser, usually at 337nm<span class="citation" data-cites="wGHAVD9r"><sup><a href="#ref-wGHAVD9r" role="doc-biblioref">444</a></sup></span>.
This laser energy is absorbed by the matrix, which then transfers that energy along with its free protons to the co-crystalized peptides without significantly breaking them.
The matrix and co-crystallized sample generate plumes, and the volatile matrix imparts its protons to the peptides as it gets ionized first.
The weak acidic conditions used as well as the acidic nature of the matrix allows easy exchange of protons for the peptides to get ionized and fly under the electrical field in the mass spectrometer.
These ionized peptides generally form the metastable ions, most of them will fragment quickly<span class="citation" data-cites="DEnAe0bs"><sup><a href="#ref-DEnAe0bs" role="doc-biblioref">445</a></sup></span>.
However, it can take several milliseconds and the mass spectrometry analysis can be performed before this time.
Peptides ionized by MALDI almost always take up a single charge and thus observed and detected as [M+H]+ species.</p>
<h4 id="maldi-mechanism">MALDI Mechanism</h4>
<div id="fig:MALDI-mechanism" class="fignos">
<figure>
<img src="images/MALDI.png" style="width:100.0%" data-tag="6" alt="Figure 6: MALDI The analyte-matrix mixture is irradiated by a laser source, leading to ablation. Desorption and proton transfer ionize the analyte molecules that can then be accelerated into a mass spectrometer." />
<figcaption aria-hidden="true"><span>Figure 6:</span> <strong>MALDI</strong>
The analyte-matrix mixture is irradiated by a laser source, leading to ablation.
Desorption and proton transfer ionize the analyte molecules that can then be accelerated into a mass spectrometer.</figcaption>
</figure>
</div>
<p>According to PubMed, the number of publications related to MALDI peaked in 2013 and has been steadily declining.
Concurrently, the usage of MALDI for bottom-up proteomics has subsided in favor of the better depth and throughput possible from using ESI.
MALDI is still widely used for mass spectrometry imaging of proteins and metabolites<span class="citation" data-cites="kL1RzcZ6"><sup><a href="#ref-kL1RzcZ6" role="doc-biblioref">446</a></sup></span>.</p>
<h3 id="electrospray-ionization">Electrospray Ionization</h3>
<p>ESI was first applied to peptides by John Fenn and coworkers in 1989<span class="citation" data-cites="4Ajjx5fo"><sup><a href="#ref-4Ajjx5fo" role="doc-biblioref">8</a></sup></span>.
Concepts related to ESI were published at least as early as 1882, when Lord Rayleigh described the number of charges that could assemble on the surface of a droplet<span class="citation" data-cites="BslEV2vj"><sup><a href="#ref-BslEV2vj" role="doc-biblioref">425</a></sup></span>.
ESI is usually coupled with reverse-phase liquid-chromatography of peptides directly interfaced to a mass spectrometer.
A high voltage (~ 2 kV) is applied between the spray needle and the mass spectrometer (<strong>Figure 7</strong>).
As solvent exits the needle, it forms droplets that take on charge at the surface, and through a debated mechanism, those charges are imparted to peptide ions.
The liquid phase is generally kept acidic to help impart protons easily to the analytes.</p>
<p>Tryptic peptides ionized by ESI usually carry one charge one the side chain of their C-terminal residue (Arg or Lys) and one charge at their n-terminal amine.
Peptides can have more than one charge if they have a longer peptide backbone, have histidine residues, or have missed cleavages leaving extra Arg and Lys.
In most cases, peptides ionized by ESI are observed at more than one charge state.
Evidence suggests that the distribution of peptide charge states can be manipulated through chemical additives<span class="citation" data-cites="11vyuKWyw"><sup><a href="#ref-11vyuKWyw" role="doc-biblioref">447</a></sup></span>.</p>
<h4 id="electrospray-mechanism">Electrospray Mechanism</h4>
<div id="fig:ESI-mechanism" class="fignos">
<figure>
<img src="images/ESI.png" style="width:100.0%" data-tag="7" alt="Figure 7: Electrospray Ionization Charged droplets are formed, their size is reduced due to evaporation until charge repulsion leads to Coulomb fission and results in charged analyte molecules." />
<figcaption aria-hidden="true"><span>Figure 7:</span> <strong>Electrospray Ionization</strong>
Charged droplets are formed, their size is reduced due to evaporation until charge repulsion leads to Coulomb fission and results in charged analyte molecules.</figcaption>
</figure>
</div>
<p>The main goal of ESI is the production of gas-phase ions from electrolyte ions in solution.
During the process of ionization, the solution emerging from the electrospray needle or capillary is distorted into a Taylor cone and charged droplets are formed.
The charged droplets subsequently decrease in size due to solvent evaporation.
As the droplets shrink, the charge density and Coulombic repulsion increase.
This process destabilizes the droplets until the repulsion between the charges is higher than the surface tension and they fission (Coulomb explosion)<span class="citation" data-cites="4mpHldYP"><sup><a href="#ref-4mpHldYP" role="doc-biblioref">448</a></sup></span> <span class="citation" data-cites="93IPP3bH"><sup><a href="#ref-93IPP3bH" role="doc-biblioref">449</a></sup></span>.
Typical bottom-up proteomics experiments make use of acidic analyte solutions which leads to the formation of positively charged analyte molecules due to an excess presence of protons.</p>
<!-- TODO, explicitly describe the taylor cone and coulombic explosion -->
<!-- TODO: discussion of ionizaation mechanisms, ion evaporation versus charge residue model -->
<h2 class="page_break_before" id="mass-spectrometers">10. Mass Spectrometers</h2>
<h3 id="mass-spectrometry">Mass spectrometry</h3>
<p>Mass spectrometry is a science of ions; mass spectrometers serve as sophisticated instruments for determining the masses of compounds and elements.
Mass spectrometers can therefore be likened to an ultra-precise weigh scale that can differentiate mass variations down to a single electron, or even lighter.
Since J.J. Thomson’s initial exploration in 1912, the field of mass spectrometry has undergone numerous improvements, spanning from isotope assessment to the interpretation of biomacromolecules<span class="citation" data-cites="VhI3IC9H"><sup><a href="#ref-VhI3IC9H" role="doc-biblioref">450</a></sup></span>, all thanks to the combined efforts of diverse fields like chemistry, physics, electronic engineering, and computer science.
With the rapid improvement of sensitivity, mass resolution, tandem mass spectrometry methods and ion dissociation methods, mass spectrometers have evolved as a core tool for proteomic (and metabolomic) analysis.
It is precisely the widespread application of mass spectrometry in proteomics analysis that has given rise to more instrument manufacturers and a greater diversity of mass spectrometer types.
This also brings a happy annoyance to many beginners or researchers in other fields who have no background in mass spectrometry: which manufacturer and which type of mass spectrometry should I choose to analyze my samples?
Here, to help new learners build a basic understanding faster, we will briefly introduce some basic concepts, common types of mass spectrometers, and their suitable application scenarios.</p>
<h3 id="mass-spectrometer-structure-and-basic-principles">Mass Spectrometer Structure and Basic Principles</h3>
<p>The fundamental principle of mass spectrometry revolves around specific physical processes that can be described by various mathematical formulas.
Since this article serves as a guide for those new to the field, particularly those from a biology background, we’ve chosen to steer clear of delving too deeply into intricate mathematical and physical explanations.
However, for those keen on a deeper understanding, we’ve included references pertaining to these foundational principles.
Our focus lies on introducing fundamental concepts and outlining the typical workflow in mass spectrometry.</p>
<p>The process of mass spectrometry (MS) is to generate gas phase ions from compounds in samples by any suitable method, to separate these ions by their mass to charge (<em>m/z</em>) ratio, and then detect them by their respective <em>m/z</em> and abundance.
The successful implementation and demonstration of this process requires participation of five fundamental systems (<strong>Figure 8</strong>):</p>
<h4 id="the-ion-source.">1) The ion source.</h4>
<p>The ion source is where gas phase ions are generated.
As discussed in the prior chapter, for proteomic analysis, soft ionization methods such as ESI and MALDI are the most widely applied techniques<span class="citation" data-cites="4Ajjx5fo 15Ye4Za7Y"><sup><a href="#ref-4Ajjx5fo" role="doc-biblioref">8</a>,<a href="#ref-15Ye4Za7Y" role="doc-biblioref">9</a></sup></span>.
Additional ionization methods used to generate ions for mass spectrometry of small molecules include atmospheric pressure chemical ionization (APCI), atmospheric pressure photo ionization (APPI), electron ionization (EI) and chemical ionization (CI)<span class="citation" data-cites="lKt4oMBO 1B2HdTk7M"><sup><a href="#ref-lKt4oMBO" role="doc-biblioref">451</a>,<a href="#ref-1B2HdTk7M" role="doc-biblioref">452</a></sup></span>.</p>
<h4 id="the-mass-analyzer.">2) The mass analyzer.</h4>
<p>The mass analyzer is where gas phase ions are separated according to their <em>m/z</em> ratio based on physical principles.
There are several types of mass analyzers applied in mass spectrometry, including the quadrupole, linear ion trap and three-dimensional ion trap, orbitrap, Fourier transform-ion cyclotron resonance (FT-ICR), time-of-flight (TOF), and the magnetic sector analyzers<span class="citation" data-cites="18qirHBwQ 5vv7ID3o"><sup><a href="#ref-18qirHBwQ" role="doc-biblioref">453</a>,<a href="#ref-5vv7ID3o" role="doc-biblioref">454</a></sup></span>, each with unique advantages and applications (Table 4).
For proteomic analysis, tandem mass spectrometry, which involves combining two or more stages of mass analysis, is typically used to achieve precursor selection, structural analysis, and improved sensitivity<span class="citation" data-cites="MooVAdBf"><sup><a href="#ref-MooVAdBf" role="doc-biblioref">455</a></sup></span>.
The mass analyzer is the core component of a mass spectrometer, it is also the most important factor that we need to take into consideration when choosing a mass spectrometer for a specific project.</p>
<h4 id="the-detector.">3) The detector.</h4>
<p>The detector is where ions are detected and their respective <em>m/z</em> values and abundances are recorded, generating a mass spectrum.
Common types of ion detectors are illustrated in Table 5, including the Electron Multiplier (EM), Photomultiplier Tube (PMT), Microchannel Plate (MP) and Faraday Cup (FC), along with a summary of their strengths and limitations.
It is worth noting that Orbitrap and FT-ICR mass analyzers don’t use conventional detectors as listed above.
Instead, these analyzers detect an image current produced by oscillating ions<span class="citation" data-cites="Uaum1m63 svXqfJ4D erGVtl2k"><sup><a href="#ref-Uaum1m63" role="doc-biblioref">456</a>–<a href="#ref-erGVtl2k" role="doc-biblioref">458</a></sup></span>.
In both mass analyzers, the detector is essentially measuring an electrical current (or more accurately, a voltage that’s proportional to the current) that’s induced by the motion of the ions.
This signal is then processed to extract the frequencies of oscillation and Fourier-transformed into a mass spectrum, which is quite different from other types of detectors that count individual ions or particles striking a surface.
Longer transients generate higher resolution spectra.</p>
<h4 id="the-vacuum-system.">4) The vacuum system.</h4>
<p>This is designed to maintain a high-vacuum environment for ions’ transmission inside the instrument.
The vacuum system consists of different type of pumps including roughing vacuum pumps (rotary vane pumps, scroll pumps) and high-vacuum pumps (turbo molecular pumps, diffusion pumps).
Maintaining a high vacuum is essential to reduce collisions between analyte ions and inert gas molecules during their transmission from one region of the mass spectrometer to another, or during oscillations within a mass analyzer.
Collisions within the vacuum chamber may lead to unstable ion trajectories, unwanted fragmentation, poorer transmission efficiency, in turn leading to lower resolving powers and poorer sensitivities.
Even so, some inert gas is intentionally plumbed into the mass spectrometer either for collisional activated dissociation (CAD), typically with nitrogen, helium, or argon, or to dampen ions’ energy.
FT-ICR and Orbitrap mass analyzers require higher vacuum in the 10-9 to 10-11 Torr range, while TOFs require medium vacuum in the 10-7 to 10-8 Torr range, and quadrupole and ion trap insturments require a relatively low vacuum in the 10-5 to 10-6 Torr range.</p>
<h4 id="the-control-system.">5) The control system.</h4>
<p>This is needed to regulate and coordinate the various parts of the mass spectrometer to ensure seamless functioning.
This typically includes ion source control, mass analyzer control, detector control, data acquisition control, interfacing with auxiliary systems (such as a liquid chromatograph and gas chromatograph), and modules for instrument diagnostics and calibration.</p>
<div id="fig:MS-diagram" class="fignos">
<figure>
<img src="images/MS_f101.svg" style="width:100.0%" data-tag="8" alt="Figure 8: Diagram of typical mass spectrometer modules. Systems must have an ion source, mass analyzer, detector, vacuum system, and control system." />
<figcaption aria-hidden="true"><span>Figure 8:</span> <strong>Diagram of typical mass spectrometer modules.</strong>
Systems must have an ion source, mass analyzer, detector, vacuum system, and control system.</figcaption>
</figure>
</div>
<p>Table 4: Common mass analyzers.</p>
<table>
<colgroup>
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 32%" />
<col style="width: 35%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Type</th>
<th style="text-align: center;">Acronym</th>
<th style="text-align: center;">Principle</th>
<th style="text-align: center;">Characteristics</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Time-of-flight</td>
<td style="text-align: center;">TOF</td>
<td style="text-align: center;">Time dispersion of a pulsed ion beam; separation by the time it takes for ions to travel a fixed distance</td>
<td style="text-align: center;">High-speed analysis, large mass range and good sensitivity. Suited for fast data acquisition and high-throughput applications. Modern TOF systems usually can achieve mass resolution well over 10,000 (m/Δm) or even higher.</td>
</tr>
<tr class="even">
<td style="text-align: center;">Linear quadrupole</td>
<td style="text-align: center;">Q</td>
<td style="text-align: center;">Continuous ion beam in linear radio frequency quadrupole field; separation due to instability of ion trajectories; rods have applied alternating DC and RF</td>
<td style="text-align: center;">High transmission efficiency, simple design, good sensitivity, and tunable mass range; relatively low mass resolution ranges from several hundreds to a thousand; often used in tandem mass spectrometry (MS/MS) experiments</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Quadrupole ion trap</td>
<td style="text-align: center;">QIT</td>
<td style="text-align: center;">Traps ions by electromagnetic fields; separation in three-dimensional radio frequency quadrupole field by resonant excitation</td>
<td style="text-align: center;">Efficient for fragmenting ions and structural elucidation, higher sensitivity, and relatively compact which good for benchtop instruments. Relatively a low mass resolution around 1,000 - 3,000.</td>
</tr>
<tr class="even">
<td style="text-align: center;">Fourier transform-ion cyclotron resonance</td>
<td style="text-align: center;">FT-ICR</td>
<td style="text-align: center;">Traps ions in a strong magnetic field by Lorentz force; separation by cyclotron frequency, image current detection and Fourier transformation of transient signal</td>
<td style="text-align: center;">Ultimate high mass resolution (over 2,700,000 with 21 telsa magnets), making it ideal for elemental and isotopic analysis. Large size, low speed, and expensive in terms of both initial purchase cost and ongoing operation and maintenance costs.</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Orbitrap</td>
<td style="text-align: center;">Orbitrap</td>
<td style="text-align: center;">Axial oscillation in inhomogeneous electric field; detection of frequency after Fourier transformation of transient signal</td>
<td style="text-align: center;">Extremely high resolution and accuracy (up to 1,000,000), capable of resolving complex mixtures with high sensitivity. Relatively low speed, expensive in terms of both initial purchase cost and ongoing operation and maintenance costs. Need high vacuum.</td>
</tr>
</tbody>
</table>
<p>Table 5: Common detectors.</p>
<table>
<colgroup>
<col style="width: 25%" />
<col style="width: 27%" />
<col style="width: 47%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Type</th>
<th style="text-align: center;">Principle</th>
<th style="text-align: center;">Characteristics</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Electron Multiplier</td>
<td style="text-align: center;">Amplifies signals by utilizing a sequence of dynodes that emit secondary electrons when struck by an incident electron, creating a cascading effect. This results in an amplified output current at the final anode, proportional to the intensity of the initial signal.</td>
<td style="text-align: center;">Very good signal amplification to even one electron (may cause more noise dependent on gain), high sensitivity, need high vacuum and high voltage, expensive. Limited dynamic range, finite lifespan and need to be replaced periodically</td>
</tr>
<tr class="even">
<td style="text-align: center;">Faraday Cup</td>
<td style="text-align: center;">Charged particles, such as ions or electrons, enter the cup and transfer their charge to it, causing a change in electric potential that can be measured over time to infer the number of particles.</td>
<td style="text-align: center;">Suitable for particles and charge state detection. Simplicity and robustness, Wide dynamic range, no need for high voltage and high vacuum. Lower sensitivity. Sensitive to Secondary Emission directional sensitivity (direction of incoming particles)</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Microchannel Plate</td>
<td style="text-align: center;">Similar to electron multiplier, a two-dimensional matrix or “plate” of many tiny, parallel, hollow channels made from a type of glass that can generate secondary electron emissions upon incident particles striking the channel walls. These secondary emissions create an electron avalanche down the channels and amplifies the original signal.</td>
<td style="text-align: center;">Signal Amplification (Not as good as electron multiplier, but lower noise), Spatial Resolution ability, shorter life expectancy due to channel aging and depletion of the secondary emission material, smaller and cheaper than electron multiplier</td>
</tr>
<tr class="even">
<td style="text-align: center;">Daly Detector</td>
<td style="text-align: center;">Directing ions onto a surface (Doorknob) to trigger the emission of electrons, which are then accelerated towards a phosphor screen to produce photons, that are subsequently detected and amplified by a photomultiplier tube, thereby converting the ion signal into a measurable electrical signal.</td>
<td style="text-align: center;">High gain, ruggedness, wide dynamic range, suitable for high mass and high energy ions. Limited mass resolution, larger size and need high voltage, finite lifespan.</td>
</tr>
</tbody>
</table>
<h3 id="types-of-mass-spectrometers-used-for-proteomics.">Types of mass spectrometers used for proteomics.</h3>
<p>Typically, mass spectrometers are named based on the abbreviations of their principal or tandem mass analyzers.
This naming convention stems from the fact that the mass analyzer forms the core component of a mass spectrometer, and it also dictates key performance attributes such as mass resolution, scanning speed, sensitivity, and cycle time.
These performance metrics, in turn, determine what type of analysis we can conduct, its speed and its accuracy.
Next, we will focus on introducing several classic tandem mass spectrometry types commonly used in proteomics.</p>
<h3 id="triple-quadrupole-qqq.">1. Triple quadrupole (QqQ).</h3>
<p>Triple quadrupole mass spectrometer (often abbreviated as QqQ, QQQ, TQ, or TQMS) is a type of tandem mass spectrometer where three quadrupole mass analyzers are combined in series (<strong>Figure 9</strong>).
Each quadrupole is essentially a set of four parallel metal rods to which radio frequency (RF) and direct current (DC) voltages are applied to each opposing pair of rods.
The QqQ operates in a synchronized manner to isolate ions of interest (according to the Mathieu function) in the first quadrupole, induce fragmentation with inert gas in the second, and then detect the resulting product ions in the third quadrupole.
Specifically, the first quadrupole (Q1) is a mass filter, where ions of a specific <em>m/z</em> are selected from the incoming ion beam.
This is achieved by adjusting the voltage applied to the pair rods within the quadrupole, allowing ions with a particular <em>m/z</em> value to pass through while deflecting others.
The second quadrupole (Q2), also known as the collision cell, is where selected ions from Q1 are fragmented into product ions.
This fragmentation happens due to the collisions between inert gas molecules (nitrogen, argon, or helium) and ions, which causes the ions to break up (fragment) into smaller pieces (fragment ions).
For more detail about peptide fragmentation, see the Tandem Mass Spectrometry section.
This process is known as collision-induced dissociation (CID)<span class="citation" data-cites="EmIJDW71 17hxbWgXF"><sup><a href="#ref-EmIJDW71" role="doc-biblioref">459</a>,<a href="#ref-17hxbWgXF" role="doc-biblioref">460</a></sup></span>.
The Q2 is usually only subjected to RF potential and does not filter ions; instead, it transmits the product ions to the third quadrupole.
In some tandem mass spectrometry, hexapoles or octupoles are also used to replace a quadrupole as the collision cell.
Lastly, the third quadrupole (Q3) acts as a secondary mass filter, similar to Q1, but with the purpose of selecting specific fragment ions produced in the collision cell while excluding other ions.
The chosen ions are then directed to the detector, where their abundance is measured (<strong>Figure 9</strong>).
This process, involving precursor ion selection, precursor ion fragmentation, and product ion detection, is a general operating principle in tandem mass spectrometry and determines what kind of scan mode you can utilize.
While discovery-based proteomics approaches can be performed on triple-quadrupole systems, the data produced would be inferior to competing high resolution options.
Instead, QQQ instruments are widely used for targeted proteomics by operating in selected reaction monitoring (SRM) mode, which is also refered to as multiple reaction monitoring (MRM)<span class="citation" data-cites="mZfV6P3P"><sup><a href="#ref-mZfV6P3P" role="doc-biblioref">461</a></sup></span>.
QQQ instruments are available from all major vendors, including the QTRAP (Sciex), TSQ (Thermo), Xevo TQ-XS (Waters), LCMS-8050 (Shimadzu), 6475 (Agilent), and EVOQ LC-TQ (Bruker).
A key characteristic and advantage of QqQ is the flexibility of choosing various scan modes<span class="citation" data-cites="1ENeUykUA 1GjxHekcM 17hxbWgXF"><sup><a href="#ref-17hxbWgXF" role="doc-biblioref">460</a>,<a href="#ref-1ENeUykUA" role="doc-biblioref">462</a>,<a href="#ref-1GjxHekcM" role="doc-biblioref">463</a></sup></span>, including the following.</p>
<h4 id="product-ion-scan">1. Product Ion Scan:</h4>
<p>Q1 is set to filter a specific precursor ion, which is then fragmented in Q2.
Q3 scans the full range of product ion masses.
This mode is usually used to identify the structure of a particular compound.</p>
<h4 id="precursor-ion-scan">2. Precursor Ion Scan:</h4>
<p>Q3 is set to filter a specific product ion.
Q1 scans the full range of precursor ions, that when fragmented in Q2, yield the selected product ion.
This mode is used to find compounds that yield a specific fragment ion, which can be particularly useful when looking for compounds with a common structural motif.</p>
<h4 id="neutral-loss-scan">3. Neutral Loss Scan:</h4>
<p>Both Q1 and Q3 scan the full range of ions, but with a mass difference equal to a specific “neutral loss”.
This mode is used to identify compounds that, when fragmented, lose a specific neutral molecule.</p>
<h4 id="multipleselected-reaction-monitoring-msrm">4. Multiple/Selected Reaction Monitoring (M/SRM):</h4>
<p>Both Q1 and Q3 are set to filter specific ions (precursor and product, respectively).
This highly selective mode is used for quantitative analysis of specific compounds, offering excellent sensitivity and specificity<span class="citation" data-cites="1B9P7OPqR 9yFTjbpI"><sup><a href="#ref-1B9P7OPqR" role="doc-biblioref">464</a>,<a href="#ref-9yFTjbpI" role="doc-biblioref">465</a></sup></span>.</p>
<p>The triple quadrupole mass spectrometer is a highly versatile instrument, capable of both qualitative and quantitative analysis.
Enke and Yost at Michigan State University developed the first working triple-quadrupole mass spectrometer in the late 1970s<span class="citation" data-cites="Ddpn8u0F"><sup><a href="#ref-Ddpn8u0F" role="doc-biblioref">466</a></sup></span>.
QqQ is particularly well-suited for targeted quantitative analysis due to its high sensitivity, selectivity, and dynamic range, which has made it a go-to instrument in areas such as drug metabolism studies, environmental monitoring, food safety analysis, pharmaceuticals, and clinical diagnostics<span class="citation" data-cites="1AOQTDs57 11skpaXK6 BaGpN1HP 1ETD5TZZ8"><sup><a href="#ref-1AOQTDs57" role="doc-biblioref">467</a>–<a href="#ref-1ETD5TZZ8" role="doc-biblioref">470</a></sup></span>.</p>
<p>However, quadrupoles suffer from inherent limitations in mass resolution due to the constraints of principles and precision in mechanical manufacturing.
Consequently, QQQ instruments face difficulties in accurately identifying unknown molecules within complex mixtures and thus not appropriate for applications like structure analysis and biomarker discovery.</p>
<div id="fig:QQQ-diagram" class="fignos">
<figure>
<img src="images/MS_f102.svg" style="width:100.0%" data-tag="9" alt="Figure 9: Schematic diagram of typical QqQ system. Three quadrupoles enable precursor selection, fragmentation, and the fragment ion selection." />
<figcaption aria-hidden="true"><span>Figure 9:</span> <strong>Schematic diagram of typical QqQ system.</strong>
Three quadrupoles enable precursor selection, fragmentation, and the fragment ion selection.</figcaption>
</figure>
</div>
<h3 id="q-tof">2. Q-TOF</h3>
<p>Even though quadrupoles face difficulties in accurately identifying unknown peptides within complex mixtures due to its mass resolution, they serve effectively as mass filters, making them an excellent choice for combining with other high-resolution mass analyzers to form tandem mass spectrometry systems.
One commonly used approach is Quadrupole-Time-of-Flight Mass Spectrometer (Q-TOF-MS), a ‘hybrid’ device, integrating quadrupole techniques with a time-of-flight mass analyzer.
W.E. Stephens constructed and published the design of the first time-of-flight (TOF) analyzer in 1946<span class="citation" data-cites="EeH1Mu0v kFLq1VDE"><sup><a href="#ref-EeH1Mu0v" role="doc-biblioref">471</a>,<a href="#ref-kFLq1VDE" role="doc-biblioref">472</a></sup></span>.
The principle of TOF is quite straightforward: ions of different <em>m/z</em> are imparted with the same initial kinetic energy (E = Uq = ½ mv2) and then separated over time as they travel along a field-free drift path of known length.
If all ions begin their flight simultaneously, or at least within a short enough time span, the lighter ions will reach the detector before the heavier ones due to their faster velocity (V)<span class="citation" data-cites="15YzDfo72"><sup><a href="#ref-15YzDfo72" role="doc-biblioref">473</a></sup></span>.
Based on this principle, the <em>m/z</em> of different ions can be calculated according to the order in which they reach the detector.
Similarly, we can easily conclude that the longer the drift path, the higher of the mass resolution can reach if keep the response time of detector the same.
In fact, in pursuit of higher mass resolution, researchers have indeed built time-of-flight (TOF) drift tubes that are tens of meters long.
However, apparently, this is not practical for widely application in a regular lab place.
An alternative way to expand drift length and achieve higher resolution is to apply reflector (often called a reflectron).
The principles and advantages of using a reflector can be summarized as follows.</p>
<p>Under ideal circumstances within a TOF mass spectrometer, ions sharing the same <em>m/z</em> would reach the detector concurrently post-acceleration, thus generating a sharp peak on the mass spectrum.
However, the inherent oscillation path variability of ions within the mass spectrometer makes it challenging to maintain uniform initial kinetic energy amongst all ions, leading to peak broadening and a substantial reduction in mass resolution.
The reflector is designed to rectify this issue. Comprising a series of electrodes that set to different voltages, the reflector generates a retarding electric field that reverses ion trajectories back through the flight tube.
Notably, the reflector is engineered such that ions carrying lower kinetic energy delve less into the reflector and have a reduced flight path, while those with higher kinetic energy permeate more deeply and follow a longer flight path.
This equalizes the variances in initial kinetic energy, enabling ions of the same <em>m/z</em> to hit the detector almost simultaneously, thereby enhancing the resolution of TOF.</p>
<p>Furthermore, the usage of reflector effectively expands the flight path length within the same physical confines, resulting in superior ion separation and consequently, higher resolution.
This reflection comes at the cost of some ion loss, and therefore some sensitivity loss.
As such, reflecting TOFs are the basis of most commercial instruments currently in use.</p>
<p>The construction of a Q-TOF bears significant resemblance to a triple-quadrupole mass spectrometer, with the critical distinction that the third quadrupole has been replaced by a time-of-flight tube.
<strong>Figure 10</strong> delineates the schematic of a typical Quadrupole-Time-of-Flight (Q-TOF) mass spectrometer, which comprises three fundamental components:</p>
<h4 id="quadrupole-mass-analyzer-q.">1. Quadrupole mass analyzer (Q).</h4>
<p>This part of the instrument is basically the same to the Q1 in QqQ, which selects specific <em>m/z</em> values to pass through by applying a combination of DC and RF voltages across the rods.</p>
<h4 id="collision-cell.">2. Collision cell.</h4>
<p>Here, selected ions undergo collision-induced dissociation (CID) by interacting with a neutral gas, leading to their fragmentation into smaller constituents.
This process yields structural information about the original molecules.
Usually, quadrupole, hexapole, or even octopoles are used as the collision cell for better focusing and transporting.</p>
<h4 id="time-of-flight-tof-mass-analyzer.">3. Time-of-Flight (TOF) mass analyzer.</h4>
<p>Upon exiting the collision cell, the fragmented ions are reaccelerated into the ion modulator region of the time-of-flight analyzer.
There, they undergo pulsing by a strong electric field (typically 20 kV or higher) and get accelerated to a field free drift tube, and then reflected to the detector.</p>
<p>TOFs generally offer mass resolutions surpassing 50,000, rendering it a reliable instrument for identifying unknown compounds.
Moreover, the rapid travel time of ions in the vacuum tube (at the nanosecond level) confers the Q-TOF with distinctive benefits in short gradient and high-throughput analyses<span class="citation" data-cites="eJibibTn KJYZ12ho cXsqAa1Y"><sup><a href="#ref-eJibibTn" role="doc-biblioref">474</a>–<a href="#ref-cXsqAa1Y" role="doc-biblioref">476</a></sup></span>.
Another advantage of TOF is its broad mass range, which allows for the detection of large proteins, nanoclusters, and even large particles<span class="citation" data-cites="OeUnezUK 2oFwzqFP 3O301V9H"><sup><a href="#ref-OeUnezUK" role="doc-biblioref">477</a>–<a href="#ref-3O301V9H" role="doc-biblioref">479</a></sup></span>.
However, it should be noted that due to ion numbers and detector limitations, mass resolution is typically difficult to maintain over a wide mass range.</p>
<p>Presently, Q-TOF related instruments are available from all leading instrument manufacturers, and the main models are listed below:
Sciex: “TripleTOF® 6600+”, “TripleTOF® 5600+” System and “X500R QTOF” System.
Bruker Corporation: “Impact II”, “timsTOF” series, “microTOF-Q III”, “ultrafleXtreme-MALDI-TOF/TOF” and “maXis II”.
Agilent Technologies: “Agilent 6530 Accurate-Mass Q-TOF”, “Agilent 6545 Accurate-Mass Q-TOF”, and “Agilent 6550 iFunnel Q-TOF”.
Waters Corporation: “SYNAPT G2-Si HDMS”, “Xevo G2-XS QToF” and “SYNAPT XS”.</p>
<div id="fig:q-tof-diagram" class="fignos">
<figure>
<img src="images/MS_f103.svg" style="width:100.0%" data-tag="10" alt="Figure 10: Schematic diagram of a typical quadrupole time-of-flight mass spectrometer. Like a QQQ, a Q-TOF will have two quadrupoles for selection and fragmentation followed by the TOF for the final higher resolution separation and detection." />
<figcaption aria-hidden="true"><span>Figure 10:</span> <strong>Schematic diagram of a typical quadrupole time-of-flight mass spectrometer.</strong>
Like a QQQ, a Q-TOF will have two quadrupoles for selection and fragmentation followed by the TOF for the final higher resolution separation and detection.</figcaption>
</figure>
</div>
<h3 id="q-orbitrap">3. Q-Orbitrap</h3>
<p>The orbitrap is a critical pillar in the field of proteomics.
In the late 20th century, Russian scientist Alexander Makarov invented the Orbitrap<span class="citation" data-cites="1CdqYFQjZ"><sup><a href="#ref-1CdqYFQjZ" role="doc-biblioref">480</a></sup></span>, which is a novel mass analyzer that operates based on the principle of electrodynamic ion trapping and Fourier Transform.
The orbitrap consists of two main components: an inner spindle-like electrode and a coaxial outer barrel-like electrode (<strong>Figure 11A</strong>).
The ions are trapped in an orbit around the spindle electrode due to the electrostatic attraction.
Once inside, the ions begin oscillating along the central axis of the device, or “orbiting”, due to the electric field formed by the inner and outer electrodes.
The oscillation frequency of an ion is inversely proportional to the square root of its mass-to-charge ratio.
The frequency at which each ion oscillates induces an image current on the detector, which can be measured and transformed into a mass spectrum using Fourier transform.</p>
<p>The biggest difference between Orbitrap and other mass spectrometers (TOF, Q) is that it does not use ions to hit an induction device like an electron multiplier.
One of the main advantages of the Orbitrap is its ultra-high mass resolution, often exceeding 240,000 or even higher.
This gives the Orbitrap a significant superiority in the identification of unknown molecules such as peptides and metabolites<span class="citation" data-cites="5vv7ID3o 1GyycMiIQ"><sup><a href="#ref-5vv7ID3o" role="doc-biblioref">454</a>,<a href="#ref-1GyycMiIQ" role="doc-biblioref">481</a></sup></span>.
Moreover, Orbitrap spectrometers are also appreciated for their compact structure, small size, robustness, and reliability.
Just like the Q-TOF, the Orbitrap is also usually used for tandem mass spectrometry.
It is important to note that many Orbitraps are sold with a linear ion trap as a complementary detector, called a “tribrid” because these models contain a quadrupole, linear ion trap, and an orbitrap.
<strong>Figure 11B</strong> demonstrates a typical 2D schematic diagram of Q-Orbitrap.
Ions first pass through an ion optics module, which consists of a high-capacity ion transfer tube (HCTT), an electrodynamic ion funnel (EDIF), and an advanced active beam guide (AABG).
These are designed to capture ions, reduce ion losses, prevent neutrals and high-velocity clusters from entering the quadrupole, and increase sensitivity.
The ions are then segmented by the quadrupole for precursor ion selection, and the selected ions are trapped by the ion-routing multipole for higher energy collisional dissociation.
Finally, the fragmented ions are captured once again by the C-trap and injected into the Orbitrap batch-by-batch for accurate mass-to-charge analysis.
Overall, this process still follows the logical sequence of precursor ions selection, precursor ions fragmentation, and fragment ions detection.</p>
<p>Compared to a TOF, one disadvantage of the Orbitrap is its longer cycle time (AGC pre-scan, ion injection, ion isolation, ion activation and mass analysis, usually &gt;100ms), which is a negative factor for the currently favored short gradient, high-throughput analysis.
Another minor flaw of Orbitrap is the challenge encountered when trying to pair it with MALDI.
This primarily stems from the fact that MALDI uses a pulsed ionization technique, whereas Orbitrap operates continuously.
This mismatch can lead to inefficiencies and challenges in coupling the two techniques.
At present, Orbitraps are widely used in almost all aspects of proteomics including biomarker discovery<span class="citation" data-cites="KRgCNqM9"><sup><a href="#ref-KRgCNqM9" role="doc-biblioref">482</a></sup></span>, post-translational modification (PTM) analysis<span class="citation" data-cites="m2ZOJt1R 184JFw6LL"><sup><a href="#ref-m2ZOJt1R" role="doc-biblioref">29</a>,<a href="#ref-184JFw6LL" role="doc-biblioref">483</a></sup></span>, quantitative proteomics (LFQ, TMT, iTRAQ)<span class="citation" data-cites="DqzDdsRo 4x97ohIq pRTFSroJ"><sup><a href="#ref-pRTFSroJ" role="doc-biblioref">22</a>,<a href="#ref-DqzDdsRo" role="doc-biblioref">23</a>,<a href="#ref-4x97ohIq" role="doc-biblioref">484</a></sup></span>, protein-protein interaction studies<span class="citation" data-cites="1B5HMmjdK"><sup><a href="#ref-1B5HMmjdK" role="doc-biblioref">485</a></sup></span> and structural proteomics<span class="citation" data-cites="BtlT8wee 1GNpqhnTj"><sup><a href="#ref-BtlT8wee" role="doc-biblioref">486</a>,<a href="#ref-1GNpqhnTj" role="doc-biblioref">487</a></sup></span>.
It can perform both top-down and bottom-up analyses owing to its broad mass range, and is suitable for both Data-Dependent Acquisition (DDA) and Data-Independent Acquisition (DIA) methods.
Right now, the Orbitrap is still under patent protection and only one company, ThermoFisher, is allowed to manufacture related products.
Classic models from ThermoFisher include Orbitrap Astral, Ascend Tribrid, Eclipse™ Tribrid™, Fusion™ Lumos™, Exploris series (120, 240, 480) and Q Exactive™ series.</p>
<div id="fig:QE-diagram" class="fignos">
<figure>
<img src="images/MS_f104.svg" style="width:100.0%" data-tag="11" alt="Figure 11: Schematic diagram of orbitrap. (A) Close up of an Orbitrap. (B) General schematic of complete Q-Orbitrap system." />
<figcaption aria-hidden="true"><span>Figure 11:</span> <strong>Schematic diagram of orbitrap.</strong>
(A) Close up of an Orbitrap.
(B) General schematic of complete Q-Orbitrap system.</figcaption>
</figure>
</div>
<h3 id="quadrupole-fourier-transform-ion-cyclotron-resonance-q-ft-icr">4. Quadrupole Fourier Transform Ion Cyclotron Resonance (Q-FT-ICR)</h3>
<p>The Fourier Transform Ion Cyclotron Resonance (FT-ICR) mass spectrometer is a type of mass spectrometry that uses magnetic fields to separate ions based on their mass-to-charge ratio.
FT-ICR was first invented in 1974 by Alan G. Marshall and Melvin B. Comisarow from the University of British Columbia<span class="citation" data-cites="wknKHPF0"><sup><a href="#ref-wknKHPF0" role="doc-biblioref">488</a></sup></span> and is widely recognized for its high mass resolution and precision, making it a highly valuable tool in many scientific fields including proteomics, metabolomics, petroleum analysis, and environmental science.
The central feature of an FT-ICR mass spectrometer is a superconducting magnet coupled with an ICR cell (<strong>Figure 12A</strong>).
This magnet creates a strong and homogeneous magnetic field in which ions are injected.
Once the ions are inside ICR cell, under the influence of the strong magnetic field, they follow a circular path with a very small orbital radius at a specific frequency directly proportional to their mass-to-charge ratio.
At this point, no detectable image current signal is generated by detector plates located inside the ICR cell.
To improve the signal, a voltage is applied by excitation plates and resonance occurs when the frequency of the strong magnetic field matches the cyclotron frequency of the ions.
The ions absorb radio frequency energy, which increases the radius of their circular path, and consequently, the excited ions move closer to the detector plates and generate a current.
The resulting signal is an oscillating pattern or a time-domain signal.</p>
<p>Similar to Orbitraps, this time-domain signal is then transformed into a frequency-domain signal using Fourier transform, hence the name Fourier Transform ion cyclotron resonance (ICR).
The Fourier transformed data forms a mass spectrum where each peak corresponds to a specific ion present in the sample.
One of the most important advantages of FT-ICR mass spectrometry is its exceptionally high mass resolution and mass accuracy, even for large and complex molecules.
This enables precise identification and characterization of a wide range of compounds in complex mixtures<span class="citation" data-cites="HLEPjTl5 64lSPaQQ"><sup><a href="#ref-HLEPjTl5" role="doc-biblioref">489</a>,<a href="#ref-64lSPaQQ" role="doc-biblioref">490</a></sup></span>.
Moreover, FT-ICR mass spectrometry can be used for multiple stages of mass analysis (MSn), including tandem mass spectrometry (MS/MS), providing detailed information about the structure of ions.
Another significant benefit of FT-ICR is its broad mass range, making it possible to identify macromolecules like proteins for top-down proteomics<span class="citation" data-cites="Gw2v7q21 CwZoo7w"><sup><a href="#ref-CwZoo7w" role="doc-biblioref">21</a>,<a href="#ref-Gw2v7q21" role="doc-biblioref">491</a></sup></span>.</p>
<p>Despite its advantages, FT-ICR mass spectrometry is not without challenges.
The technique requires high-performance superconducting magnets, which are expensive for both initial purchase and further maintenance.
This is because FT-ICR requires liquid nitrogen and liquid helium cooling systems to keep the magnet at a sufficiently low temperature to maintain its superconducting state.
Moreover, the device demands high vacuum conditions and careful temperature control to maintain the stability of the magnetic field and the ion trajectories.
A schematic representation of a Q-FT-ICR system is shown in <strong>Figure 12B</strong>.
In congruence with the tandem mass spectrometers elucidated earlier, ions pass through an array of ion optics modules which designed for ion focusing and purification.
Following this, the ions are selectively filtered by the first quadrupole.
After this filtration, precursor ions undergo fragmentation in the collision cell, which can be a quadrupole, hexapole, or octopole.
The fragmented ions are subsequently re-concentrated by the ensuing focusing lens.
Ultimately, these fragmented ions are trapped, excited, and detected within the ICR cell.
At present, commercial FT-ICR mass spectrometers are available in both Thermo Fisher Scientific (“LTQ FT Ultra” and “LTQ FT Ultra Hybrid” systems) and Bruker Daltonics (“solariX” and “apex” series).</p>
<div id="fig:FT-ICR-diagram" class="fignos">
<figure>
<img src="images/MS_f105.svg" style="width:100.0%" data-tag="12" alt="Figure 12: Schematic of FT-ICR. (A) Typical FT-ICR cell. (B) Example of complete FT-ICR system." />
<figcaption aria-hidden="true"><span>Figure 12:</span> <strong>Schematic of FT-ICR.</strong>
(A) Typical FT-ICR cell.
(B) Example of complete FT-ICR system.</figcaption>
</figure>
</div>
<h3 id="ion-mobility">5. Ion mobility</h3>
<p>In the context of omics research, a fundamental task is the separation, identification, and quantification of molecules in complex mixtures.
Mass spectrometry alone can only provide two-dimensional data including mass-to-charge ratio and their intensity.
Liquid chromatography contributes to the separation of compounds and further provides the third dimension of information, retention time (RT), which make LC-MS evolves as the “golden standard” for proteomic analysis<span class="citation" data-cites="2Qw9ppKa ovFVXSFD"><sup><a href="#ref-2Qw9ppKa" role="doc-biblioref">492</a>,<a href="#ref-ovFVXSFD" role="doc-biblioref">493</a></sup></span>.
Despite the substantial improvements in mass spectrometry resolution and liquid chromatography consistency, accurately identifying extremely similar molecules such as isomers with LC-MS remains a challenge.
Ion mobility mass spectrometry (IM-MS), a technique that utilizes electric fields to transport analytes through a buffer gas, is beneficial for separating and identifying ions based on their size, shape, and charge state.
This technique provides the fourth dimension of information, collision cross section (CCS), which allows for more comprehensive characterization of molecules<span class="citation" data-cites="ODpFT5rK"><sup><a href="#ref-ODpFT5rK" role="doc-biblioref">494</a></sup></span>.
Apparently, multi-dimensional data is always beneficial for us to understand things comprehensively and accurately, thus getting closer to the truth.</p>
<p>In terms of mass spectrometry based proteomic analysis, adding CCS data can help us better separate, identify, and quantify peptides.</p>
<p>The core principle of ion mobility spectrometry is to separate ions in an inert gas under the influence of an electric field (E), and then measure the amount of time it takes for each ion to pass through drift tube, which is defined to be the steady-state drift velocity (Vd) correlated to the specific analyte’s mobility (K), as shown in Eq. 1.
Vd = KE (Eq.1)
While the primary measurement in IMS analyses is the mobility (K), for many analytical applications, it has become routine to convert K into the calculated collision cross-section value (CCS or Ω) using Mason-Schamp equation (Eq. 3)<span class="citation" data-cites="rfF30WBX"><sup><a href="#ref-rfF30WBX" role="doc-biblioref">495</a></sup></span>.<br />
Ω=(3/16 〖(2π/μKT)〗^(1/2) ze)/N0K0 (Eq.2)<br />
The components of the equation are defined as follows: e, charge of an electron; z, ion charge; N0, buffer gas density; μ, reduced mass of the collision partners; kb, Boltzmann’s constant; and T, the drift region temperature.
Although the Mason-Schamp equation isn’t universally embraced, it is currently the primary formula the community uses to compute CCS.
In basic terms, the CCS serves as a standard metric for the size in the gas phase, generally expressed in units of square Angströms (Å2).
However, according to the Eq.2, parameters including gas composition, working pressure, temperature within the mobility region, path of analyte movement, and the strength of the applied field can influence the final CCS value and may differ for each specific IMS platform.
Hence, direct comparison of CCS value between different platforms often requires calibration.</p>
<p>Generally, ion mobility techniques can be categorized into three separation concepts: (1) temporally dispersive, (2) spatially dispersive, and (3) ion confinement (trapping) and selective release (<strong>Figure 13A</strong>)<span class="citation" data-cites="2Qw9ppKa"><sup><a href="#ref-2Qw9ppKa" role="doc-biblioref">492</a></sup></span>.
Temporally dispersive methods produce an arrival time spectrum based on differences in the time it takes for ions to traverse a similar gas-filled drift region under the influence of an electric field.
Time-dispersive technique inherently provides an extensive examination of all signals detected during a given observation window.
However, a fundamental limitation of this wide-ranging analysis is the diminished sensitivity linked to a single time dispersion occurrence, which usually requires many (10−100) events to be aggregated to achieve statistically significant ion mobility measurements.
In contrast, spatially dispersive methods separate ions based on mobility differences (charge, shape and size), leading them on distinct drift paths or trajectories, but without significant time differences.
A characteristic of spatially dispersive techniques is the scanning of voltage to obtain a broad-band ion mobility spectrum.
Types of spatially dispersive ion mobility include High Field Asymmetric Waveform Ion Mobility Spectrometry (FAIMS), uniform-field differential mobility analyzers (DMA), and the newly introduced scanned frequency ion mobility filter called transverse modulation ion mobility spectrometry (TMIMS).
Ion confinement and release strategies are recently developed techniques which trap ions in a pressurized drift cell by electric field, and then release them based on mobility distinctions.
This technique relies on the ability to control the position of ions under elevated pressure conditions using precisely adjustable electrodynamic fields.
It requires a precise fabrication craft and more complicated control system.
While it has only been perfected recently, typical products like trapped ion mobility spectrometry (TIMS)<span class="citation" data-cites="D1WlSrbO 2aWp1Ns2"><sup><a href="#ref-D1WlSrbO" role="doc-biblioref">496</a>,<a href="#ref-2aWp1Ns2" role="doc-biblioref">497</a></sup></span> and cyclic traveling wave IMS have become commercially available<span class="citation" data-cites="Gl9lNiar"><sup><a href="#ref-Gl9lNiar" role="doc-biblioref">498</a></sup></span>.
Table 6 summarizes typical ion mobility separation techniques, their separation concept, electric field direction, gas flow direction, strengths, and drawbacks.
Also, for three categories of ion mobility techniques, we have selected a typical technique from each for brief introduction.</p>
<p>Table 6: Typical ion mobility separation techniques.</p>
<table>
<colgroup>
<col style="width: 12%" />
<col style="width: 12%" />
<col style="width: 12%" />
<col style="width: 12%" />
<col style="width: 12%" />
<col style="width: 37%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Separation concept</th>
<th style="text-align: center;">Ion mobility techniques</th>
<th style="text-align: center;">Ion movement direction</th>
<th style="text-align: center;">Electric field direction</th>
<th style="text-align: center;">Drift Gas direction</th>
<th style="text-align: center;">Characteristics</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Temporally Dispersive</td>
<td style="text-align: center;">drift tube IMS (DTIMS)</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">#</td>
<td style="text-align: center;">High mobility resolution (need long drift tube), direct measurement of CCS. Low speed, large size, low sensitivity,</td>
</tr>
<tr class="even">
<td style="text-align: center;">Temporally Dispersive</td>
<td style="text-align: center;">traveling wave IMS (TWIMS)</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">→→→</td>
<td style="text-align: center;">#</td>
<td style="text-align: center;">High mobility resolution, faster than DTIMS. Low sensitivity, large size, low sensitivity, traveling electric field waves.</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Spatially Dispersive</td>
<td style="text-align: center;">high-field asymmetric IMS (FAIMS)</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">↑↓</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">Good as mass filter, Fast. No CCS measurement (Compensation voltage instead), low mobility resolution.</td>
</tr>
<tr class="even">
<td style="text-align: center;">Spatially Dispersive</td>
<td style="text-align: center;">transverse modulation IMS (TMIMS)</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">→ and ↑↓</td>
<td style="text-align: center;">#</td>
<td style="text-align: center;">Transverse Modulation, compact instrumentation, orthogonal Separation, fast and high resolution</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Confinement and Selective Release</td>
<td style="text-align: center;">trapped ion mobility spectrometry (TIMS)</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">←</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">High mobility resolution, compact instrumentation, high sensitivity, high speed, high ion utilization rate</td>
</tr>
<tr class="even">
<td style="text-align: center;">Confinement and Selective Release</td>
<td style="text-align: center;">multi-pass cyclic traveling wave IMS</td>
<td style="text-align: center;">→</td>
<td style="text-align: center;">→→→</td>
<td style="text-align: center;">#</td>
<td style="text-align: center;">High mobility resolution, improved Signal-to-Noise ratio and sensitivity, versatility (from small molecules to large biomolecules) and adjustability (number of passes can often be adjusted)</td>
</tr>
</tbody>
</table>
<p>‘#’ means stationary drift gas; →, ←, ↑↓ indicates drift gas direction or electric force direction; →→→ represents a wave and gradient electric field.</p>
<div id="fig:IMS-diagram" class="fignos">
<figure>
<img src="images/MS_f106.svg" style="width:100.0%" data-tag="13" alt="Figure 13: Ion Mobility. (A) Conceptional diagram of three types of ion mobility strategies. (B) Schematic of drift tube ion mobility spectrometry. (C) Schematic of high field asymmetric waveform ion mobility spectrometry (FAIMS). (D) Schematic of trapped ion mobility spectrometry (TIMS)." />
<figcaption aria-hidden="true"><span>Figure 13:</span> <strong>Ion Mobility.</strong>
(A) Conceptional diagram of three types of ion mobility strategies.
(B) Schematic of drift tube ion mobility spectrometry.
(C) Schematic of high field asymmetric waveform ion mobility spectrometry (FAIMS).
(D) Schematic of trapped ion mobility spectrometry (TIMS).</figcaption>
</figure>
</div>
<h4 id="drift-tube-ion-mobility-spectrometry-dtims">Drift Tube Ion Mobility Spectrometry (DTIMS)</h4>
<p>The principle of Drift Tube Ion Mobility Spectrometry (DTIMS) is based on the differential migration (time) of ions through a neutral buffer gas (commonly helium or nitrogen) under the influence of a weak uniform electric field (typically tens of V/cm).
The mobility (K) of an ion is proportional to its drift velocity (V) and inversely proportional to the strength of the applied electric field (E).
For ions with same charge states, the drift velocities are primarily determined by their collisional interactions with a buffer gas, namely, mainly affected by their shape and size.
To illustrate this process, imagine two objects with identical mass: a solid metal ball and a feather. Due to its lower density, the feather should have a larger volume than the ball.
When both are dropped from the same height, the solid ball reaches the ground before the feather because of air resistance.
This observation doesn’t contradict Newton’s law of universal gravitation, as we have accounted for air resistance.
In the context of DTIMS, the buffer gas in the drift tube acts as the “air resistance”, while the uniform electric field represents the “gravity”.
Hence, ions with the same mass-to-charge ratio are separated based on their shape and size.
This capability allows DTIMS to distinguish between isomeric compounds with identical masses but different structural configurations, given that these isomers might have distinct interactions with the drift gas.
Also, follow the intuition of the free fall example, in DTIMS, smaller ions will move faster and hit the detector earlier than larger ions in DTIMS (<strong>Figure 13B</strong>).<br />
DTIMS possess the strengths including high resolving power and allows for straightforward measurement of an ion’s CCS from first principles<span class="citation" data-cites="NcqTCZMl 6pMoYbxP"><sup><a href="#ref-NcqTCZMl" role="doc-biblioref">499</a>,<a href="#ref-6pMoYbxP" role="doc-biblioref">500</a></sup></span>.
However, DTIMS also suffers from disadvantages including: 1) separation time is too long for all ions passing through the drift tube, relative to the accumulation time, which decreases the duty cycle.
2) A longer drift tube or higher pressure is needed for greater resolving power.
However, this inevitably increases ion diffusion and ion losses unless ion focusing techniques are employed.
3) Segmentation and collision between ions and gas molecules during the traveling process in drift tube reduces the sensitivity.
Continual advancements in DTIMS design and application of ion focusing techniques further pushed the mobility resolution of same DTIMS platforms to 100 to 250 (t/Δt) range or even greater.</p>
<h4 id="high-field-asymmetric-waveform-ion-mobility-spectrometry-faims">High Field Asymmetric Waveform Ion Mobility Spectrometry (FAIMS)</h4>
<p>High Field Asymmetric Waveform Ion Mobility Spectrometry (FAIMS) represents a distinct version of spatially dispersive ion mobility spectrometry.
This technique differentiates ions utilizing a pronounced asymmetric oscillating electric field combined with a moving gas.
The principle of FAIMS is based on the different trajectories of ions as they move through a high asymmetric electric field, which are determined by their physical structure and charge states<span class="citation" data-cites="KCpIS4fP VdxvCWch E9GVLceZ"><sup><a href="#ref-KCpIS4fP" role="doc-biblioref">501</a>–<a href="#ref-E9GVLceZ" role="doc-biblioref">503</a></sup></span>.
In FAIMS, gas-phase ions are carried by a flow of carrier gas between two electrodes in a direction orthogonal to the direction of asymmetric electric field (E).
The asymmetric waveform electric field is typically characterized by a short, high-voltage pulse of one polarity followed by a longer, lower-voltage pulse of the opposite polarity.
An ion’s mobility within such an electric field is determined by its charge state, its physical structure, and the properties of the surrounding gas it moves through.
Once the ions are subjected to an asymmetric electric field, the ions will alternate between travelling toward one electrode or the other as the field oscillates in polarity, resulting in a curved trajectories between the electrodes.
Some ions move more in the high field relative to the low field, and vice versa (<strong>Figure 13C</strong>).
To differentiate between ions, a so-called “compensation voltage” (CV), which is a DC offset voltage that compensates for the differential ion movement in the high and low fields, is applied<span class="citation" data-cites="EUzlbRbi"><sup><a href="#ref-EUzlbRbi" role="doc-biblioref">504</a></sup></span>.
In this case, only ions with a specific response to the changing electric field and those that match the applied compensation voltage (CV) will have a zero net movement and are able to traverse the drift region to the detector, while others hit the electrode plate and are neutralized.
By scanning or modulating the CV, different ion species can be selectively transmitted through the FAIMS device.
In contrast to drift tube IMS in which the ion stream is sampled in discrete packets and all ions reach the detector, FAIMS is a continuous filtration technique that allows uninterrupted sampling of the ion stream, but only for a selected subset of the ion population.
One of the primary advantages of this continuous collection technique is to greatly increase the signal-to-noise ratio for the ion(s) of interest by removing unwanted chemical noise, which make FAIMS more similar to a m/z filter than other ion mobility spectrometry tools.
FAIMS also has the advantage of operating at atmospheric pressure.
Drawbacks of FAIMS, however, are that it does not produce any CCS values and it has relatively low resolution separations.
Commercial FAIMS products from vendors including Thermo Fisher and Waters are available now.</p>
<h4 id="trapped-ion-mobility-spectrometry-tims">Trapped ion mobility spectrometry (TIMS)</h4>
<p>Trapped ion mobility spectrometry (TIMS) is a common type of ion mobility which uses ion refinement and release strategy<span class="citation" data-cites="CzOaytxB"><sup><a href="#ref-CzOaytxB" role="doc-biblioref">505</a></sup></span>.
The basic idea behind TIMS is a combination of traditional ion mobility spectrometry and ion trapping techniques.
Instead of driving ions through a drift tube filled with stationary gas, TIMS holds the ions stationary in a drift cell under a moving buffer gas and then releases them by adjusting electric fields (voltages on electrodes).
This process was realized by applying two different electric fields as follows.</p>
<ol type="1">
<li>Radially confining pseudopotential.
An RF (radio frequency) voltage is applied to the electrodes of the TIMS analyzer to generate a radially confining pseudopotential, with essentially no axial component; this is only used for focusing ions to the central region of the TIMS tube, preventing them from diffusion into electrodes.</li>
<li>Axial electric field.
An axial electric field gradient, produced by superimposing DC potentials on tunnel electrodes, is applied for “trapping” ions based on the equilibrium between the force of drift gas and the opposing force from the electric field gradient, which is stronger at the entrance and becomes progressively weaker moving deeper into the tunnel.</li>
</ol>
<p>As a result, once ions entered the device, lower mobilities ones are trapped at positions where the magnitude of axially electric field is larger, while higher mobilities ones are confined to deeper positions of tunnel where axially electric field is lower.
Then, after enough ions have been accumulated in the TIMS tunnel, additional ions are prevented from entering the tunnel region and residing ions are trapped for a short time (usually few milliseconds) which can be defined by users.
Finally, the magnitude of axially electric field is decreased at a user defined rate so that ions are eluted as an order of mobilities value (K) from high to low (<strong>Figure 13D</strong>).
The axially electric field gradient is set by a resistor divider.
Importantly, like other ion mobility strategies, the resolving power of TIMS is highly dependent on the length of the gas column through which the ions traverse.
In TIMS, ions are trapped in a specific location while buffer gas continuously flows past them.
Thus, the resolving power achieved by TIMS depends on the “quantity” of gas, specifically the length of the gas column, that passes by the ions during the separation time.
This offers the direct benefit of allowing the analyzer to maintain a compact physical size (around 5 cm) and achieve a high resolving power (R ∼ 300), while the analytical gas column – the portion that flows during an analysis – can be extensive (up to 10 m) and tailored to the user’s needs.
Moreover, by leveraging the “trapping” capability (trapping time) of TIMS and the high scanning speeds of TOF, platforms such as TIMS-Q-TOF can implement a full duty cycle acquisition protocol known as Parallel Accumulation-Serial Fragmentation (PASEF)<span class="citation" data-cites="D1WlSrbO 1FmNlvpSt"><sup><a href="#ref-D1WlSrbO" role="doc-biblioref">496</a>,<a href="#ref-1FmNlvpSt" role="doc-biblioref">506</a></sup></span>.
This is particularly meaningful for identifying more peptides within a given time frame, such as capture more precursors from co-eluted peptides in the same liquid chromatography peak.
Currently, Bruker is the primary provider of commercial mass spectrometers that utilize TIMS-tof technology. (TIMS-tof pro, TIMS-tof pro2, SCP. etc.).</p>
<h4 id="structures-for-lossless-ion-manipulation-slim">10.5.4 Structures for Lossless Ion Manipulation (SLIM)</h4>
<p>A final type of ion mobility spectrometry discussed here is Structures for Lossless Ion Manipulation (SLIM), invented by Richard Smith and colleagues at Pacific Northwest National Labs<span class="citation" data-cites="KgcabL7g"><sup><a href="#ref-KgcabL7g" role="doc-biblioref">507</a></sup></span>.
SLIM uses printed circuit boards to confine ions in long path lengths for high resolution ion mobility.
Ions can be passed through the board multiple times to achieve path lengths of several meters to over 1 km for high-resolution IMS separation<span class="citation" data-cites="XOhnpKc9 XOhnpKc9"><sup><a href="#ref-XOhnpKc9" role="doc-biblioref">508</a>,<a href="#ref-XOhnpKc9" role="doc-biblioref">508</a></sup></span>.
This technology is currently under commercial development by Mobilion, in a platform named “Mobie”<span class="citation" data-cites="GwEAWw2z"><sup><a href="#ref-GwEAWw2z" role="doc-biblioref">509</a></sup></span>.</p>
<h2 class="page_break_before" id="tandem-mass-spectrometry-and-peptide-fragmentation">11. Tandem Mass Spectrometry and Peptide Fragmentation</h2>
<h3 id="tandem-mass-spectrometry">Tandem Mass Spectrometry</h3>
<p>Tandem MS, where precursor ions are selected and fragmented to generate an MS/MS spectrum containing peptide-derived product ions, is a fundamental process in modern proteomics<span class="citation" data-cites="qAVrmvwM V3WGOS1f"><sup><a href="#ref-qAVrmvwM" role="doc-biblioref">510</a>,<a href="#ref-V3WGOS1f" role="doc-biblioref">511</a></sup></span>.
This is largely because intact peptide mass alone cannot unambiguously provide a peptide’s sequence<span class="citation" data-cites="1DSKc93ie"><sup><a href="#ref-1DSKc93ie" role="doc-biblioref">512</a></sup></span>; however, MS/MS spectra provide more information due to predictable fragmentation behavior of peptide ions to generate sequence-informative fragments<span class="citation" data-cites="2MR1Pihs qAVrmvwM"><sup><a href="#ref-qAVrmvwM" role="doc-biblioref">510</a>,<a href="#ref-2MR1Pihs" role="doc-biblioref">513</a></sup></span>.
Some more advanced proteomic acquisition methods use MS1-only feature detection in combination with retention time to maximize information used for downstream quantitation<span class="citation" data-cites="zCNyqvc9"><sup><a href="#ref-zCNyqvc9" role="doc-biblioref">514</a></sup></span>.
In most of these, identifications are fundamentally based on MS/MS spectra, either acquired as part of a specific LC-MS/MS analysis that contains the MS/MS spectra themselves or on a spectral library of MS/MS spectra acquired previously<span class="citation" data-cites="175VGosTe ylZpW7Vi"><sup><a href="#ref-175VGosTe" role="doc-biblioref">515</a>,<a href="#ref-ylZpW7Vi" role="doc-biblioref">516</a></sup></span>.
True MS1-only methods that use only accurate mass and retention time for identification have been discussed, but these have yet to be widely adopted<span class="citation" data-cites="zCNyqvc9"><sup><a href="#ref-zCNyqvc9" role="doc-biblioref">514</a></sup></span>.</p>
<p>The value of MS/MS spectra for peptide identification comes from predictable fragmentation behavior of peptide ions to generate sequence-informative fragments<span class="citation" data-cites="2MR1Pihs qAVrmvwM"><sup><a href="#ref-qAVrmvwM" role="doc-biblioref">510</a>,<a href="#ref-2MR1Pihs" role="doc-biblioref">513</a></sup></span>.
Multiple dissociation methods exist to generate product ions in MS/MS spectra through various mechanisms (<strong>Figure 14</strong>).
In non-modified peptides, the most labile bonds are typically peptide bonds (i.e., amide bonds) between amino acids.
Depending on where peptides dissociate along the peptide backbone, the fragments are assigned different ion types (<strong>Figure 14A</strong>).
Fragment ion nomenclature was first developed by Roepstorff and Fohlman in 1984<span class="citation" data-cites="mZdxW2jk"><sup><a href="#ref-mZdxW2jk" role="doc-biblioref">517</a></sup></span> and then refined by Biemann in 1990<span class="citation" data-cites="ivBdKZgo"><sup><a href="#ref-ivBdKZgo" role="doc-biblioref">518</a></sup></span>.
The main ion types are the fragments that contain the original peptide N-terminus (i.e., a-, b-, and c-type ions), or the original peptide C-terminus (i.e., x-, y-, and z-type ions).
The number associated with each fragment ion indicates how many amino acids from each terminus are included.</p>
<div id="fig:SPE" class="fignos">
<figure>
<img src="images/Tandem_MS.svg" style="width:100.0%" data-tag="14" alt="Figure 14: Peptide Fragmentation Methods. (A) Sequence-informative fragment ions are termed a/x-, b/y-, and c/z-type fragments depending on which bond along the peptide backbone breaks. Fragments that explain the intact N-terminus of the peptide are a-, b-, and c-type ions, while x-, y-, and z-type ions explain the intact C-terminus of the peptide. Other panels show common dissociation methods, including collision, electron, and photon-based fragmentation. (B) Resonant collision-induced dissociation (resCID) and beam-type CID (beamCID) both produce mainly b/y-type sequencing ions through collisions with background gases like helium and nitrogen that increase the internal energy of peptide cations. (C) Electron capture and electron transfer dissociation (ECD and ETD) generate mainly c/z-type fragments through electron-mediated radical driven cleavage of the peptide backbone. (D) Infrared multi-photon dissociation (IRMPD) is a slow heating method similar in dissociation mechanism to resCID, but very different in implementation due to the IR lasers required (often with lower energy 10.6 micron photons). Ultraviolet photodissociation (UVPD) can use a range of wavelengths (popular options shown) to introduce higher energy photons to peptide cations, causing vibrational and electronic excitation that can generate all major fragment ion types depending on wavelength used." />
<figcaption aria-hidden="true"><span>Figure 14:</span> <strong>Peptide Fragmentation Methods.</strong>
(A) Sequence-informative fragment ions are termed a/x-, b/y-, and c/z-type fragments depending on which bond along the peptide backbone breaks.
Fragments that explain the intact N-terminus of the peptide are a-, b-, and c-type ions, while x-, y-, and z-type ions explain the intact C-terminus of the peptide.
Other panels show common dissociation methods, including collision, electron, and photon-based fragmentation.
(B) Resonant collision-induced dissociation (resCID) and beam-type CID (beamCID) both produce mainly b/y-type sequencing ions through collisions with background gases like helium and nitrogen that increase the internal energy of peptide cations.
(C) Electron capture and electron transfer dissociation (ECD and ETD) generate mainly c/z-type fragments through electron-mediated radical driven cleavage of the peptide backbone.
(D) Infrared multi-photon dissociation (IRMPD) is a slow heating method similar in dissociation mechanism to resCID, but very different in implementation due to the IR lasers required (often with lower energy 10.6 micron photons).
Ultraviolet photodissociation (UVPD) can use a range of wavelengths (popular options shown) to introduce higher energy photons to peptide cations, causing vibrational and electronic excitation that can generate all major fragment ion types depending on wavelength used.</figcaption>
</figure>
</div>
<p>One of the earliest and most ubiquitous peptide fragmentation methods is collision-induced dissociation (CID, also called collisionally-activated dissociation, CAD)<span class="citation" data-cites="EmIJDW71"><sup><a href="#ref-EmIJDW71" role="doc-biblioref">459</a></sup></span> (<strong>Figure 14B</strong>).
Here, collisions with inert gas molecules are used to increase the internal energy of peptide ions to reach bond dissociation energies that fragment them into products.
Various inert gases can be used; helium, nitrogen, and argon are the most common.
Preferences for which gas is used is often a function of how much energy per collision is desired.
Two main versions of CID are used in proteomics, with the most common being beam-type CID (beamCID, sometimes called higher-energy collisional dissociation, HCD)<span class="citation" data-cites="12yIRpFcQ 1DIcKpBMl"><sup><a href="#ref-12yIRpFcQ" role="doc-biblioref">519</a>,<a href="#ref-1DIcKpBMl" role="doc-biblioref">520</a></sup></span>.
BeamCID typically uses nitrogen or argon as a collision gas, and peptide ions are accelerated into a collision cell filled with several mTorr of bath gas.
The kinetic energy used to accelerate precursor ions (often generated using direct current voltage differentials between the source of the ions and the collision cell) determines the energy imparted through collisions with the bath gas, which in turn governs their fragmentation behavior.</p>
<p>Since in non-modified peptides the most labile bonds are typically peptide bonds (i.e., amide bonds) between amino acids, the increase in internal energy from beamCID generates b- and y-type ions that represent this peptide bond cleavage, as shown in Biemann fragment ion nomenclature (<strong>Figure 14A</strong>).
b-type ions provide sequence information for fragments that have an intact N-terminus, while y-type ions denote fragment ions with an intact C-terminus.
Collisions in beamCID cause near instantaneous generation of primary fragment ions.
Because the increase in internal energy happens rapidly before energy can be redistributed, beamCID can generate fragments that are not necessarily derived from cleavage of the most labile bonds (e.g., PTM-modified peptides, discussed below), but spectra are often dominated by b/y-type ions from amide bond cleavage (<strong>Figure 14B</strong>).
BeamCID can also generate secondary fragments, such as immonium ions from side chain losses<span class="citation" data-cites="CrVLd8Zt"><sup><a href="#ref-CrVLd8Zt" role="doc-biblioref">521</a></sup></span> or a-type fragment ions that come from water loss from b-type ions due to multiple collision events (note: a-type ions can form as primary fragmentation products in other dissociation methods).
The simplicity of beamCID, which simply requires an rf-only collision cell, has made it widely implemented on most instrument platforms used in modern proteomics.</p>
<p>A second form of CID is called resonant CID (resCID), where the internal energy of peptide ions is slowly increased through multiple low-energy collisions.
Here, helium gas is most often used, as it imparts less energy per collision, and activation typically happens in ion trap devices where supplemental frequencies can be used to excite ions.
In other words, ions are trapped using axial rf-frequencies, and an additional rf-frequency is applied to the electrodes of the ion trap<span class="citation" data-cites="Lx0XDSc1"><sup><a href="#ref-Lx0XDSc1" role="doc-biblioref">522</a></sup></span>.
This supplemental rf is selected to have a frequency resonant with the fundamental frequency of the ions to be fragmented, as determined by the Mathieu equations, which excites the ions of interest so that they have increased kinetic energy as they move in the ion trap<span class="citation" data-cites="8fRsJhJf 1DjnN7u2e"><sup><a href="#ref-8fRsJhJf" role="doc-biblioref">523</a>,<a href="#ref-1DjnN7u2e" role="doc-biblioref">524</a></sup></span>.
The increased kinetic energy creates more collisions with the background helium gas to slowly build up the internal energy of the precursor ions until the dissociation energy of the most labile bond is reached, causing fragmentation.
Once ions dissociate, the fragments have different <em>m/z</em> values than the precursor ions, meaning they fall out of resonance with the supplemental rf and are no longer activated.
Thus, resCID typically fragments only the most labile bonds in precursor ions and does not have secondary fragmentation behavior.
As above, for non-modified peptide ions, this typically generates sequence-informative b- and y-type product ions.
For modified peptides where the bonds connecting the modification to an amino acid are more labile than peptide bonds (e.g., phosphopeptides and glycopeptides), resCID MS/MS spectra can be dominated by product ions only of the PTM-loss rather than sequence-informative fragment ions, although many factors govern this behavior<span class="citation" data-cites="16Wowt3CO Oadeotpz"><sup><a href="#ref-16Wowt3CO" role="doc-biblioref">525</a>,<a href="#ref-Oadeotpz" role="doc-biblioref">526</a></sup></span>.
Because of this, and because this method requires an ion trap device with the ability to apply supplemental rfs, resCID is less prevalent than beamCID.
For both beamCID and resCID, the mobile proton model has been widely accepted to explain fragmentation behavior<span class="citation" data-cites="Ud7cg5Ze"><sup><a href="#ref-Ud7cg5Ze" role="doc-biblioref">527</a></sup></span>, and this largely predictable behavior has greatly helped in manual and algorithm-assisted spectral interpretation.</p>
<p>Despite the utility and broad adoption of CID, alternative dissociation methods have been explored for a variety of uses, including applications where CID is inadequate for the experimental question<span class="citation" data-cites="VOcnSdY9 oNkSkkq8 XHvsh4uo"><sup><a href="#ref-VOcnSdY9" role="doc-biblioref">528</a>–<a href="#ref-XHvsh4uo" role="doc-biblioref">530</a></sup></span>.
The most popular of these alternative dissociation methods are electron-based dissociation (ExD) approaches, which include electron capture dissociation (ECD) and electron transfer dissociation (ETD).
In both of these, peptide cations capture thermal electrons (ECD<span class="citation" data-cites="qWde4x"><sup><a href="#ref-qWde4x" role="doc-biblioref">531</a></sup></span>) or abstract an electron from a reagent anion (ETD<span class="citation" data-cites="GH22tfkO"><sup><a href="#ref-GH22tfkO" role="doc-biblioref">532</a></sup></span>) to generate radical-driven dissociation of the N-Ca bond that predominantly generates sequence-informative c- and z-type product ions (<strong>Figure 14C</strong>).
The mechanisms of ExD methods have been widely explored<span class="citation" data-cites="OVsIcoNm xhXJM8vz"><sup><a href="#ref-OVsIcoNm" role="doc-biblioref">533</a>,<a href="#ref-xhXJM8vz" role="doc-biblioref">534</a></sup></span>, and the preferential cleavage of N-Ca bonds along the peptide backbone have been particularly useful for PTM-modified species because the modifications remain largely intact even during peptide backbone bond fragmentation.
ExD methods have shown promise for analysis of numerous PTMs, including phosphorylation, glycosylation, ADP-ribosylation, and more<span class="citation" data-cites="rqc6ScJr HNURrSN7"><sup><a href="#ref-rqc6ScJr" role="doc-biblioref">535</a>,<a href="#ref-HNURrSN7" role="doc-biblioref">536</a></sup></span>.
Electron-based dissociation is also more suitable than collision-based dissociation for MS analyses of intact proteins<span class="citation" data-cites="W7m8UNum BG48TsdY"><sup><a href="#ref-W7m8UNum" role="doc-biblioref">537</a>,<a href="#ref-BG48TsdY" role="doc-biblioref">538</a></sup></span> and larger oligonucleotides<span class="citation" data-cites="ekbZzMCZ 16lajpfJg kcIkQ9U2 dRBXpy4q aQmCzpPg o0OmLG1e"><sup><a href="#ref-ekbZzMCZ" role="doc-biblioref">539</a>–<a href="#ref-o0OmLG1e" role="doc-biblioref">544</a></sup></span></p>
<p>Two fundamental challenges exist with ExD methods.
First, ExD implementation requires instruments that can manipulate cations and anions (or free electrons) within the same scan sequence and can trap both simultaneously for electron capture/transfer events to occur.
This has been successfully accomplished on a number of instruments, including FT-ICR systems, ion traps, ToFs with quadrupole ion traps, and hybrid Orbitrap instruments, but it is not a ubiquitous feature of all platforms.
That said, several exciting advances in recent years have made ExD methods more accessible on numerous instrument configurations<span class="citation" data-cites="rqc6ScJr HNURrSN7 UqS9JT46 1Gs2JKxdE Vu5rNLFc"><sup><a href="#ref-rqc6ScJr" role="doc-biblioref">535</a>,<a href="#ref-HNURrSN7" role="doc-biblioref">536</a>,<a href="#ref-UqS9JT46" role="doc-biblioref">545</a>–<a href="#ref-Vu5rNLFc" role="doc-biblioref">547</a></sup></span>.
A second challenge is the dependence of ExD dissociation efficiency on precursor ion charge density<span class="citation" data-cites="eSdFYuAj"><sup><a href="#ref-eSdFYuAj" role="doc-biblioref">548</a></sup></span>.
ExD methods generally produce robust fragmentation for charge dense precursor ions (i.e., those with relatively low <em>m/z</em> values and higher z).
Alternatively, precursors with low charge density (i.e., higher <em>m/z</em> values) have relatively condensed secondary gas-phase structure that leads to non-covalent interactions.
Even in the cases when ExD methods drive peptide backbone cleavage, product ions (i.e., c- and z-type fragments) are held together by the non-covalent interactions so that few (or no) sequence-informative product ions are produced.
This process is called non-dissociative electron-capture/transfer (ECnoD/ETnoD)<span class="citation" data-cites="qyjSSKs7"><sup><a href="#ref-qyjSSKs7" role="doc-biblioref">549</a></sup></span>.
Several strategies to mitigate ECnoD/ETnoD have been successfully explored, including supplemental activation of product ions with resCID (ETcaD<span class="citation" data-cites="c8aJxeOj"><sup><a href="#ref-c8aJxeOj" role="doc-biblioref">550</a></sup></span>) or beamCID (EThcD<span class="citation" data-cites="TEQyYNF9 bsAhaObV"><sup><a href="#ref-TEQyYNF9" role="doc-biblioref">551</a>,<a href="#ref-bsAhaObV" role="doc-biblioref">552</a></sup></span>), supplemental activation with infrared photons (AI-ECD<span class="citation" data-cites="oRY0m4xv F58LowfZ"><sup><a href="#ref-oRY0m4xv" role="doc-biblioref">553</a>,<a href="#ref-F58LowfZ" role="doc-biblioref">554</a></sup></span> and AI-ETD<span class="citation" data-cites="3mhY8AZ1 pXrOdleW 1HZ2A5iku PWUQDl3j"><sup><a href="#ref-3mhY8AZ1" role="doc-biblioref">555</a>–<a href="#ref-PWUQDl3j" role="doc-biblioref">558</a></sup></span>) or ultraviolet photons (ETuvPD<span class="citation" data-cites="1CD0nWWIP"><sup><a href="#ref-1CD0nWWIP" role="doc-biblioref">559</a></sup></span>), and use of higher energy electrons<span class="citation" data-cites="a9481fQj 3Q6WdA4H Vu5rNLFc"><sup><a href="#ref-Vu5rNLFc" role="doc-biblioref">547</a>,<a href="#ref-a9481fQj" role="doc-biblioref">560</a>,<a href="#ref-3Q6WdA4H" role="doc-biblioref">561</a></sup></span>.
Despite their successes, these methods still require instrumentation capable of ExD in addition to extra hardware needed for a given strategy (e.g., a CO<sub>2</sub> laser in AI-ETD<span class="citation" data-cites="1zm3p02B"><sup><a href="#ref-1zm3p02B" role="doc-biblioref">562</a></sup></span>).
As with ExD in general, recent advances in supplemental activation strategies for ExD are making these tools more accessible<span class="citation" data-cites="rqc6ScJr HNURrSN7"><sup><a href="#ref-rqc6ScJr" role="doc-biblioref">535</a>,<a href="#ref-HNURrSN7" role="doc-biblioref">536</a></sup></span>.</p>
<p>Photoactivation is another family of alternative dissociation strategies that has been steadily gaining popularity<span class="citation" data-cites="M8hC7nxG pP0K4nse"><sup><a href="#ref-M8hC7nxG" role="doc-biblioref">563</a>,<a href="#ref-pP0K4nse" role="doc-biblioref">564</a></sup></span>.
Infrared multi-photon dissociation (IRMPD) is canonically the photodissociation method used in early proteomic applications<span class="citation" data-cites="pP0K4nse"><sup><a href="#ref-pP0K4nse" role="doc-biblioref">564</a></sup></span>, but ultraviolet photodissociation (UVPD) has been the more widely used approach in the recent decade<span class="citation" data-cites="YqEIuvCO"><sup><a href="#ref-YqEIuvCO" role="doc-biblioref">565</a></sup></span>.
IRMPD functions similarly to resCID; it is a slow heating approach that causes vibrational excitation due to absorption of low energy photons, generally 10.6 μm photons from a CO<sub>2</sub> laser<span class="citation" data-cites="zA5rBLH4 XXMVxnKl"><sup><a href="#ref-zA5rBLH4" role="doc-biblioref">566</a>,<a href="#ref-XXMVxnKl" role="doc-biblioref">567</a></sup></span>.
Predominant fragments are b- and y-type fragments, although secondary fragmentation occurs because fragment ions remain in the photon path after the initial dissociation event (<strong>Figure 14D</strong>).
Despite limited use in the past decade, recent work shows that IRMPD, or more generally activation with IR photons, may still have value in the proteomics toolkit<span class="citation" data-cites="1EjowCoSS 1HZ2A5iku"><sup><a href="#ref-1EjowCoSS" role="doc-biblioref">279</a>,<a href="#ref-1HZ2A5iku" role="doc-biblioref">557</a></sup></span>.
UVPD has been explored with a number of wavelengths, including 157 nm, 193 nm, 213 nm, 266 nm, and 355 nm<span class="citation" data-cites="UC6BHa63 145wzrCEm UoKcvT55 168MaNzTm UrNcqFO2 cZOW0Gem"><sup><a href="#ref-UC6BHa63" role="doc-biblioref">568</a>–<a href="#ref-cZOW0Gem" role="doc-biblioref">573</a></sup></span>.
Higher-energy UVPD approaches, like 193 and 213 nm photons, are typically used for underivatized peptide and protein ions<span class="citation" data-cites="YqEIuvCO"><sup><a href="#ref-YqEIuvCO" role="doc-biblioref">565</a></sup></span>, while others, like 266 and 355 nm, can be used for directed fragmentation at specific residues with natural chromophores (e.g., tyrosine) or exogenously added chromophore tags<span class="citation" data-cites="1ACL74GmA 18sf2lBbl"><sup><a href="#ref-1ACL74GmA" role="doc-biblioref">574</a>,<a href="#ref-18sf2lBbl" role="doc-biblioref">575</a></sup></span>.
UVPD with 193 and 213 nm generate multiple fragment types, including sequence-informative a-, b-, c-, x-, y-, and z-ions in addition to other fragmentation pathways, which occur through vibrational and electronic excitation<span class="citation" data-cites="5SkV76Q3"><sup><a href="#ref-5SkV76Q3" role="doc-biblioref">576</a></sup></span>.
UVPD has been explored for bottom-up proteomic applications, but its more impactful utility, arguably, has been realized for intact protein characterization<span class="citation" data-cites="10TkrAfIp"><sup><a href="#ref-10TkrAfIp" role="doc-biblioref">577</a></sup></span>.
The laser needed for UVPD (i.e., the photon wavelength desired) determines much about its implementation.
193 nm photons are typically generated using an Excimer laser with ArF gas<span class="citation" data-cites="KWENOLlt"><sup><a href="#ref-KWENOLlt" role="doc-biblioref">578</a></sup></span>, while 213 nm photons can be generated with a solid-state laser that is easier to integrate into an instrument platform and maintain<span class="citation" data-cites="1A6jk9UQ3 UoKcvT55"><sup><a href="#ref-UoKcvT55" role="doc-biblioref">570</a>,<a href="#ref-1A6jk9UQ3" role="doc-biblioref">579</a></sup></span>.
That said, 213 nm photons tend to provide more directed, preferential cleavage pathways compared to 193 nm photons that cleave more broadly in non-directed fashion<span class="citation" data-cites="zJsJYkGj"><sup><a href="#ref-zJsJYkGj" role="doc-biblioref">580</a></sup></span>.
Outside of ExD and photoactivation approaches, other alternative dissociation methods have been explored for various proteomic applications, although they are not as widely adopted at ExD and UVPD methods<span class="citation" data-cites="M8hC7nxG"><sup><a href="#ref-M8hC7nxG" role="doc-biblioref">563</a></sup></span>.</p>
<h2 class="page_break_before" id="data-acquisition">12. Data Acquisition</h2>
<p>Hybrid mass spectrometers used for modern proteome analysis offer the flexibility to collect data in many different ways.
Data acquisition strategies differ in the sequence of precursor scans and fragment ion scans, and in how analytes are chosen for MS/MS.
Constant innovation to develop better data collection methods improves our view of the proteome, but many method options may confuse newcomers.
This section provides an overview of the general classes of data collection methods.</p>
<p>Data acquisition strategies for proteomics fall into one of two groups.</p>
<ol type="1">
<li>Data dependent acquisition (DDA), in which the exact scan sequence in each analysis depends on the data that the mass spectrometer observes.</li>
<li>Data independent acquisition (DIA), in which the exact scan sequence in each analysis DOES NOT depend on the data; the collected scans are the same whether you inject yeast peptides, human peptides, or a solvent blank.</li>
</ol>
<p>DDA and DIA can both be further subdivided in to targeted and untargeted methods.</p>
<h3 id="dda">DDA</h3>
<p>In most cases, the peptide masses that will be observed are not known before doing the experiment.
Data collection methods must account for this.
DDA was invented in the early 1990s, which enabled collecting MS/MS spectra for observed peptides as they eluting from the LC column<span class="citation" data-cites="JpKe7ymh byV7q4rP 1BU9KEnwO"><sup><a href="#ref-JpKe7ymh" role="doc-biblioref">581</a>–<a href="#ref-1BU9KEnwO" role="doc-biblioref">583</a></sup></span>.</p>
<h4 id="untargeted-dda">Untargeted DDA</h4>
<p>A common method currently used in modern proteomics is untargeted DDA.
The MS collects precursor (MS1) scans iteratively until precursor mass envelopes meeting certain criteria are detected.
Criteria for selection are usually specific charge states and a minimum signal intensity.
When those ions meet these criteria, the MS selects those masses for fragmentation.</p>
<p>Because ions are selected as they are observed, repeated DDA of the same sample will produce a different set of identifications.
This stochasticity is the main drawback of DDA.</p>
<p>Because DDA is required for quantification of proteins using isobaric tags like TMT, this stochasticity of DDA limits the ability to compare quantities across batches.
For example, if you have 30 samples, you can use two sets of the 16-plex kit to label 15 samples in each set with one channel labeled by a pooled sample to enable comparison across the groups.
When you collect DDA data from each of those sets, each set will have MS/MS data from an overlapping but different set of peptides.
If one set has MS/MS from a peptide but the other set does not, then that peptide cannot be quantified in the whole sample group.
This limits the number of quantified proteins in large TMT experiments with multiple batches.
<!-- mention intelligent DDA and real time search? --></p>
<h4 id="targeted-dda">Targeted DDA</h4>
<p>Targeted DDA is not common in modern proteomics.
In targeted DDA, in addition to general criteria like a minimum intensity and a certain charge state, the mass spectrometer looks for specific masses.
These masses might be previously observed signals that were previously missed by MS/MS<span class="citation" data-cites="2D6WnK0w yqW0xFtN"><sup><a href="#ref-2D6WnK0w" role="doc-biblioref">584</a>,<a href="#ref-yqW0xFtN" role="doc-biblioref">585</a></sup></span>.
In these studies, the sample is first analyzed by LC-MS to detect precursor ion features with some software, and then subsequent analyses target those masses for fragmentation with inclusion lists until they are all fragmented.
This was shown to increase proteome coverage.</p>
<h3 id="dia">DIA</h3>
<p>The simplest method to operate a mass spectrometer is to have predefined scans that are collected for each sample analysis.
This is data-independent acquisition (DIA); the scan sequence does not depend on the data that the instrument observes.
Thus, the scan sequence is repetitive, looping through predetermined scans, most often which are <em>m/z</em> quadrupole selection ranges followed by fragmentation in a second quadrupole and fragment ion detection in a final MS stage.
In DIA, the same scan sequence is performed if we inject air, a blank, peptides, ammonia, or anything.
Like DDA, DIA can also be either targeted or untargeted<span class="citation" data-cites="wAKbWLXc"><sup><a href="#ref-wAKbWLXc" role="doc-biblioref">586</a></sup></span>.
The two targeted DIA methods are SRM/MRM or PRM.
Untargeted DIA (uDIA) is often referred to simply as “DIA” or “SWATH” (Sequential Window Acquisition of All Theoretical Mass Spectra) (<strong>Figure 15</strong>).</p>
<div id="fig:DIA-types" class="fignos">
<figure>
<img src="images/data-acquisition-DIA-types.svg" style="width:100.0%" data-tag="15" alt="Figure 15: Types of DIA. A) SRM/MRM. Peptides are ionized by ESI and although there are many peptides entering the mass spectrometer at any time, the first quadrupole (Q1) isolates one mass, which is then fragmented by HCD. Fragment masses from the peptide are then selected in the third quadrupole (Q3). This leads to very low noise and high sensitivity. B) PRM. Like MRM, peptides are selected in the first quadrupole, but this analysis is done on a high-resolution instrument like an Orbitrap or TOF. Selectivity is gained by exploiting the high mass accuracy and resolution to monitor multiple fragment ions. C) uDIA/SWATH. Like MRM and PRM, peptides are isolated with Q1, but in this case a much wider isolation window is used. This usually results in co-isolation of many peptides simultaneously. Fragments from many peptides are measured with high resolution and high mass accuracy. Special software is used to get peptide identities and quantities from the fragment ions." />
<figcaption aria-hidden="true"><span>Figure 15:</span> <strong>Types of DIA.</strong>
A) SRM/MRM.
Peptides are ionized by ESI and although there are many peptides entering the mass spectrometer at any time, the first quadrupole (Q1) isolates one mass, which is then fragmented by HCD.
Fragment masses from the peptide are then selected in the third quadrupole (Q3).
This leads to very low noise and high sensitivity.
B) PRM.
Like MRM, peptides are selected in the first quadrupole, but this analysis is done on a high-resolution instrument like an Orbitrap or TOF.
Selectivity is gained by exploiting the high mass accuracy and resolution to monitor multiple fragment ions.
C) uDIA/SWATH.
Like MRM and PRM, peptides are isolated with Q1, but in this case a much wider isolation window is used.
This usually results in co-isolation of many peptides simultaneously.
Fragments from many peptides are measured with high resolution and high mass accuracy.
Special software is used to get peptide identities and quantities from the fragment ions.</figcaption>
</figure>
</div>
<h4 id="targeted-dia">Targeted DIA</h4>
<p>The first type of targeted DIA is called SRM or MRM<span class="citation" data-cites="NQj9Wees"><sup><a href="#ref-NQj9Wees" role="doc-biblioref">587</a></sup></span>.
The popularity of this method in the literature peaked in 2014, with just under 1,500 documents on PubMed that year resulting from a search for “MRM”.
In this strategy, the QQQ MS is set so that the first quadrupole selects the precursor mass of the peptide(s) of interest, the second quadrupole fragments the peptide, and the third quadrupole monitors the product of specific fragments from that peptide.
This strategy is very sensitive and has the benefit of very low noise.
The fragments monitored in Q3 are chosen such that it is unlikely these fragments could arise from another peptide.
Usually at least a few transitions are monitored for each peptide in order to get multiple measures for that peptide.</p>
<p>An early example of MRM applied to quantify c-reactive protein was in 2004<span class="citation" data-cites="1FAMk6Qey"><sup><a href="#ref-1FAMk6Qey" role="doc-biblioref">588</a></sup></span>.
Around the same time, SRM was combined with antibody enrichment of peptides from target proteins<span class="citation" data-cites="63Pqc9VE"><sup><a href="#ref-63Pqc9VE" role="doc-biblioref">589</a></sup></span>.
This approach was popular for analysis of plasma proteins<span class="citation" data-cites="1B9P7OPqR"><sup><a href="#ref-1B9P7OPqR" role="doc-biblioref">464</a></sup></span>.
These early examples led to many more studies that used QQQ MS instruments to get accurate quantitation of many proteins in one injection<span class="citation" data-cites="iJJwYdQ1 ba7F2Epn"><sup><a href="#ref-iJJwYdQ1" role="doc-biblioref">590</a>,<a href="#ref-ba7F2Epn" role="doc-biblioref">591</a></sup></span>.
Scheduling MRM measurement when chromatography is stable additionally enabled better utilization of instrument duty cycle and therefore monitoring of more peptides per injection<span class="citation" data-cites="jcu6ZT0x"><sup><a href="#ref-jcu6ZT0x" role="doc-biblioref">592</a></sup></span>.
Efforts even developed libraries of transitions that allow quantification of any protein in model organisms<span class="citation" data-cites="eFcH0xbv"><sup><a href="#ref-eFcH0xbv" role="doc-biblioref">593</a></sup></span>.</p>
<p>Another similar targeted DIA method is called parallel reaction monitoring (PRM)<span class="citation" data-cites="1Bt8NAhNo"><sup><a href="#ref-1Bt8NAhNo" role="doc-biblioref">594</a></sup></span>.
Instead of using a QQQ instrument, PRM uses a hybrid MS with a quadrupole and a high-resolution mass analyzer, such as an Q-TOF or Q-Exactive.
The idea is that instead of monitoring specific fragments in Q3, the high mass accuracy can be used to filter peptide fragments for high selectivity and accurate quantification.
Studies have found that PRM and MRM/SRM have comparable dynamic range and linearity<span class="citation" data-cites="HB3XvRF2"><sup><a href="#ref-HB3XvRF2" role="doc-biblioref">595</a></sup></span>.</p>
<h4 id="untargeted-dia">Untargeted DIA</h4>
<p>There were many implementations of uDIA over the years, starting in 2003 by Purvine et al from the Goodlett lab<span class="citation" data-cites="dWzeUq6d"><sup><a href="#ref-dWzeUq6d" role="doc-biblioref">596</a></sup></span>.
In this first work they demonstrated uDIA using a Q-TOF with in source fragmentation and showed that extracted ion chromatograms of precursor and fragment ions matched in shape suggesting that this could be used to identify and quantify peptides.
The following year, Venable et al from the Yates lab introduced uDIA with an ion trap<span class="citation" data-cites="yn40ByaQ"><sup><a href="#ref-yn40ByaQ" role="doc-biblioref">597</a></sup></span>.
Subsequent methods include MSE<span class="citation" data-cites="Z3Hfnak1"><sup><a href="#ref-Z3Hfnak1" role="doc-biblioref">598</a></sup></span>, PAcIFIC<span class="citation" data-cites="Wb9egjJg"><sup><a href="#ref-Wb9egjJg" role="doc-biblioref">599</a></sup></span>, all ions fragmentation (AIF)<span class="citation" data-cites="OubW7j0g"><sup><a href="#ref-OubW7j0g" role="doc-biblioref">600</a></sup></span>.
Computational methods were also developed to automate interpretation of this data, such as DeMux<span class="citation" data-cites="OubW7j0g"><sup><a href="#ref-OubW7j0g" role="doc-biblioref">600</a></sup></span>, XDIA<span class="citation" data-cites="y71ZDsTJ"><sup><a href="#ref-y71ZDsTJ" role="doc-biblioref">601</a></sup></span>, and ETISEQ<span class="citation" data-cites="ckj0XYOM"><sup><a href="#ref-ckj0XYOM" role="doc-biblioref">602</a></sup></span>.</p>
<p>The paper that is often cited for uDIA that led to widespread adoption was by Gillet et al. from the Aebersold group in 2012<span class="citation" data-cites="11ccXv5xm"><sup><a href="#ref-11ccXv5xm" role="doc-biblioref">603</a></sup></span>.
In this paper they branded the idea as SWATH.
Widespread adoption may have been facilitated by the co-marketing of this idea by ABSciex as a proteomics solution on their new 5600 Q-TOF (called “tripleTOF” despite containing only one TOF, likely a portmanteau of “triple quadrupole” and “Q-TOF”).
Importantly, in the Gillet et al. paper the authors described a computational method to extract information from SWATH where peptides of interest were queried against the data.
They also demonstrated the application of SWATH to measure proteomic changes that happen in diauxic shift, and showed that SWATH can reveal modified peptides, in this case a methionine oxidation.</p>
<p>There are also many papers describing uDIA with orbitraps.
One early example described combining random isolation windows together and then demultiplexing the chimeric spectra<span class="citation" data-cites="1GCjdtGKY"><sup><a href="#ref-1GCjdtGKY" role="doc-biblioref">604</a></sup></span>.
In another landmark paper, over 6,000 proteins were identified from mouse tissue by at least 2 peptides<span class="citation" data-cites="1GvzNjeV3"><sup><a href="#ref-1GvzNjeV3" role="doc-biblioref">605</a></sup></span>.
In 2018, the new model orbitrap at that time (HF-X) enabled identification of nearly 6,000 human proteins in only 30 minutes.
Currently orbitraps have all but replaced the Sciex Q-TOFs for DIA data collection.</p>
<p>A new direction in uDIA is the addition of ion separation by ion mobility.
This has appeared in two forms.
On the timsTOF, diaPASEF makes use of the trapped ion mobility to increase speed and sensitivity of analysis<span class="citation" data-cites="YKRUqViz"><sup><a href="#ref-YKRUqViz" role="doc-biblioref">606</a></sup></span>.
On the orbitrap, the combination of FAIMS and DIA has enabled the identification of over 10,000 proteins from one sample, which is a major milestone<span class="citation" data-cites="44oI1g68"><sup><a href="#ref-44oI1g68" role="doc-biblioref">434</a></sup></span>.</p>
<h4 id="acquisition-methods-for-ptms">Acquisition methods for PTMs</h4>
<h5 id="phosphopeptides">Phosphopeptides</h5>
<p>Resonant CID<span class="citation" data-cites="h7UB5vUH"><sup><a href="#ref-h7UB5vUH" role="doc-biblioref">607</a></sup></span> and beam-type HCD<span class="citation" data-cites="C0Uvh6dQ"><sup><a href="#ref-C0Uvh6dQ" role="doc-biblioref">608</a></sup></span> are the most popular methods for unmodified and modified peptides due to their speed, accessibility, and efficiency.
Due to the weak phosphoester bond relative to the peptide backbone, resonant CID usually produces spectra that are dominated by only the neutral loss of the phosphate.
For this reason, the optimal dissociation methods for phosphopeptide identification and phosphosite localization include HCD or ExD-based methods, discussed later in more depth<span class="citation" data-cites="YU1KHJ6Z 18YSDGQ7f"><sup><a href="#ref-YU1KHJ6Z" role="doc-biblioref">609</a>,<a href="#ref-18YSDGQ7f" role="doc-biblioref">610</a></sup></span>.
ExD methods generate phosphopeptide MS/MS spectra with many c- and z•-type fragment ions for peptide sequencing and localization of labile phosphate modifications, typically disrupted with CID<span class="citation" data-cites="GH22tfkO"><sup><a href="#ref-GH22tfkO" role="doc-biblioref">532</a></sup></span>.
Gas-phase phosphate rearrangement induced by collisional activation represents a glaring challenge for the field and several have explored site localization in the face of rearrangement<span class="citation" data-cites="12LsVxcez 1BSd7ZIJx XAV4WUNo"><sup><a href="#ref-12LsVxcez" role="doc-biblioref">611</a>–<a href="#ref-XAV4WUNo" role="doc-biblioref">613</a></sup></span>.</p>
<p>Advanced data acquisition schemes trigger predetermined MS/MS events when a specific fragment ion or neutral loss is detected in a spectrum.
Certain decision-tree strategies have arisen to increase data acquisition efficiency, including pseudo-MS3 scans which are triggered on detection of phosphate losses<span class="citation" data-cites="Oadeotpz"><sup><a href="#ref-Oadeotpz" role="doc-biblioref">526</a></sup></span> and the use of site-specific x-type ions<span class="citation" data-cites="DcoxXgj"><sup><a href="#ref-DcoxXgj" role="doc-biblioref">614</a></sup></span>.
For example, when linear ion traps were the main proteomics workhorses, resonant CID analysis of phosphopeptides would result in predominantly neutral loss of the phosphate with limited sequence ion information.
To gain sequence ions in these experiments, instruments could be set to isolate a loss of 98 Daltons for MS3 activation<span class="citation" data-cites="SngtJdsy WtrZNDoq"><sup><a href="#ref-SngtJdsy" role="doc-biblioref">615</a>,<a href="#ref-WtrZNDoq" role="doc-biblioref">616</a></sup></span>.
The newer collisional dissociation technique HCD, or beam-type collisional activation, significantly improves the detection of peptide fragments with the phosphorylation intact on fragment ions, and thus, this neutral loss scanning technique is no longer common.</p>
<p>Recently developed approaches to phosphopeptide identification include DIA-based phosphoproteomics with Spectronaut<span class="citation" data-cites="OCeJYJsg GD2lTg3N"><sup><a href="#ref-OCeJYJsg" role="doc-biblioref">617</a>,<a href="#ref-GD2lTg3N" role="doc-biblioref">618</a></sup></span>, “plug-and-play” high-resolution MS<span class="citation" data-cites="18oI6YEEE"><sup><a href="#ref-18oI6YEEE" role="doc-biblioref">619</a></sup></span>, SureQuant for phosphotyrosine<span class="citation" data-cites="nO23Yma7"><sup><a href="#ref-nO23Yma7" role="doc-biblioref">309</a></sup></span>, PIQED for direct identification and quantification of phosphorylation from DIA without a prior spectral library<span class="citation" data-cites="184JFw6LL"><sup><a href="#ref-184JFw6LL" role="doc-biblioref">483</a></sup></span>, and FAIMS front-end separations which yield 15-20% more phosphosite identifications than non-FAIMS experiments<span class="citation" data-cites="E9GVLceZ"><sup><a href="#ref-E9GVLceZ" role="doc-biblioref">503</a></sup></span>.
For quantification of phosphoproteins, Hogrebe et al. investigated several of the most common strategies and concluded that TMT-based MS2 strategies may be the current best approach<span class="citation" data-cites="rydRUvTU"><sup><a href="#ref-rydRUvTU" role="doc-biblioref">620</a></sup></span>.</p>
<h5 id="glycopeptides">Glycopeptides</h5>
<p>A similar product-dependent MS/MS triggering strategy was introduced for N-linked glycopeptides<span class="citation" data-cites="GL3PYaNa"><sup><a href="#ref-GL3PYaNa" role="doc-biblioref">621</a></sup></span>.
Collisional dissociation of glycosylated peptides produces oxonium ions, for example at <em>m/z</em> 204.09 (HexNAc) or <em>m/z</em> 366.14 (HexHexNAc).
If oxonium ions from the fragmented glycan are detected among the most abundant fragment ions of the HCD spectra, then an ETD scan is triggered.
This ETD scan provides information about the peptide sequence, while the original HCD scan provides glycan structure information.</p>
<h2 class="page_break_before" id="raw-data-analysis">13. Raw Data Analysis</h2>
<p>The goal of raw data analysis is to convert raw spectral data into lists of altered protein groups, which requires many steps, including checking data quality, peptide spectra matching, protein inference<span class="citation" data-cites="imR1dFei wtH5a4Ax"><sup><a href="#ref-imR1dFei" role="doc-biblioref">622</a>,<a href="#ref-wtH5a4Ax" role="doc-biblioref">623</a></sup></span>, quantification, and statistical hypothesis tests.
Subsequently, many additional analyses can be performed to make biological inferences, which is covered in a subsequent section.
An overview of the entire data analysis cycle is shown in (<strong>Figure 16</strong>).</p>
<div id="fig:DIA-types" class="fignos">
<figure>
<img src="images/Fig_12_Biological_interpretation.png" style="width:100.0%" data-tag="16" alt="Figure 16: Proteomics Data Analysis and Biological Interpretation. The process begins with protein identification and quantification using tools such as Proteome Discoverer, Spectronaut, Spectromine, MS Fragger, MaxQuant, and Skyline. Quality control measures ensure data integrity, leading to a biological interpretation of the results. Differential expression analyses may include relative abundance charts, heat maps, and volcano plots. Functional analysis encompasses gene ontology, protein-protein interactions, and signaling pathways." />
<figcaption aria-hidden="true"><span>Figure 16:</span> <strong>Proteomics Data Analysis and Biological Interpretation.</strong>
The process begins with protein identification and quantification using tools such as Proteome Discoverer, Spectronaut, Spectromine, MS Fragger, MaxQuant, and Skyline.
Quality control measures ensure data integrity, leading to a biological interpretation of the results.
Differential expression analyses may include relative abundance charts, heat maps, and volcano plots.
Functional analysis encompasses gene ontology, protein-protein interactions, and signaling pathways.</figcaption>
</figure>
</div>
<p>Due to the inherent differences in the data structures of DDA and DIA measurements, there exist different types of software that can facilitate the steps mentioned above.
The existing software for DDA and DIA analysis can be further divided into freeware and non-freeware.</p>
<h3 id="analysis-of-dda-data">Analysis of DDA data</h3>
<p>DDA data analysis either directly uses the vendor proprietary data format directly with a proprietary search engine like Mascot, SEQUEST (through Proteome Discoverer), Paragon (through Protein Pilot), or it can be processed through one of the many freely available search engines or pipelines, for example, Comet, MaxQuant, MSGF+, X!Tandem, Morpheus, MS-Fragger, and OMSSA.
Table 7 gives weblinks and citations for these software tools.
For analysis with freeware, raw data is converted to standard open XML formats like mzML<span class="citation" data-cites="zsmAmqZJ fQHXeOG1 r2hPariY"><sup><a href="#ref-zsmAmqZJ" role="doc-biblioref">624</a>–<a href="#ref-r2hPariY" role="doc-biblioref">626</a></sup></span>.
The appropriate FASTA file containing proteins predicted from that organism’s genome is chosen as a reference database to search the experimental spectra.
All search parameters like peptide and fragment mass errors (i.e. MS1 and MS2 tolerances), enzyme specificity, number of missed cleavages, chemical artifacts and potential biological modifications (variable/dynamic modifications) are specified before executing the search.
The search algorithm scores each query spectrum against its possible peptide matches<span class="citation" data-cites="1GxDrwPIW"><sup><a href="#ref-1GxDrwPIW" role="doc-biblioref">627</a></sup></span>.
A spectrum and its best scoring candidate peptide are called a peptide spectrum match (PSM).
The scores reflect a <em>goodness-of-fit</em> between an experimental spectrum and a theoretical one and do not necessarily depict the correctness of the peptide assignment.</p>
<p>Table 7: DDA data analysis software examples.</p>
<table>
<colgroup>
<col style="width: 11%" />
<col style="width: 34%" />
<col style="width: 38%" />
<col style="width: 15%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Publication</th>
<th style="text-align: center;">Website</th>
<th style="text-align: center;">Free/Paid</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">MaxQuant</td>
<td style="text-align: center;">Cox and Mann, 2008<span class="citation" data-cites="18PIzg2Co"><sup><a href="#ref-18PIzg2Co" role="doc-biblioref">628</a></sup></span></td>
<td style="text-align: center;"><a href="https://www.maxquant.org/">MaxQuant</a></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: center;">MSFragger</td>
<td style="text-align: center;">Kong et al., 2017<span class="citation" data-cites="bKpYR6PB"><sup><a href="#ref-bKpYR6PB" role="doc-biblioref">629</a></sup></span></td>
<td style="text-align: center;"><a href="https://msfragger.nesvilab.org/">MSFragger</a></td>
<td style="text-align: center;">free</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Mascot</td>
<td style="text-align: center;">Perkins et al., 1999<span class="citation" data-cites="rKtsv6G"><sup><a href="#ref-rKtsv6G" role="doc-biblioref">630</a></sup></span></td>
<td style="text-align: center;"><a href="https://www.matrixscience.com/">Mascot</a></td>
<td style="text-align: center;">free</td>
</tr>
<tr class="even">
<td style="text-align: center;">MS-GF+</td>
<td style="text-align: center;">Kim et al.,<span class="citation" data-cites="1B2RVFOKK"><sup><a href="#ref-1B2RVFOKK" role="doc-biblioref">631</a></sup></span></td>
<td style="text-align: center;"><a href="https://github.com/MSGFPlus/msgfplus">MS-GF+</a></td>
<td style="text-align: center;">free</td>
</tr>
<tr class="odd">
<td style="text-align: center;">X!Tandem</td>
<td style="text-align: center;">Craig et al.,<span class="citation" data-cites="IcvYCrHi iK4IJ8ED"><sup><a href="#ref-IcvYCrHi" role="doc-biblioref">632</a>,<a href="#ref-iK4IJ8ED" role="doc-biblioref">633</a></sup></span></td>
<td style="text-align: center;"><a href="https://www.thegpm.org/tandem/">GPMDB</a></td>
<td style="text-align: center;">free</td>
</tr>
<tr class="even">
<td style="text-align: center;">Comet</td>
<td style="text-align: center;">Eng et al., 2012<span class="citation" data-cites="jC0M4v7k"><sup><a href="#ref-jC0M4v7k" role="doc-biblioref">634</a></sup></span></td>
<td style="text-align: center;"><a href="https://uwpr.github.io/Comet/">Comet</a></td>
<td style="text-align: center;">free</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Skyline</td>
<td style="text-align: center;">MacLean et al., 2010<span class="citation" data-cites="CObAggjA"><sup><a href="#ref-CObAggjA" role="doc-biblioref">635</a></sup></span></td>
<td style="text-align: center;"><a href="https://skyline.ms/project/home/software/Skyline/begin.view">Skyline</a></td>
<td style="text-align: center;">free</td>
</tr>
<tr class="even">
<td style="text-align: center;">ProteomeDiscoverer</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="https://www.thermofisher.com/ch/en/home/industrial/mass-spectrometry/liquid-chromatography-mass-spectrometry-lc-ms/lc-ms-software/multi-omics-data-analysis/proteome-discoverer-software.html">ProteomeDiscoverer</a></td>
<td style="text-align: center;">paid</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Mascot</td>
<td style="text-align: center;">Perkins et al., 1999<span class="citation" data-cites="rKtsv6G"><sup><a href="#ref-rKtsv6G" role="doc-biblioref">630</a></sup></span></td>
<td style="text-align: center;"><a href="https://www.matrixscience.com/">Mascot</a></td>
<td style="text-align: center;">paid</td>
</tr>
<tr class="even">
<td style="text-align: center;">Spectromine</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="ttps://biognosys.com/software/spectromine/?gclid=Cj0KCQiAoY-PBhCNARIsABcz770mjUz6iavBr9Ql7RPUdMvaHu9RYgPNrEfZco1wExEeoFwnQXuCHscaAlgBEALw_wcB">Spectromine</a></td>
<td style="text-align: center;">paid</td>
</tr>
<tr class="odd">
<td style="text-align: center;">PEAKS</td>
<td style="text-align: center;">Tran et al., 2018<span class="citation" data-cites="IUqTUA43"><sup><a href="#ref-IUqTUA43" role="doc-biblioref">636</a></sup></span></td>
<td style="text-align: center;"><a href="https://www.bioinfor.com/peaks-studio/">PEAKS</a></td>
<td style="text-align: center;">paid</td>
</tr>
</tbody>
</table>
<p>Recall that we noted the stochasticity of DDA; every injection will select different peptide precursors for fragmentation leading to different identifications from each sample.
To ameliorate this issue, often strategies are used to transfer identifications between multiple sample analyses.
This transfer of IDs across runs is known as “match between runs”, which was originally made famous by the processing software MaxQuant<span class="citation" data-cites="vO5QsPvL Gs7HWzAD"><sup><a href="#ref-vO5QsPvL" role="doc-biblioref">637</a>,<a href="#ref-Gs7HWzAD" role="doc-biblioref">638</a></sup></span>.
There are several other similar tools and strategies, including the accurate mass and time approach<span class="citation" data-cites="66LthRpU"><sup><a href="#ref-66LthRpU" role="doc-biblioref">639</a></sup></span>, Q-MEND<span class="citation" data-cites="MaoP0MGM"><sup><a href="#ref-MaoP0MGM" role="doc-biblioref">640</a></sup></span>, IDEAL-Q<span class="citation" data-cites="5H13CEWA"><sup><a href="#ref-5H13CEWA" role="doc-biblioref">641</a></sup></span> and superHIRN<span class="citation" data-cites="rRs3KNfz"><sup><a href="#ref-rRs3KNfz" role="doc-biblioref">642</a></sup></span>.
More recent work has introducted statistical assessment of MBR methods using a two-proteome model<span class="citation" data-cites="1EHOiWEPV"><sup><a href="#ref-1EHOiWEPV" role="doc-biblioref">643</a></sup></span>.
Statistically controlled MBR is currently available in the IonQuant tool<span class="citation" data-cites="XGf3EEUa"><sup><a href="#ref-XGf3EEUa" role="doc-biblioref">644</a></sup></span>.</p>
<h3 id="strategies-for-analysis-of-dia-data">Strategies for analysis of DIA data</h3>
<p>DIA data analysis is fundamentally different from DDA data analysis because, instead of a single MS/MS spectrum for each peptide, we can observe the elution of peptide fragments for any peptide over chromatography time.
Even though DIA data analysis derives peptide matches differently, the same target-decoy analysis described above is often used.
There are two general approaches for peptide identification from DIA data: peptide-centric and spectrum-centric.</p>
<p>Peptide-centric approaches looks for evidence of specific peptides that are in some assay library of MS/MS spectra.
That library could be predicted spectra (e.g., using Prosit)<span class="citation" data-cites="50qmV6VJ"><sup><a href="#ref-50qmV6VJ" role="doc-biblioref">645</a></sup></span>, or previously measured spectra (e.g., from a organism-wide knowledge base)<span class="citation" data-cites="JejZsE21"><sup><a href="#ref-JejZsE21" role="doc-biblioref">646</a></sup></span>.
Examples of software that perform peptide-centric analysis include OpenSWATH<span class="citation" data-cites="Sd09mxFZ"><sup><a href="#ref-Sd09mxFZ" role="doc-biblioref">647</a></sup></span>, Spectronaut<span class="citation" data-cites="1GvzNjeV3"><sup><a href="#ref-1GvzNjeV3" role="doc-biblioref">605</a></sup></span>, csoDIAq<span class="citation" data-cites="ezZ8OwXe"><sup><a href="#ref-ezZ8OwXe" role="doc-biblioref">648</a></sup></span>, and DIA-NN<span class="citation" data-cites="1HLH2DWkK"><sup><a href="#ref-1HLH2DWkK" role="doc-biblioref">649</a></sup></span>.</p>
<p>Spectrum-centric approaches instead ask if there is evidence for any peptide based on analysis of the observed spectra.
Examples of spectrum-centric approaches include DIA-Umpire<span class="citation" data-cites="Mqz9Ko20"><sup><a href="#ref-Mqz9Ko20" role="doc-biblioref">650</a></sup></span> and PECAN<span class="citation" data-cites="15hTjIZkP"><sup><a href="#ref-15hTjIZkP" role="doc-biblioref">651</a></sup></span>.
Spectrum-centric approaches may assemble pseudo-MS/MS spectra from co-elution of fragments that can then be used with any DDA database search<span class="citation" data-cites="Mqz9Ko20"><sup><a href="#ref-Mqz9Ko20" role="doc-biblioref">650</a></sup></span>.
Spectrum-centric approaches may be less sensitive at peptide identification than peptide-centric approaches.</p>
<p>A non-comprehensive list of software for DIA data analysis is found in table 8.</p>
<p>Table 8: DIA data analysis software examples.</p>
<table>
<colgroup>
<col style="width: 11%" />
<col style="width: 34%" />
<col style="width: 38%" />
<col style="width: 15%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Publication</th>
<th style="text-align: center;">Website</th>
<th style="text-align: center;">Free/Paid</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">MaxDIA</td>
<td style="text-align: center;">Cox and Mann, 2008<span class="citation" data-cites="18PIzg2Co"><sup><a href="#ref-18PIzg2Co" role="doc-biblioref">628</a></sup></span></td>
<td style="text-align: center;"><a href="https://www.maxquant.org/">MaxQuant</a></td>
<td style="text-align: center;">free</td>
</tr>
<tr class="even">
<td style="text-align: center;">Skyline</td>
<td style="text-align: center;">MacLean et al., 2010<span class="citation" data-cites="CObAggjA"><sup><a href="#ref-CObAggjA" role="doc-biblioref">635</a></sup></span></td>
<td style="text-align: center;"><a href="https://skyline.ms/project/home/software/Skyline/begin.view">Skyline</a></td>
<td style="text-align: center;">free</td>
</tr>
<tr class="odd">
<td style="text-align: center;">DIA-NN</td>
<td style="text-align: center;">Demichev et al., 2019<span class="citation" data-cites="1HLH2DWkK"><sup><a href="#ref-1HLH2DWkK" role="doc-biblioref">649</a></sup></span></td>
<td style="text-align: center;"><a href="https://github.com/vdemichev/DiaNN">DIA-NN</a></td>
<td style="text-align: center;">free</td>
</tr>
<tr class="even">
<td style="text-align: center;">EncyclopeDIA</td>
<td style="text-align: center;">Searle et al., 2018<span class="citation" data-cites="rYKhRHlv"><sup><a href="#ref-rYKhRHlv" role="doc-biblioref">652</a></sup></span></td>
<td style="text-align: center;"><a href="http://www.searlelab.org/software/encyclopedia/index.html">EncyclopeDIA</a></td>
<td style="text-align: center;">free</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Spectronaut</td>
<td style="text-align: center;">Bruderer et al., 2015<span class="citation" data-cites="UKcaDUJ7"><sup><a href="#ref-UKcaDUJ7" role="doc-biblioref">653</a></sup></span></td>
<td style="text-align: center;"><a href="https://biognosys.com/software/spectronaut/">Spectronaut</a></td>
<td style="text-align: center;">paid</td>
</tr>
<tr class="even">
<td style="text-align: center;">PEAKS</td>
<td style="text-align: center;">Tran et al., 2018<span class="citation" data-cites="IUqTUA43"><sup><a href="#ref-IUqTUA43" role="doc-biblioref">636</a></sup></span></td>
<td style="text-align: center;"><a href="https://www.bioinfor.com/peaks-studio/">PEAKS</a></td>
<td style="text-align: center;">paid</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Scaffold DIA</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="https://www.proteomesoftware.com/products/scaffold-dia">Proteome Software</a></td>
<td style="text-align: center;">paid</td>
</tr>
</tbody>
</table>
<h3 id="deriving-statistical-significance-of-psms-with-the-target-decoy-approach">Deriving statistical significance of PSMs with the target-decoy approach</h3>
<p>For evaluating the probability that a PSM is not random, matches to a decoy database of shuffled or reversed sequences are used as the null model for peptide matching.
A randomized or reversed version of target database is used as a nonparametric null model.
The decoy database can be searched separate from the target database (Kall’s method)<span class="citation" data-cites="bjANWlBN"><sup><a href="#ref-bjANWlBN" role="doc-biblioref">654</a></sup></span> or it can be combined with the target database before search (Elias and Gygi method)<span class="citation" data-cites="BrkRiRe8"><sup><a href="#ref-BrkRiRe8" role="doc-biblioref">655</a></sup></span>.
Using either separate method or concatenated database search method, an estimate of false hits can be calculated which is used to estimate the false discovery rate (FDR)<span class="citation" data-cites="5mQtZh5o"><sup><a href="#ref-5mQtZh5o" role="doc-biblioref">656</a></sup></span>.
The FDR denotes the proportion of false hits in the population accepted as true.
For Kall’s method: the false hits are estimated to be the number of decoys above a given threshold.
It is assumed that the number of decoy hits that pass a threshold are the false hits.
A similar number of target population may also be false.
Therefore, the FDR is calculated as<span class="citation" data-cites="B2OZyvg5"><sup><a href="#ref-B2OZyvg5" role="doc-biblioref">657</a></sup></span>:</p>
<p><span class="math display">\[FDR = \frac{Decoy PSMs + 1}{Target PSMs}\]</span></p>
<p>For Elias and Gygi Method, the target population in which FDR is estimated changes.
The target and decoy hits coming from a joint database compete against each other.
For any spectrum, either a target or a decoy peptide can be the best hit.
It is argued that the joint target-decoy population has decoy hits as confirmed false hits.
However, due to the joint database search, the target database may also have equal number of false hits.
Thus, the number of false hits is multiplied by two for FDR estimation.</p>
<p><span class="math display">\[FDR = \frac{2 * Decoy PSMs}{Target + Decoy PSMs}\]</span></p>
<h3 id="integrated-ms-data-analysis-platforms">Integrated MS Data Analysis Platforms</h3>
<p>Given the complexity of proteomic data analysis and the requirement for many steps to get from raw data to quantified proteins, there are some integrated software enviroments that easily allow users to complete everything in one place.</p>
<h4 id="peptide-shaker">Peptide Shaker</h4>
<p>Since each search engine may give slightly different results, and peptides identified by multiple search engines may be more confident hits, integration platforms such as PeptideShaker have been developed to combine search results<span class="citation" data-cites="B10jxZjh tZ6JV4UF"><sup><a href="#ref-B10jxZjh" role="doc-biblioref">658</a>,<a href="#ref-tZ6JV4UF" role="doc-biblioref">659</a></sup></span>.
PeptideShaker gives an interactive overview of all the protein, peptide, and PSMs in a dataset.
It also has many other features, such as PTM summaries, 3D structure mapping of detected peptides, QC, validation, and GO enrichment (see biological interpretation section for more details).</p>
<h4 id="trans-proteomic-pipeline-tpp">Trans-Proteomic Pipeline (TPP)</h4>
<p>The Trans-Proteomic Pipeline (TPP) is a free and open-source mass spectrometry data analysis suite for end-to end analysis that remains in continual development to provide ever expansive data analysis capabilities since its inception over twenty years ago<span class="citation" data-cites="9bP7w1xR w5NR3Ly9 NY6IKEE4 1FG7koZ0h 15EklFALC KUnjPYby 1ArAvfAkV 7BQGCj33 SKCHzO1U KIekIPp9 bBz09mFl"><sup><a href="#ref-KIekIPp9" role="doc-biblioref">75</a>,<a href="#ref-9bP7w1xR" role="doc-biblioref">660</a>–<a href="#ref-bBz09mFl" role="doc-biblioref">669</a></sup></span>.
The current release provides tools for mass spectrometry spectral processing, spectrum searching, search validation, abundance computation, protein inference, and statistical evaluation of the data to ensure controlled false-discovery rates.
Many of the tools include machine-learning modeling to extract the most information from datasets and build robust statistical models to compute probabilities that derived information is correct.</p>
<p>One of the major advantages of TPP is its ability to be deployed in a wide variety of environments, from personal Windows laptops to extensive large Linux clusters for automated use within cloud computing environments.
While the command-line interfaces are appreciated by many power users, others prefer a graphical user interface (GUI), which is provided by the TPP GUI called Petunia, allowing users to use the TPP from any web browser on any platform.
Petunia has the advantages that the same exact GUI is available on a modest Windows laptop, a powerful expandable Linux server shared by a research group, or a remote cloud computing instance running on Amazon Web Services (AWS)<span class="citation" data-cites="15EklFALC"><sup><a href="#ref-15EklFALC" role="doc-biblioref">664</a></sup></span>.</p>
<p>The TPP incudes many statistical validation tools such as PeptideProphet<span class="citation" data-cites="10mgCOk6B"><sup><a href="#ref-10mgCOk6B" role="doc-biblioref">670</a></sup></span>, ProteinProphet<span class="citation" data-cites="4OdLMmDD"><sup><a href="#ref-4OdLMmDD" role="doc-biblioref">671</a></sup></span>, iProphet<span class="citation" data-cites="1FG7koZ0h"><sup><a href="#ref-1FG7koZ0h" role="doc-biblioref">663</a></sup></span>, and PTMProphet<span class="citation" data-cites="SKCHzO1U"><sup><a href="#ref-SKCHzO1U" role="doc-biblioref">668</a></sup></span>, where Bayesian machine learning techniques are applied to the various search engine scores to model the correct and incorrect assignment distributions and then use these models to assign a probability of being correct based on these learned models.
With these tools it is possible to validate search engine results on large-scale datasets and in short order, enabling users to select probability thresholds based on a selected tolerable false discovery rate (FDR).
The TPP is made fully interoperable via the open XML-based formats pepXML and protXML for different aspects of processing data-dependent acquisition (DDA), and Data-independent acquisition (DIA) proteomics data, resulting in a complete suite of tools for processing the increasingly larger datasets from start to finish.</p>
<p>DIA workflows are supported via the DISCO tool which reads mzML files containing the instrument-produced spectra and uses signal processing approaches to isolate the fragment ions in the multiplexed MS2 spectra that correlate with precursors in the MS1 and writes the results to new mzML files that may then be searched with standard DDA search engines and downstream tools, including target-decoy analysis.
This provides a comprehensive analysis of DIA data without the need for building a spectral library first.</p>
<p>From its inception, the TPP has been and will always be free and open-source software, allowing anyone to use it without cost and to inspect its source code, alter the source code for their own needs, or even incorporate parts of it into their own products.
Others have performed these tasks and include various analysis routines as addons such as TAILS N-terminomics analysis<span class="citation" data-cites="l6NqXxi8"><sup><a href="#ref-l6NqXxi8" role="doc-biblioref">672</a></sup></span>, quantitation analysis with PyQuant<span class="citation" data-cites="WlEQJK9j"><sup><a href="#ref-WlEQJK9j" role="doc-biblioref">673</a></sup></span>, SimPhospho<span class="citation" data-cites="DzOnRsF8"><sup><a href="#ref-DzOnRsF8" role="doc-biblioref">674</a></sup></span>, WinProphet<span class="citation" data-cites="8GgfngeF"><sup><a href="#ref-8GgfngeF" role="doc-biblioref">675</a></sup></span>, ProtyQuant<span class="citation" data-cites="2XDHE8d9"><sup><a href="#ref-2XDHE8d9" role="doc-biblioref">676</a></sup></span>, and inclusion of R-tools for metaproteomic analysis<span class="citation" data-cites="146pjMaXB"><sup><a href="#ref-146pjMaXB" role="doc-biblioref">677</a></sup></span>.
As a collection of individual tools, they are easily amenable to pipelining in a very flexible manner to support a huge variety of combinations and workflows, and a custom program may easily be inserted into the pipeline to support technology development.</p>
<h5 id="search-engines-supported-by-tpp">Search engines supported by TPP</h5>
<p>The heart of MS proteomics DDA data continues to be the “search engine” that interprets collections of mass spectra to determine the peptide or peptides that yielded them.
Spectral library search engines and de novo search engines, which are less common, are also available and are included in software suites such as the Trans Proteomic Pipeline.
A sequence search engine most commonly used is the open-source version of SEQUEST called Comet, which is actively maintained and updated with new functionality as needs arise.
For spectral library searching, SpectraST uses an approach where new spectra are matched against a library of previously identified spectra in the form of a spectral library<span class="citation" data-cites="KbIsieWP"><sup><a href="#ref-KbIsieWP" role="doc-biblioref">678</a></sup></span>.
This approach is much faster, more sensitive, and more specific than sequence database searching, although is only as good as the reference spectral library provided.
There is renewed interest in spectral libraries because of data-independent acquisition (DIA) approaches being increasingly deployed and therefore the quality and coverage of libraries is paramount and likely to improve in the coming years, aided by the standard spectral library format being developed by the PSI<span class="citation" data-cites="M0N5UGce"><sup><a href="#ref-M0N5UGce" role="doc-biblioref">679</a></sup></span>.
For de novo sequence analysis, Novor<span class="citation" data-cites="1C0ZGb77k"><sup><a href="#ref-1C0ZGb77k" role="doc-biblioref">680</a></sup></span> and Casanovo<span class="citation" data-cites="zzSgLIrm"><sup><a href="#ref-zzSgLIrm" role="doc-biblioref">681</a></sup></span> are very fast and capable de novo sequence search engines that are available.</p>
<p>For chemical crosslinking proteomics analysis, open-source programs such as Kojak<span class="citation" data-cites="10TqPZVdy KIekIPp9"><sup><a href="#ref-10TqPZVdy" role="doc-biblioref">74</a>,<a href="#ref-KIekIPp9" role="doc-biblioref">75</a></sup></span> are available for standard or cleavable MS2-based crosslinking techniques.
Crosslinking-based MS analyses are employed to elucidate protein-protein interactions and facilitate protein structure and topology predictions.
Kojak is designed to identify two independent peptides covalently bonded with a crosslinker and fragmented in a single MS2 scan event using a database search approach.
Kojak algorithm also includes support for cleavable cross-linkers, and identification of cross-links between 15N-labeled homomultimers and is integrated into the Trans-Proteomic Pipeline, enabling access to dozens of additional tools, in particular, the PeptideProphet and iProphet tools for validation of cross-links improve the sensitivity and accuracy of correct cross-link identifications at user-defined thresholds.
Development of Kojak has continued over the last ten years culminating in many improvements and new features.
These improvements include support for additional open formats and standards, further refinement to the search algorithm for efficiency, E-values to normalize the scores of the results, support for cleavable cross-linkers, and methods to identify cross-links between homomultimer subunits.</p>
<p>For open modification database searching, programs such as Magnum (<span class="citation" data-cites="fgRTHCzN"><sup><a href="#ref-fgRTHCzN" role="doc-biblioref">682</a></sup></span>) are also now available which is specialized in identification of non-peptide masses that are bound to peptides. The tool is capable of identifying xenobiotic mass adducts, in addition to PTMs that were uncharacterized in the search parameters.</p>
<h3 id="quality-control">Quality Control</h3>
<p>Quality control should be a central aspect of any mass spectrometry-based study to ensure reproducibility of generated results.
There are two types of quality controls that can be conducted for any kind of mass spectrometry experiment.
First, QC approaches should monitor instruments themselves (e.g. HPLC and mass spectrometer), and second, QC approaches should assess the quality of data from unknowns or samples.
For further reading, an entire issue was published on quality control in the journal <em>Proteomics</em> in 2011<span class="citation" data-cites="JXLIvVI6"><sup><a href="#ref-JXLIvVI6" role="doc-biblioref">683</a></sup></span>, especially the review by Köcher <em>et al.</em><span class="citation" data-cites="8dt10izl"><sup><a href="#ref-8dt10izl" role="doc-biblioref">684</a></sup></span>, as well as the review published by Bittremieux <em>et al.</em> in 2017<span class="citation" data-cites="196I1uWdL"><sup><a href="#ref-196I1uWdL" role="doc-biblioref">685</a></sup></span>.</p>
<h4 id="qc-instrument-performance">QC: Instrument Performance</h4>
<p>It is generally advisable to monitor instrument performance regularly.
Instrument calibrations in regular intervals are essential to ensure that performance is maintained.
Each instrument has a specific calibration method that is required.
During the calibration you can check injection times (for ion trap instruments) and intensity of the ions in the calibration mix.</p>
<p>After ensuring good calibration and signal with the simple calibration mixture, it is advisable to analyze complex samples, such as commercial simple peptide mixtures or even whole tryptic digests of cell lysates (e.g. K562 standard from Promega).
The additional check with a complex sample ensures all aspects of the system are working together correctly, especially the liquid chromatography and emitter.
These digests should be analyzed after every instrument calibration and periodically between samples when acquiring more extensive batches.
Data measured from tryptic digests should be analyzed by the software of your choice and the numbers of identified peptide precursors and proteins can be compared with previous controls for consistency.</p>
<p>Another strategy is to analyze digested purified proteins, which easily enable discovery of retention time shifts and mass accuracy problems.
It is advised that new practicioners perform this manually at first to understand their data; this can be done by manually by looking <em>m/z</em> values of your standard peptides across runs in Skyline or the vendor-specific software.
Looking at the intensity of the extracted peaks will help identify sensitivity fluctuations.</p>
<p>Carry-over between different measurements can be identified from blank measurements which are subsequently analyzed with your search software of choice.
Blank measurements can be injections of different buffers, water or the starting conditions of your liquid chromatography.
In case of increased detection of carry-over, injections with trifluoroethanol can be performed.</p>
<p>Another factor to take into consideration is the stability of your electrospray.
Electrospray stability tends to worsen over time as columns wear and accumulate contaminants, such as salts or detergents, which can influence the quality of the emitter or column tip.
You will notice spray instabilities either in the total ion chromatogram (TIC) as thin spikes with short periods of no measured signal or if you install cameras at your ESI source.
Suboptimal spray conditions will usually result in droplets forming on the emitter, being released into the mass spectrometer (also referred to as “spitting”).
Real-time quality control software (listed in tables 9 and 10 below) can help you identify instrument issues right away.</p>
<p>Table 9: Quality Control Software for raw file and real-time analysis.</p>
<table>
<colgroup>
<col style="width: 7%" />
<col style="width: 27%" />
<col style="width: 24%" />
<col style="width: 21%" />
<col style="width: 18%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Supported instrument vendors</th>
<th style="text-align: center;">Website/Download</th>
<th style="text-align: center;">publication</th>
<th style="text-align: center;">Note</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">QuiC</td>
<td style="text-align: center;">Thermo Scientific, AB SCIEX, Agilent, Bruker, Waters</td>
<td style="text-align: center;"><a href="https://biognosys.com/resources/quic-manual/">QuiC</a></td>
<td style="text-align: center;"></td>
<td style="text-align: center;">requires Biognosys iRT peptides</td>
</tr>
<tr class="even">
<td style="text-align: center;">AlphaPept</td>
<td style="text-align: center;">Thermo Scientific, Bruker</td>
<td style="text-align: center;"><a href="https://github.com/MannLabs/alphapept">AlphaPept</a></td>
<td style="text-align: center;"><span class="citation" data-cites="9ocWQ3g2"><sup><a href="#ref-9ocWQ3g2" role="doc-biblioref">686</a></sup></span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: center;">RawMeat 2.1</td>
<td style="text-align: center;">Thermo Scientific</td>
<td style="text-align: center;"><a href="https://proteomicsresource.washington.edu/protocols06/RawMeat_1007.exe">RawMeat</a></td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: center;">rawDiag</td>
<td style="text-align: center;">Thermo Scientific</td>
<td style="text-align: center;"><a href="https://github.com/fgcz/rawDiag">rawDiag</a></td>
<td style="text-align: center;"><span class="citation" data-cites="mKMrKqLf"><sup><a href="#ref-mKMrKqLf" role="doc-biblioref">687</a></sup></span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: center;">rawrr</td>
<td style="text-align: center;">Thermo Scientific</td>
<td style="text-align: center;"><a href="https://github.com/fgcz/rawrr">rawrr</a></td>
<td style="text-align: center;"><span class="citation" data-cites="AI1wddvh"><sup><a href="#ref-AI1wddvh" role="doc-biblioref">688</a></sup></span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: center;">rawBeans</td>
<td style="text-align: center;">Thermo or mzML</td>
<td style="text-align: center;"><a href="https://bitbucket.org/incpm/prot-qc/downloads/">rawBeans</a></td>
<td style="text-align: center;"><span class="citation" data-cites="MotUU8WD"><sup><a href="#ref-MotUU8WD" role="doc-biblioref">689</a></sup></span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: center;">SIMPATIQCO</td>
<td style="text-align: center;">Thermo Scientific</td>
<td style="text-align: center;"><a href="https://ms.imp.ac.at/index.php?action=simpatiqco">SIMPATIQCO</a></td>
<td style="text-align: center;"><span class="citation" data-cites="fVgT4RCS"><sup><a href="#ref-fVgT4RCS" role="doc-biblioref">690</a></sup></span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: center;">QC-ART</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="https://github.com/stanfill/QC-ART">QC-ART</a></td>
<td style="text-align: center;"><span class="citation" data-cites="gR6OKDGy"><sup><a href="#ref-gR6OKDGy" role="doc-biblioref">691</a></sup></span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: center;">SprayQc</td>
<td style="text-align: center;">Thermo Scientific, AB SCIEX, extensible to other instrumentation</td>
<td style="text-align: center;"><a href="http://sourceforge.org/projects/sprayqc">SprayQc</a></td>
<td style="text-align: center;"><span class="citation" data-cites="IrACRvhj"><sup><a href="#ref-IrACRvhj" role="doc-biblioref">692</a></sup></span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: center;">Metriculator</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="http://github.com/princelab/metriculator">Metriculator</a></td>
<td style="text-align: center;"><span class="citation" data-cites="N0V314Nq"><sup><a href="#ref-N0V314Nq" role="doc-biblioref">693</a></sup></span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: center;">MassQC</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="https://massqc.proteomesoftware.com/">MassQC</a></td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: center;">OpenMS</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="https://www.openms.de/">OpenMS</a></td>
<td style="text-align: center;"><span class="citation" data-cites="ZJ3s73nL"><sup><a href="#ref-ZJ3s73nL" role="doc-biblioref">694</a></sup></span></td>
<td style="text-align: center;"></td>
</tr>
</tbody>
</table>
<p>Table 10: Search result QC.</p>
<table>
<colgroup>
<col style="width: 10%" />
<col style="width: 36%" />
<col style="width: 27%" />
<col style="width: 25%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Website/Download/publication</th>
<th style="text-align: center;">publication</th>
<th style="text-align: center;">Note</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">MSStats</td>
<td style="text-align: center;"><a href="https://github.com/Vitek-Lab/MSstats">MSStats</a></td>
<td style="text-align: center;"><span class="citation" data-cites="LpMjs9Ht"><sup><a href="#ref-LpMjs9Ht" role="doc-biblioref">695</a></sup></span></td>
<td style="text-align: center;">can use output from MaxQuant, Proteome Discoverer, Skyline, Progenesis, Spectronaut</td>
</tr>
<tr class="even">
<td style="text-align: center;">MSStatsQC</td>
<td style="text-align: center;"><a href="https://msstats.org/msstatsqc/">MSStatsQC</a></td>
<td style="text-align: center;"><span class="citation" data-cites="BVhBtZ2w"><sup><a href="#ref-BVhBtZ2w" role="doc-biblioref">696</a></sup></span></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: center;">PTXQC</td>
<td style="text-align: center;"><a href="https://github.com/cbielow/PTXQC">PTXQC</a></td>
<td style="text-align: center;"><span class="citation" data-cites="R1U4dX3A"><sup><a href="#ref-R1U4dX3A" role="doc-biblioref">697</a></sup></span></td>
<td style="text-align: center;">requires MaxQuant search engine output</td>
</tr>
<tr class="even">
<td style="text-align: center;">protti</td>
<td style="text-align: center;"><a href="https://github.com/jpquast/protti">protti</a></td>
<td style="text-align: center;"><span class="citation" data-cites="Nix1GwUR"><sup><a href="#ref-Nix1GwUR" role="doc-biblioref">698</a></sup></span></td>
<td style="text-align: center;"></td>
</tr>
</tbody>
</table>
<h4 id="qc-samples">QC: Samples</h4>
<p>Apart from instrument performance, data analysis should start with QC to identify problematic measurements and to exclude them if necessary.
It is recommended to develop a standardized system for data QC early on and to keep this consistent over time.
Adding indexed retention time (iRT) peptides to samples can help identify and correct gradient and retention time inconsistencies between samples at the data analysis stage.
Including common contaminants in FASTA files, such as keratins, is not considered optional because these contaminants are always present, and monitoring them can help identify sample preparation issues that may be non-uniform across samples.
Other parameters to check in your analysis are the consistency of the number of peptide-spectrum matches, identified peptides and proteins over all samples, as well as coefficients of variation between your replicates.
Before and after data normalization (if normalization is performed) it is good to compare the median intensities of all measurements to identify potential measurement or normalization issues.
Precursor charge distributions, missed cleavage numbers, peak width, as well as the number of points per peak are additional parameters that can be checked.
In case you are analyzing different conditions, perform hierarchical clustering or a principal component analysis to check if your samples cluster as expected.</p>
<h3 id="quantitative-proteomic-data-analysis-overview">Quantitative Proteomic Data Analysis Overview</h3>
<p>This section aims to provide an overview of the best practices when conducting large scale proteomics quantitative data analysis.
A universal workflow for proteomic data analysis does not currently exist because the processing depends on specific attributes of each individual dataset<span class="citation" data-cites="mkpI0rxh"><sup><a href="#ref-mkpI0rxh" role="doc-biblioref">699</a></sup></span>.
Analyzing proteomic data requires knowledge of a multitude of pre-processing techniques where order matters, and it can be challenging knowing where to start.
This review will cover tools to reduce bias due to nonbiological variability, statistical methods to identify differential expression and machine learning (ML) methods for supervised or unsupervised interpretation of proteomic data.
For a no-code option of all processing methods described in this section, we recommend Perseus<span class="citation" data-cites="xj3O2t6x"><sup><a href="#ref-xj3O2t6x" role="doc-biblioref">700</a></sup></span>.</p>
<h4 id="data-transformation">Data Transformation</h4>
<p>Peptide or protein quantities are generally assumed to be logarithm (log) transformed before any subsequent processing<span class="citation" data-cites="1wPiXyAt 127vja7uh 1CKzlqjbM mkTOg0QU"><sup><a href="#ref-1wPiXyAt" role="doc-biblioref">701</a>–<a href="#ref-mkTOg0QU" role="doc-biblioref">704</a></sup></span>.
Log transformation allows data to more closely conform to a normal distribution and reduces the effect of highly abundant proteins<span class="citation" data-cites="127vja7uh"><sup><a href="#ref-127vja7uh" role="doc-biblioref">702</a></sup></span>.
Many normalization techniques also assume data to be symmetric, so log transformation should precede any downstream analysis in these cases<span class="citation" data-cites="127vja7uh"><sup><a href="#ref-127vja7uh" role="doc-biblioref">702</a></sup></span>.
If there are missing values present, a simple approach would be to use log(1+x) to avoid taking the log of zero.
After the transformation, zero quantities will remain as zero and the other quantities should be large enough that adding one will have a minor effect.</p>
<h4 id="data-normalization">Data Normalization</h4>
<p>Data normalization, the process for adjusting data to have comparable distributions between samples, should almost always be performed prior to batch correction and any subsequent data analysis<span class="citation" data-cites="rXBG8g4V 127vja7uh"><sup><a href="#ref-127vja7uh" role="doc-biblioref">702</a>,<a href="#ref-rXBG8g4V" role="doc-biblioref">705</a></sup></span>.
This is required when the assumption is that most proteins are not changed between conditions, which is not always true.
For example, in some studies of post translational modifications where kinases are inhibited, there may be true shifts in total signal between conditions, and normalization would mask those differences.
This is also true in co-IP experiments where one condition may truly have many fewer binding partners.
Normalization removes systematic bias in peptide/protein abundances that could mask true biological discoveries or give rise to false conclusions<span class="citation" data-cites="11NAEs2KF"><sup><a href="#ref-11NAEs2KF" role="doc-biblioref">706</a></sup></span>.
Bias may be due to factors such as measurement errors and protein degradation<span class="citation" data-cites="127vja7uh"><sup><a href="#ref-127vja7uh" role="doc-biblioref">702</a></sup></span>, although the causes for these variations are often unknown<span class="citation" data-cites="mkTOg0QU"><sup><a href="#ref-mkTOg0QU" role="doc-biblioref">704</a></sup></span>.
As data scaling methods should be kept at a minimum<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>, a normalization technique well suited to address the nuances specific to one’s data should be selected.
The assumptions for a given normalization technique should not be violated, otherwise choosing the wrong technique can lead to misleading conclusions<span class="citation" data-cites="c2k0gkjF"><sup><a href="#ref-c2k0gkjF" role="doc-biblioref">708</a></sup></span>.
There are a multitude of data normalization techniques and knowing the most suitable one for a dataset can be challenging.</p>
<p>Visualization of peptide or protein intensity distributions among samples is an important step prior to selecting a normalization technique.
Normalization is suggested to be done on the peptide level<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.
If the technical variability causes the peptide/protein abundances from each sample to be different by a constant factor, and thus intensities are graphically similar across samples, then a central tendency normalization method such as mean, median or quantile normalization may be sufficient<span class="citation" data-cites="l6oCMFXG 127vja7uh"><sup><a href="#ref-127vja7uh" role="doc-biblioref">702</a>,<a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.
However, if there is a linear relationship between bias and the peptide/protein abundances, a different method may be more appropriate.
To visualize linear and nonlinear trends due to bias, we can plot the data in a ratio versus intensity, or a M (minus) versus A (average), plot<span class="citation" data-cites="127vja7uh scfU7NxG"><sup><a href="#ref-127vja7uh" role="doc-biblioref">702</a>,<a href="#ref-scfU7NxG" role="doc-biblioref">709</a></sup></span>.
Linear regression normalization is an available technique if bias is linearly dependent on peptide/protein abundance magnitudes<span class="citation" data-cites="1CKzlqjbM 127vja7uh"><sup><a href="#ref-127vja7uh" role="doc-biblioref">702</a>,<a href="#ref-1CKzlqjbM" role="doc-biblioref">703</a></sup></span>.
Alternatively, local regression (LOESS) normalization assumes nonlinearity between protein intensity and bias<span class="citation" data-cites="1CKzlqjbM"><sup><a href="#ref-1CKzlqjbM" role="doc-biblioref">703</a></sup></span>.
Another method, removal of unwanted variation (RUV), uses information from negative controls and a linear mixed effect model to estimate unwanted noise, which is then removed from the data<span class="citation" data-cites="HCKwDoRr"><sup><a href="#ref-HCKwDoRr" role="doc-biblioref">710</a></sup></span>.</p>
<p>If sample distributions are drastically different, for example due to different treatments or samples are obtained from various tissues, one must use a method that preserves the heterogeneity in the data, including information present in outliers, meanwhile reducing systematic bias<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.
For example, Hidden Markov Model (HMM)-assisted normalization<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>, RobNorm<span class="citation" data-cites="ZgHkHm7E"><sup><a href="#ref-ZgHkHm7E" role="doc-biblioref">711</a></sup></span> or EigenMS<span class="citation" data-cites="nxZj1qHr"><sup><a href="#ref-nxZj1qHr" role="doc-biblioref">712</a></sup></span> may be suitable for this type of data.
These techniques assume error is only due to the batch and order of processing.
The first method that addresses correlation of errors between compounds by using the information from the variation of one variable to predict another is systematic error removal using random forest (SERRF)<span class="citation" data-cites="lZkCFiQ5"><sup><a href="#ref-lZkCFiQ5" role="doc-biblioref">713</a></sup></span>.
SERRF, among 14 normalization methods, was the most effective in significantly reducing systematic error<span class="citation" data-cites="lZkCFiQ5"><sup><a href="#ref-lZkCFiQ5" role="doc-biblioref">713</a></sup></span>.</p>
<p>Studies aiming to compare these methods for omics data normalization have come to different conclusions.
Ranking of different normalization methods can be done by assessing the percent decrease in median log2(standard deviation) and log2 pooled estimate of variance (PEV) in comparison to the raw data<span class="citation" data-cites="10U5RhkON"><sup><a href="#ref-10U5RhkON" role="doc-biblioref">714</a></sup></span>.
One study found linear regression ranked the highest compared to central tendency, LOESS and quantile normalization for peptide abundance normalization for replicate samples with and without biological differences<span class="citation" data-cites="127vja7uh"><sup><a href="#ref-127vja7uh" role="doc-biblioref">702</a></sup></span>.
A paper comparing multiple normalization methods using a large proteomic dataset found that mean/median centering, quantile normalization and RUV had the highest known associations between proteins and clinical variables<span class="citation" data-cites="1wPiXyAt"><sup><a href="#ref-1wPiXyAt" role="doc-biblioref">701</a></sup></span>.
Rather than individually implementing normalization techniques, which can be challenging for non-domain experts, there are several R and Python packages that automate mass spectrometry data analysis and visualization.
These tools assist with making an appropriate selection of a normalization technique.
For example, NormalyzerDE, an R package, includes several popular methods for normalization and differential expression analysis of LC-MS data<span class="citation" data-cites="DvZ7JWPs"><sup><a href="#ref-DvZ7JWPs" role="doc-biblioref">715</a></sup></span>.
AlphaPeptStats<span class="citation" data-cites="9nRMPWST"><sup><a href="#ref-9nRMPWST" role="doc-biblioref">716</a></sup></span>, a Python package, allows for comprehensive mass spectrometry data analysis, including normalization, imputation, batch correction, visualization, statistical analysis and graphical representations including heatmaps, volcano plots, and scatter plots.
AlphaPeptStats allows for analysis of label-free proteomics data from several platforms (MaxQuant, AlphaPept, DIA-NN, Spectronaut, FragPipe) in Python but also has web version that does not require installation.
For both transformation and normalization, we recommend using the options in scikit-learn in python.</p>
<h4 id="data-imputation">Data Imputation</h4>
<p>Missing peptide intensities, which are common in proteomic data, may need to be addressed, although this is a controversial topic in the field.
Normalization should be performed before imputation since bias may not be removed to detect group differences if imputation occurs prior to normalization<span class="citation" data-cites="mkTOg0QU"><sup><a href="#ref-mkTOg0QU" role="doc-biblioref">704</a></sup></span>.
Reasons for missing data include the peptide not being biologically present, being present but at too low of a quantity to be detected, or present at quantifiable abundance but misidentified or incorrectly undetected<span class="citation" data-cites="mkTOg0QU"><sup><a href="#ref-mkTOg0QU" role="doc-biblioref">704</a></sup></span>.
If the quantity is not at the detectable limit, the quantity is called censored and these values are missing not at random<span class="citation" data-cites="mkTOg0QU"><sup><a href="#ref-mkTOg0QU" role="doc-biblioref">704</a></sup></span>.
Imputing these censored values will lead to bias as the imputed values will be overestimated<span class="citation" data-cites="mkTOg0QU"><sup><a href="#ref-mkTOg0QU" role="doc-biblioref">704</a></sup></span>.
However, if the quantity is present at detectable limits but was missed due to a problem with the instrument, this peptide is missing completely at random (MCAR)<span class="citation" data-cites="mkTOg0QU"><sup><a href="#ref-mkTOg0QU" role="doc-biblioref">704</a></sup></span>.
While imputation of values that are MCAR using observed values would be a reasonable approach, censored peptides should not be imputed because their missingness is informative<span class="citation" data-cites="mkTOg0QU"><sup><a href="#ref-mkTOg0QU" role="doc-biblioref">704</a></sup></span>.
Peptides MCAR are a less frequent problem compared to censored peptides<span class="citation" data-cites="mkTOg0QU"><sup><a href="#ref-mkTOg0QU" role="doc-biblioref">704</a></sup></span>.
Understanding why the peptide is missing can be challenging<span class="citation" data-cites="mkTOg0QU"><sup><a href="#ref-mkTOg0QU" role="doc-biblioref">704</a></sup></span>, however there are techniques such as maximum likelihood model<span class="citation" data-cites="15hSlFzuu"><sup><a href="#ref-15hSlFzuu" role="doc-biblioref">717</a></sup></span> or logistic regression<span class="citation" data-cites="jmZQxVay"><sup><a href="#ref-jmZQxVay" role="doc-biblioref">718</a></sup></span> that may distinguish censored versus MCAR values.</p>
<p>Commonly used imputation methods for omics data are random forest (RF) imputation<span class="citation" data-cites="RwK8wIx4"><sup><a href="#ref-RwK8wIx4" role="doc-biblioref">719</a></sup></span>, k-nearest neighbors (kNN) imputation<span class="citation" data-cites="gcRMhkMM"><sup><a href="#ref-gcRMhkMM" role="doc-biblioref">720</a></sup></span>, and single value decomposition (SVD)<span class="citation" data-cites="PuoOF9VV"><sup><a href="#ref-PuoOF9VV" role="doc-biblioref">721</a></sup></span>.
Using the mean or median of the non-missing values for a variable is an easy approach to imputation but may lead to underestimating the true biological differences<span class="citation" data-cites="mkTOg0QU"><sup><a href="#ref-mkTOg0QU" role="doc-biblioref">704</a></sup></span>.
Choice of the appropriate imputation method is critical as how these missing values are filled in has a substantial impact on downstream analysis and conclusions<span class="citation" data-cites="7LvdRbeH"><sup><a href="#ref-7LvdRbeH" role="doc-biblioref">722</a></sup></span>.
In one study, RF imputation was the most accurate among nine imputation methods across several combinations of types and rates of missingness and does not require preprocessing (e.g., does not require normal distribution) for metabolomics data<span class="citation" data-cites="l8NlGvoh"><sup><a href="#ref-l8NlGvoh" role="doc-biblioref">723</a></sup></span>.
Another study found RF, among eight imputation methods, had the lowest normalized root mean squared error (NRMSE) between imputed values and the actual values when MCAR values were randomly replaced with missing values, followed by SVD and KNN using metabolomics data<span class="citation" data-cites="7LvdRbeH"><sup><a href="#ref-7LvdRbeH" role="doc-biblioref">722</a></sup></span>.
Lastly, a study found RF also had the lowest NRMSE when comparing seven imputation methods using a large-scale label-free proteomics dataset<span class="citation" data-cites="ZwtivmR2"><sup><a href="#ref-ZwtivmR2" role="doc-biblioref">724</a></sup></span>.
In general for imputation, we recommend using missforest, which is available as both a R and python package.</p>
<h3 id="batch-correction">Batch Correction</h3>
<p>Normalization is assumed to occur prior to batch effect correction<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.
Batch effect correction is still a critical step after normalization as proteins may still be affected by batch effects and diagnosing a batch effect may be easier once data is normalized<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.
Prior to performing any statistical analysis of data, we must start with distinguishing signals in the data due to biological versus batch effects.
A batch effect occurs when differences in preparation of samples and how data was acquired between batches results in altered quantities of peptides (or genes or metabolites) which results in reduced statistical power in detecting true differences<span class="citation" data-cites="l6oCMFXG l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a>,<a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.
This non-biological variability originates from the time of sample collection to peptide/protein quantification<span class="citation" data-cites="1wPiXyAt"><sup><a href="#ref-1wPiXyAt" role="doc-biblioref">701</a></sup></span> and is often a problem when working with large numbers of samples, involving multiple plates run by different technicians, on different instruments and/or using different reagent batches<span class="citation" data-cites="142g5e11W"><sup><a href="#ref-142g5e11W" role="doc-biblioref">725</a></sup></span>.
Results from these different batches ultimately need to be aggregated and data analysis to be performed on the whole dataset, so it may be difficult to measure and then control for exact changes due to non-biological variability once the data has been aggregated<span class="citation" data-cites="1wPiXyAt"><sup><a href="#ref-1wPiXyAt" role="doc-biblioref">701</a></sup></span>.
Batch correction methods remove technical variability, however they should not remove any true biological effect<span class="citation" data-cites="142g5e11W 1wPiXyAt"><sup><a href="#ref-1wPiXyAt" role="doc-biblioref">701</a>,<a href="#ref-142g5e11W" role="doc-biblioref">725</a></sup></span>.
Although it is agreed upon that these biases should be accounted for to prevent misleading conclusions, there is no one gold standard batch correction method.</p>
<p>Batch effects can manifest as continuous, such as from MS signal drift, or as discrete, such as a shift that affects the entire batch<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.
To visualize batch effects, one can plot the average intensity per sample in the order each was measured by the MS to see if intensities are shifted in a certain batch<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.
Measuring protein-protein correlations is another method to check for batch effects; if proteins within a batch are more correlated compared to those from other batches, there are likely batch effects<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.
Prior to batch correction, one should ensure the experimental design is not inherently flawed due to batch effects and whether a change in design should be implemented.
Studies spanning multiple days and experiments involving samples from different centers are vulnerable to batch effects<span class="citation" data-cites="mPnIAH38"><sup><a href="#ref-mPnIAH38" role="doc-biblioref">726</a></sup></span>.
One example of technical variability that may irreversibly flaw an experiment would be running samples at varying time points, or ‘as they came in’<span class="citation" data-cites="10L8J7McB"><sup><a href="#ref-10L8J7McB" role="doc-biblioref">727</a></sup></span>.
This problem can be circumvented by balancing biological groups in each batch<span class="citation" data-cites="10L8J7McB"><sup><a href="#ref-10L8J7McB" role="doc-biblioref">727</a></sup></span>.
Additionally, collection of samples at different institutions introduces non-biological variability due to differences in a multitude of conditions such as collection protocols, storage, and transportation<span class="citation" data-cites="1wPiXyAt"><sup><a href="#ref-1wPiXyAt" role="doc-biblioref">701</a></sup></span>.
A solution to this problem would be to evenly distribute samples between centers or batches<span class="citation" data-cites="1wPiXyAt"><sup><a href="#ref-1wPiXyAt" role="doc-biblioref">701</a></sup></span>.</p>
<p>There are several batch correction methods, the most popular method being Combating Batch Effects When Combining Batches of Gene Expression Microarray Data (ComBat), originally designed for genomics data<span class="citation" data-cites="1HahRBkyb 142g5e11W"><sup><a href="#ref-142g5e11W" role="doc-biblioref">725</a>,<a href="#ref-1HahRBkyb" role="doc-biblioref">728</a></sup></span>.
ComBat uses Bayesian inference to estimate batch effects across features in a batch and applies a modified mean shift, but requires peptides to be present in all batches which can lead to loss of a large number of peptides<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-142g5e11W" role="doc-biblioref">725</a></sup></span>.
Combat is available as a python package called pycombat.
Out of six batch correction methods using microarray data, ComBat was the best in reducing batch effects across several performance metrics and was effective using high dimensional data with small sample sizes<span class="citation" data-cites="cna9kCVu"><sup><a href="#ref-cna9kCVu" role="doc-biblioref">729</a></sup></span>.
ComBat may be more suitable for small datasets when the source of batch effects are known<span class="citation" data-cites="142g5e11W"><sup><a href="#ref-142g5e11W" role="doc-biblioref">725</a></sup></span>.
However if potential batch variables are not known or processing time or group does not adequately control for batch effects, surrogate variable analysis (SVA) may be used where the source of batch effect is estimated from the data<span class="citation" data-cites="142g5e11W mPnIAH38"><sup><a href="#ref-142g5e11W" role="doc-biblioref">725</a>,<a href="#ref-mPnIAH38" role="doc-biblioref">726</a></sup></span>.
A third option for batch effect correction uses negative control proteins to estimate unwanted variation, called “Remove Unwanted Variation, 2-step” (RUV-2)<span class="citation" data-cites="O7sERbhY"><sup><a href="#ref-O7sERbhY" role="doc-biblioref">730</a></sup></span>.
There are many additional batch effect correction methods for single cell data, such as mutual nearest neighbors<span class="citation" data-cites="kILCvszy"><sup><a href="#ref-kILCvszy" role="doc-biblioref">731</a></sup></span>, or Scanorama, which generalizes mutual nearest neighbors matching<span class="citation" data-cites="12Z3TdwON"><sup><a href="#ref-12Z3TdwON" role="doc-biblioref">732</a></sup></span>.</p>
<h3 id="assessment-of-transformed-data">Assessment of Transformed Data</h3>
<p>Prior to conducting any statistical analysis, the raw data matrix should be compared to the data after the above-described pre-processing steps have been performed to ensure bias is removed.
We can compare data using boxplots of peptide intensities from the raw data matrix versus corrected data in sample running order to look at batch associated patterns; after correction, we should see uniform intensities across batches<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.
We can also use clustering methods such as Principal Component analysis (PCA), Uniform Manifold Approximation and Projection (UMAP), or t-SNE (t-Distribute Stochastic Neighbor Embedding) and plot protein quantities colored by batches or technical versus biological samples to see how proteins cluster in space based on similarity.
We can measure the variability each PC contributes; we want to see similar variability among all PCs, however if see one PC contributing to overall variability highly then means variables are dependent<span class="citation" data-cites="98DcNOmC"><sup><a href="#ref-98DcNOmC" role="doc-biblioref">733</a></sup></span>.
tSNE and UMAP allow for non-linear transformations and allow for clusters to be more visually distinct<span class="citation" data-cites="98DcNOmC"><sup><a href="#ref-98DcNOmC" role="doc-biblioref">733</a></sup></span>.
Grouping of similar samples by batch or other non-biological factors, such as time or plate, indicates bias<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.
Quantitative measures of whether batch effects have been removed are principal variance components analysis (PVCA), which provides information on factors driving variance between biological and technical differences, and checking correlation of samples between different batches, within the same batch and between replicates.
When batch effects are present, samples in the same batch will have higher correlation than samples from different batches and between replicates<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.
Once batch effects are removed, proteins in the same batch should be correlated at the same level with proteins from other batches<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.
Similarity between technical replicates can be measured using pooled median absolute deviation (PMAD), pooled coefficient of variation (PCV) and pooled estimate of variance (PEV); high similarity would mean batch effects are removed and there is low non-biological effects<span class="citation" data-cites="1CKzlqjbM"><sup><a href="#ref-1CKzlqjbM" role="doc-biblioref">703</a></sup></span>.</p>
<p>Lastly, it is also important to show that batch correction leads to improvement in finding true biological differences between samples.
We can show the positive effect that batch correction has on the data by demonstrating reproducibility after batch correction.
One way to provide evidence for reproducibility is to show that prior to batch correction, there was no overlap between differentially expressed proteins between groups in one batch with those found between the same groups in another batch and, after batch correction, the differentially expressed proteins between the groups become the same between batches<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.
This applies generally datasets with large numbers (e.g., hundreds) of samples to allow for meaningful statistical comparisons<span class="citation" data-cites="l6oCMFXG"><sup><a href="#ref-l6oCMFXG" role="doc-biblioref">707</a></sup></span>.</p>
<h3 id="statistical-analysis">Statistical Analysis</h3>
<p>Once the above pre-processing steps have been applied to the dataset, we can investigate if any protein quantities differ between groups.
There is an urgent need for biomarkers for disease prediction and there is large potential for protein based biomarker candidates<span class="citation" data-cites="1GJKaTKqC"><sup><a href="#ref-1GJKaTKqC" role="doc-biblioref">734</a></sup></span>.
However, omics datasets are often limited due to having many more features than number of samples, which is termed the ‘curse of dimensionality’<span class="citation" data-cites="mkpI0rxh"><sup><a href="#ref-mkpI0rxh" role="doc-biblioref">699</a></sup></span>.
Attributes that are redundant or not informative can reduce the accuracy of a model<span class="citation" data-cites="IGJNx1OA"><sup><a href="#ref-IGJNx1OA" role="doc-biblioref">735</a></sup></span>.</p>
<p>Univariate statistical tests including t-tests and analysis of variance (ANOVA) provide p-values to allow ranking the importance of variables<span class="citation" data-cites="mkpI0rxh"><sup><a href="#ref-mkpI0rxh" role="doc-biblioref">699</a></sup></span>.
T-tests are used in pairwise comparisons, and ANOVA is used when there are multiple groups to ask whether any group is different from the rest.
After ANOVA, the Tukey’s posthoc test can reveal which pairwise differences are present among the multiple groups that were compared.
There are many posthoc tests that can be used, and guidance from an expert statistician is suggested.
Wilcoxon rank-sum tests can be used if the data are still not normal after the above approaches and therefore violates the assumptions required for a t-test.
Kruskal-Wallis test is the non-parametric version of ANOVA useful for three or more groups when assumptions of ANOVA are violated.</p>
<p>Data can be reduced using a feature selection method, which includes either feature subset selection, where irrelevant features are removed, or feature extraction, where there is a transformation that generates new, aggregated variables and do not lead to loss of information<span class="citation" data-cites="mkpI0rxh"><sup><a href="#ref-mkpI0rxh" role="doc-biblioref">699</a></sup></span>.
An example of a commonly used multivariate feature extraction method using proteomic data is principal component analysis (PCA)<span class="citation" data-cites="mkpI0rxh"><sup><a href="#ref-mkpI0rxh" role="doc-biblioref">699</a></sup></span>.</p>
<p>Multiple hypothesis test happens in proteomics when we make many statistical tests to check for differences of many measured proteins betweenc conditions.
For example, if we measure 1000 proteins between a pair of conditions, we may perform 1000 t-tests.
By random chance, even in the absence of true protein quantity changes, 10 of these tests will produce a p-value less than 0.01.
These would be false positives (i.e., a p-value will appear significant by chance)<span class="citation" data-cites="98DcNOmC"><sup><a href="#ref-98DcNOmC" role="doc-biblioref">733</a></sup></span> and multiple testing correction should be applied to main the overall false positive rate at less than a specified cut-off<span class="citation" data-cites="mkpI0rxh"><sup><a href="#ref-mkpI0rxh" role="doc-biblioref">699</a></sup></span>.
There are many methods for multiple testing correction.
Benjamini-Hochberg correction is less stringent than the Bonferroni correction, which leads to too many false negatives, and thus is a more commonly used multiple testing correction method<span class="citation" data-cites="mkpI0rxh"><sup><a href="#ref-mkpI0rxh" role="doc-biblioref">699</a></sup></span>.</p>
<p>Volcano plots allow visualization of differentially abundant proteins by displaying the negative log of the adjusted p-value as a function of the log fold change, a measure of effect size, for each protein.
Points with larger y axis values are more statistically significant and those further away from zero on the x axis have a larger fold change.<br />
There are two methods for identifying differentially expressed proteins.
The first method involves a combined adjusted p-value cut-off (y axis) and fold change cut-off (x axis) to create a ‘square cut-off’<span class="citation" data-cites="98DcNOmC"><sup><a href="#ref-98DcNOmC" role="doc-biblioref">733</a></sup></span>.
The second involves a non-linear cut-off, where a systematic error is added to all the standard deviations used in the t-tests<span class="citation" data-cites="98DcNOmC"><sup><a href="#ref-98DcNOmC" role="doc-biblioref">733</a></sup></span>.</p>
<p>There are other statistical tests to consider for quantitative proteomics data.
Another popular statistical method in proteomics when dealing with high dimensional data is lasso linear regression, which removes regression coefficients from the model by applying a penalty parameter<span class="citation" data-cites="LCdBFAHO"><sup><a href="#ref-LCdBFAHO" role="doc-biblioref">736</a></sup></span>.
Bayesian models are an emerging technique for protein based biomarker discovery that are more powerful than standard t-tests<span class="citation" data-cites="LCdBFAHO"><sup><a href="#ref-LCdBFAHO" role="doc-biblioref">736</a></sup></span> and have outperformed linear models<span class="citation" data-cites="LCdBFAHO ZhPBaP2m"><sup><a href="#ref-LCdBFAHO" role="doc-biblioref">736</a>,<a href="#ref-ZhPBaP2m" role="doc-biblioref">737</a></sup></span>.
Bayesian models incorporate external information into the prior distribution; for example, knowledge of peptides that usually have more technical variability are assigned a less informative prior<span class="citation" data-cites="LCdBFAHO"><sup><a href="#ref-LCdBFAHO" role="doc-biblioref">736</a></sup></span>.
Prior to implementing machine learning (ML), one can start with the simpler models, such as linear regression or naïve Bayes<span class="citation" data-cites="1GJKaTKqC"><sup><a href="#ref-1GJKaTKqC" role="doc-biblioref">734</a></sup></span>.</p>
<p>In general for statistical tests, can suggest using the standard test available in base R or in python packages scipy or statsmodels.</p>
<h3 id="machine-learning-ml">Machine Learning (ML)</h3>
<p>Despite the efficacy of ML for finding signals in a high dimensional feature space to distinguish between classes<span class="citation" data-cites="1GJKaTKqC"><sup><a href="#ref-1GJKaTKqC" role="doc-biblioref">734</a></sup></span>, the application of ML to proteomic data analysis is still in its early stages<span class="citation" data-cites="1GJKaTKqC"><sup><a href="#ref-1GJKaTKqC" role="doc-biblioref">734</a></sup></span> as only 2% of proteomics studies involve ML<span class="citation" data-cites="1KFEQehj"><sup><a href="#ref-1KFEQehj" role="doc-biblioref">738</a></sup></span>.
The reason for such sparse usage of ML in proteomics data is the need for very large datasets comprising 100s of samples, which is still rare in proteomics.</p>
<p>Supervised classification is the most common type of ML used for proteomic biomarker discovery, where an algorithm has been trained on variables to predict the class labels of unseen test data<span class="citation" data-cites="IGJNx1OA"><sup><a href="#ref-IGJNx1OA" role="doc-biblioref">735</a></sup></span>.
Supervised means the class labels, such as disease versus controls, is known<span class="citation" data-cites="1KFEQehj"><sup><a href="#ref-1KFEQehj" role="doc-biblioref">738</a></sup></span>.
Decision trees are common model choice due to their many advantages: variables are not assumed to be linearly related, models are able to rank more important variables on their own, and interactions between variables do not need to be pre-specified by the user<span class="citation" data-cites="LCdBFAHO"><sup><a href="#ref-LCdBFAHO" role="doc-biblioref">736</a></sup></span>.
There are three phases of model development and evaluation<span class="citation" data-cites="7uMxquUf"><sup><a href="#ref-7uMxquUf" role="doc-biblioref">739</a></sup></span>.
In the first step, the dataset is split into training and testing splits, commonly 70% training and 30% testing.
Second, the model is constructed using only the training data, which is further subdivided into training and test sets.
During this process, an internal validation strategy, or cross-validation (CV), is employed<span class="citation" data-cites="mkpI0rxh"><sup><a href="#ref-mkpI0rxh" role="doc-biblioref">699</a></sup></span>.
Commonly used CV methods in proteomics are k-fold and leave-one-out cross-validation<span class="citation" data-cites="mkpI0rxh"><sup><a href="#ref-mkpI0rxh" role="doc-biblioref">699</a></sup></span>.
The final step is to evaluate the model on the testing set that was held-out in step one.
There should not be overlap between the training and testing data, and the testing data should only be evaluated once after all training has been completed.
The dataset used for training and testing should be representative of the population that is to be eventually tested.
If underrepresented groups are lacking from models during training, these models will not generalize to these populations<span class="citation" data-cites="dqOuwBB4"><sup><a href="#ref-dqOuwBB4" role="doc-biblioref">740</a></sup></span>.
Proteomic data and patient specific factors derived from the electronic health record (EHR) like age, race, and smoking status can be employed as inputs to a model<span class="citation" data-cites="1GJKaTKqC"><sup><a href="#ref-1GJKaTKqC" role="doc-biblioref">734</a></sup></span>.
However, addition of EHR data may not be informative in some instances; in studying Alzheimer’s Disease, adding these patient specific variables were informative for non-Hispanic white participants, but not for African Americans<span class="citation" data-cites="dqOuwBB4"><sup><a href="#ref-dqOuwBB4" role="doc-biblioref">740</a></sup></span>.</p>
<p>A common mistake in proteomics ML studies is allowing the test data to leak into the feature selection step<span class="citation" data-cites="LamN4xBz 1KFEQehj"><sup><a href="#ref-1KFEQehj" role="doc-biblioref">738</a>,<a href="#ref-LamN4xBz" role="doc-biblioref">741</a></sup></span>.
It has been reported that 80% of ML studies on the gut microbiome performed feature selection using all the data, including test data<span class="citation" data-cites="LamN4xBz"><sup><a href="#ref-LamN4xBz" role="doc-biblioref">741</a></sup></span>.
Including the testing data in the feature selection step leads to development of an artificially inflated model<span class="citation" data-cites="LamN4xBz"><sup><a href="#ref-LamN4xBz" role="doc-biblioref">741</a></sup></span> that is overfit on the training data and performs poorly on new data<span class="citation" data-cites="1GJKaTKqC"><sup><a href="#ref-1GJKaTKqC" role="doc-biblioref">734</a></sup></span>.
Feature selection should occur only on the training set and final model performance should be reported using the unseen testing set.
The number of samples should be ten times the number of features to make statistically valid comparisons, however this may not be possible in many cases<span class="citation" data-cites="LamN4xBz"><sup><a href="#ref-wJCp8LYF" role="doc-biblioref">742</a></sup></span>.
If a study is limited by its number of samples, one can perform classification without feature selection<span class="citation" data-cites="1KFEQehj"><sup><a href="#ref-LamN4xBz" role="doc-biblioref">741</a></sup></span>.</p>
<p>Pitfalls also arise when a ML classifier is trained using an imbalanced dataset<span class="citation" data-cites="7uMxquUf"><sup><a href="#ref-7uMxquUf" role="doc-biblioref">739</a></sup></span>.
Proteomics biomarker studies commonly have imbalanced groups, where the number of samples in one group is drastically different from another group.
Most ML algorithms assume balanced number of samples per class and not accounting for these differences can lead to reduced performance and construction of a biased classifier<span class="citation" data-cites="16jVcFgBv"><sup><a href="#ref-16jVcFgBv" role="doc-biblioref">743</a></sup></span>.</p>
<p>Care should be practiced when choosing an appropriate metric when dealing with imbalanced data.
A high accuracy may be meaningless in the case of imbalanced classification; the number of correction predictions will be high even with a blind guess for the majority class<span class="citation" data-cites="65gghDCg"><sup><a href="#ref-65gghDCg" role="doc-biblioref">744</a></sup></span>.
F1 score, Matthews correlation coefficient (MCC), and area under the precision recall curve (AUPR) are preferred metrics for imbalanced data classification<span class="citation" data-cites="4k6ers10 LB4mDas6"><sup><a href="#ref-4k6ers10" role="doc-biblioref">745</a>,<a href="#ref-LB4mDas6" role="doc-biblioref">746</a></sup></span>.<br />
MCC, for example, is preferred since it is only high if the model predicts correctly on both the positive and negative classes<span class="citation" data-cites="7uMxquUf"><sup><a href="#ref-7uMxquUf" role="doc-biblioref">739</a></sup></span>.
Over- and under-sampling to equalize the number of samples in classes are potential methods to address class imbalance, but can be ineffective or even detrimental to the performance of the model<span class="citation" data-cites="16jVcFgBv"><sup><a href="#ref-16jVcFgBv" role="doc-biblioref">743</a></sup></span>.
These sampling methods may lead to a poorly calibrated model that overestimates the probability of the minority class samples and reduce the model’s applicability to clinical practice<span class="citation" data-cites="65gghDCg"><sup><a href="#ref-65gghDCg" role="doc-biblioref">744</a></sup></span>.</p>
<h2 class="page_break_before" id="protein-sequence-databases">14. Protein Sequence Databases</h2>
<h3 id="where-do-you-get-them">Where do you get them?</h3>
<p>For those looking for guidance on where to obtain a database for their organism of interest quickly, we recommend going to uniprot.org and using their “proteomes”: https://www.uniprot.org/proteomes?query=*.
After selecting the proteome of interest, for most applications, we recommend clicking the “download one sequence per gene (FASTA)”, for example on the left of this page for E coli: https://www.uniprot.org/proteomes/UP000000625.</p>
<h3 id="protein-database-sources-and-types">Protein Database Sources and Types</h3>
<p>Many mass spectrometry-based proteomic techniques use search algorithms that require a defined theoretical search space to identify peptide sequences based on precursor mass and peptide fragmentation patterns, which are then used to infer the presence and abundance of a protein.
Traditionally these databases are used with DDA database search algorithms, but they can also be used with new MS/MS spectra prediction algorithms to predict spectral libraries for DIA data analysis.
The search space is calculated from the potential proteins in a sample, which includes the proteome (often a single species) and expected contaminants.
This is called database searching and the flat file of protein sequences in FASTA format acts as a protein database.
In this section, we will describe major resources for proteome FASTA files (protein sequence collections), how to retrieve them, and suggested best practices for preserving FASTA file provenance to improve reproducibility.</p>
<p>In general, FASTA sequence collections can be retrieved from three central clearing houses: UniProt, RefSeq, and Ensembl.
These will be discussed separately below as they each have specific design goals, data products, and unique characteristics.
It is important to learn the following three points for each resource: the source of the underlying data, canonical versus non-canonical sequences, and how versioning works.
These points, along with general best practices, such as using a taxonomic identifier, are essential to understand and communicate search settings used in analyses of proteomic datasets.
Finally, it is critical to understand that sequence collections from these three resources are not the same, nor do they offer the same sets of species.</p>
<p>Key terminology may vary between resources, so these terms are defined here.
The term “taxon identifier” is used across resources and is based on the NCBI taxonomy database.
Every taxonomic node has a number, e.g., <em>Homo sapiens</em> (genus species) is 9606 and Mammalia (class) is 40674.
This can be useful when retrieving and describing protein sequence collections.
Another term used is “annotation”, which has different meanings in different contexts.
Broadly, a “genome annotation” is the result of an annotation pipeline to predict coding sequences, and often a gene name/symbol if possible.
Two examples are MAKER<span class="citation" data-cites="KDnDuk25"><sup><a href="#ref-KDnDuk25" role="doc-biblioref">747</a></sup></span> and the RefSeq annotation pipeline<span class="citation" data-cites="12mAW7BXf"><sup><a href="#ref-12mAW7BXf" role="doc-biblioref">748</a></sup></span>.
Alternatively, “protein annotation” (or gene annotation) often refers to the annotation of proteins (gene products) using names and ontology (i.e., protein names, gene names/symbols, functional domains, gene onotology, keywords, etc.).
Protein annotation is termed “biocuration” and described in detail by UniProt<span class="citation" data-cites="BRlMNUOW"><sup><a href="#ref-BRlMNUOW" role="doc-biblioref">749</a></sup></span>.
Lastly, there are established minimum reporting guidelines for referring to FASTA files established in MIAPE: Mass Spectrometry Informatics that are taxon identifier and number of sequences<span class="citation" data-cites="1EbCD9r1v paTtvUb8"><sup><a href="#ref-1EbCD9r1v" role="doc-biblioref">750</a>,<a href="#ref-paTtvUb8" role="doc-biblioref">751</a></sup></span>.
The FASTA file naming suggestions below are not official but are suggested as a best practice.</p>
<h5 id="uniprot">UniProt</h5>
<p>The Universal Protein Resource (UniProt)<span class="citation" data-cites="16uefmiYk nULTgvHl vPdD3ylm 1ENygunzG"><sup><a href="#ref-16uefmiYk" role="doc-biblioref">752</a>–<a href="#ref-1ENygunzG" role="doc-biblioref">755</a></sup></span>, has three different products: UniProt Knowledgebase (UniProtKB), the UniProt Reference Clusters (UniRef), and the UniProt Archive (UniParc).
The numerous resources and capabilities associated with the UniProt are not explored in this section, but these are well described on UniProt’s website.
UniProtKB is the source of proteomes across the Tree of Life and is the resource we will be describing herein.
There are broadly two types of proteome sequence collections: Swiss-Prot/TrEMBL and designated proteomes.
The Swiss-Prot/TrEMBL type can be understood by discussing how data is integrated into UniProt.
Most protein sequences in UniProt are derived from coding sequences submitted to EMBL-Bank, GenBank and DDBJ.
These translated sequences are initially imported into TrEMBL database, which is why TrEMBL is also termed “unreviewed”.
There are other sources of protein sequences, as described by UniProt<span class="citation" data-cites="1BIF8v4YE"><sup><a href="#ref-1BIF8v4YE" role="doc-biblioref">756</a></sup></span>.
These include the Protein Data Bank (PDB), direct protein sequencing, sequences derived from the literature, gene prediction (from sources such as Ensembl) or in-house prediction by UniProt itself.
Protein sequences can then be manually curated into the Swiss-Prot database using multiple outlined steps (described in detail by UniProt here<span class="citation" data-cites="Q2scu90J"><sup><a href="#ref-Q2scu90J" role="doc-biblioref">757</a></sup></span>) and is why Swiss-Prot is also termed “reviewed”.
Note that more than one TrEMBL entry may be removed and replaced by a single Swiss-Prot entry during curation.
A search of “taxonomy_id:9606” at UniProtKB will retrieve both the Swiss-Prot/reviewed and TrEMBL/unreviewed sequences for Homo sapiens.
The entries do not overlap, so users often either use just Swiss-Prot or Swiss-Prot combined with TrEMBL, the latter being the most exhaustive option.
With ever-increasing numbers of high-quality genome assemblies processed with robust automated annotation pipelines, TrEMBL entries will contain higher quality protein sequences than in the past.
In other words, if a mammal species has 20 000 to 40 000 entries in UniProtKB and many of these are TrEMBL, users should be comfortable using all the protein entries to define their search space (more on this later when discussing proteomes at UniProtKB).
Determining the expected size of a well-annotated proteome requires additional knowledge, but tools to answer these questions continue to improve.
As more and more genome annotations are generated, the backlog of manual curation continues to increase.
However, automated genome annotations are also rapidly improving, blurring the line between Swiss-Prot and TrEMBL utility.</p>
<p>The second type of protein sequence collections available at UniProtKB are designated proteomes, with subclasses of “proteome”, “reference proteome” or “pan-proteome”.
As defined by UniProt, a proteome is the set of proteins derived from the annotation of a completely sequenced genome assembly (one proteome per genome assembly).
This means that a proteome will include both Swiss-Prot and TrEMBL entries present in a single genome annotation, and that all entries in the proteome can be traced to a single complete genome assembly.
This aids in tracking provenance as assemblies change, and metrics of these assemblies are available.
These metrics include Benchmarking Universal Single-Copy Ortholog (BUSCO) score, and “Completeness” as Standard, Close Standard or Outlier based on the Complete Proteome Detector (CPD).
Given the quality of genome annotation pipelines, using a proteome as a FASTA file for a species is the preferred method of defining search spaces now.
Outside of humans, no higher eukaryotic Swiss-Prot sequence collections are complete enough for use in proteomics analyses, but this does not mean that the available Swiss-Prot plus TrEMBL protein sequence collection precludes accurate proteomic data analysis.
Lastly, the difference between reference proteome and proteome is used to highlight model organisms or organisms of interest, but not to imply improved quality.
UniProt also has support for the concept of “pan proteomes” (consensus proteomes for a closely related set of organisms) but this is mostly used for bacteria (e.g., strains of a given species will share a pan proteome).</p>
<p>When retrieving protein sequence collections as Swiss-Prot/TrEMBL or designated proteomes, there is an option of downloading “FASTA (canonical)” or “FASTA (canonical &amp; isoform)”.
The later includes additional manually annotated isoforms for Swiss-Prot sequences.
Each Swiss-Prot entry has one canonical sequence chosen by the manual curator.
Any additional sequence variants (mostly from alternative slicing) are annotated as differences with respect to the canonical sequence.
Specifying “canonical” will select only one protein sequence per Swiss-Prot entry while specifying “canonical &amp; isoforms” will download additional protein sequences by including isoforms for Swiss-Prot entries.
Recently, an option to “download one protein sequence per gene (FASTA)” has been added.
These FASTA files include Swiss-Prot and TrEMBL sequences to number about 20 000 protein sequences for a wide range of higher eukaryotic organisms.</p>
<p>The number of additional isoforms in a proteome varies considerably by species.
In the human, mouse, and rat proteomes of the total number of entries, 25 %, 40 % and 48 % are canonical, respectively.
The choice of including isoforms is related to the search algorithm and experimental goals.
For instance, if differentiating isoforms is relevant, they should be included otherwise they will not be detected.
In cases where isoforms are present in the FASTA (evident by shared protein names) but these cannot be removed prior to downloading (e.g., California sea lion, <em>Zalophus californianus</em>, proteome UP000515165, release 2023_04 has no options for downloading one protein sequence per gene), non-redundant FASTA files can be manually generated (i.e., “remove_duplicates.py” via<span class="citation" data-cites="16BHXvDGS"><sup><a href="#ref-16BHXvDGS" role="doc-biblioref">758</a></sup></span>).
If possible, retrieving canonical protein sequences via proteomes is the most straight forward approach and in general appropriate for most search algorithms, versus the method of searching and downloading Swiss-Prot and/or TrEMBL entries.</p>
<p>Though FASTA files are the typical input of many search algorithms, UniProt also offers an XML and GFF format download.
In contrast to the flat FASTA file format, the XML format includes sequence information as well as associated information like PTMs, which is used in some search algorithms like MetaMorpheus<span class="citation" data-cites="1GfGl6YGi"><sup><a href="#ref-1GfGl6YGi" role="doc-biblioref">759</a></sup></span>.</p>
<p>Once a protein sequence collection has been selected and retrieved, how can the file be named and report this to others in a way that allows them to reproduce the retrieval?
The minimum reporting information is the taxon identified and number of sequences used<span class="citation" data-cites="1EbCD9r1v paTtvUb8"><sup><a href="#ref-1EbCD9r1v" role="doc-biblioref">750</a>,<a href="#ref-paTtvUb8" role="doc-biblioref">751</a></sup></span>.
The following naming format (and those below) augments this and is suggested for UniProtKB FASTA files (the use of underscores or hyphens is not critical):</p>
<p>[common or scientific name]-[taxon id]-uniprot-[swiss-prot/trembl/proteome]-[UP# if used]-[canonical/canonical plus isoform]-[release]</p>
<p>example of a <em>Homo sapiens</em> (human) protein fasta from UniProtKB:</p>
<p>Human-9606-uniprot-proteome-UP000005640-canonical-2023_04.fasta</p>
<p>The importance of the taxon identifier has already been described above and is a consistent identifier across time and shared across resources.
The choices of Swiss-Prot and TrEMBL in some combination was discussed above, and Proteome can be “proteome”, “reference proteome” or “pan-proteome”.
The proteome identifier (‘UP’ followed by 9 digits) is conserved across releases, and release information should also be included.
A confusing issue to newcomers is what the term “release” means.
This is a year_month format (e.g., 2023_04), but it is not the date a FASTA file was downloaded or created, nor does it imply there are monthly updates.
This release “date” is a traceable release identifier that is listed on UniProt’s website.
Including all this information ensures that the exact provenance of a FASTA file is known and allows the FASTA file to be regenerated.</p>
<h5 id="refseq">RefSeq</h5>
<p>NCBI is a clearing house of numerous types of data and databases.
Specific to protein sequence collections, NCBI Reference Sequence Database (RefSeq) provides annotated genomes across the Tree of Life.
The newly developed NCBI Datasets portal<span class="citation" data-cites="uPHDl2Vm"><sup><a href="#ref-uPHDl2Vm" role="doc-biblioref">760</a></sup></span> is the preferred method for accessing the myriad of NCBI data products, though protein sequence collections can also be retrieved from RefSeq directly<span class="citation" data-cites="tg8zlLqI xpZWsuGI"><sup><a href="#ref-tg8zlLqI" role="doc-biblioref">761</a>,<a href="#ref-xpZWsuGI" role="doc-biblioref">762</a></sup></span>.
Like UniProt described above, most of the additional functionality and information available through NCBI Datasets and RefSeq will not be described here, although the Eukaryotic RefSeq annotation dashboard<span class="citation" data-cites="BfmLo0uT"><sup><a href="#ref-BfmLo0uT" role="doc-biblioref">763</a></sup></span> is a noteworthy resource to monitor the progress of new or re-annotations.
We recommend exploring the resources available from NCBI<span class="citation" data-cites="fWjGnKWy"><sup><a href="#ref-fWjGnKWy" role="doc-biblioref">764</a></sup></span>, utilizing their tutorials and help requests.</p>
<p>RefSeq is akin to the “proteome” sequence collection from UniProtKB, where a release is based on a single genome assembly.
If a more complete genome assembly is deposited or additional secondary evidence (e.g., RNA sequencing) is deposited, RefSeq can update the annotation with a new annotation release.
Every annotation release will have an annotation report that contains information on the underlying genome assembly, the new genome annotation, secondary evidence used, and various statistics about what was updated.
The current annotation release is referred to as the “reference annotation”, but each annotation is numbered sequentially starting at 100 (the first release), though a recent naming change has abandoned the sequential release numbering and instead is the RefSeq assembly “-RS” and then the year month when it was annotated (e.g., the current human reference annotation is GCF_000001405.40-RS_2023_10).
Certain species are on scheduled re-annotation, like human and mouse, while other species are updated as needed based on new data and community feedback (ex. release 100 of taxon 9704 was in 2018, but a more contiguous genome assembly resulted in re-annotation to release 101 in 2020).
This general process for new and existing species is described in Heck and Neely<span class="citation" data-cites="17gB3bs6x"><sup><a href="#ref-17gB3bs6x" role="doc-biblioref">765</a></sup></span>.</p>
<p>Since RefSeq is genome assembly-centric, its protein sequence collections are retrieved for each species.
This contrasts with being able to use a higher-level taxon identifier like 40674 (Mammalia) in UniProt to retrieve a single FASTA.
To accomplish this same search in NCBI Datasets requires a Mammalia search, followed by browsing all 2847 genomes and then filtering the results to reference genomes with RefSeq annotations, and those resulting 223 could be bulk downloaded, though this will still be 223 individual FASTA files.
It is possible to download a single FASTA from an upper-level taxon identifier using the NCBI Taxonomy Browser, though this service may be redundant with the new NCBI Datasets portal.
Given the constant development of NCBI Datasets, these functionalities may change, but the general RefSeq philosophy of single species FASTA should be kept in mind.
Likewise, when retrieving genome annotations there is no ability to specify canonical entries only, but it is possible to use computational tools to remove redundant entries (“remove_duplicates.py” from<span class="citation" data-cites="16BHXvDGS"><sup><a href="#ref-16BHXvDGS" role="doc-biblioref">758</a></sup></span>).</p>
<p>Similar to the UniProtKB FASTA file naming suggestion, the following naming format is suggested for RefSeq protein sequence collection FASTA (the use of underscores or hyphens is not critical):</p>
<p>[common or scientific name]-[taxon id]-refseq-[release number]</p>
<p>Example of a <em>Equus caballus</em> (horse) protein FASTA from RefSeq:</p>
<p>Equus_caballus-9796-refseq-103.fasta</p>
<p>The release number starts at 100 and is consecutively numbered.
Note, the human releases previously had a much longer number to be included (e.g., NCBI Release 109.20211119), then began following a consecutive numbering for Release 110, but have now switched to the new format related to assembly and annotation date.
Also, in a few species (Human, Chinese hamster, and Dog, currently), there is a reference and an alternate assembly, both with an available annotation.
In these cases, including the underlying assembly identifier would be needed. Note that when you retrieve the protein FASTA from NCBI it will include two more identifiers that aren’t required in the file name since it can be determined from the taxon identifier and release number.
These are the genome assembly used (this is generated by the depositor and follows no naming scheme) and the RefSeq identifier (GCF followed by a number string).
These aren’t essential for FASTA naming, but are for comparing between UniProt, RefSeq and Ensembl when the same underlying assembly is used (or not, indicating how up to date one is versus the other).</p>
<h5 id="ensembl">Ensembl</h5>
<p>There are two main web portals for Ensembl sequence collections: the Ensembl genome browser<span class="citation" data-cites="69obv795"><sup><a href="#ref-69obv795" role="doc-biblioref">766</a></sup></span> has vertebrate organisms and the Ensemble Genome project<span class="citation" data-cites="1EGyqGdLr"><sup><a href="#ref-1EGyqGdLr" role="doc-biblioref">767</a></sup></span> has specific web portals for different non-vertebrate branches of the Tree of Life.
This contrasts with NCBI and UniProt where all branches are centrally available.
Recently, Ensembl has created a new portal “Rapid Release” focusing on quickly making annotations available (replacing the “Pre-Ensemble” portal), albeit without the full functionality of the primary Ensembl resources.
Overall, Ensembl provides diverse comparative and genomic tools that should be explored, but, specific to this discussion, they provide species-specific genome annotation products similar to RefSeq.</p>
<p>To retrieve a protein sequence collection from Ensembl at any of the portals, a species can be searched using a name, which will then have taxon identifier displayed (but searching by identifier is not readily apparent).
From the results you can select your species and follow links for genome annotation.
Caution should be used when browsing the annotation products since the protein coding sequence (abbreviated “cds”) annotations are nucleic acid sequences (a useable via 3-frame translation if using certain software), while actual translated peptide sequences are in the “pep” folders.
The pep folders contain file names with “ab initio” and “all” in the FASTA file names (file extensions are “fa” for FASTA and “gz” indicating gzip compression algorithm), while there may only be one pep product for certain species in the “Rapid Release” portal.
The “ab initio” FASTA files contain mostly predicted gene products.
The “all” FASTA files are the usable protein sequence collections.
Ensembl FASTA files usually have some protein sequence redundancy.</p>
<p>Ensembl provides a release number for all the databases within each portal.
Similar to the UniProt file naming suggestion, the following naming format is suggested for Ensembl protein sequence collection FASTA (the use of underscores or hyphens is not critical):</p>
<p>[common or scientific name]-[taxon id]-ensembl-[abinitio/all]-[rapid]-[release number]</p>
<p>Example of a <em>Sus scrofa</em> (pig) protein FASTA from Ensembl:</p>
<p>Pig-9823-ensembl-all-106.fasta</p>
<p>Similar to the FASTA download from RefSeq, the downloaded file name can include additional identifying information related to the underlying genome assembly.
Again, this is not required for labeling, but is useful to easily compare assembly versions.</p>
<p>Since much of the data from Ensembl is also regularly processed into UniProt, using UniProt sequence collections instead may be preferred.
That said, they are not on the same release schedule nor will the FASTA files contain the same proteins.
Ensembl sequences still must go through the established protein sequence pipeline at UniProt to remove redundancy and conform to UniProt accession and FASTA header formats.
Moreover, the gene-centric and comparative tools built into Ensembl may be more experimentally appropriate and using an Ensembl protein sequence collection can better leverage those tools.</p>
<h4 id="other-resources">Other resources</h4>
<p>There are other locations of protein sequence collections, and these will likewise have different FASTA file formatting; sequences may have unusual characters, and formats of accessions and FASTA header lines may need to be reformatted to be compatible with search software.
These alternatives include institutes like the Joint Genome Institute’s microbial genome clearing house, species-specific community resource (e.g., PomBase, FlyBase, WormBase, TryTrypDB, etc.), and one-off websites tenuously hosting in-house annotations.
It is preferred to use protein sequence collection from the main three sources described here, since provenance can be tracked, and versions maintained.
It is beyond the scope of this discussion to address other genome annotation resources, how they are versioned, or the best way to describe FASTA files retrieved from those sources.
In these cases, defaulting to the minimum requirements of listing number of entries and supplying the FASTA along with data are necessary.</p>
<h4 id="contaminants">Contaminants</h4>
<p>Samples are rarely comprised of only proteins from the species of interest.
There can be protein contamination during sample collection or processing.
This may include proteins from human skin, wool from clothing, particles from latex, and even trypsin itself, all of which contain proteins that can be digested along with the intended sample and analyzed in the mass spectrometer.
Avoiding unwanted matching of mass spectra originating from contaminant proteins to the cellular proteins due to sequence similarities is important to the identification and quantitation of as many cellular proteins as possible.
To avoid these spectra matching to the wrong peptides, repositories of supplementary sequences for contaminant proteins have been added to a reference database for MS data searches.
Appending a contaminants database to the reference database allows the identification of peptides that are not exclusive to one species.</p>
<p>As early as 2004, The Global Proteome Machine was providing a protein sequence collection of these common Repository of Adventitious Proteins (cRAP), while another contaminant list was published in 2008<span class="citation" data-cites="OeXy2hrh"><sup><a href="#ref-OeXy2hrh" role="doc-biblioref">768</a></sup></span>.
The current cRAP version (v1.0) was described in 2012<span class="citation" data-cites="jwgBjKOg"><sup><a href="#ref-jwgBjKOg" role="doc-biblioref">769</a></sup></span> and is still widely in use today.
cRAP is the contaminant protein list used in nearly all modern database searching software, though the documentation, versioning or updating of many of these “built-in” contaminant sequence collections is difficult to follow.
There is also another contaminant sequence collection distributed with MaxQuant.
Together, the cRAP and MaxQuant contaminant protein sequence collections are found in some form across most software, including MetaMorpheus and Philosopher (available in FragPipe)<span class="citation" data-cites="WdwXXrRx"><sup><a href="#ref-WdwXXrRx" role="doc-biblioref">770</a></sup></span>.
This list of known frequently contaminating proteins can either be automatically included by the software or can be retrieved as a FASTA to be used along with the primary search FASTA(s).
Recently the Hao Lab has revisited these common contaminant sequences in an effort to update the protein sequences (ProtContLib), test their utility on experimental data, and add or remove entries<span class="citation" data-cites="m8bjbzJ5"><sup><a href="#ref-m8bjbzJ5" role="doc-biblioref">771</a></sup></span>.</p>
<p>In addition to these environmentally unintended contaminants, there are known contaminants that also have available protein sequence collections (or can be generated using the steps above) and should be included in the search space.
These can include the media cells were grown in (e.g., fetal bovine serum<span class="citation" data-cites="CnwgtVyY 1CIFGLtYQ"><sup><a href="#ref-CnwgtVyY" role="doc-biblioref">772</a>,<a href="#ref-1CIFGLtYQ" role="doc-biblioref">773</a></sup></span>, food fed to cells/animals (e.g., <em>Caenorhabditis elegans</em> grown on <em>Escherichia coli</em>) or known non-specific binders in affinity purification (i.e., CRAPome<span class="citation" data-cites="wUllSafI"><sup><a href="#ref-wUllSafI" role="doc-biblioref">774</a></sup></span>).
The common Repository of Fetal Bovine Serum Proteins (cRFP)<span class="citation" data-cites="EyM6Y0Xm"><sup><a href="#ref-EyM6Y0Xm" role="doc-biblioref">775</a></sup></span> are protein lists of common protein contaminants and fetal serum bovine sequences used to reduced the number of falsely identified proteins in cell culture experiments.
Cells washed or cultured in contaminant free media before harvest or the collection of secreted proteins depletes most high abundance contaminant proteins but the sequence similarity between contaminant and secreted proteins can cause false identifications and overestimation of the true protein abundance leading to wasted resources and time on validating false leads.
As emphasized throughout this section, accurately defining the search space is essential for accurate results and, especially in the case of contaminants, requires knowledge of the experiment and sample processing to adequately define possible background proteins.</p>
<h3 id="choosing-the-right-database">Choosing the right database</h3>
<p>Proteomics data analysis requires carefully matching the search space (defined by the database choice) with the expected proteins.
A properly chosen database will minimize false positives and false negatives.
Choosing a database that is too large will increase the number of false positives, or decoy hits, which in turn will reduce the total number of identifiable proteins.
For this reason it is ill advised to search against all possible protein sequences ever predicted from any genomic sequence.
On the other hand, choosing a database that is too small may increase false negatives, or missed protein identifications, because in order for a protein to be identified it must be present in the database.
Some search algorithms can self-correct when a database is overly large such that higher identity thresholds are required for identification to minimize false positives (e.g., Mascot), while smaller experiment-specific search spaces (also referred to as “subsets”) can have unintended effects on false positives if not managed appropriately<span class="citation" data-cites="qgZxt4hu 12o52Fhcj Lovg2liS"><sup><a href="#ref-qgZxt4hu" role="doc-biblioref">776</a>–<a href="#ref-Lovg2liS" role="doc-biblioref">778</a></sup></span> or may even improve protein identifications<span class="citation" data-cites="nMWQwf9T"><sup><a href="#ref-nMWQwf9T" role="doc-biblioref">779</a></sup></span>.
Whether to employ a search space that is sample-specific (i.e., subset), species-specific (with only canonical proteins, described below), exhaustive species-specific (including all isoforms), or even larger clade-level protein sequence set (e.g., the over 14 million protein sequences associated with Fungi, taxon identifier 4751) is a complex issue that is experiment and software dependent.
Moreover, in cases where no species-specific protein sequence collection exists, homology-based searching can be used (as described in<span class="citation" data-cites="17gB3bs6x"><sup><a href="#ref-17gB3bs6x" role="doc-biblioref">765</a></sup></span>).
In each of these cases, proteomics practitioners must understand their specific experimental sample and search algorithm in order to know how to best define the search space, which is essential to yielding accurate results.</p>
<h2 class="page_break_before" id="data-repositories-and-knowledge-bases">15. Data Repositories and Knowledge Bases</h2>
<h3 id="proteomics-raw-data-repositories">Proteomics raw data repositories</h3>
<p>An essential part of the proteomics publication cycle is raw data sharing.
This is important so that others can reproduce results and utilize data for new investigations.
Computational researchers may use published data to develop new algorithms or combine multiple datasets into a meta study.
There are many websites that serve as data repositories for publication.
These include: PRIDE<span class="citation" data-cites="19xkjz9Ak jpihB8r0"><sup><a href="#ref-19xkjz9Ak" role="doc-biblioref">780</a>,<a href="#ref-jpihB8r0" role="doc-biblioref">781</a></sup></span>, Massive<span class="citation" data-cites="JejZsE21"><sup><a href="#ref-JejZsE21" role="doc-biblioref">646</a></sup></span>, and Chorus<span class="citation" data-cites="BeY3BmUJ"><sup><a href="#ref-BeY3BmUJ" role="doc-biblioref">782</a></sup></span></p>
<h3 id="peptideatlas-and-srmatlas">PeptideAtlas and SRMAtlas</h3>
<p>It would be beneficial to analyze all the data <em>in toto</em> to derive a knowledge base of all detectable proteins in an organism.
A challenge in attempting this is that, given the vast array of software for MS data analysis, the results are not directly comparable nor combinable given the problem of false discovery rates (FDR) that must be added up when dataset results are combined.
For example, if we combine 3 datasets that were each filtered to 1% FDR, the maximum FDR of the combined dataset is now 3% because it is unlikely that the random decoy hits are shared across each dataset.
To address this, in 2005 the PeptideAtlas concept was started to ingest as many publically available datasets as possible per organism, search the data through a single pipeline together and arrive at a total controlled 1% protein level FDR<span class="citation" data-cites="15qh1BI32 FkUJzost"><sup><a href="#ref-15qh1BI32" role="doc-biblioref">783</a>,<a href="#ref-FkUJzost" role="doc-biblioref">784</a></sup></span>.
The PeptideAtlas website (www.peptideatlas.org) is a multi-organism, publicly accessible compendium of peptides identified in large sets of tandem mass spectrometry proteomics experiments.
Mass spectrometer output files are collected for human, mouse, yeast, and many other organisms of research interest, and searched using the latest search engines and genome derived protein sequences.
All results of sequence and spectral library searching on PeptideAtlas are processed through the Trans Proteomic Pipeline to derive a probability of correct identification for all results in a uniform manner to insure a high quality database, along with false discovery rates at the whole atlas level.</p>
<p>The most recognizable MS data compendium is the Human PeptideAtlas which is produced yearly since 2005 to derive all the peptide sequence knowledge of the current human proteome (Figure 17A).
As of 2023, the Human PeptideAtlas contains the knowledge of over 93% of the human proteome, with over 189K MS runs and 3.6B spectra searched resulting in 3.4m peptides identified and 17,245 proteins identified from the 19,600 total proteins possible.
The number of proteins has been incrementally increases year over year as new public data becomes available (Figure 17B).</p>
<div id="fig:peptide-atlas" class="fignos">
<figure>
<img src="images/peptide_atlas.jpg" style="width:100.0%" data-tag="17" alt="Figure 17: The Human PeptideAtlas as of 2023. A) The current total search space and identified elements of the 2023 Human PeptideAtlas. B) Historical cumulative plot of the identified total proteins (blue vertical bars) and the unique proteins identified per dataset (red vertical bars) over the total period of 2005-2023." />
<figcaption aria-hidden="true"><span>Figure 17:</span> <strong>The Human PeptideAtlas as of 2023.</strong>
A) The current total search space and identified elements of the 2023 Human PeptideAtlas.
B) Historical cumulative plot of the identified total proteins (blue vertical bars) and the unique proteins identified per dataset (red vertical bars) over the total period of 2005-2023.</figcaption>
</figure>
</div>
<p>For the presentation of selected reaction monitoring (SRM) targeted peptide assays, there are two components of the PeptideAtlas ecosystem where the PeptideAtlas SRM Experiment Library (PASSEL) is presented to enable submission, dissemination, and reuse of SRM experimental results from analysis of biological samples<span class="citation" data-cites="Z2Ay0IqK CJuU986M"><sup><a href="#ref-Z2Ay0IqK" role="doc-biblioref">785</a>,<a href="#ref-CJuU986M" role="doc-biblioref">786</a></sup></span>.
The PASSEL system acts as a data repository by allowing researchers with SRM data to deposit their data in parallel with journal publication, and other users can search existing data to obtain the parameters for replication in their own laboratory.
Another unique component for SRM data repositories is the SRMAtlas website, which provides definitive coordinates for all possible proteins within an organism to conduct targeted SRM assays that conclusively identify the respective peptide in biological samples.
As an example, the Human SRMAtlas provides data on 166,174 synthetic proteotypic human peptides, providing multiple, independent assays to quantify any human protein and numerous spliced variants, non-synonymous mutations, and post-translational modifications<span class="citation" data-cites="17qvo6lpw"><sup><a href="#ref-17qvo6lpw" role="doc-biblioref">787</a></sup></span>.
The data are freely accessible as a resource at http://www.srmatlas.org/.</p>
<h3 id="other-knowledgebases">Other knowledgebases</h3>
<p>There are many other knowledgebases that will be useful to proteomics researchers.
These include the proteomics standards initiative (PSI)<span class="citation" data-cites="qVEHb5K"><sup><a href="#ref-qVEHb5K" role="doc-biblioref">788</a></sup></span>, Proteometools<span class="citation" data-cites="2iJj61wl"><sup><a href="#ref-2iJj61wl" role="doc-biblioref">789</a></sup></span>, and iProX<span class="citation" data-cites="LMs52Uzn"><sup><a href="#ref-LMs52Uzn" role="doc-biblioref">790</a></sup></span>.
Panorama<span class="citation" data-cites="rKWchS8R"><sup><a href="#ref-rKWchS8R" role="doc-biblioref">791</a></sup></span> is a resource for sharing processed proteomics data including the extracted ion chromatograms, which can improve transparency by enabling easy data inspection on the web.</p>
<h2 class="page_break_before" id="biological-interpretation">16. Biological Interpretation</h2>
<p>The most common untargeted proteomics experiment will produce a list of proteins or peptides of interest which require further validation and biological interpretation.
This list usually results from statistical data analysis; the typical output of differentially expressed proteins usually contains hundreds of hits.
In this section, we aim to present a concise overview of how proteomic data can be effectively contextualized and used to generate new hypotheses.</p>
<p>The simplest approach is to start manual lookup of every protein in the list to uncover groups that function together.
Starting with a list of hundreds of protein changes, a smaller list can be prioritized by considering the level of significance and effect size.
For example, proteins with the smallest p-values (significance) and largest abundance fold-changes (effect size).
It is tempting to focus on proteins with the most extreme fold changes.
In this case, the assumption is that the more significant the fold change (in either direction, up- or down-regulation), the higher the impact of those proteins on cellular behavior.
This assumption is not always valid because protein signal in MS depends on abundance.
The manual data interpretation approach is typically infeasible due to the number of proteins that would need to be individually looked up one-by-one.</p>
<p>A better strategy is to use computational methods.
These methods may consider the whole list of proteins including some ranking by significance or fold change.
One common interpretation method is to construct a protein network, which then lends itself to network analyses.
Another method is to consider functional enrichment through annotation databases.
These databases offer insights by examining the enrichment of certain functional annotations amongst the interesting proteins.
Secondly, one could consider other evolutionary, structurally or regulatory based methods to identify interpretation of the data.
To fully interpret analysis, it may be required to perform or examine other data such as data from biophysical, biochemical and alternative proteomic approaches.
Finally, the data can further be interpreted using multi-omic, native or clinical approaches.
Below we summarize these approaches and point out potential pitfalls with these methods.</p>
<h3 id="constructing-a-protein-network">Constructing a protein network</h3>
<p>A network is a representation of the relations between objects.
Nodes are the entities of the network (e.g., users of a social platform, train stations, proteins), while edges are the connections between them (e.g., friendship, routes, and protein interactions, respectively).
In the case of protein-protein interactions, evidence for the functional associations between proteins can be obtained experimentally.
For example, co-immunoprecipitation, crosslinking, and proximity labeling can be used to reveal physical interactions<span class="citation" data-cites="43dscula"><sup><a href="#ref-43dscula" role="doc-biblioref">792</a></sup></span>.
The data is presented in a table with nodes and edges (e.g., “protein A interacts with protein B”) from which the network can be constructed.
A considerable wealth of protein-protein association data is stored in free databases like IntAct, which contain interactions derived from literature curation or direct user submissions<span class="citation" data-cites="xeKEaOVF"><sup><a href="#ref-xeKEaOVF" role="doc-biblioref">793</a></sup></span>.
Protein interactions can also be predicted by classifiers that consider many features, like orthology and co-localization, to produce a posterior odds ratio of interaction<span class="citation" data-cites="D6gNmWdp uSY25K4H"><sup><a href="#ref-D6gNmWdp" role="doc-biblioref">794</a>,<a href="#ref-uSY25K4H" role="doc-biblioref">795</a></sup></span>.</p>
<p>Large repositories like STRING (Search Tool for the Retrieval of Interacting Genes/Proteins) collect and integrate protein-protein interaction data from several databases<span class="citation" data-cites="uSY25K4H"><sup><a href="#ref-uSY25K4H" role="doc-biblioref">795</a></sup></span>.
STRING also provides a web-based interface to survey the data, and users only have to feed a search box with the identifiers of the protein(s) of interest.
STRING will retrieve the network and show the evidence supporting each interaction.
Importantly, these databases do not indicate the direction of the interaction, so they produce undirected networks.</p>
<p>There are many other options for generating and working with networks.
For example, geneMANIA can generate a network from data<span class="citation" data-cites="qwaNFUVe"><sup><a href="#ref-qwaNFUVe" role="doc-biblioref">796</a></sup></span>.
Cytoscape is a free tool useful for generating and interacting with networks<span class="citation" data-cites="1BJixYIRV"><sup><a href="#ref-1BJixYIRV" role="doc-biblioref">797</a></sup></span>.</p>
<h3 id="network-analysis">Network analysis</h3>
<p>Network analysis is a group of techniques that explore and investigate the network, yielding valuable knowledge about its structure and unveiling key players regulating the flow of information.
One of the first steps in network analysis relates to centrality measurements.
Centralities are indicators of the relative importance of a node corresponding to its position in the network, and each centrality measure provides new insights to interpret the data in new ways<span class="citation" data-cites="Es98brAH awUpRsoz"><sup><a href="#ref-Es98brAH" role="doc-biblioref">798</a>,<a href="#ref-awUpRsoz" role="doc-biblioref">799</a></sup></span>.</p>
<h4 id="degree-centrality">Degree centrality</h4>
<p>The degree of a node measures the number of edges incident to that node.
Nodes with a high degree interact with many other nodes, called first neighbors.
In particular, the node degree distribution in protein networks is highly skewed, with most nodes having a low degree and a few having high degrees, known as hubs.
Hubs are usually regulatory proteins, being notable examples oncogenes and transcription factors.
Moreover, hubs are attractive targets for directed interventions, as their alteration has a profound effect on the stability of the network<span class="citation" data-cites="12ufdjSBQ"><sup><a href="#ref-12ufdjSBQ" role="doc-biblioref">800</a></sup></span>.</p>
<h4 id="closeness-centrality">Closeness centrality</h4>
<p>The route from one node to another is a path, and the shortest path is the one connecting them in the least amount of steps.
Closeness centrality is the inverse of the average length of a node’s shortest paths to all other nodes in the network.
Nodes with a high closeness score have the shortest distances to all the others, so closeness centrality calculations detect nodes that can spread information very efficiently, as they are in a better position in the network for this task<span class="citation" data-cites="1FZ0vcua3 KQtBgFa0"><sup><a href="#ref-1FZ0vcua3" role="doc-biblioref">801</a>,<a href="#ref-KQtBgFa0" role="doc-biblioref">802</a></sup></span>.</p>
<h4 id="betweenness-centrality">Betweenness centrality</h4>
<p>This centrality index is related to the amount of shortest paths transversing a node.
Nodes with a high betweenness centrality usually bridge different parts of the network and strongly influence the flow of information, as they lie in communication paths.
These connector hubs (or bottlenecks) are also interesting for follow–up experiments because their removal can disconnect different regions of the network<span class="citation" data-cites="1BmdBEJEB"><sup><a href="#ref-1BmdBEJEB" role="doc-biblioref">803</a></sup></span>.</p>
<p>Centrality measurements add new layers of information and allow for ranking differentially expressed proteins apart from their fold-change in abundance.
<strong>Figure 18</strong> depicts a simple network consisting of proteins A to L, with A having the highest fold change (10) and L the lowest (2).
In Panel A, the fill color for the nodes indicates this metric, where it can be easily seen that A stands out.
However, protein A is a peripheral protein, only interacting with B.
In Panel B, nodes are colored according to node degree.
Clearly, protein F has the highest number of interactions and is also the closest to all other nodes, which can be appreciated when nodes are colored according to closeness centrality (Panel C).
On the other hand, protein G acts as a bridge between two regions of the network and thus, has the highest betweenness centrality (Panel D).
Except for fold change, node A has the lowest indices, and it will be up to the researcher to decide whether this protein warrants further examination.</p>
<div id="fig:simple-network" class="fignos">
<figure>
<img src="images/Simple%20network.png" style="width:100.0%" data-tag="18" alt="Figure 18: Analysis of a simple network using different centrality measurements. Nodes are colored according to each metric using a yellow-to-red gradient (yellow: lowest value, red: highest value). Network visualization and analysis were performed in Cytoscape." />
<figcaption aria-hidden="true"><span>Figure 18:</span> <strong>Analysis of a simple network using different centrality measurements.</strong>
Nodes are colored according to each metric using a yellow-to-red gradient (yellow: lowest value, red: highest value).
Network visualization and analysis were performed in Cytoscape.</figcaption>
</figure>
</div>
<h3 id="network-clustering">Network clustering</h3>
<p>In the small network presented in <strong>Figure 18</strong>, two groups of densely connected nodes exist.
This topology suggests that these communities (or “clusters”) work together or participate in a protein complex.
Dividing a network into clusters helps identify underlying relationships among nodes, which is especially useful in large networks.
In a broad sense, network clustering groups nodes according to a topological property, generally interconnectedness.
There are many network clustering algorithms, each with its own merits and approaches<span class="citation" data-cites="1H6bETSCb 11gPEmZWK"><sup><a href="#ref-1H6bETSCb" role="doc-biblioref">804</a>,<a href="#ref-11gPEmZWK" role="doc-biblioref">805</a></sup></span>.
The MCL (Markov CLustering) algorithm is suitable for protein networks in most situations .
On the other hand, the Molecular COmplex DEtection (MCODE) algorithm helps detect very densely connected nodes, thus unveiling protein complexes<span class="citation" data-cites="iqdatwR1"><sup><a href="#ref-iqdatwR1" role="doc-biblioref">806</a></sup></span>.
In this regard, network clustering is useful for tentatively assigning the function of an uncharacterized protein.
If the protein appears in a cluster, its function should be closely related to the cluster members, a principle known as “guilty by association.”<span class="citation" data-cites="WJOtOvRY"><sup><a href="#ref-WJOtOvRY" role="doc-biblioref">807</a></sup></span></p>
<h3 id="network-visualization">Network visualization</h3>
<p>A critical step in network analysis is to display the data in a structured and uncluttered graph.
Networks can rapidly become a hairball unamenable to interpretation.
Software platforms like Cytoscape can be used to visualize networks orderly by applying layout algorithms and format styles<span class="citation" data-cites="8oE5lYNw"><sup><a href="#ref-8oE5lYNw" role="doc-biblioref">808</a></sup></span>.
Since many of these platforms are open source, community-designed plugins enhance their capabilities.
In Cytoscape, the stringApp adds a search bar to query the STRING database with accession numbers or protein names<span class="citation" data-cites="jrtjRoft"><sup><a href="#ref-jrtjRoft" role="doc-biblioref">809</a></sup></span>.
The network is directly retrieved into Cytoscape, where its built-in network analyzer can be used to calculate centralities.
Moreover, user-defined information, like fold-change values, can be integrated and mapped into the network.</p>
<h3 id="functional-term-enrichment-analysis-kegg-string-go-gsea-ora-reactome-and-others">Functional term enrichment analysis: KEGG, String, GO, GSEA, ORA, Reactome, and others</h3>
<!-- To-Do: this section is redundant with the next section -->
<p>Term enrichment analysis is performed to assess whether particular ‘functional terms’ are over-represented in a list of proteins (e.g. from a proteomics experiment)<span class="citation" data-cites="h0C6mgc4 CaFLiife QlVqCOZZ"><sup><a href="#ref-h0C6mgc4" role="doc-biblioref">810</a>–<a href="#ref-QlVqCOZZ" role="doc-biblioref">812</a></sup></span>.
For example, after a differential abundance analysis, we may wish to examine whether there is any shared function amongst the proteins which were determined to have significant changes.
The simplest analysis to test whether this subset contains more of any particular functional terms than we would expect given the background of proteins.
For example, the Gene Ontology is split into the classes: Cellular Component, Molecular Function and Biological Function and we might be interested as to whether our proteins may be more likely to localize to a particular subcellular niche<span class="citation" data-cites="IFevKgND"><sup><a href="#ref-IFevKgND" role="doc-biblioref">813</a></sup></span>.
The Cellular Component terms could give us a starting point if this might be the case, by examining if Cellular Component annotations are enriched.</p>
<p>There are a number of databases and tools to perform such analysis, which can even be extended to examine whole pathways, networks, post-translational modification and literature representation.
For example, databases such as KEGG<span class="citation" data-cites="15ZNlTPZi"><sup><a href="#ref-15ZNlTPZi" role="doc-biblioref">814</a></sup></span>, String<span class="citation" data-cites="uSY25K4H"><sup><a href="#ref-uSY25K4H" role="doc-biblioref">795</a></sup></span>, Reactome<span class="citation" data-cites="n1NPYC1C"><sup><a href="#ref-n1NPYC1C" role="doc-biblioref">815</a></sup></span> and PhosphoSitePlus<span class="citation" data-cites="f3o81IaL"><sup><a href="#ref-f3o81IaL" role="doc-biblioref">816</a></sup></span> can be used to test or annotate a list of proteins.
For example, proteomics analysis of human cardiac 3D microtissue exposed to anthracyclines (drugs used in cancer chemotherapy) unearthed several proteins with altered levels<span class="citation" data-cites="10dRpS3qN"><sup><a href="#ref-10dRpS3qN" role="doc-biblioref">817</a></sup></span>.
Many of these were specifically grouped under GO terms related to mitochondrial dysfunction, indicating the detrimental effects of these drugs on the organelle.
GO terms<span class="citation" data-cites="IFevKgND"><sup><a href="#ref-IFevKgND" role="doc-biblioref">813</a></sup></span> or descriptors from other annotation libraries (like KEGG<span class="citation" data-cites="15ZNlTPZi"><sup><a href="#ref-15ZNlTPZi" role="doc-biblioref">814</a></sup></span> or REACTOME<span class="citation" data-cites="DhTe2WLI"><sup><a href="#ref-DhTe2WLI" role="doc-biblioref">818</a></sup></span>) can be retrieved from STRING when constructing a network or from other freely available compendiums.
We refer to a number of articles on the topics, including tools, reviews and best-practice<span class="citation" data-cites="rlKsqE1k 1FmAqWcpa 7xIEHm2N"><sup><a href="#ref-rlKsqE1k" role="doc-biblioref">819</a>–<a href="#ref-7xIEHm2N" role="doc-biblioref">821</a></sup></span>.
The main points from such analysis is that we can obtain an insight about protein function by looking at whether our list of proteins have similar or the same annotations.
A number of limitations should be taken into account for interpretation.
The first is that proteins that are more abundant are more likely to be studied, measured and examined in the literature.
Hence, abundant proteins will have more annotations than less abundant ones.
One key part of the analysis is also to correctly select the background set; that is, the universe of protein which our list is being compared against.
By including contaminates or proteins that are not expressed in our system within the list, the results may be unfaithful.</p>
<p>We may also have access to our own curated set of annotations derived either computational or experimental.
One may be interested in seeing whether we have enrichment of these annotations amongst the differentially abundant proteins.
Our list of proteins could be divided into two groups: differentially abundant or not. These groups could be divided into whether they have a particular annotation: yes or no.
This information can be summarized in a two-by-two table, to which we can apply a statistical test to examine whether that annotation is enriched within our differentially abundant proteins.
One test that could be used is the hypergeometric test, and another would be a Fisher Exact test.</p>
<p>There are many methods for performing functional enrichment analysis on the data, but they can mainly be classified into three categories (<strong>Figure 19</strong>), as follows.</p>
<div id="fig:enrich_methods" class="fignos">
<figure>
<img src="images/enrich_methods_v4.png" style="width:100.0%" data-tag="19" alt="Figure 19: Types of functional enrichment methods. In the volcano plot (left), proteins with altered values are colored blue or red according to arbitrarily chosen cut-off values for significance and fold change. Black bars or thick-bordered nodes indicate members of a GO category." />
<figcaption aria-hidden="true"><span>Figure 19:</span> <strong>Types of functional enrichment methods.</strong>
In the volcano plot (left), proteins with altered values are colored blue or red according to arbitrarily chosen cut-off values for significance and fold change.
Black bars or thick-bordered nodes indicate members of a GO category.</figcaption>
</figure>
</div>
<h4 id="over-representation-analysis">Over-representation analysis</h4>
<p>In modern proteomics analysis, usually thousands of proteins are identified and quantified.
Fold-change and significance thresholds are chosen (e.g., fold-change ≥ 2 and p ≤ 0.05) to obtain a list of proteins with altered levels among the tested conditions.
In over-representation methods, a contingency table is created for every protein set to establish whether proteins with altered abundance show an enrichment or a depletion of the ontology term compared to the background observed proteome<span class="citation" data-cites="HNlvs4Yk"><sup><a href="#ref-HNlvs4Yk" role="doc-biblioref">822</a></sup></span>.
For example, suppose that 2000 proteins were quantified in a proteomics analysis, being 40 of these members of the set “tricarboxylic acid cycle (TCA).”
Also, let us assume that 200 proteins showed altered abundance, with 15 belonging to the TCA set.
Then, the contingency table can be constructed as follows (Table 11):</p>
<p>Table 11: Example term enrichment analysis.</p>
<table>
<colgroup>
<col style="width: 25%" />
<col style="width: 31%" />
<col style="width: 33%" />
<col style="width: 9%" />
</colgroup>
<thead>
<tr class="header">
<th></th>
<th style="text-align: center;"><strong>Proteins with altered abundance</strong></th>
<th style="text-align: center;"><strong>Proteins with unaltered abundance</strong></th>
<th style="text-align: center;"><strong>Total</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>Proteins in TCA set</strong></td>
<td style="text-align: center;">15</td>
<td style="text-align: center;">25</td>
<td style="text-align: center;">40</td>
</tr>
<tr class="even">
<td><strong>Proteins not in TCA set</strong></td>
<td style="text-align: center;">185</td>
<td style="text-align: center;">1775</td>
<td style="text-align: center;">1960</td>
</tr>
<tr class="odd">
<td><strong>Total</strong></td>
<td style="text-align: center;">200</td>
<td style="text-align: center;">1800</td>
<td style="text-align: center;">2000</td>
</tr>
</tbody>
</table>
<p>Then, a suitable statistical test is conducted to ascertain if proteins with altered levels are enriched in members of the TCA cycle (in this case, they are; p &lt; 0.00001).
This is commonly achieved using Fisher’s exact test<span class="citation" data-cites="TWAPY3zr"><sup><a href="#ref-TWAPY3zr" role="doc-biblioref">823</a></sup></span>.
The process is then repeated for every set as desired.
Since multiple comparisons are tested, p values must be adjusted by a false discovery rate<span class="citation" data-cites="ArmIWAfa"><sup><a href="#ref-ArmIWAfa" role="doc-biblioref">824</a></sup></span>.
There are also several free tools for term enrichment analysis, including<span class="citation" data-cites="rrE2B3JZ"><sup><a href="#ref-rrE2B3JZ" role="doc-biblioref">825</a></sup></span>, GSEA<span class="citation" data-cites="15p5LWIVP"><sup><a href="#ref-15p5LWIVP" role="doc-biblioref">826</a></sup></span>, and DAVID<span class="citation" data-cites="101HEiJoS"><sup><a href="#ref-101HEiJoS" role="doc-biblioref">827</a></sup></span>.</p>
<h4 id="functional-class-scoring">Functional class scoring</h4>
<p>The caveat of over-representation methods is that they rely on a list of differentially expressed genes or proteins with altered abundance, selected due to arbitrarily chosen cut-off values.
For example, if we set a fold change cutoff of 2, a protein with a fold-change of 1.99 would not be included in the analysis.
Moreover, several proteins belonging to the same set may have altered levels but are below the fold change threshold.
However, moderate alterations of their abundance as a group could drive the observed phenotype, even more so than a single protein over the cut-off.
Functional class scoring strategies aim at countering these limitations by disregarding thresholds altogether.
GSEA (Gene Set Enrichment Analysis) is a widely used functional class scoring method in which all detected entities are first ranked according to a quantitative measurement (fold change, p-value, or their combination)<span class="citation" data-cites="tt0Vdf4C"><sup><a href="#ref-tt0Vdf4C" role="doc-biblioref">828</a></sup></span>.
Then, the distribution of members of a set is obtained.
A scoring scheme based on the Kolmogorov – Smirnov test is used to assess whether there is an enrichment of the category towards the top or bottom of the ranked list.</p>
<h4 id="pathway-topology-based-methods">Pathway topology-based methods</h4>
<p>Both methods mentioned above do not consider the functional relationships among proteins put forth by network analysis; i.e., they assume functional independence.
Topology-based enrichment methods incorporate this information by, for example, assigning an importance value to a set when its members also participate in a pathway or cluster together in a network<span class="citation" data-cites="6gI0wLAR"><sup><a href="#ref-6gI0wLAR" role="doc-biblioref">829</a></sup></span>.
<strong>Figure 19</strong> shows how topology-based methods consider non-significant hits (grey nodes) that other strategies may not pick up, due to their position in a network.</p>
<h3 id="other-computational-approaches-network-analysis-isoform-correlation-analysis-alphafold-blast-protein-language-models">Other computational approaches: Network analysis, Isoform correlation analysis, AlphaFold, BLAST, protein language models</h3>
<p>Additional computational analysis of a list of interesting proteins may uncover additional substructure, correlation or biologically useful hypothesis.
Building a network between the proteins based on the experiments performed might be a useful approach to identify additional structure.
For example, co-expression network analysis can be used to build a network from these proteins<span class="citation" data-cites="1BeN7jMiD"><sup><a href="#ref-1BeN7jMiD" role="doc-biblioref">830</a></sup></span>.
In these networks, proteins are nodes and edges describe relationships between those proteins.
Network-specific methods can then be applied, such as community detection algorithms which could uncover clusters of proteins with shared functions<span class="citation" data-cites="15Ki75h8 fXCPx62A"><sup><a href="#ref-15Ki75h8" role="doc-biblioref">831</a>,<a href="#ref-fXCPx62A" role="doc-biblioref">832</a></sup></span>.</p>
<p>One way the proteome generates complexity is through alternative-splicing, which results in protein isoforms<span class="citation" data-cites="eZTqkAjM"><sup><a href="#ref-eZTqkAjM" role="doc-biblioref">833</a></sup></span>.</p>
<p>Recently, a number of tools have been proposed to identify peptide isoforms that are quantitatively different across conditions by using a principle called peptide correlation analysis<span class="citation" data-cites="kBc66YMI YKUlYryb"><sup><a href="#ref-kBc66YMI" role="doc-biblioref">834</a>,<a href="#ref-YKUlYryb" role="doc-biblioref">835</a></sup></span>.
The idea is that the quantitative behavior of peptides should match each other.
If there are subgroups that behave coherently within the group but not across groups suggest that peptide may have come from a different proteoform.
These approaches can be used to identify specific proteoforms that are functional across different conditions.</p>
<p>For many, a protein’s structure reveals important functional details<span class="citation" data-cites="vegaUWlG"><sup><a href="#ref-vegaUWlG" role="doc-biblioref">836</a></sup></span>.
There are a plethora of approaches to predict a protein’s structure<span class="citation" data-cites="PnnbHO9a yZfcMIwh 1AwSDaTyL"><sup><a href="#ref-PnnbHO9a" role="doc-biblioref">837</a>–<a href="#ref-1AwSDaTyL" role="doc-biblioref">839</a></sup></span>.
Recently, AlphaFold and RoseTTAFold have become dominant methods for predicting protein structures with high resolution<span class="citation" data-cites="yZfcMIwh 1AwSDaTyL"><sup><a href="#ref-yZfcMIwh" role="doc-biblioref">838</a>,<a href="#ref-1AwSDaTyL" role="doc-biblioref">839</a></sup></span>.
If intrinsically disordered domains are of particular interest, methods explicitly designed for this task are recommended<span class="citation" data-cites="tIYr5DF7"><sup><a href="#ref-tIYr5DF7" role="doc-biblioref">840</a></sup></span>.
Once a structure is obtained more elaborate computational methods might be useful such as docking or molecular dynamics<span class="citation" data-cites="CRgkwTQA iOQeUEPU"><sup><a href="#ref-CRgkwTQA" role="doc-biblioref">841</a>,<a href="#ref-iOQeUEPU" role="doc-biblioref">842</a></sup></span>.
These approaches can give insight into how protein or molecules fit together and the dynamics of a protein’s structure (conformational heterogeneity).
A complete discussion of these topics is beyond the scope of this section.</p>
<p>Another way to obtain insights into a protein function is to look for protein with similar sequences or motifs.
Using BLAST, a sequence alignment tool, one can align two or more protein sequences and determine their level of similarity<span class="citation" data-cites="jcOtxABy"><sup><a href="#ref-jcOtxABy" role="doc-biblioref">843</a></sup></span>.
For example, if a human protein of unknown function has a similar sequence to a yeast protein with known function this may be a starting place for the putative function of that protein.</p>
<p>Novel approaches to representing the similarity of proteins have proved successful at predicting the functional properties of proteins.
Protein language models seek to learn “representation” of proteins, these are usually numerical vectors that represent a protein sequence<span class="citation" data-cites="daqSjHIW nf5EVAMJ"><sup><a href="#ref-daqSjHIW" role="doc-biblioref">844</a>,<a href="#ref-nf5EVAMJ" role="doc-biblioref">845</a></sup></span>.
Abstractly, these vectors preserve protein similarity or a notion of “proteinness”. This usually means that two proteins that have a close vector may share similarities in protein function.
These representations are also advantageous because they can easily become the inputs for machine learning algorithms to predict valuable protein properties; for example, thermal stability values<span class="citation" data-cites="wLlg6SzE"><sup><a href="#ref-wLlg6SzE" role="doc-biblioref">846</a></sup></span>, protein-protein binding affinities<span class="citation" data-cites="YBdYtUv"><sup><a href="#ref-YBdYtUv" role="doc-biblioref">847</a></sup></span>, secondary protein structure, and more.</p>
<h2 class="page_break_before" id="orthogonal-validation">17. Orthogonal Validation</h2>
<h3 id="the-importance-of-orthogonal-experimental-validation">The importance of orthogonal experimental validation</h3>
<p>The computational workflows to interpret mass spectrometry data are sophisticated, powerful tools, but also show important limitations and caveats due to their dependence on limited prior knowledge, specific experimental parameters or data quality restraints (see section ‘Analysis of Raw Data’).
These inherent biases can give rise to ambiguous or spurious interpretation of the data even when these workflows are applied correctly and to the best of the experimenter’s knowledge.
Therefore, researchers will oftentimes be asked by scientific journals to provide independent orthogonal validation of their proteomics data and not performing such can be a major roadblock in the publication process.</p>
<p>The aim of validating data obtained by proteomics approaches should always be two-fold by demonstrating that the conclusions arrived at by proteomics data acquisition and analysis are, firstly, valid and, secondly, relevant.
Depending on the question at hand, researchers can draw on an overabundance of techniques to validate MS-derived hypotheses in appropriate cellular, organismal or in vitro models.
In the following paragraphs we aim to present only a high-level, stringent, non-exhaustive selection of orthogonal validation approaches and emphasize the importance of implementing assays that challenge assumptions gained from proteomics data analysis pipelines.</p>
<p>Before embarking on orthogonal validation of any hit, the success of the experiment should be established by assessing (internal) positive controls.
Internal positive controls can be proteins whose behavior under the experimental conditions applied can be deduced from prior knowledge (i.e. the scientific literature or public databases).
Once the expected changes in internal controls have been confirmed by computational analysis (see the above section), the orthogonal experimental validation of novel, perhaps unexpected findings can begin.</p>
<p>Orthogonal validation of new insights obtained from quantitative proteomics experiments can be a very time-consuming process and often requires familiarity with techniques not directly related to proteomics workflows.
Given these challenges, the method(s) of choice warrant(s) careful consideration and is highly context-dependent.
Importantly, proteomics experiments in one way or another generally yield comprehensive lists of potentially interesting candidate proteins or pathways, the researcher will have to shortlist candidates to be taken forward to the validation stage of the project.
Which candidates should you validate by an orthogonal approach and which ones might not require further validation?</p>
<p>In general, candidates representing abundant proteins that show high sequence coverage and are detected with high confidence might not necessarily need extensive orthogonal validation when compared with proteins of intermediate to low abundance that might be more challenging to faithfully quantify by proteomics alone (i.e., many membrane proteins or transcription factors).
Similarly, since the proteome is rarely comprehensively quantified in any single proteomics experiment, proteins of interest (POIs) that are critical for an observed biological change might not be part of the dataset.
In these cases, additional, targeted analyses might help to support or discredit proteomics-based hypotheses.</p>
<p>Validation techniques are as manifold as biological questions and discussions thereof may easily fill multiple textbooks.
The following sections are therefore merely meant to paint with a broad brush stroke a picture of useful methodologies with which to validate and follow up MS-data derived observations.
As this is meant to orient the reader, wherever possible, we will explicitly point out useful literature reviews for a deeper dive into each of these techniques.</p>
<h3 id="general-considerations">General considerations</h3>
<p>Once POIs have been selected based on prior agreed-upon selection criteria (i.e. (adjusted) p value and/or fold change thresholds), orthogonal validation experiments should ideally be conducted under physiologically relevant conditions to mitigate artificial and misleading outcomes.
Therefore, in vitro experiments, while useful to isolate and dissect particular aspects of a biological system, can give highly artificial results as conditions are far removed from the POI’s native environment.
To investigate the biological function of a protein or pathway, direct genetic manipulation of the biological system at hand (e.g., modulating the expression of a POI by overexpression or knockout-/down experiments) can be minimally invasive when performed correctly.
Should the POI be encoded by an essential gene, by definition, a complete and stable knockout might not be advisable<span class="citation" data-cites="bUq8ZzIL 171wjW4dP"><sup><a href="#ref-bUq8ZzIL" role="doc-biblioref">848</a>,<a href="#ref-171wjW4dP" role="doc-biblioref">849</a></sup></span>.
In these extreme cases, attenuated expression (i.e., using RNA interference (RNAi) or controlled degradation, see below) rather than complete repression of a gene can be used to probe for protein function.
Epitope tagging and/or exogenous expression of a gene of interest can be a powerful approach in assessing PPIs and investigating proteins of low abundance.
However, overexpression artifacts are common<span class="citation" data-cites="pGJ2GFbE"><sup><a href="#ref-pGJ2GFbE" role="doc-biblioref">850</a></sup></span>.</p>
<p>It is not always possible to fully avoid the pleiotropic effects of protein (over-)expression or depletion, but a number of mitigation strategies (i.e., inducible expression, the use of multiple independent RNAi strategies) will be discussed below.</p>
<p>Extensive biochemical characterization of any overexpressed gene is critical to ensure it closely reflects the functions of its endogenous counterpart.
These assays might involve assessing protein localization (i.e., by imaging techniques such as microscopy and flow cytometry), protein abundance (i.e., by mass spectrometry or immunoblot analysis) and phenotypic assays where applicable and practical.</p>
<h3 id="functional-genomics-techniques-in-the-validation-of-ms-hits">Functional genomics techniques in the validation of MS hits</h3>
<p>Typical follow-up experiments to validate mass-spectrometry derived insights often involve the acute depletion or induction of a POI and assessing the impact on specific cellular phenotypes.
Here we present a selection of methodologies to effectively modulate gene expression and discuss important considerations when planning functional genomics experiments for target validation.</p>
<p>Gene deletion or knockdown to prevent production of a functional protein is a powerful means to interrogate the role of one or more proteins in the phenotype(s) under investigation.
To this end, well-established technologies deserving mention at this point are RNA interference (RNAi) in the form of siRNA/shRNA- or miRNA-mediated gene knockdown abd CRISPR/Cas9-or TALEN-mediated gene knockout<span class="citation" data-cites="DdiSq79q"><sup><a href="#ref-DdiSq79q" role="doc-biblioref">851</a></sup></span>.
Since each one of these technologies comes with its own unique advantages and caveats, the approach taken depends on the biological question at hand.</p>
<p>Clustered regularly interspaced short palindromic repeats (CRISPR)/Cas-based gene deletion technologies allow for the targeting of individual genes with relative ease, high efficiency and specificity<span class="citation" data-cites="15T32hvm6"><sup><a href="#ref-15T32hvm6" role="doc-biblioref">852</a></sup></span>.
When expressed in mammalian cells, the bacterially-derived Cas9 endonuclease can be guided with the help of a short guide RNA (gRNA) to a genomic location of interest, where it creates a DNA double strand break in a highly controlled manner (for a detailed discussion see<span class="citation" data-cites="hRIRSidU"><sup><a href="#ref-hRIRSidU" role="doc-biblioref">853</a></sup></span>).
The cell’s DNA double-stand break repair machinery then introduces base pair insertions or deletions (indels) via non-homologous-end-joining (NHEJ), thus causing missense, and frameshift mutations (i.e. resulting in premature stop codons), leading to premature termination of gene expression or non-functional, aberrant gene products.
Similarly, the concomitant provision of a complementary DNA donor template encoding a desired gene modification (i.e. insertion of a stretch of DNA or base pair modification) will trigger homology-directed repair (HDR), resulting in gene knock in or base editing<span class="citation" data-cites="hRIRSidU"><sup><a href="#ref-hRIRSidU" role="doc-biblioref">853</a></sup></span>.<br />
Practical considerations of CRISPR/Cas9-mediated gene knock-in and base editing will not be addressed in detail but are expertly discussed in<span class="citation" data-cites="1uwXNnEt EPXqJmJj rUUeN4Sh VFV60BPa"><sup><a href="#ref-1uwXNnEt" role="doc-biblioref">854</a>–<a href="#ref-VFV60BPa" role="doc-biblioref">857</a></sup></span>.</p>
<p>The relative ease-of-use and high efficiency of the CRISPR/Cas9 gene editing technology has rendered it the method of choice for gene manipulation in many fields of cell biology.
However, it should be noted that CRISPR/Cas9-mediated gene deletion is not free from off-target effects (<span class="citation" data-cites="sAa18GbX"><sup><a href="#ref-sAa18GbX" role="doc-biblioref">858</a></sup></span> for advice on how to minimize these off-target effects).
Moreover, long-term depletion (or upregulation) of a POI itself can in some cases have dramatic systemic consequences and constitute an acute selection pressure leading to compensatory stress-induced adaptation that might obfuscate primary loss-of-function phenotypes and pose a substantial hurdle to the interpretability of biological data.
As these compensatory mechanisms often manifest with time, controlled, transient genetic manipulation (gene depletion or transgene expression) is advised.
Small interfering RNA (siRNA)-mediated knockdown by transient transfection is typically achieved at shorter time frames (24 – 96h), depending on the turnover of the POI.
On an even shorter timescale, targeted, degron-based degradation systems enable depletion of a POI within minutes and further reduce off-target effects, but require the exogenous expression of a transgene and therefore some genetic manipulation.
A more comprehensive discussion of a selection of these systems (anchor-away, deGradFP, auxin-inducible degron (AID), dTAG technologies) and their advantages and potential pitfalls is presented in<span class="citation" data-cites="n2p9cQ7"><sup><a href="#ref-n2p9cQ7" role="doc-biblioref">859</a></sup></span>.</p>
<p>Multiple eukaryotic and prokaryotic transcription-based systems have been developed that allow for the controlled biosynthesis or depletion of one or more POIs.
Amongst these, a popular and dependable choice for mammalian cells are tetracycline-controlled operon systems, which allow up- or downregulation of a POI in the presence of the antibiotic tetracycline or its derivative doxycycline.
These systems rely on the insertion of a bacteria-derived Tet operon (TetO) between the promoter and coding sequence of the gene of interest.
In this configuration, the TetO binds a co-expressed Tet-repressor protein blocking transcription of the gene of interest.
When tetracycline is added to the cells, the repressor then dissociates from the operon, thus de-repressing the gene of interest.
Different variations of this potent system exist, allowing for more flexibility in experimental design.
For instance, in the Tet-OFF system, the Tet repressor is fused to a eukaryotic transactivator (the chimeric fusion construct is termed tTA) and addition of tetracycline, or the related doxycycline, abolishes TetO binding and thus suppresses transcriptional activation<span class="citation" data-cites="1G6PkBlEo"><sup><a href="#ref-1G6PkBlEo" role="doc-biblioref">860</a></sup></span>.
Alternatively, a mutant form of tTA (rtTA) binds the TetO only in the presence of tetracycline, allowing for tetracycline-induced gene expression.
For a detailed discussion of these systems, we refer the reader to an excellent review<span class="citation" data-cites="6NVK2tsy"><sup><a href="#ref-6NVK2tsy" role="doc-biblioref">861</a></sup></span>.</p>
<p>When generating stable expression cell lines, being able to precisely control the genomic integration site of the transgene reduces overall genetic heterogeneity in a cell population and thereby reduces potential off-target or pleiotropic effects.
This ability is realised in the FlpIn-T-REx technology which harnesses Flp-recombinase mediated DNA recombination at a strictly defined genomic locus (the FRT site)<span class="citation" data-cites="idh43eOQ"><sup><a href="#ref-idh43eOQ" role="doc-biblioref">862</a></sup></span>.
Site-directed isogenic integration of any gene of interest at the FRT site, which is under a tetracycline-inducible promoter and a hygromycin resistance gene, allows for facile generation of tetracycline/doxycycline-inducible isogeneic expression cell lines with minimal leaky expression (for an example, see<span class="citation" data-cites="S1IrclG6"><sup><a href="#ref-S1IrclG6" role="doc-biblioref">863</a></sup></span>).</p>
<h3 id="validation-and-interpretation-of-protein-abundance-changes">Validation and interpretation of protein abundance changes</h3>
<p>To validate protein abundance changes observed by quantitative bottom-up proteomics or simply assess the success of targeted genetic manipulation as part of an orthogonal follow-up experiment (see above), the experimenter typically resorts to antibody-based techniques such as immunoblotting analysis or immunofluorescence and immunohistological imaging of POIs.
The latter also allows for validation of protein expression and localization in intact tissue or cells.
However, these semi-quantitative methods are strongly influenced by the quality of the antibodies used and might not be sensitive enough to detect small changes in protein levels.
In this case, more accurate orthogonal quantitation of proteins might be achieved by stable isotope labelling (SILAC/TMT/iTRAQ) and/or SRM/PRM (see section ‘Experiment Types’).
SDS-PAGE and immunoblot analysis are powerful and facile low-throughput tools to quickly validate protein abundance changes.
However, short of introducing epitope tags to the endogenous POI, the success of immunoblotting is contingent on the availability of specific antibodies, which can present a formidable problem when investigating poorly characterized proteins or working with model organisms for which the commercial availability of specific antibodies is limited (this is particularly problematic for ‘unconventional’ or even well-established model organisms such as yeast).
A detailed discussion of the strengths and pitfalls of immunoblotting for validation of semi-quantitative proteomics data can be found in an excellent review by Handler <em>et al.</em><span class="citation" data-cites="BwuqP6B4"><sup><a href="#ref-BwuqP6B4" role="doc-biblioref">864</a></sup></span>.</p>
<p>Protein abundance changes detected in a proteomics experiment can be the result of a range of different cellular processes.
The abundance of a protein in a complex sample (e.g. cell lysate or biological fluid) directly reflects a combination of the protein’s intrinsic stability and the translational rate under the conditions of interest.</p>
<p>Both protein stability as well as gene expression activity can be quantified independently.
Altered protein stability might be a direct consequence of specific or global changes in protein turnover.
Radioisotope labelling is a well-established, accurate way to monitor protein synthesis, maturation and turnover<span class="citation" data-cites="gFNDbBK5 TJFHVsUt"><sup><a href="#ref-gFNDbBK5" role="doc-biblioref">865</a>,<a href="#ref-TJFHVsUt" role="doc-biblioref">866</a></sup></span>.
This ‘pulse-chase’ methodology relies on the incorporation (‘pulsing’) of radioisotopes (typically 35S-labelled cysteine and methionine) into de-novo synthesized proteins.
Upon withdrawal of the labeled amino acids from the culture medium, the decay of signal is monitored over time (‘the chase’) by SDS-PAGE and phosphoimaging, resulting in a temporal readout of protein abundances.
The advantage of this technology is that a subpopulation (newly synthesized proteins) can be monitored directly, giving an accurate assessment of protein stability.
Once a change in protein stability has been validated, the underlying mechanisms can be addressed by inhibiting protein degradation pathways; prominently proteasome-mediated degradation (using specific proteasome inhibitors such as bortemzomib/velcade or MG132), autophagy (pharmacologically inhibiting autophagic flux) or degradation by proteases (using protease inhibitors).
The type of radiolabeling described above is relatively labor-intense, of low-throughput and has the obvious disadvantage of requiring radioactive material, which needs to be handled under strict safety precautions.
Moreover, it critically depends on the presence of one or more methionines and/or cysteines in the POIs.</p>
<p>It is also possible to measure protein stability within complex protein mixtures (i.e. cell lysates or biological fluids) using an array of specialized mass spectrometry techniques as discussed in<span class="citation" data-cites="T6IYjbu4"><sup><a href="#ref-T6IYjbu4" role="doc-biblioref">867</a></sup></span> and<span class="citation" data-cites="8UcGwfym"><sup><a href="#ref-8UcGwfym" role="doc-biblioref">116</a></sup></span>.</p>
<p>For purified proteins, well-established in vitro spectrometric and calorimetric methods such as circular dichroism, differential scanning calorimetry or differential scanning fluorometry can be used, but the relatively high sample amounts might be restrictive.</p>
<p>Finally, gene expression changes can also be determined with high fidelity using quantitative real-time PCR (qRT-PCR) or RNA-Seq can measure changes in gene transcription or mRNA turnover (for an extensive discussion of both technologies, please see<span class="citation" data-cites="fYmIhIvC"><sup><a href="#ref-fYmIhIvC" role="doc-biblioref">868</a></sup></span> and<span class="citation" data-cites="9ByCq3fI"><sup><a href="#ref-9ByCq3fI" role="doc-biblioref">869</a></sup></span>, respectively).</p>
<h3 id="validation-of-protein-protein-interactions">Validation of protein-protein interactions</h3>
<p>The interaction of a protein with other proteins determines its function.
Protein-protein interactions (PPIs) can be either mostly static (i.e. core subunits of a protein complex) or dynamic, varying with cellular state (i.e. cell cycle phase or cellular stress responses, posttranslational modifications) or environmental factors (i.e. availability of nutrients, presence of extracellular ligands of cell-surface receptors).
Therefore, any given protein can typically bind a range of interaction partners in a spatially and temporally restricted manner, thus forming complex PPI networks (the interactome of a protein).
The method of choice to experimentally examine altered PPI states depends on the model system and biological question (i.e. purified proteins vs complex protein mixtures, monitoring of PPIs in live cells or cell lysate etc).
Popular methods for the validation of PPIs in vivo include protein fragment complementation (split protein systems), 2-hybrid assays (mammalian, yeast and bacterial), proximity ligation, proximity labelling and FRET / BRET.
Protein fragment complementation assays rely on the principle that the two self-associating halves of reporter proteins can be expressed in an inactive form but when in spatial proximity bind one another to complement the functional, active reporter.
When these split reporters are fused to two interacting proteins (so-called bait and prey proteins), the binding of bait to prey induces the spatial restriction needed to fully complement the reporter. Commonly used reporter complementation systems are split fluorescent proteins (i.e. GFP, YFP)<span class="citation" data-cites="1Hf5Cuj4t"><sup><a href="#ref-1Hf5Cuj4t" role="doc-biblioref">870</a></sup></span>, ubiquitin<span class="citation" data-cites="UJonPw7t"><sup><a href="#ref-UJonPw7t" role="doc-biblioref">871</a></sup></span>, luciferase<span class="citation" data-cites="btMVdgTk"><sup><a href="#ref-btMVdgTk" role="doc-biblioref">872</a></sup></span>, TEV protease<span class="citation" data-cites="zrB7Yy5d"><sup><a href="#ref-zrB7Yy5d" role="doc-biblioref">873</a></sup></span>, beta-lactamase<span class="citation" data-cites="hPao2o3e"><sup><a href="#ref-hPao2o3e" role="doc-biblioref">874</a></sup></span>, beta-galactosidase, Gal4, or DHFR<span class="citation" data-cites="LY7mqhu6"><sup><a href="#ref-LY7mqhu6" role="doc-biblioref">875</a></sup></span>.
The resulting functional readout of these complementation system depends on which split reporter is used.
In general, the split luciferase system shows enhanced sensitivity over fluorescence-based systems as background luminescence is low.</p>
<p>Two-hybrid assays are based on a similar functional complementation strategy as fragment complementation systems.
Conventionally, two self-complementing transcription factor fragments are fused to bait and prey proteins, respectively, leading to the restoration of a functional transcription factor only upon prey-bait interaction.
The complemented transcription factor then induces the expression of a reporter gene that can be measured.
Multiple variations of this system abound for different model organisms, but they almost always involve transcriptional activation or repression of a reporter gene (<span class="citation" data-cites="18Csai5qq"><sup><a href="#ref-18Csai5qq" role="doc-biblioref">876</a></sup></span> for a detailed discussion).</p>
<p>The yeast-2-hybrid system (Y2H) is deserving of mention here as it had been the very first 2-hybrid system established<span class="citation" data-cites="19CwS2HX"><sup><a href="#ref-19CwS2HX" role="doc-biblioref">877</a></sup></span> and has ever since proven to be extremely versatile (multiple auxotrophic reporters and markers of phenotypic sensitivity available), cheap, lends itself to functional high-throughput screening and variants have been developed that allow for the investigation of membrane-protein interactions (i.e. membrane Y2H)<span class="citation" data-cites="2PC5fx0I 18Csai5qq"><sup><a href="#ref-18Csai5qq" role="doc-biblioref">876</a>,<a href="#ref-2PC5fx0I" role="doc-biblioref">878</a></sup></span>.</p>
<p>Despite the many advantages the Y2H offers, critical drawbacks include the potential of misfolding of bait and prey proteins when fused to a complementation reporter, expression at non-physiological levels, the lack of control over posttranslational modifications that might be important for the PPI under investigation, and the potential requirement of kingdom- or species-specific folding factors for the bait/prey under investigation (i.e. when probing PPI of mammalian proteins in Y2H).
Principles of the Y2H technology have also been adapted to mammalian systems, which circumvent some of the aforementioned drawbacks of Y2H<span class="citation" data-cites="1DnPkHRsA"><sup><a href="#ref-1DnPkHRsA" role="doc-biblioref">879</a></sup></span>.</p>
<p>Perhaps the most commonly applied method of detecting and validating PPIs in vitro is affinity purification (AP, also known as affinity chromatography) of co-immunoprecipitation (Co-IP) either coupled with SDS-PAGE/immunoblotting or mass spectrometry to determine the identity of interacting proteins.
AP typically relies on the isolation of a transgenic POI by an epitope tag (using epitope-specific matrix-conjugated proteins (antibodies or epitope-binding proteins)), while Co-IP harnesses specific antibodies directly targeting the POI.
Specific interactors are expected to be enriched compared to the negative control (i.e an isotype control antibody, a knockout cell line or empty matrix).
AP is not solely restricted to detecting PPIs, but can also be adapted to protein interactions with other biomolecules such as RNA<span class="citation" data-cites="1G5hcMdry"><sup><a href="#ref-1G5hcMdry" role="doc-biblioref">880</a></sup></span>.
It should be noted that AP and Co-IP can return multiple potential binding partners, many of which might be artefactual due to loss of cellular compartmentalization during sample preparation.</p>
<p>To reduce the probability of such artefacts and increase the confidence of a specific interaction, reciprocal affinity purification (by pulldown of each interaction partner) or in situ imaging might be performed (i.e. using fluorescence resonance energy transfer (FRET)<span class="citation" data-cites="636RraOs"><sup><a href="#ref-636RraOs" role="doc-biblioref">881</a></sup></span>, split-protein systems<span class="citation" data-cites="zTlZpc1k"><sup><a href="#ref-zTlZpc1k" role="doc-biblioref">882</a></sup></span>, proximity ligation assay<span class="citation" data-cites="13oBuESdH"><sup><a href="#ref-13oBuESdH" role="doc-biblioref">883</a></sup></span> and immunofluorescence microscopy).</p>
<p>Forster and bioluminescence resonance energy transfer (FRET / BRET) can be used for in situ visualization of protein proximities and therefore PPIs.
In FRET, non-radiative energy transfer between donor and receptor chromophores (each fused to prey and bait proteins, respectively), results in the emission of a characteristic fluorescence signal only when both prey and bait are in very close proximity (1-10 nm distance) and a suitable light source for donor excitation is provided<span class="citation" data-cites="feT5EHvH"><sup><a href="#ref-feT5EHvH" role="doc-biblioref">884</a></sup></span>.</p>
<p>The underlying principle of BRET is similar to that of FRET but with the exception of using a chemical substrate which activates bioluminescent donor, such as luciferase, resulting in energy transfer to a fluorescent acceptor molecule<span class="citation" data-cites="o1iijZBV 168qPRxga"><sup><a href="#ref-o1iijZBV" role="doc-biblioref">885</a>,<a href="#ref-168qPRxga" role="doc-biblioref">886</a></sup></span>.
The main advantages of BRET over FRET are independence from an external light source (which can result in photobleaching), but requires at least one of the POIs to be fused to the donor (while in FRET, donor and acceptor can be chemically conjugated to POI-specific antibodies)<span class="citation" data-cites="o1iijZBV"><sup><a href="#ref-o1iijZBV" role="doc-biblioref">885</a></sup></span>.
FRET can be particularly useful in investigating cell surface protein interactions when using specific antibodies conjugated to donor and acceptor probes as antibodies are not cell-permeable and therefore restricted to targets presented on the cell surface in the absence of membrane permeabilization agents.
Other fluorescence-based PPI assays encompass Fluorescence correlation spectroscopy (FCS) and fluorescence cross-correlation spectroscopy (FCCS).
These methods use small volumes of fluorescently labelled proteins and can determine their diffusion coefficients, which change in when proteins form a complex<span class="citation" data-cites="6egqfZ9W"><sup><a href="#ref-6egqfZ9W" role="doc-biblioref">887</a></sup></span>.</p>
<p>Proximity labelling methods (Proximity ligation and enzymatic proximity labelling (BirA, APEX2, HRP) can surveil labile or transient interaction in live cells in a high-throughput format when coupled with target identification by MS<span class="citation" data-cites="ZhoKgqsC"><sup><a href="#ref-ZhoKgqsC" role="doc-biblioref">888</a></sup></span>,<span class="citation" data-cites="RXiPqFRh"><sup><a href="#ref-RXiPqFRh" role="doc-biblioref">889</a></sup></span>.
These approaches harness a biotin ligase (i.e. BirA, BioID2, AirID, BASU, APEX2, HRP) fused to a POI whose interactome is to be determined. In the presence of biotin (for BirA, BioID2, AirID, BASU, APEX2 and HRP) or a biotin-phenol derivative (for APEX2), the biotin ligase will activate the biotin(-phenol) which then covalently biotinylates any protein in close proximity.
The activated biotin has a short half-life, ensuring that the effective labelling radius is typically restricted to approximately 10 nm.
Biotinylated proteins are isolated by affinity purification with streptavidin-conjugated beads and identified by mass spectrometry or SDS-PAGE/immunoblotting.
TurboID, miniTurboID and ultraID, promiscuous biotin ligases faster than BirA, have been developed allowing for shorter treatment times and decreased background signal.
The choice of a biotin ligase variant depends on the POI and experimental setup, but in general HRP does not work in cytoplasmic environments where conditions are chemically reducing, but is suitable for labelling proteins extracellular face of the plasma membrane or in the endoplasmic reticulum and golgi apparatus.
While TurboID and similar variants have fast kinetics, they can cause depletion of endogenous biotin and therefore cytotoxicity.</p>
<p>A major drawback shared by all variants described above is that they necessitate fusion to the POI, which might alter its physiological behavior and give rise to false positives or false negatives.
Moreover, detecting a biotin-labelled protein does not unequivocally designate it as an interaction partner as spatial proximity to the POI-biotin ligase fusion protein without direct binding can result in biotinylation.
The inclusion of controls, such as expression of the biotinylating enzyme alone in the cellular compartment of interest, is therefore particularly important for enzymatic proximity labelling methods.</p>
<p>The <em>in situ</em> proximity ligation assay (PLA) combines the specificity of antibodies with the signal amplification capacity of a DNA polymerase reaction.
Here, two antibodies, each conjugated to a short single-strand DNA (ssDNA) tag and each specific to one of the two proteins whose interaction is under investigation, are added to fixed cells or tissue.
Once bound to their respective targets and only when in direct proximity, the addition of two connector oligonucleotides complementary to each tag ssDNA tag and phi29 DNA polymerase, triggers isothermal rolling circle amplification, eventually resulting in the generation of continuous stretches of repetitive DNA.
These DNA products can then be visualized by in situ hybridization with fluorescently labelled oligonucleotides (see<span class="citation" data-cites="1F8ZvcEeH"><sup><a href="#ref-1F8ZvcEeH" role="doc-biblioref">890</a></sup></span> for a detailed discussion).
PLA has the advantage of visualizing the two interacting proteins in their native environment when high-resolution microscopy is used as a readout.</p>
<p>Chemical cross-linking (XL) of proteins can determine PPIs with amino-acid level resolution, and can thereby give valuable insights into the orientation of two or more proteins relative to one another<span class="citation" data-cites="3LymzGVH"><sup><a href="#ref-3LymzGVH" role="doc-biblioref">891</a></sup></span>.
Recent technical advances also enabled the visualization of protein-RNA interaction<span class="citation" data-cites="yxw6FRju"><sup><a href="#ref-yxw6FRju" role="doc-biblioref">892</a></sup></span>.
Various XL chemistries are available (amine-reactive, sulfhydryl and photoreactive crosslinkers; reversible vs irreversible) and cross-linked proteins detected by mass spectrometry<span class="citation" data-cites="4T3MaN5d"><sup><a href="#ref-4T3MaN5d" role="doc-biblioref">893</a></sup></span>.
In general, applying XL-MS to a mixture of interacting, purified proteins is preferable to in situ XL of complex protein mixtures (i.e. cell lysate) as detection and deconvolution of XL peptides is technically and computationally challenging.</p>
<p>Surface plasmon resonance can accurately measure several key kinetics of PPIs with high accuracy (e.g. association and dissociation kinetics, stoichiometry, affinity)<span class="citation" data-cites="H3H5bwtx"><sup><a href="#ref-H3H5bwtx" role="doc-biblioref">894</a></sup></span>.
It relies on the quantification of refractive index changes of polarized light shone onto a sensor chip containing a prey protein immobilized on a metal surface (typically gold).
When prey and bait proteins interact, the mass concentration at the metal interface changes, altering the refractive index and SPR angle (intensity of the refracted light).
 </p>
<h2 class="page_break_before" id="acknowledgements">Acknowledgements</h2>
<p>The authors thank Phil Wilmarth for helpful input.
Identification of certain commercial equipment, instruments, software, or materials does not imply recommendation or endorsement by the National Institute of Standards and Technology, nor does it imply that the products identified are necessarily the best available for the purpose.
The authors thank Dasom Hwang for help with graphic design.
The authors thank Anthony Gitter and Daniem Himmelstein for assistance using manubot.
The authors thank Jordan Burton and Pierre-Alexander Mücke for minor edits to the text.
This manuscript was written collaboratively using manubot<span class="citation" data-cites="YuJbg3zO"><sup><a href="#ref-YuJbg3zO" role="doc-biblioref">895</a></sup></span>.
The live and evolving version where anyone can contribute can be found here https://github.com/jessegmeyerlab/proteomics-tutorial/tree/main.</p>
<h2 id="author-contributions">Author contributions</h2>
<p>JGM was assigned last author as the project initiator and leader.
RLM was assigned second to last author based on their leadership role in curating all sections.
All other authors were ordered by estimating their contributions using a quantitative score.
The score for each author was a sum of the number of sentences added plus 33 lines for each figure added.
Scores were adjusted for confounding factors, such as split contributions between multiple contributors.
Authors with similar scores were assigned equal contributions.</p>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-tzsjhzo9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(1) </div><div class="csl-right-inline">Martin-Baniandres, P.; Lan, W.-H.; Board, S.; Romero-Ruiz, M.; Garcia-Manyes, S.; Qing, Y.; Bayley, H. Enzyme-Less Nanopore Detection of Post-Translational Modifications Within Long Polypeptides. <em>Nat. Nanotechnol.</em> <strong>2023</strong>, <em>18</em> (11), 1335–1340. <a href="https://doi.org/10.1038/s41565-023-01462-8">https://doi.org/10.1038/s41565-023-01462-8</a>.</div>
</div>
<div id="ref-dZHIMQZQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(2) </div><div class="csl-right-inline">Wang, X.; Thomas, T.-M.; Ren, R.; Zhou, Y.; Zhang, P.; Li, J.; Cai, S.; Liu, K.; Ivanov, A. P.; Herrmann, A.; Edel, J. B. Nanopore Detection Using Supercharged Polypeptide Molecular Carriers. <em>J. Am. Chem. Soc.</em> <strong>2023</strong>, <em>145</em> (11), 6371–6382. <a href="https://doi.org/10.1021/jacs.2c13465">https://doi.org/10.1021/jacs.2c13465</a>.</div>
</div>
<div id="ref-XUmjaJtI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(3) </div><div class="csl-right-inline">Yusko, E. C.; Bruhn, B. R.; Eggenberger, O. M.; Houghtaling, J.; Rollings, R. C.; Walsh, N. C.; Nandivada, S.; Pindrus, M.; Hall, A. R.; Sept, D.; Li, J.; Kalonia, D. S.; Mayer, M. Real-Time Shape Approximation and Fingerprinting of Single Proteins Using a Nanopore. <em>Nature Nanotech</em> <strong>2016</strong>, <em>12</em> (4), 360–367. <a href="https://doi.org/10.1038/nnano.2016.267">https://doi.org/10.1038/nnano.2016.267</a>.</div>
</div>
<div id="ref-vmYTdWAl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(4) </div><div class="csl-right-inline">Afshar Bakshloo, M.; Kasianowicz, J. J.; Pastoriza-Gallego, M.; Mathé, J.; Daniel, R.; Piguet, F.; Oukhaled, A. Nanopore-Based Protein Identification. <em>J. Am. Chem. Soc.</em> <strong>2022</strong>, <em>144</em> (6), 2716–2725. <a href="https://doi.org/10.1021/jacs.1c11758">https://doi.org/10.1021/jacs.1c11758</a>.</div>
</div>
<div id="ref-rpZdECmv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(5) </div><div class="csl-right-inline">Swaminathan, J.; Boulgakov, A. A.; Hernandez, E. T.; Bardo, A. M.; Bachman, J. L.; Marotta, J.; Johnson, A. M.; Anslyn, E. V.; Marcotte, E. M. Highly Parallel Single-Molecule Identification of Proteins in Zeptomole-Scale Mixtures. <em>Nat Biotechnol</em> <strong>2018</strong>, <em>36</em> (11), 1076–1082. <a href="https://doi.org/10.1038/nbt.4278">https://doi.org/10.1038/nbt.4278</a>.</div>
</div>
<div id="ref-qRXV6ZBf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(6) </div><div class="csl-right-inline">Hunt, D. F.; Buko, A. M.; Ballard, J. M.; Shabanowitz, J.; Giordani, A. B. Sequence Analysis of Polypeptides by Collision Activated Dissociation on a Triple Quadrupole Mass Spectrometer. <em>Biol. Mass Spectrom.</em> <strong>1981</strong>, <em>8</em> (9), 397–408. <a href="https://doi.org/10.1002/bms.1200080909">https://doi.org/10.1002/bms.1200080909</a>.</div>
</div>
<div id="ref-FV1jAbXp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(7) </div><div class="csl-right-inline">Gibson, B. W.; Biemann, K. Strategy for the Mass Spectrometric Verification and Correction of the Primary Structures of Proteins Deduced from Their DNA Sequences. <em>Proc. Natl. Acad. Sci. U.S.A.</em> <strong>1984</strong>, <em>81</em> (7), 1956–1960. <a href="https://doi.org/10.1073/pnas.81.7.1956">https://doi.org/10.1073/pnas.81.7.1956</a>.</div>
</div>
<div id="ref-4Ajjx5fo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(8) </div><div class="csl-right-inline">Fenn, J. B.; Mann, M.; Meng, C. K.; Wong, S. F.; Whitehouse, C. M. Electrospray Ionization for Mass Spectrometry of Large Biomolecules. <em>Science</em> <strong>1989</strong>, <em>246</em> (4926), 64–71. <a href="https://doi.org/10.1126/science.2675315">https://doi.org/10.1126/science.2675315</a>.</div>
</div>
<div id="ref-15Ye4Za7Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(9) </div><div class="csl-right-inline">Tanaka, K.; Waki, H.; Ido, Y.; Akita, S.; Yoshida, Y.; Yoshida, T.; Matsuo, T. Protein and Polymer Analyses up to &lt;i&gt;m/z&lt;/i&gt; 100 000 by Laser Ionization Time‐of‐flight Mass Spectrometry. <em>Rapid Comm Mass Spectrometry</em> <strong>1988</strong>, <em>2</em> (8), 151–153. <a href="https://doi.org/10.1002/rcm.1290020802">https://doi.org/10.1002/rcm.1290020802</a>.</div>
</div>
<div id="ref-pG7zq91Q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(10) </div><div class="csl-right-inline">Eng, J. K.; McCormack, A. L.; Yates, J. R. An Approach to Correlate Tandem Mass Spectral Data of Peptides with Amino Acid Sequences in a Protein Database. <em>J Am Soc Mass Spectrom</em> <strong>1994</strong>, <em>5</em> (11), 976–989. <a href="https://doi.org/10.1016/1044-0305(94)80016-2">https://doi.org/10.1016/1044-0305(94)80016-2</a>.</div>
</div>
<div id="ref-ujahSo3G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(11) </div><div class="csl-right-inline">Wolters, D. A.; Washburn, M. P.; Yates, J. R. An Automated Multidimensional Protein Identification Technology for Shotgun Proteomics. <em>Anal. Chem.</em> <strong>2001</strong>, <em>73</em> (23), 5683–5690. <a href="https://doi.org/10.1021/ac010617e">https://doi.org/10.1021/ac010617e</a>.</div>
</div>
<div id="ref-mw5jxASm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(12) </div><div class="csl-right-inline">Nesvizhskii, A. I.; Keller, A.; Kolker, E.; Aebersold, R. A Statistical Model for Identifying Proteins by Tandem Mass Spectrometry. <em>Anal. Chem.</em> <strong>2003</strong>, <em>75</em> (17), 4646–4658. <a href="https://doi.org/10.1021/ac0341261">https://doi.org/10.1021/ac0341261</a>.</div>
</div>
<div id="ref-15RjczuIy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(13) </div><div class="csl-right-inline">Elias, J. E.; Gygi, S. P. Target-Decoy Search Strategy for Increased Confidence in Large-Scale Protein Identifications by Mass Spectrometry. <em>Nat Methods</em> <strong>2007</strong>, <em>4</em> (3), 207–214. <a href="https://doi.org/10.1038/nmeth1019">https://doi.org/10.1038/nmeth1019</a>.</div>
</div>
<div id="ref-16QSCuG38" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(14) </div><div class="csl-right-inline">Aebersold, R.; Mann, M. Mass-Spectrometric Exploration of Proteome Structure and Function. <em>Nature</em> <strong>2016</strong>, <em>537</em> (7620), 347–355. <a href="https://doi.org/10.1038/nature19949">https://doi.org/10.1038/nature19949</a>.</div>
</div>
<div id="ref-UsGpigc0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(15) </div><div class="csl-right-inline">Kelleher, N. L.; Lin, H. Y.; Valaskovic, G. A.; Aaserud, D. J.; Fridriksson, E. K.; McLafferty, F. W. Top Down Versus Bottom Up Protein Characterization by Tandem High-Resolution Mass Spectrometry. <em>J. Am. Chem. Soc.</em> <strong>1999</strong>, <em>121</em> (4), 806–812. <a href="https://doi.org/10.1021/ja973655h">https://doi.org/10.1021/ja973655h</a>.</div>
</div>
<div id="ref-6a8Xt7SG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(16) </div><div class="csl-right-inline">Smith, L. M.; Kelleher, N. L. Proteoform: A Single Term Describing Protein Complexity. <em>Nat Methods</em> <strong>2013</strong>, <em>10</em> (3), 186–187. <a href="https://doi.org/10.1038/nmeth.2369">https://doi.org/10.1038/nmeth.2369</a>.</div>
</div>
<div id="ref-6hFikhYz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(17) </div><div class="csl-right-inline">Kelleher, N. L. Peer Reviewed: Top-Down Proteomics. <em>Anal. Chem.</em> <strong>2004</strong>, <em>76</em> (11), 196 A–203 A. <a href="https://doi.org/10.1021/ac0415657">https://doi.org/10.1021/ac0415657</a>.</div>
</div>
<div id="ref-a6Z8FNxW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(18) </div><div class="csl-right-inline">Cupp-Sutton, K. A.; Wu, S. High-Throughput Quantitative Top-down Proteomics. <em>Mol. Omics</em> <strong>2020</strong>, <em>16</em> (2), 91–99. <a href="https://doi.org/10.1039/c9mo00154a">https://doi.org/10.1039/c9mo00154a</a>.</div>
</div>
<div id="ref-h00Trxkt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(19) </div><div class="csl-right-inline">Smith, L. M.; Kelleher, N. L. Proteoforms as the Next Proteomics Currency. <em>Science</em> <strong>2018</strong>, <em>359</em> (6380), 1106–1107. <a href="https://doi.org/10.1126/science.aat1884">https://doi.org/10.1126/science.aat1884</a>.</div>
</div>
<div id="ref-oe85BKlv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(20) </div><div class="csl-right-inline">Aebersold, R.; Agar, J. N.; Amster, I. J.; Baker, M. S.; Bertozzi, C. R.; Boja, E. S.; Costello, C. E.; Cravatt, B. F.; Fenselau, C.; Garcia, B. A.; Ge, Y.; Gunawardena, J.; Hendrickson, R. C.; Hergenrother, P. J.; Huber, C. G.; Ivanov, A. R.; Jensen, O. N.; Jewett, M. C.; Kelleher, N. L.; Kiessling, L. L.; Krogan, N. J.; Larsen, M. R.; Loo, J. A.; Ogorzalek Loo, R. R.; Lundberg, E.; MacCoss, M. J.; Mallick, P.; Mootha, V. K.; Mrksich, M.; Muir, T. W.; Patrie, S. M.; Pesavento, J. J.; Pitteri, S. J.; Rodriguez, H.; Saghatelian, A.; Sandoval, W.; Schlüter, H.; Sechi, S.; Slavoff, S. A.; Smith, L. M.; Snyder, M. P.; Thomas, P. M.; Uhlén, M.; Van Eyk, J. E.; Vidal, M.; Walt, D. R.; White, F. M.; Williams, E. R.; Wohlschlager, T.; Wysocki, V. H.; Yates, N. A.; Young, N. L.; Zhang, B. How Many Human Proteoforms Are There? <em>Nat Chem Biol</em> <strong>2018</strong>, <em>14</em> (3), 206–214. <a href="https://doi.org/10.1038/nchembio.2576">https://doi.org/10.1038/nchembio.2576</a>.</div>
</div>
<div id="ref-CwZoo7w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(21) </div><div class="csl-right-inline">Chen, B.; Brown, K. A.; Lin, Z.; Ge, Y. Top-Down Proteomics: Ready for Prime Time? <em>Anal. Chem.</em> <strong>2017</strong>, <em>90</em> (1), 110–127. <a href="https://doi.org/10.1021/acs.analchem.7b04747">https://doi.org/10.1021/acs.analchem.7b04747</a>.</div>
</div>
<div id="ref-pRTFSroJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(22) </div><div class="csl-right-inline">Meyer, J. G.; Niemi, N. M.; Pagliarini, D. J.; Coon, J. J. Quantitative Shotgun Proteome Analysis by Direct Infusion. <em>Nat Methods</em> <strong>2020</strong>, <em>17</em> (12), 1222–1228. <a href="https://doi.org/10.1038/s41592-020-00999-z">https://doi.org/10.1038/s41592-020-00999-z</a>.</div>
</div>
<div id="ref-DqzDdsRo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(23) </div><div class="csl-right-inline">Jiang, Y.; Hutton, A.; Cranney, C. W.; Meyer, J. G. Label-Free Quantification from Direct Infusion Shotgun Proteome Analysis (DISPA-LFQ) with CsoDIAq Software. <em>Anal. Chem.</em> <strong>2022</strong>. <a href="https://doi.org/10.1021/acs.analchem.2c02249">https://doi.org/10.1021/acs.analchem.2c02249</a>.</div>
</div>
<div id="ref-1339IM4Kb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(24) </div><div class="csl-right-inline">Jiang, Y.; Salladay-Perez, I.; Momenzadeh, A.; Covarrubias, A. J.; Meyer, J. G. Simultaneous Multi-Omics Analysis by Direct Infusion Mass Spectrometry (SMAD-MS), 2023. <a href="https://doi.org/10.1101/2023.06.26.546628">https://doi.org/10.1101/2023.06.26.546628</a>.</div>
</div>
<div id="ref-JTW0FD8g" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(25) </div><div class="csl-right-inline">Omenn, G. S.; Lane, L.; Overall, C. M.; Pineau, C.; Packer, N. H.; Cristea, I. M.; Lindskog, C.; Weintraub, S. T.; Orchard, S.; Roehrl, M. H. A.; Nice, E.; Liu, S.; Bandeira, N.; Chen, Y.-J.; Guo, T.; Aebersold, R.; Moritz, R. L.; Deutsch, E. W. The 2022 Report on the Human Proteome from the HUPO Human Proteome Project. <em>J Proteome Res</em> <strong>2022</strong>, <em>22</em> (4), 1024–1042. <a href="https://doi.org/10.1021/acs.jproteome.2c00498">https://doi.org/10.1021/acs.jproteome.2c00498</a>.</div>
</div>
<div id="ref-Iei0Ms6X" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(26) </div><div class="csl-right-inline">Beck, M.; Schmidt, A.; Malmstroem, J.; Claassen, M.; Ori, A.; Szymborska, A.; Herzog, F.; Rinner, O.; Ellenberg, J.; Aebersold, R. The Quantitative Proteome of a Human Cell Line. <em>Molecular Systems Biology</em> <strong>2011</strong>, <em>7</em> (1). <a href="https://doi.org/10.1038/msb.2011.82">https://doi.org/10.1038/msb.2011.82</a>.</div>
</div>
<div id="ref-OQN3Pb5c" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(27) </div><div class="csl-right-inline">Jiang, L.; Wang, M.; Lin, S.; Jian, R.; Li, X.; Chan, J.; Dong, G.; Fang, H.; Robinson, A. E.; Snyder, M. P.; Aguet, F.; Anand, S.; Ardlie, K. G.; Gabriel, S.; Getz, G.; Graubert, A.; Hadley, K.; Handsaker, R. E.; Huang, K. H.; Kashin, S.; MacArthur, D. G.; Meier, S. R.; Nedzel, J. L.; Nguyen, D. Y.; Segrè, A. V.; Todres, E.; Balliu, B.; Barbeira, A. N.; Battle, A.; Bonazzola, R.; Brown, A.; Brown, C. D.; Castel, S. E.; Conrad, D.; Cotter, D. J.; Cox, N.; Das, S.; de Goede, O. M.; Dermitzakis, E. T.; Engelhardt, B. E.; Eskin, E.; Eulalio, T. Y.; Ferraro, N. M.; Flynn, E.; Fresard, L.; Gamazon, E. R.; Garrido-Martín, D.; Gay, N. R.; Guigó, R.; Hamel, A. R.; He, Y.; Hoffman, P. J.; Hormozdiari, F.; Hou, L.; Im, H. K.; Jo, B.; Kasela, S.; Kellis, M.; Kim-Hellmuth, S.; Kwong, A.; Lappalainen, T.; Li, X.; Liang, Y.; Mangul, S.; Mohammadi, P.; Montgomery, S. B.; Muñoz-Aguirre, M.; Nachun, D. C.; Nobel, A. B.; Oliva, M.; Park, Y.; Park, Y.; Parsana, P.; Reverter, F.; Rouhana, J. M.; Sabatti, C.; Saha, A.; Skol, A. D.; Stephens, M.; Stranger, B. E.; Strober, B. J.; Teran, N. A.; Viñuela, A.; Wang, G.; Wen, X.; Wright, F.; Wucher, V.; Zou, Y.; Ferreira, P. G.; Li, G.; Melé, M.; Yeger-Lotem, E.; Barcus, M. E.; Bradbury, D.; Krubit, T.; McLean, J. A.; Qi, L.; Robinson, K.; Roche, N. V.; Smith, A. M.; Sobin, L.; Tabor, D. E.; Undale, A.; Bridge, J.; Brigham, L. E.; Foster, B. A.; Gillard, B. M.; Hasz, R.; Hunter, M.; Johns, C.; Johnson, M.; Karasik, E.; Kopen, G.; Leinweber, W. F.; McDonald, A.; Moser, M. T.; Myer, K.; Ramsey, K. D.; Roe, B.; Shad, S.; Thomas, J. A.; Walters, G.; Washington, M.; Wheeler, J.; Jewell, S. D.; Rohrer, D. C.; Valley, D. R.; Davis, D. A.; Mash, D. C.; Branton, P. A.; Barker, L. K.; Gardiner, H. M.; Mosavel, M.; Siminoff, L. A.; Flicek, P.; Haeussler, M.; Juettemann, T.; Kent, W. J.; Lee, C. M.; Powell, C. C.; Rosenbloom, K. R.; Ruffier, M.; Sheppard, D.; Taylor, K.; Trevanion, S. J.; Zerbino, D. R.; Abell, N. S.; Akey, J.; Chen, L.; Demanelis, K.; Doherty, J. A.; Feinberg, A. P.; Hansen, K. D.; Hickey, P. F.; Jasmine, F.; Kaul, R.; Kibriya, M. G.; Li, J. B.; Li, Q.; Linder, S. E.; Pierce, B. L.; Rizzardi, L. F.; Smith, K. S.; Stamatoyannopoulos, J.; Tang, H.; Carithers, L. J.; Guan, P.; Koester, S. E.; Little, A. R.; Moore, H. M.; Nierras, C. R.; Rao, A. K.; Vaught, J. B.; Volpi, S. A Quantitative Proteome Map of the Human Body. <em>Cell</em> <strong>2020</strong>, <em>183</em> (1), 269–283.e19. <a href="https://doi.org/10.1016/j.cell.2020.08.036">https://doi.org/10.1016/j.cell.2020.08.036</a>.</div>
</div>
<div id="ref-i4DXPyG5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(28) </div><div class="csl-right-inline">Sinitcyn, P.; Richards, A. L.; Weatheritt, R. J.; Brademan, D. R.; Marx, H.; Shishkova, E.; Meyer, J. G.; Hebert, A. S.; Westphall, M. S.; Blencowe, B. J.; Cox, J.; Coon, J. J. Global Detection of Human Variants and Isoforms by Deep Proteome Sequencing. <em>Nat Biotechnol</em> <strong>2023</strong>, <em>41</em> (12), 1776–1786. <a href="https://doi.org/10.1038/s41587-023-01714-x">https://doi.org/10.1038/s41587-023-01714-x</a>.</div>
</div>
<div id="ref-m2ZOJt1R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(29) </div><div class="csl-right-inline">Ramazi, S.; Zahiri, J. Post-Translational Modifications in Proteins: Resources, Tools and Prediction Methods. <em>Database</em> <strong>2021</strong>, <em>2021</em>. <a href="https://doi.org/10.1093/database/baab012">https://doi.org/10.1093/database/baab012</a>.</div>
</div>
<div id="ref-uLIaWFNx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(30) </div><div class="csl-right-inline">Ramazi, S.; Zahiri, J. Posttranslational Modifications in Proteins: Resources, Tools and Prediction Methods. <em>Database (Oxford)</em> <strong>2021</strong>, <em>2021</em>. <a href="https://doi.org/10.1093/database/baab012">https://doi.org/10.1093/database/baab012</a>.</div>
</div>
<div id="ref-OEP3MV7r" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(31) </div><div class="csl-right-inline">Wang, Y.-C.; Peterson, S. E.; Loring, J. F. Protein Post-Translational Modifications and Regulation of Pluripotency in Human Stem Cells. <em>Cell Res</em> <strong>2013</strong>, <em>24</em> (2), 143–160. <a href="https://doi.org/10.1038/cr.2013.151">https://doi.org/10.1038/cr.2013.151</a>.</div>
</div>
<div id="ref-fAHUecLj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(32) </div><div class="csl-right-inline">Rogers, L. D.; Overall, C. M. Proteolytic Post-Translational Modification of Proteins: Proteomic Tools and Methodology. <em>Mol Cell Proteomics</em> <strong>2013</strong>, <em>12</em> (12), 3532–3542. <a href="https://doi.org/10.1074/mcp.m113.031310">https://doi.org/10.1074/mcp.m113.031310</a>.</div>
</div>
<div id="ref-yYHlJgbC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(33) </div><div class="csl-right-inline">Kitamura, N.; Galligan, J. J. A Global View of the Human Post-Translational Modification Landscape. <em>Biochemical Journal</em> <strong>2023</strong>, <em>480</em> (16), 1241–1265. <a href="https://doi.org/10.1042/bcj20220251">https://doi.org/10.1042/bcj20220251</a>.</div>
</div>
<div id="ref-e078CIsS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(34) </div><div class="csl-right-inline">Ardito, F.; Giuliani, M.; Perrone, D.; Troiano, G.; Lo Muzio, L. The Crucial Role of Protein Phosphorylation in Cell Signaling and Its Use as Targeted Therapy (Review). <em>Int J Mol Med</em> <strong>2017</strong>, <em>40</em> (2), 271–280. <a href="https://doi.org/10.3892/ijmm.2017.3036">https://doi.org/10.3892/ijmm.2017.3036</a>.</div>
</div>
<div id="ref-oTq7yQcv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(35) </div><div class="csl-right-inline">Niemi, N. M.; MacKeigan, J. P. Mitochondrial Phosphorylation in Apoptosis: Flipping the Death Switch. <em>Antioxid Redox Signal</em> <strong>2012</strong>, <em>19</em> (6), 572–582. <a href="https://doi.org/10.1089/ars.2012.4982">https://doi.org/10.1089/ars.2012.4982</a>.</div>
</div>
<div id="ref-1DDSEV4N5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(36) </div><div class="csl-right-inline">Ruvolo, P. P.; Deng, X.; May, W. S. Phosphorylation of Bcl2 and Regulation of Apoptosis. <em>Leukemia</em> <strong>2001</strong>, <em>15</em> (4), 515–522. <a href="https://doi.org/10.1038/sj.leu.2402090">https://doi.org/10.1038/sj.leu.2402090</a>.</div>
</div>
<div id="ref-1CHaUvQ1A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(37) </div><div class="csl-right-inline">Lecker, S. H.; Goldberg, A. L.; Mitch, W. E. Protein Degradation by the Ubiquitin-Proteasome Pathway in Normal and Disease States. <em>J Am Soc Nephrol</em> <strong>2006</strong>, <em>17</em> (7), 1807–1819. <a href="https://doi.org/10.1681/asn.2006010083">https://doi.org/10.1681/asn.2006010083</a>.</div>
</div>
<div id="ref-M4mSYOgE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(38) </div><div class="csl-right-inline">Eifler, K.; Vertegaal, A. C. O. SUMOylation-Mediated Regulation of Cell Cycle Progression and Cancer. <em>Trends Biochem Sci</em> <strong>2015</strong>, <em>40</em> (12), 779–793. <a href="https://doi.org/10.1016/j.tibs.2015.09.006">https://doi.org/10.1016/j.tibs.2015.09.006</a>.</div>
</div>
<div id="ref-4iAess79" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(39) </div><div class="csl-right-inline">Lumpkin, R. J.; Gu, H.; Zhu, Y.; Leonard, M.; Ahmad, A. S.; Clauser, K. R.; Meyer, J. G.; Bennett, E. J.; Komives, E. A. Site-Specific Identification and Quantitation of Endogenous SUMO Modifications Under Native Conditions. <em>Nat Commun</em> <strong>2017</strong>, <em>8</em> (1), 1171. <a href="https://doi.org/10.1038/s41467-017-01271-3">https://doi.org/10.1038/s41467-017-01271-3</a>.</div>
</div>
<div id="ref-blcqlkZJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(40) </div><div class="csl-right-inline">Romancino, D. P.; Buffa, V.; Caruso, S.; Ferrara, I.; Raccosta, S.; Notaro, A.; Campos, Y.; Noto, R.; Martorana, V.; Cupane, A.; Giallongo, A.; d'Azzo, A.; Manno, M.; Bongiovanni, A. Palmitoylation Is a Post-Translational Modification of Alix Regulating the Membrane Organization of Exosome-Like Small Extracellular Vesicles. <em>Biochim Biophys Acta Gen Subj</em> <strong>2018</strong>, <em>1862</em> (12), 2879–2887. <a href="https://doi.org/10.1016/j.bbagen.2018.09.004">https://doi.org/10.1016/j.bbagen.2018.09.004</a>.</div>
</div>
<div id="ref-V8FVI0EN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(41) </div><div class="csl-right-inline">Clark, M. C.; Baum, L. G. T Cells Modulate Glycans on CD43 and CD45 During Development and Activation, Signal Regulation, and Survival. <em>Ann N Y Acad Sci</em> <strong>2012</strong>, <em>1253</em>, 58–67. <a href="https://doi.org/10.1111/j.1749-6632.2011.06304.x">https://doi.org/10.1111/j.1749-6632.2011.06304.x</a>.</div>
</div>
<div id="ref-889bLrDQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(42) </div><div class="csl-right-inline">Marth, J. D.; Grewal, P. K. Mammalian Glycosylation in Immunity. <em>Nat Rev Immunol</em> <strong>2008</strong>, <em>8</em> (11), 874–887. <a href="https://doi.org/10.1038/nri2417">https://doi.org/10.1038/nri2417</a>.</div>
</div>
<div id="ref-1FiLH8A0G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(43) </div><div class="csl-right-inline">Conibear, A. C. Deciphering Protein Post-Translational Modifications Using Chemical Biology Tools. <em>Nat Rev Chem</em> <strong>2020</strong>, <em>4</em> (12), 674–695. <a href="https://doi.org/10.1038/s41570-020-00223-8">https://doi.org/10.1038/s41570-020-00223-8</a>.</div>
</div>
<div id="ref-fzBRT8ZP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(44) </div><div class="csl-right-inline">Zhu, H.; Klemic, J. F.; Chang, S.; Bertone, P.; Casamayor, A.; Klemic, K. G.; Smith, D.; Gerstein, M.; Reed, M. A.; Snyder, M. Analysis of Yeast Protein Kinases Using Protein Chips. <em>Nat Genet</em> <strong>2000</strong>, <em>26</em> (3), 283–289. <a href="https://doi.org/10.1038/81576">https://doi.org/10.1038/81576</a>.</div>
</div>
<div id="ref-Wcw3KfEB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(45) </div><div class="csl-right-inline">Reimer, U.; Reineke, U.; Schneider-Mergener, J. Peptide Arrays: From Macro to Micro. <em>Curr Opin Biotechnol</em> <strong>2002</strong>, <em>13</em> (4), 315–320. <a href="https://doi.org/10.1016/s0958-1669(02)00339-7">https://doi.org/10.1016/s0958-1669(02)00339-7</a>.</div>
</div>
<div id="ref-QjEZsYB2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(46) </div><div class="csl-right-inline">Martyn, G. D.; Veggiani, G.; Kusebauch, U.; Morrone, S. R.; Yates, B. P.; Singer, A. U.; Tong, J.; Manczyk, N.; Gish, G.; Sun, Z.; Kurinov, I.; Sicheri, F.; Moran, M. F.; Moritz, R. L.; Sidhu, S. S. Engineered SH2 Domains for Targeted Phosphoproteomics. <em>ACS Chem Biol</em> <strong>2022</strong>, <em>17</em> (6), 1472–1484. <a href="https://doi.org/10.1021/acschembio.2c00051">https://doi.org/10.1021/acschembio.2c00051</a>.</div>
</div>
<div id="ref-UeFCwwR7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(47) </div><div class="csl-right-inline">Benjamini, Y.; Hochberg, Y. Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing. <em>Journal of the Royal Statistical Society Series B: Statistical Methodology</em> <strong>1995</strong>, <em>57</em> (1), 289–300. <a href="https://doi.org/10.1111/j.2517-6161.1995.tb02031.x">https://doi.org/10.1111/j.2517-6161.1995.tb02031.x</a>.</div>
</div>
<div id="ref-13Yxz6MsM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(48) </div><div class="csl-right-inline">Agard, N. J.; Wells, J. A. Methods for the Proteomic Identification of Protease Substrates. <em>Current Opinion in Chemical Biology</em> <strong>2009</strong>, <em>13</em> (5-6), 503–509. <a href="https://doi.org/10.1016/j.cbpa.2009.07.026">https://doi.org/10.1016/j.cbpa.2009.07.026</a>.</div>
</div>
<div id="ref-E5n1UfBA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(49) </div><div class="csl-right-inline">Villén, J.; Beausoleil, S. A.; Gerber, S. A.; Gygi, S. P. Large-Scale Phosphorylation Analysis of Mouse Liver. <em>Proc. Natl. Acad. Sci. U.S.A.</em> <strong>2007</strong>, <em>104</em> (5), 1488–1493. <a href="https://doi.org/10.1073/pnas.0609836104">https://doi.org/10.1073/pnas.0609836104</a>.</div>
</div>
<div id="ref-N0xHE81w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(50) </div><div class="csl-right-inline">Bodenmiller, B.; Malmstrom, J.; Gerrits, B.; Campbell, D.; Lam, H.; Schmidt, A.; Rinner, O.; Mueller, L. N.; Shannon, P. T.; Pedrioli, P. G.; Panse, C.; Lee, H.; Schlapbach, R.; Aebersold, R. PhosphoPep—a Phosphoproteome Resource for Systems Biology Research in
&lt;i&gt;Drosophila&lt;/i&gt;
Kc167 Cells. <em>Molecular Systems Biology</em> <strong>2007</strong>, <em>3</em> (1). <a href="https://doi.org/10.1038/msb4100182">https://doi.org/10.1038/msb4100182</a>.</div>
</div>
<div id="ref-4q5oxW6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(51) </div><div class="csl-right-inline">Kim, J.-E.; White, F. M. Quantitative Analysis of Phosphotyrosine Signaling Networks Triggered by CD3 and CD28 Costimulation in Jurkat Cells. <em>The Journal of Immunology</em> <strong>2006</strong>, <em>176</em> (5), 2833–2843. <a href="https://doi.org/10.4049/jimmunol.176.5.2833">https://doi.org/10.4049/jimmunol.176.5.2833</a>.</div>
</div>
<div id="ref-1E7oJsvFz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(52) </div><div class="csl-right-inline">Mayya, V.; Lundgren, D. H.; Hwang, S.-I.; Rezaul, K.; Wu, L.; Eng, J. K.; Rodionov, V.; Han, D. K. Quantitative Phosphoproteomic Analysis of T Cell Receptor Signaling Reveals System-Wide Modulation of Protein-Protein Interactions. <em>Sci. Signal.</em> <strong>2009</strong>, <em>2</em> (84). <a href="https://doi.org/10.1126/scisignal.2000007">https://doi.org/10.1126/scisignal.2000007</a>.</div>
</div>
<div id="ref-EiFGJCed" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(53) </div><div class="csl-right-inline">White, F. M. Quantitative Phosphoproteomic Analysis of Signaling Network Dynamics. <em>Current Opinion in Biotechnology</em> <strong>2008</strong>, <em>19</em> (4), 404–409. <a href="https://doi.org/10.1016/j.copbio.2008.06.006">https://doi.org/10.1016/j.copbio.2008.06.006</a>.</div>
</div>
<div id="ref-PNr94DXb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(54) </div><div class="csl-right-inline">Bagdonaite, I.; Malaker, S. A.; Polasky, D. A.; Riley, N. M.; Schjoldager, K.; Vakhrushev, S. Y.; Halim, A.; Aoki-Kinoshita, K. F.; Nesvizhskii, A. I.; Bertozzi, C. R.; Wandall, H. H.; Parker, B. L.; Thaysen-Andersen, M.; Scott, N. E. Glycoproteomics. <em>Nat Rev Methods Primers</em> <strong>2022</strong>, <em>2</em> (1). <a href="https://doi.org/10.1038/s43586-022-00128-4">https://doi.org/10.1038/s43586-022-00128-4</a>.</div>
</div>
<div id="ref-MH9QSnbc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(55) </div><div class="csl-right-inline">Varki, A. Biological Roles of Glycans. <em>Glycobiology</em> <strong>2016</strong>, <em>27</em> (1), 3–49. <a href="https://doi.org/10.1093/glycob/cww086">https://doi.org/10.1093/glycob/cww086</a>.</div>
</div>
<div id="ref-nSDlzuSO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(56) </div><div class="csl-right-inline">Ohtsubo, K.; Marth, J. D. Glycosylation in Cellular Mechanisms of Health and Disease. <em>Cell</em> <strong>2006</strong>, <em>126</em> (5), 855–867. <a href="https://doi.org/10.1016/j.cell.2006.08.019">https://doi.org/10.1016/j.cell.2006.08.019</a>.</div>
</div>
<div id="ref-buMJaA2S" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(57) </div><div class="csl-right-inline">Freeze, H. H.; Eklund, E. A.; Ng, B. G.; Patterson, M. C. Neurological Aspects of Human Glycosylation Disorders. <em>Annu. Rev. Neurosci.</em> <strong>2015</strong>, <em>38</em> (1), 105–125. <a href="https://doi.org/10.1146/annurev-neuro-071714-034019">https://doi.org/10.1146/annurev-neuro-071714-034019</a>.</div>
</div>
<div id="ref-Iygi3cGd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(58) </div><div class="csl-right-inline">Wiśniewski, J. R.; Zougman, A.; Nagaraj, N.; Mann, M. Universal Sample Preparation Method for Proteome Analysis. <em>Nat Methods</em> <strong>2009</strong>, <em>6</em> (5), 359–362. <a href="https://doi.org/10.1038/nmeth.1322">https://doi.org/10.1038/nmeth.1322</a>.</div>
</div>
<div id="ref-1FU2NfVYY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(59) </div><div class="csl-right-inline">Zougman, A.; Selby, P. J.; Banks, R. E. Suspension Trapping (STrap) Sample Preparation Method for Bottom‐up Proteomics Analysis. <em>Proteomics</em> <strong>2014</strong>, <em>14</em> (9), 1006–1000. <a href="https://doi.org/10.1002/pmic.201300553">https://doi.org/10.1002/pmic.201300553</a>.</div>
</div>
<div id="ref-yO8bBfo8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(60) </div><div class="csl-right-inline">Batth, T. S.; Tollenaere, MaximA. X.; Rüther, P.; Gonzalez-Franquesa, A.; Prabhakar, B. S.; Bekker-Jensen, S.; Deshmukh, A. S.; Olsen, J. V. Protein Aggregation Capture on Microparticles Enables Multipurpose Proteomics Sample Preparation*. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2019</strong>, <em>18</em> (5), 1027a–1035. <a href="https://doi.org/10.1074/mcp.tir118.001270">https://doi.org/10.1074/mcp.tir118.001270</a>.</div>
</div>
<div id="ref-7zlczgkb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(61) </div><div class="csl-right-inline">HaileMariam, M.; Eguez, R. V.; Singh, H.; Bekele, S.; Ameni, G.; Pieper, R.; Yu, Y. S-Trap, an Ultrafast Sample-Preparation Approach for Shotgun Proteomics. <em>J. Proteome Res.</em> <strong>2018</strong>, <em>17</em> (9), 2917–2924. <a href="https://doi.org/10.1021/acs.jproteome.8b00505">https://doi.org/10.1021/acs.jproteome.8b00505</a>.</div>
</div>
<div id="ref-t0hz9mu6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(62) </div><div class="csl-right-inline">Cao, J.; Boatner, L. M.; Desai, H. S.; Burton, N. R.; Armenta, E.; Chan, N. J.; Castellón, J. O.; Backus, K. M. Multiplexed CuAAC Suzuki–Miyaura Labeling for Tandem Activity-Based Chemoproteomic Profiling. <em>Anal. Chem.</em> <strong>2021</strong>, <em>93</em> (4), 2610–2618. <a href="https://doi.org/10.1021/acs.analchem.0c04726">https://doi.org/10.1021/acs.analchem.0c04726</a>.</div>
</div>
<div id="ref-VTprpeAT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(63) </div><div class="csl-right-inline">Yan, T.; Desai, H.; Boatner, L.; Yen, S.; Cao, J.; Palafox, M.; Jami-Alahmadi, Y.; Backus, K. SP3-FAIMS Chemoproteomics for High Coverage Profiling of the Human Cysteinome, 2020. <a href="https://doi.org/10.26434/chemrxiv.13487364.v1">https://doi.org/10.26434/chemrxiv.13487364.v1</a>.</div>
</div>
<div id="ref-yeQvImXS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(64) </div><div class="csl-right-inline">Meyer, J. G.; Kim, S.; Maltby, D. A.; Ghassemian, M.; Bandeira, N.; Komives, E. A. Expanding Proteome Coverage with Orthogonal-Specificity α-Lytic Proteases. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2014</strong>, <em>13</em> (3), 823–835. <a href="https://doi.org/10.1074/mcp.m113.034710">https://doi.org/10.1074/mcp.m113.034710</a>.</div>
</div>
<div id="ref-nOIi7YMn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(65) </div><div class="csl-right-inline">Oldfield, C. J.; Dunker, A. K. Intrinsically Disordered Proteins and Intrinsically Disordered Protein Regions. <em>Annu. Rev. Biochem.</em> <strong>2014</strong>, <em>83</em> (1), 553–584. <a href="https://doi.org/10.1146/annurev-biochem-072711-164947">https://doi.org/10.1146/annurev-biochem-072711-164947</a>.</div>
</div>
<div id="ref-aq0DyuaG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(66) </div><div class="csl-right-inline">Fatima, K.; Naqvi, F.; Younas, H. A Review: Molecular Chaperone-Mediated Folding, Unfolding and Disaggregation of Expressed Recombinant Proteins. <em>Cell Biochem Biophys</em> <strong>2021</strong>, <em>79</em> (2), 153–174. <a href="https://doi.org/10.1007/s12013-021-00970-5">https://doi.org/10.1007/s12013-021-00970-5</a>.</div>
</div>
<div id="ref-vsPwR5xs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(67) </div><div class="csl-right-inline">Jumper, J.; Evans, R.; Pritzel, A.; Green, T.; Figurnov, M.; Ronneberger, O.; Tunyasuvunakool, K.; Bates, R.; Žídek, A.; Potapenko, A.; Bridgland, A.; Meyer, C.; Kohl, S. A. A.; Ballard, A. J.; Cowie, A.; Romera-Paredes, B.; Nikolov, S.; Jain, R.; Adler, J.; Back, T.; Petersen, S.; Reiman, D.; Clancy, E.; Zielinski, M.; Steinegger, M.; Pacholska, M.; Berghammer, T.; Bodenstein, S.; Silver, D.; Vinyals, O.; Senior, A. W.; Kavukcuoglu, K.; Kohli, P.; Hassabis, D. Highly Accurate Protein Structure Prediction with AlphaFold. <em>Nature</em> <strong>2021</strong>, <em>596</em> (7873), 583–589. <a href="https://doi.org/10.1038/s41586-021-03819-2">https://doi.org/10.1038/s41586-021-03819-2</a>.</div>
</div>
<div id="ref-11Etsn1JV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(68) </div><div class="csl-right-inline">Jumper, J.; Hassabis, D. The Protein Structure Prediction Revolution and Its Implications for Medicine: 2023 Albert Lasker Basic Medical Research Award. <em>JAMA</em> <strong>2023</strong>, <em>330</em> (15), 1425–1426. <a href="https://doi.org/10.1001/jama.2023.17095">https://doi.org/10.1001/jama.2023.17095</a>.</div>
</div>
<div id="ref-waP2bBmk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(69) </div><div class="csl-right-inline">Holding, A. N. XL-MS: Protein Cross-Linking Coupled with Mass Spectrometry. <em>Methods</em> <strong>2015</strong>, <em>89</em>, 54–63. <a href="https://doi.org/10.1016/j.ymeth.2015.06.010">https://doi.org/10.1016/j.ymeth.2015.06.010</a>.</div>
</div>
<div id="ref-XYcoBTz1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(70) </div><div class="csl-right-inline">Steigenberger, B.; Albanese, P.; Heck, A. J. R.; Scheltema, R. A. To Cleave or Not To Cleave in XL-MS? <em>J. Am. Soc. Mass Spectrom.</em> <strong>2019</strong>, <em>31</em> (2), 196–206. <a href="https://doi.org/10.1021/jasms.9b00085">https://doi.org/10.1021/jasms.9b00085</a>.</div>
</div>
<div id="ref-v6mVdK34" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(71) </div><div class="csl-right-inline">Yu, C.; Huang, L. Cross-Linking Mass Spectrometry: An Emerging Technology for Interactomics and Structural Biology. <em>Anal. Chem.</em> <strong>2017</strong>, <em>90</em> (1), 144–165. <a href="https://doi.org/10.1021/acs.analchem.7b04431">https://doi.org/10.1021/acs.analchem.7b04431</a>.</div>
</div>
<div id="ref-JnaXzrse" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(72) </div><div class="csl-right-inline">Kaake, R. M.; Wang, X.; Burke, A.; Yu, C.; Kandur, W.; Yang, Y.; Novtisky, E. J.; Second, T.; Duan, J.; Kao, A.; Guan, S.; Vellucci, D.; Rychnovsky, S. D.; Huang, L. A New in Vivo Cross-Linking Mass Spectrometry Platform to Define Protein–Protein Interactions in Living Cells. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2014</strong>, <em>13</em> (12), 3533–3543. <a href="https://doi.org/10.1074/mcp.m114.042630">https://doi.org/10.1074/mcp.m114.042630</a>.</div>
</div>
<div id="ref-KW2lStHJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(73) </div><div class="csl-right-inline">Yang, B.; Wu, Y.-J.; Zhu, M.; Fan, S.-B.; Lin, J.; Zhang, K.; Li, S.; Chi, H.; Li, Y.-X.; Chen, H.-F.; Luo, S.-K.; Ding, Y.-H.; Wang, L.-H.; Hao, Z.; Xiu, L.-Y.; Chen, S.; Ye, K.; He, S.-M.; Dong, M.-Q. Identification of Cross-Linked Peptides from Complex Samples. <em>Nat Methods</em> <strong>2012</strong>, <em>9</em> (9), 904–906. <a href="https://doi.org/10.1038/nmeth.2099">https://doi.org/10.1038/nmeth.2099</a>.</div>
</div>
<div id="ref-10TqPZVdy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(74) </div><div class="csl-right-inline">Hoopmann, M. R.; Zelter, A.; Johnson, R. S.; Riffle, M.; MacCoss, M. J.; Davis, T. N.; Moritz, R. L. Kojak: Efficient Analysis of Chemically Cross-Linked Protein Complexes. <em>J Proteome Res</em> <strong>2015</strong>, <em>14</em> (5), 2190–2198. <a href="https://doi.org/10.1021/pr501321h">https://doi.org/10.1021/pr501321h</a>.</div>
</div>
<div id="ref-KIekIPp9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(75) </div><div class="csl-right-inline">Hoopmann, M. R.; Shteynberg, D. D.; Zelter, A.; Riffle, M.; Lyon, A. S.; Agard, D. A.; Luan, Q.; Nolen, B. J.; MacCoss, M. J.; Davis, T. N.; Moritz, R. L. Improved Analysis of Cross-Linking Mass Spectrometry Data with Kojak 2.0, Advanced by Integration into the Trans-Proteomic Pipeline. <em>J Proteome Res</em> <strong>2023</strong>, <em>22</em> (2), 647–655. <a href="https://doi.org/10.1021/acs.jproteome.2c00670">https://doi.org/10.1021/acs.jproteome.2c00670</a>.</div>
</div>
<div id="ref-jq9D2t6h" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(76) </div><div class="csl-right-inline">Rinner, O.; Seebacher, J.; Walzthoeni, T.; Mueller, L. N.; Beck, M.; Schmidt, A.; Mueller, M.; Aebersold, R. Identification of Cross-Linked Peptides from Large Sequence Databases. <em>Nat Methods</em> <strong>2008</strong>, <em>5</em> (4), 315–318. <a href="https://doi.org/10.1038/nmeth.1192">https://doi.org/10.1038/nmeth.1192</a>.</div>
</div>
<div id="ref-19ZFgWCt6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(77) </div><div class="csl-right-inline">Liu, F.; Rijkers, D. T. S.; Post, H.; Heck, A. J. R. Proteome-Wide Profiling of Protein Assemblies by Cross-Linking Mass Spectrometry. <em>Nature Methods</em> <strong>2015</strong>, <em>12</em> (12), 1179–1184. <a href="https://doi.org/doi.org/10.1038/nmeth.3603">https://doi.org/doi.org/10.1038/nmeth.3603</a>.</div>
</div>
<div id="ref-1F05pgVZ9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(78) </div><div class="csl-right-inline">Riffle, M.; Jaschob, D.; Zelter, A.; Davis, T. N. ProXL (Protein Cross-Linking Database): A Platform for Analysis, Visualization, and Sharing of Protein Cross-Linking Mass Spectrometry Data. <em>J Proteome Res</em> <strong>2016</strong>, <em>15</em> (8), 2863–2870. <a href="https://doi.org/10.1021/acs.jproteome.6b00274">https://doi.org/10.1021/acs.jproteome.6b00274</a>.</div>
</div>
<div id="ref-1EF7o7CB4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(79) </div><div class="csl-right-inline">Riffle, M.; Jaschob, D.; Zelter, A.; Davis, T. N. Proxl (Protein Cross-Linking Database): A Public Server, QC Tools, and Other Major Updates. <em>J Proteome Res</em> <strong>2018</strong>, <em>18</em> (2), 759–764. <a href="https://doi.org/10.1021/acs.jproteome.8b00726">https://doi.org/10.1021/acs.jproteome.8b00726</a>.</div>
</div>
<div id="ref-RInMQu7q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(80) </div><div class="csl-right-inline">Dominguez, C.; Boelens, R.; Bonvin, A. M. J. J. HADDOCK:  A Protein−Protein Docking Approach Based on Biochemical or Biophysical Information. <em>J. Am. Chem. Soc.</em> <strong>2003</strong>, <em>125</em> (7), 1731–1737. <a href="https://doi.org/10.1021/ja026939x">https://doi.org/10.1021/ja026939x</a>.</div>
</div>
<div id="ref-1Dqlo9aRm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(81) </div><div class="csl-right-inline">Masson, G. R.; Burke, J. E.; Ahn, N. G.; Anand, G. S.; Borchers, C.; Brier, S.; Bou-Assaf, G. M.; Engen, J. R.; Englander, S. W.; Faber, J.; Garlish, R.; Griffin, P. R.; Gross, M. L.; Guttman, M.; Hamuro, Y.; Heck, A. J. R.; Houde, D.; Iacob, R. E.; Jørgensen, T. J. D.; Kaltashov, I. A.; Klinman, J. P.; Konermann, L.; Man, P.; Mayne, L.; Pascal, B. D.; Reichmann, D.; Skehel, M.; Snijder, J.; Strutzenberg, T. S.; Underbakke, E. S.; Wagner, C.; Wales, T. E.; Walters, B. T.; Weis, D. D.; Wilson, D. J.; Wintrode, P. L.; Zhang, Z.; Zheng, J.; Schriemer, D. C.; Rand, K. D. Recommendations for Performing, Interpreting and Reporting Hydrogen Deuterium Exchange Mass Spectrometry (HDX-MS) Experiments. <em>Nat Methods</em> <strong>2019</strong>, <em>16</em> (7), 595–602. <a href="https://doi.org/10.1038/s41592-019-0459-y">https://doi.org/10.1038/s41592-019-0459-y</a>.</div>
</div>
<div id="ref-cZM0UPqp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(82) </div><div class="csl-right-inline">Konermann, L.; Pan, J.; Liu, Y.-H. Hydrogen Exchange Mass Spectrometry for Studying Protein Structure and Dynamics. <em>Chem. Soc. Rev.</em> <strong>2011</strong>, <em>40</em> (3), 1224–1234. <a href="https://doi.org/10.1039/c0cs00113a">https://doi.org/10.1039/c0cs00113a</a>.</div>
</div>
<div id="ref-SZzVEJCj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(83) </div><div class="csl-right-inline">Hvidt, A.; Nielsen, S. O. Hydrogen Exchange in Proteins. <em>Adv Protein Chem</em> <strong>1966</strong>, <em>21</em>, 287–386. <a href="https://doi.org/10.1016/s0065-3233(08)60129-1">https://doi.org/10.1016/s0065-3233(08)60129-1</a>.</div>
</div>
<div id="ref-H647Pppu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(84) </div><div class="csl-right-inline">Englander, S. W.; Kallenbach, N. R. Hydrogen Exchange and Structural Dynamics of Proteins and Nucleic Acids. <em>Quart. Rev. Biophys.</em> <strong>1983</strong>, <em>16</em> (4), 521–655. <a href="https://doi.org/10.1017/s0033583500005217">https://doi.org/10.1017/s0033583500005217</a>.</div>
</div>
<div id="ref-1CHLgMzps" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(85) </div><div class="csl-right-inline">Chalmers, M. J.; Busby, S. A.; Pascal, B. D.; West, G. M.; Griffin, P. R. Differential Hydrogen/Deuterium Exchange Mass Spectrometry Analysis of Protein–Ligand Interactions. <em>Expert Review of Proteomics</em> <strong>2011</strong>, <em>8</em> (1), 43–59. <a href="https://doi.org/10.1586/epr.10.109">https://doi.org/10.1586/epr.10.109</a>.</div>
</div>
<div id="ref-19wzQMZwm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(86) </div><div class="csl-right-inline">Iacob, R. E.; Engen, J. R. Hydrogen Exchange Mass Spectrometry: Are We Out of the Quicksand? <em>J. Am. Soc. Mass Spectrom.</em> <strong>2012</strong>, <em>23</em> (6), 1003–1010. <a href="https://doi.org//10.1007/s13361-012-0377-z">https://doi.org//10.1007/s13361-012-0377-z</a>.</div>
</div>
<div id="ref-m2HWEBQZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(87) </div><div class="csl-right-inline">Moroco, J. A.; Engen, J. R. Replication in Bioanalytical Studies with HDX MS: Aim as High as Possible. <em>Bioanalysis</em> <strong>2015</strong>, <em>7</em> (9), 1065–1067. <a href="https://doi.org/10.4155/bio.15.46">https://doi.org/10.4155/bio.15.46</a>.</div>
</div>
<div id="ref-whtWKxfa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(88) </div><div class="csl-right-inline">Rand, K. D.; Jørgensen, T. J. D. Development of a Peptide Probe for the Occurrence of Hydrogen (&lt;Sup&gt;1&lt;/Sup&gt;H/&lt;Sup&gt;2&lt;/Sup&gt;H) Scrambling Upon Gas-Phase Fragmentation. <em>Anal. Chem.</em> <strong>2007</strong>, <em>79</em> (22), 8686–8693. <a href="https://doi.org/10.1021/ac0710782">https://doi.org/10.1021/ac0710782</a>.</div>
</div>
<div id="ref-17FfyYJKn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(89) </div><div class="csl-right-inline">Xu, G.; Chance, M. R. Hydroxyl Radical-Mediated Modification of Proteins as Probes for Structural Proteomics. <em>Chem. Rev.</em> <strong>2007</strong>, <em>107</em> (8), 3514–3543. <a href="https://doi.org/10.1021/cr0682047">https://doi.org/10.1021/cr0682047</a>.</div>
</div>
<div id="ref-dyFOYdP5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(90) </div><div class="csl-right-inline">Wang, L.; Chance, M. R. Structural Mass Spectrometry of Proteins Using Hydroxyl Radical Based Protein Footprinting. <em>Anal. Chem.</em> <strong>2011</strong>, <em>83</em> (19), 7234–7241. <a href="https://doi.org/10.1021/ac200567u">https://doi.org/10.1021/ac200567u</a>.</div>
</div>
<div id="ref-8tWrfuT2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(91) </div><div class="csl-right-inline">Tullius, T. D.; Greenbaum, J. A. Mapping Nucleic Acid Structure by Hydroxyl Radical Cleavage. <em>Current Opinion in Chemical Biology</em> <strong>2005</strong>, <em>9</em> (2), 127–134. <a href="https://doi.org/10.1016/j.cbpa.2005.02.009">https://doi.org/10.1016/j.cbpa.2005.02.009</a>.</div>
</div>
<div id="ref-b8XE9cl8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(92) </div><div class="csl-right-inline">Tullius, T. D. Probing DNA Structure with Hydroxyl Radicals. <em>CP Nucleic Acid Chemistry</em> <strong>2001</strong>, <em>7</em> (1). <a href="https://doi.org/10.1002/0471142700.nc0607s07">https://doi.org/10.1002/0471142700.nc0607s07</a>.</div>
</div>
<div id="ref-14ACSyAnR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(93) </div><div class="csl-right-inline">Biswas, P. K.; Chakraborty, S. Targeted DNA Oxidation and Trajectory of Radical DNA Using DFT Based QM/MM Dynamics. <em>Nucleic Acids Research</em> <strong>2019</strong>, <em>47</em> (6), 2757–2765. <a href="https://doi.org/10.1093/nar/gkz089">https://doi.org/10.1093/nar/gkz089</a>.</div>
</div>
<div id="ref-muDfX3XR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(94) </div><div class="csl-right-inline">Niu, B.; Gross, M. L. &lt;Scp&gt;MS&lt;/Scp&gt;
‐Based Hydroxyl Radical Footprinting: Methodology and Application of Fast Photochemical Oxidation of Proteins (
&lt;Scp&gt;FPOP&lt;/Scp&gt;
). <em>Mass Spectrometry‐Based Chemical Proteomics</em>, 2019, 363–416. <a href="https://doi.org/10.1002/9781118970195.ch15">https://doi.org/10.1002/9781118970195.ch15</a>.</div>
</div>
<div id="ref-zy4wero" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(95) </div><div class="csl-right-inline">Feng, J.; Chu, C.; Ma, Z. Fenton and Fenton-Like Catalysts for Electrochemical Immunoassay: A Mini Review. <em>Electrochemistry Communications</em> <strong>2021</strong>, <em>125</em>, 106970. <a href="https://doi.org/10.1016/j.elecom.2021.106970">https://doi.org/10.1016/j.elecom.2021.106970</a>.</div>
</div>
<div id="ref-bAhX0GGr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(96) </div><div class="csl-right-inline">Nieto-Juarez, J. I.; Pierzchła, K.; Sienkiewicz, A.; Kohn, T. Inactivation of MS2 Coliphage in Fenton and Fenton-Like Systems: Role of Transition Metals, Hydrogen Peroxide and Sunlight. <em>Environ. Sci. Technol.</em> <strong>2010</strong>, <em>44</em> (9), 3351–3356. <a href="https://doi.org/10.1021/es903739f">https://doi.org/10.1021/es903739f</a>.</div>
</div>
<div id="ref-GyzG2odq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(97) </div><div class="csl-right-inline">Houée-Levin, C.; Bobrowski, K. The Use of the Methods of Radiolysis to Explore the Mechanisms of Free Radical Modifications in Proteins. <em>Journal of Proteomics</em> <strong>2013</strong>, <em>92</em>, 51–62. <a href="https://doi.org/10.1016/j.jprot.2013.02.014">https://doi.org/10.1016/j.jprot.2013.02.014</a>.</div>
</div>
<div id="ref-tcDiyhQT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(98) </div><div class="csl-right-inline">Hernández-Corroto, E.; Boussetta, N.; Marina, M. L.; García, M. C.; Vorobiev, E. High Voltage Electrical Discharges Followed by Deep Eutectic Solvents Extraction for the Valorization of Pomegranate Seeds (Punica Granatum L.). <em>Innovative Food Science &amp;amp; Emerging Technologies</em> <strong>2022</strong>, <em>79</em>, 103055. <a href="https://doi.org/10.1016/j.ifset.2022.103055">https://doi.org/10.1016/j.ifset.2022.103055</a>.</div>
</div>
<div id="ref-gryR5tjl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(99) </div><div class="csl-right-inline">Jolivet, P.; Aymé, L.; Giuliani, A.; Wien, F.; Chardot, T.; Gohon, Y. Structural Proteomics: Topology and Relative Accessibility of Plant Lipid Droplet Associated Proteins. <em>Journal of Proteomics</em> <strong>2017</strong>, <em>169</em>, 87–98. <a href="https://doi.org/10.1016/j.jprot.2017.09.005">https://doi.org/10.1016/j.jprot.2017.09.005</a>.</div>
</div>
<div id="ref-43HI8GWO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(100) </div><div class="csl-right-inline">Stulić, V.; Vukušić, T.; Butorac, A.; Popović, D.; Herceg, Z. Proteomic Analysis of Saccharomyces Cerevisiae Response to Plasma Treatment. <em>International Journal of Food Microbiology</em> <strong>2019</strong>, <em>292</em>, 171–183. <a href="https://doi.org/10.1016/j.ijfoodmicro.2018.12.017">https://doi.org/10.1016/j.ijfoodmicro.2018.12.017</a>.</div>
</div>
<div id="ref-pFnbzp8Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(101) </div><div class="csl-right-inline">Hambly, D. M.; Gross, M. L. Laser Flash Photolysis of Hydrogen Peroxide to Oxidize Protein Solvent-Accessible Residues on the Microsecond Timescale. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2005</strong>, <em>16</em> (12), 2057–2063. <a href="https://doi.org/10.1016/j.jasms.2005.09.008">https://doi.org/10.1016/j.jasms.2005.09.008</a>.</div>
</div>
<div id="ref-IH1lj0uY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(102) </div><div class="csl-right-inline">Johnson, D. T.; Di Stefano, L. H.; Jones, L. M. Fast Photochemical Oxidation of Proteins (FPOP): A Powerful Mass Spectrometry–Based Structural Proteomics Tool. <em>Journal of Biological Chemistry</em> <strong>2019</strong>, <em>294</em> (32), 11969–11979. <a href="https://doi.org/10.1074/jbc.rev119.006218">https://doi.org/10.1074/jbc.rev119.006218</a>.</div>
</div>
<div id="ref-cMJbHB1a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(103) </div><div class="csl-right-inline">Hambly, D.; Gross, M. Laser Flash Photochemical Oxidation to Locate Heme Binding and Conformational Changes in Myoglobin. <em>International Journal of Mass Spectrometry</em> <strong>2007</strong>, <em>259</em> (1-3), 124–129. <a href="https://doi.org/10.1016/j.ijms.2006.08.018">https://doi.org/10.1016/j.ijms.2006.08.018</a>.</div>
</div>
<div id="ref-LgS9wDAM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(104) </div><div class="csl-right-inline">Chea, E. E.; Jones, L. M. Modifications Generated by Fast Photochemical Oxidation of Proteins Reflect the Native Conformations of Proteins. <em>Protein Science</em> <strong>2018</strong>, <em>27</em> (6), 1047–1056. <a href="https://doi.org/10.1002/pro.3408">https://doi.org/10.1002/pro.3408</a>.</div>
</div>
<div id="ref-sqSJTywu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(105) </div><div class="csl-right-inline">Gau, B. C.; Sharp, J. S.; Rempel, D. L.; Gross, M. L. Fast Photochemical Oxidation of Protein Footprints Faster Than Protein Unfolding. <em>Anal. Chem.</em> <strong>2009</strong>, <em>81</em> (16), 6563–6571. <a href="https://doi.org/10.1021/ac901054w">https://doi.org/10.1021/ac901054w</a>.</div>
</div>
<div id="ref-sZQji0TC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(106) </div><div class="csl-right-inline">Luchini, A.; Espina, V.; Liotta, L. A. Protein Painting Reveals Solvent-Excluded Drug Targets Hidden Within Native Protein–Protein Interfaces. <em>Nat Commun</em> <strong>2014</strong>, <em>5</em> (1). <a href="https://doi.org/10.1038/ncomms5413">https://doi.org/10.1038/ncomms5413</a>.</div>
</div>
<div id="ref-17xQ2aw0Q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(107) </div><div class="csl-right-inline">Haymond, A.; Dey, D.; Carter, R.; Dailing, A.; Nara, V.; Nara, P.; Venkatayogi, S.; Paige, M.; Liotta, L.; Luchini, A. Protein Painting, an Optimized MS-Based Technique, Reveals Functionally Relevant Interfaces of the PD-1/PD-L1 Complex and the YAP2/ZO-1 Complex. <em>Journal of Biological Chemistry</em> <strong>2019</strong>, <em>294</em> (29), 11180–11198. <a href="https://doi.org/10.1074/jbc.ra118.007310">https://doi.org/10.1074/jbc.ra118.007310</a>.</div>
</div>
<div id="ref-gdTpkDDu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(108) </div><div class="csl-right-inline">Pepelnjak, M.; de Souza, N.; Picotti, P. Detecting Protein–Small Molecule Interactions Using Limited Proteolysis–Mass Spectrometry (LiP-MS). <em>Trends in Biochemical Sciences</em> <strong>2020</strong>, <em>45</em> (10), 919–920. <a href="https://doi.org/10.1016/j.tibs.2020.05.006">https://doi.org/10.1016/j.tibs.2020.05.006</a>.</div>
</div>
<div id="ref-xTQMmTt5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(109) </div><div class="csl-right-inline">Schopper, S.; Kahraman, A.; Leuenberger, P.; Feng, Y.; Piazza, I.; Müller, O.; Boersema, P. J.; Picotti, P. Measuring Protein Structural Changes on a Proteome-Wide Scale Using Limited Proteolysis-Coupled Mass Spectrometry. <em>Nat Protoc</em> <strong>2017</strong>, <em>12</em> (11), 2391–2410. <a href="https://doi.org/10.1038/nprot.2017.100">https://doi.org/10.1038/nprot.2017.100</a>.</div>
</div>
<div id="ref-6IZfi1O" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(110) </div><div class="csl-right-inline">Malinovska, L.; Cappelletti, V.; Kohler, D.; Piazza, I.; Tsai, T.-H.; Pepelnjak, M.; Stalder, P.; Dörig, C.; Sesterhenn, F.; Elsässer, F.; Kralickova, L.; Beaton, N.; Reiter, L.; de Souza, N.; Vitek, O.; Picotti, P. Proteome-Wide Structural Changes Measured with Limited Proteolysis-Mass Spectrometry: An Advanced Protocol for High-Throughput Applications. <em>Nat Protoc</em> <strong>2022</strong>, <em>18</em> (3), 659–682. <a href="https://doi.org/10.1038/s41596-022-00771-x">https://doi.org/10.1038/s41596-022-00771-x</a>.</div>
</div>
<div id="ref-UETOKcpN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(111) </div><div class="csl-right-inline">Barret, Diane C.  A.; Schuster, D.; Rodrigues, Matthew J.; Leitner, A.; Picotti, P.; Schertler, Gebhard F.  X.; Kaupp, U.  Benjamin.; Korkhov, Volodymyr M.; Marino, J. Structural Basis of Calmodulin Modulation of the Rod Cyclic Nucleotide-Gated Channel. <em>Proc. Natl. Acad. Sci. U.S.A.</em> <strong>2023</strong>, <em>120</em> (15). <a href="https://doi.org/10.1073/pnas.2300309120">https://doi.org/10.1073/pnas.2300309120</a>.</div>
</div>
<div id="ref-1BxnTxtGI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(112) </div><div class="csl-right-inline">. <a href="https://www.eubopen.org/sites/www.eubopen.org/files/attachments/2022/LiP-MS%20protocol.pdf">https://www.eubopen.org/sites/www.eubopen.org/files/attachments/2022/LiP-MS%20protocol.pdf</a>.</div>
</div>
<div id="ref-1HcZz8Apd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(113) </div><div class="csl-right-inline">Martinez Molina, D.; Jafari, R.; Ignatushchenko, M.; Seki, T.; Larsson, E. A.; Dan, C.; Sreekumar, L.; Cao, Y.; Nordlund, P. Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay. <em>Science</em> <strong>2013</strong>, <em>341</em> (6141), 84–87. <a href="https://doi.org/10.1126/science.1233606">https://doi.org/10.1126/science.1233606</a>.</div>
</div>
<div id="ref-rMCVhIUo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(114) </div><div class="csl-right-inline">Prabhu, N.; Dai, L.; Nordlund, P. CETSA in Integrated Proteomics Studies of Cellular Processes. <em>Current Opinion in Chemical Biology</em> <strong>2020</strong>, <em>54</em>, 54–62. <a href="https://doi.org/10.1016/j.cbpa.2019.11.004">https://doi.org/10.1016/j.cbpa.2019.11.004</a>.</div>
</div>
<div id="ref-JOskUci" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(115) </div><div class="csl-right-inline">Mateus, A.; Kurzawa, N.; Becher, I.; Sridharan, S.; Helm, D.; Stein, F.; Typas, A.; Savitski, M. M. Thermal Proteome Profiling for Interrogating Protein Interactions. <em>Molecular Systems Biology</em> <strong>2020</strong>, <em>16</em> (3). <a href="https://doi.org/10.15252/msb.20199232">https://doi.org/10.15252/msb.20199232</a>.</div>
</div>
<div id="ref-8UcGwfym" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(116) </div><div class="csl-right-inline">Le Sueur, C.; Hammarén, H. M.; Sridharan, S.; Savitski, M. M. Thermal Proteome Profiling: Insights into Protein Modifications, Associations, and Functions. <em>Current Opinion in Chemical Biology</em> <strong>2022</strong>, <em>71</em>, 102225. <a href="https://doi.org/10.1016/j.cbpa.2022.102225">https://doi.org/10.1016/j.cbpa.2022.102225</a>.</div>
</div>
<div id="ref-89c9sDdK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(117) </div><div class="csl-right-inline">King, D. T.; Serrano-Negrón, J. E.; Zhu, Y.; Moore, C. L.; Shoulders, M. D.; Foster, L. J.; Vocadlo, D. J. Thermal Proteome Profiling Reveals the O-GlcNAc-Dependent Meltome. <em>J. Am. Chem. Soc.</em> <strong>2022</strong>, <em>144</em> (9), 3833–3842. <a href="https://doi.org/10.1021/jacs.1c10621">https://doi.org/10.1021/jacs.1c10621</a>.</div>
</div>
<div id="ref-Ssg2QwSE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(118) </div><div class="csl-right-inline">Sridharan, S.; Günthner, I.; Becher, I.; Savitski, M.; Bantscheff, M. Target Discovery Using Thermal Proteome Profiling. <em>Mass Spectrometry‐Based Chemical Proteomics</em>, 2019, 267–291. <a href="https://doi.org/10.1002/9781118970195.ch11">https://doi.org/10.1002/9781118970195.ch11</a>.</div>
</div>
<div id="ref-ieBDHPPg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(119) </div><div class="csl-right-inline">Marcilla, M.; Albar, J. P. Quantitative Proteomics: A Strategic Ally to Map Protein Interaction Networks. <em>IUBMB Life</em> <strong>2012</strong>, <em>65</em> (1), 9–16. <a href="https://doi.org/10.1002/iub.1081">https://doi.org/10.1002/iub.1081</a>.</div>
</div>
<div id="ref-DORXXewQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(120) </div><div class="csl-right-inline">Altelaar, A. F. M.; Munoz, J.; Heck, A. J. R. Next-Generation Proteomics: Towards an Integrative View of Proteome Dynamics. <em>Nat Rev Genet</em> <strong>2012</strong>, <em>14</em> (1), 35–48. <a href="https://doi.org/10.1038/nrg3356">https://doi.org/10.1038/nrg3356</a>.</div>
</div>
<div id="ref-1BsZO9WFn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(121) </div><div class="csl-right-inline">Varjosalo, M.; Sacco, R.; Stukalov, A.; van Drogen, A.; Planyavsky, M.; Hauri, S.; Aebersold, R.; Bennett, K. L.; Colinge, J.; Gstaiger, M.; Superti-Furga, G. Interlaboratory Reproducibility of Large-Scale Human Protein-Complex Analysis by Standardized AP-MS. <em>Nat Methods</em> <strong>2013</strong>, <em>10</em> (4), 307–314. <a href="https://doi.org/10.1038/nmeth.2400">https://doi.org/10.1038/nmeth.2400</a>.</div>
</div>
<div id="ref-yeZfXPV6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(122) </div><div class="csl-right-inline">Huttlin, E. L.; Ting, L.; Bruckner, R. J.; Gebreab, F.; Gygi, M. P.; Szpyt, J.; Tam, S.; Zarraga, G.; Colby, G.; Baltier, K.; Dong, R.; Guarani, V.; Vaites, L. P.; Ordureau, A.; Rad, R.; Erickson, B. K.; Wühr, M.; Chick, J.; Zhai, B.; Kolippakkam, D.; Mintseris, J.; Obar, R. A.; Harris, T.; Artavanis-Tsakonas, S.; Sowa, M. E.; De Camilli, P.; Paulo, J. A.; Harper, J. W.; Gygi, S. P. The BioPlex Network: A Systematic Exploration of the Human Interactome. <em>Cell</em> <strong>2015</strong>, <em>162</em> (2), 425–440. <a href="https://doi.org/10.1016/j.cell.2015.06.043">https://doi.org/10.1016/j.cell.2015.06.043</a>.</div>
</div>
<div id="ref-m46yvHfn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(123) </div><div class="csl-right-inline">Huttlin, E. L.; Bruckner, R. J.; Paulo, J. A.; Cannon, J. R.; Ting, L.; Baltier, K.; Colby, G.; Gebreab, F.; Gygi, M. P.; Parzen, H.; Szpyt, J.; Tam, S.; Zarraga, G.; Pontano-Vaites, L.; Swarup, S.; White, A. E.; Schweppe, D. K.; Rad, R.; Erickson, B. K.; Obar, R. A.; Guruharsha, K. G.; Li, K.; Artavanis-Tsakonas, S.; Gygi, S. P.; Harper, J. W. Architecture of the Human Interactome Defines Protein Communities and Disease Networks. <em>Nature</em> <strong>2017</strong>, <em>545</em> (7655), 505–509. <a href="https://doi.org/10.1038/nature22366">https://doi.org/10.1038/nature22366</a>.</div>
</div>
<div id="ref-VMc1jEkd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(124) </div><div class="csl-right-inline">Huttlin, E. L.; Bruckner, R. J.; Navarrete-Perea, J.; Cannon, J. R.; Baltier, K.; Gebreab, F.; Gygi, M. P.; Thornock, A.; Zarraga, G.; Tam, S.; Szpyt, J.; Gassaway, B. M.; Panov, A.; Parzen, H.; Fu, S.; Golbazi, A.; Maenpaa, E.; Stricker, K.; Guha Thakurta, S.; Zhang, T.; Rad, R.; Pan, J.; Nusinow, D. P.; Paulo, J. A.; Schweppe, D. K.; Vaites, L. P.; Harper, J. W.; Gygi, S. P. Dual Proteome-Scale Networks Reveal Cell-Specific Remodeling of the Human Interactome. <em>Cell</em> <strong>2021</strong>, <em>184</em> (11), 3022–3040.e28. <a href="https://doi.org/10.1016/j.cell.2021.04.011">https://doi.org/10.1016/j.cell.2021.04.011</a>.</div>
</div>
<div id="ref-LkwAXvyl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(125) </div><div class="csl-right-inline">Martell, J. D.; Deerinck, T. J.; Sancak, Y.; Poulos, T. L.; Mootha, V. K.; Sosinsky, G. E.; Ellisman, M. H.; Ting, A. Y. Engineered Ascorbate Peroxidase as a Genetically Encoded Reporter for Electron Microscopy. <em>Nat Biotechnol</em> <strong>2012</strong>, <em>30</em> (11), 1143–1148. <a href="https://doi.org/10.1038/nbt.2375">https://doi.org/10.1038/nbt.2375</a>.</div>
</div>
<div id="ref-1BbkEbfDz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(126) </div><div class="csl-right-inline">Rhee, H.-W.; Zou, P.; Udeshi, N. D.; Martell, J. D.; Mootha, V. K.; Carr, S. A.; Ting, A. Y. Proteomic Mapping of Mitochondria in Living Cells via Spatially Restricted Enzymatic Tagging. <em>Science</em> <strong>2013</strong>, <em>339</em> (6125), 1328–1331. <a href="https://doi.org/10.1126/science.1230593">https://doi.org/10.1126/science.1230593</a>.</div>
</div>
<div id="ref-D5NLgmhd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(127) </div><div class="csl-right-inline">Hung, V.; Zou, P.; Rhee, H.-W.; Udeshi, Namrata D.; Cracan, V.; Svinkina, T.; Carr, Steven A.; Mootha, Vamsi K.; Ting, Alice Y. Proteomic Mapping of the Human Mitochondrial Intermembrane Space in Live Cells via Ratiometric APEX Tagging. <em>Molecular Cell</em> <strong>2014</strong>, <em>55</em> (2), 332–341. <a href="https://doi.org/10.1016/j.molcel.2014.06.003">https://doi.org/10.1016/j.molcel.2014.06.003</a>.</div>
</div>
<div id="ref-zSgLeSta" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(128) </div><div class="csl-right-inline">Roux, K. J.; Kim, D. I.; Raida, M.; Burke, B. A Promiscuous Biotin Ligase Fusion Protein Identifies Proximal and Interacting Proteins in Mammalian Cells. <em>Journal of Cell Biology</em> <strong>2012</strong>, <em>196</em> (6), 801–810. <a href="https://doi.org/10.1083/jcb.201112098">https://doi.org/10.1083/jcb.201112098</a>.</div>
</div>
<div id="ref-NFFhvScy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(129) </div><div class="csl-right-inline">Samavarchi-Tehrani, P.; Samson, R.; Gingras, A.-C. Proximity Dependent Biotinylation: Key Enzymes and Adaptation to Proteomics Approaches. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2020</strong>, <em>19</em> (5), 757–773. <a href="https://doi.org/10.1074/mcp.r120.001941">https://doi.org/10.1074/mcp.r120.001941</a>.</div>
</div>
<div id="ref-1FVEjFnHE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(130) </div><div class="csl-right-inline">Varnaitė, R.; MacNeill, S. A. Meet the Neighbors: Mapping Local Protein Interactomes by Proximity‐dependent Labeling with BioID. <em>Proteomics</em> <strong>2016</strong>, <em>16</em> (19), 2503–2518. <a href="https://doi.org/10.1002/pmic.201600123">https://doi.org/10.1002/pmic.201600123</a>.</div>
</div>
<div id="ref-jJJVaczi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(131) </div><div class="csl-right-inline">Kim, D. I.; Jensen, S. C.; Noble, K. A.; KC, B.; Roux, K. H.; Motamedchaboki, K.; Roux, K. J. An Improved Smaller Biotin Ligase for BioID Proximity Labeling. <em>MBoC</em> <strong>2016</strong>, <em>27</em> (8), 1188–1196. <a href="https://doi.org/10.1091/mbc.e15-12-0844">https://doi.org/10.1091/mbc.e15-12-0844</a>.</div>
</div>
<div id="ref-BlJpACnb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(132) </div><div class="csl-right-inline">Cho, K. F.; Branon, T. C.; Udeshi, N. D.; Myers, S. A.; Carr, S. A.; Ting, A. Y. Proximity Labeling in Mammalian Cells with TurboID and Split-TurboID. <em>Nat Protoc</em> <strong>2020</strong>, <em>15</em> (12), 3971–3999. <a href="https://doi.org/10.1038/s41596-020-0399-0">https://doi.org/10.1038/s41596-020-0399-0</a>.</div>
</div>
<div id="ref-sas7Wv3H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(133) </div><div class="csl-right-inline">Li, P.; Meng, Y.; Wang, L.; Di, L. BioID: A Proximity-Dependent Labeling Approach in Proteomics Study. In <em>Functional Proteomics</em>; Springer New York, 2018; pp 143–151. <a href="https://doi.org/10.1007/978-1-4939-8814-3_10">https://doi.org/10.1007/978-1-4939-8814-3_10</a>.</div>
</div>
<div id="ref-emz5eXnO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(134) </div><div class="csl-right-inline">Burgess, R. R.; Deutscher, M. P. <em>Guide to protein purification</em>, 2nd ed.; Methods in enzymology; Elsevier/Academic Press: Amsterdam Boston, 2009.</div>
</div>
<div id="ref-LvSVXQeu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(135) </div><div class="csl-right-inline"><em>The protein protocols handbook</em>, 3. ed.; Walker, J. M., Ed.; Springer protocols handbooks; Humana Press: New York, NY, 2009.</div>
</div>
<div id="ref-YFivV9n5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(136) </div><div class="csl-right-inline">Linn, S. Strategies and Considerations for Protein Purifications. <em>Methods Enzymol</em> <strong>2009</strong>, <em>463</em>, 9–19. <a href="https://doi.org/10.1016/s0076-6879(09)63002-0">https://doi.org/10.1016/s0076-6879(09)63002-0</a>.</div>
</div>
<div id="ref-PUTrxDaK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(137) </div><div class="csl-right-inline"><em>Proteins and Proteomics: A Laboratory Manual</em>; Simpson, R. J., Ed.; Cold Spring Harbor Laboratory Press: Cold Spring Harbor, NY, 2003.</div>
</div>
<div id="ref-Cj5ZGyu8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(138) </div><div class="csl-right-inline">Feist, P.; Hummon, A. B. Proteomic Challenges: Sample Preparation Techniques for Microgram-Quantity Protein Analysis from Biological Samples. <em>Int J Mol Sci</em> <strong>2015</strong>, <em>16</em> (2), 3537–3563. <a href="https://doi.org/10.3390/ijms16023537">https://doi.org/10.3390/ijms16023537</a>.</div>
</div>
<div id="ref-OZvOVLht" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(139) </div><div class="csl-right-inline">Salvi, G.; De Los Rios, P.; Vendruscolo, M. Effective Interactions Between Chaotropic Agents and Proteins. <em>Proteins</em> <strong>2005</strong>, <em>61</em> (3), 492–499. <a href="https://doi.org/10.1002/prot.20626">https://doi.org/10.1002/prot.20626</a>.</div>
</div>
<div id="ref-bpTXhTR5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(140) </div><div class="csl-right-inline">Proc, J. L.; Kuzyk, M. A.; Hardie, D. B.; Yang, J.; Smith, D. S.; Jackson, A. M.; Parker, C. E.; Borchers, C. H. A Quantitative Study of the Effects of Chaotropic Agents, Surfactants, and Solvents on the Digestion Efficiency of Human Plasma Proteins by Trypsin. <em>J Proteome Res</em> <strong>2010</strong>, <em>9</em> (10), 5422–5437. <a href="https://doi.org/10.1021/pr100656u">https://doi.org/10.1021/pr100656u</a>.</div>
</div>
<div id="ref-B8hLhFif" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(141) </div><div class="csl-right-inline">Sun, S.; Zhou, J.-Y.; Yang, W.; Zhang, H. Inhibition of Protein Carbamylation in Urea Solution Using Ammonium-Containing Buffers. <em>Anal Biochem</em> <strong>2013</strong>, <em>446</em>, 76–81. <a href="https://doi.org/10.1016/j.ab.2013.10.024">https://doi.org/10.1016/j.ab.2013.10.024</a>.</div>
</div>
<div id="ref-Igj2yRI0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(142) </div><div class="csl-right-inline">Donnelly, D. P.; Rawlins, C. M.; DeHart, C. J.; Fornelli, L.; Schachner, L. F.; Lin, Z.; Lippens, J. L.; Aluri, K. C.; Sarin, R.; Chen, B.; Lantz, C.; Jung, W.; Johnson, K. R.; Koller, A.; Wolff, J. J.; Campuzano, I. D. G.; Auclair, J. R.; Ivanov, A. R.; Whitelegge, J. P.; Paša-Tolić, L.; Chamot-Rooke, J.; Danis, P. O.; Smith, L. M.; Tsybin, Y. O.; Loo, J. A.; Ge, Y.; Kelleher, N. L.; Agar, J. N. Best Practices and Benchmarks for Intact Protein Analysis for Top-down Mass Spectrometry. <em>Nat Methods</em> <strong>2019</strong>, <em>16</em> (7), 587–594. <a href="https://doi.org/10.1038/s41592-019-0457-0">https://doi.org/10.1038/s41592-019-0457-0</a>.</div>
</div>
<div id="ref-ifXiaFI9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(143) </div><div class="csl-right-inline">Zhang, X. Less Is More: Membrane Protein Digestion Beyond Urea–Trypsin Solution for Next-Level Proteomics. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2015</strong>, <em>14</em> (9), 2441–2453. <a href="https://doi.org/10.1074/mcp.r114.042572">https://doi.org/10.1074/mcp.r114.042572</a>.</div>
</div>
<div id="ref-xvenBhjQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(144) </div><div class="csl-right-inline">Jiang, L.; He, L.; Fountoulakis, M. Comparison of Protein Precipitation Methods for Sample Preparation Prior to Proteomic Analysis. <em>J Chromatogr A</em> <strong>2004</strong>, <em>1023</em> (2), 317–320. <a href="https://doi.org/10.1016/j.chroma.2003.10.029">https://doi.org/10.1016/j.chroma.2003.10.029</a>.</div>
</div>
<div id="ref-OMzPHtgd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(145) </div><div class="csl-right-inline">Burgess, R. R. Protein Precipitation Techniques. <em>Methods Enzymol</em> <strong>2009</strong>, <em>463</em>, 331–342. <a href="https://doi.org/10.1016/s0076-6879(09)63020-2">https://doi.org/10.1016/s0076-6879(09)63020-2</a>.</div>
</div>
<div id="ref-Llj72YHx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(146) </div><div class="csl-right-inline">Wojtkiewicz, M.; Berg Luecke, L.; Kelly, M. I.; Gundry, R. L. Facile Preparation of Peptides for Mass Spectrometry Analysis in Bottom-Up Proteomics Workflows. <em>Curr Protoc</em> <strong>2021</strong>, <em>1</em> (3), e85. <a href="https://doi.org/10.1002/cpz1.85">https://doi.org/10.1002/cpz1.85</a>.</div>
</div>
<div id="ref-rWTbDNu2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(147) </div><div class="csl-right-inline">Wiśniewski, J. R.; Zougman, A.; Nagaraj, N.; Mann, M. Universal Sample Preparation Method for Proteome Analysis. <em>Nat Methods</em> <strong>2009</strong>, <em>6</em> (5), 359–362. <a href="https://doi.org/10.1038/nmeth.1322">https://doi.org/10.1038/nmeth.1322</a>.</div>
</div>
<div id="ref-bkON7ZR2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(148) </div><div class="csl-right-inline">Wiśniewski, J. R. Filter-Aided Sample Preparation for Proteome Analysis. <em>Methods Mol Biol</em> <strong>2018</strong>, <em>1841</em>, 3–10. <a href="https://doi.org/10.1007/978-1-4939-8695-8_1">https://doi.org/10.1007/978-1-4939-8695-8_1</a>.</div>
</div>
<div id="ref-29YpOcx3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(149) </div><div class="csl-right-inline">Kodama, H.; Tokuman, N.; Yasui, I.; Gyobu, Y.; Kashiwagi, Y. [Group and Type Distribution of Hemolytic Streptococci Isolated from Clinical Specimens--Prevalence of Group A Type 3 Isolates in 1985 in Toyama Prefecture]. <em>Kansenshogaku Zasshi</em> <strong>1987</strong>, <em>61</em> (4), 482–488. <a href="https://doi.org/10.11150/kansenshogakuzasshi1970.61.482">https://doi.org/10.11150/kansenshogakuzasshi1970.61.482</a>.</div>
</div>
<div id="ref-R9HFU5Rw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(150) </div><div class="csl-right-inline">Sielaff, M.; Kuharev, J.; Bohn, T.; Hahlbrock, J.; Bopp, T.; Tenzer, S.; Distler, U. Evaluation of FASP, SP3, and iST Protocols for Proteomic Sample Preparation in the Low Microgram Range. <em>J Proteome Res</em> <strong>2017</strong>, <em>16</em> (11), 4060–4072. <a href="https://doi.org/10.1021/acs.jproteome.7b00433">https://doi.org/10.1021/acs.jproteome.7b00433</a>.</div>
</div>
<div id="ref-2ZApD7In" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(151) </div><div class="csl-right-inline">Batth, T. S.; Tollenaere, M. X.; Rüther, P.; Gonzalez-Franquesa, A.; Prabhakar, B. S.; Bekker-Jensen, S.; Deshmukh, A. S.; Olsen, J. V. Protein Aggregation Capture on Microparticles Enables Multipurpose Proteomics Sample Preparation. <em>Mol Cell Proteomics</em> <strong>2019</strong>, <em>18</em> (5), 1027–1035. <a href="https://doi.org/10.1074/mcp.tir118.001270">https://doi.org/10.1074/mcp.tir118.001270</a>.</div>
</div>
<div id="ref-IRVtdB7A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(152) </div><div class="csl-right-inline">Ludwig, K. R.; Schroll, M. M.; Hummon, A. B. Comparison of In-Solution, FASP, and S-Trap Based Digestion Methods for Bottom-Up Proteomic Studies. <em>J Proteome Res</em> <strong>2018</strong>, <em>17</em> (7), 2480–2490. <a href="https://doi.org/10.1021/acs.jproteome.8b00235">https://doi.org/10.1021/acs.jproteome.8b00235</a>.</div>
</div>
<div id="ref-GcubNF0j" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(153) </div><div class="csl-right-inline">Masuda, T.; Tomita, M.; Ishihama, Y. Phase Transfer Surfactant-Aided Trypsin Digestion for Membrane Proteome Analysis. <em>J Proteome Res</em> <strong>2008</strong>, <em>7</em> (2), 731–740. <a href="https://doi.org/10.1021/pr700658q">https://doi.org/10.1021/pr700658q</a>.</div>
</div>
<div id="ref-120jHNa4n" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(154) </div><div class="csl-right-inline">Yeung, Y.-G.; Nieves, E.; Angeletti, R. H.; Stanley, E. R. Removal of Detergents from Protein Digests for Mass Spectrometry Analysis. <em>Analytical Biochemistry</em> <strong>2008</strong>, <em>382</em> (2), 135–137. <a href="https://doi.org/10.1016/j.ab.2008.07.034">https://doi.org/10.1016/j.ab.2008.07.034</a>.</div>
</div>
<div id="ref-G9HWYdpG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(155) </div><div class="csl-right-inline">Sun, Z.; Asmann, Y. W.; Kalari, K. R.; Bot, B.; Eckel-Passow, J. E.; Baker, T. R.; Carr, J. M.; Khrebtukova, I.; Luo, S.; Zhang, L.; Schroth, G. P.; Perez, E. A.; Thompson, E. A. Integrated Analysis of Gene Expression, CpG Island Methylation, and Gene Copy Number in Breast Cancer Cells by Deep Sequencing. <em>PLoS One</em> <strong>2011</strong>, <em>6</em> (2), e17490. <a href="https://doi.org/10.1371/journal.pone.0017490">https://doi.org/10.1371/journal.pone.0017490</a>.</div>
</div>
<div id="ref-15gFWnWy1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(156) </div><div class="csl-right-inline">Rigaut, G.; Shevchenko, A.; Rutz, B.; Wilm, M.; Mann, M.; Séraphin, B. A Generic Protein Purification Method for Protein Complex Characterization and Proteome Exploration. <em>Nat Biotechnol</em> <strong>1999</strong>, <em>17</em> (10), 1030–1032. <a href="https://doi.org/10.1038/13732">https://doi.org/10.1038/13732</a>.</div>
</div>
<div id="ref-12BtFuhxe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(157) </div><div class="csl-right-inline">Chang, Y.-H.; Gregorich, Z. R.; Chen, A. J.; Hwang, L.; Guner, H.; Yu, D.; Zhang, J.; Ge, Y. New Mass-Spectrometry-Compatible Degradable Surfactant for Tissue Proteomics. <em>J Proteome Res</em> <strong>2015</strong>, <em>14</em> (3), 1587–1599. <a href="https://doi.org/10.1021/pr5012679">https://doi.org/10.1021/pr5012679</a>.</div>
</div>
<div id="ref-1FdkzkZPe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(158) </div><div class="csl-right-inline">Knott, S. J.; Brown, K. A.; Josyer, H.; Carr, A.; Inman, D.; Jin, S.; Friedl, A.; Ponik, S. M.; Ge, Y. Photocleavable Surfactant-Enabled Extracellular Matrix Proteomics. <em>Anal Chem</em> <strong>2020</strong>, <em>92</em> (24), 15693–15698. <a href="https://doi.org/10.1021/acs.analchem.0c03104">https://doi.org/10.1021/acs.analchem.0c03104</a>.</div>
</div>
<div id="ref-KbqdsNwY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(159) </div><div class="csl-right-inline">Mbeunkui, F.; Goshe, M. B. Investigation of Solubilization and Digestion Methods for Microsomal Membrane Proteome Analysis Using Data-Independent LC-MSE. <em>Proteomics</em> <strong>2011</strong>, <em>11</em> (5), 898–911. <a href="https://doi.org/10.1002/pmic.200900698">https://doi.org/10.1002/pmic.200900698</a>.</div>
</div>
<div id="ref-1FX8gYsTy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(160) </div><div class="csl-right-inline">Lin, Y.; Huo, L.; Liu, Z.; Li, J.; Liu, Y.; He, Q.; Wang, X.; Liang, S. Sodium Laurate, a Novel Protease- and Mass Spectrometry-Compatible Detergent for Mass Spectrometry-Based Membrane Proteomics. <em>PLoS One</em> <strong>2013</strong>, <em>8</em> (3), e59779. <a href="https://doi.org/10.1371/journal.pone.0059779">https://doi.org/10.1371/journal.pone.0059779</a>.</div>
</div>
<div id="ref-7nXVfPN7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(161) </div><div class="csl-right-inline">Zhou, J.; Zhou, T.; Cao, R.; Liu, Z.; Shen, J.; Chen, P.; Wang, X.; Liang, S. Evaluation of the Application of Sodium Deoxycholate to Proteomic Analysis of Rat Hippocampal Plasma Membrane. <em>J Proteome Res</em> <strong>2006</strong>, <em>5</em> (10), 2547–2553. <a href="https://doi.org/10.1021/pr060112a">https://doi.org/10.1021/pr060112a</a>.</div>
</div>
<div id="ref-EZCh7C0x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(162) </div><div class="csl-right-inline">Ahmadi, S.; Winter, D. Identification of Poly(ethylene Glycol) and Poly(ethylene Glycol)-Based Detergents Using Peptide Search Engines. <em>Anal Chem</em> <strong>2018</strong>, <em>90</em> (11), 6594–6600. <a href="https://doi.org/10.1021/acs.analchem.8b00365">https://doi.org/10.1021/acs.analchem.8b00365</a>.</div>
</div>
<div id="ref-EK9zCZWn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(163) </div><div class="csl-right-inline">León, I. R.; Schwämmle, V.; Jensen, O. N.; Sprenger, R. R. Quantitative Assessment of in-Solution Digestion Efficiency Identifies Optimal Protocols for Unbiased Protein Analysis. <em>Mol Cell Proteomics</em> <strong>2013</strong>, <em>12</em> (10), 2992–3005. <a href="https://doi.org/10.1074/mcp.m112.025585">https://doi.org/10.1074/mcp.m112.025585</a>.</div>
</div>
<div id="ref-1A6wXg5ZL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(164) </div><div class="csl-right-inline">Tobin, A. K.; Bowsher, C. G. Subcellular Fractionation of Plant Tissues: Isolation of Plastids and Mitochondria. In <em>Protein Purification Protocols</em>; Humana Press; pp 53–64. <a href="https://doi.org/10.1385/1-59259-655-x:53">https://doi.org/10.1385/1-59259-655-x:53</a>.</div>
</div>
<div id="ref-WjXlUEEW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(165) </div><div class="csl-right-inline">Lande, N. V.; Barua, P.; Gayen, D.; Kumar, S.; Chakraborty, S.; Chakraborty, N. Proteomic Dissection of the Chloroplast: Moving Beyond Photosynthesis. <em>Journal of Proteomics</em> <strong>2020</strong>, <em>212</em>, 103542. <a href="https://doi.org/10.1016/j.jprot.2019.103542">https://doi.org/10.1016/j.jprot.2019.103542</a>.</div>
</div>
<div id="ref-TdvEcc7C" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(166) </div><div class="csl-right-inline">Millar, A. H.; Liddell, A.; Leaver, C. J. Isolation and Subfractionation of Mitochondria from Plants. <em>Methods Cell Biol</em> <strong>2007</strong>, <em>80</em>, 65–90. <a href="https://doi.org/10.1016/s0091-679x(06)80003-8">https://doi.org/10.1016/s0091-679x(06)80003-8</a>.</div>
</div>
<div id="ref-12PeVjdCj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(167) </div><div class="csl-right-inline">Wilken, L. R.; Nikolov, Z. L. Recovery and Purification of Plant-Made Recombinant Proteins. <em>Biotechnology Advances</em> <strong>2012</strong>, <em>30</em> (2), 419–433. <a href="https://doi.org/10.1016/j.biotechadv.2011.07.020">https://doi.org/10.1016/j.biotechadv.2011.07.020</a>.</div>
</div>
<div id="ref-iTMONsOP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(168) </div><div class="csl-right-inline">Plaxton, W. C. Avoiding Proteolysis During the Extraction and Purification of Active Plant Enzymes. <em>Plant and Cell Physiology</em> <strong>2019</strong>, <em>60</em> (4), 715–724. <a href="https://doi.org/10.1093/pcp/pcz028">https://doi.org/10.1093/pcp/pcz028</a>.</div>
</div>
<div id="ref-iuZ9vx4h" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(169) </div><div class="csl-right-inline">Pierpoint, W. S. The Extraction of Enzymes From Plant Tissues Rich in Phenolic Compounds. In <em>Protein Purification Protocols</em>; Humana Press; pp 65–74. <a href="https://doi.org/10.1385/1-59259-655-x:65">https://doi.org/10.1385/1-59259-655-x:65</a>.</div>
</div>
<div id="ref-J2KykVTl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(170) </div><div class="csl-right-inline">Isaacson, T.; Damasceno, C. M. B.; Saravanan, R. S.; He, Y.; Catalá, C.; Saladié, M.; Rose, J. K. C. Sample Extraction Techniques for Enhanced Proteomic Analysis of Plant Tissues. <em>Nat Protoc</em> <strong>2006</strong>, <em>1</em> (2), 769–774. <a href="https://doi.org/10.1038/nprot.2006.102">https://doi.org/10.1038/nprot.2006.102</a>.</div>
</div>
<div id="ref-1CzJQ1fro" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(171) </div><div class="csl-right-inline">Wu, X.; Gong, F.; Wang, W. Protein Extraction from Plant Tissues for 2DE and Its Application in Proteomic Analysis. <em>Proteomics</em> <strong>2014</strong>, <em>14</em> (6), 645–658. <a href="https://doi.org/10.1002/pmic.201300239">https://doi.org/10.1002/pmic.201300239</a>.</div>
</div>
<div id="ref-14eFecfij" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(172) </div><div class="csl-right-inline">Niu, L.; Yuan, H.; Gong, F.; Wu, X.; Wang, W. Protein Extraction Methods Shape Much of the Extracted Proteomes. <em>Front. Plant Sci.</em> <strong>2018</strong>, <em>9</em>. <a href="https://doi.org/10.3389/fpls.2018.00802">https://doi.org/10.3389/fpls.2018.00802</a>.</div>
</div>
<div id="ref-lDCDjYu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(173) </div><div class="csl-right-inline">Mergner, J.; Kuster, B. Plant Proteome Dynamics. <em>Annu. Rev. Plant Biol.</em> <strong>2022</strong>, <em>73</em> (1), 67–92. <a href="https://doi.org/10.1146/annurev-arplant-102620-031308">https://doi.org/10.1146/annurev-arplant-102620-031308</a>.</div>
</div>
<div id="ref-1EeOoOwwB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(174) </div><div class="csl-right-inline">Song, G.; Hsu, P. Y.; Walley, J. W. Assessment and Refinement of Sample Preparation Methods for Deep and Quantitative Plant Proteome Profiling. <em>Proteomics</em> <strong>2018</strong>, <em>18</em> (17). <a href="https://doi.org/10.1002/pmic.201800220">https://doi.org/10.1002/pmic.201800220</a>.</div>
</div>
<div id="ref-1Be3nuQby" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(175) </div><div class="csl-right-inline">Hughes, C. S.; Moggridge, S.; Müller, T.; Sorensen, P. H.; Morin, G. B.; Krijgsveld, J. Single-Pot, Solid-Phase-Enhanced Sample Preparation for Proteomics Experiments. <em>Nat Protoc</em> <strong>2018</strong>, <em>14</em> (1), 68–85. <a href="https://doi.org/10.1038/s41596-018-0082-x">https://doi.org/10.1038/s41596-018-0082-x</a>.</div>
</div>
<div id="ref-tqd1QDxn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(176) </div><div class="csl-right-inline">Mikulášek, K.; Konečná, H.; Potěšil, D.; Holánková, R.; Havliš, J.; Zdráhal, Z. SP3 Protocol for Proteomic Plant Sample Preparation Prior LC-MS/MS. <em>Front. Plant Sci.</em> <strong>2021</strong>, <em>12</em>. <a href="https://doi.org/10.3389/fpls.2021.635550">https://doi.org/10.3389/fpls.2021.635550</a>.</div>
</div>
<div id="ref-3rOHISrh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(177) </div><div class="csl-right-inline">Gupta, R.; Kim, S. T. Depletion of RuBisCO Protein Using the Protamine Sulfate Precipitation Method. In <em>Methods in Molecular Biology</em>; Springer New York, 2015; pp 225–233. <a href="https://doi.org/10.1007/978-1-4939-2550-6_17">https://doi.org/10.1007/978-1-4939-2550-6_17</a>.</div>
</div>
<div id="ref-5VlfbJMW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(178) </div><div class="csl-right-inline">Smolikova, G.; Gorbach, D.; Lukasheva, E.; Mavropolo-Stolyarenko, G.; Bilova, T.; Soboleva, A.; Tsarev, A.; Romanovskaya, E.; Podolskaya, E.; Zhukov, V.; Tikhonovich, I.; Medvedev, S.; Hoehenwarter, W.; Frolov, A. Bringing New Methods to the Seed Proteomics Platform: Challenges and Perspectives. <em>IJMS</em> <strong>2020</strong>, <em>21</em> (23), 9162. <a href="https://doi.org/10.3390/ijms21239162">https://doi.org/10.3390/ijms21239162</a>.</div>
</div>
<div id="ref-17GoHwva3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(179) </div><div class="csl-right-inline">Gomes, T. A.; Zanette, C. M.; Spier, M. R. An Overview of Cell Disruption Methods for Intracellular Biomolecules Recovery. <em>Preparative Biochemistry &amp;amp; Biotechnology</em> <strong>2020</strong>, <em>50</em> (7), 635–654. <a href="https://doi.org/10.1080/10826068.2020.1728696">https://doi.org/10.1080/10826068.2020.1728696</a>.</div>
</div>
<div id="ref-195QiKJpv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(180) </div><div class="csl-right-inline">Salomon, I.; Janssen, H.; Neefjes, J. Mechanical Forces Used for Cell Fractionation Can Create Hybrid Membrane Vesicles. <em>Int J Biol Sci</em> <strong>2010</strong>, <em>6</em> (7), 649–654. <a href="https://doi.org/10.7150/ijbs.6.649">https://doi.org/10.7150/ijbs.6.649</a>.</div>
</div>
<div id="ref-4EGQHHdM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(181) </div><div class="csl-right-inline">Molnar, A.; Lakat, T.; Hosszu, A.; Szebeni, B.; Balogh, A.; Orfi, L.; Szabo, A. J.; Fekete, A.; Hodrea, J. Lyophilization and Homogenization of Biological Samples Improves Reproducibility and Reduces Standard Deviation in Molecular Biology Techniques. <em>Amino Acids</em> <strong>2021</strong>, <em>53</em> (6), 917–928. <a href="https://doi.org/10.1007/s00726-021-02994-w">https://doi.org/10.1007/s00726-021-02994-w</a>.</div>
</div>
<div id="ref-157qUOoZe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(182) </div><div class="csl-right-inline">Cai, X.; Xue, Z.; Wu, C.; Sun, R.; Qian, L.; Yue, L.; Ge, W.; Yi, X.; Liu, W.; Chen, C.; Gao, H.; Yu, J.; Xu, L.; Zhu, Y.; Guo, T. High-Throughput Proteomic Sample Preparation Using Pressure Cycling Technology. <em>Nat Protoc</em> <strong>2022</strong>, <em>17</em> (10), 2307–2325. <a href="https://doi.org/10.1038/s41596-022-00727-1">https://doi.org/10.1038/s41596-022-00727-1</a>.</div>
</div>
<div id="ref-wXcRYG3x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(183) </div><div class="csl-right-inline">Olson, B. J. S. C.; Markwell, J. Assays for Determination of Protein Concentration. <em>Curr Protoc Protein Sci</em> <strong>2007</strong>, <em>Chapter 3</em>, Unit 3.4. <a href="https://doi.org/10.1002/0471140864.ps0304s48">https://doi.org/10.1002/0471140864.ps0304s48</a>.</div>
</div>
<div id="ref-DRVtai1B" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(184) </div><div class="csl-right-inline">Jones, L. J.; Haugland, R. P.; Singer, V. L. Development and Characterization of the NanoOrange Protein Quantitation Assay: A Fluorescence-Based Assay of Proteins in Solution. <em>Biotechniques</em> <strong>2003</strong>, <em>34</em> (4), 850–854, 856, 858 passim. <a href="https://doi.org/10.2144/03344pt03">https://doi.org/10.2144/03344pt03</a>.</div>
</div>
<div id="ref-qFKCAO2X" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(185) </div><div class="csl-right-inline">Brady, P. N.; Macnaughtan, M. A. Evaluation of Colorimetric Assays for Analyzing Reductively Methylated Proteins: Biases and Mechanistic Insights. <em>Anal Biochem</em> <strong>2015</strong>, <em>491</em>, 43–51. <a href="https://doi.org/10.1016/j.ab.2015.08.027">https://doi.org/10.1016/j.ab.2015.08.027</a>.</div>
</div>
<div id="ref-jQLWiiId" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(186) </div><div class="csl-right-inline">Contreras-Martos, S.; Nguyen, H. H.; Nguyen, P. N.; Hristozova, N.; Macossay-Castillo, M.; Kovacs, D.; Bekesi, A.; Oemig, J. S.; Maes, D.; Pauwels, K.; Tompa, P.; Lebrun, P. Quantification of Intrinsically Disordered Proteins: A Problem Not Fully Appreciated. <em>Front Mol Biosci</em> <strong>2018</strong>, <em>5</em>, 83. <a href="https://doi.org/10.3389/fmolb.2018.00083">https://doi.org/10.3389/fmolb.2018.00083</a>.</div>
</div>
<div id="ref-1F0WRIEv4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(187) </div><div class="csl-right-inline">Sapan, C. V.; Lundblad, R. L.; Price, N. C. <a href="https://www.ncbi.nlm.nih.gov/pubmed/10075906">Colorimetric Protein Assay Techniques.</a> <em>Biotechnol Appl Biochem</em> <strong>1999</strong>, <em>29</em> (2), 99–108.</div>
</div>
<div id="ref-MgeRCWZg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(188) </div><div class="csl-right-inline">Smith, P. K.; Krohn, R. I.; Hermanson, G. T.; Mallia, A. K.; Gartner, F. H.; Provenzano, M. D.; Fujimoto, E. K.; Goeke, N. M.; Olson, B. J.; Klenk, D. C. Measurement of Protein Using Bicinchoninic Acid. <em>Anal Biochem</em> <strong>1985</strong>, <em>150</em> (1), 76–85. <a href="https://doi.org/10.1016/0003-2697(85)90442-7">https://doi.org/10.1016/0003-2697(85)90442-7</a>.</div>
</div>
<div id="ref-7aeNLPmC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(189) </div><div class="csl-right-inline">Walker, J. M. The Bicinchoninic Acid (BCA) Assay for Protein Quantitation. <em>Methods Mol Biol</em> <strong>1994</strong>, <em>32</em>, 5–8. <a href="https://doi.org/10.1385/0-89603-268-x:5">https://doi.org/10.1385/0-89603-268-x:5</a>.</div>
</div>
<div id="ref-10DIgRvDQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(190) </div><div class="csl-right-inline">Bradford, M. M. A Rapid and Sensitive Method for the Quantitation of Microgram Quantities of Protein Utilizing the Principle of Protein-Dye Binding. <em>Anal Biochem</em> <strong>1976</strong>, <em>72</em>, 248–254. <a href="https://doi.org/10.1006/abio.1976.9999">https://doi.org/10.1006/abio.1976.9999</a>.</div>
</div>
<div id="ref-L2cR80nH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(191) </div><div class="csl-right-inline">Kielkopf, C. L.; Bauer, W.; Urbatsch, I. L. Bradford Assay for Determining Protein Concentration. <em>Cold Spring Harb Protoc</em> <strong>2020</strong>, <em>2020</em> (4), 102269. <a href="https://doi.org/10.1101/pdb.prot102269">https://doi.org/10.1101/pdb.prot102269</a>.</div>
</div>
<div id="ref-2lZxrpnu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(192) </div><div class="csl-right-inline">Jones, A.; Razniewska, T.; Lesser, B. H.; Siqueira, R.; Berk, D.; Behie, L. A.; Gaucher, G. M. An Assay for the Measurement of the Protein Content of Cells Immobilized in Carrageenan. <em>Can J Microbiol</em> <strong>1984</strong>, <em>30</em> (4), 475–481. <a href="https://doi.org/10.1139/m84-069">https://doi.org/10.1139/m84-069</a>.</div>
</div>
<div id="ref-q1g1rJrT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(193) </div><div class="csl-right-inline">Duncombe, T. A.; Ponti, A.; Seebeck, F. P.; Dittrich, P. S. UV-Vis Spectra-Activated Droplet Sorting for Label-Free Chemical Identification and Collection of Droplets. <em>Anal Chem</em> <strong>2021</strong>, <em>93</em> (38), 13008–13013. <a href="https://doi.org/10.1021/acs.analchem.1c02822">https://doi.org/10.1021/acs.analchem.1c02822</a>.</div>
</div>
<div id="ref-NLQoCcYj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(194) </div><div class="csl-right-inline">Wiśniewski, J. R.; Gaugaz, F. Z. Fast and Sensitive Total Protein and Peptide Assays for Proteomic Analysis. <em>Anal Chem</em> <strong>2015</strong>, <em>87</em> (8), 4110–4116. <a href="https://doi.org/10.1021/ac504689z">https://doi.org/10.1021/ac504689z</a>.</div>
</div>
<div id="ref-iRPPTjzF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(195) </div><div class="csl-right-inline">You, W. W.; Haugland, R. P.; Ryan, D. K.; Haugland, R. P. 3-(4-Carboxybenzoyl)quinoline-2-Carboxaldehyde, a Reagent with Broad Dynamic Range for the Assay of Proteins and Lipoproteins in Solution. <em>Anal Biochem</em> <strong>1997</strong>, <em>244</em> (2), 277–282. <a href="https://doi.org/10.1006/abio.1996.9920">https://doi.org/10.1006/abio.1996.9920</a>.</div>
</div>
<div id="ref-OglKN2v5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(196) </div><div class="csl-right-inline">Suttapitugsakul, S.; Xiao, H.; Smeekens, J.; Wu, R. Evaluation and Optimization of Reduction and Alkylation Methods to Maximize Peptide Identification with MS-Based Proteomics. <em>Mol Biosyst</em> <strong>2017</strong>, <em>13</em> (12), 2574–2582. <a href="https://doi.org/10.1039/c7mb00393e">https://doi.org/10.1039/c7mb00393e</a>.</div>
</div>
<div id="ref-2cCnZ6ft" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(197) </div><div class="csl-right-inline">Hale, J. E.; Butler, J. P.; Gelfanova, V.; You, J.-S.; Knierman, M. D. A Simplified Procedure for the Reduction and Alkylation of Cysteine Residues in Proteins Prior to Proteolytic Digestion and Mass Spectral Analysis. <em>Anal Biochem</em> <strong>2004</strong>, <em>333</em> (1), 174–181. <a href="https://doi.org/10.1016/j.ab.2004.04.013">https://doi.org/10.1016/j.ab.2004.04.013</a>.</div>
</div>
<div id="ref-9vICnnYM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(198) </div><div class="csl-right-inline">Müller, T.; Winter, D. Systematic Evaluation of Protein Reduction and Alkylation Reveals Massive Unspecific Side Effects by Iodine-Containing Reagents. <em>Mol Cell Proteomics</em> <strong>2017</strong>, <em>16</em> (7), 1173–1187. <a href="https://doi.org/10.1074/mcp.m116.064048">https://doi.org/10.1074/mcp.m116.064048</a>.</div>
</div>
<div id="ref-GkFAexwC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(199) </div><div class="csl-right-inline">Gao, X.-H.; Li, L.; Parisien, M.; Wu, J.; Bederman, I.; Gao, Z.; Krokowski, D.; Chirieleison, S. M.; Abbott, D.; Wang, B.; Arvan, P.; Cameron, M.; Chance, M.; Willard, B.; Hatzoglou, M. Discovery of a Redox Thiol Switch: Implications for Cellular Energy Metabolism. <em>Mol Cell Proteomics</em> <strong>2020</strong>, <em>19</em> (5), 852–870. <a href="https://doi.org/10.1074/mcp.ra119.001910">https://doi.org/10.1074/mcp.ra119.001910</a>.</div>
</div>
<div id="ref-Z6nk0TFI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(200) </div><div class="csl-right-inline">Irvine, G. W.; Stillman, M. J. Residue Modification and Mass Spectrometry for the Investigation of Structural and Metalation Properties of Metallothionein and Cysteine-Rich Proteins. <em>Int J Mol Sci</em> <strong>2017</strong>, <em>18</em> (5). <a href="https://doi.org/10.3390/ijms18050913">https://doi.org/10.3390/ijms18050913</a>.</div>
</div>
<div id="ref-s4OLOlUG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(201) </div><div class="csl-right-inline">Murray, C. I.; Van Eyk, J. E. Chasing Cysteine Oxidative Modifications: Proteomic Tools for Characterizing Cysteine Redox Status. <em>Circ Cardiovasc Genet</em> <strong>2012</strong>, <em>5</em> (5), 591. <a href="https://doi.org/10.1161/circgenetics.111.961425">https://doi.org/10.1161/circgenetics.111.961425</a>.</div>
</div>
<div id="ref-vpBg0CP7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(202) </div><div class="csl-right-inline">Vandermarliere, E.; Mueller, M.; Martens, L. Getting Intimate with Trypsin, the Leading Protease in Proteomics. <em>Mass Spec Rev</em> <strong>2013</strong>, <em>32</em> (6), 453–465. <a href="https://doi.org/10.1002/mas.21376">https://doi.org/10.1002/mas.21376</a>.</div>
</div>
<div id="ref-16FpKYZCe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(203) </div><div class="csl-right-inline">Swaney, D. L.; Wenger, C. D.; Coon, J. J. Value of Using Multiple Proteases for Large-Scale Mass Spectrometry-Based Proteomics. <em>J Proteome Res</em> <strong>2010</strong>, <em>9</em> (3), 1323–1329. <a href="https://doi.org/10.1021/pr900863u">https://doi.org/10.1021/pr900863u</a>.</div>
</div>
<div id="ref-lSFMVWD1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(204) </div><div class="csl-right-inline">Tsiatsiani, L.; Heck, A. J. R. Proteomics Beyond Trypsin. <em>FEBS J</em> <strong>2015</strong>, <em>282</em> (14), 2612–2626. <a href="https://doi.org/10.1111/febs.13287">https://doi.org/10.1111/febs.13287</a>.</div>
</div>
<div id="ref-10vHwy5gN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(205) </div><div class="csl-right-inline">Meyer, J. G. <em>ISRN Comput Biol</em> <strong>2014</strong>, <em>2014</em>. <a href="https://doi.org/10.1155/2014/960902">https://doi.org/10.1155/2014/960902</a>.</div>
</div>
<div id="ref-y9KaXde4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(206) </div><div class="csl-right-inline">Meyer, J. G. &lt;I&gt;In Silico&lt;/i&gt;Proteome Cleavage Reveals Iterative Digestion Strategy for High Sequence Coverage. <em>ISRN Computational Biology</em> <strong>2014</strong>, <em>2014</em>, 1–7. <a href="https://doi.org/10.1155/2014/960902">https://doi.org/10.1155/2014/960902</a>.</div>
</div>
<div id="ref-7aKTZT1i" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(207) </div><div class="csl-right-inline">Choudhary, G.; Wu, S.-L.; Shieh, P.; Hancock, W. S. Multiple Enzymatic Digestion for Enhanced Sequence Coverage of Proteins in Complex Proteomic Mixtures Using Capillary LC with Ion Trap MS/MS. <em>J Proteome Res</em> <strong>2003</strong>, <em>2</em> (1), 59–67. <a href="https://doi.org/10.1021/pr025557n">https://doi.org/10.1021/pr025557n</a>.</div>
</div>
<div id="ref-gXhdeFOF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(208) </div><div class="csl-right-inline">Giansanti, P.; Tsiatsiani, L.; Low, T. Y.; Heck, A. J. R. Six Alternative Proteases for Mass Spectrometry-Based Proteomics Beyond Trypsin. <em>Nat Protoc</em> <strong>2016</strong>, <em>11</em> (5), 993–1006. <a href="https://doi.org/10.1038/nprot.2016.057">https://doi.org/10.1038/nprot.2016.057</a>.</div>
</div>
<div id="ref-14bc70ELO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(209) </div><div class="csl-right-inline">Blank-Landeshammer, B.; Teichert, I.; Märker, R.; Nowrousian, M.; Kück, U.; Sickmann, A. Combination of Proteogenomics with Peptide. <em>mBio</em> <strong>2019</strong>, <em>10</em> (5). <a href="https://doi.org/10.1128/mbio.02367-19">https://doi.org/10.1128/mbio.02367-19</a>.</div>
</div>
<div id="ref-zmo81rNN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(210) </div><div class="csl-right-inline">Yang, H.; Li, Y.-C.; Zhao, M.-Z.; Wu, F.-L.; Wang, X.; Xiao, W.-D.; Wang, Y.-H.; Zhang, J.-L.; Wang, F.-Q.; Xu, F.; Zeng, W.-F.; Overall, C. M.; He, S.-M.; Chi, H.; Xu, P. Precision. <em>Mol Cell Proteomics</em> <strong>2019</strong>, <em>18</em> (4), 773–785. <a href="https://doi.org/10.1074/mcp.tir118.000918">https://doi.org/10.1074/mcp.tir118.000918</a>.</div>
</div>
<div id="ref-IZw9sVjg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(211) </div><div class="csl-right-inline">Vanuopadath, M.; Sajeev, N.; Murali, A. R.; Sudish, N.; Kangosseri, N.; Sebastian, I. R.; Jain, N. D.; Pal, A.; Raveendran, D.; Nair, B. G.; Nair, S. S. Mass Spectrometry-Assisted Venom Profiling of Hypnale Hypnale Found in the Western Ghats of India Incorporating de Novo Sequencing Approaches. <em>Int J Biol Macromol</em> <strong>2018</strong>, <em>118</em> (Pt B), 1736–1746. <a href="https://doi.org/10.1016/j.ijbiomac.2018.07.016">https://doi.org/10.1016/j.ijbiomac.2018.07.016</a>.</div>
</div>
<div id="ref-19KCAADV7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(212) </div><div class="csl-right-inline">Vanuopadath, M.; Raveendran, D.; Nair, B. G.; Nair, S. S. Venomics and Antivenomics of Indian Spectacled Cobra (Naja Naja) from the Western Ghats. <em>Acta Tropica</em> <strong>2022</strong>, <em>228</em>, 106324. <a href="https://doi.org/10.1016/j.actatropica.2022.106324">https://doi.org/10.1016/j.actatropica.2022.106324</a>.</div>
</div>
<div id="ref-7BuX2TBR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(213) </div><div class="csl-right-inline">Guthals, A.; Clauser, K. R.; Frank, A. M.; Bandeira, N. Sequencing-Grade &lt;i&gt;De Novo&lt;/i&gt; Analysis of MS/MS Triplets (CID/HCD/ETD) From Overlapping Peptides. <em>J. Proteome Res.</em> <strong>2013</strong>, <em>12</em> (6), 2846–2857. <a href="https://doi.org/10.1021/pr400173d">https://doi.org/10.1021/pr400173d</a>.</div>
</div>
<div id="ref-1GLf6FYas" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(214) </div><div class="csl-right-inline">Wiśniewski, J. R.; Wegler, C.; Artursson, P. Multiple-Enzyme-Digestion Strategy Improves Accuracy and Sensitivity of Label- and Standard-Free Absolute Quantification to a Level That Is Achievable by Analysis with Stable Isotope-Labeled Standard Spiking. <em>J Proteome Res</em> <strong>2018</strong>, <em>18</em> (1), 217–224. <a href="https://doi.org/10.1021/acs.jproteome.8b00549">https://doi.org/10.1021/acs.jproteome.8b00549</a>.</div>
</div>
<div id="ref-WubQ7yFP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(215) </div><div class="csl-right-inline">Miller, R. M.; Millikin, R. J.; Hoffmann, C. V.; Solntsev, S. K.; Sheynkman, G. M.; Shortreed, M. R.; Smith, L. M. Improved Protein Inference from Multiple Protease Bottom-Up Mass Spectrometry Data. <em>J. Proteome Res.</em> <strong>2019</strong>, <em>18</em> (9), 3429–3438. <a href="https://doi.org/10.1021/acs.jproteome.9b00330">https://doi.org/10.1021/acs.jproteome.9b00330</a>.</div>
</div>
<div id="ref-Ac0S3PrF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(216) </div><div class="csl-right-inline">Kaulich, P. T.; Cassidy, L.; Bartel, J.; Schmitz, R. A.; Tholey, A. Multi-Protease Approach for the Improved Identification and Molecular Characterization of Small Proteins and Short Open Reading Frame-Encoded Peptides. <em>J. Proteome Res.</em> <strong>2021</strong>, <em>20</em> (5), 2895–2903. <a href="https://doi.org/10.1021/acs.jproteome.1c00115">https://doi.org/10.1021/acs.jproteome.1c00115</a>.</div>
</div>
<div id="ref-P861T4mW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(217) </div><div class="csl-right-inline">Vincent, D.; Ezernieks, V.; Rochfort, S.; Spangenberg, G. A Multiple Protease Strategy to Optimise the Shotgun Proteomics of Mature Medicinal Cannabis Buds. <em>IJMS</em> <strong>2019</strong>, <em>20</em> (22), 5630. <a href="https://doi.org/10.3390/ijms20225630">https://doi.org/10.3390/ijms20225630</a>.</div>
</div>
<div id="ref-14eCb7Jfc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(218) </div><div class="csl-right-inline">Guo, X.; Trudgian, D. C.; Lemoff, A.; Yadavalli, S.; Mirzaei, H. Confetti: A Multiprotease Map of the HeLa Proteome for Comprehensive Proteomics. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2014</strong>, <em>13</em> (6), 1573–1584. <a href="https://doi.org/10.1074/mcp.m113.035170">https://doi.org/10.1074/mcp.m113.035170</a>.</div>
</div>
<div id="ref-18s9UXUGf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(219) </div><div class="csl-right-inline">Giansanti, P.; Aye, T.; van den Toorn, H.; Peng, M.; van Breukelen, B.; Heck, Albert J. R. An Augmented Multiple-Protease-Based Human Phosphopeptide Atlas. <em>Cell Reports</em> <strong>2015</strong>, <em>11</em> (11), 1834–1843. <a href="https://doi.org/10.1016/j.celrep.2015.05.029">https://doi.org/10.1016/j.celrep.2015.05.029</a>.</div>
</div>
<div id="ref-ZgqlEERS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(220) </div><div class="csl-right-inline">Jekel, P. A.; Weijer, W. J.; Beintema, J. J. Use of Endoproteinase Lys-C from Lysobacter Enzymogenes in Protein Sequence Analysis. <em>Anal Biochem</em> <strong>1983</strong>, <em>134</em> (2), 347–354. <a href="https://doi.org/10.1016/0003-2697(83)90308-1">https://doi.org/10.1016/0003-2697(83)90308-1</a>.</div>
</div>
<div id="ref-nobcST81" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(221) </div><div class="csl-right-inline">Glatter, T.; Ludwig, C.; Ahrné, E.; Aebersold, R.; Heck, A. J. R.; Schmidt, A. Large-Scale Quantitative Assessment of Different in-Solution Protein Digestion Protocols Reveals Superior Cleavage Efficiency of Tandem Lys-C/Trypsin Proteolysis over Trypsin Digestion. <em>J Proteome Res</em> <strong>2012</strong>, <em>11</em> (11), 5145–5156. <a href="https://doi.org/10.1021/pr300273g">https://doi.org/10.1021/pr300273g</a>.</div>
</div>
<div id="ref-oDNy6K9V" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(222) </div><div class="csl-right-inline">Epstein, D. M.; Wensink, P. C. <a href="https://www.ncbi.nlm.nih.gov/pubmed/3053694">The Alpha-Lytic Protease Gene of Lysobacter Enzymogenes. The Nucleotide Sequence Predicts a Large Prepro-Peptide with Homology to Pro-Peptides of Other Chymotrypsin-Like Enzymes.</a> <em>J Biol Chem</em> <strong>1988</strong>, <em>263</em> (32), 16586–16590.</div>
</div>
<div id="ref-UeP5oN6q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(223) </div><div class="csl-right-inline">Drapeau, G. R.; Boily, Y.; Houmard, J. <a href="https://www.ncbi.nlm.nih.gov/pubmed/4627743">Purification and Properties of an Extracellular Protease of Staphylococcus Aureus.</a> <em>J Biol Chem</em> <strong>1972</strong>, <em>247</em> (20), 6720–6726.</div>
</div>
<div id="ref-FcovvAwu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(224) </div><div class="csl-right-inline">Liu, S.; Moulton, K. R.; Auclair, J. R.; Zhou, Z. S. Mildly Acidic Conditions Eliminate Deamidation Artifact During Proteolysis: Digestion with Endoprotease Glu-C at pH 4.5. <em>Amino Acids</em> <strong>2016</strong>, <em>48</em> (4), 1059–1067. <a href="https://doi.org/10.1007/s00726-015-2166-z">https://doi.org/10.1007/s00726-015-2166-z</a>.</div>
</div>
<div id="ref-Z8khRFem" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(225) </div><div class="csl-right-inline">Ingrosso, D.; Fowler, A. V.; Bleibaum, J.; Clarke, S. Specificity of Endoproteinase Asp-N (Pseudomonas Fragi): Cleavage at Glutamyl Residues in Two Proteins. <em>Biochem Biophys Res Commun</em> <strong>1989</strong>, <em>162</em> (3), 1528–1534. <a href="https://doi.org/10.1016/0006-291x(89)90848-6">https://doi.org/10.1016/0006-291x(89)90848-6</a>.</div>
</div>
<div id="ref-129jHkJ7J" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(226) </div><div class="csl-right-inline">Appel, W. Chymotrypsin: Molecular and Catalytic Properties. <em>Clin Biochem</em> <strong>1986</strong>, <em>19</em> (6), 317–322. <a href="https://doi.org/10.1016/s0009-9120(86)80002-9">https://doi.org/10.1016/s0009-9120(86)80002-9</a>.</div>
</div>
<div id="ref-n8tCUNeh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(227) </div><div class="csl-right-inline">Wilhelm, M.; Schlegl, J.; Hahne, H.; Gholami, A. M.; Lieberenz, M.; Savitski, M. M.; Ziegler, E.; Butzmann, L.; Gessulat, S.; Marx, H.; Mathieson, T.; Lemeer, S.; Schnatbaum, K.; Reimer, U.; Wenschuh, H.; Mollenhauer, M.; Slotta-Huspenina, J.; Boese, J.-H.; Bantscheff, M.; Gerstmair, A.; Faerber, F.; Kuster, B. Mass-Spectrometry-Based Draft of the Human Proteome. <em>Nature</em> <strong>2014</strong>, <em>509</em> (7502), 582–587. <a href="https://doi.org/10.1038/nature13319">https://doi.org/10.1038/nature13319</a>.</div>
</div>
<div id="ref-Ddlp2JQj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(228) </div><div class="csl-right-inline">Guo, X.; Trudgian, D. C.; Lemoff, A.; Yadavalli, S.; Mirzaei, H. Confetti: A Multiprotease Map of the HeLa Proteome for Comprehensive Proteomics. <em>Mol Cell Proteomics</em> <strong>2014</strong>, <em>13</em> (6), 1573–1584. <a href="https://doi.org/10.1074/mcp.m113.035170">https://doi.org/10.1074/mcp.m113.035170</a>.</div>
</div>
<div id="ref-VbXzZ784" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(229) </div><div class="csl-right-inline">Low, T. Y.; van Heesch, S.; van den Toorn, H.; Giansanti, P.; Cristobal, A.; Toonen, P.; Schafer, S.; Hübner, N.; van Breukelen, B.; Mohammed, S.; Cuppen, E.; Heck, A. J. R.; Guryev, V. Quantitative and Qualitative Proteome Characteristics Extracted from in-Depth Integrated Genomics and Proteomics Analysis. <em>Cell Rep</em> <strong>2013</strong>, <em>5</em> (5), 1469–1478. <a href="https://doi.org/10.1016/j.celrep.2013.10.041">https://doi.org/10.1016/j.celrep.2013.10.041</a>.</div>
</div>
<div id="ref-1BDWCCTHa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(230) </div><div class="csl-right-inline">Peng, M.; Taouatas, N.; Cappadona, S.; van Breukelen, B.; Mohammed, S.; Scholten, A.; Heck, A. J. R. Protease Bias in Absolute Protein Quantitation. <em>Nat Methods</em> <strong>2012</strong>, <em>9</em> (6), 524–525. <a href="https://doi.org/10.1038/nmeth.2031">https://doi.org/10.1038/nmeth.2031</a>.</div>
</div>
<div id="ref-qJADzs4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(231) </div><div class="csl-right-inline">Porter, W. H.; Cunningham, L. W.; Mitchell, W. M. <a href="https://www.ncbi.nlm.nih.gov/pubmed/4332560">Studies on the Active Site of Clostripain. The Specific Inactivation by the Chloromethyl Ketone Derived from -N-Tosyl-L-Lysine.</a> <em>J Biol Chem</em> <strong>1971</strong>, <em>246</em> (24), 7675–7682.</div>
</div>
<div id="ref-MqKkSQNK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(232) </div><div class="csl-right-inline">Huesgen, P. F.; Lange, P. F.; Rogers, L. D.; Solis, N.; Eckhard, U.; Kleifeld, O.; Goulas, T.; Gomis-Rüth, F. X.; Overall, C. M. LysargiNase Mirrors Trypsin for Protein C-Terminal and Methylation-Site Identification. <em>Nat Methods</em> <strong>2014</strong>, <em>12</em> (1), 55–58. <a href="https://doi.org/10.1038/nmeth.3177">https://doi.org/10.1038/nmeth.3177</a>.</div>
</div>
<div id="ref-3xdcLBaR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(233) </div><div class="csl-right-inline">Hohmann, L.; Sherwood, C.; Eastham, A.; Peterson, A.; Eng, J. K.; Eddes, J. S.; Shteynberg, D.; Martin, D. B. Proteomic Analyses Using Grifola Frondosa Metalloendoprotease Lys-N. <em>J Proteome Res</em> <strong>2009</strong>, <em>8</em> (3), 1415–1422. <a href="https://doi.org/10.1021/pr800774h">https://doi.org/10.1021/pr800774h</a>.</div>
</div>
<div id="ref-O6S1GvPn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(234) </div><div class="csl-right-inline">Taouatas, N.; Drugan, M. M.; Heck, A. J. R.; Mohammed, S. Straightforward Ladder Sequencing of Peptides Using a Lys-N Metalloendopeptidase. <em>Nat Methods</em> <strong>2008</strong>, <em>5</em> (5), 405–407. <a href="https://doi.org/10.1038/nmeth.1204">https://doi.org/10.1038/nmeth.1204</a>.</div>
</div>
<div id="ref-8DGqkN3D" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(235) </div><div class="csl-right-inline">Raijmakers, R.; Neerincx, P.; Mohammed, S.; Heck, A. J. R. Cleavage Specificities of the Brother and Sister Proteases Lys-C and Lys-N. <em>Chem Commun (Camb)</em> <strong>2010</strong>, <em>46</em> (46), 8827–8829. <a href="https://doi.org/10.1039/c0cc02523b">https://doi.org/10.1039/c0cc02523b</a>.</div>
</div>
<div id="ref-eMC9mqfj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(236) </div><div class="csl-right-inline">Fruton, J. S. A History of Pepsin and Related Enzymes. <em>Q Rev Biol</em> <strong>2002</strong>, <em>77</em> (2), 127–147. <a href="https://doi.org/10.1086/340729">https://doi.org/10.1086/340729</a>.</div>
</div>
<div id="ref-pnU7apE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(237) </div><div class="csl-right-inline">Northrop, J. H. CRYSTALLINE PEPSIN : I. ISOLATION AND TESTS OF PURITY. <em>J Gen Physiol</em> <strong>1930</strong>, <em>13</em> (6), 739–766. <a href="https://doi.org/10.1085/jgp.13.6.739">https://doi.org/10.1085/jgp.13.6.739</a>.</div>
</div>
<div id="ref-9Yf3fEMC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(238) </div><div class="csl-right-inline">Northrop, J. H. CRYSTALLINE PEPSIN : II. GENERAL PROPERTIES AND EXPERIMENTAL METHODS. <em>J Gen Physiol</em> <strong>1930</strong>, <em>13</em> (6), 767–780. <a href="https://doi.org/10.1085/jgp.13.6.767">https://doi.org/10.1085/jgp.13.6.767</a>.</div>
</div>
<div id="ref-10CHY6r2E" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(239) </div><div class="csl-right-inline">Northrop, J. H. CRYSTALLINE PEPSIN. <em>Science</em> <strong>1929</strong>, <em>69</em> (1796), 580. <a href="https://doi.org/10.1126/science.69.1796.580">https://doi.org/10.1126/science.69.1796.580</a>.</div>
</div>
<div id="ref-1EtaxGZW0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(240) </div><div class="csl-right-inline"><em>The Nobel Prize in Chemistry 1946</em>. NobelPrize.org. <a href="https://www.nobelprize.org/prizes/chemistry/1946/speedread/">https://www.nobelprize.org/prizes/chemistry/1946/speedread/</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-itW7kK3s" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(241) </div><div class="csl-right-inline">Gorman, J. J.; Wallis, T. P.; Pitt, J. J. Protein Disulfide Bond Determination by Mass Spectrometry. <em>Mass Spectrom Rev</em> <strong>2002</strong>, <em>21</em> (3), 183–216. <a href="https://doi.org/10.1002/mas.10025">https://doi.org/10.1002/mas.10025</a>.</div>
</div>
<div id="ref-Vske1oHJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(242) </div><div class="csl-right-inline">Liu, F.; van Breukelen, B.; Heck, A. J. R. Facilitating Protein Disulfide Mapping by a Combination of Pepsin Digestion, Electron Transfer Higher Energy Dissociation (EThcD), and a Dedicated Search Algorithm SlinkS. <em>Mol Cell Proteomics</em> <strong>2014</strong>, <em>13</em> (10), 2776–2786. <a href="https://doi.org/10.1074/mcp.o114.039057">https://doi.org/10.1074/mcp.o114.039057</a>.</div>
</div>
<div id="ref-Azi1X1R4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(243) </div><div class="csl-right-inline">Jones, L. M.; Zhang, H.; Vidavsky, I.; Gross, M. L. Online, High-Pressure Digestion System for Protein Characterization by Hydrogen/Deuterium Exchange and Mass Spectrometry. <em>Anal. Chem.</em> <strong>2010</strong>, <em>82</em> (4), 1171–1174. <a href="https://doi.org/10.1021/ac902477u">https://doi.org/10.1021/ac902477u</a>.</div>
</div>
<div id="ref-1nMCmYmb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(244) </div><div class="csl-right-inline">Kostyukevich, Y.; Acter, T.; Zherebker, A.; Ahmed, A.; Kim, S.; Nikolaev, E. Hydrogen/Deuterium Exchange in Mass Spectrometry. <em>Mass Spectrometry Reviews</em> <strong>2018</strong>, <em>37</em> (6), 811–853. <a href="https://doi.org/10.1002/mas.21565">https://doi.org/10.1002/mas.21565</a>.</div>
</div>
<div id="ref-YOXX6An6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(245) </div><div class="csl-right-inline">Ebeling, W.; Hennrich, N.; Klockow, M.; Metz, H.; Orth, H. D.; Lang, H. Proteinase K from Tritirachium Album Limber. <em>Eur J Biochem</em> <strong>1974</strong>, <em>47</em> (1), 91–97. <a href="https://doi.org/10.1111/j.1432-1033.1974.tb03671.x">https://doi.org/10.1111/j.1432-1033.1974.tb03671.x</a>.</div>
</div>
<div id="ref-1CgjQBy2S" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(246) </div><div class="csl-right-inline">Saenger, W. Proteinase K. In <em>Handbook of Proteolytic Enzymes</em>; Elsevier, 2013; pp 3240–3242. https://doi.org/<a href="https://doi.org/10.1016/b978-0-12-382219-2.00714-6">https://doi.org/10.1016/b978-0-12-382219-2.00714-6</a>.</div>
</div>
<div id="ref-1BhNuatKH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(247) </div><div class="csl-right-inline">Schopper, S.; Kahraman, A.; Leuenberger, P.; Feng, Y.; Piazza, I.; Müller, O.; Boersema, P. J.; Picotti, P. Measuring Protein Structural Changes on a Proteome-Wide Scale Using Limited Proteolysis-Coupled Mass Spectrometry. <em>Nat Protoc</em> <strong>2017</strong>, <em>12</em> (11), 2391–2410. <a href="https://doi.org/10.1038/nprot.2017.100">https://doi.org/10.1038/nprot.2017.100</a>.</div>
</div>
<div id="ref-Ixo1jeWd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(248) </div><div class="csl-right-inline">Udenfriend, S.; Stein, S.; Böhlen, P.; Dairman, W.; Leimgruber, W.; Weigele, M. Fluorescamine: A Reagent for Assay of Amino Acids, Peptides, Proteins, and Primary Amines in the Picomole Range. <em>Science</em> <strong>1972</strong>, <em>178</em> (4063), 871–872. <a href="https://doi.org/10.1126/science.178.4063.871">https://doi.org/10.1126/science.178.4063.871</a>.</div>
</div>
<div id="ref-dBvmYbi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(249) </div><div class="csl-right-inline">Bantan-Polak, T.; Kassai, M.; Grant, K. B. A Comparison of Fluorescamine and Naphthalene-2,3-Dicarboxaldehyde Fluorogenic Reagents for Microplate-Based Detection of Amino Acids. <em>Anal Biochem</em> <strong>2001</strong>, <em>297</em> (2), 128–136. <a href="https://doi.org/10.1006/abio.2001.5338">https://doi.org/10.1006/abio.2001.5338</a>.</div>
</div>
<div id="ref-7YHVbaDH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(250) </div><div class="csl-right-inline">Wiśniewski, J. R.; Gaugaz, F. Z. Fast and Sensitive Total Protein and Peptide Assays for Proteomic Analysis. <em>Anal. Chem.</em> <strong>2015</strong>, <em>87</em> (8), 4110–4116. <a href="https://doi.org/10.1021/ac504689z">https://doi.org/10.1021/ac504689z</a>.</div>
</div>
<div id="ref-YTQzoOxb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(251) </div><div class="csl-right-inline">Ong, S.-E.; Blagoev, B.; Kratchmarova, I.; Kristensen, D. B.; Steen, H.; Pandey, A.; Mann, M. Stable Isotope Labeling by Amino Acids in Cell Culture, SILAC, as a Simple and Accurate Approach to Expression Proteomics. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2002</strong>, <em>1</em> (5), 376–386. <a href="https://doi.org/10.1074/mcp.m200025-mcp200">https://doi.org/10.1074/mcp.m200025-mcp200</a>.</div>
</div>
<div id="ref-1FOQ1H9mS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(252) </div><div class="csl-right-inline">Mertins, P.; Udeshi, N. D.; Clauser, K. R.; Mani, D.; Patel, J.; Ong, S.; Jaffe, J. D.; Carr, S. A. iTRAQ Labeling Is Superior to mTRAQ for Quantitative Global Proteomics and Phosphoproteomics. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2012</strong>, <em>11</em> (6), M111.014423. <a href="https://doi.org/10.1074/mcp.m111.014423">https://doi.org/10.1074/mcp.m111.014423</a>.</div>
</div>
<div id="ref-Qqbj9JVK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(253) </div><div class="csl-right-inline">Boersema, P. J.; Raijmakers, R.; Lemeer, S.; Mohammed, S.; Heck, A. J. R. Multiplex Peptide Stable Isotope Dimethyl Labeling for Quantitative Proteomics. <em>Nat Protoc</em> <strong>2009</strong>, <em>4</em> (4), 484–494. <a href="https://doi.org/10.1038/nprot.2009.21">https://doi.org/10.1038/nprot.2009.21</a>.</div>
</div>
<div id="ref-E7KmdZas" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(254) </div><div class="csl-right-inline">Zanivan, S.; Meves, A.; Behrendt, K.; Schoof, Erwin M.; Neilson, Lisa J.; Cox, J.; Tang, Hao R.; Kalna, G.; van Ree, Janine H.; van Deursen, Jan M.; Trempus, Carol S.; Machesky, Laura M.; Linding, R.; Wickström, Sara A.; Fässler, R.; Mann, M. In Vivo SILAC-Based Proteomics Reveals Phosphoproteome Changes During Mouse Skin Carcinogenesis. <em>Cell Reports</em> <strong>2013</strong>, <em>3</em> (2), 552–566. <a href="https://doi.org/10.1016/j.celrep.2013.01.003">https://doi.org/10.1016/j.celrep.2013.01.003</a>.</div>
</div>
<div id="ref-TGkcmyuH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(255) </div><div class="csl-right-inline">Monetti, M.; Nagaraj, N.; Sharma, K.; Mann, M. Large-Scale Phosphosite Quantification in Tissues by a Spike-in SILAC Method. <em>Nat Methods</em> <strong>2011</strong>, <em>8</em> (8), 655–658. <a href="https://doi.org/10.1038/nmeth.1647">https://doi.org/10.1038/nmeth.1647</a>.</div>
</div>
<div id="ref-QipLkzBy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(256) </div><div class="csl-right-inline">Derks, J.; Leduc, A.; Wallmann, G.; Huffman, R. G.; Willetts, M.; Khan, S.; Specht, H.; Ralser, M.; Demichev, V.; Slavov, N. Increasing the Throughput of Sensitive Proteomics by plexDIA. <em>Nat Biotechnol</em> <strong>2022</strong>, <em>41</em> (1), 50–59. <a href="https://doi.org/10.1038/s41587-022-01389-w">https://doi.org/10.1038/s41587-022-01389-w</a>.</div>
</div>
<div id="ref-ZhSU2auj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(257) </div><div class="csl-right-inline">Thielert, M.; Itang, E. C.; Ammar, C.; Rosenberger, F. A.; Bludau, I.; Schweizer, L.; Nordmann, T. M.; Skowronek, P.; Wahle, M.; Zeng, W.; Zhou, X.; Brunner, A.; Richter, S.; Levesque, M. P.; Theis, F. J.; Steger, M.; Mann, M. Robust Dimethyl‐based Multiplex‐DIA Doubles Single‐cell Proteome Depth via a Reference Channel. <em>Molecular Systems Biology</em> <strong>2023</strong>, <em>19</em> (9). <a href="https://doi.org/10.15252/msb.202211503">https://doi.org/10.15252/msb.202211503</a>.</div>
</div>
<div id="ref-WPmaUWBS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(258) </div><div class="csl-right-inline">Meyer, J. G.; D’Souza, A. K.; Sorensen, D. J.; Rardin, M. J.; Wolfe, A. J.; Gibson, B. W.; Schilling, B. Quantification of Lysine Acetylation and Succinylation Stoichiometry in Proteins Using Mass Spectrometric Data-Independent Acquisitions (SWATH). <em>J. Am. Soc. Mass Spectrom.</em> <strong>2016</strong>, <em>27</em> (11), 1758–1771. <a href="https://doi.org/10.1007/s13361-016-1476-z">https://doi.org/10.1007/s13361-016-1476-z</a>.</div>
</div>
<div id="ref-j9N2kPSr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(259) </div><div class="csl-right-inline">Thompson, A.; Schäfer, J.; Kuhn, K.; Kienle, S.; Schwarz, J.; Schmidt, G.; Neumann, T.; Hamon, C. Tandem Mass Tags:  A Novel Quantification Strategy for Comparative Analysis of Complex Protein Mixtures by MS/MS. <em>Anal. Chem.</em> <strong>2003</strong>, <em>75</em> (8), 1895–1904. <a href="https://doi.org/10.1021/ac0262560">https://doi.org/10.1021/ac0262560</a>.</div>
</div>
<div id="ref-V8IykxaB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(260) </div><div class="csl-right-inline">Ross, P. L.; Huang, Y. N.; Marchese, J. N.; Williamson, B.; Parker, K.; Hattan, S.; Khainovski, N.; Pillai, S.; Dey, S.; Daniels, S.; Purkayastha, S.; Juhasz, P.; Martin, S.; Bartlet-Jones, M.; He, F.; Jacobson, A.; Pappin, D. J. Multiplexed Protein Quantitation in Saccharomyces Cerevisiae Using Amine-Reactive Isobaric Tagging Reagents. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2004</strong>, <em>3</em> (12), 1154–1169. <a href="https://doi.org/10.1074/mcp.m400129-mcp200">https://doi.org/10.1074/mcp.m400129-mcp200</a>.</div>
</div>
<div id="ref-lbOXqNSw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(261) </div><div class="csl-right-inline">Xiang, F.; Ye, H.; Chen, R.; Fu, Q.; Li, L. &lt;I&gt;N&lt;/i&gt;,&lt;i&gt;N&lt;/i&gt;-Dimethyl Leucines as Novel Isobaric Tandem Mass Tags for Quantitative Proteomics and Peptidomics. <em>Anal. Chem.</em> <strong>2010</strong>, <em>82</em> (7), 2817–2825. <a href="https://doi.org/10.1021/ac902778d">https://doi.org/10.1021/ac902778d</a>.</div>
</div>
<div id="ref-15rS5UMb9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(262) </div><div class="csl-right-inline">Rauniyar, N.; Yates, J. R. Isobaric Labeling-Based Relative Quantification in Shotgun Proteomics. <em>J Proteome Res</em> <strong>2014</strong>, <em>13</em> (12), 5293–5309. <a href="https://doi.org/10.1021/pr500880b">https://doi.org/10.1021/pr500880b</a>.</div>
</div>
<div id="ref-16D3xg72c" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(263) </div><div class="csl-right-inline">Sinitcyn, P.; Rudolph, J. D.; Cox, J. Computational Methods for Understanding Mass Spectrometry–Based Shotgun Proteomics Data. <em>Annu. Rev. Biomed. Data Sci.</em> <strong>2018</strong>, <em>1</em> (1), 207–234. <a href="https://doi.org/10.1146/annurev-biodatasci-080917-013516">https://doi.org/10.1146/annurev-biodatasci-080917-013516</a>.</div>
</div>
<div id="ref-kIpNiFeU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(264) </div><div class="csl-right-inline">Cheng, L.; Pisitkun, T.; Knepper, M. A.; Hoffert, J. D. Peptide Labeling Using Isobaric Tagging Reagents for Quantitative Phosphoproteomics. <em>Methods Mol Biol</em> <strong>2016</strong>, <em>1355</em>, 53–70. <a href="https://doi.org/10.1007/978-1-4939-3049-4_4">https://doi.org/10.1007/978-1-4939-3049-4_4</a>.</div>
</div>
<div id="ref-ESY1qR9T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(265) </div><div class="csl-right-inline">Wu, W. W.; Wang, G.; Insel, P. A.; Hsiao, C.-T.; Zou, S.; Martin, B.; Maudsley, S.; Shen, R.-F. Discovery- and Target-Based Protein Quantification Using iTRAQ and Pulsed Q Collision Induced Dissociation (PQD). <em>J Proteomics</em> <strong>2012</strong>, <em>75</em> (8), 2480–2487. <a href="https://doi.org/10.1016/j.jprot.2012.02.027">https://doi.org/10.1016/j.jprot.2012.02.027</a>.</div>
</div>
<div id="ref-FMSpJgZ4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(266) </div><div class="csl-right-inline">Orsburn, B. C.; Yuan, Y.; Bumpus, N. N. Insights into Protein Post-Translational Modification Landscapes of Individual Human Cells by Trapped Ion Mobility Time-of-Flight Mass Spectrometry. <em>Nat Commun</em> <strong>2022</strong>, <em>13</em> (1). <a href="https://doi.org/10.1038/s41467-022-34919-w">https://doi.org/10.1038/s41467-022-34919-w</a>.</div>
</div>
<div id="ref-OlzbV8x4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(267) </div><div class="csl-right-inline">Zecha, J.; Satpathy, S.; Kanashova, T.; Avanessian, S. C.; Kane, M. H.; Clauser, K. R.; Mertins, P.; Carr, S. A.; Kuster, B. TMT Labeling for the Masses: A Robust and Cost-Efficient, In-Solution Labeling Approach. <em>Mol Cell Proteomics</em> <strong>2019</strong>, <em>18</em> (7), 1468–1478. <a href="https://doi.org/10.1074/mcp.tir119.001385">https://doi.org/10.1074/mcp.tir119.001385</a>.</div>
</div>
<div id="ref-JokZNPi7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(268) </div><div class="csl-right-inline">Ross, P. L.; Huang, Y. N.; Marchese, J. N.; Williamson, B.; Parker, K.; Hattan, S.; Khainovski, N.; Pillai, S.; Dey, S.; Daniels, S.; Purkayastha, S.; Juhasz, P.; Martin, S.; Bartlet-Jones, M.; He, F.; Jacobson, A.; Pappin, D. J. Multiplexed Protein Quantitation in Saccharomyces Cerevisiae Using Amine-Reactive Isobaric Tagging Reagents. <em>Mol Cell Proteomics</em> <strong>2004</strong>, <em>3</em> (12), 1154–1169. <a href="https://doi.org/10.1074/mcp.m400129-mcp200">https://doi.org/10.1074/mcp.m400129-mcp200</a>.</div>
</div>
<div id="ref-ZPmx6w38" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(269) </div><div class="csl-right-inline">Aggarwal, S.; Yadav, A. K. Dissecting the iTRAQ Data Analysis. In <em>Methods in Molecular Biology</em>; Springer New York, 2016; pp 277–291. <a href="https://doi.org/10.1007/978-1-4939-3106-4_18">https://doi.org/10.1007/978-1-4939-3106-4_18</a>.</div>
</div>
<div id="ref-43RxThXr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(270) </div><div class="csl-right-inline">Pichler, P.; Köcher, T.; Holzmann, J.; Mazanek, M.; Taus, T.; Ammerer, G.; Mechtler, K. Peptide Labeling with Isobaric Tags Yields Higher Identification Rates Using iTRAQ 4-Plex Compared to TMT 6-Plex and iTRAQ 8-Plex on LTQ Orbitrap. <em>Anal Chem</em> <strong>2010</strong>, <em>82</em> (15), 6549–6558. <a href="https://doi.org/10.1021/ac100890k">https://doi.org/10.1021/ac100890k</a>.</div>
</div>
<div id="ref-1At6lP9vn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(271) </div><div class="csl-right-inline">Pottiez, G.; Wiederin, J.; Fox, H. S.; Ciborowski, P. Comparison of 4-Plex to 8-Plex iTRAQ Quantitative Measurements of Proteins in Human Plasma Samples. <em>J Proteome Res</em> <strong>2012</strong>, <em>11</em> (7), 3774–3781. <a href="https://doi.org/10.1021/pr300414z">https://doi.org/10.1021/pr300414z</a>.</div>
</div>
<div id="ref-5gl2gxA4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(272) </div><div class="csl-right-inline">Palmese, A.; De Rosa, C.; Chiappetta, G.; Marino, G.; Amoresano, A. Novel Method to Investigate Protein Carbonylation by iTRAQ Strategy. <em>Anal Bioanal Chem</em> <strong>2012</strong>, <em>404</em> (6-7), 1631–1635. <a href="https://doi.org/10.1007/s00216-012-6324-9">https://doi.org/10.1007/s00216-012-6324-9</a>.</div>
</div>
<div id="ref-t4QMhmYv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(273) </div><div class="csl-right-inline">Nyström, T. Role of Oxidative Carbonylation in Protein Quality Control and Senescence. <em>EMBO J</em> <strong>2005</strong>, <em>24</em> (7), 1311–1317. <a href="https://doi.org/10.1038/sj.emboj.7600599">https://doi.org/10.1038/sj.emboj.7600599</a>.</div>
</div>
<div id="ref-12bJg7wN8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(274) </div><div class="csl-right-inline">Thompson, A.; Wölmer, N.; Koncarevic, S.; Selzer, S.; Böhm, G.; Legner, H.; Schmid, P.; Kienle, S.; Penning, P.; Höhle, C.; Berfelde, A.; Martinez-Pinna, R.; Farztdinov, V.; Jung, S.; Kuhn, K.; Pike, I. TMTpro: Design, Synthesis, and Initial Evaluation of a Proline-Based Isobaric 16-Plex Tandem Mass Tag Reagent Set. <em>Anal. Chem.</em> <strong>2019</strong>, <em>91</em> (24), 15941–15950. <a href="https://doi.org/10.1021/acs.analchem.9b04474">https://doi.org/10.1021/acs.analchem.9b04474</a>.</div>
</div>
<div id="ref-1HEOHr3h0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(275) </div><div class="csl-right-inline">Li, J.; Cai, Z.; Bomgarden, R. D.; Pike, I.; Kuhn, K.; Rogers, J. C.; Roberts, T. M.; Gygi, S. P.; Paulo, J. A. TMTpro-18plex: The Expanded and Complete Set of TMTpro Reagents for Sample Multiplexing. <em>J. Proteome Res.</em> <strong>2021</strong>, <em>20</em> (5), 2964–2972. <a href="https://doi.org/10.1021/acs.jproteome.1c00168">https://doi.org/10.1021/acs.jproteome.1c00168</a>.</div>
</div>
<div id="ref-9irghCdu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(276) </div><div class="csl-right-inline">He, Y.; Shishkova, E.; Peters-Clarke, T. M.; Brademan, D. R.; Westphall, M. S.; Bergen, D.; Huang, J.; Huguet, R.; Senko, M. W.; Zabrouskov, V.; McAlister, G. C.; Coon, J. J. Evaluation of the Orbitrap Ascend Tribrid Mass Spectrometer for Shotgun Proteomics. <em>Anal. Chem.</em> <strong>2023</strong>, <em>95</em> (28), 10655–10663. <a href="https://doi.org/10.1021/acs.analchem.3c01155">https://doi.org/10.1021/acs.analchem.3c01155</a>.</div>
</div>
<div id="ref-1GJhuH2mX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(277) </div><div class="csl-right-inline">Ting, L.; Rad, R.; Gygi, S. P.; Haas, W. MS3 Eliminates Ratio Distortion in Isobaric Multiplexed Quantitative Proteomics. <em>Nat Methods</em> <strong>2011</strong>, <em>8</em> (11), 937–940. <a href="https://doi.org/10.1038/nmeth.1714">https://doi.org/10.1038/nmeth.1714</a>.</div>
</div>
<div id="ref-agiL3n2A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(278) </div><div class="csl-right-inline">McAlister, G. C.; Nusinow, D. P.; Jedrychowski, M. P.; Wühr, M.; Huttlin, E. L.; Erickson, B. K.; Rad, R.; Haas, W.; Gygi, S. P. MultiNotch MS3 Enables Accurate, Sensitive, and Multiplexed Detection of Differential Expression Across Cancer Cell Line Proteomes. <em>Anal. Chem.</em> <strong>2014</strong>, <em>86</em> (14), 7150–7158. <a href="https://doi.org/10.1021/ac502040v">https://doi.org/10.1021/ac502040v</a>.</div>
</div>
<div id="ref-1EjowCoSS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(279) </div><div class="csl-right-inline">Lee, K. W.; Peters-Clarke, T. M.; Mertz, K. L.; McAlister, G. C.; Syka, J. E. P.; Westphall, M. S.; Coon, J. J. Infrared Photoactivation Boosts Reporter Ion Yield in Isobaric Tagging. <em>Anal. Chem.</em> <strong>2022</strong>, <em>94</em> (7), 3328–3334. <a href="https://doi.org/10.1021/acs.analchem.1c05398">https://doi.org/10.1021/acs.analchem.1c05398</a>.</div>
</div>
<div id="ref-VW0HtJln" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(280) </div><div class="csl-right-inline">Schweppe, D. K.; Prasad, S.; Belford, M. W.; Navarrete-Perea, J.; Bailey, D. J.; Huguet, R.; Jedrychowski, M. P.; Rad, R.; McAlister, G.; Abbatiello, S. E.; Woulters, E. R.; Zabrouskov, V.; Dunyach, J.-J.; Paulo, J. A.; Gygi, S. P. Characterization and Optimization of Multiplexed Quantitative Analyses Using High-Field Asymmetric-Waveform Ion Mobility Mass Spectrometry. <em>Analytical Chemistry</em> <strong>2019</strong>, <em>91</em> (6), 4010–4016. https://doi.org/<a href="https://doi.org/10.1021/acs.analchem.8b05399">https://doi.org/10.1021/acs.analchem.8b05399</a>.</div>
</div>
<div id="ref-ghHhZLY4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(281) </div><div class="csl-right-inline">Pan, K.-T.; Chen, Y.-Y.; Pu, T.-H.; Chao, Y.-S.; Yang, C.-Y.; Bomgarden, R. D.; Rogers, J. C.; Meng, T.-C.; Khoo, K.-H. Mass Spectrometry-Based Quantitative Proteomics for Dissecting Multiplexed Redox Cysteine Modifications in Nitric Oxide-Protected Cardiomyocyte Under Hypoxia. <em>Antioxid Redox Signal</em> <strong>2013</strong>, <em>20</em> (9), 1365–1381. <a href="https://doi.org/10.1089/ars.2013.5326">https://doi.org/10.1089/ars.2013.5326</a>.</div>
</div>
<div id="ref-RikGmYWm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(282) </div><div class="csl-right-inline">Qu, Z.; Meng, F.; Bomgarden, R. D.; Viner, R. I.; Li, J.; Rogers, J. C.; Cheng, J.; Greenlief, C. M.; Cui, J.; Lubahn, D. B.; Sun, G. Y.; Gu, Z. Proteomic Quantification and Site-Mapping of S-Nitrosylated Proteins Using Isobaric iodoTMT Reagents. <em>J Proteome Res</em> <strong>2014</strong>, <em>13</em> (7), 3200–3211. <a href="https://doi.org/10.1021/pr401179v">https://doi.org/10.1021/pr401179v</a>.</div>
</div>
<div id="ref-XS3gqf5Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(283) </div><div class="csl-right-inline">Hahne, H.; Neubert, P.; Kuhn, K.; Etienne, C.; Bomgarden, R.; Rogers, J. C.; Kuster, B. Carbonyl-Reactive Tandem Mass Tags for the Proteome-Wide Quantification of N-Linked Glycans. <em>Anal Chem</em> <strong>2012</strong>, <em>84</em> (8), 3716–3724. <a href="https://doi.org/10.1021/ac300197c">https://doi.org/10.1021/ac300197c</a>.</div>
</div>
<div id="ref-WvWnJWa8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(284) </div><div class="csl-right-inline">Xiang, F.; Ye, H.; Chen, R.; Fu, Q.; Li, L. N,N-Dimethyl Leucines as Novel Isobaric Tandem Mass Tags for Quantitative Proteomics and Peptidomics. <em>Anal Chem</em> <strong>2010</strong>, <em>82</em> (7), 2817–2825. <a href="https://doi.org/10.1021/ac902778d">https://doi.org/10.1021/ac902778d</a>.</div>
</div>
<div id="ref-OhLoelJI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(285) </div><div class="csl-right-inline">Chen, Z.; Wang, Q.; Lin, L.; Tang, Q.; Edwards, J. L.; Li, S.; Liu, S. Comparative Evaluation of Two Isobaric Labeling Tags, DiART and iTRAQ. <em>Anal Chem</em> <strong>2012</strong>, <em>84</em> (6), 2908–2915. <a href="https://doi.org/10.1021/ac203467q">https://doi.org/10.1021/ac203467q</a>.</div>
</div>
<div id="ref-1FMnd7AHB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(286) </div><div class="csl-right-inline">Zhang, J.; Wang, Y.; Li, S. Deuterium Isobaric Amine-Reactive Tags for Quantitative Proteomics. <em>Anal Chem</em> <strong>2010</strong>, <em>82</em> (18), 7588–7595. <a href="https://doi.org/10.1021/ac101306x">https://doi.org/10.1021/ac101306x</a>.</div>
</div>
<div id="ref-1CFiIlIUn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(287) </div><div class="csl-right-inline">Zhang, Y.; Askenazi, M.; Jiang, J.; Luckey, C. J.; Griffin, J. D.; Marto, J. A. A Robust Error Model for iTRAQ Quantification Reveals Divergent Signaling Between Oncogenic FLT3 Mutants in Acute Myeloid Leukemia. <em>Mol Cell Proteomics</em> <strong>2009</strong>, <em>9</em> (5), 780–790. <a href="https://doi.org/10.1074/mcp.m900452-mcp200">https://doi.org/10.1074/mcp.m900452-mcp200</a>.</div>
</div>
<div id="ref-bHqseFm6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(288) </div><div class="csl-right-inline">Ramsubramaniam, N.; Tao, F.; Li, S.; Marten, M. R. Cost-Effective Isobaric Tagging for Quantitative Phosphoproteomics Using DiART Reagents. <em>Mol Biosyst</em> <strong>2013</strong>, <em>9</em> (12), 2981–2987. <a href="https://doi.org/10.1039/c3mb70358d">https://doi.org/10.1039/c3mb70358d</a>.</div>
</div>
<div id="ref-1E4PqaQ4R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(289) </div><div class="csl-right-inline">Dephoure, N.; Gygi, S. P. Hyperplexing: A Method for Higher-Order Multiplexed Quantitative Proteomics Provides a Map of the Dynamic Response to Rapamycin in Yeast. <em>Sci. Signal.</em> <strong>2012</strong>, <em>5</em> (217). <a href="https://doi.org/10.1126/scisignal.2002548">https://doi.org/10.1126/scisignal.2002548</a>.</div>
</div>
<div id="ref-AP3g38e9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(290) </div><div class="csl-right-inline">Liang, Y.; Truong, T.; Saxton, A. J.; Boekweg, H.; Payne, S. H.; Van Ry, P. M.; Kelly, R. T. HyperSCP: Combining Isotopic and Isobaric Labeling for Higher Throughput Single-Cell Proteomics. <em>Anal. Chem.</em> <strong>2023</strong>, <em>95</em> (20), 8020–8027. <a href="https://doi.org/10.1021/acs.analchem.3c00906">https://doi.org/10.1021/acs.analchem.3c00906</a>.</div>
</div>
<div id="ref-KGUz4OlJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(291) </div><div class="csl-right-inline">Aggarwal, S.; Talukdar, N. C.; Yadav, A. K. Advances in Higher Order Multiplexing Techniques in Proteomics. <em>J Proteome Res</em> <strong>2019</strong>, <em>18</em> (6), 2360–2369. <a href="https://doi.org/10.1021/acs.jproteome.9b00228">https://doi.org/10.1021/acs.jproteome.9b00228</a>.</div>
</div>
<div id="ref-RdDAao5G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(292) </div><div class="csl-right-inline">Aggarwal, S.; Kumar, A.; Jamwal, S.; Midha, M. K.; Talukdar, N. C.; Yadav, A. K. HyperQuant-A Computational Pipeline for Higher Order Multiplexed Quantitative Proteomics. <em>ACS Omega</em> <strong>2020</strong>, <em>5</em> (19), 10857–10867. <a href="https://doi.org/10.1021/acsomega.0c00515">https://doi.org/10.1021/acsomega.0c00515</a>.</div>
</div>
<div id="ref-TL5Hhri7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(293) </div><div class="csl-right-inline">Bowser, B. L.; Robinson, R. A. S. Enhanced Multiplexing Technology for Proteomics. <em>Annu Rev Anal Chem (Palo Alto Calif)</em> <strong>2023</strong>, <em>16</em> (1), 379–400. <a href="https://doi.org/10.1146/annurev-anchem-091622-092353">https://doi.org/10.1146/annurev-anchem-091622-092353</a>.</div>
</div>
<div id="ref-oFWp3zgI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(294) </div><div class="csl-right-inline">Jayapal, K. P.; Sui, S.; Philp, R. J.; Kok, Y.-J.; Yap, M. G. S.; Griffin, T. J.; Hu, W.-S. Multitagging Proteomic Strategy to Estimate Protein Turnover Rates in Dynamic Systems. <em>J. Proteome Res.</em> <strong>2010</strong>, <em>9</em> (5), 2087–2097. <a href="https://doi.org/10.1021/pr9007738">https://doi.org/10.1021/pr9007738</a>.</div>
</div>
<div id="ref-kwtDZh6G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(295) </div><div class="csl-right-inline">Gu, L.; Evans, A. R.; Robinson, R. A. S. Sample Multiplexing with Cysteine-Selective Approaches: cysDML and cPILOT. <em>J Am Soc Mass Spectrom</em> <strong>2015</strong>, <em>26</em> (4), 615–630. <a href="https://doi.org/10.1007/s13361-014-1059-9">https://doi.org/10.1007/s13361-014-1059-9</a>.</div>
</div>
<div id="ref-SOsi8PV2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(296) </div><div class="csl-right-inline">Evans, A. R.; Gu, L.; Guerrero, R.; Robinson, R. A. S. Global cPILOT Analysis of the APP/PS-1 Mouse Liver Proteome. <em>Proteomics Clin Appl</em> <strong>2015</strong>, <em>9</em> (9-10), 872–884. <a href="https://doi.org/10.1002/prca.201400149">https://doi.org/10.1002/prca.201400149</a>.</div>
</div>
<div id="ref-MmHm4JiO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(297) </div><div class="csl-right-inline">Evans, A. R.; Robinson, R. A. S. Global Combined Precursor Isotopic Labeling and Isobaric Tagging (cPILOT) Approach with Selective MS(3) Acquisition. <em>Proteomics</em> <strong>2013</strong>, <em>13</em> (22), 3267–3272. <a href="https://doi.org/10.1002/pmic.201300198">https://doi.org/10.1002/pmic.201300198</a>.</div>
</div>
<div id="ref-tQGfQU9D" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(298) </div><div class="csl-right-inline">Gu, L.; Robinson, R. A. S. High-Throughput Endogenous Measurement of S-Nitrosylation in Alzheimer's Disease Using Oxidized Cysteine-Selective cPILOT. <em>Analyst</em> <strong>2016</strong>, <em>141</em> (12), 3904–3915. <a href="https://doi.org/10.1039/c6an00417b">https://doi.org/10.1039/c6an00417b</a>.</div>
</div>
<div id="ref-1CCC3i2yw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(299) </div><div class="csl-right-inline">Dephoure, N.; Gygi, S. P. Hyperplexing: A Method for Higher-Order Multiplexed Quantitative Proteomics Provides a Map of the Dynamic Response to Rapamycin in Yeast. <em>Sci Signal</em> <strong>2012</strong>, <em>5</em> (217), rs2. <a href="https://doi.org/10.1126/scisignal.2002548">https://doi.org/10.1126/scisignal.2002548</a>.</div>
</div>
<div id="ref-a0wHWh6L" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(300) </div><div class="csl-right-inline">Schlage, P.; Kockmann, T.; Kizhakkedathu, J. N.; auf dem Keller, U. Monitoring Matrix Metalloproteinase Activity at the Epidermal-Dermal Interface by SILAC-iTRAQ-TAILS. <em>Proteomics</em> <strong>2015</strong>, <em>15</em> (14), 2491–2502. <a href="https://doi.org/10.1002/pmic.201400627">https://doi.org/10.1002/pmic.201400627</a>.</div>
</div>
<div id="ref-EcjnY7Z5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(301) </div><div class="csl-right-inline">Welle, K. A.; Zhang, T.; Hryhorenko, J. R.; Shen, S.; Qu, J.; Ghaemmaghami, S. Time-Resolved Analysis of Proteome Dynamics by Tandem Mass Tags and Stable Isotope Labeling in Cell Culture (TMT-SILAC) Hyperplexing. <em>Mol Cell Proteomics</em> <strong>2016</strong>, <em>15</em> (12), 3551–3563. <a href="https://doi.org/10.1074/mcp.m116.063230">https://doi.org/10.1074/mcp.m116.063230</a>.</div>
</div>
<div id="ref-bnkhDKrK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(302) </div><div class="csl-right-inline">Kumar, A.; Jamwal, S.; Midha, M. K.; Hamza, B.; Aggarwal, S.; Yadav, A. K.; Rao, K. V. S. Dataset Generated Using Hyperplexing and Click Chemistry to Monitor Temporal Dynamics of Newly Synthesized Macrophage Secretome Post Infection by Mycobacterial Strains. <em>Data Brief</em> <strong>2016</strong>, <em>9</em>, 349–354. <a href="https://doi.org/10.1016/j.dib.2016.08.055">https://doi.org/10.1016/j.dib.2016.08.055</a>.</div>
</div>
<div id="ref-sUDoGv6r" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(303) </div><div class="csl-right-inline">Rothenberg, D. A.; Taliaferro, J. M.; Huber, S. M.; Begley, T. J.; Dedon, P. C.; White, F. M. A Proteomics Approach to Profiling the Temporal Translational Response to Stress and Growth. <em>iScience</em> <strong>2018</strong>, <em>9</em>, 367–381. <a href="https://doi.org/10.1016/j.isci.2018.11.004">https://doi.org/10.1016/j.isci.2018.11.004</a>.</div>
</div>
<div id="ref-aoK1gRMs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(304) </div><div class="csl-right-inline">Potel, C. M.; Lin, M.-H.; Heck, A. J. R.; Lemeer, S. Widespread Bacterial Protein Histidine Phosphorylation Revealed by Mass Spectrometry-Based Proteomics. <em>Nat Methods</em> <strong>2018</strong>, <em>15</em> (3), 187–190. <a href="https://doi.org/10.1038/nmeth.4580">https://doi.org/10.1038/nmeth.4580</a>.</div>
</div>
<div id="ref-17Un7LUDR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(305) </div><div class="csl-right-inline">Kleinnijenhuis, A. J.; Kjeldsen, F.; Kallipolitis, B.; Haselmann, K. F.; Jensen, O. N. Analysis of Histidine Phosphorylation Using Tandem MS and Ion−Electron Reactions. <em>Anal. Chem.</em> <strong>2007</strong>, <em>79</em> (19), 7450–7456. <a href="https://doi.org/10.1021/ac0707838">https://doi.org/10.1021/ac0707838</a>.</div>
</div>
<div id="ref-14IWO8iZc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(306) </div><div class="csl-right-inline">Leijten, N. M.; Heck, A. J. R.; Lemeer, S. Histidine Phosphorylation in Human Cells; a Needle or Phantom in the Haystack? <em>Nat Methods</em> <strong>2022</strong>, <em>19</em> (7), 827–828. <a href="https://doi.org/10.1038/s41592-022-01524-0">https://doi.org/10.1038/s41592-022-01524-0</a>.</div>
</div>
<div id="ref-xcaW4mpA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(307) </div><div class="csl-right-inline">Schmidt, A.; Ammerer, G.; Mechtler, K. Studying the Fragmentation Behavior of Peptides with Arginine Phosphorylation and Its Influence on Phospho‐site Localization. <em>Proteomics</em> <strong>2013</strong>, <em>13</em> (6), 945–954. <a href="https://doi.org/10.1002/pmic.201200240">https://doi.org/10.1002/pmic.201200240</a>.</div>
</div>
<div id="ref-1ELOuUl8I" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(308) </div><div class="csl-right-inline">Salomon, A. R.; Ficarro, S. B.; Brill, L. M.; Brinker, A.; Phung, Q. T.; Ericson, C.; Sauer, K.; Brock, A.; Horn, D. M.; Schultz, P. G.; Peters, E. C. Profiling of Tyrosine Phosphorylation Pathways in Human Cells Using Mass Spectrometry. <em>Proc. Natl. Acad. Sci. U.S.A.</em> <strong>2003</strong>, <em>100</em> (2), 443–448. <a href="https://doi.org/10.1073/pnas.2436191100">https://doi.org/10.1073/pnas.2436191100</a>.</div>
</div>
<div id="ref-nO23Yma7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(309) </div><div class="csl-right-inline">Stopfer, L. E.; Flower, C. T.; Gajadhar, A. S.; Patel, B.; Gallien, S.; Lopez-Ferrer, D.; White, F. M. High-Density, Targeted Monitoring of Tyrosine Phosphorylation Reveals Activated Signaling Networks in Human Tumors. <em>Cancer Research</em> <strong>2021</strong>, <em>81</em> (9), 2495–2509. <a href="https://doi.org/10.1158/0008-5472.can-20-3804">https://doi.org/10.1158/0008-5472.can-20-3804</a>.</div>
</div>
<div id="ref-uOTrEnEQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(310) </div><div class="csl-right-inline">Olsen, J. V.; Blagoev, B.; Gnad, F.; Macek, B.; Kumar, C.; Mortensen, P.; Mann, M. Global, In Vivo, and Site-Specific Phosphorylation Dynamics in Signaling Networks. <em>Cell</em> <strong>2006</strong>, <em>127</em> (3), 635–648. <a href="https://doi.org/10.1016/j.cell.2006.09.026">https://doi.org/10.1016/j.cell.2006.09.026</a>.</div>
</div>
<div id="ref-8FBpI8YZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(311) </div><div class="csl-right-inline">Keck, F.; Ataey, P.; Amaya, M.; Bailey, C.; Narayanan, A. Phosphorylation of Single Stranded RNA Virus Proteins and Potential for Novel Therapeutic Strategies. <em>Viruses</em> <strong>2015</strong>, <em>7</em> (10), 5257–5273. <a href="https://doi.org/10.3390/v7102872">https://doi.org/10.3390/v7102872</a>.</div>
</div>
<div id="ref-ahQQ60nZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(312) </div><div class="csl-right-inline">Ochoa, D.; Jarnuczak, A. F.; Viéitez, C.; Gehre, M.; Soucheray, M.; Mateus, A.; Kleefeldt, A. A.; Hill, A.; Garcia-Alonso, L.; Stein, F.; Krogan, N. J.; Savitski, M. M.; Swaney, D. L.; Vizcaíno, J. A.; Noh, K.-M.; Beltrao, P. The Functional Landscape of the Human Phosphoproteome. <em>Nat Biotechnol</em> <strong>2019</strong>, <em>38</em> (3), 365–373. <a href="https://doi.org/10.1038/s41587-019-0344-3">https://doi.org/10.1038/s41587-019-0344-3</a>.</div>
</div>
<div id="ref-9tmuuRlu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(313) </div><div class="csl-right-inline">Savage, S. R.; Zhang, B. Using Phosphoproteomics Data to Understand Cellular Signaling: A Comprehensive Guide to Bioinformatics Resources. <em>Clin Proteomics</em> <strong>2020</strong>, <em>17</em>, 27. <a href="https://doi.org/10.1186/s12014-020-09290-x">https://doi.org/10.1186/s12014-020-09290-x</a>.</div>
</div>
<div id="ref-FzJAr4ba" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(314) </div><div class="csl-right-inline">Urban, J. A Review on Recent Trends in the Phosphoproteomics Workflow. From Sample Preparation to Data Analysis. <em>Anal Chim Acta</em> <strong>2021</strong>, <em>1199</em>, 338857. <a href="https://doi.org/10.1016/j.aca.2021.338857">https://doi.org/10.1016/j.aca.2021.338857</a>.</div>
</div>
<div id="ref-CG2bNzkD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(315) </div><div class="csl-right-inline">Cohen, P. The Role of Protein Phosphorylation in Human Health and Disease. The Sir Hans Krebs Medal Lecture. <em>Eur J Biochem</em> <strong>2001</strong>, <em>268</em> (19), 5001–5010. <a href="https://doi.org/10.1046/j.0014-2956.2001.02473.x">https://doi.org/10.1046/j.0014-2956.2001.02473.x</a>.</div>
</div>
<div id="ref-gPKAjoqC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(316) </div><div class="csl-right-inline">Zhao, Y.; Jensen, O. N. Modification-Specific Proteomics: Strategies for Characterization of Post-Translational Modifications Using Enrichment Techniques. <em>Proteomics</em> <strong>2009</strong>, <em>9</em> (20), 4632–4641. <a href="https://doi.org/10.1002/pmic.200900398">https://doi.org/10.1002/pmic.200900398</a>.</div>
</div>
<div id="ref-ZhEOEHzl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(317) </div><div class="csl-right-inline">Iliuk, A. B.; Arrington, J. V.; Tao, W. A. Analytical Challenges Translating Mass Spectrometry-Based Phosphoproteomics from Discovery to Clinical Applications. <em>Electrophoresis</em> <strong>2014</strong>, <em>35</em> (24), 3430–3440. <a href="https://doi.org/10.1002/elps.201400153">https://doi.org/10.1002/elps.201400153</a>.</div>
</div>
<div id="ref-IZ8H05sX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(318) </div><div class="csl-right-inline">Hsu, C.-C.; Xue, L.; Arrington, J. V.; Wang, P.; Paez Paez, J. S.; Zhou, Y.; Zhu, J.-K.; Tao, W. A. Estimating the Efficiency of Phosphopeptide Identification by Tandem Mass Spectrometry. <em>J Am Soc Mass Spectrom</em> <strong>2017</strong>, <em>28</em> (6), 1127–1135. <a href="https://doi.org/10.1007/s13361-017-1603-5">https://doi.org/10.1007/s13361-017-1603-5</a>.</div>
</div>
<div id="ref-1DdZgE3oc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(319) </div><div class="csl-right-inline">Gundry, R. L.; White, M. Y.; Murray, C. I.; Kane, L. A.; Fu, Q.; Stanley, B. A.; Van Eyk, J. E. Preparation of Proteins and Peptides for Mass Spectrometry Analysis in a Bottom-up Proteomics Workflow. <em>Curr Protoc Mol Biol</em> <strong>2009</strong>, <em>Chapter 10</em>, Unit10.25. <a href="https://doi.org/10.1002/0471142727.mb1025s88">https://doi.org/10.1002/0471142727.mb1025s88</a>.</div>
</div>
<div id="ref-1AhoIVYp8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(320) </div><div class="csl-right-inline">Potel, C. M.; Lemeer, S.; Heck, A. J. R. Phosphopeptide Fragmentation and Site Localization by Mass Spectrometry: An Update. <em>Anal Chem</em> <strong>2018</strong>, <em>91</em> (1), 126–141. <a href="https://doi.org/10.1021/acs.analchem.8b04746">https://doi.org/10.1021/acs.analchem.8b04746</a>.</div>
</div>
<div id="ref-2bm470Yr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(321) </div><div class="csl-right-inline">Fíla, J.; Honys, D. Enrichment Techniques Employed in Phosphoproteomics. <em>Amino Acids</em> <strong>2011</strong>, <em>43</em> (3), 1025–1047. <a href="https://doi.org/10.1007/s00726-011-1111-z">https://doi.org/10.1007/s00726-011-1111-z</a>.</div>
</div>
<div id="ref-cXqDwSab" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(322) </div><div class="csl-right-inline">Rex, D. A. B.; Subbannayya, Y.; Modi, P. K.; Palollathil, A.; Gopalakrishnan, L.; Bhandary, Y. P.; Prasad, T. S. K.; Pinto, S. M. Temporal Quantitative Phosphoproteomics Profiling of Interleukin-33 Signaling Network Reveals Unique Modulators of Monocyte Activation. <em>Cells</em> <strong>2022</strong>, <em>11</em> (1). <a href="https://doi.org/10.3390/cells11010138">https://doi.org/10.3390/cells11010138</a>.</div>
</div>
<div id="ref-vghskTda" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(323) </div><div class="csl-right-inline">Posewitz, M. C.; Tempst, P. Immobilized Gallium(III) Affinity Chromatography of Phosphopeptides. <em>Anal Chem</em> <strong>1999</strong>, <em>71</em> (14), 2883–2892. <a href="https://doi.org/10.1021/ac981409y">https://doi.org/10.1021/ac981409y</a>.</div>
</div>
<div id="ref-1Biu5x5kI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(324) </div><div class="csl-right-inline">Ojida, A.; Inoue, M.; Mito-oka, Y.; Tsutsumi, H.; Sada, K.; Hamachi, I. Effective Disruption of Phosphoprotein-Protein Surface Interaction Using Zn(II) Dipicolylamine-Based Artificial Receptors via Two-Point Interaction. <em>J Am Chem Soc</em> <strong>2006</strong>, <em>128</em> (6), 2052–2058. <a href="https://doi.org/10.1021/ja056585k">https://doi.org/10.1021/ja056585k</a>.</div>
</div>
<div id="ref-GINSjn64" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(325) </div><div class="csl-right-inline">Andersson, L.; Porath, J. Isolation of Phosphoproteins by Immobilized Metal (Fe3+) Affinity Chromatography. <em>Anal Biochem</em> <strong>1986</strong>, <em>154</em> (1), 250–254. <a href="https://doi.org/10.1016/0003-2697(86)90523-3">https://doi.org/10.1016/0003-2697(86)90523-3</a>.</div>
</div>
<div id="ref-tH2WxrEl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(326) </div><div class="csl-right-inline">Machida, M.; Kosako, H.; Shirakabe, K.; Kobayashi, M.; Ushiyama, M.; Inagawa, J.; Hirano, J.; Nakano, T.; Bando, Y.; Nishida, E.; Hattori, S. Purification of Phosphoproteins by Immobilized Metal Affinity Chromatography and Its Application to Phosphoproteome Analysis. <em>FEBS J</em> <strong>2007</strong>, <em>274</em> (6), 1576–1587. <a href="https://doi.org/10.1111/j.1742-4658.2007.05705.x">https://doi.org/10.1111/j.1742-4658.2007.05705.x</a>.</div>
</div>
<div id="ref-LbRH4fW1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(327) </div><div class="csl-right-inline">Dubrovska, A.; Souchelnytskyi, S. Efficient Enrichment of Intact Phosphorylated Proteins by Modified Immobilized Metal-Affinity Chromatography. <em>Proteomics</em> <strong>2005</strong>, <em>5</em> (18), 4678–4683. <a href="https://doi.org/10.1002/pmic.200500002">https://doi.org/10.1002/pmic.200500002</a>.</div>
</div>
<div id="ref-f75VaG7P" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(328) </div><div class="csl-right-inline">Zhou, S.; Bailey, M. J.; Dunn, M. J.; Preedy, V. R.; Emery, P. W. A Quantitative Investigation into the Losses of Proteins at Different Stages of a Two-Dimensional Gel Electrophoresis Procedure. <em>Proteomics</em> <strong>2005</strong>, <em>5</em> (11), 2739–2747. <a href="https://doi.org/10.1002/pmic.200401178">https://doi.org/10.1002/pmic.200401178</a>.</div>
</div>
<div id="ref-OoEPX07F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(329) </div><div class="csl-right-inline">Batth, T. S.; Francavilla, C.; Olsen, J. V. Off-Line High-pH Reversed-Phase Fractionation for In-Depth Phosphoproteomics. <em>J. Proteome Res.</em> <strong>2014</strong>, <em>13</em> (12), 6176–6186. <a href="https://doi.org/10.1021/pr500893m">https://doi.org/10.1021/pr500893m</a>.</div>
</div>
<div id="ref-CkrjTtPg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(330) </div><div class="csl-right-inline">Larsen, M. R.; Thingholm, T. E.; Jensen, O. N.; Roepstorff, P.; Jørgensen, T. J. D. Highly Selective Enrichment of Phosphorylated Peptides from Peptide Mixtures Using Titanium Dioxide Microcolumns. <em>Mol Cell Proteomics</em> <strong>2005</strong>, <em>4</em> (7), 873–886. <a href="https://doi.org/10.1074/mcp.t500007-mcp200">https://doi.org/10.1074/mcp.t500007-mcp200</a>.</div>
</div>
<div id="ref-11CslpEEW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(331) </div><div class="csl-right-inline">Thingholm, T. E.; Larsen, M. R. Phosphopeptide Enrichment by Immobilized Metal Affinity Chromatography. <em>Methods Mol Biol</em> <strong>2016</strong>, <em>1355</em>, 123–133. <a href="https://doi.org/10.1007/978-1-4939-3049-4_8">https://doi.org/10.1007/978-1-4939-3049-4_8</a>.</div>
</div>
<div id="ref-SLLSf3H4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(332) </div><div class="csl-right-inline">Carr, S. A.; Huddleston, M. J.; Annan, R. S. Selective Detection and Sequencing of Phosphopeptides at the Femtomole Level by Mass Spectrometry. <em>Analytical Biochemistry</em> <strong>1996</strong>, <em>239</em> (2), 180–192. <a href="https://doi.org/10.1006/abio.1996.0313">https://doi.org/10.1006/abio.1996.0313</a>.</div>
</div>
<div id="ref-Bij8Oa4F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(333) </div><div class="csl-right-inline">Ruprecht, B.; Koch, H.; Medard, G.; Mundt, M.; Kuster, B.; Lemeer, S. Comprehensive and Reproducible Phosphopeptide Enrichment Using Iron Immobilized Metal Ion Affinity Chromatography (Fe-IMAC) Columns. <em>Mol Cell Proteomics</em> <strong>2014</strong>, <em>14</em> (1), 205–215. <a href="https://doi.org/10.1074/mcp.m114.043109">https://doi.org/10.1074/mcp.m114.043109</a>.</div>
</div>
<div id="ref-15ptCbE4S" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(334) </div><div class="csl-right-inline">Ruprecht, B.; Koch, H.; Domasinska, P.; Frejno, M.; Kuster, B.; Lemeer, S. Optimized Enrichment of Phosphoproteomes by Fe-IMAC Column Chromatography. <em>Methods Mol Biol</em> <strong>2017</strong>, <em>1550</em>, 47–60. <a href="https://doi.org/10.1007/978-1-4939-6747-6_5">https://doi.org/10.1007/978-1-4939-6747-6_5</a>.</div>
</div>
<div id="ref-fPTYjwxV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(335) </div><div class="csl-right-inline">Lai, A. C.-Y.; Tsai, C.-F.; Hsu, C.-C.; Sun, Y.-N.; Chen, Y.-J. Complementary Fe(3+)- and Ti(4+)-Immobilized Metal Ion Affinity Chromatography for Purification of Acidic and Basic Phosphopeptides. <em>Rapid Commun Mass Spectrom</em> <strong>2012</strong>, <em>26</em> (18), 2186–2194. <a href="https://doi.org/10.1002/rcm.6327">https://doi.org/10.1002/rcm.6327</a>.</div>
</div>
<div id="ref-Hii2H7k2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(336) </div><div class="csl-right-inline">Choi, S.; Kim, J.; Cho, K.; Park, G.; Yoon, J. H.; Park, S.; Yoo, J. S.; Ryu, S. H.; Kim, Y. H.; Kim, J. Sequential Fe3O4/TiO2 Enrichment for Phosphopeptide Analysis by Liquid Chromatography/Tandem Mass Spectrometry. <em>Rapid Commun Mass Spectrom</em> <strong>2010</strong>, <em>24</em> (10), 1467–1474. <a href="https://doi.org/10.1002/rcm.4541">https://doi.org/10.1002/rcm.4541</a>.</div>
</div>
<div id="ref-vuiYaWET" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(337) </div><div class="csl-right-inline">Tong, J.; Cao, B.; Martyn, G. D.; Krieger, J. R.; Taylor, P.; Yates, B.; Sidhu, S. S.; Li, S. S. C.; Mao, X.; Moran, M. F. Protein-Phosphotyrosine Proteome Profiling by Superbinder-SH2 Domain Affinity Purification Mass Spectrometry, sSH2-AP-MS. <em>Proteomics</em> <strong>2017</strong>, <em>17</em> (6). <a href="https://doi.org/10.1002/pmic.201600360">https://doi.org/10.1002/pmic.201600360</a>.</div>
</div>
<div id="ref-1G78xLdle" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(338) </div><div class="csl-right-inline">Martyn, G. D.; Veggiani, G.; Sidhu, S. S. Engineering SH2 Domains with Tailored Specificities and Affinities. <em>Methods Mol Biol</em> <strong>2023</strong>, <em>2705</em>, 307–348. <a href="https://doi.org/10.1007/978-1-0716-3393-9_17">https://doi.org/10.1007/978-1-0716-3393-9_17</a>.</div>
</div>
<div id="ref-G33wPuEv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(339) </div><div class="csl-right-inline">Li, J.; Wang, J.; Yan, Y.; Li, N.; Qing, X.; Tuerxun, A.; Guo, X.; Chen, X.; Yang, F. Comprehensive Evaluation of Different TiO. <em>Cells</em> <strong>2022</strong>, <em>11</em> (13). <a href="https://doi.org/10.3390/cells11132047">https://doi.org/10.3390/cells11132047</a>.</div>
</div>
<div id="ref-2h6JMw35" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(340) </div><div class="csl-right-inline">Spiro, R. G. Protein Glycosylation: Nature, Distribution, Enzymatic Formation, and Disease Implications of Glycopeptide Bonds. <em>Glycobiology</em> <strong>2002</strong>, <em>12</em> (4), 43R–56R. <a href="https://doi.org/10.1093/glycob/12.4.43r">https://doi.org/10.1093/glycob/12.4.43r</a>.</div>
</div>
<div id="ref-DC1ukRtO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(341) </div><div class="csl-right-inline">Riley, N. M.; Bertozzi, C. R.; Pitteri, S. J. A Pragmatic Guide to Enrichment Strategies for Mass Spectrometry–Based Glycoproteomics. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2021</strong>, <em>20</em>, 100029. <a href="https://doi.org/10.1074/mcp.r120.002277">https://doi.org/10.1074/mcp.r120.002277</a>.</div>
</div>
<div id="ref-pgTlWbfO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(342) </div><div class="csl-right-inline">Bermudez, A.; Pitteri, S. J. Enrichment of Intact Glycopeptides Using Strong Anion Exchange and Electrostatic Repulsion Hydrophilic Interaction Chromatography. In <em>Methods in Molecular Biology</em>; Springer US, 2021; pp 107–120. <a href="https://doi.org/10.1007/978-1-0716-1241-5_8">https://doi.org/10.1007/978-1-0716-1241-5_8</a>.</div>
</div>
<div id="ref-1Bj6B87md" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(343) </div><div class="csl-right-inline">Totten, S. M.; Feasley, C. L.; Bermudez, A.; Pitteri, S. J. Parallel Comparison of N-Linked Glycopeptide Enrichment Techniques Reveals Extensive Glycoproteomic Analysis of Plasma Enabled by SAX-ERLIC. <em>J. Proteome Res.</em> <strong>2017</strong>, <em>16</em> (3), 1249–1260. <a href="https://doi.org/10.1021/acs.jproteome.6b00849">https://doi.org/10.1021/acs.jproteome.6b00849</a>.</div>
</div>
<div id="ref-3T8OfCLe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(344) </div><div class="csl-right-inline">Wohlgemuth, J.; Karas, M.; Eichhorn, T.; Hendriks, R.; Andrecht, S. Quantitative Site-Specific Analysis of Protein Glycosylation by LC-MS Using Different Glycopeptide-Enrichment Strategies. <em>Analytical Biochemistry</em> <strong>2009</strong>, <em>395</em> (2), 178–188. <a href="https://doi.org/10.1016/j.ab.2009.08.023">https://doi.org/10.1016/j.ab.2009.08.023</a>.</div>
</div>
<div id="ref-17QEVShGe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(345) </div><div class="csl-right-inline">Thaysen-Andersen, M.; Packer, N. H. Advances in LC–MS/MS-Based Glycoproteomics: Getting Closer to System-Wide Site-Specific Mapping of the N- and O-Glycoproteome. <em>Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics</em> <strong>2014</strong>, <em>1844</em> (9), 1437–1452. <a href="https://doi.org/10.1016/j.bbapap.2014.05.002">https://doi.org/10.1016/j.bbapap.2014.05.002</a>.</div>
</div>
<div id="ref-ZH7oVCIW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(346) </div><div class="csl-right-inline">Thaysen-Andersen, M.; Larsen, M. R.; Packer, N. H.; Palmisano, G. Structural Analysis of Glycoprotein Sialylation – Part I: Pre-LC-MS Analytical Strategies. <em>RSC Adv.</em> <strong>2013</strong>, <em>3</em> (45), 22683. <a href="https://doi.org/10.1039/c3ra42960a">https://doi.org/10.1039/c3ra42960a</a>.</div>
</div>
<div id="ref-4qdFwSjo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(347) </div><div class="csl-right-inline">Gaunitz, S.; Nagy, G.; Pohl, N. L. B.; Novotny, M. V. Recent Advances in the Analysis of Complex Glycoproteins. <em>Anal. Chem.</em> <strong>2016</strong>, <em>89</em> (1), 389–413. <a href="https://doi.org/10.1021/acs.analchem.6b04343">https://doi.org/10.1021/acs.analchem.6b04343</a>.</div>
</div>
<div id="ref-LeABskLB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(348) </div><div class="csl-right-inline">Ghosh, R.; Gilda, J. E.; Gomes, A. V. The Necessity of and Strategies for Improving Confidence in the Accuracy of Western Blots. <em>Expert Rev Proteomics</em> <strong>2014</strong>, <em>11</em> (5), 549–560. <a href="https://doi.org/10.1586/14789450.2014.939635">https://doi.org/10.1586/14789450.2014.939635</a>.</div>
</div>
<div id="ref-xEkij3xI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(349) </div><div class="csl-right-inline">Zhao, Y.; Muir, T. W.; Kent, S. B.; Tischer, E.; Scardina, J. M.; Chait, B. T. Mapping Protein-Protein Interactions by Affinity-Directed Mass Spectrometry. <em>Proc Natl Acad Sci U S A</em> <strong>1996</strong>, <em>93</em> (9), 4020–4024. <a href="https://doi.org/10.1073/pnas.93.9.4020">https://doi.org/10.1073/pnas.93.9.4020</a>.</div>
</div>
<div id="ref-oN0RedlD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(350) </div><div class="csl-right-inline">Rush, J.; Moritz, A.; Lee, K. A.; Guo, A.; Goss, V. L.; Spek, E. J.; Zhang, H.; Zha, X.-M.; Polakiewicz, R. D.; Comb, M. J. Immunoaffinity Profiling of Tyrosine Phosphorylation in Cancer Cells. <em>Nat Biotechnol</em> <strong>2004</strong>, <em>23</em> (1), 94–101. <a href="https://doi.org/10.1038/nbt1046">https://doi.org/10.1038/nbt1046</a>.</div>
</div>
<div id="ref-11yMtXtzL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(351) </div><div class="csl-right-inline">Matsuoka, S.; Ballif, B. A.; Smogorzewska, A.; McDonald, E. R.; Hurov, K. E.; Luo, J.; Bakalarski, C. E.; Zhao, Z.; Solimini, N.; Lerenthal, Y.; Shiloh, Y.; Gygi, S. P.; Elledge, S. J. ATM and ATR Substrate Analysis Reveals Extensive Protein Networks Responsive to DNA Damage. <em>Science</em> <strong>2007</strong>, <em>316</em> (5828), 1160–1166. <a href="https://doi.org/10.1126/science.1140321">https://doi.org/10.1126/science.1140321</a>.</div>
</div>
<div id="ref-nEmRnHBE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(352) </div><div class="csl-right-inline">Moritz, A.; Li, Y.; Guo, A.; Villén, J.; Wang, Y.; MacNeill, J.; Kornhauser, J.; Sprott, K.; Zhou, J.; Possemato, A.; Ren, J. M.; Hornbeck, P.; Cantley, L. C.; Gygi, S. P.; Rush, J.; Comb, M. J. Akt-RSK-S6 Kinase Signaling Networks Activated by Oncogenic Receptor Tyrosine Kinases. <em>Sci Signal</em> <strong>2010</strong>, <em>3</em> (136), ra64. <a href="https://doi.org/10.1126/scisignal.2000998">https://doi.org/10.1126/scisignal.2000998</a>.</div>
</div>
<div id="ref-fZ6aM0wq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(353) </div><div class="csl-right-inline">Emanuele, M. J.; Elia, A. E. H.; Xu, Q.; Thoma, C. R.; Izhar, L.; Leng, Y.; Guo, A.; Chen, Y.-N.; Rush, J.; Hsu, P. W.-C.; Yen, H.-C. S.; Elledge, S. J. Global Identification of Modular Cullin-RING Ligase Substrates. <em>Cell</em> <strong>2011</strong>, <em>147</em> (2), 459–474. <a href="https://doi.org/10.1016/j.cell.2011.09.019">https://doi.org/10.1016/j.cell.2011.09.019</a>.</div>
</div>
<div id="ref-JMsurOWF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(354) </div><div class="csl-right-inline">Kim, W.; Bennett, E. J.; Huttlin, E. L.; Guo, A.; Li, J.; Possemato, A.; Sowa, M. E.; Rad, R.; Rush, J.; Comb, M. J.; Harper, J. W.; Gygi, S. P. Systematic and Quantitative Assessment of the Ubiquitin-Modified Proteome. <em>Mol Cell</em> <strong>2011</strong>, <em>44</em> (2), 325–340. <a href="https://doi.org/10.1016/j.molcel.2011.08.025">https://doi.org/10.1016/j.molcel.2011.08.025</a>.</div>
</div>
<div id="ref-11FwNutcB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(355) </div><div class="csl-right-inline">Wagner, S. A.; Beli, P.; Weinert, B. T.; Nielsen, M. L.; Cox, J.; Mann, M.; Choudhary, C. A Proteome-Wide, Quantitative Survey of in Vivo Ubiquitylation Sites Reveals Widespread Regulatory Roles. <em>Mol Cell Proteomics</em> <strong>2011</strong>, <em>10</em> (10), M111.013284. <a href="https://doi.org/10.1074/mcp.m111.013284">https://doi.org/10.1074/mcp.m111.013284</a>.</div>
</div>
<div id="ref-tWUlvotu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(356) </div><div class="csl-right-inline">Choudhary, C.; Kumar, C.; Gnad, F.; Nielsen, M. L.; Rehman, M.; Walther, T. C.; Olsen, J. V.; Mann, M. Lysine Acetylation Targets Protein Complexes and Co-Regulates Major Cellular Functions. <em>Science</em> <strong>2009</strong>, <em>325</em> (5942), 834–840. <a href="https://doi.org/10.1126/science.1175371">https://doi.org/10.1126/science.1175371</a>.</div>
</div>
<div id="ref-36Wx4WMF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(357) </div><div class="csl-right-inline">Hebert, A. S.; Dittenhafer-Reed, K. E.; Yu, W.; Bailey, D. J.; Selen, E. S.; Boersma, M. D.; Carson, J. J.; Tonelli, M.; Balloon, A. J.; Higbee, A. J.; Westphall, M. S.; Pagliarini, D. J.; Prolla, T. A.; Assadi-Porter, F.; Roy, S.; Denu, J. M.; Coon, J. J. Calorie Restriction and SIRT3 Trigger Global Reprogramming of the Mitochondrial Protein Acetylome. <em>Mol Cell</em> <strong>2012</strong>, <em>49</em> (1), 186–199. <a href="https://doi.org/10.1016/j.molcel.2012.10.024">https://doi.org/10.1016/j.molcel.2012.10.024</a>.</div>
</div>
<div id="ref-bAU4w5zL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(358) </div><div class="csl-right-inline">Svinkina, T.; Gu, H.; Silva, J. C.; Mertins, P.; Qiao, J.; Fereshetian, S.; Jaffe, J. D.; Kuhn, E.; Udeshi, N. D.; Carr, S. A. Deep, Quantitative Coverage of the Lysine Acetylome Using Novel Anti-Acetyl-Lysine Antibodies and an Optimized Proteomic Workflow. <em>Mol Cell Proteomics</em> <strong>2015</strong>, <em>14</em> (9), 2429–2440. <a href="https://doi.org/10.1074/mcp.o114.047555">https://doi.org/10.1074/mcp.o114.047555</a>.</div>
</div>
<div id="ref-RMueHUu6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(359) </div><div class="csl-right-inline">Schilling, B.; Meyer, J. G.; Wei, L.; Ott, M.; Verdin, E. High-Resolution Mass Spectrometry to Identify and Quantify Acetylation Protein Targets. <em>Methods Mol Biol</em> <strong>2019</strong>, <em>1983</em>, 3–16. <a href="https://doi.org/10.1007/978-1-4939-9434-2_1">https://doi.org/10.1007/978-1-4939-9434-2_1</a>.</div>
</div>
<div id="ref-58knZUn0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(360) </div><div class="csl-right-inline">Basisty, N.; Meyer, J. G.; Wei, L.; Gibson, B. W.; Schilling, B. Simultaneous Quantification of the Acetylome and Succinylome by ‘One‐Pot’ Affinity Enrichment. <em>Proteomics</em> <strong>2018</strong>, <em>18</em> (17). <a href="https://doi.org/10.1002/pmic.201800123">https://doi.org/10.1002/pmic.201800123</a>.</div>
</div>
<div id="ref-I7T7nB5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(361) </div><div class="csl-right-inline">Guo, A.; Gu, H.; Zhou, J.; Mulhern, D.; Wang, Y.; Lee, K. A.; Yang, V.; Aguiar, M.; Kornhauser, J.; Jia, X.; Ren, J.; Beausoleil, S. A.; Silva, J. C.; Vemulapalli, V.; Bedford, M. T.; Comb, M. J. Immunoaffinity Enrichment and Mass Spectrometry Analysis of Protein Methylation. <em>Mol Cell Proteomics</em> <strong>2013</strong>, <em>13</em> (1), 372–387. <a href="https://doi.org/10.1074/mcp.o113.027870">https://doi.org/10.1074/mcp.o113.027870</a>.</div>
</div>
<div id="ref-82ISyiE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(362) </div><div class="csl-right-inline">Zhao, Y.; Zhang, Y.; Sun, H.; Maroto, R.; Brasier, A. R. Selective Affinity Enrichment of Nitrotyrosine-Containing Peptides for Quantitative Analysis in Complex Samples. <em>J Proteome Res</em> <strong>2017</strong>, <em>16</em> (8), 2983–2992. <a href="https://doi.org/10.1021/acs.jproteome.7b00275">https://doi.org/10.1021/acs.jproteome.7b00275</a>.</div>
</div>
<div id="ref-1CF6MyQQ3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(363) </div><div class="csl-right-inline">Boersema, P. J.; Foong, L. Y.; Ding, V. M. Y.; Lemeer, S.; van Breukelen, B.; Philp, R.; Boekhorst, J.; Snel, B.; den Hertog, J.; Choo, A. B. H.; Heck, A. J. R. In-Depth Qualitative and Quantitative Profiling of Tyrosine Phosphorylation Using a Combination of Phosphopeptide Immunoaffinity Purification and Stable Isotope Dimethyl Labeling. <em>Mol Cell Proteomics</em> <strong>2009</strong>, <em>9</em> (1), 84–99. <a href="https://doi.org/10.1074/mcp.m900291-mcp200">https://doi.org/10.1074/mcp.m900291-mcp200</a>.</div>
</div>
<div id="ref-PgRmFpK6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(364) </div><div class="csl-right-inline">Zhang, Y.; Wolf-Yadlin, A.; Ross, P. L.; Pappin, D. J.; Rush, J.; Lauffenburger, D. A.; White, F. M. Time-Resolved Mass Spectrometry of Tyrosine Phosphorylation Sites in the Epidermal Growth Factor Receptor Signaling Network Reveals Dynamic Modules. <em>Mol Cell Proteomics</em> <strong>2005</strong>, <em>4</em> (9), 1240–1250. <a href="https://doi.org/10.1074/mcp.m500089-mcp200">https://doi.org/10.1074/mcp.m500089-mcp200</a>.</div>
</div>
<div id="ref-hAQepsrJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(365) </div><div class="csl-right-inline">Lu, Q.; Zhang, X.; Liang, T.; Bai, X. O-GlcNAcylation: An Important Post-Translational Modification and a Potential Therapeutic Target for Cancer Therapy. <em>Mol Med</em> <strong>2022</strong>, <em>28</em> (1), 115. <a href="https://doi.org/10.1186/s10020-022-00544-y">https://doi.org/10.1186/s10020-022-00544-y</a>.</div>
</div>
<div id="ref-UV2xOP8u" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(366) </div><div class="csl-right-inline">Burt, R. A.; Dejanovic, B.; Peckham, H. J.; Lee, K. A.; Li, X.; Ounadjela, J. R.; Rao, A.; Malaker, S. A.; Carr, S. A.; Myers, S. A. Novel Antibodies for the Simple and Efficient Enrichment of Native O-GlcNAc Modified Peptides. <em>Mol Cell Proteomics</em> <strong>2021</strong>, <em>20</em>, 100167. <a href="https://doi.org/10.1016/j.mcpro.2021.100167">https://doi.org/10.1016/j.mcpro.2021.100167</a>.</div>
</div>
<div id="ref-Zhiw1CQm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(367) </div><div class="csl-right-inline">Fu, Q.; Kowalski, M. P.; Mastali, M.; Parker, S. J.; Sobhani, K.; van den Broek, I.; Hunter, C. L.; Van Eyk, J. E. Highly Reproducible Automated Proteomics Sample Preparation Workflow for Quantitative Mass Spectrometry. <em>J. Proteome Res.</em> <strong>2017</strong>, <em>17</em> (1), 420–428. <a href="https://doi.org/10.1021/acs.jproteome.7b00623">https://doi.org/10.1021/acs.jproteome.7b00623</a>.</div>
</div>
<div id="ref-3DOhQZCg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(368) </div><div class="csl-right-inline">Overmyer, K. A.; Shishkova, E.; Miller, I. J.; Balnis, J.; Bernstein, M. N.; Peters-Clarke, T. M.; Meyer, J. G.; Quan, Q.; Muehlbauer, L. K.; Trujillo, E. A.; He, Y.; Chopra, A.; Chieng, H. C.; Tiwari, A.; Judson, M. A.; Paulson, B.; Brademan, D. R.; Zhu, Y.; Serrano, L. R.; Linke, V.; Drake, L. A.; Adam, A. P.; Schwartz, B. S.; Singer, H. A.; Swanson, S.; Mosher, D. F.; Stewart, R.; Coon, J. J.; Jaitovich, A. Large-Scale Multi-Omic Analysis of COVID-19 Severity. <em>Cell Systems</em> <strong>2021</strong>, <em>12</em> (1), 23–40.e7. <a href="https://doi.org/10.1016/j.cels.2020.10.003">https://doi.org/10.1016/j.cels.2020.10.003</a>.</div>
</div>
<div id="ref-xwhmqv6G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(369) </div><div class="csl-right-inline">Tu, C.; Rudnick, P. A.; Martinez, M. Y.; Cheek, K. L.; Stein, S. E.; Slebos, R. J. C.; Liebler, D. C. Depletion of Abundant Plasma Proteins and Limitations of Plasma Proteomics. <em>J Proteome Res</em> <strong>2010</strong>, <em>9</em> (10), 4982–4991. <a href="https://doi.org/10.1021/pr100646w">https://doi.org/10.1021/pr100646w</a>.</div>
</div>
<div id="ref-MZe4do4v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(370) </div><div class="csl-right-inline">Zolotarjova, N.; Martosella, J.; Nicol, G.; Bailey, J.; Boyes, B. E.; Barrett, W. C. Differences Among Techniques for High-Abundant Protein Depletion. <em>Proteomics</em> <strong>2005</strong>, <em>5</em> (13), 3304–3313. <a href="https://doi.org/10.1002/pmic.200402021">https://doi.org/10.1002/pmic.200402021</a>.</div>
</div>
<div id="ref-CkoUIvms" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(371) </div><div class="csl-right-inline">Denizli, A.; Pişkin, E. Dye-Ligand Affinity Systems. <em>J Biochem Biophys Methods</em> <strong>2001</strong>, <em>49</em> (1-3), 391–416. <a href="https://doi.org/10.1016/s0165-022x(01)00209-3">https://doi.org/10.1016/s0165-022x(01)00209-3</a>.</div>
</div>
<div id="ref-mxcAPQRD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(372) </div><div class="csl-right-inline">Andaç, M. Cibacron Blue Immobilized Poly(glycidyl-Methacrylate) Nanobeads for Albumin Removal in Proteome Studies. <em>Artif Cells Nanomed Biotechnol</em> <strong>2013</strong>, <em>43</em> (2), 133–139. <a href="https://doi.org/10.3109/21691401.2013.852102">https://doi.org/10.3109/21691401.2013.852102</a>.</div>
</div>
<div id="ref-LonwNp1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(373) </div><div class="csl-right-inline">Sjöbring, U.; Björck, L.; Kastern, W. Protein G Genes: Structure and Distribution of IgG-Binding and Albumin-Binding Domains. <em>Mol Microbiol</em> <strong>1989</strong>, <em>3</em> (3), 319–327. <a href="https://doi.org/10.1111/j.1365-2958.1989.tb00177.x">https://doi.org/10.1111/j.1365-2958.1989.tb00177.x</a>.</div>
</div>
<div id="ref-23zI8SGX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(374) </div><div class="csl-right-inline">Moks, T.; Abrahmsén, L.; Nilsson, B.; Hellman, U.; Sjöquist, J.; Uhlén, M. Staphylococcal Protein A Consists of Five IgG-Binding Domains. <em>Eur J Biochem</em> <strong>1986</strong>, <em>156</em> (3), 637–643. <a href="https://doi.org/10.1111/j.1432-1033.1986.tb09625.x">https://doi.org/10.1111/j.1432-1033.1986.tb09625.x</a>.</div>
</div>
<div id="ref-LRMcEp9I" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(375) </div><div class="csl-right-inline">Graille, M.; Stura, E. A.; Corper, A. L.; Sutton, B. J.; Taussig, M. J.; Charbonnier, J. B.; Silverman, G. J. Crystal Structure of a Staphylococcus Aureus Protein A Domain Complexed with the Fab Fragment of a Human IgM Antibody: Structural Basis for Recognition of B-Cell Receptors and Superantigen Activity. <em>Proc Natl Acad Sci U S A</em> <strong>2000</strong>, <em>97</em> (10), 5399–5404. <a href="https://doi.org/10.1073/pnas.97.10.5399">https://doi.org/10.1073/pnas.97.10.5399</a>.</div>
</div>
<div id="ref-gxra2RsF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(376) </div><div class="csl-right-inline">Lee, P. Y.; Osman, J.; Low, T. Y.; Jamal, R. Plasma/Serum Proteomics: Depletion Strategies for Reducing High-Abundance Proteins for Biomarker Discovery. <em>Bioanalysis</em> <strong>2019</strong>, <em>11</em> (19), 1799–1812. <a href="https://doi.org/10.4155/bio-2019-0145">https://doi.org/10.4155/bio-2019-0145</a>.</div>
</div>
<div id="ref-13El9kECw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(377) </div><div class="csl-right-inline">Garcia, S.; Baldasso, P. A.; Guest, P. C.; Martins-de-Souza, D. Depletion of Highly Abundant Proteins of the Human Blood Plasma: Applications in Proteomics Studies of Psychiatric Disorders. <em>Methods Mol Biol</em> <strong>2017</strong>, <em>1546</em>, 195–204. <a href="https://doi.org/10.1007/978-1-4939-6730-8_16">https://doi.org/10.1007/978-1-4939-6730-8_16</a>.</div>
</div>
<div id="ref-xeb6grTF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(378) </div><div class="csl-right-inline">Guerrier, L.; Righetti, P. G.; Boschetti, E. Reduction of Dynamic Protein Concentration Range of Biological Extracts for the Discovery of Low-Abundance Proteins by Means of Hexapeptide Ligand Library. <em>Nat Protoc</em> <strong>2008</strong>, <em>3</em> (5), 883–890. <a href="https://doi.org/10.1038/nprot.2008.59">https://doi.org/10.1038/nprot.2008.59</a>.</div>
</div>
<div id="ref-1AFK16JwO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(379) </div><div class="csl-right-inline">Righetti, P. G.; Boschetti, E. Sample Treatment Methods Involving Combinatorial Peptide Ligand Libraries for Improved Proteomes Analyses. <em>Methods Mol Biol</em> <strong>2015</strong>, <em>1243</em>, 55–82. <a href="https://doi.org/10.1007/978-1-4939-1872-0_4">https://doi.org/10.1007/978-1-4939-1872-0_4</a>.</div>
</div>
<div id="ref-hcs5DCXe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(380) </div><div class="csl-right-inline">Mahn, A.; Ismail, M. Depletion of Highly Abundant Proteins in Blood Plasma by Ammonium Sulfate Precipitation for 2D-PAGE Analysis. <em>J Chromatogr B Analyt Technol Biomed Life Sci</em> <strong>2011</strong>, <em>879</em> (30), 3645–3648. <a href="https://doi.org/10.1016/j.jchromb.2011.09.024">https://doi.org/10.1016/j.jchromb.2011.09.024</a>.</div>
</div>
<div id="ref-jOj0DZcZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(381) </div><div class="csl-right-inline">Liu, G.; Zhao, Y.; Angeles, A.; Hamuro, L. L.; Arnold, M. E.; Shen, J. X. A Novel and Cost Effective Method of Removing Excess Albumin from Plasma/Serum Samples and Its Impacts on LC-MS/MS Bioanalysis of Therapeutic Proteins. <em>Anal Chem</em> <strong>2014</strong>, <em>86</em> (16), 8336–8343. <a href="https://doi.org/10.1021/ac501837t">https://doi.org/10.1021/ac501837t</a>.</div>
</div>
<div id="ref-1DUeuj8TB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(382) </div><div class="csl-right-inline">Liu, Z.; Fan, S.; Liu, H.; Yu, J.; Qiao, R.; Zhou, M.; Yang, Y.; Zhou, J.; Xie, P. Enhanced Detection of Low-Abundance Human Plasma Proteins by Integrating Polyethylene Glycol Fractionation and Immunoaffinity Depletion. <em>PLoS One</em> <strong>2016</strong>, <em>11</em> (11), e0166306. <a href="https://doi.org/10.1371/journal.pone.0166306">https://doi.org/10.1371/journal.pone.0166306</a>.</div>
</div>
<div id="ref-jv9v3twJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(383) </div><div class="csl-right-inline">Warder, S. E.; Tucker, L. A.; Strelitzer, T. J.; McKeegan, E. M.; Meuth, J. L.; Jung, P. M.; Saraf, A.; Singh, B.; Lai-Zhang, J.; Gagne, G.; Rogers, J. C. Reducing Agent-Mediated Precipitation of High-Abundance Plasma Proteins. <em>Anal Biochem</em> <strong>2009</strong>, <em>387</em> (2), 184–193. <a href="https://doi.org/10.1016/j.ab.2009.01.013">https://doi.org/10.1016/j.ab.2009.01.013</a>.</div>
</div>
<div id="ref-a6kEMboa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(384) </div><div class="csl-right-inline">Göktürk, I.; Tamahkar, E.; Yılmaz, F.; Denizli, A. Protein Depletion with Bacterial Cellulose Nanofibers. <em>J Chromatogr B Analyt Technol Biomed Life Sci</em> <strong>2018</strong>, <em>1099</em>, 1–9. <a href="https://doi.org/10.1016/j.jchromb.2018.08.030">https://doi.org/10.1016/j.jchromb.2018.08.030</a>.</div>
</div>
<div id="ref-1KROnN78" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(385) </div><div class="csl-right-inline">Andac, M.; Galaev, I. Y.; Denizli, A. Molecularly Imprinted Poly(hydroxyethyl Methacrylate) Based Cryogel for Albumin Depletion from Human Serum. <em>Colloids Surf B Biointerfaces</em> <strong>2013</strong>, <em>109</em>, 259–265. <a href="https://doi.org/10.1016/j.colsurfb.2013.03.054">https://doi.org/10.1016/j.colsurfb.2013.03.054</a>.</div>
</div>
<div id="ref-7cVO0ON7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(386) </div><div class="csl-right-inline">Tamahkar, E.; Babaç, C.; Kutsal, T.; Pişkin, E.; Denizli, A. Bacterial Cellulose Nanofibers for Albumin Depletion from Human Serum. <em>Process Biochemistry</em> <strong>2010</strong>, <em>45</em> (10), 1713–1719. <a href="https://doi.org/10.1016/j.procbio.2010.07.007">https://doi.org/10.1016/j.procbio.2010.07.007</a>.</div>
</div>
<div id="ref-iOe9QsbS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(387) </div><div class="csl-right-inline">Blume, J. E.; Manning, W. C.; Troiano, G.; Hornburg, D.; Figa, M.; Hesterberg, L.; Platt, T. L.; Zhao, X.; Cuaresma, R. A.; Everley, P. A.; Ko, M.; Liou, H.; Mahoney, M.; Ferdosi, S.; Elgierari, E. M.; Stolarczyk, C.; Tangeysh, B.; Xia, H.; Benz, R.; Siddiqui, A.; Carr, S. A.; Ma, P.; Langer, R.; Farias, V.; Farokhzad, O. C. Rapid, Deep and Precise Profiling of the Plasma Proteome with Multi-Nanoparticle Protein Corona. <em>Nat Commun</em> <strong>2020</strong>, <em>11</em> (1). <a href="https://doi.org/10.1038/s41467-020-17033-7">https://doi.org/10.1038/s41467-020-17033-7</a>.</div>
</div>
<div id="ref-37v9j349" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(388) </div><div class="csl-right-inline">Huang, T.; Wang, J.; Stukalov, A.; Donovan, M. K. R.; Ferdosi, S.; Williamson, L.; Just, S.; Castro, G.; Cantrell, L. S.; Elgierari, E.; Benz, R. W.; Huang, Y.; Motamedchaboki, K.; Hakimi, A.; Arrey, T.; Damoc, E.; Kreimer, S.; Farokhzad, O. C.; Batzoglou, S.; Siddiqui, A.; Van Eyk, J. E.; Hornburg, D. Protein Coronas on Functionalized Nanoparticles Enable Quantitative and Precise Large-Scale Deep Plasma Proteomics, 2023. <a href="https://doi.org/10.1101/2023.08.28.555225">https://doi.org/10.1101/2023.08.28.555225</a>.</div>
</div>
<div id="ref-1EP8DknKE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(389) </div><div class="csl-right-inline">Wu, C. C.; Tsantilas, K. A.; Park, J.; Plubell, D.; Sanders, J. A.; Naicker, P.; Govender, I.; Buthelezi, S.; Stoychev, S.; Jordaan, J.; Merrihew, G.; Huang, E.; Parker, E. D.; Riffle, M.; Hoofnagle, A. N.; Noble, W. S.; Poston, K. L.; Montine, T. J.; MacCoss, M. J. Mag-Net: Rapid Enrichment of Membrane-Bound Particles Enables High Coverage Quantitative Analysis of the Plasma Proteome, 2023. <a href="https://doi.org/10.1101/2023.06.10.544439">https://doi.org/10.1101/2023.06.10.544439</a>.</div>
</div>
<div id="ref-d0vk8527" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(390) </div><div class="csl-right-inline">Huang, T.; Wang, J.; Stukalov, A.; Donovan, M. K. R.; Ferdosi, S.; Williamson, L.; Just, S.; Castro, G.; Cantrell, L. S.; Elgierari, E.; Benz, R. W.; Huang, Y.; Motamedchaboki, K.; Hakimi, A.; Arrey, T.; Damoc, E.; Kreimer, S.; Farokhzad, O. C.; Batzoglou, S.; Siddiqui, A.; Van Eyk, J. E.; Hornburg, D. Protein Coronas on Functionalized Nanoparticles Enable Quantitative and Precise Large-Scale Deep Plasma Proteomics. <em>bioRxiv</em> <strong>2023</strong>. <a href="https://doi.org/10.1101/2023.08.28.555225">https://doi.org/10.1101/2023.08.28.555225</a>.</div>
</div>
<div id="ref-12wGIM8jB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(391) </div><div class="csl-right-inline">Blume, J. E.; Manning, W. C.; Troiano, G.; Hornburg, D.; Figa, M.; Hesterberg, L.; Platt, T. L.; Zhao, X.; Cuaresma, R. A.; Everley, P. A.; Ko, M.; Liou, H.; Mahoney, M.; Ferdosi, S.; Elgierari, E. M.; Stolarczyk, C.; Tangeysh, B.; Xia, H.; Benz, R.; Siddiqui, A.; Carr, S. A.; Ma, P.; Langer, R.; Farias, V.; Farokhzad, O. C. Rapid, Deep and Precise Profiling of the Plasma Proteome with Multi-Nanoparticle Protein Corona. <em>Nat Commun</em> <strong>2020</strong>, <em>11</em> (1), 3662. <a href="https://doi.org/10.1038/s41467-020-17033-7">https://doi.org/10.1038/s41467-020-17033-7</a>.</div>
</div>
<div id="ref-nCqm9thz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(392) </div><div class="csl-right-inline">Shen, Y.; Tolić, N.; Masselon, C.; Pasa-Tolić, L.; Camp, D. G.; Hixson, K. K.; Zhao, R.; Anderson, G. A.; Smith, R. D. Ultrasensitive Proteomics Using High-Efficiency on-Line Micro-SPE-nanoLC-nanoESI MS and MS/MS. <em>Anal Chem</em> <strong>2004</strong>, <em>76</em> (1), 144–154. <a href="https://doi.org/10.1021/ac030096q">https://doi.org/10.1021/ac030096q</a>.</div>
</div>
<div id="ref-167ADUQ9x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(393) </div><div class="csl-right-inline">Magni, F.; Van Der Burgt, Y. E. M.; Chinello, C.; Mainini, V.; Gianazza, E.; Squeo, V.; Deelder, A. M.; Kienle, M. G. Biomarkers Discovery by Peptide and Protein Profiling in Biological Fluids Based on Functionalized Magnetic Beads Purification and Mass Spectrometry. <em>Blood Transfus</em> <strong>2010</strong>, <em>8 Suppl 3</em> (Suppl 3), s92–7. <a href="https://doi.org/10.2450/2010.015s">https://doi.org/10.2450/2010.015s</a>.</div>
</div>
<div id="ref-klXQ49fD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(394) </div><div class="csl-right-inline">Peter, J. F.; Otto, A. M. Magnetic Particles as Powerful Purification Tool for High Sensitive Mass Spectrometric Screening Procedures. <em>Proteomics</em> <strong>2010</strong>, <em>10</em> (4), 628–633. <a href="https://doi.org/10.1002/pmic.200900535">https://doi.org/10.1002/pmic.200900535</a>.</div>
</div>
<div id="ref-18rJRCf42" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(395) </div><div class="csl-right-inline">Ruhaak, L. R.; Huhn, C.; Koeleman, C. A. M.; Deelder, A. M.; Wuhrer, M. Robust and High-Throughput Sample Preparation for (Semi-)Quantitative Analysis of N-Glycosylation Profiles from Plasma Samples. <em>Methods Mol Biol</em> <strong>2012</strong>, <em>893</em>, 371–385. <a href="https://doi.org/10.1007/978-1-61779-885-6_23">https://doi.org/10.1007/978-1-61779-885-6_23</a>.</div>
</div>
<div id="ref-kPeMPVtT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(396) </div><div class="csl-right-inline">Mysling, S.; Palmisano, G.; Højrup, P.; Thaysen-Andersen, M. Utilizing Ion-Pairing Hydrophilic Interaction Chromatography Solid Phase Extraction for Efficient Glycopeptide Enrichment in Glycoproteomics. <em>Anal Chem</em> <strong>2010</strong>, <em>82</em> (13), 5598–5609. <a href="https://doi.org/10.1021/ac100530w">https://doi.org/10.1021/ac100530w</a>.</div>
</div>
<div id="ref-16XWFxoc3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(397) </div><div class="csl-right-inline">Luo, Q.; Yue, G.; Valaskovic, G. A.; Gu, Y.; Wu, S.-L.; Karger, B. L. On-Line 1D and 2D Porous Layer Open Tubular/LC-ESI-MS Using 10-Microm-i.d. Poly(styrene-Divinylbenzene) Columns for Ultrasensitive Proteomic Analysis. <em>Anal Chem</em> <strong>2007</strong>, <em>79</em> (16), 6174–6181. <a href="https://doi.org/10.1021/ac070583w">https://doi.org/10.1021/ac070583w</a>.</div>
</div>
<div id="ref-us64YkZ0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(398) </div><div class="csl-right-inline">Miyazaki, S.; Morisato, K.; Ishizuka, N.; Minakuchi, H.; Shintani, Y.; Furuno, M.; Nakanishi, K. Development of a Monolithic Silica Extraction Tip for the Analysis of Proteins. <em>J Chromatogr A</em> <strong>2004</strong>, <em>1043</em> (1), 19–25. <a href="https://doi.org/10.1016/j.chroma.2004.03.025">https://doi.org/10.1016/j.chroma.2004.03.025</a>.</div>
</div>
<div id="ref-nPfahhRa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(399) </div><div class="csl-right-inline">Bladergroen, M. R.; van der Burgt, Y. E. M. Solid-Phase Extraction Strategies to Surmount Body Fluid Sample Complexity in High-Throughput Mass Spectrometry-Based Proteomics. <em>J Anal Methods Chem</em> <strong>2015</strong>, <em>2015</em>, 250131. <a href="https://doi.org/10.1155/2015/250131">https://doi.org/10.1155/2015/250131</a>.</div>
</div>
<div id="ref-TY5C1ch7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(400) </div><div class="csl-right-inline">Rappsilber, J.; Ishihama, Y.; Mann, M. Stop and Go Extraction Tips for Matrix-Assisted Laser Desorption/Ionization, Nanoelectrospray, and LC/MS Sample Pretreatment in Proteomics. <em>Anal. Chem.</em> <strong>2002</strong>, <em>75</em> (3), 663–670. <a href="https://doi.org/10.1021/ac026117i">https://doi.org/10.1021/ac026117i</a>.</div>
</div>
<div id="ref-j0ZcPOCn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(401) </div><div class="csl-right-inline">Kulak, N. A.; Pichler, G.; Paron, I.; Nagaraj, N.; Mann, M. Minimal, Encapsulated Proteomic-Sample Processing Applied to Copy-Number Estimation in Eukaryotic Cells. <em>Nat Methods</em> <strong>2014</strong>, <em>11</em> (3), 319–324. <a href="https://doi.org/10.1038/nmeth.2834">https://doi.org/10.1038/nmeth.2834</a>.</div>
</div>
<div id="ref-SWsK5zPt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(402) </div><div class="csl-right-inline">Müller, T.; Kalxdorf, M.; Longuespée, R.; Kazdal, D. N.; Stenzinger, A.; Krijgsveld, J. Automated Sample Preparation with SP3 for Low-Input Clinical Proteomics. <em>Mol Syst Biol</em> <strong>2020</strong>, <em>16</em> (1), e9111. <a href="https://doi.org/10.15252/msb.20199111">https://doi.org/10.15252/msb.20199111</a>.</div>
</div>
<div id="ref-ncGFPcxC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(403) </div><div class="csl-right-inline">Li, T.; Wang, W.; Zhao, H.; He, F.; Zhong, K.; Yuan, S.; Wang, Z. National Continuous Surveys on Internal Quality Control for HbA1c in 306 Clinical Laboratories of China from 2012 to 2016: Continual Improvement. <em>J Clin Lab Anal</em> <strong>2016</strong>, <em>31</em> (5). <a href="https://doi.org/10.1002/jcla.22099">https://doi.org/10.1002/jcla.22099</a>.</div>
</div>
<div id="ref-Bm9Z2DVx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(404) </div><div class="csl-right-inline"><a href="https://www.ncbi.nlm.nih.gov/pubmed/35777803">Abstracts of Presentations at the Association of Clinical Scientists 143</a>. <em>Ann Clin Lab Sci</em> <strong>2022</strong>, <em>52</em> (3), 511–525.</div>
</div>
<div id="ref-L1FaRtpm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(405) </div><div class="csl-right-inline">Essader, A. S.; Cargile, B. J.; Bundy, J. L.; Stephenson, J. L. A Comparison of Immobilized pH Gradient Isoelectric Focusing and Strong-Cation-Exchange Chromatography as a First Dimension in Shotgun Proteomics. <em>Proteomics</em> <strong>2005</strong>, <em>5</em> (1), 24–34. <a href="https://doi.org/10.1002/pmic.200400888">https://doi.org/10.1002/pmic.200400888</a>.</div>
</div>
<div id="ref-GjvtcEz1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(406) </div><div class="csl-right-inline">Manadas, B.; Mendes, V. M.; English, J.; Dunn, M. J. Peptide Fractionation in Proteomics Approaches. <em>Expert Rev Proteomics</em> <strong>2010</strong>, <em>7</em> (5), 655–663. <a href="https://doi.org/10.1586/epr.10.46">https://doi.org/10.1586/epr.10.46</a>.</div>
</div>
<div id="ref-1FrVJlffz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(407) </div><div class="csl-right-inline">McCalley, D. V. The Challenges of the Analysis of Basic Compounds by High Performance Liquid Chromatography: Some Possible Approaches for Improved Separations. <em>J Chromatogr A</em> <strong>2009</strong>, <em>1217</em> (6), 858–880. <a href="https://doi.org/10.1016/j.chroma.2009.11.068">https://doi.org/10.1016/j.chroma.2009.11.068</a>.</div>
</div>
<div id="ref-1HPaIsist" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(408) </div><div class="csl-right-inline">Gilar, M.; Olivova, P.; Daly, A. E.; Gebler, J. C. Two-Dimensional Separation of Peptides Using RP-RP-HPLC System with Different pH in First and Second Separation Dimensions. <em>J Sep Sci</em> <strong>2005</strong>, <em>28</em> (14), 1694–1703. <a href="https://doi.org/10.1002/jssc.200500116">https://doi.org/10.1002/jssc.200500116</a>.</div>
</div>
<div id="ref-190oCO8Ki" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(409) </div><div class="csl-right-inline">Yang, F.; Shen, Y.; Camp, D. G.; Smith, R. D. High-pH Reversed-Phase Chromatography with Fraction Concatenation for 2D Proteomic Analysis. <em>Expert Rev Proteomics</em> <strong>2012</strong>, <em>9</em> (2), 129–134. <a href="https://doi.org/10.1586/epr.12.15">https://doi.org/10.1586/epr.12.15</a>.</div>
</div>
<div id="ref-15mGcvpNQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(410) </div><div class="csl-right-inline">Simpson, R. J.; Moritz, R. L.; Nice, E. E.; Grego, B. A High-Performance Liquid Chromatography Procedure for Recovering Subnanomole Amounts of Protein from SDS-Gel Electroeluates for Gas-Phase Sequence Analysis. <em>Eur J Biochem</em> <strong>1987</strong>, <em>165</em> (1), 21–29. <a href="https://doi.org/10.1111/j.1432-1033.1987.tb11189.x">https://doi.org/10.1111/j.1432-1033.1987.tb11189.x</a>.</div>
</div>
<div id="ref-g9BMjmYA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(411) </div><div class="csl-right-inline">Boersema, P. J.; Mohammed, S.; Heck, A. J. R. Hydrophilic Interaction Liquid Chromatography (HILIC) in Proteomics. <em>Anal Bioanal Chem</em> <strong>2008</strong>, <em>391</em> (1), 151–159. <a href="https://doi.org/10.1007/s00216-008-1865-7">https://doi.org/10.1007/s00216-008-1865-7</a>.</div>
</div>
<div id="ref-KqYbI8ep" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(412) </div><div class="csl-right-inline">Mizukami, Y.; Iwamatsu, A.; Aki, T.; Kimura, M.; Nakamura, K.; Nao, T.; Okusa, T.; Matsuzaki, M.; Yoshida, K.-I.; Kobayashi, S. ERK1/2 Regulates Intracellular ATP Levels Through Alpha-Enolase Expression in Cardiomyocytes Exposed to Ischemic Hypoxia and Reoxygenation. <em>J Biol Chem</em> <strong>2004</strong>, <em>279</em> (48), 50120–50131. <a href="https://doi.org/10.1074/jbc.m402299200">https://doi.org/10.1074/jbc.m402299200</a>.</div>
</div>
<div id="ref-16k9ga0WM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(413) </div><div class="csl-right-inline">Buszewski, B.; Noga, S. Hydrophilic Interaction Liquid Chromatography (HILIC)--a Powerful Separation Technique. <em>Anal Bioanal Chem</em> <strong>2011</strong>, <em>402</em> (1), 231–247. <a href="https://doi.org/10.1007/s00216-011-5308-5">https://doi.org/10.1007/s00216-011-5308-5</a>.</div>
</div>
<div id="ref-nrDJzBh3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(414) </div><div class="csl-right-inline">Jandera, P. Stationary Phases for Hydrophilic Interaction Chromatography, Their Characterization and Implementation into Multidimensional Chromatography Concepts. <em>J Sep Sci</em> <strong>2008</strong>, <em>31</em> (9), 1421–1437. <a href="https://doi.org/10.1002/jssc.200800051">https://doi.org/10.1002/jssc.200800051</a>.</div>
</div>
<div id="ref-Kf2BNqap" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(415) </div><div class="csl-right-inline">Alpert, A. J. Electrostatic Repulsion Hydrophilic Interaction Chromatography for Isocratic Separation of Charged Solutes and Selective Isolation of Phosphopeptides. <em>Anal. Chem.</em> <strong>2007</strong>, <em>80</em> (1), 62–76. <a href="https://doi.org/10.1021/ac070997p">https://doi.org/10.1021/ac070997p</a>.</div>
</div>
<div id="ref-1HJnUznNp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(416) </div><div class="csl-right-inline">Hao, P.; Guo, T.; Li, X.; Adav, S. S.; Yang, J.; Wei, M.; Sze, S. K. Novel Application of Electrostatic Repulsion-Hydrophilic Interaction Chromatography (ERLIC) in Shotgun Proteomics: Comprehensive Profiling of Rat Kidney Proteome. <em>J. Proteome Res.</em> <strong>2010</strong>, <em>9</em> (7), 3520–3526. <a href="https://doi.org/10.1021/pr100037h">https://doi.org/10.1021/pr100037h</a>.</div>
</div>
<div id="ref-Nk7uUlzO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(417) </div><div class="csl-right-inline">Hörth, P.; Miller, C. A.; Preckel, T.; Wenz, C. Efficient Fractionation and Improved Protein Identification by Peptide OFFGEL Electrophoresis. <em>Mol Cell Proteomics</em> <strong>2006</strong>, <em>5</em> (10), 1968–1974. <a href="https://doi.org/10.1074/mcp.t600037-mcp200">https://doi.org/10.1074/mcp.t600037-mcp200</a>.</div>
</div>
<div id="ref-Qf7HZAh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(418) </div><div class="csl-right-inline">Ernoult, E.; Gamelin, E.; Guette, C. Improved Proteome Coverage by Using iTRAQ Labelling and Peptide OFFGEL Fractionation. <em>Proteome Sci</em> <strong>2008</strong>, <em>6</em>, 27. <a href="https://doi.org/10.1186/1477-5956-6-27">https://doi.org/10.1186/1477-5956-6-27</a>.</div>
</div>
<div id="ref-LSDxad5T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(419) </div><div class="csl-right-inline">Fraterman, S.; Zeiger, U.; Khurana, T. S.; Rubinstein, N. A.; Wilm, M. Combination of Peptide OFFGEL Fractionation and Label-Free Quantitation Facilitated Proteomics Profiling of Extraocular Muscle. <em>Proteomics</em> <strong>2007</strong>, <em>7</em> (18), 3404–3416. <a href="https://doi.org/10.1002/pmic.200700382">https://doi.org/10.1002/pmic.200700382</a>.</div>
</div>
<div id="ref-rNop1hIz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(420) </div><div class="csl-right-inline">François, I.; Sandra, K.; Sandra, P. Comprehensive Liquid Chromatography: Fundamental Aspects and Practical Considerations—A Review. <em>Analytica Chimica Acta</em> <strong>2009</strong>, <em>641</em> (1-2), 14–31. <a href="https://doi.org/10.1016/j.aca.2009.03.041">https://doi.org/10.1016/j.aca.2009.03.041</a>.</div>
</div>
<div id="ref-71TlBiJy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(421) </div><div class="csl-right-inline">Annesley, T. M. Ion Suppression in Mass Spectrometry. <em>Clinical Chemistry</em> <strong>2003</strong>, <em>49</em> (7), 1041–1044. <a href="https://doi.org/10.1373/49.7.1041">https://doi.org/10.1373/49.7.1041</a>.</div>
</div>
<div id="ref-pRj1odLy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(422) </div><div class="csl-right-inline">Xie, F.; Smith, R. D.; Shen, Y. Advanced Proteomic Liquid Chromatography. <em>Journal of Chromatography A</em> <strong>2012</strong>, <em>1261</em>, 78–90. <a href="https://doi.org/10.1016/j.chroma.2012.06.098">https://doi.org/10.1016/j.chroma.2012.06.098</a>.</div>
</div>
<div id="ref-T9Yv2Omz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(423) </div><div class="csl-right-inline">Zhang, X.; Fang, A.; Riley, C. P.; Wang, M.; Regnier, F. E.; Buck, C. Multi-Dimensional Liquid Chromatography in Proteomics—A Review. <em>Analytica Chimica Acta</em> <strong>2010</strong>, <em>664</em> (2), 101–113. <a href="https://doi.org/10.1016/j.aca.2010.02.001">https://doi.org/10.1016/j.aca.2010.02.001</a>.</div>
</div>
<div id="ref-WG5jq46X" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(424) </div><div class="csl-right-inline">SHI, Y.; XIANG, R.; HORVATH, C.; WILKINS, J. The Role of Liquid Chromatography in Proteomics. <em>Journal of Chromatography A</em> <strong>2004</strong>, <em>1053</em> (1-2), 27–36. <a href="https://doi.org/10.1016/j.chroma.2004.07.044">https://doi.org/10.1016/j.chroma.2004.07.044</a>.</div>
</div>
<div id="ref-BslEV2vj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(425) </div><div class="csl-right-inline">Rayleigh, L. XX. &lt;I&gt;On the Equilibrium of Liquid Conducting Masses Charged with Electricity&lt;/i&gt;. <em>The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science</em> <strong>1882</strong>, <em>14</em> (87), 184–186. <a href="https://doi.org/10.1080/14786448208628425">https://doi.org/10.1080/14786448208628425</a>.</div>
</div>
<div id="ref-hwO7hkQN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(426) </div><div class="csl-right-inline">Luo, Q.; Tang, K.; Yang, F.; Elias, A.; Shen, Y.; Moore, R. J.; Zhao, R.; Hixson, K. K.; Rossie, S. S.; Smith, R. D. More Sensitive and Quantitative Proteomic Measurements Using Very Low Flow Rate Porous Silica Monolithic LC Columns with Electrospray Ionization-Mass Spectrometry. <em>J. Proteome Res.</em> <strong>2006</strong>, <em>5</em> (5), 1091–1097. <a href="https://doi.org/10.1021/pr050424y">https://doi.org/10.1021/pr050424y</a>.</div>
</div>
<div id="ref-156fxE3m7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(427) </div><div class="csl-right-inline">van Deemter, J. J.; Zuiderweg, F. J.; Klinkenberg, A. Longitudinal Diffusion and Resistance to Mass Transfer as Causes of Nonideality in Chromatography. <em>Chemical Engineering Science</em> <strong>1956</strong>, <em>5</em> (6), 271–289. <a href="https://doi.org/10.1016/0009-2509(56)80003-1">https://doi.org/10.1016/0009-2509(56)80003-1</a>.</div>
</div>
<div id="ref-XQiQIxLc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(428) </div><div class="csl-right-inline">Midha, M. K.; Kapil, C.; Maes, M.; Baxter, D. H.; Morrone, S. R.; Prokop, T. J.; Moritz, R. L. Vacuum Insulated Probe Heated Electrospray Ionization Source Enhances Microflow Rate Chromatography Signals in the Bruker timsTOF Mass Spectrometer. <em>J Proteome Res</em> <strong>2023</strong>, <em>22</em> (7), 2525–2537. <a href="https://doi.org/10.1021/acs.jproteome.3c00305">https://doi.org/10.1021/acs.jproteome.3c00305</a>.</div>
</div>
<div id="ref-dvtPvLcY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(429) </div><div class="csl-right-inline">Kreimer, S.; Haghani, A.; Binek, A.; Hauspurg, A.; Seyedmohammad, S.; Rivas, A.; Momenzadeh, A.; Meyer, J. G.; Raedschelders, K.; Van Eyk, J. E. Parallelization with Dual-Trap Single-Column Configuration Maximizes Throughput of Proteomic Analysis. <em>Anal. Chem.</em> <strong>2022</strong>, <em>94</em> (36), 12452–12460. <a href="https://doi.org/10.1021/acs.analchem.2c02609">https://doi.org/10.1021/acs.analchem.2c02609</a>.</div>
</div>
<div id="ref-L2sT906m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(430) </div><div class="csl-right-inline">Kreimer, S.; Binek, A.; Chazarin, B.; Cho, J. H.; Haghani, A.; Hutton, A.; Marbán, E.; Mastali, M.; Meyer, J. G.; Mesquita, T.; Song, Y.; Van Eyk, J.; Parker, S. High-Throughput Single-Cell Proteomic Analysis of Organ-Derived Heterogeneous Cell Populations by Nanoflow Dual-Trap Single-Column Liquid Chromatography. <em>Anal. Chem.</em> <strong>2023</strong>, <em>95</em> (24), 9145–9150. <a href="https://doi.org/10.1021/acs.analchem.3c00213">https://doi.org/10.1021/acs.analchem.3c00213</a>.</div>
</div>
<div id="ref-1Gtuii4F4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(431) </div><div class="csl-right-inline">Pellett, J.; Lukulay, P.; Mao, Y.; Bowen, W.; Reed, R.; Ma, M.; Munger, R. C.; Dolan, J. W.; Wrisley, L.; Medwid, K.; Toltl, N. P.; Chan, C. C.; Skibic, M.; Biswas, K.; Wells, K. A.; Snyder, L. R. “Orthogonal” Separations for Reversed-Phase Liquid Chromatography. <em>Journal of Chromatography A</em> <strong>2006</strong>, <em>1101</em> (1-2), 122–135. <a href="https://doi.org/10.1016/j.chroma.2005.09.080">https://doi.org/10.1016/j.chroma.2005.09.080</a>.</div>
</div>
<div id="ref-1FnaHa1s9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(432) </div><div class="csl-right-inline">Washburn, M. P.; Wolters, D.; Yates, J. R., III. Large-Scale Analysis of the Yeast Proteome by Multidimensional Protein Identification Technology. <em>Nat Biotechnol</em> <strong>2001</strong>, <em>19</em> (3), 242–247. <a href="https://doi.org/10.1038/85686">https://doi.org/10.1038/85686</a>.</div>
</div>
<div id="ref-dl5RDy6j" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(433) </div><div class="csl-right-inline">Wang, Y.; Yang, F.; Gritsenko, M. A.; Wang, Y.; Clauss, T.; Liu, T.; Shen, Y.; Monroe, M. E.; Lopez‐Ferrer, D.; Reno, T.; Moore, R. J.; Klemke, R. L.; Camp, D. G., II; Smith, R. D. Reversed‐phase Chromatography with Multiple Fraction Concatenation Strategy for Proteome Profiling of Human MCF10A Cells. <em>Proteomics</em> <strong>2011</strong>, <em>11</em> (10), 2019–2026. <a href="https://doi.org/10.1002/pmic.201000722">https://doi.org/10.1002/pmic.201000722</a>.</div>
</div>
<div id="ref-44oI1g68" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(434) </div><div class="csl-right-inline">Kawashima, Y.; Nagai, H.; Konno, R.; Ishikawa, M.; Nakajima, D.; Sato, H.; Nakamura, R.; Furuyashiki, T.; Ohara, O. Single-Shot 10K Proteome Approach: Over 10,000 Protein Identifications by Data-Independent Acquisition-Based Single-Shot Proteomics with Ion Mobility Spectrometry. <em>J. Proteome Res.</em> <strong>2022</strong>, <em>21</em> (6), 1418–1427. <a href="https://doi.org/10.1021/acs.jproteome.2c00023">https://doi.org/10.1021/acs.jproteome.2c00023</a>.</div>
</div>
<div id="ref-11c7CVfcj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(435) </div><div class="csl-right-inline">Biemann, K.; Gapp, G.; Seibl, J. APPLICATION OF MASS SPECTROMETRY TO STRUCTURE PROBLEMS. I. AMINO ACID SEQUENCE IN PEPTIDES. <em>J. Am. Chem. Soc.</em> <strong>1959</strong>, <em>81</em> (9), 2274–2275. <a href="https://doi.org/10.1021/ja01518a069">https://doi.org/10.1021/ja01518a069</a>.</div>
</div>
<div id="ref-189tZteJw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(436) </div><div class="csl-right-inline">Nau, H.; Kelley, J. A.; Biemann, K. Determination of the Amino Acid Sequence of the C-Terminal Cyanogen Bromide Fragment of Actin by Computer-Assisted Gas Chromatography-Mass Spectrometry. <em>J. Am. Chem. Soc.</em> <strong>1973</strong>, <em>95</em> (21), 7162–7164. <a href="https://doi.org/10.1021/ja00802a048">https://doi.org/10.1021/ja00802a048</a>.</div>
</div>
<div id="ref-Tr33h2Fs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(437) </div><div class="csl-right-inline">Hunt, D. F.; Bone, W. M.; Shabanowitz, J.; Rhodes, J.; Ballard, J. M. Sequence Analysis of Oligopeptides by Secondary Ion/Collision Activated Dissociation Mass Spectrometry. <em>Anal. Chem.</em> <strong>1981</strong>, <em>53</em> (11), 1704–1706. <a href="https://doi.org/10.1021/ac00234a035">https://doi.org/10.1021/ac00234a035</a>.</div>
</div>
<div id="ref-lZwv1GwF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(438) </div><div class="csl-right-inline">Yates, J. R., III. The Revolution and Evolution of Shotgun Proteomics for Large-Scale Proteome Analysis. <em>J. Am. Chem. Soc.</em> <strong>2013</strong>, <em>135</em> (5), 1629–1640. <a href="https://doi.org/10.1021/ja3094313">https://doi.org/10.1021/ja3094313</a>.</div>
</div>
<div id="ref-YdLSljVE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(439) </div><div class="csl-right-inline"><em>The Nobel Prize in Chemistry 2002</em>. NobelPrize.org. <a href="https://www.nobelprize.org/prizes/chemistry/2002/summary/">https://www.nobelprize.org/prizes/chemistry/2002/summary/</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-zfiYj2co" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(440) </div><div class="csl-right-inline">Karas, Michael.; Bachmann, Doris.; Hillenkamp, Franz. Influence of the Wavelength in High-Irradiance Ultraviolet Laser Desorption Mass Spectrometry of Organic Molecules. <em>Analytical Chemistry</em> <strong>1985</strong>, <em>57</em> (14), 2935–2939. <a href="https://doi.org/10.1021/ac00291a042">https://doi.org/10.1021/ac00291a042</a>.</div>
</div>
<div id="ref-tnQHCFV7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(441) </div><div class="csl-right-inline">Karas, Michael.; Hillenkamp, Franz. Laser Desorption Ionization of Proteins with Molecular Masses Exceeding 10,000 Daltons. <em>Anal. Chem.</em> <strong>1988</strong>, <em>60</em> (20), 2299–2301. <a href="https://doi.org/10.1021/ac00171a028">https://doi.org/10.1021/ac00171a028</a>.</div>
</div>
<div id="ref-1HBJs2tAB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(442) </div><div class="csl-right-inline"><em>The Scientist :: Nobel Prize controversy</em>. <a href="https://web.archive.org/web/20070517202246/http://cmbi.bjmu.edu.cn/news/0212/55.htm">https://web.archive.org/web/20070517202246/http://cmbi.bjmu.edu.cn/news/0212/55.htm</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-5ZJdaya1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(443) </div><div class="csl-right-inline">Yang, H.; Wan, D.; Song, F.; Liu, Z.; Liu, S. Α-Cyano-4-Hydroxycinnamic Acid, Sinapinic Acid, and Ferulic Acid as Matrices and Alkylating Agents for Matrix-Assisted Laser Desorption/Ionization Time-of-Flight Mass Spectrometric Analysis of Cysteine-Containing Peptides. <em>Rapid Commun Mass Spectrom</em> <strong>2013</strong>, <em>27</em> (12), 1410–1412. <a href="https://doi.org/10.1002/rcm.6587">https://doi.org/10.1002/rcm.6587</a>.</div>
</div>
<div id="ref-wGHAVD9r" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(444) </div><div class="csl-right-inline">Dreisewerd, K. The Desorption Process in MALDI. <em>Chem. Rev.</em> <strong>2003</strong>, <em>103</em> (2), 395–426. <a href="https://doi.org/10.1021/cr010375i">https://doi.org/10.1021/cr010375i</a>.</div>
</div>
<div id="ref-DEnAe0bs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(445) </div><div class="csl-right-inline">Karas, Michael.; Bahr, Ute.; Strupat, Kerstin.; Hillenkamp, Franz.; Tsarbopoulos, Anthony.; Pramanik, B. N. Matrix Dependence of Metastable Fragmentation of Glycoproteins in MALDI TOF Mass Spectrometry. <em>Anal. Chem.</em> <strong>1995</strong>, <em>67</em> (3), 675–679. <a href="https://doi.org/10.1021/ac00099a029">https://doi.org/10.1021/ac00099a029</a>.</div>
</div>
<div id="ref-kL1RzcZ6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(446) </div><div class="csl-right-inline">Buchberger, A. R.; DeLaney, K.; Johnson, J.; Li, L. Mass Spectrometry Imaging: A Review of Emerging Advancements and Future Insights. <em>Anal Chem</em> <strong>2017</strong>, <em>90</em> (1), 240–265. <a href="https://doi.org/10.1021/acs.analchem.7b04733">https://doi.org/10.1021/acs.analchem.7b04733</a>.</div>
</div>
<div id="ref-11vyuKWyw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(447) </div><div class="csl-right-inline">Meyer, J. G.; A Komives, E. Charge State Coalescence During Electrospray Ionization Improves Peptide Identification by Tandem Mass Spectrometry. <em>J Am Soc Mass Spectrom</em> <strong>2012</strong>, <em>23</em> (8), 1390–1399. <a href="https://doi.org/10.1007/s13361-012-0404-0">https://doi.org/10.1007/s13361-012-0404-0</a>.</div>
</div>
<div id="ref-4mpHldYP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(448) </div><div class="csl-right-inline">Kebarle, P.; Verkerk, U. H. Electrospray: From Ions in Solution to Ions in the Gas Phase, What We Know Now. <em>Mass Spectrom Rev</em> <strong>2009</strong>, <em>28</em> (6), 898–917. <a href="https://doi.org/10.1002/mas.20247">https://doi.org/10.1002/mas.20247</a>.</div>
</div>
<div id="ref-93IPP3bH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(449) </div><div class="csl-right-inline">Konermann, L.; Ahadi, E.; Rodriguez, A. D.; Vahidi, S. Unraveling the Mechanism of Electrospray Ionization. <em>Anal Chem</em> <strong>2012</strong>, <em>85</em> (1), 2–9. <a href="https://doi.org/10.1021/ac302789c">https://doi.org/10.1021/ac302789c</a>.</div>
</div>
<div id="ref-VhI3IC9H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(450) </div><div class="csl-right-inline">Griffiths, J. A Brief History of Mass Spectrometry. <em>Anal. Chem.</em> <strong>2008</strong>, <em>80</em> (15), 5678–5683. <a href="https://doi.org/10.1021/ac8013065">https://doi.org/10.1021/ac8013065</a>.</div>
</div>
<div id="ref-lKt4oMBO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(451) </div><div class="csl-right-inline">Klampfl, C. W.; Himmelsbach, M. Direct Ionization Methods in Mass Spectrometry: An Overview. <em>Analytica Chimica Acta</em> <strong>2015</strong>, <em>890</em>, 44–59. <a href="https://doi.org/10.1016/j.aca.2015.07.012">https://doi.org/10.1016/j.aca.2015.07.012</a>.</div>
</div>
<div id="ref-1B2HdTk7M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(452) </div><div class="csl-right-inline">Awad, H.; Khamis, M. M.; El-Aneed, A. Mass Spectrometry, Review of the Basics: Ionization. <em>Applied Spectroscopy Reviews</em> <strong>2014</strong>, <em>50</em> (2), 158–175. <a href="https://doi.org/10.1080/05704928.2014.954046">https://doi.org/10.1080/05704928.2014.954046</a>.</div>
</div>
<div id="ref-18qirHBwQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(453) </div><div class="csl-right-inline">Li, C.; Chu, S.; Tan, S.; Yin, X.; Jiang, Y.; Dai, X.; Gong, X.; Fang, X.; Tian, D. Towards Higher Sensitivity of Mass Spectrometry: A Perspective From the Mass Analyzers. <em>Front. Chem.</em> <strong>2021</strong>, <em>9</em>. <a href="https://doi.org/10.3389/fchem.2021.813359">https://doi.org/10.3389/fchem.2021.813359</a>.</div>
</div>
<div id="ref-5vv7ID3o" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(454) </div><div class="csl-right-inline">Eliuk, S.; Makarov, A. Evolution of Orbitrap Mass Spectrometry Instrumentation. <em>Annual Rev. Anal. Chem.</em> <strong>2015</strong>, <em>8</em> (1), 61–80. <a href="https://doi.org/10.1146/annurev-anchem-071114-040325">https://doi.org/10.1146/annurev-anchem-071114-040325</a>.</div>
</div>
<div id="ref-MooVAdBf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(455) </div><div class="csl-right-inline">Glish, G. L.; Burinsky, D. J. Hybrid Mass Spectrometers for Tandem Mass Spectrometry. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2008</strong>, <em>19</em> (2), 161–172. <a href="https://doi.org/10.1016/j.jasms.2007.11.013">https://doi.org/10.1016/j.jasms.2007.11.013</a>.</div>
</div>
<div id="ref-Uaum1m63" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(456) </div><div class="csl-right-inline">Hecht, E. S.; Scigelova, M.; Eliuk, S.; Makarov, A. Fundamentals and Advances of Orbitrap Mass Spectrometry. <em>Encyclopedia of Analytical Chemistry</em>, 2019, 1–40. <a href="https://doi.org/10.1002/9780470027318.a9309.pub2">https://doi.org/10.1002/9780470027318.a9309.pub2</a>.</div>
</div>
<div id="ref-svXqfJ4D" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(457) </div><div class="csl-right-inline">Zubarev, R. A.; Makarov, A. Orbitrap Mass Spectrometry. <em>Anal. Chem.</em> <strong>2013</strong>, <em>85</em> (11), 5288–5296. <a href="https://doi.org/10.1021/ac4001223">https://doi.org/10.1021/ac4001223</a>.</div>
</div>
<div id="ref-erGVtl2k" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(458) </div><div class="csl-right-inline">Tsybin, Y. O.; Nagornov, K. O.; Kozhinov, A. N. Advanced Fundamentals in Fourier Transform Mass Spectrometry. In <em>Fundamentals and Applications of Fourier Transform Mass Spectrometry</em>; Elsevier, 2019; pp 113–132. <a href="https://doi.org/10.1016/b978-0-12-814013-0.00005-3">https://doi.org/10.1016/b978-0-12-814013-0.00005-3</a>.</div>
</div>
<div id="ref-EmIJDW71" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(459) </div><div class="csl-right-inline">Wells, J. M.; McLuckey, S. A. Collision-Induced Dissociation (CID) of Peptides and Proteins. <em>Methods Enzymol</em> <strong>2005</strong>, <em>402</em>, 148–185. <a href="https://doi.org/10.1016/s0076-6879(05)02005-7">https://doi.org/10.1016/s0076-6879(05)02005-7</a>.</div>
</div>
<div id="ref-17hxbWgXF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(460) </div><div class="csl-right-inline">Thomson, B. A.; Douglas, D. J.; Corr, J. J.; Hager, J. W.; Jolliffe, C. L. Improved Collisionally Activated Dissociation Efficiency and Mass Resolution on a Triple Quadrupole Mass Spectrometer System. <em>Anal. Chem.</em> <strong>1995</strong>, <em>67</em> (10), 1696–1704. <a href="https://doi.org/10.1021/ac00106a008">https://doi.org/10.1021/ac00106a008</a>.</div>
</div>
<div id="ref-mZfV6P3P" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(461) </div><div class="csl-right-inline">Lange, V.; Picotti, P.; Domon, B.; Aebersold, R. Selected Reaction Monitoring for Quantitative Proteomics: A Tutorial. <em>Molecular Systems Biology</em> <strong>2008</strong>, <em>4</em> (1). <a href="https://doi.org/10.1038/msb.2008.61">https://doi.org/10.1038/msb.2008.61</a>.</div>
</div>
<div id="ref-1ENeUykUA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(462) </div><div class="csl-right-inline">Johnson, J. V.; Yost, R. A.; Kelley, P. E.; Bradford, D. C. Tandem-in-Space and Tandem-in-Time Mass Spectrometry: Triple Quadrupoles and Quadrupole Ion Traps. <em>Anal. Chem.</em> <strong>1990</strong>, <em>62</em> (20), 2162–2172. <a href="https://doi.org/10.1021/ac00219a003">https://doi.org/10.1021/ac00219a003</a>.</div>
</div>
<div id="ref-1GjxHekcM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(463) </div><div class="csl-right-inline">Sleno, L.; Volmer, D. A. Ion Activation Methods for Tandem Mass Spectrometry. <em>J. Mass Spectrom.</em> <strong>2004</strong>, <em>39</em> (10), 1091–1112. <a href="https://doi.org/10.1002/jms.703">https://doi.org/10.1002/jms.703</a>.</div>
</div>
<div id="ref-1B9P7OPqR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(464) </div><div class="csl-right-inline">Anderson, L.; Hunter, C. L. Quantitative Mass Spectrometric Multiple Reaction Monitoring Assays for Major Plasma Proteins. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2006</strong>, <em>5</em> (4), 573–588. <a href="https://doi.org/10.1074/mcp.m500331-mcp200">https://doi.org/10.1074/mcp.m500331-mcp200</a>.</div>
</div>
<div id="ref-9yFTjbpI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(465) </div><div class="csl-right-inline">Hager, J. W. Recent Trends in Mass Spectrometer Development. <em>Anal Bioanal Chem</em> <strong>2003</strong>, <em>378</em> (4), 845–850. <a href="https://doi.org/10.1007/s00216-003-2287-1">https://doi.org/10.1007/s00216-003-2287-1</a>.</div>
</div>
<div id="ref-Ddpn8u0F" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(466) </div><div class="csl-right-inline">Yost, R. A.; Enke, C. G. Selected Ion Fragmentation with a Tandem Quadrupole Mass Spectrometer. <em>J. Am. Chem. Soc.</em> <strong>1978</strong>, <em>100</em> (7), 2274–2275. <a href="https://doi.org/10.1021/ja00475a072">https://doi.org/10.1021/ja00475a072</a>.</div>
</div>
<div id="ref-1AOQTDs57" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(467) </div><div class="csl-right-inline">Hopfgartner, G.; Varesio, E.; Tschäppät, V.; Grivet, C.; Bourgogne, E.; Leuthold, L. A. Triple Quadrupole Linear Ion Trap Mass Spectrometer for the Analysis of Small Molecules and Macromolecules. <em>J. Mass Spectrom.</em> <strong>2004</strong>, <em>39</em> (8), 845–855. <a href="https://doi.org/10.1002/jms.659">https://doi.org/10.1002/jms.659</a>.</div>
</div>
<div id="ref-11skpaXK6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(468) </div><div class="csl-right-inline">Ma, S.; Zhu, M. Recent Advances in Applications of Liquid Chromatography–Tandem Mass Spectrometry to the Analysis of Reactive Drug Metabolites. <em>Chemico-Biological Interactions</em> <strong>2009</strong>, <em>179</em> (1), 25–37. <a href="https://doi.org/10.1016/j.cbi.2008.09.014">https://doi.org/10.1016/j.cbi.2008.09.014</a>.</div>
</div>
<div id="ref-BaGpN1HP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(469) </div><div class="csl-right-inline">Hernández, F.; Cervera, M. I.; Portolés, T.; Beltrán, J.; Pitarch, E. The Role of GC-MS/MS with Triple Quadrupole in Pesticide Residue Analysis in Food and the Environment. <em>Anal. Methods</em> <strong>2013</strong>, <em>5</em> (21), 5875. <a href="https://doi.org/10.1039/c3ay41104d">https://doi.org/10.1039/c3ay41104d</a>.</div>
</div>
<div id="ref-1ETD5TZZ8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(470) </div><div class="csl-right-inline">van der Gugten, J. G. Tandem Mass Spectrometry in the Clinical Laboratory: A Tutorial Overview. <em>Clinical Mass Spectrometry</em> <strong>2020</strong>, <em>15</em>, 36–43. <a href="https://doi.org/10.1016/j.clinms.2019.09.002">https://doi.org/10.1016/j.clinms.2019.09.002</a>.</div>
</div>
<div id="ref-EeH1Mu0v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(471) </div><div class="csl-right-inline">Anonymous. Proceedings of the American Physical Society. <em>Phys. Rev.</em> <strong>1946</strong>, <em>69</em> (11-12), 674–674. <a href="https://doi.org/10.1103/physrev.69.674.2">https://doi.org/10.1103/physrev.69.674.2</a>.</div>
</div>
<div id="ref-kFLq1VDE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(472) </div><div class="csl-right-inline">Wolff, M. M.; Stephens, W. E. A Pulsed Mass Spectrometer with Time Dispersion. <em>Review of Scientific Instruments</em> <strong>1953</strong>, <em>24</em> (8), 616–617. <a href="https://doi.org/10.1063/1.1770801">https://doi.org/10.1063/1.1770801</a>.</div>
</div>
<div id="ref-15YzDfo72" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(473) </div><div class="csl-right-inline">Wiley, W. C.; McLaren, I. H. Time-of-Flight Mass Spectrometer with Improved Resolution. <em>Review of Scientific Instruments</em> <strong>1955</strong>, <em>26</em> (12), 1150–1157. <a href="https://doi.org/10.1063/1.1715212">https://doi.org/10.1063/1.1715212</a>.</div>
</div>
<div id="ref-eJibibTn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(474) </div><div class="csl-right-inline">Sandow, J. J.; Infusini, G.; Dagley, L. F.; Larsen, R.; Webb, A. I. Simplified High-Throughput Methods for Deep Proteome Analysis on the timsTOF Pro, 2019. <a href="https://doi.org/10.1101/657908">https://doi.org/10.1101/657908</a>.</div>
</div>
<div id="ref-KJYZ12ho" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(475) </div><div class="csl-right-inline">Aballo, T. J.; Roberts, D. S.; Melby, J. A.; Buck, K. M.; Brown, K. A.; Ge, Y. Ultrafast and Reproducible Proteomics from Small Amounts of Heart Tissue Enabled by Azo and timsTOF Pro. <em>J. Proteome Res.</em> <strong>2021</strong>, <em>20</em> (8), 4203–4211. <a href="https://doi.org/10.1021/acs.jproteome.1c00446">https://doi.org/10.1021/acs.jproteome.1c00446</a>.</div>
</div>
<div id="ref-cXsqAa1Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(476) </div><div class="csl-right-inline">Pan, S.; Zhang, H.; Rush, J.; Eng, J.; Zhang, N.; Patterson, D.; Comb, M. J.; Aebersold, R. High Throughput Proteome Screening for Biomarker Detection. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2005</strong>, <em>4</em> (2), 182–190. <a href="https://doi.org/10.1074/mcp.m400161-mcp200">https://doi.org/10.1074/mcp.m400161-mcp200</a>.</div>
</div>
<div id="ref-OeUnezUK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(477) </div><div class="csl-right-inline">Wörner, T. P.; Shamorkina, T. M.; Snijder, J.; Heck, A. J. R. Mass Spectrometry-Based Structural Virology. <em>Anal. Chem.</em> <strong>2020</strong>, <em>93</em> (1), 620–640. <a href="https://doi.org/10.1021/acs.analchem.0c04339">https://doi.org/10.1021/acs.analchem.0c04339</a>.</div>
</div>
<div id="ref-2oFwzqFP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(478) </div><div class="csl-right-inline">Snijder, J.; Heck, A. J. R. Analytical Approaches for Size and Mass Analysis of Large Protein Assemblies. <em>Annual Rev. Anal. Chem.</em> <strong>2014</strong>, <em>7</em> (1), 43–64. <a href="https://doi.org/10.1146/annurev-anchem-071213-020015">https://doi.org/10.1146/annurev-anchem-071213-020015</a>.</div>
</div>
<div id="ref-3O301V9H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(479) </div><div class="csl-right-inline">Comby-Zerbino, C.; Dagany, X.; Chirot, F.; Dugourd, P.; Antoine, R. The Emergence of Mass Spectrometry for Characterizing Nanomaterials. Atomically Precise Nanoclusters and Beyond. <em>Mater. Adv.</em> <strong>2021</strong>, <em>2</em> (15), 4896–4913. <a href="https://doi.org/10.1039/d1ma00261a">https://doi.org/10.1039/d1ma00261a</a>.</div>
</div>
<div id="ref-1CdqYFQjZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(480) </div><div class="csl-right-inline">Makarov, A. Electrostatic Axially Harmonic Orbital Trapping:  A High-Performance Technique of Mass Analysis. <em>Anal. Chem.</em> <strong>2000</strong>, <em>72</em> (6), 1156–1162. <a href="https://doi.org/10.1021/ac991131p">https://doi.org/10.1021/ac991131p</a>.</div>
</div>
<div id="ref-1GyycMiIQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(481) </div><div class="csl-right-inline">Makarov, A.; Scigelova, M. Coupling Liquid Chromatography to Orbitrap Mass Spectrometry. <em>Journal of Chromatography A</em> <strong>2010</strong>, <em>1217</em> (25), 3938–3945. <a href="https://doi.org/10.1016/j.chroma.2010.02.022">https://doi.org/10.1016/j.chroma.2010.02.022</a>.</div>
</div>
<div id="ref-KRgCNqM9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(482) </div><div class="csl-right-inline">Crutchfield, C. A.; Thomas, S. N.; Sokoll, L. J.; Chan, D. W. Advances in Mass Spectrometry-Based Clinical Biomarker Discovery. <em>Clin Proteom</em> <strong>2016</strong>, <em>13</em> (1). <a href="https://doi.org/10.1186/s12014-015-9102-9">https://doi.org/10.1186/s12014-015-9102-9</a>.</div>
</div>
<div id="ref-184JFw6LL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(483) </div><div class="csl-right-inline">Meyer, J. G.; Mukkamalla, S.; Steen, H.; Nesvizhskii, A. I.; Gibson, B. W.; Schilling, B. PIQED: Automated Identification and Quantification of Protein Modifications from DIA-MS Data. <em>Nat Methods</em> <strong>2017</strong>, <em>14</em> (7), 646–647. <a href="https://doi.org/10.1038/nmeth.4334">https://doi.org/10.1038/nmeth.4334</a>.</div>
</div>
<div id="ref-4x97ohIq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(484) </div><div class="csl-right-inline">Momenzadeh, A.; Jiang, Y.; Kreimer, S.; Teigen, L. E.; Zepeda, C. S.; Haghani, A.; Mastali, M.; Song, Y.; Hutton, A.; Parker, S. J.; Van Eyk, J. E.; Sundberg, C. W.; Meyer, J. G. A Complete Workflow for High Throughput Human Single Skeletal Muscle Fiber Proteomics. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2023</strong>, <em>34</em> (9), 1858–1867. <a href="https://doi.org/10.1021/jasms.3c00072">https://doi.org/10.1021/jasms.3c00072</a>.</div>
</div>
<div id="ref-1B5HMmjdK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(485) </div><div class="csl-right-inline">Smits, A. H.; Vermeulen, M. Characterizing Protein–Protein Interactions Using Mass Spectrometry: Challenges and Opportunities. <em>Trends in Biotechnology</em> <strong>2016</strong>, <em>34</em> (10), 825–834. <a href="https://doi.org/10.1016/j.tibtech.2016.02.014">https://doi.org/10.1016/j.tibtech.2016.02.014</a>.</div>
</div>
<div id="ref-BtlT8wee" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(486) </div><div class="csl-right-inline">Sun, J.; Liu, X. R.; Li, S.; He, P.; Li, W.; Gross, M. L. Nanoparticles and Photochemistry for Native-Like Transmembrane Protein Footprinting. <em>Nat Commun</em> <strong>2021</strong>, <em>12</em> (1). <a href="https://doi.org/10.1038/s41467-021-27588-8">https://doi.org/10.1038/s41467-021-27588-8</a>.</div>
</div>
<div id="ref-1GNpqhnTj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(487) </div><div class="csl-right-inline">Sun, J.; Li, S.; Li, W.; Gross, M. L. Carbocation Footprinting of Soluble and Transmembrane Proteins. <em>Anal. Chem.</em> <strong>2021</strong>, <em>93</em> (39), 13101–13105. <a href="https://doi.org/10.1021/acs.analchem.1c03274">https://doi.org/10.1021/acs.analchem.1c03274</a>.</div>
</div>
<div id="ref-wknKHPF0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(488) </div><div class="csl-right-inline">Comisarow, M. B.; Marshall, A. G. Fourier Transform Ion Cyclotron Resonance Spectroscopy. <em>Chemical Physics Letters</em> <strong>1974</strong>, <em>25</em> (2), 282–283. <a href="https://doi.org/10.1016/0009-2614(74)89137-2">https://doi.org/10.1016/0009-2614(74)89137-2</a>.</div>
</div>
<div id="ref-HLEPjTl5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(489) </div><div class="csl-right-inline">Cho, Y.; Ahmed, A.; Islam, A.; Kim, S. Developments in FT‐ICR MS Instrumentation, Ionization Techniques, and Data Interpretation Methods for Petroleomics. <em>Mass Spectrometry Reviews</em> <strong>2014</strong>, <em>34</em> (2), 248–263. <a href="https://doi.org/10.1002/mas.21438">https://doi.org/10.1002/mas.21438</a>.</div>
</div>
<div id="ref-64lSPaQQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(490) </div><div class="csl-right-inline">Jiang, B.; Kuang, B. Y.; Liang, Y.; Zhang, J.; Huang, X. H. H.; Xu, C.; Yu, J. Z.; Shi, Q. Molecular Composition of Urban Organic Aerosols on Clear and Hazy Days in Beijing: A Comparative Study Using FT-ICR MS. <em>Environ. Chem.</em> <strong>2016</strong>, <em>13</em> (5), 888. <a href="https://doi.org/10.1071/en15230">https://doi.org/10.1071/en15230</a>.</div>
</div>
<div id="ref-Gw2v7q21" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(491) </div><div class="csl-right-inline">Bogdanov, B.; Smith, R. D. Proteomics by FTICR Mass Spectrometry: Top down and Bottom Up. <em>Mass Spectrometry Reviews</em> <strong>2004</strong>, <em>24</em> (2), 168–200. <a href="https://doi.org/10.1002/mas.20015">https://doi.org/10.1002/mas.20015</a>.</div>
</div>
<div id="ref-2Qw9ppKa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(492) </div><div class="csl-right-inline">Motoyama, A.; Yates, J. R., III. Multidimensional LC Separations in Shotgun Proteomics. <em>Anal. Chem.</em> <strong>2008</strong>, <em>80</em> (19), 7187–7193. <a href="https://doi.org/10.1021/ac8013669">https://doi.org/10.1021/ac8013669</a>.</div>
</div>
<div id="ref-ovFVXSFD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(493) </div><div class="csl-right-inline">Chen, G.; Pramanik, B. N. Application of LC/MS to Proteomics Studies: Current Status and Future Prospects. <em>Drug Discovery Today</em> <strong>2009</strong>, <em>14</em> (9-10), 465–471. <a href="https://doi.org/10.1016/j.drudis.2009.02.007">https://doi.org/10.1016/j.drudis.2009.02.007</a>.</div>
</div>
<div id="ref-ODpFT5rK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(494) </div><div class="csl-right-inline">May, J. C.; McLean, J. A. Ion Mobility-Mass Spectrometry: Time-Dispersive Instrumentation. <em>Anal. Chem.</em> <strong>2015</strong>, <em>87</em> (3), 1422–1436. <a href="https://doi.org/10.1021/ac504720m">https://doi.org/10.1021/ac504720m</a>.</div>
</div>
<div id="ref-rfF30WBX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(495) </div><div class="csl-right-inline">Dodds, J. N.; Baker, E. S. Ion Mobility Spectrometry: Fundamental Concepts, Instrumentation, Applications, and the Road Ahead. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2019</strong>, <em>30</em> (11), 2185–2195. <a href="https://doi.org/10.1007/s13361-019-02288-2">https://doi.org/10.1007/s13361-019-02288-2</a>.</div>
</div>
<div id="ref-D1WlSrbO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(496) </div><div class="csl-right-inline">Meier, F.; Beck, S.; Grassl, N.; Lubeck, M.; Park, M. A.; Raether, O.; Mann, M. Parallel Accumulation–Serial Fragmentation (PASEF): Multiplying Sequencing Speed and Sensitivity by Synchronized Scans in a Trapped Ion Mobility Device. <em>J. Proteome Res.</em> <strong>2015</strong>, <em>14</em> (12), 5378–5387. <a href="https://doi.org/10.1021/acs.jproteome.5b00932">https://doi.org/10.1021/acs.jproteome.5b00932</a>.</div>
</div>
<div id="ref-2aWp1Ns2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(497) </div><div class="csl-right-inline">Ridgeway, M. E.; Bleiholder, C.; Mann, M.; Park, M. A. Trends in Trapped Ion Mobility – Mass Spectrometry Instrumentation. <em>TrAC Trends in Analytical Chemistry</em> <strong>2019</strong>, <em>116</em>, 324–331. <a href="https://doi.org/10.1016/j.trac.2019.03.030">https://doi.org/10.1016/j.trac.2019.03.030</a>.</div>
</div>
<div id="ref-Gl9lNiar" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(498) </div><div class="csl-right-inline">Giles, K.; Ujma, J.; Wildgoose, J.; Pringle, S.; Richardson, K.; Langridge, D.; Green, M. A Cyclic Ion Mobility-Mass Spectrometry System. <em>Anal. Chem.</em> <strong>2019</strong>, <em>91</em> (13), 8564–8573. <a href="https://doi.org/10.1021/acs.analchem.9b01838">https://doi.org/10.1021/acs.analchem.9b01838</a>.</div>
</div>
<div id="ref-NcqTCZMl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(499) </div><div class="csl-right-inline">Odenkirk, M. T.; Baker, E. S. Utilizing Drift Tube Ion Mobility Spectrometry for the Evaluation of Metabolites and Xenobiotics. In <em>Methods in Molecular Biology</em>; Springer US, 2019; pp 35–54. <a href="https://doi.org/10.1007/978-1-0716-0030-6_2">https://doi.org/10.1007/978-1-0716-0030-6_2</a>.</div>
</div>
<div id="ref-6pMoYbxP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(500) </div><div class="csl-right-inline">Jiang, W.; Chung, N. A.; May, J. C.; McLean, J. A.; Robinson, R. A. S. Ion Mobility–Mass Spectrometry. <em>Encyclopedia of Analytical Chemistry</em>, 2019, 1–34. <a href="https://doi.org/10.1002/9780470027318.a9292.pub2">https://doi.org/10.1002/9780470027318.a9292.pub2</a>.</div>
</div>
<div id="ref-KCpIS4fP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(501) </div><div class="csl-right-inline">Swearingen, K. E.; Moritz, R. L. High-Field Asymmetric Waveform Ion Mobility Spectrometry for Mass Spectrometry-Based Proteomics. <em>Expert Review of Proteomics</em> <strong>2012</strong>, <em>9</em> (5), 505–517. <a href="https://doi.org/10.1586/epr.12.50">https://doi.org/10.1586/epr.12.50</a>.</div>
</div>
<div id="ref-VdxvCWch" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(502) </div><div class="csl-right-inline">Hale, O. J.; Illes-Toth, E.; Mize, T. H.; Cooper, H. J. High-Field Asymmetric Waveform Ion Mobility Spectrometry and Native Mass Spectrometry: Analysis of Intact Protein Assemblies and Protein Complexes. <em>Anal. Chem.</em> <strong>2020</strong>, <em>92</em> (10), 6811–6816. <a href="https://doi.org/10.1021/acs.analchem.0c00649">https://doi.org/10.1021/acs.analchem.0c00649</a>.</div>
</div>
<div id="ref-E9GVLceZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(503) </div><div class="csl-right-inline">Hebert, A. S.; Prasad, S.; Belford, M. W.; Bailey, D. J.; McAlister, G. C.; Abbatiello, S. E.; Huguet, R.; Wouters, E. R.; Dunyach, J.-J.; Brademan, D. R.; Westphall, M. S.; Coon, J. J. Comprehensive Single-Shot Proteomics with FAIMS on a Hybrid Orbitrap Mass Spectrometer. <em>Anal. Chem.</em> <strong>2018</strong>, <em>90</em> (15), 9529–9537. <a href="https://doi.org/10.1021/acs.analchem.8b02233">https://doi.org/10.1021/acs.analchem.8b02233</a>.</div>
</div>
<div id="ref-EUzlbRbi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(504) </div><div class="csl-right-inline">GUEVREMONT, R. High-Field Asymmetric Waveform Ion Mobility Spectrometry: A New Tool for Mass Spectrometry. <em>Journal of Chromatography A</em> <strong>2004</strong>, <em>1058</em> (1-2), 3–19. <a href="https://doi.org/10.1016/j.chroma.2004.08.119">https://doi.org/10.1016/j.chroma.2004.08.119</a>.</div>
</div>
<div id="ref-CzOaytxB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(505) </div><div class="csl-right-inline">Ridgeway, M. E.; Lubeck, M.; Jordens, J.; Mann, M.; Park, M. A. Trapped Ion Mobility Spectrometry: A Short Review. <em>International Journal of Mass Spectrometry</em> <strong>2018</strong>, <em>425</em>, 22–35. <a href="https://doi.org/10.1016/j.ijms.2018.01.006">https://doi.org/10.1016/j.ijms.2018.01.006</a>.</div>
</div>
<div id="ref-1FmNlvpSt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(506) </div><div class="csl-right-inline">Meier, F.; Park, M. A.; Mann, M. Trapped Ion Mobility Spectrometry and Parallel Accumulation–Serial Fragmentation in Proteomics. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2021</strong>, <em>20</em>, 100138. <a href="https://doi.org/10.1016/j.mcpro.2021.100138">https://doi.org/10.1016/j.mcpro.2021.100138</a>.</div>
</div>
<div id="ref-KgcabL7g" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(507) </div><div class="csl-right-inline">Tolmachev, A. V.; Webb, I. K.; Ibrahim, Y. M.; Garimella, S. V. B.; Zhang, X.; Anderson, G. A.; Smith, R. D. Characterization of Ion Dynamics in Structures for Lossless Ion Manipulations. <em>Anal. Chem.</em> <strong>2014</strong>, <em>86</em> (18), 9162–9168. <a href="https://doi.org/10.1021/ac502054p">https://doi.org/10.1021/ac502054p</a>.</div>
</div>
<div id="ref-XOhnpKc9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(508) </div><div class="csl-right-inline">Deng, L.; Webb, I. K.; Garimella, S. V. B.; Hamid, A. M.; Zheng, X.; Norheim, R. V.; Prost, S. A.; Anderson, G. A.; Sandoval, J. A.; Baker, E. S.; Ibrahim, Y. M.; Smith, R. D. Serpentine Ultralong Path with Extended Routing (SUPER) High Resolution Traveling Wave Ion Mobility-MS Using Structures for Lossless Ion Manipulations. <em>Anal. Chem.</em> <strong>2017</strong>, <em>89</em> (8), 4628–4634. <a href="https://doi.org/10.1021/acs.analchem.7b00185">https://doi.org/10.1021/acs.analchem.7b00185</a>.</div>
</div>
<div id="ref-GwEAWw2z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(509) </div><div class="csl-right-inline">Arndt, J. R.; Wormwood Moser, K. L.; Van Aken, G.; Doyle, R. M.; Talamantes, T.; DeBord, D.; Maxon, L.; Stafford, G.; Fjeldsted, J.; Miller, B.; Sherman, M. High-Resolution Ion-Mobility-Enabled Peptide Mapping for High-Throughput Critical Quality Attribute Monitoring. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2021</strong>, <em>32</em> (8), 2019–2032. <a href="https://doi.org/10.1021/jasms.0c00434">https://doi.org/10.1021/jasms.0c00434</a>.</div>
</div>
<div id="ref-qAVrmvwM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(510) </div><div class="csl-right-inline">Shuken, S. R. An Introduction to Mass Spectrometry-Based Proteomics. <em>J. Proteome Res.</em> <strong>2023</strong>, <em>22</em> (7), 2151–2171. <a href="https://doi.org/10.1021/acs.jproteome.2c00838">https://doi.org/10.1021/acs.jproteome.2c00838</a>.</div>
</div>
<div id="ref-V3WGOS1f" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(511) </div><div class="csl-right-inline">Zhang, Y.; Fonslow, B. R.; Shan, B.; Baek, M.-C.; Yates, J. R., III. Protein Analysis by Shotgun/Bottom-up Proteomics. <em>Chem. Rev.</em> <strong>2013</strong>, <em>113</em> (4), 2343–2394. <a href="https://doi.org/10.1021/cr3003533">https://doi.org/10.1021/cr3003533</a>.</div>
</div>
<div id="ref-1DSKc93ie" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(512) </div><div class="csl-right-inline">Yu, L.; Xiong, Y.-M.; Polfer, N. C. Periodicity of Monoisotopic Mass Isomers and Isobars in Proteomics. <em>Anal. Chem.</em> <strong>2011</strong>, <em>83</em> (20), 8019–8023. <a href="https://doi.org/10.1021/ac201624t">https://doi.org/10.1021/ac201624t</a>.</div>
</div>
<div id="ref-2MR1Pihs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(513) </div><div class="csl-right-inline">Hunt, D. F.; Yates, J. R., 3rd; Shabanowitz, J.; Winston, S.; Hauer, C. R. Protein Sequencing by Tandem Mass Spectrometry. <em>Proc. Natl. Acad. Sci. U.S.A.</em> <strong>1986</strong>, <em>83</em> (17), 6233–6237. <a href="https://doi.org/10.1073/pnas.83.17.6233">https://doi.org/10.1073/pnas.83.17.6233</a>.</div>
</div>
<div id="ref-zCNyqvc9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(514) </div><div class="csl-right-inline">Ivanov, M. V.; Bubis, J. A.; Gorshkov, V.; Tarasova, I. A.; Levitsky, L. I.; Lobas, A. A.; Solovyeva, E. M.; Pridatchenko, M. L.; Kjeldsen, F.; Gorshkov, M. V. DirectMS1: MS/MS-Free Identification of 1000 Proteins of Cellular Proteomes in 5 Minutes. <em>Anal. Chem.</em> <strong>2020</strong>, <em>92</em> (6), 4326–4333. <a href="https://doi.org/10.1021/acs.analchem.9b05095">https://doi.org/10.1021/acs.analchem.9b05095</a>.</div>
</div>
<div id="ref-175VGosTe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(515) </div><div class="csl-right-inline">Jaitly, N.; Monroe, M. E.; Petyuk, V. A.; Clauss, T. R. W.; Adkins, J. N.; Smith, R. D. Robust Algorithm for Alignment of Liquid Chromatography−Mass Spectrometry Analyses in an Accurate Mass and Time Tag Data Analysis Pipeline. <em>Anal. Chem.</em> <strong>2006</strong>, <em>78</em> (21), 7397–7409. <a href="https://doi.org/10.1021/ac052197p">https://doi.org/10.1021/ac052197p</a>.</div>
</div>
<div id="ref-ylZpW7Vi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(516) </div><div class="csl-right-inline">Meier, F.; Geyer, P. E.; Virreira Winter, S.; Cox, J.; Mann, M. BoxCar Acquisition Method Enables Single-Shot Proteomics at a Depth of 10,000 Proteins in 100 Minutes. <em>Nat Methods</em> <strong>2018</strong>, <em>15</em> (6), 440–448. <a href="https://doi.org/10.1038/s41592-018-0003-5">https://doi.org/10.1038/s41592-018-0003-5</a>.</div>
</div>
<div id="ref-mZdxW2jk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(517) </div><div class="csl-right-inline">Roepstorff, P.; Fohlman, J. Letter to the Editors. <em>Biol. Mass Spectrom.</em> <strong>1984</strong>, <em>11</em> (11), 601–601. <a href="https://doi.org/10.1002/bms.1200111109">https://doi.org/10.1002/bms.1200111109</a>.</div>
</div>
<div id="ref-ivBdKZgo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(518) </div><div class="csl-right-inline">McCloskey, J. A. <em>Mass spectrometry</em>; Methods in enzymology; Academic Press: San Diego, 1990.</div>
</div>
<div id="ref-12yIRpFcQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(519) </div><div class="csl-right-inline">Goeringer, D. E.; McLuckey, S. A. Evolution of Ion Internal Energy During Collisional Excitation in the Paul Ion Trap: A Stochastic Approach. <em>The Journal of Chemical Physics</em> <strong>1996</strong>, <em>104</em> (6), 2214–2221. <a href="https://doi.org/10.1063/1.471812">https://doi.org/10.1063/1.471812</a>.</div>
</div>
<div id="ref-1DIcKpBMl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(520) </div><div class="csl-right-inline">Olsen, J. V.; Macek, B.; Lange, O.; Makarov, A.; Horning, S.; Mann, M. Higher-Energy C-Trap Dissociation for Peptide Modification Analysis. <em>Nat Methods</em> <strong>2007</strong>, <em>4</em> (9), 709–712. <a href="https://doi.org/10.1038/nmeth1060">https://doi.org/10.1038/nmeth1060</a>.</div>
</div>
<div id="ref-CrVLd8Zt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(521) </div><div class="csl-right-inline">Hohmann, L. J.; Eng, J. K.; Gemmill, A.; Klimek, J.; Vitek, O.; Reid, G. E.; Martin, D. B. Quantification of the Compositional Information Provided by Immonium Ions on a Quadrupole-Time-of-Flight Mass Spectrometer. <em>Anal. Chem.</em> <strong>2008</strong>, <em>80</em> (14), 5596–5606. <a href="https://doi.org/10.1021/ac8006076">https://doi.org/10.1021/ac8006076</a>.</div>
</div>
<div id="ref-Lx0XDSc1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(522) </div><div class="csl-right-inline">Nolting, D.; Malek, R.; Makarov, A. Ion Traps in Modern Mass Spectrometry. <em>Mass Spectrometry Reviews</em> <strong>2017</strong>, <em>38</em> (2), 150–168. <a href="https://doi.org/10.1002/mas.21549">https://doi.org/10.1002/mas.21549</a>.</div>
</div>
<div id="ref-8fRsJhJf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(523) </div><div class="csl-right-inline">Kovacic, I.; Rand, R.; Mohamed Sah, S. Mathieu's Equation and Its Generalizations: Overview of Stability Charts and Their Features. <em>Applied Mechanics Reviews</em> <strong>2018</strong>, <em>70</em> (2). <a href="https://doi.org/10.1115/1.4039144">https://doi.org/10.1115/1.4039144</a>.</div>
</div>
<div id="ref-1DjnN7u2e" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(524) </div><div class="csl-right-inline">March, R. E. Ion Trap Mass Spectrometers. In <em>Encyclopedia of Spectroscopy and Spectrometry</em>; Elsevier, 1999; pp 1000–1009. <a href="https://doi.org/10.1006/rwsp.2000.0143">https://doi.org/10.1006/rwsp.2000.0143</a>.</div>
</div>
<div id="ref-16Wowt3CO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(525) </div><div class="csl-right-inline">Palumbo, A. M.; Tepe, J. J.; Reid, G. E. Mechanistic Insights into the Multistage Gas-Phase Fragmentation Behavior of Phosphoserine- and Phosphothreonine-Containing Peptides. <em>J. Proteome Res.</em> <strong>2008</strong>, <em>7</em> (2), 771–779. <a href="https://doi.org/10.1021/pr0705136">https://doi.org/10.1021/pr0705136</a>.</div>
</div>
<div id="ref-Oadeotpz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(526) </div><div class="csl-right-inline">Schroeder, M. J.; Shabanowitz, J.; Schwartz, J. C.; Hunt, D. F.; Coon, J. J. A Neutral Loss Activation Method for Improved Phosphopeptide Sequence Analysis by Quadrupole Ion Trap Mass Spectrometry. <em>Anal. Chem.</em> <strong>2004</strong>, <em>76</em> (13), 3590–3598. <a href="https://doi.org/10.1021/ac0497104">https://doi.org/10.1021/ac0497104</a>.</div>
</div>
<div id="ref-Ud7cg5Ze" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(527) </div><div class="csl-right-inline">Wysocki, V. H.; Tsaprailis, G.; Smith, L. L.; Breci, L. A. Mobile and Localized Protons: A Framework for Understanding Peptide Dissociation. <em>J Mass Spectrom</em> <strong>2000</strong>, <em>35</em> (12), 1399–1406. <a href="https://doi.org/10.1002/1096-9888(200012)35:12&lt;1399::aid-jms86&gt;3.0.co;2-r">https://doi.org/10.1002/1096-9888(200012)35:12&lt;1399::aid-jms86&gt;3.0.co;2-r</a>.</div>
</div>
<div id="ref-VOcnSdY9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(528) </div><div class="csl-right-inline">Coon, J. J. Collisions or Electrons? Protein Sequence Analysis in the 21st Century. <em>Anal. Chem.</em> <strong>2009</strong>, <em>81</em> (9), 3208–3215. <a href="https://doi.org/10.1021/ac802330b">https://doi.org/10.1021/ac802330b</a>.</div>
</div>
<div id="ref-oNkSkkq8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(529) </div><div class="csl-right-inline">Udeshi, N. D.; Compton, P. D.; Shabanowitz, J.; Hunt, D. F.; Rose, K. L. Methods for Analyzing Peptides and Proteins on a Chromatographic Timescale by Electron-Transfer Dissociation Mass Spectrometry. <em>Nat Protoc</em> <strong>2008</strong>, <em>3</em> (11), 1709–1717. <a href="https://doi.org/10.1038/nprot.2008.159">https://doi.org/10.1038/nprot.2008.159</a>.</div>
</div>
<div id="ref-XHvsh4uo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(530) </div><div class="csl-right-inline">Sarbu, M.; Ghiulai, R. M.; Zamfir, A. D. Recent Developments and Applications of Electron Transfer Dissociation Mass Spectrometry in Proteomics. <em>Amino Acids</em> <strong>2014</strong>, <em>46</em> (7), 1625–1634. <a href="https://doi.org/10.1007/s00726-014-1726-y">https://doi.org/10.1007/s00726-014-1726-y</a>.</div>
</div>
<div id="ref-qWde4x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(531) </div><div class="csl-right-inline">Zubarev, R. A.; Kelleher, N. L.; McLafferty, F. W. Electron Capture Dissociation of Multiply Charged Protein Cations. A Nonergodic Process. <em>J. Am. Chem. Soc.</em> <strong>1998</strong>, <em>120</em> (13), 3265–3266. <a href="https://doi.org/10.1021/ja973478k">https://doi.org/10.1021/ja973478k</a>.</div>
</div>
<div id="ref-GH22tfkO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(532) </div><div class="csl-right-inline">Syka, J. E. P.; Coon, J. J.; Schroeder, M. J.; Shabanowitz, J.; Hunt, D. F. Peptide and Protein Sequence Analysis by Electron Transfer Dissociation Mass Spectrometry. <em>Proc. Natl. Acad. Sci. U.S.A.</em> <strong>2004</strong>, <em>101</em> (26), 9528–9533. <a href="https://doi.org/10.1073/pnas.0402700101">https://doi.org/10.1073/pnas.0402700101</a>.</div>
</div>
<div id="ref-OVsIcoNm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(533) </div><div class="csl-right-inline">Chen, X.; Hao, C. Where Does the Electron Go? Electron Distribution and Reactivity of Peptide Cation Radicals Formed by Electron Transfer in the Gas Phase. <em>J. Am. Chem. Soc.</em> <strong>2008</strong>, <em>130</em> (27), 8818–8833. <a href="https://doi.org/10.1021/ja8019005">https://doi.org/10.1021/ja8019005</a>.</div>
</div>
<div id="ref-xhXJM8vz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(534) </div><div class="csl-right-inline">Syrstad, E. A.; Turecček, F. Toward a General Mechanism of Electron Capture Dissociation. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2005</strong>, <em>16</em> (2), 208–224. <a href="https://doi.org/10.1016/j.jasms.2004.11.001">https://doi.org/10.1016/j.jasms.2004.11.001</a>.</div>
</div>
<div id="ref-rqc6ScJr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(535) </div><div class="csl-right-inline">Riley, N. M.; Coon, J. J. The Role of Electron Transfer Dissociation in Modern Proteomics. <em>Anal. Chem.</em> <strong>2017</strong>, <em>90</em> (1), 40–64. <a href="https://doi.org/10.1021/acs.analchem.7b04810">https://doi.org/10.1021/acs.analchem.7b04810</a>.</div>
</div>
<div id="ref-HNURrSN7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(536) </div><div class="csl-right-inline">Lermyte, F.; Valkenborg, D.; Loo, J. A.; Sobott, F. Radical Solutions: Principles and Application of Electron‐based Dissociation in Mass Spectrometry‐based Analysis of Protein Structure. <em>Mass Spectrometry Reviews</em> <strong>2018</strong>, <em>37</em> (6), 750–771. <a href="https://doi.org/10.1002/mas.21560">https://doi.org/10.1002/mas.21560</a>.</div>
</div>
<div id="ref-W7m8UNum" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(537) </div><div class="csl-right-inline">Ge, Y.; Lawhorn, B. G.; ElNaggar, M.; Strauss, E.; Park, J.-H.; Begley, T. P.; McLafferty, F. W. Top Down Characterization of Larger Proteins (45 kDa) by Electron Capture Dissociation Mass Spectrometry. <em>Journal of the American Chemical Society</em> <strong>2002</strong>, <em>124</em> (4), 672–678. https://doi.org/<a href="https://doi.org/10.1021/ja011335z">https://doi.org/10.1021/ja011335z</a>.</div>
</div>
<div id="ref-BG48TsdY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(538) </div><div class="csl-right-inline">Brunner, A. M.; Lössl, P.; Liu, F.; Huguet, R.; Mullen, C.; Yamashita, M.; Zabrouskov, V.; Makarov, A.; Altelaar, A. F. M.; Heck, A. J. R. Benchmarking Multiple Fragmentation Methods on an Orbitrap Fusion for Top-down Phospho-Proteoform Characterization. <em>Analytical Chemistry</em> <strong>2015</strong>, <em>87</em> (8), 4152–4158. https://doi.org/<a href="https://doi.org/10.1021/acs.analchem.5b00162">https://doi.org/10.1021/acs.analchem.5b00162</a>.</div>
</div>
<div id="ref-ekbZzMCZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(539) </div><div class="csl-right-inline">Schürch, S. Characterization of Nucleic Acids by Tandem Mass Spectrometry ‐ The Second Decade (2004–2013): From DNA to RNA and Modified Sequences. <em>Mass Spectrometry Reviews</em> <strong>2014</strong>, <em>35</em> (4), 483–523. <a href="https://doi.org/10.1002/mas.21442">https://doi.org/10.1002/mas.21442</a>.</div>
</div>
<div id="ref-16lajpfJg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(540) </div><div class="csl-right-inline">Peters-Clarke, T. M.; Quan, Q.; Brademan, D. R.; Hebert, A. S.; Westphall, M. S.; Coon, J. J. Ribonucleic Acid Sequence Characterization by Negative Electron Transfer Dissociation Mass Spectrometry. <em>Anal. Chem.</em> <strong>2020</strong>, <em>92</em> (6), 4436–4444. <a href="https://doi.org/10.1021/acs.analchem.9b05388">https://doi.org/10.1021/acs.analchem.9b05388</a>.</div>
</div>
<div id="ref-kcIkQ9U2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(541) </div><div class="csl-right-inline">Pimentel, E. B.; Peters-Clarke, T. M.; Coon, J. J.; Martell, J. D. DNA-Scaffolded Synergistic Catalysis. <em>J. Am. Chem. Soc.</em> <strong>2021</strong>, <em>143</em> (50), 21402–21409. <a href="https://doi.org/10.1021/jacs.1c10757">https://doi.org/10.1021/jacs.1c10757</a>.</div>
</div>
<div id="ref-dRBXpy4q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(542) </div><div class="csl-right-inline">Taucher, M.; Breuker, K. Characterization of Modified RNA by Top‐Down Mass Spectrometry. <em>Angewandte Chemie</em> <strong>2012</strong>, <em>124</em> (45), 11451–11454. <a href="https://doi.org/10.1002/ange.201206232">https://doi.org/10.1002/ange.201206232</a>.</div>
</div>
<div id="ref-aQmCzpPg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(543) </div><div class="csl-right-inline">Crittenden, C. M.; Lanzillotti, M. B.; Chen, B. Top-Down Mass Spectrometry of Synthetic Single Guide Ribonucleic Acids Enabled by Facile Sample Clean-Up. <em>Anal. Chem.</em> <strong>2023</strong>, <em>95</em> (6), 3180–3186. <a href="https://doi.org/10.1021/acs.analchem.2c03030">https://doi.org/10.1021/acs.analchem.2c03030</a>.</div>
</div>
<div id="ref-o0OmLG1e" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(544) </div><div class="csl-right-inline">Merrifield, J. L.; Pimentel, E. B.; Peters-Clarke, T. M.; Nesbitt, D. J.; Coon, J. J.; Martell, J. D. DNA-Compatible Copper/TEMPO Oxidation for DNA-Encoded Libraries. <em>Bioconjugate Chem.</em> <strong>2023</strong>, <em>34</em> (8), 1380–1386. <a href="https://doi.org/10.1021/acs.bioconjchem.3c00254">https://doi.org/10.1021/acs.bioconjchem.3c00254</a>.</div>
</div>
<div id="ref-UqS9JT46" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(545) </div><div class="csl-right-inline">Shaw, J. B.; Malhan, N.; Vasil’ev, Y. V.; Lopez, N. I.; Makarov, A.; Beckman, J. S.; Voinov, V. G. Sequencing Grade Tandem Mass Spectrometry for Top–Down Proteomics Using Hybrid Electron Capture Dissociation Methods in a Benchtop Orbitrap Mass Spectrometer. <em>Anal. Chem.</em> <strong>2018</strong>, <em>90</em> (18), 10819–10827. <a href="https://doi.org/10.1021/acs.analchem.8b01901">https://doi.org/10.1021/acs.analchem.8b01901</a>.</div>
</div>
<div id="ref-1Gs2JKxdE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(546) </div><div class="csl-right-inline">Fort, K. L.; Cramer, C. N.; Voinov, V. G.; Vasil’ev, Y. V.; Lopez, N. I.; Beckman, J. S.; Heck, A. J. R. Exploring ECD on a Benchtop Q Exactive Orbitrap Mass Spectrometer. <em>J. Proteome Res.</em> <strong>2017</strong>, <em>17</em> (2), 926–933. <a href="https://doi.org/10.1021/acs.jproteome.7b00622">https://doi.org/10.1021/acs.jproteome.7b00622</a>.</div>
</div>
<div id="ref-Vu5rNLFc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(547) </div><div class="csl-right-inline">Baba, T.; Ryumin, P.; Duchoslav, E.; Chen, K.; Chelur, A.; Loyd, B.; Chernushevich, I. Dissociation of Biomolecules by an Intense Low-Energy Electron Beam in a High Sensitivity Time-of-Flight Mass Spectrometer. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2021</strong>, <em>32</em> (8), 1964–1975. <a href="https://doi.org/10.1021/jasms.0c00425">https://doi.org/10.1021/jasms.0c00425</a>.</div>
</div>
<div id="ref-eSdFYuAj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(548) </div><div class="csl-right-inline">Good, D. M.; Wirtala, M.; McAlister, G. C.; Coon, J. J. Performance Characteristics of Electron Transfer Dissociation Mass Spectrometry. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2007</strong>, <em>6</em> (11), 1942–1951. <a href="https://doi.org/10.1074/mcp.m700073-mcp200">https://doi.org/10.1074/mcp.m700073-mcp200</a>.</div>
</div>
<div id="ref-qyjSSKs7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(549) </div><div class="csl-right-inline">Pitteri, S. J.; Chrisman, P. A.; McLuckey, S. A. Electron-Transfer Ion/Ion Reactions of Doubly Protonated Peptides:  Effect of Elevated Bath Gas Temperature. <em>Anal. Chem.</em> <strong>2005</strong>, <em>77</em> (17), 5662–5669. <a href="https://doi.org/10.1021/ac050666h">https://doi.org/10.1021/ac050666h</a>.</div>
</div>
<div id="ref-c8aJxeOj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(550) </div><div class="csl-right-inline">Swaney, D. L.; McAlister, G. C.; Wirtala, M.; Schwartz, J. C.; Syka, J. E. P.; Coon, J. J. Supplemental Activation Method for High-Efficiency Electron-Transfer Dissociation of Doubly Protonated Peptide Precursors. <em>Anal. Chem.</em> <strong>2006</strong>, <em>79</em> (2), 477–485. <a href="https://doi.org/10.1021/ac061457f">https://doi.org/10.1021/ac061457f</a>.</div>
</div>
<div id="ref-TEQyYNF9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(551) </div><div class="csl-right-inline">Campbell, J. L.; Hager, J. W.; Le Blanc, J. C. Y. On Performing Simultaneous Electron Transfer Dissociation and Collision-Induced Dissociation on Multiply Protonated Peptides in a Linear Ion Trap. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2009</strong>, <em>20</em> (9), 1672–1683. <a href="https://doi.org/10.1016/j.jasms.2009.05.009">https://doi.org/10.1016/j.jasms.2009.05.009</a>.</div>
</div>
<div id="ref-bsAhaObV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(552) </div><div class="csl-right-inline">Frese, C. K.; Altelaar, A. F. M.; van den Toorn, H.; Nolting, D.; Griep-Raming, J.; Heck, A. J. R.; Mohammed, S. Toward Full Peptide Sequence Coverage by Dual Fragmentation Combining Electron-Transfer and Higher-Energy Collision Dissociation Tandem Mass Spectrometry. <em>Anal. Chem.</em> <strong>2012</strong>, <em>84</em> (22), 9668–9673. <a href="https://doi.org/10.1021/ac3025366">https://doi.org/10.1021/ac3025366</a>.</div>
</div>
<div id="ref-oRY0m4xv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(553) </div><div class="csl-right-inline">Mikhailov, V. A.; Cooper, H. J. Activated Ion Electron Capture Dissociation (AI ECD) of Proteins: Synchronization of Infrared and Electron Irradiation with Ion Magnetron Motion. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2009</strong>, <em>20</em> (5), 763–771. <a href="https://doi.org/10.1016/j.jasms.2008.12.015">https://doi.org/10.1016/j.jasms.2008.12.015</a>.</div>
</div>
<div id="ref-F58LowfZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(554) </div><div class="csl-right-inline">Horn, D. M.; Ge, Y.; McLafferty, F. W. Activated Ion Electron Capture Dissociation for Mass Spectral Sequencing of Larger (42 kDa) Proteins. <em>Anal. Chem.</em> <strong>2000</strong>, <em>72</em> (20), 4778–4784. <a href="https://doi.org/10.1021/ac000494i">https://doi.org/10.1021/ac000494i</a>.</div>
</div>
<div id="ref-3mhY8AZ1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(555) </div><div class="csl-right-inline">Riley, N. M.; Westphall, M. S.; Coon, J. J. Activated Ion Electron Transfer Dissociation for Improved Fragmentation of Intact Proteins. <em>Anal. Chem.</em> <strong>2015</strong>, <em>87</em> (14), 7109–7116. <a href="https://doi.org/10.1021/acs.analchem.5b00881">https://doi.org/10.1021/acs.analchem.5b00881</a>.</div>
</div>
<div id="ref-pXrOdleW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(556) </div><div class="csl-right-inline">Ledvina, Aaron R.; McAlister, Graeme C.; Gardner, Myles W.; Smith, Suncerae I.; Madsen, James A.; Schwartz, Jae C.; Stafford, George C., Jr.; Syka, John E.  P.; Brodbelt, Jennifer S.; Coon, Joshua J. Infrared Photoactivation Reduces Peptide Folding and Hydrogen‐Atom Migration Following ETD Tandem Mass Spectrometry. <em>Angew Chem Int Ed</em> <strong>2009</strong>, <em>48</em> (45), 8526–8528. <a href="https://doi.org/10.1002/anie.200903557">https://doi.org/10.1002/anie.200903557</a>.</div>
</div>
<div id="ref-1HZ2A5iku" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(557) </div><div class="csl-right-inline">Peters-Clarke, T. M.; Schauer, K. L.; Riley, N. M.; Lodge, J. M.; Westphall, M. S.; Coon, J. J. Optical Fiber-Enabled Photoactivation of Peptides and Proteins. <em>Anal. Chem.</em> <strong>2020</strong>, <em>92</em> (18), 12363–12370. <a href="https://doi.org/10.1021/acs.analchem.0c02087">https://doi.org/10.1021/acs.analchem.0c02087</a>.</div>
</div>
<div id="ref-PWUQDl3j" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(558) </div><div class="csl-right-inline">Peters-Clarke, T. M.; Riley, N. M.; Westphall, M. S.; Coon, J. J. Practical Effects of Intramolecular Hydrogen Rearrangement in Electron Transfer Dissociation-Based Proteomics. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2021</strong>, <em>33</em> (1), 100–110. <a href="https://doi.org/10.1021/jasms.1c00284">https://doi.org/10.1021/jasms.1c00284</a>.</div>
</div>
<div id="ref-1CD0nWWIP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(559) </div><div class="csl-right-inline">Cannon, J. R.; Holden, D. D.; Brodbelt, J. S. Hybridizing Ultraviolet Photodissociation with Electron Transfer Dissociation for Intact Protein Characterization. <em>Anal. Chem.</em> <strong>2014</strong>, <em>86</em> (21), 10970–10977. <a href="https://doi.org/10.1021/ac5036082">https://doi.org/10.1021/ac5036082</a>.</div>
</div>
<div id="ref-a9481fQj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(560) </div><div class="csl-right-inline">Kjeldsen, F.; Haselmann, K. F.; Budnik, B. A.; Jensen, F.; Zubarev, R. A. Dissociative Capture of Hot (3–13 eV) Electrons by Polypeptide Polycations: An Efficient Process Accompanied by Secondary Fragmentation. <em>Chemical Physics Letters</em> <strong>2002</strong>, <em>356</em> (3), 201–206. <a href="https://doi.org/10.1016/S0009-2614(02)00149-5">https://doi.org/10.1016/S0009-2614(02)00149-5</a>.</div>
</div>
<div id="ref-3Q6WdA4H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(561) </div><div class="csl-right-inline">Fung, Y. M. E.; Adams, C. M.; Zubarev, R. A. Electron Ionization Dissociation of Singly and Multiply Charged Peptides. <em>J. Am. Chem. Soc.</em> <strong>2009</strong>, <em>131</em> (29), 9977–9985. <a href="https://doi.org/10.1021/ja8087407">https://doi.org/10.1021/ja8087407</a>.</div>
</div>
<div id="ref-1zm3p02B" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(562) </div><div class="csl-right-inline">Riley, N. M.; Westphall, M. S.; Hebert, A. S.; Coon, J. J. Implementation of Activated Ion Electron Transfer Dissociation on a Quadrupole-Orbitrap-Linear Ion Trap Hybrid Mass Spectrometer. <em>Anal. Chem.</em> <strong>2017</strong>, <em>89</em> (12), 6358–6366. <a href="https://doi.org/10.1021/acs.analchem.7b00213">https://doi.org/10.1021/acs.analchem.7b00213</a>.</div>
</div>
<div id="ref-M8hC7nxG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(563) </div><div class="csl-right-inline">Macias, L. A.; Santos, I. C.; Brodbelt, J. S. Ion Activation Methods for Peptides and Proteins. <em>Anal. Chem.</em> <strong>2019</strong>, <em>92</em> (1), 227–251. <a href="https://doi.org/10.1021/acs.analchem.9b04859">https://doi.org/10.1021/acs.analchem.9b04859</a>.</div>
</div>
<div id="ref-pP0K4nse" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(564) </div><div class="csl-right-inline">Brodbelt, J. S. Photodissociation Mass Spectrometry: New Tools for Characterization of Biological Molecules. <em>Chem. Soc. Rev.</em> <strong>2014</strong>, <em>43</em> (8), 2757–2783. <a href="https://doi.org/10.1039/c3cs60444f">https://doi.org/10.1039/c3cs60444f</a>.</div>
</div>
<div id="ref-YqEIuvCO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(565) </div><div class="csl-right-inline">Brodbelt, J. S.; Morrison, L. J.; Santos, I. Ultraviolet Photodissociation Mass Spectrometry for Analysis of Biological Molecules. <em>Chem. Rev.</em> <strong>2019</strong>, <em>120</em> (7), 3328–3380. <a href="https://doi.org/10.1021/acs.chemrev.9b00440">https://doi.org/10.1021/acs.chemrev.9b00440</a>.</div>
</div>
<div id="ref-zA5rBLH4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(566) </div><div class="csl-right-inline">Maitre, P.; Scuderi, D.; Corinti, D.; Chiavarino, B.; Crestoni, M. E.; Fornarini, S. Applications of Infrared Multiple Photon Dissociation (IRMPD) to the Detection of Posttranslational Modifications. <em>Chem. Rev.</em> <strong>2019</strong>, <em>120</em> (7), 3261–3295. <a href="https://doi.org/10.1021/acs.chemrev.9b00395">https://doi.org/10.1021/acs.chemrev.9b00395</a>.</div>
</div>
<div id="ref-XXMVxnKl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(567) </div><div class="csl-right-inline">Crowe, M. C.; Brodbelt, J. S. Infrared Multiphoton Dissociation (IRMPD) and Collisionally Activated Dissociationof Peptides in a Quadrupole Ion Trapwith Selective IRMPD of Phosphopeptides. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2004</strong>, <em>15</em> (11), 1581–1592. <a href="https://doi.org/10.1016/j.jasms.2004.07.016">https://doi.org/10.1016/j.jasms.2004.07.016</a>.</div>
</div>
<div id="ref-UC6BHa63" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(568) </div><div class="csl-right-inline">Thompson, M. S.; Cui, W.; Reilly, J. P. Fragmentation of Singly Charged Peptide Ions by Photodissociation at &lt;i&gt;λ&lt;/i&gt;=157 Nm. <em>Angew Chem Int Ed</em> <strong>2004</strong>, <em>43</em> (36), 4791–4794. <a href="https://doi.org/10.1002/anie.200460788">https://doi.org/10.1002/anie.200460788</a>.</div>
</div>
<div id="ref-145wzrCEm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(569) </div><div class="csl-right-inline">Madsen, J. A.; Boutz, D. R.; Brodbelt, J. S. Ultrafast Ultraviolet Photodissociation at 193 Nm and Its Applicability to Proteomic Workflows. <em>J. Proteome Res.</em> <strong>2010</strong>, <em>9</em> (8), 4205–4214. <a href="https://doi.org/10.1021/pr100515x">https://doi.org/10.1021/pr100515x</a>.</div>
</div>
<div id="ref-UoKcvT55" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(570) </div><div class="csl-right-inline">Fornelli, L.; Srzentić, K.; Toby, T. K.; Doubleday, P. F.; Huguet, R.; Mullen, C.; Melani, R. D.; dos Santos Seckler, H.; DeHart, C. J.; Weisbrod, C. R.; Durbin, K. R.; Greer, J. B.; Early, B. P.; Fellers, R. T.; Zabrouskov, V.; Thomas, P. M.; Compton, P. D.; Kelleher, N. L. Thorough Performance Evaluation of 213 Nm Ultraviolet Photodissociation for Top-down Proteomics. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2020</strong>, <em>19</em> (2), 405–420. <a href="https://doi.org/10.1074/mcp.tir119.001638">https://doi.org/10.1074/mcp.tir119.001638</a>.</div>
</div>
<div id="ref-168MaNzTm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(571) </div><div class="csl-right-inline">Yeh, G. K.; Sun, Q.; Meneses, C.; Julian, R. R. Rapid Peptide Fragmentation Without Electrons, Collisions, Infrared Radiation, or Native Chromophores. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2009</strong>, <em>20</em> (3), 385–393. <a href="https://doi.org/10.1016/j.jasms.2008.10.019">https://doi.org/10.1016/j.jasms.2008.10.019</a>.</div>
</div>
<div id="ref-UrNcqFO2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(572) </div><div class="csl-right-inline">Park, S.; Ahn, W.; Lee, S.; Han, S. Y.; Rhee, B. K.; Oh, H. B. Ultraviolet Photodissociation at 266 Nm of Phosphorylated Peptide Cations. <em>Rapid Comm Mass Spectrometry</em> <strong>2009</strong>, <em>23</em> (23), 3609–3620. <a href="https://doi.org/10.1002/rcm.4184">https://doi.org/10.1002/rcm.4184</a>.</div>
</div>
<div id="ref-cZOW0Gem" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(573) </div><div class="csl-right-inline">Wilson, J. J.; Brodbelt, J. S. MS/MS Simplification by 355 Nm Ultraviolet Photodissociation of Chromophore-Derivatized Peptides in a Quadrupole Ion Trap. <em>Anal. Chem.</em> <strong>2007</strong>, <em>79</em> (20), 7883–7892. <a href="https://doi.org/10.1021/ac071241t">https://doi.org/10.1021/ac071241t</a>.</div>
</div>
<div id="ref-1ACL74GmA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(574) </div><div class="csl-right-inline">Ly, T.; Julian, R. R. Residue-Specific Radical-Directed Dissociation of Whole Proteins in the Gas Phase. <em>J. Am. Chem. Soc.</em> <strong>2007</strong>, <em>130</em> (1), 351–358. <a href="https://doi.org/10.1021/ja076535a">https://doi.org/10.1021/ja076535a</a>.</div>
</div>
<div id="ref-18sf2lBbl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(575) </div><div class="csl-right-inline">Ly, T.; Julian, Ryan R. Ultraviolet Photodissociation: Developments Towards Applications for Mass‐Spectrometry‐Based Proteomics. <em>Angew Chem Int Ed</em> <strong>2009</strong>, <em>48</em> (39), 7130–7137. <a href="https://doi.org/10.1002/anie.200900613">https://doi.org/10.1002/anie.200900613</a>.</div>
</div>
<div id="ref-5SkV76Q3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(576) </div><div class="csl-right-inline">R. Julian, R. The Mechanism Behind Top-Down UVPD Experiments: Making Sense of Apparent Contradictions. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2017</strong>, <em>28</em> (9), 1823–1826. <a href="https://doi.org/10.1007/s13361-017-1721-0">https://doi.org/10.1007/s13361-017-1721-0</a>.</div>
</div>
<div id="ref-10TkrAfIp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(577) </div><div class="csl-right-inline">Brodbelt, J. S. Deciphering Combinatorial Post-Translational Modifications by Top-down Mass Spectrometry. <em>Current Opinion in Chemical Biology</em> <strong>2022</strong>, <em>70</em>, 102180. <a href="https://doi.org/10.1016/j.cbpa.2022.102180">https://doi.org/10.1016/j.cbpa.2022.102180</a>.</div>
</div>
<div id="ref-KWENOLlt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(578) </div><div class="csl-right-inline">Shaw, J. B.; Li, W.; Holden, D. D.; Zhang, Y.; Griep-Raming, J.; Fellers, R. T.; Early, B. P.; Thomas, P. M.; Kelleher, N. L.; Brodbelt, J. S. Complete Protein Characterization Using Top-Down Mass Spectrometry and Ultraviolet Photodissociation. <em>J. Am. Chem. Soc.</em> <strong>2013</strong>, <em>135</em> (34), 12646–12651. <a href="https://doi.org/10.1021/ja4029654">https://doi.org/10.1021/ja4029654</a>.</div>
</div>
<div id="ref-1A6jk9UQ3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(579) </div><div class="csl-right-inline">Kolbowski, L.; Belsom, A.; Rappsilber, J. Ultraviolet Photodissociation of Tryptic Peptide Backbones at 213 Nm. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2020</strong>, <em>31</em> (6), 1282–1290. <a href="https://doi.org/10.1021/jasms.0c00106">https://doi.org/10.1021/jasms.0c00106</a>.</div>
</div>
<div id="ref-zJsJYkGj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(580) </div><div class="csl-right-inline">Lanzillotti, M.; Brodbelt, J. S. Comparison of Top-Down Protein Fragmentation Induced by 213 and 193 Nm UVPD. <em>J. Am. Soc. Mass Spectrom.</em> <strong>2023</strong>, <em>34</em> (2), 279–285. <a href="https://doi.org/10.1021/jasms.2c00288">https://doi.org/10.1021/jasms.2c00288</a>.</div>
</div>
<div id="ref-JpKe7ymh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(581) </div><div class="csl-right-inline">Davis, M. T.; Stahl, D. C.; Swiderek, K. M.; Lee, T. D. Capillary Liquid Chromatography/Mass Spectrometry for Peptide and Protein Characterization. <em>Methods</em> <strong>1994</strong>, <em>6</em> (3), 304–314. <a href="https://doi.org/10.1006/meth.1994.1031">https://doi.org/10.1006/meth.1994.1031</a>.</div>
</div>
<div id="ref-byV7q4rP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(582) </div><div class="csl-right-inline">Yates, J. R.; Eng, J. K.; McCormack, A. L.; Schieltz, David. Method to Correlate Tandem Mass Spectra of Modified Peptides to Amino Acid Sequences in the Protein Database. <em>Anal. Chem.</em> <strong>1995</strong>, <em>67</em> (8), 1426–1436. <a href="https://doi.org/10.1021/ac00104a020">https://doi.org/10.1021/ac00104a020</a>.</div>
</div>
<div id="ref-1BU9KEnwO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(583) </div><div class="csl-right-inline">Stahl, D. C.; Swiderek, K. M.; Davis, M. T.; Lee, T. D. Data-Controlled Automation of Liquid Chromatography/Tandem Mass Spectrometry Analysis of Peptide Mixtures. <em>J Am Soc Mass Spectrom</em> <strong>1996</strong>, <em>7</em> (6), 532–540. <a href="https://doi.org/10.1016/1044-0305(96)00057-8">https://doi.org/10.1016/1044-0305(96)00057-8</a>.</div>
</div>
<div id="ref-2D6WnK0w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(584) </div><div class="csl-right-inline">Hoopmann, M. R.; Merrihew, G. E.; von Haller, P. D.; MacCoss, M. J. Post Analysis Data Acquisition for the Iterative MS/MS Sampling of Proteomics Mixtures. <em>J. Proteome Res.</em> <strong>2009</strong>, <em>8</em> (4), 1870–1875. <a href="https://doi.org/10.1021/pr800828p">https://doi.org/10.1021/pr800828p</a>.</div>
</div>
<div id="ref-yqW0xFtN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(585) </div><div class="csl-right-inline">Picotti, P.; Aebersold, R.; Domon, B. The Implications of Proteolytic Background for Shotgun Proteomics. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2007</strong>, <em>6</em> (9), 1589–1598. <a href="https://doi.org/10.1074/mcp.m700029-mcp200">https://doi.org/10.1074/mcp.m700029-mcp200</a>.</div>
</div>
<div id="ref-wAKbWLXc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(586) </div><div class="csl-right-inline">Meyer, J. G.; Schilling, B. Clinical Applications of Quantitative Proteomics Using Targeted and Untargeted Data-Independent Acquisition Techniques. <em>Expert Review of Proteomics</em> <strong>2017</strong>, <em>14</em> (5), 419–429. <a href="https://doi.org/10.1080/14789450.2017.1322904">https://doi.org/10.1080/14789450.2017.1322904</a>.</div>
</div>
<div id="ref-NQj9Wees" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(587) </div><div class="csl-right-inline">Colangelo, C. M.; Chung, L.; Bruce, C.; Cheung, K.-H. Review of Software Tools for Design and Analysis of Large Scale MRM Proteomic Datasets. <em>Methods</em> <strong>2013</strong>, <em>61</em> (3), 287–298. <a href="https://doi.org/10.1016/j.ymeth.2013.05.004">https://doi.org/10.1016/j.ymeth.2013.05.004</a>.</div>
</div>
<div id="ref-1FAMk6Qey" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(588) </div><div class="csl-right-inline">Kuhn, E.; Wu, J.; Karl, J.; Liao, H.; Zolg, W.; Guild, B. Quantification of C‐reactive Protein in the Serum of Patients with Rheumatoid Arthritis Using Multiple Reaction Monitoring Mass Spectrometry and &lt;Sup&gt;13&lt;/Sup&gt;C‐labeled Peptide Standards. <em>Proteomics</em> <strong>2004</strong>, <em>4</em> (4), 1175–1186. <a href="https://doi.org/10.1002/pmic.200300670">https://doi.org/10.1002/pmic.200300670</a>.</div>
</div>
<div id="ref-63Pqc9VE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(589) </div><div class="csl-right-inline">Anderson, N. L.; Anderson, N. G.; Haines, L. R.; Hardie, D. B.; Olafson, R. W.; Pearson, T. W. Mass Spectrometric Quantitation of Peptides and Proteins Using Stable Isotope Standards and Capture by Anti-Peptide Antibodies (SISCAPA). <em>J. Proteome Res.</em> <strong>2004</strong>, <em>3</em> (2), 235–244. <a href="https://doi.org/10.1021/pr034086h">https://doi.org/10.1021/pr034086h</a>.</div>
</div>
<div id="ref-iJJwYdQ1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(590) </div><div class="csl-right-inline">Vidova, V.; Spacil, Z. A Review on Mass Spectrometry-Based Quantitative Proteomics: Targeted and Data Independent Acquisition. <em>Analytica Chimica Acta</em> <strong>2017</strong>, <em>964</em>, 7–23. <a href="https://doi.org/10.1016/j.aca.2017.01.059">https://doi.org/10.1016/j.aca.2017.01.059</a>.</div>
</div>
<div id="ref-ba7F2Epn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(591) </div><div class="csl-right-inline">Lemoine, J.; Fortin, T.; Salvador, A.; Jaffuel, A.; Charrier, J.-P.; Choquet-Kastylevsky, G. The Current Status of Clinical Proteomics and the Use of MRM and MRM&lt;sup&gt;3&lt;/Sup&gt;for Biomarker Validation. <em>Expert Review of Molecular Diagnostics</em> <strong>2012</strong>, <em>12</em> (4), 333–342. <a href="https://doi.org/10.1586/erm.12.32">https://doi.org/10.1586/erm.12.32</a>.</div>
</div>
<div id="ref-jcu6ZT0x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(592) </div><div class="csl-right-inline">Stahl-Zeng, J.; Lange, V.; Ossola, R.; Eckhardt, K.; Krek, W.; Aebersold, R.; Domon, B. High Sensitivity Detection of Plasma Proteins by Multiple Reaction Monitoring of N-Glycosites. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2007</strong>, <em>6</em> (10), 1809–1817. <a href="https://doi.org/10.1074/mcp.m700132-mcp200">https://doi.org/10.1074/mcp.m700132-mcp200</a>.</div>
</div>
<div id="ref-eFcH0xbv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(593) </div><div class="csl-right-inline">Picotti, P.; Lam, H.; Campbell, D.; Deutsch, E. W.; Mirzaei, H.; Ranish, J.; Domon, B.; Aebersold, R. A Database of Mass Spectrometric Assays for the Yeast Proteome. <em>Nat Methods</em> <strong>2008</strong>, <em>5</em> (11), 913–914. <a href="https://doi.org/10.1038/nmeth1108-913">https://doi.org/10.1038/nmeth1108-913</a>.</div>
</div>
<div id="ref-1Bt8NAhNo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(594) </div><div class="csl-right-inline">Peterson, A. C.; Russell, J. D.; Bailey, D. J.; Westphall, M. S.; Coon, J. J. Parallel Reaction Monitoring for High Resolution and High Mass Accuracy Quantitative, Targeted Proteomics. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2012</strong>, <em>11</em> (11), 1475–1488. <a href="https://doi.org/10.1074/mcp.o112.020131">https://doi.org/10.1074/mcp.o112.020131</a>.</div>
</div>
<div id="ref-HB3XvRF2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(595) </div><div class="csl-right-inline">Ronsein, G. E.; Pamir, N.; von Haller, P. D.; Kim, D. S.; Oda, M. N.; Jarvik, G. P.; Vaisar, T.; Heinecke, J. W. Parallel Reaction Monitoring (PRM) and Selected Reaction Monitoring (SRM) Exhibit Comparable Linearity, Dynamic Range and Precision for Targeted Quantitative HDL Proteomics. <em>Journal of Proteomics</em> <strong>2015</strong>, <em>113</em>, 388–399. <a href="https://doi.org/10.1016/j.jprot.2014.10.017">https://doi.org/10.1016/j.jprot.2014.10.017</a>.</div>
</div>
<div id="ref-dWzeUq6d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(596) </div><div class="csl-right-inline">Purvine, S.; Eppel*, J.; Yi, E. C.; Goodlett, D. R. Shotgun Collision‐induced Dissociation of Peptides Using a Time of Flight Mass Analyzer. <em>Proteomics</em> <strong>2003</strong>, <em>3</em> (6), 847–850. <a href="https://doi.org/10.1002/pmic.200300362">https://doi.org/10.1002/pmic.200300362</a>.</div>
</div>
<div id="ref-yn40ByaQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(597) </div><div class="csl-right-inline">Venable, J. D.; Dong, M.-Q.; Wohlschlegel, J.; Dillin, A.; Yates, J. R., III. Automated Approach for Quantitative Analysis of Complex Peptide Mixtures from Tandem Mass Spectra. <em>Nat Methods</em> <strong>2004</strong>, <em>1</em> (1), 39–45. <a href="https://doi.org/10.1038/nmeth705">https://doi.org/10.1038/nmeth705</a>.</div>
</div>
<div id="ref-Z3Hfnak1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(598) </div><div class="csl-right-inline">Plumb, R. S.; Johnson, K. A.; Rainville, P.; Smith, B. W.; Wilson, I. D.; Castro‐Perez, J. M.; Nicholson, J. K. UPLC/MS&lt;sup&gt;E&lt;/Sup&gt;; a New Approach for Generating Molecular Fragment Information for Biomarker Structure Elucidation. <em>Rapid Comm Mass Spectrometry</em> <strong>2006</strong>, <em>20</em> (13), 1989–1994. <a href="https://doi.org/10.1002/rcm.2550">https://doi.org/10.1002/rcm.2550</a>.</div>
</div>
<div id="ref-Wb9egjJg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(599) </div><div class="csl-right-inline">Panchaud, A.; Scherl, A.; Shaffer, S. A.; von Haller, P. D.; Kulasekara, H. D.; Miller, S. I.; Goodlett, D. R. Precursor Acquisition Independent From Ion Count: How to Dive Deeper into the Proteomics Ocean. <em>Anal. Chem.</em> <strong>2009</strong>, <em>81</em> (15), 6481–6488. <a href="https://doi.org/10.1021/ac900888s">https://doi.org/10.1021/ac900888s</a>.</div>
</div>
<div id="ref-OubW7j0g" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(600) </div><div class="csl-right-inline">Geiger, T.; Cox, J.; Mann, M. Proteomics on an Orbitrap Benchtop Mass Spectrometer Using All-Ion Fragmentation. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2010</strong>, <em>9</em> (10), 2252–2261. <a href="https://doi.org/10.1074/mcp.m110.001537">https://doi.org/10.1074/mcp.m110.001537</a>.</div>
</div>
<div id="ref-y71ZDsTJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(601) </div><div class="csl-right-inline">Carvalho, P. C.; Han, X.; Xu, T.; Cociorva, D.; Carvalho, M. da G.; Barbosa, V. C.; Yates, J. R., III. XDIA: Improving on the Label-Free Data-Independent Analysis. <em>Bioinformatics</em> <strong>2010</strong>, <em>26</em> (6), 847–848. <a href="https://doi.org/10.1093/bioinformatics/btq031">https://doi.org/10.1093/bioinformatics/btq031</a>.</div>
</div>
<div id="ref-ckj0XYOM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(602) </div><div class="csl-right-inline">Wong, J. W.; Schwahn, A. B.; Downard, K. M. ETISEQ – an Algorithm for Automated Elution Time Ion Sequencing of Concurrently Fragmented Peptides for Mass Spectrometry-Based Proteomics. <em>BMC Bioinformatics</em> <strong>2009</strong>, <em>10</em> (1). <a href="https://doi.org/10.1186/1471-2105-10-244">https://doi.org/10.1186/1471-2105-10-244</a>.</div>
</div>
<div id="ref-11ccXv5xm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(603) </div><div class="csl-right-inline">Gillet, L. C.; Navarro, P.; Tate, S.; Röst, H.; Selevsek, N.; Reiter, L.; Bonner, R.; Aebersold, R. Targeted Data Extraction of the MS/MS Spectra Generated by Data-Independent Acquisition: A New Concept for Consistent and Accurate Proteome Analysis. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2012</strong>, <em>11</em> (6), O111.016717. <a href="https://doi.org/10.1074/mcp.o111.016717">https://doi.org/10.1074/mcp.o111.016717</a>.</div>
</div>
<div id="ref-1GCjdtGKY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(604) </div><div class="csl-right-inline">Egertson, J. D.; Kuehn, A.; Merrihew, G. E.; Bateman, N. W.; MacLean, B. X.; Ting, Y. S.; Canterbury, J. D.; Marsh, D. M.; Kellmann, M.; Zabrouskov, V.; Wu, C. C.; MacCoss, M. J. Multiplexed MS/MS for Improved Data-Independent Acquisition. <em>Nat Methods</em> <strong>2013</strong>, <em>10</em> (8), 744–746. <a href="https://doi.org/10.1038/nmeth.2528">https://doi.org/10.1038/nmeth.2528</a>.</div>
</div>
<div id="ref-1GvzNjeV3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(605) </div><div class="csl-right-inline">Bruderer, R.; Bernhardt, O. M.; Gandhi, T.; Xuan, Y.; Sondermann, J.; Schmidt, M.; Gomez-Varela, D.; Reiter, L. Optimization of Experimental Parameters in Data-Independent Mass Spectrometry Significantly Increases Depth and Reproducibility of Results. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2017</strong>, <em>16</em> (12), 2296–2309. <a href="https://doi.org/10.1074/mcp.ra117.000314">https://doi.org/10.1074/mcp.ra117.000314</a>.</div>
</div>
<div id="ref-YKRUqViz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(606) </div><div class="csl-right-inline">Meier, F.; Brunner, A.-D.; Frank, M.; Ha, A.; Bludau, I.; Voytik, E.; Kaspar-Schoenefeld, S.; Lubeck, M.; Raether, O.; Bache, N.; Aebersold, R.; Collins, B. C.; Röst, H. L.; Mann, M. diaPASEF: Parallel Accumulation–Serial Fragmentation Combined with Data-Independent Acquisition. <em>Nat Methods</em> <strong>2020</strong>, <em>17</em> (12), 1229–1236. <a href="https://doi.org/10.1038/s41592-020-00998-0">https://doi.org/10.1038/s41592-020-00998-0</a>.</div>
</div>
<div id="ref-h7UB5vUH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(607) </div><div class="csl-right-inline">Jedrychowski, M. P.; Huttlin, E. L.; Haas, W.; Sowa, M. E.; Rad, R.; Gygi, S. P. Evaluation of HCD- and CID-Type Fragmentation Within Their Respective Detection Platforms For Murine Phosphoproteomics. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2011</strong>, <em>10</em> (12), M111.009910. <a href="https://doi.org/10.1074/mcp.m111.009910">https://doi.org/10.1074/mcp.m111.009910</a>.</div>
</div>
<div id="ref-C0Uvh6dQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(608) </div><div class="csl-right-inline">Nagaraj, N.; D’Souza, R. C. J.; Cox, J.; Olsen, J. V.; Mann, M. Feasibility of Large-Scale Phosphoproteomics with Higher Energy Collisional Dissociation Fragmentation. <em>J. Proteome Res.</em> <strong>2010</strong>, <em>9</em> (12), 6786–6794. <a href="https://doi.org/10.1021/pr100637q">https://doi.org/10.1021/pr100637q</a>.</div>
</div>
<div id="ref-YU1KHJ6Z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(609) </div><div class="csl-right-inline">Riley, N. M.; Hebert, A. S.; Dürnberger, G.; Stanek, F.; Mechtler, K.; Westphall, M. S.; Coon, J. J. Phosphoproteomics with Activated Ion Electron Transfer Dissociation. <em>Anal. Chem.</em> <strong>2017</strong>, <em>89</em> (12), 6367–6376. <a href="https://doi.org/10.1021/acs.analchem.7b00212">https://doi.org/10.1021/acs.analchem.7b00212</a>.</div>
</div>
<div id="ref-18YSDGQ7f" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(610) </div><div class="csl-right-inline">Frese, C. K.; Zhou, H.; Taus, T.; Altelaar, A. F. M.; Mechtler, K.; Heck, A. J. R.; Mohammed, S. Unambiguous Phosphosite Localization Using Electron-Transfer/Higher-Energy Collision Dissociation (EThcD). <em>J. Proteome Res.</em> <strong>2013</strong>, <em>12</em> (3), 1520–1525. <a href="https://doi.org/10.1021/pr301130k">https://doi.org/10.1021/pr301130k</a>.</div>
</div>
<div id="ref-12LsVxcez" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(611) </div><div class="csl-right-inline">Palumbo, A. M.; Reid, G. E. Evaluation of Gas-Phase Rearrangement and Competing Fragmentation Reactions on Protein Phosphorylation Site Assignment Using Collision Induced Dissociation-MS/MS and MS&lt;sup&gt;3&lt;/Sup&gt;. <em>Anal. Chem.</em> <strong>2008</strong>, <em>80</em> (24), 9735–9747. <a href="https://doi.org/10.1021/ac801768s">https://doi.org/10.1021/ac801768s</a>.</div>
</div>
<div id="ref-1BSd7ZIJx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(612) </div><div class="csl-right-inline">Cui, L.; Reid, G. E. Examining Factors That Influence Erroneous Phosphorylation Site Localization via Competing Fragmentation and Rearrangement Reactions During Ion Trap &lt;Scp&gt;CID&lt;/Scp&gt;‐&lt;scp&gt;MS&lt;/Scp&gt;/&lt;Scp&gt;MS&lt;/Scp&gt; and ‐&lt;Scp&gt;MS&lt;/Scp&gt;&lt;sup&gt;3&lt;/Sup&gt;. <em>Proteomics</em> <strong>2013</strong>, <em>13</em> (6), 964–973. <a href="https://doi.org/10.1002/pmic.201200384">https://doi.org/10.1002/pmic.201200384</a>.</div>
</div>
<div id="ref-XAV4WUNo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(613) </div><div class="csl-right-inline">Mischerikow, N.; Altelaar, A. F. M.; Navarro, J. D.; Mohammed, S.; Heck, A. J. R. Comparative Assessment of Site Assignments in CID and Electron Transfer Dissociation Spectra of Phosphopeptides Discloses Limited Relocation of Phosphate Groups. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2010</strong>, <em>9</em> (10), 2140–2148. <a href="https://doi.org/10.1074/mcp.m900619-mcp200">https://doi.org/10.1074/mcp.m900619-mcp200</a>.</div>
</div>
<div id="ref-DcoxXgj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(614) </div><div class="csl-right-inline">Kelstrup, C. D.; Hekmat, O.; Francavilla, C.; Olsen, J. V. Pinpointing Phosphorylation Sites: Quantitative Filtering and a Novel Site-Specific x-Ion Fragment. <em>J. Proteome Res.</em> <strong>2011</strong>, <em>10</em> (7), 2937–2948. <a href="https://doi.org/10.1021/pr200154t">https://doi.org/10.1021/pr200154t</a>.</div>
</div>
<div id="ref-SngtJdsy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(615) </div><div class="csl-right-inline">Beausoleil, S. A.; Jedrychowski, M.; Schwartz, D.; Elias, J. E.; Villén, J.; Li, J.; Cohn, M. A.; Cantley, L. C.; Gygi, S. P. Large-Scale Characterization of HeLa Cell Nuclear Phosphoproteins. <em>Proc. Natl. Acad. Sci. U.S.A.</em> <strong>2004</strong>, <em>101</em> (33), 12130–12135. <a href="https://doi.org/10.1073/pnas.0404720101">https://doi.org/10.1073/pnas.0404720101</a>.</div>
</div>
<div id="ref-WtrZNDoq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(616) </div><div class="csl-right-inline">Villén, J.; Beausoleil, S. A.; Gygi, S. P. Evaluation of the Utility of Neutral‐loss‐dependent MS3 Strategies in Large‐scale Phosphorylation Analysis. <em>Proteomics</em> <strong>2008</strong>, <em>8</em> (21), 4444–4452. <a href="https://doi.org/10.1002/pmic.200800283">https://doi.org/10.1002/pmic.200800283</a>.</div>
</div>
<div id="ref-OCeJYJsg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(617) </div><div class="csl-right-inline">Parker, B. L.; Yang, G.; Humphrey, S. J.; Chaudhuri, R.; Ma, X.; Peterman, S.; James, D. E. Targeted Phosphoproteomics of Insulin Signaling Using Data-Independent Acquisition Mass Spectrometry. <em>Sci. Signal.</em> <strong>2015</strong>, <em>8</em> (380). <a href="https://doi.org/10.1126/scisignal.aaa3139">https://doi.org/10.1126/scisignal.aaa3139</a>.</div>
</div>
<div id="ref-GD2lTg3N" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(618) </div><div class="csl-right-inline">Martinez-Val, A.; Bekker-Jensen, D. B.; Hogrebe, A.; Olsen, J. V. Data Processing and Analysis for DIA-Based Phosphoproteomics Using Spectronaut. In <em>Methods in Molecular Biology</em>; Springer US, 2021; pp 95–107. <a href="https://doi.org/10.1007/978-1-0716-1641-3_6">https://doi.org/10.1007/978-1-0716-1641-3_6</a>.</div>
</div>
<div id="ref-18oI6YEEE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(619) </div><div class="csl-right-inline">Lawrence, R. T.; Searle, B. C.; Llovet, A.; Villén, J. Plug-and-Play Analysis of the Human Phosphoproteome by Targeted High-Resolution Mass Spectrometry. <em>Nat Methods</em> <strong>2016</strong>, <em>13</em> (5), 431–434. <a href="https://doi.org/10.1038/nmeth.3811">https://doi.org/10.1038/nmeth.3811</a>.</div>
</div>
<div id="ref-rydRUvTU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(620) </div><div class="csl-right-inline">Hogrebe, A.; von Stechow, L.; Bekker-Jensen, D. B.; Weinert, B. T.; Kelstrup, C. D.; Olsen, J. V. Benchmarking Common Quantification Strategies for Large-Scale Phosphoproteomics. <em>Nat Commun</em> <strong>2018</strong>, <em>9</em> (1). <a href="https://doi.org/10.1038/s41467-018-03309-6">https://doi.org/10.1038/s41467-018-03309-6</a>.</div>
</div>
<div id="ref-GL3PYaNa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(621) </div><div class="csl-right-inline">Singh, C.; Zampronio, C. G.; Creese, A. J.; Cooper, H. J. Higher Energy Collision Dissociation (HCD) Product Ion-Triggered Electron Transfer Dissociation (ETD) Mass Spectrometry for the Analysis of &lt;i&gt;N&lt;/i&gt;-Linked Glycoproteins. <em>J. Proteome Res.</em> <strong>2012</strong>, <em>11</em> (9), 4517–4525. <a href="https://doi.org/10.1021/pr300257c">https://doi.org/10.1021/pr300257c</a>.</div>
</div>
<div id="ref-imR1dFei" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(622) </div><div class="csl-right-inline">Nesvizhskii, A. I.; Aebersold, R. Interpretation of Shotgun Proteomic Data: The Protein Inference Problem. <em>Mol Cell Proteomics</em> <strong>2005</strong>, <em>4</em> (10), 1419–1440. <a href="https://doi.org/10.1074/mcp.r500012-mcp200">https://doi.org/10.1074/mcp.r500012-mcp200</a>.</div>
</div>
<div id="ref-wtH5a4Ax" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(623) </div><div class="csl-right-inline">Audain, E.; Uszkoreit, J.; Sachsenberg, T.; Pfeuffer, J.; Liang, X.; Hermjakob, H.; Sanchez, A.; Eisenacher, M.; Reinert, K.; Tabb, D. L.; Kohlbacher, O.; Perez-Riverol, Y. In-Depth Analysis of Protein Inference Algorithms Using Multiple Search Engines and Well-Defined Metrics. <em>Journal of Proteomics</em> <strong>2017</strong>, <em>150</em>, 170–182. <a href="https://doi.org/10.1016/j.jprot.2016.08.002">https://doi.org/10.1016/j.jprot.2016.08.002</a>.</div>
</div>
<div id="ref-zsmAmqZJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(624) </div><div class="csl-right-inline">Martens, L.; Chambers, M.; Sturm, M.; Kessner, D.; Levander, F.; Shofstahl, J.; Tang, W. H.; Römpp, A.; Neumann, S.; Pizarro, A. D.; Montecchi-Palazzi, L.; Tasman, N.; Coleman, M.; Reisinger, F.; Souda, P.; Hermjakob, H.; Binz, P.-A.; Deutsch, E. W. mzML—a Community Standard for Mass Spectrometry Data. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2011</strong>, <em>10</em> (1), R110.000133. <a href="https://doi.org/10.1074/mcp.r110.000133">https://doi.org/10.1074/mcp.r110.000133</a>.</div>
</div>
<div id="ref-fQHXeOG1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(625) </div><div class="csl-right-inline">Deutsch, E. W. Mass Spectrometer Output File Format mzML. <em>Methods Mol Biol</em> <strong>2010</strong>, <em>604</em>, 319–331. <a href="https://doi.org/10.1007/978-1-60761-444-9_22">https://doi.org/10.1007/978-1-60761-444-9_22</a>.</div>
</div>
<div id="ref-r2hPariY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(626) </div><div class="csl-right-inline">Deutsch, E. W. File Formats Commonly Used in Mass Spectrometry Proteomics. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2012</strong>, <em>11</em> (12), 1612–1621. <a href="https://doi.org/10.1074/mcp.r112.019695">https://doi.org/10.1074/mcp.r112.019695</a>.</div>
</div>
<div id="ref-1GxDrwPIW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(627) </div><div class="csl-right-inline">Verheggen, K.; Ræder, H.; Berven, F. S.; Martens, L.; Barsnes, H.; Vaudel, M. Anatomy and Evolution of Database Search Engines—a Central Component of Mass Spectrometry Based Proteomic Workflows. <em>Mass Spectrometry Reviews</em> <strong>2017</strong>, <em>39</em> (3), 292–306. <a href="https://doi.org/10.1002/mas.21543">https://doi.org/10.1002/mas.21543</a>.</div>
</div>
<div id="ref-18PIzg2Co" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(628) </div><div class="csl-right-inline">Cox, J.; Mann, M. MaxQuant Enables High Peptide Identification Rates, Individualized p.p.b.-Range Mass Accuracies and Proteome-Wide Protein Quantification. <em>Nat Biotechnol</em> <strong>2008</strong>, <em>26</em> (12), 1367–1372. <a href="https://doi.org/10.1038/nbt.1511">https://doi.org/10.1038/nbt.1511</a>.</div>
</div>
<div id="ref-bKpYR6PB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(629) </div><div class="csl-right-inline">Kong, A. T.; Leprevost, F. V.; Avtonomov, D. M.; Mellacheruvu, D.; Nesvizhskii, A. I. MSFragger: Ultrafast and Comprehensive Peptide Identification in Mass Spectrometry–Based Proteomics. <em>Nat Methods</em> <strong>2017</strong>, <em>14</em> (5), 513–520. <a href="https://doi.org/10.1038/nmeth.4256">https://doi.org/10.1038/nmeth.4256</a>.</div>
</div>
<div id="ref-rKtsv6G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(630) </div><div class="csl-right-inline">Perkins, D. N.; Pappin, D. J.; Creasy, D. M.; Cottrell, J. S. Probability-Based Protein Identification by Searching Sequence Databases Using Mass Spectrometry Data. <em>Electrophoresis</em> <strong>1999</strong>, <em>20</em> (18), 3551–3567. <a href="https://doi.org/10.1002/(sici)1522-2683(19991201)20:18&lt;3551::aid-elps3551&gt;3.0.co;2-2">https://doi.org/10.1002/(sici)1522-2683(19991201)20:18&lt;3551::aid-elps3551&gt;3.0.co;2-2</a>.</div>
</div>
<div id="ref-1B2RVFOKK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(631) </div><div class="csl-right-inline">Kim, S.; Pevzner, P. A. MS-GF+ Makes Progress Towards a Universal Database Search Tool for Proteomics. <em>Nat Commun</em> <strong>2014</strong>, <em>5</em> (1). <a href="https://doi.org/10.1038/ncomms6277">https://doi.org/10.1038/ncomms6277</a>.</div>
</div>
<div id="ref-IcvYCrHi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(632) </div><div class="csl-right-inline">Craig, R.; Beavis, R. C. A Method for Reducing the Time Required to Match Protein Sequences with Tandem Mass Spectra. <em>Rapid Comm Mass Spectrometry</em> <strong>2003</strong>, <em>17</em> (20), 2310–2316. <a href="https://doi.org/10.1002/rcm.1198">https://doi.org/10.1002/rcm.1198</a>.</div>
</div>
<div id="ref-iK4IJ8ED" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(633) </div><div class="csl-right-inline">Craig, R.; Beavis, R. C. TANDEM: Matching Proteins with Tandem Mass Spectra. <em>Bioinformatics</em> <strong>2004</strong>, <em>20</em> (9), 1466–1467. <a href="https://doi.org/10.1093/bioinformatics/bth092">https://doi.org/10.1093/bioinformatics/bth092</a>.</div>
</div>
<div id="ref-jC0M4v7k" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(634) </div><div class="csl-right-inline">Eng, J. K.; Jahan, T. A.; Hoopmann, M. R. Comet: An Open‐source &lt;Scp&gt;MS&lt;/Scp&gt;/&lt;Scp&gt;MS&lt;/Scp&gt; Sequence Database Search Tool. <em>Proteomics</em> <strong>2012</strong>, <em>13</em> (1), 22–24. <a href="https://doi.org/10.1002/pmic.201200439">https://doi.org/10.1002/pmic.201200439</a>.</div>
</div>
<div id="ref-CObAggjA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(635) </div><div class="csl-right-inline">MacLean, B.; Tomazela, D. M.; Shulman, N.; Chambers, M.; Finney, G. L.; Frewen, B.; Kern, R.; Tabb, D. L.; Liebler, D. C.; MacCoss, M. J. Skyline: An Open Source Document Editor for Creating and Analyzing Targeted Proteomics Experiments. <em>Bioinformatics</em> <strong>2010</strong>, <em>26</em> (7), 966–968. <a href="https://doi.org/10.1093/bioinformatics/btq054">https://doi.org/10.1093/bioinformatics/btq054</a>.</div>
</div>
<div id="ref-IUqTUA43" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(636) </div><div class="csl-right-inline">Tran, N. H.; Qiao, R.; Xin, L.; Chen, X.; Liu, C.; Zhang, X.; Shan, B.; Ghodsi, A.; Li, M. Deep Learning Enables de Novo Peptide Sequencing from Data-Independent-Acquisition Mass Spectrometry. <em>Nat Methods</em> <strong>2018</strong>, <em>16</em> (1), 63–66. <a href="https://doi.org/10.1038/s41592-018-0260-3">https://doi.org/10.1038/s41592-018-0260-3</a>.</div>
</div>
<div id="ref-vO5QsPvL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(637) </div><div class="csl-right-inline">Cox, J.; Hein, M. Y.; Luber, C. A.; Paron, I.; Nagaraj, N.; Mann, M. Accurate Proteome-Wide Label-Free Quantification by Delayed Normalization and Maximal Peptide Ratio Extraction, Termed MaxLFQ. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2014</strong>, <em>13</em> (9), 2513–2526. <a href="https://doi.org/10.1074/mcp.m113.031591">https://doi.org/10.1074/mcp.m113.031591</a>.</div>
</div>
<div id="ref-Gs7HWzAD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(638) </div><div class="csl-right-inline">Tyanova, S.; Temu, T.; Cox, J. The MaxQuant Computational Platform for Mass Spectrometry-Based Shotgun Proteomics. <em>Nat Protoc</em> <strong>2016</strong>, <em>11</em> (12), 2301–2319. <a href="https://doi.org/10.1038/nprot.2016.136">https://doi.org/10.1038/nprot.2016.136</a>.</div>
</div>
<div id="ref-66LthRpU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(639) </div><div class="csl-right-inline">Zimmer, J. S. D.; Monroe, M. E.; Qian, W.; Smith, R. D. Advances in Proteomics Data Analysis and Display Using an Accurate Mass and Time Tag Approach. <em>Mass Spectrometry Reviews</em> <strong>2006</strong>, <em>25</em> (3), 450–482. <a href="https://doi.org/10.1002/mas.20071">https://doi.org/10.1002/mas.20071</a>.</div>
</div>
<div id="ref-MaoP0MGM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(640) </div><div class="csl-right-inline">Andreev, V. P.; Li, L.; Cao, L.; Gu, Y.; Rejtar, T.; Wu, S.-L.; Karger, B. L. A New Algorithm Using Cross-Assignment for Label-Free Quantitation with LC−LTQ-FT MS. <em>J. Proteome Res.</em> <strong>2007</strong>, <em>6</em> (6), 2186–2194. <a href="https://doi.org/10.1021/pr0606880">https://doi.org/10.1021/pr0606880</a>.</div>
</div>
<div id="ref-5H13CEWA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(641) </div><div class="csl-right-inline">Tsou, C.-C.; Tsai, C.-F.; Tsui, Y.-H.; Sudhir, P.-R.; Wang, Y.-T.; Chen, Y.-J.; Chen, J.-Y.; Sung, T.-Y.; Hsu, W.-L. IDEAL-Q, an Automated Tool for Label-Free Quantitation Analysis Using an Efficient Peptide Alignment Approach and Spectral Data Validation. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2010</strong>, <em>9</em> (1), 131–144. <a href="https://doi.org/10.1074/mcp.m900177-mcp200">https://doi.org/10.1074/mcp.m900177-mcp200</a>.</div>
</div>
<div id="ref-rRs3KNfz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(642) </div><div class="csl-right-inline">Mueller, L. N.; Rinner, O.; Schmidt, A.; Letarte, S.; Bodenmiller, B.; Brusniak, M.; Vitek, O.; Aebersold, R.; Müller, M. &lt;B&gt;&lt;i&gt;SuperHirn&lt;/i&gt;&lt;/b&gt; – a Novel Tool for High Resolution LC‐MS‐based Peptide/Protein Profiling. <em>Proteomics</em> <strong>2007</strong>, <em>7</em> (19), 3470–3480. <a href="https://doi.org/10.1002/pmic.200700057">https://doi.org/10.1002/pmic.200700057</a>.</div>
</div>
<div id="ref-1EHOiWEPV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(643) </div><div class="csl-right-inline">Lim, M. Y.; Paulo, J. A.; Gygi, S. P. Evaluating False Transfer Rates from the Match-Between-Runs Algorithm with a Two-Proteome Model. <em>J. Proteome Res.</em> <strong>2019</strong>, <em>18</em> (11), 4020–4026. <a href="https://doi.org/10.1021/acs.jproteome.9b00492">https://doi.org/10.1021/acs.jproteome.9b00492</a>.</div>
</div>
<div id="ref-XGf3EEUa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(644) </div><div class="csl-right-inline">Yu, F.; Haynes, S. E.; Nesvizhskii, A. I. IonQuant Enables Accurate and Sensitive Label-Free Quantification With FDR-Controlled Match-Between-Runs. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2021</strong>, <em>20</em>, 100077. <a href="https://doi.org/10.1016/j.mcpro.2021.100077">https://doi.org/10.1016/j.mcpro.2021.100077</a>.</div>
</div>
<div id="ref-50qmV6VJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(645) </div><div class="csl-right-inline">Gessulat, S.; Schmidt, T.; Zolg, D. P.; Samaras, P.; Schnatbaum, K.; Zerweck, J.; Knaute, T.; Rechenberger, J.; Delanghe, B.; Huhmer, A.; Reimer, U.; Ehrlich, H.-C.; Aiche, S.; Kuster, B.; Wilhelm, M. Prosit: Proteome-Wide Prediction of Peptide Tandem Mass Spectra by Deep Learning. <em>Nat Methods</em> <strong>2019</strong>, <em>16</em> (6), 509–518. <a href="https://doi.org/10.1038/s41592-019-0426-7">https://doi.org/10.1038/s41592-019-0426-7</a>.</div>
</div>
<div id="ref-JejZsE21" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(646) </div><div class="csl-right-inline">Wang, M.; Wang, J.; Carver, J.; Pullman, B. S.; Cha, S. W.; Bandeira, N. Assembling the Community-Scale Discoverable Human Proteome. <em>Cell Syst</em> <strong>2018</strong>, <em>7</em> (4), 412–421.e5. <a href="https://doi.org/10.1016/j.cels.2018.08.004">https://doi.org/10.1016/j.cels.2018.08.004</a>.</div>
</div>
<div id="ref-Sd09mxFZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(647) </div><div class="csl-right-inline">Röst, H. L.; Rosenberger, G.; Navarro, P.; Gillet, L.; Miladinović, S. M.; Schubert, O. T.; Wolski, W.; Collins, B. C.; Malmström, J.; Malmström, L.; Aebersold, R. OpenSWATH Enables Automated, Targeted Analysis of Data-Independent Acquisition MS Data. <em>Nat Biotechnol</em> <strong>2014</strong>, <em>32</em> (3), 219–223. <a href="https://doi.org/10.1038/nbt.2841">https://doi.org/10.1038/nbt.2841</a>.</div>
</div>
<div id="ref-ezZ8OwXe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(648) </div><div class="csl-right-inline">Cranney, C. W.; Meyer, J. G. CsoDIAq Software for Direct Infusion Shotgun Proteome Analysis. <em>Anal. Chem.</em> <strong>2021</strong>, <em>93</em> (36), 12312–12319. <a href="https://doi.org/10.1021/acs.analchem.1c02021">https://doi.org/10.1021/acs.analchem.1c02021</a>.</div>
</div>
<div id="ref-1HLH2DWkK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(649) </div><div class="csl-right-inline">Demichev, V.; Messner, C. B.; Vernardis, S. I.; Lilley, K. S.; Ralser, M. DIA-NN: Neural Networks and Interference Correction Enable Deep Proteome Coverage in High Throughput. <em>Nat Methods</em> <strong>2019</strong>, <em>17</em> (1), 41–44. <a href="https://doi.org/10.1038/s41592-019-0638-x">https://doi.org/10.1038/s41592-019-0638-x</a>.</div>
</div>
<div id="ref-Mqz9Ko20" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(650) </div><div class="csl-right-inline">Tsou, C.-C.; Avtonomov, D.; Larsen, B.; Tucholska, M.; Choi, H.; Gingras, A.-C.; Nesvizhskii, A. I. DIA-Umpire: Comprehensive Computational Framework for Data-Independent Acquisition Proteomics. <em>Nat Methods</em> <strong>2015</strong>, <em>12</em> (3), 258–264. <a href="https://doi.org/10.1038/nmeth.3255">https://doi.org/10.1038/nmeth.3255</a>.</div>
</div>
<div id="ref-15hTjIZkP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(651) </div><div class="csl-right-inline">Ting, Y. S.; Egertson, J. D.; Bollinger, J. G.; Searle, B. C.; Payne, S. H.; Noble, W. S.; MacCoss, M. J. PECAN: Library-Free Peptide Detection for Data-Independent Acquisition Tandem Mass Spectrometry Data. <em>Nat Methods</em> <strong>2017</strong>, <em>14</em> (9), 903–908. <a href="https://doi.org/10.1038/nmeth.4390">https://doi.org/10.1038/nmeth.4390</a>.</div>
</div>
<div id="ref-rYKhRHlv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(652) </div><div class="csl-right-inline">Searle, B. C.; Pino, L. K.; Egertson, J. D.; Ting, Y. S.; Lawrence, R. T.; MacLean, B. X.; Villén, J.; MacCoss, M. J. Chromatogram Libraries Improve Peptide Detection and Quantification by Data Independent Acquisition Mass Spectrometry. <em>Nat Commun</em> <strong>2018</strong>, <em>9</em> (1). <a href="https://doi.org/10.1038/s41467-018-07454-w">https://doi.org/10.1038/s41467-018-07454-w</a>.</div>
</div>
<div id="ref-UKcaDUJ7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(653) </div><div class="csl-right-inline">Bruderer, R.; Bernhardt, O. M.; Gandhi, T.; Miladinović, S. M.; Cheng, L.-Y.; Messner, S.; Ehrenberger, T.; Zanotelli, V.; Butscheid, Y.; Escher, C.; Vitek, O.; Rinner, O.; Reiter, L. Extending the Limits of Quantitative Proteome Profiling with Data-Independent Acquisition and Application to Acetaminophen-Treated Three-Dimensional Liver Microtissues. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2015</strong>, <em>14</em> (5), 1400–1410. <a href="https://doi.org/10.1074/mcp.m114.044305">https://doi.org/10.1074/mcp.m114.044305</a>.</div>
</div>
<div id="ref-bjANWlBN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(654) </div><div class="csl-right-inline">Käll, L.; Storey, J. D.; MacCoss, M. J.; Noble, W. S. Assigning Significance to Peptides Identified by Tandem Mass Spectrometry Using Decoy Databases. <em>J. Proteome Res.</em> <strong>2007</strong>, <em>7</em> (1), 29–34. <a href="https://doi.org/10.1021/pr700600n">https://doi.org/10.1021/pr700600n</a>.</div>
</div>
<div id="ref-BrkRiRe8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(655) </div><div class="csl-right-inline">Elias, J. E.; Gygi, S. P. Target-Decoy Search Strategy for Increased Confidence in Large-Scale Protein Identifications by Mass Spectrometry. <em>Nat Methods</em> <strong>2007</strong>, <em>4</em> (3), 207–214. <a href="https://doi.org/10.1038/nmeth1019">https://doi.org/10.1038/nmeth1019</a>.</div>
</div>
<div id="ref-5mQtZh5o" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(656) </div><div class="csl-right-inline">Aggarwal, S.; Yadav, A. K. False Discovery Rate Estimation in Proteomics. In <em>Methods in Molecular Biology</em>; Springer New York, 2016; pp 119–128. <a href="https://doi.org/10.1007/978-1-4939-3106-4_7">https://doi.org/10.1007/978-1-4939-3106-4_7</a>.</div>
</div>
<div id="ref-B2OZyvg5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(657) </div><div class="csl-right-inline">Levitsky, L. I.; Ivanov, M. V.; Lobas, A. A.; Gorshkov, M. V. Unbiased False Discovery Rate Estimation for Shotgun Proteomics Based on the Target-Decoy Approach. <em>J. Proteome Res.</em> <strong>2016</strong>, <em>16</em> (2), 393–397. <a href="https://doi.org/10.1021/acs.jproteome.6b00144">https://doi.org/10.1021/acs.jproteome.6b00144</a>.</div>
</div>
<div id="ref-B10jxZjh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(658) </div><div class="csl-right-inline">Vaudel, M.; Burkhart, J. M.; Zahedi, R. P.; Oveland, E.; Berven, F. S.; Sickmann, A.; Martens, L.; Barsnes, H. PeptideShaker Enables Reanalysis of MS-Derived Proteomics Data Sets. <em>Nat Biotechnol</em> <strong>2015</strong>, <em>33</em> (1), 22–24. <a href="https://doi.org/10.1038/nbt.3109">https://doi.org/10.1038/nbt.3109</a>.</div>
</div>
<div id="ref-tZ6JV4UF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(659) </div><div class="csl-right-inline">Farag, Y. M.; Horro, C.; Vaudel, M.; Barsnes, H. PeptideShaker Online: A User-Friendly Web-Based Framework for the Identification of Mass Spectrometry-Based Proteomics Data. <em>J. Proteome Res.</em> <strong>2021</strong>, <em>20</em> (12), 5419–5423. <a href="https://doi.org/10.1021/acs.jproteome.1c00678">https://doi.org/10.1021/acs.jproteome.1c00678</a>.</div>
</div>
<div id="ref-9bP7w1xR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(660) </div><div class="csl-right-inline">Keller, A.; Eng, J.; Zhang, N.; Li, X.; Aebersold, R. A Uniform Proteomics MS/MS Analysis Platform Utilizing Open XML File Formats. <em>Mol Syst Biol</em> <strong>2005</strong>, <em>1</em>, 2005.0017. <a href="https://doi.org/10.1038/msb4100024">https://doi.org/10.1038/msb4100024</a>.</div>
</div>
<div id="ref-w5NR3Ly9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(661) </div><div class="csl-right-inline">Pedrioli, P. G. A. Trans-Proteomic Pipeline: A Pipeline for Proteomic Analysis. <em>Methods Mol Biol</em> <strong>2010</strong>, <em>604</em>, 213–238. <a href="https://doi.org/10.1007/978-1-60761-444-9_15">https://doi.org/10.1007/978-1-60761-444-9_15</a>.</div>
</div>
<div id="ref-NY6IKEE4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(662) </div><div class="csl-right-inline">Keller, A.; Shteynberg, D. Software Pipeline and Data Analysis for MS/MS Proteomics: The Trans-Proteomic Pipeline. <em>Methods Mol Biol</em> <strong>2011</strong>, <em>694</em>, 169–189. <a href="https://doi.org/10.1007/978-1-60761-977-2_12">https://doi.org/10.1007/978-1-60761-977-2_12</a>.</div>
</div>
<div id="ref-1FG7koZ0h" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(663) </div><div class="csl-right-inline">Shteynberg, D.; Deutsch, E. W.; Lam, H.; Eng, J. K.; Sun, Z.; Tasman, N.; Mendoza, L.; Moritz, R. L.; Aebersold, R.; Nesvizhskii, A. I. iProphet: Multi-Level Integrative Analysis of Shotgun Proteomic Data Improves Peptide and Protein Identification Rates and Error Estimates. <em>Mol Cell Proteomics</em> <strong>2011</strong>, <em>10</em> (12), M111.007690. <a href="https://doi.org/10.1074/mcp.m111.007690">https://doi.org/10.1074/mcp.m111.007690</a>.</div>
</div>
<div id="ref-15EklFALC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(664) </div><div class="csl-right-inline">Slagel, J.; Mendoza, L.; Shteynberg, D.; Deutsch, E. W.; Moritz, R. L. Processing Shotgun Proteomics Data on the Amazon Cloud with the Trans-Proteomic Pipeline. <em>Mol Cell Proteomics</em> <strong>2014</strong>, <em>14</em> (2), 399–404. <a href="https://doi.org/10.1074/mcp.o114.043380">https://doi.org/10.1074/mcp.o114.043380</a>.</div>
</div>
<div id="ref-KUnjPYby" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(665) </div><div class="csl-right-inline">Deutsch, E. W.; Mendoza, L.; Shteynberg, D.; Slagel, J.; Sun, Z.; Moritz, R. L. Trans-Proteomic Pipeline, a Standardized Data Processing Pipeline for Large-Scale Reproducible Proteomics Informatics. <em>Proteomics Clin Appl</em> <strong>2015</strong>, <em>9</em> (7-8), 745–754. <a href="https://doi.org/10.1002/prca.201400164">https://doi.org/10.1002/prca.201400164</a>.</div>
</div>
<div id="ref-1ArAvfAkV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(666) </div><div class="csl-right-inline">Shteynberg, D.; Mendoza, L.; Hoopmann, M. R.; Sun, Z.; Schmidt, F.; Deutsch, E. W.; Moritz, R. L. reSpect: Software for Identification of High and Low Abundance Ion Species in Chimeric Tandem Mass Spectra. <em>J Am Soc Mass Spectrom</em> <strong>2015</strong>, <em>26</em> (11), 1837–1847. <a href="https://doi.org/10.1007/s13361-015-1252-5">https://doi.org/10.1007/s13361-015-1252-5</a>.</div>
</div>
<div id="ref-7BQGCj33" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(667) </div><div class="csl-right-inline">Hoopmann, M. R.; Winget, J. M.; Mendoza, L.; Moritz, R. L. StPeter: Seamless Label-Free Quantification with the Trans-Proteomic Pipeline. <em>J Proteome Res</em> <strong>2018</strong>, <em>17</em> (3), 1314–1320. <a href="https://doi.org/10.1021/acs.jproteome.7b00786">https://doi.org/10.1021/acs.jproteome.7b00786</a>.</div>
</div>
<div id="ref-SKCHzO1U" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(668) </div><div class="csl-right-inline">Shteynberg, D. D.; Deutsch, E. W.; Campbell, D. S.; Hoopmann, M. R.; Kusebauch, U.; Lee, D.; Mendoza, L.; Midha, M. K.; Sun, Z.; Whetton, A. D.; Moritz, R. L. PTMProphet: Fast and Accurate Mass Modification Localization for the Trans-Proteomic Pipeline. <em>J Proteome Res</em> <strong>2019</strong>, <em>18</em> (12), 4262–4272. <a href="https://doi.org/10.1021/acs.jproteome.9b00205">https://doi.org/10.1021/acs.jproteome.9b00205</a>.</div>
</div>
<div id="ref-bBz09mFl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(669) </div><div class="csl-right-inline">Deutsch, E. W.; Mendoza, L.; Shteynberg, D. D.; Hoopmann, M. R.; Sun, Z.; Eng, J. K.; Moritz, R. L. Trans-Proteomic Pipeline: Robust Mass Spectrometry-Based Proteomics Data Analysis Suite. <em>J Proteome Res</em> <strong>2023</strong>, <em>22</em> (2), 615–624. <a href="https://doi.org/10.1021/acs.jproteome.2c00624">https://doi.org/10.1021/acs.jproteome.2c00624</a>.</div>
</div>
<div id="ref-10mgCOk6B" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(670) </div><div class="csl-right-inline">Keller, A.; Nesvizhskii, A. I.; Kolker, E.; Aebersold, R. Empirical Statistical Model to Estimate the Accuracy of Peptide Identifications Made by MS/MS and Database Search. <em>Anal Chem</em> <strong>2002</strong>, <em>74</em> (20), 5383–5392. <a href="https://doi.org/10.1021/ac025747h">https://doi.org/10.1021/ac025747h</a>.</div>
</div>
<div id="ref-4OdLMmDD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(671) </div><div class="csl-right-inline">Nesvizhskii, A. I.; Keller, A.; Kolker, E.; Aebersold, R. A Statistical Model for Identifying Proteins by Tandem Mass Spectrometry. <em>Anal Chem</em> <strong>2003</strong>, <em>75</em> (17), 4646–4658. <a href="https://doi.org/10.1021/ac0341261">https://doi.org/10.1021/ac0341261</a>.</div>
</div>
<div id="ref-l6NqXxi8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(672) </div><div class="csl-right-inline">auf dem Keller, U.; Overall, C. M. CLIPPER: An Add-on to the Trans-Proteomic Pipeline for the Automated Analysis of TAILS N-Terminomics Data. <em>Biol Chem</em> <strong>2012</strong>, <em>393</em> (12), 1477–1483. <a href="https://doi.org/10.1515/hsz-2012-0269">https://doi.org/10.1515/hsz-2012-0269</a>.</div>
</div>
<div id="ref-WlEQJK9j" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(673) </div><div class="csl-right-inline">Mitchell, C. J.; Kim, M.-S.; Na, C. H.; Pandey, A. PyQuant: A Versatile Framework for Analysis of Quantitative Mass Spectrometry Data. <em>Mol Cell Proteomics</em> <strong>2016</strong>, <em>15</em> (8), 2829–2838. <a href="https://doi.org/10.1074/mcp.o115.056879">https://doi.org/10.1074/mcp.o115.056879</a>.</div>
</div>
<div id="ref-DzOnRsF8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(674) </div><div class="csl-right-inline">Suni, V.; Suomi, T.; Tsubosaka, T.; Imanishi, S. Y.; Elo, L. L.; Corthals, G. L. SimPhospho: A Software Tool Enabling Confident Phosphosite Assignment. <em>Bioinformatics</em> <strong>2018</strong>, <em>34</em> (15), 2690–2692. <a href="https://doi.org/10.1093/bioinformatics/bty151">https://doi.org/10.1093/bioinformatics/bty151</a>.</div>
</div>
<div id="ref-8GgfngeF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(675) </div><div class="csl-right-inline">Chen, C.-T.; Ko, C.-L.; Choong, W.-K.; Wang, J.-H.; Hsu, W.-L.; Sung, T.-Y. WinProphet: A User-Friendly Pipeline Management System for Proteomics Data Analysis Based on Trans-Proteomic Pipeline. <em>Anal Chem</em> <strong>2019</strong>, <em>91</em> (15), 9403–9406. <a href="https://doi.org/10.1021/acs.analchem.9b01556">https://doi.org/10.1021/acs.analchem.9b01556</a>.</div>
</div>
<div id="ref-2XDHE8d9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(676) </div><div class="csl-right-inline">Winkler, R. ProtyQuant: Comparing Label-Free Shotgun Proteomics Datasets Using Accumulated Peptide Probabilities. <em>J Proteomics</em> <strong>2020</strong>, <em>230</em>, 103985. <a href="https://doi.org/10.1016/j.jprot.2020.103985">https://doi.org/10.1016/j.jprot.2020.103985</a>.</div>
</div>
<div id="ref-146pjMaXB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(677) </div><div class="csl-right-inline">He, S.; Chakraborty, R.; Ranganathan, S. Metaproteomic Analysis of an Oral Squamous Cell Carcinoma Dataset Suggests Diagnostic Potential of the Mycobiome. <em>Int J Mol Sci</em> <strong>2023</strong>, <em>24</em> (2). <a href="https://doi.org/10.3390/ijms24021050">https://doi.org/10.3390/ijms24021050</a>.</div>
</div>
<div id="ref-KbIsieWP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(678) </div><div class="csl-right-inline">Lam, H.; Deutsch, E. W.; Eddes, J. S.; Eng, J. K.; Stein, S. E.; Aebersold, R. Building Consensus Spectral Libraries for Peptide Identification in Proteomics. <em>Nat Methods</em> <strong>2008</strong>, <em>5</em> (10), 873–875. <a href="https://doi.org/10.1038/nmeth.1254">https://doi.org/10.1038/nmeth.1254</a>.</div>
</div>
<div id="ref-M0N5UGce" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(679) </div><div class="csl-right-inline"><em>mzSpecLib – HUPO Proteomics Standards Initiative</em>. <a href="https://www.psidev.info/mzspeclib">https://www.psidev.info/mzspeclib</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-1C0ZGb77k" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(680) </div><div class="csl-right-inline">Ma, B. Novor: Real-Time Peptide de Novo Sequencing Software. <em>J Am Soc Mass Spectrom</em> <strong>2015</strong>, <em>26</em> (11), 1885–1894. <a href="https://doi.org/10.1007/s13361-015-1204-0">https://doi.org/10.1007/s13361-015-1204-0</a>.</div>
</div>
<div id="ref-zzSgLIrm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(681) </div><div class="csl-right-inline">Yilmaz, M.; Fondrie, W. E.; Bittremieux, W.; Oh, S.; Noble, W. S. &lt;I&gt;De Novo&lt;/i&gt; Mass Spectrometry Peptide Sequencing with a Transformer Model, 2022. <a href="https://doi.org/10.1101/2022.02.07.479481">https://doi.org/10.1101/2022.02.07.479481</a>.</div>
</div>
<div id="ref-fgRTHCzN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(682) </div><div class="csl-right-inline">Riffle, M.; Hoopmann, M. R.; Jaschob, D.; Zhong, G.; Moritz, R. L.; MacCoss, M. J.; Davis, T. N.; Isoherranen, N.; Zelter, A. Discovery and Visualization of Uncharacterized Drug-Protein Adducts Using Mass Spectrometry. <em>Anal Chem</em> <strong>2022</strong>, <em>94</em> (8), 3501–3509. <a href="https://doi.org/10.1021/acs.analchem.1c04101">https://doi.org/10.1021/acs.analchem.1c04101</a>.</div>
</div>
<div id="ref-JXLIvVI6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(683) </div><div class="csl-right-inline">Quality Control in Proteomics. <em>Proteomics</em> <strong>2011</strong>, <em>11</em> (6), 1015–1016. <a href="https://doi.org/10.1002/pmic.201190020">https://doi.org/10.1002/pmic.201190020</a>.</div>
</div>
<div id="ref-8dt10izl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(684) </div><div class="csl-right-inline">Köcher, T.; Pichler, P.; Swart, R.; Mechtler, K. Quality Control in LC‐MS/MS. <em>Proteomics</em> <strong>2011</strong>, <em>11</em> (6), 1026–1030. <a href="https://doi.org/10.1002/pmic.201000578">https://doi.org/10.1002/pmic.201000578</a>.</div>
</div>
<div id="ref-196I1uWdL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(685) </div><div class="csl-right-inline">Bittremieux, W.; Tabb, D. L.; Impens, F.; Staes, A.; Timmerman, E.; Martens, L.; Laukens, K. Quality Control in Mass Spectrometry‐based Proteomics. <em>Mass Spectrometry Reviews</em> <strong>2017</strong>, <em>37</em> (5), 697–711. <a href="https://doi.org/10.1002/mas.21544">https://doi.org/10.1002/mas.21544</a>.</div>
</div>
<div id="ref-9ocWQ3g2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(686) </div><div class="csl-right-inline">Strauss, M. T.; Bludau, I.; Zeng, W.-F.; Voytik, E.; Ammar, C.; Schessner, J.; Ilango, R.; Gill, M.; Meier, F.; Willems, S.; Mann, M. AlphaPept, a Modern and Open Framework for MS-Based Proteomics, 2021. <a href="https://doi.org/10.1101/2021.07.23.453379">https://doi.org/10.1101/2021.07.23.453379</a>.</div>
</div>
<div id="ref-mKMrKqLf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(687) </div><div class="csl-right-inline">Trachsel, C.; Panse, C.; Kockmann, T.; Wolski, W. E.; Grossmann, J.; Schlapbach, R. rawDiag: An R Package Supporting Rational LC–MS Method Optimization for Bottom-up Proteomics. <em>J. Proteome Res.</em> <strong>2018</strong>, <em>17</em> (8), 2908–2914. <a href="https://doi.org/10.1021/acs.jproteome.8b00173">https://doi.org/10.1021/acs.jproteome.8b00173</a>.</div>
</div>
<div id="ref-AI1wddvh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(688) </div><div class="csl-right-inline">Kockmann, T.; Panse, C. The Rawrr R Package: Direct Access to Orbitrap Data and Beyond. <em>J. Proteome Res.</em> <strong>2021</strong>, <em>20</em> (4), 2028–2034. <a href="https://doi.org/10.1021/acs.jproteome.0c00866">https://doi.org/10.1021/acs.jproteome.0c00866</a>.</div>
</div>
<div id="ref-MotUU8WD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(689) </div><div class="csl-right-inline">Morgenstern, D.; Barzilay, R.; Levin, Y. RawBeans: A Simple, Vendor-Independent, Raw-Data Quality-Control Tool. <em>J. Proteome Res.</em> <strong>2021</strong>, <em>20</em> (4), 2098–2104. <a href="https://doi.org/10.1021/acs.jproteome.0c00956">https://doi.org/10.1021/acs.jproteome.0c00956</a>.</div>
</div>
<div id="ref-fVgT4RCS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(690) </div><div class="csl-right-inline">Pichler, P.; Mazanek, M.; Dusberger, F.; Weilnböck, L.; Huber, C. G.; Stingl, C.; Luider, T. M.; Straube, W. L.; Köcher, T.; Mechtler, K. SIMPATIQCO: A Server-Based Software Suite Which Facilitates Monitoring the Time Course of LC–MS Performance Metrics on Orbitrap Instruments. <em>J. Proteome Res.</em> <strong>2012</strong>, <em>11</em> (11), 5540–5547. <a href="https://doi.org/10.1021/pr300163u">https://doi.org/10.1021/pr300163u</a>.</div>
</div>
<div id="ref-gR6OKDGy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(691) </div><div class="csl-right-inline">Stanfill, B. A.; Nakayasu, E. S.; Bramer, L. M.; Thompson, A. M.; Ansong, C. K.; Clauss, T. R.; Gritsenko, M. A.; Monroe, M. E.; Moore, R. J.; Orton, D. J.; Piehowski, P. D.; Schepmoes, A. A.; Smith, R. D.; Webb-Robertson, B.-J. M.; Metz, T. O. Quality Control Analysis in Real-Time (QC-ART): A Tool for Real-Time Quality Control Assessment of Mass Spectrometry-Based Proteomics Data. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2018</strong>, <em>17</em> (9), 1824–1836. <a href="https://doi.org/10.1074/mcp.ra118.000648">https://doi.org/10.1074/mcp.ra118.000648</a>.</div>
</div>
<div id="ref-IrACRvhj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(692) </div><div class="csl-right-inline">Scheltema, R. A.; Mann, M. SprayQc: A Real-Time LC–MS/MS Quality Monitoring System To Maximize Uptime Using Off the Shelf Components. <em>J. Proteome Res.</em> <strong>2012</strong>, <em>11</em> (6), 3458–3466. <a href="https://doi.org/10.1021/pr201219e">https://doi.org/10.1021/pr201219e</a>.</div>
</div>
<div id="ref-N0V314Nq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(693) </div><div class="csl-right-inline">Taylor, R. M.; Dance, J.; Taylor, R. J.; Prince, J. T. Metriculator: Quality Assessment for Mass Spectrometry-Based Proteomics. <em>Bioinformatics</em> <strong>2013</strong>, <em>29</em> (22), 2948–2949. <a href="https://doi.org/10.1093/bioinformatics/btt510">https://doi.org/10.1093/bioinformatics/btt510</a>.</div>
</div>
<div id="ref-ZJ3s73nL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(694) </div><div class="csl-right-inline">Röst, H. L.; Sachsenberg, T.; Aiche, S.; Bielow, C.; Weisser, H.; Aicheler, F.; Andreotti, S.; Ehrlich, H.-C.; Gutenbrunner, P.; Kenar, E.; Liang, X.; Nahnsen, S.; Nilse, L.; Pfeuffer, J.; Rosenberger, G.; Rurik, M.; Schmitt, U.; Veit, J.; Walzer, M.; Wojnar, D.; Wolski, W. E.; Schilling, O.; Choudhary, J. S.; Malmström, L.; Aebersold, R.; Reinert, K.; Kohlbacher, O. OpenMS: A Flexible Open-Source Software Platform for Mass Spectrometry Data Analysis. <em>Nat Methods</em> <strong>2016</strong>, <em>13</em> (9), 741–748. <a href="https://doi.org/10.1038/nmeth.3959">https://doi.org/10.1038/nmeth.3959</a>.</div>
</div>
<div id="ref-LpMjs9Ht" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(695) </div><div class="csl-right-inline">Choi, M.; Chang, C.-Y.; Clough, T.; Broudy, D.; Killeen, T.; MacLean, B.; Vitek, O. MSstats: An R Package for Statistical Analysis of Quantitative Mass Spectrometry-Based Proteomic Experiments. <em>Bioinformatics</em> <strong>2014</strong>, <em>30</em> (17), 2524–2526. <a href="https://doi.org/10.1093/bioinformatics/btu305">https://doi.org/10.1093/bioinformatics/btu305</a>.</div>
</div>
<div id="ref-BVhBtZ2w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(696) </div><div class="csl-right-inline">Dogu, E.; Mohammad-Taheri, S.; Abbatiello, S. E.; Bereman, M. S.; MacLean, B.; Schilling, B.; Vitek, O. MSstatsQC: Longitudinal System Suitability Monitoring and Quality Control for Targeted Proteomic Experiments. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2017</strong>, <em>16</em> (7), 1335–1347. <a href="https://doi.org/10.1074/mcp.m116.064774">https://doi.org/10.1074/mcp.m116.064774</a>.</div>
</div>
<div id="ref-R1U4dX3A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(697) </div><div class="csl-right-inline">Bielow, C.; Mastrobuoni, G.; Kempa, S. Proteomics Quality Control: Quality Control Software for MaxQuant Results. <em>J. Proteome Res.</em> <strong>2015</strong>, <em>15</em> (3), 777–787. <a href="https://doi.org/10.1021/acs.jproteome.5b00780">https://doi.org/10.1021/acs.jproteome.5b00780</a>.</div>
</div>
<div id="ref-Nix1GwUR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(698) </div><div class="csl-right-inline">Quast, J.-P.; Schuster, D.; Picotti, P. Protti: An R Package for Comprehensive Data Analysis of Peptide- and Protein-Centric Bottom-up Proteomics Data. <em>Bioinformatics Advances</em> <strong>2021</strong>, <em>2</em> (1). <a href="https://doi.org/10.1093/bioadv/vbab041">https://doi.org/10.1093/bioadv/vbab041</a>.</div>
</div>
<div id="ref-mkpI0rxh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(699) </div><div class="csl-right-inline">Lualdi, M.; Fasano, M. Statistical Analysis of Proteomics Data: A Review on Feature Selection. <em>Journal of Proteomics</em> <strong>2019</strong>, <em>198</em>, 18–26. <a href="https://doi.org/10.1016/j.jprot.2018.12.004">https://doi.org/10.1016/j.jprot.2018.12.004</a>.</div>
</div>
<div id="ref-xj3O2t6x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(700) </div><div class="csl-right-inline">Tyanova, S.; Temu, T.; Sinitcyn, P.; Carlson, A.; Hein, M. Y.; Geiger, T.; Mann, M.; Cox, J. The Perseus Computational Platform for Comprehensive Analysis of (Prote)omics Data. <em>Nat Methods</em> <strong>2016</strong>, <em>13</em> (9), 731–740. <a href="https://doi.org/10.1038/nmeth.3901">https://doi.org/10.1038/nmeth.3901</a>.</div>
</div>
<div id="ref-1wPiXyAt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(701) </div><div class="csl-right-inline">Dubois, E.; Galindo, A. N.; Dayon, L.; Cominetti, O. Assessing Normalization Methods in Mass Spectrometry-Based Proteome Profiling of Clinical Samples. <em>Biosystems</em> <strong>2022</strong>, <em>215-216</em>, 104661. <a href="https://doi.org/10.1016/j.biosystems.2022.104661">https://doi.org/10.1016/j.biosystems.2022.104661</a>.</div>
</div>
<div id="ref-127vja7uh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(702) </div><div class="csl-right-inline">Callister, S. J.; Barry, R. C.; Adkins, J. N.; Johnson, E. T.; Qian, W.; Webb-Robertson, B.-J. M.; Smith, R. D.; Lipton, M. S. Normalization Approaches for Removing Systematic Biases Associated with Mass Spectrometry and Label-Free Proteomics. <em>J. Proteome Res.</em> <strong>2006</strong>, <em>5</em> (2), 277–286. <a href="https://doi.org/10.1021/pr050300l">https://doi.org/10.1021/pr050300l</a>.</div>
</div>
<div id="ref-1CKzlqjbM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(703) </div><div class="csl-right-inline">Välikangas, T.; Suomi, T.; Elo, L. L. A Systematic Evaluation of Normalization Methods in Quantitative Label-Free Proteomics. <em>Brief Bioinform</em> <strong>2016</strong>, bbw095. <a href="https://doi.org/10.1093/bib/bbw095">https://doi.org/10.1093/bib/bbw095</a>.</div>
</div>
<div id="ref-mkTOg0QU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(704) </div><div class="csl-right-inline">Karpievitch, Y. V.; Dabney, A. R.; Smith, R. D. Normalization and Missing Value Imputation for Label-Free LC-MS Analysis. <em>BMC Bioinformatics</em> <strong>2012</strong>, <em>13</em> (S16). <a href="https://doi.org/10.1186/1471-2105-13-s16-s5">https://doi.org/10.1186/1471-2105-13-s16-s5</a>.</div>
</div>
<div id="ref-rXBG8g4V" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(705) </div><div class="csl-right-inline">Chawade, A.; Alexandersson, E.; Levander, F. Normalyzer: A Tool for Rapid Evaluation of Normalization Methods for Omics Data Sets. <em>J. Proteome Res.</em> <strong>2014</strong>, <em>13</em> (6), 3114–3120. <a href="https://doi.org/10.1021/pr401264n">https://doi.org/10.1021/pr401264n</a>.</div>
</div>
<div id="ref-11NAEs2KF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(706) </div><div class="csl-right-inline">Molania, R.; Foroutan, M.; Gagnon-Bartsch, J. A.; Gandolfo, L. C.; Jain, A.; Sinha, A.; Olshansky, G.; Dobrovic, A.; Papenfuss, A. T.; Speed, T. P. Removing Unwanted Variation from Large-Scale RNA Sequencing Data with PRPS. <em>Nat Biotechnol</em> <strong>2022</strong>, <em>41</em> (1), 82–95. <a href="https://doi.org/10.1038/s41587-022-01440-w">https://doi.org/10.1038/s41587-022-01440-w</a>.</div>
</div>
<div id="ref-l6oCMFXG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(707) </div><div class="csl-right-inline">Čuklina, J.; Lee, C. H.; Williams, E. G.; Sajic, T.; Collins, B. C.; Rodríguez Martínez, M.; Sharma, V. S.; Wendt, F.; Goetze, S.; Keele, G. R.; Wollscheid, B.; Aebersold, R.; Pedrioli, P. G. A. Diagnostics and Correction of Batch Effects in Large‐scale Proteomic Studies: A Tutorial. <em>Molecular Systems Biology</em> <strong>2021</strong>, <em>17</em> (8). <a href="https://doi.org/10.15252/msb.202110240">https://doi.org/10.15252/msb.202110240</a>.</div>
</div>
<div id="ref-c2k0gkjF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(708) </div><div class="csl-right-inline">Evans, C.; Hardin, J.; Stoebel, D. M. Selecting Between-Sample RNA-Seq Normalization Methods from the Perspective of Their Assumptions. <em>Briefings in Bioinformatics</em> <strong>2017</strong>, <em>19</em> (5), 776–792. <a href="https://doi.org/10.1093/bib/bbx008">https://doi.org/10.1093/bib/bbx008</a>.</div>
</div>
<div id="ref-scfU7NxG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(709) </div><div class="csl-right-inline">Bolstad, B. M.; Irizarry, R. A.; Åstrand, M.; Speed, T. P. A Comparison of Normalization Methods for High Density
oligonucleotide Array Data Based on Variance and Bias. <em>Bioinformatics</em> <strong>2003</strong>, <em>19</em> (2), 185–193. <a href="https://doi.org/10.1093/bioinformatics/19.2.185">https://doi.org/10.1093/bioinformatics/19.2.185</a>.</div>
</div>
<div id="ref-HCKwDoRr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(710) </div><div class="csl-right-inline">Risso, D.; Ngai, J.; Speed, T. P.; Dudoit, S. Normalization of RNA-Seq Data Using Factor Analysis of Control Genes or Samples. <em>Nat Biotechnol</em> <strong>2014</strong>, <em>32</em> (9), 896–902. <a href="https://doi.org/10.1038/nbt.2931">https://doi.org/10.1038/nbt.2931</a>.</div>
</div>
<div id="ref-ZgHkHm7E" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(711) </div><div class="csl-right-inline">Wang, M.; Jiang, L.; Jian, R.; Chan, J. Y.; Liu, Q.; Snyder, M. P.; Tang, H. RobNorm: Model-Based Robust Normalization Method for Labeled Quantitative Mass Spectrometry Proteomics Data. <em>Bioinformatics</em> <strong>2020</strong>, <em>37</em> (6), 815–821. <a href="https://doi.org/10.1093/bioinformatics/btaa904">https://doi.org/10.1093/bioinformatics/btaa904</a>.</div>
</div>
<div id="ref-nxZj1qHr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(712) </div><div class="csl-right-inline">Karpievitch, Y. V.; Taverner, T.; Adkins, J. N.; Callister, S. J.; Anderson, G. A.; Smith, R. D.; Dabney, A. R. Normalization of Peak Intensities in Bottom-up MS-Based Proteomics Using Singular Value Decomposition. <em>Bioinformatics</em> <strong>2009</strong>, <em>25</em> (19), 2573–2580. <a href="https://doi.org/10.1093/bioinformatics/btp426">https://doi.org/10.1093/bioinformatics/btp426</a>.</div>
</div>
<div id="ref-lZkCFiQ5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(713) </div><div class="csl-right-inline">Fan, S.; Kind, T.; Cajka, T.; Hazen, S. L.; Tang, W. H. W.; Kaddurah-Daouk, R.; Irvin, M. R.; Arnett, D. K.; Barupal, D. K.; Fiehn, O. Systematic Error Removal Using Random Forest for Normalizing Large-Scale Untargeted Lipidomics Data. <em>Anal. Chem.</em> <strong>2019</strong>, <em>91</em> (5), 3590–3596. <a href="https://doi.org/10.1021/acs.analchem.8b05592">https://doi.org/10.1021/acs.analchem.8b05592</a>.</div>
</div>
<div id="ref-10U5RhkON" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(714) </div><div class="csl-right-inline">Kultima, K.; Nilsson, A.; Scholz, B.; Rossbach, U. L.; Fälth, M.; Andrén, P. E. Development and Evaluation of Normalization Methods for Label-Free Relative Quantification of Endogenous Peptides. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2009</strong>, <em>8</em> (10), 2285–2295. <a href="https://doi.org/10.1074/mcp.m800514-mcp200">https://doi.org/10.1074/mcp.m800514-mcp200</a>.</div>
</div>
<div id="ref-DvZ7JWPs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(715) </div><div class="csl-right-inline">Willforss, J.; Chawade, A.; Levander, F. NormalyzerDE: Online Tool for Improved Normalization of Omics Expression Data and High-Sensitivity Differential Expression Analysis. <em>J. Proteome Res.</em> <strong>2018</strong>, <em>18</em> (2), 732–740. <a href="https://doi.org/10.1021/acs.jproteome.8b00523">https://doi.org/10.1021/acs.jproteome.8b00523</a>.</div>
</div>
<div id="ref-9nRMPWST" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(716) </div><div class="csl-right-inline">Krismer, E.; Bludau, I.; Strauss, M. T.; Mann, M. AlphaPeptStats: An Open-Source Python Package for Automated and Scalable Statistical Analysis of Mass Spectrometry-Based Proteomics. <em>Bioinformatics</em> <strong>2023</strong>, <em>39</em> (8). <a href="https://doi.org/10.1093/bioinformatics/btad461">https://doi.org/10.1093/bioinformatics/btad461</a>.</div>
</div>
<div id="ref-15hSlFzuu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(717) </div><div class="csl-right-inline">Karpievitch, Y.; Stanley, J.; Taverner, T.; Huang, J.; Adkins, J. N.; Ansong, C.; Heffron, F.; Metz, T. O.; Qian, W.-J.; Yoon, H.; Smith, R. D.; Dabney, A. R. A Statistical Framework for Protein Quantitation in Bottom-up MS-Based Proteomics. <em>Bioinformatics</em> <strong>2009</strong>, <em>25</em> (16), 2028–2034. <a href="https://doi.org/10.1093/bioinformatics/btp362">https://doi.org/10.1093/bioinformatics/btp362</a>.</div>
</div>
<div id="ref-jmZQxVay" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(718) </div><div class="csl-right-inline">Luo, R.; Colangelo, C. M.; Sessa, W. C.; Zhao, H. Bayesian Analysis of iTRAQ Data with Nonrandom Missingness: Identification of Differentially Expressed Proteins. <em>Stat Biosci</em> <strong>2009</strong>, <em>1</em> (2), 228–245. <a href="https://doi.org/10.1007/s12561-009-9013-2">https://doi.org/10.1007/s12561-009-9013-2</a>.</div>
</div>
<div id="ref-RwK8wIx4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(719) </div><div class="csl-right-inline">Stekhoven, D. J.; Bühlmann, P. MissForest—Non-Parametric Missing Value Imputation for Mixed-Type Data. <em>Bioinformatics</em> <strong>2011</strong>, <em>28</em> (1), 112–118. <a href="https://doi.org/10.1093/bioinformatics/btr597">https://doi.org/10.1093/bioinformatics/btr597</a>.</div>
</div>
<div id="ref-gcRMhkMM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(720) </div><div class="csl-right-inline">Troyanskaya, O.; Cantor, M.; Sherlock, G.; Brown, P.; Hastie, T.; Tibshirani, R.; Botstein, D.; Altman, R. B. Missing Value Estimation Methods for DNA Microarrays. <em>Bioinformatics</em> <strong>2001</strong>, <em>17</em> (6), 520–525. <a href="https://doi.org/10.1093/bioinformatics/17.6.520">https://doi.org/10.1093/bioinformatics/17.6.520</a>.</div>
</div>
<div id="ref-PuoOF9VV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(721) </div><div class="csl-right-inline">Stacklies, W.; Redestig, H.; Scholz, M.; Walther, D.; Selbig, J. pcaMethods—a Bioconductor Package Providing PCA Methods for Incomplete Data. <em>Bioinformatics</em> <strong>2007</strong>, <em>23</em> (9), 1164–1167. <a href="https://doi.org/10.1093/bioinformatics/btm069">https://doi.org/10.1093/bioinformatics/btm069</a>.</div>
</div>
<div id="ref-7LvdRbeH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(722) </div><div class="csl-right-inline">Wei, R.; Wang, J.; Su, M.; Jia, E.; Chen, S.; Chen, T.; Ni, Y. Missing Value Imputation Approach for Mass Spectrometry-Based Metabolomics Data. <em>Sci Rep</em> <strong>2018</strong>, <em>8</em> (1). <a href="https://doi.org/10.1038/s41598-017-19120-0">https://doi.org/10.1038/s41598-017-19120-0</a>.</div>
</div>
<div id="ref-l8NlGvoh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(723) </div><div class="csl-right-inline">Kokla, M.; Virtanen, J.; Kolehmainen, M.; Paananen, J.; Hanhineva, K. Random Forest-Based Imputation Outperforms Other Methods for Imputing LC-MS Metabolomics Data: A Comparative Study. <em>BMC Bioinformatics</em> <strong>2019</strong>, <em>20</em> (1). <a href="https://doi.org/10.1186/s12859-019-3110-0">https://doi.org/10.1186/s12859-019-3110-0</a>.</div>
</div>
<div id="ref-ZwtivmR2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(724) </div><div class="csl-right-inline">Jin, L.; Bi, Y.; Hu, C.; Qu, J.; Shen, S.; Wang, X.; Tian, Y. A Comparative Study of Evaluating Missing Value Imputation Methods in Label-Free Proteomics. <em>Sci Rep</em> <strong>2021</strong>, <em>11</em> (1). <a href="https://doi.org/10.1038/s41598-021-81279-4">https://doi.org/10.1038/s41598-021-81279-4</a>.</div>
</div>
<div id="ref-142g5e11W" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(725) </div><div class="csl-right-inline">Goh, W. W. B.; Wang, W.; Wong, L. Why Batch Effects Matter in Omics Data, and How to Avoid Them. <em>Trends in Biotechnology</em> <strong>2017</strong>, <em>35</em> (6), 498–507. <a href="https://doi.org/10.1016/j.tibtech.2017.02.012">https://doi.org/10.1016/j.tibtech.2017.02.012</a>.</div>
</div>
<div id="ref-mPnIAH38" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(726) </div><div class="csl-right-inline">Leek, J. T.; Scharpf, R. B.; Bravo, H. C.; Simcha, D.; Langmead, B.; Johnson, W. E.; Geman, D.; Baggerly, K.; Irizarry, R. A. Tackling the Widespread and Critical Impact of Batch Effects in High-Throughput Data. <em>Nat Rev Genet</em> <strong>2010</strong>, <em>11</em> (10), 733–739. <a href="https://doi.org/10.1038/nrg2825">https://doi.org/10.1038/nrg2825</a>.</div>
</div>
<div id="ref-10L8J7McB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(727) </div><div class="csl-right-inline">Hu, J. The Importance of Experimental Design in Proteomic Mass Spectrometry Experiments: Some Cautionary Tales. <em>Briefings in Functional Genomics and Proteomics</em> <strong>2005</strong>, <em>3</em> (4), 322–331. <a href="https://doi.org/10.1093/bfgp/3.4.322">https://doi.org/10.1093/bfgp/3.4.322</a>.</div>
</div>
<div id="ref-1HahRBkyb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(728) </div><div class="csl-right-inline">Johnson, W. E.; Li, C.; Rabinovic, A. Adjusting Batch Effects in Microarray Expression Data Using Empirical Bayes Methods. <em>Biostatistics</em> <strong>2006</strong>, <em>8</em> (1), 118–127. <a href="https://doi.org/10.1093/biostatistics/kxj037">https://doi.org/10.1093/biostatistics/kxj037</a>.</div>
</div>
<div id="ref-cna9kCVu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(729) </div><div class="csl-right-inline">Chen, C.; Grennan, K.; Badner, J.; Zhang, D.; Gershon, E.; Jin, L.; Liu, C. Removing Batch Effects in Analysis of Expression Microarray Data: An Evaluation of Six Batch Adjustment Methods. <em>PLoS ONE</em> <strong>2011</strong>, <em>6</em> (2), e17238. <a href="https://doi.org/10.1371/journal.pone.0017238">https://doi.org/10.1371/journal.pone.0017238</a>.</div>
</div>
<div id="ref-O7sERbhY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(730) </div><div class="csl-right-inline">Gagnon-Bartsch, J. A.; Speed, T. P. Using Control Genes to Correct for Unwanted Variation in Microarray Data. <em>Biostatistics</em> <strong>2011</strong>, <em>13</em> (3), 539–552. <a href="https://doi.org/10.1093/biostatistics/kxr034">https://doi.org/10.1093/biostatistics/kxr034</a>.</div>
</div>
<div id="ref-kILCvszy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(731) </div><div class="csl-right-inline">Haghverdi, L.; Lun, A. T. L.; Morgan, M. D.; Marioni, J. C. Batch Effects in Single-Cell RNA-Sequencing Data Are Corrected by Matching Mutual Nearest Neighbors. <em>Nat Biotechnol</em> <strong>2018</strong>, <em>36</em> (5), 421–427. <a href="https://doi.org/10.1038/nbt.4091">https://doi.org/10.1038/nbt.4091</a>.</div>
</div>
<div id="ref-12Z3TdwON" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(732) </div><div class="csl-right-inline">Hie, B.; Bryson, B.; Berger, B. Efficient Integration of Heterogeneous Single-Cell Transcriptomes Using Scanorama. <em>Nat Biotechnol</em> <strong>2019</strong>, <em>37</em> (6), 685–691. <a href="https://doi.org/10.1038/s41587-019-0113-3">https://doi.org/10.1038/s41587-019-0113-3</a>.</div>
</div>
<div id="ref-98DcNOmC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(733) </div><div class="csl-right-inline">Schessner, J. P.; Voytik, E.; Bludau, I. A Practical Guide to Interpreting and Generating Bottom‐up Proteomics Data Visualizations. <em>Proteomics</em> <strong>2022</strong>, <em>22</em> (8). <a href="https://doi.org/10.1002/pmic.202100103">https://doi.org/10.1002/pmic.202100103</a>.</div>
</div>
<div id="ref-1GJKaTKqC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(734) </div><div class="csl-right-inline">Mann, M.; Kumar, C.; Zeng, W.-F.; Strauss, M. T. Artificial Intelligence for Proteomics and Biomarker Discovery. <em>Cell Systems</em> <strong>2021</strong>, <em>12</em> (8), 759–770. <a href="https://doi.org/10.1016/j.cels.2021.06.006">https://doi.org/10.1016/j.cels.2021.06.006</a>.</div>
</div>
<div id="ref-IGJNx1OA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(735) </div><div class="csl-right-inline">Swan, A. L.; Mobasheri, A.; Allaway, D.; Liddell, S.; Bacardit, J. Application of Machine Learning to Proteomics Data: Classification and Biomarker Identification in Postgenomics Biology. <em>OMICS: A Journal of Integrative Biology</em> <strong>2013</strong>, <em>17</em> (12), 595–610. <a href="https://doi.org/10.1089/omi.2013.0017">https://doi.org/10.1089/omi.2013.0017</a>.</div>
</div>
<div id="ref-LCdBFAHO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(736) </div><div class="csl-right-inline">Hernández, B.; Pennington, S. R.; Parnell, A. C. Bayesian Methods for Proteomic Biomarker Development. <em>EuPA Open Proteomics</em> <strong>2015</strong>, <em>9</em>, 54–64. <a href="https://doi.org/10.1016/j.euprot.2015.08.001">https://doi.org/10.1016/j.euprot.2015.08.001</a>.</div>
</div>
<div id="ref-ZhPBaP2m" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(737) </div><div class="csl-right-inline">Kuschner, K. W.; Malyarenko, D. I.; Cooke, W. E.; Cazares, L. H.; Semmes, O.; Tracy, E. R. A Bayesian Network Approach to Feature Selection in Mass Spectrometry Data. <em>BMC Bioinformatics</em> <strong>2010</strong>, <em>11</em> (1). <a href="https://doi.org/10.1186/1471-2105-11-177">https://doi.org/10.1186/1471-2105-11-177</a>.</div>
</div>
<div id="ref-1KFEQehj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(738) </div><div class="csl-right-inline">Desaire, H.; Go, E. P.; Hua, D. Advances, Obstacles, and Opportunities for Machine Learning in Proteomics. <em>Cell Reports Physical Science</em> <strong>2022</strong>, <em>3</em> (10), 101069. <a href="https://doi.org/10.1016/j.xcrp.2022.101069">https://doi.org/10.1016/j.xcrp.2022.101069</a>.</div>
</div>
<div id="ref-7uMxquUf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(739) </div><div class="csl-right-inline">Hicks, S. A.; Strümke, I.; Thambawita, V.; Hammou, M.; Riegler, M. A.; Halvorsen, P.; Parasa, S. On Evaluation Metrics for Medical Applications of Artificial Intelligence. <em>Sci Rep</em> <strong>2022</strong>, <em>12</em> (1). <a href="https://doi.org/10.1038/s41598-022-09954-8">https://doi.org/10.1038/s41598-022-09954-8</a>.</div>
</div>
<div id="ref-dqOuwBB4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(740) </div><div class="csl-right-inline">Khan, M. J.; Desaire, H.; Lopez, O. L.; Kamboh, M. I.; Robinson, R. A. S. Why Inclusion Matters for Alzheimer’s Disease Biomarker Discovery in Plasma. <em>JAD</em> <strong>2021</strong>, <em>79</em> (3), 1327–1344. <a href="https://doi.org/10.3233/jad-201318">https://doi.org/10.3233/jad-201318</a>.</div>
</div>
<div id="ref-LamN4xBz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(741) </div><div class="csl-right-inline">Desaire, H. How (Not) to Generate a Highly Predictive Biomarker Panel Using Machine Learning. <em>J. Proteome Res.</em> <strong>2022</strong>, <em>21</em> (9), 2071–2074. <a href="https://doi.org/10.1021/acs.jproteome.2c00117">https://doi.org/10.1021/acs.jproteome.2c00117</a>.</div>
</div>
<div id="ref-wJCp8LYF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(742) </div><div class="csl-right-inline">Ransohoff, D. F. Rules of Evidence for Cancer Molecular-Marker Discovery and Validation. <em>Nat Rev Cancer</em> <strong>2004</strong>, <em>4</em> (4), 309–314. <a href="https://doi.org/10.1038/nrc1322">https://doi.org/10.1038/nrc1322</a>.</div>
</div>
<div id="ref-16jVcFgBv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(743) </div><div class="csl-right-inline">Kim, M.; Hwang, K.-B. An Empirical Evaluation of Sampling Methods for the Classification of Imbalanced Data. <em>PLoS ONE</em> <strong>2022</strong>, <em>17</em> (7), e0271260. <a href="https://doi.org/10.1371/journal.pone.0271260">https://doi.org/10.1371/journal.pone.0271260</a>.</div>
</div>
<div id="ref-65gghDCg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(744) </div><div class="csl-right-inline">van den Goorbergh, R.; van Smeden, M.; Timmerman, D.; Van Calster, B. The Harm of Class Imbalance Corrections for Risk Prediction Models: Illustration and Simulation Using Logistic Regression. <em>Journal of the American Medical Informatics Association</em> <strong>2022</strong>, <em>29</em> (9), 1525–1534. <a href="https://doi.org/10.1093/jamia/ocac093">https://doi.org/10.1093/jamia/ocac093</a>.</div>
</div>
<div id="ref-4k6ers10" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(745) </div><div class="csl-right-inline">Movahedi, F.; Antaki, J. F. Limitation of ROC in Evaluation of Classifiers for Imbalanced Data. <em>The Journal of Heart and Lung Transplantation</em> <strong>2021</strong>, <em>40</em> (4), S413. <a href="https://doi.org/10.1016/j.healun.2021.01.1160">https://doi.org/10.1016/j.healun.2021.01.1160</a>.</div>
</div>
<div id="ref-LB4mDas6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(746) </div><div class="csl-right-inline">Saito, T.; Rehmsmeier, M. The Precision-Recall Plot Is More Informative Than the ROC Plot When Evaluating Binary Classifiers on Imbalanced Datasets. <em>PLoS ONE</em> <strong>2015</strong>, <em>10</em> (3), e0118432. <a href="https://doi.org/10.1371/journal.pone.0118432">https://doi.org/10.1371/journal.pone.0118432</a>.</div>
</div>
<div id="ref-KDnDuk25" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(747) </div><div class="csl-right-inline">Holt, C.; Yandell, M. MAKER2: An Annotation Pipeline and Genome-Database Management Tool for Second-Generation Genome Projects. <em>BMC Bioinformatics</em> <strong>2011</strong>, <em>12</em>, 491. <a href="https://doi.org/10.1186/1471-2105-12-491">https://doi.org/10.1186/1471-2105-12-491</a>.</div>
</div>
<div id="ref-12mAW7BXf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(748) </div><div class="csl-right-inline"><em>The NCBI Eukaryotic Genome Annotation Pipeline</em>. <a href="https://www.ncbi.nlm.nih.gov/refseq/annotation_euk/process/">https://www.ncbi.nlm.nih.gov/refseq/annotation_euk/process/</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-BRlMNUOW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(749) </div><div class="csl-right-inline"><em>UniProt</em>. <a href="https://www.uniprot.org/help/biocuration">https://www.uniprot.org/help/biocuration</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-1EbCD9r1v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(750) </div><div class="csl-right-inline">. <a href="https://www.psidev.info/sites/default/files/2018-03/MIAPE_MSI_1.1.pdf">https://www.psidev.info/sites/default/files/2018-03/MIAPE_MSI_1.1.pdf</a>.</div>
</div>
<div id="ref-paTtvUb8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(751) </div><div class="csl-right-inline">Martínez-Bartolomé, S.; Deutsch, E. W.; Binz, P.-A.; Jones, A. R.; Eisenacher, M.; Mayer, G.; Campos, A.; Canals, F.; Bech-Serra, J.-J.; Carrascal, M.; Gay, M.; Paradela, A.; Navajas, R.; Marcilla, M.; Hernáez, M. L.; Gutiérrez-Blázquez, M. D.; Velarde, L. F. C.; Aloria, K.; Beaskoetxea, J.; Medina-Aunon, J. A.; Albar, J. P. Guidelines for Reporting Quantitative Mass Spectrometry Based Experiments in Proteomics. <em>J Proteomics</em> <strong>2013</strong>, <em>95</em>, 84–88. <a href="https://doi.org/10.1016/j.jprot.2013.02.026">https://doi.org/10.1016/j.jprot.2013.02.026</a>.</div>
</div>
<div id="ref-16uefmiYk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(752) </div><div class="csl-right-inline">Apweiler, R.; Bairoch, A.; Wu, C. H.; Barker, W. C.; Boeckmann, B.; Ferro, S.; Gasteiger, E.; Huang, H.; Lopez, R.; Magrane, M.; Martin, M. J.; Natale, D. A.; O'Donovan, C.; Redaschi, N.; Yeh, L.-S. L. UniProt: The Universal Protein Knowledgebase. <em>Nucleic Acids Res</em> <strong>2004</strong>, <em>32</em> (Database issue), D115–9. <a href="https://doi.org/10.1093/nar/gkh131">https://doi.org/10.1093/nar/gkh131</a>.</div>
</div>
<div id="ref-nULTgvHl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(753) </div><div class="csl-right-inline">Lussi, Y. C.; Magrane, M.; Martin, M. J.; Orchard, S. Searching and Navigating UniProt Databases. <em>Current Protocols</em> <strong>2023</strong>, <em>3</em> (3). <a href="https://doi.org/10.1002/cpz1.700">https://doi.org/10.1002/cpz1.700</a>.</div>
</div>
<div id="ref-vPdD3ylm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(754) </div><div class="csl-right-inline">Bowler-Barnett, E. H.; Fan, J.; Luo, J.; Magrane, M.; Martin, M. J.; Orchard, S. UniProt and Mass Spectrometry-Based Proteomics—A 2-Way Working Relationship. <em>Molecular &amp;amp; Cellular Proteomics</em> <strong>2023</strong>, <em>22</em> (8), 100591. <a href="https://doi.org/10.1016/j.mcpro.2023.100591">https://doi.org/10.1016/j.mcpro.2023.100591</a>.</div>
</div>
<div id="ref-1ENygunzG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(755) </div><div class="csl-right-inline">; Bateman, A.; Martin, M.-J.; Orchard, S.; Magrane, M.; Ahmad, S.; Alpi, E.; Bowler-Barnett, E. H.; Britto, R.; Bye-A-Jee, H.; Cukura, A.; Denny, P.; Dogan, T.; Ebenezer, T.; Fan, J.; Garmiri, P.; da Costa Gonzales, L. J.; Hatton-Ellis, E.; Hussein, A.; Ignatchenko, A.; Insana, G.; Ishtiaq, R.; Joshi, V.; Jyothi, D.; Kandasaamy, S.; Lock, A.; Luciani, A.; Lugaric, M.; Luo, J.; Lussi, Y.; MacDougall, A.; Madeira, F.; Mahmoudy, M.; Mishra, A.; Moulang, K.; Nightingale, A.; Pundir, S.; Qi, G.; Raj, S.; Raposo, P.; Rice, D. L.; Saidi, R.; Santos, R.; Speretta, E.; Stephenson, J.; Totoo, P.; Turner, E.; Tyagi, N.; Vasudev, P.; Warner, K.; Watkins, X.; Zaru, R.; Zellner, H.; Bridge, A. J.; Aimo, L.; Argoud-Puy, G.; Auchincloss, A. H.; Axelsen, K. B.; Bansal, P.; Baratin, D.; Batista Neto, T. M.; Blatter, M.-C.; Bolleman, J. T.; Boutet, E.; Breuza, L.; Gil, B. C.; Casals-Casas, C.; Echioukh, K. C.; Coudert, E.; Cuche, B.; de Castro, E.; Estreicher, A.; Famiglietti, M. L.; Feuermann, M.; Gasteiger, E.; Gaudet, P.; Gehant, S.; Gerritsen, V.; Gos, A.; Gruaz, N.; Hulo, C.; Hyka-Nouspikel, N.; Jungo, F.; Kerhornou, A.; Le Mercier, P.; Lieberherr, D.; Masson, P.; Morgat, A.; Muthukrishnan, V.; Paesano, S.; Pedruzzi, I.; Pilbout, S.; Pourcel, L.; Poux, S.; Pozzato, M.; Pruess, M.; Redaschi, N.; Rivoire, C.; Sigrist, C. J. A.; Sonesson, K.; Sundaram, S.; Wu, C. H.; Arighi, C. N.; Arminski, L.; Chen, C.; Chen, Y.; Huang, H.; Laiho, K.; McGarvey, P.; Natale, D. A.; Ross, K.; Vinayaka, C. R.; Wang, Q.; Wang, Y.; Zhang, J. UniProt: The Universal Protein Knowledgebase in 2023. <em>Nucleic Acids Research</em> <strong>2022</strong>, <em>51</em> (D1), D523–D531. <a href="https://doi.org/10.1093/nar/gkac1052">https://doi.org/10.1093/nar/gkac1052</a>.</div>
</div>
<div id="ref-1BIF8v4YE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(756) </div><div class="csl-right-inline"><em>UniProt</em>. <a href="https://www.uniprot.org/help/sequence_origin">https://www.uniprot.org/help/sequence_origin</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-Q2scu90J" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(757) </div><div class="csl-right-inline"><em>UniProt</em>. <a href="https://www.uniprot.org/help/manual_curation">https://www.uniprot.org/help/manual_curation</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-16BHXvDGS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(758) </div><div class="csl-right-inline">Wilmarth, P. Pwilmart/Fasta_utilities, 2023. <a href="https://github.com/pwilmart/fasta_utilities">https://github.com/pwilmart/fasta_utilities</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-1GfGl6YGi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(759) </div><div class="csl-right-inline">Shortreed, M. R.; Wenger, C. D.; Frey, B. L.; Sheynkman, G. M.; Scalf, M.; Keller, M. P.; Attie, A. D.; Smith, L. M. Global Identification of Protein Post-Translational Modifications in a Single-Pass Database Search. <em>J Proteome Res</em> <strong>2015</strong>, <em>14</em> (11), 4714–4720. <a href="https://doi.org/10.1021/acs.jproteome.5b00599">https://doi.org/10.1021/acs.jproteome.5b00599</a>.</div>
</div>
<div id="ref-uPHDl2Vm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(760) </div><div class="csl-right-inline"><em>Datasets</em>. NCBI. <a href="https://www.ncbi.nlm.nih.gov/datasets">https://www.ncbi.nlm.nih.gov/datasets</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-tg8zlLqI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(761) </div><div class="csl-right-inline"><em>Eukaryotic genomes annotated at NCBI</em>. <a href="https://www.ncbi.nlm.nih.gov/refseq/annotation_euk/all/">https://www.ncbi.nlm.nih.gov/refseq/annotation_euk/all/</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-xpZWsuGI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(762) </div><div class="csl-right-inline"><em>Prokaryotic RefSeq Genomes</em>. <a href="https://www.ncbi.nlm.nih.gov/refseq/about/prokaryotes/">https://www.ncbi.nlm.nih.gov/refseq/about/prokaryotes/</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-BfmLo0uT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(763) </div><div class="csl-right-inline"><em>Eukaryotic RefSeq Genome Annotation Status</em>. <a href="https://www.ncbi.nlm.nih.gov/refseq/annotation_euk/status/">https://www.ncbi.nlm.nih.gov/refseq/annotation_euk/status/</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-fWjGnKWy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(764) </div><div class="csl-right-inline"><em>Training &amp; Tutorials - Site Guide - NCBI</em>. <a href="https://www.ncbi.nlm.nih.gov/guide/training-tutorials/">https://www.ncbi.nlm.nih.gov/guide/training-tutorials/</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-17gB3bs6x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(765) </div><div class="csl-right-inline">Heck, M.; Neely, B. A. Proteomics in Non-Model Organisms: A New Analytical Frontier. <em>J Proteome Res</em> <strong>2020</strong>, <em>19</em> (9), 3595–3606. <a href="https://doi.org/10.1021/acs.jproteome.0c00448">https://doi.org/10.1021/acs.jproteome.0c00448</a>.</div>
</div>
<div id="ref-69obv795" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(766) </div><div class="csl-right-inline"><em>Ensembl genome browser 111</em>. <a href="https://useast.ensembl.org/index.html">https://useast.ensembl.org/index.html</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-1EGyqGdLr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(767) </div><div class="csl-right-inline"><em>Ensembl Genomes</em>. <a href="http://ensemblgenomes.org/">http://ensemblgenomes.org/</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-OeXy2hrh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(768) </div><div class="csl-right-inline">Keller, B. O.; Sui, J.; Young, A. B.; Whittal, R. M. Interferences and Contaminants Encountered in Modern Mass Spectrometry. <em>Anal Chim Acta</em> <strong>2008</strong>, <em>627</em> (1), 71–81. <a href="https://doi.org/10.1016/j.aca.2008.04.043">https://doi.org/10.1016/j.aca.2008.04.043</a>.</div>
</div>
<div id="ref-jwgBjKOg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(769) </div><div class="csl-right-inline"><em>cRAP protein sequences</em>. <a href="https://www.thegpm.org/crap/">https://www.thegpm.org/crap/</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-WdwXXrRx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(770) </div><div class="csl-right-inline">da Veiga Leprevost, F.; Haynes, S. E.; Avtonomov, D. M.; Chang, H.-Y.; Shanmugam, A. K.; Mellacheruvu, D.; Kong, A. T.; Nesvizhskii, A. I. Philosopher: A Versatile Toolkit for Shotgun Proteomics Data Analysis. <em>Nat Methods</em> <strong>2020</strong>, <em>17</em> (9), 869–870. <a href="https://doi.org/10.1038/s41592-020-0912-y">https://doi.org/10.1038/s41592-020-0912-y</a>.</div>
</div>
<div id="ref-m8bjbzJ5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(771) </div><div class="csl-right-inline">Frankenfield, A. M.; Ni, J.; Ahmed, M.; Hao, L. Protein Contaminants Matter: Building Universal Protein Contaminant Libraries for DDA and DIA Proteomics, 2022. <a href="https://doi.org/10.1101/2022.04.27.489766">https://doi.org/10.1101/2022.04.27.489766</a>.</div>
</div>
<div id="ref-CnwgtVyY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(772) </div><div class="csl-right-inline">Bunkenborg, J.; García, G. E.; Paz, M. I. P.; Andersen, J. S.; Molina, H. The Minotaur Proteome: Avoiding Cross-Species Identifications Deriving from Bovine Serum in Cell Culture Models. <em>Proteomics</em> <strong>2010</strong>, <em>10</em> (16), 3040–3044. <a href="https://doi.org/10.1002/pmic.201000103">https://doi.org/10.1002/pmic.201000103</a>.</div>
</div>
<div id="ref-1CIFGLtYQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(773) </div><div class="csl-right-inline">Lehrich, B. M.; Liang, Y.; Fiandaca, M. S. Foetal Bovine Serum Influence on in Vitro Extracellular Vesicle Analyses. <em>J Extracell Vesicles</em> <strong>2021</strong>, <em>10</em> (3), e12061. <a href="https://doi.org/10.1002/jev2.12061">https://doi.org/10.1002/jev2.12061</a>.</div>
</div>
<div id="ref-wUllSafI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(774) </div><div class="csl-right-inline">Mellacheruvu, D.; Wright, Z.; Couzens, A. L.; Lambert, J.-P.; St-Denis, N. A.; Li, T.; Miteva, Y. V.; Hauri, S.; Sardiu, M. E.; Low, T. Y.; Halim, V. A.; Bagshaw, R. D.; Hubner, N. C.; Al-Hakim, A.; Bouchard, A.; Faubert, D.; Fermin, D.; Dunham, W. H.; Goudreault, M.; Lin, Z.-Y.; Badillo, B. G.; Pawson, T.; Durocher, D.; Coulombe, B.; Aebersold, R.; Superti-Furga, G.; Colinge, J.; Heck, A. J. R.; Choi, H.; Gstaiger, M.; Mohammed, S.; Cristea, I. M.; Bennett, K. L.; Washburn, M. P.; Raught, B.; Ewing, R. M.; Gingras, A.-C.; Nesvizhskii, A. I. The CRAPome: A Contaminant Repository for Affinity Purification-Mass Spectrometry Data. <em>Nat Methods</em> <strong>2013</strong>, <em>10</em> (8), 730–736. <a href="https://doi.org/10.1038/nmeth.2557">https://doi.org/10.1038/nmeth.2557</a>.</div>
</div>
<div id="ref-EyM6Y0Xm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(775) </div><div class="csl-right-inline">Shin, J.; Kwon, Y.; Lee, S.; Na, S.; Hong, E. Y.; Ju, S.; Jung, H.-G.; Kaushal, P.; Shin, S.; Back, J. H.; Choi, S. Y.; Kim, E. H.; Lee, S. J.; Park, Y. E.; Ahn, H.-S.; Ahn, Y.; Kabir, M. H.; Park, S.-J.; Yang, W. S.; Yeom, J.; Bang, O. Y.; Ha, C.-W.; Lee, J.-W.; Kang, U.-B.; Kim, H.-J.; Park, K.-S.; Lee, J. E.; Lee, J. E.; Kim, J. Y.; Kim, K. P.; Kim, Y.; Hirano, H.; Yi, E. C.; Cho, J.-Y.; Paek, E.; Lee, C. Common Repository of FBS Proteins (cRFP) To Be Added to a Search Database for Mass Spectrometric Analysis of Cell Secretome. <em>J Proteome Res</em> <strong>2019</strong>, <em>18</em> (10), 3800–3806. <a href="https://doi.org/10.1021/acs.jproteome.9b00475">https://doi.org/10.1021/acs.jproteome.9b00475</a>.</div>
</div>
<div id="ref-qgZxt4hu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(776) </div><div class="csl-right-inline">Lin, A.; Plubell, D. L.; Keich, U.; Noble, W. S. Accurately Assigning Peptides to Spectra When Only a Subset of Peptides Are Relevant. <em>J Proteome Res</em> <strong>2021</strong>, <em>20</em> (8), 4153–4164. <a href="https://doi.org/10.1021/acs.jproteome.1c00483">https://doi.org/10.1021/acs.jproteome.1c00483</a>.</div>
</div>
<div id="ref-12o52Fhcj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(777) </div><div class="csl-right-inline">Keich, U.; Tamura, K.; Noble, W. S. Averaging Strategy To Reduce Variability in Target-Decoy Estimates of False Discovery Rate. <em>J Proteome Res</em> <strong>2019</strong>, <em>18</em> (2), 585–593. <a href="https://doi.org/10.1021/acs.jproteome.8b00802">https://doi.org/10.1021/acs.jproteome.8b00802</a>.</div>
</div>
<div id="ref-Lovg2liS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(778) </div><div class="csl-right-inline">Noble, W. S. Mass Spectrometrists Should Search Only for Peptides They Care About. <em>Nat Methods</em> <strong>2015</strong>, <em>12</em> (7), 605–608. <a href="https://doi.org/10.1038/nmeth.3450">https://doi.org/10.1038/nmeth.3450</a>.</div>
</div>
<div id="ref-nMWQwf9T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(779) </div><div class="csl-right-inline">Kumar, D.; Yadav, A. K.; Dash, D. Choosing an Optimal Database for Protein Identification from Tandem Mass Spectrometry Data. <em>Methods Mol Biol</em> <strong>2017</strong>, <em>1549</em>, 17–29. <a href="https://doi.org/10.1007/978-1-4939-6740-7_3">https://doi.org/10.1007/978-1-4939-6740-7_3</a>.</div>
</div>
<div id="ref-19xkjz9Ak" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(780) </div><div class="csl-right-inline">Jones, P. PRIDE: A Public Repository of Protein and Peptide Identifications for the Proteomics Community. <em>Nucleic Acids Research</em> <strong>2006</strong>, <em>34</em> (90001), D659–D663. <a href="https://doi.org/10.1093/nar/gkj138">https://doi.org/10.1093/nar/gkj138</a>.</div>
</div>
<div id="ref-jpihB8r0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(781) </div><div class="csl-right-inline">Perez-Riverol, Y.; Bai, J.; Bandla, C.; García-Seisdedos, D.; Hewapathirana, S.; Kamatchinathan, S.; Kundu, D.; Prakash, A.; Frericks-Zipper, A.; Eisenacher, M.; Walzer, M.; Wang, S.; Brazma, A.; Vizcaíno, J. The PRIDE Database Resources in 2022: A Hub for Mass Spectrometry-Based Proteomics Evidences. <em>Nucleic Acids Research</em> <strong>2021</strong>, <em>50</em> (D1), D543–D552. <a href="https://doi.org/10.1093/nar/gkab1038">https://doi.org/10.1093/nar/gkab1038</a>.</div>
</div>
<div id="ref-BeY3BmUJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(782) </div><div class="csl-right-inline"><em>Chorus - Home</em>. <a href="https://chorusproject.org/pages/index.html">https://chorusproject.org/pages/index.html</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-15qh1BI32" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(783) </div><div class="csl-right-inline">Deutsch, E. W.; Eng, J. K.; Zhang, H.; King, N. L.; Nesvizhskii, A. I.; Lin, B.; Lee, H.; Yi, E. C.; Ossola, R.; Aebersold, R. Human Plasma PeptideAtlas. <em>Proteomics</em> <strong>2005</strong>, <em>5</em> (13), 3497–3500. <a href="https://doi.org/10.1002/pmic.200500160">https://doi.org/10.1002/pmic.200500160</a>.</div>
</div>
<div id="ref-FkUJzost" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(784) </div><div class="csl-right-inline">Desiere, F.; Deutsch, E. W.; King, N. L.; Nesvizhskii, A. I.; Mallick, P.; Eng, J.; Chen, S.; Eddes, J.; Loevenich, S. N.; Aebersold, R. The PeptideAtlas Project. <em>Nucleic Acids Res</em> <strong>2006</strong>, <em>34</em> (Database issue), D655–8. <a href="https://doi.org/10.1093/nar/gkj040">https://doi.org/10.1093/nar/gkj040</a>.</div>
</div>
<div id="ref-Z2Ay0IqK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(785) </div><div class="csl-right-inline">Farrah, T.; Deutsch, E. W.; Kreisberg, R.; Sun, Z.; Campbell, D. S.; Mendoza, L.; Kusebauch, U.; Brusniak, M.-Y.; Hüttenhain, R.; Schiess, R.; Selevsek, N.; Aebersold, R.; Moritz, R. L. PASSEL: The PeptideAtlas SRMexperiment Library. <em>Proteomics</em> <strong>2012</strong>, <em>12</em> (8), 1170–1175. <a href="https://doi.org/10.1002/pmic.201100515">https://doi.org/10.1002/pmic.201100515</a>.</div>
</div>
<div id="ref-CJuU986M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(786) </div><div class="csl-right-inline">Kusebauch, U.; Deutsch, E. W.; Campbell, D. S.; Sun, Z.; Farrah, T.; Moritz, R. L. Using PeptideAtlas, SRMAtlas, and PASSEL: Comprehensive Resources for Discovery and Targeted Proteomics. <em>Curr Protoc Bioinformatics</em> <strong>2014</strong>, <em>46</em>, 13.25.1–13.25.28. <a href="https://doi.org/10.1002/0471250953.bi1325s46">https://doi.org/10.1002/0471250953.bi1325s46</a>.</div>
</div>
<div id="ref-17qvo6lpw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(787) </div><div class="csl-right-inline">Kusebauch, U.; Campbell, D. S.; Deutsch, E. W.; Chu, C. S.; Spicer, D. A.; Brusniak, M.-Y.; Slagel, J.; Sun, Z.; Stevens, J.; Grimes, B.; Shteynberg, D.; Hoopmann, M. R.; Blattmann, P.; Ratushny, A. V.; Rinner, O.; Picotti, P.; Carapito, C.; Huang, C.-Y.; Kapousouz, M.; Lam, H.; Tran, T.; Demir, E.; Aitchison, J. D.; Sander, C.; Hood, L.; Aebersold, R.; Moritz, R. L. Human SRMAtlas: A Resource of Targeted Assays to Quantify the Complete Human Proteome. <em>Cell</em> <strong>2016</strong>, <em>166</em> (3), 766–778. <a href="https://doi.org/10.1016/j.cell.2016.06.041">https://doi.org/10.1016/j.cell.2016.06.041</a>.</div>
</div>
<div id="ref-qVEHb5K" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(788) </div><div class="csl-right-inline">Deutsch, E. W.; Orchard, S.; Binz, P.-A.; Bittremieux, W.; Eisenacher, M.; Hermjakob, H.; Kawano, S.; Lam, H.; Mayer, G.; Menschaert, G.; Perez-Riverol, Y.; Salek, R. M.; Tabb, D. L.; Tenzer, S.; Vizcaíno, J. A.; Walzer, M.; Jones, A. R. Proteomics Standards Initiative: Fifteen Years of Progress and Future Work. <em>J. Proteome Res.</em> <strong>2017</strong>, <em>16</em> (12), 4288–4298. <a href="https://doi.org/10.1021/acs.jproteome.7b00370">https://doi.org/10.1021/acs.jproteome.7b00370</a>.</div>
</div>
<div id="ref-2iJj61wl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(789) </div><div class="csl-right-inline">Zolg, D. P.; Wilhelm, M.; Schnatbaum, K.; Zerweck, J.; Knaute, T.; Delanghe, B.; Bailey, D. J.; Gessulat, S.; Ehrlich, H.-C.; Weininger, M.; Yu, P.; Schlegl, J.; Kramer, K.; Schmidt, T.; Kusebauch, U.; Deutsch, E. W.; Aebersold, R.; Moritz, R. L.; Wenschuh, H.; Moehring, T.; Aiche, S.; Huhmer, A.; Reimer, U.; Kuster, B. Building ProteomeTools Based on a Complete Synthetic Human Proteome. <em>Nat Methods</em> <strong>2017</strong>, <em>14</em> (3), 259–262. <a href="https://doi.org/10.1038/nmeth.4153">https://doi.org/10.1038/nmeth.4153</a>.</div>
</div>
<div id="ref-LMs52Uzn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(790) </div><div class="csl-right-inline">Ma, J.; Chen, T.; Wu, S.; Yang, C.; Bai, M.; Shu, K.; Li, K.; Zhang, G.; Jin, Z.; He, F.; Hermjakob, H.; Zhu, Y. iProX: An Integrated Proteome Resource. <em>Nucleic Acids Research</em> <strong>2018</strong>, <em>47</em> (D1), D1211–D1217. <a href="https://doi.org/10.1093/nar/gky869">https://doi.org/10.1093/nar/gky869</a>.</div>
</div>
<div id="ref-rKWchS8R" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(791) </div><div class="csl-right-inline">Sharma, V.; Eckels, J.; Taylor, G. K.; Shulman, N. J.; Stergachis, A. B.; Joyner, S. A.; Yan, P.; Whiteaker, J. R.; Halusa, G. N.; Schilling, B.; Gibson, B. W.; Colangelo, C. M.; Paulovich, A. G.; Carr, S. A.; Jaffe, J. D.; MacCoss, M. J.; MacLean, B. Panorama: A Targeted Proteomics Knowledge Base. <em>J. Proteome Res.</em> <strong>2014</strong>, <em>13</em> (9), 4205–4210. <a href="https://doi.org/10.1021/pr5006636">https://doi.org/10.1021/pr5006636</a>.</div>
</div>
<div id="ref-43dscula" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(792) </div><div class="csl-right-inline">Low, T. Y.; Syafruddin, S. E.; Mohtar, M. A.; Vellaichamy, A.; A Rahman, N. S.; Pung, Y.-F.; Tan, C. S. H. Recent Progress in Mass Spectrometry-Based Strategies for Elucidating Protein-Protein Interactions. <em>Cell Mol Life Sci</em> <strong>2021</strong>, <em>78</em> (13), 5325–5339. <a href="https://doi.org/10.1007/s00018-021-03856-0">https://doi.org/10.1007/s00018-021-03856-0</a>.</div>
</div>
<div id="ref-xeKEaOVF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(793) </div><div class="csl-right-inline">Del Toro, N.; Shrivastava, A.; Ragueneau, E.; Meldal, B.; Combe, C.; Barrera, E.; Perfetto, L.; How, K.; Ratan, P.; Shirodkar, G.; Lu, O.; Mészáros, B.; Watkins, X.; Pundir, S.; Licata, L.; Iannuccelli, M.; Pellegrini, M.; Martin, M. J.; Panni, S.; Duesbury, M.; Vallet, S. D.; Rappsilber, J.; Ricard-Blum, S.; Cesareni, G.; Salwinski, L.; Orchard, S.; Porras, P.; Panneerselvam, K.; Hermjakob, H. The IntAct Database: Efficient Access to Fine-Grained Molecular Interaction Data. <em>Nucleic Acids Res</em> <strong>2022</strong>, <em>50</em> (D1), D648–D653. <a href="https://doi.org/10.1093/nar/gkab1006">https://doi.org/10.1093/nar/gkab1006</a>.</div>
</div>
<div id="ref-D6gNmWdp" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(794) </div><div class="csl-right-inline">Lees, J. G.; Heriche, J. K.; Morilla, I.; Ranea, J. A.; Orengo, C. A. Systematic Computational Prediction of Protein Interaction Networks. <em>Phys Biol</em> <strong>2011</strong>, <em>8</em> (3), 035008. <a href="https://doi.org/10.1088/1478-3975/8/3/035008">https://doi.org/10.1088/1478-3975/8/3/035008</a>.</div>
</div>
<div id="ref-uSY25K4H" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(795) </div><div class="csl-right-inline">Szklarczyk, D.; Gable, A. L.; Nastou, K. C.; Lyon, D.; Kirsch, R.; Pyysalo, S.; Doncheva, N. T.; Legeay, M.; Fang, T.; Bork, P.; Jensen, L. J.; von Mering, C. The STRING Database in 2021: Customizable Protein-Protein Networks, and Functional Characterization of User-Uploaded Gene/Measurement Sets. <em>Nucleic Acids Res</em> <strong>2021</strong>, <em>49</em> (D1), D605–D612. <a href="https://doi.org/10.1093/nar/gkaa1074">https://doi.org/10.1093/nar/gkaa1074</a>.</div>
</div>
<div id="ref-qwaNFUVe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(796) </div><div class="csl-right-inline">Warde-Farley, D.; Donaldson, S. L.; Comes, O.; Zuberi, K.; Badrawi, R.; Chao, P.; Franz, M.; Grouios, C.; Kazi, F.; Lopes, C. T.; Maitland, A.; Mostafavi, S.; Montojo, J.; Shao, Q.; Wright, G.; Bader, G. D.; Morris, Q. The GeneMANIA Prediction Server: Biological Network Integration for Gene Prioritization and Predicting Gene Function. <em>Nucleic Acids Research</em> <strong>2010</strong>, <em>38</em> (suppl_2), W214–W220. <a href="https://doi.org/10.1093/nar/gkq537">https://doi.org/10.1093/nar/gkq537</a>.</div>
</div>
<div id="ref-1BJixYIRV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(797) </div><div class="csl-right-inline">Shannon, P.; Markiel, A.; Ozier, O.; Baliga, N. S.; Wang, J. T.; Ramage, D.; Amin, N.; Schwikowski, B.; Ideker, T. Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks. <em>Genome Res.</em> <strong>2003</strong>, <em>13</em> (11), 2498–2504. <a href="https://doi.org/10.1101/gr.1239303">https://doi.org/10.1101/gr.1239303</a>.</div>
</div>
<div id="ref-Es98brAH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(798) </div><div class="csl-right-inline">Jalili, M.; Salehzadeh-Yazdi, A.; Gupta, S.; Wolkenhauer, O.; Yaghmaie, M.; Resendis-Antonio, O.; Alimoghaddam, K. Evolution of Centrality Measurements for the Detection of Essential Proteins in Biological Networks. <em>Front Physiol</em> <strong>2016</strong>, <em>7</em>, 375. <a href="https://doi.org/10.3389/fphys.2016.00375">https://doi.org/10.3389/fphys.2016.00375</a>.</div>
</div>
<div id="ref-awUpRsoz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(799) </div><div class="csl-right-inline">Ashtiani, M.; Salehzadeh-Yazdi, A.; Razaghi-Moghadam, Z.; Hennig, H.; Wolkenhauer, O.; Mirzaie, M.; Jafari, M. A Systematic Survey of Centrality Measures for Protein-Protein Interaction Networks. <em>BMC Syst Biol</em> <strong>2018</strong>, <em>12</em> (1), 80. <a href="https://doi.org/10.1186/s12918-018-0598-2">https://doi.org/10.1186/s12918-018-0598-2</a>.</div>
</div>
<div id="ref-12ufdjSBQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(800) </div><div class="csl-right-inline">He, X.; Zhang, J. Why Do Hubs Tend to Be Essential in Protein Networks? <em>PLoS Genet</em> <strong>2006</strong>, <em>2</em> (6), e88. <a href="https://doi.org/10.1371/journal.pgen.0020088">https://doi.org/10.1371/journal.pgen.0020088</a>.</div>
</div>
<div id="ref-1FZ0vcua3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(801) </div><div class="csl-right-inline">Hahn, M. W.; Kern, A. D. Comparative Genomics of Centrality and Essentiality in Three Eukaryotic Protein-Interaction Networks. <em>Mol Biol Evol</em> <strong>2004</strong>, <em>22</em> (4), 803–806. <a href="https://doi.org/10.1093/molbev/msi072">https://doi.org/10.1093/molbev/msi072</a>.</div>
</div>
<div id="ref-KQtBgFa0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(802) </div><div class="csl-right-inline">EMBL-EBI. <em>Closeness centrality | Network analysis of protein interaction data</em>. <a href="https://www.ebi.ac.uk/training/online/courses/network-analysis-of-protein-interaction-data-an-introduction/building-and-analysing-ppins/topological-ppin-analysis/closeness-centrality/">https://www.ebi.ac.uk/training/online/courses/network-analysis-of-protein-interaction-data-an-introduction/building-and-analysing-ppins/topological-ppin-analysis/closeness-centrality/</a> (accessed 2024-05-07).</div>
</div>
<div id="ref-1BmdBEJEB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(803) </div><div class="csl-right-inline">Yu, H.; Kim, P. M.; Sprecher, E.; Trifonov, V.; Gerstein, M. The Importance of Bottlenecks in Protein Networks: Correlation with Gene Essentiality and Expression Dynamics. <em>PLoS Comput Biol</em> <strong>2007</strong>, <em>3</em> (4), e59. <a href="https://doi.org/10.1371/journal.pcbi.0030059">https://doi.org/10.1371/journal.pcbi.0030059</a>.</div>
</div>
<div id="ref-1H6bETSCb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(804) </div><div class="csl-right-inline">Li, X.; Wu, M.; Kwoh, C.-K.; Ng, S.-K. Computational Approaches for Detecting Protein Complexes from Protein Interaction Networks: A Survey. <em>BMC Genomics</em> <strong>2010</strong>, <em>11 Suppl 1</em> (Suppl 1), S3. <a href="https://doi.org/10.1186/1471-2164-11-s1-s3">https://doi.org/10.1186/1471-2164-11-s1-s3</a>.</div>
</div>
<div id="ref-11gPEmZWK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(805) </div><div class="csl-right-inline">Brohée, S.; van Helden, J. Evaluation of Clustering Algorithms for Protein-Protein Interaction Networks. <em>BMC Bioinformatics</em> <strong>2006</strong>, <em>7</em>, 488. <a href="https://doi.org/10.1186/1471-2105-7-488">https://doi.org/10.1186/1471-2105-7-488</a>.</div>
</div>
<div id="ref-iqdatwR1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(806) </div><div class="csl-right-inline">Bader, G. D.; Hogue, C. W. V. An Automated Method for Finding Molecular Complexes in Large Protein Interaction Networks. <em>BMC Bioinformatics</em> <strong>2003</strong>, <em>4</em>, 2. <a href="https://doi.org/10.1186/1471-2105-4-2">https://doi.org/10.1186/1471-2105-4-2</a>.</div>
</div>
<div id="ref-WJOtOvRY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(807) </div><div class="csl-right-inline">Piovesan, D.; Giollo, M.; Ferrari, C.; Tosatto, S. C. E. Protein Function Prediction Using Guilty by Association from Interaction Networks. <em>Amino Acids</em> <strong>2015</strong>, <em>47</em> (12), 2583–2592. <a href="https://doi.org/10.1007/s00726-015-2049-3">https://doi.org/10.1007/s00726-015-2049-3</a>.</div>
</div>
<div id="ref-8oE5lYNw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(808) </div><div class="csl-right-inline">Shannon, P.; Markiel, A.; Ozier, O.; Baliga, N. S.; Wang, J. T.; Ramage, D.; Amin, N.; Schwikowski, B.; Ideker, T. Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks. <em>Genome Res</em> <strong>2003</strong>, <em>13</em> (11), 2498–2504. <a href="https://doi.org/10.1101/gr.1239303">https://doi.org/10.1101/gr.1239303</a>.</div>
</div>
<div id="ref-jrtjRoft" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(809) </div><div class="csl-right-inline">Doncheva, N. T.; Morris, J. H.; Holze, H.; Kirsch, R.; Nastou, K. C.; Cuesta-Astroz, Y.; Rattei, T.; Szklarczyk, D.; von Mering, C.; Jensen, L. J. Cytoscape stringApp 2.0: Analysis and Visualization of Heterogeneous Biological Networks. <em>J Proteome Res</em> <strong>2022</strong>, <em>22</em> (2), 637–646. <a href="https://doi.org/10.1021/acs.jproteome.2c00651">https://doi.org/10.1021/acs.jproteome.2c00651</a>.</div>
</div>
<div id="ref-h0C6mgc4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(810) </div><div class="csl-right-inline"><em>Protein Bioinformatics</em>; Wu, C. H., Arighi, C. N., Ross, K. E., Eds.; Springer New York, 2017. <a href="https://doi.org/10.1007/978-1-4939-6783-4">https://doi.org/10.1007/978-1-4939-6783-4</a>.</div>
</div>
<div id="ref-CaFLiife" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(811) </div><div class="csl-right-inline">Halder, A.; Verma, A.; Biswas, D.; Srivastava, S. Recent Advances in Mass-Spectrometry Based Proteomics Software, Tools and Databases. <em>Drug Discovery Today: Technologies</em> <strong>2021</strong>, <em>39</em>, 69–79. <a href="https://doi.org/10.1016/j.ddtec.2021.06.007">https://doi.org/10.1016/j.ddtec.2021.06.007</a>.</div>
</div>
<div id="ref-QlVqCOZZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(812) </div><div class="csl-right-inline">Deutsch, E. W.; Lam, H.; Aebersold, R. Data Analysis and Bioinformatics Tools for Tandem Mass Spectrometry in Proteomics. <em>Physiological Genomics</em> <strong>2008</strong>, <em>33</em> (1), 18–25. <a href="https://doi.org/10.1152/physiolgenomics.00298.2007">https://doi.org/10.1152/physiolgenomics.00298.2007</a>.</div>
</div>
<div id="ref-IFevKgND" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(813) </div><div class="csl-right-inline">Ashburner, M.; Ball, C. A.; Blake, J. A.; Botstein, D.; Butler, H.; Cherry, J. M.; Davis, A. P.; Dolinski, K.; Dwight, S. S.; Eppig, J. T.; Harris, M. A.; Hill, D. P.; Issel-Tarver, L.; Kasarskis, A.; Lewis, S.; Matese, J. C.; Richardson, J. E.; Ringwald, M.; Rubin, G. M.; Sherlock, G. Gene Ontology: Tool for the Unification of Biology. The Gene Ontology Consortium. <em>Nat Genet</em> <strong>2000</strong>, <em>25</em> (1), 25–29. <a href="https://doi.org/10.1038/75556">https://doi.org/10.1038/75556</a>.</div>
</div>
<div id="ref-15ZNlTPZi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(814) </div><div class="csl-right-inline">Kanehisa, M.; Goto, S. KEGG: Kyoto Encyclopedia of Genes and Genomes. <em>Nucleic Acids Res</em> <strong>2000</strong>, <em>28</em> (1), 27–30. <a href="https://doi.org/10.1093/nar/28.1.27">https://doi.org/10.1093/nar/28.1.27</a>.</div>
</div>
<div id="ref-n1NPYC1C" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(815) </div><div class="csl-right-inline">Jassal, B.; Matthews, L.; Viteri, G.; Gong, C.; Lorente, P.; Fabregat, A.; Sidiropoulos, K.; Cook, J.; Gillespie, M.; Haw, R.; Loney, F.; May, B.; Milacic, M.; Rothfels, K.; Sevilla, C.; Shamovsky, V.; Shorser, S.; Varusai, T.; Weiser, J.; Wu, G.; Stein, L.; Hermjakob, H.; D'Eustachio, P. The Reactome Pathway Knowledgebase. <em>Nucleic Acids Res</em> <strong>2020</strong>, <em>48</em> (D1), D498–D503. <a href="https://doi.org/10.1093/nar/gkz1031">https://doi.org/10.1093/nar/gkz1031</a>.</div>
</div>
<div id="ref-f3o81IaL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(816) </div><div class="csl-right-inline">Hornbeck, P. V.; Kornhauser, J. M.; Latham, V.; Murray, B.; Nandhikonda, V.; Nord, A.; Skrzypek, E.; Wheeler, T.; Zhang, B.; Gnad, F. 15 Years of PhosphoSitePlus®: Integrating Post-Translationally Modified Sites, Disease Variants and Isoforms. <em>Nucleic Acids Res</em> <strong>2019</strong>, <em>47</em> (D1), D433–D441. <a href="https://doi.org/10.1093/nar/gky1159">https://doi.org/10.1093/nar/gky1159</a>.</div>
</div>
<div id="ref-10dRpS3qN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(817) </div><div class="csl-right-inline">Selevsek, N.; Caiment, F.; Nudischer, R.; Gmuender, H.; Agarkova, I.; Atkinson, F. L.; Bachmann, I.; Baier, V.; Barel, G.; Bauer, C.; Boerno, S.; Bosc, N.; Clayton, O.; Cordes, H.; Deeb, S.; Gotta, S.; Guye, P.; Hersey, A.; Hunter, F. M. I.; Kunz, L.; Lewalle, A.; Lienhard, M.; Merken, J.; Minguet, J.; Oliveira, B.; Pluess, C.; Sarkans, U.; Schrooders, Y.; Schuchhardt, J.; Smit, I.; Thiel, C.; Timmermann, B.; Verheijen, M.; Wittenberger, T.; Wolski, W.; Zerck, A.; Heymans, S.; Kuepfer, L.; Roth, A.; Schlapbach, R.; Niederer, S.; Herwig, R.; Kleinjans, J. Network Integration and Modelling of Dynamic Drug Responses at Multi-Omics Levels. <em>Commun Biol</em> <strong>2020</strong>, <em>3</em> (1), 573. <a href="https://doi.org/10.1038/s42003-020-01302-8">https://doi.org/10.1038/s42003-020-01302-8</a>.</div>
</div>
<div id="ref-DhTe2WLI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(818) </div><div class="csl-right-inline">Gillespie, M.; Jassal, B.; Stephan, R.; Milacic, M.; Rothfels, K.; Senff-Ribeiro, A.; Griss, J.; Sevilla, C.; Matthews, L.; Gong, C.; Deng, C.; Varusai, T.; Ragueneau, E.; Haider, Y.; May, B.; Shamovsky, V.; Weiser, J.; Brunson, T.; Sanati, N.; Beckman, L.; Shao, X.; Fabregat, A.; Sidiropoulos, K.; Murillo, J.; Viteri, G.; Cook, J.; Shorser, S.; Bader, G.; Demir, E.; Sander, C.; Haw, R.; Wu, G.; Stein, L.; Hermjakob, H.; D'Eustachio, P. The Reactome Pathway Knowledgebase 2022. <em>Nucleic Acids Res</em> <strong>2022</strong>, <em>50</em> (D1), D687–D692. <a href="https://doi.org/10.1093/nar/gkab1028">https://doi.org/10.1093/nar/gkab1028</a>.</div>
</div>
<div id="ref-rlKsqE1k" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(819) </div><div class="csl-right-inline">Laukens, K.; Naulaerts, S.; Berghe, W. V. Bioinformatics Approaches for the Functional Interpretation of Protein Lists: From Ontology Term Enrichment to Network Analysis. <em>Proteomics</em> <strong>2015</strong>, <em>15</em> (5-6), 981–996. <a href="https://doi.org/10.1002/pmic.201400296">https://doi.org/10.1002/pmic.201400296</a>.</div>
</div>
<div id="ref-1FmAqWcpa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(820) </div><div class="csl-right-inline">The Gene Ontology Resource: Enriching a GOld Mine. <em>Nucleic Acids Res</em> <strong>2021</strong>, <em>49</em> (D1), D325–D334. <a href="https://doi.org/10.1093/nar/gkaa1113">https://doi.org/10.1093/nar/gkaa1113</a>.</div>
</div>
<div id="ref-7xIEHm2N" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(821) </div><div class="csl-right-inline">Wijesooriya, K.; Jadaan, S. A.; Perera, K. L.; Kaur, T.; Ziemann, M. Urgent Need for Consistent Standards in Functional Enrichment Analysis. <em>PLoS Comput Biol</em> <strong>2022</strong>, <em>18</em> (3), e1009935. <a href="https://doi.org/10.1371/journal.pcbi.1009935">https://doi.org/10.1371/journal.pcbi.1009935</a>.</div>
</div>
<div id="ref-HNlvs4Yk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(822) </div><div class="csl-right-inline">Maleki, F.; Ovens, K.; Hogan, D. J.; Kusalik, A. J. Gene Set Analysis: Challenges, Opportunities, and Future Research. <em>Front Genet</em> <strong>2020</strong>, <em>11</em>, 654. <a href="https://doi.org/10.3389/fgene.2020.00654">https://doi.org/10.3389/fgene.2020.00654</a>.</div>
</div>
<div id="ref-TWAPY3zr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(823) </div><div class="csl-right-inline">Fisher, R. A. On the Interpretation of χ 2 from Contingency Tables, and the Calculation of P. <em>Journal of the Royal Statistical Society</em> <strong>1922</strong>, <em>85</em> (1), 87. <a href="https://doi.org/10.2307/2340521">https://doi.org/10.2307/2340521</a>.</div>
</div>
<div id="ref-ArmIWAfa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(824) </div><div class="csl-right-inline">Wijesooriya, K.; Jadaan, S. A.; Perera, K. L.; Kaur, T.; Ziemann, M. Urgent Need for Consistent Standards in Functional Enrichment Analysis. <em>PLoS Comput Biol</em> <strong>2022</strong>, <em>18</em> (3), e1009935. <a href="https://doi.org/10.1371/journal.pcbi.1009935">https://doi.org/10.1371/journal.pcbi.1009935</a>.</div>
</div>
<div id="ref-rrE2B3JZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(825) </div><div class="csl-right-inline">Chen, E. Y.; Tan, C. M.; Kou, Y.; Duan, Q.; Wang, Z.; Meirelles, G. V.; Clark, N. R.; Ma'ayan, A. Enrichr: Interactive and Collaborative HTML5 Gene List Enrichment Analysis Tool. <em>BMC Bioinformatics</em> <strong>2013</strong>, <em>14</em>, 128. <a href="https://doi.org/10.1186/1471-2105-14-128">https://doi.org/10.1186/1471-2105-14-128</a>.</div>
</div>
<div id="ref-15p5LWIVP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(826) </div><div class="csl-right-inline">Subramanian, A.; Tamayo, P.; Mootha, V. K.; Mukherjee, S.; Ebert, B. L.; Gillette, M. A.; Paulovich, A.; Pomeroy, S. L.; Golub, T. R.; Lander, E. S.; Mesirov, J. P. Gene Set Enrichment Analysis: A Knowledge-Based Approach for Interpreting Genome-Wide Expression Profiles. <em>Proc. Natl. Acad. Sci. U.S.A.</em> <strong>2005</strong>, <em>102</em> (43), 15545–15550. <a href="https://doi.org/10.1073/pnas.0506580102">https://doi.org/10.1073/pnas.0506580102</a>.</div>
</div>
<div id="ref-101HEiJoS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(827) </div><div class="csl-right-inline">Sherman, B. T.; Hao, M.; Qiu, J.; Jiao, X.; Baseler, M. W.; Lane, H. C.; Imamichi, T.; Chang, W. DAVID: A Web Server for Functional Enrichment Analysis and Functional Annotation of Gene Lists (2021 Update). <em>Nucleic Acids Res</em> <strong>2022</strong>, <em>50</em> (W1), W216–W221. <a href="https://doi.org/10.1093/nar/gkac194">https://doi.org/10.1093/nar/gkac194</a>.</div>
</div>
<div id="ref-tt0Vdf4C" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(828) </div><div class="csl-right-inline">Subramanian, A.; Tamayo, P.; Mootha, V. K.; Mukherjee, S.; Ebert, B. L.; Gillette, M. A.; Paulovich, A.; Pomeroy, S. L.; Golub, T. R.; Lander, E. S.; Mesirov, J. P. Gene Set Enrichment Analysis: A Knowledge-Based Approach for Interpreting Genome-Wide Expression Profiles. <em>Proc Natl Acad Sci U S A</em> <strong>2005</strong>, <em>102</em> (43), 15545–15550. <a href="https://doi.org/10.1073/pnas.0506580102">https://doi.org/10.1073/pnas.0506580102</a>.</div>
</div>
<div id="ref-6gI0wLAR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(829) </div><div class="csl-right-inline">Mitrea, C.; Taghavi, Z.; Bokanizad, B.; Hanoudi, S.; Tagett, R.; Donato, M.; Voichiţa, C.; Drăghici, S. Methods and Approaches in the Topology-Based Analysis of Biological Pathways. <em>Front Physiol</em> <strong>2013</strong>, <em>4</em>, 278. <a href="https://doi.org/10.3389/fphys.2013.00278">https://doi.org/10.3389/fphys.2013.00278</a>.</div>
</div>
<div id="ref-1BeN7jMiD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(830) </div><div class="csl-right-inline">Masuda, N. Faculty Opinions Recommendation of Gene Co-Expression Analysis for Functional Classification and Gene-Disease Predictions. <em>Faculty Opinions – Post-Publication Peer Review of the Biomedical Literature</em>, 2021. <a href="https://doi.org/10.3410/f.727201931.793589848">https://doi.org/10.3410/f.727201931.793589848</a>.</div>
</div>
<div id="ref-15Ki75h8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(831) </div><div class="csl-right-inline">Pardo-Diaz, J.; Bozhilova, L. V.; Beguerisse-Díaz, M.; Poole, P. S.; Deane, C. M.; Reinert, G. Robust Gene Coexpression Networks Using Signed Distance Correlation, 2020. <a href="https://doi.org/10.1101/2020.06.21.163543">https://doi.org/10.1101/2020.06.21.163543</a>.</div>
</div>
<div id="ref-fXCPx62A" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(832) </div><div class="csl-right-inline">Fortunato, S. Community Detection in Graphs. <em>Physics Reports</em> <strong>2010</strong>, <em>486</em> (3-5), 75–174. <a href="https://doi.org/10.1016/j.physrep.2009.11.002">https://doi.org/10.1016/j.physrep.2009.11.002</a>.</div>
</div>
<div id="ref-eZTqkAjM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(833) </div><div class="csl-right-inline">Gunning, P. W.; Hardeman, E. C. Fundamental Differences. <em>eLife</em> <strong>2018</strong>, <em>7</em>, e34477. <a href="https://doi.org/10.7554/eLife.34477">https://doi.org/10.7554/eLife.34477</a>.</div>
</div>
<div id="ref-kBc66YMI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(834) </div><div class="csl-right-inline">Bludau, I.; Frank, M.; Dörig, C.; Cai, Y.; Heusel, M.; Rosenberger, G.; Picotti, P.; Collins, B. C.; Röst, H.; Aebersold, R. Systematic Detection of Functional Proteoform Groups from Bottom-up Proteomic Datasets, 2020. <a href="https://doi.org/10.1101/2020.12.22.423928">https://doi.org/10.1101/2020.12.22.423928</a>.</div>
</div>
<div id="ref-YKUlYryb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(835) </div><div class="csl-right-inline">Dermit, M.; Peters-Clarke, T. M.; Shishkova, E.; Meyer, J. G. Peptide Correlation Analysis (PeCorA) Reveals Differential Proteoform Regulation. <em>J Proteome Res</em> <strong>2020</strong>, <em>20</em> (4), 1972–1980. <a href="https://doi.org/10.1021/acs.jproteome.0c00602">https://doi.org/10.1021/acs.jproteome.0c00602</a>.</div>
</div>
<div id="ref-vegaUWlG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(836) </div><div class="csl-right-inline">Sun, P. D.; Foster, C. E.; Boyington, J. C. Overview of Protein Structural and Functional Folds. <em>Curr Protoc Protein Sci</em> <strong>2004</strong>, <em>Chapter 17</em> (1), Unit 17.1. <a href="https://doi.org/10.1002/0471140864.ps1701s35">https://doi.org/10.1002/0471140864.ps1701s35</a>.</div>
</div>
<div id="ref-PnnbHO9a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(837) </div><div class="csl-right-inline">Xu, J. Distance-Based Protein Folding Powered by Deep Learning. <em>Proc. Natl. Acad. Sci. U.S.A.</em> <strong>2019</strong>, <em>116</em> (34), 16856–16865. <a href="https://doi.org/10.1073/pnas.1821309116">https://doi.org/10.1073/pnas.1821309116</a>.</div>
</div>
<div id="ref-yZfcMIwh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(838) </div><div class="csl-right-inline">Jumper, J.; Evans, R.; Pritzel, A.; Green, T.; Figurnov, M.; Ronneberger, O.; Tunyasuvunakool, K.; Bates, R.; Žídek, A.; Potapenko, A.; Bridgland, A.; Meyer, C.; Kohl, S. A. A.; Ballard, A. J.; Cowie, A.; Romera-Paredes, B.; Nikolov, S.; Jain, R.; Adler, J.; Back, T.; Petersen, S.; Reiman, D.; Clancy, E.; Zielinski, M.; Steinegger, M.; Pacholska, M.; Berghammer, T.; Bodenstein, S.; Silver, D.; Vinyals, O.; Senior, A. W.; Kavukcuoglu, K.; Kohli, P.; Hassabis, D. Highly Accurate Protein Structure Prediction with AlphaFold. <em>Nature</em> <strong>2021</strong>, <em>596</em> (7873), 583–589. <a href="https://doi.org/10.1038/s41586-021-03819-2">https://doi.org/10.1038/s41586-021-03819-2</a>.</div>
</div>
<div id="ref-1AwSDaTyL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(839) </div><div class="csl-right-inline">Baek, M.; DiMaio, F.; Anishchenko, I.; Dauparas, J.; Ovchinnikov, S.; Lee, G. R.; Wang, J.; Cong, Q.; Kinch, L. N.; Schaeffer, R. D.; Millán, C.; Park, H.; Adams, C.; Glassman, C. R.; DeGiovanni, A.; Pereira, J. H.; Rodrigues, A. V.; van Dijk, A. A.; Ebrecht, A. C.; Opperman, D. J.; Sagmeister, T.; Buhlheller, C.; Pavkov-Keller, T.; Rathinaswamy, M. K.; Dalwadi, U.; Yip, C. K.; Burke, J. E.; Garcia, K. C.; Grishin, N. V.; Adams, P. D.; Read, R. J.; Baker, D. Accurate Prediction of Protein Structures and Interactions Using a Three-Track Neural Network. <em>Science</em> <strong>2021</strong>, <em>373</em> (6557), 871–876. <a href="https://doi.org/10.1126/science.abj8754">https://doi.org/10.1126/science.abj8754</a>.</div>
</div>
<div id="ref-tIYr5DF7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(840) </div><div class="csl-right-inline">Oates, M. E.; Romero, P.; Ishida, T.; Ghalwash, M.; Mizianty, M. J.; Xue, B.; Dosztányi, Z.; Uversky, V. N.; Obradovic, Z.; Kurgan, L.; Dunker, A. K.; Gough, J. D²P²: Database of Disordered Protein Predictions. <em>Nucleic Acids Res</em> <strong>2012</strong>, <em>41</em> (Database issue), D508–16. <a href="https://doi.org/10.1093/nar/gks1226">https://doi.org/10.1093/nar/gks1226</a>.</div>
</div>
<div id="ref-CRgkwTQA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(841) </div><div class="csl-right-inline">Meng, X.-Y.; Zhang, H.-X.; Mezei, M.; Cui, M. Molecular Docking: A Powerful Approach for Structure-Based Drug Discovery. <em>Curr Comput Aided Drug Des</em> <strong>2011</strong>, <em>7</em> (2), 146–157. <a href="https://doi.org/10.2174/157340911795677602">https://doi.org/10.2174/157340911795677602</a>.</div>
</div>
<div id="ref-iOQeUEPU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(842) </div><div class="csl-right-inline">Gelpi, J.; Hospital, A.; Goñi, R.; Orozco, M. Molecular Dynamics Simulations: Advances and Applications. <em>AABC</em> <strong>2015</strong>, 37. <a href="https://doi.org/10.2147/aabc.s70333">https://doi.org/10.2147/aabc.s70333</a>.</div>
</div>
<div id="ref-jcOtxABy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(843) </div><div class="csl-right-inline">Altschul, S. F.; Gish, W.; Miller, W.; Myers, E. W.; Lipman, D. J. Basic Local Alignment Search Tool. <em>J Mol Biol</em> <strong>1990</strong>, <em>215</em> (3), 403–410. <a href="https://doi.org/10.1016/s0022-2836(05)80360-2">https://doi.org/10.1016/s0022-2836(05)80360-2</a>.</div>
</div>
<div id="ref-daqSjHIW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(844) </div><div class="csl-right-inline">Unsal, S.; Atas, H.; Albayrak, M.; Turhan, K.; Acar, A. C.; Doğan, T. Learning Functional Properties of Proteins with Language Models. <em>Nat Mach Intell</em> <strong>2022</strong>, <em>4</em> (3), 227–245. <a href="https://doi.org/10.1038/s42256-022-00457-9">https://doi.org/10.1038/s42256-022-00457-9</a>.</div>
</div>
<div id="ref-nf5EVAMJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(845) </div><div class="csl-right-inline">Bepler, T.; Berger, B. Learning the Protein Language: Evolution, Structure, and Function. <em>Cell Systems</em> <strong>2021</strong>, <em>12</em> (6), 654–669.e3. <a href="https://doi.org/10.1016/j.cels.2021.05.017">https://doi.org/10.1016/j.cels.2021.05.017</a>.</div>
</div>
<div id="ref-wLlg6SzE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(846) </div><div class="csl-right-inline">Mateus, A.; Kurzawa, N.; Becher, I.; Sridharan, S.; Helm, D.; Stein, F.; Typas, A.; Savitski, M. M. Thermal Proteome Profiling for Interrogating Protein Interactions. <em>Mol Syst Biol</em> <strong>2020</strong>, <em>16</em> (3), e9232. <a href="https://doi.org/10.15252/msb.20199232">https://doi.org/10.15252/msb.20199232</a>.</div>
</div>
<div id="ref-YBdYtUv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(847) </div><div class="csl-right-inline">Siebenmorgen, T.; Zacharias, M. Computational Prediction of Protein–Protein Binding Affinities. <em>WIREs Comput Mol Sci</em> <strong>2019</strong>, <em>10</em> (3). <a href="https://doi.org/10.1002/wcms.1448">https://doi.org/10.1002/wcms.1448</a>.</div>
</div>
<div id="ref-bUq8ZzIL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(848) </div><div class="csl-right-inline">Luo, H.; Lin, Y.; Gao, F.; Zhang, C.-T.; Zhang, R. DEG 10, an Update of the Database of Essential Genes That Includes Both Protein-Coding Genes and Noncoding Genomic Elements: Table 1. <em>Nucl. Acids Res.</em> <strong>2013</strong>, <em>42</em> (D1), D574–D580. <a href="https://doi.org/10.1093/nar/gkt1131">https://doi.org/10.1093/nar/gkt1131</a>.</div>
</div>
<div id="ref-171wjW4dP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(849) </div><div class="csl-right-inline">Funk, L.; Su, K.-C.; Ly, J.; Feldman, D.; Singh, A.; Moodie, B.; Blainey, P. C.; Cheeseman, I. M. The Phenotypic Landscape of Essential Human Genes. <em>Cell</em> <strong>2022</strong>, <em>185</em> (24), 4634–4653.e22. <a href="https://doi.org/10.1016/j.cell.2022.10.017">https://doi.org/10.1016/j.cell.2022.10.017</a>.</div>
</div>
<div id="ref-pGJ2GFbE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(850) </div><div class="csl-right-inline">Stepanenko, A. A.; Heng, H. H. Transient and Stable Vector Transfection: Pitfalls, Off-Target Effects, Artifacts. <em>Mutation Research/Reviews in Mutation Research</em> <strong>2017</strong>, <em>773</em>, 91–103. <a href="https://doi.org/10.1016/j.mrrev.2017.05.002">https://doi.org/10.1016/j.mrrev.2017.05.002</a>.</div>
</div>
<div id="ref-DdiSq79q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(851) </div><div class="csl-right-inline">Boettcher, M.; McManus, Michael T. Choosing the Right Tool for the Job: RNAi, TALEN, or CRISPR. <em>Molecular Cell</em> <strong>2015</strong>, <em>58</em> (4), 575–585. <a href="https://doi.org/10.1016/j.molcel.2015.04.028">https://doi.org/10.1016/j.molcel.2015.04.028</a>.</div>
</div>
<div id="ref-15T32hvm6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(852) </div><div class="csl-right-inline">Jinek, M.; Chylinski, K.; Fonfara, I.; Hauer, M.; Doudna, J. A.; Charpentier, E. A Programmable Dual-RNA–Guided DNA Endonuclease in Adaptive Bacterial Immunity. <em>Science</em> <strong>2012</strong>, <em>337</em> (6096), 816–821. <a href="https://doi.org/10.1126/science.1225829">https://doi.org/10.1126/science.1225829</a>.</div>
</div>
<div id="ref-hRIRSidU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(853) </div><div class="csl-right-inline">Barman, A.; Deb, B.; Chakraborty, S. A Glance at Genome Editing with CRISPR–Cas9 Technology. <em>Curr Genet</em> <strong>2019</strong>, <em>66</em> (3), 447–462. <a href="https://doi.org/10.1007/s00294-019-01040-3">https://doi.org/10.1007/s00294-019-01040-3</a>.</div>
</div>
<div id="ref-1uwXNnEt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(854) </div><div class="csl-right-inline">Rezazade Bazaz, M.; Dehghani, H. From DNA Break Repair Pathways to CRISPR/Cas-Mediated Gene Knock-in Methods. <em>Life Sciences</em> <strong>2022</strong>, <em>295</em>, 120409. <a href="https://doi.org/10.1016/j.lfs.2022.120409">https://doi.org/10.1016/j.lfs.2022.120409</a>.</div>
</div>
<div id="ref-EPXqJmJj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(855) </div><div class="csl-right-inline">Li, G.; Li, X.; Zhuang, S.; Wang, L.; Zhu, Y.; Chen, Y.; Sun, W.; Wu, Z.; Zhou, Z.; Chen, J.; Huang, X.; Wang, J.; Li, D.; Li, W.; Wang, H.; Wei, W. Gene Editing and Its Applications in Biomedicine. <em>Sci. China Life Sci.</em> <strong>2022</strong>, <em>65</em> (4), 660–700. <a href="https://doi.org/10.1007/s11427-021-2057-0">https://doi.org/10.1007/s11427-021-2057-0</a>.</div>
</div>
<div id="ref-rUUeN4Sh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(856) </div><div class="csl-right-inline">Saber Sichani, A.; Ranjbar, M.; Baneshi, M.; Torabi Zadeh, F.; Fallahi, J. A Review on Advanced CRISPR-Based Genome-Editing Tools: Base Editing and Prime Editing. <em>Mol Biotechnol</em> <strong>2022</strong>, <em>65</em> (6), 849–860. <a href="https://doi.org/10.1007/s12033-022-00639-1">https://doi.org/10.1007/s12033-022-00639-1</a>.</div>
</div>
<div id="ref-VFV60BPa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(857) </div><div class="csl-right-inline">Boti, M. A.; Athanasopoulou, K.; Adamopoulos, P. G.; Sideris, D. C.; Scorilas, A. Recent Advances in Genome-Engineering Strategies. <em>Genes</em> <strong>2023</strong>, <em>14</em> (1), 129. <a href="https://doi.org/10.3390/genes14010129">https://doi.org/10.3390/genes14010129</a>.</div>
</div>
<div id="ref-sAa18GbX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(858) </div><div class="csl-right-inline">Naeem, M.; Majeed, S.; Hoque, M. Z.; Ahmad, I. Latest Developed Strategies to Minimize the Off-Target Effects in CRISPR-Cas-Mediated Genome Editing. <em>Cells</em> <strong>2020</strong>, <em>9</em> (7), 1608. <a href="https://doi.org/10.3390/cells9071608">https://doi.org/10.3390/cells9071608</a>.</div>
</div>
<div id="ref-n2p9cQ7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(859) </div><div class="csl-right-inline">Prozzillo, Y.; Fattorini, G.; Santopietro, M. V.; Suglia, L.; Ruggiero, A.; Ferreri, D.; Messina, G. Targeted Protein Degradation Tools: Overview and Future Perspectives. <em>Biology</em> <strong>2020</strong>, <em>9</em> (12), 421. <a href="https://doi.org/10.3390/biology9120421">https://doi.org/10.3390/biology9120421</a>.</div>
</div>
<div id="ref-1G6PkBlEo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(860) </div><div class="csl-right-inline">T. Das, A.; Tenenbaum, L.; Berkhout, B. Tet-On Systems For Doxycycline-Inducible Gene Expression. <em>CGT</em> <strong>2016</strong>, <em>16</em> (3), 156–167. <a href="https://doi.org/10.2174/1566523216666160524144041">https://doi.org/10.2174/1566523216666160524144041</a>.</div>
</div>
<div id="ref-6NVK2tsy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(861) </div><div class="csl-right-inline">Kallunki; Barisic; Jäättelä; Liu. How to Choose the Right Inducible Gene Expression System for Mammalian Studies? <em>Cells</em> <strong>2019</strong>, <em>8</em> (8), 796. <a href="https://doi.org/10.3390/cells8080796">https://doi.org/10.3390/cells8080796</a>.</div>
</div>
<div id="ref-idh43eOQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(862) </div><div class="csl-right-inline">Spitzer, J.; Landthaler, M.; Tuschl, T. Rapid Creation of Stable Mammalian Cell Lines for Regulated Expression of Proteins Using the Gateway® Recombination Cloning Technology and Flp-In T-REx® Lines. In <em>Methods in Enzymology</em>; Elsevier, 2013; pp 99–124. <a href="https://doi.org/10.1016/b978-0-12-418687-3.00008-2">https://doi.org/10.1016/b978-0-12-418687-3.00008-2</a>.</div>
</div>
<div id="ref-S1IrclG6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(863) </div><div class="csl-right-inline">Ward, R. J.; Alvarez-Curto, E.; Milligan, G. Using the Flp-In™ T-Rex™ System to Regulate GPCR Expression. In <em>Methods in Molecular Biology</em>; Humana Press, 2011; pp 21–37. <a href="https://doi.org/10.1007/978-1-61779-126-0_2">https://doi.org/10.1007/978-1-61779-126-0_2</a>.</div>
</div>
<div id="ref-BwuqP6B4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(864) </div><div class="csl-right-inline">Handler, D. C.; Pascovici, D.; Mirzaei, M.; Gupta, V.; Salekdeh, G. H.; Haynes, P. A. The Art of Validating Quantitative Proteomics Data. <em>Proteomics</em> <strong>2018</strong>, <em>18</em> (23). <a href="https://doi.org/10.1002/pmic.201800222">https://doi.org/10.1002/pmic.201800222</a>.</div>
</div>
<div id="ref-gFNDbBK5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(865) </div><div class="csl-right-inline">Fritzsche, S.; Springer, S. Pulse‐Chase Analysis for Studying Protein Synthesis and Maturation. <em>CP Protein Science</em> <strong>2014</strong>, <em>78</em> (1). <a href="https://doi.org/10.1002/0471140864.ps3003s78">https://doi.org/10.1002/0471140864.ps3003s78</a>.</div>
</div>
<div id="ref-TJFHVsUt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(866) </div><div class="csl-right-inline">Sin, C.; Chiarugi, D.; Valleriani, A. Degradation Parameters from Pulse-Chase Experiments. <em>PLoS ONE</em> <strong>2016</strong>, <em>11</em> (5), e0155028. <a href="https://doi.org/10.1371/journal.pone.0155028">https://doi.org/10.1371/journal.pone.0155028</a>.</div>
</div>
<div id="ref-T6IYjbu4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(867) </div><div class="csl-right-inline">Vallejo, D. D.; Rojas Ramírez, C.; Parson, K. F.; Han, Y.; Gadkari, V. V.; Ruotolo, B. T. Mass Spectrometry Methods for Measuring Protein Stability. <em>Chem. Rev.</em> <strong>2022</strong>, <em>122</em> (8), 7690–7719. <a href="https://doi.org/10.1021/acs.chemrev.1c00857">https://doi.org/10.1021/acs.chemrev.1c00857</a>.</div>
</div>
<div id="ref-fYmIhIvC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(868) </div><div class="csl-right-inline">New Developments in Quantitative Real-Time Polymerase Chain Reaction Technology. <em>Current Issues in Molecular Biology</em> <strong>2014</strong>. <a href="https://doi.org/10.21775/cimb.016.001">https://doi.org/10.21775/cimb.016.001</a>.</div>
</div>
<div id="ref-9ByCq3fI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(869) </div><div class="csl-right-inline">Wang, Z.; Gerstein, M.; Snyder, M. RNA-Seq: A Revolutionary Tool for Transcriptomics. <em>Nat Rev Genet</em> <strong>2009</strong>, <em>10</em> (1), 57–63. <a href="https://doi.org/10.1038/nrg2484">https://doi.org/10.1038/nrg2484</a>.</div>
</div>
<div id="ref-1Hf5Cuj4t" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(870) </div><div class="csl-right-inline">Pedelacq, J.-D.; Cabantous, S. Development and Applications of Superfolder and Split Fluorescent Protein Detection Systems in Biology. <em>IJMS</em> <strong>2019</strong>, <em>20</em> (14), 3479. <a href="https://doi.org/10.3390/ijms20143479">https://doi.org/10.3390/ijms20143479</a>.</div>
</div>
<div id="ref-UJonPw7t" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(871) </div><div class="csl-right-inline">Lehming, N. Analysis of Protein-Protein Proximities Using the Split-Ubiquitin System. <em>Briefings in Functional Genomics and Proteomics</em> <strong>2002</strong>, <em>1</em> (3), 230–238. <a href="https://doi.org/10.1093/bfgp/1.3.230">https://doi.org/10.1093/bfgp/1.3.230</a>.</div>
</div>
<div id="ref-btMVdgTk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(872) </div><div class="csl-right-inline">Azad, T.; Tashakor, A.; Hosseinkhani, S. Split-Luciferase Complementary Assay: Applications, Recent Developments, and Future Perspectives. <em>Anal Bioanal Chem</em> <strong>2014</strong>, <em>406</em> (23), 5541–5560. <a href="https://doi.org/10.1007/s00216-014-7980-8">https://doi.org/10.1007/s00216-014-7980-8</a>.</div>
</div>
<div id="ref-zrB7Yy5d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(873) </div><div class="csl-right-inline">Cesaratto, F.; Burrone, O. R.; Petris, G. Tobacco Etch Virus Protease: A Shortcut Across Biotechnologies. <em>Journal of Biotechnology</em> <strong>2016</strong>, <em>231</em>, 239–249. <a href="https://doi.org/10.1016/j.jbiotec.2016.06.012">https://doi.org/10.1016/j.jbiotec.2016.06.012</a>.</div>
</div>
<div id="ref-hPao2o3e" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(874) </div><div class="csl-right-inline">Hinton, S. R.; Corpuz, E. L. S.; McFarlane Holman, K. L.; Meyer, S. C. A Split β-Lactamase Sensor for the Detection of DNA Modification by Cisplatin and Ruthenium-Based Chemotherapeutic Drugs. <em>Journal of Inorganic Biochemistry</em> <strong>2022</strong>, <em>236</em>, 111986. <a href="https://doi.org/10.1016/j.jinorgbio.2022.111986">https://doi.org/10.1016/j.jinorgbio.2022.111986</a>.</div>
</div>
<div id="ref-LY7mqhu6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(875) </div><div class="csl-right-inline">Wang, T.; Yang, N.; Liang, C.; Xu, H.; An, Y.; Xiao, S.; Zheng, M.; Liu, L.; Wang, G.; Nie, L. Detecting Protein-Protein Interaction Based on Protein Fragment Complementation Assay. <em>CPPS</em> <strong>2020</strong>, <em>21</em> (6), 598–610. <a href="https://doi.org/10.2174/1389203721666200213102829">https://doi.org/10.2174/1389203721666200213102829</a>.</div>
</div>
<div id="ref-18Csai5qq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(876) </div><div class="csl-right-inline">Pichlerova, K.; Hanes, J. Technologies for the Identification and Validation of Protein-Protein Interactions. <em>gpb</em> <strong>2021</strong>, <em>40</em> (06), 495–522. <a href="https://doi.org/10.4149/gpb_2021035">https://doi.org/10.4149/gpb_2021035</a>.</div>
</div>
<div id="ref-19CwS2HX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(877) </div><div class="csl-right-inline">Fields, S.; Song, O. A Novel Genetic System to Detect Protein–Protein Interactions. <em>Nature</em> <strong>1989</strong>, <em>340</em> (6230), 245–246. <a href="https://doi.org/10.1038/340245a0">https://doi.org/10.1038/340245a0</a>.</div>
</div>
<div id="ref-2PC5fx0I" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(878) </div><div class="csl-right-inline">Brückner, A.; Polge, C.; Lentze, N.; Auerbach, D.; Schlattner, U. Yeast Two-Hybrid, a Powerful Tool for Systems Biology. <em>IJMS</em> <strong>2009</strong>, <em>10</em> (6), 2763–2788. <a href="https://doi.org/10.3390/ijms10062763">https://doi.org/10.3390/ijms10062763</a>.</div>
</div>
<div id="ref-1DnPkHRsA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(879) </div><div class="csl-right-inline">Patrício, D.; Fardilha, M. The Mammalian Two-Hybrid System as a Powerful Tool for High-Throughput Drug Screening. <em>Drug Discovery Today</em> <strong>2020</strong>, <em>25</em> (4), 764–771. <a href="https://doi.org/10.1016/j.drudis.2020.01.022">https://doi.org/10.1016/j.drudis.2020.01.022</a>.</div>
</div>
<div id="ref-1G5hcMdry" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(880) </div><div class="csl-right-inline">Dong, Y.; Yang, J.; Ye, W.; Wang, Y.; Ye, C.; Weng, D.; Gao, H.; Zhang, F.; Xu, Z.; Lei, Y. Isolation of Endogenously Assembled RNA-Protein Complexes Using Affinity Purification Based on Streptavidin Aptamer S1. <em>IJMS</em> <strong>2015</strong>, <em>16</em> (9), 22456–22472. <a href="https://doi.org/10.3390/ijms160922456">https://doi.org/10.3390/ijms160922456</a>.</div>
</div>
<div id="ref-636RraOs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(881) </div><div class="csl-right-inline">Algar, W. R.; Hildebrandt, N.; Vogel, S. S.; Medintz, I. L. FRET as a Biomolecular Research Tool — Understanding Its Potential While Avoiding Pitfalls. <em>Nat Methods</em> <strong>2019</strong>, <em>16</em> (9), 815–829. <a href="https://doi.org/10.1038/s41592-019-0530-8">https://doi.org/10.1038/s41592-019-0530-8</a>.</div>
</div>
<div id="ref-zTlZpc1k" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(882) </div><div class="csl-right-inline">Shekhawat, S. S.; Ghosh, I. Split-Protein Systems: Beyond Binary Protein–Protein Interactions. <em>Current Opinion in Chemical Biology</em> <strong>2011</strong>, <em>15</em> (6), 789–797. <a href="https://doi.org/10.1016/j.cbpa.2011.10.014">https://doi.org/10.1016/j.cbpa.2011.10.014</a>.</div>
</div>
<div id="ref-13oBuESdH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(883) </div><div class="csl-right-inline">Wang, P.; Yang, Y.; Hong, T.; Zhu, G. Proximity Ligation Assay: An Ultrasensitive Method for Protein Quantification and Its Applications in Pathogen Detection. <em>Appl Microbiol Biotechnol</em> <strong>2021</strong>, <em>105</em> (3), 923–935. <a href="https://doi.org/10.1007/s00253-020-11049-1">https://doi.org/10.1007/s00253-020-11049-1</a>.</div>
</div>
<div id="ref-feT5EHvH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(884) </div><div class="csl-right-inline">Shrestha, D.; Jenei, A.; Nagy, P.; Vereb, G.; Szöllősi, J. Understanding FRET as a Research Tool for Cellular Studies. <em>IJMS</em> <strong>2015</strong>, <em>16</em> (12), 6718–6756. <a href="https://doi.org/10.3390/ijms16046718">https://doi.org/10.3390/ijms16046718</a>.</div>
</div>
<div id="ref-o1iijZBV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(885) </div><div class="csl-right-inline">Pfleger, K. D. G.; Eidne, K. A. Illuminating Insights into Protein-Protein Interactions Using Bioluminescence Resonance Energy Transfer (BRET). <em>Nat Methods</em> <strong>2006</strong>, <em>3</em> (3), 165–174. <a href="https://doi.org/10.1038/nmeth841">https://doi.org/10.1038/nmeth841</a>.</div>
</div>
<div id="ref-168qPRxga" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(886) </div><div class="csl-right-inline">Wu, Y.; Jiang, T. Developments in FRET- and BRET-Based Biosensors. <em>Micromachines</em> <strong>2022</strong>, <em>13</em> (10), 1789. <a href="https://doi.org/10.3390/mi13101789">https://doi.org/10.3390/mi13101789</a>.</div>
</div>
<div id="ref-6egqfZ9W" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(887) </div><div class="csl-right-inline">Martinez-Moro, M.; Di Silvio, D.; Moya, S. E. Fluorescence Correlation Spectroscopy as a Tool for the Study of the Intracellular Dynamics and Biological Fate of Protein Corona. <em>Biophysical Chemistry</em> <strong>2019</strong>, <em>253</em>, 106218. <a href="https://doi.org/10.1016/j.bpc.2019.106218">https://doi.org/10.1016/j.bpc.2019.106218</a>.</div>
</div>
<div id="ref-ZhoKgqsC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(888) </div><div class="csl-right-inline">Zafra, F.; Piniella, D. Proximity Labeling Methods for Proteomic Analysis of Membrane Proteins. <em>Journal of Proteomics</em> <strong>2022</strong>, <em>264</em>, 104620. <a href="https://doi.org/10.1016/j.jprot.2022.104620">https://doi.org/10.1016/j.jprot.2022.104620</a>.</div>
</div>
<div id="ref-RXiPqFRh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(889) </div><div class="csl-right-inline">Che, Y.; Khavari, P. A. Research Techniques Made Simple: Emerging Methods to Elucidate Protein Interactions Through Spatial Proximity. <em>Journal of Investigative Dermatology</em> <strong>2017</strong>, <em>137</em> (12), e197–e203. <a href="https://doi.org/10.1016/j.jid.2017.09.028">https://doi.org/10.1016/j.jid.2017.09.028</a>.</div>
</div>
<div id="ref-1F8ZvcEeH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(890) </div><div class="csl-right-inline">Koos, B.; Andersson, L.; Clausson, C.-M.; Grannas, K.; Klaesson, A.; Cane, G.; Söderberg, O. Analysis of Protein Interactions in Situ by Proximity Ligation Assays. In <em>Current Topics in Microbiology and Immunology</em>; Springer Berlin Heidelberg, 2013; pp 111–126. <a href="https://doi.org/10.1007/82_2013_334">https://doi.org/10.1007/82_2013_334</a>.</div>
</div>
<div id="ref-3LymzGVH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(891) </div><div class="csl-right-inline">Sarnowski, C. P.; Bikaki, M.; Leitner, A. Cross-Linking and Mass Spectrometry as a Tool for Studying the Structural Biology of Ribonucleoproteins. <em>Structure</em> <strong>2022</strong>, <em>30</em> (4), 441–461. <a href="https://doi.org/10.1016/j.str.2022.03.003">https://doi.org/10.1016/j.str.2022.03.003</a>.</div>
</div>
<div id="ref-yxw6FRju" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(892) </div><div class="csl-right-inline">Steinmetz, B.; Smok, I.; Bikaki, M.; Leitner, A. Protein–RNA Interactions: From Mass Spectrometry to Drug Discovery. <em>Essays in Biochemistry</em> <strong>2023</strong>, <em>67</em> (2), 175–186. <a href="https://doi.org/10.1042/ebc20220177">https://doi.org/10.1042/ebc20220177</a>.</div>
</div>
<div id="ref-4T3MaN5d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(893) </div><div class="csl-right-inline">Belsom, A.; Rappsilber, J. Anatomy of a Crosslinker. <em>Current Opinion in Chemical Biology</em> <strong>2021</strong>, <em>60</em>, 39–46. <a href="https://doi.org/10.1016/j.cbpa.2020.07.008">https://doi.org/10.1016/j.cbpa.2020.07.008</a>.</div>
</div>
<div id="ref-H3H5bwtx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(894) </div><div class="csl-right-inline">Drescher, D. G.; Selvakumar, D.; Drescher, M. J. Analysis of Protein Interactions by Surface Plasmon Resonance. In <em>Protein-Protein Interactions in Human Disease, Part A</em>; Elsevier, 2018; pp 1–30. <a href="https://doi.org/10.1016/bs.apcsb.2017.07.003">https://doi.org/10.1016/bs.apcsb.2017.07.003</a>.</div>
</div>
<div id="ref-YuJbg3zO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">(895) </div><div class="csl-right-inline">Himmelstein, D. S.; Rubinetti, V.; Slochower, D. R.; Hu, D.; Malladi, V. S.; Greene, C. S.; Gitter, A. Open Collaborative Writing with Manubot. <em>PLoS Comput Biol</em> <strong>2019</strong>, <em>15</em> (6), e1007128. <a href="https://doi.org/10.1371/journal.pcbi.1007128">https://doi.org/10.1371/journal.pcbi.1007128</a>.</div>
</div>
</div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
